#!/bin/sh

20140419

cp -p /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.Samtools.ptb.MergeVCFsAcrossChromosomes.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.Samtools.ptb.MergeVCFsAcrossChromosomes.vs2.sh

bash /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.Samtools.ptb.MergeVCFsAcrossChromosomes.vs2.sh

gzip /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.vcf

zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.vcf.gz | fill-aa -a /home/michaelt/Data/1000G/AncestralAlignments/human_ancestor_GRCh37_e59/human_ancestor_ | gzip -c > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.vcf.gz 

#zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.vcf.gz | perl -lane 'if ($F[0] =~ m/^#/) { print join("\t", @F); } else { if ($F[4] =~ m/\,/) { my @vals3 = split(/,/, $F[4]); if ((scalar(split(//, $vals3[0])) == 1) && (scalar(split(//, $vals3[1])) == 1)) { print join("\t", @F); } } else { my @vals1 = split(//, $F[3]); my @vals2 = split(//, $F[4]); if (($#vals1 == 0) && ($#vals2 == 0)) { print join("\t", @F); } } } ' \
#|  perl -lane 'if ($F[0] =~ m/^#/) { print join("\t", @F); } else { my @alleles2 = split(/,/, $F[4]); if ($#alleles2 == 0) { print join("\t", @F); } }' | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.vcf.gz

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.vcf.gz --remove-indels --recode --recode-INFO-all --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP

mv AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.forRealz.SNP.recode.vcf AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.recode.vcf

gzip /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.recode.vcf

zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.recode.vcf.gz | perl -lane 'if ($F[0] =~ m/^#/) { print join("\t", @F); } else { my @alleles2 = split(/,/, $F[4]); if ($#alleles2 == 0) { print join("\t", @F); } }' | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.vcf.gz

python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.DropOff.VCFVariants.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.vcf.gz --window 1000 | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.vcf.gz

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.vcf.gz --geno .95 --recode --recode-INFO-all --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95

gzip -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.recode.vcf

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz --hwe .0001 --recode --recode-INFO-all --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4

mv /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.recode.vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf

/home/shared/software/java/jre1.7.0_09/bin/java -Xmx5g -jar /home/shared/software/GATK/2.5.2/GenomeAnalysisTK.jar -T VariantAnnotator -R /home/michaelt/Data/HumanGenome/GRCh37/Version4/human_g1k_v37.fasta -I /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.bam  -o /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.HRun.vcf -A HomopolymerRun --variant /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf -nt 6 

mv /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.HRun.vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf

mv /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.HRun.vcf.idx /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.idx

###zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | perl -lane 'my @vals1 = split(/;/, $F[7]); foreach my $SNPinfo (@vals1) { if ($SNPinfo =~ m/HRun/) { print $SNPinfo; } };' | sort | uniq -c 

###[  michaelt@node1  /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | perl -lane 'my @vals1 = split(/;/, $F[7]); foreach my $SNPinfo (@vals1) { if ($SNPinfo =~ m/HRun/) { print $SNPinfo; } };' | sort | uniq -c

gzip /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --plink --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4

plink --file /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4 --make-bed --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4 --genome --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4


cp -p /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/VLookUp.Python.DataHapMap3Overlap.vs1.py /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/VLookUp.Python.DataHapMap3Overlap.vs1.py

python /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/VLookUp.Python.DataHapMap3Overlap.vs1.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.map --file2 /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.liftOver.hg18tohg19.map | sort -u -k 1,1 -k 4,4 | awk '{ print $2 }' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.rsIDs
#python /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/VLookUp.Python.DataHapMap3Overlap.vs1.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.map --file2 /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.liftOver.hg18tohg19.map | grep -v AFF | awk '{ print $2 }' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.rsIDs
#Edit /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/VLookUp.Python.DataHapMap3Overlap.vs1.py
python /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/VLookUp.Python.DataHapMap3Overlap.vs1.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.map --file2 /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.liftOver.hg18tohg19.map | sort -u -k 1,1 -k 4,4 | awk '{ print $2 }' > /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.liftOver.hg18tohg19.Samtools.Vs2.rsIDs
#python /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/VLookUp.Python.DataHapMap3Overlap.vs1.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.map --file2 /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.liftOver.hg18tohg19.map | grep -v AFF | awk '{ print $2 }' > /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.liftOver.hg18tohg19.Samtools.Vs2.rsIDs

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4 --extract /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.rsIDs --recode --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3

plink --file /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3 --make-bed --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3

plink --bfile /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly --extract /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.liftOver.hg18tohg19.Samtools.Vs2.rsIDs --keep /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.CEUCHBYRI.Unrelated.FIDIIDs --recode --out /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Samtools.Vs2

plink --file /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Samtools.Vs2 --make-bed --out /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Samtools.Vs2

paste /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.bim /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Samtools.Vs2.bim | perl -lane 'if ((($F[4] eq $F[10]) || ($F[4] eq $F[11])) && (($F[5] eq $F[10]) || ($F[5] eq $F[11]))) { print join("\t", @F); } ' | awk '{ print $2 }' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.FinalListwDrops.rsIDs

paste /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.bim /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Samtools.Vs2.bim | perl -lane 'if ((($F[4] eq $F[10]) || ($F[4] eq $F[11])) && (($F[5] eq $F[10]) || ($F[5] eq $F[11]))) { print join("\t", @F); } ' | awk '{ print $8 }' > /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.liftOver.hg18tohg19.Samtools.Vs2.FinalListwDrops.rsIDs

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3 --extract /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.FinalListwDrops.rsIDs --recode --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.vs2

plink --bfile /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Samtools.Vs2 --extract /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.liftOver.hg18tohg19.Samtools.Vs2.FinalListwDrops.rsIDs --recode --out /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Samtools.Vs2.vs2

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.vs2.ped /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Samtools.Vs2.vs2.ped > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.ped

plink --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.ped --map /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.vs2.map --make-bed --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.phenoColEdit.ped

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.phenoColEdit.pedind

ln -s /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.bim /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.pedsnp

#Directory /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools
mkdir EigenstratFiles

cp -p /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.wHapMap3Data.CEUCHBYRI.parfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.parfile

/home/michaelt/Software/EIG5.0.1/bin/smartpca -p /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.parfile

python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca | cat - <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca | grep -E 'POOL43_10RL|POOL101_11RL|POOL18S_07RL|POOL2S_03RL' | perl -lane '$F[$#F] = "NoPhenotype"; print join("\t", @F); ') > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt1

#Added in the following four individuals with the tag 'NoPhenotype' at the end manually in the pt1 file
#POOL43_10RL POOL101_11RL POOL18S_07RL POOL2S_03RL

python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.HapMap3vs.py --file1 /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.txt --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt2

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt2 > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race

sed -i 's/American_Indian_or_Alaskan_Native/AIndian_or_Alaskan_Native/g' /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.1vs2.xtxt

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.3vs4.xtxt

scp /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.*pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/Samtools/EigenstratFiles/.







scp /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.1vs2.pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/.

#python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt1
python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca | cat - <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca | grep -E 'POOL43_10RL|POOL101_11RL|POOL18S_07RL|POOL2S_03RL' | perl -lane '$F[$#F] = "NoPhenotype"; print join("\t", @F); ') > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt1

#Added in the following four individuals with the tag 'NoPhenotype' at the end manually in the pt1 file
#POOL43_10RL POOL101_11RL POOL18S_07RL POOL2S_03RL
