#!/bin/sh

#qsub -I -l walltime=72:00:00,mem=8gb,nodes=1:ppn=6

#sffFile contains a list of every sequencing read/run and the breakdown/separation of each original .sff file into each MID tag
#ALL_MAPPING is where each individual is mapped back to the appropriate MID tag/.sff file combination from sffFile directory 

#Create appropriate main directories in ALL_MAPPING and then appropriate subdirectories in each main directory (main directory i.e. Pools/mapping_pool_1-46/, and subdirecory i.e. POOL14-01RL/). Then creating appropriate directories for each .sff file original location in sffFile/,  

20130823 - Going through past runs of code and figuring out what was done initially; collecting this information here as the first entry in this MainScript1/readme file for documentation (tends to be what I use instead of the LABDIRECTORY notebook thing I had previously tried to setup/use)

ls -lrt /home/pg/ALL_MAPPING/. | awk '{ print $9 } ' > home_pg_ALL_MAPPING_dirList.20130513.txt

bash CollectLSInfoPerDir.sh

bash DataOrganization.CollectOrigSFFFileInformation.sh

cat ../sffFiles/home_pg_ALL_MAPPING_ALL_Pools_sff_OrigLocations.Unedited.20130513.txt | sort | uniq > ../sffFiles/home_pg_ALL_MAPPING_ALL_Pools_sff_OrigLocations.Unedited.20130513.SortUniq.txt

#bash ../sffFiles/BreakUpsffFiles.sh
bash ../sffFiles/MakeDir.sh

bash DataOrganization.CombinePoolNameswsffFileIDs.sh

#Edit DataOrganization.CombinePoolNameswsffFileIDs.Thu_May_16_14:48:16_CDT_2013.output -- commenting out lines that are incorrect with '#'

bash DataOrganization.ConnectPoolsffFileswithIndvMIDRLsffFiles.sh DataOrganization.CombinePoolNameswsffFileIDs.Thu_May_16_14:48:16_CDT_2013.Edited.output

bash DataOrganization.CollectAllIndvMIDRLsffFiles.sh

bash ../sffFiles/Process_sffFile.BreakUpMIDs.Launcher.vs1.sh

bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs1.sh DataOrganization.CollectAllIndvMIDRLsffFiles.Thu_May_16_14:54:21_CDT_2013.RedundantDirectoriesRemoved.output

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs1.sh DataOrganization.CollectAllIndvMIDRLsffFiles.Thu_May_16_14:54:21_CDT_2013.RedundantDirectoriesRemoved.output 0


2nd run with pools mapping_pool_91-97 and mapping_pool_98-105, as well as edits + updating code
~~~~~~~~~~~

ls -lrt /home/pg/ALL_MAPPING/. | awk '{ print $9 } ' > home_pg_ALL_MAPPING_dirList.20130827.txt

cat home_pg_ALL_MAPPING_dirList.20130827.txt > home_pg_ALL_MAPPING_dirList.20130827.NotCreatedYet.txt
#Then manually edited to remove directories already used/processed before -- kept mapping_pool_91-97 and mapping_pool_98-105

bash DataOrganization.CollectLSInfoPerDir.vs2.sh home_pg_ALL_MAPPING_dirList.20130827.NotCreatedYet.txt #Creates subdirectory folder-list per directory

bash DataOrganization.MakeDir.vs2.sh home_pg_ALL_MAPPING_dirList.20130827.NotCreatedYet.txt #Creates the rest of the sub-subdirectories, etc. in ALL_MAPPING

bash DataOrganization.CollectOrigSFFFileInformation.vs2.sh home_pg_ALL_MAPPING_dirList.20130827.NotCreatedYet.txt #Creates home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.${date1}.txt file

cat ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130828_123042.txt | sort | uniq > ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130828_123042.SortUniq.txt

#Edit -- Directory location for sff file H37BKPH01 was changed from /data/454/raw/2013/R_2013_02_08_11_30_15_FLX08100638_Administrator_CS_KK_PG_MM5_Pool_91-2/D_2013_02_09_15_00_17_node1_altSignalProcessing/sff/H37BKPH01.sff to /data/454/raw/2013/R_2013_02_08_11_30_15_FLX08100638_Administrator_CS_KK_PG_MM5_Pool_91-2/D_2013_02_12_10_29_52_node2_signalProcessing/sff/H37BKPH01.sff (and *02.sff)
(bash DataOrganization.CorrectPool91Script.vs1.sh)
#Edit -- Change made in DataOrganization.CombinePoolNameswsffFileIDs.vs2.20130828_124947.Edited.output as well too 

#bash ../sffFiles/BreakUpsffFiles.vs2.sh ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130828_123042.SortUniq.txt
bash ../sffFiles/DataOrganization.MakeDir.vs2.sh ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130828_123042.SortUniq.Edited.txt

bash DataOrganization.CombinePoolNameswsffFileIDs.vs2.sh home_pg_ALL_MAPPING_dirList.20130827.NotCreatedYet.txt ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130828_123042.SortUniq.Edited.txt #Creates file such as DataOrganization.CombinePoolNameswsffFileIDs.vs2.${date1}.output

bash DataOrganization.ConnectPoolsffFileswithIndvMIDRLsffFiles.vs2.sh home_pg_ALL_MAPPING_dirList.20130827.NotCreatedYet.txt DataOrganization.CombinePoolNameswsffFileIDs.vs2.20130828_124947.Edited.output 

bash DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.sh home_pg_ALL_MAPPING_dirList.20130827.NotCreatedYet.txt #Creates DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.${date1}.output file 

bash ../sffFiles/DataProcessing.BreakUpMIDs.Launcher.vs2.sh ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130828_123042.SortUniq.Edited.txt

bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130828_141053.output

#cat DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130828_141053.output | grep -E 'H37BKPH|H4KITRJ|H4PUW3D' > DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130828_141053.Leftover.output

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130828_141053.output

#cp -p home_pg_ALL_MAPPING_dirList.20130827.txt home_pg_ALL_MAPPING_dirList.WorkingMainList.txt
#cat DataOrganization.CombinePoolNameswsffFileIDs.Thu_May_16_14\:48\:16_CDT_2013.Edited.output DataOrganization.CombinePoolNameswsffFileIDs.vs2.20130828_124947.output > DataOrganization.CombinePoolNameswsffFileIDs.vs2.WorkingMainFile.output #Combining all sections. Trying to use 'WorkingMainFile' as means to keep all parts together
#cat DataOrganization.CollectAllIndvMIDRLsffFiles.Thu_May_16_14\:54\:21_CDT_2013.RedundantDirectoriesRemoved.JustFirstTwoCols.output DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130828_141053.output > DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.output
#cp -p DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.output DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output 
#Made DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output  file by changing all earlier directories for mapping_pools.... to include the /Pools/... directory via the following command in vi .,$ s/ING\/mapping_pool/ING\/Pools\/mapping_pool/g 

#At this point catching any .fastq files or other downstream processed files that were lost or not properly processed in order for them to be resubmitted
bash DataProcessing.Pipeline.CheckForErroneousFiles.vs1.sh home_pg_ALL_MAPPING_dirList.WorkingMainList.txt DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output fastq #Create DataProcessing.Pipeline.CheckForErroneousFiles.vs1.DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.Redo.${SearchTerm1}.${date1}.output file

bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataProcessing.Pipeline.CheckForErroneousFiles.vs1.DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.Redo.fastq.20130830_165929.output

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataProcessing.Pipeline.CheckForErroneousFiles.vs1.DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.Redo.fastq.20130830_165929.output


3rd run with mapping_pool_106-124
~~~~~~~~~~~

ls -lrt /home/pg/ALL_MAPPING/. | awk '{ print $9 } ' > home_pg_ALL_MAPPING_dirList.20130831.txt

cat home_pg_ALL_MAPPING_dirList.20130831.txt > home_pg_ALL_MAPPING_dirList.20130831.NotCreatedYet.txt
#Then manually edited to remove directories already used/processed before -- kept mapping_pool_106-124

bash DataOrganization.CollectLSInfoPerDir.vs2.sh home_pg_ALL_MAPPING_dirList.20130831.NotCreatedYet.txt #Creates subdirectory folder-list per directory

bash DataOrganization.MakeDir.vs2.sh home_pg_ALL_MAPPING_dirList.20130831.NotCreatedYet.txt #Creates the rest of the sub-subdirectories, etc. in ALL_MAPPING

bash DataOrganization.CollectOrigSFFFileInformation.vs2.sh home_pg_ALL_MAPPING_dirList.20130831.NotCreatedYet.txt #Creates home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.${date1}.txt file

cat ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130831_103251.txt | sort | uniq > ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130831_103251.SortUniq.txt

bash ../sffFiles/DataOrganization.MakeDir.vs2.sh ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130831_103251.SortUniq.txt

bash DataOrganization.CombinePoolNameswsffFileIDs.vs2.sh home_pg_ALL_MAPPING_dirList.20130831.NotCreatedYet.txt ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130831_103251.SortUniq.txt #Creates file such as DataOrganization.CombinePoolNameswsffFileIDs.vs2.${date1}.output

bash DataOrganization.ConnectPoolsffFileswithIndvMIDRLsffFiles.vs2.sh home_pg_ALL_MAPPING_dirList.20130831.NotCreatedYet.txt DataOrganization.CombinePoolNameswsffFileIDs.vs2.20130831_103251.output 

bash DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.sh home_pg_ALL_MAPPING_dirList.20130831.NotCreatedYet.txt #Creates DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.${date1}.output file 

bash ../sffFiles/DataProcessing.BreakUpMIDs.Launcher.vs2.sh ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130831_103251.SortUniq.txt

bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130831_103352.output

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130831_103352.output

cat ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130831_103251.SortUniq.txt | grep H85C36T01 > ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130831_103251.SortUniq.Leftover.txt #Then ran ../sffFiles/DataProcessing.BreakUpMIDs.Launcher.vs2.sh

cat DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130831_103352.output | grep H85C36T01 > DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130831_103352.Leftover.output #Then ran DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh and DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh

#Also H9H1D9W01

#cat DataOrganization.CombinePoolNameswsffFileIDs.Thu_May_16_14\:48\:16_CDT_2013.Edited.output DataOrganization.CombinePoolNameswsffFileIDs.vs2.20130828_124947.output DataOrganization.CombinePoolNameswsffFileIDs.vs2.20130831_103251.output > DataOrganization.CombinePoolNameswsffFileIDs.vs2.WorkingMainFile.output
#cat DataOrganization.CollectAllIndvMIDRLsffFiles.Thu_May_16_14\:54\:21_CDT_2013.RedundantDirectoriesRemoved.JustFirstTwoCols.output DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130828_141053.output DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130831_103352.output > DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.output
#cp -p DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.output DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output 

bash DataProcessing.Pipeline.CheckForErroneousFiles.vs2.sh home_pg_ALL_MAPPING_dirList.WorkingMainList.txt DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output fastq

bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataProcessing.Pipeline.CheckForErroneousFiles.vs1.DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.Redo.fastq.20130831_235915.output

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataProcessing.Pipeline.CheckForErroneousFiles.vs1.DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.Redo.fastq.20130831_235915.output

cat home_pg_ALL_MAPPING_ALL_Pools_sff_OrigLocations.Unedited.20130513.SortUniq.txt home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130828_123042.SortUniq.Edited.txt home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130831_103251.SortUniq.txt > home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.WorkingMainList.SortUniq.txt

bash ../sffFiles/DataProcessing.BreakUpMIDs.Launcher.vs2.sh ../sffFiles/DataProcessing.Pipeline.CheckForErroneousFiles.vs1.DataProcessing.BreakUpMIDs.vs1.WorkingMainFile.Edited.Redo.fastq.20130831_235915.output


20130903 -- Created the PreMerge and PostMerge Pools/separation in the folder hierarchy; began continued development of pt3/the merging/dupl/BQSR steps in the pipeline
~~~~~~~~~~~
PreMerge

bash DataProcessing.RemoveaCertainFile.vs1.sh home_pg_ALL_MAPPING_dirList.WorkingMainList.txt PreMerge #Cleaned up old .preGATK* and *sam.gz files so can more easily locate folders where the pt2 code did not run properly (e.g. a lack of these files existing versus trying to find files with the proper timestamp signature)

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output #Redoing everything

#Note -- at this point changed original .pt3 of the pipeline scripts to .pt4 and made the 'merge file' portion of the original .pt3 into a stand alone .pt3

bash DataOrganization.AddOrRemoveRGInformation.Launcher.vs1.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output #Run of DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh included an error for the @RG information so used an extra script to correct every files. This is not intended to be normally used in the pipeline and the DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.vs2.sh script has been corrected

bash DataOrganization.CreatebamMergeList.vs1.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output #Creates file DataOrganization.CreatebamMergeList.vs1.${date1}.output

bash DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20130909_231450.output

bash DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20130909_231450.output

#Possibly was going to drop reads that did not map to expected probes but delaying this decisions since some issues came out. Will come back to figure out if this is proper after putting the rest of the pipeline together
#samtools view POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.bam | python ../../../../DataProcessing.Pipeline.DropOffTargetReads.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/OID41305_HG19_Match5_probes/OID41305_HG19_Match5_probe_coverage.bed --file2 - | vi -
#R --vanilla --slave --args < ../../../../DataProcessing.Pipeline.DropOffTargetReads.plot.R /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/OID41305_HG19_Match5_probes/OID41305_HG19_Match5_probe_coverage.chr4.bed POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.chr4.formattted POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.chr4.OverlapPlot.jpeg
#R --vanilla --slave --args < ../../../../DataProcessing.Pipeline.DropOffTargetReads.plot.R /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/OID41305_HG19_Match5_probes/OID41305_HG19_Match5_probe_coverage.chr4.bed 454Contigs.chr4.formatted 454Contigs.chr4.OverlapPlot.jpeg
#
#20131029 edited
#R --vanilla --slave --args < ../../../../DataProcessing.Pipeline.DropOffTargetReads.plot.R /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.chr4.bed POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.chr4.formattted POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.chr4.OverlapPlot.120326_HG19_SW_MM5_EZ.jpeg

#Created pt5 to be run on a single interactive node, but running ReduceReads and creating a raw .vcf file for the entire, merged .bam file at once was too long. Created support scripts to break up merged .bam file into separate chromosome .bam files so that raw .vcfs could be made and ReduceReads could be run
#bash DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged DataOrganization.CreatebamMergeList.vs1.20130916_105823.output

bash DataProcessing.Pipeline.pt5a.SplitBamIntoChromosomes.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.preGATK.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt5b.CreateVCFfilesPerChromosome.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.preGATK.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt5c.RunReduceReadsPerChromosome.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.preGATK.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged


20130923 - 4th run with mapping_pool_SOLID
~~~~~~~~~~~

#Manually created directories in ALL_MAPPING Pools subfolders following the excel spreadsheet POP_genomics_sample_information_2 and used 'touch' to create expected .sff files using ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130925_103251.SortUniq.txt created below to determine who got what .sff file

ls -lrt /data/454/raw/2011/*PG*/*/sff/. | grep sff | grep -E 'raw|H' | perl -sane 'if ($F[0] =~ m/raw/) { $tempVal1 = $F[0]; } else { if ($tempVal1 =~ m/(.*)(\.\:)/) { print $1, $F[8], "\n"; } }' > ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130925_103251.SortUniq.txt 

#Manually added in SOLID pools from 2012 directory

bash ../sffFiles/DataOrganization.MakeDir.vs2.sh ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130925_103251.SortUniq.txt

bash DataOrganization.CombinePoolNameswsffFileIDs.vs2.sh home_pg_ALL_MAPPING_dirList.20130831.NotCreatedYet.txt ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130925_103251.SortUniq.txt #Creates file such as DataOrganization.CombinePoolNameswsffFileIDs.vs2.${date1}.output

#Had to clean up file DataOrganization.CombinePoolNameswsffFileIDs.vs2.20130925_150954.output manually

bash DataOrganization.ConnectPoolsffFileswithIndvMIDRLsffFiles.vs2.sh home_pg_ALL_MAPPING_dirList.20130831.NotCreatedYet.txt DataOrganization.CombinePoolNameswsffFileIDs.vs2.20130925_150954.output

bash DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.sh home_pg_ALL_MAPPING_dirList.20130831.NotCreatedYet.txt #Creates DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.${date1}.output file

cp -p DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130925_152507.output DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130925_152507.Edited.output

#Manually removed some erroneous entries from DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130925_152507.Edited.output 

bash ../sffFiles/DataProcessing.BreakUpMIDs.Launcher.vs2.sh ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130925_103251.SortUniq.txt

bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130925_152507.Edited.output

#NOTE: At this point rerunning everything with QC pipeline components

DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh

bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130925_152507.Edited.output

#DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output did not include mapping_SOLID runs yet at this point. Updated the file afterwards (since there were different needs for this pt1 script in particular for each group of folder/files that would not matter with later downstream pts).
bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output

cat DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130925_152507.Edited.output > asdadad

mv asdadad DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output 

bash DataProcessing.Pipeline.CheckForPresenceOfCertainFile.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output PreMerge QCed.preGATK.QCed.bam

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataProcessing.Pipeline.CheckForPresenceOfCertainFile.vs2.QCed.preGATK.QCed.bam.20130928_134627.output

bash DataOrganization.CreatebamMergeList.vs1.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output #Creates file DataOrganization.CreatebamMergeList.vs1.${date1}.output

#Manually merged POOL39_2-... with POOL39-... lines

bash DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20130928_145300.output

bash DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20130928_145300.output

#Manually curated 20130930_POP_genomics_sample_information_2.edited.csv to look for duplicate entries and remove those that had less coverage
#Ran below code to cut down DataOrganization.CreatebamMergeList.vs1.20130928_145300.output to only include those individuals being used (and not those either missing IDs, exist as duplicates, etc.). For the moment deciding not to merge individuals who were run twice on MM5 (maybe bias with extra coverage on just a few individuals?) or run on SOLID and MM5. Also generally opted to use MM5 over SOLID unless coverage in SOLID was much better than MM5 (which was generally not the case). A few individuals were run twice on MM5, versus on SOLID and MM5
#cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20130930_POP_genomics_sample_information_2.edited.noMeta.dropIndvs.csv | grep ool | sort | wc -- to help check expected total number of entries to be kept when using DataProcessing.Pipeline.DropIndividualsBasedOnPopGenomFile.vs2.py with 20130930_POP_genomics_sample_information_2.edited.noMeta.dropIndvs.csv

python DataProcessing.Pipeline.DropIndividualsBasedOnPopGenomFile.vs2.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20130930_POP_genomics_sample_information_2.edited.noMeta.dropIndvs.csv --file2 DataOrganization.CreatebamMergeList.vs1.20130928_145300.output > DataOrganization.CreatebamMergeList.vs1.20130928_145300.dropIndvs.output

#bash DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged DataOrganization.CreatebamMergeList.vs1.20130928_145300.dropIndvs.output

bash DataProcessing.Pipeline.pt6a.SplitBamIntoChromosomes.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6b.CreateVCFfilesPerChromosome.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6c.RunReduceReadsPerChromosome.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

python DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20130930_POP_genomics_sample_information_2.edited.noMeta.dropIndvs.csv --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Sample_status_list_9-16-13_CT.edited.noMeta.csv > DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output
echo "macsid macsid MID# POOLID-MID# race posage hesn lesn lesc othersc vrapid rapid vslow slow timetoART timeaids" > DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output.header 
#20131004 Note -- four individuals not present in Sample_status_list file, 41478,10,POOL43-10RL
#20040,7,POOL18S-07RL
#30550,11,POOL101-11RL
#20770,3,POOL2S-03RL
#Unsure why at current point, will consult with Kevin/Sudhir after first run through of downstream analysis is done. Also, with these four individuals, total count is 756, not 757. Kevin suggested should have 757 individuals in total, so possibly missing someone? Unless POOL39 is being double counted by their count as well
#20131004 Note -- POOL57 MID10 has folder name ...-010RL for whatever reason originally, probably typo. Just fyi to be aware.

#20131004 Note -- at this point changed naming schematics to go from pt6a, pt6b, pt6c, etc... to go to pt6.All.pta, pt6.Samtools.pta, pt6.GATK.pta, etc... to delineate the different branching paths of the analysis steps for each analysis/variant-calling tool being used, with steps/code being used by all branches given the pt6.All.pt.... designation

python DataProcessing.Pipeline.CreatePhenotypeFile.vs1.py --file1 DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno

python DataProcessing.Pipeline.CreateCovariateFile.vs1.py --file1 DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar 

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar | awk '{ if ($5 == "1") { print $1, "\t", $2 } } ' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDIIDs

bash DataProcessing.Pipeline.pt6.All.ptb.ConvertVCFtoPEDPerChromosome.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools 

bash DataProcessing.Pipeline.pt6.All.ptc.CollapsePedFilesIntoSingleFile.vs2.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools

bash DataProcessing.Pipeline.pt6.All.ptd.PLINKlogisticRegression.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools

(20131012 -- time pt check)

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt --extract AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.bim.SNPsonly.snpIDs --r2 --ld-window-r2 0.8 --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r8 &

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt --extract AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.bim.SNPsonly.snpIDs --r2 --ld-window-r2 0 --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0 &

python ../../../../DataProcessing.Pipeline.pruneLDbased.py --file1 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld --r2 .1 > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt &

python ../../../../DataProcessing.Pipeline.pruneLDbased.py --file1 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld --r2 .8 > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump80.txt &

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt | awk '{ print $1 }' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt.snpList

perl /home/michaelt/Scripts/Perl/ListSubset.pl --file1 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt.snpList --subset 250000 --output AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt.snpList.250k

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt --extract AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt.snpList.250k --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k --recode --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.ped

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.pedind

ln -s AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.bim AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.pedsnp

/home/michaelt/Software/EIG5.0.1/bin/smartpca -p AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.parfile
#Note -- ran this on spudhead b/c smartpca was working already there but wasn't on olivia/miranda yet. Will ask Aleksandar to install eigenstrat if possible at some point

python ../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.Results.pca.altrd.race
#scp'ing the necessary files back and forth from work computer and spudhead

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.Results.pca.altrd.race -c 1:2 -p American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic:White_non-Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.Results.pca.altrd.race.xtxt

#/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.Results.pca.altrd.race -c 1:2 -p American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic:White_non-Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.Results.pca.altrd.race.xtxt

20131013
~~~~~~~~~~~
#At this point noticed names were off in that POOL01S, POOL02S, etc... were not included, just POOL01, POOL02, etc...so had duplicates, i.e. SOLID array names not captured properly in the .bam files and all subsequent processes. Reran SOLID arrays and then merge/post-merge scripts

cat DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output | grep SOLID > DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output.SOLID

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output.SOLID

cat DataOrganization.CreatebamMergeList.vs1.20130928_145300.dropIndvs.output | grep SOLID > DataOrganization.CreatebamMergeList.vs1.20130928_145300.dropIndvs.output.SOLID

bash DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20130928_145300.dropIndvs.output.SOLID

bash DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20130928_145300.dropIndvs.output.SOLID

#bash DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged DataOrganization.CreatebamMergeList.vs1.20130928_145300.dropIndvs.output

bash DataProcessing.Pipeline.pt6.All.pta.SplitBamIntoChromosomes.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.Samtools.pta.CreateVCFfilesPerChromosome.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.GATK.pta.RunReduceReadsPerChromosome.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.All.ptb.ConvertVCFtoPEDPerChromosome.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/Samtools AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools

bash DataProcessing.Pipeline.pt6.All.ptc.CollapsePedFilesIntoSingleFile.vs2.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/Samtools AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.bim | perl -lane 'if (($F[4] =~ m/^[AGTC]$/) && ($F[5] =~ m/^[AGTC]$/)) { print join("\t", @F);}' > Pools/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.bim.SNPsonly

cat Pools/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.bim.SNPsonly | awk '{ print $2 }' > Pools/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.bim.SNPsonly.snpIDs

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt --extract AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.bim.SNPsonly.snpIDs --r2 --ld-window-r2 0.8 --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r8 &

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt --extract AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.bim.SNPsonly.snpIDs --r2 --ld-window-r2 0 --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0 &

python ../../../../DataProcessing.Pipeline.pruneLDbased.py --file1 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld --r2 .1 > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt &

python ../../../../DataProcessing.Pipeline.pruneLDbased.py --file1 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld --r2 .8 > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump80.txt &

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt | awk '{ print $1 }' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt.snpList

perl /home/michaelt/Scripts/Perl/ListSubset.pl --file1 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt.snpList --subset 250000 --output AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt.snpList.250k

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt --extract AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt.snpList.250k --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k --recode --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.ped

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.pedind

ln -s AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.bim AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.pedsnp

/home/michaelt/Software/EIG5.0.1/bin/smartpca -p AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.parfile
#Note -- ran this on spudhead b/c smartpca was working already there but wasn't on olivia/miranda yet. Will ask Aleksandar to install eigenstrat if possible at some point

python ../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.Results.pca.altrd.race

python ../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.noOutlierRmvl.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.Results.pca.altrd.race -c 1:2 -p American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic:White_non-Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.Results.pca.altrd.race.xtxt


bash DataProcessing.Pipeline.pt6.All.ptd.PLINKlogisticRegression.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools

bash DataProcessing.Pipeline.pt6.GATK.ptb.CreateVCFfilesPerChromosome.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged 

bash ../../../../DataProcessing.Pipeline.pt6.GATK.ptc.MergeVCFsAcrossChromosomes.vs1.sh

bash DataProcessing.Pipeline.pt6.GATK.ptd.RecalAndApplyVCFfilesPerChromosome.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

vcftools --gzvcf AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz --plink --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP

plink --file AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP

bash DataProcessing.Pipeline.pt6.All.ptd.PLINKlogisticRegression.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --maf .01 --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01

perl /home/michaelt/Scripts/Perl/ListSubset.pl --file1 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.bim --subset 250000 --output AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.bim

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.bim | awk '{ print $2 }' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.bim.onlySNPids

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01 --extract AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.bim.onlySNPids --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k --recode --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.ped

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.pedind 

ln -s AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.bim AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.pedsnp

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --maf .01 --geno .95 --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95 --recode --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.ped

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.pedind

ln -s AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.bim AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.pedsnp

/home/michaelt/Software/EIG5.0.1/bin/smartpca -p AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.parfile

python ../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.Results.pca.altrd.race

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.Results.pca.altrd.race.plot.1vs2.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.Results.pca.altrd.race.plot.3vs4.xtxt

python ../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.Results.pca.altrd.race

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.Results.pca.altrd.race.plot.1vs2.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.Results.pca.altrd.race.plot.3vs4.xtxt

python ../../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.noOutlierRmvl.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race

python ../../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.noOutlierRmvl.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race.plot.1vs2.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race.plot.3vs4.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race.plot.1vs2.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race.plot.3vs4.xtxt


plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt --maf .01 --geno .95 --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.bim | perl -lane 'if (($F[4] =~ m/^[AGTC]$/) && ($F[5] =~ m/^[AGTC]$/)) { print join("\t", @F);}' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.bim.SNPsonly

perl /home/michaelt/Scripts/Perl/ListSubset.pl --file1 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.bim.SNPsonly --subset 250000 --output AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.250k.bim.SNPsonly

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.250k.bim.SNPsonly | awk '{ print $2 }' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.250k.bim.SNPsonly.onlySNPids

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95 --extract AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.250k.bim.SNPsonly.onlySNPids --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k --recode --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.ped

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.pedind

ln -s AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.bim AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.pedsnp

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/smartpca -p AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.250k.parfile

python ../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.Results.pca.altrd.race

python ../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.noOutlierRmvl.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.Results.pca.altrd.race.plot.1vs2.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.Results.pca.altrd.race.plot.3vs4.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race.plot.1vs2.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race.plot.3vs4.xtxt


python DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.Results.pca --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.5PCs.covar
#Edit code in between runs
python DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.Results.pca --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.5PCs.covar | awk '{ print $1, "\t", $2 } ' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.5PCs.covar.FIDIIDs

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.5PCs.covar | awk '{ if ($5 == "1") { print $1, "\t", $2 } } ' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.5PCs.covar.WhiteOnly.FIDIIDs

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar | awk '{ print $1, "\t", $2 } ' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar | awk '{ if ($5 == "1") { print $1, "\t", $2 } } ' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.WhiteOnly.FIDIIDs

bash DataProcessing.Pipeline.pt6.All.ptd.PLINKlogisticRegression.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP
#Edit code in between runs
bash DataProcessing.Pipeline.pt6.All.ptd.PLINKlogisticRegression.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.5PCs.covar --covar-number 1,2,4,5,6,7,8 --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.5PCs.covar.WhiteOnly.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno1.5PCsmaf01geno95.Assoc 

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.5PCs.covar --covar-number 1,2,4,5,6,7,8 --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.5PCs.covar.WhiteOnly.FIDIIDs --maf .01 --geno .95 --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.WhitesOnly.Pheno1.5PCsmaf01geno95.Assoc 

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8,9,10,11,12,13 --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.WhiteOnly.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno1.10PCsmaf01geno95.Assoc 

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8,9,10,11,12,13 --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.WhiteOnly.FIDIIDs --maf .01 --geno .95 --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.WhitesOnly.Pheno1.10PCsmaf01geno95.Assoc 

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2 --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.Assoc

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8 --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.5PCsmaf01geno05.Assoc

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8,9,10,11,12,13 --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.10PCsmaf01geno05.Assoc

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno1.5PCsmaf01geno95.Assoc.assoc.logistic Pheno1_Logistic_Whites_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno1.5PCsmaf01geno95.Assoc.assoc.logistic.QQplot.jpeg & 

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.WhitesOnly.Pheno1.5PCsmaf01geno95.Assoc.assoc.logistic Pheno1_Logistic_Whites_maf01geno95_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.WhitesOnly.Pheno1.5PCsmaf01geno95.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno1.10PCsmaf01geno95.Assoc.assoc.logistic Pheno1_Logistic_Whites_10PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno1.10PCsmaf01geno95.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.WhitesOnly.Pheno1.10PCsmaf01geno95.Assoc.assoc.logistic Pheno1_Logistic_Whites_maf01geno95_10PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.WhitesOnly.Pheno1.10PCsmaf01geno95.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.Assoc.assoc.logistic Pheno1_Logistic AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.5PCsmaf01geno05.Assoc.assoc.logistic Pheno1_Logistic_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.5PCsmaf01geno05.Assoc.assoc.logistic.QQPlot.jpeg &

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.10PCsmaf01geno05.Assoc.assoc.logistic Pheno1_Logistic_10PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.10PCsmaf01geno05.Assoc.assoc.logistic.QQPlot.jpeg &

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8 --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --maf .01 --geno .95 --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno1.5PCsmaf01geno05.Assoc

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8,9,10,11,12,13 --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --maf .01 --geno .95 --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno1.10PCsmaf01geno05.Assoc

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno1.5PCsmaf01geno05.Assoc.assoc.logistic Pheno1_Logistic_maf01geno95_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno1.5PCsmaf01geno05.Assoc.assoc.logistic.QQplot.jpeg & 

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno1.10PCsmaf01geno05.Assoc.assoc.logistic Pheno1_Logistic_maf01geno95_10PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno1.10PCsmaf01geno05.Assoc.assoc.logistic.QQplot.jpeg & 

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.NoCovar.Assoc

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1 --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.justPOOLcovar.Assoc

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 2 --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.justMIDcovar.Assoc

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 4,5,6,7,8 --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.noPOOLMIDcovar.5PCsmaf01geno05.Assoc

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.NoCovar.Assoc.assoc.logistic Pheno1_Logistic_noCovar AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.NoCovar.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.justPOOLcovar.Assoc.assoc.logistic Pheno1_Logistic_POOLCovar AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.justPOOLcovar.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.justMIDcovar.Assoc.assoc.logistic Pheno1_Logistic_MIDCovar AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.justMIDcovar.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.noPOOLMIDcovar.5PCsmaf01geno05.Assoc.assoc.logistic Pheno1_Logistic_noPOOLMID_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.noPOOLMIDcovar.5PCsmaf01geno05.Assoc.assoc.logistic.QQplot.jpeg &

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 2 --covar /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8 --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno2.5PCsmaf01geno05.Assoc

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno2.5PCsmaf01geno05.Assoc.assoc.logistic Pheno2_Logistic_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno2.5PCsmaf01geno05.Assoc.assoc.logistic.QQplot.jpeg &  

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 2 --covar /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2 --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno2.Assoc

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 2 --covar /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8 --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.WhiteOnly.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno2.5PCsmaf01geno05.Assoc

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 2 --covar /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8 --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --maf .01 --geno .95 --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno2.5PCsmaf01geno05.Assoc

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno2.Assoc.assoc.logistic Pheno2_Logistic AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno2.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno2.5PCsmaf01geno05.Assoc.assoc.logistic Pheno2_Logistic_WhitesOnly_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno2.5PCsmaf01geno05.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno2.5PCsmaf01geno05.Assoc.assoc.logistic Pheno2_Logistic_maf01geno95_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno2.5PCsmaf01geno05.Assoc.assoc.logistic.QQplot.jpeg &

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz | grep PASS | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.body.gz

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz | wc
1272767 968575687 5440128566

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz | awk '{ if ($1 == "12" ) { print $0 } } ' | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr12.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz

perl /home/michaelt/Scripts/Perl/ListSubset.pl --file1 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr12.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz --subset 10000 --output AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr12.GATK.ReduceReads.UG.VQSR.SNP.vcf.ran10000

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz | head -n 123 | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.header.gz

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.header.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.body.gz | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz

rm AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.body.gz 

vcftools --gzvcf AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz --plink --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS

plink --file AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS

perl /home/michaelt/Scripts/Perl/ListSubset.pl --file1 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.bim --subset 250000 --output AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.bim.250k

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.bim.250k | awk '{ print $2 }' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.bim.250k.snpIDs

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS --extract AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.bim.250k.snpIDs --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k --recode --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.ped

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.pedind

ln -s AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.bim AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.pedsnp

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/smartpca -p AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.parfile

python ../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.Results.pca.altrd.race

python ../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.noOutlierRmvl.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.Results.pca.altrd.race.plot.1vs2.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.Results.pca.altrd.race.plot.3vs4.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race.plot.1vs2.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race.plot.3vs4.xtxt

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 3 --covar /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8 --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno3.5PCsmaf01geno05.Assoc

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno3.5PCsmaf01geno05.Assoc.assoc.logistic Pheno3_Logistic_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno3.5PCsmaf01geno05.Assoc.assoc.logistic.QQplot.jpeg &  

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 3 --covar /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2 --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno3.Assoc

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 3 --covar /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8 --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.WhiteOnly.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno3.5PCsmaf01geno05.Assoc

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 3 --covar /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8 --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --maf .01 --geno .95 --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno3.5PCsmaf01geno05.Assoc

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno3.Assoc.assoc.logistic Pheno3_Logistic AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno3.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno3.5PCsmaf01geno05.Assoc.assoc.logistic Pheno3_Logistic_WhitesOnly_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno3.5PCsmaf01geno05.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno3.5PCsmaf01geno05.Assoc.assoc.logistic Pheno3_Logistic_maf01geno95_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno3.5PCsmaf01geno05.Assoc.assoc.logistic.QQplot.jpeg &


PASS run

python DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.Results.pca --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.PASS.maf01.geno95.phenoColEdit.5PCs.covar
#Edit code in between runs
python DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.Results.pca --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.PASS.maf01.geno95.phenoColEdit.10PCs.covar


Annovar (timepoint chk: 20131029)
perl /home/michaelt/Software/annovar/annotate_variation.pl --geneanno  /home/michaelt/Software/annovar/ -buildver hg19 #annotation: 1        861283          861283          G       C
perl /mnt/lustre/home/mturchin20/Software/annovar/annotate_variation.pl --geneanno hg19_Variants4ESP6515.AnnovarFormat.txt /mnt/lustre/home/mturchin20/Software/annovar/humandb/ -buildver hg19
perl /mnt/lustre/home/mturchin20/Software/annovar/summarize_annovar.pl hg19_Variants4ESP6515.AnnovarFormat.txt /mnt/lustre/home/mturchin20/Software/annovar/humandb/ -buildver hg19

python DataPrcoessing.Pipeline.Utility.LinkGeneInfoListwTxStStpInfo.vs1.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/liquid_array.csv > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/liquid_array.wChrTrxStartStop.vs2.csv
#This script used the geneID symbol to compare the two datasets. Decided this was not adequate and used the UCSC ID and HugoID in the second version of the code below

python DataPrcoessing.Pipeline.Utility.LinkGeneInfoListwTxStStpInfo.vs2.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /home/michaelt/Data/Hugo/HGNC_GeneInformation_wUCSCIDs.txt --file3 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/liquid_array.csv > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/liquid_array.wChrTrxStartStop.vs3.csv

python DataProcessing.Pipeline.pt6.All.pte.annotateVariantsManuallyGeneID.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/liquid_array.wChrTrxStartStop.vs3.csv --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr12.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr12.GATK.ReduceReads.UG.VQSR.SNP.vcf.selfAnnotation.gz

python DataProcessing.Pipeline.pt6.All.pte.annotateVariantsManuallyGeneID.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/liquid_array.wChrTrxStartStop.vs3.csv --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.selfAnnotation.gz

python DataProcessing.Pipeline.pt6.All.pte.annotateVariantsManuallyGeneID.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/liquid_array.wChrTrxStartStop.vs3.csv --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr12.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz --findClosest 1 | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr12.GATK.ReduceReads.UG.VQSR.SNP.vcf.selfAnnotation.gz

python DataProcessing.Pipeline.pt6.All.pte.annotateVariantsManuallyGeneID.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/liquid_array.wChrTrxStartStop.vs3.csv --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz --findClosest 1 | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.selfAnnotation.gz

zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz | awk '{ print $1, "\t", $2, "\t", $2, "\t", $4, "\t", $5 }' | grep -v ^# > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.AnnovarFormat

#perl /home/michaelt/Software/annovar/annotate_variation.pl --downdb refGene /home/michaelt/Software/annovar -build hg19
#PWD: /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK

perl /home/michaelt/Software/annovar/annotate_variation.pl --geneanno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.AnnovarFormat /home/michaelt/Software/annovar/ -buildver hg19 

perl /home/michaelt/Software/annovar/summarize_annovar.pl /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.AnnovarFormat /home/michaelt/Software/annovar/ --ver1000g 1000g2012apr --verdbsnp 129 --veresp 6500 --buildver hg19 --remove

#perl /home/michaelt/Software/annovar/annotate_variation.pl -filter -dbtype ljb2_all -buildver hg19 -outfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.AnnovarFormat.ljb2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.AnnovarFormat /home/michaelt/Software/annovar/ -otherinfo




20131111 - 5th run fine-tuning QC setup from beginning of fastq file processing, so redoing everything from the start; still all SOLID and MM5 individuals
~~~~~~~~~~~

mv DataPrcoessing.Pipeline.Utility.LinkGeneInfoListwTxStStpInfo.vs2.py DataProcessing.Pipeline.Utility.LinkGeneInfoListwTxStStpInfo.vs2.py
mv DataPrcoessing.Pipeline.Utility.LinkGeneInfoListwTxStStpInfo.vs1.py DataProcessing.Pipeline.Utility.LinkGeneInfoListwTxStStpInfo.vs1.py

bash DataProcessing.Pipeline.Utility.GetFastqPreQCStats.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output

bash DataProcessing.Pipeline.Utility.RemoveaCertainFile.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output #To remove QCed.fastq.gz and associated stats files for new run through

bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output

mv DataProcessing.Pipeline.Utility.GetFastqPreQCStats.sh DataProcessing.Pipeline.Utility.GetFastqPreQCStats.vs1.sh

cp -p DataProcessing.Pipeline.Utility.GetFastqPreQCStats.vs1.sh DataProcessing.Pipeline.Utility.GetFastqQCStats.vs1.sh

bash DataProcessing.Pipeline.Utility.GetFastqQCStats.vs1.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output

#Notes: original sff location of /home/pg/michaelt/Data/ALL_MAPPING/Pools/PreMerge/mapping_pool_1-46/POOL20-12RL/. has 4 MIDs with good quality -- maybe the MID shouldn't be 12? Nah, there are 7 MIDs for POOL20 so it's unlikely to just be a MID12 thin
#Redo sff of /home/pg/michaelt/Data/ALL_MAPPING/Pools/PreMerge/mapping_pool_69-77/POOL75-09RL/ HX05RSL01.RL9.sff and HX05RSL02.RL9.sff

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output

cp -p DataProcessing.Pipeline.CheckForPresenceOfCertainFile.vs2.sh DataProcessing.Pipeline.CheckForPresenceOfCertainFile.vs3.sh
 
mv DataProcessing.Pipeline.CheckForPresenceOfCertainFile.vs3.sh DataProcessing.Pipeline.Utility.CheckForPresenceOfCertainFile.vs3.sh

bash DataProcessing.Pipeline.Utility.CheckForPresenceOfCertainFile.vs3.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output #searching for $baseFile1.QCed.preGATK.QCed.bam

mv DataOrganization.Utility.CreatebamMergeList.vs1.sh DataOrganization.Utility.CreatebamMergeList.vs1.sh

cp -p DataOrganization.Utility.CreatebamMergeList.vs1.sh DataOrganization.Utility.CreatebamMergeList.vs2.sh

bash DataOrganization.Utility.CreatebamMergeList.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output #Double-checking that this file was made appropriately and that all lines of the original file were used to create the eventual output. Just crossing t's and dotting i's here
#Note do the dropped individual thing to the output file from this run when I get back -- can delete this line later when I've done what I've asked in it, e.g. edit the output file to drop the individuals like was done earlier in this list/history of code

bash DataProcessing.Pipeline.Utility.RemoveaCertainFile.vs2.sh DataOrganization.CreatebamMergeList.vs1.20130928_145300.output 

python DataProcessing.Pipeline.DropIndividualsBasedOnPopGenomFile.vs2.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20130930_POP_genomics_sample_information_2.edited.noMeta.dropIndvs.csv --file2 DataOrganization.CreatebamMergeList.vs1.20131114_193315.output > DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output

bash DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output

bash DataOrganization.Utility.pt3.CheckOrigDirectoriesForFilesMissed.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output 
#Checking to see, given the directories specified in DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output and grepping all the sff files from them, are those sff files present in the single line being used to merge that particular individual's total bam file. E.g. seeing if files present in the directories are properly in the 'CreatebamMergeList' line representing that/those directories for that individual

bash DataOrganization.Utility.pt3.CheckMergedBamsForContainingProperFiles.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output
#Checking to see whether the merged bam files contain all the files we expect it to given the source line of files to combine from DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output

cp -p DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.vs1.sh DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.vs2.sh

bash DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output

bash DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs1.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output

cat 

bash DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output

cat DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output | perl -lane 'my @vals = split(/;/, $F[0]); if ($vals[0] !~ m/(POOL[a-zA-Z0-9]+-\d\dRL)/) { print join("\t", @F) ; } ' > DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output.Missing
#Realized had POOL[a-zA-Z0-9]+_\d\dRL statement in DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.vs2.sh instead of POOL[a-zA-Z0-9]+.*\d\dRL so stats file not properly formatted and needed to redo them

bash DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output.Missing

cat DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.output | perl -lane 'my @array1; push(@array1, $F[1]); foreach my $entry (@F) { if ($entry !~m/[a-zA-Z]/) { push (@array1, $entry); } } print join(",", @array1);' > DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.output.reformatted


#bash DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output

bash DataProcessing.Pipeline.pt6.All.pta.SplitBamIntoChromosomes.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.Samtools.pta.CreateVCFfilesPerChromosome.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.GATK.pta.RunReduceReadsPerChromosome.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.All.ptb.ConvertVCFtoPEDPerChromosome.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/Samtools AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools

bash DataProcessing.Pipeline.pt6.All.ptc.CollapsePedFilesIntoSingleFile.vs2.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/Samtools AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools


#Do reformatting of other statsoutput from earlier in the pipeline (the per-region/run stat file vs per-individual stat file that's already been done) and figure out how or what to graph visually via histograms and such, what's most important to look at
#Think through individual information file using to make sure it's all right and such


bash DataProcessing.Pipeline.pt6.GATK.ptb.CreateVCFfilesPerChromosome.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash ../../../../DataProcessing.Pipeline.pt6.GATK.ptc.MergeVCFsAcrossChromosomes.vs1.sh

bash DataProcessing.Pipeline.pt6.GATK.ptd.RecalAndApplyVCFfilesPerChromosome.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

vcftools --gzvcf AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz --plink --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP

plink --file AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP

#Decided on a change to how I incorporated individuals with multiple runs across Pools. Originally I was just keeping one pool and removing the other from the total dataset (demarcated using carrots, hence the 'noCarrot' version of files to preserve the original markings but remove them for downstream use), but have now decided to not manually choose which pool to use but to collect and use them all for every individual that has multiple pool runs. /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.Utility.CheckForDupEntriesPOPSampleInfoFile.vs1.py identifies individuals with multiple pools and creates a list of each pool for that individual. DataProcessing.Pipeline.Utility.CombineMultPoolsForSameIndvbamMergeList.vs1.sh then removes the individual lines for each pool from the main Createbam file, collects them into a single line and reattaches the new single-line-per-individual to the removed file (producing a list of the compeltely removed main file, a list of all the new lines, and the to-be-used list of removed main file + new list
#Going to rerun the 'new list' of individuals and then continue doing analysis with the removed maine file + new list, e.g. DataOrganization.CreatebamMergeList.vs1.20131114_193315.output.multPOOLperIndvCollected
#There may be one or two individuals being dropped in the mix that should not be so, but for the time being I will move forward with the analysis. I believe, after spot-checking most parts of the analysis and lines of code, that things are functioning more or less as they should be. When the rest of the individuals are added this will allow a second go-around to investigate if other individuals are being dropped unintentionally and to fully iron-out these portions of the code
#python /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.Utility.CheckForDupEntriesPOPSampleInfoFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20131202_POP_genomics_sample_information_Solid_and_MM5.noMeta.noStar.csv

python /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.Utility.CheckForDupEntriesPOPSampleInfoFile.vs1.py --file1 20130930_POP_genomics_sample_information_2.edited.noMeta.noCarrot.csv | perl -F: -ane 'my @vals1 = split(/,/, $F[0]); my $id1 = $vals1[1]; print $id1, ":"; foreach my $entry1 (@F) { my @valsN = split(/,/, $entry1); my @MID = split(//, $valsN[2]); if ($#MID == "3") { print "0", $valsN[2], ","; } else { print $valsN[2], ","; } foreach my $entry2 (@valsN) { if ($entry2 =~ m/ool/) { print $entry2; } } print ":";} print "\n";' | sed "s/'//g" | sed 's/\s//g' | perl -F: -lane 'print join(":", @F[1..$#F]);' > 20130930_POP_genomics_sample_information_2.edited.noMeta.noCarrot.DupListPrepforMergeListUse.txt
#Manually delete first line of this resulting file since it somewhat not needed and adding in an arbitrary piece of code to just 'delete the first line of the output' seems too hard-coded to just put in there. Would rather do it manually and observe the need for this each time this may have to be done

bash DataProcessing.Pipeline.Utility.CombineMultPoolsForSameIndvbamMergeList.vs1.sh /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20130930_POP_genomics_sample_information_2.edited.noMeta.noCarrot.DupListPrepforMergeListUse.txt DataOrganization.CreatebamMergeList.vs1.20131114_193315.output #Creates DataOrganization.CreatebamMergeList.vs1.20131114_193315.output.multPOOLperIndvCollected* files 

cat 20130930_POP_genomics_sample_information_2.edited.noMeta.noCarrot.csv | grep -v \* > 20130930_POP_genomics_sample_information_2.edited.noMeta.noCarrot.dropIndvs.csv

python DataProcessing.Pipeline.DropIndividualsBasedOnPopGenomFile.vs2.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20130930_POP_genomics_sample_information_2.edited.noMeta.noCarrot.dropIndvs.csv --file2 DataOrganization.CreatebamMergeList.vs1.20131114_193315.output.multPOOLperIndvCollected > DataOrganization.CreatebamMergeList.vs1.20131114_193315.multPOOLperIndvCollected.dropIndvs.output 

bash DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.output.multPOOLperIndvCollected.newEntries

bash DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.output.multPOOLperIndvCollected.newEntries

#bash DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged DataOrganization.CreatebamMergeList.vs1.20131114_193315.multPOOLperIndvCollected.dropIndvs.output

bash DataProcessing.Pipeline.pt6.All.pta.SplitBamIntoChromosomes.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.Samtools.pta.CreateVCFfilesPerChromosome.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.GATK.pta.RunReduceReadsPerChromosome.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.GATK.ptb.CreateVCFfilesPerChromosome.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash ../../../../DataProcessing.Pipeline.pt6.GATK.ptc.MergeVCFsAcrossChromosomes.vs1.sh

bash DataProcessing.Pipeline.pt6.GATK.ptd.RecalAndApplyVCFfilesPerChromosome.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

gzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.vcf

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz | grep PASS > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.temp1

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz | grep \# > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.header

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.header AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.temp1 | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz

rm AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.temp1

vcftools --gzvcf AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz --geno .9 --recode --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.geno90.vcf

#NOTE (20131211) -- so realized when I merged multiple POOL#s into one file I kept the POOL#s the same in the .bam file's sample ID tag, so the variant calling partitioned the merged .bam file into separate 'individuals' again. So updated DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.vs3.sh to now change the SM:... ID in the merged .bam file to only contain one POOL# to account for the file only supposed to be representing a single individual now. As a result, redoing everything from the pt3 merge step onwards again

bash DataProcessing.Pipeline.Utility.RemoveaCertainFile.vs2.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.multPOOLperIndvCollected.dropIndvs.output 

bash DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.multPOOLperIndvCollected.dropIndvs.output

bash DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.multPOOLperIndvCollected.dropIndvs.output

#bash DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged DataOrganization.CreatebamMergeList.vs1.20131114_193315.multPOOLperIndvCollected.dropIndvs.output

bash DataProcessing.Pipeline.pt6.All.pta.SplitBamIntoChromosomes.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.Samtools.pta.CreateVCFfilesPerChromosome.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.GATK.pta.RunReduceReadsPerChromosome.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.GATK.ptb.CreateVCFfilesPerChromosome.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash ../../../../DataProcessing.Pipeline.pt6.GATK.ptc.MergeVCFsAcrossChromosomes.vs1.sh

bash DataProcessing.Pipeline.pt6.GATK.ptd.RecalAndApplyVCFfilesPerChromosome.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

gzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.vcf

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz | grep PASS > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.temp1

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz | grep \# > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.header

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.header AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.temp1 | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz

rm AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.temp1

vcftools --gzvcf AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz --geno .9 --recode --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.geno90.vcf

vcftools --gzvcf AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz --geno .85 --recode --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.geno85.vcf

vcftools --gzvcf AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz --geno .95 --recode --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.geno95.vcf

vcftools --gzvcf AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz --missing --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss | awk '{ print $6 }' | perl -lane 'printf "%.2f\n", $F[0];' | sort -g -k 1,1 | uniq -c
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss | awk '{ print $6 }' | perl -lane 'printf "%.2f\n", $F[0];' | sort -g -k 1,1 | uniq -c | perl -lane '$cumulativeSum += $F[0]; push(@F, $cumulativeSum/2292557); print join("\t", @F);'

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/OID41305_HG19_Match5_probes/OID41305_HG19_Match5_probe_coverage.bed | perl -lane 'if ($F[0] =~ m/chr([a-zA-Z0-9]*).*/) { $F[0] = $1; print join("\t", @F); } ' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/OID41305_HG19_Match5_probes/OID41305_HG19_Match5_probe_coverage.formatted.bed

samtools view  ../../mapping_pool_106-124/POOL106_01RL/POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.bam | perl -lane 'my @vals1 = split(//, $F[9]); print $F[2], "\t", $F[3], "\t", $F[3] + scalar(@vals1), "\t", scalar(@vals1);' > ../../mapping_pool_106-124/POOL106_01RL/POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.OverlapPlot.formatted

python DataProcessing.Pipeline.Utility.AddColorInfotoVCFtoolslmissFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.0kbWindow.plotReady

#R --vanilla --slave --args < ../../../../DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs1.R /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/OID41305_HG19_Match5_probes/OID41305_HG19_Match5_probe_coverage.formatted.bed /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_106-124/POOL106_01RL/POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.OverlapPlot.formatted /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs1

R --vanilla --slave --args < ../../../../DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs2.R /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_106-124/POOL106_01RL/POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.OverlapPlot.formatted /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.0kbWindow.plotReady /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs2

python ../../../../DataProcessing.Pipeline.Utility.AddColorInfotoVCFtoolslmissFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss --window 500 > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.500bWindow.plotReady

python ../../../../DataProcessing.Pipeline.Utility.AddColorInfotoVCFtoolslmissFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss --window 1000 > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.1kbWindow.plotReady

python ../../../../DataProcessing.Pipeline.Utility.AddColorInfotoVCFtoolslmissFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss --window 5000 > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.5kbWindow.plotReady

python ../../../../DataProcessing.Pipeline.Utility.AddColorInfotoVCFtoolslmissFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss --window 10000 > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.10kbWindow.plotReady

python ../../../../DataProcessing.Pipeline.Utility.AddColorInfotoVCFtoolslmissFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss --window 25000 > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.25kbWindow.plotReady

python ../../../../DataProcessing.Pipeline.Utility.AddColorInfotoVCFtoolslmissFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss --window 50000 > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.50kbWindow.plotReady

R --vanilla --slave --args < ../../../../DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs2.R /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_106-124/POOL106_01RL/POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.OverlapPlot.formatted /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.500bWindow.plotReady /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs2.500bWindow

R --vanilla --slave --args < ../../../../DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs2.R /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_106-124/POOL106_01RL/POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.OverlapPlot.formatted /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.1kbWindow.plotReady /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs2.1kbWindow

R --vanilla --slave --args < ../../../../DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs3.R /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_106-124/POOL106_01RL/POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.OverlapPlot.formatted /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.1kbWindow.plotReady /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs3.1kbWindow

zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz | awk '{ print $1, "\t", $2, "\t", $2, "\t", $4, "\t", $5 }' | grep -v ^# > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat

perl /home/michaelt/Software/annovar/summarize_annovar.pl /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat /home/michaelt/Software/annovar/ --ver1000g 1000g2012apr --verdbsnp 129 --veresp 6500 --buildver hg19 --remove

mv DataProcessing.Pipeline.DropOffTargetReads.vs1.py DataProcessing.Pipeline.Utility.DropOff.TargetReads.vs1.py

python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.DropOff.VCFVariants.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz --window 1000 | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.DropOffTargetVariants.1kbWindow.vcf.gz

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.DropOffTargetVariants.1kbWindow.vcf.gz --geno .85 --recode --recode-INFO-all --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.DropOffTargetVariants.1kbWindow.85geno

gzip /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.DropOffTargetVariants.1kbWindow.85geno.recode.vcf &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.DropOffTargetVariants.1kbWindow.vcf.gz --geno .95 --recode --recode-INFO-all --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.DropOffTargetVariants.1kbWindow.95geno

gzip /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.DropOffTargetVariants.1kbWindow.95geno.recode.vcf &


#In directory /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles
#Using phenotype file descriptions in DataProcessing.Pipeline.CreatePhenotypeFile.vs1.py to help determine which column + values to use
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($3 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.WhiteOnly.justSeroNeg.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($3 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.WhiteOnly.justSeroPos.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($4 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.WhiteOnly.justSeroHighlyExposedNeg.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.WhiteOnly.justRapidVRapid.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.WhiteOnly.justSlowVSlow.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($6 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.WhiteOnly.justVeryRapid.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($6 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.WhiteOnly.justVerySlow.FID

#In directory /home/michaelt/Data/1000G/AncestralAlignments online sources say these reference files based on Gch37 which is the equivalent of hg19 so should be same reference bases between these files and my own 
wget ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/pilot_data/technical/reference/ancestral_alignments/human_ancestor*
wget http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/analysis_results/supporting/ancestral_alignments/human_ancestor_GRCh37_e59.tar.bz2

#Trying to get AA information for each variant called in the dataset. Following code was copied/pasted from http://seqanswers.com/forums/showthread.php?t=15026 "1000 genomes ancestral alleles?"
ls human_ancestor_*.fa.bz2 | while read IN; do
OUT=`echo $IN | sed 's,bz2$,gz,'`
CHR=`echo $IN | sed 's,human_ancestor_,, ; s,.fa.bz2,,'`
bzcat $IN | sed "s,^>.*,>$CHR," | gzip -c > $OUT
samtools faidx $OUT
done
#

#Test line
samtools faidx human_ancestor_1.fa.gz 1:1000000-1000020 #Expect to see tgggcacagcctcacccagga

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz | fill-aa -a /home/michaelt/Data/1000G/AncestralAlignments/human_ancestor_GRCh37_e59/human_ancestor_ | gzip -c > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz
#EDIT NOTE 03162014 -- when redoing this later just drop out SNPs that don't have an AA too at this point (might as well since they're not being used in our analysis, but probably double check how many are being dropped by this, probably not a lot, e.g. under 1,000?)

vcftools --gzvcf AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --plink --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped | awk '{ print $1, "\t", $2 } ' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDIIDs

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDIIDs | awk '{ print $1 } ' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs

python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.DropOff.VCFVariants.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --window 1000 | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.vcf.gz

python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.DropOff.VCFVariants.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --window 0 | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.0kbWindow.vcf.gz

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.vcf.gz --geno .95 --recode --recode-INFO-all --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno

gzip /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.FID --freq --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg.FID --freq --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos.FID --freq --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg.FID --freq --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid.FID --freq --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow.FID --freq --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid.FID --freq --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow.FID --freq --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow &

#
#20140101 NOTE -- These *.pheno and *.covar files contain FID and IID information for /every/ POOL_* individual, even if the main .vcf/.ped/.whatever file does not contain duplicate entries of the same FID/IID (e.g. multiple POOL_* were combined for a single FID/IID, as should be). So 'total line counts' for some of these *.pheno and *.covar files may be 'off' but in fact this is fine since the main .vcf/.ped/.whatever file is properly delimited. This is just to allow /all/ entries to have information so if the leading POOL_* identifier for a given FID/IID group changes for whatever reason, these information files can still be used since each POOL_* in any particular FID/IID group should be annotated
#

python DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20130930_POP_genomics_sample_information_2.edited.noMeta.noCarrot.dropIndvs.csv --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Sample_status_list_9-16-13_CT.edited.noMeta.csv > DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140101_POP_genomics_sample.status_list_9-16-13.output

python DataProcessing.Pipeline.CreatePhenotypeFile.vs1.py --file1 DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140101_POP_genomics_sample.status_list_9-16-13.output > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno

python DataProcessing.Pipeline.CreateCovariateFile.vs1.py --file1 DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140101_POP_genomics_sample.status_list_9-16-13.output > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar 

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar | awk '{ if ($5 == "1") { print $1, "\t", $2 } } ' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDIIDs

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDIIDs | awk '{ print $1 } ' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs

#Reran creating PLINK pheno files that contain only certain sero+/- and slow/rapid individuals

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($3 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($4 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($6 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($6 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow.FID

#So order is: justSeroHighlyExposedNeg 
paste AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow.frq | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.AllPhenos.frq.gz

gzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow.frq 

#gunzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.frq.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg.frq.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg.frq.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid.frq.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos.frq.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow.frq.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid.frq.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow.frq.gz

#paste <(zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz | tail -n +125 | perl -lane 'print join("\t", @F[0..8]);') <(zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.AllPhenos.frq.gz | tail -n +2) | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.wFreqsRplcGenos.noHeader.vcf.gz

###cat <(zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz | head -n 124) <(zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.wFreqsRplcGenos.noHeader.vcf.gz) | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.wFreqsRplcGenos.vcf.gz
#cat <(zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz | awk '{ if (NR < 125) { print $0 } }' ) <(zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.wFreqsRplcGenos.noHeader.vcf.gz) | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.wFreqsRplcGenos.asddadadad.vcf.gz
#rm /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.wFreqsRplcGenos.noHeader.vcf.gz
#rm AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.*

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | grep exonic | grep -v ncRNA | head -n 10 | perl -F, -lane 'print join("\t", @F[$#F-6..$#F]);' | perl -lane 'print join("\t", @F[$#F-4..$#F]);' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AnnovarTest/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.TestGroup.vs1

perl /home/michaelt/Software/annovar/summarize_annovar.pl /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AnnovarTest/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.TestGroup.vs1 /home/michaelt/Software/annovar/ --ver1000g 1000g2012apr --verdbsnp 129 --veresp 6500 --buildver hg19 --remove

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.TestGroup.vs1 | perl -lane '($F[3], $F[4]) = ($F[4], $F[3]); print join("\t", @F);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.TestGroup.vs2

perl /home/michaelt/Software/annovar/summarize_annovar.pl /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AnnovarTest/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.TestGroup.vs2 /home/michaelt/Software/annovar/ --ver1000g 1000g2012apr --verdbsnp 129 --veresp 6500 --buildver hg19 --remove

python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs1.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.AllPhenos.frq.gz --file3 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 25; $i <= 31; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.txt.gz

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CalcMeandAF.vs1.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.txt.gz

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/DataProcessing.Pipeline.Utility.CalcMeandAF.vs1.GeneSpecificAnalyses.txt | grep PP2D | grep -v PP2DP | sort -g -k 9,9

Timepoint check: 20140106

bash DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs1.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output

cat DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs1.output | perl -lane 'my @array1; push(@array1, $F[1]); foreach my $entry (@F) { if ($entry !~ m/[a-zA-Z]/) { push (@array1, $entry); } } print join(",", @array1);' | awk 'BEGIN{ print "ID,TotalReadsPreQC,TotalReadsQ10,TotalReadsQ10Prcnt,TotalReadsF4,TotalReadsF4Prcnt,TotalReadsF256,TotalReadsF256Prcnt,TotalReadsPostQC,TotalReadsDropped,TotalReadsDroppedPrcnt,AverageReadLengthPreQC,AverageReadLengthPostQC,BedFileCoveragePostQC" } { print $0 } ' > DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs1.output.reformatted 

bash DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.multPOOLperIndvCollected.dropIndvs.output

#Total column numbers post formatting = 27
cat DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.output | perl -lane 'my @array1; push(@array1, $F[1]); foreach my $entry (@F) { if ($entry !~ m/[a-zA-Z]/) { push (@array1, $entry); } } print join(",", @array1);' | awk 'BEGIN{ print "ID,NumberOfOriginalBamFiles,TotalReadsPreQC,TotalReadsPreQCForward,TotalReadsPreQCReverse,TotalReadsPreQCForward_Reverse,PreQCMAPQmean,PreQCMAPQsd,PreQCMean,PreQCSD,PreQCMax,PreQCMin,CoveragePreQC,TotalReadsPostRmdup,TotalReadsPostRmdupDropped,TotalReadsPostRmdupDroppedPercentage,TotalReadsPostQC,TotalReadsPostQCForward,TotalReadsPostQCReverse,TotalReadsPostQCForward_Reverse,PostQCMAPQmean,PostQCMAPQsd,PostQCMean,PostQCSD,PostQCMax,PostQCMin,CoveragePostQC" } { print $0 } ' > DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.output.reformatted

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs1.R /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs1.output.reformatted /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.output.reformatted

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --plink --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode

plink --file /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --make-bed --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.phenoColEdit.ped

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.phenoColEdit.pedind

ln -s /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.bim /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.pedsnp

scp /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.phenoColEdit.ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.phenoColEdit.pedind /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.pedsnp mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/Eigenstrat/.  

cp -p AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.parfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.parfile

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/smartpca -p AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.parfile

scp mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.Results.pca /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/.

python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20140101_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.Results.pca.altrd.race

scp /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.Results.pca.altrd.race mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/Eigenstrat/.

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.Results.pca.altrd.race.plot.1vs2.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.Results.pca.altrd.race.plot.3vs4.xtxt

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.Results.pca.altrd.race | grep White_non-Hispanic | perl lane 'my @vals1 = split(/:/, $F[0]); $F[0] = $vals1[0]; print join("\t", @F);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.Results.pca.altrd.race.FIDs 

#Directory /home/michaelt/Data/HapMap/HapMap3
wget ftp://ftp.ncbi.nlm.nih.gov/hapmap/genotypes/2009-01_phaseIII/plink_format/*
wget ftp://ftp.ncbi.nlm.nih.gov/hapmap/genotypes/2009-01_phaseIII/draft2_samples_QC+.txt

#Direcotry /home/michaelt/Software/liftOver
wget http://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64/liftOver
wget http://hgdownload.cse.ucsc.edu/goldenPath/hg18/liftOver/hg18ToHg19.over.chain.gz

cat hapmap3_r2_b36_fwd.consensus.qc.poly.map | perl -lane '$val1++; print "chr", $F[0], "\t", $F[3], "\t", $F[3]+1, "\t", $val1;' > hapmap3_r2_b36_fwd.consensus.qc.poly.map.liftOverFormat

./../../Software/liftOver/liftOver hapmap3_r2_b36_fwd.consensus.qc.poly.map.liftOverFormat ../../Software/liftOver/hg18ToHg19.over.chain.gz hapmap3_r2_b36_fwd.consensus.qc.poly.map.liftOverFormat.hg19 hapmap3_r2_b36_fwd.consensus.qc.poly.map.liftOverFormat.dropped

paste hapmap3_r2_b36_fwd.consensus.qc.poly.map hapmap3_r2_b36_fwd.consensus.qc.poly.map.liftOverFormat > hapmap3_r2_b36_fwd.consensus.qc.poly.map.liftOverFormat.ForPostProcessing

python /home/michaelt/Data/HapMap/HapMap3/VLookUp.Python.vs1.py --file1 /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.map.liftOverFormat.hg19 --file2 /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.map.liftOverFormat.ForPostProcessing > /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.liftOver.hg18tohg19.map

python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/VLookUp.Python.vs1.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.map --file2 /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.liftOver.hg18tohg19.map | awk '{ print $2 }' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.rsIDs

cat /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.txt | grep -E 'CEU|CHB|YRI' | awk '{ if ($8 =="x") { print $0 } } ' > /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.CEUCHBYRI.Unrelated.txt

cat /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.CEUCHBYRI.Unrelated.txt | awk '{ print $1, "\t", $2 }' > /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.CEUCHBYRI.Unrelated.FIDIIDs

plink --file /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly --make-bed --out /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly

plink --bfile /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly --extract /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.rsIDs --keep /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.CEUCHBYRI.Unrelated.FIDIIDs --recode --out /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData

cat /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.map | awk '{ print $2 }' > /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.map.rsIDs 

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --extract /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.map.rsIDs --recode --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3

plink --file /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3 --make-bed --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3

#There are a few SNPs that do not match up between the two files. Removing them (about ~30?) and recreating the overlap .ped files once more
paste /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.bim /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.bim | awk '{ if (($5 == $11) || ($5 == $12) ) { print $0 } } ' | awk '{ print $2 }' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.FinalListwDrops.rsIDs

plink --bfile /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly --extract /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.FinalListwDrops.rsIDs --keep /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.CEUCHBYRI.Unrelated.FIDIIDs --recode --out /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.vs2

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --extract /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.FinalListwDrops.rsIDs --recode --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.vs2

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.vs2.ped /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.vs2.ped > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.ped

plink --ped AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.ped --map AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.vs2.map --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.phenoColEdit.ped

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.phenoColEdit.pedind

ln -s AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.bim AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.pedsnp

scp AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.phenoColEdit.ped AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.phenoColEdit.pedind AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.pedsnp mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/Eigenstrat/.

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/smartpca -p AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.parfile

scp mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/.

python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20140101_POP_genomics_sample.status_list_9-16-13.output --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt1

cp -p /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.HapMap3vs.py

python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.HapMap3vs.py --file1 /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.txt --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt2

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt2 > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race

sed -i 's/American_Indian_or_Alaskan_Native/AIndian_or_Alaskan_Native/g' AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race

scp /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/Eigenstrat/.

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI -x -o AllPools.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.1vs2.xtxt

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race | awk '{ if ((($2 >= 0) && ($2 <= .2)) && (($3 >= -.01) && ($3 <= .03))) { print $0 } } ' | grep White_non-Hispanic > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI -x -o AllPools.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.3vs4.xtxt

ln -s AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed AllPools.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i  AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed -c 1:2 -p White_non-Hispanic -x -o  AllPools.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.xtxt

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed | perl -lane 'my @vals1 = split(/:/, $F[0]); print $vals1[0];' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs

scp mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/.

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($3 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($4 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($3 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($6 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($6 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow.FID

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs1.R /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs1.output.reformatted /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.output.reformatted

scp /home/pg/michaelt/Data/ALL_MAPPING/Results/Graphics/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs1.*.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/Results/Graphics/.

bash DataProcessing.Pipeline.Utility.GetperfastQpostQCStats.vs1.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output 
#Deleted 4 lines with nothing in them
sed -i 's/(//g' /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperfastQpostQCStats.vs1.output
sed -i 's/)//g' /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperfastQpostQCStats.vs1.output



R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs1.R /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperfastQpostQCStats.vs1.output /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs1.output.reformatted /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.output.reformatted

python DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.5PCs.covar
#Edit code in between runs
python DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | awk '{ print $1, "\t", $2 } ' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.FIDIIDs

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.FIDIIDs | awk '{ print $1 }' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.FIDs

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 1,2,3,4,5,6,7,8 --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.FIDIIDs --logistic --allow-no-sex --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.5PCs

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.FIDIIDs --logistic --allow-no-sex --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.5PCs

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.FIDIIDs --maf .01 --logistic --allow-no-sex --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.5PCs.maf01

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 1,2,3,4,5,6,7,8,9,10,11,12,13 --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.FIDIIDs --logistic --allow-no-sex --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.10PCs

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.5PCs.assoc.logistic Pheno1_Logistic_5PCs /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.5PCs.assoc.logsitic.QQplot.jpeg &

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.10PCs.assoc.logistic Pheno1_Logistic_10PCs /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.10PCs.assoc.logsitic.QQplot.jpeg &

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.frq.gz | perl -lane 'my @vals1 = split(/:/, $F[4]); my @vals2 = split(/:/, $F[5]); push(@F, sprintf("%.0f", $vals1[1]*$F[3])); push(@F, sprintf("%.0f", $vals2[1]*$F[3])); print join("\t", @F); ' | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.frq.wAlleleCounts.gz

#Edited VLookUp.Python.vs1.py
python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/VLookUp.Python.vs1.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.5PCs.assoc.logistic --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.frq.wAlleleCounts.gz | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.5PCs.assoc.logistic.wAC.gz

R --vanilla --slave --args  < /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.ComparePValvsAC.vs1.R /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.5PCs.assoc.logistic.wAC.gz /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.ComparePValvsAC.vs1.plot.jpeg

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.PlotSFS.vs1.R /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.txt.gz /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/Graphics/DataProcessing.Pipeline.Utility.PlotSFS.vs1.plot.jpeg

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.txt.gz | grep exonic | awk -F, '{ print $19 } ' | sort | uniq > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.txt.GeneIDList

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.txt.gz | grep exonic | awk -F, '{ print $19 } ' | sort | uniq > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.txt.GeneIDList.Exonic

zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.txt.gz | grep exonic | grep -v RNA | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.txt.gz

python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs2.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.txt.gz > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs2.ExonicVariants.All.output

#Edited code
python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs2.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.txt.gz > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs2.ExonicVariants.Nonsynonymous.output

#Edited code
python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs2.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.txt.gz > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs2.ExonicVariants.PP2_D.output

#Edited code
python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs2.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.txt.gz > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs2.ExonicVariants.PP2_DP.output

R --vanilla --slave --args < DataProcessing.Pipeline.Analysis.VariantDensity.FollowUpCalculations.vs1.R DataProcessing.Pipeline.Analysis.VariantDensity.vs2.ExonicVariants.PP2_D.output | awk '{ if (($4 != "NA") && ($2 >= 4)) { print $0 } } ' | sort -rg -k 3,3
R --vanilla --slave --args < DataProcessing.Pipeline.Analysis.VariantDensity.FollowUpCalculations.vs1.R DataProcessing.Pipeline.Analysis.VariantDensity.vs2.ExonicVariants.PP2_DP.output | awk '{ if (($4 != "NA") && ($2 >= 4)) { print $0 } } ' | sort -rg -k 3,3

#MAPK12 specifically looking at here

B 22,50691870_1,rs138582408,C,T,C,1154,PASS,AC=6;AF=3.995e-03;AN=1502;BaseQRankSum=7.966;DB;DP=13733;Dels=0.00;FS=2.273;HaplotypeScore=3.8969;InbreedingCoeff=-0.0070;MLEAC=5;MLEAF=3.329e-03;MQ=59.99;MQ0=0;MQRankSum=-0.801;QD=12.54;ReadPosRankSum=-1.530;VQSLOD=0.233;culprit=HaplotypeScore;AA=C,0.00451807,0.00319489,0.00310559,0.00569801,0.02,0.00892857,0,0,6,2,1,4,2,1,0,0,exonic,MAPK12,nonsynonymous SNV,,0.41,0.811214,N,0.22,T,0.0,B,0.921332,N,0.539884,D,0.925,22,50691870,50691870,C,T
D 22,50693697_1,rs201016559,T,C,t,221.31,PASS,AC=2;AF=1.332e-03;AN=1502;BaseQRankSum=-4.072;DB;DP=16220;Dels=0.00;FS=0.000;HaplotypeScore=1.5329;InbreedingCoeff=-0.0031;MLEAC=1;MLEAF=6.658e-04;MQ=59.99;MQ0=0;MQRankSum=0.761;QD=8.20;ReadPosRankSum=0.915;VQSLOD=0.809;culprit=FS;AA=t,0.000753012,0,0,0.0014245,0,0,0,0,1,0,0,1,0,0,0,0,exonic,MAPK12,nonsynonymous SNV,,0.03,0.99724,C,0.86,T,0.885,D,0.997985,U,0.841807,D,4.43,22,50693697,50693697,T,C
P 22,50694078_1,rs141322209,G,A,G,97.36,PASS,AC=3;AF=2.003e-03;AN=1498;BaseQRankSum=5.053;DB;DP=14056;Dels=0.00;FS=8.309;HaplotypeScore=2.3858;InbreedingCoeff=-0.0041;MLEAC=1;MLEAF=6.676e-04;MQ=59.99;MQ0=0;MQRankSum=-0.100;QD=3.14;ReadPosRankSum=-1.239;VQSLOD=0.164;culprit=QD;AA=G,0.00226586,0.00159744,0,0.00286533,0.01,0,0,0,3,1,0,2,1,0,0,0,exonic,MAPK12,nonsynonymous SNV,,0.02,0.999418,C,0.92,T,0.17,P,0.997696,U,0.241217,N,4.38,22,50694078,50694078,G,A
D 22,50694084_1,rs2066776,G,A,G,690.67,PASS,AC=3;AF=2.003e-03;AN=1498;BaseQRankSum=6.300;DB;DP=14065;Dels=0.00;FS=0.000;HaplotypeScore=1.9649;InbreedingCoeff=-0.0018;MLEAC=3;MLEAF=2.003e-03;MQ=59.99;MQ0=0;MQRankSum=0.104;QD=14.39;ReadPosRankSum=-0.306;VQSLOD=1.54;culprit=FS;AA=G,0.00226586,0,0,0.00429799,0,0.0181818,0,0.0178571,3,0,0,3,0,2,0,1,exonic,MAPK12,nonsynonymous SNV,rs2066776,0,0.999259,C,1.0,D,0.996,D,0.998205,U,0.999993,D,4.2,22,50694084,50694084,G,A
B 22,50694251_1,rs200047423,T,C,T,456.62,PASS,AC=4;AF=2.670e-03;AN=1498;BaseQRankSum=-7.928;DB;DP=14144;Dels=0.00;FS=2.627;HaplotypeScore=1.6631;InbreedingCoeff=-0.0051;MLEAC=2;MLEAF=1.335e-03;MQ=59.99;MQ0=0;MQRankSum=-0.851;QD=6.92;ReadPosRankSum=-1.241;VQSLOD=0.931;culprit=QD;AA=T,0.00226586,0.00159744,0,0.00286533,0,0,0,0,3,1,0,2,0,0,0,0,exonic,MAPK12,nonsynonymous SNV,,0,0.997358,C,1.0,D,0.087,B,0.998359,U,0.998098,D,4.49,22,50694251,50694251,T,C
D 22,50694280_1,.,G,A,G,480.31,PASS,AC=1;AF=6.676e-04;AN=1498;BaseQRankSum=5.983;DP=14170;Dels=0.00;FS=6.285;HaplotypeScore=1.2801;InbreedingCoeff=-0.0033;MLEAC=1;MLEAF=6.676e-04;MQ=59.99;MQ0=0;MQRankSum=0.507;QD=12.64;ReadPosRankSum=-1.984;VQSLOD=0.763;culprit=HaplotypeScore;AA=G,0.000755287,0,0,0.00143266,0,0,0,0,1,0,0,1,0,0,0,0,exonic,MAPK12,nonsynonymous SNV,,0,0.999458,C,1.0,D,1.0,D,0.997712,U,0.999999,D,4.44,22,50694280,50694280,G,A
D 22,50694529_1,rs2230823,G,A,G,213.68,PASS,AC=3;AF=1.997e-03;AN=1502;BaseQRankSum=5.839;DB;DP=14376;Dels=0.00;FS=9.486;HaplotypeScore=1.0269;InbreedingCoeff=-0.0038;MLEAC=1;MLEAF=6.658e-04;MQ=59.99;MQ0=0;MQRankSum=0.313;POSITIVE_TRAIN_SITE;QD=6.28;ReadPosRankSum=2.196;VQSLOD=1.17;culprit=QD;AA=G,0.00150602,0,0,0.002849,0,0,0,0,2,0,0,2,0,0,0,0,exonic,MAPK12,nonsynonymous SNV,rs2230823,0,0.986757,C,1.0,D,0.9,D,0.6867,U,0.075359,N,3.17,22,50694529,50694529,G,A
B 22,50696678_1,rs34422484,G,A,G,7444.34,PASS,AC=57;AF=0.038;AN=1488;BaseQRankSum=19.808;DB;DP=7340;Dels=0.00;FS=4.423;HaplotypeScore=0.9100;InbreedingCoeff=0.0409;MLEAC=53;MLEAF=0.036;MQ=60.00;MQ0=0;MQRankSum=-0.703;POSITIVE_TRAIN_SITE;QD=12.53;ReadPosRankSum=1.973;VQSLOD=2.50;culprit=DP;AA=G,0.0333839,0.0273312,0.03125,0.0387931,0.0306122,0.0535714,0.0454545,0.0344828,44,17,10,27,3,6,2,2,exonic,MAPK12,nonsynonymous SNV,rs34422484,0.04,0.982381,C,0.94,T,0.0,B,0.945872,N,0.506249,D,2.11,22,50696678,50696678,G,A


zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50691870 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'

zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50693697 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'
346-0/1:6,10:16:99:271,0,158    398-0/1:10,1:11:1:1,0,294
zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50694078 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'
301-0/1:9,1:10:4:4,0,238        316-0/1:10,1:11:2:2,0,278       379-0/1:5,5:10:99:147,0,116
zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50694084 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'
220-0/1:2,12:15:20:373,0,20     234-0/1:12,9:20:99:241,0,282    435-0/1:6,6:12:99:176,0,150
zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50694251 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'
428-0/1:5,1:6:4:4,0,156 493-0/1:10,2:12:32:32,0,302     495-0/1:9,1:10:4:4,0,280        571-0/1:18,20:37:99:499,0,506
zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50694280 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'
569-0/1:18,20:37:99:530,0,453
zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50694529 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'
440-0/1:4,1:5:21:21,0,105       492-0/1:8,1:9:10:10,0,223       554-0/1:11,9:19:99:262,0,261
zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50696678 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'
33-0/1:1,2:3:26:64,0,26 54-0/1:10,1:11:2:2,0,281        75-0/1:10,1:11:3:3,0,293        82-0/1:4,3:7:79:79,0,115        94-0/1:10,1:11:2:2,0,301        100-0/1:7,8:15:99:241,0,188     105-0/1:10,21:30:99:665,0,220   117-0/1:8,3:11:74:74,0,241      188-0/1:3,1:4:20:20,0,91        216-0/1:1,1:2:27:29,0,27        220-0/1:3,6:9:66:183,0,66  234-0/1:5,7:12:99:210,0,137     235-0/1:9,8:17:99:227,0,235     260-0/1:1,3:4:23:93,0,23        265-0/1:8,10:18:99:294,0,208    304-1/1:0,1:1:3:35,3,0  313-0/1:1,4:5:17:124,0,17       318-0/1:3,1:4:23:23,0,82        323-0/1:2,4:6:51:117,0,51       349-0/1:4,4:8:99:120,0,101      354-0/1:3,3:6:74:88,0,74   361-0/1:10,5:15:99:127,0,263    388-0/1:3,2:5:58:58,0,79        393-1/1:0,5:5:15:181,15,0       399-1/1:0,11:11:33:390,33,0     414-0/1:6,3:9:80:80,0,148       417-0/1:3,4:7:73:122,0,73       423-0/1:8,5:13:99:132,0,203     430-0/1:5,2:7:49:49,0,147       435-0/1:4,3:7:87:87,0,106       448-0/1:4,6:10:99:183,0,100        451-1/1:0,2:2:6:67,6,0  460-0/1:2,3:5:49:87,0,49        514-0/1:1,2:3:24:61,0,24        515-0/1:4,1:5:19:19,0,113       544-0/1:5,5:10:99:139,0,127     554-0/1:3,5:8:70:153,0,70       568-0/1:5,23:27:50:713,0,50     570-0/1:9,8:17:99:223,0,255     589-0/1:7,5:12:99:131,0,189     590-0/1:7,4:11:99:106,0,190        592-0/1:14,13:26:99:358,0,345   594-0/1:21,20:40:99:570,0,518   599-0/1:3,6:9:64:174,0,64       616-0/1:7,6:13:99:171,0,168     622-0/1:8,3:11:64:64,0,215      631-0/1:7,5:12:99:141,0,175     632-0/1:15,6:20:99:113,0,410    633-0/1:10,10:20:99:286,0,247   662-0/1:3,2:5:47:47,0,69        687-0/1:1,3:4:20:92,0,20   691-0/1:6,2:8:48:48,0,175       738-0/1:22,15:36:99:422,0,551


R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs3.R /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_106-124/POOL106_01RL/POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.OverlapPlot.formatted /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.0kbWindow.plotReady /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs3.0kbWindow

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.FIDIIDs --make-bed --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.FIDs > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.pheno
#Expecting 619 individuals based on plink .fam output and FIDs file -- get 619 in '...justWhite.pheno' output file

cp -p VLookUp.Python.vs1.py VLookUp.Python.vs1.vs2.py 

#Later did mv VLookUp.Python.vs1.vs2.py DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs1.py 

python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs1.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.txt.gz --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { print $8, "\t", $2 } }' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.SetIDFile.Exonic

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $8, "\t", $2 } } } ' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($11 == "P") || ($11 == "D") ) { print $8, "\t", $2 } } } } ' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_PD

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $8, "\t", $2 } } } } ' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_D

#Hardcoding SKAT analysis script because the tools require 'file location' as input so it seems like I'll need multiple file sets to run the code on multiple SetID sets for example

R --vanilla --slave --args < DataProcessing.Pipeline.Analysis.Skat.vs1.R

#EPACTS
#Using same 'file creation setup' instead of doing these commands within the Python script to help maintain consistency with how I'm partitioning variants into exonic, nonsynonymous, Polyphen2 categories, etc...these partitions are based on the commands I use in my CalcMeandAF.vs1.R scripts 

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { print $0 } }' | python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $0 } } } ' | python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($11 == "P") || ($11 == "D") ) { print $0 } } } } ' | python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } } ' | python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D

zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "23") { $1 = "X"} print $0 } ' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip

/home/shared/software/tabix-0.2.5/bgzip /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip

/home/shared/software/tabix-0.2.5/tabix -pvcf -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.pheno | awk '{ print $1, "\t", $2, "\t0\t0\t0\t", $3, "\t", $4, "\t", $5, "\t", $6 } ' | sed 's/-9/NA/g' | perl -lane 'print join("\t", @F); ' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped

#Add line #FAM_ID IND_ID FAT_ID MOT_ID SEX HIVPROG HIVPROGHE AIDS AIDSEXTR to /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.pheno

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic \
--ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test b.collapse \
--run 2 \
--out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.EPACTS.collapse

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD \
--ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test b.collapse \
--max-maf .9 \
-min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD.EPACTS.collapse

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic \
--ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test b.madsen \
--max-maf .9 \
-min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.EPACTS.madsen

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous \
--ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test b.madsen \
--max-maf .9 \
-min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.EPACTS.madsen

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD \
--ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test b.madsen \
--max-maf .9 \
-min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD.EPACTS.madsen

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D \
--ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test b.madsen \
--max-maf .9 \
-min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D.EPACTS.madsen

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous \
--ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test b.wcnt \
--max-maf .9 \
--min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.EPACTS.wcnt

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD \
--ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test b.wcnt \
--max-maf .9 \
-min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD.EPACTS.wcnt


epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic \
--ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test skat --skat-o --skat-adjust \
--max-maf .9 \
--min-callrate .01 \ 
--run 2 \
--out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.EPACTS.Skat0

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous \
--ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test skat --skat-o --skat-adjust \
--max-maf .9 \
-min-maf .000000000000001 \ 
--min-callrate .01 \ 
--run 2 \
--out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.EPACTS.Skat0

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD \
--ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test skat --skat-o --skat-adjust \
--max-maf .9 \
-min-maf .000000000000001 \ 
--min-callrate .01 \ 
--run 2 \
--out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D \
--ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test skat --skat-o --skat-adjust \
--max-maf .9 \
-min-maf .000000000000001 \ 
--min-callrate .01 \ 
--run 2 \
--out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD \
--ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROGHE \
--test b.madsen \
--max-maf .9 \
-min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Pheno2.Exonic.Nonsynonymous.PP2_PD.EPACTS.madsen &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D \
--ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROGHE \
--test b.madsen \
--max-maf .9 \
-min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Pheno2.Exonic.Nonsynonymous.PP2_D.EPACTS.madsen &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD \
--ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno AIDS \
--test b.madsen \
--max-maf .9 \
-min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Pheno3.Exonic.Nonsynonymous.PP2_PD.EPACTS.madsen &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D \
--ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno AIDS \
--test b.madsen \
--max-maf .9 \
-min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Pheno3.Exonic.Nonsynonymous.PP2_D.EPACTS.madsen &


epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD \
--ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROGHE \
--test skat --skat-o --skat-adjust \
--max-maf .9 \
-min-maf .000000000000001 \ 
--min-callrate .01 \ 
--run 2 \
--out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Pheno2.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0 &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D \
--ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROGHE \
--test skat --skat-o --skat-adjust \
--max-maf .9 \
-min-maf .000000000000001 \ 
--min-callrate .01 \ 
--run 2 \
--out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Pheno2.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0 &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD \
--ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno AIDS \
--test skat --skat-o --skat-adjust \
--max-maf .9 \
-min-maf .000000000000001 \ 
--min-callrate .01 \ 
--run 2 \
--out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Pheno3.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0 &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D \
--ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno AIDS \
--test skat --skat-o --skat-adjust \
--max-maf .9 \
-min-maf .000000000000001 \ 
--min-callrate .01 \ 
--run 2 \
--out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Pheno3.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0 &

#KIAA1429
zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "8") { print $0 } } ' | grep 95501065 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'

zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.txt.gz | grep KIAA1429 | awk -F, '{ if (($28 == "D") || ($28 == "P")) { print $2, "\t", $28 } }' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SpecificGenes/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.Exonic.KIAA1429.PP2_PD.Locations

bash DataProcessing.Pipeline.Analysis.IdentifyIndividualsContainingVariants.vs1.sh 8 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SpecificGenes/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.Exonic.KIAA1429.PP2_PD.Locations

95501065(D): 31-0/1:9,1:10:6:6,0,264    350-0/1:6,12:18:99:365,0,129    411-0/1:5,4:9:99:108,0,122      649-0/1:7,1:8:11:11,0,197
95508103(D): 102-0/1:11,7:18:99:175,0,337
95518785(D): 362-0/1:14,18:31:99:469,0,376
95522750(D): 715-0/1:11,12:22:99:296,0,326
95522824(D): 163-0/1:5,1:6:16:16,0,152  686-0/1:13,9:21:99:248,0,354
95530099(D): 107-0/1:5,1:6:17:17,0,160  192-0/1:6,1:7:3:3,0,178 243-0/1:13,13:26:99:377,0,379   
95531388(P): 721-0/1:3,4:7:69:110,0,69  
95531525(P): 473-0/1:10,8:18:99:221,0,273       705-0/1:9,1:10:4:4,0,278
95538947(P): 96-0/1:21,26:45:99:638,0,623       759-0/1:7,1:8:7:7,0,218 
95541400(P): 625-0/1:9,5:14:99:125,0,259        719-0/1:8,1:9:5:5,0,248 
95549374(P): 153-0/1:8,1:9:8:8,0,249    296-0/1:5,5:10:99:103,0,141     297-0/1:7,1:8:2:2,0,214 421-0/1:9,1:10:4:4,0,285
95550513(P): 92-0/1:8,5:13:99:131,0,242 296-0/1:4,1:5:9:9,0,95  683-0/1:6,1:7:13:13,0,175


vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.FID --counts --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg.FID --counts --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos.FID --counts --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg.FID --counts --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid.FID --counts --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow.FID --counts --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid.FID --counts --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow.FID --counts --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow &

paste AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow.frq.count | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.AllPhenos.frq.count.gz

gzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow.frq.count 

#gunzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.frq.count.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg.frq.count.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg.frq.count.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid.frq.count.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos.frq.count.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow.frq.count.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid.frq.count.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow.frq.count.gz

paste AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq.count | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.AllPhenos.frq.AndCount.gz

gzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq.count

#gunzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq.count

#20140219
#(for AWK) 26 exonic/intronic, 27 gene, 28 synon/non-synon, 29 rsID from annovari/'dbsnp129' (vs from GATK variant caller at beginning of array), 30-40 prediction stuff (30 is AVSIFT, then predic prog # / predic prog Category for PhyloP, SIFT, Polyphen2, LRT, MT)
#Now including allele counts as well as allele frequencies
#Also now outputting multiple lines for the same variant if it covers multiple genes. This is being recorded by adding a '_1', '_2', etc... to te end of the basepair position at array entry 2 for each line to designate the # of times the variant is present in the dataset to represent different loci for that same variant

python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.AllPhenos.frq.AndCount.gz --file3 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 29; $i <= 39; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.vs2.txt.gz

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | grep exonic | awk -F, '{ print $3 }' | sort | uniq > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.AnnotatingVariantsCoveringMultTranscripts
#The manually edited /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.AnnotatingVariantsCoveringMultTranscripts comparing the output to /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv to see variants that have multiple gene-hits (e.g. overlap multiple genes).
#Decided to output multiple lines for variants that cover multiple transcripts/genes, e.g. have 3 lines for the same variant if that variant hits 3 genes at once
#So stopped this train of thought here and didn't finish annotating the above filename


#Used vs1 of DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20140101_POP_genomics_sample.status_list_9-16-13.output instead of vs2 earlier. Redoing these and downstream steps with the proper vs2 of this file
python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140101_POP_genomics_sample.status_list_9-16-13.output --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt1

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt2 > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race

sed -i 's/American_Indian_or_Alaskan_Native/AIndian_or_Alaskan_Native/g' AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race

scp /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/Eigenstrat/.

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI -x -o AllPools.AllPoolsMerged.ChrAll.GATK.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.1vs2.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI -x -o AllPools.AllPoolsMerged.ChrAll.GATK.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.3vs4.xtxt

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race | awk '{ if ((($2 >= 0) && ($2 <= .2)) && (($3 >= -.01) && ($3 <= .03))) { print $0 } } ' | grep White_non-Hispanic > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed

ln -s AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed AllPools.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i  AllPools.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed -c 1:2 -p White_non-Hispanic -x -o  AllPools.AllPoolsMerged.ChrAll.GATK.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.xtxt

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed | perl -lane 'my @vals1 = split(/:/, $F[0]); print $vals1[0];' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs

scp mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/.

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar | awk '{ print $1, "\t", $2 } ' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar.FIDIIDs

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar.FIDIIDs | awk '{ print $1 }' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.QCed.FIDs

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 1,2,3,4,5,6,7,8 --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar.FIDIIDs --logistic --allow-no-sex --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.5PCs

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar.FIDIIDs --logistic --allow-no-sex --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.5PCs

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar.FIDIIDs --maf .01 --logistic --allow-no-sex --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.5PCs.maf01

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 1,2,3,4,5,6,7,8,9,10,11,12,13 --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar.FIDIIDs --logistic --allow-no-sex --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.10PCs

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 2 --covar /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar.FIDIIDs --maf .01 --logistic --allow-no-sex --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.5PCs.maf01.pheno2

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 3 --covar /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar.FIDIIDs --maf .01 --logistic --allow-no-sex --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.5PCs.maf01.pheno3

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 4 --covar /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar.FIDIIDs --maf .01 --logistic --allow-no-sex --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.5PCs.maf01.pheno4

!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.5PCs.assoc.logistic Pheno1_WhiteOnly_Logistic_5PCs /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.5PCs.assoc.logsitic.QQplot.jpeg &

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.5PCs.maf01.assoc.logistic Pheno1_WhiteOnly_Logistic_maf01_5PCs /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.5PCs.maf01.assoc.logsitic.QQplot.jpeg &

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.10PCs.assoc.logistic Pheno1_WhiteOnly_Logistic_10PCs /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.10PCs.assoc.logsitic.QQplot.jpeg &

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ print $1 }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.FIDs
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($3 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.FIDs
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($4 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.FIDs
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($3 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.FIDs
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.FIDs
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.FIDs
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($6 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.FIDs
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($6 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.FIDs

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.FIDs --freq --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.FIDs --freq --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.FIDs --freq --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.FIDs --freq --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.FIDs --freq --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.FIDs --freq --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.FIDs --freq --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.FIDs --freq --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.FIDs --counts --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.FIDs --counts --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.FIDs --counts --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.FIDs --counts --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.FIDs --counts --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.FIDs --counts --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.FIDs --counts --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.FIDs --counts --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow &

paste AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq.count | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.AllPhenos.frq.AndCount.gz

gzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq.count

#gunzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq.count

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.FIDs --counts --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.FIDs --counts --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.FIDs --counts --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.FIDs --counts --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow &

gzip /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.frq.count /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.frq.count /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.frq.count /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.frq.count 

python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.AllPhenos.frq.AndCount.gz --file3 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 29; $i <= 39; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.vs2.txt.gz

zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.vs2.txt.gz | awk -F, '{ if ($26 =="exonic") { print $0 } }' | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.vs2.txt.gz

python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs3.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.vs2.txt.gz > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs3.ExonicVariants.All.output

#Edited code
python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs3.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.vs2.txt.gz > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs3.ExonicVariants.Nonsynonymous.output

#Edited code
python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs3.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.vs2.txt.gz > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs3.ExonicVariants.PP2_DP.output

#Edited code
python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs3.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.vs2.txt.gz > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs3.ExonicVariants.PP2_D.output

R --vanilla --slave --args < DataProcessing.Pipeline.Analysis.VariantDensity.FollowUpCalculations.vs1.R DataProcessing.Pipeline.Analysis.VariantDensity.vs3.ExonicVariants.PP2_DP.output | awk '{ if (($4 != "NA") && ($2 >= 3)) { print $0 } } ' | sort -rg -k 3,3
R --vanilla --slave --args < DataProcessing.Pipeline.Analysis.VariantDensity.FollowUpCalculations.vs1.R DataProcessing.Pipeline.Analysis.VariantDensity.vs3.ExonicVariants.PP2_D.output | awk '{ if (($4 != "NA") && ($2 >= 3)) { print $0 } } ' | sort -rg -k 3,3

#MAPK12 specifically looking at here
zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50691870 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'

zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50693697 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'
346-0/1:6,10:16:99:271,0,158    398-0/1:10,1:11:1:1,0,294

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar.FIDIIDs --make-bed --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed

python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs2.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.vs2.txt.gz --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { print $8, "\t", $2 } }' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.SetIDFile.Exonic

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $8, "\t", $2 } } } ' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($11 == "P") || ($11 == "D") ) { print $8, "\t", $2 } } } } ' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_PD

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $8, "\t", $2 } } } } ' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_D

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { print $0 } }' | python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $0 } } } ' | python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($11 == "P") || ($11 == "D") ) { print $0 } } } } ' | python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } } ' | python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.QCed.FIDs > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.pheno

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.pheno | awk '{ print $1, "\t", $2, "\t0\t0\t0\t", $3, "\t", $4, "\t", $5, "\t", $6 } ' | sed 's/-9/NA/g' | perl -lane 'print join("\t", @F); ' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar | awk '{ print $4, "\t", $6, "\t", $7 "\t", $8, "\t", $9, "\t", $10, "\t", $11,  "\t", $12, "\t", $13, "\t", $14,  "\t", $15 }' | paste /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped - > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped.vs2

mv /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped.vs2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped

#Add line #FAM_ID IND_ID FAT_ID MOT_ID SEX HIVPROG HIVPROGHE AIDS AIDSEXTR RMID PC1 PC2 PC3 PC4 PC5 PC6 PC7 PC8 PC9 PC10 to /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \

!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!EPACTS Stuff

#EPACTS Variables Running Through
#Pheno: HIVPROG HIVPROGHE AIDS AIDSEXTR
#Filetypes: Exonic Exonic.Nonsynonymous Exonic.Nonsynonymous.PP2_PD Exonic.Nonsynonymous.PP2_D
#Test b.collapse b.madsen b.wcnt  skat --skat-o --skat-adjust

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --test skat --skat-o --skat-adjust --run 1 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.Exonic.EPACTS.Skat0 &

#epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --test skat --skat-o --skat-adjust --max-maf .9 --min-maf .000000000000001 --min-callrate .01 --run 1 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.Exonic.EPACTS.Skat0.vs2 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --test skat --skat-o --skat-adjust --min-callrate .01 --run 1 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.Exonic.EPACTS.Skat0.vs3 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --test skat --skat-o --skat-adjust --max-maf .9 --run 1 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.Exonic.EPACTS.Skat0.vs4 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --test skat --skat-o --skat-adjust --min-maf .000000000000001 --run 1 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.Exonic.EPACTS.Skat0.vs5 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --test skat --skat-o --skat-adjust --max-maf .5 --run 1 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.Exonic.EPACTS.Skat0.vs6 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --test skat --skat-o --skat-adjust --max-maf 1 --run 1 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.Exonic.EPACTS.Skat0.vs7 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --test skat --skat-o --skat-adjust --max-maf .05 --run 1 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.Exonic.EPACTS.Skat0.vs8 &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --test b.collapse --run 1 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.Exonic.EPACTS.collapse &

#epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --test b.collapse --max-maf .9 --min-maf .000000000000001 --min-callrate .01 --run 1 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.Exonic.EPACTS.collapse.vs2 &


#Note -- not really running the code, opening up an R console and copying/pasting the commands since I'm moreso playing around with things and testing them out
R --vanilla --slave --args < DataProcessing.Pipeline.Analysis.Skat.vs3.R
34: 1 SNPs with either high missing rates or no-variation are excluded!
35: The missing genotype rate is 0.004442. Imputation is applied.
36: 5 SNPs with either high missing rates or no-variation are excluded!
37: The missing genotype rate is 0.006094. Imputation is applied.
38: 5 SNPs with either high missing rates or no-variation are excluded!
39: The missing genotype rate is 0.006094. Imputation is applied.
40: 1 SNPs with either high missing rates or no-variation are excluded!
41: The missing genotype rate is 0.002703. Imputation is applied.
42: 3 SNPs with either high missing rates or no-variation are excluded!
43: The missing genotype rate is 0.003392. Imputation is applied.
44: The missing genotype rate is 0.003941. Imputation is applied.
45: The missing genotype rate is 0.003003. Imputation is applied.

30: The missing genotype rate is 0.002336. Imputation is applied.
31: 4 SNPs with either high missing rates or no-variation are excluded!
32: The missing genotype rate is 0.002336. Imputation is applied.
33: The missing genotype rate is 0.001802. Imputation is applied.
34: The missing genotype rate is 0.002252. Imputation is applied.
35: The missing genotype rate is 0.000751. Imputation is applied.
36: 1 SNPs with either high missing rates or no-variation are excluded!
37: The missing genotype rate is 0.005005. Imputation is applied.
38: The missing genotype rate is 0.004880. Imputation is applied.
39: 1 SNPs with either high missing rates or no-variation are excluded!
40: The missing genotype rate is 0.019520. Imputation is applied.
41: The missing genotype rate is 0.004692. Imputation is applied.
42: The missing genotype rate is 0.005255. Imputation is applied.
43: 1 SNPs with either high missing rates or no-variation are excluded!
44: The missing genotype rate is 0.003003. Imputation is applied.
45: 1 SNPs with either high missing rates or no-variation are excluded!
46: The missing genotype rate is 0.001802. Imputation is applied.
47: The missing genotype rate is 0.003003. Imputation is applied.
48: ALL SNPs have either high missing rates or no-variation. P-value=1
49: The missing genotype rate is 0.003003. Imputation is applied.
50: Only one SNP in the SNP set!

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0 &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0 &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0 &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.EPACTS.Skat0 &
make -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.EPACTS.Skat0.Makefile -j 6
Note: Get odd error when trying to run skat-o w/ adjustment on the above combination of parameters, e.g. pheno4 + exonic set. Does not occur elsewhere. Can run skat-o fine manually use R code and SKAT directly, so doing that for this set of results. However, none of the results for Pheno4 are coming out well due to the low sample numbers so it's a moot point.  
Error in out[i, ] <- c(nona.collapse, maf.collapse, nrow(G), nrow(genos),  :
  number of items to replace is not a multiple of replacement length
  Calls: source -> withVisible -> eval -> eval
  In addition: There were 14 warnings (use warnings() to see them)
  Execution halted
  make: *** [/home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.EPACTS.Skat0.0.epacts] Error 1
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0 &


epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.EPACTS.Collapse &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Collapse &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Collapse &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Collapse &

EPACTS Group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.EPACTS.Collapse &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Collapse &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Collapse &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Collapse &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.EPACTS.Collapse &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Collapse &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Collapse &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Collapse &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.EPACTS.Collapse &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Collapse &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Collapse &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Collapse &


epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.EPACTS.Madsen &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Madsen &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Madsen &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Madsen &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.EPACTS.Madsen &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Madsen &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Madsen &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Madsen &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.EPACTS.Madsen &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Madsen &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Madsen &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Madsen &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.EPACTS.Madsen &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Madsen &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Madsen &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Madsen &


epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.EPACTS.wcnt &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.EPACTS.wcnt &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.wcnt &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.wcnt &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.EPACTS.wcnt &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.EPACTS.wcnt &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.wcnt &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.wcnt &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.EPACTS.wcnt &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.EPACTS.wcnt &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.wcnt &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.wcnt &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.EPACTS.wcnt &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.EPACTS.wcnt &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.wcnt &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.wcnt &




python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs4.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.vs2.txt.gz > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs4.ExonicVariants.All.output

#Edited code
python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs4.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.vs2.txt.gz > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs4.ExonicVariants.Nonsynonymous.output

#Edited code
python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs4.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.vs2.txt.gz > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs4.ExonicVariants.PP2_DP.output

#Edited code
python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs4.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.vs2.txt.gz > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs4.ExonicVariants.PP2_D.output

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.FollowUpCalculations.vs2.R /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs4.ExonicVariants.PP2_DP.output | awk '{ if (($5 != "NA") && ($2 >= 3)) { print $0 } } ' | sort -rg -k 4,4
R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.FollowUpCalculations.vs2.R /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs4.ExonicVariants.PP2_D.output | awk '{ if (($4 != "NA") && ($2 >= 3)) { print $0 } } ' | sort -rg -k 4,4

zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | head -n 124 >  /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.header

zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | tail -n +125 | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK//home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.noHeader.gz

perl /home/michaelt/Scripts/Perl/ListSubset.pl --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.noHeader.gz --subset 1000 --output /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.random1000.vcf

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.random1000.vcf | sort -g -k 1,1 -k 2,2 | cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.header - | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.random1000.vcf.gz

#NOTE: Manually editted top _ X chromosome variants and put them at the bottom

scp AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.random1000.vcf.gz mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/.

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.pheno | awk '{ print $3, "\t", $4, "\t", $5, "\t", $6 }' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.pheno.noRowHeader

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.pheno | awk '{ print $1, "\t", $2 }' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.pheno.RowHeader

bash /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.AlleleCountComparison.Simulations.Launcher.vs1.sh 1 100

PasteCommandjustSeroNeg=`perl -e 'print "paste - "; for (my $i = 1; $i <= 100; $i++) { print "/home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVs", $i, ".CtDwn.frq.count "; }'`
PasteCommandjustSeroPos=`perl -e 'print "paste - "; for (my $i = 1; $i <= 100; $i++) { print "/home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVs", $i, ".CtDwn.frq.count "; }'`
PasteCommandjustRapidVRapid=`perl -e 'print "paste - "; for (my $i = 1; $i <= 100; $i++) { print "/home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVs", $i, ".CtDwn.frq.count "; }'`
PasteCommandjustSlowVSlow=`perl -e 'print "paste - "; for (my $i = 1; $i <= 100; $i++) { print "/home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVs", $i, ".CtDwn.frq.count "; }'`

#zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.frq.count.gz | tail -n +2 | paste - /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVs*.CtDwn.frq.count | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVsAll.frq.count.gz
#zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.frq.count.gz | tail -n +2 | paste - /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVs*.CtDwn.frq.count | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVsAll.frq.count.gz
#zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.frq.count.gz | tail -n +2 | paste - /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVs*.CtDwn.frq.count | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.gz
#zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.frq.count.gz | tail -n +2 | paste - /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVs*.CtDwn.frq.count | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVsAll.frq.count.gz
zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.frq.count.gz | tail -n +2 | $PasteCommandjustSeroNeg | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVsAll.frq.count.gz
zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.frq.count.gz | tail -n +2 | $PasteCommandjustSeroPos | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVsAll.frq.count.gz
zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.frq.count.gz | tail -n +2 | $PasteCommandjustRapidVRapid | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.gz
zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.frq.count.gz | tail -n +2 | $PasteCommandjustSlowVSlow | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVsAll.frq.count.gz

#Tri-Allelic and Quad-Allelic SNPs. They are eventually dropped out when including annotation information in the original, main source files since the program doesn't accept non-bi-allelic SNP setups. Could potentially divide these SNPs into multiple entries for the same variant but only including each bi-allelic state amongst the original tri- or quad-alleles, but this is currently decided as being too much work for a supposedly very low return on information gained (387 SNPs << ~60k which is around the full On-Target, 95% geno list).
[  michaelt@node1  /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK]$zcat SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.gz | perl -lane 'if ($#F == 206) { print join("\t", @F); } ' | wc
    387   80109  377798
[  michaelt@node1  /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK]$zcat SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.gz | perl -lane 'if ($#F == 207) { print join("\t", @F); } ' | wc
      3     624    2798

[  michaelt@node1  /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK]$zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ print $4, "\t", $5 }' | sort | uniq -c | grep -E 'A|G|T|C' | sort -rg -k 1,1
    15295 C        T
    15107 G        A
    9604 T        C
    8993 A        G
    3308 G        C
    3105 C        G
    2628 G        T
    2572 C        A
    2164 T        G
    2069 A        C
    1635 A        T
    1577 T        A
    60 C        G,T
    59 G        A,C
    58 G        A,T
    56 C        A,T
    37 T        C,G
    26 T        A,C
    22 G        C,T
    20 A        G,T
    20 A        C,G
    12 C        A,G
    10 A        C,T
    7 T        A,G
    3 G        A,C,T
    2 phone_home=STANDARD      gatk_key=null
    1 each     ALT
    1 REF      ALT
    1 Containing       Spanning

cp -p /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.py /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.ForSimulation.AlleleCountAnalysis.vs1.py

python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.ForSimulation.AlleleCountAnalysis.vs1.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVsAll.frq.count.gz --file3 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 13; $i <= 24; $i++) { if (($F[$i] eq "NA") || ($F[$i] eq "")) { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz
python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.ForSimulation.AlleleCountAnalysis.vs1.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVsAll.frq.count.gz --file3 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 13; $i <= 24; $i++) { if (($F[$i] eq "NA") || ($F[$i] eq "")) { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz
python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.ForSimulation.AlleleCountAnalysis.vs1.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.gz --file3 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 13; $i <= 24; $i++) { if (($F[$i] eq "NA") || ($F[$i] eq "")) { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz
python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.ForSimulation.AlleleCountAnalysis.vs1.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVsAll.frq.count.gz --file3 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 13; $i <= 24; $i++) { if (($F[$i] eq "NA") || ($F[$i] eq "")) { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.AlleleCountComparison.AnalyzeResults.vs1.R /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Analysis/AlleleCounts/DataProcessing.Pipeline.Analysis.AlleleCountComparison.AnalyzeResults.vs1.output


bash /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.AlleleCountComparison.Simulations.Launcher.vs1.sh 101 1000
bash /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.AlleleCountComparison.Simulations.Launcher.vs1.sh 850 1000

PasteCommandjustSeroNeg=`perl -e 'print "paste - "; for (my $i = 1; $i <= 1000; $i++) { print "/home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVs", $i, ".CtDwn.frq.count "; }'`
PasteCommandjustSeroPos=`perl -e 'print "paste - "; for (my $i = 1; $i <= 1000; $i++) { print "/home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVs", $i, ".CtDwn.frq.count "; }'`
PasteCommandjustRapidVRapid=`perl -e 'print "paste - "; for (my $i = 1; $i <= 1000; $i++) { print "/home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVs", $i, ".CtDwn.frq.count "; }'`
PasteCommandjustSlowVSlow=`perl -e 'print "paste - "; for (my $i = 1; $i <= 1000; $i++) { print "/home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVs", $i, ".CtDwn.frq.count "; }'`

zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.frq.count.gz | tail -n +2 | $PasteCommandjustSeroNeg | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVsAll.frq.count.gz
zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.frq.count.gz | tail -n +2 | $PasteCommandjustSeroPos | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVsAll.frq.count.gz
zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.frq.count.gz | tail -n +2 | $PasteCommandjustRapidVRapid | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.gz
zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.frq.count.gz | tail -n +2 | $PasteCommandjustSlowVSlow | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVsAll.frq.count.gz

python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.ForSimulation.AlleleCountAnalysis.vs1.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVsAll.frq.count.gz --file3 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 13; $i <= 24; $i++) { if (($F[$i] eq "NA") || ($F[$i] eq "")) { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz
python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.ForSimulation.AlleleCountAnalysis.vs1.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVsAll.frq.count.gz --file3 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 13; $i <= 24; $i++) { if (($F[$i] eq "NA") || ($F[$i] eq "")) { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz
python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.ForSimulation.AlleleCountAnalysis.vs1.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.gz --file3 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 13; $i <= 24; $i++) { if (($F[$i] eq "NA") || ($F[$i] eq "")) { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz
python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.ForSimulation.AlleleCountAnalysis.vs1.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVsAll.frq.count.gz --file3 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 13; $i <= 24; $i++) { if (($F[$i] eq "NA") || ($F[$i] eq "")) { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.AlleleCountComparison.AnalyzeResults.vs2.R /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Analysis/AlleleCounts/DataProcessing.Pipeline.Analysis.AlleleCountComparison.AnalyzeResults.vs2.output

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Analysis/AlleleCounts/DataProcessing.Pipeline.Analysis.AlleleCountComparison.AnalyzeResults.vs2.output | awk '{ if (($13 != "NA") && ($13 != 1)) { print $0 } } ' | sort -rg -k 14,14 -k 13,13

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($10 == "D") { print $0 } } } } ' | python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $0 } } } } ' | python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0 &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Collapse &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Collapse &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Collapse &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Collapse &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Collapse &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Collapse &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Collapse &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Collapse &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Madsen &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Madsen &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Madsen &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Madsen &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Madsen &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Madsen &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Madsen &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Madsen &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.wcnt &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.wcnt &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.wcnt &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.wcnt &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.wcnt &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.wcnt &
#Two warning messages for above run of b.wcnt (Pheno3, LRT_D)
#Warning messages:
#1: glm.fit: fitted probabilities numerically 0 or 1 occurred
#2: glm.fit: fitted probabilities numerically 0 or 1 occurred
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.wcnt &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.wcnt &


#cat <( paste <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <( cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
cat  <( paste <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.All.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.All.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.All.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.All.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
> /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/All.GATK.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.justWhite.QCed.Pheno1.RMID5PCs.All.EPACTS.All.Summary.txt

cat <( paste <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.All.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.All.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.All.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.All.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
> /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/All.GATK.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.justWhite.QCed.Pheno2.RMID5PCs.All.EPACTS.All.Summary.txt

cat <( paste <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.All.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.All.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.All.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.All.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
> /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/All.GATK.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.justWhite.QCed.Pheno3.RMID5PCs.All.EPACTS.All.Summary.txt

cat <( paste <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.All.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.All.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.All.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.All.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
> /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/All.GATK.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.justWhite.QCed.Pheno4.RMID5PCs.All.EPACTS.All.Summary.txt

#Accidentally ran in /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/. and converted some files that were not supposed to be. Think I fixed it
ls -lrt | awk '{ print $9 }' | perl -lane 'if ($F[0] =~ m/(.*)(\.wAA)(\.White.*)/) { my $val1 = $1 . $2 . $3; my $val2 = $1 . $2 . ".DropOffTargetVariants.1kbWindow.95geno.recode" . $3; system("mv $val1 $val2");}'
#Fix ls -lrt | grep WhiteOnly | grep -v WhiteOnly.QCed | awk '{ print $9 } ' | perl -lane 'if ($F[0] =~ m/(.*)(\.wAA)(.DropOffTargetVariants.1kbWindow.95geno.recode)(\.White.*)/) { my $val1 = $1 . $2 . $3 . $4; my $val2 = $1 . $2 . $4; system("mv $val1 $val2");}'

cp -p DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.py DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.FullVCFwAnnovar.vs1.py

python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.FullVCFwAnnovar.vs1.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.AllGenesPerVariant.vcf.gz

#Edited code in /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.FullVCFwAnnovar.vs1.py
python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.FullVCFwAnnovar.vs1.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.vcf.gz

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py DataPrcoessing.Pipeline.Utility.GetVCFQCStats.vs1.py

python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.AllGenesPerVariant.vcf.gz > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.AllGenesPerVariant.GetVCFQCStats.vs1.output

python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.vcf.gz > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output

zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.vcf.gz | grep exonic | grep -v ncRNA | python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py --file1 - > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.Exonic.GetVCFQCStats.vs1.output

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py DataPrcoessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs1.py

python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs1.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.vcf.gz > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.FIDs > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justSeroNeg 
cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.FIDs > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justSeroPos 
cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.FIDs > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justSeroHighlyExposedNeg 
cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.FIDs > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justRapidVRapid 
cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.FIDs > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justSlowVSlow 

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
[1] "MeanSingletons: 20.6626826029216 MeanDoubletons: 25.2881806108898 MeanCommon: 5033.1567065073 MeanRare: 504.302788844621 MeanHeterozy: 3457.57503320053 MeanCoverage: 16.6629860920849 sdCoverage: 6.17039709917999 Exonic: 342.65604249668 Intronic: 4139.33997343958 Nonsynonymous: 127.176626826029 Synonymous: 212.610889774236 PP2_D: 9.80610889774236 PP2_P: 11.6693227091633 Sift_D: 22.0823373173971 LRT_D: 21.9189907038513"


cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justSeroNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
[1] "MeanSingletons: 16.1948881789137 MeanDoubletons: 21.1086261980831 MeanCommon: 5038.1661341853 MeanRare: 435.638977635783 MeanHeterozy: 3385.56869009585 MeanCoverage: 15.8726865928358 sdCoverage: 5.50509121070413 Exonic: 340.05750798722 Intronic: 4090.77635782748 Nonsynonymous: 126.159744408946 Synonymous: 211.268370607029 PP2_D: 9.77635782747604 PP2_P: 11.3897763578275 Sift_D: 22.0095846645367 LRT_D: 21.6932907348243"

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justSeroHighlyExposedNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
[1] "MeanSingletons: 16.3291925465839 MeanDoubletons: 21.1987577639752 MeanCommon: 5043.50310559006 MeanRare: 436.596273291925 MeanHeterozy: 3384.70186335404 MeanCoverage: 15.7976001524024 sdCoverage: 4.99886275938732 Exonic: 339.596273291925 Intronic: 4096.49689440994 Nonsynonymous: 125.944099378882 Synonymous: 210.795031055901 PP2_D: 9.64596273291925 PP2_P: 11.3416149068323 Sift_D: 21.9316770186335 LRT_D: 21.5465838509317"

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
[1] "MeanSingletons: 16.9517045454545 MeanDoubletons: 19.8977272727273 MeanCommon: 5046.28125 MeanRare: 426.855113636364 MeanHeterozy: 3405.65340909091 MeanCoverage: 17.0944802710762 sdCoverage: 6.42723639615973 Exonic: 337.542613636364 Intronic: 4091.76420454545 Nonsynonymous: 125.235795454545 Synonymous: 209.323863636364 PP2_D: 9.69886363636364 PP2_P: 11.4403409090909 Sift_D: 21.3238636363636 LRT_D: 21.9403409090909"

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justRapidVRapid | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
[1] "MeanSingletons: 16.6734693877551 MeanDoubletons: 17.3673469387755 MeanCommon: 5088.89795918367 MeanRare: 403.591836734694 MeanHeterozy: 3419.16326530612 MeanCoverage: 22.2127566842631 sdCoverage: 9.1245435877065 Exonic: 338.673469387755 Intronic: 4112.5306122449 Nonsynonymous: 125.061224489796 Synonymous: 210.69387755102 PP2_D: 9.12244897959184 PP2_P: 11.469387755102 Sift_D: 21.0408163265306 LRT_D: 21.3469387755102"

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justSlowVSlow | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
[1] "MeanSingletons: 17.3571428571429 MeanDoubletons: 20.3571428571429 MeanCommon: 5025.71428571429 MeanRare: 426.107142857143 MeanHeterozy: 3386.69642857143 MeanCoverage: 16.4965406672807 sdCoverage: 5.79915172393244 Exonic: 333.803571428571 Intronic: 4087.39285714286 Nonsynonymous: 123.196428571429 Synonymous: 207.392857142857 PP2_D: 9.80357142857143 PP2_P: 11.375 Sift_D: 21.3571428571429 LRT_D: 21.8035714285714"

cp -p /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perGene.vs1.py

python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perGene.vs1.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.AllGenesPerVariant.vcf.gz > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perGene.vs1.output

python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs3.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.AllPhenos.frq.AndCount.gz --file3 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 29; $i <= 39; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.vs3.txt.gz

#Including SNPs that don't have clear ancestral/derived alleles. That was the major change to the vs2 to vs3 of the CombineVCFwFreqsAnnovarInfo.vs#.py script
python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs2.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.vs3.txt.gz --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { print $8, "\t", $2 } }' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.SetIDFile.Exonic

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $8, "\t", $2 } } } ' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($11 == "P") || ($11 == "D") ) { print $8, "\t", $2 } } } } ' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_PD

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $8, "\t", $2 } } } } ' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_D

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { print $0 } }' | python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $0 } } } ' | python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($11 == "P") || ($11 == "D") ) { print $0 } } } } ' | python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } } ' | python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($10 == "D") { print $0 } } } } ' | python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D
cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $0 } } } } ' | python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { print $0 } }' | python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $0 } } } ' | python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($11 == "P") || ($11 == "D") ) { print $0 } } } } ' | python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } } ' | python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_D

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($10 == "D") { print $0 } } } } ' | python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.Sift_D
cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $0 } } } } ' | python /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.LRT_D

#TimeptChck: 20140319

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.Exonic.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.Exonic.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.Exonic.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID.Exonic.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.Exonic.EPACTS.Skat0 &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0 &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0 &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0 &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0 &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0 &


cat DataProcessing.Pipeline.Utility.GetFastqQCStats.vs1.output > DataProcessing.Pipeline.Utility.GetFastqQCStats.vs1.output.forR
#Removed empty lines with /0.00000.0000 0.0000 and dd in vi
cat DataProcessing.Pipeline.Utility.GetFastqQCStats.vs1.output.forR | R -q -e "Data1 <- read.table(file('stdin'), header=F); sum(as.numeric(Data1[,4])); sum(as.numeric(Data1[,6]));"
> Data1 <- read.table(file('stdin'), header=F); sum(as.numeric(Data1[,4])); sum(as.numeric(Data1[,6]));
[1] 156661507032
[1] 258365498
cat DataProcessing.Pipeline.Utility.GetFastqQCStats.vs1.output | wc
   2896   83858  645395

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.EPACTS.Skat0.epacts  | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); summary(qvalue(Data1[,10]));"

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.EPACTS.Skat0.epacts  | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); summary(qvalue(Data1[,10]));"


cp -p /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs3.py /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs4.py

#Removing '(NM_...' from 'splicing' variants so that their gene IDs can be included. Also split up gene IDs that had ;symbol;smbol' which only affectes really 3 variants that had 'exonic;splcing'
python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs4.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.AllPhenos.frq.AndCount.gz --file3 /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 29; $i <= 39; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.vs4.txt.gz

python /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs2.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.vs4.txt.gz --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing")) { print $0 } }' | python /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { print $0 } }' | python /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $0 } } } ' | python /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($11 == "P") || ($11 == "D") ) { print $0 } } } } ' | python /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } } ' | python /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($10 == "D") { print $0 } } } } ' | python /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D
cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $0 } } } } ' | python /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing")) { print $0 } }' | python /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.All.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.All.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.All.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.All.EPACTS.Skat0 &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.All.EPACTS.Collapse &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.All.EPACTS.Collapse &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.All.EPACTS.Collapse &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.All.EPACTS.Collapse &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.All.EPACTS.Madsen &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.All.EPACTS.Madsen &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.All.EPACTS.Madsen &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.All.EPACTS.Madsen &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.All.EPACTS.wcnt &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.All.EPACTS.wcnt &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.All.EPACTS.wcnt &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.All.EPACTS.wcnt &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.All.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.All.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.All.EPACTS.Skat0 &


cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.All.EPACTS.Skat0.epacts  | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); summary(qvalue(Data1[,10]));"



plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --genome --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.IBD

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | awk '{ print $1, "\t", $1 }' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --genome --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.IBD

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --maf .05 --genome --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.IBD

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --maf .10 --genome --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf10.IBD

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --maf .10 --geno .01 --genome --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf10.geno01.IBD

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.bim | awk '{ if ($1 == "23") { print $2 } } ' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.bim.justX

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --exclude /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.bim.justX --maf .10 --geno .01 --genome --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.noX.maf10.geno01.IBD

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --maf .1 --indep-pairwise 50 5 0.1 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf10.r2_10

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --extract /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf10.r2_10.prune.in --genome --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf10.r2_10

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --hardy --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.FIDs --hardy --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.hwe | awk '{ if ($6 < .00001) { print $2 } }' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.hwe.1eNeg5.rsIDs

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --maf .05 --hwe .00001 --genome --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.hwe1eNeg5.IBD

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --maf .05 --hwe .0001 --genome --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.hwe1eNeg4.IBD

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --maf .05 --hwe .00001 --relatedness --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.maf05.hwe1Neg5

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --relatedness2 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.IBD.genome | grep -v POOL38_03 | grep -v POOL99_02 | grep -v POOL59_09 | grep -v POOL78_08 | sort -rg -k 8,8 | vi -

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.IBD.genome | grep -v POOL38_03 | grep -v POOL99_02 | grep -v POOL59_09 | grep -v POOL78_08 | sort -rg -k 8,8 | vi -

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --transpose --recode --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.tfam | grep -E 'POOL27_10|POOL63_08' | awk '{ print $1, "\t", $2 }' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.tfam.2Indvs

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.tfam.2Indvs --transpose --recode --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.just2Indvs

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.tfam.2Indvs --transpose --recode --maf .05 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.just2Indvs.maf05

#Based on cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.hwe1eNeg4.IBD.genome | grep -v POOL38_03 | grep -v POOL99_02 | grep -v POOL59_09 | grep -v POOL78_08 | sort -rg -k 9,9 | vi -
zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ print $1, "\t", $2, "\t", $314, "\t", $492 }' | grep -v ^# | grep -v "\.\/\." | perl -lane 'my @vals1 = split(/:/, $F[2]); my @vals2 = split(/:/, $F[3]); my @vals1sub = split(/\//, $vals1[0]); my @vals2sub = split(/\//, $vals2[0]); my $flag1; if ($vals1[0] eq $vals2[0]) { $flag1 = 2; } elsif (($vals1sub[0] eq $vals2sub[0]) || ($vals1sub[0] eq $vals2sub[1]) || ($vals1sub[1] eq $vals2sub[0]) || ($vals1sub[1] eq $vals2sub[1])) { $flag1 = 1; } else { $flag1 = 0; } print join("\t", @F), "\t", $flag1;' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); xVals <- c(Data1[1,2]); chrm <- Data1[1,1]; adj <- 0; for (i in 2:nrow(Data1)) { if (Data1[i,1] != chrm) { chrm <- Data1[i,1]; adj <- adj + Data1[i-1,2]; }; xVals <- c(xVals, Data1[i,2] + adj); }; jpeg(\"/home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.POOL27_10RL_POOL63_08RL.IBSPlot.jpg\", width=5000, height=2000, res=300); plot(xVals, Data1[,5], main=\"Bad - POOL27_10 vs. POOL63_08\", type=\"l\"); dev.off();" &

zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ print $1, "\t", $2, "\t", $496, "\t", $541 }' | grep -v ^# | grep -v "\.\/\." | perl -lane 'my @vals1 = split(/:/, $F[2]); my @vals2 = split(/:/, $F[3]); my @vals1sub = split(/\//, $vals1[0]); my @vals2sub = split(/\//, $vals2[0]); my $flag1; if ($vals1[0] eq $vals2[0]) { $flag1 = 2; } elsif (($vals1sub[0] eq $vals2sub[0]) || ($vals1sub[0] eq $vals2sub[1]) || ($vals1sub[1] eq $vals2sub[0]) || ($vals1sub[1] eq $vals2sub[1])) { $flag1 = 1; } else { $flag1 = 0; } print join("\t", @F), "\t", $flag1;' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); xVals <- c(Data1[1,2]); chrm <- Data1[1,1]; adj <- 0; for (i in 2:nrow(Data1)) { if (Data1[i,1] != chrm) { chrm <- Data1[i,1]; adj <- adj + Data1[i-1,2]; }; xVals <- c(xVals, Data1[i,2] + adj); }; jpeg(\"/home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.POOL64_05RL_POOL6S_12RL.IBSPlot.jpg\", width=5000, height=2000, res=300); plot(xVals, Data1[,5], main=\"Bad - POOL64_05 vs. POOL6S_12\", type=\"l\"); dev.off();" &

zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ print $1, "\t", $2, "\t", $130, "\t", $483 }' | grep -v ^# | grep -v "\.\/\." | perl -lane 'my @vals1 = split(/:/, $F[2]); my @vals2 = split(/:/, $F[3]); my @vals1sub = split(/\//, $vals1[0]); my @vals2sub = split(/\//, $vals2[0]); my $flag1; if ($vals1[0] eq $vals2[0]) { $flag1 = 2; } elsif (($vals1sub[0] eq $vals2sub[0]) || ($vals1sub[0] eq $vals2sub[1]) || ($vals1sub[1] eq $vals2sub[0]) || ($vals1sub[1] eq $vals2sub[1])) { $flag1 = 1; } else { $flag1 = 0; } print join("\t", @F), "\t", $flag1;' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); xVals <- c(Data1[1,2]); chrm <- Data1[1,1]; adj <- 0; for (i in 2:nrow(Data1)) { if (Data1[i,1] != chrm) { chrm <- Data1[i,1]; adj <- adj + Data1[i-1,2]; }; xVals <- c(xVals, Data1[i,2] + adj); }; jpeg(\"/home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.POOL111_10RL_POOL62_04RL.IBSPlot.jpg\", width=5000, height=2000, res=300); plot(xVals, Data1[,5], main=\"Bad - POOL111_10 vs. POOL62_04\", type=\"l\"); dev.off();" &

zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ print $1, "\t", $2, "\t", $181, "\t", $699 }' | grep -v ^# | grep -v "\.\/\." | perl -lane 'my @vals1 = split(/:/, $F[2]); my @vals2 = split(/:/, $F[3]); my @vals1sub = split(/\//, $vals1[0]); my @vals2sub = split(/\//, $vals2[0]); my $flag1; if ($vals1[0] eq $vals2[0]) { $flag1 = 2; } elsif (($vals1sub[0] eq $vals2sub[0]) || ($vals1sub[0] eq $vals2sub[1]) || ($vals1sub[1] eq $vals2sub[0]) || ($vals1sub[1] eq $vals2sub[1])) { $flag1 = 1; } else { $flag1 = 0; } print join("\t", @F), "\t", $flag1;' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); xVals <- c(Data1[1,2]); chrm <- Data1[1,1]; adj <- 0; for (i in 2:nrow(Data1)) { if (Data1[i,1] != chrm) { chrm <- Data1[i,1]; adj <- adj + Data1[i-1,2]; }; xVals <- c(xVals, Data1[i,2] + adj); }; jpeg(\"/home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.POOL118_06RL_POOL91_07RL.IBSPlot.jpg\", width=5000, height=2000, res=300); plot(xVals, Data1[,5], main=\"Bad - POOL118_06 vs. POOL91_07\", type=\"l\"); dev.off();" &

zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ print $1, "\t", $2, "\t", $456, "\t", $524 }' | grep -v ^# | grep -v "\.\/\." | perl -lane 'my @vals1 = split(/:/, $F[2]); my @vals2 = split(/:/, $F[3]); my @vals1sub = split(/\//, $vals1[0]); my @vals2sub = split(/\//, $vals2[0]); my $flag1; if ($vals1[0] eq $vals2[0]) { $flag1 = 2; } elsif (($vals1sub[0] eq $vals2sub[0]) || ($vals1sub[0] eq $vals2sub[1]) || ($vals1sub[1] eq $vals2sub[0]) || ($vals1sub[1] eq $vals2sub[1])) { $flag1 = 1; } else { $flag1 = 0; } print join("\t", @F), "\t", $flag1;' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); xVals <- c(Data1[1,2]); chrm <- Data1[1,1]; adj <- 0; for (i in 2:nrow(Data1)) { if (Data1[i,1] != chrm) { chrm <- Data1[i,1]; adj <- adj + Data1[i-1,2]; }; xVals <- c(xVals, Data1[i,2] + adj); }; jpeg(\"/home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.POOL59_05RL_POOL68_086L.IBSPlot.jpg\", width=5000, height=2000, res=300); plot(xVals, Data1[,5], main=\"Bad - POOL59_05 vs. POOL68_08\", type=\"l\"); dev.off();" &

zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ print $1, "\t", $2, "\t", $259, "\t", $308 }' | grep -v ^# | grep -v "\.\/\." | perl -lane 'my @vals1 = split(/:/, $F[2]); my @vals2 = split(/:/, $F[3]); my @vals1sub = split(/\//, $vals1[0]); my @vals2sub = split(/\//, $vals2[0]); my $flag1; if ($vals1[0] eq $vals2[0]) { $flag1 = 2; } elsif (($vals1sub[0] eq $vals2sub[0]) || ($vals1sub[0] eq $vals2sub[1]) || ($vals1sub[1] eq $vals2sub[0]) || ($vals1sub[1] eq $vals2sub[1])) { $flag1 = 1; } else { $flag1 = 0; } print join("\t", @F), "\t", $flag1;' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); xVals <- c(Data1[1,2]); chrm <- Data1[1,1]; adj <- 0; for (i in 2:nrow(Data1)) { if (Data1[i,1] != chrm) { chrm <- Data1[i,1]; adj <- adj + Data1[i-1,2]; }; xVals <- c(xVals, Data1[i,2] + adj); }; jpeg(\"/home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.POOL17S_09RL_POOL26_01RL.IBSPlot.jpg\", width=5000, height=2000, res=300); plot(xVals, Data1[,5], main=\"Good - POOL17S_09 vs. POOL26_01\", type=\"l\"); dev.off();" &

zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ print $1, "\t", $2, "\t", $229, "\t", $496 }' | grep -v ^# | grep -v "\.\/\." | perl -lane 'my @vals1 = split(/:/, $F[2]); my @vals2 = split(/:/, $F[3]); my @vals1sub = split(/\//, $vals1[0]); my @vals2sub = split(/\//, $vals2[0]); my $flag1; if ($vals1[0] eq $vals2[0]) { $flag1 = 2; } elsif (($vals1sub[0] eq $vals2sub[0]) || ($vals1sub[0] eq $vals2sub[1]) || ($vals1sub[1] eq $vals2sub[0]) || ($vals1sub[1] eq $vals2sub[1])) { $flag1 = 1; } else { $flag1 = 0; } print join("\t", @F), "\t", $flag1;' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); xVals <- c(Data1[1,2]); chrm <- Data1[1,1]; adj <- 0; for (i in 2:nrow(Data1)) { if (Data1[i,1] != chrm) { chrm <- Data1[i,1]; adj <- adj + Data1[i-1,2]; }; xVals <- c(xVals, Data1[i,2] + adj); }; jpeg(\"/home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.POOL13S_04RL_POOL64_05RL.IBSPlot.jpg\", width=5000, height=2000, res=300); plot(xVals, Data1[,5], main=\"Good - POOL13S_04 vs. POOL64_05\", type=\"l\"); dev.off();" &

zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ print $1, "\t", $2, "\t", $181, "\t", $708 }' | grep -v ^# | grep -v "\.\/\." | perl -lane 'my @vals1 = split(/:/, $F[2]); my @vals2 = split(/:/, $F[3]); my @vals1sub = split(/\//, $vals1[0]); my @vals2sub = split(/\//, $vals2[0]); my $flag1; if ($vals1[0] eq $vals2[0]) { $flag1 = 2; } elsif (($vals1sub[0] eq $vals2sub[0]) || ($vals1sub[0] eq $vals2sub[1]) || ($vals1sub[1] eq $vals2sub[0]) || ($vals1sub[1] eq $vals2sub[1])) { $flag1 = 1; } else { $flag1 = 0; } print join("\t", @F), "\t", $flag1;' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); xVals <- c(Data1[1,2]); chrm <- Data1[1,1]; adj <- 0; for (i in 2:nrow(Data1)) { if (Data1[i,1] != chrm) { chrm <- Data1[i,1]; adj <- adj + Data1[i-1,2]; }; xVals <- c(xVals, Data1[i,2] + adj); }; jpeg(\"/home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.POOL118_06RL_POOL92_10RL.IBSPlot.jpg\", width=5000, height=2000, res=300); plot(xVals, Data1[,5], main=\"Good - POOL18_06 vs. POOL92_10\", type=\"l\"); dev.off();" &


zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ print $1, "\t", $2, "\t", $314, "\t", $492 }' | grep -v ^# | grep -v "\.\/\." | perl -lane 'my @vals1 = split(/:/, $F[2]); my @vals2 = split(/:/, $F[3]); my @vals1sub = split(/\//, $vals1[0]); my @vals2sub = split(/\//, $vals2[0]); my $flag1; if ($vals1[0] eq $vals2[0]) { $flag1 = "BLUE"; } elsif (($vals1sub[0] eq $vals2sub[0]) || ($vals1sub[0] eq $vals2sub[1]) || ($vals1sub[1] eq $vals2sub[0]) || ($vals1sub[1] eq $vals2sub[1])) { $flag1 = "BLUE"; } else { $flag1 = "RED"; } if ($F[0] eq "X") { $F[0] = "23"; } print join("\t", @F), "\t", $flag1;' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); yAxis <- c(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, \"X\"); jpeg(\"/home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.POOL27_10RL_POOL63_08RL.IBSPlot.vs2.jpg\", width=5000, height=2500, res=300); Data1.red <- Data1[Data1[,5]==\"RED\",]; plot(Data1[,2], Data1[,1], main=\"Bad - POOL27_10 vs. POOL63_08\", yaxt=\"n\", col=as.character(Data1[,5])); points(Data1.red[,2], Data1.red[,1], col=Data1.red[,5]); axis(2, at=sort(unique(Data1[,1])), labels=yAxis); dev.off();" &







###### Unsure where this all belongs or what i was doing with these steps before
#vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --maf .05 --recode --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.maf05
#
#gzip /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.maf05.recode.vcf
#
#zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.maf05.recode.vcf.gz | awk '{ print $314, "\t", $492 }' | tail -n +124 | perl -lane
#
#plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --make-bed --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed
######

#Outputs in directory run in, so go to directory /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/
/home/michaelt/Software/admixture_linux-1.23/admixture /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.bed --cv 2 | tee /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.Admixture.2.log &
/home/michaelt/Software/admixture_linux-1.23/admixture /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.bed --cv 3 | tee /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.Admixture.3.log &
/home/michaelt/Software/admixture_linux-1.23/admixture /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.bed --cv 4 | tee /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.Admixture.4.log &
/home/michaelt/Software/admixture_linux-1.23/admixture /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.bed --cv 5 | tee /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.Admixture.5.log &
/home/michaelt/Software/admixture_linux-1.23/admixture /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.bed --cv 6 | tee /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.Admixture.6.log &


#Manually created this file and added the entries, based on the heterzygous-issue individuals seen in above statement (the grep -v individuals) and the individuals in the .genome file with rather high Z2 values (~80%+)
vi /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.20140324.IBDDropIndvs.FIDs

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --remove /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.20140324.IBDDropIndvs.FIDs --recode --recode-INFO-all --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs

gzip /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf

zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.gz | awk '{ if ($1 == "23") { $1 = "X"} print $0 } ' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip

/home/shared/software/tabix-0.2.5/bgzip /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped | grep -v -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.20140324.IBDDropIndvs.FIDs > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.All.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.All.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.All.DropIBDProbIndvs.EPACTS.Skat0 &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.DropIBDProbIndvs.EPACTS.Skat0 &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.DropIBDProbIndvs.EPACTS.Skat0 &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.DropIBDProbIndvs.EPACTS.Skat0 &

#Complex
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.All.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.All.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.All.DropIBDProbIndvs.EPACTS.Skat0 &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.Exonic.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.Exonic.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.Exonic.DropIBDProbIndvs.EPACTS.Skat0 &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.DropIBDProbIndvs.EPACTS.Skat0 &

epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.DropIBDProbIndvs.EPACTS.Skat0 &


[  michaelt@login2  /home/pg/michaelt/Data/ALL_MAPPING]$cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.pheno | grep -E 'POOL103_03|POOL105_03'
POOL103_03RL    POOL103_03RL    2       2       -9      -9
POOL105_03RL    POOL105_03RL    2       2       -9      -9
[  michaelt@login2  /home/pg/michaelt/Data/ALL_MAPPING]$cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.pheno | grep -E 'POOL47_10|POOL87_11'
POOL87_11RL     POOL87_11RL     1       -9      -9      -9
POOL47_10RL     POOL47_10RL     2       2       -9      -9
[  michaelt@login2  /home/pg/michaelt/Data/ALL_MAPPING]$cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.pheno | grep -E 'POOL07_01|POOL22_01'
POOL07_01RL     POOL07_01RL     1       -9      -9      -9
POOL22_01RL     POOL22_01RL     1       -9      -9      -9
[  michaelt@login2  /home/pg/michaelt/Data/ALL_MAPPING]$cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.pheno | grep -E 'POOL36_11|POOL50_12'
POOL50_12RL     POOL50_12RL     2       2       1       -9
POOL36_11RL     POOL36_11RL     2       2       1       -9
[  michaelt@login2  /home/pg/michaelt/Data/ALL_MAPPING]$cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.pheno | grep -E 'POOL112_10|POOL116_10'
POOL112_10RL    POOL112_10RL    1       1       -9      -9
POOL116_10RL    POOL116_10RL    2       2       -9      -9


zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50693697 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'


cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.hwe1eNeg4.IBD.genome | perl -ane 'if ($F[8] > 0) { print $F[0], "\n", $F[2], "\n"; } ' | sort | uniq -c | sort -g -k 1,1 | awk '{ if ($1 >= 100) { print $2 } } ' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.hwe1eNeg4.IBD.genome.AppearMoreThan100Times.FIDs

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.hwe1eNeg4.IBD.genome | perl -ane 'if ($F[8] > 0) { print $F[0], "\n", $F[2], "\n"; } ' | sort | uniq -c | sort -g -k 1,1 | awk '{ if ($1 >= 150) { print $2 } } ' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.hwe1eNeg4.IBD.genome.AppearMoreThan150Times.FIDs



#Samtools
bash /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.Samtools.ptb.MergeVCFsAcrossChromosomes.vs1.sh

gzip /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.vcf

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.vcf.gz | head -n 119 > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.vcf.header

zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.vcf.gz | grep -v INDEL | perl -lane 'if (($F[3] =~ m/^[AGTC]$/) && ($F[4] =~ m/^[AGTC]$/)) { print join("\t", @F); }' | cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.vcf.header - |  gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.vcf.gz

zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.vcf.gz | perl -lane 'if ($F[5] >= 50) { print join("\t", @F); } ' | cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.vcf.header - | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.vcf.gz 

python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.DropOff.VCFVariants.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.vcf.gz --window 1000 | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.vcf.gz

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.vcf.gz --geno .95 --recode --recode-INFO-all --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.geno95

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.vcf.gz --geno .99 --recode --recode-INFO-all --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.geno99

gzip /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.geno95.recode.vcf

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.geno95.recode.vcf.gz --plink --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.geno95.recode

plink --file /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.geno95.recode --make-bed --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.geno95.recode

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.geno95.recode --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --maf .01 --genome --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.geno95.recode.WhiteOnly.QCed

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.geno95.recode.WhiteOnly.QCed.genome | grep -v POOL116_06RL | grep -v POOL116_08RL | grep -v POOL104_06RL | grep -v POOL122_11RL | sort -rg -k 9,9 | vi - 



#Redoing tranches -- doing ts90 resulted in almost no SNPs being present after the 'DropOffTarg_1kb' step, so stepped up to ts_99
/home/shared/software/java/jre1.7.0_09/bin/java -Xmx5g -jar /home/shared/software/GATK/2.5.2/GenomeAnalysisTK.jar -T ApplyRecalibration -R /home/michaelt/Data/HumanGenome/GRCh37/Version8/human_g1k_v37.fasta -input /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.vcf -recalFile /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.recal -tranchesFile /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.tranches -o /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts90.vcf --ts_filter_level 90 -mode SNP

gzip /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts90.vcf

#Takes forever (with zlib 1.2.3), almost an hour
#vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts90.vcf.gz --remove-filtered-all --recode --recode-INFO-all --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts90.PASS

zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts90.vcf.gz | head -n 125 > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts90.vcf.header

zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts90.vcf.gz | grep -v '^#' | grep -E 'PASS|VQSRTrancheSNP90.00to99.00' | cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts90.vcf.header - | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.vcf.gz 

zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.vcf.gz | fill-aa -a /home/michaelt/Data/1000G/AncestralAlignments/human_ancestor_GRCh37_e59/human_ancestor_ | gzip -c > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.vcf.gz

python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.DropOff.VCFVariants.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.vcf.gz --window 1000 | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.DropOffTarg_1kb.vcf.gz

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.DropOffTarg_1kb.vcf.gz --geno .95 --recode --recode-INFO-all --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.DropOffTarg_1kb.95geno

gzip /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.DropOffTarg_1kb.95geno.recode.vcf

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.DropOffTarg_1kb.95geno.recode.vcf.gz --plink --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.DropOffTarg_1kb.95geno

plink --file /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.DropOffTarg_1kb.95geno --make-bed --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.DropOffTarg_1kb.95geno

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.DropOffTarg_1kb.95geno --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --maf .05 --genome --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.DropOffTarg_1kb.95geno.maf05


#Doing IBD with variants covering arrays of 1000G individuals to see if get similar results, so doing this on the 4_CS cluster
scp /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/.

vi /KG/1000_Genomes/Samples/VCFs/HIVProj/DataProcessing.Pipeline.Utility.Create1000GOverlapWithArrayVCFs.sh

mv /KG/1000_Genomes/Samples/VCFs/HIVProj/DataProcessing.Pipeline.Utility.Create1000GOverlapWithArrayVCFs.sh /KG/1000_Genomes/Samples/VCFs/HIVProj/DataProcessing.Pipeline.Utility.Create1000GOverlapWithArrayVCFs.vs1.sh

cp -p /KG/1000_Genomes/Samples/VCFs/HIVProj/DataProcessing.Pipeline.Utility.Create1000GOverlapWithArrayVCFs.vs1.sh /KG/1000_Genomes/Samples/VCFs/HIVProj/DataProcessing.Pipeline.Utility.Create1000GOverlapWithArrayVCFs.Launcher.vs1.sh

bash /KG/1000_Genomes/Samples/VCFs/HIVProj/DataProcessing.Pipeline.Utility.Create1000GOverlapWithArrayVCFs.Launcher.vs1.sh

vcf-concat /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr1.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr2.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr3.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr4.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr5.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr6.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr7.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr8.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr9.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr10.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr11.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr12.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr13.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr14.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr15.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr16.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr17.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr18.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr19.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr20.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr21.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr22.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrX.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz | gzip > /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrALL.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz

vcftools --gzvcf /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrALL.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz --maf .05 --geno .95 --hwe .00001 --recode --recode-INFO-all --out /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrALL.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.maf05.geno95.hwe1eNeg5 

gzip /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrALL.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.maf05.geno95.hwe1eNeg5.recode.vcf

vcftools --gzvcf /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrALL.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.maf05.geno95.hwe1eNeg5.recode.vcf.gz --plink-tped --out /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrALL.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.maf05.geno95.hwe1eNeg5

plink --tfile /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrALL.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.maf05.geno95.hwe1eNeg5 --make-bed --out /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrALL.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.maf05.geno95.hwe1eNeg5

plink --bfile /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrALL.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.maf05.geno95.hwe1eNeg5 --genome --out /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrALL.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.maf05.geno95.hwe1eNeg5




#TimeptChck: 20140319

#For Sudhir
cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.csv | perl -F, -lane 'my $poolID; foreach my $entry (@F) { if ($entry =~ m/(.*ool)\s+([a-zA-Z0-9]*)/) { my $store2 = $2; if ($store2 =~ m/^[0-9]$/) { $poolID = "POOL0" . $store2; } else { $poolID = "POOL" . $2; } } } if ($F[2] =~ m/^[0-9]$/) { $F[2] = "0" . $F[2]; } if ($poolID) { print $F[1], "\t", $poolID, "_", $F[2], "RL"; }' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.LinkMACID_POOLID.txt

#Manually correct POOL110_04RL line -- it's wrong

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.LinkMACID_POOLID.txt | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.FIDs | sed 's/\^//g' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.justEuropean.QCed.txt

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.LinkMACID_POOLID.txt | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.hwe1eNeg4.IBD.genome.AppearMoreThan100Times.FIDs | sed 's/\^//g' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.justEuropean.PresentInMoreThan100PairsZ2.txt
cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.LinkMACID_POOLID.txt | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.hwe1eNeg4.IBD.genome.AppearMoreThan150Times.FIDs | sed 's/\^//g' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.justEuropean.PresentInMoreThan150PairsZ2.txt




#At this point began moving files without any 'Vs#' signifier in their names to Vs1/

#mapping_pool_merged directory
mkdir Vs1
ls -lrt | grep -v Vs2 | grep AllPools | awk '{ print $9 }' | perl -ane 'system("mv $F[0] Vs1/$F[0]");'

#GATK Directory
gzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.raw.vcf &	
mkdir Vs1
ls -lrt | grep -v Vs2 | grep AllPools | awk '{ print $9 }' | perl -ane 'system("mv $F[0] Vs1/$F[0]");'

#Samtools Directory
mkdir Vs1
mv AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.Chr* Vs1/.





6th run with mapping_pool_125_
~~~~~~~~~~~

ls -lrt /home/pg/MM5_ALL_MAPPING/. | awk '{ print $9 } ' > /home/pg/michaelt/Data/ALL_MAPPING/home_pg_ALL_MAPPING_dirList.20140321.txt

cp -p /home/pg/michaelt/Data/ALL_MAPPING/home_pg_ALL_MAPPING_dirList.20140321.txt /home/pg/michaelt/Data/ALL_MAPPING/home_pg_ALL_MAPPING_dirList.20140321.NotCreatedYet.txt
#Then manually edited to remove directories already used/processed before -- kept mapping_pool_125_

cp -p /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectLSInfoPerDir.vs2.sh /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectLSInfoPerDir.vs3.sh

bash /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectLSInfoPerDir.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.NotCreatedYet.txt MM5 #Creates subdirectory folder-list per directory

cp -p /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.MakeDir.vs2.sh /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.MakeDir.vs3.sh

bash /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.MakeDir.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.NotCreatedYet.txt MM5 #Creates the rest of the sub-subdirectories, etc. in ALL_MAPPING

cp -p /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectOrigSFFFileInformation.vs2.sh /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectOrigSFFFileInformation.vs3.sh

bash /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectOrigSFFFileInformation.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.NotCreatedYet.txt MM5 #Creates home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.${date1}.txt file

cat /home/pg/michaelt/Data/sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.20140321_160330.txt | sort | uniq > /home/pg/michaelt/Data/sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.20140321_160330.SortUniq.txt

bash /home/pg/michaelt/Data/sffFiles/DataOrganization.MakeDir.vs2.sh /home/pg/michaelt/Data/sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.20140321_160330.SortUniq.txt

cp -p /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CombinePoolNameswsffFileIDs.vs2.sh /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CombinePoolNameswsffFileIDs.vs3.sh

bash /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CombinePoolNameswsffFileIDs.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.NotCreatedYet.txt /home/pg/michaelt/Data/sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.20140321_160330.SortUniq.txt MM5 #Creates file such as DataOrganization.CombinePoolNameswsffFileIDs.vs3.${date1}.output

cp -p /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.ConnectPoolsffFileswithIndvMIDRLsffFiles.vs2.sh /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.ConnectPoolsffFileswithIndvMIDRLsffFiles.vs3.sh

bash /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.ConnectPoolsffFileswithIndvMIDRLsffFiles.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.NotCreatedYet.txt /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CombinePoolNameswsffFileIDs.vs3.20140321_161846.output MM5

cp -p /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.sh /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.sh

bash /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.NotCreatedYet.txt MM5 #Creates DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.${date1}.output file 

bash /home/pg/michaelt/Data/sffFiles/DataProcessing.BreakUpMIDs.Launcher.vs2.sh /home/pg/michaelt/Data/sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.20140321_160330.SortUniq.txt

bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.20140321_164539.output

cp -p DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.vs4.sh DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.vs5.sh

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.20140321_164539.output

#Edited DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output via :.,$ s/Pools/Pools\/MM5/g 

cat DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.20140321_164539.output > DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output

cp -p DataOrganization.Utility.CreatebamMergeList.vs2.sh DataOrganization.Utility.CreatebamMergeList.vs3.sh

bash DataOrganization.Utility.CreatebamMergeList.vs3.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output

#Remove carrots from file, not dropping individuals. Then creating a file which takes all the IDs from the second column in the POP_genomics file, hashes them, then prints out the list of multiple entries per ID plus their RMID. The extra perl stuff is formatting this because I print out multiple python arrays which have that funny [' ', ' ', ' '] formatting (joined by ;). I then grep out the multiple entries of the main 'CreatebamMergeList' file given the list of multiple entries for the single ID, add them together in a single line and reappend them to the original CreatebamMergeList file. * individuals are dropped because their ID names have the star so they don't get matched up to non-star ID# versions of that ID#. I then have to manually remove the star entries from the POP_genomics file so that its matched up to the CreatebamMergeList which future downstream steps are based on

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.csv | sed 's/\^//g' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.noCarrot.csv

python /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.Utility.CheckForDupEntriesPOPSampleInfoFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.noCarrot.csv | perl -F: -ane 'my @vals1 = split(/,/, $F[0]); my $id1 = $vals1[1]; print $id1, ":"; foreach my $entry1 (@F) { my @valsN = split(/,/, $entry1); my @MID = split(//, $valsN[2]); if ($#MID == "3") { print "0", $valsN[2], ","; } else { print $valsN[2], ","; } foreach my $entry2 (@valsN) { if ($entry2 =~ m/ool/) { print $entry2; } } print ":";} print "\n";' | sed "s/'//g" | sed 's/\s//g' | perl -F: -lane 'print join(":", @F[1..$#F]);' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.noCarrot.DupListPrepForMergeListUse.txt

bash /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineMultPoolsForSameIndvbamMergeList.vs1.sh /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.noCarrot.DupListPrepForMergeListUse.txt /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140323_234600.output #Creates DataOrganization.CreatebamMergeList.vs3.20140323_234600.output.multPOOLperIndvCollected* files

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.noCarrot.csv | grep -v \* > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.noCarrot.dropIndvs.csv 

python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.DropIndividualsBasedOnPopGenomFile.vs3.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.noCarrot.dropIndvs.csv --file2 /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140323_234600.output.multPOOLperIndvCollected > /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140323_234600.output.multPOOLperIndvCollected.dropIndvs

cat /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.20140321_164539.output | awk '{ print $2 }' > /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.20140321_164539.output.justSFFFiles

cat /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140323_234600.output.multPOOLperIndvCollected.dropIndvs | grep -f /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.20140321_164539.output.justSFFFiles > /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140323_234600.output.multPOOLperIndvCollected.dropIndvs.justRecentAdditions

cp -p /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs2.sh

cp -p /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.vs3.sh /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.vs4.sh

bash /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs2.sh /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140323_234600.output.multPOOLperIndvCollected.dropIndvs.justRecentAdditions 

cp -p /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs2.sh

bash /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs2.sh /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140323_234600.output.multPOOLperIndvCollected.dropIndvs.justRecentAdditions

cp -p /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs2.sh

#bash /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs2.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140323_234600.output.multPOOLperIndvCollected.dropIndvs


#Troublehsooting

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.noCarrot.DupListPrepForMergeListUse.txt | grep pool7:01 > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.noCarrot.DupListPrepForMergeListUse.01_pool7_01_Pool22.txt

bash /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineMultPoolsForSameIndvbamMergeList.vs1.sh /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.noCarrot.DupListPrepForMergeListUse.01_pool7_01_Pool22.txt /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140323_234600.output.multPOOLperIndvCollected.test



#At this point realized there were some differences between the earlier 20140322_POP_genomics_sample_information_2... files I was using and the most recent one (I had just been appending the most recent additions of the new files to my on-going, edited file from earlier versions. However, a few entries had been changed beyond just the new additions, so needed to more thoroughly compare IDs between my edited, on-going version and the newest file. So redoing things from the 'MergeallSampleBamFiles' point onwards. Also included individuals from POOL86 which had been skipped at some point

ls -lrt /home/pg/MM5_ALL_MAPPING/mapping_pool_78-90/. | awk '{ print $9 } ' > Pools/MM5/PreMerge/mapping_pool_78-90/home_pg_ALL_MAPPING_dirList.WorkingMainList.txt.mapping_pool_78-90.Full

cat Pools/MM5/PreMerge/mapping_pool_78-90/home_pg_ALL_MAPPING_dirList.WorkingMainList.txt.mapping_pool_78-90.Full | grep 86 > Pools/MM5/PreMerge/mapping_pool_78-90/home_pg_ALL_MAPPING_dirList.WorkingMainList.txt.mapping_pool_78-90

cp -p home_pg_ALL_MAPPING_dirList.20140321.NotCreatedYet.txt home_pg_ALL_MAPPING_dirList.20140321.86Redo.txt

cp -p /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PreMerge/mapping_pool_78-90/home_pg_ALL_MAPPING_dirList.WorkingMainList.txt.mapping_pool_78-90 /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PreMerge/mapping_pool_78-90/home_pg_ALL_MAPPING_dirList.20140321.86Redo.txt.mapping_pool_78-90

mv /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PreMerge/mapping_pool_78-90/home_pg_ALL_MAPPING_dirList.WorkingMainList.txt.mapping_pool_78-90.Full home_pg_ALL_MAPPING_dirList.WorkingMainList.txt.mapping_pool_78-90

bash /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.MakeDir.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.86Redo.txt MM5 #Creates the rest of the sub-subdirectories, etc. in ALL_MAPPING

bash /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectOrigSFFFileInformation.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.86Redo.txt MM5 #Creates home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.${date1}.txt file

cat /home/pg/michaelt/Data/sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.20140328_154417.txt | sort | uniq > /home/pg/michaelt/Data/sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.20140328_154417.SortUniq.txt

bash /home/pg/michaelt/Data/sffFiles/DataOrganization.MakeDir.vs2.sh /home/pg/michaelt/Data/sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.20140328_154417.SortUniq.txt

bash /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CombinePoolNameswsffFileIDs.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.86Redo.txt /home/pg/michaelt/Data/sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.20140328_154417.SortUniq.txt MM5 #Creates file such as DataOrganization.CombinePoolNameswsffFileIDs.vs3.${date1}.output

bash /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.ConnectPoolsffFileswithIndvMIDRLsffFiles.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.86Redo.txt /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CombinePoolNameswsffFileIDs.vs3.20140328_154612.output MM5

bash /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.86Redo.txt MM5 #Creates DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.${date1}.output file 

bash /home/pg/michaelt/Data/sffFiles/DataProcessing.BreakUpMIDs.Launcher.vs2.sh /home/pg/michaelt/Data/sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.20140328_154417.SortUniq.txt

bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.20140328_173953.output

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.20140328_173953.output

cat DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.20140328_173953.output > DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output.temp

mv DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output.temp DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output

bash DataOrganization.Utility.CreatebamMergeList.vs3.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output

scp mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.csv mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.worksheet.csv /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/.

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.csv | sed 's/\r/\n/g' | grep -v '^$' | sed 's/\^//g' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.noMeta.noBlank.noCarrot.csv

#MANUALLY included *s next to POOL106-01 (there are two of them and for each MACID there is a duplicate so should be fine) and POOL112-01 because there are two entries in the POP_genomics files for this POOL#_MID# combination, so unsure which is correct MACID. But since there are duplicate entries for each MACID should be fine 

python /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.Utility.CheckForDupEntriesPOPSampleInfoFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.noMeta.noBlank.noCarrot.csv | perl -F: -ane 'chomp(@F); my @vals1 = split(/,/, $F[0]); my $id1 = $vals1[1]; print $id1, ":"; foreach my $entry1 (@F) { my @valsN = split(/,/, $entry1); my @MID = split(//, $valsN[2]); if ($#MID == "3") { print "0", $valsN[2], ","; } else { print $valsN[2], ","; } foreach my $entry2 (@valsN) { if ($entry2 =~ m/ool/) { print $entry2; } } print ":";} print "\n";' | sed "s/'//g" | sed 's/\s//g' | sed 's/\]//g' | perl -F: -lane 'print join(":", @F[1..$#F]);' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.noMeta.noBlank.noCarrot.DupListPrepForMergeListUse.txt

cp -p DataProcessing.Pipeline.Utility.CombineMultPoolsForSameIndvbamMergeList.vs1.sh DataProcessing.Pipeline.Utility.CombineMultPoolsForSameIndvbamMergeList.vs2.sh

bash /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineMultPoolsForSameIndvbamMergeList.vs2.sh /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.noMeta.noBlank.noCarrot.DupListPrepForMergeListUse.txt /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output #Creates DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected* files

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.noMeta.noBlank.noCarrot.csv | grep -v \* > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.noMeta.noBlank.noCarrot.dropIndvs.csv 

python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.DropIndividualsBasedOnPopGenomFile.vs3.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.noMeta.noBlank.noCarrot.dropIndvs.csv --file2 /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected > /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs

cat /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs | perl -F\; -lane 'my @vals1 = split(/\|/, $F[0]); my @vals2 = split(/\//, $vals1[0]); if ($vals2[$#vals2] =~ m/(POOL[a-zA-Z0-9]*).*(\d\d+RL)/) { print $1 . "_" . $2 ; } ' > /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.POOLIDs

#cat /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs | awk '{ if (NR >= 833) { print $0 } } ' > DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.Redos
cat /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs | awk '{ if (NR >= 837) { print $0 } } ' > DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.Redos

bash /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs2.sh /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.Redos

bash /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs2.sh /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.Redos

#Did this process twice, once originally and again with the POOL106/POOL112 issue

cat /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs | grep -E 'POOL157-01|POOL158-08|POOL158-01|POOL171-05' > /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.Redos2

#Once more for POOL15S-04 issue

cat DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs | grep POOL16S-02 > DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.Redos3

bash /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs2.sh /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.Redos2

bash /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs2.sh /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.Redos3

bash /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs2.sh /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.Redos2

bash /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs2.sh /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.Redos3

#bash /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs2.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged /home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs

bash /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.pta.SplitBamIntoChromosomes.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

mkdir /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Vs1

ls -lrt | grep -v Vs2 | grep AllPools | awk '{ print $9 }' | perl -ane 'system("mv $F[0] Vs1/$F[0]");'

bash /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.pta.RunReduceReadsPerChromosome.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptb.CreateVCFfilesPerChromosome.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.Samtools.pta.CreateVCFfilesPerChromosome.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

cp -p /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptc.MergeVCFsAcrossChromosomes.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptc.MergeVCFsAcrossChromosomes.vs2.sh

bash /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptc.MergeVCFsAcrossChromosomes.vs2.sh

cp -p DataProcessing.Pipeline.pt6.GATK.ptd.RecalAndApplyVCFfilesPerChromosome.vs1.sh DataProcessing.Pipeline.pt6.GATK.ptd.RecalAndApplyVCFfilesPerChromosome.vs2.sh 

bash DataProcessing.Pipeline.pt6.GATK.ptd.RecalAndApplyVCFfilesPerChromosome.Launcher.vs1.sh /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

scp Pools/MM5/PostMerge/mapping_pool_merged/GATK/*tranches.pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/.

gzip -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf &

gzip -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.vcf &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz --remove-filtered VQSRTrancheSNP99.90to100.00 --remove-filtered LowQual --recode --recode-INFO-all --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9

#For zlib installing, directory /home/michaelt/Software/zlib-1.2.8
./configure --prefix=/home/michaelt/lib; make test
#Unsure how to make vcftools use this particular zlib library

gzip /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.recode.vcf

zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.recode.vcf.gz | fill-aa -a /home/michaelt/Data/1000G/AncestralAlignments/human_ancestor_GRCh37_e59/human_ancestor_ | gzip -c > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.vcf.gz

zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.vcf.gz | perl -lane 'if ($F[0] =~ m/^#/) { print join("\t", @F); } else { my @alleles2 = split(/,/, $F[4]); if ($#alleles2 == 0) { print join("\t", @F); } }' | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz

python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.DropOff.VCFVariants.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --window 1000 | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.vcf.gz

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.vcf.gz --geno .95 --recode --recode-INFO-all --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95

gzip -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf

#Exploring what HWE cutoff to use

#vcftools --gzvcf  /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz --hardy --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95

#Going with .0001 HWE cutoff. At .00001 1809 SNPs are dropped and at .0001 2470 SNPs are dropped, so not much is lost for another magnitude increase in strictness. Between .0001 and .001 not a large number of SNPs would be lost either, however see examples where obs is 900/0/1 and exp is 900/2/0 on a number of X chromosome locations and this produces a p-value between .0001 and .001. It appears that vcf file encoding gives males genotypes for both 'chromosomes' even though they only have one, so 900/0/1 is not unexpected. Therefore I feel like these SNPs should be kept and going up to .001 may start leading to a greater proportion of false positives

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz --hwe .0001 --recode --recode-INFO-all --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4

mv /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.recode.vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf

/home/shared/software/java/jre1.7.0_09/bin/java -Xmx5g -jar /home/shared/software/GATK/2.5.2/GenomeAnalysisTK.jar -T VariantAnnotator -R /home/michaelt/Data/HumanGenome/GRCh37/Version4/human_g1k_v37.fasta -I /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.bam  -o /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.HRun.vcf -A HomopolymerRun --variant /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf -nt 6

######
######Here and stuff
######

mv /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.HRun.vcf  /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf

#Do something to just check number of HRuns seen among variants? a cat | perl -lane ' ' | sort | uniq -c type thing?

gzip -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf 


vcftools --vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf --plink --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4

plink --file /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --make-bed --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --genome --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4

###IBD pair list -- decided going to do eigenstrat first and see if 1) dropping white individuals due to QC and only looking at white individuals lowers the set of removed individuals overall 2) potentially look for clustering amongst those that are dropping or have high z1 vs z0, z2 (e.g. always at the end of the cloud of white non-hispanics)

POOL103_03RL POOL103_03RL POOL105_03RL POOL105_03RL UN    NA  0.0000  0.0480  0.9520  0.9760  -1  0.998919  1.0000      NA
POOL47_10RL  POOL47_10RL  POOL87_11RL  POOL87_11RL UN    NA  0.0038  0.1065  0.8896  0.9429  -1  0.997455  1.0000 313.0000
POOL175_11RL POOL175_11RL  POOL43_10RL  POOL43_10RL UN    NA  0.0019  0.1445  0.8535  0.9258  -1  0.996671  1.0000      NA
POOL36_11RL  POOL36_11RL  POOL50_12RL  POOL50_12RL UN    NA  0.0077  0.1560  0.8363  0.9143  -1  0.996195  1.0000 306.0000
POOL176_12RL POOL176_12RL  POOL36_12RL  POOL36_12RL UN    NA  0.0270  0.1440  0.8291  0.9010  -1  0.995736  1.0000 345.0000
POOL104_07RL POOL104_07RL  POOL33_05RL  POOL33_05RL UN    NA  0.0155  0.1616  0.8230  0.9037  -1  0.995775  1.0000 149.5000
POOL165_12RL POOL165_12RL  POOL71_11RL  POOL71_11RL UN    NA  0.0213  0.1740  0.8048  0.8918  -1  0.995276  1.0000 297.0000
POOL28_06RL  POOL28_06RL  POOL91_11RL  POOL91_11RL UN    NA  0.0194  0.1825  0.7981  0.8893  -1  0.995154  1.0000 73.7500
POOL112_10RL POOL112_10RL POOL116_10RL POOL116_10RL UN    NA  0.0275  0.1951  0.7775  0.8750  -1  0.994566  1.0000 145.0000
POOL163_12RL POOL163_12RL  POOL69_08RL  POOL69_08RL UN    NA  0.0372  0.2257  0.7371  0.8500  -1  0.993508  1.0000 96.3333

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --genome --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.WhiteOnly.QCed

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.WhiteOnly.QCed.genome | sort -rg -k 9,9 | vi -

POOL103_03RL POOL103_03RL POOL105_03RL POOL105_03RL UN    NA  0.0000  0.0434  0.9566  0.9783  -1  0.998919  1.0000      NA
POOL47_10RL  POOL47_10RL  POOL87_11RL  POOL87_11RL UN    NA  0.0034  0.0963  0.9003  0.9484  -1  0.997455  1.0000 313.0000
POOL36_11RL  POOL36_11RL  POOL50_12RL  POOL50_12RL UN    NA  0.0068  0.1412  0.8521  0.9226  -1  0.996195  1.0000 306.0000
POOL112_10RL POOL112_10RL POOL116_10RL POOL116_10RL UN    NA  0.0242  0.1772  0.7985  0.8871  -1  0.994566  1.0000 145.0000
POOL163_12RL POOL163_12RL  POOL69_08RL  POOL69_08RL UN    NA  0.0328  0.2053  0.7619  0.8645  -1  0.993508  1.0000 96.3333

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -E 'POOL103_03RL|POOL105_03RL'
POOL103_03RL    POOL103_03RL    2       2       -9      -9
POOL105_03RL    POOL105_03RL    2       2       -9      -9

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -E 'POOL47_10RL|POOL87_11RL'
POOL87_11RL     POOL87_11RL     1       -9      -9      -9
POOL47_10RL     POOL47_10RL     2       2       -9      -9

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -E 'POOL36_11RL|POOL50_12RL'
POOL50_12RL     POOL50_12RL     2       2       1       -9
POOL36_11RL     POOL36_11RL     2       2       1       -9

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -E 'POOL112_10RL|POOL116_10RL'
POOL112_10RL    POOL112_10RL    1       1       -9      -9
POOL116_10RL    POOL116_10RL    2       2       -9      -9

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -E 'POOL163_12RL|POOL69_08RL'
POOL69_08RL     POOL69_08RL     1       1       -9      -9
POOL163_12RL    POOL163_12RL    1       -9      -9      -9

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.WhiteOnly.QCed.genome | sort -rg -k 9,9 | head -n 10000 | awk '{ print $1, "\n", $3 }' | perl -lane 'chomp(@F); print join("\t", @F);' | sort | uniq -c | sort -g -k 1,1 | awk '{ if ($1 >= 100 ) { print $2 } } ' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.WhiteOnly.QCed.genome.HighZ2Countgte100.FIDs

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.WhiteOnly.QCed.genome | sort -rg -k 8,8 | head -n 10000 | awk '{ print $1, "\n", $3 }' | perl -lane 'chomp(@F); print join("\t", @F);' | sort | uniq -c | sort -g -k 1,1 > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.WhiteOnly.QCed.genome.Z1CountList

#NOTE -- dropping these individuals because they are starkly greater than anyone else in the list. Individuals from the 'high z2' count list are not being dropped because they appear as just to be the tail of a distribution versus being such a group of outliers
cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.WhiteOnly.QCed.genome | sort -rg -k 8,8 | head -n 10000 | awk '{ print $1, "\n", $3 }' | perl -lane 'chomp(@F); print join("\t", @F);' | sort | uniq -c | sort -g -k 1,1 | awk '{ if ($1 >= 200) { print $2 } } ' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.WhiteOnly.QCed.genome.HighZ1Countgte200.FIDs

fileforQCremovingdroppingindividualsbasedonIBDhere


#EIGENSTRAT stuff here

cp -p /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/VLookUp.Python.vs1.py /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/VLookUp.Python.DataHapMap3Overlap.vs1.py

python /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/VLookUp.Python.DataHapMap3Overlap.vs1.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.map --file2 /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.liftOver.hg18tohg19.map | awk '{ print $2 }' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.rsIDs

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --extract /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.rsIDs --recode --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3

plink --file /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3 --make-bed --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3

#Note -- there are a few SNPs that don't match up, about ~30? Just letting them drop since troubleshooting almost got most of them back but not all of them and yeah

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.map | awk '{ print $2 }' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.map.rsIDs 

plink --bfile /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly --extract /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.map.rsIDs --keep  /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.CEUCHBYRI.Unrelated.FIDIIDs --recode --out  /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Vs2

plink --file /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Vs2 --make-bed --out /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Vs2

paste /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.bim /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Vs2.bim | awk '{ if (($5 == $11) || ($5 == $12) ) { print $0 } } ' | awk '{ print $2 }' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.FinalListwDrops.rsIDs 

plink --bfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3 --extract /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.FinalListwDrops.rsIDs --recode --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.vs2

plink --bfile /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly --extract /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.FinalListwDrops.rsIDs --keep /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.CEUCHBYRI.Unrelated.FIDIIDs --recode --out /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Vs2.vs2

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.vs2.ped /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Vs2.vs2.ped > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.ped

plink --ped /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.ped --map /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.vs2.map --make-bed --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.phenoColEdit.ped

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.phenoColEdit.pedind

ln -s /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.bim /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.pedsnp

#Created EigenstratFiles in GATK and moved Eigenstrat/ files into EigenstratFiles/.

scp mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.parfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/.

mv /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.parfile /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.wHapMap3Data.CEUCHBYRI.parfile

/home/michaelt/Software/EIG5.0.1/bin/smartpca -p /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.wHapMap3Data.CEUCHBYRI.parfile

#REDO WITH 4 INDVS MISSING FROM PHENO FILE
python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt1

python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.HapMap3vs.py --file1 /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.txt --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt2

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt2 > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race

sed -i 's/American_Indian_or_Alaskan_Native/AIndian_or_Alaskan_Native/g' /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI -x -o /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.1vs2.xtxt

scp /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.1vs2.pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/.

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race | awk '{ if ((($2 >= 0) && ($2 <= .22)) && (($3 >= -.01) && ($3 <= .021))) { print $0 } } ' | grep White_non-Hispanic > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed -c 1:2 -p White_non-Hispanic -x -o /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.xtxt

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed | perl -lane 'my @vals1 = split(/:/, $F[0]); print $vals1[0];' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed | perl -lane 'my @vals1 = split(/:/, $F[0]); print $vals1[0], "\t", $vals1[0];' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs

scp /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteACed.plot.1vs2.pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/.


#FIgure out phenotype/covar file stuff here now

scp mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/Sample_status_list_12-10-13_CT.edited.csv /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/.

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Sample_status_list_12-10-13_CT.edited.csv | sed 's/\r/\n/g' | sed 's/"//g ' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Sample_status_list_12-10-13_CT.edited.noMeta.csv

cp -p /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.py /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.py

python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.noMeta.noBlank.noCarrot.dropIndvs.csv --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Sample_status_list_12-10-13_CT.edited.noMeta.csv > /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output

python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs1.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno

#4 individuals missing from the /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Sample_status_list_12-10-13_CT.edited.noMeta.csv file and unsure why -- emailed Sudhir about it
cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk 'BEGIN{ while(getline < "/home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.POOLIDs") { POOLID[$1] = 1 }  } { if (POOLID[$1]) { print $1 } } ' > nana
cat "/home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.POOLIDs" | grep -v -f nana
#POOL43_10RL
#POOL101_11RL
#POOL18S_07RL
#POOL2S_03RL

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk 'BEGIN{ while(getline < "/home/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.POOLIDs") { POOLID[$1] = 1; } } { if (POOLID[$1]) { print $0 } } ' > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno.MainPOOLIDs

python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateCovariateFile.vs1.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar


#Allele Frequency stuff/collection stuff here now

#Should drop individuals from IBD stuff here
cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ print $1 }' | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.WhiteOnly.QCed.DropIBD.pheno.FIDs
cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($3 == "1") { print $1 } }' | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.WhiteOnly.QCed.DropIBD.pheno.justSeroNeg.FIDs
cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($4 == "1") { print $1 } }' | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.WhiteOnly.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs
cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($3 == "2") { print $1 } }' | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.WhiteOnly.QCed.DropIBD.pheno.justSeroPos.FIDs
cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "1") { print $1 } }' | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.WhiteOnly.QCed.DropIBD.pheno.justRapidVRapid.FIDs
cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "2") { print $1 } }' | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.WhiteOnly.QCed.DropIBD.pheno.justSlowVSlow.FIDs
cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($6 == "1") { print $1 } }' | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.WhiteOnly.QCed.DropIBD.pheno.justVeryRapid.FIDs
cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($6 == "2") { print $1 } }' | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.WhiteOnly.QCed.DropIBD.pheno.justVerySlow.FIDs

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.FIDs --freq --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.FIDs --freq --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.FIDs --freq --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.FIDs --freq --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.FIDs --freq --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.FIDs --freq --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.FIDs --freq --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.FIDs --freq --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.FIDs --counts --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.FIDs --counts --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.FIDs --counts --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.FIDs --counts --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.FIDs --counts --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.FIDs --counts --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.FIDs --counts --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid &

vcftools --gzvcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.FIDs --counts --out /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow &


#Annovar stuff here now



bash DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs1.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output

cat DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs1.vs2.output | perl -lane 'my @array1; push(@array1, $F[1]); foreach my $entry (@F) { if ($entry !~ m/[a-zA-Z]/) { push (@array1, $entry); } } print join(",", @array1);' | awk 'BEGIN{ print "ID,TotalReadsPreQC,TotalReadsQ10,TotalReadsQ10Prcnt,TotalReadsF4,TotalReadsF4Prcnt,TotalReadsF256,TotalReadsF256Prcnt,TotalReadsPostQC,TotalReadsDropped,TotalReadsDroppedPrcnt,AverageReadLengthPreQC,AverageReadLengthPostQC,BedFileCoveragePostQC" } { print $0 } ' > DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs1.vs2.output.reformatted 

bash DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.sh DataOrganization.CreatebamMergeList.vs3.20140328_174052.multPOOLperIndvCollected.dropIndvs.output

#Total column numbers post formatting = 27
cat DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.vs2.output | perl -lane 'my @array1; push(@array1, $F[1]); foreach my $entry (@F) { if ($entry !~ m/[a-zA-Z]/) { push (@array1, $entry); } } print join(",", @array1);' | awk 'BEGIN{ print "ID,NumberOfOriginalBamFiles,TotalReadsPreQC,TotalReadsPreQCForward,TotalReadsPreQCReverse,TotalReadsPreQCForward_Reverse,PreQCMAPQmean,PreQCMAPQsd,PreQCMean,PreQCSD,PreQCMax,PreQCMin,CoveragePreQC,TotalReadsPostRmdup,TotalReadsPostRmdupDropped,TotalReadsPostRmdupDroppedPercentage,TotalReadsPostQC,TotalReadsPostQCForward,TotalReadsPostQCReverse,TotalReadsPostQCForward_Reverse,PostQCMAPQmean,PostQCMAPQsd,PostQCMean,PostQCSD,PostQCMax,PostQCMin,CoveragePostQC" } { print $0 } ' > DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.vs2.output.reformatted

R --vanilla --slave --args < /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs1.R /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs1.output.reformatted /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.output.reformatted

bash DataProcessing.Pipeline.Utility.GetFastqPreQCStats.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output

bash DataProcessing.Pipeline.Utility.RemoveaCertainFile.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output #To remove QCed.fastq.gz and associated stats files for new run through

bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output

mv DataProcessing.Pipeline.Utility.GetFastqPreQCStats.sh DataProcessing.Pipeline.Utility.GetFastqPreQCStats.vs1.sh

cp -p DataProcessing.Pipeline.Utility.GetFastqPreQCStats.vs1.sh DataProcessing.Pipeline.Utility.GetFastqQCStats.vs1.sh

bash DataProcessing.Pipeline.Utility.GetFastqQCStats.vs1.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output

cat DataProcessing.Pipeline.Utility.GetFastqQCStats.vs1.output > DataProcessing.Pipeline.Utility.GetFastqQCStats.vs1.output.forR
#Removed empty lines with /0.00000.0000 0.0000 and dd in vi
cat DataProcessing.Pipeline.Utility.GetFastqQCStats.vs1.output.forR | R -q -e "Data1 <- read.table(file('stdin'), header=F); sum(as.numeric(Data1[,4])); sum(as.numeric(Data1[,6]));"
> Data1 <- read.table(file('stdin'), header=F); sum(as.numeric(Data1[,4])); sum(as.numeric(Data1[,6]));
[1] 156661507032
[1] 258365498
cat DataProcessing.Pipeline.Utility.GetFastqQCStats.vs1.output | wc
   2896   83858  645395


python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.FullVCFwAnnovar.vs1.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.AllGenesPerVariant.vcf.gz

#Edited code in /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.FullVCFwAnnovar.vs1.py
python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.FullVCFwAnnovar.vs1.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv --file2 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.vcf.gz

python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.AllGenesPerVariant.vcf.gz > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.AllGenesPerVariant.GetVCFQCStats.vs1.output

python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.vcf.gz > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output

zcat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.vcf.gz | grep exonic | grep -v ncRNA | python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py --file1 - > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.Exonic.GetVCFQCStats.vs1.output

python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs1.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.vcf.gz > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.FIDs > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justSeroNeg 
cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.FIDs > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justSeroPos 
cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.FIDs > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justSeroHighlyExposedNeg 
cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.FIDs > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justRapidVRapid 
cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output | grep -f /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.FIDs > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justSlowVSlow 

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
[1] "MeanSingletons: 20.6626826029216 MeanDoubletons: 25.2881806108898 MeanCommon: 5033.1567065073 MeanRare: 504.302788844621 MeanHeterozy: 3457.57503320053 MeanCoverage: 16.6629860920849 sdCoverage: 6.17039709917999 Exonic: 342.65604249668 Intronic: 4139.33997343958 Nonsynonymous: 127.176626826029 Synonymous: 212.610889774236 PP2_D: 9.80610889774236 PP2_P: 11.6693227091633 Sift_D: 22.0823373173971 LRT_D: 21.9189907038513"


cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justSeroNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
[1] "MeanSingletons: 16.1948881789137 MeanDoubletons: 21.1086261980831 MeanCommon: 5038.1661341853 MeanRare: 435.638977635783 MeanHeterozy: 3385.56869009585 MeanCoverage: 15.8726865928358 sdCoverage: 5.50509121070413 Exonic: 340.05750798722 Intronic: 4090.77635782748 Nonsynonymous: 126.159744408946 Synonymous: 211.268370607029 PP2_D: 9.77635782747604 PP2_P: 11.3897763578275 Sift_D: 22.0095846645367 LRT_D: 21.6932907348243"

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justSeroHighlyExposedNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
[1] "MeanSingletons: 16.3291925465839 MeanDoubletons: 21.1987577639752 MeanCommon: 5043.50310559006 MeanRare: 436.596273291925 MeanHeterozy: 3384.70186335404 MeanCoverage: 15.7976001524024 sdCoverage: 4.99886275938732 Exonic: 339.596273291925 Intronic: 4096.49689440994 Nonsynonymous: 125.944099378882 Synonymous: 210.795031055901 PP2_D: 9.64596273291925 PP2_P: 11.3416149068323 Sift_D: 21.9316770186335 LRT_D: 21.5465838509317"

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
[1] "MeanSingletons: 16.9517045454545 MeanDoubletons: 19.8977272727273 MeanCommon: 5046.28125 MeanRare: 426.855113636364 MeanHeterozy: 3405.65340909091 MeanCoverage: 17.0944802710762 sdCoverage: 6.42723639615973 Exonic: 337.542613636364 Intronic: 4091.76420454545 Nonsynonymous: 125.235795454545 Synonymous: 209.323863636364 PP2_D: 9.69886363636364 PP2_P: 11.4403409090909 Sift_D: 21.3238636363636 LRT_D: 21.9403409090909"

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justRapidVRapid | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
[1] "MeanSingletons: 16.6734693877551 MeanDoubletons: 17.3673469387755 MeanCommon: 5088.89795918367 MeanRare: 403.591836734694 MeanHeterozy: 3419.16326530612 MeanCoverage: 22.2127566842631 sdCoverage: 9.1245435877065 Exonic: 338.673469387755 Intronic: 4112.5306122449 Nonsynonymous: 125.061224489796 Synonymous: 210.69387755102 PP2_D: 9.12244897959184 PP2_P: 11.469387755102 Sift_D: 21.0408163265306 LRT_D: 21.3469387755102"

cat /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justSlowVSlow | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
[1] "MeanSingletons: 17.3571428571429 MeanDoubletons: 20.3571428571429 MeanCommon: 5025.71428571429 MeanRare: 426.107142857143 MeanHeterozy: 3386.69642857143 MeanCoverage: 16.4965406672807 sdCoverage: 5.79915172393244 Exonic: 333.803571428571 Intronic: 4087.39285714286 Nonsynonymous: 123.196428571429 Synonymous: 207.392857142857 PP2_D: 9.80357142857143 PP2_P: 11.375 Sift_D: 21.3571428571429 LRT_D: 21.8035714285714"

python /home/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perGene.vs1.py --file1 /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.AllGenesPerVariant.vcf.gz > /home/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perGene.vs1.output








#Setting up lists


#Running analyses


/home/shared/software/java/jre1.7.0_09/bin/java -Xmx5g -jar /home/shared/software/GATK/2.5.2/GenomeAnalysisTK.jar -T VariantAnnotator -R /home/michaelt/Data/HumanGenome/GRCh37/Version4/human_g1k_v37.fasta -I /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.bam -o /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.HRun.vcf -A HomopolymherRun --variant /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.recode.vcf

mv /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.HRun.vcf /home/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.recode.vcf

