#!/bin/sh

20140517 - 1st run
~~~~~~~~~~~

ls -lrt /data/userdata/kevin/P2_ALL_MAPPING/FINAL/. | awk '{ print $9 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/data_kevin_P2_ALL_MAPPING_FINAL_dirList.20140519.txt

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.MakeDir.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.MakeDir.vs3.ForP2.vs1.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.MakeDir.vs3.ForP2.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/data_kevin_P2_ALL_MAPPING_FINAL_dirList.20140519.txt P2

#date "+%F_%k%M%S" | sed 's/\-//g' 
#20140519_154626
ls -lrt /data/454/raw/2013/*/*node*/sff/. | grep P2 | sed 's/\://g' | sed 's/\.//g' > /data/userdata/pg/michaelt/Data/sffFiles/data_kevin_P2_ALL_MAPPING_FINAL_Pools_sff_OrigLocations.Unedited.vs3.20140519_154626.pt1.txt

ls -lrt /data/454/raw/2013/*/*node*/sff/. | grep P2 | sed 's/\://g' | perl -lane 'print "ls -lrt ", $F[0];' | bash - | grep sff | perl -lane 'if ($F[8] =~ m/(.*)(01)(\.sff)/) { print $F[8]; }' > /data/userdata/pg/michaelt/Data/sffFiles/data_kevin_P2_ALL_MAPPING_FINAL_Pools_sff_OrigLocations.Unedited.vs3.20140519_154626.pt2.txt

paste /data/userdata/pg/michaelt/Data/sffFiles/data_kevin_P2_ALL_MAPPING_FINAL_Pools_sff_OrigLocations.Unedited.vs3.20140519_154626.pt1.txt /data/userdata/pg/michaelt/Data/sffFiles/data_kevin_P2_ALL_MAPPING_FINAL_Pools_sff_OrigLocations.Unedited.vs3.20140519_154626.pt2.txt | perl -lane 'if ($F[1] =~ m/(.*)(01)(\.sff)/) { print $F[0], $1, $2, $3; print $F[0], $1, "02", $3; }' > /data/userdata/pg/michaelt/Data/sffFiles/data_kevin_P2_ALL_MAPPING_FINAL_Pools_sff_OrigLocations.Unedited.vs3.20140519_154626.ptAll.txt 

cp -p /data/userdata/pg/michaelt/Data/sffFiles/DataOrganization.MakeDir.vs2.sh /data/userdata/pg/michaelt/Data/sffFiles/DataOrganization.MakeDir.vs3.sh

bash /data/userdata/pg/michaelt/Data/sffFiles/DataOrganization.MakeDir.vs3.sh /data/userdata/pg/michaelt/Data/sffFiles/data_kevin_P2_ALL_MAPPING_FINAL_Pools_sff_OrigLocations.Unedited.vs3.20140519_154626.ptAll.txt

cat /data/userdata/pg/michaelt/Data/sffFiles/data_kevin_P2_ALL_MAPPING_FINAL_Pools_sff_OrigLocations.Unedited.vs3.20140519_154626.ptAll.txt | perl -F/ -lane 'if ($F[5] =~ m/(.*)(P2)(\D*)(\d+)(-|_)(.*)/) { my $poolVal1 = "POOL_P2_" . $4; print join("/", @F), "\t", $F[8];}' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CombinePoolNameswsffFileIDs.vs3.ForP2.vs1.20140519_171823.pt1.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/data_kevin_P2_ALL_MAPPING_FINAL_dirList.20140519.txt | perl -F- -ane 'print join("-", @F); system("cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CombinePoolNameswsffFileIDs.vs3.ForP2.vs1.20140519_171823.pt1.output | grep $F[0] ");' | perl -lane 'if (!$F[1]) { $usePool1 = $F[0]; } else { print $usePool1, "\t", join("\t", @F);}' | perl -lane 'if ($F[2] =~ m/(.*)(\.sff)/) { print $F[0], "\t", $1, "\t", $F[3];}' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CombinePoolNameswsffFileIDs.vs3.ForP2.vs1.20140519_171823.ptAll.output

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.ConnectPoolsffFileswithIndvMIDRLsffFiles.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.ConnectPoolsffFileswithIndvMIDRLsffFiles.vs3.ForP2.vs1.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.ConnectPoolsffFileswithIndvMIDRLsffFiles.vs3.ForP2.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/data_kevin_P2_ALL_MAPPING_FINAL_dirList.20140519.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CombinePoolNameswsffFileIDs.vs3.ForP2.vs1.20140519_171823.ptAll.output P2 #Creates  

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForP2.vs1.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForP2.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/data_kevin_P2_ALL_MAPPING_FINAL_dirList.20140519.txt P2 #Creates DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG.vs1.${date1}.output file

cp -p /data/userdata/pg/michaelt/Data/sffFiles/DataProcessing.BreakUpMIDs.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/sffFiles/DataProcessing.BreakUpMIDs.Launcher.vs3.sh

cp -p /data/userdata/pg/michaelt/Data/sffFiles/DataProcessing.BreakUpMIDs.vs1.sh /data/userdata/pg/michaelt/Data/sffFiles/DataProcessing.BreakUpMIDs.vs2.sh 

bash /data/userdata/pg/michaelt/Data/sffFiles/DataProcessing.BreakUpMIDs.Launcher.vs3.sh /data/userdata/pg/michaelt/Data/sffFiles/data_kevin_P2_ALL_MAPPING_FINAL_Pools_sff_OrigLocations.Unedited.vs3.20140519_154626.ptAll.txt

cat DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.20140519_181702.output | head -n 10 > DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.20140519_181702.output.top10

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.20140519_181702.output

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/OID41305_HG19_Match5_probes/OID41305_HG19_Match5_probe_coverage.bed | perl -lane 'if ($F[0] =~ m/chr(.*)/) { $F[0] = $1; } print join("\t", @F);' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/OID41305_HG19_Match5_probes/OID41305_HG19_Match5_probe_coverage.altrd.bed

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.vs5.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.vs5.ForP2.vs1.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.20140519_181702.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.20140519_181702.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output 

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CheckForPresenceOfCertainFile.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CheckForPresenceOfCertainFile.vs3.ForP2.vs1.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CheckForPresenceOfCertainFile.vs3.ForP2.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output QCed.fastq.gz

mv DataProcessing.Pipeline.Utility.CheckForPresenceOfCertainFile.vs3.output DataProcessing.Pipeline.Utility.CheckForPresenceOfCertainFile.vs3.output.orig1

cat DataProcessing.Pipeline.Utility.CheckForPresenceOfCertainFile.vs3.output.orig1 | awk '{ print $2, "\t", $3 }' > DataProcessing.Pipeline.Utility.CheckForPresenceOfCertainFile.vs3.output.orig1.forSubmission

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataProcessing.Pipeline.Utility.CheckForPresenceOfCertainFile.vs3.output.orig1.forSubmission

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataProcessing.Pipeline.Utility.CheckForPresenceOfCertainFile.vs3.output.orig1.forSubmission

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CheckForPresenceOfCertainFile.vs3.ForP2.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output QCed.preGATK.QCed.bam

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataProcessing.Pipeline.Utility.CheckForPresenceOfCertainFile.vs3.ForP2.vs1.20140521_101415.output

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.Utility.CreatebamMergeList.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.Utility.CreatebamMergeList.vs3.ForP2.vs1.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.Utility.CreatebamMergeList.vs3.ForP2.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output

#Got P2_master_pool_sheet_12-10-13.xls from Wolinsky lab wiki http://swlab.virology.northwestern.edu/groups/454_sequencing_group/
#Note -- starred out POOL 112 01RL because it has two entries and both are resequenced elsewhere

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/P2_master_pool_sheet_12-10-13.edited.csv | perl -F, -lane 'if ($F[4] =~ m/ool/) { $poolVal = $F[4]; } else { if ($poolVal) { $F[4] = $poolVal; } } print join(",", @F);' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/P2_master_pool_sheet_12-10-13.edited.PoolFilledOut.csv

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/P2_master_pool_sheet_12-10-13.edited.PoolFilledOut.csv | grep -v \* > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/P2_master_pool_sheet_12-10-13.edited.PoolFilledOut.DropIndv.csv

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.Utility.CheckForDupEntriesPOPSampleInfoFile.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.Utility.CheckForDupEntriesPOPSampleInfoFile.vs1.ForP2.vs1.py

#Remove carrots from file, not dropping individuals. Then creating a file which takes all the IDs from the second column in the POP_genomics file, hashes them, then prints out the list of multiple entries per ID plus their RMID. The extra perl stuff is formatting this because I print out multiple python arrays which have that funny [' ', ' ', ' '] formatting (joined by ;). I then grep out the multiple entries of the main 'CreatebamMergeList' file given the list of multiple entries for the single ID, add them together in a single line and reappend them to the original CreatebamMergeList file.

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.Utility.CheckForDupEntriesPOPSampleInfoFile.vs1.ForP2.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/P2_master_pool_sheet_12-10-13.edited.PoolFilledOut.DropIndv.csv | perl -F: -ane 'my @vals1 = split(/,/, $F[0]); my $id1 = $vals1[0]; print $id1, ":"; foreach my $entry1 (@F) { my @valsN = split(/,/, $entry1); my @MID = split(//, $valsN[1]); if ($#MID == "3") { print "0", $valsN[1], ","; } else { print $valsN[1], ","; } foreach my $entry2 (@valsN) { if ($entry2 =~ m/ool/) { my @POOL = split(/\s+/, $entry2); print uc($POOL[1]), "_P2_", $POOL[2]; } } print ":";} print "\n";' | sed "s/'//g" | sed 's/\s//g' | perl -F: -lane 'print join(":", @F[1..$#F]);' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/P2_master_pool_sheet_12-10-13.edited.PoolFilledOut.DropIndv.DupListPrepForMergeListUse.txt

scp -p P2_master_pool_sheet_12-10-13_Original.xlsx P2_master_pool_sheet_12-10-13.edited.xlsx P2_master_pool_sheet_12-10-13.edited.csv P2_master_pool_sheet_12-10-13.edited.PoolFilledOut.csv mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/.

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineMultPoolsForSameIndvbamMergeList.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineMultPoolsForSameIndvbamMergeList.vs2.ForP2.vs1.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineMultPoolsForSameIndvbamMergeList.vs2.ForP2.vs1.sh /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/P2_master_pool_sheet_12-10-13.edited.PoolFilledOut.DropIndv.DupListPrepForMergeListUse.txt DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output #Creates DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected*

#Line to check the new entries at the bottom contain only the individuals they should
cat DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected | tail -n 30 | perl -F\; -ane 'my $poolID; foreach my $entry1 (@F) { if ($entry1 =~ m/.*(POOL_P2.*)\|\D+.*.sff/) { if (!$poolID) { $poolID = $1; } if ($1 ne $poolID) { print $poolID, "\t"; $poolID = $1;} } } print $poolID, "\n";'

#Line to check that every entry in this file is unique so the right number of individuals are exepcted to be found in final .vcf file
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected | perl -F\; -lane 'if ($F[0] =~ m/.*(POOL_P2.*)\|\D+.*/) { print $1; } ' | sort | uniq -d

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs2.ForP2.vs1.sh

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.vs4.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.vs4.ForP2.vs1.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs2.ForP2.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs2.ForP2.Vs1.sh

cp -p DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.vs2.sh DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.vs2.ForP2.vs1.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs2.ForP2.Vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected

#Forgot that 112 01RL needs to be dropped so redoing these two POOLs that were originally both grouped with 112 01RL independently
cat DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected | grep -E 'POOL_P2_158-08|POOL_P2_157-01' > DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected.redos

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs2.ForP2.vs1.sh DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected.redos

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs2.ForP2.Vs1.sh DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected.redos

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs2.ForP2.vs1.sh

#bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs2.ForP2.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.pta.SplitBamIntoChromosomes.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1 AllPools.P2.Vs1.AllPoolsMerged

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/Samtools
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.pta.RunReduceReadsPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1 AllPools.P2.Vs1.AllPoolsMerged

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptb.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1 AllPools.P2.Vs1.AllPoolsMerged

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.Samtools.pta.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1 AllPools.P2.Vs1.AllPoolsMerged

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptc.MergeVCFsAcrossChromosomes.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptc.MergeVCFsAcrossChromosomes.vs2.ForP2.Vs1.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptc.MergeVCFsAcrossChromosomes.vs2.ForP2.Vs1.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptd.RecalAndApplyVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK AllPools.P2.Vs1.AllPoolsMerged

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/*tranches.pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/.

gzip -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf &

gzip -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.vcf &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz --remove-filtered VQSRTrancheSNP99.90to100.00 --remove-filtered LowQual --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.recode.vcf

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.re
code.vcf.gz | fill-aa -a /home/michaelt/Data/1000G/AncestralAlignments/human_ancestor_GRCh37_e59/human_ancestor_ | gzip -c > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.vcf.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.vcf.gz | perl -lane 'if ($F[0] =~ m/^#/) { print join("\t", @F); } else { my @alleles2 = split(/,/, $F[4]); if ($#alleles2 == 0) { print join("\t", @F); } }' | gzip >
 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.DropOff.VCFVariants.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/OID41305_HG19_Match5_probes/OID41305_HG19_Match5_probe_coverage.altrd.bed --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --window 1000 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.vcf.gz

/home/michaelt/Software/vcftools_0.1.12a/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.vcf.gz --missing-indv --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.imiss | awk '{ if ($5 > .99) { print $1 } } ' | grep -v INDV > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.imiss.Above99Miss.justIIDs

######HERE######

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.vcf.gz --remove /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.imiss.Above99Miss.justIIDs --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99

gzip -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.recode.vcf

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.recode.vcf.gz --geno .95 --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95

gzip -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.recode.vcf

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.recode.vcf.gz --hwe .0001 --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.recode.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf

#Note -- takes about ~2-3 days to complete even with -nt 6? With -nt 32 it takes
/home/shared/software/java/jre1.7.0_09/bin/java -Xmx10g -jar /home/shared/software/GATK/2.5.2/GenomeAnalysisTK.jar -T VariantAnnotator -R /home/michaelt/Data/HumanGenome/GRCh37/Version4/human_g1k_v37.fasta -I /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/AllPools.P2.Vs1.AllPoolsMerged.bam -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.HRun.vcf -A HomopolymerRun --variant /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf -nt 32 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.HRun.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.HRun.vcf.idx /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.idx

gzip -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz | perl -lane 'my @vals1 = split(/;/, $F[7]); foreach my $SNPinfo (@vals1) { if ($SNPinfo =~ m/HRun/) { print $SNPinfo; } };' | sort | uniq -c

61165 HRun=0                  
24643 HRun=1
7165 HRun=2
2342 HRun=3
633 HRun=4
156 HRun=5
30 HRun=6
4 HRun=7
1 HRun=8
1 HRun=9

vcftools --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf --plink --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4

plink --file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4 --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4 --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --genome --out  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome | sort -rg -k 9,9 | vi -

POOL_P2_103_03RL  POOL_P2_103_03RL  POOL_P2_105_03RL  POOL_P2_105_03RL UN    NA  0.0000  0.0575  0.9425  0.9713  -1  0.998376  1.0000      NA
POOL_P2_101_01RL  POOL_P2_101_01RL  POOL_P2_104_01RL  POOL_P2_104_01RL UN    NA  0.0249  0.1971  0.7780  0.8765  -1  0.993254  1.0000 77.6667
POOL_P2_163_12RL  POOL_P2_163_12RL   POOL_P2_69_08RL   POOL_P2_69_08RL UN    NA  0.0442  0.2246  0.7312  0.8435  -1  0.991569  1.0000 67.0000
POOL_P2_140_10RL  POOL_P2_140_10RL   POOL_P2_87_11RL   POOL_P2_87_11RL UN    NA  0.0252  0.2551  0.7197  0.8473  -1  0.991603  1.0000 64.5714
POOL_P2_145_11RL  POOL_P2_145_11RL  POOL_P2_150_12RL  POOL_P2_150_12RL UN    NA  0.0455  0.2412  0.7133  0.8339  -1  0.991037  1.0000 67.5714
#Not considering the below two pairs to be 'identical', and the last two are provided just to show the dropoff
POOL_P2_164_05RL  POOL_P2_164_05RL   POOL_P2_86_04RL   POOL_P2_86_04RL UN    NA  0.0011  0.6718  0.3271  0.6630  -1  0.980962  1.0000      NA
POOL_P2_59_09RL   POOL_P2_59_09RL   POOL_P2_61_12RL   POOL_P2_61_12RL UN    NA  0.0112  0.6874  0.3014  0.6451  -1  0.980046  1.0000 164.6667
POOL_P2_155_08RL  POOL_P2_155_08RL   POOL_P2_66_03RL   POOL_P2_66_03RL UN    NA  0.7293  0.0000  0.2707  0.2707  -1  0.965164  0.1030  1.3684
POOL_P2_66_03RL   POOL_P2_66_03RL   POOL_P2_93_07RL   POOL_P2_93_07RL UN    NA  0.7300  0.0000  0.2700  0.2700  -1  0.964634  0.1792  1.5263

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | grep -E 'POOL_P2_103_03RL|POOL_P2_105_03RL'
POOL_P2_103_03RL        POOL_P2_103_03RL        2       2       -9      -9
POOL_P2_105_03RL        POOL_P2_105_03RL        2       2       -9      -9

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | grep -E 'POOL_P2_101_01RL|POOL_P2_104_01RL'
POOL_P2_101_01RL        POOL_P2_101_01RL        1       1       -9      -9
POOL_P2_104_01RL        POOL_P2_104_01RL        1       1       -9      -9

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | grep -E 'POOL_P2_163_12RL|POOL_P2_69_08RL'
POOL_P2_69_08RL POOL_P2_69_08RL 1       1       -9      -9
POOL_P2_163_12RL        POOL_P2_163_12RL        1       -9      -9      -9

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | grep -E 'POOL_P2_140_10RL|POOL_P2_87_11RL'
POOL_P2_87_11RL POOL_P2_87_11RL 1       -9      -9      -9
POOL_P2_140_10RL        POOL_P2_140_10RL        2       2       -9      -9

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | grep -E 'POOL_P2_145_11RL|POOL_P2_150_12RL'
POOL_P2_150_12RL        POOL_P2_150_12RL        2       2       1       -9
POOL_P2_145_11RL        POOL_P2_145_11RL        2       2       1       -9

perl -e 'print "POOL_P2_103_03RL\nPOOL_P2_105_03RL\nPOOL_P2_101_01RL\nPOOL_P2_104_01RL\nPOOL_P2_163_12RL\nPOOL_P2_69_08RL\nPOOL_P2_140_10RL\nPOOL_P2_87_11RL\nPOOL_P2_145_11RL\nPOOL_P2_150_12RL\n";' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.FIDs

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.FIDs --depth --missing --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs


cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.idepth

INDV    N_SITES MEAN_DEPTH
POOL_P2_101_01RL        96013   11.6469
POOL_P2_103_03RL        96129   18.5583
POOL_P2_104_01RL        94656   6.22495
POOL_P2_105_03RL        96130   18.1951
POOL_P2_140_10RL        93623   5.21629
POOL_P2_145_11RL        95675   9.08535
POOL_P2_150_12RL        94502   5.87342
POOL_P2_163_12RL        95352   7.56006
POOL_P2_69_08RL 95232   8.13951
POOL_P2_87_11RL 96087   13.8943


cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.imiss

INDV    N_DATA  N_GENOTYPES_FILTERED    N_MISS  F_MISS
POOL_P2_101_01RL        96140   0       127     0.00132099
POOL_P2_103_03RL        96140   0       11      0.000114416
POOL_P2_104_01RL        96140   0       1484    0.0154358
POOL_P2_105_03RL        96140   0       10      0.000104015
POOL_P2_140_10RL        96140   0       2517    0.0261806
POOL_P2_145_11RL        96140   0       465     0.0048367
POOL_P2_150_12RL        96140   0       1638    0.0170377
POOL_P2_163_12RL        96140   0       788     0.00819638
POOL_P2_69_08RL 96140   0       908     0.00944456
POOL_P2_87_11RL 96140   0       53      0.000551279

So choosing:
POOL_P2_105_03RL
POOL_P2_101_01RL
POOL_P2_145_11RL

perl -e 'print "POOL_P2_103_03RL\nPOOL_P2_104_01RL\nPOOL_P2_140_10RL\nPOOL_P2_150_12RL\nPOOL_P2_163_12RL\nPOOL_P2_69_08RL\nPOOL_P2_87_11RL\n";' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome | sort -rg -k 9,9 | head -n 10000 | awk '{ print $1, "\n", $3 }' | perl -lane 'chomp(@F); print join("\t", @F);' | sort | uniq -c | sort -g -k 1,1 | awk '{ if ($1 >= 110 ) { print $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ2Countgte110.FIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome | sort -rg -k 8,8 | head -n 10000 | awk '{ print $1, "\n", $3 }' | perl -lane 'chomp(@F); print join("\t", @F);' | sort | uniq -c | sort -g -k 1,1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.Z1CountList

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome | sort -rg -k 8,8 | head -n 10000 | awk '{ print $1, "\n", $3 }' | perl -lane 'chomp(@F); print join("\t", @F);' | sort | uniq -c | sort -g -k 1,1 | awk '{ if ($1 >= 100) { print $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.FIDs

#NOTE -- dropping these individuals because they are starkly greater than anyone else in the list. Individuals from the 'high z2' count list are not being dropped because they appear as just to be the tail of a distribution versus being such a group of outliers

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs

cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ2Countgte110.FIDs | perl -lane 'push(@F, "Z2"); print join("\t", @F);') <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.FIDs | perl -lane 'push(@F, "Z1"); print join("\t", @F);') > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.OddHighZ1OrZ2.FID_ZNums 

#EIGENSTRAT stuff here

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/VLookUp.Python.DataHapMap3Overlap.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.map --file2 /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.liftOver.hg18tohg19.map | awk '{ print $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.overlapHapMap3.rsIDs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4 --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.overlapHapMap3.rsIDs --recode --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.overlapHapMap3

plink --file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.overlapHapMap3 --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.overlapHapMap3

#Note -- there are a few SNPs that don't match up, about ~30? Just letting them drop since troubleshooting almost got most of them back but not all of them and yeah

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.overlapHapMap3.map | awk '{ print $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.overlapHapMap3.map.rsIDs

plink --bfile /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.overlapHapMap3.map.rsIDs --keep /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.CEUCHBYRI.Unrelated.FIDIIDs --recode --out /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Array2.Vs1

plink --file /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Array2.Vs1 --make-bed --out /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Array2.Vs1

paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.overlapHapMap3.bim /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Array2.Vs1.bim | awk '{ if (($5 == $11) || ($5 == $12) ) { print $0 } } ' | awk '{ print $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.overlapHapMap3.FinalListwDrops.rsIDs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.overlapHapMap3 --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.overlapHapMap3.FinalListwDrops.rsIDs --recode --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.overlapHapMap3.vs2

plink --bfile /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.overlapHapMap3.FinalListwDrops.rsIDs --keep /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.CEUCHBYRI.Unrelated.FIDIIDs --recode --out /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Array2.Vs1.vs2

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.overlapHapMap3.vs2.ped /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Array2.Vs1.vs2.ped > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.ped

plink --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.ped --map /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.overlapHapMap3.vs2.map --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.phenoColEdit.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.phenoColEdit.pedind

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.bim /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.pedsnp

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.wHapMap3Data.CEUCHBYRI.parfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.parfile

/home/michaelt/Software/EIG5.0.1/bin/smartpca -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.parfile

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/P2_master_pool_sheet_12-10-13.edited.PoolFilledOut.DropIndv.csv --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Sample_status_list_12-10-13_CT.edited.noMeta.csv > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs1.ForP2.vs1.py

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs1.ForP2.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateCovariateFile.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateCovariateFile.vs1.ForP2.vs1.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateCovariateFile.vs1.ForP2.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.covar

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.vs1.ForP2.vs1.py

#Troubleshooting
#cat EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca | grep POOL > EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.justArray2
#python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.vs1.ForP2.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.justArray2 > asdouasdoui

#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_100_06RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_101_08RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_101_11RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_112_01RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_112_08RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_119_02RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_149_10RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_154_02RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_154_12RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_67_10RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_71_03RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_72_06RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_73_09RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_87_09RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_87_12RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_88_10RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_89_11RL)

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.vs1.ForP2.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca | cat - <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca | grep -E 'POOL_P2_100_06RL|POOL_P2_101_08RL|POOL_P2_101_11RL|POOL_P2_112_01RL|POOL_P2_112_08RL|POOL_P2_119_02RL|POOL_P2_149_10RL|POOL_P2_154_02RL|POOL_P2_154_12RL|POOL_P2_67_10RL|POOL_P2_71_03RL|POOL_P2_72_06RL|POOL_P2_73_09RL|POOL_P2_87_09RL|POOL_P2_87_12RL|POOL_P2_88_10RL|POOL_P2_89_11RL' | perl -lane '$F[$#F] = "NoPhenotype"; print join("\t", @F); ') > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt1

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.HapMap3vs.py --file1 /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.txt --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt2

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race

sed -i 's/American_Indian_or_Alaskan_Native/AIndian_or_Alaskan_Native/g' /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.1vs2.xtxt

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.3vs4.xtxt

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race -c 5:6 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.5vs6.xtxt

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.*.pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/.

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race | awk '{ if ((($2 >= -.001) && ($2 <= .22)) && (($3 >= -.005) && ($3 <= .021))) { print $0 } } ' | grep White_non-Hispanic > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed -c 1:2 -p White_non-Hispanic -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.plot.1vs2.xtxt

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.plot.1vs2.pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/.

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed | perl -lane 'my @vals1 = split(/:/, $F[0]); print $vals1[0];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed | perl -lane 'my @vals1 = split(/:/, $F[0]); print $vals1[0], "\t", $vals1[0];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race | awk 'BEGIN{ while(getline < "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.OddHighZ1OrZ2.FID_ZNums") { FID = $1":"$1; FIDList[FID] = $2; }} { if (FIDList[$1]) { if (FIDList[$1] == "Z1") { $12 = "Z1High"; } else if (FIDList[$1] == "Z2") { $12 = "Z2High"; } else{ print "blah" }  } else { $12 = "NormalIBD"; } print $0; } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.IBDissuesTagged

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.IBDissuesTagged -c 1:2 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.IBDissuesTagged.plot.1vs2.xtxt
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output.top10

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed | awk 'BEGIN{ while(getline < "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.OddHighZ1OrZ2.FID_ZNums") { FID = $1":"$1; FIDList[FID] = $2; }} { if (FIDList[$1]) { if (FIDList[$1] == "Z1") { $12 = "Z1High"; } else if (FIDList[$1] == "Z2") { $12 = "Z2High"; } else{ print "blah" }  } else { $12 = "NormalIBD"; } print $0; } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 1:2 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.1vs2.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 3:4 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.3vs4.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 5:6 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.5vs6.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 7:8 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.7vs8.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 9:10 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.9vs10.xtxt

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/*pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/.

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.phenoColEdit.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.phenoColEdit.pedind

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.bim /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.pedsnp

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.parfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.parfile 

/home/michaelt/Software/EIG5.0.1/bin/smartpca -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.parfile

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.vs1.ForP2.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.Results.pca > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.Results.pca.altrd.race 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.Results.pca.altrd.race | awk 'BEGIN{ while(getline < "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.OddHighZ1OrZ2.FID_ZNums") { FID = $1":"$1; FIDList[FID] = $2; }} { if (FIDList[$1]) { if (FIDList[$1] == "Z1") { $12 = "Z1High"; } else if (FIDList[$1] == "Z2") { $12 = "Z2High"; } else{ print "blah" }  } else { $12 = "NormalIBD"; } print $0; } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.Results.pca.altrd.race.IBDissuesTagged

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.Results.pca.altrd.race.plot.1vs2.xtxt

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.Results.pca.altrd.race.plot.3vs4.xtxt

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.Results.pca.altrd.race.IBDissuesTagged -c 1:2 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.Results.pca.altrd.race.IBDissuesTagged.plot.1vs2.xtxt

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.Results.pca.altrd.race.IBDissuesTagged -c 3:4 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.Results.pca.altrd.race.IBDissuesTagged.plot.3vs4.xtxt

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/*pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/.

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.covar > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.5PCs.covar
#Edit code in between runs
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.covar > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.pheno 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar | awk '{ print $1, "\t", $2 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar | awk '{ print $1 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDs

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDs --non-ref-ac 1 --recode --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.recode.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.gz --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.allSNPs

#plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD --freq --out 

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs & 

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05 &

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.10PCs &

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.10PCs.maf05 &

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 2 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno2 &

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 3 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno3 &

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 4 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno4 &

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.assoc.logistic Pheno1_justWhite_Logistic_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.assoc.logistic.QQplot.jpeg & 

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.assoc.logistic Pheno1_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.assoc.logistic.QQplot.jpeg & 

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.10PCs.assoc.logistic Pheno1_justWhite_Logistic_10PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.10PCs.assoc.logistic.QQplot.jpeg & 

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.10PCs.maf05.assoc.logistic Pheno1_justWhite_Logistic_maf05_10PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.10PCs.maf05.assoc.logistic.QQplot.jpeg & 

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno2.assoc.logistic Pheno2_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno2.assoc.logistic.QQplot.jpeg & 

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno3.assoc.logistic Pheno3_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno3.assoc.logistic.QQplot.jpeg & 

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno4.assoc.logistic Pheno4_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno4.assoc.logistic.QQplot.jpeg & 

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/GATK/PLINKfiles/. 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ print $1 }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ if ($3 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroNeg.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ if ($4 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ if ($3 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroPos.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ if ($5 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justRapidVRapid.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ if ($5 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSlowVSlow.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ if ($6 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justVeryRapid.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ if ($6 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justVerySlow.FIDs

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroNeg.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroPos.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justRapidVRapid.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSlowVSlow.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justVeryRapid.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justVerySlow.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroPos.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justRapidVRapid.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSlowVSlow.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justVeryRapid.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justVerySlow.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow &

paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq.count | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.AllPhenos.frq.AndCount.gz

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq.count

#gunzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq.count

#Annovar stuff here now

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz | awk '{ print $1, "\t", $2, "\t", $2, "\t", $4, "\t", $5 }' | grep -v ^# > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat

perl /home/michaelt/Software/annovar/summarize_annovar.pl /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat /home/michaelt/Software/annovar/ --ver1000g 1000g2012apr --verdbsnp 129 --veresp 6500 --buildver hg19 --remove

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs5.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.AllPhenos.frq.AndCount.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 30; $i <= 40; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz | grep -v \# | wc
96140 88738882 2030932886

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | wc
98735  109719 44353043

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | perl -F, -lane 'if ($F[1] =~ m/\_2/) { print join("\t", @F); }' | wc
 2551   83674 1149243

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | perl -F, -lane 'if ($F[1] =~ m/\_3/) { print join("\t", @F); }' | wc
 41      41   18201

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | perl -F, -lane 'if ($F[1] =~ m/\_4/) { print join("\t", @F); }' | wc
 2       2     858

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | perl -F, -lane 'if ($F[1] =~ m/\_5/) { print join("\t", @F); }' | wc
 1       1     419


R -q -e 'val1 = 2551 + 41 + 2 + 1; print (val1);'
> val1 = 2551 + 41 + 2 + 1; print (val1);
[1] 2595
> 
> 

R -q -e 'val1 = 2551 + 41 + 2 + 1; print (val1); val2 = 98735 - 2595; print (val2)'
> val1 = 2551 + 41 + 2 + 1; print (val1); val2 = 98735 - 2595; print (val2)
[1] 2595
[1] 96140
> 
> 

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs3.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs

mkdir SKATFiles

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing") || ($7 == "UTR5;UTR3")) { print $8, "\t", $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.All

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { print $8, "\t", $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.Exonic

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $8, "\t", $2 } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($12 == "P") || ($12 == "D") ) { print $8, "\t", $2 } } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_PD

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $8, "\t", $2 } } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_D



mkdir EPACTSFiles

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing") || ($7 == "UTR5;UTR3")) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.All

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $0 } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($12 == "P") || ($12 == "D") ) { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($13 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D

##These are based on array1 -- don't expect them to work here
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing")) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All

#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic

#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $0 } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous

#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($12 == "P") || ($12 == "D") ) { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD

#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { ($12 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_D

#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.Sift_D
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($13 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.LRT_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.pheno | awk '{ print $1, "\t", $2, "\t0\t0\t0\t", $3, "\t", $4, "\t", $5, "\t", $6 } ' | sed 's/-9/NA/g' | perl -lane 'print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar | awk '{ print $4, "\t", $6, "\t", $7 "\t", $8, "\t", $9, "\t", $10, "\t", $11,  "\t", $12, "\t", $13, "\t", $14,  "\t", $15 }' | paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped.vs2

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped.vs2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped

#Add line #FAM_ID IND_ID FAT_ID MOT_ID SEX HIVPROG HIVPROGHE AIDS AIDSEXTR RMID PC1 PC2 PC3 PC4 PC5 PC6 PC7 PC8 PC9 PC10 to /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.gz | awk '{ if ($1 == "23") { $1 = "X"} print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip

/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip 
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz

#EPACTS analyses

#EPACTS Variables Running Through
#Pheno: HIVPROG HIVPROGHE AIDS AIDSEXTR
#Filetypes: All Exonic Exonic.Nonsynonymous Exonic.Nonsynonymous.PP2_PD Exonic.Nonsynonymous.PP2_D Exonic.Nonsynonymous.Sift_D Exonic.Nonsynonymous.LRT_D
#Tests b.collapse b.madsen b.wcnt  skat --skat-adjust skat --skat-o --skat-adjust
#MAF cutoffs: .05 1.0 (e.g. treat all variants the same)

#Ran .,$ s/\/home\/pg\//\/data\/userdata\/pg\//g in DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.pt1.sh and /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.pt2.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs 

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/ExtraFiles

#Directory /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/
ls -lrt | grep RMID5PCs | grep -v epacts$ | awk '{ print $9 }' | perl -lane 'system("mv $F[0] ExtraFiles/$F[0]");'

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.CollectResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.CollectResults.vs2.sh

#NOTE -- odd thing happening where I can copy and paste the commands from the file and they work, but if I run the script it doesn't work. Doesn't like the \ separating the lines
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.CollectResults.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/ 1

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.CollectResults.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.CollectResults.vs2.CpyPst.sh

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs Pheno1 HIV_Acquisition_justWhite_RMID5PCs_skat0 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.RMID5PCs.Everything.Pheno1.maf1.skat0.QQplot.jpeg

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs Pheno2 HIV_AcquisitionHE_justWhite_RMID5PCs_skat0 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.RMID5PCs.Everything.Pheno2.maf1.skat0.QQplot.jpeg

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs Pheno3 AIDS_Progress_justWhite_RMID5PCs_skat0 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.RMID5PCs.Everything.Pheno3.maf1.skat0.QQplot.jpeg

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs Pheno4 AIDS_ProgressExtr_justWhite_RMID5PCs_skat0 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.RMID5PCs.Everything.Pheno4.maf1.skat0.QQplot.jpeg

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.RMID5PCs.Everything.Pheno*QQplot.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/.

perl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs1.pl --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --seed 10000 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped 

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno1.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno1.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno1.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno1.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno1.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno1.maf1.skat0.perm1 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno2.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno2.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno2.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno2.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno2.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno2.maf1.skat0.perm1 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno3.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno3.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno3.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno3.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno3.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno3.maf1.skat0.perm1 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno4.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno4.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno4.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno4.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno4.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno4.maf1.skat0.perm1 &

#install.packages("/home/michaelt/Software/qvalue_1.1.try2.tar.gz", repos=NULL, type="source")

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.EPACTS.RMID5PCs.Every.Pheno2.maf1.skat0.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.EPACTS.RMID5PCs.Every.Pheno3.maf1.skat0.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.EPACTS.RMID5PCs.Every.Pheno4.maf1.skat0.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno1.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno1.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno1.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno1.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno1.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno1.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.perm1.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno2.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno2.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno2.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno2.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno2.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno2.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.EPACTS.RMID5PCs.Every.Pheno2.maf1.skat0.perm1.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno3.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno3.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno3.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno3.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno3.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno3.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.EPACTS.RMID5PCs.Every.Pheno3.maf1.skat0.perm1.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno4.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno4.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno4.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno4.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno4.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno4.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.EPACTS.RMID5PCs.Every.Pheno4.maf1.skat0.perm1.wQvals.Summary



cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno1.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno1.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno1.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno1.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno1.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno1.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.EPACTS.RMID5PCs.Every.Pheno1.maf05.skat0.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno2.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno2.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno2.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno2.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno2.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno2.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.EPACTS.RMID5PCs.Every.Pheno2.maf05.skat0.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno3.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno3.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno3.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno3.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno3.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno3.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.EPACTS.RMID5PCs.Every.Pheno3.maf05.skat0.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno4.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno4.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno4.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno4.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno4.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno4.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.EPACTS.RMID5PCs.Every.Pheno4.maf05.skat0.wQvals.Summary



mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses















#QC Stuff

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs2.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output.GetperRegionpostQCStats.vs2.output | perl -lane 'my @array1; push(@array1, $F[1]); foreach my $entry (@F) { if ($entry !~ m/[a-zA-Z]/) { push (@array1, $entry); } } print join(",", @array1);' | awk 'BEGIN{ print "ID,TotalReadsPreQC,TotalReadsQ10,TotalReadsQ10Prcnt,TotalReadsF4,TotalReadsF4Prcnt,TotalReadsF256,TotalReadsF256Prcnt,TotalReadsPostQC,TotalReadsDropped,TotalReadsDroppedPrcnt,AverageReadLengthPreQC,AverageReadLengthPostQC,BedFileCoveragePostQC" } { print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output.GetperRegionpostQCStats.vs2.output.reformatted

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs2.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected



bash DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.sh DataOrganization.CreatebamMergeList.vs3.20140328_174052.multPOOLperIndvCollected.dropIndvs.output

#Total column numbers post formatting = 27
cat DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.vs2.output | perl -lane 'my @array1; push(@array1, $F[1]); foreach my $entry (@F) { if ($entry !~ m/[a-zA-Z]/) { push (@array1, $entry); } } print join(",", @array1);' | awk 'BEGIN{ print "ID,NumberOfOriginalBamFiles,TotalReadsPreQC,TotalReadsPreQCForward,TotalReadsPreQCReverse,TotalReadsPreQCForward_Reverse,PreQCMAPQmean,PreQCMAPQsd,PreQCMean,PreQCSD,PreQCMax,PreQCMin,CoveragePreQC,TotalReadsPostRmdup,TotalReadsPostRmdupDropped,TotalReadsPostRmdupDroppedPercentage,TotalReadsPostQC,TotalReadsPostQCForward,TotalReadsPostQCReverse,TotalReadsPostQCForward_Reverse,PostQCMAPQmean,PostQCMAPQsd,PostQCMean,PostQCSD,PostQCMax,PostQCMin,CoveragePostQC" } { print $0 } ' > DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.vs2.output.reformatted

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperfastQpostQCStats.vs1.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs1.output.reformatted /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.output.reformatted





#20140808 For F31 Grant
vcftools --gzvcf Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz --freq --out Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.ex1

#20150129
#Jumping back in to start running true value multiple runs and fake value permutations to get final p-values/FDRs/results 

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.gz --missing --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD

~~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.imiss | sort -rg -k 5,5 | head -n 10
POOL_P2_66_03RL 86483   0       75173   0.869223
POOL_P2_175_01RL        86483   0       70385   0.813859
POOL_P2_139_08RL        86483   0       61414   0.710128
POOL_P2_155_12RL        86483   0       51092   0.590775
POOL_P2_105_08RL        86483   0       40091   0.463571
POOL_P2_133_05RL        86483   0       40029   0.462854
POOL_P2_138_08RL        86483   0       34303   0.396644
POOL_P2_170_11RL        86483   0       23038   0.266388
POOL_P2_133_04RL        86483   0       22483   0.25997
POOL_P2_113_09RL        86483   0       14051   0.162471
~~~

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/*Pheno1*perm1* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/*Pheno2*perm1* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/*Pheno3*perm1* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/*Pheno4*perm1* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/.

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/*Pheno1* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/*Pheno2* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/*Pheno3* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/*Pheno4* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/.

perl /home/michaelt/Scripts/Perl/RandomList.pl --total 10000 --seed 5697324 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.MultiRunList.vs1.txt
perl /home/michaelt/Scripts/Perl/RandomList.pl --total 10000 --seed 2138074 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.PermutationList.vs1.txt

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations

for i in {1..1000}
do

        SEED1=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.PermutationList.vs1.txt | awk '{ if (NR == '$i') { print $0 } } '`

	perl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs3.pl --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --seed $SEED1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm${i}.vs3.ped 

#	echo $SEED1

done

~~~
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.vs3.ped | head -n 10                                                                                                                                                                                                                                                                                                       
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL_P2_83_08RL POOL_P2_83_08RL 0       0       0       2       NA      NA      NA      8       0.0156  0.0127  -0.0161 -0.0306 -0.0460 0.0286  0.0071  -0.0276 0.0150  -0.0125                                                                                                                                            
POOL_P2_156_08RL        POOL_P2_156_08RL        0       0       0       2       NA      NA      NA      8       0.0126  0.0086  -0.0303 0.0166  -0.0120 -0.0239 -0.0156 -0.0252 -0.0331 -0.0006
POOL_P2_135_03RL        POOL_P2_135_03RL        0       0       0       1       2       NA      NA      3       0.0088  0.0041  0.0147  0.0344  0.0045  -0.0063 0.0018  -0.0247 0.0196  -0.0216                                                                                                                            
POOL_P2_141_03RL        POOL_P2_141_03RL        0       0       0       1       2       NA      NA      3       0.0128  0.0106  -0.0328 0.0322  -0.0089 -0.0540 -0.0393 0.0170  0.0503  0.0290
POOL_P2_157_04RL        POOL_P2_157_04RL        0       0       0       1       NA      NA      NA      4       0.0124  0.0006  -0.0328 0.0068  0.0043  0.0227  -0.0153 -0.0068 -0.0462 -0.0168                                                                                                                            
POOL_P2_131_05RL        POOL_P2_131_05RL        0       0       0       1       2       NA      NA      5       0.0134  0.0059  0.0123  0.0390  0.0145  -0.0268 0.0012  0.0261  -0.0256 0.0273
POOL_P2_88_02RL POOL_P2_88_02RL 0       0       0       2       2       NA      NA      2       0.0116  0.0033  -0.0360 0.0206  0.0383  -0.0404 -0.0117 -0.0159 0.0066  0.0251                                                                                                                                             
POOL_P2_158_08RL        POOL_P2_158_08RL        0       0       0       2       1       2       NA      8       0.0088  0.0044  0.0137  0.0112  0.0499  0.0253  -0.0095 0.0353  -0.0019 0.0059
POOL_P2_169_01RL        POOL_P2_169_01RL        0       0       0       1       NA      NA      NA      1       0.0123  0.0039  0.0122  0.0155  0.0095  0.0183  0.0229  -0.0000 0.0239  0.0068                                                                                                                             
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm2.vs3.ped | head -n 10                                                                                                                                                                                                                                                                                                       
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL_P2_83_08RL POOL_P2_83_08RL 0       0       0       1       NA      NA      NA      8       0.0156  0.0127  -0.0161 -0.0306 -0.0460 0.0286  0.0071  -0.0276 0.0150  -0.0125                                                                                                                                            
POOL_P2_156_08RL        POOL_P2_156_08RL        0       0       0       1       NA      NA      NA      8       0.0126  0.0086  -0.0303 0.0166  -0.0120 -0.0239 -0.0156 -0.0252 -0.0331 -0.0006
POOL_P2_135_03RL        POOL_P2_135_03RL        0       0       0       2       2       NA      NA      3       0.0088  0.0041  0.0147  0.0344  0.0045  -0.0063 0.0018  -0.0247 0.0196  -0.0216                                                                                                                            
POOL_P2_141_03RL        POOL_P2_141_03RL        0       0       0       2       1       NA      NA      3       0.0128  0.0106  -0.0328 0.0322  -0.0089 -0.0540 -0.0393 0.0170  0.0503  0.0290
POOL_P2_157_04RL        POOL_P2_157_04RL        0       0       0       1       NA      NA      NA      4       0.0124  0.0006  -0.0328 0.0068  0.0043  0.0227  -0.0153 -0.0068 -0.0462 -0.0168                                                                                                                            
POOL_P2_131_05RL        POOL_P2_131_05RL        0       0       0       2       2       NA      NA      5       0.0134  0.0059  0.0123  0.0390  0.0145  -0.0268 0.0012  0.0261  -0.0256 0.0273
POOL_P2_88_02RL POOL_P2_88_02RL 0       0       0       1       2       NA      NA      2       0.0116  0.0033  -0.0360 0.0206  0.0383  -0.0404 -0.0117 -0.0159 0.0066  0.0251                                                                                                                                             
POOL_P2_158_08RL        POOL_P2_158_08RL        0       0       0       2       2       1       NA      8       0.0088  0.0044  0.0137  0.0112  0.0499  0.0253  -0.0095 0.0353  -0.0019 0.0059
POOL_P2_169_01RL        POOL_P2_169_01RL        0       0       0       2       NA      NA      NA      1       0.0123  0.0039  0.0122  0.0155  0.0095  0.0183  0.0229  -0.0000 0.0239  0.0068                                                                                                                             
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm3.vs3.ped | head -n 10                                                                                                                                                                                                                                                                                                       
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL_P2_83_08RL POOL_P2_83_08RL 0       0       0       2       NA      NA      NA      8       0.0156  0.0127  -0.0161 -0.0306 -0.0460 0.0286  0.0071  -0.0276 0.0150  -0.0125                                                                                                                                            
POOL_P2_156_08RL        POOL_P2_156_08RL        0       0       0       1       NA      NA      NA      8       0.0126  0.0086  -0.0303 0.0166  -0.0120 -0.0239 -0.0156 -0.0252 -0.0331 -0.0006
POOL_P2_135_03RL        POOL_P2_135_03RL        0       0       0       2       1       NA      NA      3       0.0088  0.0041  0.0147  0.0344  0.0045  -0.0063 0.0018  -0.0247 0.0196  -0.0216                                                                                                                            
POOL_P2_141_03RL        POOL_P2_141_03RL        0       0       0       1       2       NA      NA      3       0.0128  0.0106  -0.0328 0.0322  -0.0089 -0.0540 -0.0393 0.0170  0.0503  0.0290
POOL_P2_157_04RL        POOL_P2_157_04RL        0       0       0       2       NA      NA      NA      4       0.0124  0.0006  -0.0328 0.0068  0.0043  0.0227  -0.0153 -0.0068 -0.0462 -0.0168                                                                                                                            
POOL_P2_131_05RL        POOL_P2_131_05RL        0       0       0       2       2       NA      NA      5       0.0134  0.0059  0.0123  0.0390  0.0145  -0.0268 0.0012  0.0261  -0.0256 0.0273
POOL_P2_88_02RL POOL_P2_88_02RL 0       0       0       1       2       NA      NA      2       0.0116  0.0033  -0.0360 0.0206  0.0383  -0.0404 -0.0117 -0.0159 0.0066  0.0251                                                                                                                                             
POOL_P2_158_08RL        POOL_P2_158_08RL        0       0       0       2       1       2       NA      8       0.0088  0.0044  0.0137  0.0112  0.0499  0.0253  -0.0095 0.0353  -0.0019 0.0059
POOL_P2_169_01RL        POOL_P2_169_01RL        0       0       0       1       NA      NA      NA      1       0.0123  0.0039  0.0122  0.0155  0.0095  0.0183  0.0229  -0.0000 0.0239  0.0068                                                                                                                             
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm4.vs3.ped | head -n 10                                                                                                                                                                                                                                                                                                       
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL_P2_83_08RL POOL_P2_83_08RL 0       0       0       2       NA      NA      NA      8       0.0156  0.0127  -0.0161 -0.0306 -0.0460 0.0286  0.0071  -0.0276 0.0150  -0.0125                                                                                                                                            
POOL_P2_156_08RL        POOL_P2_156_08RL        0       0       0       2       NA      NA      NA      8       0.0126  0.0086  -0.0303 0.0166  -0.0120 -0.0239 -0.0156 -0.0252 -0.0331 -0.0006
POOL_P2_135_03RL        POOL_P2_135_03RL        0       0       0       2       2       NA      NA      3       0.0088  0.0041  0.0147  0.0344  0.0045  -0.0063 0.0018  -0.0247 0.0196  -0.0216                                                                                                                            
POOL_P2_141_03RL        POOL_P2_141_03RL        0       0       0       2       1       NA      NA      3       0.0128  0.0106  -0.0328 0.0322  -0.0089 -0.0540 -0.0393 0.0170  0.0503  0.0290
POOL_P2_157_04RL        POOL_P2_157_04RL        0       0       0       2       NA      NA      NA      4       0.0124  0.0006  -0.0328 0.0068  0.0043  0.0227  -0.0153 -0.0068 -0.0462 -0.0168                                                                                                                            
POOL_P2_131_05RL        POOL_P2_131_05RL        0       0       0       2       1       NA      NA      5       0.0134  0.0059  0.0123  0.0390  0.0145  -0.0268 0.0012  0.0261  -0.0256 0.0273
POOL_P2_88_02RL POOL_P2_88_02RL 0       0       0       1       2       NA      NA      2       0.0116  0.0033  -0.0360 0.0206  0.0383  -0.0404 -0.0117 -0.0159 0.0066  0.0251                                                                                                                                             
POOL_P2_158_08RL        POOL_P2_158_08RL        0       0       0       1       2       2       NA      8       0.0088  0.0044  0.0137  0.0112  0.0499  0.0253  -0.0095 0.0353  -0.0019 0.0059
POOL_P2_169_01RL        POOL_P2_169_01RL        0       0       0       2       NA      NA      NA      1       0.0123  0.0039  0.0122  0.0155  0.0095  0.0183  0.0229  -0.0000 0.0239  0.0068                                                                                                                             
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm5.vs3.ped | head -n 10                                                                                                                                                                                                                                                                                                       
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL_P2_83_08RL POOL_P2_83_08RL 0       0       0       2       NA      NA      NA      8       0.0156  0.0127  -0.0161 -0.0306 -0.0460 0.0286  0.0071  -0.0276 0.0150  -0.0125                                                                                                                                            
POOL_P2_156_08RL        POOL_P2_156_08RL        0       0       0       1       NA      NA      NA      8       0.0126  0.0086  -0.0303 0.0166  -0.0120 -0.0239 -0.0156 -0.0252 -0.0331 -0.0006
POOL_P2_135_03RL        POOL_P2_135_03RL        0       0       0       2       2       NA      NA      3       0.0088  0.0041  0.0147  0.0344  0.0045  -0.0063 0.0018  -0.0247 0.0196  -0.0216                                                                                                                            
POOL_P2_141_03RL        POOL_P2_141_03RL        0       0       0       2       2       NA      NA      3       0.0128  0.0106  -0.0328 0.0322  -0.0089 -0.0540 -0.0393 0.0170  0.0503  0.0290
POOL_P2_157_04RL        POOL_P2_157_04RL        0       0       0       1       NA      NA      NA      4       0.0124  0.0006  -0.0328 0.0068  0.0043  0.0227  -0.0153 -0.0068 -0.0462 -0.0168                                                                                                                            
POOL_P2_131_05RL        POOL_P2_131_05RL        0       0       0       2       2       NA      NA      5       0.0134  0.0059  0.0123  0.0390  0.0145  -0.0268 0.0012  0.0261  -0.0256 0.0273
POOL_P2_88_02RL POOL_P2_88_02RL 0       0       0       1       1       NA      NA      2       0.0116  0.0033  -0.0360 0.0206  0.0383  -0.0404 -0.0117 -0.0159 0.0066  0.0251                                                                                                                                             
POOL_P2_158_08RL        POOL_P2_158_08RL        0       0       0       1       2       2       NA      8       0.0088  0.0044  0.0137  0.0112  0.0499  0.0253  -0.0095 0.0353  -0.0019 0.0059
POOL_P2_169_01RL        POOL_P2_169_01RL        0       0       0       1       NA      NA      NA      1       0.0123  0.0039  0.0122  0.0155  0.0095  0.0183  0.0229  -0.0000 0.0239  0.0068                                                                                                                            
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.vs3.ped | head -n 150 | tail -n 10
POOL_P2_80_01RL POOL_P2_80_01RL 0       0       0       2       NA      NA      NA      1       0.0148  0.0059  -0.0233 -0.0426 0.0418  -0.0008 -0.0098 0.0051  -0.0213 0.0252
POOL_P2_154_06RL        POOL_P2_154_06RL        0       0       0       2       2       NA      NA      6       0.0164  0.0064  0.0211  0.0186  0.0552  -0.0092 -0.0025 -0.0114 0.0258  -0.0670
POOL_P2_158_12RL        POOL_P2_158_12RL        0       0       0       2       2       NA      NA      12      0.0138  0.0202  0.0158  0.0525  -0.0343 -0.0234 0.0504  0.0353  0.0001  0.0148
POOL_P2_166_09RL        POOL_P2_166_09RL        0       0       0       1       2       2       NA      9       0.0140  0.0055  -0.0177 0.0377  -0.0096 -0.0398 -0.0454 0.0344  0.0272  -0.0319
POOL_P2_149_02RL        POOL_P2_149_02RL        0       0       0       2       1       NA      NA      2       0.0112  0.0123  0.0139  0.0321  -0.0063 -0.0345 -0.0308 0.0064  0.0266  0.0154
POOL_P2_67_06RL POOL_P2_67_06RL 0       0       0       2       2       NA      NA      6       0.0145  0.0064  0.0266  -0.0201 -0.0388 0.0020  0.0063  -0.0134 0.0308  0.0171
POOL_P2_75_02RL POOL_P2_75_02RL 0       0       0       2       1       NA      NA      2       0.0106  0.0043  -0.0382 0.0131  -0.0260 0.0097  -0.0206 -0.0053 0.0158  -0.0481
POOL_P2_60_03RL POOL_P2_60_03RL 0       0       0       2       1       NA      NA      3       0.0120  0.0105  0.0506  -0.0095 -0.0124 -0.0044 0.0005  -0.0130 0.0393  -0.0497
POOL_P2_91_12RL POOL_P2_91_12RL 0       0       0       1       1       NA      NA      12      0.0177  0.0039  -0.0203 -0.0247 -0.0437 -0.0020 -0.0005 0.0200  0.0203  -0.0326
POOL_P2_86_10RL POOL_P2_86_10RL 0       0       0       2       NA      NA      NA      10      0.0111  0.0091  -0.0263 0.0054  0.0154  0.0027  -0.0567 -0.0416 -0.0390 0.0190
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm2.vs3.ped | head -n 150 | tail -n 10
POOL_P2_80_01RL POOL_P2_80_01RL 0       0       0       2       NA      NA      NA      1       0.0148  0.0059  -0.0233 -0.0426 0.0418  -0.0008 -0.0098 0.0051  -0.0213 0.0252
POOL_P2_154_06RL        POOL_P2_154_06RL        0       0       0       1       2       NA      NA      6       0.0164  0.0064  0.0211  0.0186  0.0552  -0.0092 -0.0025 -0.0114 0.0258  -0.0670
POOL_P2_158_12RL        POOL_P2_158_12RL        0       0       0       2       2       NA      NA      12      0.0138  0.0202  0.0158  0.0525  -0.0343 -0.0234 0.0504  0.0353  0.0001  0.0148
POOL_P2_166_09RL        POOL_P2_166_09RL        0       0       0       1       1       2       NA      9       0.0140  0.0055  -0.0177 0.0377  -0.0096 -0.0398 -0.0454 0.0344  0.0272  -0.0319
POOL_P2_149_02RL        POOL_P2_149_02RL        0       0       0       2       2       NA      NA      2       0.0112  0.0123  0.0139  0.0321  -0.0063 -0.0345 -0.0308 0.0064  0.0266  0.0154
POOL_P2_67_06RL POOL_P2_67_06RL 0       0       0       2       2       NA      NA      6       0.0145  0.0064  0.0266  -0.0201 -0.0388 0.0020  0.0063  -0.0134 0.0308  0.0171
POOL_P2_75_02RL POOL_P2_75_02RL 0       0       0       2       2       NA      NA      2       0.0106  0.0043  -0.0382 0.0131  -0.0260 0.0097  -0.0206 -0.0053 0.0158  -0.0481
POOL_P2_60_03RL POOL_P2_60_03RL 0       0       0       1       2       NA      NA      3       0.0120  0.0105  0.0506  -0.0095 -0.0124 -0.0044 0.0005  -0.0130 0.0393  -0.0497
POOL_P2_91_12RL POOL_P2_91_12RL 0       0       0       2       1       NA      NA      12      0.0177  0.0039  -0.0203 -0.0247 -0.0437 -0.0020 -0.0005 0.0200  0.0203  -0.0326
POOL_P2_86_10RL POOL_P2_86_10RL 0       0       0       2       NA      NA      NA      10      0.0111  0.0091  -0.0263 0.0054  0.0154  0.0027  -0.0567 -0.0416 -0.0390 0.0190
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm3.vs3.ped | head -n 150 | tail -n 10
POOL_P2_80_01RL POOL_P2_80_01RL 0       0       0       2       NA      NA      NA      1       0.0148  0.0059  -0.0233 -0.0426 0.0418  -0.0008 -0.0098 0.0051  -0.0213 0.0252
POOL_P2_154_06RL        POOL_P2_154_06RL        0       0       0       2       2       NA      NA      6       0.0164  0.0064  0.0211  0.0186  0.0552  -0.0092 -0.0025 -0.0114 0.0258  -0.0670
POOL_P2_158_12RL        POOL_P2_158_12RL        0       0       0       1       1       NA      NA      12      0.0138  0.0202  0.0158  0.0525  -0.0343 -0.0234 0.0504  0.0353  0.0001  0.0148
POOL_P2_166_09RL        POOL_P2_166_09RL        0       0       0       2       2       2       NA      9       0.0140  0.0055  -0.0177 0.0377  -0.0096 -0.0398 -0.0454 0.0344  0.0272  -0.0319
POOL_P2_149_02RL        POOL_P2_149_02RL        0       0       0       2       2       NA      NA      2       0.0112  0.0123  0.0139  0.0321  -0.0063 -0.0345 -0.0308 0.0064  0.0266  0.0154
POOL_P2_67_06RL POOL_P2_67_06RL 0       0       0       1       2       NA      NA      6       0.0145  0.0064  0.0266  -0.0201 -0.0388 0.0020  0.0063  -0.0134 0.0308  0.0171
POOL_P2_75_02RL POOL_P2_75_02RL 0       0       0       2       1       NA      NA      2       0.0106  0.0043  -0.0382 0.0131  -0.0260 0.0097  -0.0206 -0.0053 0.0158  -0.0481
POOL_P2_60_03RL POOL_P2_60_03RL 0       0       0       2       2       NA      NA      3       0.0120  0.0105  0.0506  -0.0095 -0.0124 -0.0044 0.0005  -0.0130 0.0393  -0.0497
POOL_P2_91_12RL POOL_P2_91_12RL 0       0       0       1       2       NA      NA      12      0.0177  0.0039  -0.0203 -0.0247 -0.0437 -0.0020 -0.0005 0.0200  0.0203  -0.0326
POOL_P2_86_10RL POOL_P2_86_10RL 0       0       0       1       NA      NA      NA      10      0.0111  0.0091  -0.0263 0.0054  0.0154  0.0027  -0.0567 -0.0416 -0.0390 0.0190
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm4.vs3.ped | head -n 150 | tail -n 10
POOL_P2_80_01RL POOL_P2_80_01RL 0       0       0       1       NA      NA      NA      1       0.0148  0.0059  -0.0233 -0.0426 0.0418  -0.0008 -0.0098 0.0051  -0.0213 0.0252
POOL_P2_154_06RL        POOL_P2_154_06RL        0       0       0       2       2       NA      NA      6       0.0164  0.0064  0.0211  0.0186  0.0552  -0.0092 -0.0025 -0.0114 0.0258  -0.0670
POOL_P2_158_12RL        POOL_P2_158_12RL        0       0       0       2       2       NA      NA      12      0.0138  0.0202  0.0158  0.0525  -0.0343 -0.0234 0.0504  0.0353  0.0001  0.0148
POOL_P2_166_09RL        POOL_P2_166_09RL        0       0       0       1       1       1       NA      9       0.0140  0.0055  -0.0177 0.0377  -0.0096 -0.0398 -0.0454 0.0344  0.0272  -0.0319
POOL_P2_149_02RL        POOL_P2_149_02RL        0       0       0       2       2       NA      NA      2       0.0112  0.0123  0.0139  0.0321  -0.0063 -0.0345 -0.0308 0.0064  0.0266  0.0154
POOL_P2_67_06RL POOL_P2_67_06RL 0       0       0       2       2       NA      NA      6       0.0145  0.0064  0.0266  -0.0201 -0.0388 0.0020  0.0063  -0.0134 0.0308  0.0171
POOL_P2_75_02RL POOL_P2_75_02RL 0       0       0       2       2       NA      NA      2       0.0106  0.0043  -0.0382 0.0131  -0.0260 0.0097  -0.0206 -0.0053 0.0158  -0.0481
POOL_P2_60_03RL POOL_P2_60_03RL 0       0       0       1       1       NA      NA      3       0.0120  0.0105  0.0506  -0.0095 -0.0124 -0.0044 0.0005  -0.0130 0.0393  -0.0497
POOL_P2_91_12RL POOL_P2_91_12RL 0       0       0       1       1       NA      NA      12      0.0177  0.0039  -0.0203 -0.0247 -0.0437 -0.0020 -0.0005 0.0200  0.0203  -0.0326
POOL_P2_86_10RL POOL_P2_86_10RL 0       0       0       1       NA      NA      NA      10      0.0111  0.0091  -0.0263 0.0054  0.0154  0.0027  -0.0567 -0.0416 -0.0390 0.0190
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm5.vs3.ped | head -n 150 | tail -n 10
POOL_P2_80_01RL POOL_P2_80_01RL 0       0       0       2       NA      NA      NA      1       0.0148  0.0059  -0.0233 -0.0426 0.0418  -0.0008 -0.0098 0.0051  -0.0213 0.0252
POOL_P2_154_06RL        POOL_P2_154_06RL        0       0       0       1       2       NA      NA      6       0.0164  0.0064  0.0211  0.0186  0.0552  -0.0092 -0.0025 -0.0114 0.0258  -0.0670
POOL_P2_158_12RL        POOL_P2_158_12RL        0       0       0       2       2       NA      NA      12      0.0138  0.0202  0.0158  0.0525  -0.0343 -0.0234 0.0504  0.0353  0.0001  0.0148
POOL_P2_166_09RL        POOL_P2_166_09RL        0       0       0       2       2       1       NA      9       0.0140  0.0055  -0.0177 0.0377  -0.0096 -0.0398 -0.0454 0.0344  0.0272  -0.0319
POOL_P2_149_02RL        POOL_P2_149_02RL        0       0       0       1       2       NA      NA      2       0.0112  0.0123  0.0139  0.0321  -0.0063 -0.0345 -0.0308 0.0064  0.0266  0.0154
POOL_P2_67_06RL POOL_P2_67_06RL 0       0       0       2       1       NA      NA      6       0.0145  0.0064  0.0266  -0.0201 -0.0388 0.0020  0.0063  -0.0134 0.0308  0.0171
POOL_P2_75_02RL POOL_P2_75_02RL 0       0       0       2       2       NA      NA      2       0.0106  0.0043  -0.0382 0.0131  -0.0260 0.0097  -0.0206 -0.0053 0.0158  -0.0481
POOL_P2_60_03RL POOL_P2_60_03RL 0       0       0       2       2       NA      NA      3       0.0120  0.0105  0.0506  -0.0095 -0.0124 -0.0044 0.0005  -0.0130 0.0393  -0.0497
POOL_P2_91_12RL POOL_P2_91_12RL 0       0       0       2       1       NA      NA      12      0.0177  0.0039  -0.0203 -0.0247 -0.0437 -0.0020 -0.0005 0.0200  0.0203  -0.0326
POOL_P2_86_10RL POOL_P2_86_10RL 0       0       0       2       NA      NA      NA      10      0.0111  0.0091  -0.0263 0.0054  0.0154  0.0027  -0.0567 -0.0416 -0.0390 0.0190
~~~

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs

#mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/*Pheno*run* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/.

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs





#20150214
#2nd run -- investigating applying a more stringent imiss threshold (just to see how it may or may not change results....this run may not be used, just investigating)
#Using .Vs2. in titles

#mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs2/GATK
#mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs2/GATK/PLINKfiles
#mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs2/GATK/EPACTSFiles

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.vcf.gz --geno .95 --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95

gzip -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz --hwe .0001 --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.recode.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf

#CHECK_0
#/home/shared/software/java/jre1.7.0_09/bin/java -Xmx10g -jar /home/shared/software/GATK/2.5.2/GenomeAnalysisTK.jar -T VariantAnnotator -R /home/michaelt/Data/HumanGenome/GRCh37/Version4/human_g1k_v37.fasta -I /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/AllPools.P2.Vs2.AllPoolsMerged.bam -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.HRun.vcf -A HomopolymerRun --variant /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf -nt 32

#mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.HRun.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf

#mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.HRun.vcf.idx /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.idx

gzip -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf

#zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | perl -lane 'my @vals1 = split(/;/, $F[7]); foreach my $SNPinfo (@vals1) { if ($SNPinfo =~ m/HRun/) { print $SNPinfo; } };' | sort | uniq -c 

#1 HRun=8
#1 HRun=9

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --plink --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --missing --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4

plink --file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4

#Final SNP-QC'ed .vcf created by this point
#Indv QC processes next

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.imiss | awk '{ if ($5 > .99) { print $1, "\t", $1 } } ' | grep -v INDV > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.imiss.Above99Miss.justFIDIIDs

#Dropping these individuals for --genome just so the program finishes running; with individuals that have >.99 missingness the program fails to run. This is consistent with P1 because no individuals had missingness >.99 so the program was not interrupted (also empirically no individuals were close to .99 missingness)
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --remove /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.imiss.Above99Miss.justFIDIIDs --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Indv99

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/VLookUp.Python.DataHapMap3Overlap.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.map --file2 /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.liftOver.hg18tohg19.map | awk '{ print $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.rsIDs 

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.rsIDs --recode --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3

plink --file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3 --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.map | awk '{ print $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.map.rsIDs

plink --bfile /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.map.rsIDs --keep /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.CEUCHBYRI.Unrelated.FIDIIDs --recode --out /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Array2.Vs2

plink --file /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Array2.Vs2 --make-bed /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Array2.Vs2

paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.bim /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Array2.Vs2.bim  | awk '{ if (($5 == $11) || ($5 == $12) ) { print $0 } } ' | awk '{ print $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.FinalListwDrops.rsIDs 

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3 --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.FinalListwDrops.rsIDs --recode --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.vs2

plink --bfile /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.FinalListwDrops.rsIDs --keep /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.CEUCHBYRI.Unrelated.FIDIIDs --recode --out /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Array2.Vs2.vs2

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.vs2.ped /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Array2.Vs2.vs2.ped > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.ped

plink --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.ped --map /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.vs2.map --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.phenoColEdit.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.phenoColEdit.pedind 

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.bim /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.pedsnp

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.parfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.parfile

/home/michaelt/Software/EIG5.0.1/bin/smartpca -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.parfile

#Troubleshooting
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_100_06RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_101_08RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_101_11RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_112_01RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_112_08RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_119_02RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_149_10RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_154_02RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_154_12RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_172_02RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_172_03RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_172_04RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_172_05RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_172_06RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_67_10RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_72_06RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_73_09RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_87_09RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_87_12RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_89_11RL)


python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.vs1.ForP2.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca | cat - <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca | grep -E 'POOL_P2_100_06RL|POOL_P2_101_08RL|POOL_P2_101_11RL|POOL_P2_112_01RL|POOL_P2_112_08RL|POOL_P2_119_02RL|POOL_P2_149_10RL|POOL_P2_154_02RL|POOL_P2_154_12RL|POOL_P2_172_02RL|POOL_P2_172_03RL|POOL_P2_172_04RL|POOL_P2_172_05RL|POOL_P2_172_06RL|POOL_P2_67_10RL|POOL_P2_71_03RL|POOL_P2_72_06RL|POOL_P2_73_09RL|POOL_P2_87_09RL|POOL_P2_87_12RL|POOL_P2_88_10RL|POOL_P2_89_11RL' | perl -lane '$F[$#F] = "NoPhenotype"; print join("\t", @F); ') > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.pt1

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.HapMap3vs.py --file1 /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.txt --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.pt2

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.pt1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.pt2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race

sed -i 's/American_Indian_or_Alaskan_Native/AIndian_or_Alaskan_Native/g' /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.plot.1vs2.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.plot.3vs4.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race -c 5:6 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.plot.5vs6.xtxt

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.plot.*.pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/.

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race | awk '{ if ((($2 >= -.001) && ($2 <= .22)) && (($3 >= -.005) && ($3 <= .021))) { print $0 } } ' | grep White_non-Hispanic > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed

~~~
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles]$cmp <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed | awk '{ print $1 }') <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed | awk '{ print $1 }')
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles]$paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed | awk '{ print $1 }') <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed | awk '{ print $1 }') | head -n 10
POOL_P2_100_01RL:POOL_P2_100_01RL       POOL_P2_100_01RL:POOL_P2_100_01RL
POOL_P2_100_03RL:POOL_P2_100_03RL       POOL_P2_100_03RL:POOL_P2_100_03RL
POOL_P2_100_05RL:POOL_P2_100_05RL       POOL_P2_100_05RL:POOL_P2_100_05RL
POOL_P2_100_09RL:POOL_P2_100_09RL       POOL_P2_100_09RL:POOL_P2_100_09RL
POOL_P2_100_10RL:POOL_P2_100_10RL       POOL_P2_100_10RL:POOL_P2_100_10RL
~~~

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed -c 1:2 -p White_non-Hispanic -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.plot.1vs2.xtxt

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.plot.*.pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/.

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed | perl -lane 'my @vals1 = split(/:/, $F[0]); print $vals1[0];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed | perl -lane 'my @vals1 = split(/:/, $F[0]); print $vals1[0], "\t", $vals1[0];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed

#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome | sort -rg -k 9,9 | head -n 20

~~~
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome | sort -rg -k 9,9 | head -n 20
POOL_P2_103_03RL  POOL_P2_103_03RL  POOL_P2_105_03RL  POOL_P2_105_03RL UN    NA  0.0000  0.0570  0.9430  0.9715  -1  0.998377  1.0000      NA
POOL_P2_101_01RL  POOL_P2_101_01RL  POOL_P2_104_01RL  POOL_P2_104_01RL UN    NA  0.0238  0.1964  0.7798  0.8780  -1  0.993273  1.0000 77.6667
POOL_P2_163_12RL  POOL_P2_163_12RL   POOL_P2_69_08RL   POOL_P2_69_08RL UN    NA  0.0409  0.2277  0.7314  0.8453  -1  0.991571  1.0000 67.0000
POOL_P2_140_10RL  POOL_P2_140_10RL   POOL_P2_87_11RL   POOL_P2_87_11RL UN    NA  0.0241  0.2550  0.7210  0.8485  -1  0.991593  1.0000 75.1667
POOL_P2_145_11RL  POOL_P2_145_11RL  POOL_P2_150_12RL  POOL_P2_150_12RL UN    NA  0.0456  0.2393  0.7151  0.8348  -1  0.991017  1.0000 67.5714
POOL_P2_164_05RL  POOL_P2_164_05RL   POOL_P2_86_04RL   POOL_P2_86_04RL UN    NA  0.0011  0.6676  0.3313  0.6651  -1  0.980924  1.0000      NA
POOL_P2_59_09RL   POOL_P2_59_09RL   POOL_P2_61_12RL   POOL_P2_61_12RL UN    NA  0.0101  0.6867  0.3032  0.6466  -1  0.979954  1.0000 164.3333
POOL_P2_155_08RL  POOL_P2_155_08RL   POOL_P2_66_03RL   POOL_P2_66_03RL UN    NA  0.7268  0.0000  0.2732  0.2732  -1  0.965022  0.1030  1.3684
POOL_P2_66_03RL   POOL_P2_66_03RL   POOL_P2_93_07RL   POOL_P2_93_07RL UN    NA  0.7278  0.0000  0.2722  0.2722  -1  0.964485  0.1792  1.5263
POOL_P2_66_03RL   POOL_P2_66_03RL   POOL_P2_97_03RL   POOL_P2_97_03RL UN    NA  0.7384  0.0000  0.2616  0.2616  -1  0.963792  0.3400  1.7647

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | grep -E 'POOL_P2_103_03RL|POOL_P2_105_03RL'
POOL_P2_103_03RL        POOL_P2_103_03RL        2       2       -9      -9
POOL_P2_105_03RL        POOL_P2_105_03RL        2       2       -9      -9

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | grep -E 'POOL_P2_101_01RL|POOL_P2_104_01RL'
POOL_P2_101_01RL        POOL_P2_101_01RL        1       1       -9      -9
POOL_P2_104_01RL        POOL_P2_104_01RL        1       1       -9      -9

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | grep -E 'POOL_P2_163_12RL|POOL_P2_69_08RL'
POOL_P2_69_08RL POOL_P2_69_08RL 1       1       -9      -9
POOL_P2_163_12RL        POOL_P2_163_12RL        1       -9      -9      -9

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | grep -E 'POOL_P2_140_10RL|POOL_P2_87_11RL'
POOL_P2_87_11RL POOL_P2_87_11RL 1       -9      -9      -9
POOL_P2_140_10RL        POOL_P2_140_10RL        2       2       -9      -9

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | grep -E 'POOL_P2_145_11RL|POOL_P2_150_12RL'
POOL_P2_150_12RL        POOL_P2_150_12RL        2       2       1       -9
POOL_P2_145_11RL        POOL_P2_145_11RL        2       2       1       -9
~~~

perl -e 'print "POOL_P2_103_03RL\nPOOL_P2_105_03RL\nPOOL_P2_101_01RL\nPOOL_P2_104_01RL\nPOOL_P2_163_12RL\nPOOL_P2_69_08RL\nPOOL_P2_140_10RL\nPOOL_P2_87_11RL\nPOOL_P2_145_11RL\nPOOL_P2_150_12RL\n";' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.FIDs

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.FIDs --depth --missing --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs

~~~
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.idepth
INDV    N_SITES MEAN_DEPTH
POOL_P2_101_01RL        94775   11.7298
POOL_P2_103_03RL        94881   18.6866
POOL_P2_104_01RL        93535   6.26491
POOL_P2_105_03RL        94879   18.3223
POOL_P2_140_10RL        92567   5.24853
POOL_P2_145_11RL        94480   9.1442
POOL_P2_150_12RL        93373   5.90921
POOL_P2_163_12RL        94183   7.61423
POOL_P2_69_08RL 94066   8.19753
POOL_P2_87_11RL 94851   14.0026

[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.imiss
INDV    N_DATA  N_GENOTYPES_FILTERED    N_MISS  F_MISS
POOL_P2_101_01RL        94889   0       114     0.0012014
POOL_P2_103_03RL        94889   0       8       8.4309e-05
POOL_P2_104_01RL        94889   0       1354    0.0142693
POOL_P2_105_03RL        94889   0       10      0.000105386
POOL_P2_140_10RL        94889   0       2322    0.0244707
POOL_P2_145_11RL        94889   0       409     0.0043103
POOL_P2_150_12RL        94889   0       1516    0.0159766
POOL_P2_163_12RL        94889   0       706     0.00744027
POOL_P2_69_08RL 94889   0       823     0.00867329
POOL_P2_87_11RL 94889   0       38      0.000400468
~~~

So choosing:
POOL_P2_105_03RL
POOL_P2_101_01RL
POOL_P2_145_11RL

perl -e 'print "POOL_P2_103_03RL\nPOOL_P2_104_01RL\nPOOL_P2_140_10RL\nPOOL_P2_150_12RL\nPOOL_P2_163_12RL\nPOOL_P2_69_08RL\nPOOL_P2_87_11RL\n";' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome | sort -rg -k 9,9 | head -n 10000 | awk '{ print $1, "\n", $3 }' | perl -lane 'chomp(@F); print join("\t", @F);' | sort | uniq -c | sort -g -k 1,1 | awk '{ if ($1 >= 110 ) { print $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ2Countgte110.FIDs  

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome | sort -rg -k 8,8 | head -n 10000 | awk '{ print $1, "\n", $3 }' | perl -lane 'chomp(@F); print join("\t", @F);' | sort | uniq -c | sort -g -k 1,1 | awk '{ if ($1 >= 100) { print $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.FIDs

#NOTE -- dropping these individuals because they are starkly greater than anyone else in the list. Individuals from the 'high z2' count list are not being dropped because they appear as just to be the tail of a distribution versus being such a group of outliers

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs

cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ2Countgte110.FIDs | perl -lane 'push(@F, "Z2"); print join("\t", @F);') <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.FIDs | perl -lane 'push(@F, "Z1"); print join("\t", @F);') > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.OddHighZ1OrZ2.FID_ZNums

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race | awk 'BEGIN{ while(getline < "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.OddHighZ1OrZ2.FID_ZNums") { FID = $1":"$1; FIDList[FID] = $2; }} { if (FIDList[$1]) { if (FIDList[$1] == "Z1") { $12 = "Z1High"; } else if (FIDList[$1] == "Z2") { $12 = "Z2High"; } else{ print "blah" }  } else { $12 = "NormalIBD"; } print $0; } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.IBDissuesTagged 

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.IBDissuesTagged -c 1:2 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.IBDissuesTagged.plot.1vs2.xtxt

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed | awk 'BEGIN{ while(getline < "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.OddHighZ1OrZ2.FID_ZNums") { FID = $1":"$1; FIDList[FID] = $2; }} { if (FIDList[$1]) { if (FIDList[$1] == "Z1") { $12 = "Z1High"; } else if (FIDList[$1] == "Z2") { $12 = "Z2High"; } else{ print "blah" }  } else { $12 = "NormalIBD"; } print $0; } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 1:2 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.1vs2.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 3:4 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.3vs4.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 5:6 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.5vs6.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 7:8 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.7vs8.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 9:10 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.9vs10.xtxt

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.*pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/.

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.covar > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.5PCs.covar

~~~
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.5PCs.covar | wc
890    8900   69165
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.5PCs.covar | wc
889    8890   69081
~~~

#Edit code in between runs
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.covar > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar

~~~
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles]$head -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar
POOL_P2_137_06RL        POOL_P2_137_06RL        137     6       3       -0.0601 0.0109  0.0150  0.0031  0.0413  -0.0396 0.0464  0.0252  -0.0499 -0.0633
POOL_P2_83_08RL POOL_P2_83_08RL 83      8       1       0.0156  0.0127  -0.0161 -0.0306 -0.0460 0.0286  0.0071  -0.0276 0.0150  -0.0125
POOL_P2_156_08RL        POOL_P2_156_08RL        156     8       1       0.0126  0.0086  -0.0303 0.0166  -0.0120 -0.0239 -0.0156 -0.0252 -0.0331 -0.0006
POOL_P2_85_09RL POOL_P2_85_09RL 85      9       2       0.0023  -0.0281 0.0279  0.0265  -0.0593 0.0079  -0.0928 0.0291  -0.0548 0.0129
POOL_P2_147_06RL        POOL_P2_147_06RL        147     6       3       -0.0632 0.0108  0.0065  0.0254  -0.0178 0.0509  0.0797  -0.0028 -0.0256 0.0074
POOL_P2_135_03RL        POOL_P2_135_03RL        135     3       1       0.0088  0.0041  0.0147  0.0344  0.0045  -0.0063 0.0018  -0.0247 0.0196  -0.0216
POOL_P2_141_03RL        POOL_P2_141_03RL        141     3       1       0.0128  0.0106  -0.0328 0.0322  -0.0089 -0.0540 -0.0393 0.0170  0.0503  0.0290
POOL_P2_157_04RL        POOL_P2_157_04RL        157     4       1       0.0124  0.0006  -0.0328 0.0068  0.0043  0.0227  -0.0153 -0.0068 -0.0462 -0.0168
POOL_P2_131_05RL        POOL_P2_131_05RL        131     5       1       0.0134  0.0059  0.0123  0.0390  0.0145  -0.0268 0.0012  0.0261  -0.0256 0.0273
POOL_P2_88_02RL POOL_P2_88_02RL 88      2       1       0.0116  0.0033  -0.0360 0.0206  0.0383  -0.0404 -0.0117 -0.0159 0.0066  0.0251
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles]$tail -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar
POOL_P2_163_10RL        POOL_P2_163_10RL        163     10      1       0.0138  0.0081  0.0540  0.0400  -0.0431 -0.0153 0.0136  -0.0065 0.0269  0.0156
POOL_P2_175_01RL        POOL_P2_175_01RL        175     1       1       0.0027  0.0034  0.0113  0.0076  0.0042  0.0168  0.0104  0.0005  -0.0106 0.0102
POOL_P2_57_04RL POOL_P2_57_04RL 57      4       1       0.0139  0.0090  0.0151  0.0513  -0.0042 -0.0391 0.0460  0.0427  -0.0206 -0.0140
POOL_P2_135_06RL        POOL_P2_135_06RL        135     6       1       0.0114  0.0113  0.0157  -0.0421 0.0131  0.0041  -0.0043 0.0104  -0.0126 -0.0211
POOL_P2_167_03RL        POOL_P2_167_03RL        167     3       3       -0.0373 0.0066  -0.0036 -0.0227 0.0269  0.0191  -0.0293 -0.0145 -0.0032 0.0271
POOL_P2_161_12RL        POOL_P2_161_12RL        161     12      1       0.0114  0.0051  0.0079  0.0162  0.0471  0.0437  0.0110  -0.0227 -0.0313 -0.0262
POOL_P2_88_11RL POOL_P2_88_11RL 88      11      2       0.0014  -0.0190 0.0395  -0.0262 0.0197  -0.0256 -0.0052 0.0645  0.0065  0.0502
POOL_P2_62_11RL POOL_P2_62_11RL 62      11      1       0.0158  0.0099  -0.0148 -0.0108 -0.0054 0.0104  0.0120  0.0468  -0.0107 0.0272
POOL_P2_82_02RL POOL_P2_82_02RL 82      2       1       0.0111  0.0044  -0.0247 0.0102  0.0175  -0.0198 -0.0190 -0.0370 -0.0294 -0.0593
POOL_P2_173_01RL        POOL_P2_173_01RL        173     1       1       0.0084  0.0039  0.0102  0.0066  0.0002  0.0389  -0.0032 0.0030  0.0288  0.0046
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles]$tail -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar
POOL_P2_163_10RL        POOL_P2_163_10RL        163     10      1       0.0137  0.0081  0.0540  0.0399  -0.0426 -0.0161 0.0133  -0.0057 0.0275  0.0134
POOL_P2_175_01RL        POOL_P2_175_01RL        175     1       1       0.0027  0.0035  0.0112  0.0077  0.0038  0.0172  0.0103  0.0004  -0.0102 0.0110
POOL_P2_57_04RL POOL_P2_57_04RL 57      4       1       0.0139  0.0089  0.0152  0.0513  -0.0036 -0.0389 0.0455  0.0418  -0.0215 -0.0127
POOL_P2_135_06RL        POOL_P2_135_06RL        135     6       1       0.0114  0.0112  0.0158  -0.0421 0.0132  0.0042  -0.0037 0.0099  -0.0143 -0.0211
POOL_P2_167_03RL        POOL_P2_167_03RL        167     3       3       -0.0370 0.0066  -0.0035 -0.0226 0.0263  0.0195  -0.0307 -0.0125 -0.0016 0.0295
POOL_P2_161_12RL        POOL_P2_161_12RL        161     12      1       0.0114  0.0051  0.0079  0.0164  0.0461  0.0446  0.0108  -0.0239 -0.0310 -0.0235
POOL_P2_88_11RL POOL_P2_88_11RL 88      11      2       0.0014  -0.0188 0.0394  -0.0265 0.0194  -0.0251 -0.0056 0.0647  0.0063  0.0488
POOL_P2_62_11RL POOL_P2_62_11RL 62      11      1       0.0157  0.0099  -0.0147 -0.0106 -0.0055 0.0104  0.0118  0.0472  -0.0113 0.0257
POOL_P2_82_02RL POOL_P2_82_02RL 82      2       1       0.0111  0.0042  -0.0245 0.0101  0.0180  -0.0200 -0.0182 -0.0384 -0.0303 -0.0577
POOL_P2_173_01RL        POOL_P2_173_01RL        173     1       1       0.0083  0.0039  0.0101  0.0070  -0.0010 0.0391  -0.0028 0.0032  0.0291  0.0023
~~~

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss | awk '{ if ($5 >= .2) { print $1 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.pheno 

~~~
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.pheno | wc
747    4482   32288
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.pheno | wc
756    4536   32680
~~~

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar | awk '{ print $1, "\t", $2 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar | awk '{ print $1 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDs

#CHECK_0
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDs  --non-ref-ac 1 --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.recode.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf

gzip -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.allSNPs
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar --covar-number 2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.10PCs
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar --covar-number 2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.10PCs.maf05
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 2 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno2
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 3 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno3
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 4 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno4

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.assoc.logistic Pheno1_justWhite_Logistic_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.assoc.logistic.QQplot.jpeg
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.assoc.logistic Pheno1_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.assoc.logistic.QQplot.jpeg
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.10PCs.assoc.logistic Pheno1_justWhite_Logistic_10PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.10PCs.assoc.logistic.QQplot.jpeg
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.10PCs.maf05.assoc.logistic Pheno1_justWhite_Logistic_maf05_10PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.10PCs.maf05.assoc.logistic.QQplot.jpeg
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno2.assoc.logistic Pheno2_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno2.assoc.logistic.QQplot.jpeg
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno3.assoc.logistic Pheno3_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno3.assoc.logistic.QQplot.jpeg
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno4.assoc.logistic Pheno4_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno4.assoc.logistic.QQplot.jpeg

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/.

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ print $1 }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.FIDs 
~~~
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.FIDs | wc
747     747   12415
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.FIDs | wc
756     756   12567
~~~
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ if ($3 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroNeg.FIDs 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ if ($4 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroHighlyExposedNeg.FIDs 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ if ($3 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroPos.FIDs 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ if ($5 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justRapidVRapid.FIDs 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ if ($5 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSlowVSlow.FIDs 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ if ($6 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVeryRapid.FIDs 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ if ($6 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVerySlow.FIDs 

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80 &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroNeg.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroHighlyExposedNeg.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroPos.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justRapidVRapid.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSlowVSlow.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVeryRapid.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVerySlow.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80 &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroHighlyExposedNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroPos.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justRapidVRapid.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSlowVSlow.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVeryRapid.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVerySlow.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow & 

paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow.frq.count | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.AllPhenos.frq.AndCount.gz
gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow.frq.count 
#gunzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow.frq.count.gz 

#Technically I cannot create the first file below because frequencies are only for the QC'ed, white subset individuals
##python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.AllPhenos.frq.AndCount.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 30; $i <= 40; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs6.txt.gz 
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.AllPhenos.frq.AndCount.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 30; $i <= 40; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz 

~~~
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v \# | wc
85200 64411200 1485394397
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | wc
87510   97379 38910992
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/\_2/) { print join("\t", @F); }' | wc
2272   74181 1012838
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/\_3/) { print join("\t", @F); }' | wc
35    1133   15436
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/\_4/) { print join("\t", @F); }' | wc
2      64     844
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/\_5/) { print join("\t", @F); }' | wc
1      32     412
~~~

R -q -e 'val1 = 2272 + 35 + 2 + 1; print (val1);'
> val1 = 2272 + 35 + 2 + 1; print (val1);
[1] 2310
>
>


R -q -e 'val1 = 2272 + 35 + 2 + 1; print(val1); val2 = 87510 - 2310; print(val2);'
> val1 = 2272 + 35 + 2 + 1; print(val1); val2 = 87510 - 2310; print(val2);
[1] 2310
[1] 85200
>
>

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80 

##python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs3.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs6.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs3.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing") || ($7 == "UTR5;UTR3")) { print $8, "\t", $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.All
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "splicing") || ($7 == "exonic;splicing")) { print $8, "\t", $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.Exonic
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") )  { print $8, "\t", $2 } } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if ((($7 == "exonic") && (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV"))) || (($7 == "splicing") || ($7 == "exonic;splicing"))) { print $8, "\t", $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.wSplice
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($12 == "P") || ($12 == "D") ) { print $8, "\t", $2 } } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_PD
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $8, "\t", $2 } } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_D

~~~
[  michaelt@node6  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if ((($7 == "exonic") && (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV"))) || (($7 == "splicing") || ($7 == "exonic;splicing"))) { print $9 } } ' | sort | uniq -c
67
5296 nonsynonymous_SNV
119 stopgain_SNV                                                                                                                                                                                                                                                                                                       
10 stoploss_SNV
1 synonymous_SNV 
1 unknown
[  michaelt@node6  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if ((($7 == "exonic") && (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV"))) || (($7 == "splicing") || ($7 == "exonic;splicing"))) { print $7 } } ' | sort | uniq -c
5423 exonic
4 exonic;splicing
67 splicing
[  michaelt@node6  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if ((($7 == "exonic") && (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV"))) || (($7 == "splicing") || ($7 == "exonic;splicing"))) { print $0 } } ' | grep exonic\;splicing
7,rs114540180,0,150325378,T,C,exonic;splicing,GIMAP6,nonsynonymous_SNV,NM_001244072:c.A518G:p.Q173R,T,B,N,N
9,9:136521661,0,136521661,T,C,exonic;splicing,DBH,nonsynonymous_SNV,NM_000787:c.T1451C:p.L484P,T,B,D,N
15,rs75010033,0,101905203,C,T,exonic;splicing,PCSK6,unknown,UNKNOWN,,,,
20,rs139806443,0,56188385,T,C,exonic;splicing,ZBP1,synonymous_SNV,NM_001160418:c.A279G:p.E93E,,,,
~~~

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing") || ($7 == "UTR5;UTR3")) { print $0 } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.All 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "splicing") || ($7 == "exonic;splicing")) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $0 } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if ((($7 == "exonic") && (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV"))) || (($7 == "splicing") || ($7 == "exonic;splicing"))) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($12 == "P") || ($12 == "D") ) { print $0 } } } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $0 } } } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($13 == "D") { print $0 } } } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing") || ($7 == "UTR5;UTR3")) { print $0 } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All 
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "splicing") || ($7 == "exonic;splicing")) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $0 } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if ((($7 == "exonic") && (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV"))) || (($7 == "splicing") || ($7 == "exonic;splicing"))) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.wSplices
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($12 == "P") || ($12 == "D") ) { print $0 } } } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $0 } } } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_D
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.Sift_D
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($13 == "D") { print $0 } } } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.LRT_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.pheno | awk '{ print $1, "\t", $2, "\t0\t0\t0\t", $3, "\t", $4, "\t", $5, "\t", $6 } ' | sed 's/-9/NA/g' | perl -lane 'print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit.ped 

#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar | awk '{ print $4, "\t", $6, "\t", $7 "\t", $8, "\t", $9, "\t", $10, "\t", $11,  "\t", $12, "\t", $13, "\t", $14,  "\t", $15 }' | paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit.ped  -  | perl -lane 'print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit.ped.vs3 
join /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar | perl -lane 'splice(@F, 9, 2); splice(@F, 10, 1); print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit.ped.vs2 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit.ped.vs2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit.ped

~~~
[  michaelt@node6  ~]$cmp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit.ped.vs3
[  michaelt@node6  ~]$                                                                                                                                                  
~~~

#Add line #FAM_ID IND_ID FAT_ID MOT_ID SEX HIVPROG HIVPROGHE AIDS AIDSEXTR RMID PC1 PC2 PC3 PC4 PC5 PC6 PC7 PC8 PC9 PC10 to /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit.ped 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | awk '{ if ($1 == "23") { $1 = "X"} print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip

/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz

perl /home/michaelt/Scripts/Perl/RandomList.pl --total 10000 --seed 98560234 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.PermutationList.ForP2.vs1.txt

for i in {1..1000}
do
	SEED1=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.PermutationList.ForP2.vs1.txt | awk '{ if (NR == '$i') { print $0 } } '`
	perl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs3.pl --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit.ped --seed $SEED1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit.perm${i}.vs3.ped

done

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs 

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT 
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/SUMSTAT 
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/SUMSTAT 
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/SUMSTAT 
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT 
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/SUMSTAT 
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/SUMSTAT 
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/SUMSTAT 

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs TruePheno Pheno1 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs TruePheno Pheno2 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs TruePheno Pheno3 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs TruePheno Pheno4 Exonic.Nonsynonymous

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs TruePheno Pheno1 Exonic.Nonsynonymous OLD1
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs TruePheno Pheno2 Exonic.Nonsynonymous OLD1
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs TruePheno Pheno3 Exonic.Nonsynonymous OLD1
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs TruePheno Pheno4 Exonic.Nonsynonymous OLD1

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs FakePheno Pheno1 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs FakePheno Pheno2 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs FakePheno Pheno3 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs FakePheno Pheno4 Exonic.Nonsynonymous

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs FakePheno Pheno1 Exonic.Nonsynonymous OLD1
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs FakePheno Pheno2 Exonic.Nonsynonymous OLD1
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs FakePheno Pheno3 Exonic.Nonsynonymous OLD1
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs FakePheno Pheno4 Exonic.Nonsynonymous OLD1

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno1 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno2 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno3 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno4 Exonic.Nonsynonymous

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno1 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno2 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno3 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno4 Exonic.Nonsynonymous

for i in {1..1000}
do 
        if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm${i}.epacts ] ; then
		echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno1.Exonic.Nonsynonymous
	fi
        if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm${i}.epacts ] ; then
		echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno2.Exonic.Nonsynonymous
	fi
        if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm${i}.epacts ] ; then
		echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno3.Exonic.Nonsynonymous
	fi
        if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm${i}.epacts ] ; then
		echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno4.Exonic.Nonsynonymous
	fi
done

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.QQPlotUsingPermutations.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs2.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg P2_HIVAcquisition_Nonsyn_SkatO 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.QQPlotUsingPermutations.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs2.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg P2_HIVAcquisitionHE_Nonsyn_SkatO 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.QQPlotUsingPermutations.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs2.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg P2_AIDSProgression_Nonsyn_SkatO 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.QQPlotUsingPermutations.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs2.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg P2_AIDSProgressionExtr_Nonsyn_SkatO 

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/.

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/.

#mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/WRONG1.AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs
#mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/OLD1.NoRecodeInfoAll.AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs2.sh 
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs2.sh 
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs2.sh 
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs2.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs2.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno1 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs2.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno2 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs2.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno3 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs2.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno4 Exonic.Nonsynonymous

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs2.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno1 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs2.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno2 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs2.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno3 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs2.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno4 Exonic.Nonsynonymous

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Results
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Results/Checkpoints
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Results/Checkpoints/2ndRun
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Results/Checkpoints/2ndRun/20150302

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno*/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno*.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno*/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno*.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno*/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno*.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Results/Checkpoints/2ndRun/20150302/.

#Should have done this with all four phenotypes but only did with first phenotype, accidentally clobbered the remaining three. Moved all the MM5 files though correctly so can see the before/after there, and if need be can recreate the original files by using the 'gene+var groupings' based on the improper .bim.wGenes file from the wrong annovar file
#mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/WRONG1.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts

~~~
1150  cat TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ print $1 }' > jndfgondfg
1151  cat TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -vF jndfgondfg 
1152  cat TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -vF jndfgondfg  | wc
1153  cat TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -v -F jndfgondfg  
1154  cat TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -v -F jndfgondfg  | wc
1155  cat jndfgondfg | head -n 10
1156  grep --help
1157  cat TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -v -f jndfgondfg  | wc
1158  cat TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -v -f jndfgondfg
1159  cat TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output | grep CTGF
1160  cat TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output | grep LUZP6
1161  history
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations]$cat TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -v -f jndfgondfg
ID
DNAJC7
HS3ST4
AICDA
IFITM2
CALM2
SFPQ
TIMP3
RNF5
CTGF
NCS1
LUZP6
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations]$cat TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output | grep CTGF
CTGF    NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NANA       NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NANA       NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations]$cat TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output | grep LUZP6
LUZP6   NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NANA       NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NANA       NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA
~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.permAll.epacts.NullDistrQQPlotCheck.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.permAll.epacts.NullDistrQQPlotCheck.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.permAll.epacts.NullDistrQQPlotCheck.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.permAll.epacts.NullDistrQQPlotCheck.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/.

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/.

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justpVals /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justpVals /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justpVals /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justpVals /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/.

#20150330
#Retroactively including code for these 'gene/variant level result summary outputs', following the logic of excel sheet exploration, here

/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplicecat

echo ".5" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".6" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".7" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".8" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".85" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".9" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .9) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .9) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`

~~~
[  michaelt@node1  ~]$echo ".5" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.5 24 184
[  michaelt@node1  ~]$echo ".6" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'` 
.6 51 400
[  michaelt@node1  ~]$echo ".7" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'` 
.7 91 654
[  michaelt@node1  ~]$echo ".8" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'` 
.8 135 1011
[  michaelt@node1  ~]$echo ".85" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'` 
.85 188 1384
[  michaelt@node1  ~]$echo ".9" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .9) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .9) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'` 
.9 376 2763
~~~

#GetperFastQC
#GetperRegionpostQC
#GetperFullBam
#GetVCF? for all indvs, just white indvs, then subgroups
#for full file, for per gene, for per indv (and then subgroups)

#QC Stuff

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetFastqQCStats.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output

#CHECK_0: Github backup/hard-copy current QC output files for P1 and P2

#Removed empty lines in /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetFastqQCStats.vs2.20150315_234317.output with /0.00000.0000 0.0000 and dd in vi 
#PreMerge IEUL1WW01.RL4 0.00000.0000 0.0000               totalDropped(%):   minDropped(%):   UpperBoundThresh:  maxDropped(%):
#PreMerge IEUL1WW01.RL5 0.00000.0000 0.0000               totalDropped(%):   minDropped(%):   UpperBoundThresh:  maxDropped(%):
#PreMerge IEUL1WW01.RL6 0.00000.0000 0.0000               totalDropped(%):   minDropped(%):   UpperBoundThresh:  maxDropped(%):
#PreMerge IEUL1WW01.RL7 0.00000.0000 0.0000               totalDropped(%):   minDropped(%):   UpperBoundThresh:  maxDropped(%):
#PreMerge IEUL1WW01.RL9 0.00000.0000 0.0000               totalDropped(%):   minDropped(%):   UpperBoundThresh:  maxDropped(%):
#PreMerge IEUL1WW01.RL10 0.00000.0000 0.0000               totalDropped(%):   minDropped(%):   UpperBoundThresh:  maxDropped(%):
#PreMerge IEUL1WW01.RL11 0.00000.0000 0.0000               totalDropped(%):   minDropped(%):   UpperBoundThresh:  maxDropped(%):
#PreMerge IEUL1WW01.RL12 0.00000.0000 0.0000               totalDropped(%):   minDropped(%):   UpperBoundThresh:  maxDropped(%):

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetFastqQCStats.vs2.20150315_234317.output | R -q -e "Data1 <- read.table(file('stdin'), header=F); sum(as.numeric(Data1[,4])); sum(as.numeric(Data1[,6]));"
[1] 172890654704
[1] 288065985
>
>

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetFastqQCStats.vs2.20150315_234317.output | wc
   3776  109504  828231

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output.GetperRegionpostQCStats.vs3.20150316_\ 00554.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output.GetperRegionpostQCStats.vs3.20150316_00554.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output.GetperRegionpostQCStats.vs3.20150316_00554.output | perl -lane 'my @array1; push(@array1, $F[1]); foreach my $entry (@F) { if ($entry !~ m/[a-zA-Z]/) { push (@array1, $entry); } } print join(",", @array1);' | awk 'BEGIN{ print "ID,TotalReadsPreQC,TotalReadsQ10,TotalReadsQ10Prcnt,TotalReadsF4,TotalReadsF4Prcnt,TotalReadsF256,TotalReadsF256Prcnt,TotalReadsPostQC,TotalReadsDropped,TotalReadsDroppedPrcnt,AverageReadLengthPreQC,AverageReadLengthPostQC,BedFileCoveragePostQC" } { print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output.GetperRegionpostQCStats.vs3.20150316_00554.output.reformatted

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs3.ForP2.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs3.ForP2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected.GetperFullBampostQCStats.vs3.ForP2.20150316_02534.output | perl -lane 'my @array1; push(@array1, $F[1]); foreach my $entry (@F) { if ($entry !~ m/[a-zA-Z]/) { push (@array1, $entry); } } print join(",", @array1);' | awk 'BEGIN{ print "ID,NumberOfOriginalBamFiles,TotalReadsPreQC,TotalReadsPreQCForward,TotalReadsPreQCReverse,TotalReadsPreQCForward_Reverse,PreQCMAPQmean,PreQCMAPQsd,PreQCMean,PreQCSD,PreQCMax,PreQCMin,CoveragePreQC,TotalReadsPostRmdup,TotalReadsPostRmdupDropped,TotalReadsPostRmdupDroppedPercentage,TotalReadsPostQC,TotalReadsPostQCForward,TotalReadsPostQCReverse,TotalReadsPostQCForward_Reverse,PostQCMAPQmean,PostQCMAPQsd,PostQCMean,PostQCSD,PostQCMax,PostQCMin,CoveragePostQC" } { print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected.GetperFullBampostQCStats.vs3.ForP2.20150316_02534.output.reformatted 

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetFastqQCStats.vs2.20150315_234317.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output.GetperRegionpostQCStats.vs3.20150316_00554.output.reformatted /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected.GetperFullBampostQCStats.vs3.ForP2.20150316_02534.output.reformatted 

~~~
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations]$/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetFastqQCStats.vs2.20150315_234317.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output.GetperRegionpostQCStats.vs3.20150316_00554.output.reformatted /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected.GetperFullBampostQCStats.vs3.ForP2.20150316_02534.output.reformatted
[1] 6.75321
[1] 6.294703
[1] 0.2864592
[1] 0.9647855
null device
	1
[1] 254981
[1] 617.3847
[1] 239.1003
[1] 13.57618
[1] 38316.19
[1] 12.64409
null device
	1
~~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/OutputFiles/Graphics/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs2.2015-03-16_00:48:03.SummaryRegionPlots.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/OutputFiles/Graphics/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/OutputFiles/Graphics/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs1.vs2.2015-03-16_00:48:06.SummaryIndvPlots.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/OutputFiles/Graphics/.

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz 
#Edited code in /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs2.py
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.GetVCFQCStats.vs2.output

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.GetVCFQCStats.vs2.output

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz | grep exonic | grep -v ncRNA | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.GetVCFQCStats.vs2.output

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.PerGene.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.PerGene.vs2.py

#python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.PerGene.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.perGene.vs2.output
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz | grep -v intergenic | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.PerGene.vs2.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.perGene.vs2.output

#Coverage/DP is column 18

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz | grep exonic | grep -v ncRNA | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.PerGene.vs2.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.Exonic.perGene.vs2.output

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs3.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz | grep -E '^#|exonic' | grep -v ncRNA | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs3.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroNeg 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroHighlyExposedNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroHighlyExposedNeg 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroPos.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroPos 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justRapidVRapid.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justRapidVRapid 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSlowVSlow.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSlowVSlow 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVeryRapid.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justVeryRapid 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVerySlow.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justVerySlow 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroNeg 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroHighlyExposedNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroHighlyExposedNeg 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroPos.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroPos 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justRapidVRapid.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justRapidVRapid 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSlowVSlow.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSlowVSlow 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVeryRapid.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justVeryRapid 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVerySlow.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justVerySlow 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 12.8567603748327 MeanDoubletons: 21.6559571619813 MeanCommon: 6842.5890227577 MeanRare: 593.401606425703 MeanHeterozy: 4661.906291834 MeanCoverage: 12.5231536370384 sdCoverage: 4.89279817826893 Exonic: 653.61311914324 Intronic: 5008.33065595716 Nonsynonymous: 257.259705488621 Synonymous: 390.409638554217 PP2_D: 28.1954484605087 PP2_P: 18.7028112449799 Sift_D: 41.3373493975904 LRT_D: 30.6398929049531"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 12.4660766961652 MeanDoubletons: 21.7256637168142 MeanCommon: 6828.32448377581 MeanRare: 596.554572271386 MeanHeterozy: 4648.38348082596 MeanCoverage: 12.3044067539243 sdCoverage: 4.93589584565892 Exonic: 654.171091445428 Intronic: 5000.91150442478 Nonsynonymous: 257.262536873156 Synonymous: 390.896755162242 PP2_D: 28.0766961651917 PP2_P: 18.6873156342183 Sift_D: 41.7079646017699 LRT_D: 30.5486725663717"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroHighlyExposedNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 12.796178343949 MeanDoubletons: 21.0382165605096 MeanCommon: 6830.45859872611 MeanRare: 586.955414012739 MeanHeterozy: 4643.3949044586 MeanCoverage: 12.8067167758079 sdCoverage: 4.95028888995383 Exonic: 653.949044585987 Intronic: 4996.75159235669 Nonsynonymous: 257.757961783439 Synonymous: 390.235668789809 PP2_D: 27.6624203821656 PP2_P: 18.6751592356688 Sift_D: 41.5923566878981 LRT_D: 30.4458598726115"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 13.1813725490196 MeanDoubletons: 21.5980392156863 MeanCommon: 6854.44117647059 MeanRare: 590.781862745098 MeanHeterozy: 4673.14215686274 MeanCoverage: 12.7049065619788 sdCoverage: 4.85525920103541 Exonic: 653.149509803922 Intronic: 5014.49509803922 Nonsynonymous: 257.257352941176 Synonymous: 390.004901960784 PP2_D: 28.2941176470588 PP2_P: 18.7156862745098 Sift_D: 41.0294117647059 LRT_D: 30.7156862745098"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justRapidVRapid | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 12.7777777777778 MeanDoubletons: 20.3333333333333 MeanCommon: 6868.83333333333 MeanRare: 579.740740740741 MeanHeterozy: 4691.14814814815 MeanCoverage: 13.1737065707157 sdCoverage: 3.78571833702897 Exonic: 652.111111111111 Intronic: 5019.25925925926 Nonsynonymous: 255.351851851852 Synonymous: 390.666666666667 PP2_D: 27.8703703703704 PP2_P: 18.3333333333333 Sift_D: 39.9814814814815 LRT_D: 29.5740740740741"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSlowVSlow | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 13.0246913580247 MeanDoubletons: 22.6049382716049 MeanCommon: 6866.0987654321 MeanRare: 598.456790123457 MeanHeterozy: 4689.83950617284 MeanCoverage: 12.8425294439885 sdCoverage: 6.90751896295189 Exonic: 656.444444444444 Intronic: 5027.16049382716 Nonsynonymous: 259.679012345679 Synonymous: 391.074074074074 PP2_D: 28.4938271604938 PP2_P: 18.7037037037037 Sift_D: 41.8765432098765 LRT_D: 31.1604938271605"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justVeryRapid | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 14.3809523809524 MeanDoubletons: 19.6666666666667 MeanCommon: 6923.42857142857 MeanRare: 573.714285714286 MeanHeterozy: 4751.57142857143 MeanCoverage: 13.398101176621 sdCoverage: 3.24540374835087 Exonic: 651.333333333333 Intronic: 5064.47619047619 Nonsynonymous: 257.333333333333 Synonymous: 387.952380952381 PP2_D: 29 PP2_P: 18.8095238095238 Sift_D: 40.6190476190476 LRT_D: 31.4285714285714"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justVerySlow | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 13.1578947368421 MeanDoubletons: 21 MeanCommon: 6848.76315789474 MeanRare: 595.631578947368 MeanHeterozy: 4674.86842105263 MeanCoverage: 14.4278295823871 sdCoverage: 9.32553269310467 Exonic: 653.5 Intronic: 5009.71052631579 Nonsynonymous: 259.184210526316 Synonymous: 388.921052631579 PP2_D: 28.1842105263158 PP2_P: 18.4473684210526 Sift_D: 41.6052631578947 LRT_D: 30.6842105263158"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 1.95850066934404 MeanDoubletons: 3.33868808567604 MeanCommon: 591.321285140562 MeanRare: 62.3092369477912 MeanHeterozy: 403.706827309237 MeanCoverage: 14.8163023694426 sdCoverage: 5.89911671487659 Exonic: 653.61311914324 Intronic: 0 Nonsynonymous: 257.259705488621 Synonymous: 390.409638554217 PP2_D: 28.1954484605087 PP2_P: 18.7028112449799 Sift_D: 41.0883534136546 LRT_D: 30.6398929049531"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 1.87610619469027 MeanDoubletons: 3.25663716814159 MeanCommon: 591.082595870207 MeanRare: 63.1061946902655 MeanHeterozy: 404.362831858407 MeanCoverage: 14.6616951893061 sdCoverage: 5.96973879035129 Exonic: 654.171091445428 Intronic: 0 Nonsynonymous: 257.262536873156 Synonymous: 390.896755162242 PP2_D: 28.0766961651917 PP2_P: 18.6873156342183 Sift_D: 41.4660766961652 LRT_D: 30.5486725663717"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroHighlyExposedNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 1.82165605095541 MeanDoubletons: 3.18471337579618 MeanCommon: 591.808917197452 MeanRare: 62.1592356687898 MeanHeterozy: 404.146496815287 MeanCoverage: 15.3278965971024 sdCoverage: 6.0433291355643 Exonic: 653.949044585987 Intronic: 0 Nonsynonymous: 257.757961783439 Synonymous: 390.235668789809 PP2_D: 27.6624203821656 PP2_P: 18.6751592356688 Sift_D: 41.3630573248408 LRT_D: 30.4458598726115"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.02696078431373 MeanDoubletons: 3.40686274509804 MeanCommon: 591.519607843137 MeanRare: 61.6470588235294 MeanHeterozy: 403.161764705882 MeanCoverage: 14.944762747056 sdCoverage: 5.84401646067709 Exonic: 653.149509803922 Intronic: 0 Nonsynonymous: 257.257352941176 Synonymous: 390.004901960784 PP2_D: 28.2941176470588 PP2_P: 18.7156862745098 Sift_D: 40.7745098039216 LRT_D: 30.7156862745098"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justRapidVRapid | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 1.74074074074074 MeanDoubletons: 3.40740740740741 MeanCommon: 592.814814814815 MeanRare: 59.3148148148148 MeanHeterozy: 399.259259259259 MeanCoverage: 15.4838212840091 sdCoverage: 4.5829707196305 Exonic: 652.111111111111 Intronic: 0 Nonsynonymous: 255.351851851852 Synonymous: 390.666666666667 PP2_D: 27.8703703703704 PP2_P: 18.3333333333333 Sift_D: 39.6851851851852 LRT_D: 29.5740740740741"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSlowVSlow | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.08641975308642 MeanDoubletons: 3.5679012345679 MeanCommon: 593.246913580247 MeanRare: 63.2222222222222 MeanHeterozy: 406.345679012346 MeanCoverage: 15.0660673762291 sdCoverage: 8.2663373355639 Exonic: 656.444444444444 Intronic: 0 Nonsynonymous: 259.679012345679 Synonymous: 391.074074074074 PP2_D: 28.4938271604938 PP2_P: 18.7037037037037 Sift_D: 41.5185185185185 LRT_D: 31.1604938271605"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justVeryRapid | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.0952380952381 MeanDoubletons: 3.28571428571429 MeanCommon: 592.142857142857 MeanRare: 59.2380952380952 MeanHeterozy: 397.619047619048 MeanCoverage: 15.6974873657967 sdCoverage: 3.9523316947915 Exonic: 651.333333333333 Intronic: 0 Nonsynonymous: 257.333333333333 Synonymous: 387.952380952381 PP2_D: 29 PP2_P: 18.8095238095238 Sift_D: 40.1904761904762 LRT_D: 31.4285714285714"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justVerySlow | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 1.94736842105263 MeanDoubletons: 3.28947368421053 MeanCommon: 591.631578947368 MeanRare: 61.9210526315789 MeanHeterozy: 404.631578947368 MeanCoverage: 16.9681495161637 sdCoverage: 11.1593773020957 Exonic: 653.5 Intronic: 0 Nonsynonymous: 259.184210526316 Synonymous: 388.921052631579 PP2_D: 28.1842105263158 PP2_P: 18.4473684210526 Sift_D: 41.2894736842105 LRT_D: 30.6842105263158"
> 
> 


#Comparing P2 to P1
~~~
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.GetVCFQCStats.vs2.output
NumVar: 9844 Exonic: 9840 Intronic: 0 Intergenic: 0 UTR5: 0 UTR3: 0                                                                                                                                                                                                                                                        
Synonymous: 4411 Nonsynonymous: 5233 Stopgain: 116 Stoploss: 10
PP2_D: 1572 PP2_P: 842 PP2_B: 2059 PP2_-9: 467 Sift_D: 1991 Sift_T: 2691 Sift_-9: 258 LRT_D: 2050 LRT_N: 2343 LRT_U: 234 LRT_-9: 313                                                                                                                                                                                       
Common: 1144 Rare: 8700 Singletons: 1463 Doubletons: 1547
MeanQuality: 17860.6449553 QualityStd: 65869.6226881 MeanDepth: 14.9857453366 DepthStd: 4.77660674994 TiTvRatio: 4.24453915823 Transitions: 7967 Transversions: 1877                                                                                                                                                       
MeanHapScore: 1.79340668427 HapScoreStd: 1.06580996397 HapScore0Quart: 0.2499 HapScore25Quart: 1.071825 HapScore50Quart: 1.5222 HapScore75Quart: 2.202675 HapScore100Quart: 8.7767
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGenePerVariant.Exonic.GetVCFQCStats.vs1.output
NumVar: 7617 Exonic: 7614 Intronic: 0 Intergenic: 0 UTR5: 0 UTR3: 0
Synonymous: 3339 Nonsynonymous: 4083 Stopgain: 54 Stoploss: 13
PP2_D: 1129 PP2_P: 660 PP2_B: 1546 PP2_-9: 379 Sift_D: 1477 Sift_T: 1927 Sift_-9: 310 LRT_D: 1852 LRT_N: 1624 LRT_U: 96 LRT_-9: 142
Common: 588 Rare: 7029 Singletons: 2592 Doubletons: 1387
MeanQuality: 16993.8342957 QualityStd: 79340.471072 MeanDepth: 23.3232939376 DepthStd: 8.87203927559 TiTvRatio: 3.39780600462 Transitions: 5885 Transversions: 1732
MeanHapScore: 2.75564990154 HapScoreStd: 1.67229214079 HapScore0Quart: 0.181 HapScore25Quart: 1.5964 HapScore50Quart: 2.3674 HapScore75Quart: 3.4397 HapScore100Quart: 14.6402

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 1.87610619469027 MeanDoubletons: 3.25663716814159 MeanCommon: 591.082595870207 MeanRare: 63.1061946902655 MeanHeterozy: 404.362831858407 MeanCoverage: 14.6616951893061 sdCoverage: 5.96973879035129 Exonic: 654.171091445428 Intronic: 0 Nonsynonymous: 257.262536873156 Synonymous: 390.896755162242 PP2_D: 28.0766961651917 PP2_P: 18.6873156342183 Sift_D: 41.4660766961652 LRT_D: 30.5486725663717"
>
>
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output.justSeroNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.43909348441926 MeanDoubletons: 2.86685552407932 MeanCommon: 299.28045325779 MeanRare: 35.2549575070822 MeanHeterozy: 219.773371104816 MeanCoverage: 21.9810675566897 sdCoverage: 8.58968008315038 Exonic: 334.082152974504 Intronic: 0 Nonsynonymous: 122.872521246459 Synonymous: 208.801699716714 PP2_D: 9.90934844192635 PP2_P: 12.6345609065156 Sift_D: 22.9206798866856 LRT_D: 21.6345609065156"
>
>

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.02696078431373 MeanDoubletons: 3.40686274509804 MeanCommon: 591.519607843137 MeanRare: 61.6470588235294 MeanHeterozy: 403.161764705882 MeanCoverage: 14.944762747056 sdCoverage: 5.84401646067709 Exonic: 653.149509803922 Intronic: 0 Nonsynonymous: 257.257352941176 Synonymous: 390.004901960784 PP2_D: 28.2941176470588 PP2_P: 18.7156862745098 Sift_D: 40.7745098039216 LRT_D: 30.7156862745098"
>
>
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.53537735849057 MeanDoubletons: 2.54009433962264 MeanCommon: 298.695754716981 MeanRare: 34.4528301886792 MeanHeterozy: 219.007075471698 MeanCoverage: 22.9559819110576 sdCoverage: 9.25012765969131 Exonic: 332.719339622641 Intronic: 0 Nonsynonymous: 122.200471698113 Synonymous: 207.785377358491 PP2_D: 9.78301886792453 PP2_P: 12.6768867924528 Sift_D: 22.2452830188679 LRT_D: 21.8915094339623"
>
>
~~~

#20150325
#Adding new annotated .vcf format option now using vcf-annotate (http://vcftools.sourceforge.net/perl_module.html#vcf-annotate)
#Currently mainly being used to help pass on information to Sudhir for evaluation cost/design-strategy for variant follow-up genotyping
#CHECK_0: Getting vcf-annotate to work properly for the AC_... columns if this setup/format is desired to be continually used

cat <(echo "#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        REF     ALT     AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW") \
<(join <( python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | sort -k 1,1) <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^\# | perl -lane 'print join("_", @F[0..1]), "\t", $F[3], "\t", $F[4];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
perl -lane 'splice(@F, 0, 1, (((split(/_/, $F[0]))[0]), ((split(/_/, $F[0]))[1]), ((split(/_/, $F[0]))[1]))); print join("\t", @F);' |
perl -ane 'print join("\t", @F[0..8]); my $alt = $F[8]; my $totAC = join("", ((split(/:/, $F[9]))[1])) + join("", ((split(/:/, $F[10]))[1])); print "\t", $totAC; for (my $i=9; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", $F[$i]; } } print "\n";' | sort -g -k 1,1 -k 2,2 ) |
bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs1.wACs.txt.gz

cat <(echo "#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        REF     ALT     AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW") <(join <( python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | sort -k 1,1) <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^\# | perl -lane 'print join("_", @F[0..1]), "\t", $F[3], "\t", $F[4];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | perl -lane 'splice(@F, 0, 1, (((split(/_/, $F[0]))[0]), ((split(/_/, $F[0]))[1]), ((split(/_/, $F[0]))[1]))); print join("\t", @F);' | perl -ane 'print join("\t", @F[0..8]); my $alt = $F[8]; my $totAC = join("", ((split(/:/, $F[9]))[1])) + join("", ((split(/:/, $F[10]))[1])); print "\t", $totAC; for (my $i=9; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", $F[$i]; } } print "\n";' | sort -g -k 1,1 -k 2,2 ) | bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs1.wACs.txt.gz

tabix -s 1 -b 2 -e 3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs1.wACs.txt.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | vcf-annotate -a /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs1.wACs.txt.gz -d /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.PrepFor_vcfannotate.vs1.desc.txt -c CHROM,FROM,TO,INFO/LOC,INFO/GENE,INFO/VARTYPE,INFO/AACHANGE,INFO/AC_ALL,INFO/AC_TOTAL,INFO/AC_SERONEG,INFO/AC_HESERONEG,INFO/AC_SEROPOS,INFO/AC_RAPID,INFO/AC_SLOW | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.annotate.vs1.wACs.vcf.gz

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO

cat <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.annotate.vs1.wACs.vcf.gz | grep ^# | perl -lane 'if ($F[0] =~ /#CHROM/) { print join("\t", @F[0..7]); } else { print join(" ", @F); }' ) \
<(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.annotate.vs1.wACs.vcf.gz | perl -lane 'print join("\t", @F[0..7]);' | grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1; } }' | grep -vw NA ) | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq )) | \
gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.P2.Vs2.qVal_lte85.annotated.vcf.gz 

~~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1; } }'i | grep -vw NA | wc                                                                                                                                                                                                                                                         
    188     188    1154
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.annotate.vs1.wACs.vcf.gz | perl -lane 'print join("\t", @F[0..7]);' | grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1; } }' | grep -vw NA ) | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq ) | wc
   1384   11072  668030                                                                                                                                                                                                                                                                                                    
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.P2.Vs2.qVal_lte85.annotated.vcf.gz | awk '{ print $1, "\t", $2}' | grep -v ^# | wc
   1384    2768   18998
~~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.P2.Vs2.qVal_lte85.annotated.vcf.gz mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/.

cat <(echo "#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        PVAL	QVAL	REF     ALT	AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW") \
<(join <( python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | sort -k 1,1) <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^\# | perl -lane 'print join("_", @F[0..1]), "\t", $F[3], "\t", $F[4];' | sort -k 1,1) | \
perl -lane '$F[1] = join("|", split(/[\;,\,]/, $F[1])); $F[2] = join("|", split(/[\;,\,]/, $F[2])); print join("\t", @F);' | \
perl -lane 'my @geneEntries = split(/\|/, $F[2]); my @pVals; my @qVals; foreach my $gene (@geneEntries) { $gene = ((split(/\(/, $gene))[0]); my $geneEntry = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w "$gene"`; if ($geneEntry) { my @geneVals1 = split(/\s+/, $geneEntry); push(@pVals, $geneVals1[1]); push(@qVals, $geneVals1[3]); } else { push(@pVals, "NA"); push(@qVals, "NA"); } } splice(@F, 5, 0, (join("|", @pVals), join("|", @qVals))); print join("\t", @F);' | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
perl -lane 'splice(@F, 0, 1, (((split(/_/, $F[0]))[0]), ((split(/_/, $F[0]))[1]), ((split(/_/, $F[0]))[1]))); print join("\t", @F);' | \
perl -ane 'print join("\t", @F[0..10]); my $alt = $F[10]; my $totAC = join("", ((split(/:/, $F[11]))[1])) + join("", ((split(/:/, $F[12]))[1])); print "\t", $totAC; for (my $i=11; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", ((split(/\:/, $F[$i]))[1]); } } print "\n";' | sort -g -k 1,1 -k 2,2 ) | \
bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz

cat <(echo "#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        PVAL	QVAL	REF     ALT     AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW") <(join <( python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | sort -k 1,1) <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^\# | perl -lane 'print join("_", @F[0..1]), "\t", $F[3], "\t", $F[4];' | sort -k 1,1) | perl -lane '$F[1] = join("|", split(/[\;,\,]/, $F[1])); $F[2] = join("|", split(/[\;,\,]/, $F[2])); print join("\t", @F);' | perl -lane 'my @geneEntries = split(/\|/, $F[2]); my @pVals; my @qVals; foreach my $gene (@geneEntries) { $gene = ((split(/\(/, $gene))[0]); my $geneEntry = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w "$gene"`; if ($geneEntry) { my @geneVals1 = split(/\s+/, $geneEntry); push(@pVals, $geneVals1[1]); push(@qVals, $geneVals1[3]); } else { push(@pVals, "NA"); push(@qVals, "NA"); } } splice(@F, 5, 0, (join("|", @pVals), join("|", @qVals))); print join("\t", @F);' | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | perl -lane 'splice(@F, 0, 1, (((split(/_/, $F[0]))[0]), ((split(/_/, $F[0]))[1]), ((split(/_/, $F[0]))[1]))); print join("\t", @F);' | perl -ane 'print join("\t", @F[0..10]); my $alt = $F[10]; my $totAC = join("", ((split(/:/, $F[11]))[1])) + join("", ((split(/:/, $F[12]))[1])); print "\t", $totAC; for (my $i=11; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", ((split(/\:/, $F[$i]))[1]); } } print "\n";' | sort -g -k 1,1 -k 2,2 ) | /home/shared/software/tabix-0.2.5/bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz
perl -ane 'print join("\t", @F[0..8]); my $alt = $F[8]; my $totAC = join("", ((split(/:/, $F[9]))[1])) + join("", ((split(/:/, $F[10]))[1])); print "\t", $totAC; for (my $i=9; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", $F[$i]; } } print "\n";' | sort -g -k 1,1 -k 2,2 ) | /home/shared/software/tabix-0.2.5/bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz

~~~
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | head -n 10
#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        PVAL    QVAL    REF     ALT     AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW
X       2821584 2821584 downstream      ARSD    .       .       0.6429427000    0.90298 C       G       1466    2       2       0       0       0       0
X       2821782 2821782 downstream      ARSD    .       .       0.6429427000    0.90298 G       A       1478    1       0       0       1       0       0
X       2821848 2821848 downstream      ARSD    .       .       0.6429427000    0.90298 G       A       1484    3       3       3       0       0       0
X       2822103 2822103 UTR3    ARSD    .       .       0.6429427000    0.90298 C       G       1486    2       0       0       2       0       2
X       2823133 2823133 UTR3    ARSD    .       .       0.6429427000    0.90298 G       A       1486    2       1       1       1       0       0
X       2824218 2824218 UTR3    ARSD    .       .       0.6429427000    0.90298 T       C       1486    15      7       3       8       2       2
X       2825026 2825026 UTR3    ARSD    .       .       0.6429427000    0.90298 C       T       1474    5       2       0       3       0       0
X       2825083 2825083 UTR3    ARSD    .       .       0.6429427000    0.90298 G       A       1476    12      6       4       6       2       2
X       2825149 2825149 UTR3    ARSD    .       .       0.6429427000    0.90298 C       T       1474    4       2       0       2       1       0
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | perl -lane 'print $#F;' | sort | uniq -c
  85201 17
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | wc                                      
  85201 1533618 7483870
~~~

tabix -s 1 -b 2 -e 3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | vcf-annotate -a /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz -d /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.PrepFor_vcfannotate.vs2.desc.txt -c CHROM,FROM,TO,INFO/LOC,INFO/GENE,INFO/VARTYPE,INFO/AACHANGE,INFO/PVAL,INFO/QVAL,REF,ALT,INFO/AC_ALL,INFO/AC_TOTAL,INFO/AC_SERONEG,INFO/AC_HESERONEG,INFO/AC_SEROPOS,INFO/AC_RAPID,INFO/AC_SLOW | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz

~~~
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz | perl -lane 'print $#F;' | sort | uniq -c
87 0
3 1
3 10
1 109
7 11
5 12
1 13
4 14
1 16
2 2
1 20
2 24
2 3
1 38
3 4
1 41
4 5
3 6
1 69
85201 755
4 8
3 9
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | wc
85201 1533618 7483870
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz | wc                         
85340 64412794 1498116300
~~~

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.P2.Vs2.annotated.vcf.gz

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.P2.Vs2.annotated.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.ForSudhir.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/.

##
##20150825 20150930 CHECK_1: Redo certain analyses because of the single "30430   POOL87_02RL" individual. Check subset side analysis below for details regarding these revisions and what this is referring to specifically
##20150907 201510__ CHECK_1: Redo all this in P2 as well
##

#20150825 CHECK_0: Also figure out how to check alignments of reads/bam file stuff around specific/certain variants, such as those that look interesting on the lollipop charts, to make sure misaligning reads are not the cause of multiple variants popping up in localized places (and aberrantly causing a region to look interesting)
#20150825 CHECK_0: Really need to redo those subsampling-to-P2 analyses, but on the individual .bam level files first?
#20150825 CHECK_0: Do those p-value comparisons or rank comparison between top hits in my analyses and the other analyses the molecular people have done?
#20150914 CEHCK_0: Look at variants that are not present in ExAC or 1000G, but are present in my dataset, to see if there are any systematic qualities to them (poorly mapped reads, certain regions of the genome, correlation or overabundance in significant-trending genes?..correlations with gene p-values perhaps?)
#20150825 CHECK_0: Regarding the fastqc stuff -- Check this stuff?

#20150909 CHECK_0: Ask Sudhir about timetoART (i.e. is it 'time to Anti-Retroviral Therapy'?)

#See /data/userdata/pg/michaelt/Data/ALL_MAPPING/MainScript1.PG_HIVArray2.sh for analyses and creationg of new phenotype file
#From /data/userdata/pg/michaelt/Data/ALL_MAPPING/MainScript1.PG_HIVArray2.sh
~~~
...
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs2.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs3.py

#20150909 -- Changing the ...DataProcessing.Pipeline.CreatePhenotypeFile.vs3.py file to now check whether posage is >0 to determine seropositive status, and I will be doing this as the initial determining step. I will also now add an extra 'else' step that define a phenotype as 'NA' if none of the conditions are met anywhere in the rules provided

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs3.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.vs3.pheno

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.vs2.pheno
...
~~~

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs3.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs1.ForP2.vs2.py

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs1.pheno

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs1.ForP2.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4 }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $5 }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $6 }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $5, "\t", $6 }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $7 }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $8 }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $9 }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $10 }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $5 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $6 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $5, "\t", $6 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $7 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $8 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $9 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $10 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $5 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $6 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $5, "\t", $6 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $7 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $8 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $9 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $10 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{print $3, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $8, "\t", $9, "\t", $10 }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $8, "\t", $9, "\t", $10 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $8, "\t", $9, "\t", $10 } }' | sort | uniq -c

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | awk '{ print $3 }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | awk '{ print $4 }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | awk '{ print $5 }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | awk '{ print $6 }' | sort | uniq -c

~~~
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | head -n 10
20446,6,50.35,POOL_P2_137_06RL,Black non-Hispanic,35.2,0,0,0,1,0,0,0,0,3.42
41011,8,19.13,POOL_P2_83_08RL,White non-Hispanic,0,0,1,0,0,0,0,0,0,0
41013,8,67.85,POOL_P2_156_08RL,White non-Hispanic,0,0,1,0,0,0,0,0,0,0
40157,9,88.62,POOL_P2_85_09RL,White Hispanic,0,1,0,0,0,0,0,0,0,0
10290,6,24.94,POOL_P2_147_06RL,Black non-Hispanic,39.3,0,0,0,1,1,1,0,0,1.62
10292,3,50.78,POOL_P2_135_03RL,White non-Hispanic,0,1,0,0,0,0,0,0,0,0
10453,3,82.8,POOL_P2_141_03RL,White non-Hispanic,28.3,0,0,0,1,0,0,0,0,7.91
10519,4,5.75,POOL_P2_157_04RL,White non-Hispanic,0,0,1,0,0,0,0,0,0,0
20501,5,56.03,POOL_P2_131_05RL,White non-Hispanic,0,1,0,0,0,0,0,0,0,0
40553,2,13.47,POOL_P2_88_02RL,White non-Hispanic,31.4,0,0,0,1,0,0,0,0,4.09
[  michaelt@login2  ~]$cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Sample_status_list_12-10-13_CT.edited.noMeta.csv | head -n 10 | perl -F\, -lane 'print join("\t", @F[10..19]);'
posage  hesn    lesn    lesc    othersc vrapid  rapid   vslow   slow    timetoART
36.4    0       0       0       1       0       0       0       0       2.13
40.2    0       0       1       1       0       1       0       0       1.25
0       0       1       0       0       0       0       0       0       0
35.6    0       0       0       0       0       0       1       1       24.03
34      0       0       0       1       0       0       0       0       2.42
45.7    0       0       1       1       0       0       0       0       1.24
0       0       1       0       0       0       0       0       0       0
47.9    0       0       0       1       1       1       0       0       3.15
0       1       0       0       0       0       0       0       0       0
[  michaelt@login2  ~]$head -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno
POOL_P2_137_06RL        POOL_P2_137_06RL        2       2       -9      -9
POOL_P2_83_08RL POOL_P2_83_08RL 1       -9      -9      -9
POOL_P2_156_08RL        POOL_P2_156_08RL        1       -9      -9      -9
POOL_P2_85_09RL POOL_P2_85_09RL 1       1       -9      -9
POOL_P2_147_06RL        POOL_P2_147_06RL        2       2       1       1
POOL_P2_135_03RL        POOL_P2_135_03RL        1       1       -9      -9
POOL_P2_141_03RL        POOL_P2_141_03RL        2       2       -9      -9
POOL_P2_157_04RL        POOL_P2_157_04RL        1       -9      -9      -9
POOL_P2_131_05RL        POOL_P2_131_05RL        1       1       -9      -9
POOL_P2_88_02RL POOL_P2_88_02RL 2       2       -9      -9
[  michaelt@login2  ~]$head -n 19 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print join("\t", @F[5..14]);' | head -n 10
35.2    0       0       0       1       0       0       0       0       3.42
0       0       1       0       0       0       0       0       0       0
0       0       1       0       0       0       0       0       0       0
0       1       0       0       0       0       0       0       0       0
39.3    0       0       0       1       1       1       0       0       1.62
0       1       0       0       0       0       0       0       0       0
28.3    0       0       0       1       0       0       0       0       7.91
0       0       1       0       0       0       0       0       0       0
0       1       0       0       0       0       0       0       0       0
31.4    0       0       0       1       0       0       0       0       4.09
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4 }' | sort | uniq -c
529 0        0
204 0        1
200 1        0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $5 }' | sort | uniq -c
346 0        0       0
183 0        0       1
204 0        1       0
200 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $6 }' | sort | uniq -c
124 0        0       0
405 0        0       1
204 0        1       0
200 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $5, "\t", $6 }' | sort | uniq -c
113 0        0       0       0
233 0        0       0       1
11 0        0       1       0
172 0        0       1       1
204 0        1       0       0
200 1        0       0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $7 }' | sort | uniq -c
504 0        0       0
25 0        0       1
204 0        1       0
200 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $8 }' | sort | uniq -c
460 0        0       0
69 0        0       1
204 0        1       0
200 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $9 }' | sort | uniq -c
460 0        0       0
69 0        0       1
204 0        1       0
200 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $10 }' | sort | uniq -c
401 0        0       0
128 0        0       1
204 0        1       0
200 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4 } }' | sort | uniq -c
529 0        0
1 1        0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $5 } }' | sort | uniq -c
346 0        0       0
183 0        0       1
1 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $6 } }' | sort | uniq -c
124 0        0       0
405 0        0       1
1 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $5, "\t", $6 } }' | sort | uniq -c
113 0        0       0       0
233 0        0       0       1
11 0        0       1       0
172 0        0       1       1
1 1        0       0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $7 } }' | sort | uniq -c
504 0        0       0
25 0        0       1
1 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $8 } }' | sort | uniq -c
460 0        0       0
69 0        0       1
1 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $9 } }' | sort | uniq -c
460 0        0       0
69 0        0       1
1 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $10 } }' | sort | uniq -c
401 0        0       0
128 0        0       1
1 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4 } }' | sort | uniq -c
527 0        0
1 1        0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $5 } }' | sort | uniq -c
344 0        0       0
183 0        0       1
1 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $6 } }' | sort | uniq -c
124 0        0       0
403 0        0       1
1 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $5, "\t", $6 } }' | sort | uniq -c
113 0        0       0       0
231 0        0       0       1
11 0        0       1       0
172 0        0       1       1
1 1        0       0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{print $3, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $8, "\t", $9, "\t", $10 }' | sort | uniq -c
53 0        0       0       0       0       0       0       1
52 0        0       0       0       0       0       1       1
7 0        0       0       0       0       1       0       0
1 0        0       0       0       1       1       0       0
181 0        0       0       1       0       0       0       0
6 0        0       0       1       0       0       0       1
8 0        0       0       1       0       0       1       1
20 0        0       0       1       0       1       0       0
18 0        0       0       1       1       1       0       0
10 0        0       1       0       0       0       0       0
1 0        0       1       0       0       1       0       0
141 0        0       1       1       0       0       0       0
9 0        0       1       1       0       0       1       1
16 0        0       1       1       0       1       0       0
6 0        0       1       1       1       1       0       0
204 0        1       0       0       0       0       0       0
200 1        0       0       0       0       0       0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $8, "\t", $9, "\t", $10 } }' | sort | uniq -c
53 0        0       0       0       0       0       0       1
52 0        0       0       0       0       0       1       1
7 0        0       0       0       0       1       0       0
1 0        0       0       0       1       1       0       0
181 0        0       0       1       0       0       0       0
6 0        0       0       1       0       0       0       1
8 0        0       0       1       0       0       1       1
20 0        0       0       1       0       1       0       0
18 0        0       0       1       1       1       0       0
10 0        0       1       0       0       0       0       0
1 0        0       1       0       0       1       0       0
141 0        0       1       1       0       0       0       0
9 0        0       1       1       0       0       1       1
16 0        0       1       1       0       1       0       0
6 0        0       1       1       1       1       0       0
1 1        0       0       0       0       0       0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $8, "\t", $9, "\t", $10 } }' | sort | uniq -c
53 0        0       0       0       0       0       0       1
52 0        0       0       0       0       0       1       1
7 0        0       0       0       0       1       0       0
1 0        0       0       0       1       1       0       0
179 0        0       0       1       0       0       0       0
6 0        0       0       1       0       0       0       1
8 0        0       0       1       0       0       1       1
20 0        0       0       1       0       1       0       0
18 0        0       0       1       1       1       0       0
10 0        0       1       0       0       0       0       0
1 0        0       1       0       0       1       0       0
141 0        0       1       1       0       0       0       0
9 0        0       1       1       0       0       1       1
16 0        0       1       1       0       1       0       0
6 0        0       1       1       1       1       0       0
1 1        0       0       0       0       0       0       0
[  michaelt@login2  ~]$cmp <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $8, "\t", $9, "\t", $10 } }' | sort | uniq -c) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $8, "\t", $9, "\t", $10 } }' | sort | uniq -c)
/dev/fd/63 /dev/fd/62 differ: byte 158, line 5
[  michaelt@login2  ~]$python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs1.ForP2.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno
[  michaelt@login2  ~]$ 
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | awk '{ print $3 }' | sort | uniq -c
403 1
530 2
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | awk '{ print $4 }' | sort | uniq -c
199 1
530 2
204 -9
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | awk '{ print $5 }' | sort | uniq -c
69 1
128 2
736 -9
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | awk '{ print $6 }' | sort | uniq -c
25 1
69 2
839 -9
~~~

#201510__ Note -- the below code is copy/pasted from the end of the rerun done at ##20150825 as a point of reference

~~
.
.
.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.P2.Vs2.annotated.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/OLD2.P2.AllPools.P2.Vs2.annotated.vcf.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/OLD2.P2.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz 
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.ForSudhir.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/OLD2.P2.skatO.nonsyn.qVals.ACs.GeneLevel.ForSudhir.txt

rm -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt

for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs`
do

	paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane 'if ($F[0] =~ m/^'$gene1'$/) { print join("\t", @F); }' | awk '{ print $1, "\t", $2, "\t", $4, "\t", $5, "\t", $8, "\t", $9, "\t", $11 }' ) \
	<(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | \
	grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'if ($F[0] =~ m/^'$gene1'$/) { print join("\t", @F); }' | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }') | \
	awk '{ print $12, "\t", $13, "\t", $14, "\t", $15, "\t", $16, "\t", $17, "\t", $18 }' | \
	R -q -e "Data1 <- read.table(file('stdin'), header=F); print(paste(apply(Data1, 2, sum)), quote=FALSE);" | grep -v ^\> | perl -lane 'splice(@F, 0, 1); print join("\t", @F);') \
	>> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt

done

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt | perl -lane 'splice(@F, 4, 3); print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt

#Included the below header in file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt 
#GENE   PVAL    QVAL    NUMVAR    AC_ALL        AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW

#QC Checks on file

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt | wc
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt | perl -lane 'if (($F[3] == $F[4]) && ($F[3] == $F[5]) && ($F[4] == $F[5])) { print join("\t", @F); } '  | wc
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt | perl -lane 'if (($F[3] != $F[4]) || ($F[3] != $F[5]) || ($F[4] != $F[5])) { print join("\t", @F); } '
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[5] != $val2) { print join("\t", @F); }'
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[5] == $val2) { print join("\t", @F); }' | wc
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[7] > $F[6]) { print join("\t", @F); }'
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[7] <= $F[6]) { print join("\t", @F); }' | wc
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[9] + $F[10]; if ($F[8] < $val2) { print join("\t", @F); }'
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[9] + $F[10]; if ($F[8] >= $val2) { print join("\t", @F); }' | wc

#20151003 CHECK_0: Once have annotation file working properly (e.g. Total_AC working properly), go back and check why one variant is dropping out of the analysis for this one gene here?
#Original date was sometime back in April/May 2015
~~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt | wc
786    8646   39442
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt | perl -lane 'if (($F[3] == $F[4]) && ($F[3] == $F[5]) && ($F[4] == $F[5])) { print join("\t", @F); } '  | wc
785   10983   48835
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt | perl -lane 'if (($F[3] != $F[4]) || ($F[3] != $F[5]) || ($F[4] != $F[5])) { print join("\t", @F); } '
WNK1    0.418770600     0.88734 42      42      41      0.41574 62674   3926    1794    820     2132    290     422
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GenLevel.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[5] != $val2) { print join("\t", @F); }'
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[5] == $val2) { print join("\t", @F); }' | wc
786    8646   39402
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[7] > $F[6]) { print join("\t", @F); }'
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[7] <= $F[6]) { print join("\t", @F); }' | wc
786    8646   39402
[  michaelt@node1  ~]$
~~~

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.P2.Vs2.annotated.vcf.gz

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz

#Difference between 141 and 142 due to '#CHROM...' header in /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz
~~~
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz | wc
85201 1533618 7403208
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz | wc
85340 64412794 1498035646
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.P2.Vs2.annotated.vcf.gz | wc
85340 64412794 1498035646
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz | wc
85201 1533618 7403208
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz | wc
85340 64412794 1498035646
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.P2.Vs2.annotated.vcf.gz | wc
85340 64412794 1498035646
[  michaelt@node1  ~]$R -q -e "85340 - 85201"
> 85340 - 85201
[1] 139
>
>
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.P2.Vs2.annotated.vcf.gz | grep ^# | wc
140    1594   29457
~~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/OLD2.P2.AllPools.P2.Vs2.annotated.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/OLD2.P2.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/OLD2.P2.skatO.nonsyn.qVals.ACs.GeneLevel.ForSudhir.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/.

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.P2.Vs2.annotated.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/.

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt.P2.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.msigdb_c2

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt.P2.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.msigdb_c2
~~~






















###
### Current end of main analyses
### CURRENTEND
###
### Below begin specific subset side analyses
###



















##20140925
##Getting variants present in specific group of invidiuals for Kevin

#20150418
#Getting any variant whose alternative allele copies only exist in these subset of individuals. By definition then only including variants whose alternative ACs are equal to or less than 24 (b/c 12 individuals are specifically being looked at here)
#Still using /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz file since geno80 step was done post subsetting to QCed white non-hispanic individuals
#E.g. there is no /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz -- only a ...Vs2... exists

#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss | grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | grep -E '20572|20306|20219|20170|20123|20093|20501|20698|20132|20661|20471|20575' | awk -F, '{ print $4 }' | perl -lane 'if ($F[0] =~ m/(POOL)(\d+)-(\d+RL)/) { print "POOL_P2_", $2, "_", $3; } ')
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss | grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | grep -E '20572|20306|20219|20170|20123|20093|20501|20698|20132|20661|20471|20575' | awk -F, '{ print $4 }')

~~~
[  michaelt@node2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss | grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | grep -E '20572|20306|20219|20170|20123|20093|20501|20698|20132|20661|20471|20575' | awk -F, '{ print $4 }')
POOL_P2_102_06RL        94889   0       30      0.000316159
POOL_P2_109_07RL        94889   0       772     0.00813582
POOL_P2_109_09RL        94889   0       684     0.00720842
POOL_P2_114_02RL        94889   0       128     0.00134894
POOL_P2_131_05RL        94889   0       122     0.00128571
POOL_P2_144_09RL        94889   0       2       2.10773e-05
POOL_P2_153_07RL        94889   0       1084    0.0114239
POOL_P2_159_01RL        94889   0       97      0.00102225
POOL_P2_163_12RL        94889   0       706     0.00744027
POOL_P2_174_02RL        94889   0       51      0.00053747
POOL_P2_87_11RL 94889   0       38      0.000400468
POOL_P2_92_11RL 94889   0       208     0.00219203
~~~

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.ExtractPatientIDsFromVCF.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --IDs 20572,20306,20219,20170,20123,20093,20501,20698,20132,20661,20471,20575 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf.gz | wc

~~~
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | wc
  95015 88153459 2022066686
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf.gz | wc
  95015 1993339 56484166
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf.gz | perl -lane 'print $#F;' | sort | uniq -c
87 0
3 1
1 10
1 109
5 11
5 12
1 13
4 14
1 16
2 2
94891 20
2 24
2 3
2 4
3 5
1 6
1 69
1 8
2 9
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf.gz | tail -n +127 | perl -lane 'print $#F;' | sort | uniq -c
  94889 20
~~~

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf.gz | perl -lane 'my @SNPinfo = split(/;/, $F[7]); my $AC; foreach my $info1 (@SNPinfo) { if ($info1 =~ /^AC\=/) { $AC = ((split(/=/, $info1))[1]); } } my $genoCount = 0; foreach (my $i = 9; $i <= $#F; $i++) { my @indvInfo = split(/\:/, $F[$i]); my @genoInfo = split(/\//, $indvInfo[0]); if ($genoInfo[0] == 1) { $genoCount++; } if ($genoInfo[1] == 1) { $genoCount++; } } if ($genoCount == $AC) { print join("\t", @F); } ' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.vcf.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs6.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.wAnnotVs6.vs2.vcf.gz

~~~
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.wAnnotVs6.vs2.vcf.gz | wc                     289    4257  114803
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.vcf.gz | wc                                   284    3988  107801
~~~

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85 ) { print $1 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.qVal_lte85.GeneIDs

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.wAnnotVs6.vs2.vcf.gz | perl -lane 'my @snpInfo2 = split(/;/, $F[9]); push(@F, $snpInfo2[1]); print join("\t", @F);' | grep -w -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.qVal_lte85.GeneIDs | perl -lane 'pop(@F); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.wAnnotVs6.vs2.TopGenes_qVallte85.vcf.gz 

cat <(echo "CHR BP SNPID REF ALT LOC GENE VARTYPE AACHANGE NUMAC 20123   20219   20572   20575   20501   20132   20306   20471   20661   20170   20093   20698") <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.wAnnotVs6.vs2.TopGenes_qVallte85.vcf.gz | grep -v ^# | perl -ane 'print join("\t", @F[0..4]);
my @SNPinfo2 = split(/;/, $F[9]); for (my $j = 0; $j <= 3; $j++) { if ($SNPinfo2[$j]) { print "\t", $SNPinfo2[$j]; } else { print "\t", "."; } }
my @SNPinfo1 = split(/;/, $F[7]); my $AC; foreach my $info1 (@SNPinfo1) { if ($info1 =~ /^AC\=/) { $AC = ((split(/=/, $info1))[1]); } } print "\t", $AC;
for (my $i = 10; $i <= $#F; $i++) { my @indvInfo1 = split(/\:/, $F[$i]); print "\t", $indvInfo1[0]; }
print "\n";') > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.wAnnotVs6.vs2.TopGenes_qVallte85.ForKevin

~~~
[  michaelt@node2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.wAnnotVs6.vs2.TopGenes_qVallte85.ForKevin | perl -lane 'print $#F;' | sort | uniq -c
    34 21
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf.gz | perl -lane 'my @SNPinfo1 = split(/;/, $F[7]); my $AC; foreach my $info1 (@SNPinfo1) { if ($info1 =~ /^AC\=/) { $AC = ((split(/=/, $info1))[1]); } } if ($AC == 2) { print join("\t", @F); }' | grep \/1 | perl -lane 'my $genoCount = 0; foreach (my $i = 9; $i <= $#F; $i++) { my @indvInfo = split(/\:/, $F[$i]); my @genoInfo = split(/\//, $indvInfo[0]); if ($genoInfo[0] == 1) { $genoCount++; } if ($genoInfo[1] == 1) { $genoCount++; } } print $genoCount; ' | sort | uniq -c
    279 1
    3 2
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf.gz | perl -lane 'my @SNPinfo1 = split(/;/, $F[7]); my $AC; foreach my $info1 (@SNPinfo1) { if ($info1 =~ /^AC\=/) { $AC = ((split(/=/, $info1))[1]); } } if ($AC == 2) { print join("\t", @F); }' | grep 1\/1 | wc
    2      42    1149
~~~

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.wAnnotVs6.vs2.TopGenes_qVallte85.ForKevin /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/HESNIndvs.P2.PrivateVariants.AllVariants.TopGenes_qVal85.txt

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/HESNIndvs.P2.PrivateVariants.AllVariants.TopGenes_qVal85.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/. 


#20150427
#Specifically pulling out variant level information for SULT1C3 based on Kevin's request

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz | grep -E '^#|SULT1C3' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/P2.ForKevin.SULT1C3.ACs.txt 


scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/P2.ForKevin.SULT1C3.ACs.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/.

~~~
[  michaelt@node2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroNeg.FIDs | wc
339     339    5617
[  michaelt@node2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroHighlyExposedNeg.FIDs | wc
157     157    2633
[  michaelt@node2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroPos.FIDs | wc
408     408    6798
[  michaelt@node2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justRapidVRapid.FIDs | wc
54      54     901
[  michaelt@node2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSlowVSlow.FIDs | wc
81      81    1368
~~~





##20141114 
#Redoing hesn stuff because of issue found previously

1160  vi /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs1.ForP2.vs1.py
#Made edit --
if (line1[7] == "1"): #Note -- 1 is highly-exposed sero-negative and 2 is sero-positive                                                                                                                                                                                                                            
	Pheno2 = "1"
elif (line1[6] == "1"):                                                                                                                                                                                                                                                                                            
	Pheno2 = "-9"

to

if (line1[6] == "1"): #Note -- 1 is highly-exposed sero-negative and 2 is sero-positive                                                                                                                                                                                                                            
	Pheno2 = "1"
elif (line1[7] == "1"):                                                                                                                                                                                                                                                                                            
	Pheno2 = "-9"

1161  cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.pheno
  1162  plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 2 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno2 &
   1163  cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ if ($4 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs
    1164  history

1167  /home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno2.assoc.logistic Pheno2_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno2.assoc.logistic.QQplot.jpeg &

1007  gunzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq.count
 1008  vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg &
  1009  vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg &
   1010  paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq.count | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.AllPhenos.frq.AndCount.gz
    1011  gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq.count

1013  python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs5.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.AllPhenos.frq.AndCount.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 30; $i <= 40; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.pheno | awk '{ print $1, "\t", $2, "\t0\t0\t0\t", $3, "\t", $4, "\t", $5, "\t", $6 } ' | sed 's/-9/NA/g' | perl -lane 'print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped                                                                                                                                                         

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar | awk '{ print $4, "\t", $6, "\t", $7 "\t", $8, "\t", $9, "\t", $10, "\t", $11,  "\t", $12, "\t", $13, "\t", $14,  "\t", $15 }' | paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped.vs2                                                                                                                                                                               

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped.vs2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped
																																							   #Add line #FAM_ID IND_ID FAT_ID MOT_ID SEX HIVPROG HIVPROGHE AIDS AIDSEXTR RMID PC1 PC2 PC3 PC4 PC5 PC6 PC7 PC8 PC9 PC10 to /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped

1162  bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs

#20150108 didn't actually rerun /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs1.ForP2.vs1.py code after making the edit described above.....so had to rerun my rerun of everything

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs1.ForP2.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.pheno

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 2 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno2 &

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ if ($4 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno2.assoc.logistic Pheno2_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno2.assoc.logistic.QQplot.jpeg &

gunzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq.count

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg &

paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq.count | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.AllPhenos.frq.AndCount.gz

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq.count

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.AllPhenos.frq.AndCount.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 30; $i <= 40; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs6.txt.gz

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.pheno | awk '{ print $1, "\t", $2, "\t0\t0\t0\t", $3, "\t", $4, "\t", $5, "\t", $6 } ' | sed 's/-9/NA/g' | perl -lane 'print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar | awk '{ print $4, "\t", $6, "\t", $7 "\t", $8, "\t", $9, "\t", $10, "\t", $11,  "\t", $12, "\t", $13, "\t", $14,  "\t", $15 }' | paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped.vs2

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped.vs2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped

#Add line #FAM_ID IND_ID FAT_ID MOT_ID SEX HIVPROG HIVPROGHE AIDS AIDSEXTR RMID PC1 PC2 PC3 PC4 PC5 PC6 PC7 PC8 PC9 PC10 to /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped

1049  ls -lrt | grep Pheno2 | grep skat0 | grep -v flat | grep .epacts$
 1050  mv AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno2.maf1.skat0.epacts HESNError.AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno2.maf1.skat0.epacts
  1051  mv AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts HESNError.AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts
   1052  mv AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno2.maf1.skat0.epacts HESNError.AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno2.maf1.skat0.epacts
    1053  mv AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno2.maf1.skat0.epacts HESNError.AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno2.maf1.skat0.epacts
     1054  mv AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno2.maf1.skat0.epacts HESNError.AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno2.maf1.skat0.epacts
      1055  mv AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno2.maf1.skat0.epacts HESNError.AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno2.maf1.skat0.epacts
       1056  mv AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno2.maf1.skat0.perm1.epacts HESNError.AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno2.maf1.skat0.perm1.epacts
        1057  mv AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno2.maf1.skat0.perm1.epacts HESNError.AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno2.maf1.skat0.perm1.epacts
	 1058  mv AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts HESNError.AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts
	  1059  ls -lrt | grep Pheno2 | grep skat0 | grep -v flat | grep .epacts$
	   1060  mv HESNError.AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno2.maf1.skat0.perm1.epacts AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno2.maf1.skat0.perm1.epacts
	    1061  mv HESNError.AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno2.maf1.skat0.perm1.epacts AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno2.maf1.skat0.perm1.epacts
	     1062  ls -lrt | grep Pheno2 | grep skat0 | grep -v flat | grep .epacts$ | grep maf1
	      1063  mv AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno2.maf1.skat0.epacts HESNError.AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno2.maf1.skat0.epacts

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs




##20141114
##Getting Malim gene results for Steve from P2

scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/Second_capture_P2_gene-list.MalimSpecific.csv /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/.
scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/Second_capture_P2_gene-list.MalimSpecific.txt /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/.

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Second_capture_P2_gene-list.MalimSpecific.noMeta.txt | sed 's/\r/\n/g' | perl -lane 'print join("\t", @F);' > /home/michaelt/LabStuff/Stephe/HIVStuff/PG/Second_capture_P2_gene-list.MalimSpecific.noMeta.noMeta.txt

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.epacts | grep -f /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Second_capture_P2_gene-list.MalimSpecific.noMeta.txt | awk '{ if ($10 != "NA") { print $0 } } ' | sort -rg -k 10,10 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.epacts.MalimGenes 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts | grep -f /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Second_capture_P2_gene-list.MalimSpecific.noMeta.txt | awk '{ if ($10 != "NA") { print $0 } } ' | sort -rg -k 10,10 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts.MalimGenes 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.epacts | grep -f /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Second_capture_P2_gene-list.MalimSpecific.noMeta.txt | awk '{ if ($10 != "NA") { print $0 } } ' | sort -rg -k 10,10 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.epacts.MalimGenes 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.epacts.MalimGenes /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.epacts.MalimGenes | sort -rg -k 10,10 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); print join("\t", @F); }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/ForSteve.MalimGenes.HIVAcqui.Nonsynonymous.skat0.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts.MalimGenes /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts.MalimGenes | sort -rg -k 10,10 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); print join("\t", @F); }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/ForSteve.MalimGenes.HIVAcquiHighlyExposed.Nonsynonymous.skat0.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.epacts.MalimGenes /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.epacts.MalimGenes | sort -rg -k 10,10 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); print join("\t", @F); }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/ForSteve.MalimGenes.AIDSProg.Nonsynonymous.skat0.output

#Added in header for above 3 'MalimGenes' file results from both P2 and MM5
#chrom  BEGIN   END     MARKER_ID       NS      FRAC_WITH_RARE  NUM_ALL_VARS    NUM_PASS_VARS   NUM_SING_VARS   PVALUE	QVALUE  STATRHO

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/ForSteve.MalimGenes* mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/.

[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/ForSteve.MalimGenes.HIVAcqui.Nonsynonymous.skat0.output | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' | sort | uniq -c | awk '{ if ($1 > 1) { print $0 } } '
	2 APOBEC3A
	2 CBFB
	2 CUL5
	2 DDB1
	2 GIMAP1-GIMAP5
	2 GIMAP5
	2 HEATR1
	2 LGALS3BP
	2 PARP9
	2 TCEB2
	2 TNPO3
	2 VPRBP
	2 ZEB2

[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/ForSteve.MalimGenes.HIVAcquiHighlyExposed.Nonsynonymous.skat0.output | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' | sort | uniq -c | awk '{ if ($1 > 1) { print $0 } } '
	2 APOBEC3A
	2 CBFB
	2 CUL5
	2 DDB1
	2 GIMAP1-GIMAP5
	2 GIMAP5
	2 HEATR1
	2 LGALS3BP
	2 PARP9
	2 TCEB2
	2 TNPO3
	2 VPRBP
	2 ZEB2

[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/ForSteve.MalimGenes.AIDSProg.Nonsynonymous.skat0.output | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' | sort | uniq -c | awk '{ if ($1 > 1) { print $0 } } '
	2 APOBEC3A
	2 CUL5
	2 DDB1
	2 GIMAP1-GIMAP5
	2 GIMAP5
	2 HEATR1
	2 LGALS3BP
	2 PARP9
	2 TCEB2
	2 TNPO3
	2 ZEB2


##20150106
##Creating code/scripts to get list of individuals per variant that contain the alternative allele for that variant, given a list of specific variants

#Make note about doing the grep nonsynonymous vs. grep -E 'nonsynonymous|stop' thing with the wAnnovar files matching up to the EPACTS # of variants/passed variants results

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs

#python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.vcf.gz 
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz | grep -E '^#|XAF1' | grep -E '^#|nonsynonymous|stop' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.vs2.XAF1.Exonic.Nonsyn.vcf.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.vs2.XAF1.Exonic.Nonsyn.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.vs1.XAF1.Exonic.Nonsyn.output 

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.vs1.XAF1.Exonic.Nonsyn.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs1.XAF1.Exonic.Nonsyn.output

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.vs1.XAF1.Exonic.Nonsyn.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/MACIDs.XAF1.Exonic.Nonsyn.vcf

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs1.XAF1.Exonic.Nonsyn.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/MACIDs.XAF1.Exonic.Nonsyn.condensed

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/MACIDs.XAF1.Exonic.Nonsyn.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/MACIDs.XAF1.Exonic.Nonsyn.condensed mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/. 

#20150108 -- realized the below issues with HESN while doing the 'SpecificGeneVCFs' analyses so leaving this code here, but after I ran these lines I redid the code again 'officially' up above
 911  python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs1.ForP2.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.pheno
   912  paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.pheno | vi -
     914  cat AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.pheno | head -n 20
       915  cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.pheno | head -n 20
         921  cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.pheno | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.pheno
	   923  cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.pheno | wc
	     925  paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.pheno | vi -
	       926  cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.pheno  | awk '{ print $1, "\t", $2, "\t0\t0\t0\t", $3, "\t", $4, "\t", $5, "\t", $6 } ' | sed 's/-9/NA/g' | perl -lane 'print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.EPACTSedit.pheno
	         927  cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.pheno  | awk '{ print $1, "\t", $2, "\t0\t0\t0\t", $3, "\t", $4, "\t", $5, "\t", $6 } ' | sed 's/-9/NA/g' | perl -lane 'print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.EPACTSedit.ped
		   928  rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.EPACTSedit.pheno
		     929  cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar | awk '{ print $4, "\t", $6, "\t", $7 "\t", $8, "\t", $9, "\t", $10, "\t", $11,  "\t", $12, "\t", $13, "\t", $14,  "\t", $15 }' | paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.EPACTSedit.ped - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.EPACTSedit.ped.vs2
		       930  mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.EPACTSedit.ped.vs2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.EPACTSedit.ped
1188  bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.HESNUpdate.justWhite.QCed.DropIBD.EPACTS.RMID5PCs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.pheno | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.pheno 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.pheno  | awk '{ print $1, "\t", $2, "\t0\t0\t0\t", $3, "\t", $4, "\t", $5, "\t", $6 } ' | sed 's/-9/NA/g' | perl -lane 'print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.EPACTSedit.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar | awk '{ print $4, "\t", $6, "\t", $7 "\t", $8, "\t", $9, "\t", $10, "\t", $11,  "\t", $12, "\t", $13, "\t", $14,  "\t", $15 }' | paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.EPACTSedit.ped - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.EPACTSedit.ped.vs2 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.EPACTSedit.ped.vs2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.EPACTSedit.ped

#Add line #FAM_ID IND_ID FAT_ID MOT_ID SEX HIVPROG HIVPROGHE AIDS AIDSEXTR RMID PC1 PC2 PC3 PC4 PC5 PC6 PC7 PC8 PC9 PC10 to /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.EPACTSedit.ped

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.HESNUpdate.justWhite.QCed.DropIBD.EPACTS.RMID5PCs
#20150109

#20150129 -- it appears that one -rw-rw-r-- 1 michaelt pg    20 Jan 22 23:24 AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.XAF1.Exonic.Nonsyn.vcf.gz a number of files were '/dev/null/'ed or something, so to say, because files that were used on or around Jan 22 (2015) were similarly 'wiped' at this time point of 23:24, e.g.

~~~
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK]$ls -lrt Analyses/SpecificGeneVCFs/.
total 768                                                                                                                                                                                                                                                                                                                  
lrwxrwxrwx 1 michaelt pg 216 Jan  8 15:55 MACIDs.XAF1.Exonic.Nonsyn.vcf -> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.vs1.XAF1.Exonic.Nonsyn.output
lrwxrwxrwx 1 michaelt pg 225 Jan  8 15:55 MACIDs.XAF1.Exonic.Nonsyn.condensed -> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs1.XAF1.Exonic.Nonsyn.output         
-rw-rw-r-- 1 michaelt pg  20 Jan 22 23:24 AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.XAF1.Exonic.Nonsyn.vcf.gz
-rw-rw-r-- 1 michaelt pg  20 Jan 22 23:24 AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.vs2.XAF1.Exonic.Nonsyn.vcf.gz                                                                      
-rw-rw-r-- 1 michaelt pg   0 Jan 22 23:24 DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.vs1.XAF1.Exonic.Nonsyn.output
-rw-rw-r-- 1 michaelt pg   0 Jan 22 23:24 DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs1.XAF1.Exonic.Nonsyn.output           

[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK]$ls -lrt PLINKfiles/.
total 216832
-rw-rw-r-- 1 michaelt pg     40309 Jan  8 18:44 AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.pheno                                                                                                                                                                                                                            
-rw-rw-r-- 1 michaelt pg     32680 Jan  8 22:35 AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.pheno
-rw-rw-r-- 1 michaelt pg    109260 Jan  8 22:46 AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.EPACTSedit.ped                                                                                                                                                                                            
-rw-rw-r-- 1 michaelt pg     30404 Jan  9 13:54 AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno2.nosex
-rw-rw-r-- 1 michaelt pg   8404937 Jan  9 13:54 AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno2.assoc.logistic                                                                      
-rw-rw-r-- 1 michaelt pg      6263 Jan  9 13:54 AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno2.log
-rw-rw-r-- 1 michaelt pg    157039 Jan  9 13:59 AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno2.assoc.logistic.QQplot.jpeg                                                          
-rw-rw-r-- 1 michaelt pg         0 Jan 22 23:24 AllPools.P2.Vs1.AllPoolsMerged.pheno
-rw-rw-r-- 1 michaelt pg         0 Jan 22 23:24 AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.pheno                                                                                                                                                                                                                
-rw-rw-r-- 1 michaelt pg         0 Jan 22 23:24 AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs
-rw-rw-r-- 1 michaelt pg         0 Jan 22 23:24 AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped                                                                                                                                                                                                       
-rw-rw-r-- 1 michaelt pg         0 Jan 22 23:24 AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped.vs2
~~~

#etc...
#so just reran them on 20150129 -- doesn't appear to be many and saw this in MM5 folder in a few files but didn't realize it had happened in P2 too, and a bit more extensively

#20150223
#Rerunning the 'grep XAF1 individuals' specifically again with the 2nd run of the data

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz | grep -E '^#|XAF1' | grep -E '^#|nonsynonymous|stop|splicing' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.XAF1.Exonic.Nonsyn.vcf.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.XAF1.Exonic.Nonsyn.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.vs1.AllPools.P2.Vs2.XAF1.Exonic.Nonsyn.output

#Including PrintLine1.extend([",".join(line1[8].split(",")[0:3])]) line to show information about specific variant along with the individual IDs per variant
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs2.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.vs1.AllPools.P2.Vs2.XAF1.Exonic.Nonsyn.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs2.AllPools.P2.Vs2.XAF1.Exonic.Nonsyn.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs2.AllPools.P2.Vs2.XAF1.Exonic.Nonsyn.output | perl -ane 'my @vals1 = split(/,/, $F[2]); foreach my $indv1 (@vals1) { print $indv1, "\n"; } ;' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs2.AllPools.P2.Vs2.XAF1.Exonic.Nonsyn.output.IIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs2.AllPools.P2.Vs2.XAF1.Exonic.Nonsyn.output.IIDs  

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs2.AllPools.P2.Vs2.XAF1.Exonic.Nonsyn.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/MACIDs.Vs2.XAF1.Exonic.Nonsyn.condensed

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs2.AllPools.P2.Vs2.XAF1.Exonic.Nonsyn.output.IIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/MACIDs.Vs2.XAF1.Exonic.Nonsyn.condensed.IIDs

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/MACIDs.Vs2.XAF1.Exonic.Nonsyn.condensed /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/MACIDs.Vs2.XAF1.Exonic.Nonsyn.condensed.IIDs mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/.



##20150318
##Adding up and/or binning rho values at different cutoffs/thresholds to just get an idea if there's any information on top gene architecture in relation to HIV-infection                                                                                                                                                 


cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane '$hash1{$F[10]}++; if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };' 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .3) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'


~~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarC
ounts | perl -lane '$hash1{$F[10]}++; if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'                                                                                                                                                                                                 
0.09    11
0.25    20                                                                                                                                                                                                                                                                                                                 
1       354
0.01    4                                                                                                                                                                                                                                                                                                                  
0       243
0.04    8                                                                                                                                                                                                                                                                                                                  
0.5     28
PVALUE  1                                                                                                                                                                                                                                                                                                                  
NA      128
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .3) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
NA      11
PVALUE  1
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .4) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
1       3
0.01    1
0.09    1
0.04    1
NA      11
PVALUE  1
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .5) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.09    2
0.25    1
1       9
0.01    2
0       2
0.04    3
0.5     3
PVALUE  1
NA      13
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .6) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.09    5
0.25    6
1       16
0.01    3
0       8
0.04    4
0.5     4
PVALUE  1
NA      16
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .7) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.09    7
0.25    9
1       31
0.01    3
0       22
0.04    6
0.5     6
PVALUE  1
NA      18
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .8) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.09    8
0.25    11
1       47
0.01    4
0       40
0.04    6
0.5     7
PVALUE  1
NA      23
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .85) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.09    10
0.25    13
1       64
0.01    4
0       55
0.04    7
0.5     11
PVALUE  1
NA      35
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .9) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.09    10
0.25    20
1       147
0.01    4
0       113
0.04    8
0.5     26
PVALUE  1
NA      59
~~~

#Pheno2
~~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane '$hash1{$F[10]}++; if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.09    7
0.25    7
1       300
0.01    5
0       318
0.04    9
0.5     19
PVALUE  1
NA      131
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .4) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.5     1
NA      11
PVALUE  1
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .5) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.5     1
NA      11
PVALUE  1
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .6) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.25    1
1       7
0.01    2
0       12
0.04    2
0.5     1
PVALUE  1
NA      11
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .7) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.25    1
1       7
0.01    2
0       12
0.04    2
0.5     1
PVALUE  1
NA      12
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .8) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.25    2
1       20
0.01    4
0       17
0.04    3
0.5     1
PVALUE  1
NA      22
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .85) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.25    2
1       21
0.01    4
0       17
0.04    3
0.5     1
PVALUE  1
NA      22
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .9) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.09    2
0.25    3
1       26
0.01    5
0       24
0.04    3
0.5     1
PVALUE  1
NA      23
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .95) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.09    7
0.25    7
1       271
0.01    5
0       256
0.04    9
0.5     19
PVALUE  1
NA      101
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .93) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.09    7
0.25    7
1       260
0.01    5
0       246
0.04    9
0.5     19
PVALUE  1
NA      100
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .92) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.09    4
0.25    7
1       202
0.01    5
0       160
0.04    6
0.5     19
PVALUE  1
NA      57
~~~






##20150320
##Looking for APOE on P2 for Sudhir

#From P2
~~~
[  michaelt@login2  ~]$vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --chr 19 --from-bp 45409039 --to-bp 45412650 --recode --recode-INFO-all --out nasasnoas.vcf

VCFtools - v0.1.11
(C) Adam Auton 2009

Parameters as interpreted:
--gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz
--chr 19
--to-bp 45412650
--recode-INFO-all
--out nasasnoas.vcf
--recode
--from-bp 45409039

Using zlib version: 1.2.3
Versions of zlib >= 1.2.4 will be *much* faster when reading zipped VCF files.
Reading Index file.
File contains 85200 entries and 747 individuals.
Filtering by chromosome.
Chromosome: 1
Chromosome: 2
Chromosome: 3
Chromosome: 4
Chromosome: 5
Chromosome: 6
Chromosome: 7
Chromosome: 8
Chromosome: 9
Chromosome: 10
Chromosome: 11
Chromosome: 12
Chromosome: 13
Chromosome: 14
Chromosome: 15
Chromosome: 16
Chromosome: 17
Chromosome: 18
Chromosome: 19
Chromosome: 20
Skipping Remainder.
Keeping 3459 entries on specified chromosomes.
Applying Required Filters.
Filtering sites by chromosome and/or position
After filtering, kept 747 out of 747 Individuals
After filtering, kept 13 out of a possible 3459 Sites
Outputting VCF file... Done
Run Time = 17.00 seconds
~~~

#From MM5
~~~
[  michaelt@node1  ~]$vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --chr 19 --from-bp 45409039 --to-bp 45492650 --recode --recode-INFO-all --out nasasnoas_MM5_vs2

VCFtools - v0.1.11
(C) Adam Auton 2009

Parameters as interpreted:
--gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz
--chr 19
--to-bp 45492650
--recode-INFO-all
--out nasasnoas_MM5_vs2
--recode
--from-bp 45409039

Using zlib version: 1.2.3
Versions of zlib >= 1.2.4 will be *much* faster when reading zipped VCF files.
Reading Index file.
File contains 63863 entries and 775 individuals.
Filtering by chromosome.
Chromosome: 1
Chromosome: 2
Chromosome: 3
Chromosome: 4
Chromosome: 5
Chromosome: 6
Chromosome: 7
Chromosome: 8
Chromosome: 9
Chromosome: 10
Chromosome: 11
Chromosome: 12
Chromosome: 13
Chromosome: 14
Chromosome: 15
Chromosome: 16
Chromosome: 17
Chromosome: 18
Chromosome: 19
Chromosome: 20
Skipping Remainder.
Keeping 2467 entries on specified chromosomes.
Applying Required Filters.
Filtering sites by chromosome and/or position
After filtering, kept 775 out of 775 Individuals
After filtering, kept 0 out of a possible 2467 Sites
Error:No data left for analysis!
~~~




##20150315
##Downsampling MM5 array to allSample.bam wc level comparable to P2 and rerunning GATK protocol as a real rough way to see if 'lower coverage' would affect the results we see in MM5 vs. P2
##Should also do QC level things on P2 to see singleton, doubleton calls, SFS, compared to MM5 results
##Downsampling randomly from allSample.bam file as a real, real rough first-attempt at this. Making assumption that 'all reads across all idnividuals' are more or less uniformly distributed so can just remove a 'proportion' of them from the overall .bam file to recapitulate a similar level of coverage between MM5 and P2

samtools view /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/AllPools.P2.Vs1.AllPoolsMerged.bam | wc
234582491 4458515601 618142147413

samtools flagstat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/AllPools.P2.Vs1.AllPoolsMerged.bam
234582491 + 0 in total (QC-passed reads + QC-failed reads)
0 + 0 duplicates
234582491 + 0 mapped (100.00%:-nan%)
0 + 0 paired in sequencing
0 + 0 read1
0 + 0 read2
0 + 0 properly paired (-nan%:-nan%)
0 + 0 with itself and mate mapped
0 + 0 singletons (-nan%:-nan%)
0 + 0 with mate mapped to a different chr
0 + 0 with mate mapped to a different chr (mapQ>=5)








##20150324
##Downloading and comparing overlap between ExAC data and my own for QC sake
##Data/ExAC data overlap QC analysis

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons 

python /home/michaelt/Scripts/Python/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --file2 /home/michaelt/Data/ExAC/ExAC.r0.3.sites.vep.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.gz 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.gz | grep VCF1 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF1.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.gz | grep VCF3a | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.gz | grep VCF3b | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3b.gz 

[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF1.gz | wc
69426 52555482 1206293273
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a.gz | wc
15774 11940918 279542898
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3b.gz | wc
15774  141966 49346091

paste <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a.gz | perl -ane 'print $F[1], "_", $F[2], "\t"; my @vals1 = split(/\;/, $F[8]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { print $entry1; } } print "\n";') <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3b.gz | perl -ane 'print $F[1], "_", $F[2], "\t"; my @vals1 = split(/\;/, $F[8]); my $AC_NFE; my $AN_NFE; foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AC_NFE=/) { $AC_NFE = (split(/,/, (split(/=/, $entry1))[1]))[0]; } if ($entry1 =~ m/^AN_NFE=/) { $AN_NFE = (split(/=/, $entry1))[1]; } } if (($AC_NFE) && ($AN_NFE)) { print "AF_NFE=", $AC_NFE/$AN_NFE; } else { print "NA"} print "\n";') > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs | grep -v NA | perl -lane '$F[1] = (split(/=/, $F[1]))[1]; $F[3] = (split(/=/, $F[3]))[1]; print join("\t", @F);' | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs.jpeg\", width=4000, height=5000, res=300); par(mfrow=c(3,2)); Data2 <- Data1[,1:2]; head(Data1); head(Data2); Data3a <- Data1[,3:4]; head(Data3a); Data3b <- merge(Data2, Data3a, by.x='V1', by.y='V3'); colnames(Data3b) <- c(\"Pos\", \"HIV\", \"ExAC\"); head(Data3b); plot(Data3b[,2], Data3b[,3], xlab=\"P2_HIV AF\", ylab=\"ExAC_NFE AF\", main=\"P2_HIV vs. ExAC_NFE AF\"); abline(a=0,b=1); plot(-log10(Data3b[,2]), -log10(Data3b[,3]), xlab=\"P2_HIV -log10(AF)\", ylab=\"ExAC_NFE -log10(AF)\", main=\"P2_HIV vs. ExAC_NFE -log10(AF)\"); abline(a=0,b=1); hist(Data3b[,2], xlab=\"P2_HIV AF\", main=\"P2_HIV AF Histogram\", breaks=100); hist(-log10(Data3b[,2]), xlab=\"P2_HIV -log10(AF)\", main=\"P2_HIV -log10(AF) Histogram\", breaks=100); hist(Data3b[,3], xlab=\"ExAC_NFE AF\", main=\"ExAC_NFE AF Histogram\", breaks=100); hist(-log10(Data3b[,3]), xlab=\"ExAC_NFE -log10(AF)\", main=\"ExAC_NFE -log10(AF) Histogram\", breaks=100); dev.off()"

~~
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs | grep -v NA | perl -lane '$F[1] = (split(/=/, $F[1]))[1]; $F[3] = (split(/=/, $F[3]))[1]; print join("\t", @F);' | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs.jpeg\", width=4000, height=5000, res=300); par(mfrow=c(3,2)); Data2 <- Data1[,1:2]; head(Data1); head(Data2); Data3a <- Data1[,3:4]; head(Data3a); Data3b <- merge(Data2, Data3a, by.x='V1', by.y='V3'); colnames(Data3b) <- c(\"Pos\", \"HIV\", \"ExAC\"); head(Data3b); plot(Data3b[,2], Data3b[,3], xlab=\"P2_HIV AF\", ylab=\"ExAC_NFE AF\", main=\"P2_HIV vs. ExAC_NFE AF\"); abline(a=0,b=1); plot(-log10(Data3b[,2]), -log10(Data3b[,3]), xlab=\"P2_HIV -log10(AF)\", ylab=\"ExAC_NFE -log10(AF)\", main=\"P2_HIV vs. ExAC_NFE -log10(AF)\"); abline(a=0,b=1); hist(Data3b[,2], xlab=\"P2_HIV AF\", main=\"P2_HIV AF Histogram\", breaks=100); hist(-log10(Data3b[,2]), xlab=\"P2_HIV -log10(AF)\", main=\"P2_HIV -log10(AF) Histogram\", breaks=100); hist(Data3b[,3], xlab=\"ExAC_NFE AF\", main=\"ExAC_NFE AF Histogram\", breaks=100); hist(-log10(Data3b[,3]), xlab=\"ExAC_NFE -log10(AF)\", main=\"ExAC_NFE -log10(AF) Histogram\", breaks=100); dev.off()"
> Data1 <- read.table(file('stdin'), header=F); jpeg("/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs.jpeg", width=4000, height=5000, res=300); par(mfrow=c(3,2)); Data2 <- Data1[,1:2]; head(Data1); head(Data2); Data3a <- Data1[,3:4]; head(Data3a); Data3b <- merge(Data2, Data3a, by.x='V1', by.y='V3'); colnames(Data3b) <- c("Pos", "HIV", "ExAC"); head(Data3b); plot(Data3b[,2], Data3b[,3], xlab="P2_HIV AF", ylab="ExAC_NFE AF", main="P2_HIV vs. ExAC_NFE AF"); abline(a=0,b=1); plot(-log10(Data3b[,2]), -log10(Data3b[,3]), xlab="P2_HIV -log10(AF)", ylab="ExAC_NFE -log10(AF)", main="P2_HIV vs. ExAC_NFE -log10(AF)"); abline(a=0,b=1); hist(Data3b[,2], xlab="P2_HIV AF", main="P2_HIV AF Histogram", breaks=100); hist(-log10(Data3b[,2]), xlab="P2_HIV -log10(AF)", main="P2_HIV -log10(AF) Histogram", breaks=100); hist(Data3b[,3], xlab="ExAC_NFE AF", main="ExAC_NFE AF Histogram", breaks=100); hist(-log10(Data3b[,3]), xlab="ExAC_NFE -log10(AF)", main="ExAC_NFE -log10(AF) Histogram", breaks=100); dev.off()
	V1        V2           V3           V4
1   7_38502677 0.0022030   7_38502677 7.550133e-05
2 12_113446944 0.0005525 12_113446944 3.747452e-04
3  17_66524802 0.0033260  5_149677245 1.962227e-03
4  5_149677245 0.0038500  11_45672261 8.466093e-01
5  19_40430459 0.0027780  19_40430459 5.187658e-04
6  9_140036587 0.0016540  9_140036587 2.998950e-05
	V1        V2
1   7_38502677 0.0022030
2 12_113446944 0.0005525
3  17_66524802 0.0033260
4  5_149677245 0.0038500
5  19_40430459 0.0027780
6  9_140036587 0.0016540
	V3           V4
1   7_38502677 7.550133e-05
2 12_113446944 3.747452e-04
3  5_149677245 1.962227e-03
4  11_45672261 8.466093e-01
5  19_40430459 5.187658e-04
6  9_140036587 2.998950e-05
	Pos       HIV         ExAC
1 10_112262561 0.0016560 4.180652e-03
2 10_112262625 0.0005519 2.998501e-05
3 10_112266771 0.3130000 3.108683e-01
4 10_112266805 0.0011040 8.991728e-05
5 10_112266827 0.0038630 4.422126e-03
6 10_112269846 0.0022050 1.509525e-04
null device 
	1 
> 
> 
~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/.

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF1.gz | perl -lane 'my @vals1 = split(/\;/, $F[8]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { print ((split(/=/, $entry1))[1]); } }' | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF1.AFplots.jpeg\", width=4000, height=1750, res=300); par(mfrow=c(1,2)); hist(Data1[,1], xlab=\"P2_HIV AF\", main=\"P2_HIV AF non-ExAC Overlap\", breaks=50); hist(-log10(Data1[,1]), xlab=\"P2_HIV -log10(AF)\", main=\"P2_HIV -log10(AF) non-ExAC Overlap\", breaks=50); dev.off();"
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF1.gz | perl -lane 'my @vals1 = split(/\;/, $F[8]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AC=/) { print ((split(/=/, $entry1))[1]); } }' | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF1.ACplots.jpeg\", width=4000, height=1750, res=300); par(mfrow=c(1,2)); hist(Data1[,1], xlab=\"P2_HIV AC\", main=\"P2_HIV AC non-ExAC Overlap\", breaks=50); hist(-log10(Data1[,1]), xlab=\"P2_HIV -log10(AC)\", main=\"P2_HIV -log10(AC) non-ExAC Overlap\", breaks=50); dev.off();"

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF1.AFplots.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF1.ACplots.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/.

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs6.txt.gz | grep exonic | perl -F, -lane 'print $F[0], "\t", (split(/_/, $F[1]))[0];' | sort | uniq > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs6.Exonic.justChrBP 

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --positions /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs6.Exonic.justChrBP  --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.Exonic 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.Exonic.recode.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.Exonic.vcf

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.Exonic.vcf

python /home/michaelt/Scripts/Python/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.Exonic.vcf.gz --file2 /home/michaelt/Data/ExAC/ExAC.r0.3.sites.vep.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.gz | grep VCF1 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.gz | grep VCF3a | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.gz | grep VCF3b | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3b.gz 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.gz | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a.gz | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3b.gz | wc

[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.gz | wc
   1784 1350488 31485648
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a.gz | wc
   8528 6455696 151386060
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3b.gz | wc
   8528   76752 27949001

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.gz | perl -lane 'my @vals1 = split(/\;/, $F[8]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { print ((split(/=/, $entry1))[1]); } }' | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.AFplots.jpeg\", width=4000, height=1750, res=300); par(mfrow=c(1,2)); hist(Data1[,1], xlab=\"HIV AF\", main=\"HIV Exonic AF non-ExAC Overlap\", breaks=50); hist(-log10(Data1[,1]), xlab=\"HIV -log10(AF)\", main=\"HIV Exonic -log10(AF) non-ExAC Overlap\", breaks=50); dev.off();"
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.gz | perl -lane 'my @vals1 = split(/\;/, $F[8]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AC=/) { print ((split(/=/, $entry1))[1]); } }' | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.ACplots.jpeg\", width=4000, height=1750, res=300); par(mfrow=c(1,2)); hist(Data1[,1], xlab=\"HIV AC\", main=\"HIV Exonic AC non-ExAC Overlap\", breaks=50); hist(-log10(Data1[,1]), xlab=\"HIV -log10(AC)\", main=\"HIV Exonic -log10(AC) non-ExAC Overlap\", breaks=50); dev.off();"

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.AFplots.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.ACplots.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/.

paste <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a.gz | perl -ane 'print $F[1], "_", $F[2], "\t"; my @vals1 = split(/\;/, $F[8]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { print $entry1; } } print "\n";') <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3b.gz | perl -ane 'print $F[1], "_", $F[2], "\t"; my @vals1 = split(/\;/, $F[8]); my $AC_NFE; my $AN_NFE; foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AC_NFE=/) { $AC_NFE = (split(/,/, (split(/=/, $entry1))[1]))[0]; } if ($entry1 =~ m/^AN_NFE=/) { $AN_NFE = (split(/=/, $entry1))[1]; } } if (($AC_NFE) && ($AN_NFE)) { print "AF_NFE=", $AC_NFE/$AN_NFE; } else { print "NA"} print "\n";') > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs

##cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs | grep -v NA | perl -lane '$F[1] = (split(/=/, $F[1]))[1]; $F[3] = (split(/=/, $F[3]))[1]; print join("\t", @F);' | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs.AFplots.jpeg\", width=4000, height=5000, res=300); par(mfrow=c(3,2)); Data2 <- Data1[,1:2]; head(Data1); head(Data2); Data3a <- Data1[,3:4]; head(Data3a); Data3b <- merge(Data2, Data3a, by.x='V1', by.y='V3'); colnames(Data3b) <- c(\"Pos\", \"HIV\", \"ExAC\"); head(Data3b); plot(Data3b[,2], Data3b[,3], xlab=\"HIV AF\", ylab=\"ExAC_NFE AF\", main=\"HIV vs. ExAC_NFE AF\"); abline(a=0,b=1); plot(-log10(Data3b[,2]), -log10(Data3b[,3]), xlab=\"HIV -log10(AF)\", ylab=\"ExAC_NFE -log10(AF)\", main=\"HIV vs. ExAC_NFE -log10(AF)\"); abline(a=0,b=1); hist(Data3b[,2], xlab=\"HIV AF\", main=\"HIV AF Histogram\", breaks=100); hist(-log10(Data3b[,2]), xlab=\"HIV -log10(AF)\", main=\"HIV -log10(AF) Histogram\", breaks=100); hist(Data3b[,3], xlab=\"ExAC_NFE AF\", main=\"ExAC_NFE AF Histogram\", breaks=100); hist(-log10(Data3b[,3]), xlab=\"ExAC_NFE -log10(AF)\", main=\"ExAC_NFE -log10(AF) Histogram\", breaks=100); dev.off()"

~~~
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs | grep -v NA | perl -lane '$F[1] = (split(/=/, $F[1]))[1]; $F[3] = (split(/=/, $F[3]))[1]; print join("\t", @F);' | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs.AFplots.jpeg\", width=4000, height=5000, res=300); par(mfrow=c(3,2)); Data2 <- Data1[,1:2]; head(Data1); head(Data2); Data3a <- Data1[,3:4]; head(Data3a); Data3b <- merge(Data2, Data3a, by.x='V1', by.y='V3'); colnames(Data3b) <- c(\"Pos\", \"HIV\", \"ExAC\"); head(Data3b); plot(Data3b[,2], Data3b[,3], xlab=\"HIV AF\", ylab=\"ExAC_NFE AF\", main=\"HIV vs. ExAC_NFE AF\"); abline(a=0,b=1); plot(-log10(Data3b[,2]), -log10(Data3b[,3]), xlab=\"HIV -log10(AF)\", ylab=\"ExAC_NFE -log10(AF)\", main=\"HIV vs. ExAC_NFE -log10(AF)\"); abline(a=0,b=1); hist(Data3b[,2], xlab=\"HIV AF\", main=\"HIV AF Histogram\", breaks=100); hist(-log10(Data3b[,2]), xlab=\"HIV -log10(AF)\", main=\"HIV -log10(AF) Histogram\", breaks=100); hist(Data3b[,3], xlab=\"ExAC_NFE AF\", main=\"ExAC_NFE AF Histogram\", breaks=100); hist(-log10(Data3b[,3]), xlab=\"ExAC_NFE -log10(AF)\", main=\"ExAC_NFE -log10(AF) Histogram\", breaks=100); dev.off()"
> Data1 <- read.table(file('stdin'), header=F); jpeg("/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs.AFplots.jpeg", width=4000, height=5000, res=300); par(mfrow=c(3,2)); Data2 <- Data1[,1:2]; head(Data1); head(Data2); Data3a <- Data1[,3:4]; head(Data3a); Data3b <- merge(Data2, Data3a, by.x='V1', by.y='V3'); colnames(Data3b) <- c("Pos", "HIV", "ExAC"); head(Data3b); plot(Data3b[,2], Data3b[,3], xlab="HIV AF", ylab="ExAC_NFE AF", main="HIV vs. ExAC_NFE AF"); abline(a=0,b=1); plot(-log10(Data3b[,2]), -log10(Data3b[,3]), xlab="HIV -log10(AF)", ylab="ExAC_NFE -log10(AF)", main="HIV vs. ExAC_NFE -log10(AF)"); abline(a=0,b=1); hist(Data3b[,2], xlab="HIV AF", main="HIV AF Histogram", breaks=100); hist(-log10(Data3b[,2]), xlab="HIV -log10(AF)", main="HIV -log10(AF) Histogram", breaks=100); hist(Data3b[,3], xlab="ExAC_NFE AF", main="ExAC_NFE AF Histogram", breaks=100); hist(-log10(Data3b[,3]), xlab="ExAC_NFE -log10(AF)", main="ExAC_NFE -log10(AF) Histogram", breaks=100); dev.off()
            V1        V2           V3           V4
1   7_38502677 0.0022030   7_38502677 7.550133e-05
2 12_113446944 0.0005525 12_113446944 3.747452e-04
3  5_149677245 0.0038500  5_149677245 1.962227e-03
4  19_40430459 0.0027780  19_40430459 5.187658e-04
5  9_140036587 0.0016540  9_140036587 2.998950e-05
6    3_4490950 0.0010980  15_74328206 1.007855e-01
            V1        V2
1   7_38502677 0.0022030
2 12_113446944 0.0005525
3  5_149677245 0.0038500
4  19_40430459 0.0027780
5  9_140036587 0.0016540
6    3_4490950 0.0010980
            V3           V4
1   7_38502677 7.550133e-05
2 12_113446944 3.747452e-04
3  5_149677245 1.962227e-03
4  19_40430459 5.187658e-04
5  9_140036587 2.998950e-05
6  15_74328206 1.007855e-01
           Pos       HIV         ExAC
1 10_112262561 0.0016560 4.180652e-03
2 10_112262625 0.0005519 2.998501e-05
3 10_112266827 0.0038630 4.422126e-03
4 10_112269846 0.0022050 1.509525e-04
5 10_112269883 0.0011030 1.354076e-04
6 10_112269908 0.0130000 1.489016e-02
null device 
          1 
> 
> 
~~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs.AFplots.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/. 

#20151231 NOTE -- Noticed that in the .justAFs files the SNPs were not automatically aligned with one another row-by-row. Occasionally SNPs would be off by a row or so. This looks to be because I just pasted together VCF3a and VCF3b assuming the two would match, but I should have used join (I don't think I knew about join at the time I wrote this exact code). However, doing join produces the same number of lines as the original justAFs file, so at least all SNPs are present in both columns it appears (if a bit out of order)? See below code and the redo of the AF vs. AF plots
#20151231 NOTE -- Apparently I did a 'merge' of sorts in R that was supposed to align the SNPs together by ChrBP. Still, some of the plot points showed differences in AF that the join --> AF1 - AF2 did not show. So redoing the plots with the justAF based on the linux join command and seeing if it looks different or not.
#20151231 NOTE -- The two plots don't look different actually, at least for the ExAC setup. 1000G might be different?

#join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) |

join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) | grep -v NA | perl -lane 'my $AF1 = (split(/\=/, $F[1]))[1]; my $AF2 = (split(/\=/, $F[2]))[1]; print $AF1, "\t", $AF2, "\t", $AF1 - $AF2;' | sort -rg -k 3,3 | vi -

#use table code on VCF1 AFs to get bin #s?
# table(cut(Data1[,2], seq(0,200,10), right=FALSE));"

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF1.gz | perl -lane 'splice(@F, 0, 1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 - | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a.gz | perl -lane 'splice(@F, 0, 1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 - | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz

#python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz 
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.gz | perl -lane 'splice(@F, 0, 1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 - | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz 

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a.gz | perl -lane 'splice(@F, 0, 1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 - | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz

join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) | grep -v NA | perl -lane 'my $AF1 = (split(/\=/, $F[1]))[1]; my $AF2 = (split(/\=/, $F[2]))[1]; print $AF1, "\t", $AF2, "\t", $AF1 - $AF2;' | grep -v NA | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs.AFplots.joinVers.jpeg\", width=4000, height=5000, res=300); par(mfrow=c(3,2)); plot(Data1[,1], Data1[,2], xlab=\"HIV AF\", ylab=\"ExAC_NFE AF\", main=\"HIV vs. ExAC_NFE AF\"); abline(a=0,b=1); plot(-log10(Data1[,1]), -log10(Data1[,2]), xlab=\"HIV -log10(AF)\", ylab=\"ExAC_NFE -log10(AF)\", main=\"HIV vs. ExAC_NFE -log10(AF)\"); abline(a=0,b=1); hist(Data1[,1], xlab=\"HIV AF\", main=\"HIV AF Histogram\", breaks=100); hist(-log10(Data1[,1]), xlab=\"HIV -log10(AF)\", main=\"HIV -log10(AF) Histogram\", breaks=100); hist(Data1[,2], xlab=\"ExAC_NFE AF\", main=\"ExAC_NFE AF Histogram\", breaks=100); hist(-log10(Data1[,2]), xlab=\"ExAC_NFE -log10(AF)\", main=\"ExAC_NFE -log10(AF) Histogram\", breaks=100); dev.off()"

join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) | grep -v NA | perl -lane 'my $AF1 = (split(/\=/, $F[1]))[1]; my $AF2 = (split(/\=/, $F[2]))[1]; print $AF1, "\t", $AF2, "\t", $AF1 - $AF2;' | grep -v NA | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs.AFplots.joinVers.jpeg\", width=4000, height=5000, res=300); par(mfrow=c(3,2)); plot(Data1[,1], Data1[,2], xlab=\"HIV AF\", ylab=\"ExAC_NFE AF\", main=\"HIV vs. ExAC_NFE AF\"); abline(a=0,b=1); plot(-log10(Data1[,1]), -log10(Data1[,2]), xlab=\"HIV -log10(AF)\", ylab=\"ExAC_NFE -log10(AF)\", main=\"HIV vs. ExAC_NFE -log10(AF)\"); abline(a=0,b=1); hist(Data1[,1], xlab=\"HIV AF\", main=\"HIV AF Histogram\", breaks=100); hist(-log10(Data1[,1]), xlab=\"HIV -log10(AF)\", main=\"HIV -log10(AF) Histogram\", breaks=100); hist(Data1[,2], xlab=\"ExAC_NFE AF\", main=\"ExAC_NFE AF Histogram\", breaks=100); hist(-log10(Data1[,2]), xlab=\"ExAC_NFE -log10(AF)\", main=\"ExAC_NFE -log10(AF) Histogram\", breaks=100); dev.off()"

~~~
[  michaelt@login2  ~]$python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz
NumVar: 69426 Exonic: 1417 Intronic: 48292 Intergenic: 962 UTR5: 1669 UTR3: 10522
Synonymous: 433 Nonsynonymous: 936 Stopgain: 35 Stoploss: 2
PP2_D: 359 PP2_P: 181 PP2_B: 286 PP2_-9: 83 Sift_D: 431 Sift_T: 413 Sift_-9: 65 LRT_D: 537 LRT_N: 297 LRT_U: 33 LRT_-9: 42
Common: 11123 Rare: 58303 Singletons: 7836 Doubletons: 8050
MeanQuality: 18914.5172718 QualityStd: 60874.4040481 MeanDepth: 12.2053472886 DepthStd: 4.97330009258 TiTvRatio: 2.90802139037 Transitions: 51661 Transversions: 17765
MeanHapScore: 1.71500629159 HapScoreStd: 1.11251912466 HapScore0Quart: 0.154 HapScore25Quart: 0.9356 HapScore50Quart: 1.4069 HapScore75Quart: 2.158875 HapScore100Quart: 9.8582
[  michaelt@login2  ~]$python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz
NumVar: 15774 Exonic: 8423 Intronic: 6007 Intergenic: 27 UTR5: 267 UTR3: 775
Synonymous: 3978 Nonsynonymous: 4297 Stopgain: 81 Stoploss: 8
PP2_D: 1213 PP2_P: 661 PP2_B: 1774 PP2_-9: 392 Sift_D: 1563 Sift_T: 2278 Sift_-9: 199 LRT_D: 1513 LRT_N: 2048 LRT_U: 201 LRT_-9: 278
Common: 2592 Rare: 13182 Singletons: 1768 Doubletons: 2077
MeanQuality: 23177.3606111 QualityStd: 71665.2223989 MeanDepth: 14.7409410837 DepthStd: 4.77960280091 TiTvRatio: 3.73267326733 Transitions: 12441 Transversions: 3333
MeanHapScore: 1.92702658172 HapScoreStd: 1.15662955628 HapScore0Quart: 0.2296 HapScore25Quart: 1.1292 HapScore50Quart: 1.62755 HapScore75Quart: 2.402275 HapScore100Quart: 8.9553
[  michaelt@login2  ~]$python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz
NumVar: 1784 Exonic: 1417 Intronic: 0 Intergenic: 0 UTR5: 0 UTR3: 0
Synonymous: 433 Nonsynonymous: 936 Stopgain: 35 Stoploss: 2
PP2_D: 359 PP2_P: 181 PP2_B: 286 PP2_-9: 83 Sift_D: 431 Sift_T: 413 Sift_-9: 65 LRT_D: 537 LRT_N: 297 LRT_U: 33 LRT_-9: 42
Common: 64 Rare: 1720 Singletons: 491 Doubletons: 371
MeanQuality: 4609.21646861 QualityStd: 32949.4964869 MeanDepth: 14.4698851207 DepthStd: 5.15583696527 TiTvRatio: 2.97327394209 Transitions: 1335 Transversions: 449
MeanHapScore: 1.72194960762 HapScoreStd: 1.01524647667 HapScore0Quart: 0.2587 HapScore25Quart: 1.015175 HapScore50Quart: 1.4738 HapScore75Quart: 2.13005 HapScore100Quart: 7.2113
[  michaelt@login2  ~]$python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz
NumVar: 8528 Exonic: 8423 Intronic: 0 Intergenic: 0 UTR5: 0 UTR3: 0
Synonymous: 3978 Nonsynonymous: 4297 Stopgain: 81 Stoploss: 8
PP2_D: 1213 PP2_P: 661 PP2_B: 1774 PP2_-9: 390 Sift_D: 1561 Sift_T: 2278 Sift_-9: 199 LRT_D: 1513 LRT_N: 2047 LRT_U: 201 LRT_-9: 277
Common: 1163 Rare: 7365 Singletons: 1034 Doubletons: 1233
MeanQuality: 20916.1048605 QualityStd: 70809.5132381 MeanDepth: 15.0552298089 DepthStd: 4.79125148866 TiTvRatio: 4.53047989624 Transitions: 6986 Transversions: 1542
MeanHapScore: 1.81009148687 HapScoreStd: 1.08358141385 HapScore0Quart: 0.2499 HapScore25Quart: 1.0786 HapScore50Quart: 1.53115 HapScore75Quart: 2.211225 HapScore100Quart: 8.7767
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ if ($8 >= .05) { print $0 } } ' | perl -lane 'my @vals1 = split(/,/, $F[8]); print $vals1[0];' | sort | uniq -c
   1541 UTR3
    207 UTR5
      1 UTR5;UTR3
    572 downstream
      2 exonic
    155 intergenic
   8164 intronic
     12 ncRNA_UTR3
      1 ncRNA_UTR5
     62 ncRNA_exonic
    122 ncRNA_intronic
    276 upstream
      8 upstream;downstream
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ if ($8 < .05) { print $0 } } ' | perl -lane 'my @vals1 = split(/,/, $F[8]); print $vals1[0];' | sort | uniq -c
   8981 UTR3
   1462 UTR5
     11 UTR5;UTR3
   2866 downstream
   1415 exonic
    807 intergenic
  40128 intronic
     61 ncRNA_UTR3
     25 ncRNA_UTR5
      1 ncRNA_UTR5;ncRNA_UTR3
    305 ncRNA_exonic
    694 ncRNA_intronic
     34 splicing
   1416 upstream
     97 upstream;downstream
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ if ($8 >= .05) { print $0 } } ' | perl -lane 'my @vals1 = split(/,/, $F[8]); print $vals1[0];' | sort | uniq -c
    164 UTR3
     56 UTR5
      1 downstream
   1142 exonic
      8 intergenic
   1175 intronic
     21 ncRNA_exonic
     15 ncRNA_intronic
      1 splicing
      6 upstream
      3 upstream;downstream
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ if ($8 < .05) { print $0 } } ' | perl -lane 'my @vals1 = split(/,/, $F[8]); print $vals1[0];' | sort | uniq -c
    611 UTR3
    211 UTR5
      6 UTR5;UTR3
      9 downstream
   7281 exonic
      4 exonic;splicing
     19 intergenic
   4832 intronic
      7 ncRNA_UTR3
      5 ncRNA_UTR5
     80 ncRNA_exonic
     59 ncRNA_intronic
     32 splicing
     22 upstream
      4 upstream;downstream
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ if ($8 >= .05) { print $0 } } ' | perl -lane 'my @vals1 = split(/,/, $F[8]); print $vals1[0];' | sort | uniq -c
      2 exonic
     62 ncRNA_exonic
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ if ($8 < .05) { print $0 } } ' | perl -lane 'my @vals1 = split(/,/, $F[8]); print $vals1[0];' | sort | uniq -c
   1415 exonic
    305 ncRNA_exonic
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ if ($8 >= .05) { print $0 } } ' | perl -lane 'my @vals1 = split(/,/, $F[8]); print $vals1[0];' | sort | uniq -c
   1142 exonic
     21 ncRNA_exonic
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ if ($8 < .05) { print $0 } } ' | perl -lane 'my @vals1 = split(/,/, $F[8]); print $vals1[0];' | sort | uniq -c
   7281 exonic
      4 exonic;splicing
     80 ncRNA_exonic
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ print $8 } ' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.05), right=FALSE));"
> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.05), right=FALSE));

  [0,0.05) [0.05,0.1) [0.1,0.15) [0.15,0.2) [0.2,0.25) [0.25,0.3) [0.3,0.35)
     58303       2021       1390       1019        906        729        566
[0.35,0.4) [0.4,0.45) [0.45,0.5) [0.5,0.55) [0.55,0.6) [0.6,0.65) [0.65,0.7)
       521        487        434        407        400        309        389
[0.7,0.75) [0.75,0.8) [0.8,0.85) [0.85,0.9) [0.9,0.95)   [0.95,1)
       305        255        227        192        148        415
>
>
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ print $8 } ' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.05), right=FALSE));"
> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.05), right=FALSE));

  [0,0.05) [0.05,0.1) [0.1,0.15) [0.15,0.2) [0.2,0.25) [0.25,0.3) [0.3,0.35)
     13182        486        299        242        210        153        133
[0.35,0.4) [0.4,0.45) [0.45,0.5) [0.5,0.55) [0.55,0.6) [0.6,0.65) [0.65,0.7)
       169        112         88         92         85         87         70
[0.7,0.75) [0.75,0.8) [0.8,0.85) [0.85,0.9) [0.9,0.95)   [0.95,1)
        61         50         44         56         45        109
>
>
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ print $8 } ' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.05), right=FALSE));"
> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.05), right=FALSE));

  [0,0.05) [0.05,0.1) [0.1,0.15) [0.15,0.2) [0.2,0.25) [0.25,0.3) [0.3,0.35)
      1720         19          3          6          4          4          2
[0.35,0.4) [0.4,0.45) [0.45,0.5) [0.5,0.55) [0.55,0.6) [0.6,0.65) [0.65,0.7)
         6          2          1          1          1          2          0
[0.7,0.75) [0.75,0.8) [0.8,0.85) [0.85,0.9) [0.9,0.95)   [0.95,1)
         1          3          2          0          0          7
>
>
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ print $8 } ' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.05), right=FALSE));"
> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.05), right=FALSE));

  [0,0.05) [0.05,0.1) [0.1,0.15) [0.15,0.2) [0.2,0.25) [0.25,0.3) [0.3,0.35)
      7365        205        124        120         85         66         60
[0.35,0.4) [0.4,0.45) [0.45,0.5) [0.5,0.55) [0.55,0.6) [0.6,0.65) [0.65,0.7)
        72         47         39         40         41         37         38
[0.7,0.75) [0.75,0.8) [0.8,0.85) [0.85,0.9) [0.9,0.95)   [0.95,1)
        33         26         19         25         22         63
>
>
[  michaelt@login2  ~]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) | grep -v NA | perl -lane 'my $AF1 = (split(/\=/, $F[1]))[1]; my $AF2 = (split(/\=/, $F[2]))[1]; print $AF1, "\t", $AF2, "\t", $AF1 - $AF2;' | grep -v NA | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs.AFplots.joinVers.jpeg\", width=4000, height=5000, res=300); par(mfrow=c(3,2)); plot(Data1[,1], Data1[,2], xlab=\"HIV AF\", ylab=\"ExAC_NFE AF\", main=\"HIV vs. ExAC_NFE AF\"); abline(a=0,b=1); plot(-log10(Data1[,1]), -log10(Data1[,2]), xlab=\"HIV -log10(AF)\", ylab=\"ExAC_NFE -log10(AF)\", main=\"HIV vs. ExAC_NFE -log10(AF)\"); abline(a=0,b=1); hist(Data1[,1], xlab=\"HIV AF\", main=\"HIV AF Histogram\", breaks=100); hist(-log10(Data1[,1]), xlab=\"HIV -log10(AF)\", main=\"HIV -log10(AF) Histogram\", breaks=100); hist(Data1[,2], xlab=\"ExAC_NFE AF\", main=\"ExAC_NFE AF Histogram\", breaks=100); hist(-log10(Data1[,2]), xlab=\"ExAC_NFE -log10(AF)\", main=\"ExAC_NFE -log10(AF) Histogram\", breaks=100); dev.off()"
> Data1 <- read.table(file('stdin'), header=F); jpeg("/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs.AFplots.joinVers.jpeg", width=4000, height=5000, res=300); par(mfrow=c(3,2)); plot(Data1[,1], Data1[,2], xlab="HIV AF", ylab="ExAC_NFE AF", main="HIV vs. ExAC_NFE AF"); abline(a=0,b=1); plot(-log10(Data1[,1]), -log10(Data1[,2]), xlab="HIV -log10(AF)", ylab="ExAC_NFE -log10(AF)", main="HIV vs. ExAC_NFE -log10(AF)"); abline(a=0,b=1); hist(Data1[,1], xlab="HIV AF", main="HIV AF Histogram", breaks=100); hist(-log10(Data1[,1]), xlab="HIV -log10(AF)", main="HIV -log10(AF) Histogram", breaks=100); hist(Data1[,2], xlab="ExAC_NFE AF", main="ExAC_NFE AF Histogram", breaks=100); hist(-log10(Data1[,2]), xlab="ExAC_NFE -log10(AF)", main="ExAC_NFE -log10(AF) Histogram", breaks=100); dev.off()
null device
          1
>
>
[  michaelt@login2  ~]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) | grep -v NA | perl -lane 'my $AF1 = (split(/\=/, $F[1]))[1]; my $AF2 = (split(/\=/, $F[2]))[1]; print $AF1, "\t", $AF2, "\t", $AF1 - $AF2;' | grep -v NA | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs.AFplots.joinVers.jpeg\", width=4000, height=5000, res=300); par(mfrow=c(3,2)); plot(Data1[,1], Data1[,2], xlab=\"HIV AF\", ylab=\"ExAC_NFE AF\", main=\"HIV vs. ExAC_NFE AF\"); abline(a=0,b=1); plot(-log10(Data1[,1]), -log10(Data1[,2]), xlab=\"HIV -log10(AF)\", ylab=\"ExAC_NFE -log10(AF)\", main=\"HIV vs. ExAC_NFE -log10(AF)\"); abline(a=0,b=1); hist(Data1[,1], xlab=\"HIV AF\", main=\"HIV AF Histogram\", breaks=100); hist(-log10(Data1[,1]), xlab=\"HIV -log10(AF)\", main=\"HIV -log10(AF) Histogram\", breaks=100); hist(Data1[,2], xlab=\"ExAC_NFE AF\", main=\"ExAC_NFE AF Histogram\", breaks=100); hist(-log10(Data1[,2]), xlab=\"ExAC_NFE -log10(AF)\", main=\"ExAC_NFE -log10(AF) Histogram\", breaks=100); dev.off()"
> Data1 <- read.table(file('stdin'), header=F); jpeg("/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs.AFplots.joinVers.jpeg", width=4000, height=5000, res=300); par(mfrow=c(3,2)); plot(Data1[,1], Data1[,2], xlab="HIV AF", ylab="ExAC_NFE AF", main="HIV vs. ExAC_NFE AF"); abline(a=0,b=1); plot(-log10(Data1[,1]), -log10(Data1[,2]), xlab="HIV -log10(AF)", ylab="ExAC_NFE -log10(AF)", main="HIV vs. ExAC_NFE -log10(AF)"); abline(a=0,b=1); hist(Data1[,1], xlab="HIV AF", main="HIV AF Histogram", breaks=100); hist(-log10(Data1[,1]), xlab="HIV -log10(AF)", main="HIV -log10(AF) Histogram", breaks=100); hist(Data1[,2], xlab="ExAC_NFE AF", main="ExAC_NFE AF Histogram", breaks=100); hist(-log10(Data1[,2]), xlab="ExAC_NFE -log10(AF)", main="ExAC_NFE -log10(AF) Histogram", breaks=100); dev.off()
null device 
          1 
> 
> 
~~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs.AFplots.joinVers.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs.AFplots.joinVers.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/.

#From Windows netbook via cmder
scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/*AFplots* .
#(C:\Users\PortMCT2\Desktop\Lab_Stuff\StephensLab\NUWrk\HIVStff\PG\ALL_MAPPING\Pools\P2\PostMerge\mapping_pool_merged\GATK\Analyses\VCFComparisons)

#zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'print join("\t", @F[0..9]);' | grep -f <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) | grep -v NA | perl -lane 'my $AF1 = (split(/\=/, $F[1]))[1]; my $AF2 = (split(/\=/, $F[2]))[1]; print $F[0], "\t", $AF1, "\t", $AF2, "\t", $AF1 - $AF2;' | sort -rg -k 4,4 | perl -lane 'if (abs($F[3]) > .1) { print join("\t", split(/_/, $F[0])); } ') - | perl -lane 'print $F[0], "_", $F[1], "\t", join("\t", ((split(/,/, $F[8]))[0..2]));' | sort -k 3,3
#zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'print join("\t", @F[0..9]);' | grep -f <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) | grep -v NA | perl -lane 'my $AF1 = (split(/\=/, $F[1]))[1]; my $AF2 = (split(/\=/, $F[2]))[1]; print $F[0], "\t", $AF1, "\t", $AF2, "\t", $AF1 - $AF2;' | sort -rg -k 4,4 | perl -lane 'if (abs($F[3]) > .1) { print join("\t", split(/_/, $F[0])); } ') - | perl -lane 'print $F[0], "_", $F[1], "\t", join("\t", ((split(/,/, $F[8]))[0..2]));' | sort -k 3,3
join <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) | grep -v NA | perl -lane 'my $AF1 = (split(/\=/, $F[1]))[1]; my $AF2 = (split(/\=/, $F[2]))[1]; print $F[0], "\t", $AF1, "\t", $AF2, "\t", $AF1 - $AF2;' | sort -rg -k 4,4 | perl -lane 'if (abs($F[3]) > .1) { print join("\t", @F); } ' | sort) <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'print join("\t", @F[0..9]);' | grep -f <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) | grep -v NA | perl -lane 'my $AF1 = (split(/\=/, $F[1]))[1]; my $AF2 = (split(/\=/, $F[2]))[1]; print $F[0], "\t", $AF1, "\t", $AF2, "\t", $AF1 - $AF2;' | sort -rg -k 4,4 | perl -lane 'if (abs($F[3]) > .1) { print join("\t", split(/_/, $F[0])); } ') - | perl -lane 'print $F[0], "_", $F[1], "\t", join("\t", ((split(/,/, $F[8]))[0..2]));' | sort )

~~~
[  michaelt@login2  ~]$join <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) | grep -v NA | perl -lane 'my $AF1 = (split(/\=/, $F[1]))[1]; my $AF2 = (split(/\=/, $F[2]))[1]; print $F[0], "\t", $AF1, "\t", $AF2, "\t", $AF1 - $AF2;' | sort -rg -k 4,4 | perl -lane 'if (abs($F[3]) > .1) { print join("\t", @F); } ' | sort) <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'print join("\t", @F[0..9]);' | grep -f <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) | grep -v NA | perl -lane 'my $AF1 = (split(/\=/, $F[1]))[1]; my $AF2 = (split(/\=/, $F[2]))[1]; print $F[0], "\t", $AF1, "\t", $AF2, "\t", $AF1 - $AF2;' | sort -rg -k 4,4 | perl -lane 'if (abs($F[3]) > .1) { print join("\t", split(/_/, $F[0])); } ') - | perl -lane 'print $F[0], "_", $F[1], "\t", $F[2], "\t", join("\t", ((split(/,/, $F[8]))[0..2]));' | sort ) | sort -rg -k 4,4
21_47422639 0.856 0.000130087344359784 0.85586991265564 rs2236485 intronic COL6A1 
20_1895820 0.366 0.000801254136909946 0.36519874586309 rs17855610 exonic SIRPA nonsynonymous_SNV
20_1895826 0.365 0.000833970394051011 0.364166029605949 rs17855611 exonic SIRPA nonsynonymous_SNV
20_1895815 0.364 0.000856613405125695 0.363143386594874 rs17853846 exonic SIRPA synonymous_SNV
20_1895835 0.364 0.00124863705110619 0.362751362948894 rs17855612 exonic SIRPA nonsynonymous_SNV
20_1895797 0.365 0.00226293103448276 0.362737068965517 rs146490405 exonic SIRPA synonymous_SNV
20_1895794 0.364 0.00270615650605127 0.361293843493949 rs1135192 exonic SIRPA synonymous_SNV
20_1895842 0.364 0.00326002873245663 0.360739971267543 rs17853847 exonic SIRPA synonymous_SNV
11_320166 0.883 0.70417124039517 0.17882875960483 rs28655829 intronic IFITM3 
21_47422644 0.341 0.193504506036388 0.147495493963612 rs2236486 intronic COL6A1 
12_52435691 0.115 0.218810364350896 -0.103810364350896 rs7316491 exonic NR4A1 synonymous_SNV
1_161090597 1.643e-03 0.1088428103178 -0.1071998103178 rs201942515 UTR3 NIT1 
15_101968084 0.204 0.311569544874521 -0.107569544874521 rs2277585 intronic PCSK6 
17_6662782 0.198 0.307096774193548 -0.109096774193548 rs3752861 intronic XAF1 
11_993745 0.216 0.325671852899576 -0.109671852899576 rs7104956 intronic AP2A2 
8_141780818 0.132 0.241970665663783 -0.109970665663783 rs79812711 intronic PTK2 
19_40421612 0.175 0.285531187695813 -0.110531187695813 rs34939346 exonic FCGBP nonsynonymous_SNV
19_50167726 0.155 0.266394779771615 -0.111394779771615 rs2304207 exonic IRF3 nonsynonymous_SNV
22_18222089 0.152 0.267609439740587 -0.115609439740587 rs372897 intronic BID 
8_39785321 0.368 0.484942372041145 -0.116942372041145 rs3739319 intronic IDO1 
17_44071294 0.229 0.348512585812357 -0.119512585812357 rs62063845 exonic MAPT synonymous_SNV
11_70253475 0.203 0.323137284778303 -0.120137284778303 rs2298397 exonic CTTN synonymous_SNV
3_4508742 0.255 0.376048360745017 -0.121048360745017 rs2819590 exonic SUMF1 nonsynonymous_SNV
17_21730703 0.225 0.352712765957447 -0.127712765957447 rs72842072 intergenic FAM27L(dist=94667) 
12_49332092 0.370 0.50253036437247 -0.13253036437247 rs1054376 UTR3 ARF3 
17_56618030 0.201 0.343065693430657 -0.142065693430657 rs8073754 exonic SEPT4 nonsynonymous_SNV
2_145155414 0.092 0.237762237762238 -0.145762237762238 rs13003631 intronic ZEB2 
14_95062574 0.463 0.619047619047619 -0.156047619047619 rs74834647 intergenic SERPINA3(dist=16140) 
6_44222047 0.095 0.251724137931034 -0.156724137931034 rs3187 UTR3 SLC35B2 
11_64577620 0.112 0.27763739734681 -0.16563739734681 rs509606 intronic MEN1 
17_43902861 0.228 0.4375 -0.2095 rs112126453 intronic CRHR1 
17_43902842 0.229 0.473684210526316 -0.244684210526316 rs41280116 intronic CRHR1 
17_43902944 0.229 0.481481481481481 -0.252481481481481 rs1912151 intronic CRHR1 
2_132236963 2.193e-03 0.27727952167414 -0.27508652167414 rs72992284 exonic TUBA3D synonymous_SNV
18_43332255 0.536 0.833333333333333 -0.297333333333333 rs3087560 UTR3 SLC14A1 
[  michaelt@login2  ~]$zcat /home/michaelt/Data/ExAC/ExAC.r0.3.sites.vep.vcf.gz | grep -v ^# | perl -lane 'print join("\t", @F[0..9]);' | grep 47422639  
21      47422639        .       ACCCGGGCAGTCCCAGATCTGCGTAGGTGCGCGCGGGGCCG       A,GCCCGGGCAGTCCCAGATCTGCGTAGGTGCGCGCGGGGCCG,TCCCGGGCAGTCCCAGATCTGCGTAGGTGCGCGCGGGGCCG   75971445.28     PASS    AC=94,91278,1;AC_AFR=0,8430,0;AC_AMR=0,6467,0;AC_Adj=8,75092,1;AC_EAS=1,3139,0;AC_FIN=0,4161,0;AC_Het=1,12544,0,7,0,1;AC_Hom=0,31270,0;AC_NFE=7,43135,1;AC_OTH=0,533,0;AC_SAS=0,9227,0;AF=8.145e-04,0.791,8.665e-06;AN=115402;AN_AFR=9434;AN_AMR=7486;AN_Adj=93912;AN_EAS=4398;AN_FIN=5154;AN_NFE=53810;AN_OTH=686;AN_SAS=12944;BaseQRankSum=1.71;ClippingRankSum=-4.100e-01;DP=1926443;FS=0.853;GQ_MEAN=161.82;GQ_STDDEV=198.20;Het_AFR=0,910,0,0,0,0;Het_AMR=0,721,0,0,0,0;Het_EAS=0,684,0,1,0,0;Het_FIN=0,727,0,0,0,0;Het_NFE=1,7312,0,6,0,1;Het_OTH=0,95,0,0,0,0;Het_SAS=0,2095,0,0,0,0;Hom_AFR=0,3760,0;Hom_AMR=0,2873,0;Hom_EAS=0,1227,0;Hom_FIN=0,1717,0;Hom_NFE=0,17908,0;Hom_OTH=0,219,0;Hom_SAS=0,3566,0;InbreedingCoeff=0.1933;MQ=59.51;MQ0=0;MQRankSum=0.100;NCC=5392;QD=31.50;ReadPosRankSum=0.424;VQSLOD=1.99;culprit=QD;DP_HIST=7589|1337|2761|3958|5006|6234|6642|6174|4991|3823|2630|1816|1273|804|592|478|361|304|260|668,38|18|28|6|4|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0,7518|1098|2249|3292|4408|5467|6014|5794|4731|3626|2500|1717|1203|748|563|442|343|286|242|628,0|0|0|0|0|0|1|0|0|0|0|0|0|0|0|0|0|0|0|0;GQ_HIST=1743|649|980|1710|2226|1773|1484|1386|980|1279|1308|940|1888|1654|997|1552|1667|1067|1679|30739,24|30|13|5|7|5|4|1|2|1|1|0|0|0|1|0|0|0|0|0,489|546|893|1576|2005|1605|1270|1184|793|1074|1141|779|1229|1350|862|1434|1576|1013|1610|30440,0|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0|1;CSQ=A|ENSG00000142156|ENST00000463060|Transcript|intron_variant&non_coding_transcript_variant&feature_truncation|||||||1||1|COL6A1|HGNC|2211|processed_transcript||||||||||5/5||ENST00000463060.1:n.833+15_833+55delACCCGGGCAGTCCCAGATCTGCGTAGGTGCGCGCGGGGCCGinsA|||||||||||||||||||,TCCCGGGCAGTCCCAGATCTGCGTAGGTGCGCGCGGGGCCG|ENSG00000142156|ENST00000463060|Transcript|intron_variant&non_coding_transcript_variant|||||||3||1|COL6A1|HGNC|2211|processed_transcript||||||||||5/5||ENST00000463060.1:n.833+15_833+55delACCCGGGCAGTCCCAGATCTGCGTAGGTGCGCGCGGGGCCGinsTCCCGGGCAGTCCCAGATCTGCGTAGGTGCGCGCGGGGCCG|||||||||||||||||||,GCCCGGGCAGTCCCAGATCTGCGTAGGTGCGCGCGGGGCCG|ENSG00000142156|ENST00000463060|Transcript|intron_variant&non_coding_transcript_variant|||||||2||1|COL6A1|HGNC|2211|processed_transcript||||||||||5/5||ENST00000463060.1:n.833+15_833+55delACCCGGGCAGTCCCAGATCTGCGTAGGTGCGCGCGGGGCCGinsGCCCGGGCAGTCCCAGATCTGCGTAGGTGCGCGCGGGGCCG|||||||||||||||||||,A|ENSG00000142156|ENST00000361866|Transcript|intron_variant&feature_truncation|||||||1||1|COL6A1|HGNC|2211|protein_coding|YES|CCDS13727.1|ENSP00000355180|CO6A1_HUMAN|Q05BT9_HUMAN|UPI000019B179||||33/34||ENST00000361866.3:c.2434+15_2434+55delACCCGGGCAGTCCCAGATCTGCGTAGGTGCGCGCGGGGCCGinsA|||||||||||||||||||,TCCCGGGCAGTCCCAGATCTGCGTAGGTGCGCGCGGGGCCG|ENSG00000142156|ENST00000361866|Transcript|intron_variant|||||||3||1|COL6A1|HGNC|2211|protein_coding|YES|CCDS13727.1|ENSP00000355180|CO6A1_HUMAN|Q05BT9_HUMAN|UPI000019B179||||33/34||ENST00000361866.3:c.2434+15_2434+55delACCCGGGCAGTCCCAGATCTGCGTAGGTGCGCGCGGGGCCGinsTCCCGGGCAGTCCCAGATCTGCGTAGGTGCGCGCGGGGCCG|||||||||||||||||||,GCCCGGGCAGTCCCAGATCTGCGTAGGTGCGCGCGGGGCCG|ENSG00000142156|ENST00000361866|Transcript|intron_variant|||||||2||1|COL6A1|HGNC|2211|protein_coding|YES|CCDS13727.1|ENSP00000355180|CO6A1_HUMAN|Q05BT9_HUMAN|UPI000019B179||||33/34||ENST00000361866.3:c.2434+15_2434+55delACCCGGGCAGTCCCAGATCTGCGTAGGTGCGCGCGGGGCCGinsGCCCGGGCAGTCCCAGATCTGCGTAGGTGCGCGCGGGGCCG|||||||||||||||||||,A|ENSG00000142156|ENST00000486023|Transcript|intron_variant&non_coding_transcript_variant&feature_truncation|||||||1||1|COL6A1|HGNC|2211|processed_transcript||||||||||1/2||ENST00000486023.1:n.222+15_222+55delACCCGGGCAGTCCCAGATCTGCGTAGGTGCGCGCGGGGCCGinsA|||||||||||||||||||,TCCCGGGCAGTCCCAGATCTGCGTAGGTGCGCGCGGGGCCG|ENSG00000142156|ENST00000486023|Transcript|intron_variant&non_coding_transcript_variant|||||||3||1|COL6A1|HGNC|2211|processed_transcript||||||||||1/2||ENST00000486023.1:n.222+15_222+55delACCCGGGCAGTCCCAGATCTGCGTAGGTGCGCGCGGGGCCGinsTCCCGGGCAGTCCCAGATCTGCGTAGGTGCGCGCGGGGCCG|||||||||||||||||||,GCCCGGGCAGTCCCAGATCTGCGTAGGTGCGCGCGGGGCCG|ENSG00000142156|ENST00000486023|Transcript|intron_variant&non_coding_transcript_variant|||||||2||1|COL6A1|HGNC|2211|processed_transcript||||||||||1/2||ENST00000486023.1:n.222+15_222+55delACCCGGGCAGTCCCAGATCTGCGTAGGTGCGCGCGGGGCCGinsGCCCGGGCAGTCCCAGATCTGCGTAGGTGCGCGCGGGGCCG|||||||||||||||||||,A|ENSG00000142156|ENST00000498614|Transcript|intron_variant&non_coding_transcript_variant&feature_truncation|||||||1||1|COL6A1|HGNC|2211|processed_transcript||||||||||4/5||ENST00000498614.1:n.668+15_668+55delACCCGGGCAGTCCCAGATCTGCGTAGGTGCGCGCGGGGCCGinsA|||||||||||||||||||,TCCCGGGCAGTCCCAGATCTGCGTAGGTGCGCGCGGGGCCG|ENSG00000142156|ENST00000498614|Transcript|intron_variant&non_coding_transcript_variant|||||||3||1|COL6A1|HGNC|2211|processed_transcript||||||||||4/5||ENST00000498614.1:n.668+15_668+55delACCCGGGCAGTCCCAGATCTGCGTAGGTGCGCGCGGGGCCGinsTCCCGGGCAGTCCCAGATCTGCGTAGGTGCGCGCGGGGCCG|||||||||||||||||||,GCCCGGGCAGTCCCAGATCTGCGTAGGTGCGCGCGGGGCCG|ENSG00000142156|ENST00000498614|Transcript|intron_variant&non_coding_transcript_variant|||||||2||1|COL6A1|HGNC|2211|processed_transcript||||||||||4/5||ENST00000498614.1:n.668+15_668+55delACCCGGGCAGTCCCAGATCTGCGTAGGTGCGCGCGGGGCCGinsGCCCGGGCAGTCCCAGATCTGCGTAGGTGCGCGCGGGGCCG|||||||||||||||||||   
[  michaelt@login2  ~]$zcat /home/michaelt/Data/ExAC/ExAC.r0.3.sites.vep.vcf.gz | grep -v ^# | perl -lane 'print join("\t", @F[0..9]);' | grep -w 1895820
.
.
.
20      1895820 rs17855610      C       T       334321.83       PASS    AC=1026;AC_AFR=12;AC_AMR=5;AC_Adj=82;AC_EAS=7;AC_FIN=0;AC_Het=78;AC_Hom=2;AC_NFE=46;AC_OTH=0;AC_SAS=12;AF=9.810e-03;AN=104586;AN_AFR=9354;AN_AMR=8468;AN_Adj=99632;AN_EAS=4928;AN_FIN=5678;AN_NFE=57410;AN_OTH=758;AN_SAS=13036;BaseQRankSum=-7.270e-01;ClippingRankSum=0.731;DB;DP=1805855;FS=2.421;GQ_MEAN=74.20;GQ_STDDEV=27.80;Het_AFR=12;Het_AMR=5;Het_EAS=7;Het_FIN=0;Het_NFE=42;Het_OTH=0;Het_SAS=12;Hom_AFR=0;Hom_AMR=0;Hom_EAS=0;Hom_FIN=0;Hom_NFE=2;Hom_OTH=0;Hom_SAS=0;InbreedingCoeff=0.5606;MQ=29.00;MQ0=0;MQRankSum=-1.733e+00;NCC=12181;QD=26.45;ReadPosRankSum=-2.900e-02;VQSLOD=106.32;culprit=MQ;DP_HIST=1689|376|587|1008|8382|9591|8729|6008|4802|3621|2574|1656|1099|655|444|318|207|141|92|228,391|61|5|3|8|8|14|8|11|4|6|2|1|5|0|1|0|2|2|14;GQ_HIST=782|845|135|181|768|474|464|471|389|590|581|471|13491|6669|3839|3734|2916|1958|2036|11499,3|89|61|76|51|18|31|21|16|18|12|8|13|16|10|14|13|5|6|151;CSQ=T|ENSG00000198053|ENST00000356025|Transcript|missense_variant|178|155|52|T/I|aCt/aTt|rs17855610|1||1|SIRPA|HGNC|9662|protein_coding||CCDS13022.1|ENSP00000348307|SHPS1_HUMAN|B4DP97_HUMAN|UPI0000073ADE|tolerated(0.08)|benign(0.101)|3/9||Pfam_domain:PF07686&Pfam_domain:PF00047&PROSITE_profiles:PS50835&SMART_domains:SM00409&SMART_domains:SM00406&Superfamily_domains:SSF48726|ENST00000356025.3:c.155C>T|ENSP00000348307.3:p.Thr52Ile|T:0.3108|T:0.18|T:0.31|T:0.53|T:0.23|||||||||||||,T|ENSG00000198053|ENST00000400068|Transcript|missense_variant|174|155|52|T/I|aCt/aTt|rs17855610|1||1|SIRPA|HGNC|9662|protein_coding|||ENSP00000382941|SHPS1_HUMAN||UPI0000071D10|tolerated(0.08)|benign(0.433)|3/9||Pfam_domain:PF07686&Pfam_domain:PF00047&PROSITE_profiles:PS50835&SMART_domains:SM00409&SMART_domains:SM00406&Superfamily_domains:SSF48726|ENST00000400068.3:c.155C>T|ENSP00000382941.3:p.Thr52Ile|T:0.3108|T:0.18|T:0.31|T:0.53|T:0.23|||||||||||||,T|ENSG00000198053|ENST00000358771|Transcript|missense_variant|307|155|52|T/I|aCt/aTt|rs17855610|1||1|SIRPA|HGNC|9662|protein_coding|YES|CCDS13022.1|ENSP00000351621|SHPS1_HUMAN|B4DP97_HUMAN|UPI0000073ADE|tolerated(0.08)|benign(0.101)|2/8||Pfam_domain:PF07686&Pfam_domain:PF00047&PROSITE_profiles:PS50835&SMART_domains:SM00409&SMART_domains:SM00406&Superfamily_domains:SSF48726|ENST00000358771.4:c.155C>T|ENSP00000351621.4:p.Thr52Ile|T:0.3108|T:0.18|T:0.31|T:0.53|T:0.23|||||||||||||
.
.
.
~~~

#20160105 NOTE -- dbSNP shows rs2236485 as triallelic too, though only w/ 3 different alleles and not the odd setup the ExAC data is showing. I think the results I'm presenting in the HIV data is more correct than what is being seen in the ExAC data

#20160105 NOTE -- Below is some information from dbSNP re: rs17855610 as a representative of the SIRPA region. Seems like the region has a bit of a track record for differing AFs across previous datasets (note how it's 'fixed' in HapMap, then avrying in 1000G, but also nearly fixed in ExAC). SIRPA in general seems to be varying among populations and for some reason a low frequency in ExAC in particular, eg other SNPs seem to follow these patterns it looks like
#~~~
#	Population Diversity (Alleles in RefSNP orientation) . See additional population frequency from 1000Genome [here]	back to top
#
#Sample Ascertainment	Genotype Detail	Alleles
#ss#	Population	Individual
#Group	Chrom.
#Sample Cnt.
#Source	C/C
#
#HWP	C
#	T
#
#ss1363819986	EAS		1008	AF			0.57539999	0.42460001	
#			
#EUR		1006	AF			0.84890002	0.15110001	
#			
#AFR		1322	AF			0.90850002	0.09150000	
#			
#AMR		694	AF			0.74640000	0.25360000	
#			
#SAS		978	AF			0.79240000	0.20760001	
#			
#ss1693874268	ExAc_Aggregated_Populations		104586	AF			0.99018991	0.00981011	
#			
#ss28509912	HapMap-CEU	European	116	IG	1.00000000		1.00000000		
#			
#HapMap-HCB	Asian	86	IG	1.00000000		1.00000000		
#			
#HapMap-JPT	Asian	88	IG	1.00000000		1.00000000		
#			
#HapMap-YRI	Sub-Saharan African	118	IG	1.00000000		1.00000000		
#			
#ENSEMBL_Watson		2	IG	1.00000000		1.00000000		
#			
#ENSEMBL_Venter		2	IG	1.00000000		1.00000000		
#~~~			

#20160105 CHECK_0 -- Prob: Followup with checking out SIRPA a little bit more? Look at reads in the region via IGV somehow? 
#20160105 CHECK_0 -- Prob: Or try seeing coverage in the area of SIRPA using bedtools coverage option also?
#20160105 CHECK_0 -- Prob: Would like to see a tool that shows pop AFs across a region?
#20160105 NOTE -- The point here is that it would be nice to see across the region, the AF is consistently different between multiple datasets so as to say, as a whole, ExAC seems unreasonable (for whatever reason) for this region. Or that it's hard to make a case that our data is any less or more believable than say 1000G, ExAC, HapMap, etc...









##20150324
##Downloading and comparing overlap between 1000G data and my own for QC sake
##Data/1000G data overlap QC analysis

python /home/michaelt/Scripts/Python/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --file2 /home/michaelt/Data/1000G/VCFs/ALL.chrAll.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.gz 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.gz | grep VCF1 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.gz | grep VCF3a | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.gz | fgrep VCF3b | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3b.gz

[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.gz | wc
30290 22929530 528593026
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a.gz | wc
54910 41566870 957282281
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3b.gz | wc
54910 138043740 558622312

paste <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a.gz | perl -ane 'print $F[1], "_", $F[2], "\t"; my @vals1 = split(/\;/, $F[8]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { print $entry1; } } print "\n";') <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3b.gz | perl -ane 'print $F[1], "_", $F[2], "\t"; my @vals1 = split(/\;/, $F[8]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^EUR_AF=/) { print $entry1; } } print "\n";') > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | grep -v NA | perl -lane '$F[1] = (split(/=/, $F[1]))[1]; $F[3] = (split(/,/, (split(/=/, $F[3]))[1]))[0]; print join("\t", @F);' | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs.AFplots.jpeg\", width=4000, height=5000, res=300); par(mfrow=c(3,2)); Data2 <- Data1[,1:2]; head(Data1); head(Data2); Data3a <- Data1[,3:4]; head(Data3a); Data3b <- merge(Data2, Data3a, by.x='V1', by.y='V3'); colnames(Data3b) <- c(\"Pos\", \"HIV\", \"1000G\"); head(Data3b); plot(Data3b[,2], Data3b[,3], xlab=\"P2_HIV AF\", ylab=\"1000G_EUR AF\", main=\"P2_HIV vs. 1000G_EUR AF\"); abline(a=0,b=1); plot(-log10(Data3b[,2]), -log10(Data3b[,3]), xlab=\"P2_HIV -log10(AF)\", ylab=\"1000G_EUR -log10(AF)\", main=\"P2_HIV vs. 1000G_EUR -log10(AF)\"); abline(a=0,b=1); hist(Data3b[,2], xlab=\"P2_HIV AF\", main=\"P2_HIV AF Histogram\", breaks=100); hist(-log10(Data3b[,2]), xlab=\"P2_HIV -log10(AF)\", main=\"P2_HIV -log10(AF) Histogram\", breaks=100); hist(Data3b[,3], xlab=\"1000G_EUR AF\", main=\"1000G_EUR AF Histogram\", breaks=100); hist(-log10(Data3b[,3]), xlab=\"1000G_EUR -log10(AF)\", main=\"1000G_EUR -log10(AF) Histogram\", breaks=100); dev.off()"

~~~
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | grep -v NA | perl -lane '$F[1] = (split(/=/, $F[1]))[1]; $F[3] = (split(/,/, (split(/=/, $F[3]))[1]))[0]; print join("\t", @F);' | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs.AFplots.jpeg\", width=4000, height=5000, res=300); par(mfrow=c(3,2)); Data2 <- Data1[,1:2]; head(Data1); head(Data2); Data3a <- Data1[,3:4]; head(Data3a); Data3b <- merge(Data2, Data3a, by.x='V1', by.y='V3'); colnames(Data3b) <- c(\"Pos\", \"HIV\", \"1000G\"); head(Data3b); plot(Data3b[,2], Data3b[,3], xlab=\"P2_HIV AF\", ylab=\"1000G_EUR AF\", main=\"P2_HIV vs. 1000G_EUR AF\"); abline(a=0,b=1); plot(-log10(Data3b[,2]), -log10(Data3b[,3]), xlab=\"P2_HIV -log10(AF)\", ylab=\"1000G_EUR -log10(AF)\", main=\"P2_HIV vs. 1000G_EUR -log10(AF)\"); abline(a=0,b=1); hist(Data3b[,2], xlab=\"P2_HIV AF\", main=\"P2_HIV AF Histogram\", breaks=100); hist(-log10(Data3b[,2]), xlab=\"P2_HIV -log10(AF)\", main=\"P2_HIV -log10(AF) Histogram\", breaks=100); hist(Data3b[,3], xlab=\"1000G_EUR AF\", main=\"1000G_EUR AF Histogram\", breaks=100); hist(-log10(Data3b[,3]), xlab=\"1000G_EUR -log10(AF)\", main=\"1000G_EUR -log10(AF) Histogram\", breaks=100); dev.off()"
> Data1 <- read.table(file('stdin'), header=F); jpeg("/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs.AFplots.jpeg", width=4000, height=5000, res=300); par(mfrow=c(3,2)); Data2 <- Data1[,1:2]; head(Data1); head(Data2); Data3a <- Data1[,3:4]; head(Data3a); Data3b <- merge(Data2, Data3a, by.x='V1', by.y='V3'); colnames(Data3b) <- c("Pos", "HIV", "1000G"); head(Data3b); plot(Data3b[,2], Data3b[,3], xlab="P2_HIV AF", ylab="1000G_EUR AF", main="P2_HIV vs. 1000G_EUR AF"); abline(a=0,b=1); plot(-log10(Data3b[,2]), -log10(Data3b[,3]), xlab="P2_HIV -log10(AF)", ylab="1000G_EUR -log10(AF)", main="P2_HIV vs. 1000G_EUR -log10(AF)"); abline(a=0,b=1); hist(Data3b[,2], xlab="P2_HIV AF", main="P2_HIV AF Histogram", breaks=100); hist(-log10(Data3b[,2]), xlab="P2_HIV -log10(AF)", main="P2_HIV -log10(AF) Histogram", breaks=100); hist(Data3b[,3], xlab="1000G_EUR AF", main="1000G_EUR AF Histogram", breaks=100); hist(-log10(Data3b[,3]), xlab="1000G_EUR -log10(AF)", main="1000G_EUR -log10(AF) Histogram", breaks=100); dev.off()
	V1       V2          V3     V4
1  6_32818629 0.032000  6_32818629 0.0338
2  20_1974571 0.002778  1_79109917 0.0010
3  1_79109917 0.002193  1_25881114 0.0050
4  1_25881114 0.004391 19_50868981 0.0050
5 12_49333248 0.137000 12_49333248 0.1441
6 4_108576106 0.004400 17_66528427 0.0268
	V1       V2
1  6_32818629 0.032000
2  20_1974571 0.002778
3  1_79109917 0.002193
4  1_25881114 0.004391
5 12_49333248 0.137000
6 4_108576106 0.004400
	V3     V4
1  6_32818629 0.0338
2  1_79109917 0.0010
3  1_25881114 0.0050
4 19_50868981 0.0050
5 12_49333248 0.1441
6 17_66528427 0.0268
	Pos      HIV  1000G
1 10_112261958 0.002765 0.0050
2 10_112262017 0.018000 0.0239
3 10_112262028 0.004425 0.0080
4 10_112262086 0.003315 0.0040
5 10_112262125 0.670000 0.6998
6 10_112262561 0.001656 0.0080
null device 
	1 
> 
> 
~~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs.AFplots.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/. 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.gz | perl -lane 'my @vals1 = split(/\;/, $F[8]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { print ((split(/=/, $entry1))[1]); } }' | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.AFplots.jpeg\", width=4000, height=1750, res=300); par(mfrow=c(1,2)); hist(Data1[,1], xlab=\"P2_HIV AF\", main=\"P2_HIV AF non-1000G Overlap\", breaks=50); hist(-log10(Data1[,1]), xlab=\"P2_HIV -log10(AF)\", main=\"P2_HIV -log10(AF) non-1000G Overlap\", breaks=50); dev.off();"
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.gz | perl -lane 'my @vals1 = split(/\;/, $F[8]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AC=/) { print ((split(/=/, $entry1))[1]); } }' | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.ACplots.jpeg\", width=4000, height=1750, res=300); par(mfrow=c(1,2)); hist(Data1[,1], xlab=\"P2_HIV AC\", main=\"P2_HIV AC non-1000G Overlap\", breaks=50); hist(-log10(Data1[,1]), xlab=\"P2_HIV -log10(AC)\", main=\"P2_HIV -log10(AC) non-1000G Overlap\", breaks=50); dev.off(); print(c(mean(Data1[,1]),sd(Data1[,1]),min(Data1[,1]),max(Data1[,1])));"

~~~
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.gz | perl -lane 'my @vals1 = split(/\;/, $F[8]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AC=/) { print ((split(/=/, $entry1))[1]); } }' | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.ACplots.jpeg\", width=4000, height=1750, res=300); par(mfrow=c(1,2)); hist(Data1[,1], xlab=\"P2_HIV AC\", main=\"P2_HIV AC non-1000G Overlap\", breaks=50); hist(-log10(Data1[,1]), xlab=\"P2_HIV -log10(AC)\", main=\"P2_HIV -log10(AC) non-1000G Overlap\", breaks=50); dev.off(); print(c(mean(Data1[,1]),sd(Data1[,1]),min(Data1[,1]),max(Data1[,1])));"
> Data1 <- read.table(file('stdin'), header=F); jpeg("/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.ACplots.jpeg", width=4000, height=1750, res=300); par(mfrow=c(1,2)); hist(Data1[,1], xlab="P2_HIV AC", main="P2_HIV AC non-1000G Overlap", breaks=50); hist(-log10(Data1[,1]), xlab="P2_HIV -log10(AC)", main="P2_HIV -log10(AC) non-1000G Overlap", breaks=50); dev.off(); print(c(mean(Data1[,1]),sd(Data1[,1]),min(Data1[,1]),max(Data1[,1])));
null device
	1
[1]    4.938825   41.328417    1.000000 1798.000000
>
>
~~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.AFplots.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.ACplots.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/.

#20151231 NOTE -- Continuing from the ExAC notes of this same time period...
#20151231 NOTE -- It would appear there are some SNPs that are encoded as tri-allelic in the 1000G vcf so, given the two AF possibilities present in the 1000G column of AFs, sometimes the 'wrong' AF was chosen (whichever one was 'first' was taken by default). This is why some SNPs appeared to have AF differences >.5 or something like that; just going to drop SNPs that have greater than two alleles in the 1000G data for the purposes of this analysis

#join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) |
join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) | grep -v NA | perl -lane 'my $AF1 = (split(/\=/, $F[1]))[1]; my $AF2 = (split(/\=/, $F[2]))[1]; print $AF1, "\t", $AF2, "\t", $AF1 - $AF2;' | sort -rg -k 3,3 | vi -

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.gz | perl -lane 'splice(@F, 0, 1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 - | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a.gz | perl -lane 'splice(@F, 0, 1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 - | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz

join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) | grep -v NA | perl -lane 'my $AF1 = (split(/\=/, $F[1]))[1]; my $AF2 = (split(/\=/, $F[2]))[1]; print $AF1, "\t", $AF2, "\t", $AF1 - $AF2;' | grep -v \, | grep -v NA | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs.AFplots.joinVers.jpeg\", width=4000, height=5000, res=300); par(mfrow=c(3,2)); plot(Data1[,1], Data1[,2], xlab=\"HIV AF\", ylab=\"1000G_EUR AF\", main=\"HIV vs. 1000G_EUR AF\"); abline(a=0,b=1); plot(-log10(Data1[,1]), -log10(Data1[,2]), xlab=\"HIV -log10(AF)\", ylab=\"1000G_EUR -log10(AF)\", main=\"HIV vs. 1000G_EUR -log10(AF)\"); abline(a=0,b=1); hist(Data1[,1], xlab=\"HIV AF\", main=\"HIV AF Histogram\", breaks=100); hist(-log10(Data1[,1]), xlab=\"HIV -log10(AF)\", main=\"HIV -log10(AF) Histogram\", breaks=100); hist(Data1[,2], xlab=\"1000G_EUR AF\", main=\"1000G_EUR AF Histogram\", breaks=100); hist(-log10(Data1[,2]), xlab=\"1000G_EUR -log10(AF)\", main=\"1000G_EUR -log10(AF) Histogram\", breaks=100); dev.off()"

~~~
[  michaelt@login2  ~]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) | grep -v NA | perl -lane 'my $AF1 = (split(/\=/, $F[1]))[1]; my $AF2 = (split(/\=/, $F[2]))[1]; print $AF1, "\t", $AF2, "\t", $AF1 - $AF2;' | sort -rg -k 3,3 | head -n 10
0.999   0,1     0.999
0.891   0,0.9284        0.891
0.727   0,0.7087        0.727
0.684   0,0.661 0.684
0.581   0,0.5875        0.581
0.544   0,0.5109        0.544
0.360   0,0.3926        0.36
0.330   0.002,0.3559    0.328
0.364   0.0537  0.3103
0.306   0,0.2823        0.306
[  michaelt@login2  ~]$python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz
NumVar: 30290 Exonic: 4225 Intronic: 18277 Intergenic: 335 UTR5: 765 UTR3: 4287
Synonymous: 1509 Nonsynonymous: 2610 Stopgain: 75 Stoploss: 3
PP2_D: 936 PP2_P: 438 PP2_B: 896 PP2_-9: 233 Sift_D: 1124 Sift_T: 1227 Sift_-9: 152 LRT_D: 1276 LRT_N: 973 LRT_U: 121 LRT_-9: 133
Common: 73 Rare: 30217 Singletons: 7845 Doubletons: 6594
MeanQuality: 532.95353417 QualityStd: 10785.5655271 MeanDepth: 12.9590036815 DepthStd: 5.1058709517 TiTvRatio: 2.71565260059 Transitions: 22138 Transversions: 8152
MeanHapScore: 1.71024142291 HapScoreStd: 1.053553403 HapScore0Quart: 0.154 HapScore25Quart: 0.961125 HapScore50Quart: 1.43565 HapScore75Quart: 2.16565 HapScore100Quart: 9.8582
[  michaelt@login2  ~]$python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz
NumVar: 54910 Exonic: 5615 Intronic: 36022 Intergenic: 654 UTR5: 1171 UTR3: 7010
Synonymous: 2902 Nonsynonymous: 2623 Stopgain: 41 Stoploss: 7
PP2_D: 636 PP2_P: 404 PP2_B: 1164 PP2_-9: 242 Sift_D: 870 Sift_T: 1464 Sift_-9: 112 LRT_D: 774 LRT_N: 1372 LRT_U: 113 LRT_-9: 187
Common: 13642 Rare: 41268 Singletons: 1759 Doubletons: 3533
MeanQuality: 30278.9255116 QualityStd: 76067.3096149 MeanDepth: 12.5180080859 DepthStd: 4.98896623488 TiTvRatio: 3.24146454503 Transitions: 41964 Transversions: 12946
MeanHapScore: 1.77854182116 HapScoreStd: 1.16007465813 HapScore0Quart: 0.1598 HapScore25Quart: 0.970025 HapScore50Quart: 1.4582 HapScore75Quart: 2.22815 HapScore100Quart: 8.9553
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ if ($8 < .05) { print $0 } } ' | perl -lane 'my @vals1 = split(/,/, $F[8]); print $vals1[0];' | sort | uniq -c
   4280 UTR3
    765 UTR5
      4 UTR5;UTR3
   1170 downstream
   4217 exonic
      1 exonic;splicing
    335 intergenic
  18224 intronic
     32 ncRNA_UTR3
     16 ncRNA_UTR5
    152 ncRNA_exonic
    312 ncRNA_intronic
     49 splicing
    622 upstream
     38 upstream;downstream
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ if ($8 >= .05) { print $0 } } ' | perl -lane 'my @vals1 = split(/,/, $F[8]); print $vals1[0];' | sort | uniq -c
      7 UTR3
      8 exonic
     53 intronic
      1 ncRNA_exonic
      2 ncRNA_intronic
      2 upstream
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ if ($8 >= .05) { print $0 } } ' | perl -lane 'my @vals1 = split(/,/, $F[8]); if ($vals1[0] eq "exonic") { print join("\t", @F[0..9]); }'
16      28617485        rs1801030       C       T       768860  VQSRTrancheSNP99.00to99.90      0.980   exonic,SULT1A1,nonsynonymous_SNV,NM_177536:c.G433A:p.V145M      GT:AD:DP:GQ:PL
5       149360877       rs30832 T       C       895158  VQSRTrancheSNP99.00to99.90      0.989   exonic,SLC26A2,nonsynonymous_SNV,NM_000112:c.T1721C:p.I574T     GT:AD:DP:GQ:PL
6       29523676        rs8337  C       G       554518  VQSRTrancheSNP99.00to99.90      0.730   exonic,UBD,nonsynonymous_SNV,NM_006398:c.G479C:p.C160S  GT:AD:DP:GQ:PL
2       132238169       rs2463337       A       G       2451.35 VQSRTrancheSNP90.00to99.00      0.062   exonic,TUBA3D,synonymous_SNV,NM_080386:c.A903G:p.Q301Q  GT:AD:DP:GQ:PL
20      1895984 rs17855615      C       A       141043  VQSRTrancheSNP99.00to99.90      0.367   exonic,SIRPA,nonsynonymous_SNV,NM_001040023:c.C319A:p.R107S     GT:AD:DP:GQ:PL
20      1896100 rs6136375       C       T       156990  VQSRTrancheSNP99.00to99.90      0.384   exonic,SIRPA,synonymous_SNV,NM_001040023:c.C435T:p.R145R        GT:AD:DP:GQ:PL
20      1895990 rs17855616      G       A       145816  VQSRTrancheSNP99.00to99.90      0.367   exonic,SIRPA,nonsynonymous_SNV,NM_001040023:c.G325A:p.G109S     GT:AD:DP:GQ:PL
12      65146532        rs1147096       C       T       207766  VQSRTrancheSNP99.00to99.90      0.607   exonic,GNS,synonymous_SNV,NM_002076:c.G198A:p.P66P      GT:AD:DP:GQ:PL
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ if ($8 < .05) { print $0 } } ' | perl -lane 'my @vals1 = split(/,/, $F[8]); print $vals1[0];' | sort | uniq -c
   5312 UTR3
    908 UTR5
     13 UTR5;UTR3
   1705 downstream
   4479 exonic
      3 exonic;splicing
    491 intergenic
  26736 intronic
     36 ncRNA_UTR3
     14 ncRNA_UTR5
      1 ncRNA_UTR5;ncRNA_UTR3
    233 ncRNA_exonic
    441 ncRNA_intronic
     17 splicing
    816 upstream
     63 upstream;downstream
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ if ($8 >= .05) { print $0 } } ' | perl -lane 'my @vals1 = split(/,/, $F[8]); print $vals1[0];' | sort | uniq -c
   1698 UTR3
    263 UTR5
      1 UTR5;UTR3
    573 downstream
   1136 exonic
    163 intergenic
   9286 intronic
     12 ncRNA_UTR3
      1 ncRNA_UTR5
     82 ncRNA_exonic
    135 ncRNA_intronic
      1 splicing
    280 upstream
     11 upstream;downstream
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ print $8 } ' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.05), right=FALSE));"
> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.05), right=FALSE));

  [0,0.05) [0.05,0.1) [0.1,0.15) [0.15,0.2) [0.2,0.25) [0.25,0.3) [0.3,0.35) 
     30217         17          3          7         10          1          3 
[0.35,0.4) [0.4,0.45) [0.45,0.5) [0.5,0.55) [0.55,0.6) [0.6,0.65) [0.65,0.7) 
         8          1          1          3          3          2          3 
[0.7,0.75) [0.75,0.8) [0.8,0.85) [0.85,0.9) [0.9,0.95)   [0.95,1) 
         1          0          2          3          0          5 
> 
> 
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ print $8 } ' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.05), right=FALSE));"
> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.05), right=FALSE));

  [0,0.05) [0.05,0.1) [0.1,0.15) [0.15,0.2) [0.2,0.25) [0.25,0.3) [0.3,0.35) 
     41268       2490       1686       1254       1106        881        696 
[0.35,0.4) [0.4,0.45) [0.45,0.5) [0.5,0.55) [0.55,0.6) [0.6,0.65) [0.65,0.7) 
       682        598        521        496        482        394        456 
[0.7,0.75) [0.75,0.8) [0.8,0.85) [0.85,0.9) [0.9,0.95)   [0.95,1) 
       365        305        269        245        193        519 
> 
> 
[  michaelt@login2  ~]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) | grep -v NA | perl -lane 'my $AF1 = (split(/\=/, $F[1]))[1]; my $AF2 = (split(/\=/, $F[2]))[1]; print $AF1, "\t", $AF2, "\t", $AF1 - $AF2;' | wc
  54910  164730 1260025
[  michaelt@login2  ~]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) | grep -v NA | perl -lane 'my $AF1 = (split(/\=/, $F[1]))[1]; my $AF2 = (split(/\=/, $F[2]))[1]; print $AF1, "\t", $AF2, "\t", $AF1 - $AF2;' | grep -v \, | wc
  54630  163890 1253124
[  michaelt@login2  ~]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) | grep -v NA | perl -lane 'my $AF1 = (split(/\=/, $F[1]))[1]; my $AF2 = (split(/\=/, $F[2]))[1]; print $AF1, "\t", $AF2, "\t", $AF1 - $AF2;' | grep -v \, | grep -v NA | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs.AFplots.joinVers.jpeg\", width=4000, height=5000, res=300); par(mfrow=c(3,2)); plot(Data1[,1], Data1[,2], xlab=\"HIV AF\", ylab=\"1000G_EUR AF\", main=\"HIV vs. 1000G_EUR AF\"); abline(a=0,b=1); plot(-log10(Data1[,1]), -log10(Data1[,2]), xlab=\"HIV -log10(AF)\", ylab=\"1000G_EUR -log10(AF)\", main=\"HIV vs. 1000G_EUR -log10(AF)\"); abline(a=0,b=1); hist(Data1[,1], xlab=\"HIV AF\", main=\"HIV AF Histogram\", breaks=100); hist(-log10(Data1[,1]), xlab=\"HIV -log10(AF)\", main=\"HIV -log10(AF) Histogram\", breaks=100); hist(Data1[,2], xlab=\"1000G_EUR AF\", main=\"1000G_EUR AF Histogram\", breaks=100); hist(-log10(Data1[,2]), xlab=\"1000G_EUR -log10(AF)\", main=\"1000G_EUR -log10(AF) Histogram\", breaks=100); dev.off()"
> Data1 <- read.table(file('stdin'), header=F); jpeg("/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs.AFplots.joinVers.jpeg", width=4000, height=5000, res=300); par(mfrow=c(3,2)); plot(Data1[,1], Data1[,2], xlab="HIV AF", ylab="1000G_EUR AF", main="HIV vs. 1000G_EUR AF"); abline(a=0,b=1); plot(-log10(Data1[,1]), -log10(Data1[,2]), xlab="HIV -log10(AF)", ylab="1000G_EUR -log10(AF)", main="HIV vs. 1000G_EUR -log10(AF)"); abline(a=0,b=1); hist(Data1[,1], xlab="HIV AF", main="HIV AF Histogram", breaks=100); hist(-log10(Data1[,1]), xlab="HIV -log10(AF)", main="HIV -log10(AF) Histogram", breaks=100); hist(Data1[,2], xlab="1000G_EUR AF", main="1000G_EUR AF Histogram", breaks=100); hist(-log10(Data1[,2]), xlab="1000G_EUR -log10(AF)", main="1000G_EUR -log10(AF) Histogram", breaks=100); dev.off()
null device
          1
>
>
~~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs.AFplots.joinVers.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/.

#From Windows netbook via cmder
scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/*AFplots* .
#(C:\Users\PortMCT2\Desktop\Lab_Stuff\StephensLab\NUWrk\HIVStff\PG\ALL_MAPPING\Pools\P2\PostMerge\mapping_pool_merged\GATK\Analyses\VCFComparisons)




















##20150324
##Trying to use /home/michaelt/Software/lollipops-v0.96-linux64 on top20/35 or so genes to see how things look and how they come out
##First run of lollipop graphs on top20/25 genes

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | sort -rg -k 2,2 |  head -n -12 | awk '{ print $1 }' | tail -n 35 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top35 

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/LollipopPlots

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top35 | perl -ane 'print $F[0], "\t"; for (my $i = 1; $i<=$#F; $i++) { my $chrBP1 = (split(/_/, $F[$i]))[0]; $chrBP1 =~ s/:/,/g; my @annoInfo1 = split(/,/, `zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | grep $F[0] | grep $chrBP1`); chomp(@annoInfo1); print ((split(/\./, ((split(/:/, $annoInfo1[28]))[2])))[1]); print "\t";} print "\n";' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/LollipopPlots/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top35.LollipopFormat 

OLD_IFS=$IFS
IFS=$'\n'

for command1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/LollipopPlots/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top35.LollipopFormat`
do

	IFS=$OLD_IFS

	geneID1=`echo $command1 | perl -ane 'print $F[0];'` 

#	echo $geneID1 $command1

	/home/michaelt/Software/lollipops-v0.96-linux64/lollipops -o=/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/LollipopPlots/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top35.${geneID1}.LollipopPlot.svg $command1
#	/home/michaelt/Software/lollipops-v0.96-linux64/lollipops -o=/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/LollipopPlots/P2.LollipopPlot.${geneID1}.Nonsyn.svg $command1

#	cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/LollipopPlots/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top35.${geneID1}.LollipopPlot.svg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/LollipopPlots/P2.LollipopPlot.${geneID1}.Nonsyn.svg

done

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/LollipopPlots/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top35.*.LollipopPlot.svg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/LollipopPlots/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/LollipopPlots/P2.LollipopPlot.*.Nonsyn.svg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/LollipopPlots/.


##20150330
##Getting 'excel sheets' prepped for sending on to Sumit/Kevin
##Also editting them to send a version to Sudhir as well

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ print $1 } ' | grep -vw NA | grep -vw NS > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs | head -n 5 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs.top5

rm -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/P2.skatO.nonsyn.qVals.ACs.Orig.txt

echo "Gene      pValue  qValue  NumVar  NumVar  NumVar  rho     AC_Total        AC_SeroNeg      AC_HESeroNeg    AC_SeroPos      AC_Rapid        AC_Slow" >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/P2.skatO.nonsyn.qVals.ACs.Orig.txt

for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs.top5`
do

	paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane 'if ($F[0] =~ m/^'$gene1'$/) { print join("\t", @F); }' | awk '{ print $1, "\t", $2, "\t", $4, "\t", $5, "\t", $8, "\t", $9, "\t", $11 }' ) \
	<(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs1.wACs.txt.gz | \
	grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'if ($F[0] =~ m/^'$gene1'$/) { print join("\t", @F); }' | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }') | \
	perl -lane 'print join("", ((split(/:/, $F[10]))[1])), "\t", ((split(/:/, $F[11]))[1]), "\t", ((split(/:/, $F[12]))[1]), "\t", ((split(/:/, $F[13]))[1]), "\t", ((split(/:/, $F[14]))[1]), "\t", ((split(/:/, $F[15]))[1]);' | \
	R -q -e "Data1 <- read.table(file('stdin'), header=F); print(paste(apply(Data1, 2, sum)), quote=FALSE);" | grep -v ^\> | perl -lane 'splice(@F, 0, 1); print join("\t", @F);') \
	>> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/P2.skatO.nonsyn.qVals.ACs.Orig.txt

done

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/P2.skatO.nonsyn.qVals.ACs.Orig.txt | perl -lane 'splice(@F, 4, 2); print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.ForSudhir.txt

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.ForSudhir.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/.

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/P2.skatO.nonsyn.qVals.ACs.Orig.txt | perl -lane 'splice(@F, 4, 3); print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/P2.skatO.nonsyn.qVals.ACs.txt

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/P2.skatO.nonsyn.qVals.ACs.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/.

#QC Checks on file

~~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/P2.skatO.nonsyn.qVals.ACs.txt | wc        
    786    7860   35959

#Check_0: Once have annotation file working properly (e.g. Total_AC working properly), go back and check why one variant is dropping out of the analysis for this one gene here?
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/P2.skatO.nonsyn.qVals.ACs.Orig.txt | perl -lane 'if (($F[3] == $F[4]) && ($F[3] == $F[5]) && ($F[4] == $F[5])) { print join("\t", @F); } ' | wc                                   
    785   10205   41299
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/P2.skatO.nonsyn.qVals.ACs.Orig.txt | perl -lane 'if (($F[3] != $F[4]) || ($F[3] != $F[5]) || ($F[4] != $F[5])) { print join("\t", @F); } ' 
WNK1    0.4272612000    0.88854 42      42      41      0       3926    1800    826     2126    290     422

[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/P2.skatO.nonsyn.qVals.ACs.txt | perl -lane 'my $val2 = $F[5] + $F[7]; if ($F[4] != $val2) { print join("\t", @F); }'
[  michaelt@node1  ~]$
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/P2.skatO.nonsyn.qVals.ACs.txt | perl -lane 'my $val2 = $F[5] + $F[7]; if ($F[4] == $val2) { print join("\t", @F); }' | wc
    786    7860   35959

[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/P2.skatO.nonsyn.qVals.ACs.txt | perl -lane 'my $val2 = $F[5] + $F[7]; if ($F[6] > $F[5]) { print join("\t", @F); }'
[  michaelt@node1  ~]$
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/P2.skatO.nonsyn.qVals.ACs.txt | perl -lane 'my $val2 = $F[5] + $F[7]; if ($F[6] <= $F[5]) { print join("\t", @F); }' | wc                                                         
    786    7860   35959

[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/P2.skatO.nonsyn.qVals.ACs.txt | perl -lane 'my $val2 = $F[8] + $F[9]; if ($F[7] < $val2) { print join("\t", @F); }'
[  michaelt@node1  ~]$
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/P2.skatO.nonsyn.qVals.ACs.txt | perl -lane 'my $val2 = $F[8] + $F[9]; if ($F[7] >= $val2) { print join("\t", @F); }' | wc
    786    7860   35959
~~~

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.ForSudhir.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.ForSudhir.txt



##20150423
##Working up code to run SKAT-O on msigdb pathways as a form of pathway analysis
##Also looking at prepping some of Kevin O's code/groupings to be used for this as well

cat /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt | perl -ane 'print $F[0], "\t"; my @genes; my $geneCount = 0; my @vars; my @varsPerGene; for (my $i = 2; $i <= $#F; $i++) {my $geneGrep = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -P "^$F[$i]\\s"`; if ($geneGrep) { my @geneGrepArray = split(/\s+/, $geneGrep); push(@genes, $F[$i]); $geneCount++; push(@varsPerGene, $#geneGrepArray); push(@vars, @geneGrepArray[1..$#geneGrepArray]); } } if ($geneCount == 0) { print $geneCount, "\t.\t.\t.\t.\n"; } else { print $geneCount, "\t", scalar(@vars), "\t", join(",", @varsPerGene), "\t", join(",", @genes), "\t", join("\t", @vars), "\n"; }' > /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt.P2.GroupFile.Exonic.Nonsynonymous.wSplice.pre 

cat /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt.P2.GroupFile.Exonic.Nonsynonymous.wSplice.pre | perl -lane 'if ($F[1] >= 2) { print $F[0], "\t", join("\t", @F[5..$#F]); }' > /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt.P2.GroupFile.Exonic.Nonsynonymous.wSplice

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/TruePheno/
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/TruePheno/Pheno1/
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/TruePheno/Pheno2/
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt.P2.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.msigdb_c2

P2TopPathways=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.msigdb_c2.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | sort -rg -k 10,10 | tail -n 11 | head -n 10 | awk '{ print $4 }' | perl -ane 'my @pathwayArray = split(/_/, $F[0]); if (eof()) { print join("_", @pathwayArray[1..$#pathwayArray]), "\n"; } else { print join("_", @pathwayArray[1..$#pathwayArray]), ";"; }'`

#rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.P2.Vs2.EPACTS.RMID5PCs.msigdb_c2.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneLevel.AllPathways

OLD_IFS=$IFS
IFS=$';'

for P2Pathway in $P2TopPathways
do
	IFS=$OLD_IFS
	echo $P2Pathway

#	rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.P2.Vs2.EPACTS.RMID5PCs.msigdb_c2.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneLevel.$P2Pathway

	echo $P2Pathway `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.msigdb_c2.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep $P2Pathway | awk '{ print $10 }'` >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.P2.Vs2.EPACTS.RMID5PCs.msigdb_c2.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneLevel.AllPathways

	GenesInPathway=`cat /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt.P2.GroupFile.Exonic.Nonsynonymous.wSplice.pre | grep -w $P2Pathway | awk '{ print $5 }' | perl -lane 'print join(";", split(/,/, $F[0]));'`

	OLD_IFS=$IFS
	IFS=$';'

	for GenePathway in $GenesInPathway
	do
		IFS=$OLD_IFS

		echo $GenePathway

#		cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.ForSudhir.txt | grep -w -P "^$GenePathway\\s" >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.P2.Vs2.EPACTS.RMID5PCs.msigdb_c2.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneLevel.$P2Pathway 
		cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.ForSudhir.txt | grep -w -P "^$GenePathway\\s" >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.P2.Vs2.EPACTS.RMID5PCs.msigdb_c2.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneLevel.AllPathways

	done

	echo "" >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.P2.Vs2.EPACTS.RMID5PCs.msigdb_c2.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneLevel.AllPathways

done

~~~
[  michaelt@node2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.P2.Vs2.EPACTS.RMID5PCs.msigdb_c2.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneLevel.* | awk '{ print $1 }' | sort | uniq -c | sort -k 1,1 | tail -n 25
1 RGS2
1 RSAD2
1 SAMD9L
1 SLC1A3
1 SLC2A3
1 SOD2
1 SPARC
1 SPP1
1 STXBP2
1 TCEB2
1 TNFSF10
1 TRA2A
1 UBE2J1
1 UBE2M
1 VPRBP
1 XAF1
2 COL1A2
2 COL6A1
2 LBH
2 MME
2 PLAU
2 PRDM1
2 PRRC2A
2 PSME4
3 ENPP2
~~~

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt.P2.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.msigdb_c2




##20150825
##Redoing .pheno file and specific, associated analyses, or at least creating a log here and editting them back in the main file above, because of the single indinvidual mix-up Sudhir identified
##Also going to be creating a specific script here, or at least a one-liner, to check various aspects of the file on which the phenotype codes are based on for QC/cross-checking purposes
##Going to be copying/pasting all the lines of code I am rerunning from above, so there may a large number of duplicate entries in terms of 'logging' specifically-run code. But this section will provide lines of code I did specifically rerun even if naming is same as before

#20150825 CHECK_0: Do this stuff

#From Sudhir 20150403 email chain -- redoing/fixing/correcting "30430   POOL87_02RL" indivdual -- only individual found with issue
~~
...
I checked the CAMACS_phenotypes against the master status list and found one discrepancy.

        (1)     MACSID  30430   POOL87_02RL     is listed in the CAMACS dataset as HESN and is seropositive.  I moved to seropositive in my reconstruction.

	In deidentifying the sequencing pool-midtag ID<80><99>s from Michael<80><99>s phenotype data, I could not resolve three samples.
	...
~~

#20150825? 20150913 CHECK_1: Create phenotype file QC check script #Ran a bunch of a one-liners, which can be seen in the main log portion around this time frame, that looks at and QCs the entries both for the original phenotype file and the coded phenotype file

#Put pheno redo correction here?
#
#Have to redo group-pheno level QC stats
#Perm files
#Pathway analyses
#Mktest?
#20150825
#Current to-do list
#
#Redo MultinRun TarAndClean and PermutationRun TarAndClean


#CHECK_0: Really need to redo those subsampling-to-P2 analyses, but on the individual .bam level files first?

#20150907 -- First is to just redo all the prep code and prep files, worry about redoing analyses once all the prep work/steps have been redone properly
#Then run the base-level SKAT-O analyses, run the scripts that collect that information, then the analyses that operate on top of those results
#Note -- all the code below was determined by copying/pasting a large portion of the log from '7th run...' onwards and determining which lines needed to be rerun. Additionaly lines of code were also included, generally referring to renaming previous '...pheno' files to '...vs2.pheno' files to begin this organization system of 'vs2', 'vs3', etc...


mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs1.pheno

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.pheno

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar --covar-number 2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.10PCs
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar --covar-number 2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.10PCs.maf05
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno --mpheno 2 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno2
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno --mpheno 3 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno3
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno --mpheno 4 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno4

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.assoc.logistic Pheno1_justWhite_Logistic_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.assoc.logistic.QQplot.jpeg
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.assoc.logistic Pheno1_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.assoc.logistic.QQplot.jpeg
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.10PCs.assoc.logistic Pheno1_justWhite_Logistic_10PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.10PCs.assoc.logistic.QQplot.jpeg
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.10PCs.maf05.assoc.logistic Pheno1_justWhite_Logistic_maf05_10PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.10PCs.maf05.assoc.logistic.QQplot.jpeg
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno2.assoc.logistic Pheno2_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno2.assoc.logistic.QQplot.jpeg
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno3.assoc.logistic Pheno3_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno3.assoc.logistic.QQplot.jpeg
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno4.assoc.logistic Pheno4_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno4.assoc.logistic.QQplot.jpeg

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/.

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.pheno.FIDs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroNeg.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.pheno.justSeroNeg.FIDs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroHighlyExposedNeg.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.pheno.justSeroHighlyExposedNeg.FIDs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroPos.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.pheno.justSeroPos.FIDs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justRapidVRapid.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.pheno.justRapidVRapid.FIDs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSlowVSlow.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.pheno.justSlowVSlow.FIDs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVeryRapid.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.pheno.justVeryRapid.FIDs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVerySlow.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.pheno.justVerySlow.FIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | awk '{ print $1 }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.FIDs 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | awk '{ if ($3 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroNeg.FIDs 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | awk '{ if ($4 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroHighlyExposedNeg.FIDs 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | awk '{ if ($3 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroPos.FIDs 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | awk '{ if ($5 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justRapidVRapid.FIDs 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | awk '{ if ($5 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSlowVSlow.FIDs 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | awk '{ if ($6 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justVeryRapid.FIDs 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | awk '{ if ($6 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justVerySlow.FIDs 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.frq.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.frq.count.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.justSeroNeg.frq.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.justSeroNeg.frq.count.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.justSeroHighlyExposedNeg.frq.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.justSeroHighlyExposedNeg.frq.count.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.justSeroPos.frq.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.justSeroPos.frq.count.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.justRapidVRapid.frq.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.justRapidVRapid.frq.count.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.justSlowVSlow.frq.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.justSlowVSlow.frq.count.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.justVeryRapid.frq.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.justVeryRapid.frq.count.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.justVerySlow.frq.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.justVerySlow.frq.count.gz

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2 &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroNeg.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroNeg &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroHighlyExposedNeg.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroHighlyExposedNeg &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroPos.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroPos &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justRapidVRapid.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justRapidVRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSlowVSlow.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSlowVSlow &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justVeryRapid.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVeryRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justVerySlow.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVerySlow & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2 &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroNeg &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroHighlyExposedNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroHighlyExposedNeg &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroPos.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroPos &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justRapidVRapid.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justRapidVRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSlowVSlow.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSlowVSlow &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justVeryRapid.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVeryRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justVerySlow.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVerySlow & 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.AllPhenos.frq.AndCount.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.AllPhenos.frq.AndCount.gz

paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVerySlow.frq.count | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.AllPhenos.frq.AndCount.gz
gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVerySlow.frq.count 
#gunzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroNeg.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroHighlyExposedNeg.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroPos.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justRapidVRapid.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSlowVSlow.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVeryRapid.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVerySlow.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroNeg.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroHighlyExposedNeg.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroPos.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justRapidVRapid.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSlowVSlow.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVeryRapid.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVerySlow.frq.count.gz 



python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.AllPhenos.frq.AndCount.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 30; $i <= 40; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v \# | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/\_2/) { print join("\t", @F); }' | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/\_3/) { print join("\t", @F); }' | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/\_4/) { print join("\t", @F); }' | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/\_5/) { print join("\t", @F); }' | wc

~~~
[  michaelt@node18  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v \# | wc
85200 64411200 1485394397
[  michaelt@node18  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz | wc
87510   97379 38909004
[  michaelt@node18  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/\_2/) { print join("\t", @F); }' | wc
2272   74181 1012781
[  michaelt@node18  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/\_3/) { print join("\t", @F); }' | wc
35    1133   15428
[  michaelt@node18  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/\_4/) { print join("\t", @F); }' | wc
2      64     844
[  michaelt@node18  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/\_5/) { print join("\t", @F); }' | wc
1      32     412
R -q -e 'val1 = 2272 + 35 + 2 + 1; print (val1);'
> val1 = 2272 + 35 + 2 + 1; print (val1);
[1] 2310
>
>
R -q -e 'val1 = 2272 + 35 + 2 + 1; print(val1); val2 = 87510 - 2310; print(val2);'
> val1 = 2272 + 35 + 2 + 1; print(val1); val2 = 87510 - 2310; print(val2);
[1] 2310
[1] 85200
>
>
~~~

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.bim.wGeneIDs

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs3.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.All /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.bim.wGeneIDs.SetIDFile.All 
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.wSplice /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.wSplice
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_PD /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_PD
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_D /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing") || ($7 == "UTR5;UTR3")) { print $8, "\t", $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.SetIDFile.All
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "splicing") || ($7 == "exonic;splicing")) { print $8, "\t", $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.SetIDFile.Exonic
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") )  { print $8, "\t", $2 } } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs | awk -F, '{ if ((($7 == "exonic") && (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV"))) || (($7 == "splicing") || ($7 == "exonic;splicing"))) { print $8, "\t", $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.wSplice
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($12 == "P") || ($12 == "D") ) { print $8, "\t", $2 } } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_PD
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $8, "\t", $2 } } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_D

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.All /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.bim.wGeneIDs.GroupFile.All
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.bim.wGeneIDs.GroupFile.Exonic
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing") || ($7 == "UTR5;UTR3")) { print $0 } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.All 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "splicing") || ($7 == "exonic;splicing")) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $0 } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs | awk -F, '{ if ((($7 == "exonic") && (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV"))) || (($7 == "splicing") || ($7 == "exonic;splicing"))) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($12 == "P") || ($12 == "D") ) { print $0 } } } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $0 } } } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($13 == "D") { print $0 } } } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs1.EPACTSedit.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.pheno | awk '{ print $1, "\t", $2, "\t0\t0\t0\t", $3, "\t", $4, "\t", $5, "\t", $6 } ' | sed 's/-9/NA/g' | perl -lane 'print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.EPACTSedit.ped 

join /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar | perl -lane 'splice(@F, 9, 2); splice(@F, 10, 1); print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.EPACTSedit.ped.vs2 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.EPACTSedit.ped.vs2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.EPACTSedit.ped

#Add line #FAM_ID IND_ID FAT_ID MOT_ID SEX HIVPROG HIVPROGHE AIDS AIDSEXTR RMID PC1 PC2 PC3 PC4 PC5 PC6 PC7 PC8 PC9 PC10 to /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.EPACTSedit.ped 

ls -lrt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/. | awk '{ print $9 }' | perl -lane 'if ($F[0] =~ m/(AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.)(EPACTSedit.perm\d+.vs3.ped)/) { my $val1 = "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/" . $1 . $2; my $val2 = "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/" . $1 . "vs1." . $2; system("mv $val1 $val2"); }' 

for i in {1..1500}
do
	SEED1=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.PermutationList.ForP2.vs1.txt | awk '{ if (NR == '$i') { print $0 } } '`
	perl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs3.pl --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.EPACTSedit.ped --seed $SEED1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.EPACTSedit.perm${i}.vs3.ped

done

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs 

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno1 Exonic.Nonsynonymous OLD2
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs.top5  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs.top5
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.qVal_lte85.GeneIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.qVal_lte85.GeneIDs
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno2 Exonic.Nonsynonymous OLD2
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno3 Exonic.Nonsynonymous OLD2
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno4 Exonic.Nonsynonymous OLD2

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno1 Exonic.Nonsynonymous OLD2
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno2 Exonic.Nonsynonymous OLD2
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno3 Exonic.Nonsynonymous OLD2
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno4 Exonic.Nonsynonymous OLD2

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs3.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs TruePheno Pheno1 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs3.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs TruePheno Pheno2 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs3.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs TruePheno Pheno3 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs3.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs TruePheno Pheno4 Exonic.Nonsynonymous

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs3.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs FakePheno Pheno1 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs3.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs FakePheno Pheno2 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs3.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs FakePheno Pheno3 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs3.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs FakePheno Pheno4 Exonic.Nonsynonymous

~~~
[  michaelt@node1  ~]$ls -lrt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno*/Exonic.Nonsynonymous/. | grep RedoList
[  michaelt@node1  ~]$
~~~

for i in {1..100}
do 
        if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run${i}.epacts ] ; then
		echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.RedoList.TruePheno.Pheno1.Exonic.Nonsynonymous
	fi
        if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run${i}.epacts ] ; then
		echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.RedoList.TruePheno.Pheno2.Exonic.Nonsynonymous
	fi
        if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run${i}.epacts ] ; then
		echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.RedoList.TruePheno.Pheno3.Exonic.Nonsynonymous
	fi
        if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run${i}.epacts ] ; then
		echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.RedoList.TruePheno.Pheno4.Exonic.Nonsynonymous
	fi
done

~~~
[  michaelt@node1  ~]$for i in {1..100}
> do
>         if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run${i}.epacts ] ; then
>                 echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.RedoList.TruePheno.Pheno1.Exonic.Nonsynonymous
>         fi
>         if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run${i}.epacts ] ; then
>                 echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.RedoList.TruePheno.Pheno2.Exonic.Nonsynonymous
>         fi
>         if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run${i}.epacts ] ; then
>                 echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.RedoList.TruePheno.Pheno3.Exonic.Nonsynonymous
>         fi
>         if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run${i}.epacts ] ; then
>                 echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.RedoList.TruePheno.Pheno4.Exonic.Nonsynonymous
>         fi
> done
[  michaelt@node1  ~]$ls -lrt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno*/Exonic.Nonsynonymous/. | grep RedoList
[  michaelt@node1  ~]$
~~~

~~~
[  michaelt@node1  ~]$ls -lrt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno*/Exonic.Nonsynonymous/. | grep RedoList    
[  michaelt@node1  ~]$
~~~

for i in {1..1000}
do 
        if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm${i}.epacts ] ; then
		echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno1.Exonic.Nonsynonymous
	fi
        if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm${i}.epacts ] ; then
		echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno2.Exonic.Nonsynonymous
	fi
        if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm${i}.epacts ] ; then
		echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno3.Exonic.Nonsynonymous
	fi
        if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm${i}.epacts ] ; then
		echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno4.Exonic.Nonsynonymous
	fi
done

~~~
[  michaelt@node1  ~]$for i in {1..1000}                                                                                                                                > do                                                                                                                                                                    >         if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm${i}.epacts ] ; then                                                                                                         >                 echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno1.Exonic.Nonsynonymous                           >         fi                                                                                                                                                            >         if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm${i}.epacts ] ; then                                                                                                         >                 echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno2.Exonic.Nonsynonymous                           >         fi                                                                                                                                                            >         if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm${i}.epacts ] ; then                                                                                                         >                 echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno3.Exonic.Nonsynonymous
>         fi
>         if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm${i}.epacts ] ; then
>                 echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno4.Exonic.Nonsynonymous
>         fi
> done
[  michaelt@node1  ~]$
[  michaelt@node1  ~]$ls -lrt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno*/Exonic.Nonsynonymous/. | grep RedoList    
-rw-rw-r-- 1 michaelt pg       8 Oct  3 02:03 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno3.Exonic.Nonsynonymous
[  michaelt@node1  ~]$
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno3.Exonic.Nonsynonymous                                                                    167                                                                                                                                                                     643                   
~~~

#Ran for Pheno3 167 and 643
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno3.Exonic.Nonsynonymous /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/OLD3.a.DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno3.Exonic.Nonsynonymous

#Redid missing file check
~~~
[  michaelt@node1  ~]$ls -lrt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno*/Exonic.Nonsynonymous/. | grep RedoList
-rw-rw-r-- 1 michaelt pg       8 Oct  3 02:03 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/OLD3.a.DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno3.Exonic.Nonsynonymous
[  michaelt@node1  ~]$                                          
[  michaelt@node1  ~]$ls -lrt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno*/Exonic.Nonsynonymous/. | grep RedoList | grep -v OLD
[  michaelt@node1  ~]$
~~~

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Results/Checkpoints/2ndRun/20151003

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno*/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno*.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno*/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno*.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno*/Exonic.Nonsynonymous/Data*.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno*/Exonic.Nonsynonymous/All*.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno*/Exonic.Nonsynonymous/Data*.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Results/Checkpoints/2ndRun/20151003/.

#20150925 CHECK_0: Look at/over work desktop to make sure proper files/folders were overwritten with the below system/commands. Maybe just use the 'checkpoint' file/folder system instead for the desktop specifically??
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justpVals /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/Data*.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/All*.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/Data*.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justpVals /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/Data*.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/All*.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/Data*.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justpVals /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/Data*.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/All*.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/Data*.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justpVals /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/Data*.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/All*.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/Data*.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/.

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno*/Exonic.Nonsynonymous/Data*jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno*/Exonic.Nonsynonymous/All*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/TempStuff/.
mkdir /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/TempStuff
ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.permAll.epacts.NullDistrQQPlotCheck.vs1.jpeg /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/TempStuff/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.permAll.epacts.NullDistrQQPlotCheck.vs1.jpeg
ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.permAll.epacts.NullDistrQQPlotCheck.vs1.jpeg /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/TempStuff/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.permAll.epacts.NullDistrQQPlotCheck.vs1.jpeg
ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.permAll.epacts.NullDistrQQPlotCheck.vs1.jpeg /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/TempStuff/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.permAll.epacts.NullDistrQQPlotCheck.vs1.jpeg
ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.permAll.epacts.NullDistrQQPlotCheck.vs1.jpeg /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/TempStuff/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.permAll.epacts.NullDistrQQPlotCheck.vs1.jpeg

scp -p /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/TempStuff/*Null*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/TempStuff/.

#20150925 Note -- things vary a little as the p-values become closer but the same groups of IDs exist in the same portions of the p-value distribution. Encouragingly, the top set of genes are essentially the same (like top 5-10 or so)
#paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput | vi -

#Retroactively including code for these 'gene/variant level result summary outputs', following the logic of excel sheet exploration, here

echo ".5" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".6" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".7" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".8" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".85" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".9" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .9) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .9) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`

~~~
[  michaelt@node1  ~]$echo ".5" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.5 5 44
[  michaelt@node1  ~]$echo ".6" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.6 43 340
[  michaelt@node1  ~]$echo ".7" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.7 95 703
[  michaelt@node1  ~]$echo ".8" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.8 145 1071
[  michaelt@node1  ~]$echo ".85" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.85 183 1353
[  michaelt@node1  ~]$echo ".9" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .9) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .9) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.9 357 2627
~~~

#GetperFastQC
#GetperRegionpostQC
#GetperFullBam
#GetVCF? for all indvs, just white indvs, then subgroups
#for full file, for per gene, for per indv (and then subgroups)

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroNeg 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroHighlyExposedNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroHighlyExposedNeg 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroPos.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroPos 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroNeg 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroHighlyExposedNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroHighlyExposedNeg 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroPos.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroPos 

~~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroNeg | wc
339   20340  147851
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroNeg | wc
338   20280  147416
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroHighlyExposedNeg | wc
157    9420   68503
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroHighlyExposedNeg | wc
156    9360   68068
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroPos | wc
408   24480  177987
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroPos | wc
409   24540  178422
~~~

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroHighlyExposedNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroHighlyExposedNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"

~~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
> print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 12.4615384615385 MeanDoubletons: 21.7366863905325 MeanCommon: 6828.18047337278 MeanRare: 596.733727810651 MeanHeterozy: 4648.60355029586 MeanCoverage: 12.284834838914 sdCoverage: 4.93002222552126 Exonic: 654.284023668639 Intronic: 5000.81656804734 Nonsynonymous: 257.313609467456 Synonymous: 390.958579881657 PP2_D: 28.0710059171598 PP2_P: 18.7011834319527 Sift_D: 41.7189349112426 LRT_D: 30.5562130177515"
>
>
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroHighlyExposedNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
> print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 12.7884615384615 MeanDoubletons: 21.0576923076923 MeanCommon: 6830.16025641026 MeanRare: 587.282051282051 MeanHeterozy: 4643.83974358974 MeanCoverage: 12.76753089599 sdCoverage: 4.94174312728489 Exonic: 654.192307692308 Intronic: 4996.51923076923 Nonsynonymous: 257.871794871795 Synonymous: 390.365384615385 PP2_D: 27.6474358974359 PP2_P: 18.7051282051282 Sift_D: 41.6153846153846 LRT_D: 30.4615384615385"
>
>
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
> print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 13.1833740831296 MeanDoubletons: 21.5892420537897 MeanCommon: 6854.49633251834 MeanRare: 590.647921760391 MeanHeterozy: 4672.89975550122 MeanCoverage: 12.7201016902561 sdCoverage: 4.8590326626708 Exonic: 653.058679706601 Intronic: 5014.54034229829 Nonsynonymous: 257.215158924205 Synonymous: 389.955990220049 PP2_D: 28.2982885085575 PP2_P: 18.7041564792176 Sift_D: 41.0220048899755 LRT_D: 30.7090464547677"
>
>

[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
> print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 1.8698224852071 MeanDoubletons: 3.2603550295858 MeanCommon: 591.153846153846 MeanRare: 63.1479289940828 MeanHeterozy: 404.491124260355 MeanCoverage: 14.6397166875744 sdCoverage: 5.9648377939256 Exonic: 654.284023668639 Intronic: 0 Nonsynonymous: 257.313609467456 Synonymous: 390.958579881657 PP2_D: 28.0710059171598 PP2_P: 18.7011834319527 Sift_D: 41.4763313609467 LRT_D: 30.5562130177515"
>
>
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroHighlyExposedNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
> print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 1.80769230769231 MeanDoubletons: 3.19230769230769 MeanCommon: 591.967948717949 MeanRare: 62.2435897435897 MeanHeterozy: 404.423076923077 MeanCoverage: 15.284547031862 sdCoverage: 6.03825440705908 Exonic: 654.192307692308 Intronic: 0 Nonsynonymous: 257.871794871795 Synonymous: 390.365384615385 PP2_D: 27.6474358974359 PP2_P: 18.7051282051282 Sift_D: 41.3846153846154 LRT_D: 30.4615384615385"
>
>
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
> print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.03178484107579 MeanDoubletons: 3.40342298288509 MeanCommon: 591.459657701712 MeanRare: 61.6161369193154 MeanHeterozy: 403.058679706601 MeanCoverage: 14.962233813138 sdCoverage: 5.84753483261101 Exonic: 653.058679706601 Intronic: 0 Nonsynonymous: 257.215158924205 Synonymous: 389.955990220049 PP2_D: 28.2982885085575 PP2_P: 18.7041564792176 Sift_D: 40.7677261613692 LRT_D: 30.7090464547677"
>
>
~~~





#20150925 Note -- I started using a new form of the grep statement associated with the "...wVarCounts" file here, e.g. grep ^"$gene"[^-0-9A-Za-z_] instead of grep -w. This is the correct way to go about things.

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz  
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz.tbi /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz.tbi  
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.vcfannotate.vs3.wACs.vcf.gz 

cat <(echo "#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        PVAL	QVAL	REF     ALT	AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW") \
<(join <( python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz | sort -k 1,1) <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^\# | perl -lane 'print join("_", @F[0..1]), "\t", $F[3], "\t", $F[4];' | sort -k 1,1) | \
perl -lane '$F[1] = join("|", split(/[\;,\,]/, $F[1])); $F[2] = join("|", split(/[\;,\,]/, $F[2])); print join("\t", @F);' | \
perl -lane 'my @geneEntries = split(/\|/, $F[2]); my @pVals; my @qVals; foreach my $gene (@geneEntries) { $gene = ((split(/\(/, $gene))[0]); my $geneEntry = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep ^"$gene"[^-0-9A-Za-z_]`; if ($geneEntry) { my @geneVals1 = split(/\s+/, $geneEntry); push(@pVals, $geneVals1[1]); push(@qVals, $geneVals1[3]); } else { push(@pVals, "NA"); push(@qVals, "NA"); } } splice(@F, 5, 0, (join("|", @pVals), join("|", @qVals))); print join("\t", @F);' | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroHighlyExposedNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroPos.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justRapidVRapid.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSlowVSlow.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
perl -lane 'splice(@F, 0, 1, (((split(/_/, $F[0]))[0]), ((split(/_/, $F[0]))[1]), ((split(/_/, $F[0]))[1]))); print join("\t", @F);' | \
perl -ane 'print join("\t", @F[0..10]); my $alt = $F[10]; my $totAC = join("", ((split(/:/, $F[11]))[1])) + join("", ((split(/:/, $F[12]))[1])); print "\t", $totAC; for (my $i=11; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", ((split(/\:/, $F[$i]))[1]); } } print "\n";' | sort -g -k 1,1 -k 2,2 ) | \
bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz

~~~
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | head -n 10
#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        PVAL    QVAL    REF     ALT     AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW
X       2821584 2821584 downstream      ARSD    .       .       0.637302600     0.90482 C       G       1466    2       2       0       0       0       0
X       2821782 2821782 downstream      ARSD    .       .       0.637302600     0.90482 G       A       1478    1       0       0       1       0       0
X       2821848 2821848 downstream      ARSD    .       .       0.637302600     0.90482 G       A       1484    3       3       3       0       0       0
X       2822103 2822103 UTR3    ARSD    .       .       0.637302600     0.90482 C       G       1486    2       0       0       2       0       2
X       2823133 2823133 UTR3    ARSD    .       .       0.637302600     0.90482 G       A       1486    2       1       1       1       0       0
X       2824218 2824218 UTR3    ARSD    .       .       0.637302600     0.90482 T       C       1486    15      7       3       8       2       2
X       2825026 2825026 UTR3    ARSD    .       .       0.637302600     0.90482 C       T       1474    5       2       0       3       0       0
X       2825083 2825083 UTR3    ARSD    .       .       0.637302600     0.90482 G       A       1476    12      6       4       6       2       2
X       2825149 2825149 UTR3    ARSD    .       .       0.637302600     0.90482 C       T       1474    4       2       0       2       1       0

[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | perl -lane 'print $#F;' | sort | uniq -c
  85201 17
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | wc
  85201 1533618 7403208
~~~

#Note -- Notice in the first version, produced by using 'grep -w', the p-/q-values for ANKHD1 and ANKHD1-EIF4EBP3 are the same, whereas in the second version ANKHD1 has a different (and the correct) p-/q-values. This is because if you just 'grep -w' with ANKHD1 you get ANKHD1-EIF4EBP3 first as well as ANKHD1, and so the perl code takes the ANKHD1-EIF4EBP3 line/values first
~~~
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ print $1 }' | grep \-
GIMAP1-GIMAP5
HLA-DRA
HLA-DRB1
INS-IGF2
TMED7-TICAM2
RPS10-NUDT3
TNFSF12-TNFSF13
PPAN-P2RY11
TMEM110-MUSTN1
C8orf44-SGK3
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep RPS10  RPS10-NUDT3     0.877824000 1.075540e-03 0.90730        3       747     0.0093708       3       3       2       0.87758 0      747      0.0093708       3       3      2
0.87848       0       747     0.0093708       3       3       2       0.87837 0
RPS10           0.217620000 0.000000e+00 0.85875        1       747     0.0013387       1       1       1       0.21762 NA     747      0.0013387       1       1      1
0.21762       NA      747     0.0013387       1       1       1       0.21762 NA
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | grep RPS10 | tail -n 10
6 34393866      34393866        UTR5|UTR5       RPS10|RPS10-NUDT3       .,.     .,.     0.8799025000|0.8799025000       0.90298|0.90298 G       A       1492    206    88 28    118     14      27
6 34393914      34393914        upstream|upstream       RPS10|RPS10-NUDT3       .,.     .,.     0.8799025000|0.8799025000      0.90298|0.90298  G       C       1494   1
1     0       0       0       0
6 34393964      34393964        upstream|upstream       RPS10|RPS10-NUDT3       .,.     .,.     0.8799025000|0.8799025000      0.90298|0.90298  C       G       1494   1
0     0       1       0       0
6 34393989      34393989        upstream|upstream       RPS10|RPS10-NUDT3       .,.     .,.     0.8799025000|0.8799025000      0.90298|0.90298  A       T       1492   1
0     0       1       0       0
6 34394080      34394080        upstream|upstream       RPS10|RPS10-NUDT3       .,.     .,.     0.8799025000|0.8799025000      0.90298|0.90298  C       A       1486   1
0     0       1       0       0
6 34394084      34394084        upstream|upstream       RPS10|RPS10-NUDT3       .,.     .,.     0.8799025000|0.8799025000      0.90298|0.90298  T       G       1486   3
0     0       3       1       0
6 34394099      34394099        upstream|upstream       RPS10|RPS10-NUDT3       .,.     .,.     0.8799025000|0.8799025000      0.90298|0.90298  G       A       1482   3
1     0       2       1       1
6 34394125      34394125        upstream|upstream       RPS10|RPS10-NUDT3       .,.     .,.     0.8799025000|0.8799025000      0.90298|0.90298  T       A       1480   1
1     1       0       0       0
6 34394191      34394191        upstream|upstream       RPS10|RPS10-NUDT3       .,.     .,.     0.8799025000|0.8799025000      0.90298|0.90298  A       T       1472   1
1     1       0       0       0
6 34394226      34394226        upstream|upstream       RPS10|RPS10-NUDT3       .,.     .,.     0.8799025000|0.8799025000      0.90298|0.90298  G       A       1468   2
2     0       0       0       0
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | grep RPS10 | tail -n 10
6 34393866      34393866        UTR5|UTR5       RPS10|RPS10-NUDT3       .,.     .,.     0.217620000|0.877824000 0.85875|0.90730 G       A       1492    206     88     28 118   14      27
6 34393914      34393914        upstream|upstream       RPS10|RPS10-NUDT3       .,.     .,.     0.217620000|0.877824000 0.85875|0.90730 G       C       1494    1      1
0     0       0       0
6 34393964      34393964        upstream|upstream       RPS10|RPS10-NUDT3       .,.     .,.     0.217620000|0.877824000 0.85875|0.90730 C       G       1494    1      0
0     1       0       0
6 34393989      34393989        upstream|upstream       RPS10|RPS10-NUDT3       .,.     .,.     0.217620000|0.877824000 0.85875|0.90730 A       T       1492    1      0
0     1       0       0
6 34394080      34394080        upstream|upstream       RPS10|RPS10-NUDT3       .,.     .,.     0.217620000|0.877824000 0.85875|0.90730 C       A       1486    1      0
0     1       0       0
6 34394084      34394084        upstream|upstream       RPS10|RPS10-NUDT3       .,.     .,.     0.217620000|0.877824000 0.85875|0.90730 T       G       1486    3      0
0     3       1       0
6 34394099      34394099        upstream|upstream       RPS10|RPS10-NUDT3       .,.     .,.     0.217620000|0.877824000 0.85875|0.90730 G       A       1482    3      1
0     2       1       1
6 34394125      34394125        upstream|upstream       RPS10|RPS10-NUDT3       .,.     .,.     0.217620000|0.877824000 0.85875|0.90730 T       A       1480    1      1
1     0       0       0
6 34394191      34394191        upstream|upstream       RPS10|RPS10-NUDT3       .,.     .,.     0.217620000|0.877824000 0.85875|0.90730 A       T       1472    1      1
1     0       0       0
6 34394226      34394226        upstream|upstream       RPS10|RPS10-NUDT3       .,.     .,.     0.217620000|0.877824000 0.85875|0.90730 G       A       1468    2      2
0     0       0       0
~~~

tabix -s 1 -b 2 -e 3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | vcf-annotate -a /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz -d /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.PrepFor_vcfannotate.vs2.desc.txt -c CHROM,FROM,TO,INFO/LOC,INFO/GENE,INFO/VARTYPE,INFO/AACHANGE,INFO/PVAL,INFO/QVAL,REF,ALT,INFO/AC_ALL,INFO/AC_TOTAL,INFO/AC_SERONEG,INFO/AC_HESERONEG,INFO/AC_SEROPOS,INFO/AC_RAPID,INFO/AC_SLOW | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz

~~~
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz | perl -lane 'print $#F;' | sort | uniq -c
87 0
3 1
3 10
1 109
7 11
5 12
1 13
4 14
1 16
2 2
1 20
2 24
2 3
1 38
3 4
1 41
4 5
3 6
1 69
85201 755
4 8
3 9
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | wc
85201 1533618 7403208
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz | wc
85340 64412794 1498035646
~~~

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz | grep -v ^# | perl -lane 'print $#F;' | sort | uniq -c 
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_TOTAL=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | head -n 10
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_TOTAL=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | perl -lane 'if ($F[1] ne $F[2]) { print join("\t", @F); }' | wc
/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroNeg.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'print $#F;' | sort | uniq -c 
/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroNeg.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_SERONEG=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | head -n 10
/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroNeg.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_SERONEG=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | perl -lane 'if ($F[1] ne $F[2]) { print join("\t", @F); }' | wc
/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroHighlyExposedNeg.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'print $#F;' | sort | uniq -c 
/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroHighlyExposedNeg.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_HESERONEG=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | head -n 10
/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroHighlyExposedNeg.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_HESERONEG=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | perl -lane 'if ($F[1] ne $F[2]) { print join("\t", @F); }' | wc
/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroPos.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'print $#F;' | sort | uniq -c 
/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroPos.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_SEROPOS=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | head -n 10
/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroPos.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_SEROPOS=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | perl -lane 'if ($F[1] ne $F[2]) { print join("\t", @F); }' | wc
/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justRapidVRapid.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'print $#F;' | sort | uniq -c 
/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justRapidVRapid.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_RAPID=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | head -n 10
/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justRapidVRapid.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_RAPID=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | perl -lane 'if ($F[1] ne $F[2]) { print join("\t", @F); }' | wc
/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSlowVSlow.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'print $#F;' | sort | uniq -c 
/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSlowVSlow.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_SLOW=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | head -n 10
/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSlowVSlow.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_SLOW=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | perl -lane 'if ($F[1] ne $F[2]) { print join("\t", @F); }' | wc


zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz | grep -v ^# | perl -ane 'my $GENE1 = "NA"; $PVAL1 = "NA"; $QVAL1 = "NA"; $GENE2 = "NA"; if ($F[7] =~ m/.*GENE=([0-9a-zA-Z]+)(;|\|).*/) { $GENE1 = $1; } if ($F[7] =~ m/.*PVAL=(\d\.\d+)(;|\|).*/) { $PVAL1 = $1; } if ($F[7] =~ m/.*QVAL=(\d\.\d+)(;|\|).*/) { $QVAL1 = $1; } if ($PVAL1 ne "NA") { $GENE2=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep ^'"'"'$GENE1'"'"'[^-0-9a-zA-Z_] | awk '"'"'{ print \$1, "\t", \$2, "\t", \$4 } '"'"'`; print $GENE1, "\t", $PVAL1, "\t", $QVAL1, "\t", $GENE2; } else { print $., "\t", $GENE1, "\t", $PVAL1, "\t", $QVAL1, "\t", $GENE2, "\n";} ' | grep -v NA | head -n 10
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz | grep -v ^# | perl -ane 'my $GENE1 = "NA"; $PVAL1 = "NA"; $QVAL1 = "NA"; $GENE2 = "NA"; if ($F[7] =~ m/.*GENE=([0-9a-zA-Z]+)(;|\|).*/) { $GENE1 = $1; } if ($F[7] =~ m/.*PVAL=(\d\.\d+)(;|\|).*/) { $PVAL1 = $1; } if ($F[7] =~ m/.*QVAL=(\d\.\d+)(;|\|).*/) { $QVAL1 = $1; } if ($PVAL1 ne "NA") { $GENE2=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep ^'"'"'$GENE1'"'"'[^-0-9a-zA-Z_] | awk '"'"'{ print \$1, "\t", \$2, "\t", \$4 } '"'"'`; print $GENE1, "\t", $PVAL1, "\t", $QVAL1, "\t", $GENE2; } else { print $., "\t", $GENE1, "\t", $PVAL1, "\t", $QVAL1, "\t", $GENE2, "\n";} ' | grep -v NA | awk '{ if ($1 != $4) { print $0 } } ' | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz | grep -v ^# | perl -ane 'my $GENE1 = "NA"; $PVAL1 = "NA"; $QVAL1 = "NA"; $GENE2 = "NA"; if ($F[7] =~ m/.*GENE=([0-9a-zA-Z]+)(;|\|).*/) { $GENE1 = $1; } if ($F[7] =~ m/.*PVAL=(\d\.\d+)(;|\|).*/) { $PVAL1 = $1; } if ($F[7] =~ m/.*QVAL=(\d\.\d+)(;|\|).*/) { $QVAL1 = $1; } if ($PVAL1 ne "NA") { $GENE2=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep ^'"'"'$GENE1'"'"'[^-0-9a-zA-Z_] | awk '"'"'{ print \$1, "\t", \$2, "\t", \$4 } '"'"'`; print $GENE1, "\t", $PVAL1, "\t", $QVAL1, "\t", $GENE2; } else { print $., "\t", $GENE1, "\t", $PVAL1, "\t", $QVAL1, "\t", $GENE2, "\n";} ' | grep -v NA | awk '{ if ($2 != $5) { print $0 } } ' | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz | grep -v ^# | perl -ane 'my $GENE1 = "NA"; $PVAL1 = "NA"; $QVAL1 = "NA"; $GENE2 = "NA"; if ($F[7] =~ m/.*GENE=([0-9a-zA-Z]+)(;|\|).*/) { $GENE1 = $1; } if ($F[7] =~ m/.*PVAL=(\d\.\d+)(;|\|).*/) { $PVAL1 = $1; } if ($F[7] =~ m/.*QVAL=(\d\.\d+)(;|\|).*/) { $QVAL1 = $1; } if ($PVAL1 ne "NA") { $GENE2=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep ^'"'"'$GENE1'"'"'[^-0-9a-zA-Z_] | awk '"'"'{ print \$1, "\t", \$2, "\t", \$4 } '"'"'`; print $GENE1, "\t", $PVAL1, "\t", $QVAL1, "\t", $GENE2; } else { print $., "\t", $GENE1, "\t", $PVAL1, "\t", $QVAL1, "\t", $GENE2, "\n";} ' | grep -v NA | awk '{ if ($3 != $6) { print $0 } } ' | wc

~~~
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz | grep -v ^# | perl -lane 'print $#F;' | sort | uniq -c
  85200 755
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_TOTAL=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | head -n 10
1       16      16
2       1       1
3       5       5
4       2       2
5       4       4
6       1       1
7       1400    1400
8       1412    1412
9       17      17
10      1       1
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_TOTAL=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | perl -lane 'if ($F[1] ne $F[2]) { print join("\t", @F); }' | wc
  0       0       0
[  michaelt@node1  ~]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroNeg.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'print $#F;' | sort | uniq -c
  85200 346
[  michaelt@node1  ~]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroNeg.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_SERONEG=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | head -n 10
1       10      10
2       1       1
3       2       2
4       1       1
5       1       1
6       0       0
7       633     633
8       634     634
9       7       7
10      1       1
[  michaelt@node1  ~]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroNeg.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_SERONEG=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | perl -lane 'if ($F[1] ne $F[2]) { print join("\t", @F); }' | wc
   0       0       0
[  michaelt@node1  ~]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs
1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.
wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.
UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroHighlyExposedNeg.FIDs --recode --recode-INFO-all --stdo
ut | grep -v ^# | perl -lane 'print $#F;' | sort | uniq -c
  85200 164
[  michaelt@node1  ~]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroHighlyExposedNeg.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_HESERONEG=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | head -n 10
1       4       4
2       0       0
3       2       2
4       0       0
5       1       1
6       0       0
7       288     288
8       286     286
9       5       5
10      1       1
[  michaelt@node1  ~]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroHighlyExposedNeg.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_HESERONEG=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | perl -lane 'if ($F[1] ne $F[2]) { print join("\t", @F); }' | wc
   0       0       0
[  michaelt@login2  ~/Data]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroPos.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'print $#F;' | sort | uniq -c
/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroPos.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_SEROPOS=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | perl -lane 'if ($F[1] ne $F[2]) { print join("\t", @F); }' | wc
  85200 417
[  michaelt@login2  ~/Data]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroPos.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_SEROPOS=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | head -n 10
1       6       6
2       0       0
3       3       3
4       1       1
5       3       3
6       1       1
7       767     767
8       778     778
9       10      10
10      0       0
[  michaelt@login2  ~/Data]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroPos.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_SEROPOS=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | perl -lane 'if ($F[1] ne $F[2]) { print join("\t", @F); }' | wc
   0       0       0
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justRapidVRapid.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'print $#F;' | sort | uniq -c
/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justRapidVRapid.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_RAPID=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | perl -lane 'if ($F[1] ne $F[2]) { print join("\t", @F); }' | wc
  85200 62
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justRapidVRapid.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_RAPID=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | head -n 10
1       0       0
2       0       0
3       1       1
4       0       0
5       2       2
6       1       1
7       100     100
8       102     102
9       2       2
10      0       0
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justRapidVRapid.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_RAPID=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | perl -lane 'if ($F[1] ne $F[2]) { print join("\t", @F); }' | wc
   0       0       0
[  michaelt@login2  ~/Data/UCSC/Cow/bosTau7]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSlowVSlow.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'print $#F;' | sort | uniq -c
  85200 89
[  michaelt@login2  ~/Data/UCSC/Cow/bosTau7]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSlowVSlow.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_SLOW=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | head -n 10
1       2       2
2       0       0
3       0       0
4       0       0
5       0       0
6       0       0
7       153     153
8       156     156
9       2       2
10      0       0
[  michaelt@login2  ~/Data/UCSC/Cow/bosTau7]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSlowVSlow.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_SLOW=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | perl -lane 'if ($F[1] ne $F[2]) { print join("\t", @F); }' | wc
   0       0       0
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz | grep -v ^# | perl -ane 'my $GENE1 = "NA"; $PVAL1 = "NA"; $QVAL1 = "NA"; $GENE2 = "NA"; if ($F[7] =~ m/.*GENE=([0-9a-zA-Z]+)(;|\|).*/) { $GENE1 = $1; } if ($F[7] =~ m/.*PVAL=(\d\.\d+)(;|\|).*/) { $PVAL1 = $1; } if ($F[7] =~ m/.*QVAL=(\d\.\d+)(;|\|).*/) { $QVAL1 = $1; } if ($PVAL1 ne "NA") { $GENE2=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep ^'"'"'$GENE1'"'"'[^-0-9a-zA-Z_] | awk '"'"'{ print \$1, "\t", \$2, "\t", \$4 } '"'"'`; print $GENE1, "\t", $PVAL1, "\t", $QVAL1, "\t", $GENE2; } else { print $., "\t", $GENE1, "\t", $PVAL1, "\t", $QVAL1, "\t", $GENE2, "\n";} ' | grep -v NA | head -n 10                         ISG15   0.801272300     0.90482 ISG15    0.801272300     0.90482
ISG15   0.801272300     0.90482 ISG15    0.801272300     0.90482
ISG15   0.801272300     0.90482 ISG15    0.801272300     0.90482
ISG15   0.801272300     0.90482 ISG15    0.801272300     0.90482
ISG15   0.801272300     0.90482 ISG15    0.801272300     0.90482
ISG15   0.801272300     0.90482 ISG15    0.801272300     0.90482
ISG15   0.801272300     0.90482 ISG15    0.801272300     0.90482
ISG15   0.801272300     0.90482 ISG15    0.801272300     0.90482
ISG15   0.801272300     0.90482 ISG15    0.801272300     0.90482
ISG15   0.801272300     0.90482 ISG15    0.801272300     0.90482
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz | grep -v ^# | perl -ane 'my $GENE1 = "NA"; $PVAL1 = "NA"; $QVAL1 = "NA"; $GENE2 = "NA"; if ($F[7] =~ m/.*GENE=([0-9a-zA-Z]+)(;|\|).*/) { $GENE1 = $1; } if ($F[7] =~ m/.*PVAL=(\d\.\d+)(;|\|).*/) { $PVAL1 = $1; } if ($F[7] =~ m/.*QVAL=(\d\.\d+)(;|\|).*/) { $QVAL1 = $1; } if ($PVAL1 ne "NA") { $GENE2=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep ^'"'"'$GENE1'"'"'[^-0-9a-zA-Z_] | awk '"'"'{ print \$1, "\t", \$2, "\t", \$4 } '"'"'`; print $GENE1, "\t", $PVAL1, "\t", $QVAL1, "\t", $GENE2; } else { print $., "\t", $GENE1, "\t", $PVAL1, "\t", $QVAL1, "\t", $GENE2, "\n";} ' | grep -v NA | awk '{ if ($1 != $4) { print $0 } } ' | wc
   0       0       0
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz | grep -v ^# | perl -ane 'my $GENE1 = "NA"; $PVAL1 = "NA"; $QVAL1 = "NA"; $GENE2 = "NA"; if ($F[7] =~ m/.*GENE=([0-9a-zA-Z]+)(;|\|).*/) { $GENE1 = $1; } if ($F[7] =~ m/.*PVAL=(\d\.\d+)(;|\|).*/) { $PVAL1 = $1; } if ($F[7] =~ m/.*QVAL=(\d\.\d+)(;|\|).*/) { $QVAL1 = $1; } if ($PVAL1 ne "NA") { $GENE2=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep ^'"'"'$GENE1'"'"'[^-0-9a-zA-Z_] | awk '"'"'{ print \$1, "\t", \$2, "\t", \$4 } '"'"'`; print $GENE1, "\t", $PVAL1, "\t", $QVAL1, "\t", $GENE2; } else { print $., "\t", $GENE1, "\t", $PVAL1, "\t", $QVAL1, "\t", $GENE2, "\n";} ' | grep -v NA | awk '{ if ($2 != $5) { print $0 } } ' | wc
   0       0       0
[  michaelt@login2  ~/Data/UCSC/Cow/bosTau7]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz | grep -v ^# | perl -ane 'my $GENE1 = "NA"; $PVAL1 = "NA"; $QVAL1 = "NA"; $GENE2 = "NA"; if ($F[7] =~ m/.*GENE=([0-9a-zA-Z]+)(;|\|).*/) { $GENE1 = $1; } if ($F[7] =~ m/.*PVAL=(\d\.\d+)(;|\|).*/) { $PVAL1 = $1; } if ($F[7] =~ m/.*QVAL=(\d\.\d+)(;|\|).*/) { $QVAL1 = $1; } if ($PVAL1 ne "NA") { $GENE2=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep ^'"'"'$GENE1'"'"'[^-0-9a-zA-Z_] | awk '"'"'{ print \$1, "\t", \$2, "\t", \$4 } '"'"'`; print $GENE1, "\t", $PVAL1, "\t", $QVAL1, "\t", $GENE2; } else { print $., "\t", $GENE1, "\t", $PVAL1, "\t", $QVAL1, "\t", $GENE2, "\n";} ' | grep -v NA | awk '{ if ($3 != $6) { print $0 } } ' | wc
   0       0       0
~~~

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.P2.Vs2.annotated.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/OLD2.P2.AllPools.P2.Vs2.annotated.vcf.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/OLD2.P2.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz 
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.ForSudhir.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/OLD2.P2.skatO.nonsyn.qVals.ACs.GeneLevel.ForSudhir.txt

rm -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt

for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs`
do

	paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane 'if ($F[0] =~ m/^'$gene1'$/) { print join("\t", @F); }' | awk '{ print $1, "\t", $2, "\t", $4, "\t", $5, "\t", $8, "\t", $9, "\t", $11 }' ) \
	<(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | \
	grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'if ($F[0] =~ m/^'$gene1'$/) { print join("\t", @F); }' | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }') | \
	awk '{ print $12, "\t", $13, "\t", $14, "\t", $15, "\t", $16, "\t", $17, "\t", $18 }' | \
	R -q -e "Data1 <- read.table(file('stdin'), header=F); print(paste(apply(Data1, 2, sum)), quote=FALSE);" | grep -v ^\> | perl -lane 'splice(@F, 0, 1); print join("\t", @F);') \
	>> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt

done

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt | perl -lane 'splice(@F, 4, 3); print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt

#Included the below header in file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt 
#GENE   PVAL    QVAL    NUMVAR    AC_ALL        AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW

#QC Checks on file

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt | wc
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt | perl -lane 'if (($F[3] == $F[4]) && ($F[3] == $F[5]) && ($F[4] == $F[5])) { print join("\t", @F); } '  | wc
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt | perl -lane 'if (($F[3] != $F[4]) || ($F[3] != $F[5]) || ($F[4] != $F[5])) { print join("\t", @F); } '
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[5] != $val2) { print join("\t", @F); }'
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[5] == $val2) { print join("\t", @F); }' | wc
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[7] > $F[6]) { print join("\t", @F); }'
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[7] <= $F[6]) { print join("\t", @F); }' | wc
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[9] + $F[10]; if ($F[8] < $val2) { print join("\t", @F); }'
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[9] + $F[10]; if ($F[8] >= $val2) { print join("\t", @F); }' | wc

#20151003 CHECK_0: Once have annotation file working properly (e.g. Total_AC working properly), go back and check why one variant is dropping out of the analysis for this one gene here?
#Original date was sometime back in April/May 2015
~~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt | wc
786    8646   39442
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt | perl -lane 'if (($F[3] == $F[4]) && ($F[3] == $F[5]) && ($F[4] == $F[5])) { print join("\t", @F); } '  | wc
785   10983   48835
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt | perl -lane 'if (($F[3] != $F[4]) || ($F[3] != $F[5]) || ($F[4] != $F[5])) { print join("\t", @F); } '
WNK1    0.418770600     0.88734 42      42      41      0.41574 62674   3926    1794    820     2132    290     422
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GenLevel.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[5] != $val2) { print join("\t", @F); }'
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[5] == $val2) { print join("\t", @F); }' | wc
786    8646   39402
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[7] > $F[6]) { print join("\t", @F); }'
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[7] <= $F[6]) { print join("\t", @F); }' | wc
786    8646   39402
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[9] + $F[10]; if ($F[8] < $val2) { print join("\t", @F); }'
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[9] + $F[10]; if ($F[8] >= $val2) { print join("\t", @F); }' | wc
786    8646   39402
[  michaelt@node1  ~]$
~~~

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.P2.Vs2.annotated.vcf.gz

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz

#Difference between 141 and 142 due to '#CHROM...' header in /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz
~~~
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz | wc
85201 1533618 7403208
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz | wc
85340 64412794 1498035646
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.P2.Vs2.annotated.vcf.gz | wc
85340 64412794 1498035646
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz | wc
85201 1533618 7403208
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz | wc
85340 64412794 1498035646
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.P2.Vs2.annotated.vcf.gz | wc
85340 64412794 1498035646
[  michaelt@node1  ~]$R -q -e "85340 - 85201"
> 85340 - 85201
[1] 139
>
>
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.P2.Vs2.annotated.vcf.gz | grep ^# | wc
140    1594   29457
~~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/OLD2.P2.AllPools.P2.Vs2.annotated.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/OLD2.P2.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/OLD2.P2.skatO.nonsyn.qVals.ACs.GeneLevel.ForSudhir.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/.

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.P2.Vs2.annotated.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/.

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt.P2.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.msigdb_c2

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt.P2.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.msigdb_c2



##20151005 Note -- Comparing old results to new results some more                                                  
##Specifically looking at what genes are being included/not included given comparable q-value cutoffs between current results and past results                                                                                                                                                                               

paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk 'BEGIN{ print "nana"} { print $4 }' | sort -g ) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ print $4 }' | sort -g) | vi -
comm <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ print $1 }' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ print $1 }' | sort -g) | vi -

for i in {1..800..100}; do
	BEGIN=$i
	END=$(($i + 99))

	echo $i

	comm -23 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | sort -g -k 4,4 | awk '{ if ((NR>='$BEGIN') && (NR<='$END')) { print $1 } } ' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | sort -g -k 4,4 | awk '{ if ((NR>='$BEGIN') && (NR<='$END')) { print $1 } } ' | sort -g) | wc	
	comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | sort -g -k 4,4 | awk '{ if ((NR>='$BEGIN') && (NR<='$END')) { print $1 } } ' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | sort -g -k 4,4 | awk '{ if ((NR>='$BEGIN') && (NR<='$END')) { print $1 } } ' | sort -g) | wc	

done
	

comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .4) { print $1 } }' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .4) { print $1  } }' | sort -g) | wc
paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .4) { print $1 } }' | sort -g | wc) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .4) { print $1  } }' | wc )
paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .4) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .4) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10)

comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .5) { print $1 } }' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1  } }' | sort -g) | wc
paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .5) { print $1 } }' | sort -g | wc) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1  } }' | wc )
paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .5) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10)

comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .6) { print $1 } }' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1  } }' | sort -g) | wc
paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .6) { print $1 } }' | sort -g | wc) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1  } }' | wc )
paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .6) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10)

comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .7) { print $1 } }' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1  } }' | sort -g) | wc
paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .7) { print $1 } }' | sort -g | wc) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1  } }' | wc )
paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .7) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10)

comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .75) { print $1 } }' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .75) { print $1  } }' | sort -g) | wc
paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .75) { print $1 } }' | sort -g | wc) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .75) { print $1  } }' | wc )
paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .75) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .75) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10)

comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .8) { print $1 } }' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1  } }' | sort -g) | wc
paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .8) { print $1 } }' | sort -g | wc) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1  } }' | wc )
paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .8) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10)

comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .85) { print $1 } }' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1  } }' | sort -g) | wc
paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .85) { print $1 } }' | sort -g | wc) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1  } }' | wc )
paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .85) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10)



~~~
[  michaelt@login2  ~]$comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .8) { print $1 } }' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1  } }' | sort -g) | wc^C
[  michaelt@login2  ~]$
[  michaelt@login2  ~]$for i in {1..800..25}; do
>         BEGIN=$i
>         END=$(($i + 24))
>
>         echo $i
>
>         comm -23 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | sort -g -k 4,4 | awk '{ if ((NR>='$BEGIN') && (NR<='$END')) { print $1 } } ' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | sort -g -k 4,4 | awk '{ if ((NR>='$BEGIN') && (NR<='$END')) { print $1 } } ' | sort -g) | wc
>         comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | sort -g -k 4,4 | awk '{ if ((NR>='$BEGIN') && (NR<='$END')) { print $1 } } ' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | sort -g -k 4,4 | awk '{ if ((NR>='$BEGIN') && (NR<='$END')) { print $1 } } ' | sort -g) | wc
>
> done
1
1       1       6
1       1       3
26
3       3      18
3       3      18
51
5       5      28
5       5      31
76
4       4      29
4       4      22
101
3       3      17
3       3      23
126
2       2      13
2       2      13
151
6       6      40
6       6      36
176
7       7      38
7       7      45
201
12      12      77
12      12      66
226
13      13      76
13      13      79
251
5       5      31
5       5      26
276
8       8      50
8       8      49
301
6       6      35
6       6      41
326
12      12      68
12      12      73
351
17      17     111
17      17      97
376
15      15      82
15      15      98
401
16      16     104
16      16      97
426
17      17     101
17      17      97
451
16      16      91
16      16      87
476
19      19     102
19      19     129
501
22      22     148
22      22     127
526
25      25     148
25      25     143
551
25      25     143
25      25     142
576
25      25     137
25      25     150
601
25      25     151
25      25     163
626
25      25     162
25      25     171
651
25      25     152
25      25     171
676
25      25     193
25      25     152
701
17      17     103
17      17     104
726
3       3      17
3       3      18
751
1       1       6
1       1       3
776
0       0       0
1       1       6
[  michaelt@login2  ~]$for i in {1..800..50}; do
>         BEGIN=$i
>         END=$(($i + 49))
>
>         echo $i
>
>         comm -23 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | sort -g -k 4,4 | awk '{ if ((NR>='$BEGIN') && (NR<='$END')) { print $1 } } ' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | sort -g -k 4,4 | awk '{ if ((NR>='$BEGIN') && (NR<='$END')) { print $1 } } ' | sort -g) | wc
>         comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | sort -g -k 4,4 | awk '{ if ((NR>='$BEGIN') && (NR<='$END')) { print $1 } } ' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | sort -g -k 4,4 | awk '{ if ((NR>='$BEGIN') && (NR<='$END')) { print $1 } } ' | sort -g) | wc
>
> done
1
4       4      24
4       4      21
51
6       6      41
6       6      37
101
4       4      23
4       4      29
151
5       5      29
5       5      32
201
7       7      46
7       7      38
251
10      10      62
10      10      56
301
11      11      60
11      11      71
351
17      17     104
17      17     106
401
18      18     115
18      18     104
451
21      21     116
21      21     139
501
26      26     175
26      26     149
551
31      31     171
31      31     183
601
31      31     189
31      31     210
651
32      32     217
32      32     195
701
18      18     107
18      18     109
751
1       1       6
2       2       9
[  michaelt@login2  ~]$for i in {1..800..100}; do
>         BEGIN=$i
>         END=$(($i + 99))
>
>         echo $i
>
>         comm -23 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | sort -g -k 4,4 | awk '{ if ((NR>='$BEGIN') && (NR<='$END')) { print $1 } } ' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | sort -g -k 4,4 | awk '{ if ((NR>='$BEGIN') && (NR<='$END')) { print $1 } } ' | sort -g) | wc
>         comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | sort -g -k 4,4 | awk '{ if ((NR>='$BEGIN') && (NR<='$END')) { print $1 } } ' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | sort -g -k 4,4 | awk '{ if ((NR>='$BEGIN') && (NR<='$END')) { print $1 } } ' | sort -g) | wc
>
> done
1
4       4      29
4       4      22
101
7       7      39
7       7      48
201
10      10      64
10      10      50
301
21      21     117
21      21     130
401
23      23     140
23      23     152
501
32      32     203
32      32     189
601
33      33     201
33      33     200
701
17      17     101
18      18     106
[  michaelt@login2  ~]$comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .4) { print $1 } }' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .4) { print $1  } }' | sort -g) | wc
0       0       0
[  michaelt@login2  ~]$paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .4) { print $1 } }' | sort -g | wc) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .4) { print $1  } }' | wc )
6       6      35       4       4      23
[  michaelt@login2  ~]$paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .4) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .4) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10)
NLRX1    0.0030547940    4.848680e-04    0.36952        XAF1     0.001356961     2.615152e-04    0.39392
XAF1     0.0013443327    2.553850e-04    0.33061        PRDM1    0.001441239     3.511077e-04    0.39392
PSME4    0.0008706566    1.492251e-04    0.33061        PDXK     0.001957196     3.937028e-06    0.39392
PRDM1    0.0015569464    2.761238e-04    0.33061        MAPK14   0.002168403     1.344193e-04    0.39392
PDXK     0.0020633760    7.761132e-06    0.33061
MAPK14   0.0022776170    1.526260e-04    0.33061
[  michaelt@login2  ~]$comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .5) { print $1 } }' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1  } }' | sort -g) | wc
0       0       0
[  michaelt@login2  ~]$paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .5) { print $1 } }' | sort -g | wc) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1  } }' | wc )
24      24     150       5       5      29
[  michaelt@login2  ~]$paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .5) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10)
VSNL1    0.0148400000    0.000000e+00    0.49252        NLRX1    0.003075420     4.997193e-04    0.44695
UBE2J1   0.0107958100    2.970450e-05    0.49252        XAF1     0.001356961     2.615152e-04    0.39392
THRB     0.0143895200    5.881470e-04    0.49252        PRDM1    0.001441239     3.511077e-04    0.39392
SULT1C2          0.0151408400    1.130803e-03    0.49252        PDXK     0.001957196     3.937028e-06    0.39392
SPP1     0.0104157690    7.426805e-04    0.49252        MAPK14   0.002168403     1.344193e-04    0.39392
S1PR5    0.0135579900    1.200745e-03    0.49252
PRRC2A   0.0056639110    8.175604e-04    0.49252
PPP3CB   0.0153910000    4.599971e-04    0.49252
PPAN-P2RY11      0.0162864900    1.446203e-03    0.49252
PPAN     0.0100173360    1.028251e-03    0.49252
[  michaelt@login2  ~]$comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .6) { print $1 } }' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1  } }' | sort -g) | wc
0       0       0
[  michaelt@login2  ~]$paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .6) { print $1 } }' | sort -g | wc) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1  } }' | wc )
51      51     319      43      43     272
[  michaelt@login2  ~]$paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .6) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10)
NTRK3    0.0403896200    7.202468e-04    0.57479        RNASEH2A         0.034614480     6.398863e-04    0.58495
NR4A3    0.0401172300    1.195204e-03    0.57479        SPIN1    0.032521000     0.000000e+00    0.57607
FOSL1    0.0402649300    1.568881e-04    0.57479        PI4KA    0.030313740     1.307542e-03    0.57607
DBH      0.0368510000    1.849710e-03    0.55721        PENK     0.030582150     3.621237e-03    0.57607
RNASEH2A         0.0354072400    6.052573e-04    0.55229        DDR1     0.028966630     1.703077e-03    0.57607
NKG7     0.0357645500    7.763389e-04    0.55229        CARD17   0.033296110     9.385207e-04    0.57607
SPIN1    0.0319030000    0.000000e+00    0.54742        CAPRIN1          0.033110380     3.048877e-04    0.57607
RGS2     0.0291143500    3.549713e-03    0.54742        ATP1B3   0.031899620     4.807614e-04    0.57607
PLA2G1B          0.0339408300    2.369117e-03    0.54742        RGS2     0.026626800     3.129408e-03    0.55281
PI4KA    0.0312807300    1.258456e-03    0.54742        YWHAB    0.025820600     6.430483e-05    0.55184
[  michaelt@login2  ~]$comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .7) { print $1 } }' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1  } }' | sort -g) | wc
5       5      35
[  michaelt@login2  ~]$paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .7) { print $1 } }' | sort -g | wc) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1  } }' | wc )
91      91     551      95      95     580
[  michaelt@login2  ~]$paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .7) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10)
UGT8     0.0859641800    8.365431e-04    0.68562        AGT      0.090813480     2.839891e-04    0.69463
SOCS4    0.0854904800    1.073860e-03    0.68562        CDKN1A   0.087939860     2.452654e-03    0.67981
PAPSS2   0.0840494600    8.498805e-04    0.68562        DNASE1L3         0.085923820     3.364053e-04    0.67136
OMG      0.0855150700    2.917224e-04    0.68562        UGT8     0.079861020     1.136743e-03    0.67037
IFI44L   0.0821624900    3.149448e-04    0.68543        TFAP2B   0.072787000     0.000000e+00    0.67037
TFAP2B   0.0789600000    0.000000e+00    0.67624        SULT1E1          0.082505510     6.053445e-04    0.67037
SULT1E1          0.0800569800    5.575090e-04    0.67624        SULT1C3          0.084873900     1.014261e-03    0.67037
HINT1    0.0796820000    0.000000e+00    0.67624        SOCS4    0.083278560     9.278423e-04    0.67037
GBP5     0.0796527200    2.437168e-03    0.67624        PAPSS2   0.083186470     8.041503e-04    0.67037
EGR2     0.0746000000    0.000000e+00    0.67624        OMG      0.084424770     6.519716e-04    0.67037
[  michaelt@login2  ~]$comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .75) { print $1 } }' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .75) { print $1  } }' | sort -g) | wc
1       1       9
[  michaelt@login2  ~]$paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .75) { print $1 } }' | sort -g | wc) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .75) { print $1  } }' | wc )
119     119     725     111     111     677
[  michaelt@login2  ~]$paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .75) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .75) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10)
XCL1     0.1113600000    0.000000e+00    0.73945        TIRAP    0.114341300     9.991458e-04    0.74853
TRAF3IP3         0.1173024000    1.255849e-03    0.73945        NFX1     0.108849000     6.042116e-04    0.74499
TIRAP    0.1201393000    1.216947e-03    0.73945        IFIH1    0.107676700     9.368894e-04    0.74499
TCEB2    0.0993303400    2.810022e-03    0.73945        GEM      0.112741900     3.374195e-04    0.74499
TALDO1   0.1046679000    6.478002e-04    0.73945        GAL3ST4          0.109840400     8.640215e-04    0.74499
SULF1    0.1204102000    1.177940e-03    0.73945        ARHGAP21         0.112775100     1.008642e-03    0.74499
SLC6A2   0.1160057000    1.091932e-03    0.73945        ADRB2    0.111227800     1.409007e-03    0.74499
SLC1A3   0.1081398000    9.180029e-04    0.73945        XCL1     0.104600000     0.000000e+00    0.74457
SESN1    0.1202596000    1.168246e-03    0.73945        TCEB2    0.105940500     2.768146e-03    0.74457
RBM34    0.1185119000    9.290548e-04    0.73945        TALDO1   0.100848690     5.904511e-04    0.74457
[  michaelt@login2  ~]$comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .8) { print $1 } }' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1  } }' | sort -g) | wc
11      11      68
[  michaelt@login2  ~]$paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .8) { print $1 } }' | sort -g | wc) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1  } }' | wc )
135     135     822     145     145     884
[  michaelt@login2  ~]$paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .8) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10)
ADARB1   0.1486090000    1.581209e-03    0.79895        VPRBP    0.151679000     1.668257e-03    0.79772
FCHO1    0.1464108000    1.822636e-03    0.79556        SULF2    0.155393600     3.684350e-03    0.79772
DBP      0.1468815000    3.438189e-03    0.79556        RPS29    0.156480000     0.000000e+00    0.79772
TNF      0.1415698000    9.369615e-04    0.78691        MAST2    0.146653500     2.378122e-03    0.79772
SLC3A2   0.1431163000    1.236016e-03    0.78691        GABRB1   0.156890800     2.250381e-03    0.79772
PABPC1   0.1391848000    5.153668e-04    0.78691        FCHO1    0.148466100     3.343051e-03    0.79772
GNS      0.1420601000    2.368130e-03    0.78691        DBP      0.149288700     3.667894e-03    0.79772
EDIL3    0.1416100000    0.000000e+00    0.78691        DAK      0.158130000     0.000000e+00    0.79772
MAST2    0.1367640000    2.332162e-03    0.78583        COL1A2   0.151372600     2.929550e-03    0.79772
CHST8    0.1375061000    4.434778e-04    0.78583        CISH     0.157456800     4.580920e-04    0.79772
[  michaelt@login2  ~]$comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .85) { print $1 } }' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1  } }' | sort -g) | wc
1       1       6
[  michaelt@login2  ~]$paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .85) { print $1 } }' | sort -g | wc) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1  } }' | wc )
188     188    1154     183     183    1127
[  michaelt@login2  ~]$paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^\\s | awk '{ if ($4 <= .85) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10)
RPS10    0.2188900000    0.000000e+00    0.84504        TUBA3D   0.204950000     0.000000e+00    0.84673
PCED1B   0.2184900000    0.000000e+00    0.84504        TMEM173          0.206374500     6.187334e-04    0.84673
OASL     0.2177440000    1.124608e-03    0.84504        SULT4A1          0.213060000     0.000000e+00    0.84673
NUAK2    0.2135684000    2.480456e-03    0.84501        SULT1C4          0.199718600     4.935303e-03    0.84673
LRCH4    0.2135596000    2.435095e-03    0.84501        PTK2     0.207835200     5.439669e-04    0.84673
IFI44    0.2133526000    1.161741e-03    0.84501        NUAK2    0.212316900     2.424809e-03    0.84673
GKN1     0.2144500000    0.000000e+00    0.84501        MUS81    0.202700800     5.842271e-03    0.84673
CCR6     0.2153900000    0.000000e+00    0.84501        IFI44    0.207658500     1.029257e-03    0.84673
TMEM173          0.2091638000    7.549524e-04    0.84338        GKN1     0.202720000     0.000000e+00    0.84673
SULT4A1          0.2079100000    0.000000e+00    0.84338        ETV5     0.213238800     3.231413e-03    0.84673
~~~




##20151231
##Including analysis run for intronic regions/variants

~~~
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs | perl -F, -lane 'print $F[6];' | sort | uniq -c
  11445 UTR3
   1969 UTR5
     18 UTR5;UTR3
   3698 downstream
   9939 exonic
      4 exonic;splicing
   1978 intergenic
  54958 intronic
     80 ncRNA_UTR3
     31 ncRNA_UTR5
      1 ncRNA_UTR5;ncRNA_UTR3
    468 ncRNA_exonic
    910 ncRNA_intronic
     67 splicing
   1820 upstream
    124 upstream;downstream
~~~

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs | awk -F, '{ if ($7 == "intronic") { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Intronic.wSplice

#Including '.wSplice' at end due to /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh//data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs3.pt2.sh naming convention, ie code inserts '.wSplice' at end of a variable expecting it to say 'Exonic.Nonsynonymous'. So rather than edit a random single line in a somewhat important file, I'm just going to name the 'Intronic' file here with '.wSplice' knowing that, in fact, there is now 'splice' in this file (though I could there coudl have been possibly? just hypothetically speaking)

~~~
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Intronic.wSplice | perl -lane 'print $#F;' | sort -g | uniq -c | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,2], seq(0,1000,25), right=FALSE));"
> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,2], seq(0,1000,25), right=FALSE));

     [0,25)     [25,50)     [50,75)    [75,100)   [100,125)   [125,150)
         24          25          25          24          20          21
  [150,175)   [175,200)   [200,225)   [225,250)   [250,275)   [275,300)
         12          12          15           8           2           5
  [300,325)   [325,350)   [350,375)   [375,400)   [400,425)   [425,450)
          3           2           2           0           0           0
  [450,475)   [475,500)   [500,525)   [525,550)   [550,575)   [575,600)
          0           2           0           2           0           0
  [600,625)   [625,650)   [650,675)   [675,700)   [700,725)   [725,750)
          0           0           1           0           0           0
  [750,775)   [775,800)   [800,825)   [825,850)   [850,875)   [875,900)
          0           0           0           0           0           0
  [900,925)   [925,950)   [950,975) [975,1e+03)
          0           0           0           0
>
>
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Intronic.wSplice | perl -lane 'print $#F;' | sort -g | uniq -c | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,2], seq(0,400,10), right=FALSE));"
> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,2], seq(0,400,10), right=FALSE));

   [0,10)   [10,20)   [20,30)   [30,40)   [40,50)   [50,60)   [60,70)   [70,80)
        9        10        10        10        10        10        10        10
  [80,90)  [90,100) [100,110) [110,120) [120,130) [130,140) [140,150) [150,160)
       10         9         9         7         8         7        10         5
[160,170) [170,180) [180,190) [190,200) [200,210) [210,220) [220,230) [230,240)
        5         4         5         5         8         5         5         4
[240,250) [250,260) [260,270) [270,280) [280,290) [290,300) [300,310) [310,320)
        1         1         1         0         4         1         2         1
[320,330) [330,340) [340,350) [350,360) [360,370) [370,380) [380,390) [390,400)
        0         1         1         1         1         0         0         0
>
>
~~~










#Edit /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs

#Edit /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs

#20151223 NOTE -- Code takes about ~xxxhrs to run

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Intronic
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Intronic

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs3.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs TruePheno Pheno1 Intronic

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs3.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs FakePheno Pheno1 Intronic







##20151216
##Doing some followup work on SNPs that failed HWE to make sure/see nothing was really lost from them

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/CheckHWESNPs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | wc

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz --hardy --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe | awk '{ if ($6 < .0001) { print $0 } } ' | awk '{ print $1 "," $2 }' ) > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/CheckHWESNPs/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.genome_summary.csv

~~
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/CheckHWESNPs/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.genome_summary.csv | perl -F, -lane 'print $F[0];' | sort | uniq -c
    263 "UTR3"
     57 "UTR5"
    181 "downstream"
    170 "exonic"
     65 "intergenic"
   2859 "intronic"
      1 "ncRNA_UTR3"
     13 "ncRNA_exonic"
     54 "ncRNA_intronic"
      1 "splicing"
    122 "upstream"
      8 "upstream;downstream"
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/CheckHWESNPs/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.genome_summary.csv | perl -F, -lane 'foreach my $entry1 (@F) { if ($entry1 =~ /synonymous/) { print $entry1; break; } } ' | sort | uniq -c
     76 "nonsynonymous SNV"
     92 "synonymous SNV"
~~

paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe | awk '{ if ($6 < .0001) { print $0 } } ' | awk '{ print $1 "," $2 }' )) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe | grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe | awk '{ if ($6 < .0001) { print $0 } } ' | awk '{ print $1 "\t" $2 }' )) > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/CheckHWESNPs/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.genome_summary.wHardyPLINK.txt



#20151217 CHECK_0: Redo analyses on X chromosome



















##20151231
##Flanking sequence of SNPs for Sudhir custom genotyping array

#On NHS
mkdir /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork
mkdir /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence
#dl'ed P2.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz and P2.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz from e-mail to NHS to get files ready & prepped

#/home/shared/software/bedtools/BEDTools-Version-2.16.2/bin/.
mkdir /mnt/lustre/home/mturchin20/Data/HumanGenome/GRCh37
mkdir /mnt/lustre/home/mturchin20/Data/HumanGenome/GRCh37/Version1
cd /mnt/lustre/home/mturchin20/Data/HumanGenome/GRCh37/Version1
wget ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/reference/human_g1k_v37.fasta.gz
wget ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/reference/human_g1k_v37.fasta.fai

cd /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence
[  mturchin20@spudhead  ~/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence]$zcat /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz | awk '{ print "chr" $1, "\t", $2, "\t", $3 }' | tail -n +2 | perl -lane 'print join("\t", @F);' > /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.bed
[  mturchin20@spudhead  ~/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence]$zcat /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz | awk '{ print "chr" $1, "\t", $2, "\t", $3 }' | tail -n +2 | perl -lane 'print join("\t", @F);' > /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.bed

wget http://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64/fetchChromSizes
#From https://www.biostars.org/p/70795/
bash fetchChromSizes hg19 > hg19.chrom.sizes

[  mturchin20@spudling77  ~/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence]$bedtools flank -i /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.bed -g hg19.chrom.sizes -b 100 > /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Flank100.bed
[  mturchin20@spudling77  ~/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence]$bedtools flank -i /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.bed -g hg19.chrom.sizes -b 100 > /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Flank100.bed

~~~
[  mturchin20@spudling77  ~/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence]$cat /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Flank100.bed | head -n 10
chrX    16863025        16863125
chrX    16863127        16863227
chrX    16863034        16863134
chrX    16863136        16863236
chrX    16863590        16863690
chrX    16863692        16863792
chrX    16863682        16863782
chrX    16863784        16863884
chrX    16863770        16863870
chrX    16863872        16863972
[  mturchin20@spudling77  ~/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence]$cat /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Flank100.bed | head -n 10
chrX    2821483 2821583
chrX    2821585 2821685
chrX    2821681 2821781
chrX    2821783 2821883
chrX    2821747 2821847
chrX    2821849 2821949
chrX    2822002 2822102
chrX    2822104 2822204
chrX    2823032 2823132
chrX    2823134 2823234
~~

bedtools getfasta -fi /mnt/lustre/home/mturchin20/Data/HumanGenome/GRCh37/Version1/human_g1k_v37.fasta.gz -bed /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Flank100.bed -fo /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Flank100.fasta
[  mturchin20@bigmem03  ~]$bedtools getfasta -fi /mnt/lustre/home/mturchin20/Data/HumanGenome/GRCh37/Version1/human_g1k_v37.fasta -bed <( cat /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Flank100.bed | sed 's/chr//g') -fo /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Flank100.fasta

#Realized flank creates two outputs per single entry, where it increases the left and right side of the given bed sequence separately. Using slop instead should increase both edges and keep the ouptut as a single entry, i.e. both sides should be increased on the same single line of output

bedtools slop -i /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.bed -g /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/hg19.chrom.sizes -b 100 > /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Slop100.bed
bedtools slop -i /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.bed -g /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/hg19.chrom.sizes -b 100 > /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Slop100.bed

bedtools getfasta -fi /mnt/lustre/home/mturchin20/Data/HumanGenome/GRCh37/Version1/human_g1k_v37.fasta -bed <(cat /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Slop100.bed | sed 's/chr//g') -fo /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Slop100.fasta
bedtools getfasta -fi /mnt/lustre/home/mturchin20/Data/HumanGenome/GRCh37/Version1/human_g1k_v37.fasta -bed <(cat /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Slop100.bed | sed 's/chr//g') -fo /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Slop100.fasta

bedtools getfasta [OPTIONS] -fi <input FASTA> -bed <BED/GFF/VCF> -fo <output FASTA>

#20160117 -- Picking this back up here now on the Miranda servers since they are back online

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | awk '{ print "chr" $1, "\t", $2, "\t", $3 }' | tail -n +2 | perl -lane 'print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.bed

#mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.bed /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.WRONG1.bed

#in /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence

wget http://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64/fetchChromSizes
#From https://www.biostars.org/p/70795/
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/fetchChromSizes hg19 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/hg19.chrom.sizes
#20160117 NOTE -- Used GRCh37 from 1000G, which is hg19 (see https://www.biostars.org/p/13879/ & human_g1k_v37.fasta)

~~~
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/hg19.chrom.sizes | head -n 24
chr1    249250621
chr2    243199373
chr3    198022430
chr4    191154276
chr5    180915260
chr6    171115067
chr7    159138663
chrX    155270560
chr8    146364022
chr9    141213431
chr10   135534747
chr11   135006516
chr12   133851895
chr13   115169878
chr14   107349540
chr15   102531392
chr16   90354753
chr17   81195210
chr18   78077248
chr20   63025520
chrY    59373566
chr19   59128983
chr22   51304566
chr21   48129895
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/hg19.chrom.sizes | wc
     93     186    1971
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/hg19.chrom.sizes | grep "_" | wc
     68     136    1595
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/hg19.chrom.sizes | grep -v "_" | wc
     25      50     376
~~~

bedtools flank -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.bed -g /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/hg19.chrom.sizes -b 100 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Flank100.bed
bedtools slop -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.bed -g /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/hg19.chrom.sizes -b 100 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.bed

~~~
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Flank100.bed | head -n 10
chrX    2821349 2821449
chrX    2821451 2821551
chrX    2821483 2821583
chrX    2821585 2821685
chrX    2821489 2821589
chrX    2821591 2821691
chrX    2821681 2821781
chrX    2821783 2821883
chrX    2821747 2821847
chrX    2821849 2821949
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.bed | head -n 10
chrX    2821350 2821550
chrX    2821484 2821684
chrX    2821490 2821690
chrX    2821682 2821882
chrX    2821748 2821948
chrX    2821768 2821968
chrX    2822003 2822203
chrX    2822153 2822353
chrX    2822501 2822701
chrX    2822780 2822980
~~~

bedtools getfasta -fi /home/michaelt/Data/HumanGenome/GRCh37/Version1/human_g1k_v37.fasta -bed <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.bed | sed 's/chr//g') -fo /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.fasta

join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'print $F[0], "_", $F[1], "\t$F[0]\t$F[1]\t$F[3]\t$F[4]\t$F[9]\t$F[10]";' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.fasta | perl -ane 'if ($F[0] =~ m/^>/) { print join("\t", @F), "\t"; } else { print join("\t", @F), "\n"; }' | perl -lane 'my $Chr = ((split(/\>/, ((split(/\:/, $F[0]))[0])))[1]); my $BP = ((split(/\-/, ((split(/\:/, $F[0]))[1])))[0]) + 100; print $Chr, "_", $BP, "\t$F[0]\t$Chr\t$BP\t$F[1]";' | sort) | perl -lane 'my @fasta = split(//, $F[$#F]); my $origRef = $fasta[99]; $fasta[99] = "[$F[5]/$F[6]]"; $F[$#F] = join("", @fasta); push(@F, $origRef); print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.FullAnnotation.vs1.txt

~~~
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.fasta | head -n 10
>X:2821350-2821550
TCTATGCAAATCAGAGGCTCATCCTAGATCCAAGGGGAGGAAAAAGGCTCCAACTTGTAGCTGGGCTCATTTGGAAGGTAATGGGAGGACTTGGCAGCTGTAAGACATGTACCACATCTAGACTTCAACCTACAGAAAAGTCCAAAGATACACCAACAGCAGCCCAAGTTAGTCAACTGAAGCTGGCTCAAGGCTGCCCC
>X:2821484-2821684
GAAAAGTCCAAAGATACACCAACAGCAGCCCAAGTTAGTCAACTGAAGCTGGCTCAAGGCTGCCCCCTTTAGGTGGAGACTAACCAGCTCCCTGACATCCTTCAGTTAACTGCACCTTCCTCCTTTTTGCTGCCTGGAATGTCTACATGCTGCCTGGAGCTATGGCAGCCATCTTGTGACCATGAGGCAAACCCTATGTG
>X:2821490-2821690
TCCAAAGATACACCAACAGCAGCCCAAGTTAGTCAACTGAAGCTGGCTCAAGGCTGCCCCCTTTAGGTGGAGACTAACCAGCTCCCTGACATCCTTCAGTTAACTGCACCTTCCTCCTTTTTGCTGCCTGGAATGTCTACATGCTGCCTGGAGCTATGGCAGCCATCTTGTGACCATGAGGCAAACCCTATGTGCAAGAG
>X:2821682-2821882
TGCAAGAGCAGGAAGCTGGAAAGAACCTGTCAGCCCTGGATTGCTTCCTCTGTGACCCCCATTACCTGGGAGAGAACACATCTTTCTCTTGTTTAAGGCGTTCTTATTCCCAGTCGCCCTCTCTCCCCTGTATAACCCTAAAAGCACAGGAGTACACTTGTGACTGTGACTTAAGTTTTTCAGAACGGGGGAGTTTGGAT
>X:2821748-2821948
TGGGAGAGAACACATCTTTCTCTTGTTTAAGGCGTTCTTATTCCCAGTCGCCCTCTCTCCCCTGTATAACCCTAAAAGCACAGGAGTACACTTGTGACTGTGACTTAAGTTTTTCAGAACGGGGGAGTTTGGATTGTTTAGTCCTCTCTCCATGAATAGTGAATAGCAATGCCAGGTGAGTTCCTTCAAAGCTGGGGGGT
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.fasta | perl -ane 'if ($F[0] =~ m/^>/) { print join("\t", @F), "\t"; } else { print join("\t", @F), "\n"; }' | perl -lane 'my $Chr = ((split(/\>/, ((split(/\:/, $F[0]))[0])))[1]); my $BP = ((split(/\-/, ((split(/\:/, $F[0]))[1])))[0]) + 100; print $Chr, "_", $BP, "\t", join("\t", @F);' | head -n 10
X_2821450       >X:2821350-2821550      TCTATGCAAATCAGAGGCTCATCCTAGATCCAAGGGGAGGAAAAAGGCTCCAACTTGTAGCTGGGCTCATTTGGAAGGTAATGGGAGGACTTGGCAGCTGTAAGACATGTACCACATCTAGACTTCAACCTACAGAAAAGTCCAAAGATACACCAACAGCAGCCCAAGTTAGTCAACTGAAGCTGGCTCAAGGCTGCCCC
X_2821584       >X:2821484-2821684      GAAAAGTCCAAAGATACACCAACAGCAGCCCAAGTTAGTCAACTGAAGCTGGCTCAAGGCTGCCCCCTTTAGGTGGAGACTAACCAGCTCCCTGACATCCTTCAGTTAACTGCACCTTCCTCCTTTTTGCTGCCTGGAATGTCTACATGCTGCCTGGAGCTATGGCAGCCATCTTGTGACCATGAGGCAAACCCTATGTG
X_2821590       >X:2821490-2821690      TCCAAAGATACACCAACAGCAGCCCAAGTTAGTCAACTGAAGCTGGCTCAAGGCTGCCCCCTTTAGGTGGAGACTAACCAGCTCCCTGACATCCTTCAGTTAACTGCACCTTCCTCCTTTTTGCTGCCTGGAATGTCTACATGCTGCCTGGAGCTATGGCAGCCATCTTGTGACCATGAGGCAAACCCTATGTGCAAGAG
X_2821782       >X:2821682-2821882      TGCAAGAGCAGGAAGCTGGAAAGAACCTGTCAGCCCTGGATTGCTTCCTCTGTGACCCCCATTACCTGGGAGAGAACACATCTTTCTCTTGTTTAAGGCGTTCTTATTCCCAGTCGCCCTCTCTCCCCTGTATAACCCTAAAAGCACAGGAGTACACTTGTGACTGTGACTTAAGTTTTTCAGAACGGGGGAGTTTGGAT
X_2821848       >X:2821748-2821948      TGGGAGAGAACACATCTTTCTCTTGTTTAAGGCGTTCTTATTCCCAGTCGCCCTCTCTCCCCTGTATAACCCTAAAAGCACAGGAGTACACTTGTGACTGTGACTTAAGTTTTTCAGAACGGGGGAGTTTGGATTGTTTAGTCCTCTCTCCATGAATAGTGAATAGCAATGCCAGGTGAGTTCCTTCAAAGCTGGGGGGT
X_2821868       >X:2821768-2821968      TCTTGTTTAAGGCGTTCTTATTCCCAGTCGCCCTCTCTCCCCTGTATAACCCTAAAAGCACAGGAGTACACTTGTGACTGTGACTTAAGTTTTTCAGAACGGGGGAGTTTGGATTGTTTAGTCCTCTCTCCATGAATAGTGAATAGCAATGCCAGGTGAGTTCCTTCAAAGCTGGGGGGTGAAGGGGTGGAGGTGGAAAA
X_2822103       >X:2822003-2822203      GACACCATCCAGGGAGATTTGTCACATTTTATTCAGTATTTCTGCTGCACTGCCAGCCTAGGGATGCACTTGATTCCCAAGAAATGCAACTGTCCTATTCGCAGAGCCGTCCACAGGTACCTACCCCCTGGACTGCAGCAACTTTATTACCTTAACTAGCACAGAACAGAGGTTGATTTAAACTCCTTACACTCACTTCT
X_2822253       >X:2822153-2822353      CTTAACTAGCACAGAACAGAGGTTGATTTAAACTCCTTACACTCACTTCTCAGATCAATGAATGGGCAAAGAAACACCTCATGGCTCTGGGAAGGCATGCTGAGACCCGTTTTTGCAAGTCCTGAGGAATGGAAGAATATAGCTGCCAGGTATCCCAAGTCTAGGGCAGGGAGGGTAGTATCGGCATCACTTTCACTGCA
X_2822601       >X:2822501-2822701      AAAGTCCAAAGATGCACTCACAGGAGCCCGTTAGTCAACTGAAGCTGGTTCATGGCTCTCTTCTTCAGGTGGAGACTAACCAACTCCCTGACCTCCTTCGGTTAAGTGCTCCTTCCTTTTTCCTGCTGCCTGGAATGTCTATGTGCCATCTGGAGCTATGGCAGCCTTCTTGCAACCATGAGGCAAACTCTATGTGCAAG
X_2822880       >X:2822780-2822980      GCCATTTTCATCAAATGCCAGGAAATTCTTAGGCTGGGATTGTTGGGATTGTTTGAATGGTTGAGCGTGAAAGACTTGTTATAATGGGCTTGTTGAATACGGTGAAAGATGAGATGGCTTTTAGCTGCCTCCACTCCCCGTGTCCCATGCTCCATAGATGTTTCTTTTTTTTTTTTTTTTTAATCATTTTTTGGTTATGC
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'print $F[0], "_", $F[1], "\t$F[0]\t$F[1]\t$F[3]\t$F[4]\t$F[9]\t$F[10]";' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.fasta | perl -ane 'if ($F[0] =~ m/^>/) { print join("\t", @F), "\t"; } else { print join("\t", @F), "\n"; }' | perl -lane 'my $Chr = ((split(/\>/, ((split(/\:/, $F[0]))[0])))[1]); my $BP = ((split(/\-/, ((split(/\:/, $F[0]))[1])))[0]) + 100; print $Chr, "_", $BP, "\t$F[0]\t$Chr\t$BP\t$F[1]";' | sort) | head -n 10
10_112261958 10 112261958 intronic DUSP5 G A >10:112261858-112262058 10 112261958 TTGATTTAGAAGGTAATTAACTGGATTGATGACTCCAAATTGGACCTAGAAAGGATTGTCAGCAGTGTTCTTCTTAGCACCTTGGGAGTGAGAAGCAGTGAGGCTGTTTTACTGCCCTGAGTCGAAATGAAACCTCGTATCAGCCCCCATTTCCTTATAAATGGAGGAGGTTCCAGCTAGAATCCTGCTTAGAGCAGATG
10_112262017 10 112262017 intronic DUSP5 A G >10:112261917-112262117 10 112262017 CAGCAGTGTTCTTCTTAGCACCTTGGGAGTGAGAAGCAGTGAGGCTGTTTTACTGCCCTGAGTCGAAATGAAACCTCGTATCAGCCCCCATTTCCTTATAAATGGAGGAGGTTCCAGCTAGAATCCTGCTTAGAGCAGATGGGCTGGGGAAGAAGCCACTTCAGGAGCCTGTCTTCTCGCCAATTAATGAAATTTTGGGT
10_112262028 10 112262028 intronic DUSP5 G A >10:112261928-112262128 10 112262028 TTCTTAGCACCTTGGGAGTGAGAAGCAGTGAGGCTGTTTTACTGCCCTGAGTCGAAATGAAACCTCGTATCAGCCCCCATTTCCTTATAAATGGAGGAGGTTCCAGCTAGAATCCTGCTTAGAGCAGATGGGCTGGGGAAGAAGCCACTTCAGGAGCCTGTCTTCTCGCCAATTAATGAAATTTTGGGTAATCACACCTT
10_112262055 10 112262055 intronic DUSP5 G T >10:112261955-112262155 10 112262055 GTGAGGCTGTTTTACTGCCCTGAGTCGAAATGAAACCTCGTATCAGCCCCCATTTCCTTATAAATGGAGGAGGTTCCAGCTAGAATCCTGCTTAGAGCAGATGGGCTGGGGAAGAAGCCACTTCAGGAGCCTGTCTTCTCGCCAATTAATGAAATTTTGGGTAATCACACCTTCCTTTTAGATAAACAAGAGAGAAAATA
10_112262082 10 112262082 intronic DUSP5 G T >10:112261982-112262182 10 112262082 AAATGAAACCTCGTATCAGCCCCCATTTCCTTATAAATGGAGGAGGTTCCAGCTAGAATCCTGCTTAGAGCAGATGGGCTGGGGAAGAAGCCACTTCAGGAGCCTGTCTTCTCGCCAATTAATGAAATTTTGGGTAATCACACCTTCCTTTTAGATAAACAAGAGAGAAAATAGCCTTTGTTTTCTGGAGGGATTTGGTT
10_112262086 10 112262086 intronic DUSP5 C A >10:112261986-112262186 10 112262086 GAAACCTCGTATCAGCCCCCATTTCCTTATAAATGGAGGAGGTTCCAGCTAGAATCCTGCTTAGAGCAGATGGGCTGGGGAAGAAGCCACTTCAGGAGCCTGTCTTCTCGCCAATTAATGAAATTTTGGGTAATCACACCTTCCTTTTAGATAAACAAGAGAGAAAATAGCCTTTGTTTTCTGGAGGGATTTGGTTAGTA
10_112262096 10 112262096 intronic DUSP5 G A >10:112261996-112262196 10 112262096 ATCAGCCCCCATTTCCTTATAAATGGAGGAGGTTCCAGCTAGAATCCTGCTTAGAGCAGATGGGCTGGGGAAGAAGCCACTTCAGGAGCCTGTCTTCTCGCCAATTAATGAAATTTTGGGTAATCACACCTTCCTTTTAGATAAACAAGAGAGAAAATAGCCTTTGTTTTCTGGAGGGATTTGGTTAGTATTTGCCAGAT
10_112262125 10 112262125 intronic DUSP5 C T >10:112262025-112262225 10 112262125 AGGTTCCAGCTAGAATCCTGCTTAGAGCAGATGGGCTGGGGAAGAAGCCACTTCAGGAGCCTGTCTTCTCGCCAATTAATGAAATTTTGGGTAATCACACCTTCCTTTTAGATAAACAAGAGAGAAAATAGCCTTTGTTTTCTGGAGGGATTTGGTTAGTATTTGCCAGATGCTAGACACTTTCTTTCACAGTTTAATCT
10_112262561 10 112262561 exonic DUSP5 G C >10:112262461-112262661 10 112262561 TTTGTTTGTTTTTGTAGGGGGATATGAGACTTTCTACTCGGAATATCCTGAGTGTTGCGTGGATGTAAAACCCATTTCACAAGAGAAGATTGAGAGTGAGAGAGCCCTCATCAGCCAGTGTGGAAAACCAGTGGTAAATGTCAGCTACAGGCCAGCTTATGACCAGGTACGTGATGTGATGGGGAAGAGGTATCCTGAGT
10_112262625 10 112262625 exonic DUSP5 C G >10:112262525-112262725 10 112262625 GTAAAACCCATTTCACAAGAGAAGATTGAGAGTGAGAGAGCCCTCATCAGCCAGTGTGGAAAACCAGTGGTAAATGTCAGCTACAGGCCAGCTTATGACCAGGTACGTGATGTGATGGGGAAGAGGTATCCTGAGTGGTATTCTGGGCTCCTGTCTCCCTTCCTTCCTCAGCACCTGACTGTTCTCTCCCCACTCAGCAA
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'print $F[0], "_", $F[1], "\t$F[0]\t$F[1]\t$F[3]\t$F[4]\t$F[9]\t$F[10]";' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.fasta | perl -ane 'if ($F[0] =~ m/^>/) { print join("\t", @F), "\t"; } else { print join("\t", @F), "\n"; }' | perl -lane 'my $Chr = ((split(/\>/, ((split(/\:/, $F[0]))[0])))[1]); my $BP = ((split(/\-/, ((split(/\:/, $F[0]))[1])))[0]) + 100; print $Chr, "_", $BP, "\t$F[0]\t$Chr\t$BP\t$F[1]";' | sort) | perl -lane 'my @fasta = split(//
> , $F[$#F]); my $origRef = $fasta[99]; $fasta[99] = "[$F[5]/$F[6]]"; $F[$#F] = join("", @fasta); push(@F, $origRef); print join("\t", @F);' | head -n 10
10_112261958    10      112261958       intronic        DUSP5   G       A       >10:112261858-112262058 10      112261958       TTGATTTAGAAGGTAATTAACTGGATTGATGACTCCAAATTGGACCTAGAAAGGATTGTCAGCAGTGTTCTTCTTAGCACCTTGGGAGTGAGAAGCAGT[G/A]AGGCTGTTTTACTGCCCTGAGTCGAAATGAAACCTCGTATCAGCCCCCATTTCCTTATAAATGGAGGAGGTTCCAGCTAGAATCCTGCTTAGAGCAGATG       G
10_112262017    10      112262017       intronic        DUSP5   A       G       >10:112261917-112262117 10      112262017       CAGCAGTGTTCTTCTTAGCACCTTGGGAGTGAGAAGCAGTGAGGCTGTTTTACTGCCCTGAGTCGAAATGAAACCTCGTATCAGCCCCCATTTCCTTAT[A/G]AATGGAGGAGGTTCCAGCTAGAATCCTGCTTAGAGCAGATGGGCTGGGGAAGAAGCCACTTCAGGAGCCTGTCTTCTCGCCAATTAATGAAATTTTGGGT       A
10_112262028    10      112262028       intronic        DUSP5   G       A       >10:112261928-112262128 10      112262028       TTCTTAGCACCTTGGGAGTGAGAAGCAGTGAGGCTGTTTTACTGCCCTGAGTCGAAATGAAACCTCGTATCAGCCCCCATTTCCTTATAAATGGAGGAG[G/A]TTCCAGCTAGAATCCTGCTTAGAGCAGATGGGCTGGGGAAGAAGCCACTTCAGGAGCCTGTCTTCTCGCCAATTAATGAAATTTTGGGTAATCACACCTT       G
10_112262055    10      112262055       intronic        DUSP5   G       T       >10:112261955-112262155 10      112262055       GTGAGGCTGTTTTACTGCCCTGAGTCGAAATGAAACCTCGTATCAGCCCCCATTTCCTTATAAATGGAGGAGGTTCCAGCTAGAATCCTGCTTAGAGCA[G/T]ATGGGCTGGGGAAGAAGCCACTTCAGGAGCCTGTCTTCTCGCCAATTAATGAAATTTTGGGTAATCACACCTTCCTTTTAGATAAACAAGAGAGAAAATA       G
10_112262082    10      112262082       intronic        DUSP5   G       T       >10:112261982-112262182 10      112262082       AAATGAAACCTCGTATCAGCCCCCATTTCCTTATAAATGGAGGAGGTTCCAGCTAGAATCCTGCTTAGAGCAGATGGGCTGGGGAAGAAGCCACTTCAG[G/T]AGCCTGTCTTCTCGCCAATTAATGAAATTTTGGGTAATCACACCTTCCTTTTAGATAAACAAGAGAGAAAATAGCCTTTGTTTTCTGGAGGGATTTGGTT       G
10_112262086    10      112262086       intronic        DUSP5   C       A       >10:112261986-112262186 10      112262086       GAAACCTCGTATCAGCCCCCATTTCCTTATAAATGGAGGAGGTTCCAGCTAGAATCCTGCTTAGAGCAGATGGGCTGGGGAAGAAGCCACTTCAGGAGC[C/A]TGTCTTCTCGCCAATTAATGAAATTTTGGGTAATCACACCTTCCTTTTAGATAAACAAGAGAGAAAATAGCCTTTGTTTTCTGGAGGGATTTGGTTAGTA       C
10_112262096    10      112262096       intronic        DUSP5   G       A       >10:112261996-112262196 10      112262096       ATCAGCCCCCATTTCCTTATAAATGGAGGAGGTTCCAGCTAGAATCCTGCTTAGAGCAGATGGGCTGGGGAAGAAGCCACTTCAGGAGCCTGTCTTCTC[G/A]CCAATTAATGAAATTTTGGGTAATCACACCTTCCTTTTAGATAAACAAGAGAGAAAATAGCCTTTGTTTTCTGGAGGGATTTGGTTAGTATTTGCCAGAT       G
10_112262125    10      112262125       intronic        DUSP5   C       T       >10:112262025-112262225 10      112262125       AGGTTCCAGCTAGAATCCTGCTTAGAGCAGATGGGCTGGGGAAGAAGCCACTTCAGGAGCCTGTCTTCTCGCCAATTAATGAAATTTTGGGTAATCACA[C/T]CTTCCTTTTAGATAAACAAGAGAGAAAATAGCCTTTGTTTTCTGGAGGGATTTGGTTAGTATTTGCCAGATGCTAGACACTTTCTTTCACAGTTTAATCT       C
10_112262561    10      112262561       exonic  DUSP5   G       C       >10:112262461-112262661 10      112262561       TTTGTTTGTTTTTGTAGGGGGATATGAGACTTTCTACTCGGAATATCCTGAGTGTTGCGTGGATGTAAAACCCATTTCACAAGAGAAGATTGAGAGTGA[G/C]AGAGCCCTCATCAGCCAGTGTGGAAAACCAGTGGTAAATGTCAGCTACAGGCCAGCTTATGACCAGGTACGTGATGTGATGGGGAAGAGGTATCCTGAGT       G
10_112262625    10      112262625       exonic  DUSP5   C       G       >10:112262525-112262725 10      112262625       GTAAAACCCATTTCACAAGAGAAGATTGAGAGTGAGAGAGCCCTCATCAGCCAGTGTGGAAAACCAGTGGTAAATGTCAGCTACAGGCCAGCTTATGAC[C/G]AGGTACGTGATGTGATGGGGAAGAGGTATCCTGAGTGGTATTCTGGGCTCCTGTCTCCCTTCCTTCCTCAGCACCTGACTGTTCTCTCCCCACTCAGCAA       C
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.FullAnnotation.vs1.txt | head -n 10
10_112261958    10      112261958       intronic        DUSP5   G       A       >10:112261858-112262058 10      112261958       TTGATTTAGAAGGTAATTAACTGGATTGATGACTCCAAATTGGACCTAGAAAGGATTGTCAGCAGTGTTCTTCTTAGCACCTTGGGAGTGAGAAGCAGT[G/A]AGGCTGTTTTACTGCCCTGAGTCGAAATGAAACCTCGTATCAGCCCCCATTTCCTTATAAATGGAGGAGGTTCCAGCTAGAATCCTGCTTAGAGCAGATG       G
10_112262017    10      112262017       intronic        DUSP5   A       G       >10:112261917-112262117 10      112262017       CAGCAGTGTTCTTCTTAGCACCTTGGGAGTGAGAAGCAGTGAGGCTGTTTTACTGCCCTGAGTCGAAATGAAACCTCGTATCAGCCCCCATTTCCTTAT[A/G]AATGGAGGAGGTTCCAGCTAGAATCCTGCTTAGAGCAGATGGGCTGGGGAAGAAGCCACTTCAGGAGCCTGTCTTCTCGCCAATTAATGAAATTTTGGGT       A
10_112262028    10      112262028       intronic        DUSP5   G       A       >10:112261928-112262128 10      112262028       TTCTTAGCACCTTGGGAGTGAGAAGCAGTGAGGCTGTTTTACTGCCCTGAGTCGAAATGAAACCTCGTATCAGCCCCCATTTCCTTATAAATGGAGGAG[G/A]TTCCAGCTAGAATCCTGCTTAGAGCAGATGGGCTGGGGAAGAAGCCACTTCAGGAGCCTGTCTTCTCGCCAATTAATGAAATTTTGGGTAATCACACCTT       G
10_112262055    10      112262055       intronic        DUSP5   G       T       >10:112261955-112262155 10      112262055       GTGAGGCTGTTTTACTGCCCTGAGTCGAAATGAAACCTCGTATCAGCCCCCATTTCCTTATAAATGGAGGAGGTTCCAGCTAGAATCCTGCTTAGAGCA[G/T]ATGGGCTGGGGAAGAAGCCACTTCAGGAGCCTGTCTTCTCGCCAATTAATGAAATTTTGGGTAATCACACCTTCCTTTTAGATAAACAAGAGAGAAAATA       G
10_112262082    10      112262082       intronic        DUSP5   G       T       >10:112261982-112262182 10      112262082       AAATGAAACCTCGTATCAGCCCCCATTTCCTTATAAATGGAGGAGGTTCCAGCTAGAATCCTGCTTAGAGCAGATGGGCTGGGGAAGAAGCCACTTCAG[G/T]AGCCTGTCTTCTCGCCAATTAATGAAATTTTGGGTAATCACACCTTCCTTTTAGATAAACAAGAGAGAAAATAGCCTTTGTTTTCTGGAGGGATTTGGTT       G
10_112262086    10      112262086       intronic        DUSP5   C       A       >10:112261986-112262186 10      112262086       GAAACCTCGTATCAGCCCCCATTTCCTTATAAATGGAGGAGGTTCCAGCTAGAATCCTGCTTAGAGCAGATGGGCTGGGGAAGAAGCCACTTCAGGAGC[C/A]TGTCTTCTCGCCAATTAATGAAATTTTGGGTAATCACACCTTCCTTTTAGATAAACAAGAGAGAAAATAGCCTTTGTTTTCTGGAGGGATTTGGTTAGTA       C
10_112262096    10      112262096       intronic        DUSP5   G       A       >10:112261996-112262196 10      112262096       ATCAGCCCCCATTTCCTTATAAATGGAGGAGGTTCCAGCTAGAATCCTGCTTAGAGCAGATGGGCTGGGGAAGAAGCCACTTCAGGAGCCTGTCTTCTC[G/A]CCAATTAATGAAATTTTGGGTAATCACACCTTCCTTTTAGATAAACAAGAGAGAAAATAGCCTTTGTTTTCTGGAGGGATTTGGTTAGTATTTGCCAGAT       G
10_112262125    10      112262125       intronic        DUSP5   C       T       >10:112262025-112262225 10      112262125       AGGTTCCAGCTAGAATCCTGCTTAGAGCAGATGGGCTGGGGAAGAAGCCACTTCAGGAGCCTGTCTTCTCGCCAATTAATGAAATTTTGGGTAATCACA[C/T]CTTCCTTTTAGATAAACAAGAGAGAAAATAGCCTTTGTTTTCTGGAGGGATTTGGTTAGTATTTGCCAGATGCTAGACACTTTCTTTCACAGTTTAATCT       C
10_112262561    10      112262561       exonic  DUSP5   G       C       >10:112262461-112262661 10      112262561       TTTGTTTGTTTTTGTAGGGGGATATGAGACTTTCTACTCGGAATATCCTGAGTGTTGCGTGGATGTAAAACCCATTTCACAAGAGAAGATTGAGAGTGA[G/C]AGAGCCCTCATCAGCCAGTGTGGAAAACCAGTGGTAAATGTCAGCTACAGGCCAGCTTATGACCAGGTACGTGATGTGATGGGGAAGAGGTATCCTGAGT       G
10_112262625    10      112262625       exonic  DUSP5   C       G       >10:112262525-112262725 10      112262625       GTAAAACCCATTTCACAAGAGAAGATTGAGAGTGAGAGAGCCCTCATCAGCCAGTGTGGAAAACCAGTGGTAAATGTCAGCTACAGGCCAGCTTATGAC[C/G]AGGTACGTGATGTGATGGGGAAGAGGTATCCTGAGTGGTATTCTGGGCTCCTGTCTCCCTTCCTTCCTCAGCACCTGACTGTTCTCTCCCCACTCAGCAA       C
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.FullAnnotation.vs1.txt | awk '{ print $2, "\t", $3, "\t", $6, "\t", $7, "\t", $11, "\t", $12 }' | head -n 10
10       112261958       G       A       TTGATTTAGAAGGTAATTAACTGGATTGATGACTCCAAATTGGACCTAGAAAGGATTGTCAGCAGTGTTCTTCTTAGCACCTTGGGAGTGAGAAGCAGT[G/A]AGGCTGTTTTACTGCCCTGAGTCGAAATGAAACCTCGTATCAGCCCCCATTTCCTTATAAATGGAGGAGGTTCCAGCTAGAATCCTGCTTAGAGCAGATG    G
10       112262017       A       G       CAGCAGTGTTCTTCTTAGCACCTTGGGAGTGAGAAGCAGTGAGGCTGTTTTACTGCCCTGAGTCGAAATGAAACCTCGTATCAGCCCCCATTTCCTTAT[A/G]AATGGAGGAGGTTCCAGCTAGAATCCTGCTTAGAGCAGATGGGCTGGGGAAGAAGCCACTTCAGGAGCCTGTCTTCTCGCCAATTAATGAAATTTTGGGT    A
10       112262028       G       A       TTCTTAGCACCTTGGGAGTGAGAAGCAGTGAGGCTGTTTTACTGCCCTGAGTCGAAATGAAACCTCGTATCAGCCCCCATTTCCTTATAAATGGAGGAG[G/A]TTCCAGCTAGAATCCTGCTTAGAGCAGATGGGCTGGGGAAGAAGCCACTTCAGGAGCCTGTCTTCTCGCCAATTAATGAAATTTTGGGTAATCACACCTT    G
10       112262055       G       T       GTGAGGCTGTTTTACTGCCCTGAGTCGAAATGAAACCTCGTATCAGCCCCCATTTCCTTATAAATGGAGGAGGTTCCAGCTAGAATCCTGCTTAGAGCA[G/T]ATGGGCTGGGGAAGAAGCCACTTCAGGAGCCTGTCTTCTCGCCAATTAATGAAATTTTGGGTAATCACACCTTCCTTTTAGATAAACAAGAGAGAAAATA    G
10       112262082       G       T       AAATGAAACCTCGTATCAGCCCCCATTTCCTTATAAATGGAGGAGGTTCCAGCTAGAATCCTGCTTAGAGCAGATGGGCTGGGGAAGAAGCCACTTCAG[G/T]AGCCTGTCTTCTCGCCAATTAATGAAATTTTGGGTAATCACACCTTCCTTTTAGATAAACAAGAGAGAAAATAGCCTTTGTTTTCTGGAGGGATTTGGTT    G
10       112262086       C       A       GAAACCTCGTATCAGCCCCCATTTCCTTATAAATGGAGGAGGTTCCAGCTAGAATCCTGCTTAGAGCAGATGGGCTGGGGAAGAAGCCACTTCAGGAGC[C/A]TGTCTTCTCGCCAATTAATGAAATTTTGGGTAATCACACCTTCCTTTTAGATAAACAAGAGAGAAAATAGCCTTTGTTTTCTGGAGGGATTTGGTTAGTA    C
10       112262096       G       A       ATCAGCCCCCATTTCCTTATAAATGGAGGAGGTTCCAGCTAGAATCCTGCTTAGAGCAGATGGGCTGGGGAAGAAGCCACTTCAGGAGCCTGTCTTCTC[G/A]CCAATTAATGAAATTTTGGGTAATCACACCTTCCTTTTAGATAAACAAGAGAGAAAATAGCCTTTGTTTTCTGGAGGGATTTGGTTAGTATTTGCCAGAT    G
10       112262125       C       T       AGGTTCCAGCTAGAATCCTGCTTAGAGCAGATGGGCTGGGGAAGAAGCCACTTCAGGAGCCTGTCTTCTCGCCAATTAATGAAATTTTGGGTAATCACA[C/T]CTTCCTTTTAGATAAACAAGAGAGAAAATAGCCTTTGTTTTCTGGAGGGATTTGGTTAGTATTTGCCAGATGCTAGACACTTTCTTTCACAGTTTAATCT    C
10       112262561       G       C       TTTGTTTGTTTTTGTAGGGGGATATGAGACTTTCTACTCGGAATATCCTGAGTGTTGCGTGGATGTAAAACCCATTTCACAAGAGAAGATTGAGAGTGA[G/C]AGAGCCCTCATCAGCCAGTGTGGAAAACCAGTGGTAAATGTCAGCTACAGGCCAGCTTATGACCAGGTACGTGATGTGATGGGGAAGAGGTATCCTGAGT    G
10       112262625       C       G       GTAAAACCCATTTCACAAGAGAAGATTGAGAGTGAGAGAGCCCTCATCAGCCAGTGTGGAAAACCAGTGGTAAATGTCAGCTACAGGCCAGCTTATGAC[C/G]AGGTACGTGATGTGATGGGGAAGAGGTATCCTGAGTGGTATTCTGGGCTCCTGTCTCCCTTCCTTCCTCAGCACCTGACTGTTCTCTCCCCACTCAGCAA    C
~~~

#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.FullAnnotation.vs1.txt | awk '{ print $2, "\t", $3, "\t", $6, "\t", $7, "\t", $11, "\t", $12 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.VariantLevel.100bpFlanking.Update1.txt 
#
##Included the below header in file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.VariantLevel.100bpFlanking.Update1.txt
##CHR	BP	REF	ALT	SEQUENCE	ORIG_POS_100
#
#~~~
#[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.VariantLevel.100bpFlanking.Update1.txt | wc
#  85580  513480 19909658
#~~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.VariantLevel.100bpFlanking.Update1.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/.

join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'print $F[0], "_", $F[1], "\t$F[0]\t$F[1]\t$F[3]\t$F[4]\t$F[9]\t$F[10]";' | sort) <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.frq.gz | perl -lane 'print $F[0], "_", $F[1], "\t", $F[0], "_", $F[1], "\t", join("\t", split(/\:/, $F[4])), "\t", join("\t", split(/\:/, $F[5]));' | sort ) | join - <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.fasta | perl -ane 'if ($F[0] =~ m/^>/) { print join("\t", @F), "\t"; } else { print join("\t", @F), "\n"; }' | perl -lane 'my $Chr = ((split(/\>/, ((split(/\:/, $F[0]))[0])))[1]); my $BP = ((split(/\-/, ((split(/\:/, $F[0]))[1])))[0]) + 100; print $Chr, "_", $BP, "\t$F[0]\t$Chr\t$BP\t$F[1]";' | sort) | perl -lane 'my @fasta = split(//, $F[$#F]); my $origRef = $fasta[99]; $fasta[99] = "[$F[5]/$F[6]]"; $F[$#F] = join("", @fasta); push(@F, $origRef); print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.FullAnnotation.vs2.txt

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.FullAnnotation.vs2.txt | perl -lane 'if ($F[5] eq $F[8]) { splice (@F, 10, 2); splice(@F, 7, 2); } elsif ($F[5] eq $F[10]) { splice(@F, 7, 4); } else { print STDERR "Error1a -- $F[0] does not have either ref or alt allele matching with allelef requency"; } print join("\t", @F);' | awk '{ print $2, "\t", $3, "\t", $6, "\t", $7, "\t", $8, "\t", $12, "\t", $13 }' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.VariantLevel.100bpFlanking.Update1.txt.gz

~~~
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.FullAnnotation.vs2.txt | perl -lane 'if ($F[5] eq $F[8]) { print "REF"; } elsif ($F[5] eq $F[10]) { print "ALT"; } else { print "NA"; }' | sort | uniq -c
  85579 REF
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.FullAnnotation.vs2.txt | perl -lane 'if ($F[5] eq $F[8]) { splice (@F, 10, 2); splice(@F, 7, 2); } elsif ($F[5] eq $F[10]) { splice(@F, 7, 4); } else { print STDERR "Error1a -- $F[0] does not have either ref or alt allele matching with allelef requency"; } print join("\t", @F);' | awk '{ print $2, "\t", $3, "\t", $6, "\t", $7, "\t", $8, "\t", $12, "\t", $13 }' | head -n 10
10       112261958       G       A       0.996644        TTGATTTAGAAGGTAATTAACTGGATTGATGACTCCAAATTGGACCTAGAAAGGATTGTCAGCAGTGTTCTTCTTAGCACCTTGGGAGTGAGAAGCAGT[G/A]AGGCTGTTTTACTGCCCTGAGTCGAAATGAAACCTCGTATCAGCCCCCATTTCCTTATAAATGGAGGAGGTTCCAGCTAGAATCCTGCTTAGAGCAGATG    G
10       112262017       A       G       0.980537        CAGCAGTGTTCTTCTTAGCACCTTGGGAGTGAGAAGCAGTGAGGCTGTTTTACTGCCCTGAGTCGAAATGAAACCTCGTATCAGCCCCCATTTCCTTAT[A/G]AATGGAGGAGGTTCCAGCTAGAATCCTGCTTAGAGCAGATGGGCTGGGGAAGAAGCCACTTCAGGAGCCTGTCTTCTCGCCAATTAATGAAATTTTGGGT    A
10       112262028       G       A       0.998658        TTCTTAGCACCTTGGGAGTGAGAAGCAGTGAGGCTGTTTTACTGCCCTGAGTCGAAATGAAACCTCGTATCAGCCCCCATTTCCTTATAAATGGAGGAG[G/A]TTCCAGCTAGAATCCTGCTTAGAGCAGATGGGCTGGGGAAGAAGCCACTTCAGGAGCCTGTCTTCTCGCCAATTAATGAAATTTTGGGTAATCACACCTT    G
10       112262055       G       T       0.99933         GTGAGGCTGTTTTACTGCCCTGAGTCGAAATGAAACCTCGTATCAGCCCCCATTTCCTTATAAATGGAGGAGGTTCCAGCTAGAATCCTGCTTAGAGCA[G/T]ATGGGCTGGGGAAGAAGCCACTTCAGGAGCCTGTCTTCTCGCCAATTAATGAAATTTTGGGTAATCACACCTTCCTTTTAGATAAACAAGAGAGAAAATA    G
10       112262082       G       T       0.99933         AAATGAAACCTCGTATCAGCCCCCATTTCCTTATAAATGGAGGAGGTTCCAGCTAGAATCCTGCTTAGAGCAGATGGGCTGGGGAAGAAGCCACTTCAG[G/T]AGCCTGTCTTCTCGCCAATTAATGAAATTTTGGGTAATCACACCTTCCTTTTAGATAAACAAGAGAGAAAATAGCCTTTGTTTTCTGGAGGGATTTGGTT    G
10       112262086       C       A       0.997319        GAAACCTCGTATCAGCCCCCATTTCCTTATAAATGGAGGAGGTTCCAGCTAGAATCCTGCTTAGAGCAGATGGGCTGGGGAAGAAGCCACTTCAGGAGC[C/A]TGTCTTCTCGCCAATTAATGAAATTTTGGGTAATCACACCTTCCTTTTAGATAAACAAGAGAGAAAATAGCCTTTGTTTTCTGGAGGGATTTGGTTAGTA    C
10       112262096       G       A       0.997989        ATCAGCCCCCATTTCCTTATAAATGGAGGAGGTTCCAGCTAGAATCCTGCTTAGAGCAGATGGGCTGGGGAAGAAGCCACTTCAGGAGCCTGTCTTCTC[G/A]CCAATTAATGAAATTTTGGGTAATCACACCTTCCTTTTAGATAAACAAGAGAGAAAATAGCCTTTGTTTTCTGGAGGGATTTGGTTAGTATTTGCCAGAT    G
10       112262125       C       T       0.325737        AGGTTCCAGCTAGAATCCTGCTTAGAGCAGATGGGCTGGGGAAGAAGCCACTTCAGGAGCCTGTCTTCTCGCCAATTAATGAAATTTTGGGTAATCACA[C/T]CTTCCTTTTAGATAAACAAGAGAGAAAATAGCCTTTGTTTTCTGGAGGGATTTGGTTAGTATTTGCCAGATGCTAGACACTTTCTTTCACAGTTTAATCT    C
10       112262561       G       C       0.999331        TTTGTTTGTTTTTGTAGGGGGATATGAGACTTTCTACTCGGAATATCCTGAGTGTTGCGTGGATGTAAAACCCATTTCACAAGAGAAGATTGAGAGTGA[G/C]AGAGCCCTCATCAGCCAGTGTGGAAAACCAGTGGTAAATGTCAGCTACAGGCCAGCTTATGACCAGGTACGTGATGTGATGGGGAAGAGGTATCCTGAGT    G
10       112262625       C       G       0.999331        GTAAAACCCATTTCACAAGAGAAGATTGAGAGTGAGAGAGCCCTCATCAGCCAGTGTGGAAAACCAGTGGTAAATGTCAGCTACAGGCCAGCTTATGAC[C/G]AGGTACGTGATGTGATGGGGAAGAGGTATCCTGAGTGGTATTCTGGGCTCCTGTCTCCCTTCCTTCCTCAGCACCTGACTGTTCTCTCCCCACTCAGCAA    C
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.VariantLevel.100bpFlanking.Update1.txt.gz | perl -lane 'if ($F[2] eq $F[$#F]) { print "REF"; } elsif ($F[3] eq $F[$#F]) { print "ALT"; } else { print "NA"; }' | sort | uniq -c
      1 NA
  85579 REF
~~~

#Included the below header in file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.VariantLevel.100bpFlanking.Update1.txt.gz
#CHR	BP	REF	ALT	AF_REF	SEQUENCE	ORIG_POS_100

~~~
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.VariantLevel.100bpFlanking.Update1.txt.gz | wc
  85580  599060 20842627
~~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.VariantLevel.100bpFlanking.Update1.txt.gz mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/.

#On MacBook Air
#scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.VariantLevel.100bpFlanking.Update1.txt.gz /Users/mturchin20/Documents/Work/StephensLab/HIVStuff/PG/.
#scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.Vs2.annotated.Update1.vcf.gz mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt /Users/mturchin20/Documents/Work/StephensLab/HIVStuff/PG/.















##20151231
##Redoing X-chromosome analyses -- or all analyses using proper X-chromosome SNPs now?

#/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz
#AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.annotate.vs1.wACs.vcf.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz | grep ^X | perl -lane 'print join("\t", @F[0..9]);' | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz | grep ^X | perl -lane 'print join("\t", @F[0..9]);' | head -n 10

~~
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz | grep ^X | perl -lane 'print join("\t", @F[0..9]);' | wc
   1128   11280  380074
   [  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz | grep ^X | perl -lane 'print join("\t", @F[0..9]);' | head -n 10
   X       2821450 rs5982611       G       A       8092.19 VQSRTrancheSNP90.00to99.00      AC=127;AF=0.073;AN=1750;BaseQRankSum=-12.203;DB;DP=4370;Dels=0.00;FS=0.441;HaplotypeScore=0.4401;InbreedingCoeff=0.5212;MLEAC=87;MLEAF=0.050;MQ=59.99;MQ0=0;MQRankSum=1.622;POSITIVE_TRAIN_SITE;QD=25.05;ReadPosRankSum=-0.805;VQSLOD=3.61;culprit=DP;AA=g     GT:AD:DP:GQ:PL  0/0:4,0:4:12:0,12,141
   X       2821584 .       C       G       91.23   VQSRTrancheSNP99.00to99.90      AC=2;AF=1.129e-03;AN=1772;BaseQRankSum=-1.618;DP=5281;Dels=0.00;FS=0.000;HaplotypeScore=0.5452;InbreedingCoeff=0.0018;MLEAC=1;MLEAF=5.643e-04;MQ=59.99;MQ0=0;MQRankSum=-0.137;QD=18.25;ReadPosRankSum=2.389;VQSLOD=1.90;culprit=DP;AA=c        GT:AD:DP:GQ:PL  0/0:4,0:4:12:0,12,134
   X       2821590 rs148854344     T       C       1911.83 VQSRTrancheSNP90.00to99.00      AC=24;AF=0.014;AN=1774;BaseQRankSum=-0.570;DB;DP=5340;Dels=0.00;FS=2.564;HaplotypeScore=0.6757;InbreedingCoeff=0.2385;MLEAC=16;MLEAF=9.019e-03;MQ=59.99;MQ0=0;MQRankSum=-0.470;POSITIVE_TRAIN_SITE;QD=23.60;ReadPosRankSum=1.260;VQSLOD=4.23;culprit=DP;AA=t   GT:AD:DP:GQ:PL  0/0:5,0:5:15:0,15,164
   X       2821782 rs147988346     G       A       600.96  VQSRTrancheSNP90.00to99.00      AC=4;AF=2.240e-03;AN=1786;BaseQRankSum=6.544;DB;DP=6664;Dels=0.00;FS=2.933;HaplotypeScore=0.8430;InbreedingCoeff=0.0276;MLEAC=3;MLEAF=1.680e-03;MQ=60.00;MQ0=0;MQRankSum=0.252;POSITIVE_TRAIN_SITE;QD=23.11;ReadPosRankSum=1.023;VQSLOD=4.35;culprit=DP;AA=g   GT:AD:DP:GQ:PL  0/0:9,0:9:27:0,27,282
   X       2821848 .       G       A       67.15   VQSRTrancheSNP99.00to99.90      AC=3;AF=1.672e-03;AN=1794;BaseQRankSum=-1.159;DP=7332;Dels=0.00;FS=0.000;HaplotypeScore=1.8314;InbreedingCoeff=0.0389;MLEAC=1;MLEAF=5.574e-04;MQ=59.99;MQ0=0;MQRankSum=0.314;QD=6.10;ReadPosRankSum=0.297;VQSLOD=0.456;culprit=DP;AA=g GT:AD:DP:GQ:PL  0/0:11,0:11:33:0,33,383
   X       2821868 rs17335121      C       T       12337   VQSRTrancheSNP99.00to99.90      AC=136;AF=0.076;AN=1790;BaseQRankSum=10.182;DB;DP=7365;Dels=0.02;FS=7.042;HaplotypeScore=3.7518;InbreedingCoeff=0.7198;MLEAC=98;MLEAF=0.055;MQ=59.99;MQ0=0;MQRankSum=-0.365;POSITIVE_TRAIN_SITE;QD=21.53;ReadPosRankSum=1.776;VQSLOD=1.33;culprit=HaplotypeScore;AA=t  GT:AD:DP:GQ:PL  0/0:10,0:10:30:0,30,315
   X       2821978 rs61619902      A       C       52849.3 VQSRTrancheSNP90.00to99.00      AC=415;AF=0.231;AN=1796;BaseQRankSum=-10.620;DB;DP=8382;Dels=0.00;FS=3.000;HaplotypeScore=1.3924;InbreedingCoeff=0.8503;MLEAC=366;MLEAF=0.204;MQ=60.00;MQ0=0;MQRankSum=-0.412;POSITIVE_TRAIN_SITE;QD=27.90;ReadPosRankSum=-0.305;VQSLOD=3.39;culprit=DP;AA=c   GT:AD:DP:GQ:PL  0/0:11,0:11:33:0,33,372
   X       2821979 rs4141211       G       T       49114.1 VQSRTrancheSNP99.00to99.90      AC=385;AF=0.214;AN=1796;BaseQRankSum=34.472;DB;DP=8388;Dels=0.00;FS=7.039;HaplotypeScore=1.9785;InbreedingCoeff=0.8418;MLEAC=336;MLEAF=0.187;MQ=60.00;MQ0=0;MQRankSum=-1.683;POSITIVE_TRAIN_SITE;QD=28.77;ReadPosRankSum=-0.630;VQSLOD=1.60;culprit=HaplotypeScore;AA=g        GT:AD:DP:GQ:PL  0/0:11,0:11:33:0,33,368
   X       2822103 .       C       G       255.75  VQSRTrancheSNP99.00to99.90      AC=2;AF=1.112e-03;AN=1798;BaseQRankSum=2.030;DP=9765;Dels=0.00;FS=0.000;HaplotypeScore=0.6849;InbreedingCoeff=0.0995;MLEAC=1;MLEAF=5.562e-04;MQ=59.99;MQ0=0;MQRankSum=-0.184;QD=23.25;ReadPosRankSum=1.903;VQSLOD=1.90;culprit=FS;AA=c GT:AD:DP:GQ:PL  0/0:14,0:14:42:0,42,457
   X       2822253 rs9334  C       T       22885.5 VQSRTrancheSNP99.00to99.90      AC=137;AF=0.076;AN=1794;BaseQRankSum=1.268;DB;DP=9838;Dels=0.00;FS=1.362;HaplotypeScore=2.0943;InbreedingCoeff=0.8628;MLEAC=129;MLEAF=0.072;MQ=60.00;MQ0=0;MQRankSum=2.095;POSITIVE_TRAIN_SITE;QD=29.27;ReadPosRankSum=-1.029;VQSLOD=1.97;culprit=HaplotypeScore;AA=t  GT:AD:DP:GQ:PL  0/0:17,0:17:51:0,51,570
~~~~

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz | grep ^X | awk '{ print $1, "\t", $2}' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.XChr.ChrBPs

~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.XChr.ChrBPs | wc
   1128    2256   14665
~~

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz --positions /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.XChr.ChrBPs --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.recode.vcf

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.recode.vcf.gz | grep -v ^# | perl -lane 'my $hetCounts = 0; for (my $i=9; $i <= $#F; $i++) { my $altCounts = 0; my @indvInfo1 = split(/\:/, $F[$i]); my @genoInfo1 = split(/\//, $indvInfo1[0]); if ($genoInfo1[0] eq "1") { $altCounts++; } if ($genoInfo1[1] eq "1") { $altCounts++; } if ($altCounts eq "1") { $hetCounts++; } } print join("\t", @F[0..8]), "\t", $hetCounts; ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.perSNPhetCounts.txt

~~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOff
Targ_1kb.geno95.XChr.perSNPhetCounts.txt | awk '{ print $10 }' | sort | uniq -c | sort -g -k 2,2                                                                                                                               
    154 0                                                                                                                                                                                                                      
    146 1
    158 2
    148 3
    118 4
     83 5
     59 6
     46 7
     32 8
     22 9
     13 10
      8 11
     10 12
      6 13
      6 14
      8 15
      9 16
     11 17
      7 18
      7 19
      5 20
      6 21
      3 22
      4 23
      5 24
      4 25
      2 26
      7 27
      5 28
      2 29
      3 30
      4 31
      3 32
      1 33
      1 34
      2 35
      1 36
      2 37
      2 38
      1 39
      2 41
      2 45
      1 46
      1 51
      1 52
      1 53
      1 54
      1 56
      1 58
      1 67
      1 229
      1 383
~~~

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDs --positions /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.XChr.ChrBPs  --non-ref-ac 1 --recode  --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.recode.vcf

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.recode.vcf.gz | grep -v ^# | perl -lane 'my $hetCounts = 0; for (my $i=9; $i <= $#F; $i++) { my $altCounts = 0; my @indvInfo1 = split(/\:/, $F[$i]); my @genoInfo1 = split(/\//, $indvInfo1[0]); if ($genoInfo1[0] eq "1") { $altCounts++; } if ($genoInfo1[1] eq "1") { $altCounts++; } if ($altCounts eq "1") { $hetCounts++; } } print join("\t", @F[0..8]), "\t", $hetCounts; ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.perSNPhetCounts.txt

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.perSNPhetCounts.txt | awk '{ print $10 }' | sort | uniq -c | sort -g -k 2,2

~~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.perSNPhetCounts.txt | awk '{ print $10 }' | sort | uniq -c | sort -g -k 2,2
    126 0
    188 1
    187 2
    139 3
    117 4
     70 5
     45 6
     37 7
     17 8
     13 9
     22 10
     11 11
     14 12
      7 13
      6 14
      5 15
     10 16
      1 17
      1 18
      4 19
      3 20
      1 21
      2 22
      1 23
      2 24
      1 25
      1 26
      3 27
      1 29
      1 30
      1 34
      1 35
      2 38
      1 39
      1 40
      1 41
      1 47
      1 48
      1 200
      1 302
~~~

#So looks like maybe cutting off variants with het calls above 10 or something like that? Probably a nice balance of 'conservative but also not removing too many variants by doing so'


#Just looking at some basic SNP quality gradients, etc...
~~
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz | grep -v ^# | awk '{ print $6 }' | sort -rg | tail -n 10
50.06
50.06
50.04
50.04
50.03
50
50
50
50
50
~~~

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.recode.vcf.gz | grep -v ^# | perl -lane 'BEGIN { my %indvCounts; } my $hetCounts = 0; for (my $i=9; $i <= $#F; $i++) { my $altCounts = 0; my @indvInfo1 = split(/\:/, $F[$i]); my @genoInfo1 = split(/\//, $indvInfo1[0]); if ($genoInfo1[0] eq "1") { $altCounts++; } if ($genoInfo1[1] eq "1") { $altCounts++; } if ($altCounts eq "1") { $hetCounts++; if (!$indvCounts{$i}) { $indvCounts{$i} = 1; } else { $indvCounts{$i}++; } } } END { foreach my $indv1 (keys %indvCounts) { print $indv1, "\t", $indvCounts{$indv1};} } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.perIndvhetCounts.txt

~~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.perIndvhetCounts.txt | wc
    739    1478    4439
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.perIndvhetCounts.txt | head -n 10
559     4
32      10
127     9
118     9
71      4
443     11
206     5
358     5
331     7
560     2
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.perIndvhetCounts.txt | awk '{ print $2 }' | sort | uniq -c
     21 1
     37 10
     20 11
     10 12
      4 13
      7 14
      3 15
      3 16
      1 17
      1 18
     28 2
      1 21
      1 22
      1 23
      1 28
     65 3
      1 30
      2 32
      1 34
      1 35
     94 4
      1 47
    119 5
    108 6
      1 64
     89 7
     74 8
     44 9
~~~

#So looks like a few individuals have a 'larger than normal # of het calls' but 30-60 out of say ~734 is still a huge percentage. Doesn't seem to warrant considering how to not include one or two individuals at this stage and mesh it back up with the full autosomal analysis done earlier on.

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.perSNPhetCounts.txt | awk '{ if ($10 > 10) { print $1, "\t", $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.perSNPhetCounts.gt10Hets.ChrBPs

~~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.perSNPhetCounts.gt10Hets.ChrBPs | wc
     86     172    1122
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.perSNPhetCounts.txt | awk '{ print $10 }' | sort | uniq -c | sort -g -k 2,2 | awk '{ if ($2 > 10) { print $1 } } ' | paste -s -d+ - | bc       
86
~~~

#This matches the # of SNPs with > 10 het calls expected as from above

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.recode.vcf.gz --exclude-positions /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.perSNPhetCounts.gt10Hets.ChrBPs --recode  --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.recode.vcf

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2 & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroNeg.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroNeg & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroHighlyExposedNeg.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroHighlyExposedNeg & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroPos.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroPos & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justRapidVRapid.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justRapidVRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSlowVSlow.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSlowVSlow & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justVeryRapid.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVeryRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justVerySlow.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVerySlow & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2 & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroNeg & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroHighlyExposedNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroHighlyExposedNeg & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroPos.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroPos & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justRapidVRapid.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justRapidVRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSlowVSlow.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSlowVSlow & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justVeryRapid.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVeryRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justVerySlow.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVerySlow & 

paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVerySlow.frq.count | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.AllPhenos.frq.AndCount.gz
gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVerySlow.frq.count
#gunzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVerySlow.frq.count

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.recode.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.AllPhenos.frq.AndCount.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 30; $i <= 40; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.FreqInfo.AnnovarAnnotation.vs6.txt.gz

~~
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.FreqInfo.AnnovarAnnotation.vs6.txt.gz | wc
    995    1128  432856
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.recode.vcf.gz | grep -v \# | perl -lane 'print join("\t", @F[0..9]);' | wc
    961    9610  322461
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/_2/) { print join(",", @F); } ' | wc                                       
     34      34   14817                                                                                                                                                                                                        
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/_3/) { print join(",", @F); } ' | wc                                        
      0       0       0                                                                                                                                                                                                        
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/_4/) { print join(",", @F); } ' | wc                                        
      0       0       0
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/_5/) { print join(",", @F); } ' | wc
      0       0       0
~~~


vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.recode.vcf.gz --plink --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het

plink --file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs3.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.FreqInfo.AnnovarAnnotation.vs6.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.bim > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.bim.wGeneIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.bim.wGeneIDs | awk -F, '{ if ((($7 == "exonic") && (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV"))) || (($7 == "splicing") || ($7 == "exonic;splicing"))) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice 


~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.bim.wGeneIDs | grep -E 'nonsynonymous|stop|splicing' | wc
     74      74    7019
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'print $#F;'
3
3
6
4
4
12
2
6
4
5
3
4
2
3
8
4
1
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'print $#F;' | paste -s -d+ -
3+3+6+4+4+12+2+6+4+5+3+4+2+3+8+4+1
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'print $#F;' | paste -s -d+ - | bc
74
~~

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.recode.vcf.gz | awk '{ if ($1 == "23") { $1 = "X"} print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.recode.vcf.bgzip

/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.recode.vcf.bgzip
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.recode.vcf.bgzip.gz

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.recode.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.recode.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs

#ls -lrt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/. | awk '{ print $9 }' | perl -lane 'if ($F[0] =~ /AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9/) { my @vals1 = split(/EPACTS.RMID5PCs/, $F[0]); my $val1 = $vals1[0] . "EPACTS.RMID5PCs" . $vals1[1]; my $val2 = "AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het." . "EPACTS.RMID5PCs" . $F[1]; system("mv $val1 $val2"); }'

#20160104 NOTE -- Accidentally clobbered files when trying to do the 'mv' command above? It was done during a system-upgrade so maybe the command  failed midway and lost all the files? So writing some quick hack code to run the commands manually since I cannot submit jobs either at the moment
###

for i in {751..1000}
do
#	mainCommands1="--vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --beta 1,25 --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run${i}"
	mainCommands1="--vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.EPACTSedit.perm${i}.vs3.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --beta 1,25 --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm${i}"
#	bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.vs2.sh $mainCommands1 
	/home/michaelt/Software/EPACTS-3.2.4/bin/epacts group $mainCommands1
done

###

#20151230 NOTE -- So that I don't have to change my core code, I'm going to create /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous to run the analysis but then I'm going to move all files into /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/XChr once finished. If I need to zip/log/save files now in the ...XChr location I'll just mv * back to the ...TruePheno_XChr... location. Doing this because I don't think the ...TruePheno_XChr... organization is intuitive whereas ....Exonic.Nonsynonymous/XChr is. But I also don't want to actively edit or make a 'subfile' version of ...MultiRuns.MvAndProcessResults.vs3.sh (etc...) to suddenly begin maintaining as well

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno_XChr
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1/Exonic.Nonsynonymous

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs3.sh PP2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs TruePheno_XChr Pheno1 Exonic.Nonsynonymous

#20160119

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs3.sh PP2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs FakePheno_XChr Pheno1 Exonic.Nonsynonymous

#Substitution lines for moving the MM5 code over to here
~~~
/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.recode.NonOverlapWithMain.vcf.gz
/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2
AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1
AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs
AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.FreqInfo.AnnovarAnnotation.vs6.
/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.FreqInfo.AnnovarAnnotation.vs6.txt.gz

/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz
/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz
AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz
AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz

/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSeroHighlyExposedNeg.frq.count.gz
/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroNeg.frq.count
/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice.justGeneIDs
/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice

/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts
/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs
~~~

~~~
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | perl -lane 'print join("\t", @F[0..9]);' | tail -n 10
X       153061782       rs149689429     A       C       173.15  VQSRTrancheSNP90.00to99.00      AC=3;AF=1.680e-03;AN=1786;BaseQRankSum=-0.061;DB;DP=7104;Dels=0.00;FS=1.372;HaplotypeScore=0.8530;InbreedingCoeff=0.0310;MLEAC=1;MLEAF=5.599e-04;MQ=60.00;MQ0=0;MQRankSum=0.142;POSITIVE_TRAIN_SITE;QD=12.37;ReadPosRankSum=-0.528;VQSLOD=3.61;culprit=DP;AA=G  GT:AD:DP:GQ:PL  0/0:3,0:3:9:0,9,108
X       153061900       .       C       T       603.82  VQSRTrancheSNP99.00to99.90      AC=2;AF=1.117e-03;AN=1790;BaseQRankSum=2.866;DP=6876;Dels=0.00;FS=0.000;HaplotypeScore=1.0571;InbreedingCoeff=0.0260;MLEAC=2;MLEAF=1.117e-03;MQ=60.00;MQ0=0;MQRankSum=-0.837;QD=30.19;ReadPosRankSum=2.125;VQSLOD=1.64;culprit=DP;AA=C  GT:AD:DP:GQ:PL 0/0:5,0:5:15:0,15,172
X       153062095       .       A       T       202.34  VQSRTrancheSNP99.00to99.90      AC=3;AF=1.684e-03;AN=1782;BaseQRankSum=0.072;DP=6566;Dels=0.00;FS=6.907;HaplotypeScore=1.0904;InbreedingCoeff=0.0217;MLEAC=1;MLEAF=5.612e-04;MQ=59.99;MQ0=0;MQRankSum=0.142;QD=12.65;ReadPosRankSum=0.469;VQSLOD=1.39;culprit=DP;AA=A   GT:AD:DP:GQ:PL 0/0:7,0:7:21:0,21,240
X       153062120       .       C       T       140.26  VQSRTrancheSNP99.00to99.90      AC=3;AF=1.684e-03;AN=1782;BaseQRankSum=3.605;DP=6552;Dels=0.00;FS=0.000;HaplotypeScore=1.0142;InbreedingCoeff=0.0204;MLEAC=1;MLEAF=5.612e-04;MQ=59.99;MQ0=0;MQRankSum=-1.361;QD=10.79;ReadPosRankSum=-0.979;VQSLOD=1.50;culprit=DP;AA=C GT:AD:DP:GQ:PL 0/0:7,0:7:21:0,21,228
X       153062321       .       T       C       72.16   VQSRTrancheSNP99.00to99.90      AC=2;AF=1.126e-03;AN=1776;BaseQRankSum=1.979;DP=5763;Dels=0.00;FS=2.030;HaplotypeScore=3.4466;InbreedingCoeff=0.0058;MLEAC=1;MLEAF=5.631e-04;MQ=59.98;MQ0=0;MQRankSum=-0.998;QD=18.04;ReadPosRankSum=-1.836;VQSLOD=-7.100e-03;culprit=DP;AA=T   GT:AD:DP:GQ:PL  0/0:5,0:5:15:0,15,142
X       153062485       .       C       T       331.57  VQSRTrancheSNP99.00to99.90      AC=6;AF=3.375e-03;AN=1778;BaseQRankSum=-0.111;DP=6048;Dels=0.00;FS=1.943;HaplotypeScore=0.7010;InbreedingCoeff=0.0099;MLEAC=2;MLEAF=1.125e-03;MQ=59.96;MQ0=0;MQRankSum=-1.654;QD=8.09;ReadPosRankSum=0.313;VQSLOD=1.54;culprit=DP;AA=C  GT:AD:DP:GQ:PL 0/0:4,0:4:12:0,12,133
X       153062801       rs11795888      C       T       164.57  VQSRTrancheSNP90.00to99.00      AC=6;AF=3.386e-03;AN=1772;BaseQRankSum=4.858;DB;DP=5828;Dels=0.00;FS=1.058;HaplotypeScore=1.2315;InbreedingCoeff=0.0078;MLEAC=1;MLEAF=5.643e-04;MQ=59.99;MQ0=0;MQRankSum=-0.153;POSITIVE_TRAIN_SITE;QD=4.33;ReadPosRankSum=0.019;VQSLOD=3.11;culprit=QD;AA=C    GT:AD:DP:GQ:PL  0/0:5,0:5:15:0,15,164
X       153062844       .       G       A       128.12  VQSRTrancheSNP99.00to99.90      AC=2;AF=1.133e-03;AN=1766;BaseQRankSum=0.687;DP=5910;Dels=0.00;FS=3.253;HaplotypeScore=0.5298;InbreedingCoeff=-0.0268;MLEAC=1;MLEAF=5.663e-04;MQ=59.99;MQ0=0;MQRankSum=0.741;QD=6.41;ReadPosRankSum=-3.509;VQSLOD=-9.100e-03;culprit=DP;AA=G    GT:AD:DP:GQ:PL  0/0:6,0:6:18:0,18,206
X       153062846       .       G       A       73.15   VQSRTrancheSNP99.00to99.90      AC=7;AF=3.964e-03;AN=1766;BaseQRankSum=1.968;DP=5906;Dels=0.00;FS=1.089;HaplotypeScore=0.5647;InbreedingCoeff=-0.0281;MLEAC=1;MLEAF=5.663e-04;MQ=59.99;MQ0=0;MQRankSum=-0.161;QD=1.31;ReadPosRankSum=-2.505;VQSLOD=0.589;culprit=QD;AA=G        GT:AD:DP:GQ:PL  0/0:6,0:6:18:0,18,193
X       153062936       rs201630102     C       T       79.15   VQSRTrancheSNP99.00to99.90      AC=9;AF=5.114e-03;AN=1760;BaseQRankSum=3.236;DB;DP=5999;Dels=0.00;FS=0.000;HaplotypeScore=0.8918;InbreedingCoeff=0.0069;MLEAC=3;MLEAF=1.705e-03;MQ=60.00;MQ0=0;MQRankSum=0.230;QD=1.65;ReadPosRankSum=-0.853;VQSLOD=1.60;culprit=QD;AA=C       GT:AD:DP:GQ:PL   0/0:6,0:6:18:0,18,187
~~~



#20160217 CHECK_0 -- Problem: redo vcf concat by doing auto only from original main file and including all XChr from redone X vcf version



#zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^# | perl -lane 'print $F[0], "\t", $F[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.ChrBP.txt
#
#zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.recode.vcf.gz | grep -v ^# | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.ChrBP.txt | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.recode.NonOverlapWithMain.vcf.gz
#
#~~~
#[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.recode.vcf.gz | grep -v ^# | wc
#    961  726516 16072832
#[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.recode.NonOverlapWithMain.vcf.gz | wc
#    396  299376 6627476
#[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.recode.vcf.gz | grep -v ^# | awk '{ print $1, "\t", $2 }' | sort | uniq -c | head -n 10
#      1 1        11126202
#      1 1        11126324
#      1 1        11126399
#      1 1        11126439
#      1 1        11126569
#      1 1        11126672
#      1 1        11126697
#      1 1        11126709
#      1 1        11126840
#      1 1        11126848
#[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.recode.vcf.gz | grep -v ^# | awk '{ print $1, "\t", $2 }' | sort | uniq -c | awk '{ if ($1 > 1) { print $0 } } ' | wc
#    565    1695   11870
#[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.recode.NonOverlapWithMain.vcf.gz | grep -v ^# | awk '{ print $1, "\t", $2 }' | sort | uniq -c | awk '{ if ($1 > 1) { print $0 } } ' | wc
#      0       0       0
#[  michaelt@node4  ~]$R -q -e '396 + 565'
#> 396 + 565
#[1] 961
#>
#>
#~~~
#
##zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.recode.NonOverlapWithMain.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wProperXChr.vcf.gz
#zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.recode.NonOverlapWithMain.vcf.gz | /home/michaelt/Software/vcftools_0.1.11/bin/vcf-sort | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wProperXChr.vcf.gz
#
#~~~
#[  michaelt@node4  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wProperXChr.vcf.gz | wc
#  85722 64711981 1492048391
#[  michaelt@node4  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | wc
#  85326 64412605 1485420915
#[  michaelt@login2  ~]$R -q -e "85722 - 85326"
#> 85722 - 85326
#[1] 396
#>
#>
#~~~
#
#zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'print $F[0], ",", $F[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.ChrBP.txt
#
#zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.FreqInfo.AnnovarAnnotation.vs6.txt.gz | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.ChrBP.txt | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.FreqInfo.AnnovarAnnotation.vs6.NonOverlapWithMain.txt.gz
#
#zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.FreqInfo.AnnovarAnnotation.vs6.NonOverlapWithMain.txt.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.txt.gz
#
#~~~
#[  michaelt@node4  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz  | wc
#  87510   97379 38909004
#[  michaelt@node4  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz | wc
#  87510   97379 38909004
#[  michaelt@node4  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.FreqInfo.AnnovarAnnotation.vs6.NonOverlapWithMain.txt.gz | grep _1 | wc
#    396     451  180293
#[  michaelt@node4  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.FreqInfo.AnnovarAnnotation.vs6.NonOverlapWithMain.txt.gz | grep _2 | wc
#     16      16    7392
#[  michaelt@node4  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.FreqInfo.AnnovarAnnotation.vs6.NonOverlapWithMain.txt.gz | wc
#    412     467  187685
#[  michaelt@node4  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.txt.gz | wc
#  87922   97846 39096689
#[  michaelt@node4  ~]$R -q -e "87510 + 412"
#> 87510 + 412
#[1] 87922
#>
#>
#~~~

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wProperXChr.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wProperXChr.WRONG1.vcf.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.WRONG1.txt.gz

cat <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^X) <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.recode.vcf.gz | grep -v ^#) | /home/michaelt/Software/vcftools_0.1.11/bin/vcf-sort | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wProperXChr.vcf.gz 

cat <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz | grep -v ^X) <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.FreqInfo.AnnovarAnnotation.vs6.txt.gz) | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.txt.gz

~~~
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep ^X | perl -lane 'print join("\t", @F[0..9]);' | head -n 10
X       2821584 .       C       G       91.23   VQSRTrancheSNP99.00to99.90      AC=2;AF=1.129e-03;AN=1772;BaseQRankSum=-1.618;DP=5281;Dels=0.00;FS=0.000;HaplotypeScore=0.5452;InbreedingCoeff=0.0018;MLEAC=1;MLEAF=5.643e-04;MQ=59.99;MQ0=0;MQRankSum=-0.137;QD=18.25;ReadPosRankSum=2.389;VQSLOD=1.90;culprit=DP;AA=c GT:AD:DP:GQ:PL     0/0:4,0:4:12:0,12,134
X       2821782 rs147988346     G       A       600.96  VQSRTrancheSNP90.00to99.00      AC=4;AF=2.240e-03;AN=1786;BaseQRankSum=6.544;DB;DP=6664;Dels=0.00;FS=2.933;HaplotypeScore=0.8430;InbreedingCoeff=0.0276;MLEAC=3;MLEAF=1.680e-03;MQ=60.00;MQ0=0;MQRankSum=0.252;POSITIVE_TRAIN_SITE;QD=23.11;ReadPosRankSum=1.023;VQSLOD=4.35;culprit=DP;AA=g       GT:AD:DP:GQ:PL  0/0:9,0:9:27:0,27,282
X       2821848 .       G       A       67.15   VQSRTrancheSNP99.00to99.90      AC=3;AF=1.672e-03;AN=1794;BaseQRankSum=-1.159;DP=7332;Dels=0.00;FS=0.000;HaplotypeScore=1.8314;InbreedingCoeff=0.0389;MLEAC=1;MLEAF=5.574e-04;MQ=59.99;MQ0=0;MQRankSum=0.314;QD=6.10;ReadPosRankSum=0.297;VQSLOD=0.456;culprit=DP;AA=g  GT:AD:DP:GQ:PL     0/0:11,0:11:33:0,33,383
X       2822103 .       C       G       255.75  VQSRTrancheSNP99.00to99.90      AC=2;AF=1.112e-03;AN=1798;BaseQRankSum=2.030;DP=9765;Dels=0.00;FS=0.000;HaplotypeScore=0.6849;InbreedingCoeff=0.0995;MLEAC=1;MLEAF=5.562e-04;MQ=59.99;MQ0=0;MQRankSum=-0.184;QD=23.25;ReadPosRankSum=1.903;VQSLOD=1.90;culprit=FS;AA=c  GT:AD:DP:GQ:PL     0/0:14,0:14:42:0,42,457
X       2823133 .       G       A       704.43  VQSRTrancheSNP99.00to99.90      AC=4;AF=2.225e-03;AN=1798;BaseQRankSum=3.421;DP=10496;Dels=0.00;FS=0.000;HaplotypeScore=1.8012;InbreedingCoeff=0.1333;MLEAC=2;MLEAF=1.112e-03;MQ=60.00;MQ0=0;MQRankSum=-0.129;QD=16.77;ReadPosRankSum=1.018;VQSLOD=1.37;culprit=HaplotypeScore;AA=g        GT:AD:DP:GQ:PL  0/0:10,0:10:30:0,30,304
X       2824218 rs142005381     T       C       742.79  VQSRTrancheSNP99.00to99.90      AC=24;AF=0.013;AN=1798;BaseQRankSum=-11.574;DB;DP=9486;Dels=0.00;FS=33.597;HaplotypeScore=4.5387;InbreedingCoeff=0.0885;MLEAC=8;MLEAF=4.449e-03;MQ=58.78;MQ0=0;MQRankSum=3.221;QD=4.08;ReadPosRankSum=-0.668;VQSLOD=-1.334e+00;culprit=FS;AA=c     GT:AD:DP:GQ:PL  0/0:7,0:7:21:0,21,247
X       2825026 .       C       T       136.32  VQSRTrancheSNP99.00to99.90      AC=5;AF=2.800e-03;AN=1786;BaseQRankSum=0.525;DP=7516;Dels=0.00;FS=3.131;HaplotypeScore=0.9232;InbreedingCoeff=0.0419;MLEAC=1;MLEAF=5.599e-04;MQ=57.45;MQ0=0;MQRankSum=1.991;QD=4.70;ReadPosRankSum=-0.482;VQSLOD=1.09;culprit=QD;AA=c   GT:AD:DP:GQ:PL     0/0:16,0:16:48:0,48,542
X       2825083 rs191504623     G       A       376.85  VQSRTrancheSNP99.00to99.90      AC=15;AF=8.389e-03;AN=1788;BaseQRankSum=7.155;DB;DP=7466;Dels=0.01;FS=3.290;HaplotypeScore=1.7525;InbreedingCoeff=0.0228;MLEAC=3;MLEAF=1.678e-03;MQ=58.02;MQ0=0;MQRankSum=2.792;POSITIVE_TRAIN_SITE;QD=3.17;ReadPosRankSum=-0.376;VQSLOD=-1.840e-01;culprit=QD;AA=g        GT:AD:DP:GQ:PL  0/0:11,0:11:33:0,33,345
X       2825149 .       C       T       344.46  VQSRTrancheSNP99.00to99.90      AC=5;AF=2.800e-03;AN=1786;BaseQRankSum=-2.234;DP=7315;Dels=0.00;FS=0.917;HaplotypeScore=2.3573;InbreedingCoeff=0.0430;MLEAC=2;MLEAF=1.120e-03;MQ=58.31;MQ0=0;MQRankSum=2.171;QD=8.40;ReadPosRankSum=0.452;VQSLOD=-5.057e-01;culprit=DP;AA=c
        GT:AD:DP:GQ:PL  0/0:11,0:11:33:0,33,383
X       2825295 .       T       C       71.27   VQSRTrancheSNP99.00to99.90      AC=6;AF=3.382e-03;AN=1774;BaseQRankSum=-3.611;DP=5358;Dels=0.00;FS=0.000;HaplotypeScore=0.5341;InbreedingCoeff=-0.0346;MLEAC=1;MLEAF=5.637e-04;MQ=59.07;MQ0=0;MQRankSum=0.779;QD=1.98;ReadPosRankSum=-0.782;VQSLOD=1.62;culprit=QD;AA=t GT:AD:DP:GQ:PL     0/0:8,0:8:24:0,24,271
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep ^X | wc
    582  439992 9724949
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^X | wc
  84744 63972613 1475695966
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | wc
  85326 64412605 1485420915
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.recode.vcf.gz | grep -v ^# | wc
    961  726516 16072832
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.recode.vcf.gz | wc
   1087  727921 16099350
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.recode.vcf.gz | grep -v ^# | wc
    961  726516 16072832
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.recode.vcf.gz | grep ^X | wc
    961  726516 16072832
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$cat <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^X) <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.recode.vcf.gz | grep -v ^#) | wc
  85705 64699129 1491768798
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$R -q -e "84744 + 961"
> 84744 + 961
[1] 85705
> 
> 
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wProperXChr.vcf.gz | wc
  85705 64699129 1491768798
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz | wc
  87510   97379 38909004
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz | grep ^X | wc
    600     681  252946
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz | grep -v ^X | wc
  86910   96698 38656058
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.FreqInfo.AnnovarAnnotation.vs6.txt.gz | wc
    995    1128  432856
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$R -q -e "86910 + 995"
> 86910 + 995
[1] 87905
> 
> 
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.txt.gz | wc
  87905   97826 39088914
~~~







#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ print $1 }' | grep -v ^NS$ > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs
#
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.NonOverlapWithProperXChr
#
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.NonOverlapWithProperXChr <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w -v NS) > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr
#
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | tail -n +2 | awk '{ print $1, "\t", $2, "\t", $4 }' | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); print(cbind(Data1, signif(qvalue(Data1[,2])\$qvalues, 5)));" | head -n 10

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.WRONG1

cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep ^X | perl -lane 'print join("_", (split(/_/, $F[3]))[1..scalar((split(/_/, $F[3])))-1]);')) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w -v NS) > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | tail -n +2 | awk '{ print $1, "\t", $2, "\t", $4 }' | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); print(cbind(Data1, signif(qvalue(Data1[,2])\$qvalues, 5)));" | head -n 10

~~~
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | wc
    788   20483  109758
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs | wc
     17      17      94
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | wc
    786   20431  109641
#[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs | wc
#     15     390    2068
#[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs | wc
#    771   20041  107573
#[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | wc
#    788   20483  109758
#[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | tail -n +2 | awk '{ print $1, "\t", $2, "\t", $4 }' | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); print(cbind(Data1, signif(qvalue(Data1[,2])\$qvalues, 5)));" | head -n 10
#> library(qvalue); Data1 <- read.table(file('stdin'), header=F); print(cbind(Data1, signif(qvalue(Data1[,2])$qvalues, 5)));
#                 V1          V2      V3 signif(qvalue(Data1[, 2])$qvalues, 5)
#1              TAP1 0.763706900 0.90482                               0.90392
#2            ERRFI1 0.645490100 0.90482                               0.90392
#3            PPP3R2 0.251009700 0.87315                               0.87483
#4            PPAP2A 0.530352000 0.90482                               0.90392
#5             HSPA4 0.991395000 0.92567                               0.92510
#6             HSPA8 0.709080000 0.90482                               0.90392
#7             PCSK6 0.285120000 0.87315                               0.87483
#8              SCG2 0.642601100 0.90482                               0.90392
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep ^X | head -n 10
X       7177768 7268296 X:7177768-7268296_STS   747     0.0080321       4       4       2       0.89948 1
X       2825352 2843789 X:2825352-2843789_ARSD  747     0.0066934       2       2       0       0.63926 1
X       2873565 2878462 X:2873565-2878462_ARSE  747     0.01071 2       2       0       0.40521 1
X       3019164 3030346 X:3019164-3030346_ARSF  747     0.0040161       2       2       1       0.060549        1
X       2933110 2951253 X:2933110-2951253_ARSH  747     0.036145        5       5       0       0.22349 1
X       47442909        47445978        X:47442909-47445978_TIMP1       747     0.012048        2       2       0       0.49838 0
X       119565269       119581776       X:119565269-119581776_LAMP2     747     0.016064        4       4       0       0.91657 1
X       47486590        47486590        X:47486590-47486590_CFP 747     0.0053548       1       1       0       0.067585        NA
X       49047965        49055440        X:49047965-49055440_SYP 747     0.014726        4       4       1       0.63386 0
X       77373565        77381307        X:77373565-77381307_PGK1        747     0.0066934       2       2       0       1       1
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep ^X | wc
     16     176    1181
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep -v ^X | wc
    770    8470   59474
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep ^X | perl -lane 'print join("_", (split(/_/, $F[3]))[1..scalar((split(/_/, $F[3])))-1]);'
STS
ARSD
ARSE
ARSF
ARSH
TIMP1
LAMP2
CFP
SYP
PGK1
SYN1
CASK
COX7B
ITGB1BP2
SH3KBP1
IDS
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep ^X | perl -lane 'print join("_", (split(/_/, $F[3]))[1..scalar((split(/_/, $F[3])))-1]);' | wc
     16      16      86
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep ^X | perl -lane 'print scalar((split(/_/, $F[3])));'
2
2
2
2
2
2
.
.
.
[  michaelt@login2  ~]$cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep ^X | perl -lane 'print join("_", (split(/_/, $F[3]))[1..scalar((split(/_/, $F[3])))-1]);')) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w -v NS) | wc
    787   20457  109617
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | tail -n +2 | awk '{ print $1, "\t", $2, "\t", $4 }' | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); print(cbind(Data1, signif(qvalue(Data1[,2])\$qvalues, 5)));" | head -n 10
> library(qvalue); Data1 <- read.table(file('stdin'), header=F); print(cbind(Data1, signif(qvalue(Data1[,2])$qvalues, 5)));
                 V1          V2      V3 signif(qvalue(Data1[, 2])$qvalues, 5)
1              TAP1 0.763706900 0.90482                               0.90543
2            ERRFI1 0.645490100 0.90482                               0.90543
3            PPP3R2 0.251009700 0.87315                               0.87502
4            PPAP2A 0.530352000 0.90482                               0.90543
5             HSPA4 0.991395000 0.92567                               0.92647
6             HSPA8 0.709080000 0.90482                               0.90543
7             PCSK6 0.285120000 0.87315                               0.87502
8              SCG2 0.642601100 0.90482                               0.90543
~~~

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals.WRONG1

join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | tail -n +2 | sort ) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | tail -n +2 | awk '{ print $1, "\t", $2, "\t", $4 }' | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); print(cbind(Data1, signif(qvalue(Data1[,2])\$qvalues, 5)));" | awk '{ print $2, "\t", $5 }' | sort -k 1,1 ) | perl -lane 'splice(@F, 3, 1, $F[$#F]); splice(@F, $#F, 1); print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals

~~~
#[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | wc
#    787   20462  101888
#[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | wc
#    788   20483  109758
#[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | tail -n +2 | wc
#    787   20462  109534
#[  michaelt@node4  ~]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | tail -n +2 | awk '{ print $1, "\t", $4 }' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ print $1, "\t", $4 }') | wc
#    787    2361   17419
#[  michaelt@node4  ~]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | tail -n +2 | awk '{ print $1, "\t", $4 }' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ print $1, "\t", $4 }') | head -n 10
#AADAC 0.67037 0.67904
#ABAT 0.90482 0.90392
#ABCB1 0.90482 0.90392
#ABI1 0.87514 0.87497
#ACO1 0.89648 0.89820
#ACOT7 0.74457 0.75325
#ACOX1 0.90482 0.90392
#ACSL3 0.92567 0.92510
#ACTN2 0.74457 0.75325
#ADARB1 0.79772 0.79925
#[  michaelt@node4  ~]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | tail -n +2 | awk '{ print $1, "\t", $4 }' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ print $1, "\t", $4 }') | perl -lane 'my $val1 = $F[2] - $F[1]; if (abs($val1) > .01) { print join("\t", @F), "\t", $val1; }'
#ARSD    0.95698 0.87497 -0.08201
#ARSE    0.95698 0.87497 -0.08201
#ARSF    0.95698 0.67904 -0.27794
#ARSH    1.00000 0.90392 -0.0960800000000001
#CFP     0.95698 0.75722 -0.19976
#COL6A1  0.66481 0.67904 0.01423
#COX7B   0.95698 0.90392 -0.0530600000000001
#HS6ST2  0.95698 0.90392 -0.0530600000000001
#HTR2C   1.00000 0.92510 -0.0749
#IDS     0.95698 0.90392 -0.0530600000000001
#IKZF1   0.66481 0.67904 0.01423
#ITGB1BP2        1.00000 0.90605 -0.09395
#LAMP2   0.95698 0.90392 -0.0530600000000001
#LPAR1   0.66481 0.67904 0.01423
#LYZ     0.66546 0.67904 0.0135799999999999
#NAPSA   0.66481 0.67904 0.01423
#PGK1    1.00000 0.92510 -0.0749
#PTPN6   0.66546 0.67904 0.0135799999999999
#SH3KBP1 0.95698 0.87483 -0.0821500000000001
#SLC6A2  0.66481 0.67904 0.01423
#STS     1.00000 0.90867 -0.09133
#SYN1    0.95698 0.90392 -0.0530600000000001
#SYP     0.95698 0.90392 -0.0530600000000001
#TIMP1   0.95698 0.90392 -0.0530600000000001
#TMX1    0.66481 0.67904 0.01423
#[  michaelt@node4  ~]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | tail -n +2 | awk '{ print $1, "\t", $4 }' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ print $1, "\t", $4 }') | perl -lane 'my $val1 = $F[2] - $F[1]; if (abs($val1) > .01) { print join("\t", @F), "\t", $val1; }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs
#ARSD    0.95698 0.87497 -0.08201
#ARSE    0.95698 0.87497 -0.08201
#ARSF    0.95698 0.67904 -0.27794
#ARSH    1.00000 0.90392 -0.0960800000000001
#CFP     0.95698 0.75722 -0.19976
#COX7B   0.95698 0.90392 -0.0530600000000001
#HS6ST2  0.95698 0.90392 -0.0530600000000001
#HTR2C   1.00000 0.92510 -0.0749
#IDS     0.95698 0.90392 -0.0530600000000001
#ITGB1BP2        1.00000 0.90605 -0.09395
#LAMP2   0.95698 0.90392 -0.0530600000000001
#PGK1    1.00000 0.92510 -0.0749
#SH3KBP1 0.95698 0.87483 -0.0821500000000001
#STS     1.00000 0.90867 -0.09133
#SYN1    0.95698 0.90392 -0.0530600000000001
#SYP     0.95698 0.90392 -0.0530600000000001
#TIMP1   0.95698 0.90392 -0.0530600000000001
#[  michaelt@node4  ~]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | tail -n +2 | awk '{ print $1, "\t", $4 }' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ print $1, "\t", $4 }') | perl -lane 'my $val1 = $F[2] - $F[1]; if (abs($val1) > .01) { print join("\t", @F), "\t", $val1; }' | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs
#COL6A1  0.66481 0.67904 0.01423
#IKZF1   0.66481 0.67904 0.01423
#LPAR1   0.66481 0.67904 0.01423
#LYZ     0.66546 0.67904 0.0135799999999999
#NAPSA   0.66481 0.67904 0.01423
#PTPN6   0.66546 0.67904 0.0135799999999999
#SLC6A2  0.66481 0.67904 0.01423
#TMX1    0.66481 0.67904 0.01423
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | wc
    786   20436  101758
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | wc
    787   20457  109617
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | tail -n +2 | wc
    786   20436  109393
[  michaelt@login2  ~]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | tail -n +2 | awk '{ print $1, "\t", $4 }' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ print $1, "\t", $4 }') | wc
    786    2358   17398
[  michaelt@login2  ~]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | tail -n +2 | awk '{ print $1, "\t", $4 }' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ print $1, "\t", $4 }') | head -n 10
AADAC 0.67037 0.67919
ABAT 0.90482 0.90543
ABCB1 0.90482 0.90543
ABI1 0.87514 0.87516
ACO1 0.89648 0.89839
ACOT7 0.74457 0.75341
ACOX1 0.90482 0.90543
ACSL3 0.92567 0.92647
ACTN2 0.74457 0.75341
ADARB1 0.79772 0.79942
[  michaelt@login2  ~]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | tail -n +2 | awk '{ print $1, "\t", $4 }' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ print $1, "\t", $4 }') | perl -lane 'my $val1 = $F[2] - $F[1]; if (abs($val1) > .01) { print join("\t", @F), "\t", $val1; }'
ARSD    0.95698 0.87516 -0.08182
ARSE    0.95698 0.87516 -0.08182
ARSF    0.95698 0.67919 -0.27779
ARSH    1.00000 0.90543 -0.09457
CFP     0.95698 0.75738 -0.1996
COL6A1  0.66481 0.67919 0.0143799999999999
COX7B   0.95698 0.90543 -0.0515500000000001
HS6ST2  0.95698 0.90543 -0.0515500000000001
HTR2C   1.00000 0.92647 -0.07353
IDS     0.95698 0.90543 -0.0515500000000001
IKZF1   0.66481 0.67919 0.0143799999999999
ITGB1BP2        1.00000 0.90755 -0.09245
LAMP2   0.95698 0.90543 -0.0515500000000001
LPAR1   0.66481 0.67919 0.0143799999999999
LYZ     0.66546 0.67919 0.0137299999999999
NAPSA   0.66481 0.67919 0.0143799999999999
PGK1    1.00000 0.92647 -0.07353
PTPN6   0.66546 0.67919 0.0137299999999999
SH3KBP1 0.95698 0.87502 -0.08196
SLC6A2  0.66481 0.67919 0.0143799999999999
STS     1.00000 0.91012 -0.08988
SYN1    0.95698 0.90543 -0.0515500000000001
SYP     0.95698 0.90543 -0.0515500000000001
TIMP1   0.95698 0.90543 -0.0515500000000001
TMX1    0.66481 0.67919 0.0143799999999999
[  michaelt@login2  ~]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | tail -n +2 | awk '{ print $1, "\t", $4 }' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ print $1, "\t", $4 }') | perl -lane 'my $val1 = $F[2] - $F[1]; if (abs($val1) > .01) { print join("\t", @F), "\t", $val1; }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs
ARSD    0.95698 0.87516 -0.08182
ARSE    0.95698 0.87516 -0.08182
ARSF    0.95698 0.67919 -0.27779
ARSH    1.00000 0.90543 -0.09457
CFP     0.95698 0.75738 -0.1996
COX7B   0.95698 0.90543 -0.0515500000000001
HS6ST2  0.95698 0.90543 -0.0515500000000001
HTR2C   1.00000 0.92647 -0.07353
IDS     0.95698 0.90543 -0.0515500000000001
ITGB1BP2        1.00000 0.90755 -0.09245
LAMP2   0.95698 0.90543 -0.0515500000000001
PGK1    1.00000 0.92647 -0.07353
SH3KBP1 0.95698 0.87502 -0.08196
STS     1.00000 0.91012 -0.08988
SYN1    0.95698 0.90543 -0.0515500000000001
SYP     0.95698 0.90543 -0.0515500000000001
TIMP1   0.95698 0.90543 -0.0515500000000001
[  michaelt@login2  ~]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | tail -n +2 | awk '{ print $1, "\t", $4 }' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ print $1, "\t", $4 }') | perl -lane 'my $val1 = $F[2] - $F[1]; if (abs($val1) > .01) { print join("\t", @F), "\t", $val1; }' | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs
COL6A1  0.66481 0.67919 0.0143799999999999
IKZF1   0.66481 0.67919 0.0143799999999999
LPAR1   0.66481 0.67919 0.0143799999999999
LYZ     0.66546 0.67919 0.0137299999999999
NAPSA   0.66481 0.67919 0.0143799999999999
PTPN6   0.66546 0.67919 0.0137299999999999
SLC6A2  0.66481 0.67919 0.0143799999999999
TMX1    0.66481 0.67919 0.0143799999999999
~~~

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.WRONG1.txt.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.vcfannotate.vs3.wACs.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.vcfannotate.vs3.wACs.WRONG1.vcf.gz

cat <(echo "#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        PVAL    QVAL    REF     ALT     AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW") <(join <( python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.txt.gz | sort -k 1,1) <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wProperXChr.vcf.gz | grep -v ^\# | perl -lane 'print join("_", @F[0..1]), "\t", $F[3], "\t", $F[4];' | sort -k 1,1) | perl -lane '$F[1] = join("|", split(/[\;,\,]/, $F[1])); $F[2] = join("|", split(/[\;,\,]/, $F[2])); print join("\t", @F);' | perl -lane 'my @geneEntries = split(/\|/, $F[2]); my @pVals; my @qVals; foreach my $gene (@geneEntries) { $gene = ((split(/\(/, $gene))[0]); my $geneEntry = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | grep ^"$gene"[^-0-9A-Za-z_]`; if ($geneEntry) { my @geneVals1 = split(/\s+/, $geneEntry); push(@pVals, $geneVals1[1]); push(@qVals, $geneVals1[3]); } else { push(@pVals, "NA"); push(@qVals, "NA"); } } splice(@F, 5, 0, (join("|", @pVals), join("|", @qVals))); print join("\t", @F);' | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroHighlyExposedNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroPos.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justRapidVRapid.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSlowVSlow.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | perl -lane 'splice(@F, 0, 1, (((split(/_/, $F[0]))[0]), ((split(/_/, $F[0]))[1]), ((split(/_/, $F[0]))[1]))); print join("\t", @F);' |
perl -ane 'print join("\t", @F[0..10]); my $alt = $F[10]; my $totAC = join("", ((split(/:/, $F[11]))[1])) + join("", ((split(/:/, $F[12]))[1])); print "\t", $totAC; for (my $i=11; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", ((split(/\:/, $F[$i]))[1]); } } print "\n";' | sort -g -k 1,1 -k 2,2 ) | /home/shared/software/tabix-0.2.5/bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz
###perl -ane 'print join("\t", @F[0..10]); my $alt = $F[10]; my $totAC = join("", ((split(/:/, $F[11]))[1])) + join("", ((split(/:/, $F[12]))[1])); print "\t", $totAC; for (my $i=11; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", ((split(/\:/, $F[$i]))[1]); } } print "\n";' | sort -g -k 1,1 -k 2,2 ) | perl -lane 'splice(@F, 3, 0, ($F[9], $F[10])); splice(@F, 11, 2); print join("\t", @F);' | /home/shared/software/tabix-0.2.5/bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz

tabix -s 1 -b 2 -e 3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wProperXChr.vcf.gz | vcf-annotate -a /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz -d /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.PrepFor_vcfannotate.vs2.desc.txt -c CHROM,FROM,TO,INFO/LOC,INFO/GENE,INFO/VARTYPE,INFO/AACHANGE,INFO/PVAL,INFO/QVAL,REF,ALT,INFO/AC_ALL,INFO/AC_TOTAL,INFO/AC_SERONEG,INFO/AC_HESERONEG,INFO/AC_SEROPOS,INFO/AC_RAPID,INFO/AC_SLOW | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.vcfannotate.vs3.wACs.vcf.gz 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.P2.Vs2.annotated.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/OLD3.P2.AllPools.P2.Vs2.annotated.vcf.gz
ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.vcfannotate.vs3.wACs.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.Vs2.annotated.Update1.vcf.gz

#20160113 NOTE -- No differences between two files except for QVAL, which is the $F[8] column being removed via splice in the perl statements below. This then allows the cmp command to pass without any differences being found (whereas 'char 248, line 2' is quickly found as differing when the splice command isn't conducted, e.g. $F[8]/QVAL column remains intact)
~~~
[  michaelt@node4  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | grep -v ^X | wc
  84619 1523142 7354849
[  michaelt@node4  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | grep -v ^X | wc
  84619 1523142 7354849
[  michaelt@node4  ~]$cmp <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | grep -v ^X | perl -lane 'splice(@F, 8, 1); print join("\t", @F);') <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | grep -v ^X | perl -lane 'splice(@F, 8, 1); print join("\t", @F);')
[  michaelt@node4  ~]$
[  michaelt@node4  ~]$cmp <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | grep -v ^X) <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | grep -v ^X)
/dev/fd/63 /dev/fd/62 differ: char 248, line 2
~~~

#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs | sort | uniq > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.GeneIDList

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.GeneIDList /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.GeneIDList.WRONG1

cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep ^X | perl -lane 'print join("_", (split(/_/, $F[3]))[1..scalar((split(/_/, $F[3])))-1]);')) /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs | sort > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.GeneIDList

#Including the below code on 20160726
cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep ^X | perl -lane 'print join("_", (split(/_/, $F[3]))[1..scalar((split(/_/, $F[3])))-1]);')) /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | sort > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr

~~~
#[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.GeneIDList | wc
#    788     788    4830
#[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs | wc
#    786     786    4817
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | wc
     18      18      97
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs | wc
      2       2      13
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs
HS6ST2
HTR2C
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep ^X | perl -lane 'print join("_", (split(/_/, $F[3]))[1..scalar((split(/_/, $F[3])))-1]);') | wc
    770     770    4731
[  michaelt@login2  ~]$cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep ^X | perl -lane 'print join("_", (split(/_/, $F[3]))[1..scalar((split(/_/, $F[3])))-1]);')) /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs | wc
    787     787    4825
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | sort -g -k 2,2 | grep -v NUM_PASS_VARS | wc
    786   20436  109393
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | sort -g -k 2,2 | grep -v NUM_PASS_VARS | head -n 10
XAF1            0.001356961 2.615152e-04 0.39392        7       747     0.037483        7       7       1       0.0016809       1       747     0.037483        7       7       1       0.0014564       1       747     0.037483       7       7       1       0.0010727       1
PRDM1           0.001441239 3.511077e-04 0.39392        12      747     0.41633 12      12      3       0.0014301       1       747     0.41633 12      12      3       0.002057        1       747     0.41633 12      12    3
        0.0011171       1
PDXK            0.001957196 3.937028e-06 0.39392        3       747     0.0066934       3       3       1       0.001967        0.09    747     0.0066934       3       3       1       0.0019625       0.09    747     0.0066934      3       3       1       0.0019527       0.09
MAPK14          0.002168403 1.344193e-04 0.39392        3       747     0.0080321       3       3       2       0.0022884       1       747     0.0080321       3       3       2       0.0021681       1       747     0.0080321      3       3       2       0.0020264       1
NLRX1           0.003075420 4.997193e-04 0.44695        19      747     0.73762 19      19      8       0.0038118       0.04    747     0.73762 19      19      8       0.0025784       0.04    747     0.73762 19      19    8
        0.0033909       0.04
PRRC2A          0.005027917 6.694080e-04 0.51390        48      747     0.91968 48      48      15      0.0053474       1       747     0.91968 48      48      15      0.0049166       1       747     0.91968 48      48    15       0.0046948       0.5
CAPZB           0.005132807 3.361092e-04 0.51390        3       747     0.017403        3       3       0       0.0052729       0.04    747     0.017403        3       3       0       0.0047568       0.04    747     0.017403       3       3       0       0.0046395       0.01
C9orf69         0.008967305 4.244869e-04 0.51390        1       747     0.0093708       1       1       0       0.0088953       NA      747     0.0093708       1       1       0       0.0090003       NA      747     0.0093708      1       1       0       0.0090988       NA
PPAN            0.009379527 1.126217e-03 0.51390        13      747     0.084337        13      13      3       0.0083465       0.01    747     0.084337        13      13      3       0.0092775       0.04    747     0.084337       13      13      3       0.0094964       0.04
KLF6            0.010195799 4.530083e-04 0.51390        3       747     0.0053548       3       3       2       0.01038 1       747     0.0053548       3       3       2       0.0095274       0.25    747     0.0053548     3
        3       2       0.0098653       0.25
~~~

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | awk '{ print $1 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice.justGeneIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | awk '{ print $1 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice.justGeneIDs

#cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -v -w -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice.justGeneIDs) /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice.WRONG1

cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -v "X\:") /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice

~~~
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | wc
    785    6279   90853
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | wc
     17      91    1194
#[  michaelt@node4  ~]$cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -v -w -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice.justGeneIDs) /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice.justGeneIDs | wc
#    787    6238   90227
#[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -v -w -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice.justGeneIDs | wc
#    770    6221   90133
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice.justGeneIDs | grep -w -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice.justGeneIDs
HS6ST2
HTR2C
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -v "X\:" | wc
    769    6219   90113
[  michaelt@login2  ~]$cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -v "X\:") /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | wc
    786    6310   91307
~~~

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.Orig.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.Orig.WRONG1.txt

for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.GeneIDList`
do
	echo $gene1;
	paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | perl -lane 'if ($F[0] =~ m/^'$gene1'$/) { print join("\t", @F); }' | awk '{ print $1, "\t", $2, "\t", $4, "\t", $5, "\t", $8, "\t", $9, "\t", $11 }' ) \ 
	<(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | \
	grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'if ($F[0] =~ m/^'$gene1'$/) { print join("\t", @F); }' | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }') | \
	awk '{ print $12, "\t", $13, "\t", $14, "\t", $15, "\t", $16, "\t", $17, "\t", $18 }' | \
	R -q -e "Data1 <- read.table(file('stdin'), header=F); print(paste(apply(Data1, 2, sum)), quote=FALSE);" | grep -v ^\> | perl -lane 'splice(@F, 0, 1); print join("\t", @F);') \
	>> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.Orig.txt

done

~~~
.
.
.
NR4A2
NR4A3
NRCAM
NRP1
NS
Error in read.table(file("stdin"), header = F) :
  no lines available in input
Execution halted
NSMAF
NTF3
NTRK3
NUAK2
NUDT3
.
.
.
[  michaelt@node4  ~]$cmp <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt | sort -k 1,1 | grep -w -v NS | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs | perl -lane 'splice(@F, 2, 1); print join("\t", @F);') <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.Orig.txt | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs | perl -lane 'splice(@F, 2, 1); print join("\t", @F);')
[  michaelt@node4  ~]$
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.Orig.txt | perl -lane 'print $#F;' | sort | uniq -c
    786 13
      1 6
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.Orig.txt | perl -lane 'if ($#F == 6) { print join("\t", @F); }' 
CASK    0.621622500     0.90392 1       1       1       0.61995
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.WRONG1.txt.gz | grep CASK | grep exonic
X       41390299        41390299        exonic  CASK    synonymous_SNV  NM_001126055:c.C2394T:p.H798H   0.621622500     0.90392 G       A       1482    4       2       1       2       0       0
X       41469173        41469173        exonic  CASK    nonsynonymous_SNV       NM_001126055:c.T1121C:p.L374P   0.621622500     0.90392 A       G       1484    11      6       2       5       0       1
X       41469235        41469235        exonic  CASK    synonymous_SNV  NM_001126055:c.G1059A:p.A353A   0.621622500     0.90392 C       T       1484    6       2       1       4       1       1
X       41530742        41530742        exonic  CASK    synonymous_SNV  NM_001126054:c.T471C:p.P157P    0.621622500     0.90392 A       G       1480    2       2       0       0       0       0
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | grep CASK | grep exonic
X       41390299        41390299        exonic  CASK    synonymous_SNV  NM_001126055:c.C2394T:p.H798H   0.621622500     0.90392 G       A       1482    4       2       1       2       0       0
X       41469235        41469235        exonic  CASK    synonymous_SNV  NM_001126055:c.G1059A:p.A353A   0.621622500     0.90392 C       T       1484    6       2       1       4       1       1
X       41530742        41530742        exonic  CASK    synonymous_SNV  NM_001126054:c.T471C:p.P157P    0.621622500     0.90392 A       G       1480    2       2       0       0       0       0
~~~

#20160114 NOTE -- Excluding (redone) qvalues and the 11 XChr genes, the new updated P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.Orig.txt file and the original are the same (re: cmp statement above)
#20160217 NOTE -- CASK entry manually removed. CASK had a single non-synonymous variant that was excluded based upon heterozygous calls, so gene no longer part of results (hence difference in the above results).
#20160219 NOTE -- Updated /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.GeneIDList so CASK should not be included anymore at the above for loop where I then have to manually remove it. Ie updated /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr to be created using the 'remove all X from original file' + 'all X from new file' vs. old 'remove overlap from new file' + 'include nonoverlap from new to old' 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/OLD3.P2.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/OLD3.P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.Orig.txt | perl -lane 'splice(@F, 4, 3); print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt 

#Included the below header in file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt
#GENE   PVAL    QVAL    NUMVAR    AC_ALL        AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW

#QC Checks on file
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | wc
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.Orig.txt | perl -lane 'if (($F[3] == $F[4]) && ($F[3] == $F[5]) && ($F[4] == $F[5])) { print join("\t", @F); } '  | wc
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.Orig.txt | perl -lane 'if (($F[3] != $F[4]) || ($F[3] != $F[5]) || ($F[4] != $F[5])) { print join("\t", @F); } '
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[5] != $val2) { print join("\t", @F); }'
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[5] == $val2) { print join("\t", @F); }' | wc
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[7] > $F[6]) { print join("\t", @F); }'
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[7] <= $F[6]) { print join("\t", @F); }' | wc
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | perl -lane 'my $val2 = $F[9] + $F[10]; if ($F[8] < $val2) { print join("\t", @F); }'
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | perl -lane 'my $val2 = $F[9] + $F[10]; if ($F[8] >= $val2) { print join("\t", @F); }' | wc

~~
#[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | wc
#    788    8668   39564
#[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.Orig.txt | perl -lane 'if (($F[3] == $F[4]) && ($F[3] == $F[5]) && ($F[4] == $F[5])) { print join("\t", @F); } '  | wc
#    786   11004   48899
#[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.Orig.txt | perl -lane 'if (($F[3] != $F[4]) || ($F[3] != $F[5]) || ($F[4] != $F[5])) { print join("\t", @F); } '
#WNK1    0.418770600     0.88905 42      42      41      0.41574 62674   3926    1794    820     2132    290     422
#[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[5] != $val2) { print join("\t", @F); }'
#[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[5] == $val2) { print join("\t", @F); }' | wc
#    788    8668   39519
#[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[7] > $F[6]) { print join("\t", @F); }'
#[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[7] <= $F[6]) { print join("\t", @F); }' | wc
#    788    8668   39519
#[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | perl -lane 'my $val2 = $F[9] + $F[10]; if ($F[8] < $val2) { print join("\t", @F); }'
#[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | perl -lane 'my $val2 = $F[9] + $F[10]; if ($F[8] >= $val2) { print join("\t", @F); }' | wc
#    788    8668   39519
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | wc    787    8657   39519
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.Orig.txt | perl -lane 'if (($F[3] == $F[4]) && ($F[3] == $F[5]) && ($F[4] == $F[5])) { print join("\t", @F); } '  | wc
    785   10990   48842
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.Orig.txt | perl -lane 'if (($F[3] != $F[4]) || ($F[3] != $F[5]) || ($F[4] != $F[5])) { print join("\t", @F); } '
WNK1    0.418770600     0.88905 42      42      41      0.41574 62674   3926    1794    820     2132    290     422
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[5] != $val2) { print join("\t", @F); }'
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[5] == $val2) { print join("\t", @F); }' | wc
    787    8657   39474
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[7] > $F[6]) { print join("\t", @F); }'
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[7] <= $F[6]) { print join("\t", @F); }' | wc
    787    8657   39474
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | perl -lane 'my $val2 = $F[9] + $F[10]; if ($F[8] < $val2) { print join("\t", @F); }'
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | perl -lane 'my $val2 = $F[9] + $F[10]; if ($F[8] >= $val2) { print join("\t", @F); }' | wc
    787    8657   39474
~~

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/OLD3.P2.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz 

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.Vs2.annotated.Update1.vcf.gz | wc

#Difference between 141 and 142 due to '#CHROM...' header in /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz
~~~
[  michaelt@node4  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | wc
  85597 1540746 7436738
[  michaelt@node4  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | wc
  85597 1540746 7436738
[  michaelt@node4  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.Vs3.annotated.Update1.vcf.gz | wc
gzip: /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.Vs3.annotated.Update1.vcf.gz: No such file or directory
      0       0       0
[  michaelt@node4  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.Vs3.annotated.Update1.vcf.gz | wc
gzip: /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.Vs3.annotated.Update1.vcf.gz: No such file or directory
      0       0       0
[  michaelt@node4  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.Vs^Cannotated.Update1.vcf.gz | wc
[  michaelt@node4  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.Vs2.annotated.Update1.vcf.gz | wc
  85736 64712170 1504721514
[  michaelt@node4  ~]$R -q -e "85736 - 85597"
> 85736 - 85597
[1] 139
>
>
~~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/OLD3.P2.AllPools.P2.Vs2.annotated.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/OLD3.P2.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/OLD3.P2.skatO.nonsyn.qVals.ACs.GeneLevel.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/.

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.Vs2.annotated.Update1.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/.

#On MacBook Air
#scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.Vs2.annotated.Update1.vcf.gz mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt /Users/mturchin20/Documents/Work/StephensLab/HIVStuff/PG/.


mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1/Exonic.Nonsynonymous/* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/.










~~~
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | wc
    786    8646   60655
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -w -v ID) | wc
    771    8481   59549
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | wc
     18     198    1355
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList
ID
ARSD
ARSE
ARSF
HS6ST2
TIMP1
ARSH
PGK1
IDS
SYP
LAMP2
HTR2C
STS
ITGB1BP2
COX7B
SH3KBP1
CFP
SYN1
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -w -v ID | wc
     17      17      94
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | wc
     18      18      97
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs | wc
     17      17      94
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep -v \#CHROM | wc
     17     187    1254
[  michaelt@node4  ~]$cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -w -v ID)) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep -v \#CHROM) | wc
    788    8668   60803
[  michaelt@node4  ~]$cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -w -v ID)) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep -v \#CHROM) | head -n 10
#CHROM  BEGIN   END     MARKER_ID       NS      FRAC_WITH_RARE  NUM_ALL_VARS    NUM_PASS_VARS   NUM_SING_VARS   PVALUE  STATRHO
6       32814909        32821472        6:32814909-32821472_TAP1        747     0.31593 17      17      5       0.76116 0
1       8073379 8075753 1:8073379-8075753_ERRFI1        747     0.033467        6       6       5       0.6651  0
9       104356833       104357062       9:104356833-104357062_PPP3R2    747     0.014726        4       4       1       0.25439 0.5
5       54721062        54771245        5:54721062-54771245_PPAP2A      747     0.021419        5       5       0       0.5406  1
5       132387979       132439949       5:132387979-132439949_HSPA4     747     0.17001 15      15      4       1       0
11      122929407       122930839       11:122929407-122930839_HSPA8    747     0.29719 4       4       3       0.70908 0
15      101905203       101905203       15:101905203-101905203_PCSK6    747     0.0026774       1       1       0       0.28512 NA
2       224462224       224463115       2:224462224-224463115_SCG2      747     0.058902        5       5       1       0.64017 1
6       35543653        35610553        6:35543653-35610553_FKBP5       747     0.01071 3       3       0       0.736   1
[  michaelt@node4  ~]$cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -w -v ID)) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep -v \#CHROM) | awk '{ print $4 }' | sort | uniq -c | head -n 10
      1 10:112262561-112269908_DUSP5
      1 10:125769682-125805683_CHST15
      1 10:133786587-133786587_BNIP3
      1 10:24874423-24910265_ARHGAP21
      1 10:27040654-27057895_ABI1
      1 10:30727902-30749641_MAP3K8
      1 10:3143134-3178017_PFKP
      1 10:33196031-33218860_ITGB1
      1 10:33469276-33552771_NRP1
      1 10:35500673-35500673_CREM
[  michaelt@node4  ~]$cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -w -v ID)) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep -v \#CHROM) | awk '{ print $4 }' | sort | uniq -c | awk '{ if ($1 > 1) { print $0 } } '
[  michaelt@node4  ~]$
[  michaelt@node4  ~]$cmp <(cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -w -v ID)) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep -v \#CHROM) | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -w -v ID)) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -w -v ID))
[  michaelt@node4  ~]$

~~~

for i in {1..100}
do                                                                                                                                                                              cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run${i}.epacts | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -w -v ID)) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run${i}.epacts | grep -v \#CHROM | perl -lane 'push (@F, "XChr"); print join("\t", @F);') > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.AutoAndX.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run${i}.epacts
done

~~~
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.AutoAndX.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts  | grep XChr | wc
     17     204    1339
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.AutoAndX.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | tail -n 20
12      32893124        32893124        12:32893124-32893124_DNM1L      747     0.0013387       1       1       1       0.6691  NA
6       128304509       128718818       6:128304509-128718818_PTPRK     747     0.033467        7       7       1       0.82793 0
12      111884608       111886023       12:111884608-111886023_SH2B3    747     0.75502 6       6       2       0.90112 1
X       2825352 2843789 X:2825352-2843789_ARSD  747     0.01071 3       3       0       0.35283 0       XChr
X       2867651 2878462 X:2867651-2878462_ARSE  747     0.016064        3       3       0       0.36244 1       XChr
X       2999043 3030403 X:2999043-3030403_ARSF  747     0.17805 6       6       1       0.081585        1       XChr
X       131762541       132092485       X:131762541-132092485_HS6ST2    747     0.044177        4       4       0       0.48236 0       XChr
X       47442909        47446041        X:47442909-47446041_TIMP1       747     0.025435        4       4       0       0.5672  1       XChr
X       2933067 2951253 X:2933067-2951253_ARSH  747     0.15797 12      12      2       0.83114 1       XChr
X       77373565        77381307        X:77373565-77381307_PGK1        747     0.0066934       2       2       0       1       1       XChr
X       148568536       148582520       X:148568536-148582520_IDS       747     0.028112        6       6       1       0.5126  1       XChr
X       49047965        49055440        X:49047965-49055440_SYP 747     0.014726        4       4       1       0.63955 0       XChr
X       119565269       119581776       X:119565269-119581776_LAMP2     747     0.018742        5       5       0       0.55613 1       XChr
X       113965735       114141856       X:113965735-114141856_HTR2C     747     0.15395 3       3       0       1       1       XChr
X       7177768 7268296 X:7177768-7268296_STS   747     0.0080321       4       4       2       0.89882 1       XChr
X       70524420        70524936        X:70524420-70524936_ITGB1BP2    747     0.0053548       2       2       0       0.86122 0       XChr
X       77158195        77158206        X:77158195-77158206_COX7B       747     0.01071 3       3       0       0.67475 1       XChr
X       19555842        19854343        X:19555842-19854343_SH3KBP1     747     0.049531        8       8       0       0.24253 0.5     XChr
X       47486217        47486695        X:47486217-47486695_CFP 747     0.016064        4       4       0       0.12498 0       XChr
X       47432351        47432351        X:47432351-47432351_SYN1        747     0.0040161       1       1       0       0.59564 NA      XChr
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep  -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -w -v ID)
X       7177768 7268296 X:7177768-7268296_STS   747     0.0080321       4       4       2       0.89948 1
X       2825352 2843789 X:2825352-2843789_ARSD  747     0.0066934       2       2       0       0.63926 1
X       2873565 2878462 X:2873565-2878462_ARSE  747     0.01071 2       2       0       0.40521 1
X       3019164 3030346 X:3019164-3030346_ARSF  747     0.0040161       2       2       1       0.060549        1
X       2933110 2951253 X:2933110-2951253_ARSH  747     0.036145        5       5       0       0.22349 1
X       47442909        47445978        X:47442909-47445978_TIMP1       747     0.012048        2       2       0       0.49838 0
X       119565269       119581776       X:119565269-119581776_LAMP2     747     0.016064        4       4       0       0.91657 1
X       47486590        47486590        X:47486590-47486590_CFP 747     0.0053548       1       1       0       0.067585        NA
X       49047965        49055440        X:49047965-49055440_SYP 747     0.014726        4       4       1       0.63386 0
X       77373565        77381307        X:77373565-77381307_PGK1        747     0.0066934       2       2       0       1       1
X       47432351        47432351        X:47432351-47432351_SYN1        747     0.0040161       1       1       0       0.59276 NA
X       77158195        77158206        X:77158195-77158206_COX7B       747     0.01071 3       3       0       0.67837 1
X       70524420        70524936        X:70524420-70524936_ITGB1BP2    747     0.0053548       2       2       0       0.86268 0
X       19560130        19854343        X:19560130-19854343_SH3KBP1     747     0.022758        4       4       0       0.6636  0
X       148568536       148582520       X:148568536-148582520_IDS       747     0.018742        5       5       1       0.39746 1
~~~

for i in {1..1000}
do                                                                                                                                                                              cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm${i}.epacts | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList | grep -w -v ID)) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm${i}.epacts | grep -v \#CHROM | perl -lane 'push (@F, "XChr"); print join("\t", @F);') > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.AutoAndX.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm${i}.epacts
done

~~~
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.AutoAndX.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts | wc
    788    8685   60856
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.AutoAndX.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts | grep XChr | wc
     17     204    1357
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.AutoAndX.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts | tail -n 20
12      32893124        32893124        12:32893124-32893124_DNM1L      747     0.0013387       1       1       1       0.14014 NA
6       128304509       128718818       6:128304509-128718818_PTPRK     747     0.033467        7       7       1       0.82457 0
12      111884608       111886023       12:111884608-111886023_SH2B3    747     0.75502 6       6       2       0.17581 1
X       2825352 2843789 X:2825352-2843789_ARSD  747     0.01071 3       3       0       0.78459 1       XChr
X       2867651 2878462 X:2867651-2878462_ARSE  747     0.016064        3       3       0       0.043319        0       XChr
X       2999043 3030403 X:2999043-3030403_ARSF  747     0.17805 6       6       1       0.23843 0.5     XChr
X       131762541       132092485       X:131762541-132092485_HS6ST2    747     0.044177        4       4       0       0.82388 1       XChr
X       47442909        47446041        X:47442909-47446041_TIMP1       747     0.025435        4       4       0       0.79322 1       XChr
X       2933067 2951253 X:2933067-2951253_ARSH  747     0.15797 12      12      2       0.31268 0       XChr
X       77373565        77381307        X:77373565-77381307_PGK1        747     0.0066934       2       2       0       0.8358  1       XChr
X       148568536       148582520       X:148568536-148582520_IDS       747     0.028112        6       6       1       0.12472 1       XChr
X       49047965        49055440        X:49047965-49055440_SYP 747     0.014726        4       4       1       0.51067 1       XChr
X       119565269       119581776       X:119565269-119581776_LAMP2     747     0.018742        5       5       0       0.91921 1       XChr
X       113965735       114141856       X:113965735-114141856_HTR2C     747     0.15395 3       3       0       0.0056883       0.09    XChr
X       7177768 7268296 X:7177768-7268296_STS   747     0.0080321       4       4       2       0.13546 1       XChr
X       70524420        70524936        X:70524420-70524936_ITGB1BP2    747     0.0053548       2       2       0       0.59371 0       XChr
X       77158195        77158206        X:77158195-77158206_COX7B       747     0.01071 3       3       0       0.56651 1       XChr
X       19555842        19854343        X:19555842-19854343_SH3KBP1     747     0.049531        8       8       0       0.56246 1       XChr
X       47486217        47486695        X:47486217-47486695_CFP 747     0.016064        4       4       0       0.17961 1       XChr
X       47432351        47432351        X:47432351-47432351_SYN1        747     0.0040161       1       1       0       0.64466 NA      XChr
~~~

cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList | grep -w -v ID)) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice) > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice

~~~
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | wc
    785    6279   90853
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList | grep -w -v ID) | wc
    770    6221   90133
[  michaelt@node4  ~]$cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList | grep -w -v ID)) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice) | wc
    787    6312   91327
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep  -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList | grep -w -v ID) | perl -lane 'print "$F[0]\t$#F";' | sort -k 1,1
ARSD    2
ARSE    2
ARSF    2
ARSH    5
CFP     1
COX7B   3
IDS     5
ITGB1BP2        2
LAMP2   4
PGK1    2
SH3KBP1 4
STS     4
SYN1    1
SYP     4
TIMP1   2
~~~

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/SUMSTAT/ 
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1/Exonic.Nonsynonymous/SUMSTAT/ 

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs3.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.AutoAndX.EPACTS.RMID5PCs TruePheno_XChr Pheno1 Exonic.Nonsynonymous

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs3.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.AutoAndX.EPACTS.RMID5PCs FakePheno_XChr Pheno1 Exonic.Nonsynonymous

#20160121 NOTE -- The '.rnk' and 'SUMSTAT' files are failing because I am including an extra column, the 'XChr', in the appended genes to the original file. I can remove this or do something about this later if I need to redo the pathway analyses at this stage with the finalized data. At the moment I'm not going to change around /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs3.sh since I don't need these files. It may be more helpful to let the need drive how the changes should be made if they are eventually needed.




















#20160216 NOTE -- doing per qval threshold gene/numvar counting with updated XChr data now, also more fine-grain to help finalize follow-up genoptyping results

#echo ".5" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`

echo ".5" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ if ($4 <= .5) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ if ($4 <= .5) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'` 
echo ".6" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ if ($4 <= .6) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ if ($4 <= .6) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'` 
echo ".7" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ if ($4 <= .7) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ if ($4 <= .7) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'` 
echo ".8" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ if ($4 <= .8) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ if ($4 <= .8) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'` 
echo ".85" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ if ($4 <= .85) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ if ($4 <= .85) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'` 
echo ".88" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ if ($4 <= .88) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ if ($4 <= .88) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'` 
echo ".9" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ if ($4 <= .9) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ if ($4 <= .9) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'` 
echo "1" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ if ($4 <= 1) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ if ($4 <= 1) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'` 


echo ".3" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ if ($4 <= .3) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ if ($4 <= .3) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`



echo ".6" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".7" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".8" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".85" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".88" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .88) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .88) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".9" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .9) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .9) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo "1" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= 1) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= 1) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`

~~~
#[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$echo ".5" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.5 5 44

[  michaelt@login2  ~]$echo ".5" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ if ($4 <= .5) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ if ($4 <= .5) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.5 5 44
[  michaelt@login2  ~]$echo ".6" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ if ($4 <= .6) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ if ($4 <= .6) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.6 43 340
[  michaelt@login2  ~]$echo ".7" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ if ($4 <= .7) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ if ($4 <= .7) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.7 93 694
[  michaelt@login2  ~]$echo ".8" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ if ($4 <= .8) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ if ($4 <= .8) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.8 145 1078
[  michaelt@login2  ~]$echo ".85" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ if ($4 <= .85) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ if ($4 <= .85) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.85 183 1360
[  michaelt@login2  ~]$echo ".88" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ if ($4 <= .88) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ if ($4 <= .88) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.88 329 2396
[  michaelt@login2  ~]$echo ".9" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .9) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .9) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.9 356 2633
[  michaelt@login2  ~]$echo "1" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= 1) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= 1) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
1 787 5459
~~~

##20160219 OLD -- below are old results, prior to the 'remove old X Chr + new X Chr' approach vs. 'remove new X Chr overlap + old X Chr'
##.5 5 44
##.6 43 340
##.7 93 694
##.8 145 1078
##.85 183 1360
##.88 329 2396
##.9 356 2633
##1 787 5459

.5 5 44
.6 43 340
.7 93 694
.8 145 1078
.85 183 1360
.88 329 2396
.9 356 2633
1 787 5459









#20151230 CHECK_0 -- Prob: Create a ...MvAndProcessResults.vs4.sh version of file that includes the X-chr analysis grep/replace action?
#20151230 CHECK_0 -- Prob: Create a ...Vs4... set of files with the additional X-chr SNPs in .vcf format. Don't think I should need to redo the main analysis, just need to grep/replace the X-chr genes into the results file. Keep a copy of the old one with '_Autosomal' or something like that? Do this for the '.Routput' as well as 'noNAs' set files too (so can do perm analyses as well)
#20160111 NOTE -- I currently do not feel I should actually create a 'vs4' of these files, as of yet at least. For the moment I will now be creating files that extend the results from 'vs3', adding a suffix such as 'XChr' to the pre-existing files. I will leave the original files from 'vs3' untouched which means if I automatically use them down-the-line, I will in fact not be using the updated, additional XChr information. I will have to remember to use the file versions with the suffix I choose. However, I think for the moment this is ultimately better since it's unclear how much extra work I'll be doing on these files versus moving towards follow-up genotyping when that data becomes available. If in fact I don't really end up redoing the latest round of these analyses, recreating a new naming and organization scheme would not be a great investment. If, however, I end up having to redo these analyses extensively, I'll consider revamping the organization process at that point to then include a 'Vs4' that more handily/stream-wised deals with the XChr version of results
#20160113 CHECK_0 -- Prob: Go back and create now a ...Vs4... set of files interjecting the 'hetChr' step somewhere, figure out where. As a complement to hwe1e4 makes sense -- include in filename too? Check to see if indvs dropped based on 80% missingness and IBD params still be same? If so then yeah. 
#20160113 CHECK_0 -- Prob: Do on P2 as well.
#20160113 CHECK_0 -- Prob: Put corrected wVarCounts results final in analysis? Replace/Edit .epacts files to include new XChr from XChr files? So if run 'results collection' scripts get correct things then? Should be able to create a bash loop scrip from 1..100 or 1..1000 that can switch out/in the proper XChr gene information from the runs in this section into the correct filenames present in the analysis files then. Also cp -p orig results file and then add in new file?

##Files for Sudhir == priorty
##.vcf, annotated gene level, annotated variant level
#
##Subsampling (P2)
##ExAC/1000G (P2/P2)
##Intron (P2/P2)
##XChr (P2/P2)
##Coverage (P2/P2)

#20151228 CHECK_0 -- Prob: Do P2 for these too
#20151228 CHECK_0 -- Prob: Do Pheno2, Pheno3 & Pheno4 for these








##20151231
##Matthew p-val vs. coverage analysis

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.PerGene.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenes.GetVCFQCStats.PerGene.vs2.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenes.GetVCFQCStats.PerGene.vs2.output | awk -F, '{ print $1, "\t", $18, "\t", $2 }' | sort -gk 3,3 | vi -

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz | grep -E 'nonsynonymous|splicing' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.PerGene.vs2.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenes.Exonic.NonSyn_Splicing.GetVCFQCStats.PerGene.vs2.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenes.Exonic.NonSyn_Splicing.GetVCFQCStats.PerGene.vs2.output | awk -F, '{ print $1, "\t", $18, "\t", $2 }' | sort -gk 3,3 | vi -

#join w/ gene pvals

join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenes.Exonic.NonSyn_Splicing.GetVCFQCStats.PerGene.vs2.output | awk -F, '{ print $1, "\t", $18, "\t", $2 }' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ print $1, "\t", $2 }' | sort) | head -n 10
#join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenes.Exonic.NonSyn_Splicing.GetVCFQCStats.PerGene.vs2.output | awk -F, '{ print $1, "\t", $18, "\t", $2 }' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ print $1, "\t", $2 }' | sort) > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/MatthewPvalVsCoverage/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenes.Exonic.NonSyn_Splicing.PvalVsCoverage.txt

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/MatthewPvalVsCoverage

join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenes.Exonic.NonSyn_Splicing.GetVCFQCStats.PerGene.vs2.output | awk -F, '{ print $1, "\t", $18, "\t", $2 }' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ print $1, "\t", $2 }' | sort) | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); cor(Data1[,2], -log10(Data1[,4])); summary(lm(-log10(Data1[,4])~Data1[,2])); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/MatthewPvalVsCoverage/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenes.Exonic.NonSyn_Splicing.PvalVsCoverage.jpeg\", height=2000, width=4000, res=300); par(mfrow=c(1,2)); plot(Data1[,2], Data1[,4], main=\"P2 -- per gene Coverage vs. p-Value\", xlab=\"Coverage\", ylab=\"p-Value\"); abline(lm(Data1[,4]~Data1[,2]), col=\"RED\"); plot(Data1[,2], -log10(Data1[,4]), main=\"P2 -- per gene Coverage vs. -log10 p-Value\", xlab=\"Coverage\", ylab=\"-log10 p-Value\"); abline(lm(-log10(Data1[,4])~Data1[,2]), col=\"RED\"); dev.off();"

#On wolfy.uchicago.edu
#mkdir /Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/MatthewPvalVsCoverage/

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/MatthewPvalVsCoverage/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenes.Exonic.NonSyn_Splicing.PvalVsCoverage.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/MatthewPvalVsCoverage/

#20151230 20160111 CHECK_1 -- Prob: Do this analysis on P2 as well Soln: Did so

~~~
[  michaelt@node3  ~]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenes.Exonic.NonSyn_Splicing.GetVCFQCStats.PerGene.vs2.output | awk -F, '{ print $1, "\t", $18, "\t", $2 }' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ print $1, "\t", $2 }' | sort) | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); cor(Data1[,2], -log10(Data1[,4])); summary(lm(-log10(Data1[,4])~Data1[,2])); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/MatthewPvalVsCoverage/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenes.Exonic.NonSyn_Splicing.PvalVsCoverage.jpeg\", height=2000, width=4000, res=300); par(mfrow=c(1,2)); plot(Data1[,2], Data1[,4], main=\"P2 -- per gene Coverage vs. p-Value\", xlab=\"Coverage\", ylab=\"p-Value\"); abline(lm(Data1[,4]~Data1[,2]), col=\"RED\"); plot(Data1[,2], -log10(Data1[,4]), main=\"P2 -- per gene Coverage vs. -log10 p-Value\", xlab=\"Coverage\", ylab=\"-log10 p-Value\"); abline(lm(-log10(Data1[,4])~Data1[,2]), col=\"RED\"); dev.off();"
> Data1 <- read.table(file('stdin'), header=F); cor(Data1[,2], -log10(Data1[,4])); summary(lm(-log10(Data1[,4])~Data1[,2])); jpeg("/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/MatthewPvalVsCoverage/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenes.Exonic.NonSyn_Splicing.PvalVsCoverage.jpeg", height=2000, width=4000, res=300); par(mfrow=c(1,2)); plot(Data1[,2], Data1[,4], main="P2 -- per gene Coverage vs. p-Value", xlab="Coverage", ylab="p-Value"); abline(lm(Data1[,4]~Data1[,2]), col="RED"); plot(Data1[,2], -log10(Data1[,4]), main="P2 -- per gene Coverage vs. -log10 p-Value", xlab="Coverage", ylab="-log10 p-Value"); abline(lm(-log10(Data1[,4])~Data1[,2]), col="RED"); dev.off();
[1] 0.02546725

Call:
lm(formula = -log10(Data1[, 4]) ~ Data1[, 2])

Residuals:
    Min      1Q  Median      3Q     Max
-0.4899 -0.3341 -0.1735  0.1462  2.4245

Coefficients:
            Estimate Std. Error t value Pr(>|t|)
(Intercept) 0.417817   0.062046   6.734  3.2e-11 ***
Data1[, 2]  0.002899   0.004074   0.711    0.477
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Residual standard error: 0.4822 on 780 degrees of freedom
Multiple R-squared:  0.0006486, Adjusted R-squared:  -0.0006326
F-statistic: 0.5062 on 1 and 780 DF,  p-value: 0.477

null device
          1
>
>
~~~






##20160209
##Looking at genotype array prediction scores for later phone call with Sudhir

#join <(cat P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | perl -lane 'print $F[$#F];' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.1), right=FALSE));"
join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | perl -lane 'print $F[$#F];' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.1), right=FALSE));"

genotypeScoreThresh=".9"
#join <(cat P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 > '$genotypeScoreThresh') { print $3 } }' | head -n 10
join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 > '$genotypeScoreThresh') { print $3 } }' | head -n 10

join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ print $3 }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $12 }' | awk '{ if ($4 != $5) { print $0 } } '

join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 
) | grep -E "nonsynonymous|stop|splicing" | awk '{ print $3 }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $12 }' | awk '{ if ($4 != $5) { print $0 }
 } '

#cat P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt | grep \| | head -n 10 | perl -lane 'for (my $i = 0; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine = @F; $printLine[3] = (split(/\|/, $F[3]))[$i]; $printLine[4] = (split(/\|/, $F[4]))[$i]; $printLine[5] = (split(/\,/, $F[5]))[$i]; print join("\t", @printLine); }'
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | grep \| | head -n 10 | perl -lane 'for (my $i = 0; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine = @F; $printLine[3] = (split(/\|/, $F[3]))[$i]; $printLine[4] = (split(/\|/, $F[4]))[$i]; $printLine[5] = (split(/\,/, $F[5]))[$i]; print join("\t", @printLine); }'

join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ print $3 }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $12 }' | awk '{ if ($4 != $5) { print $0 } } '
join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ print $3 }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $12 }' | wc
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | wc
join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ print $3 }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $12 }' | wc

~~~
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | perl -lane 'print $F[$#F];' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.1), right=FALSE));"

> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.1), right=FALSE));

  [0,0.1) [0.1,0.2) [0.2,0.3) [0.3,0.4) [0.4,0.5) [0.5,0.6) [0.6,0.7) [0.7,0.8)
        0        11        35        39       128       224       417       703
[0.8,0.9)   [0.9,1)
      941      1754
>
>
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ print $3 }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $12 }' | awk '{ if ($4 != $5) { print $0 } } '
###ARSH     0.83681224      0.90392         12      13   #This is legacy output -- it was fixed by correcting out X-chromosome update was included with original data, eg by removing X from original data + new X versus removing overlap from new X and adjoining the remainder
DEDD     0.704006200     0.90392         3       2
GIMAP1-GIMAP5    0.307273400     0.87497         9       2
IPCEF1   0.775136900     0.90392         2       1
NIT1     0.884042400     0.90651         2       1
OPRM1    0.850462100     0.90392         27      26
SEPT8    0.404930200     0.88866         10      9
TNFSF13          0.535701800     0.90392         4       2
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | grep \| | head -n 10 | perl -lane 'for (my $i = 0; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine = @F; $printLine[3] = (split(/\|/, $F[3]))[$i]; $printLine[4] = (split(/\|/, $F[4]))[$i]; $printLine[5] = (split(/\,/, $F[5]))[$i]; print join("\t", @printLine); }'
X       7077556 7077556 intergenic      HDHD1(dist=11325)       .       .,.     NA|0.89950704   NA|0.90867      G       C       1442    4       1       1       3       0       0
X       7077556 7077556 intergenic      STS(dist=59916) .       .,.     NA|0.89950704   NA|0.90867      G       C       1442    4       1       1       3       0       0
X       7077704 7077704 intergenic      HDHD1(dist=11473)       .       .,.     NA|0.89950704   NA|0.90867      T       C       1466    5       3       0       2       0       0
X       7077704 7077704 intergenic      STS(dist=59768) .       .,.     NA|0.89950704   NA|0.90867      T       C       1466    5       3       0       2       0       0
X       7077964 7077964 intergenic      HDHD1(dist=11733)       .       .,.     NA|0.89950704   NA|0.90867      C       A       1466    2       0       0       2       0       0
X       7077964 7077964 intergenic      STS(dist=59508) .       .,.     NA|0.89950704   NA|0.90867      C       A       1466    2       0       0       2       0       0
X       7077965 7077965 intergenic      HDHD1(dist=11734)       .       .,.     NA|0.89950704   NA|0.90867      G       C       1466    4       0       0       4       2       0
X       7077965 7077965 intergenic      STS(dist=59507) .       .,.     NA|0.89950704   NA|0.90867      G       C       1466    4       0       0       4       2       0
X       7077972 7077972 intergenic      HDHD1(dist=11741)       .       .,.     NA|0.89950704   NA|0.90867      G       A       1464    14      2       2       12      2       0
X       7077972 7077972 intergenic      STS(dist=59500) .       .,.     NA|0.89950704   NA|0.90867      G       A       1464    14      2       2       12      2       0
X       7078058 7078058 intergenic      HDHD1(dist=11827)       .       .,.     NA|0.89950704   NA|0.90867      C       T       1458    11      7       5       4       0       2
X       7078058 7078058 intergenic      STS(dist=59414) .       .,.     NA|0.89950704   NA|0.90867      C       T       1458    11      7       5       4       0       2
X       47442048        47442048        intronic        SYN1    .       .,.     0.59692153|0.56854612   0.90392|0.90392 C       A       1472    100     53      28      47      9       2
X       47442048        47442048        intronic        TIMP1   .       .,.     0.59692153|0.56854612   0.90392|0.90392 C       A       1472    100     53      28      47      9       2
X       47443077        47443077        intronic        SYN1    .       .,.     0.59692153|0.56854612   0.90392|0.90392 C       T       1462    1       0       0       1       0       0
X       47443077        47443077        intronic        TIMP1   .       .,.     0.59692153|0.56854612   0.90392|0.90392 C       T       1462    1       0       0       1       0       0
X       47443107        47443107        intronic        SYN1    .       .,.     0.59692153|0.56854612   0.90392|0.90392 C       T       1460    2       2       2       0       0       0
X       47443107        47443107        intronic        TIMP1   .       .,.     0.59692153|0.56854612   0.90392|0.90392 C       T       1460    2       2       2       0       0       0
X       47443116        47443116        intronic        SYN1    .       .,.     0.59692153|0.56854612   0.90392|0.90392 T       C       1458    4       3       2       1       0       0
X       47443116        47443116        intronic        TIMP1   .       .,.     0.59692153|0.56854612   0.90392|0.90392 T       C       1458    4       3       2       1       0       0
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | grep \| | grep exonic | perl -lane 'for (my $i = 0; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine = @F; $printLine[3] = (split(/\|/, $F[3]))[$i]; $printLine[4] = (split(/\|/, $F[4]))[$i]; $printLine[5] = (split(/\,/, $F[5]))[$i]; print join("\t", @printLine); }' | head -n 10
1       161093935       161093935       exonic  DEDD    nonsynonymous_SNV       NM_001185092:c.T722C:p.V241A,NM_001185092:c.T722C:p.V241A       0.704006200|0.884042400 0.90392|0.90651 T       C       1494    4       2     1
        2       0       1
1       161093935       161093935       exonic  NIT1    nonsynonymous_SNV       NM_001185092:c.T722C:p.V241A,NM_001185092:c.T722C:p.V241A       0.704006200|0.884042400 0.90392|0.90651 T       C       1494    4       2     1
        2       0       1
3       52867353        52867353        exonic  MUSTN1  nonsynonymous_SNV       NM_205853:c.G236A:p.S79N,NM_205853:c.G236A:p.S79N       0.658050300|0.824432700 0.90392|0.90392 C       T       1494    6       1       0     5
        0       1
3       52867353        52867353        exonic  TMEM110-MUSTN1  nonsynonymous_SNV       NM_205853:c.G236A:p.S79N,NM_205853:c.G236A:p.S79N       0.658050300|0.824432700 0.90392|0.90392 C       T       1494    6       1     0
        5       0       1
3       52867404        52867404        exonic  MUSTN1  nonsynonymous_SNV       NM_205853:c.G185A:p.R62H,NM_205853:c.G185A:p.R62H       0.658050300|0.824432700 0.90392|0.90392 C       T       1490    4       1       0     3
        0       0
3       52867404        52867404        exonic  TMEM110-MUSTN1  nonsynonymous_SNV       NM_205853:c.G185A:p.R62H,NM_205853:c.G185A:p.R62H       0.658050300|0.824432700 0.90392|0.90392 C       T       1490    4       1     0
        3       0       0
3       52867411        52867411        exonic  MUSTN1  nonsynonymous_SNV       NM_205853:c.C178T:p.R60C,NM_205853:c.C178T:p.R60C       0.658050300|0.824432700 0.90392|0.90392 G       A       1490    4       2       1     2
        0       2
3       52867411        52867411        exonic  TMEM110-MUSTN1  nonsynonymous_SNV       NM_205853:c.C178T:p.R60C,NM_205853:c.C178T:p.R60C       0.658050300|0.824432700 0.90392|0.90392 G       A       1490    4       2     1
        2       0       2
3       52867715        52867715        exonic  MUSTN1  nonsynonymous_SNV       NM_205853:c.C60G:p.D20E,NM_205853:c.C60G:p.D20E 0.658050300|0.824432700 0.90392|0.90392 G       C       1484    1477    667     306     810   107      162
3       52867715        52867715        exonic  TMEM110-MUSTN1  nonsynonymous_SNV       NM_205853:c.C60G:p.D20E,NM_205853:c.C60G:p.D20E 0.658050300|0.824432700 0.90392|0.90392 G       C       1484    1477    667     306   810      107     162
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ print $3 }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $12 }' | awk '{ if ($4 != $5) { print $0 } } '
###ARSH     0.83681224      0.90392         12      13   #This is legacy output -- it was fixed by correcting out X-chromosome update was included with original data, eg by removing X from original data + new X versus removing overlap from new X and adjoining the remainder
ZBP1     0.803230900     0.90392         23      22
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | grep ZBP1 | grep -E "nonsynonymous|stop|splicing" | wc
     23     414    2811
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | grep ZBP1 | grep splicing
20      56188385        56188385        exonic|splicing ZBP1    synonymous_SNV  NM_001160418:c.A279G:p.E93E     0.803230900     0.90392 T       C       1480    5       3       2       2       0       2
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ print $3 }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $12 }' | wc
    786    3930   30332
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | wc
    788    8668   39564
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ print $3 }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $12 }' | wc
    769    3845   29622
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | grep -v -w -f <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ print $3 }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $12 }' | awk '{ print $1 }')
#GENE   PVAL    QVAL    NUMVAR    AC_ALL        AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW
PCSK6   0.285120000     0.87483 1       1494    2       0       0       2       0       0
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | grep PCSK6 | grep exonic
.
.
.
15      101905126       101905126       exonic  PCSK6   unknown UNKNOWN 0.285120000     0.87483 G       A       1494    5       1       1       4       0       1
15      101905170       101905170       exonic  PCSK6   unknown UNKNOWN 0.285120000     0.87483 C       T       1494    10      4       0       6       1       2
15      101905192       101905192       exonic  PCSK6   unknown UNKNOWN 0.285120000     0.87483 G       A       1494    1       1       0       0       0       0
15      101905203       101905203       exonic|splicing PCSK6   unknown UNKNOWN 0.285120000     0.87483 C       T       1494    2       0       0       2       0       0
15      101910550       101910550       exonic  PCSK6   unknown UNKNOWN 0.285120000     0.87483 G       A       1488    577     263     123     314     36      54
15      101910675       101910675       exonic  PCSK6   unknown UNKNOWN 0.285120000     0.87483 G       A       1478    6       4       2       2       1       0
15      101922312       101922312       exonic  PCSK6   unknown UNKNOWN 0.285120000     0.87483 G       A       1494    2       0       0       2       0       1
.
.
.
~

join -a 1 -a 2 -e 0 -o 0 1.2 1.3 1.4 1.5 1.6 1.7 1.8 1.9 1.10 1.11 2.2 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 >= .9) { print $3 } }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $12, "\t", $4-$12 }' | sort -k 1,1 | \
join -a 1 -a 2 -e NA -o 0 1.2 1.3 1.4 1.5 1.6 2.2 2.3 - <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 >= .8) { print $3 } }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | sort -k 1,1 | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | head -n 10

join -a 1 -a 2 -e 0 -o 0 1.2 1.3 1.4 1.5 1.6 1.7 1.8 1.9 1.10 1.11 2.2 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 == 1) { print $3 } }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $12, "\t", $4-$12 }' | sort -k 1,1 | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | \
join -a 1 -a 2 -e NA -o 0 1.2 1.3 1.4 1.5 1.6 2.2 2.3 - <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 >= .9) { print $3 } }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | sort -k 1,1 | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | \
join -a 1 -a 2 -e NA -o 0 1.2 1.3 1.4 1.5 1.6 1.7 1.8 2.2 2.3 - <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 >= .8) { print $3 } }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | sort -k 1,1 | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | \
join -a 1 -a 2 -e NA -o 0 1.2 1.3 1.4 1.5 1.6 1.7 1.8 1.9 1.10 2.2 2.3 - <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 >= .7) { print $3 } }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | sort -k 1,1 | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | \
join -a 1 -a 2 -e NA -o 0 1.2 1.3 1.4 1.5 1.6 1.7 1.8 1.9 1.10 1.11 1.12 2.2 2.3 - <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 >= .6) { print $3 } }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | sort -k 1,1 | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | \
join -a 1 -a 2 -e NA -o 0 1.2 1.3 1.4 1.5 1.6 1.7 1.8 1.9 1.10 1.11 1.12 1.13 1.14 2.2 2.3 - <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 >= .5) { print $3 } }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | sort -k 1,1 | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | \
join -a 1 -a 2 -e NA -o 0 1.2 1.3 1.4 1.5 1.6 1.7 1.8 1.9 1.10 1.11 1.12 1.13 1.14 1.15 1.16 2.2 2.3 - <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 >= .4) { print $3 } }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | sort -k 1,1 | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | \
join -a 1 -a 2 -e NA -o 0 1.2 1.3 1.4 1.5 1.6 1.7 1.8 1.9 1.10 1.11 1.12 1.13 1.14 1.15 1.16 1.17 1.18 2.2 2.3 - <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 >= .3) { print $3 } }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | sort -k 1,1 | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' |\
join -a 1 -a 2 -e NA -o 0 1.2 1.3 1.4 1.5 1.6 1.7 1.8 1.9 1.10 1.11 1.12 1.13 1.14 1.15 1.16 1.17 1.18 1.19 1.20 2.2 2.3 - <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 >= .2) { print $3 } }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | sort -k 1,1 | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | \
join -a 1 -a 2 -e NA -o 0 1.2 1.3 1.4 1.5 1.6 1.7 1.8 1.9 1.10 1.11 1.12 1.13 1.14 1.15 1.16 1.17 1.18 1.19 1.20 1.21 1.22 2.2 2.3 - <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 >= .1) { print $3 } }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | sort -k 1,1 | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | \
join -a 1 -a 2 -e NA -o 0 1.2 1.3 1.4 1.5 1.6 1.7 1.8 1.9 1.10 1.11 1.12 1.13 1.14 1.15 1.16 1.17 1.18 1.19 1.20 1.21 1.22 1.23 1.24 2.2 2.3 - <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ print $3 }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | sort -k 2,2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/WGGTPrelimScoreResults-P2.GeneAnnotations.Update1.txt
#join -a 1 -a 2 -e NA -o 0 1.2 1.3 1.4 1.5 1.6 1.7 1.8 1.9 1.10 1.11 1.12 1.13 1.14 1.15 1.16 1.17 1.18 1.19 1.20 1.21 1.22 1.23 1.24 2.2 2.3 - <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ print $3 }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | sort -k 2,2 | head -n 10

#Deleted following lines from /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/WGGTPrelimScoreResults-P2.GeneAnnotations.Update1.txt
#NM_001160418:c.A279G:p.E93E 0 0 0 1 -1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
#UNKNOWN 0 0 0 1 -1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
#GENE PVAL QVAL NUMVAR 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0

#Added the following header to /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/WGGTPrelimScoreResults-P2.GeneAnnotations.Update1.txt
#Gene pVal qVal Tot_numVar 1_numVar 1_diff .9_numVar .9_diff .8_numVar .8_diff .7_numVar .7_diff .6_numVar .6_diff .5_numVar .5_diff .4_numVar .4_diff .3_numVar .3_diff .2_numVar .2_diff .1_numVar .1_diff 0_numVar 0_diff

join <(cat P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(cat P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 >= .8) { print $3 } }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $12, "\t", $4-$12 }' | sort -k 2,2

R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); SummaryScoreThreshMissingSNPs <- c();
SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[1:25,6]), sum(Data1[1:25,8]), sum(Data1[1:25,10]), sum(Data1[1:25,12]), sum(Data1[1:25,14]), sum(Data1[1:25,16]), sum(Data1[1:25,18]), sum(Data1[1:25,20]), sum(Data1[1:25,22]), sum(Data1[1:25,24]), sum(Data1[1:25,26]), sum(Data1[1:25,25])));
SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[1:50,6]), sum(Data1[1:50,8]), sum(Data1[1:50,10]), sum(Data1[1:50,12]), sum(Data1[1:50,14]), sum(Data1[1:50,16]), sum(Data1[1:50,18]), sum(Data1[1:50,20]), sum(Data1[1:50,22]), sum(Data1[1:50,24]), sum(Data1[1:50,26]), sum(Data1[1:50,25])));
SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[1:100,6]), sum(Data1[1:100,8]), sum(Data1[1:100,10]), sum(Data1[1:100,12]), sum(Data1[1:100,14]), sum(Data1[1:100,16]), sum(Data1[1:100,18]), sum(Data1[1:100,20]), sum(Data1[1:100,22]), sum(Data1[1:100,24]), sum(Data1[1:100,26]), sum(Data1[1:100, 25])));
SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[1:200,6]), sum(Data1[1:200,8]), sum(Data1[1:200,10]), sum(Data1[1:200,12]), sum(Data1[1:200,14]), sum(Data1[1:200,16]), sum(Data1[1:200,18]), sum(Data1[1:200,20]), sum(Data1[1:200,22]), sum(Data1[1:200,24]), sum(Data1[1:200,26]), sum(Data1[1:200,25])));
SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[1:400,6]), sum(Data1[1:400,8]), sum(Data1[1:400,10]), sum(Data1[1:400,12]), sum(Data1[1:400,14]), sum(Data1[1:400,16]), sum(Data1[1:400,18]), sum(Data1[1:400,20]), sum(Data1[1:400,22]), sum(Data1[1:400,24]), sum(Data1[1:400,26]), sum(Data1[1:400,25])));
SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[,6]), sum(Data1[,8]), sum(Data1[,10]), sum(Data1[,12]), sum(Data1[,14]), sum(Data1[,16]), sum(Data1[,18]), sum(Data1[,20]), sum(Data1[,22]), sum(Data1[,24]), sum(Data1[,26]), paste("(", sum(Data1[,25]), ")", sep="")));
colnames(SummaryScoreThreshMissingSNPs) <- c(1, .9, .8, .7, .6 , .5, .4, .3, .2, .1, 0, \"(NumVarCheck)\"); rownames(SummaryScoreThreshMissingSNPs) <- c(\"Top 25\", \"Top 50\", \"Top 100\", \"Top 200\", \"Top 400\", \"All\");
print(SummaryScoreThreshMissingSNPs);"

R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); SummaryScoreThreshMissingSNPs <- c(); SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[1:25,6]), sum(Data1[1:25,8]), sum(Data1[1:25,10]), sum(Data1[1:25,12]), sum(Data1[1:25,14]), sum(Data1[1:25,16]), sum(Data1[1:25,18]), sum(Data1[1:25,20]), sum(Data1[1:25,22]), sum(Data1[1:25,24]), sum(Data1[1:25,26]), sum(Data1[1:25,25]))); SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[1:50,6]), sum(Data1[1:50,8]), sum(Data1[1:50,10]), sum(Data1[1:50,12]), sum(Data1[1:50,14]), sum(Data1[1:50,16]), sum(Data1[1:50,18]), sum(Data1[1:50,20]), sum(Data1[1:50,22]), sum(Data1[1:50,24]), sum(Data1[1:50,26]), sum(Data1[1:50,25]))); SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[1:100,6]), sum(Data1[1:100,8]), sum(Data1[1:100,10]), sum(Data1[1:100,12]), sum(Data1[1:100,14]), sum(Data1[1:100,16]), sum(Data1[1:100,18]), sum(Data1[1:100,20]), sum(Data1[1:100,22]), sum(Data1[1:100,24]), sum(Data1[1:100,26]), sum(Data1[1:100, 25]))); SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[1:200,6]), sum(Data1[1:200,8]), sum(Data1[1:200,10]), sum(Data1[1:200,12]), sum(Data1[1:200,14]), sum(Data1[1:200,16]), sum(Data1[1:200,18]), sum(Data1[1:200,20]), sum(Data1[1:200,22]), sum(Data1[1:200,24]), sum(Data1[1:200,26]), sum(Data1[1:200,25]))); SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[1:400,6]), sum(Data1[1:400,8]), sum(Data1[1:400,10]), sum(Data1[1:400,12]), sum(Data1[1:400,14]), sum(Data1[1:400,16]), sum(Data1[1:400,18]), sum(Data1[1:400,20]), sum(Data1[1:400,22]), sum(Data1[1:400,24]), sum(Data1[1:400,26]), sum(Data1[1:400,25]))); SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[,6]), sum(Data1[,8]), sum(Data1[,10]), sum(Data1[,12]), sum(Data1[,14]), sum(Data1[,16]), sum(Data1[,18]), sum(Data1[,20]), sum(Data1[,22]), sum(Data1[,24]), sum(Data1[,26]), sum(Data1[,25]))); colnames(SummaryScoreThreshMissingSNPs) <- c(1, .9, .8, .7, .6 , .5, .4, .3, .2, .1, 0, \"(NumVarCheck)\"); rownames(SummaryScoreThreshMissingSNPs) <- c(\"Top 25\", \"Top 50\", \"Top 100\", \"Top 200\", \"Top 400\", \"All\"); print(SummaryScoreThreshMissingSNPs);"

~
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$join -a 1 -a 2 -e 0 -o 0 1.2 1.3 1.4 1.5 1.6 1.7 1.8 1.9 1.10 1.11 2.2 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 >= .9) { print $3 } }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $12, "\t", $4-$12 }' | sort -k 1,1 | join -a 1 -a 2 -e NA -o 0 1.2 1.3 1.4 1.5 1.6 2.2 2.3 - <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 >= .8) { print $3 } }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | sort -k 1,1 | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | head -n 10
#GENE   PVAL    QVAL    NUMVAR  0       0       0       NUMVAR
AADAC   0.074287760     0.67904 7       4       3       7       0
ABAT    0.652072100     0.90392 4       3       1       4       0
ABCB1   0.545355600     0.90392 20      14      6       20      0
ABI1    0.387812000     0.87497 4       3       1       3       1
ACO1    0.427887900     0.89820 14      11      3       14      0
ACOT7   0.106564000     0.75325 3       2       1       2       1
ACOX1   0.837041800     0.90392 5       4       1       4       1
ACSL3   0.950070700     0.92510 9       8       1       9       0
ACTN2   0.102378180     0.75325 18      7       11      11      7
.
.
.
> join -a 1 -a 2 -e NA -o 0 1.2 1.3 1.4 1.5 1.6 1.7 1.8 1.9 1.10 1.11 1.12 1.13 1.14 1.15 1.16 1.17 1.18 1.19 1.20 1.21 1.22 2.2 2.3 - <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 >= .1) { print $3 } }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | sort -k 1,1 | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | \
> join -a 1 -a 2 -e NA -o 0 1.2 1.3 1.4 1.5 1.6 1.7 1.8 1.9 1.10 1.11 1.12 1.13 1.14 1.15 1.16 1.17 1.18 1.19 1.20 1.21 1.22 1.23 1.24 2.2 2.3 - <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ print $3 }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | sort -k 2,2 | head -n 10
NM_001160418:c.A279G:p.E93E     0       0       0       1       -1      0       0       0       0       0       0       0       0       0       0       0       0       0       0       0       0       0       0       0     0
UNKNOWN 0       0       0       1       -1      0       0       0       0       0       0       0       0       0       0       0       0       0       0       0       0       0       0       0       0
XAF1    0.001356961     0.39468 7       0       7       3       4       5       2       7       0       7       0       7       0       7       0       7       0       7       0       7       0       7       0
PRDM1   0.001441239     0.39468 12      5       7       11      1       11      1       12      0       12      0       12      0       12      0       12      0       12      0       12      0       12      0
PDXK    0.001957196     0.39468 3       1       2       1       2       2       1       3       0       3       0       3       0       3       0       3       0       3       0       3       0       3       0
MAPK14  0.002168403     0.39468 3       2       1       3       0       3       0       3       0       3       0       3       0       3       0       3       0       3       0       3       0       3       0
NLRX1   0.003075420     0.44781 19      0       19      4       15      8       11      16      3       18      1       19      0       19      0       19      0       19      0       19      0       19      0
PRRC2A  0.005027917     0.51489 48      1       47      12      36      22      26      29      19      38      10      43      5       48      0       48      0       48      0       48      0       48      0
CAPZB   0.005132807     0.51489 3       2       1       3       0       3       0       3       0       3       0       3       0       3       0       3       0       3       0       3       0       3       0
C9orf69 0.008967305     0.51489 1       0       1       0       1       0       1       0       1       0       1       1       0       1       0       1       0       1       0       1       0       1       0
.
.
.
MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults-P2.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ print $3 }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | sort -k 2,2 | grep -w NA
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$
.
.
.
> Data1 <- read.table(file('stdin'), header=FALSE); SummaryScoreThreshMissingSNPs <- c(); SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[1:25,6]), sum(Data1[1:25,8]), sum(Data1[1:25,10]), sum(Data1[1:25,12]), sum(Data1[1:25,14]), sum(Data1[1:25,16]), sum(Data1[1:25,18]), sum(Data1[1:25,20]), sum(Data1[1:25,22]), sum(Data1[1:25,24]), sum(Data1[1:25,26]), sum(Data1[1:25,25]))); SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[1:50,6]), sum(Data1[1:50,8]), sum(Data1[1:50,10]), sum(Data1[1:50,12]), sum(Data1[1:50,14]), sum(Data1[1:50,16]), sum(Data1[1:50,18]), sum(Data1[1:50,20]), sum(Data1[1:50,22]), sum(Data1[1:50,24]), sum(Data1[1:50,26]), sum(Data1[1:50,25]))); SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[1:100,6]), sum(Data1[1:100,8]), sum(Data1[1:100,10]), sum(Data1[1:100,12]), sum(Data1[1:100,14]), sum(Data1[1:100,16]), sum(Data1[1:100,18]), sum(Data1[1:100,20]), sum(Data1[1:100,22]), sum(Data1[1:100,24]), sum(Data1[1:100,26]), sum(Data1[1:100, 25]))); SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[1:200,6]), sum(Data1[1:200,8]), sum(Data1[1:200,10]), sum(Data1[1:200,12]), sum(Data1[1:200,14]), sum(Data1[1:200,16]), sum(Data1[1:200,18]), sum(Data1[1:200,20]), sum(Data1[1:200,22]), sum(Data1[1:200,24]), sum(Data1[1:200,26]), sum(Data1[1:200,25]))); SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[1:400,6]), sum(Data1[1:400,8]), sum(Data1[1:400,10]), sum(Data1[1:400,12]), sum(Data1[1:400,14]), sum(Data1[1:400,16]), sum(Data1[1:400,18]), sum(Data1[1:400,20]), sum(Data1[1:400,22]), sum(Data1[1:400,24]), sum(Data1[1:400,26]), sum(Data1[1:400,25]))); SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[,6]), sum(Data1[,8]), sum(Data1[,10]), sum(Data1[,12]), sum(Data1[,14]), sum(Data1[,16]), sum(Data1[,18]), sum(Data1[,20]), sum(Data1[,22]), sum(Data1[,24]), sum(Data1[,26]), sum(Data1[,25]))); colnames(SummaryScoreThreshMissingSNPs) <- c(1, .9, .8, .7, .6 , .5, .4, .3, .2, .1, 0, "(NumVarCheck)"); rownames(SummaryScoreThreshMissingSNPs) <- c("Top 25", "Top 50", "Top 100", "Top 200", "Top 400", "All"); print(SummaryScoreThreshMissingSNPs);
#           1  0.9  0.8 0.7 0.6 0.5 0.4 0.3 0.2 0.1 0 (NumVarCheck)
#Top 25   170  115   88  51  26   7   0   0   0   0 0           208
#Top 50   302  182  125  62  30   7   0   0   0   0 0           388
#Top 100  578  342  210 113  45  12   1   1   0   0 0           753
#Top 200 1154  692  428 223 107  49   9   4   1   0 0          1473
#Top 400 2258 1308  791 424 205  90  21  11   5   1 1          2900
#All     4313 2546 1593 876 450 218  88  48  13   1 1          5524
#> 
#> 
           1  0.9  0.8 0.7 0.6 0.5 0.4 0.3 0.2 0.1 0 (NumVarCheck)
Top 25   170  115   88  51  26   7   0   0   0   0 0           208
Top 50   302  182  125  62  30   7   0   0   0   0 0           388
Top 100  578  342  210 113  45  12   1   1   0   0 0           753
Top 200 1154  692  428 223 107  49   9   4   1   0 0          1473
Top 400 2258 1308  791 424 205  90  21  11   5   1 1          2900
All     4313 2547 1594 877 451 219  89  49  14   2 2          5522
>
>
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); sum(Data1[,4]);"
> Data1 <- read.table(file('stdin'), header=FALSE); sum(Data1[,4]);
[1] 5525
> 
> 
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence]$head -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/WGGTPrelimScoreResults-P2.GeneAnnotations.Update1.txt
#Gene pVal qVal Tot_numVar 1_numVar 1_diff .9_numVar .9_diff .8_numVar .8_diff .7_numVar .7_diff .6_numVar .6_diff .5_numVar .5_diff .4_numVar .4_diff .3_numVar .3_diff .2_numVar .2_diff .1_numVar .1_diff 0_numVar 0_diff
XAF1    0.001356961     0.39468 7       0       7       3       4       5       2       7       0       7       0       7       0       7       0       7       0       7       0       7       0       7       0
PRDM1   0.001441239     0.39468 12      5       7       11      1       11      1       12      0       12      0       12      0       12      0       12      0       12      0       12      0       12      0
PDXK    0.001957196     0.39468 3       1       2       1       2       2       1       3       0       3       0       3       0       3       0       3       0       3       0       3       0       3       0
MAPK14  0.002168403     0.39468 3       2       1       3       0       3       0       3       0       3       0       3       0       3       0       3       0       3       0       3       0       3       0
NLRX1   0.003075420     0.44781 19      0       19      4       15      8       11      16      3       18      1       19      0       19      0       19      0       19      0       19      0       19      0
PRRC2A  0.005027917     0.51489 48      1       47      12      36      22      26      29      19      38      10      43      5       48      0       48      0       48      0       48      0       48      0
CAPZB   0.005132807     0.51489 3       2       1       3       0       3       0       3       0       3       0       3       0       3       0       3       0       3       0       3       0       3       0
C9orf69 0.008967305     0.51489 1       0       1       0       1       0       1       0       1       0       1       1       0       1       0       1       0       1       0       1       0       1       0
PPAN    0.009379527     0.51489 13      1       12      2       11      2       11      10      3       11      2       13      0       13      0       13      0       13      0       13      0       13      0
~~~

#20160225
#Reran with updated X-chr files
#mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/WGGTPrelimScoreResults-P2.GeneAnnotations.Update1.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/OLD4.WGGTPrelimScoreResults-P2.GeneAnnotations.Update1.txt

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/WGGTPrelimScoreResults-P2.GeneAnnotations.Update1.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/.
#On MacBook Air
#scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/WGGTPrelimScoreResults-P2.GeneAnnotations.Update1.txt /Users/mturchin20/Documents/Work/StephensLab/HIVStuff/PG/. 








##20160725
##Prepping/exploring full data for expanding list of follow-up genoptying variants by going down the q-value sorted list of variants
##This includes downloading/annotating most recent ExAC data to get an idea for followup genoptying full list of variants for particular genes of interest
#20160725 NOTE -- Using the end tag of 'vs3' to indicate this is a step moving forward from previous analyzes/runs
#20160725 NOTE -- Getting "Input/output error"'s for a bunch of 'Vs2' files and pulling the fil from the backup. Need to contact Aleksandar about restoring the backups for any files that are now corrupt, if that is possible?

#20160725 20160725 CHECK_1 -- Prob: Backup main, source variant files from MM5 & P2 onto work comp if not there already? Soln: Did a few, may want to do more at another point?

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping 

##/backup/home/pg/michaelt/Data/ALL_MAPPING/.
#cp -p /backup/home/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz.20160725_BACKUP1
#cp -p /backup/home/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz.20160725_BACKUP1

#scp -p /backup/home/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/Backup/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/.
#scp -p /backup/home/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wProperXChr.vcf.gz mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/Backup/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/.
#scp -p /backup/home/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.vcfannotate.vs3.wACs.vcf.gz mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/Backup/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/.

#zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz.20160725_BACKUP1 | grep -v ^# | grep ^X | perl -lane 'BEGIN { my %indvCounts; } my $hetCounts = 0; for (my $i=9; $i <= $#F; $i++) { my $altCounts = 0; my @indvInfo1 = split(/\:/, $F[$i]); my @genoInfo1 = split(/\//, $indvInfo1[0]); if ($genoInfo1[0] eq "1") { $altCounts++; } if ($genoInfo1[1] eq "1") { $altCounts++; } if ($altCounts eq "1") { $hetCounts++; if (!$indvCounts{$i}) { $indvCounts{$i} = 1; } else { $indvCounts{$i}++; } } } END { foreach my $indv1 (keys %indvCounts) { print $indv1, "\t", $indvCounts{$indv1};} } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.vs3.perIndvhetCounts.txt
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz.20160725_BACKUP1 | grep -v ^# | grep ^X | perl -lane 'my $hetCounts = 0; for (my $i=9; $i <= $#F; $i++) { my $altCounts = 0; my @indvInfo1 = split(/\:/, $F[$i]); my @genoInfo1 = split(/\//, $indvInfo1[0]); if ($genoInfo1[0] eq "1") { $altCounts++; } if ($genoInfo1[1] eq "1") { $altCounts++; } if ($altCounts eq "1") { $hetCounts++; } } print join("\t", @F[0..8]), "\t", $hetCounts; ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.vs3.perSNPhetCounts.txt

#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.vs3.perIndvhetCounts.txt | awk '{ print $2 }' | sort | uniq -c | sort -g -k 2,2
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.vs3.perSNPhetCounts.txt | awk '{ print $10 }' | sort | uniq -c | sort -g -k 2,2

~~~
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz.20160725_BACKUP1 | wc
  98789 91659505 2097997764
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz.20160725_BACKUP1 | grep -v ^X | wc
  97661 90611593 2074997530
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz.20160725_BACKUP1 | grep ^X | wc
   1128 1047912 23000234
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$R -q -e "98789 - 97661"
> 98789 - 97661
[1] 1128
> 
> 
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.perIndvhetCounts.txt | awk '{ print $2 }' | sort | uniq -c | sort -g -k 2,2
     21 1
     28 2
     65 3
     94 4
    119 5
    108 6
     89 7
     74 8
     44 9
     37 10
     20 11
     10 12
      4 13
      7 14
      3 15
      3 16
      1 17
      1 18
      1 21
      1 22
      1 23
      1 28
      1 30
      2 32
      1 34
      1 35
      1 47
      1 64
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs
2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.vs3.perIndvhetCounts.txt | awk '{ print $2 }' | sort | uniq -c | sort -g -k 2,2
     24 1
     34 2
     78 3
    110 4
    138 5
    128 6
    102 7
     86 8
     51 9
     42 10
     22 11
     13 12
      5 13
     10 14
      3 15
      3 16
      2 17
      1 18
      2 19
      1 21
      1 22
      1 23
      1 27
      2 28
      1 29
      1 30
      2 32
      1 34
      1 35
      1 37
      3 38
      1 39
      2 42
      2 43
      1 44
      1 46
      4 47
      1 49
      1 50
      1 52
      2 53
      1 55
      1 56
      1 57
      1 58
      2 59
      1 61
      1 64
      1 73
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs2.perSNPhetCounts.txt | awk '{ print $10 }' | sort | uniq -c | sort -g -k 2,2
    126 0
    188 1
    187 2
    139 3
    117 4
     70 5
     45 6
     37 7
     17 8
     13 9
     22 10
     11 11
     14 12
      7 13
      6 14
      5 15
     10 16
      1 17
      1 18
      4 19
      3 20
      1 21
      2 22
      1 23
      2 24
      1 25
      1 26
      3 27
      1 29
      1 30
      1 34
      1 35
      2 38
      1 39
      1 40
      1 41
      1 47
      1 48
      1 200
      1 302
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs
2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.vs3.perSNPhetCounts.txt | awk '{ print $10 }' | sort | uniq -c | sort -g -k 2,2
    154 0
    146 1
    158 2
    148 3
    118 4
     83 5
     59 6
     46 7
     32 8
     22 9
     13 10
      8 11
     10 12
      6 13
      6 14
      8 15
      9 16
     11 17
      7 18
      7 19
      5 20
      6 21
      3 22
      4 23
      5 24
      4 25
      2 26
      7 27
      5 28
      2 29
      3 30
      4 31
      3 32
      1 33
      1 34
      2 35
      1 36
      2 37
      2 38
      1 39
      2 41
      2 45
      1 46
      1 51
      1 52
      1 53
      1 54
      1 56
      1 58
      1 67
      1 229
      1 383
[  michaelt@node3  ~]$vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz.20160725_BACKUP1 --chr X --exclude-positions /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.vs3.perSNPhetCounts.gt10Hets.ChrBPs --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.vs3.gthet10
.
.
.
Keeping 1128 entries on specified chromosomes.
Applying Required Filters.
Filtering sites by include/exclude positions files
After filtering, kept 920 out of 920 Individuals
After filtering, kept 979 out of a possible 1128 Sites
Outputting VCF file... 
.
.
.
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.vs3.perSNPhetCounts.gt10Hets.ChrBPs | wc
    149     298    1923
[  michaelt@node3  ~]$R -q -e "1128 - 149"
> 1128 - 149
[1] 979
> 
> 
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz.20160725_BACKUP1 | wc
  95015 88153459 2022066686
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz.20160725_BACKUP1 | grep -v ^X | wc
  94385 87568189 2009230741
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz.20160725_BACKUP1 | grep ^X | wc
    630  585270 12835945
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz.20160725_BACKUP1 | wc
  95015 88153459 2022066686
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz.20160725_BACKUP1 | grep -v ^X | wc
  94385 87568189 2009230741
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz.20160725_BACKUP1 | grep ^X | wc
    630  585270 12835945
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.vcf.gz | wc
  95364 88477680 2029202427
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.vcf.gz | grep -v ^X | wc
  94385 87568189 2009230741
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.vcf.gz | grep ^X | wc
    979  909491 19971686
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat.genome_summary.csv | wc
  95239  106108 7631359
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat.exome_summary.csv | wc
  11030   21899 1668931
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat.genome_summary.csv | perl -F, -lane 'print $F[$#F-4], "\t", $F[$#F-3];' | uniq -c | wc
  95239  285717 1871084
[  michaelt@node3  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.vcf.gz | grep -v ^# | wc
  95238 88476102 2029173020





~~~

#20160726 NOTE -- probably going to do gt10 again since you can see that jump once again as like before (see the above two examples, the first one is the results from before)

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.vs3.perSNPhetCounts.txt | awk '{ if ($10 > 10) { print $1, "\t", $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.vs3.perSNPhetCounts.gt10Hets.ChrBPs

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz.20160725_BACKUP1 --chr X --exclude-positions /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.vs3.perSNPhetCounts.gt10Hets.ChrBPs --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.vs3.gthet10 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.vs3.gthet10.recode.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.vs3.gthet10.vcf

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.vs3.gthet10.vcf

cat <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz.20160725_BACKUP1 | grep -v ^X ) <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.vs3.gthet10.vcf.gz | grep -v ^#) | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.vcf.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.vcf.gz | grep -v ^# | awk '{ print $1, "\t", $2, "\t", $2, "\t", $4, "\t", $5 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat

perl /home/michaelt/Software/annovar/summarize_annovar.pl /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat /home/michaelt/Software/annovar/ --ver1000g 1000g2012apr --verdbsnp 129 --veresp 6500 --buildver hg19 --remove

~~~
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'print join("\t", @F[1..$#F]);' | perl -ane 'foreach my $variant1 (@F) { print $variant1, "\n"; }' | sed 's/_/ /g' | sed 's/:/_/' | awk '{ print $1 }' | wc
   5524    5524   64389
#20160726 NOTE -- 5524 (5523 + 1 extra space at end) matches expected total # of nonsynonymous SNPs found in the dataset/used for analysis originally)
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat.genome_summary.csv | perl -F, -lane 'print $F[0];' | sort | uniq -c
  13075 "UTR3"
   2164 "UTR5"
     23 "UTR5;UTR3"
   3798 "downstream"
  10946 "exonic"
      4 "exonic;splicing"
   1089 "intergenic"
  60378 "intronic"
     94 "ncRNA_UTR3"
     33 "ncRNA_UTR5"
      1 "ncRNA_UTR5;ncRNA_UTR3"
    513 "ncRNA_exonic"
    994 "ncRNA_intronic"
     79 "splicing"
   1907 "upstream"
    140 "upstream;downstream"
      1 Func
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat.genome_summary.csv | perl -F, -lane 'print $F[$#F-23];' | sort | uniq -c
  84288
   5832 "nonsynonymous SNV"
    127 "stopgain SNV"
     11 "stoploss SNV"
   4899 "synonymous SNV"
     81 "unknown"
      1 AAChange
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /home/michaelt/Data/ExAC/0.3.1/ExAC.r0.3.1.sites.vep.vcf.AnnovarFormat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat | awk '{ print $1, "\t", $2 }' | wc
8262755 16525510 113002431
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /home/michaelt/Data/ExAC/0.3.1/ExAC.r0.3.1.sites.vep.vcf.AnnovarFormat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat | awk '{ print $1, "\t", $2 }' | sort | uniq | wc
8247300 16494600 112791415
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /home/michaelt/Data/ExAC/0.3.1/ExAC.r0.3.1.sites.vep.vcf.AnnovarFormat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat | awk '{ print $1, "\t", $2 }' | sort | uniq -d | wc
  15455   30910  211016
[  michaelt@node3  ~]$cat /home/michaelt/Data/ExAC/0.3.1/ExAC.r0.3.1.sites.vep.vcf.AnnovarFormat.genome_summary.csv | wc
8167518 12927577 951417204
[  michaelt@node3  ~]$cat /home/michaelt/Data/ExAC/0.3.1/ExAC.r0.3.1.sites.vep.vcf.AnnovarFormat.UniqueFromHIV_P2.genome_summary.vs3.csv | wc
8152062 8152062 941458373
[  michaelt@node12  ~]$join -j 1 -e NA1 -o 0 2.2 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat.genome_summary.csv | sed 's/ /_/g' | perl -F, -lane 'print $F[$#F-4], "_", $F[$#F-3], "\t", join(",", @F);' | sort -k 1,1) <(cat /home/michaelt/Data/ExAC/0.3.1/ExAC.r0.3.1.sites.vep.vcf.AnnovarFormat.genome_summary.csv | sed 's/ /_/g' | perl -F, -lane 'print $F[$#F-4], "_", $F[$#F-3], "\t", join(",", @F);' | sort -k 1,1) | awk '{ print $2 }' | wc
  15456   15456 1791314
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$R -q -e "8167518 - 8152062"
> 8167518 - 8152062
[1] 15456
> 
> 
[  michaelt@node3  ~]$cat /home/michaelt/Data/ExAC/0.3.1/ExAC.r0.3.1.sites.vep.vcf.AnnovarFormat.UniqueFromHIV_P2.genome_summary.vs3.csv | perl -F, -lane 'print $F[$#F-23];' | sort | uniq -c
3347961 
3064737 "nonsynonymous_SNV"
  87339 "stopgain_SNV"
   7751 "stoploss_SNV"
1592102 "synonymous_SNV"
  52172 "unknown"
[  michaelt@node3  ~]$cat /home/michaelt/Data/ExAC/0.3.1/ExAC.r0.3.1.sites.vep.vcf.AnnovarFormat.UniqueFromHIV_P2.genome_summary.vs3.csv | grep -E 'exonic|splicing|intronic|UTR5|UTR3' | perl -F, -lane 'print $F[$#F-23];' | > sort | uniq -c
3188073 
3064737 "nonsynonymous_SNV"
  87339 "stopgain_SNV"
   7751 "stoploss_SNV"
1592102 "synonymous_SNV"
  52172 "unknown"
[  michaelt@node3  ~]$cat /home/michaelt/Data/ExAC/0.3.1/ExAC.r0.3.1.sites.vep.vcf.AnnovarFormat.UniqueFromHIV_P2.genome_summary.vs3.csv | grep -E 'nonsynonymous|splicing|stopgain|stoploss' | wc
3209393 3209393 515252921
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /home/michaelt/Data/ExAC/0.3.1/ExAC.r0.3.1.sites.vep.vcf.AnnovarFormat.UniqueFromHIV_P2.genome_summary.vs3.csv | grep -E 'nonsynonymous|splicing|stopgain|stoploss' | grep -v ncRNA | wc
3208710 3208710 515169559
[  michaelt@node3  ~]$cat /home/michaelt/Data/ExAC/0.3.1/ExAC.r0.3.1.sites.vep.vcf.AnnovarFormat.UniqueFromHIV_P2.genome_summary.vs3.csv | grep -E 'nonsynonymous|splicing|stopgain|stoploss' | perl -F, -lane 'print $F[0];' | sort | uniq -c
3159138 "exonic"
   1013 "exonic;splicing"
    683 "ncRNA_splicing"
  48559 "splicing"
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /home/michaelt/Data/ExAC/0.3.1/ExAC.r0.3.1.sites.vep.vcf.AnnovarFormat.UniqueFromHIV_P2.genome_summary.vs3.csv | grep -E 'nonsynonymous|splicing|stopgain|stoploss' | grep -v ncRNA | perl -F, -lane 'print $F[0];' | sort | uniq -c
3159138 "exonic"
   1013 "exonic;splicing"
  48559 "splicing"
[  michaelt@node3  ~]$cat /home/michaelt/Data/ExAC/0.3.1/ExAC.r0.3.1.sites.vep.vcf.AnnovarFormat.UniqueFromHIV_P2.genome_summary.vs3.csv | grep -E 'nonsynonymous|splicing|stopgain|stoploss' | grep -v ncRNA | perl -F, -lane 'print $F[$#F-23];' | sort | uniq -c
  48561 
3064737 "nonsynonymous_SNV"
  87339 "stopgain_SNV"
   7751 "stoploss_SNV"
    270 "synonymous_SNV"
     52 "unknown"
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'print join("\t", @F[1..$#F]);' | perl -ane 'foreach my $variant1 (@F) { print $variant1, "\n"; }' | sed 's/_/ /g' | sed 's/:/_/' | awk '{ print $1 }' | sort | wc
   5524    5524   64389
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat.genome_summary.csv | sed 's/ /_/g' | perl -F, -lane 'print $F[$#F-4], "_", $F[$#F-3], "\t", join(",", @F);' | sort -k 1,1 | wc
  95239  190478 8645293
[  michaelt@node12  ~]$cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'print join("\t", @F[1..$#F]);' | perl -ane 'foreach my $variant1 (@F) { print $variant1, "\n"; }' | sed 's/_/ /g' | sed 's/:/_/' | awk '{ print $1 }') <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat | awk '{ print $1 "_" $2 }') | sort | wc
 100762  100762 1173551
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$R -q -e "95239 + 5524"
> 95239 + 5524
[1] 100763
> 
> 
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'print join("\t", @F[1..$#F]);' | perl -ane 'foreach my $variant1 (@F) { print $variant1, "\n"; }' | sed 's/_/ /g' | sed 's/:/_/' | awk '{ print $1 }' | sort | uniq | wc
   5458    5458   63615
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'print join("\t", @F[1..$#F]);' | perl -ane 'foreach my $variant1 (@F) { print $variant1, "\n"; }' | sed 's/_/ /g' | sed 's/:/_/' | awk '{ print $1 }') <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat | awk '{ print $1 "_" $2 }') | perl -lane 'print $#F;' | sort | uniq -c
 100762 0
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'print join("\t", @F[1..$#F]);' | perl -ane 'foreach my $variant1 (@F) { print $variant1, "\n"; }' | sed 's/_/ /g' | sed 's/:/_/' | awk '{ print $1 }') <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat | awk '{ print $1 "_" $2 }') | sort | uniq | wc
  95241   95241 1109192
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'print join("\t", @F[1..$#F]);' | perl -ane 'foreach my $variant1 (@F) { print $variant1, "\n"; }' | sed 's/_/ /g' | sed 's/:/_/' | awk '{ print $1 }') <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat | awk '{ print $1 "_" $2 }') | sort | uniq -d | wc
   5455    5455   63585
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$R -q -e "95241 - 5455"
> 95241 - 5455
[1] 89786
> 
> 
[  michaelt@node3  ~]$join -j 1 -v 2 -e NA1 -o 0 2.2 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'print join("\t", @F[1..$#F]);' | perl -ane 'foreach my $variant1 (@F) { print $variant1, "\n"; }' | sed 's/_/ /g' | sed 's/:/_/' | awk '{ print $1 }' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat.genome_summary.csv | sed 's/ /_/g' | perl -F, -lane 'print $F[$#F-4], "_", $F[$#F-3], "\t", join(",", @F);' | sort -k 1,1) | awk '{ print $2 }'  | wc
  89784   89784 6617254
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$join -j 1 -v 1 -e NA1 -o 0 1.1 2.2 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'print join("\t", @F[1..$#F]);' | perl -ane 'foreach my $variant1 (@F) { print $variant1, "\n"; }' | sed 's/_/ /g' | sed 's/:/_/' | awk '{ print $1 }' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat.genome_summary.csv | sed 's/ /_/g' | perl -F, -lane 'print $F[$#F-4], "_", $F[$#F-3], "\t", join(",", @F);' | sort -k 1,1) | awk '{ print $2 }' 
X_2945477
X_3002475
X_3030403
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat.UniqueFromHIV_P2NonSyn.genome_summary.csv | wc
  89784   89784 6617254
~~~

#20160721 NOTE -- Looks like 'upstream'/'downstream' is within 1kb, did some manual inspection via UCSC genome browser

cat /home/michaelt/Data/ExAC/0.3.1/ExAC.r0.3.1.sites.vep.vcf.AnnovarFormat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat | awk '{ print $1, "\t", $2 }' | wc
cat /home/michaelt/Data/ExAC/0.3.1/ExAC.r0.3.1.sites.vep.vcf.AnnovarFormat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat | awk '{ print $1, "\t", $2 }' | sort | uniq | wc
cat /home/michaelt/Data/ExAC/0.3.1/ExAC.r0.3.1.sites.vep.vcf.AnnovarFormat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat | awk '{ print $1, "\t", $2 }' | sort | uniq -d | wc

##join -a 2 -e NA1 -o 0 2.2 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat.genome_summary.csv | sed 's/ /_/g' | perl -F, -lane 'print $F[$#F-4], "_", $F[$#F-3], "\t", join(",", @F);' | sort -g -k 1,1) <(cat /home/michaelt/Data/ExAC/0.3.1/ExAC.r0.3.1.sites.vep.vcf.AnnovarFormat.genome_summary.csv | sed 's/ /_/g' | perl -F, -lane 'print $F[$#F-4], "_", $F[$#F-3], "\t", join(",", @F);' | sort -g -k 1,1) | awk '{ print $2 }' > /home/michaelt/Data/ExAC/0.3.1/ExAC.r0.3.1.sites.vep.vcf.AnnovarFormat.UniqueFromHIV_P2vs4.genome_summary.csv
join -j 1 -v 2 -e NA1 -o 0 2.2 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat.genome_summary.csv | sed 's/ /_/g' | perl -F, -lane 'print $F[$#F-4], "_", $F[$#F-3], "\t", join(",", @F);' | sort -k 1,1) <(cat /home/michaelt/Data/ExAC/0.3.1/ExAC.r0.3.1.sites.vep.vcf.AnnovarFormat.genome_summary.csv | sed 's/ /_/g' | perl -F, -lane 'print $F[$#F-4], "_", $F[$#F-3], "\t", join(",", @F);' | sort -k 1,1) | awk '{ print $2 }' > /home/michaelt/Data/ExAC/0.3.1/ExAC.r0.3.1.sites.vep.vcf.AnnovarFormat.UniqueFromHIV_P2.genome_summary.vs3.csv

cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'print join("\t", @F[1..$#F]);' | perl -ane 'foreach my $variant1 (@F) { print $variant1, "\n"; }' | sed 's/_/ /g' | sed 's/:/_/' | awk '{ print $1 }') <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat | awk '{ print $1 "_" $2 }') | sort | wc
cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'print join("\t", @F[1..$#F]);' | perl -ane 'foreach my $variant1 (@F) { print $variant1, "\n"; }' | sed 's/_/ /g' | sed 's/:/_/' | awk '{ print $1 }') <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat | awk '{ print $1 "_" $2 }') | sort | uniq | wc
cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'print join("\t", @F[1..$#F]);' | perl -ane 'foreach my $variant1 (@F) { print $variant1, "\n"; }' | sed 's/_/ /g' | sed 's/:/_/' | awk '{ print $1 }') <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat | awk '{ print $1 "_" $2 }') | sort | uniq -d | wc

##/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat.genome_summary.csv <(cat cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'print join("\t", @F[1..$#F]);' | perl -ane 'foreach my $variant1 (@F) { print $variant1, "\n"; }' | sed 's/_/ /g' | sed 's/:/_/' | awk '{ print $1 }' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat.genome_summary.csv | sed 's/ /_/g' | perl -F, -lane 'print $F[$#F-4], "_", $F[$#F-3], "\t", join(",", @F);' | sort -k 1,1)
join -j 1 -v 2 -e NA1 -o 0 2.2 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'print join("\t", @F[1..$#F]);' | perl -ane 'foreach my $variant1 (@F) { print $variant1, "\n"; }' | sed 's/_/ /g' | sed 's/:/_/' | awk '{ print $1 }' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat.genome_summary.csv | sed 's/ /_/g' | perl -F, -lane 'print $F[$#F-4], "_", $F[$#F-3], "\t", join(",", @F);' | sort -k 1,1) | awk '{ print $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat.UniqueFromHIV_P2NonSyn.genome_summary.csv 











##/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat.genome_summary.csv
#/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat.UniqueFromHIV_P2NonSyn.genome_summary.csv
#/home/m#ichaelt/Data/ExAC/0.3.1/ExAC.r0.3.1.sites.vep.vcf.AnnovarFormat.UniqueFromHIV_P2.genome_summary.vs3.csv
#grep -E 'exonic|splicing|intronic|UTR5|UTR3'
#grep -E 'exonic|splicing|intronic|UTR5|UTR3|upstream|downstream|intergenic'

#20160728 NOTE -- including '_' in the 'not-match' grep parameter due to APOBEC3A & APOBEC3A_B

#rm -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/DataAnalysis.All.FollowupGenotypingList_AllGenesVarsHIVExAC.txt
for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | sort -g -k 2,2 | grep -v NUM_PASS_VARS | awk '{ print $1 }'`; do

	pValue=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | grep ^$gene1[^-0-9a-zA-Z_] | awk '{ print $2 }'`
	qValue=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | grep ^$gene1[^-0-9a-zA-Z_] | awk '{ print $4 }'`
	PrevNumber=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/XChrFollowup/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep ^$gene1[^-0-9a-zA-Z_] | perl -lane 'print join("\t", @F[1..$#F]);' | perl -lane 'foreach my $variant1 (@F) { print $variant1; }' | wc | awk '{ print $1 }'`

	GenicNonSyn=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat.UniqueFromHIV_P2NonSyn.genome_summary.csv | grep [^-0-9a-zA-Z]$gene1[^-0-9a-zA-Z_] | grep -E 'nonsynonymous|splicing|stopgain|stoploss' | wc | awk '{ print $1 }'`
	Genic=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat.UniqueFromHIV_P2NonSyn.genome_summary.csv | grep [^-0-9a-zA-Z]$gene1[^-0-9a-zA-Z_] | grep -E 'exonic|splicing|intronic|UTR5|UTR3' | wc | awk '{ print $1 }'`  
	GenicPlus25=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat.UniqueFromHIV_P2NonSyn.genome_summary.csv | grep [^-0-9a-zA-Z]$gene1[^-0-9a-zA-Z_] | grep -E 'exonic|splicing|intronic|UTR5|UTR3|upstream|downstream|intergenic' | perl -F, -sane 'if ($F[0] =~ m/intergenic/) { foreach my $entry1 (@F[0..9]) { my $flag1 = 0; if ($entry1 =~ m/(.*)($gene1\()(dist=)(\d+)(.*)/) { if ($4 < 25000) { print join(",", @F); } } } } else { print join(",", @F); }' -- -gene1=$gene1 | wc | awk '{ print $1 }'` 
	GenicPlus50=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat.UniqueFromHIV_P2NonSyn.genome_summary.csv | grep [^-0-9a-zA-Z]$gene1[^-0-9a-zA-Z_] | grep -E 'exonic|splicing|intronic|UTR5|UTR3|upstream|downstream|intergenic' | perl -F, -sane 'if ($F[0] =~ m/intergenic/) { foreach my $entry1 (@F[0..9]) { my $flag1 = 0; if ($entry1 =~ m/(.*)($gene1\()(dist=)(\d+)(.*)/) { if ($4 < 50000) { print join(",", @F); } } } } else { print join(",", @F); }' -- -gene1=$gene1 | wc | awk '{ print $1 }'`
	GenicPlus75=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat.UniqueFromHIV_P2NonSyn.genome_summary.csv | grep [^-0-9a-zA-Z]$gene1[^-0-9a-zA-Z_] | grep -E 'exonic|splicing|intronic|UTR5|UTR3|upstream|downstream|intergenic' | perl -F, -sane 'if ($F[0] =~ m/intergenic/) { foreach my $entry1 (@F[0..9]) { my $flag1 = 0; if ($entry1 =~ m/(.*)($gene1\()(dist=)(\d+)(.*)/) { if ($4 < 75000) { print join(",", @F); } } } } else { print join(",", @F); }' -- -gene1=$gene1 | wc | awk '{ print $1 }'`
	GenicPlus100=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat.UniqueFromHIV_P2NonSyn.genome_summary.csv | grep [^-0-9a-zA-Z]$gene1[^-0-9a-zA-Z_] | grep -E 'exonic|splicing|intronic|UTR5|UTR3|upstream|downstream|intergenic' | perl -F, -sane 'if ($F[0] =~ m/intergenic/) { foreach my $entry1 (@F[0..9]) { my $flag1 = 0; if ($entry1 =~ m/(.*)($gene1\()(dist=)(\d+)(.*)/) { if ($4 < 100000) { print join(",", @F); } } } } else { print join(",", @F); }' -- -gene1=$gene1 | wc | awk '{ print $1 }'`
	GenicOver100=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wProperXChr.AnnovarFormat.UniqueFromHIV_P2NonSyn.genome_summary.csv | grep [^-0-9a-zA-Z]$gene1[^-0-9a-zA-Z_] | grep -E 'exonic|splicing|intronic|UTR5|UTR3|upstream|downstream|intergenic' | wc | awk '{ print $1 }'`

	ExACNonSyn=`cat /home/michaelt/Data/ExAC/0.3.1/ExAC.r0.3.1.sites.vep.vcf.AnnovarFormat.UniqueFromHIV_P2.genome_summary.vs3.csv | grep [^-0-9a-zA-Z]$gene1[^-0-9a-zA-Z_] | grep -E 'nonsynonymous|splicing|stopgain|stoploss' | wc | awk '{ print $1 }'`
	ExAC=`cat /home/michaelt/Data/ExAC/0.3.1/ExAC.r0.3.1.sites.vep.vcf.AnnovarFormat.UniqueFromHIV_P2.genome_summary.vs3.csv | grep [^-0-9a-zA-Z]$gene1[^-0-9a-zA-Z_] | grep -E 'exonic|splicing|intronic|UTR5|UTR3' | wc | awk '{ print $1 }'`
	ExACPlus25=`cat /home/michaelt/Data/ExAC/0.3.1/ExAC.r0.3.1.sites.vep.vcf.AnnovarFormat.UniqueFromHIV_P2.genome_summary.vs3.csv | grep [^-0-9a-zA-Z]$gene1[^-0-9a-zA-Z_] | grep -E 'exonic|splicing|intronic|UTR5|UTR3|upstream|downstream|intergenic' | perl -F, -sane 'if ($F[0] =~ m/intergenic/) { foreach my $entry1 (@F[0..9]) { my $flag1 = 0; if ($entry1 =~ m/(.*)($gene1\()(dist=)(\d+)(.*)/) { if ($4 < 25000) { print join(",", @F); } } } } else { print join(",", @F); }' -- -gene1=$gene1 | wc | awk '{ print $1 }'`
	ExACPlus50=`cat /home/michaelt/Data/ExAC/0.3.1/ExAC.r0.3.1.sites.vep.vcf.AnnovarFormat.UniqueFromHIV_P2.genome_summary.vs3.csv | grep [^-0-9a-zA-Z]$gene1[^-0-9a-zA-Z_] | grep -E 'exonic|splicing|intronic|UTR5|UTR3|upstream|downstream|intergenic' | perl -F, -sane 'if ($F[0] =~ m/intergenic/) { foreach my $entry1 (@F[0..9]) { my $flag1 = 0; if ($entry1 =~ m/(.*)($gene1\()(dist=)(\d+)(.*)/) { if ($4 < 50000) { print join(",", @F); } } } } else { print join(",", @F); }' -- -gene1=$gene1 | wc | awk '{ print $1 }'`
	ExACPlus75=`cat /home/michaelt/Data/ExAC/0.3.1/ExAC.r0.3.1.sites.vep.vcf.AnnovarFormat.UniqueFromHIV_P2.genome_summary.vs3.csv | grep [^-0-9a-zA-Z]$gene1[^-0-9a-zA-Z_] | grep -E 'exonic|splicing|intronic|UTR5|UTR3|upstream|downstream|intergenic' | perl -F, -sane 'if ($F[0] =~ m/intergenic/) { foreach my $entry1 (@F[0..9]) { my $flag1 = 0; if ($entry1 =~ m/(.*)($gene1\()(dist=)(\d+)(.*)/) { if ($4 < 75000) { print join(",", @F); } } } } else { print join(",", @F); }' -- -gene1=$gene1 | wc | awk '{ print $1 }'`
	ExACPlus100=`cat /home/michaelt/Data/ExAC/0.3.1/ExAC.r0.3.1.sites.vep.vcf.AnnovarFormat.UniqueFromHIV_P2.genome_summary.vs3.csv | grep [^-0-9a-zA-Z]$gene1[^-0-9a-zA-Z_] | grep -E 'exonic|splicing|intronic|UTR5|UTR3|upstream|downstream|intergenic' | perl -F, -sane 'if ($F[0] =~ m/intergenic/) { foreach my $entry1 (@F[0..9]) { my $flag1 = 0; if ($entry1 =~ m/(.*)($gene1\()(dist=)(\d+)(.*)/) { if ($4 < 100000) { print join(",", @F); } } } } else { print join(",", @F); }' -- -gene1=$gene1 | wc | awk '{ print $1 }'`
	ExACOver100=`cat /home/michaelt/Data/ExAC/0.3.1/ExAC.r0.3.1.sites.vep.vcf.AnnovarFormat.UniqueFromHIV_P2.genome_summary.vs3.csv | grep [^-0-9a-zA-Z]$gene1[^-0-9a-zA-Z_] | grep -E 'exonic|splicing|intronic|UTR5|UTR3|upstream|downstream|intergenic' | wc | awk '{ print $1 }'`

echo $gene1 $pValue $qValue $PrevNumber $GenicNonSyn $Genic $GenicPlus25 $GenicPlus50 $GenicPlus75 $GenicPlus100 $GenicOver100 $ExACNonSyn $ExAC $ExACPlus25 $ExACPlus50 $ExACPlus75 $ExACPlus100 $ExACOver100 >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/DataAnalysis.All.FollowupGenotypingList_AllGenesVarsHIVExAC.txt.vs3

done 

##cat 
##<(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/DataAnalysis.All.FollowupGenotypingList_AllGenesVarsHIVExAC.txt | head -n 25 | perl -lane 'push (@temp1, $F[0]); if (eof()) { print join(",", @temp1); print join("\t", @F[1..$#F]); } else { print join("\t", @F); }' | R -q -e "Data1 <- read.table(file('stdin')); Data2 <- as.numeric(as.character(Data1[,2:ncol(Data1)])); print(c(Data1[nrow(Data1),1], apply(Data2, 2, sum));"
cat <(echo "PrevNumber HIVNonSyn HIVNoFlank HIV25kb HIV50kb HIV75kb HIV100kb HIVOver100kb ExACNonSyn ExACNoFlank ExAC25kb ExAC50kb ExAC75kb ExAC100kb ExACOver100kb Genes") \ 
<(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/DataAnalysis.All.FollowupGenotypingList_AllGenesVarsHIVExAC.txt | head -n 25 | perl -lane 'push (@temp1, $F[0]); print join("\t", @F[0..$#F]);' | R -q -e "Data1 <- read.table(file('stdin')); Data2 <- as.matrix(Data1[,2:ncol(Data1)]); write.table(t(c(apply(Data2, 2, sum), paste(Data1[,1], collapse=\",\"))));" | grep -v V16 | grep -v \> | sed 's/"//g' | perl -lane 'print join("\t", @F[1..$#F]);') \
<(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/DataAnalysis.All.FollowupGenotypingList_AllGenesVarsHIVExAC.txt | head -n 50 | perl -lane 'push (@temp1, $F[0]); print join("\t", @F[0..$#F]);' | R -q -e "Data1 <- read.table(file('stdin')); Data2 <- as.matrix(Data1[,2:ncol(Data1)]); write.table(t(c(apply(Data2, 2, sum), paste(Data1[,1], collapse=\",\"))));" | grep -v V16 | grep -v \> | sed 's/"//g' | perl -lane 'print join("\t", @F[1..$#F]);') \
<(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/DataAnalysis.All.FollowupGenotypingList_AllGenesVarsHIVExAC.txt | head -n 100 | perl -lane 'push (@temp1, $F[0]); print join("\t", @F[0..$#F]);' | R -q -e "Data1 <- read.table(file('stdin')); Data2 <- as.matrix(Data1[,2:ncol(Data1)]); write.table(t(c(apply(Data2, 2, sum), paste(Data1[,1], collapse=\",\"))));" | grep -v V16 | grep -v \> | sed 's/"//g' | perl -lane 'print join("\t", @F[1..$#F]);') \
<(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/DataAnalysis.All.FollowupGenotypingList_AllGenesVarsHIVExAC.txt | head -n 200 | perl -lane 'push (@temp1, $F[0]); print join("\t", @F[0..$#F]);' | R -q -e "Data1 <- read.table(file('stdin')); Data2 <- as.matrix(Data1[,2:ncol(Data1)]); write.table(t(c(apply(Data2, 2, sum), paste(Data1[,1], collapse=\",\"))));" | grep -v V16 | grep -v \> | sed 's/"//g' | perl -lane 'print join("\t", @F[1..$#F]);') \
<(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/DataAnalysis.All.FollowupGenotypingList_AllGenesVarsHIVExAC.txt | head -n 400 | perl -lane 'push (@temp1, $F[0]); print join("\t", @F[0..$#F]);' | R -q -e "Data1 <- read.table(file('stdin')); Data2 <- as.matrix(Data1[,2:ncol(Data1)]); write.table(t(c(apply(Data2, 2, sum), paste(Data1[,1], collapse=\",\"))));" | grep -v V16 | grep -v \> | sed 's/"//g' | perl -lane 'print join("\t", @F[1..$#F]);') \
<(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/DataAnalysis.All.FollowupGenotypingList_AllGenesVarsHIVExAC.txt | perl -lane 'push (@temp1, $F[0]); print join("\t", @F[0..$#F]);' | R -q -e "Data1 <- read.table(file('stdin')); Data2 <- as.matrix(Data1[,2:ncol(Data1)]); write.table(t(c(apply(Data2, 2, sum), paste(Data1[,1], collapse=\",\"))));" | grep -v V16 | grep -v \> | sed 's/"//g' | perl -lane 'print join("\t", @F[1..$#F]);') > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/DataAnalysis.All.FollowupGenotypingList_AllGenesVarsHIVExAC.SummedOverGroups.txt

cat <(echo "qValThresh NumGenes PrevNonSyn HIVNonSyn HIVNoFlank HIV25kb HIV50kb HIV75kb HIV100kb HIVOver100kb ExACNonSyn ExACNoFlank ExAC25kb ExAC50kb ExAC75kb ExAC100kb ExACOver100kb Genes") \ 
<(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/DataAnalysis.All.FollowupGenotypingList_AllGenesVarsHIVExAC.txt.vs2 | awk '{ if ($3 <= .5 ) { print $0 } } ' | perl -lane 'push (@temp1, $F[0]); print $F[0], "\t", join("\t", @F[3..$#F]);' | R -q -e "Data1 <- read.table(file('stdin')); Data2 <- as.matrix(Data1[,2:ncol(Data1)]); write.table(t(c(apply(Data2, 2, sum), paste(Data1[,1], collapse=\",\"))));" | grep -v V16 | grep -v \> | sed 's/"//g' | perl -lane 'print ".5\t", scalar((split(/,/, $F[$#F]))), "\t", join("\t", @F[1..$#F]);') \
<(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/DataAnalysis.All.FollowupGenotypingList_AllGenesVarsHIVExAC.txt.vs2 | awk '{ if ($3 <= .55 ) { print $0 } } ' | perl -lane 'push (@temp1, $F[0]); print $F[0], "\t", join("\t", @F[3..$#F]);' | R -q -e "Data1 <- read.table(file('stdin')); Data2 <- as.matrix(Data1[,2:ncol(Data1)]); write.table(t(c(apply(Data2, 2, sum), paste(Data1[,1], collapse=\",\"))));" | grep -v V16 | grep -v \> | sed 's/"//g' | perl -lane 'print ".55\t", scalar((split(/,/, $F[$#F]))), "\t", join("\t", @F[1..$#F]);') \
<(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/DataAnalysis.All.FollowupGenotypingList_AllGenesVarsHIVExAC.txt.vs2 | awk '{ if ($3 <= .6 ) { print $0 } } ' | perl -lane 'push (@temp1, $F[0]); print $F[0], "\t", join("\t", @F[3..$#F]);' | R -q -e "Data1 <- read.table(file('stdin')); Data2 <- as.matrix(Data1[,2:ncol(Data1)]); write.table(t(c(apply(Data2, 2, sum), paste(Data1[,1], collapse=\",\"))));" | grep -v V16 | grep -v \> | sed 's/"//g' | perl -lane 'print ".6\t", scalar((split(/,/, $F[$#F]))), "\t", join("\t", @F[1..$#F]);') \
<(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/DataAnalysis.All.FollowupGenotypingList_AllGenesVarsHIVExAC.txt.vs2 | awk '{ if ($3 <= .65 ) { print $0 } } ' | perl -lane 'push (@temp1, $F[0]); print $F[0], "\t", join("\t", @F[3..$#F]);' | R -q -e "Data1 <- read.table(file('stdin')); Data2 <- as.matrix(Data1[,2:ncol(Data1)]); write.table(t(c(apply(Data2, 2, sum), paste(Data1[,1], collapse=\",\"))));" | grep -v V16 | grep -v \> | sed 's/"//g' | perl -lane 'print ".65\t", scalar((split(/,/, $F[$#F]))), "\t", join("\t", @F[1..$#F]);') \
<(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/DataAnalysis.All.FollowupGenotypingList_AllGenesVarsHIVExAC.txt.vs2 | awk '{ if ($3 <= .7 ) { print $0 } } ' | perl -lane 'push (@temp1, $F[0]); print $F[0], "\t", join("\t", @F[3..$#F]);' | R -q -e "Data1 <- read.table(file('stdin')); Data2 <- as.matrix(Data1[,2:ncol(Data1)]); write.table(t(c(apply(Data2, 2, sum), paste(Data1[,1], collapse=\",\"))));" | grep -v V16 | grep -v \> | sed 's/"//g' | perl -lane 'print ".7\t", scalar((split(/,/, $F[$#F]))), "\t", join("\t", @F[1..$#F]);') \
<(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/DataAnalysis.All.FollowupGenotypingList_AllGenesVarsHIVExAC.txt.vs2 | awk '{ if ($3 <= .75 ) { print $0 } } ' | perl -lane 'push (@temp1, $F[0]); print $F[0], "\t", join("\t", @F[3..$#F]);' | R -q -e "Data1 <- read.table(file('stdin')); Data2 <- as.matrix(Data1[,2:ncol(Data1)]); write.table(t(c(apply(Data2, 2, sum), paste(Data1[,1], collapse=\",\"))));" | grep -v V16 | grep -v \> | sed 's/"//g' | perl -lane 'print ".75\t", scalar((split(/,/, $F[$#F]))), "\t", join("\t", @F[1..$#F]);') \
<(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/DataAnalysis.All.FollowupGenotypingList_AllGenesVarsHIVExAC.txt.vs2 | awk '{ if ($3 <= .8 ) { print $0 } } ' | perl -lane 'push (@temp1, $F[0]); print $F[0], "\t", join("\t", @F[3..$#F]);' | R -q -e "Data1 <- read.table(file('stdin')); Data2 <- as.matrix(Data1[,2:ncol(Data1)]); write.table(t(c(apply(Data2, 2, sum), paste(Data1[,1], collapse=\",\"))));" | grep -v V16 | grep -v \> | sed 's/"//g' | perl -lane 'print ".8\t", scalar((split(/,/, $F[$#F]))), "\t", join("\t", @F[1..$#F]);') \
<(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/DataAnalysis.All.FollowupGenotypingList_AllGenesVarsHIVExAC.txt.vs2 | awk '{ if ($3 <= .85 ) { print $0 } } ' | perl -lane 'push (@temp1, $F[0]); print $F[0], "\t", join("\t", @F[3..$#F]);' | R -q -e "Data1 <- read.table(file('stdin')); Data2 <- as.matrix(Data1[,2:ncol(Data1)]); write.table(t(c(apply(Data2, 2, sum), paste(Data1[,1], collapse=\",\"))));" | grep -v V16 | grep -v \> | sed 's/"//g' | perl -lane 'print ".85\t", scalar((split(/,/, $F[$#F]))), "\t", join("\t", @F[1..$#F]);') \
<(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/DataAnalysis.All.FollowupGenotypingList_AllGenesVarsHIVExAC.txt.vs2 | awk '{ if ($3 <= .9 ) { print $0 } } ' | perl -lane 'push (@temp1, $F[0]); print $F[0], "\t", join("\t", @F[3..$#F]);' | R -q -e "Data1 <- read.table(file('stdin')); Data2 <- as.matrix(Data1[,2:ncol(Data1)]); write.table(t(c(apply(Data2, 2, sum), paste(Data1[,1], collapse=\",\"))));" | grep -v V16 | grep -v \> | sed 's/"//g' | perl -lane 'print ".9\t", scalar((split(/,/, $F[$#F]))), "\t", join("\t", @F[1..$#F]);') \
<(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/DataAnalysis.All.FollowupGenotypingList_AllGenesVarsHIVExAC.txt.vs2 | awk '{ if ($3 <= .95 ) { print $0 } } ' | perl -lane 'push (@temp1, $F[0]); print $F[0], "\t", join("\t", @F[3..$#F]);' | R -q -e "Data1 <- read.table(file('stdin')); Data2 <- as.matrix(Data1[,2:ncol(Data1)]); write.table(t(c(apply(Data2, 2, sum), paste(Data1[,1], collapse=\",\"))));" | grep -v V16 | grep -v \> | sed 's/"//g' | perl -lane 'print ".95\t", scalar((split(/,/, $F[$#F]))), "\t", join("\t", @F[1..$#F]);') \
<(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/DataAnalysis.All.FollowupGenotypingList_AllGenesVarsHIVExAC.txt.vs2 | perl -lane 'push (@temp1, $F[0]); print $F[0], "\t", join("\t", @F[3..$#F]);' | R -q -e "Data1 <- read.table(file('stdin')); Data2 <- as.matrix(Data1[,2:ncol(Data1)]); write.table(t(c(apply(Data2, 2, sum), paste(Data1[,1], collapse=\",\"))));" | grep -v V16 | grep -v \> | sed 's/"//g' | perl -lane 'print "1.0\t", scalar((split(/,/, $F[$#F]))), "\t", join("\t", @F[1..$#F]);') > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/DataAnalysis.All.FollowupGenotypingList_AllGenesVarsHIVExAC.SummedOverGroups.qVals.txt 

~~~
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/DataAnalysis.All.FollowupGenotypingList_AllGenesVarsHIVExAC.txt | head -n 25 | perl -lane 'push (@temp1, $F[0]); print join("\t", @F[0..$#F]);' | R -q -e "Data1 <- read.table(file('stdin')); Data2 <- as.matrix(Data1[,2:ncol(Data1)]); write.table(t(c(apply(Data2, 2, sum), paste(Data1[,1], collapse=\",\"))));" | grep -v V2 | grep -v \> | sed 's/"//g' | perl -lane 'print join("\t", @F[1..$#F]);'
2843    3051    3052    3052    3052    3052    45564   45682   45682   45682   45682   46140   GEMIN2,VTI1A,VDAC3,SDHD,NUDC,CUL5,EDC4,FUT8,MRPS9,CIAO1,GEMIN6,ICAM2,KIR3DL1,EIF3L,EEF1E1,PHKG1,FBN2,MLLT1,ATG3,GPATCH4,COPB1,TPR,CCNT1,OSBPL6,GIMAP5
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/DataAnalysis.All.FollowupGenotypingList_AllGenesVarsHIVExAC.SummedOverGroups.txt | perl -lane 'print join("\t", @F[0..$#F-1]);' | awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $7, "\t", $8, "\t", $9, "\t", $10, "\t", $11, "\t", $14, "\t", $15 }'
PrevNumber       HIVNonSyn       HIVNoFlank      HIV25kb         HIV100kb        HIVOver100kb    ExACNonSyn      ExACNoFlank     ExAC25kb        ExAC100kb       ExACOver100kb
226      26      2863    3040    3040    3040    4140    11473   11488   11884   11885
396      42      5337    5737    5737    5737    7681    20687   20754   21215   21216
761      76      10271   11089   11089   11089   15395   40165   40328   40806   40813
1501     151     19636   21077   21096   21096   30704   79930   80634   81516   81563
2924     303     39487   42526   42676   42750   61828   159908          161054          162588          162962
5528     589     77960   83794   83974   84048   119060          307875          310129          312491          313674
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/DataAnalysis.All.FollowupGenotypingList_AllGenesVarsHIVExAC.txt.vs2 | awk '{ print $3 }' | sort | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.05), right=FALSE));"
> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.05), right=FALSE));

  [0,0.05) [0.05,0.1) [0.1,0.15) [0.15,0.2) [0.2,0.25) [0.25,0.3) [0.3,0.35) 
         0          0          0          0          0          0          0 
[0.35,0.4) [0.4,0.45) [0.45,0.5) [0.5,0.55) [0.55,0.6) [0.6,0.65) [0.65,0.7) 
         4          1          0         27         11         24         26 
[0.7,0.75) [0.75,0.8) [0.8,0.85) [0.85,0.9) [0.9,0.95)   [0.95,1) 
        16         35         38        171        416         12 
> 
> 
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/DataAnalysis.All.FollowupGenotypingList_AllGenesVarsHIVExAC.txt.vs2 | awk '{ print $3 }' | sort | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.01), right=FALSE));"
> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.01), right=FALSE));

   [0,0.01) [0.01,0.02) [0.02,0.03) [0.03,0.04) [0.04,0.05) [0.05,0.06) 
          0           0           0           0           0           0 
[0.06,0.07) [0.07,0.08) [0.08,0.09)  [0.09,0.1)  [0.1,0.11) [0.11,0.12) 
          0           0           0           0           0           0 
[0.12,0.13) [0.13,0.14) [0.14,0.15) [0.15,0.16) [0.16,0.17) [0.17,0.18) 
          0           0           0           0           0           0 
[0.18,0.19)  [0.19,0.2)  [0.2,0.21) [0.21,0.22) [0.22,0.23) [0.23,0.24) 
          0           0           0           0           0           0 
[0.24,0.25) [0.25,0.26) [0.26,0.27) [0.27,0.28) [0.28,0.29)  [0.29,0.3) 
          0           0           0           0           0           0 
 [0.3,0.31) [0.31,0.32) [0.32,0.33) [0.33,0.34) [0.34,0.35) [0.35,0.36) 
          0           0           0           0           0           0 
[0.36,0.37) [0.37,0.38) [0.38,0.39)  [0.39,0.4)  [0.4,0.41) [0.41,0.42) 
          0           0           0           4           0           0 
[0.42,0.43) [0.43,0.44) [0.44,0.45) [0.45,0.46) [0.46,0.47) [0.47,0.48) 
          0           0           1           0           0           0 
[0.48,0.49)  [0.49,0.5)  [0.5,0.51) [0.51,0.52) [0.52,0.53) [0.53,0.54) 
          0           0           0          18           5           4 
[0.54,0.55) [0.55,0.56) [0.56,0.57) [0.57,0.58) [0.58,0.59)  [0.59,0.6) 
          0           3           0           7           1           0 
 [0.6,0.61) [0.61,0.62) [0.62,0.63) [0.63,0.64) [0.64,0.65) [0.65,0.66) 
          2          20           2           0           0           0 
[0.66,0.67) [0.67,0.68) [0.68,0.69)  [0.69,0.7)  [0.7,0.71) [0.71,0.72) 
          8          17           0           1           0           0 
[0.72,0.73) [0.73,0.74) [0.74,0.75) [0.75,0.76) [0.76,0.77) [0.77,0.78) 
          0           0          16          11           2           1 
[0.78,0.79)  [0.79,0.8)  [0.8,0.81) [0.81,0.82) [0.82,0.83) [0.83,0.84) 
          3          18           3           1           0           1 
[0.84,0.85) [0.85,0.86) [0.86,0.87) [0.87,0.88) [0.88,0.89)  [0.89,0.9) 
         33           2           5         136          14          14 
 [0.9,0.91) [0.91,0.92) [0.92,0.93) [0.93,0.94) [0.94,0.95) [0.95,0.96) 
        362           4          50           0           0          12 
[0.96,0.97) [0.97,0.98) [0.98,0.99)    [0.99,1) 
          0           0           0           0 
> 
> 
[  michaelt@node12  ~]$cat <(echo "qValThresh NumGenes PrevNonSyn HIVNonSyn HIVNoFlank HIV25kb HIV50kb HIV75kb HIV100kb HIVOver100kb ExACNonSyn ExACNoFlank ExAC25kb ExAC50kb ExAC75kb ExAC100kb ExACOver100kb Genes") <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GatheringVariantsForFollowupGenotyping/DataAnalysis.All.FollowupGenotypingList_AllGenesVarsHIVExAC.txt.vs2 | awk '{ if ($3 <= .5 ) { print $0 } } ' | perl -lane 'push (@temp1, $F[0]); print $F[0], "\t", join("\t", @F[3..$#F]);' | R -q -e "Data1 <- read.table(file('stdin')); Data2 <- as.matrix(Data1[,2:ncol(Data1)]); write.table(t(c(apply(Data2, 
.
.
qValThresh      NumGenes        PrevNonSyn      HIVNonSyn       HIVNoFlank      HIV25kb HIV50kb HIV75kb HIV100kb        HIVOver100kb    ExACNonSyn      ExACNoFlank     ExAC25kb        ExAC50kb        ExAC75kb        ExAC100kb      ExACOver100kb
.5      5       44      7       657     720     720     720     720     720     756     2158    2162    2162    2162    2162    2162
.55     32      289     32      3492    3736    3736    3736    3736    3736    5229    13846   13871   13871   13882   14267   14268
.6      43      353     37      4545    4887    4887    4887    4887    4887    6751    18284   18341   18341   18394   18802   18803
.65     67      533     56      7015    7557    7557    7557    7557    7557    10637   27679   27797   27806   27867   28275   28281
.7      93      713     70      9382    10139   10139   10139   10139   10139   14183   37011   37156   37165   37226   37634   37641
.75     109     857     86      11057   11904   11904   11904   11904   11904   17204   44173   44348   44357   44418   44826   44850
.8      143     1081    111     14579   15660   15660   15660   15660   15660   22089   57846   58142   58165   58226   58634   58658
.85     181     1363    142     18224   19585   19585   19585   19585   19585   27828   72596   73296   73688   73749   74175   74222
.9      281     2135    233     28529   30735   30745   30776   30790   30799   44235   115584  116434  116845  116906  117333  117401
.95     281     2135    233     28529   30735   30745   30776   30790   30799   44235   115584  116434  116845  116906  117333  117401
1.0     285     2156    233     28771   30997   31007   31038   31052   31061   44769   117131  117981  118392  118453  118880  118948






[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/GatheringVariantsForFollowupGenotyping/DataAnalysis.All.FollowupGenotypingList_AllGenesVarsHIVExAC.SummedOverGroups.qVals.txt | perl -lane 'print join("\t", @F[0..$#F-1]);' | awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $5, "\t", $6, "\t", $9, "\t", $10, "\t", $11, "\t", $12, "\t", $13, "\t", $16, "\t", $17 }'
qValThresh       NumGenes        PrevNonSyn      HIVNonSyn       HIVNoFlank      HIV25kb         HIV100kb        HIVOver100kb    ExACNonSyn      ExACNoFlank     ExAC25kb        ExAC100kb       ExACOver100kb
.35      29      183     18      2938    3175    3176    3176    4702    11966   12031   12031   12071
.5       61      394     54      6518    7074    7075    7075    10389   26233   26391   26413   26453
.54      67      494     81      6988    7584    7585    7585    11635   28963   29121   29143   29183

~~~








#Do hwe1e4 as well as XChr het stuff? do --genome and look at distributions, remove z1 again? Drop indv80....do same for P2 as well?
#Then look at/include ExAC material as well? Compare 'all' and 'exonic' to see where go with too? Maybe just go with 'all' and 'exonic' to begin with anyways?
#Need to do the QC procedures of hwe1e4 and DropIBD, indv80 on all individuals (plus + nonwhites too) to get full list of variants? do QC? Don't do QC?
#May not need to dorp individuals since only care about variants too?
#Hwe14 matters but that may be it
#Do hwe1e4, remove all X, do het stuff, bring back X after het, then do the Annovar All/Exonic stuff

#anno
#perl /home/michaelt/Software/annovar/summarize_annovar.pl 
#/home/michaelt/Data/ExAC/0.3.1/ExAC.r0.3.1.sites.vep.vcf.AnnovarFormat 
#/home/michaelt/Software/annovar/ --ver1000g 1000g2012apr --verdbsnp 129 --veresp 6500 --buildver hg19 --remove

#Including the below code on 20160726
cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep ^X | perl -lane 'print join("_", (split(/_/, $F[3]))[1..scalar((split(/_/, $F[3])))-1]);')) /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | sort > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr

~~~
.
.
.

[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | sort -g -k 2,2 | grep -v NUM_PASS_VARS | wc
    786   20436  109393
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | sort -g -k 2,2 | grep -v NUM_PASS_VARS | head -n 10
XAF1            0.001356961 2.615152e-04 0.39392        7       747     0.037483        7       7       1       0.0016809       1       747     0.037483        7       7       1       0.0014564       1       747     0.037483       7       7       1       0.0010727       1
PRDM1           0.001441239 3.511077e-04 0.39392        12      747     0.41633 12      12      3       0.0014301       1       747     0.41633 12      12      3       0.002057        1       747     0.41633 12      12    3
        0.0011171       1
PDXK            0.001957196 3.937028e-06 0.39392        3       747     0.0066934       3       3       1       0.001967        0.09    747     0.0066934       3       3       1       0.0019625       0.09    747     0.0066934      3       3       1       0.0019527       0.09
MAPK14          0.002168403 1.344193e-04 0.39392        3       747     0.0080321       3       3       2       0.0022884       1       747     0.0080321       3       3       2       0.0021681       1       747     0.0080321      3       3       2       0.0020264       1
NLRX1           0.003075420 4.997193e-04 0.44695        19      747     0.73762 19      19      8       0.0038118       0.04    747     0.73762 19      19      8       0.0025784       0.04    747     0.73762 19      19    8
        0.0033909       0.04
PRRC2A          0.005027917 6.694080e-04 0.51390        48      747     0.91968 48      48      15      0.0053474       1       747     0.91968 48      48      15      0.0049166       1       747     0.91968 48      48    15       0.0046948       0.5
CAPZB           0.005132807 3.361092e-04 0.51390        3       747     0.017403        3       3       0       0.0052729       0.04    747     0.017403        3       3       0       0.0047568       0.04    747     0.017403       3       3       0       0.0046395       0.01
C9orf69         0.008967305 4.244869e-04 0.51390        1       747     0.0093708       1       1       0       0.0088953       NA      747     0.0093708       1       1       0       0.0090003       NA      747     0.0093708      1       1       0       0.0090988       NA
PPAN            0.009379527 1.126217e-03 0.51390        13      747     0.084337        13      13      3       0.0083465       0.01    747     0.084337        13      13      3       0.0092775       0.04    747     0.084337       13      13      3       0.0094964       0.04
KLF6            0.010195799 4.530083e-04 0.51390        3       747     0.0053548       3       3       2       0.01038 1       747     0.0053548       3       3       2       0.0095274       0.25    747     0.0053548     3
        3       2       0.0098653       0.25
~~~





































##20160729
##GTEX whole blood eQTL investigation work

~~~
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$join -1 1 -2 3 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | awk '{ print $1 }' | sort) <(zcat /home/michaelt/Data/UCSC/UCSC_hg19_ENST_ENSG_geneName.txt.gz | awk '{ print $2, "\t", $3, "\t", $4 }' | sort | uniq | sort -k 3,3) | wc
    862    2586   24434
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | wc
    788   20478  109841
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$join -1 1 -2 3 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | awk '{ print $1 }' | sort) <(zcat /home/michaelt/Data/UCSC/UCSC_hg19_ENST_ENSG_geneName.txt.gz | awk '{ print $2, "\t", $3, "\t", $4 }' | sort | uniq | sort -k 3,3) | awk '{ print $1 }' | uniq -cd
      7 AGER
      7 DDR1
      7 GABBR1
      7 HLA-DRA
      6 HLA-DRB1
      5 HSPA1A
      5 HSPA1B
      2 IDS
      8 MOG
      7 PRRC2A
      8 PSMB9
      8 TAP1
      8 TNF
      7 UBD
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$join -1 1 -2 3 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | awk '{ print $1 }' | sort) <(zcat /home/michaelt/Data/UCSC/UCSC_hg19_ENST_ENSG_geneName.txt.gz | awk '{ print $2, "\t", $3, "\t", $4 }' | sort | uniq | sort -k 3,3) | grep AGER                     
AGER chr6 ENSG00000204305
AGER chr6_apd_hap1 ENSG00000231268
AGER chr6_cox_hap2 ENSG00000234729
AGER chr6_mann_hap4 ENSG00000229058
AGER chr6_mcf_hap5 ENSG00000230514
AGER chr6_qbl_hap6 ENSG00000206320
AGER chr6_ssto_hap7 ENSG00000237405
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$join -1 1 -2 3 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | awk '{ print $1 }' | sort) <(zcat /home/michaelt/Data/UCSC/UCSC_hg19_ENST_ENSG_geneName.txt.gz | awk '{ print $2, "\t", $3, "\t", $4 }' | sort | uniq | sort -k 3,3) | grep UBD 
UBD chr6 ENSG00000213886
UBD chr6_apd_hap1 ENSG00000226898
UBD chr6_cox_hap2 ENSG00000228913
UBD chr6_dbb_hap3 ENSG00000231968
UBD chr6_mann_hap4 ENSG00000224654
UBD chr6_mcf_hap5 ENSG00000206468
UBD chr6_qbl_hap6 ENSG00000206513
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$join -1 1 -2 3 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | awk '{ print $1 }' | sort) <(zcat /home/michaelt/Data/UCSC/UCSC_hg19_ENST_ENSG_geneName.txt.gz | awk '{ print $2, "\t", $3, "\t", $4 }' | sort | uniq | sort -k 3,3) | grep IDS
IDS chrX ENSG00000010404
IDS chrX ENSG00000241489
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$join -1 1 -2 3 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | awk '{ print $1 }' | sort) <(zcat /home/michaelt/Data/UCSC/UCSC_hg19_ENST_ENSG_geneName.txt.gz | awk '{ print $2, "\t", $3, "\t", $4 }' | sort | uniq | sort -k 3,3) | grep MOG
MOG chr6 ENSG00000204655
MOG chr6_apd_hap1 ENSG00000234096
MOG chr6_cox_hap2 ENSG00000237834
MOG chr6_dbb_hap3 ENSG00000234623
MOG chr6_mann_hap4 ENSG00000230885
MOG chr6_mcf_hap5 ENSG00000232641
MOG chr6_qbl_hap6 ENSG00000137345
MOG chr6_ssto_hap7 ENSG00000236561
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$join -1 3 -2 2 <(join -1 1 -2 3 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | awk '{ print $1 }' | sort) <(zcat /home/michaelt/Data/UCSC/UCSC_hg19_ENST_ENSG_geneName.txt.gz | awk '{ print $2, "\t", $3, "\t", $4 }' | sort | uniq | sort -k 3,3) | sort -k 3,3) <(zcat /home/michaelt/Data/GTEX/Vs6/Whole_Blood_Analysis_cis-eQTLs.justIDs.uniq.txt.gz | sed 's/\./ /g' | sort -k 2,2) | wc
    698    3490   24398
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$ls -lrt /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets/ | grep .P2. | wc
    797    7173   75052
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$ls -lrt /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets/ | grep Whole_Blood_Analysis_cis-eQTLs.P2 | wc
    786    7074   73990
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$ls -lrt /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets/ | grep .P2. | grep MM5
-rw-rw-r-- 1 michaelt michaelt     20 Jul 29 14:49 Whole_Blood_Analysis_cis-eQTLs.MM5.PPAN-P2RY11.gz
-rw-rw-r-- 1 michaelt michaelt 131644 Jul 29 14:51 Whole_Blood_Analysis_cis-eQTLs.MM5.G3BP2.gz
-rw-rw-r-- 1 michaelt michaelt 104474 Jul 29 14:51 Whole_Blood_Analysis_cis-eQTLs.MM5.RANBP2.gz
-rw-rw-r-- 1 michaelt michaelt 123264 Jul 29 14:51 Whole_Blood_Analysis_cis-eQTLs.MM5.TOR1AIP2.gz
-rw-rw-r-- 1 michaelt michaelt 122554 Jul 29 14:51 Whole_Blood_Analysis_cis-eQTLs.MM5.P2RY11.gz
-rw-rw-r-- 1 michaelt michaelt 159061 Jul 29 14:52 Whole_Blood_Analysis_cis-eQTLs.MM5.ATP2A3.gz
-rw-rw-r-- 1 michaelt michaelt  88000 Jul 29 14:52 Whole_Blood_Analysis_cis-eQTLs.MM5.ATP2A2.gz
-rw-rw-r-- 1 michaelt michaelt 112589 Jul 29 14:54 Whole_Blood_Analysis_cis-eQTLs.MM5.CASP8AP2.gz
-rw-rw-r-- 1 michaelt michaelt 174438 Jul 29 14:55 Whole_Blood_Analysis_cis-eQTLs.MM5.PGAP2.gz
-rw-rw-r-- 1 michaelt michaelt 183637 Jul 29 14:56 Whole_Blood_Analysis_cis-eQTLs.MM5.AIMP2.gz
-rw-rw-r-- 1 michaelt michaelt  88021 Jul 29 14:56 Whole_Blood_Analysis_cis-eQTLs.MM5.ATP2B1.gz
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$ls -lrt /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets/ | grep .P2. | grep MM5 | wc
     11      99    1062
~~~

for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | sort -g -k 2,2 | grep -v NUM_PASS_VARS | awk '{ print $1 }'`; do 

#       ensg1=`join -1 1 -2 2 <(zcat /home/michaelt/Data/GTEX/Vs6/Whole_Blood_Analysis_cis-eQTLs.justIDs.uniq.txt.gz | sed 's/\./ /g' | awk '{ print $2 }' | sort | uniq | sort)  <(mysql -h genome-mysql.cse.ucsc.edu -u genome -D hg19 -N -A -e 'select ens.name, ens.chrom, ens.name2, gene.value from hg19.ensGene ens INNER JOIN hg19.ensemblToGeneName gene ON ens.name = gene.name' | awk '{ print $2, "\t", $3, "\t", $4 }' | sort | uniq | sort -k 2,2) | grep [^-0-9a-zA-Z]$gene1$ | awk '{ print $1}'`
        ensg1=`join -1 1 -2 2 <(zcat /home/michaelt/Data/GTEX/Vs6/Whole_Blood_Analysis_cis-eQTLs.justIDs.uniq.txt.gz | sed 's/\./ /g' | awk '{ print $2 }' | sort | uniq | sort)  <(zcat /home/michaelt/Data/UCSC/UCSC_hg19_ENST_ENSG_geneName.txt.gz | awk '{ print $2, "\t", $3, "\t", $4 }' | sort | uniq | sort -k 2,2) | grep [^-0-9a-zA-Z_]$gene1$ | awk '{ print $1}'`
#       ensg2=`mysql -h genome-mysql.cse.ucsc.edu -u genome -D hg19 -N -A -e 'select ens.name, ens.chrom, ens.name2, gene.value from hg19.ensGene ens INNER JOIN hg19.ensemblToGeneName gene ON ens.name = gene.name' | awk '{ print $2, "\t", $3, "\t", $4 }' | sort | uniq | grep [^-0-9a-zA-Z]$gene1$ | awk '{ print $2 }'`
        echo $gene1 $ensg1 $ensg2
        bsub "zcat /home/michaelt/Data/GTEX/Vs6/Whole_Blood_Analysis_cis-eQTLs.txt.gz | grep $ensg1 | gzip > /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets/Whole_Blood_Analysis_cis-eQTLs.P2.$gene1.gz"

done

~~~
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$du -h /home/michaelt/Data/GTEX/Vs6/
140M    /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets
3.0G    /home/michaelt/Data/GTEX/Vs6/
~~~

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup 

rm -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.BPdiffTop2.txt
rm -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.10kbWindowTopResult.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.50kbWindowTopResult.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.100kbWindowTopResult.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.250kbWindowTopResult.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.500kbWindowTopResult.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.1MbWindowTopResult.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.30kbWindowTopResult.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.40kbWindowTopResult.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.60kbWindowTopResult.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.70kbWindowTopResult.txt 
for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | sort -g -k 2,2 | grep -v NUM_PASS_VARS | awk '{ print $1 }'`; do

#	zcat /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets/Whole_Blood_Analysis_cis-eQTLs.P2.$gene1.gz | sort -g -k 5,5 | sed 's/_/ /g' | head -n 2 | perl -slane 'if ($. == 1) { $topBP = $F[2]; } else { print $gene1, "\t", abs($F[2] - $topBP); }' -- -gene1=$gene1 >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.BPdiffTop2.txt 
	zcat /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets/Whole_Blood_Analysis_cis-eQTLs.P2.$gene1.gz | sort -g -k 5,5 | sed 's/_/ /g' | perl -slane 'if ($. == 1) { $topBP = $F[2]; $topPval = $F[8]; } else { print $gene1, "\t", $topBP, "\t", $topPval, "\t", $F[2], "\t", $F[8], "\t", abs($F[2] - $topBP), "\t", abs($F[8] - $topPval); }' -- -gene1=$gene1 | head -n 1 >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.NoWindowTopResult.txt 
	zcat /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets/Whole_Blood_Analysis_cis-eQTLs.P2.$gene1.gz | sort -g -k 5,5 | sed 's/_/ /g' | perl -slane 'if ($. == 1) { $topBP = $F[2]; $topPval = $F[8]; } if (abs($F[2] - $topBP) > 10000) { print $gene1, "\t", $topBP, "\t", $topPval, "\t", $F[2], "\t", $F[8], "\t", abs($F[2] - $topBP), "\t", abs($F[8] - $topPval); }' -- -gene1=$gene1 | head -n 1 >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.10kbWindowTopResult.txt 
	zcat /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets/Whole_Blood_Analysis_cis-eQTLs.P2.$gene1.gz | sort -g -k 5,5 | sed 's/_/ /g' | perl -slane 'if ($. == 1) { $topBP = $F[2]; $topPval = $F[8]; } if (abs($F[2] - $topBP) > 20000) { print $gene1, "\t", $topBP, "\t", $topPval, "\t", $F[2], "\t", $F[8], "\t", abs($F[2] - $topBP), "\t", abs($F[8] - $topPval); }' -- -gene1=$gene1 | head -n 1 >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt 
	zcat /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets/Whole_Blood_Analysis_cis-eQTLs.P2.$gene1.gz | sort -g -k 5,5 | sed 's/_/ /g' | perl -slane 'if ($. == 1) { $topBP = $F[2]; $topPval = $F[8]; } if (abs($F[2] - $topBP) > 30000) { print $gene1, "\t", $topBP, "\t", $topPval, "\t", $F[2], "\t", $F[8], "\t", abs($F[2] - $topBP), "\t", abs($F[8] - $topPval); }' -- -gene1=$gene1 | head -n 1 >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.30kbWindowTopResult.txt 
	zcat /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets/Whole_Blood_Analysis_cis-eQTLs.P2.$gene1.gz | sort -g -k 5,5 | sed 's/_/ /g' | perl -slane 'if ($. == 1) { $topBP = $F[2]; $topPval = $F[8]; } if (abs($F[2] - $topBP) > 40000) { print $gene1, "\t", $topBP, "\t", $topPval, "\t", $F[2], "\t", $F[8], "\t", abs($F[2] - $topBP), "\t", abs($F[8] - $topPval); }' -- -gene1=$gene1 | head -n 1 >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.40kbWindowTopResult.txt 
	zcat /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets/Whole_Blood_Analysis_cis-eQTLs.P2.$gene1.gz | sort -g -k 5,5 | sed 's/_/ /g' | perl -slane 'if ($. == 1) { $topBP = $F[2]; $topPval = $F[8]; } if (abs($F[2] - $topBP) > 60000) { print $gene1, "\t", $topBP, "\t", $topPval, "\t", $F[2], "\t", $F[8], "\t", abs($F[2] - $topBP), "\t", abs($F[8] - $topPval); }' -- -gene1=$gene1 | head -n 1 >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.60kbWindowTopResult.txt 
	zcat /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets/Whole_Blood_Analysis_cis-eQTLs.P2.$gene1.gz | sort -g -k 5,5 | sed 's/_/ /g' | perl -slane 'if ($. == 1) { $topBP = $F[2]; $topPval = $F[8]; } if (abs($F[2] - $topBP) > 70000) { print $gene1, "\t", $topBP, "\t", $topPval, "\t", $F[2], "\t", $F[8], "\t", abs($F[2] - $topBP), "\t", abs($F[8] - $topPval); }' -- -gene1=$gene1 | head -n 1 >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.70kbWindowTopResult.txt 
	zcat /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets/Whole_Blood_Analysis_cis-eQTLs.P2.$gene1.gz | sort -g -k 5,5 | sed 's/_/ /g' | perl -slane 'if ($. == 1) { $topBP = $F[2]; $topPval = $F[8]; } if (abs($F[2] - $topBP) > 50000) { print $gene1, "\t", $topBP, "\t", $topPval, "\t", $F[2], "\t", $F[8], "\t", abs($F[2] - $topBP), "\t", abs($F[8] - $topPval); }' -- -gene1=$gene1 | head -n 1 >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.50kbWindowTopResult.txt 
	zcat /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets/Whole_Blood_Analysis_cis-eQTLs.P2.$gene1.gz | sort -g -k 5,5 | sed 's/_/ /g' | perl -slane 'if ($. == 1) { $topBP = $F[2]; $topPval = $F[8]; } if (abs($F[2] - $topBP) > 100000) { print $gene1, "\t", $topBP, "\t", $topPval, "\t", $F[2], "\t", $F[8], "\t", abs($F[2] - $topBP), "\t", abs($F[8] - $topPval); }' -- -gene1=$gene1 | head -n 1 >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.100kbWindowTopResult.txt 
	zcat /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets/Whole_Blood_Analysis_cis-eQTLs.P2.$gene1.gz | sort -g -k 5,5 | sed 's/_/ /g' | perl -slane 'if ($. == 1) { $topBP = $F[2]; $topPval = $F[8]; } if (abs($F[2] - $topBP) > 250000) { print $gene1, "\t", $topBP, "\t", $topPval, "\t", $F[2], "\t", $F[8], "\t", abs($F[2] - $topBP), "\t", abs($F[8] - $topPval); }' -- -gene1=$gene1 | head -n 1 >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.250kbWindowTopResult.txt 
	zcat /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets/Whole_Blood_Analysis_cis-eQTLs.P2.$gene1.gz | sort -g -k 5,5 | sed 's/_/ /g' | perl -slane 'if ($. == 1) { $topBP = $F[2]; $topPval = $F[8]; } if (abs($F[2] - $topBP) > 500000) { print $gene1, "\t", $topBP, "\t", $topPval, "\t", $F[2], "\t", $F[8], "\t", abs($F[2] - $topBP), "\t", abs($F[8] - $topPval); }' -- -gene1=$gene1 | head -n 1 >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.500kbWindowTopResult.txt 
	zcat /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets/Whole_Blood_Analysis_cis-eQTLs.P2.$gene1.gz | sort -g -k 5,5 | sed 's/_/ /g' | perl -slane 'if ($. == 1) { $topBP = $F[2]; $topPval = $F[8]; } if (abs($F[2] - $topBP) > 1000000) { print $gene1, "\t", $topBP, "\t", $topPval, "\t", $F[2], "\t", $F[8], "\t", abs($F[2] - $topBP), "\t", abs($F[8] - $topPval); }' -- -gene1=$gene1 | head -n 1 >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.1MbWindowTopResult.txt 

	zcat /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets/Whole_Blood_Analysis_cis-eQTLs.P2.$gene1.gz | sort -g -k 5,5 | sed 's/_/ /g' | perl -lane 'my @refNTs = split(//, $F[3]); my @altNTs = split(//, $F[4]); if (($#refNTs == 0) && ($#altNTs == 0)) { print join("\t", @F); }' | perl -slane 'if ($. == 1) { $topBP = $F[2]; $topPval = $F[8]; } else { print $gene1, "\t", $topBP, "\t", $topPval, "\t", $F[2], "\t", $F[8], "\t", abs($F[2] - $topBP), "\t", abs($F[8] - $topPval); }' -- -gene1=$gene1 | head -n 1 >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.NoWindowTopResult.txt 
	zcat /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets/Whole_Blood_Analysis_cis-eQTLs.P2.$gene1.gz | sort -g -k 5,5 | sed 's/_/ /g' | perl -lane 'my @refNTs = split(//, $F[3]); my @altNTs = split(//, $F[4]); if (($#refNTs == 0) && ($#altNTs == 0)) { print join("\t", @F); }' | perl -slane 'if ($. == 1) { $topBP = $F[2]; $topPval = $F[8]; } if (abs($F[2] - $topBP) > 10000) { print $gene1, "\t", $topBP, "\t", $topPval, "\t", $F[2], "\t", $F[8], "\t", abs($F[2] - $topBP), "\t", abs($F[8] - $topPval); }' -- -gene1=$gene1 | head -n 1 >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.10kbWindowTopResult.txt 
	zcat /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets/Whole_Blood_Analysis_cis-eQTLs.P2.$gene1.gz | sort -g -k 5,5 | sed 's/_/ /g' | perl -lane 'my @refNTs = split(//, $F[3]); my @altNTs = split(//, $F[4]); if (($#refNTs == 0) && ($#altNTs == 0)) { print join("\t", @F); }' | perl -slane 'if ($. == 1) { $topBP = $F[2]; $topPval = $F[8]; } if (abs($F[2] - $topBP) > 20000) { print $gene1, "\t", $topBP, "\t", $topPval, "\t", $F[2], "\t", $F[8], "\t", abs($F[2] - $topBP), "\t", abs($F[8] - $topPval); }' -- -gene1=$gene1 | head -n 1 >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt 
	zcat /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets/Whole_Blood_Analysis_cis-eQTLs.P2.$gene1.gz | sort -g -k 5,5 | sed 's/_/ /g' | perl -lane 'my @refNTs = split(//, $F[3]); my @altNTs = split(//, $F[4]); if (($#refNTs == 0) && ($#altNTs == 0)) { print join("\t", @F); }' | perl -slane 'if ($. == 1) { $topBP = $F[2]; $topPval = $F[8]; } if (abs($F[2] - $topBP) > 30000) { print $gene1, "\t", $topBP, "\t", $topPval, "\t", $F[2], "\t", $F[8], "\t", abs($F[2] - $topBP), "\t", abs($F[8] - $topPval); }' -- -gene1=$gene1 | head -n 1 >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.30kbWindowTopResult.txt 
	zcat /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets/Whole_Blood_Analysis_cis-eQTLs.P2.$gene1.gz | sort -g -k 5,5 | sed 's/_/ /g' | perl -lane 'my @refNTs = split(//, $F[3]); my @altNTs = split(//, $F[4]); if (($#refNTs == 0) && ($#altNTs == 0)) { print join("\t", @F); }' | perl -slane 'if ($. == 1) { $topBP = $F[2]; $topPval = $F[8]; } if (abs($F[2] - $topBP) > 40000) { print $gene1, "\t", $topBP, "\t", $topPval, "\t", $F[2], "\t", $F[8], "\t", abs($F[2] - $topBP), "\t", abs($F[8] - $topPval); }' -- -gene1=$gene1 | head -n 1 >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.40kbWindowTopResult.txt 
	zcat /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets/Whole_Blood_Analysis_cis-eQTLs.P2.$gene1.gz | sort -g -k 5,5 | sed 's/_/ /g' | perl -lane 'my @refNTs = split(//, $F[3]); my @altNTs = split(//, $F[4]); if (($#refNTs == 0) && ($#altNTs == 0)) { print join("\t", @F); }' | perl -slane 'if ($. == 1) { $topBP = $F[2]; $topPval = $F[8]; } if (abs($F[2] - $topBP) > 60000) { print $gene1, "\t", $topBP, "\t", $topPval, "\t", $F[2], "\t", $F[8], "\t", abs($F[2] - $topBP), "\t", abs($F[8] - $topPval); }' -- -gene1=$gene1 | head -n 1 >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.60kbWindowTopResult.txt 
	zcat /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets/Whole_Blood_Analysis_cis-eQTLs.P2.$gene1.gz | sort -g -k 5,5 | sed 's/_/ /g' | perl -lane 'my @refNTs = split(//, $F[3]); my @altNTs = split(//, $F[4]); if (($#refNTs == 0) && ($#altNTs == 0)) { print join("\t", @F); }' | perl -slane 'if ($. == 1) { $topBP = $F[2]; $topPval = $F[8]; } if (abs($F[2] - $topBP) > 70000) { print $gene1, "\t", $topBP, "\t", $topPval, "\t", $F[2], "\t", $F[8], "\t", abs($F[2] - $topBP), "\t", abs($F[8] - $topPval); }' -- -gene1=$gene1 | head -n 1 >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.70kbWindowTopResult.txt 
	zcat /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets/Whole_Blood_Analysis_cis-eQTLs.P2.$gene1.gz | sort -g -k 5,5 | sed 's/_/ /g' | perl -lane 'my @refNTs = split(//, $F[3]); my @altNTs = split(//, $F[4]); if (($#refNTs == 0) && ($#altNTs == 0)) { print join("\t", @F); }' | perl -slane 'if ($. == 1) { $topBP = $F[2]; $topPval = $F[8]; } if (abs($F[2] - $topBP) > 50000) { print $gene1, "\t", $topBP, "\t", $topPval, "\t", $F[2], "\t", $F[8], "\t", abs($F[2] - $topBP), "\t", abs($F[8] - $topPval); }' -- -gene1=$gene1 | head -n 1 >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.50kbWindowTopResult.txt 
	zcat /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets/Whole_Blood_Analysis_cis-eQTLs.P2.$gene1.gz | sort -g -k 5,5 | sed 's/_/ /g' | perl -lane 'my @refNTs = split(//, $F[3]); my @altNTs = split(//, $F[4]); if (($#refNTs == 0) && ($#altNTs == 0)) { print join("\t", @F); }' | perl -slane 'if ($. == 1) { $topBP = $F[2]; $topPval = $F[8]; } if (abs($F[2] - $topBP) > 100000) { print $gene1, "\t", $topBP, "\t", $topPval, "\t", $F[2], "\t", $F[8], "\t", abs($F[2] - $topBP), "\t", abs($F[8] - $topPval); }' -- -gene1=$gene1 | head -n 1 >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.100kbWindowTopResult.txt 
	zcat /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets/Whole_Blood_Analysis_cis-eQTLs.P2.$gene1.gz | sort -g -k 5,5 | sed 's/_/ /g' | perl -lane 'my @refNTs = split(//, $F[3]); my @altNTs = split(//, $F[4]); if (($#refNTs == 0) && ($#altNTs == 0)) { print join("\t", @F); }' | perl -slane 'if ($. == 1) { $topBP = $F[2]; $topPval = $F[8]; } if (abs($F[2] - $topBP) > 250000) { print $gene1, "\t", $topBP, "\t", $topPval, "\t", $F[2], "\t", $F[8], "\t", abs($F[2] - $topBP), "\t", abs($F[8] - $topPval); }' -- -gene1=$gene1 | head -n 1 >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.250kbWindowTopResult.txt 
	zcat /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets/Whole_Blood_Analysis_cis-eQTLs.P2.$gene1.gz | sort -g -k 5,5 | sed 's/_/ /g' | perl -lane 'my @refNTs = split(//, $F[3]); my @altNTs = split(//, $F[4]); if (($#refNTs == 0) && ($#altNTs == 0)) { print join("\t", @F); }' | perl -slane 'if ($. == 1) { $topBP = $F[2]; $topPval = $F[8]; } if (abs($F[2] - $topBP) > 500000) { print $gene1, "\t", $topBP, "\t", $topPval, "\t", $F[2], "\t", $F[8], "\t", abs($F[2] - $topBP), "\t", abs($F[8] - $topPval); }' -- -gene1=$gene1 | head -n 1 >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.500kbWindowTopResult.txt 
	zcat /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets/Whole_Blood_Analysis_cis-eQTLs.P2.$gene1.gz | sort -g -k 5,5 | sed 's/_/ /g' | perl -lane 'my @refNTs = split(//, $F[3]); my @altNTs = split(//, $F[4]); if (($#refNTs == 0) && ($#altNTs == 0)) { print join("\t", @F); }' | perl -slane 'if ($. == 1) { $topBP = $F[2]; $topPval = $F[8]; } if (abs($F[2] - $topBP) > 1000000) { print $gene1, "\t", $topBP, "\t", $topPval, "\t", $F[2], "\t", $F[8], "\t", abs($F[2] - $topBP), "\t", abs($F[8] - $topPval); }' -- -gene1=$gene1 | head -n 1 >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.1MbWindowTopResult.txt 
	
done

cat <(echo "pValThresh 10kb 20kb 50kb 100kb 250kb 500kb 1Mb") \ 
<(echo "1e-4" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.10kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.50kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.100kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.250kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.500kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.1MbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'`) \ 
<(echo "1e-5" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.10kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.50kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.100kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.250kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.500kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.1MbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc | awk '{ print $1 }'`) \ 
<(echo "1e-6" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.10kbWindowTopResult.txt | awk '{ if (($3 <= .000001) && ($5 <= .000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if (($3 <= .000001) && ($5 <= .000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.50kbWindowTopResult.txt | awk '{ if (($3 <= .000001) && ($5 <= .000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.100kbWindowTopResult.txt | awk '{ if (($3 <= .000001) && ($5 <= .000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.250kbWindowTopResult.txt | awk '{ if (($3 <= .000001) && ($5 <= .000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.500kbWindowTopResult.txt | awk '{ if (($3 <= .000001) && ($5 <= .000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.1MbWindowTopResult.txt | awk '{ if (($3 <= .000001) && ($5 <= .000001)) { print $0; } } ' | wc | awk '{ print $1 }'`) \ 
<(echo "1e-7" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.10kbWindowTopResult.txt | awk '{ if (($3 <= .0000001) && ($5 <= .0000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if (($3 <= .0000001) && ($5 <= .0000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.50kbWindowTopResult.txt | awk '{ if (($3 <= .0000001) && ($5 <= .0000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.100kbWindowTopResult.txt | awk '{ if (($3 <= .0000001) && ($5 <= .0000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.250kbWindowTopResult.txt | awk '{ if (($3 <= .0000001) && ($5 <= .0000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.500kbWindowTopResult.txt | awk '{ if (($3 <= .0000001) && ($5 <= .0000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.1MbWindowTopResult.txt | awk '{ if (($3 <= .0000001) && ($5 <= .0000001)) { print $0; } } ' | wc | awk '{ print $1 }'`) \ 
<(echo "1e-8" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.10kbWindowTopResult.txt | awk '{ if (($3 <= .00000001) && ($5 <= .00000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if (($3 <= .00000001) && ($5 <= .00000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.50kbWindowTopResult.txt | awk '{ if (($3 <= .00000001) && ($5 <= .00000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.100kbWindowTopResult.txt | awk '{ if (($3 <= .00000001) && ($5 <= .00000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.250kbWindowTopResult.txt | awk '{ if (($3 <= .00000001) && ($5 <= .00000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.500kbWindowTopResult.txt | awk '{ if (($3 <= .00000001) && ($5 <= .00000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.1MbWindowTopResult.txt | awk '{ if (($3 <= .00000001) && ($5 <= .00000001)) { print $0; } } ' | wc | awk '{ print $1 }'`) \ 
<(echo "1e-9" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.10kbWindowTopResult.txt | awk '{ if (($3 <= .000000001) && ($5 <= .000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if (($3 <= .000000001) && ($5 <= .000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.50kbWindowTopResult.txt | awk '{ if (($3 <= .000000001) && ($5 <= .000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.100kbWindowTopResult.txt | awk '{ if (($3 <= .000000001) && ($5 <= .000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.250kbWindowTopResult.txt | awk '{ if (($3 <= .000000001) && ($5 <= .000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.500kbWindowTopResult.txt | awk '{ if (($3 <= .000000001) && ($5 <= .000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.1MbWindowTopResult.txt | awk '{ if (($3 <= .000000001) && ($5 <= .000000001)) { print $0; } } ' | wc | awk '{ print $1 }'`) \ 
<(echo "1e-10" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.10kbWindowTopResult.txt | awk '{ if (($3 <= .0000000001) && ($5 <= .0000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if (($3 <= .0000000001) && ($5 <= .0000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.50kbWindowTopResult.txt | awk '{ if (($3 <= .0000000001) && ($5 <= .0000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.100kbWindowTopResult.txt | awk '{ if (($3 <= .0000000001) && ($5 <= .0000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.250kbWindowTopResult.txt | awk '{ if (($3 <= .0000000001) && ($5 <= .0000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.500kbWindowTopResult.txt | awk '{ if (($3 <= .0000000001) && ($5 <= .0000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.1MbWindowTopResult.txt | awk '{ if (($3 <= .0000000001) && ($5 <= .0000000001)) { print $0; } } ' | wc | awk '{ print $1 }'`)

cat <(echo "SNPs only") <(echo "pValThresh 10kb 20kb 50kb 100kb 250kb 500kb 1Mb") \ 
<(echo "1e-4" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.10kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.50kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.100kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.250kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.500kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.1MbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'`) \ 
<(echo "1e-5" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.10kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.50kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.100kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.250kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.500kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.1MbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc | awk '{ print $1 }'`) \ 
<(echo "1e-6" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.10kbWindowTopResult.txt | awk '{ if (($3 <= .000001) && ($5 <= .000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if (($3 <= .000001) && ($5 <= .000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.50kbWindowTopResult.txt | awk '{ if (($3 <= .000001) && ($5 <= .000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.100kbWindowTopResult.txt | awk '{ if (($3 <= .000001) && ($5 <= .000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.250kbWindowTopResult.txt | awk '{ if (($3 <= .000001) && ($5 <= .000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.500kbWindowTopResult.txt | awk '{ if (($3 <= .000001) && ($5 <= .000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.1MbWindowTopResult.txt | awk '{ if (($3 <= .000001) && ($5 <= .000001)) { print $0; } } ' | wc | awk '{ print $1 }'`) \ 
<(echo "1e-7" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.10kbWindowTopResult.txt | awk '{ if (($3 <= .0000001) && ($5 <= .0000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if (($3 <= .0000001) && ($5 <= .0000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.50kbWindowTopResult.txt | awk '{ if (($3 <= .0000001) && ($5 <= .0000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.100kbWindowTopResult.txt | awk '{ if (($3 <= .0000001) && ($5 <= .0000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.250kbWindowTopResult.txt | awk '{ if (($3 <= .0000001) && ($5 <= .0000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.500kbWindowTopResult.txt | awk '{ if (($3 <= .0000001) && ($5 <= .0000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.1MbWindowTopResult.txt | awk '{ if (($3 <= .0000001) && ($5 <= .0000001)) { print $0; } } ' | wc | awk '{ print $1 }'`) \ 
<(echo "1e-8" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.10kbWindowTopResult.txt | awk '{ if (($3 <= .00000001) && ($5 <= .00000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if (($3 <= .00000001) && ($5 <= .00000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.50kbWindowTopResult.txt | awk '{ if (($3 <= .00000001) && ($5 <= .00000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.100kbWindowTopResult.txt | awk '{ if (($3 <= .00000001) && ($5 <= .00000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.250kbWindowTopResult.txt | awk '{ if (($3 <= .00000001) && ($5 <= .00000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.500kbWindowTopResult.txt | awk '{ if (($3 <= .00000001) && ($5 <= .00000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.1MbWindowTopResult.txt | awk '{ if (($3 <= .00000001) && ($5 <= .00000001)) { print $0; } } ' | wc | awk '{ print $1 }'`) \ 
<(echo "1e-9" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.10kbWindowTopResult.txt | awk '{ if (($3 <= .000000001) && ($5 <= .000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if (($3 <= .000000001) && ($5 <= .000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.50kbWindowTopResult.txt | awk '{ if (($3 <= .000000001) && ($5 <= .000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.100kbWindowTopResult.txt | awk '{ if (($3 <= .000000001) && ($5 <= .000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.250kbWindowTopResult.txt | awk '{ if (($3 <= .000000001) && ($5 <= .000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.500kbWindowTopResult.txt | awk '{ if (($3 <= .000000001) && ($5 <= .000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.1MbWindowTopResult.txt | awk '{ if (($3 <= .000000001) && ($5 <= .000000001)) { print $0; } } ' | wc | awk '{ print $1 }'`) \ 
<(echo "1e-10" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.10kbWindowTopResult.txt | awk '{ if (($3 <= .0000000001) && ($5 <= .0000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if (($3 <= .0000000001) && ($5 <= .0000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.50kbWindowTopResult.txt | awk '{ if (($3 <= .0000000001) && ($5 <= .0000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.100kbWindowTopResult.txt | awk '{ if (($3 <= .0000000001) && ($5 <= .0000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.250kbWindowTopResult.txt | awk '{ if (($3 <= .0000000001) && ($5 <= .0000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.500kbWindowTopResult.txt | awk '{ if (($3 <= .0000000001) && ($5 <= .0000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.1MbWindowTopResult.txt | awk '{ if (($3 <= .0000000001) && ($5 <= .0000000001)) { print $0; } } ' | wc | awk '{ print $1 }'`)

cat <(echo "pValThresh 10kb 20kb 30kb 40kb 50kb 60kb 70kb") \
<(echo "1e-4" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.10kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.30kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.40kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.50kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.60kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.70kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'`) \ 
<(echo "1e-5" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.10kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.30kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.40kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.50kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.60kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.70kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc | awk '{ print $1 }'`) \ 
<(echo "1e-6" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.10kbWindowTopResult.txt | awk '{ if (($3 <= .000001) && ($5 <= .000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if (($3 <= .000001) && ($5 <= .000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.30kbWindowTopResult.txt | awk '{ if (($3 <= .000001) && ($5 <= .000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.40kbWindowTopResult.txt | awk '{ if (($3 <= .000001) && ($5 <= .000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.50kbWindowTopResult.txt | awk '{ if (($3 <= .000001) && ($5 <= .000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.60kbWindowTopResult.txt | awk '{ if (($3 <= .000001) && ($5 <= .000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.70kbWindowTopResult.txt | awk '{ if (($3 <= .000001) && ($5 <= .000001)) { print $0; } } ' | wc | awk '{ print $1 }'`) \ 
<(echo "1e-7" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.10kbWindowTopResult.txt | awk '{ if (($3 <= .0000001) && ($5 <= .0000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if (($3 <= .0000001) && ($5 <= .0000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.30kbWindowTopResult.txt | awk '{ if (($3 <= .0000001) && ($5 <= .0000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.40kbWindowTopResult.txt | awk '{ if (($3 <= .0000001) && ($5 <= .0000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.50kbWindowTopResult.txt | awk '{ if (($3 <= .0000001) && ($5 <= .0000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.60kbWindowTopResult.txt | awk '{ if (($3 <= .0000001) && ($5 <= .0000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.70kbWindowTopResult.txt | awk '{ if (($3 <= .0000001) && ($5 <= .0000001)) { print $0; } } ' | wc | awk '{ print $1 }'`) \ 
<(echo "1e-8" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.10kbWindowTopResult.txt | awk '{ if (($3 <= .00000001) && ($5 <= .00000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if (($3 <= .00000001) && ($5 <= .00000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.30kbWindowTopResult.txt | awk '{ if (($3 <= .00000001) && ($5 <= .00000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.40kbWindowTopResult.txt | awk '{ if (($3 <= .00000001) && ($5 <= .00000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.50kbWindowTopResult.txt | awk '{ if (($3 <= .00000001) && ($5 <= .00000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.60kbWindowTopResult.txt | awk '{ if (($3 <= .00000001) && ($5 <= .00000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.70kbWindowTopResult.txt | awk '{ if (($3 <= .00000001) && ($5 <= .00000001)) { print $0; } } ' | wc | awk '{ print $1 }'`) \ 
<(echo "1e-9" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.10kbWindowTopResult.txt | awk '{ if (($3 <= .000000001) && ($5 <= .000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if (($3 <= .000000001) && ($5 <= .000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.30kbWindowTopResult.txt | awk '{ if (($3 <= .000000001) && ($5 <= .000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.40kbWindowTopResult.txt | awk '{ if (($3 <= .000000001) && ($5 <= .000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.50kbWindowTopResult.txt | awk '{ if (($3 <= .000000001) && ($5 <= .000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.60kbWindowTopResult.txt | awk '{ if (($3 <= .000000001) && ($5 <= .000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.70kbWindowTopResult.txt | awk '{ if (($3 <= .000000001) && ($5 <= .000000001)) { print $0; } } ' | wc | awk '{ print $1 }'`) 

cat <(echo "SNPs only") <(echo "pValThresh 10kb 20kb 30kb 40kb 50kb 60kb 70kb") \
<(echo "1e-4" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.10kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.30kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.40kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.50kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.60kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.70kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'`) \ 
<(echo "1e-5" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.10kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.30kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.40kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.50kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.60kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.70kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc | awk '{ print $1 }'`) \ 
<(echo "1e-6" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.10kbWindowTopResult.txt | awk '{ if (($3 <= .000001) && ($5 <= .000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if (($3 <= .000001) && ($5 <= .000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.30kbWindowTopResult.txt | awk '{ if (($3 <= .000001) && ($5 <= .000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.40kbWindowTopResult.txt | awk '{ if (($3 <= .000001) && ($5 <= .000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.50kbWindowTopResult.txt | awk '{ if (($3 <= .000001) && ($5 <= .000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.60kbWindowTopResult.txt | awk '{ if (($3 <= .000001) && ($5 <= .000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.70kbWindowTopResult.txt | awk '{ if (($3 <= .000001) && ($5 <= .000001)) { print $0; } } ' | wc | awk '{ print $1 }'`) \ 
<(echo "1e-7" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.10kbWindowTopResult.txt | awk '{ if (($3 <= .0000001) && ($5 <= .0000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if (($3 <= .0000001) && ($5 <= .0000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.30kbWindowTopResult.txt | awk '{ if (($3 <= .0000001) && ($5 <= .0000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.40kbWindowTopResult.txt | awk '{ if (($3 <= .0000001) && ($5 <= .0000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.50kbWindowTopResult.txt | awk '{ if (($3 <= .0000001) && ($5 <= .0000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.60kbWindowTopResult.txt | awk '{ if (($3 <= .0000001) && ($5 <= .0000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.70kbWindowTopResult.txt | awk '{ if (($3 <= .0000001) && ($5 <= .0000001)) { print $0; } } ' | wc | awk '{ print $1 }'`) \ 
<(echo "1e-8" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.10kbWindowTopResult.txt | awk '{ if (($3 <= .00000001) && ($5 <= .00000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if (($3 <= .00000001) && ($5 <= .00000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.30kbWindowTopResult.txt | awk '{ if (($3 <= .00000001) && ($5 <= .00000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.40kbWindowTopResult.txt | awk '{ if (($3 <= .00000001) && ($5 <= .00000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.50kbWindowTopResult.txt | awk '{ if (($3 <= .00000001) && ($5 <= .00000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.60kbWindowTopResult.txt | awk '{ if (($3 <= .00000001) && ($5 <= .00000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.70kbWindowTopResult.txt | awk '{ if (($3 <= .00000001) && ($5 <= .00000001)) { print $0; } } ' | wc | awk '{ print $1 }'`) \ 
<(echo "1e-9" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.10kbWindowTopResult.txt | awk '{ if (($3 <= .000000001) && ($5 <= .000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if (($3 <= .000000001) && ($5 <= .000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.30kbWindowTopResult.txt | awk '{ if (($3 <= .000000001) && ($5 <= .000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.40kbWindowTopResult.txt | awk '{ if (($3 <= .000000001) && ($5 <= .000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.50kbWindowTopResult.txt | awk '{ if (($3 <= .000000001) && ($5 <= .000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.60kbWindowTopResult.txt | awk '{ if (($3 <= .000000001) && ($5 <= .000000001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.70kbWindowTopResult.txt | awk '{ if (($3 <= .000000001) && ($5 <= .000000001)) { print $0; } } ' | wc | awk '{ print $1 }'`) 















cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.NoWindowTopResult.txt | awk '{ print $6 }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));"
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.NoWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $6 } }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));"
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.10kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $6 } }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));"
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $6 } }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));"
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.30kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $6 } }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));"
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.40kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $6 } }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));"
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.50kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $6 } }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));"
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.60kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $6 } }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));"
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.70kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $6 } }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));"
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.100kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $6 } }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));"
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.250kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $6 } }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));"
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $5 } }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,1e-3,1e-4,1e-5,1e-6,1e-7,1e-8,1e-9,1e-10), right=FALSE));"
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.30kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $5 } }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,1e-3,1e-4,1e-5,1e-6,1e-7,1e-8,1e-9,1e-10), right=FALSE));"

~~~
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.10kbWindowTopResult.txt | awk '{ if ($3 <= .00001) { print $0; } }' | wc
    245    1715   16420
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.10kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 > .00001)) { print $0; } } ' | wc 
     60     420    4112
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.10kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc                                                                                                              
    185    1295   12308                                                                                                                                                              
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc                                                                                                              
    170    1190   11413                                                                                                                                                                                                        
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.50kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc                                                                                                              
    127     889    8612                                                                                                                                                                                                        
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.100kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc                                                                                                              
     92     644    6338                                                                                                                                                                                                        
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.500kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } ' | wc                                                                                                              
     21     147    1444                                                                                                                                                                                                        
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.NoWindowTopResult.txt | awk '{ if ($3 < .0000000001) { print $0 } } ' | wc
     91     637    5797
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.NoWindowTopResult.txt | awk '{ if ($3 < .000000001) { print $0 } } ' | wc
    103     721    6591
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.NoWindowTopResult.txt | awk '{ if ($3 < .00000001) { print $0 } } ' | wc
    119     833    7654
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.NoWindowTopResult.txt | awk '{ if ($3 < .0000001) { print $0 } } ' | wc
    147    1029    9486
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.NoWindowTopResult.txt | awk '{ if ($3 < .000001) { print $0 } } ' | wc
    181    1267   11731
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.NoWindowTopResult.txt | awk '{ if ($3 < .00001) { print $0 } } ' | wc
    245    1715   15984
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.NoWindowTopResult.txt | awk '{ if ($3 < .0001) { print $0 } } ' | wc
    376    2632   24663
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.NoWindowTopResult.txt | awk '{ if ($3 < .001) { print $0 } } ' | wc
    622    4354   41063
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.500kbWindowTopResult.txt | awk '{ if (($3 <= .00001) && ($5 <= .00001)) { print $0; } } '                                                                                                              
PRRC2A  31595487        4.91654e-12     32099817        2.64337e-06     504330  2.64336508346e-06                                                                                                                              
TRAF3IP3        209887718       2.03414e-17     210460909       1.1133e-13      573191  1.113096586e-13                                                                                                                        
MAST2   46193146        9.2444e-08      46698490        6.65488e-06     505344  6.562436e-06
PML     74256841        2.04209e-08     74974908        4.75402e-06     718067  4.7335991e-06
SPNS1   28995757        1.66078e-24     28383243        5.08136e-12     612514  5.08135999999834e-12
MZF1    58372507        1.93825e-06     59083594        6.9923e-06      711087  5.05405e-06
P2RY11  10221808        1.26922e-06     9714156 6.08131e-06     507652  4.81209e-06
CARHSP1 8930984 1.17062e-12     9673492 4.10983e-06     742508  4.10982882938e-06
SGSH    77267349        4.48985e-06     78180243        5.7801e-06      912894  1.29025e-06
GDPD3   29834697        1.11177e-10     30435601        1.67332e-09     600904  1.562143e-09
CRHR1   43493834        2.95915e-08     44337903        9.63501e-08     844069  6.67586e-08
NFASC   204940780       5.29496e-17     204156147       4.09883e-08     784633  4.09882999470504e-08
PSMB9   32830013        4.03845e-31     32131510        3.97569e-06     698503  3.97569e-06
CHRNB2  154554357       3.10873e-09     153661845       6.91625e-06     892512  6.91314127e-06
F2R     76037551        1.20082e-06     75292649        7.93979e-06     744902  6.73897e-06
ATP5G3  175924184       3.15161e-15     176429275       8.29344e-06     505091  8.29343999684839e-06
CST7    24914590        9.14127e-22     24063534        9.52812e-06     851056  9.52812e-06
HLA-DRB1        32419959        0       31845985        6.78005e-21     573974  6.78005e-21
ISG15   948870  9.29576e-17     1692046 4.42004e-07     743176  4.42003999907042e-07
CHST13  126228410       0       126796501       5.51028e-07     568091  5.51028e-07
SULT1A2 28485141        0       28995757        8.28204e-25     510616  8.28204e-25
[  michaelt@node12  ~]$cat <(echo "pValThresh 10kb 20kb 50kb 100kb 250kb 500kb 1Mb") <(echo "1e-4" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.10kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.50kbWindowTopRes
.
.
pValThresh 10kb 20kb 50kb 100kb 250kb 500kb 1Mb
1e-4 289 276 229 184 118 70 8
1e-5 185 170 127 92 46 21 1
1e-6 137 125 94 58 26 9 1
1e-7 109 104 71 48 19 7 1
1e-8 91 80 59 38 13 5 1
1e-9 76 66 50 29 10 4 0
1e-10 69 61 42 25 8 4 0
[  michaelt@node3  ~]$cat <(echo "pValThresh 10kb 20kb 30kb 40kb 50kb 60kb 70kb") \
> <(echo "1e-4" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.10kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.30kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/A.
.
pValThresh 10kb 20kb 30kb 40kb 50kb 60kb 70kb
1e-4 289 276 257 245 229 224 212
1e-5 185 170 151 136 127 120 110
1e-6 137 125 112 102 94 81 68
1e-7 109 104 90 77 71 67 55
1e-8 91 80 69 62 59 55 44
1e-9 76 66 59 51 50 47 38
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.NoWindowTopResult.txt | awk '{ print $6 }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));"
> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));

      [0,5e+03)   [5e+03,1e+04)   [1e+04,2e+04)   [2e+04,3e+04)   [3e+04,4e+04) 
            217              62              55              32              28 
  [4e+04,5e+04)   [5e+04,6e+04)   [6e+04,7e+04)   [7e+04,1e+05) [1e+05,2.5e+05) 
             20              12               6              24              50 
[2.5e+05,5e+05)   [5e+05,1e+06) 
             52              93 
> 
> 
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.NoWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $6 } }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));"
> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));

      [0,5e+03)   [5e+03,1e+04)   [1e+04,2e+04)   [2e+04,3e+04)   [3e+04,4e+04) 
            143              40              27              24              17 
  [4e+04,5e+04)   [5e+04,6e+04)   [6e+04,7e+04)   [7e+04,1e+05) [1e+05,2.5e+05) 
             11               6               4              12              14 
[2.5e+05,5e+05)   [5e+05,1e+06) 
              7              12 
> 
> 
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.10kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $6 } }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));"
> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));

      [0,5e+03)   [5e+03,1e+04)   [1e+04,2e+04)   [2e+04,3e+04)   [3e+04,4e+04) 
              0               0             110              47              33 
  [4e+04,5e+04)   [5e+04,6e+04)   [6e+04,7e+04)   [7e+04,1e+05) [1e+05,2.5e+05) 
             15              14               5              16              22 
[2.5e+05,5e+05)   [5e+05,1e+06) 
             11              13 
> 
> 
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $6 } }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));"
> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));

      [0,5e+03)   [5e+03,1e+04)   [1e+04,2e+04)   [2e+04,3e+04)   [3e+04,4e+04) 
              0               0               0             106              46 
  [4e+04,5e+04)   [5e+04,6e+04)   [6e+04,7e+04)   [7e+04,1e+05) [1e+05,2.5e+05) 
             22              15              10              20              26 
[2.5e+05,5e+05)   [5e+05,1e+06) 
             12              16 
> 
> 
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.30kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $6 } }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));"
> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));

      [0,5e+03)   [5e+03,1e+04)   [1e+04,2e+04)   [2e+04,3e+04)   [3e+04,4e+04) 
              0               0               0               0              90 
  [4e+04,5e+04)   [5e+04,6e+04)   [6e+04,7e+04)   [7e+04,1e+05) [1e+05,2.5e+05) 
             31              21              19              28              32 
[2.5e+05,5e+05)   [5e+05,1e+06) 
             14              19 
> 
> 
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.40kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $6 } }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));"
> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));

      [0,5e+03)   [5e+03,1e+04)   [1e+04,2e+04)   [2e+04,3e+04)   [3e+04,4e+04) 
              0               0               0               0               0 
  [4e+04,5e+04)   [5e+04,6e+04)   [6e+04,7e+04)   [7e+04,1e+05) [1e+05,2.5e+05) 
             69              31              25              38              38 
[2.5e+05,5e+05)   [5e+05,1e+06) 
             17              24 
> 
> 
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.50kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $6 } }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));"
> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));

      [0,5e+03)   [5e+03,1e+04)   [1e+04,2e+04)   [2e+04,3e+04)   [3e+04,4e+04) 
              0               0               0               0               0 
  [4e+04,5e+04)   [5e+04,6e+04)   [6e+04,7e+04)   [7e+04,1e+05) [1e+05,2.5e+05) 
              0              58              32              47              43 
[2.5e+05,5e+05)   [5e+05,1e+06) 
             20              26 
> 
> 
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.60kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $6 } }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));"
> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));

      [0,5e+03)   [5e+03,1e+04)   [1e+04,2e+04)   [2e+04,3e+04)   [3e+04,4e+04) 
              0               0               0               0               0 
  [4e+04,5e+04)   [5e+04,6e+04)   [6e+04,7e+04)   [7e+04,1e+05) [1e+05,2.5e+05) 
              0               0              65              56              51 
[2.5e+05,5e+05)   [5e+05,1e+06) 
             21              28 
> 
> 
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.70kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $6 } }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));"
> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));

      [0,5e+03)   [5e+03,1e+04)   [1e+04,2e+04)   [2e+04,3e+04)   [3e+04,4e+04) 
              0               0               0               0               0 
  [4e+04,5e+04)   [5e+04,6e+04)   [6e+04,7e+04)   [7e+04,1e+05) [1e+05,2.5e+05) 
              0               0               0              91              66 
[2.5e+05,5e+05)   [5e+05,1e+06) 
             21              30 
> 
> 
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.100kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $6 } }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));"
> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));

      [0,5e+03)   [5e+03,1e+04)   [1e+04,2e+04)   [2e+04,3e+04)   [3e+04,4e+04) 
              0               0               0               0               0 
  [4e+04,5e+04)   [5e+04,6e+04)   [6e+04,7e+04)   [7e+04,1e+05) [1e+05,2.5e+05) 
              0               0               0               0             116 
[2.5e+05,5e+05)   [5e+05,1e+06) 
             30              34 
> 
> 
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.250kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $6 } }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));"
> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,5000,10000,20000,30000,40000,50000,60000,70000,100000,250000,500000,1000000), right=FALSE));

      [0,5e+03)   [5e+03,1e+04)   [1e+04,2e+04)   [2e+04,3e+04)   [3e+04,4e+04)
              0               0               0               0               0
  [4e+04,5e+04)   [5e+04,6e+04)   [6e+04,7e+04)   [7e+04,1e+05) [1e+05,2.5e+05)
              0               0               0               0               0
[2.5e+05,5e+05)   [5e+05,1e+06)
             67              47
>
>
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $5 } }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,1e-3,1e-4,1e-5,1e-6,1e-7,1e-8,1e-9,1e-10), right=FALSE));"
> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,1e-3,1e-4,1e-5,1e-6,1e-7,1e-8,1e-9,1e-10), right=FALSE));

     [0,1e-10)  [1e-10,1e-09)  [1e-09,1e-08)  [1e-08,1e-07)  [1e-07,1e-06) 
            61              5             14             24             21 
 [1e-06,1e-05) [1e-05,0.0001) [0.0001,0.001) 
            45            106              0 
> 
> 
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.30kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $5 } }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,1e-3,1e-4,1e-5,1e-6,1e-7,1e-8,1e-9,1e-10), right=FALSE));"
> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1],c(0,1e-3,1e-4,1e-5,1e-6,1e-7,1e-8,1e-9,1e-10), right=FALSE));

     [0,1e-10)  [1e-10,1e-09)  [1e-09,1e-08)  [1e-08,1e-07)  [1e-07,1e-06) 
            52              7             10             21             22 
 [1e-06,1e-05) [1e-05,0.0001) [0.0001,0.001) 
            39            106              0 
> 
> 
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | sort -g -k 6,6 | head -n 10                                                                                                                                                                                                                                                                                                                      MPP2    41987441        7.06877e-06     41967346        1.01197e-05     20095   3.05093e-06                                                                                                                                                                                                                                HMOX1   36289714        8.51361e-05     36269528        8.52212e-05     20186   8.50999999999901e-08                                                                                                                                                                                                                       HERC6   89306659        2.42847e-07     89286459        2.22922e-06     20200   1.986373e-06                                                                                                                                                                                                                               IFITM3  306920  1.92589e-33     327143  2.33435e-11     20223   2.33435e-11                                                                                                                                                                                                                                                PDXK    45158689        2.23577e-11     45138420        1.02534e-08     20269   1.02310423e-08                                                                                                                                                                                                                             APOBEC3H        39487258        3.58035e-17     39507832        1.81897e-12     20574   1.8189341965e-12                                                                                                                                                                                                                   CASP3   185574807       1.7972e-08      185595472       1.29152e-06     20665   1.273548e-06                                                                                                                                                                                                                               C1orf86 2139307 9.15955e-10     2118539 2.66592e-09     20768   1.749965e-09                                                                                                                                                                                                                                               DCHS1   6687199 7.43523e-10     6708096 9.51388e-06     20897   9.513136477e-06                                                                                                                                                                                                                                            HS2ST1  86446548        3.6799e-06      86425628        3.33935e-05     20920   2.97136e-05         
#20160802 NOTE -- Looked at some of the 'top genes that are closest to one another' given our cutoff criteria in the GTEX eQTL browser. The goal was to see if there was any visual evidence to help suggest 20kb or 30kb was a better cutoff window. Looks like eQTL windows rarely reach 20kb -- in some instances (like TMPRSS9) there 'may be a clear' secondary eQTL signal that's closer within say 10kb-15kb. But that type of example doesn't seem consistent, at least from looking at most of the top 10 example shown above. Does seem like by 20kb you really are getting outside the range of 'tagging the same region' in terms of signal/locus redundancy. 30kb is probably unnecessary then?
[  michaelt@node3  ~]$cat <(echo "SNPs only") <(echo "pValThresh 10kb 20kb 50kb 100kb 250kb 500kb 1Mb") \                                                                                                                       > <(echo "1e-4" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.10kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.50kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print.
.
SNPs only
pValThresh 10kb 20kb 50kb 100kb 250kb 500kb 1Mb
1e-4 283 267 222 179 109 63 8
1e-5 184 169 125 93 43 19 1
1e-6 135 124 93 58 25 9 1
1e-7 109 102 71 47 19 7 1
1e-8 89 79 58 35 13 5 1
1e-9 75 65 49 26 9 4 0
1e-10 69 61 41 23 8 4 0
[  michaelt@node3  ~]$cat <(echo "SNPs only") <(echo "pValThresh 10kb 20kb 30kb 40kb 50kb 60kb 70kb") \
> <(echo "1e-4" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.10kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } } ' | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.30kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print.
.
pValThresh 10kb 20kb 30kb 40kb 50kb 60kb 70kb
1e-4 283 267 247 235 222 216 206
1e-5 184 169 149 136 125 117 110
1e-6 135 124 111 102 93 80 69
1e-7 109 102 88 77 71 66 56
1e-8 89 79 68 61 58 54 43
1e-9 75 65 58 51 49 44 37
~~~

#20160803 NOTE -- Doing less than p-value thresh of 1e-4, not less than or equal to thresh of 1e-4

for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160120_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | sort -g -k 2,2 | grep -v NUM_PASS_VARS | awk '{ print $1 }'`; do

#        zcat /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets/Whole_Blood_Analysis_cis-eQTLs.P2.$gene1.gz | sort -g -k 5,5 | sed 's/_/ /g' | perl -slane 'if ($. == 1) { if ($F[8] < .0001) { $topBP = $F[2]; $topPval = $F[8]; print $gene1, "\t", join("\t", @F); } } if (abs($F[2] - $topBP) > 20000) { if ($F[8] < .0001) { print $gene1, "\t", join("\t", @F); } }' -- -gene1=$gene1 | head -n 2 >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.txt
        zcat /home/michaelt/Data/GTEX/Vs6/WholeBloodSubsets/Whole_Blood_Analysis_cis-eQTLs.P2.$gene1.gz | sort -g -k 5,5 | sed 's/_/ /g' | perl -lane 'my @refNTs = split(//, $F[3]); my @altNTs = split(//, $F[4]); if (($#refNTs == 0) && ($#altNTs == 0)) { print join("\t", @F); }' | perl -slane 'if ($. == 1) { if ($F[8] < .0001) { $topBP = $F[2]; $topPval = $F[8]; print $gene1, "\t", join("\t", @F); } } if (abs($F[2] - $topBP) > 20000) { if ($F[8] < .0001) { print $gene1, "\t", join("\t", @F); } }' -- -gene1=$gene1 | head -n 2 >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.FinalResults.20kbWindow1eNeg4pVal.txt

done

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.FinalResults.20kbWindow1eNeg4pVal.txt | wc
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.FinalResults.20kbWindow1eNeg4pVal.txt | awk '{ print $1 }' | sort | uniq -c | awk '{ print $1 }' | sort | uniq -c
join -j 2 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; print $F[0], "\t", $F[3], "\t", $F[4];' | sort -k 2,2) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if ($3 <= .0001) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; ' | sort -k 2,2) | wc
join -j 2 -v 1 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; print $F[0], "\t", $F[3], "\t", $F[4];' | sort -k 2,2) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if ($3 <= .0001) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; ' | sort -k 2,2) | wc
join -j 2 -a 1 -a 2 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; print $F[0], "\t", $F[3], "\t", $F[4];' | sort -k 2,2) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if ($3 <= .0001) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; ' | sort -k 2,2) | wc
join -j 2 -v 2 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; print $F[0], "\t", $F[3], "\t", $F[4];' | sort -k 2,2) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if ($3 <= .0001) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; ' | sort -k 2,2) | wc
join -j 2 -a 1 -a 2 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; print $F[0], "\t", $F[3], "\t", $F[4];' | sort -k 2,2) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if ($3 <= .0001) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; ' | sort -k 2,2) | awk '{ print $1 }' | sort | uniq -c | sort -r -k 1,1 | head -n 10

~~~
[  michaelt@node12  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.txt | wc
    652    6520   48189
[  michaelt@node12  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.txt | head -n 10
XAF1    ENSG00000132530.12      17      6496584 C       T       b37     0.212161        4.14956 4.35848e-05
XAF1    ENSG00000132530.12      17      5746484 AG      A       b37     0.409112        4.10525 5.22899e-05
PDXK    ENSG00000160209.14      21      45158689        T       C       b37     0.210145        6.95699 2.23577e-11
PDXK    ENSG00000160209.14      21      45138420        C       G       b37     0.176919        5.89402 1.02534e-08
MAPK14  ENSG00000112062.6       6       36653477        C       T       b37     -0.348144       -4.70505        3.90095e-06
MAPK14  ENSG00000112062.6       6       36015536        A       G       b37     -0.125767       -4.43858        1.27935e-05
PRRC2A  ENSG00000204469.8       6       31595487        A       G       b37     0.157237        7.20246 4.91654e-12
PRRC2A  ENSG00000204469.8       6       31637734        A       T       b37     0.142092        5.86154 1.22214e-08
UBE2J1  ENSG00000198833.6       6       89670409        G       A       b37     0.428912        4.29303 2.39131e-05
UBE2J1  ENSG00000198833.6       6       89606741        G       A       b37     0.408173        4.23664 3.03358e-05
[  michaelt@node12  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.txt | awk '{ print $1 }' | sort | uniq -c | awk '{ print $1 }' | sort | uniq -c
    100 1
    276 2
[  michaelt@node12  ~]$join -j 2 -a 1 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; print $F[0], "\t", $F[3], "\t", $F[4];' | sort -k 2,2) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if ($3 <= .0001) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; ' | sort -k 2,2) | wc 
    552    2210   19794
[  michaelt@node12  ~]$join -j 2 -v 1 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; print $F[0], "\t", $F[3], "\t", $F[4];' | sort -k 2,2) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if ($3 <= .0001) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; ' | sort -k 2,2) | wc 
    275     825    7430
[  michaelt@node12  ~]$join -j 2 -a 1 -a 2 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; print $F[0], "\t", $F[3], "\t", $F[4];' | sort -k 2,2) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if ($3 <= .0001) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; ' | sort -k 2,2) | wc 
    652    2510   22497
[  michaelt@node12  ~]$join -j 2 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; print $F[0], "\t", $F[3], "\t", $F[4];' | sort -k 2,2) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if ($3 <= .0001) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; ' | sort -k 2,2) | wc
    277    1385   12364
[  michaelt@node12  ~]$join -j 2 -a 2 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; print $F[0], "\t", $F[3], "\t", $F[4];' | sort -k 2,2) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if ($3 <= .0001) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; ' | sort -k 2,2) | wc
    377    1685   15067
[  michaelt@node12  ~]$join -j 2 -v 2 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; print $F[0], "\t", $F[3], "\t", $F[4];' | sort -k 2,2) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if ($3 <= .0001) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; ' | sort -k 2,2) | wc
    100     300    2703
[  michaelt@node12  ~]$join -j 2 -v 2 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; print $F[0], "\t", $F[3], "\t", $F[4];' | sort -k 2,2) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if ($3 <= .0001) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; ' | sort -k 2,2) | head -n 10
102307500 PABPC1 6.3755e-05
102477453 IL1R1 7.15887e-05
104873339 CHST11 3.83571e-06
10639533 S1PR5 1.83209e-05
108915394 TNFSF13B 1.01635e-08
110252794 COL4A1 7.03689e-06
112717138 NCAM1 6.31209e-05
114832028 UGCG 1.63329e-06
11924950 LPIN1 4.89868e-22
120475436 ENPP2 8.32811e-05
[  michaelt@node12  ~]$join -j 2 -a 1 -a 2 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; print $F[0], "\t", $F[3], "\t", $F[4];' | sort -k 2,2) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.20kbWindowTopResult.txt | awk '{ if ($3 <= .0001) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; ' | sort -k 2,2) | awk '{ print $1 }' | sort | uniq -c | sort -r -k 1,1 | head -n 10
      2 32852569
      1 99605399
      1 99560873
      1 98966499
      1 9889937
      1 9850212
      1 98216270
      1 98181056
      1 978388
      1 97190545
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.FinalResults.20kbWindow1eNeg4pVal.txt | wc
    637    6370   46913
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.FinalResults.20kbWindow1eNeg4pVal.txt | awk '{ print $1 }' | sort | uniq -c | awk '{ print $1 }' | sort | uniq -c
    103 1
    267 2
[  michaelt@node12  ~]$join -j 2 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; print $F[0], "\t", $F[3], "\t", $F[4];' | sort -k 2,2) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if ($3 <= .0001) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; ' | sort -k 2,2) | wc
    268    1340   11955
[  michaelt@node12  ~]$join -j 2 -v 1 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; print $F[0], "\t", $F[3], "\t", $F[4];' | sort -k 2,2) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if ($3 <= .0001) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; ' | sort -k 2,2) | wc
    266     798    7188
[  michaelt@node12  ~]$join -j 2 -a 1 -a 2 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; print $F[0], "\t", $F[3], "\t", $F[4];' | sort -k 2,2) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if ($3 <= .0001) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; ' | sort -k 2,2) | wc
    637    2447   21931
[  michaelt@node12  ~]$join -j 2 -v 2 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; print $F[0], "\t", $F[3], "\t", $F[4];' | sort -k 2,2) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if ($3 <= .0001) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; ' | sort -k 2,2) | wc
    103     309    2788
[  michaelt@node12  ~]$join -j 2 -a 1 -a 2 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if (($3 <= .0001) && ($5 <= .0001)) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; print $F[0], "\t", $F[3], "\t", $F[4];' | sort -k 2,2) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.20kbWindowTopResult.txt | awk '{ if ($3 <= .0001) { print $0; } }' | perl -lane 'print $F[0], "\t", $F[1], "\t", $F[2]; ' | sort -k 2,2) | awk '{ print $1 }' | sort | uniq -c | sort -r -k 1,1 | head -n 10
      2 32852569
      1 99605399
      1 98966499
      1 9889937
      1 9850212
      1 98216270
      1 98181056
      1 978388
      1 97190545
      1 96995923
~~~

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.txt | awk '{ print "chr" $3 "\t" $4 "\t" $4 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.bed
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.FinalResults.20kbWindow1eNeg4pVal.txt | awk '{ print "chr" $3 "\t" $4 "\t" $4 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.FinalResults.20kbWindow1eNeg4pVal.bed

bedtools slop -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.bed -g /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/hg19.chrom.sizes -b 100 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.Slop100.bed
bedtools slop -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.FinalResults.20kbWindow1eNeg4pVal.bed -g /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SudhirSNPFlankingSequence/hg19.chrom.sizes -b 100 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.FinalResults.20kbWindow1eNeg4pVal.Slop100.bed

bedtools getfasta -fi /home/michaelt/Data/HumanGenome/GRCh37/Version1/human_g1k_v37.fasta -bed <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.Slop100.bed | sed 's/chr//g') -fo /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.Slop100.fasta
bedtools getfasta -fi /home/michaelt/Data/HumanGenome/GRCh37/Version1/human_g1k_v37.fasta -bed <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.FinalResults.20kbWindow1eNeg4pVal.Slop100.bed | sed 's/chr//g') -fo /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.FinalResults.20kbWindow1eNeg4pVal.Slop100.fasta

join -j 1 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.txt | perl -lane 'print $F[2], "_", $F[3], "\t", join("\t", @F);' | sort -k 1,1) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.Slop100.fasta | perl -ane 'if ($F[0] =~ m/^>/) { print join("\t", @F), "\t"; } else { print join("\t", @F), "\n"; }' | perl -lane 'my $Chr = ((split(/\>/, ((split(/\:/, $F[0]))[0])))[1]); my $BP = ((split(/\-/, ((split(/\:/, $F[0]))[1])))[0]) + 100; print $Chr, "_", $BP, "\t$F[0]\t$Chr\t$BP\t$F[1]";' | sort -k 1,1) | perl -lane 'my @fasta = split(//, $F[$#F]); my $origRef = $fasta[99]; $fasta[99] = "[$F[5]/$F[6]]"; $F[$#F] = join("", @fasta); push(@F, $origRef); print join("\t", @F);' | awk '{ print $1, "\t", $2, "\t", $6, "\t", $7, "\t", "\t", $15, "\t", $16 }' | sed 's/_/ /g' | perl -lane 'print join("\t", @F);' | awk '{ print $4 }' | perl -lane 'my @vals1 = split(//, $F[0]); print $#vals1;' | sort | uniq -c
join -j 1 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.txt | perl -lane 'print $F[2], "_", $F[3], "\t", join("\t", @F);' | sort -k 1,1) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.Slop100.fasta | perl -ane 'if ($F[0] =~ m/^>/) { print join("\t", @F), "\t"; } else { print join("\t", @F), "\n"; }' | perl -lane 'my $Chr = ((split(/\>/, ((split(/\:/, $F[0]))[0])))[1]); my $BP = ((split(/\-/, ((split(/\:/, $F[0]))[1])))[0]) + 100; print $Chr, "_", $BP, "\t$F[0]\t$Chr\t$BP\t$F[1]";' | sort -k 1,1) | perl -lane 'my @fasta = split(//, $F[$#F]); my $origRef = $fasta[99]; $fasta[99] = "[$F[5]/$F[6]]"; $F[$#F] = join("", @fasta); push(@F, $origRef); print join("\t", @F);' | awk '{ print $1, "\t", $2, "\t", $6, "\t", $7, "\t", "\t", $15, "\t", $16 }' | sed 's/_/ /g' | perl -lane 'print join("\t", @F);' | awk '{ print $5 }' | perl -lane 'my @vals1 = split(//, $F[0]); print $#vals1;' | sort | uniq -c

join -j 1 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.FinalResults.20kbWindow1eNeg4pVal.txt | perl -lane 'print $F[2], "_", $F[3], "\t", join("\t", @F);' | sort -k 1,1) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.FinalResults.20kbWindow1eNeg4pVal.Slop100.fasta | perl -ane 'if ($F[0] =~ m/^>/) { print join("\t", @F), "\t"; } else { print join("\t", @F), "\n"; }' | perl -lane 'my $Chr = ((split(/\>/, ((split(/\:/, $F[0]))[0])))[1]); my $BP = ((split(/\-/, ((split(/\:/, $F[0]))[1])))[0]) + 100; print $Chr, "_", $BP, "\t$F[0]\t$Chr\t$BP\t$F[1]";' | sort -k 1,1) | perl -lane 'my @fasta = split(//, $F[$#F]); my $origRef = $fasta[99]; $fasta[99] = "[$F[5]/$F[6]]"; $F[$#F] = join("", @fasta); push(@F, $origRef); print join("\t", @F);' | awk '{ print $1, "\t", $2, "\t", $6, "\t", $7, "\t", "\t", $15, "\t", $16 }' | sed 's/_/ /g' | perl -lane 'print join("\t", @F);' | awk '{ print $4 }' | perl -lane 'my @vals1 = split(//, $F[0]); print $#vals1;' | sort | uniq -c
join -j 1 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.FinalResults.20kbWindow1eNeg4pVal.txt | perl -lane 'print $F[2], "_", $F[3], "\t", join("\t", @F);' | sort -k 1,1) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.FinalResults.20kbWindow1eNeg4pVal.Slop100.fasta | perl -ane 'if ($F[0] =~ m/^>/) { print join("\t", @F), "\t"; } else { print join("\t", @F), "\n"; }' | perl -lane 'my $Chr = ((split(/\>/, ((split(/\:/, $F[0]))[0])))[1]); my $BP = ((split(/\-/, ((split(/\:/, $F[0]))[1])))[0]) + 100; print $Chr, "_", $BP, "\t$F[0]\t$Chr\t$BP\t$F[1]";' | sort -k 1,1) | perl -lane 'my @fasta = split(//, $F[$#F]); my $origRef = $fasta[99]; $fasta[99] = "[$F[5]/$F[6]]"; $F[$#F] = join("", @fasta); push(@F, $origRef); print join("\t", @F);' | awk '{ print $1, "\t", $2, "\t", $6, "\t", $7, "\t", "\t", $15, "\t", $16 }' | sed 's/_/ /g' | perl -lane 'print join("\t", @F);' | awk '{ print $5 }' | perl -lane 'my @vals1 = split(//, $F[0]); print $#vals1;' | sort | uniq -c

##join -j 1 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.txt | perl -lane 'print $F[2], "_", $F[3], "\t", join("\t", @F);' | sort -k 1,1) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.Slop100.fasta | perl -ane 'if ($F[0] =~ m/^>/) { print join("\t", @F), "\t"; } else { print join("\t", @F), "\n"; }' | perl -lane 'my $Chr = ((split(/\>/, ((split(/\:/, $F[0]))[0])))[1]); my $BP = ((split(/\-/, ((split(/\:/, $F[0]))[1])))[0]) + 100; print $Chr, "_", $BP, "\t$F[0]\t$Chr\t$BP\t$F[1]";' | sort -k 1,1) | perl -lane 'my @fasta = split(//, $F[$#F]); my $origRef = $fasta[99]; $fasta[99] = "[$F[5]/$F[6]]"; $F[$#F] = join("", @fasta); push(@F, $origRef); print join("\t", @F);' | awk '{ print $1, "\t", $2, "\t", $6, "\t", $7, "\t", "\t", $15, "\t", $16 }' | sed 's/_/ /g' | perl -lane 'print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.Slop100.wfasta.formatted.vs1.txt 
join -j 1 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.txt | perl -lane 'print $F[2], "_", $F[3], "\t", join("\t", @F);' | sort -k 1,1) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.Slop100.fasta | perl -ane 'if ($F[0] =~ m/^>/) { print join("\t", @F), "\t"; } else { print join("\t", @F), "\n"; }' | perl -lane 'my $Chr = ((split(/\>/, ((split(/\:/, $F[0]))[0])))[1]); my $BP = ((split(/\-/, ((split(/\:/, $F[0]))[1])))[0]) + 100; print $Chr, "_", $BP, "\t$F[0]\t$Chr\t$BP\t$F[1]";' | sort -k 1,1) | perl -lane 'my @fasta = split(//, $F[$#F]); my $origRef = $fasta[99]; $fasta[99] = "[$F[5]/$F[6]]"; $F[$#F] = join("", @fasta); push(@F, $origRef); print join("\t", @F);' | awk '{ print $1, "\t", $2, "\t", $6, "\t", $7, "\t", "\t", $15, "\t", $16 }' | sort | uniq | perl -lane 'if ($. == 1) { $bp1 = $F[0]; @genes1 = ($F[1]); $entry1 = join("|", @F); } elsif ($F[0] eq $bp1) { push (@genes1, $F[1]); } else { @entry1 = split(/\|/, $entry1); $entry1[1] = join(",", @genes1); print join("\t", @entry1); $bp1 = $F[0]; @genes1 = ($F[1]); $entry1 = join("|", @F); } if (eof()) { @entry1 = split(/\|/, $entry1); $entry1[1] = join(",", @genes1); print join("\t", @entry1); }' | perl -lane 'splice(@F, 1, 0, ("NA", "WholeBlood_eQTL")); splice(@F, 4, 0, "NA"); print join("\t", @F);' | sed 's/_/ /' | perl -lane 'print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.Slop100.wfasta.formatted.vs2.txt
join -j 1 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.FinalResults.20kbWindow1eNeg4pVal.txt | perl -lane 'print $F[2], "_", $F[3], "\t", join("\t", @F);' | sort -k 1,1) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.FinalResults.20kbWindow1eNeg4pVal.Slop100.fasta | perl -ane 'if ($F[0] =~ m/^>/) { print join("\t", @F), "\t"; } else { print join("\t", @F), "\n"; }' | perl -lane 'my $Chr = ((split(/\>/, ((split(/\:/, $F[0]))[0])))[1]); my $BP = ((split(/\-/, ((split(/\:/, $F[0]))[1])))[0]) + 100; print $Chr, "_", $BP, "\t$F[0]\t$Chr\t$BP\t$F[1]";' | sort -k 1,1) | perl -lane 'my @fasta = split(//, $F[$#F]); my $origRef = $fasta[99]; $fasta[99] = "[$F[5]/$F[6]]"; $F[$#F] = join("", @fasta); push(@F, $origRef); print join("\t", @F);' | awk '{ print $1, "\t", $2, "\t", $6, "\t", $7, "\t", "\t", $15, "\t", $16 }' | sort | uniq | perl -lane 'if ($. == 1) { $bp1 = $F[0]; @genes1 = ($F[1]); $entry1 = join("|", @F); } elsif ($F[0] eq $bp1) { push (@genes1, $F[1]); } else { @entry1 = split(/\|/, $entry1); $entry1[1] = join(",", @genes1); print join("\t", @entry1); $bp1 = $F[0]; @genes1 = ($F[1]); $entry1 = join("|", @F); } if (eof()) { @entry1 = split(/\|/, $entry1); $entry1[1] = join(",", @genes1); print join("\t", @entry1); }' | perl -lane 'splice(@F, 1, 0, ("NA", "WholeBlood_eQTL")); splice(@F, 4, 0, "NA"); print join("\t", @F);' | sed 's/_/ /' | perl -lane 'print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.FinalResults.20kbWindow1eNeg4pVal.Slop100.wfasta.formatted.vs2.txt

#Included the below header in file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.FinalResults.20kbWindow1eNeg4pVal.Slop100.wfasta.formatted.vs2.txt 
#CHR    BP      RSID    CATEGORY        GENE    FUNC    REF     ALT     SEQUENCE        ORIG_POS_100

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.Slop100.wfasta.formatted.vs1.txt | perl -lane 'if ($F[3] eq $F[6]) { print "REF"; } elsif ($F[4] eq $F[6]) { print "ALT"; } else { print "NA"; }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.Slop100.wfasta.formatted.vs2.txt | perl -lane 'if ($F[6] eq $F[9]) { print "REF"; } elsif ($F[7] eq $F[9]) { print "ALT"; } else { print "NA"; }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.FinalResults.20kbWindow1eNeg4pVal.Slop100.wfasta.formatted.vs2.txt | perl -lane 'if ($F[6] eq $F[9]) { print "REF"; } elsif ($F[7] eq $F[9]) { print "ALT"; } else { print "NA"; }' | sort | uniq -c

~~~
[  michaelt@node12  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.txt | awk '{ print "chr" $3, "\t", $4, "\t", $4 }' | head -n 10
chr17    6496584         6496584
chr17    5746484         5746484
chr21    45158689        45158689
chr21    45138420        45138420
chr6     36653477        36653477
chr6     36015536        36015536
chr6     31595487        31595487
chr6     31637734        31637734
chr6     89670409        89670409
chr6     89606741        89606741
[  michaelt@node12  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.bed | head -n 10
chr17    6496584         6496584
chr17    5746484         5746484
chr21    45158689        45158689
chr21    45138420        45138420
chr6     36653477        36653477
chr6     36015536        36015536
chr6     31595487        31595487
chr6     31637734        31637734
chr6     89670409        89670409
chr6     89606741        89606741
[  michaelt@node12  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.Slop100.bed | wc
    652    1956   15468
[  michaelt@node12  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.Slop100.bed | head -n 10
chr17   6496484 6496684
chr17   5746384 5746584
chr21   45158589        45158789
chr21   45138320        45138520
chr6    36653377        36653577
chr6    36015436        36015636
chr6    31595387        31595587
chr6    31637634        31637834
chr6    89670309        89670509
chr6    89606641        89606841
[  michaelt@node12  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.Slop100.fasta | head -n 10
>17:6496484-6496684
AGACAAGCAACCCCAAAAAAAAACTGACAAAAGATTTTGGACGCTTCACAAGGGAAGATATACAAATGCCTAATAATCACCTGAAAAGGGGCCAGGTGACGGGGCTCATGCCTGTAACTCTAGCACTTCGGGAGGCCGGGAGCAGTGGTTCACGCCTGTAATCCCAGCACTTCGGGAGGCCGGGCGCAATGGCTCACACC
>17:5746384-5746584
TGTTGGCCAGGCTGGTCTGGAACTCCTGACCTCAAGTGATCTGCCTGCCTTGGCCTCCCAAAATGCTGGGACTACAGGCATGAGCCACCATGCCCAGCCAGGGTCTTGTTTTTTATTTTATTTATTTATTTATTTATTTTTTTGAGAGGGAGTCTGGCACTGTTGCCAGGGTTCGAGTGCAGTGGCACAATCTCGTCTTA
>21:45158589-45158789
AAGATACGCTGTTTGAGCACAAAGGAGATTCATTGAAACCGCTATTGCTGTAGATTACGCCTGTGACGCACTGCCTCCCTTTCACTGTTTCGCCCTGAATATCTGCTTCTCAGATCTAAGTGACTGTACTCAATAAATAGCGTGGAGACCAGAGCTCAGTGCCTTTTGCAGCCTCCATTTTGCCACTGGCCCCCTGGCTC
>21:45138320-45138520
TGTGGATGGACGGCCGGGCATGTGGCCATCGGCTGGGGGCCTCTGCAGGCCGCGCCACCAGAGTGGGCACTCAGGGTCCGCAAGGGTCTGGGGCGCCCCCAGAGAAGCCCGCCGCCCCAGGCCGGTGCGGAGTTGACTGGGTCCCCCCGGCCCCGTGTCTGTCAGCTCCCGCGCGGCCCCTGGGAAGGGACGCGGATCGG
>6:36653377-36653577
TGGACGTTCCCCGAGTTCTTCCTGTTCTCAGCAGTCGGCCCCATTGGCCCCAGGGAAGGGTGTCCTGGCCCCCCACTGTCTTCCTCAGTTGGGCAGCTCCGCCGCGTCCTCTTCTTCTTGGCCTGGCTGACTTCTGCTGTCTCTCCTCAGATTTCTACCACTCCAAACGCCGGCTGATCTTCTCCAAGAGGAAGCCCTAA
[  michaelt@node12  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.Slop100.fasta | perl -ane 'if ($F[0] =~ m/^>/) { print join("\t", @F), "\t"; } else { print join("\t", @F), "\n"; }' | perl -lane 'my $Chr = ((split(/\>/, ((split(/\:/, $F[0]))[0])))[1]); my $BP = ((split(/\-/, ((split(/\:/, $F[0]))[1])))[0]) + 100; print $Chr, "_", $BP, "\t$F[0]\t$Chr\t$BP\t$F[1]";' | sort | head -n 10
10_125757093    >10:125756993-125757193 10      125757093       CAAGCGCCTTCCAGTTCTCATGTCCCTTTAGAAACACTGAAACCAGAAATCAGTGCTGATTCACTGCTCTGTGGTTTCTGCCACAGAGATGACCCAGAGCTCCAATCAGTGGGCTCATGCTCAGAGAAAAGGTTTTGGCTCCATACCAAAAGGCAGACAAATAGATGTGCTGTACTTCACATATTTGAAGTAAAAATAAA
10_125785392    >10:125785292-125785492 10      125785392       ACAGACAGCACCACAGAAAGACGGAGACAGAGCCTGGAGTGCGGCGTCTATGAGCCAAGGAACGCCAAGGACTGCTGGAGGCCACCAAGAGCTGGGAGCAGGGGACGGCTTTTATCCCTTAGGGCCACCAGAGGGAACCAACCCTGCAGACATCTTGATTTAGGACTTCTGGCCTCCAGAATCATAAGAGACTACATTTC
10_133539754    >10:133539654-133539854 10      133539754       AAAAACCTGCCTGCTGCTTTACCTCAGTGTTAAACCTATGTGTTGGAGTGATTCGTTACACACCATGGATTATGGGTACTGAGATGACCATGGCAGACAGGAATGCAGAGAGGAACACGCTCAGATGCAGACACACACAGCTGTTTGATCAGGAGTCCCAATGCTGGATTTGCTAAGAGAACCAACACATCTTTAGAAAA
10_3137445      >10:3137345-3137545     10      3137445 TGCTGCAACACAACGGTATAATTTCCCTTTGTCTGTAACTTTAAAAAAAAAAACTTACATATGAGACGTCCGAGAAAGCCACCCATAGGTGAACTACGCGCATAGGTGAACTAGCTTACTCCATTTATGATGGATTCGGCGATTGCCTAAAAATTCTAGTCAAAACCACCTCTACATATAGAATGATCTCCTACATGGTT
10_3182686      >10:3182586-3182786     10      3182686 AACCATGGAAGGCAAACCCACGGATGGAGGCACTACCGTGTTCTGCCAAAATATGCTGGCTGAAGCCAGGGTTTACTGAAGGCAGAATGCTTCCTGTTTCGGCTGCTGGCCCTAATCCTCGCGCCAGAAAGGACATTGCTGGGTGTGGCTAAGTGCCCAAGGAGTGAAAAGAAGCACACAGAACAACTTTTGTCTAAAAT
10_34167001     >10:34166901-34167101   10      34167001        GAGTGAATTATAATACCCACCATCATTCTCCAGCTTTAAATAAAGTAAGAAAAATAACAGAGTATCTTATCATAGAAGTGACTTTTAAGTAATCATTTCGTCAAGATTCTTTTGGTAAATACAGGATTTAATTTTCCTACAGTCACTTCTGTAGGAAAATTAAAGTAGTTTATGGAAAAGAAAATGTGAGAGGTTTATTA
10_35250388     >10:35250288-35250488   10      35250388        GGCGCAATCTCGGCTCACTGCAGCCCCCGTCTCTCGGGTTCAAGCGATTCTCCTGTCTCAGCCTTTCAAGTAGCTGAGATTACAGGCACACGCTGCCACGCCCAGCTAATTTTTTGTATTTTAGTAGAGATGGGGTTTCACCGAGTTGCCTAGGCTGGACTTGAACTCCTGAGCTCAGGCAATCCACCCGCCTCGGCCTC
10_35454002     >10:35453902-35454102   10      35454002        TTGAACCCGGAAGGCAGAAGTTGCAGTAAGCCAGATTGCACCACTGCTCTCCAGCCTGGGCCACAGAGCGAGGCTCCATCTCAAAAAAAAAAAAAAAAAAAAAAGAAAAGGAAAAAGAAATACTTGAGGCTGGATAATATATAAAGAAAAGAAGTTTGTTTTGGCTCACGGTTCTGCAGGCTGCACAAGAAGCATGGCAC
10_49609011     >10:49608911-49609111   10      49609011        ATCCTACAAGTAGTAAAGGAATGGTAGTCAGAAGTTGAATCGTACTTTTAATGCCTCAGGCAGGATAGAATAGTATTGTTTTGTTTTGTATCCCAATATGCCTAACTACTTCTCTCTCTCTCCTTTCTGTTTTGCTTCTCATCCTTTCACCCATTACAGTCTCAGTATGTGTGTTACACCAGGAGGTTGCCTTGGCAGGT
10_49660001     >10:49659901-49660101   10      49660001        CCCTACACACACACAAGTGAACACACAAATCCCCTACCCAGCCTCCCCAAATCCCCCCATACAGAATCGCACAATGAACCCCACCCACTGCCCAAAATCCGCCACACACAATTGCACACACAATGCACCCTCCGCAGCCTCCCCAAATTCCCCCATACACAACTGCACACATAGTGCACCCTGCTCACCCCCCAGAATCC
[  michaelt@node12  ~]$join -j 1 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.txt | perl -lane 'print $F[2], "_", $F[3], "\t", join("\t", @F);' | sort -k 1,1) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.Slop100.fasta | perl -ane 'if ($F[0] =~ m/^>/) { print join("\t", @F), "\t"; } else { print join("\t", @F), "\n"; }' | perl -lane 'my $Chr = ((split(/\>/, ((split(/\:/, $F[0]))[0])))[1]); my $BP = ((split(/\-/, ((split(/\:/, $F[0]))[1])))[0]) + 100; print $Chr, "_", $BP, "\t$F[0]\t$Chr\t$BP\t$F[1]";' | sort -k 1,1) | wc
    654    9810  209167
[  michaelt@node12  ~]$join -j 1 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.txt | perl -lane 'print $F[2], "_", $F[3], "\t", join("\t", @F);' | sort -k 1,1) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.Slop100.fasta | perl -ane 'if ($F[0] =~ m/^>/) { print join("\t", @F), "\t"; } else { print join("\t", @F), "\n"; }' | perl -lane 'my $Chr = ((split(/\>/, ((split(/\:/, $F[0]))[0])))[1]); my $BP = ((split(/\-/, ((split(/\:/, $F[0]))[1])))[0]) + 100; print $Chr, "_", $BP, "\t$F[0]\t$Chr\t$BP\t$F[1]";' | sort -k 1,1) | awk '{ print $1; }' | sort | uniq | wc
    651     651    7555
[  michaelt@node12  ~]$join -j 1 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.txt | perl -lane 'print $F[2], "_", $F[3], "\t", join("\t", @F);' | sort -k 1,1) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.Slop100.fasta | perl -ane 'if ($F[0] =~ m/^>/) { print join("\t", @F), "\t"; } else { print join("\t", @F), "\n"; }' | perl -lane 'my $Chr = ((split(/\>/, ((split(/\:/, $F[0]))[0])))[1]); my $BP = ((split(/\-/, ((split(/\:/, $F[0]))[1])))[0]) + 100; print $Chr, "_", $BP, "\t$F[0]\t$Chr\t$BP\t$F[1]";' | sort -k 1,1) | awk '{ print $1; }' | sort | uniq -c | sort -r -k 1,1 | head -n 10
      4 6_32852569
      1 9_72650952
      1 9_35711806
      1 9_35690530
      1 9_34635337
      1 9_34582849
      1 9_33294037
      1 9_33286927
      1 9_33251747
      1 9_32455320
[  michaelt@node12  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.txt | grep 32852569
PSMB9   ENSG00000240065.3       6       32852569        T       A       b37     0.343051        12.3083 2.15279e-28
TAP1    ENSG00000168394.9       6       32852569        T       A       b37     0.12638 4.60073 6.25072e-06
[  michaelt@node12  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.Slop100.fasta | grep 32852569
[  michaelt@node12  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.Slop100.fasta | perl -ane 'if ($F[0] =~ m/^>/) { print join("\t", @F), "\t"; } else { print join("\t", @F), "\n"; }' | perl -lane 'my $Chr = ((split(/\>/, ((split(/\:/, $F[0]))[0])))[1]); my $BP = ((split(/\-/, ((split(/\:/, $F[0]))[1])))[0]) + 100; print $Chr, "_", $BP, "\t$F[0]\t$Chr\t$BP\t$F[1]";' | sort -k 1,1 | grep 32852569
6_32852569      >6:32852469-32852669    6       32852569        ATTTATGGAACAAGTGCTGACTTCAGATATCTAGTAATCTAAGGTTTTTCATCTCCAAAGACCTTTCTTTCATTTGGCCTCTACTGGGTTTTCTATTTTTTTTATTTATTTATTTATTTATTGAGACAAGGTCTCACTCTGTCACCCAGGCTGGAGTACAGTGACCTGAACATGGCTTGCTGTATCCCTAACCTCCTGTG
6_32852569      >6:32852469-32852669    6       32852569        ATTTATGGAACAAGTGCTGACTTCAGATATCTAGTAATCTAAGGTTTTTCATCTCCAAAGACCTTTCTTTCATTTGGCCTCTACTGGGTTTTCTATTTTTTTTATTTATTTATTTATTTATTGAGACAAGGTCTCACTCTGTCACCCAGGCTGGAGTACAGTGACCTGAACATGGCTTGCTGTATCCCTAACCTCCTGTG
[  michaelt@node12  ~]$join -j 1 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.txt | perl -lane 'print $F[2], "_", $F[3], "\t", join("\t", @F);' | sort -k 1,1) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.Slop100.fasta | perl -ane 'if ($F[0] =~ m/^>/) { print join("\t", @F), "\t"; } else { print join("\t", @F), "\n"; }' | perl -lane 'my $Chr = ((split(/\>/, ((split(/\:/, $F[0]))[0])))[1]); my $BP = ((split(/\-/, ((split(/\:/, $F[0]))[1])))[0]) + 100; print $Chr, "_", $BP, "\t$F[0]\t$Chr\t$BP\t$F[1]";' | sort -k 1,1) | grep 6_32852569
6_32852569 PSMB9 ENSG00000240065.3 6 32852569 T A b37 0.343051 12.3083 2.15279e-28 >6:32852469-32852669 6 32852569 ATTTATGGAACAAGTGCTGACTTCAGATATCTAGTAATCTAAGGTTTTTCATCTCCAAAGACCTTTCTTTCATTTGGCCTCTACTGGGTTTTCTATTTTTTTTATTTATTTATTTATTTATTGAGACAAGGTCTCACTCTGTCACCCAGGCTGGAGTACAGTGACCTGAACATGGCTTGCTGTATCCCTAACCTCCTGTG
6_32852569 PSMB9 ENSG00000240065.3 6 32852569 T A b37 0.343051 12.3083 2.15279e-28 >6:32852469-32852669 6 32852569 ATTTATGGAACAAGTGCTGACTTCAGATATCTAGTAATCTAAGGTTTTTCATCTCCAAAGACCTTTCTTTCATTTGGCCTCTACTGGGTTTTCTATTTTTTTTATTTATTTATTTATTTATTGAGACAAGGTCTCACTCTGTCACCCAGGCTGGAGTACAGTGACCTGAACATGGCTTGCTGTATCCCTAACCTCCTGTG
6_32852569 TAP1 ENSG00000168394.9 6 32852569 T A b37 0.12638 4.60073 6.25072e-06 >6:32852469-32852669 6 32852569 ATTTATGGAACAAGTGCTGACTTCAGATATCTAGTAATCTAAGGTTTTTCATCTCCAAAGACCTTTCTTTCATTTGGCCTCTACTGGGTTTTCTATTTTTTTTATTTATTTATTTATTTATTGAGACAAGGTCTCACTCTGTCACCCAGGCTGGAGTACAGTGACCTGAACATGGCTTGCTGTATCCCTAACCTCCTGTG
6_32852569 TAP1 ENSG00000168394.9 6 32852569 T A b37 0.12638 4.60073 6.25072e-06 >6:32852469-32852669 6 32852569 ATTTATGGAACAAGTGCTGACTTCAGATATCTAGTAATCTAAGGTTTTTCATCTCCAAAGACCTTTCTTTCATTTGGCCTCTACTGGGTTTTCTATTTTTTTTATTTATTTATTTATTTATTGAGACAAGGTCTCACTCTGTCACCCAGGCTGGAGTACAGTGACCTGAACATGGCTTGCTGTATCCCTAACCTCCTGTG
[  michaelt@node12  ~]$join -j 1 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.txt | perl -lane 'print $F[2], "_", $F[3], "\t", join("\t", @F);' | sort -k 1,1) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.Slop100.fasta | perl -ane 'if ($F[0] =~ m/^>/) { print join("\t", @F), "\t"; } else { print join("\t", @F), "\n"; }' | perl -lane 'my $Chr = ((split(/\>/, ((split(/\:/, $F[0]))[0])))[1]); my $BP = ((split(/\-/, ((split(/\:/, $F[0]))[1])))[0]) + 100; print $Chr, "_", $BP, "\t$F[0]\t$Chr\t$BP\t$F[1]";' | sort -k 1,1) | perl -lane 'my @fasta = split(//, $F[$#F]); my $origRef = $fasta[99]; $fasta[99] = "[$F[5]/$F[6]]"; $F[$#F] = join("", @fasta); push(@F, $origRef); print join("\t", @F);' | awk '{ print $1, "\t", $2, "\t", $6, "\t", $7, "\t", "\t", $15, "\t", $16 }' | sed 's/_/ /g' | perl -lane 'print join("\t", @F);' | awk '{ print $4 }' | perl -lane 'my @vals1 = split(//, $F[0]); print $#vals1;' | sort | uniq -c
    613 0
     10 1
      1 10
      1 12
     11 2
      7 3
      6 4
      1 5
      1 6
      1 7
      1 8
      1 9
[  michaelt@node12  ~]$join -j 1 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.txt | perl -lane 'print $F[2], "_", $F[3], "\t", join("\t", @F);' | sort -k 1,1) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.Slop100.fasta | perl -ane 'if ($F[0] =~ m/^>/) { print join("\t", @F), "\t"; } else { print join("\t", @F), "\n"; }' | perl -lane 'my $Chr = ((split(/\>/, ((split(/\:/, $F[0]))[0])))[1]); my $BP = ((split(/\-/, ((split(/\:/, $F[0]))[1])))[0]) + 100; print $Chr, "_", $BP, "\t$F[0]\t$Chr\t$BP\t$F[1]";' | sort -k 1,1) | perl -lane 'my @fasta = split(//, $F[$#F]); my $origRef = $fasta[99]; $fasta[99] = "[$F[5]/$F[6]]"; $F[$#F] = join("", @fasta); push(@F, $origRef); print join("\t", @F);' | awk '{ print $1, "\t", $2, "\t", $6, "\t", $7, "\t", "\t", $15, "\t", $16 }' | sed 's/_/ /g' | perl -lane 'print join("\t", @F);' | awk '{ print $5 }' | perl -lane 'my @vals1 = split(//, $F[0]); print $#vals1;' | sort | uniq -c
    636 0
      9 1
      1 10
      3 2
      3 4
      1 5
      1 7
[  michaelt@node12  ~]$join -j 1 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.txt | perl -lane 'print $F[2], "_", $F[3], "\t", join("\t", @F);' | sort -k 1,1) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.Slop100.fasta | perl -ane 'if ($F[0] =~ m/^>/) { print join("\t", @F), "\t"; } else { print join("\t", @F), "\n"; }' | perl -lane 'my $Chr = ((split(/\>/, ((split(/\:/, $F[0]))[0])))[1]); my $BP = ((split(/\-/, ((split(/\:/, $F[0]))[1])))[0]) + 100; print $Chr, "_", $BP, "\t$F[0]\t$Chr\t$BP\t$F[1]";' | sort -k 1,1) | perl -lane 'my @fasta = split(//, $F[$#F]); my $origRef = $fasta[99]; $fasta[99] = "[$F[5]/$F[6]]"; $F[$#F] = join("", @fasta); push(@F, $origRef); print join("\t", @F);' | awk '{ print $1, "\t", $2, "\t", $6, "\t", $7, "\t", "\t", $15, "\t", $16 }' | sed 's/_/ /g' | perl -lane 'print join("\t", @F);' | awk '{ print $4 }' | perl -lane 'my @vals1 = split(//, $F[0]); if ($#vals1 > 0) { print join("\t", @F); }' | head -n 10
GC
ACTT
TTTTC
GGC
AATAAT
CT
AT
AATTT
GT
GTGTGTA
[  michaelt@node12  ~]$join -j 1 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.txt | perl -lane 'print $F[2], "_", $F[3], "\t", join("\t", @F);' | sort -k 1,1) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.Slop100.fasta | perl -ane 'if ($F[0] =~ m/^>/) { print join("\t", @F), "\t"; } else { print join("\t", @F), "\n"; }' | perl -lane 'my $Chr = ((split(/\>/, ((split(/\:/, $F[0]))[0])))[1]); my $BP = ((split(/\-/, ((split(/\:/, $F[0]))[1])))[0]) + 100; print $Chr, "_", $BP, "\t$F[0]\t$Chr\t$BP\t$F[1]";' | sort -k 1,1) | perl -lane 'my @fasta = split(//, $F[$#F]); my $origRef = $fasta[99]; $fasta[99] = "[$F[5]/$F[6]]"; $F[$#F] = join("", @fasta); push(@F, $origRef); print join("\t", @F);' | awk '{ print $1, "\t", $2, "\t", $6, "\t", $7, "\t", "\t", $15, "\t", $16 }' | sed 's/_/ /g' | perl -lane 'print join("\t", @F);' | head -n 10
10      125757093       CHST15  C       T       CAAGCGCCTTCCAGTTCTCATGTCCCTTTAGAAACACTGAAACCAGAAATCAGTGCTGATTCACTGCTCTGTGGTTTCTGCCACAGAGATGACCCAGAG[C/T]TCCAATCAGTGGGCTCATGCTCAGAGAAAAGGTTTTGGCTCCATACCAAAAGGCAGACAAATAGATGTGCTGTACTTCACATATTTGAAGTAAAAATAAA    C
10      125785392       CHST15  A       T       ACAGACAGCACCACAGAAAGACGGAGACAGAGCCTGGAGTGCGGCGTCTATGAGCCAAGGAACGCCAAGGACTGCTGGAGGCCACCAAGAGCTGGGAGC[A/T]GGGGACGGCTTTTATCCCTTAGGGCCACCAGAGGGAACCAACCCTGCAGACATCTTGATTTAGGACTTCTGGCCTCCAGAATCATAAGAGACTACATTTC    A
10      133539754       BNIP3   G       C       AAAAACCTGCCTGCTGCTTTACCTCAGTGTTAAACCTATGTGTTGGAGTGATTCGTTACACACCATGGATTATGGGTACTGAGATGACCATGGCAGACA[G/C]GAATGCAGAGAGGAACACGCTCAGATGCAGACACACACAGCTGTTTGATCAGGAGTCCCAATGCTGGATTTGCTAAGAGAACCAACACATCTTTAGAAAA    G
10      3137445 PFKP    G       A       TGCTGCAACACAACGGTATAATTTCCCTTTGTCTGTAACTTTAAAAAAAAAAACTTACATATGAGACGTCCGAGAAAGCCACCCATAGGTGAACTACGC[G/A]CATAGGTGAACTAGCTTACTCCATTTATGATGGATTCGGCGATTGCCTAAAAATTCTAGTCAAAACCACCTCTACATATAGAATGATCTCCTACATGGTT    G
10      3182686 PFKP    C       A       AACCATGGAAGGCAAACCCACGGATGGAGGCACTACCGTGTTCTGCCAAAATATGCTGGCTGAAGCCAGGGTTTACTGAAGGCAGAATGCTTCCTGTTT[C/A]GGCTGCTGGCCCTAATCCTCGCGCCAGAAAGGACATTGCTGGGTGTGGCTAAGTGCCCAAGGAGTGAAAAGAAGCACACAGAACAACTTTTGTCTAAAAT    C
10      34167001        NRP1    G       C       GAGTGAATTATAATACCCACCATCATTCTCCAGCTTTAAATAAAGTAAGAAAAATAACAGAGTATCTTATCATAGAAGTGACTTTTAAGTAATCATTTC[G/C]TCAAGATTCTTTTGGTAAATACAGGATTTAATTTTCCTACAGTCACTTCTGTAGGAAAATTAAAGTAGTTTATGGAAAAGAAAATGTGAGAGGTTTATTA    G
10      35250388        CREM    G       A       GGCGCAATCTCGGCTCACTGCAGCCCCCGTCTCTCGGGTTCAAGCGATTCTCCTGTCTCAGCCTTTCAAGTAGCTGAGATTACAGGCACACGCTGCCAC[G/A]CCCAGCTAATTTTTTGTATTTTAGTAGAGATGGGGTTTCACCGAGTTGCCTAGGCTGGACTTGAACTCCTGAGCTCAGGCAATCCACCCGCCTCGGCCTC    G
10      35454002        CREM    A       T       TTGAACCCGGAAGGCAGAAGTTGCAGTAAGCCAGATTGCACCACTGCTCTCCAGCCTGGGCCACAGAGCGAGGCTCCATCTCAAAAAAAAAAAAAAAAA[A/T]AAAAGAAAAGGAAAAAGAAATACTTGAGGCTGGATAATATATAAAGAAAAGAAGTTTGTTTTGGCTCACGGTTCTGCAGGCTGCACAAGAAGCATGGCAC    A
10      49609011        MAPK8   GC      G       ATCCTACAAGTAGTAAAGGAATGGTAGTCAGAAGTTGAATCGTACTTTTAATGCCTCAGGCAGGATAGAATAGTATTGTTTTGTTTTGTATCCCAATAT[GC/G]CCTAACTACTTCTCTCTCTCTCCTTTCTGTTTTGCTTCTCATCCTTTCACCCATTACAGTCTCAGTATGTGTGTTACACCAGGAGGTTGCCTTGGCAGGT   G
10      49660001        MAPK8   C       T       CCCTACACACACACAAGTGAACACACAAATCCCCTACCCAGCCTCCCCAAATCCCCCCATACAGAATCGCACAATGAACCCCACCCACTGCCCAAAATC[C/T]GCCACACACAATTGCACACACAATGCACCCTCCGCAGCCTCCCCAAATTCCCCCATACACAACTGCACACATAGTGCACCCTGCTCACCCCCCAGAATCC    C
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.Slop100.wfasta.formatted.vs2.txt | head -n 10
10      125757093       NA      WholeBlood_eQTL CHST15  NA      C       T       CAAGCGCCTTCCAGTTCTCATGTCCCTTTAGAAACACTGAAACCAGAAATCAGTGCTGATTCACTGCTCTGTGGTTTCTGCCACAGAGATGACCCAGAG[C/T]TCCAATCAGTGGGCTCATGCTCAGAGAAAAGGTTTTGGCTCCATACCAAAAGGCAGACAAATAGATGTGCTGTACTTCACATATTTGAAGTAAAAATAAA    C
10      125785392       NA      WholeBlood_eQTL CHST15  NA      A       T       ACAGACAGCACCACAGAAAGACGGAGACAGAGCCTGGAGTGCGGCGTCTATGAGCCAAGGAACGCCAAGGACTGCTGGAGGCCACCAAGAGCTGGGAGC[A/T]GGGGACGGCTTTTATCCCTTAGGGCCACCAGAGGGAACCAACCCTGCAGACATCTTGATTTAGGACTTCTGGCCTCCAGAATCATAAGAGACTACATTTC    A
10      133539754       NA      WholeBlood_eQTL BNIP3   NA      G       C       AAAAACCTGCCTGCTGCTTTACCTCAGTGTTAAACCTATGTGTTGGAGTGATTCGTTACACACCATGGATTATGGGTACTGAGATGACCATGGCAGACA[G/C]GAATGCAGAGAGGAACACGCTCAGATGCAGACACACACAGCTGTTTGATCAGGAGTCCCAATGCTGGATTTGCTAAGAGAACCAACACATCTTTAGAAAA    G
10      3137445 NA      WholeBlood_eQTL PFKP    NA      G       A       TGCTGCAACACAACGGTATAATTTCCCTTTGTCTGTAACTTTAAAAAAAAAAACTTACATATGAGACGTCCGAGAAAGCCACCCATAGGTGAACTACGC[G/A]CATAGGTGAACTAGCTTACTCCATTTATGATGGATTCGGCGATTGCCTAAAAATTCTAGTCAAAACCACCTCTACATATAGAATGATCTCCTACATGGTT    G
10      3182686 NA      WholeBlood_eQTL PFKP    NA      C       A       AACCATGGAAGGCAAACCCACGGATGGAGGCACTACCGTGTTCTGCCAAAATATGCTGGCTGAAGCCAGGGTTTACTGAAGGCAGAATGCTTCCTGTTT[C/A]GGCTGCTGGCCCTAATCCTCGCGCCAGAAAGGACATTGCTGGGTGTGGCTAAGTGCCCAAGGAGTGAAAAGAAGCACACAGAACAACTTTTGTCTAAAAT    C
10      34167001        NA      WholeBlood_eQTL NRP1    NA      G       C       GAGTGAATTATAATACCCACCATCATTCTCCAGCTTTAAATAAAGTAAGAAAAATAACAGAGTATCTTATCATAGAAGTGACTTTTAAGTAATCATTTC[G/C]TCAAGATTCTTTTGGTAAATACAGGATTTAATTTTCCTACAGTCACTTCTGTAGGAAAATTAAAGTAGTTTATGGAAAAGAAAATGTGAGAGGTTTATTA    G
10      35250388        NA      WholeBlood_eQTL CREM    NA      G       A       GGCGCAATCTCGGCTCACTGCAGCCCCCGTCTCTCGGGTTCAAGCGATTCTCCTGTCTCAGCCTTTCAAGTAGCTGAGATTACAGGCACACGCTGCCAC[G/A]CCCAGCTAATTTTTTGTATTTTAGTAGAGATGGGGTTTCACCGAGTTGCCTAGGCTGGACTTGAACTCCTGAGCTCAGGCAATCCACCCGCCTCGGCCTC    G
10      35454002        NA      WholeBlood_eQTL CREM    NA      A       T       TTGAACCCGGAAGGCAGAAGTTGCAGTAAGCCAGATTGCACCACTGCTCTCCAGCCTGGGCCACAGAGCGAGGCTCCATCTCAAAAAAAAAAAAAAAAA[A/T]AAAAGAAAAGGAAAAAGAAATACTTGAGGCTGGATAATATATAAAGAAAAGAAGTTTGTTTTGGCTCACGGTTCTGCAGGCTGCACAAGAAGCATGGCAC    A
10      49609011        NA      WholeBlood_eQTL MAPK8   NA      GC      G       ATCCTACAAGTAGTAAAGGAATGGTAGTCAGAAGTTGAATCGTACTTTTAATGCCTCAGGCAGGATAGAATAGTATTGTTTTGTTTTGTATCCCAATAT[GC/G]CCTAACTACTTCTCTCTCTCTCCTTTCTGTTTTGCTTCTCATCCTTTCACCCATTACAGTCTCAGTATGTGTGTTACACCAGGAGGTTGCCTTGGCAGGT   G
10      49660001        NA      WholeBlood_eQTL MAPK8   NA      C       T       CCCTACACACACACAAGTGAACACACAAATCCCCTACCCAGCCTCCCCAAATCCCCCCATACAGAATCGCACAATGAACCCCACCCACTGCCCAAAATC[C/T]GCCACACACAATTGCACACACAATGCACCCTCCGCAGCCTCCCCAAATTCCCCCATACACAACTGCACACATAGTGCACCCTGCTCACCCCCCAGAATCC    C
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.Slop100.wfasta.formatted.vs1.txt | perl -lane 'if ($F[3] eq $F[6]) { print "REF"; } elsif ($F[4] eq
>  $F[6]) { print "ALT"; } else { print "NA"; }' | sort | uniq -c
     41 ALT
    613 REF
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.FinalResults.20kbWindow1eNeg4pVal.Slop100.wfasta.formatted.vs2.txt | perl -lane 'if ($F[5] eq $F[8]) { print "REF"; } elsif ($F[6] eq $F[8]) { print "ALT"; } else { print "NA"; }' | sort | uniq -c
     41 ALT
    610 REF
[  michaelt@node3  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.FinalResults.20kbWindow1eNeg4pVal.Slop100.wfasta.formatted.vs2.txt | perl -lane 'if ($F[5] eq $F[8]) { print "REF"; } elsif ($F[6] eq $F[8]) { print "ALT"; } else { print "NA"; }' | sort | uniq -c
    636 REF
[  michaelt@node3  ~]$join -j 1 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.FinalResults.20kbWindow1eNeg4pVal.txt | perl -lane 'print $F[2], "_", $F[3], "\t", join("\t", @F);' | sort -k 1,1) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.FinalResults.20kbWindow1eNeg4pVal.Slop100.fasta | perl -ane 'if ($F[0] =~ m/^>/) { print join("\t", @F), "\t"; } else { print join("\t", @F), "\n"; }' | perl -lane 'my $Chr = ((split(/\>/, ((split(/\:/, $F[0]))[0])))[1]); my $BP = ((split(/\-/, ((split(/\:/, $F[0]))[1])))[0]) + 100; print $Chr, "_", $BP, "\t$F[0]\t$Chr\t$BP\t$F[1]";' | sort -k 1,1) | perl -lane 'my @fasta = split(//, $F[$#F]); my $origRef = $fasta[99]; $fasta[99] = "[$F[5]/$F[6]]"; $F[$#F] = join("", @fasta); push(@F, $origRef); print join("\t", @F);' | awk '{ print $1, "\t", $2, "\t", $6, "\t", $7, "\t", "\t", $15, "\t", $16 }' | sed 's/_/ /g' | perl -lane 'print join("\t", @F);' | awk '{ print $4 }' | perl -lane 'my @vals1 = split(//, $F[0]); print $#vals1;' | sort | uniq -c
    639 0
[  michaelt@node3  ~]$join -j 1 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.FinalResults.20kbWindow1eNeg4pVal.txt | perl -lane 'print $F[2], "_", $F[3], "\t", join("\t", @F);' | sort -k 1,1) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/GTEXWholeBloodForFollowup/DataAnalysis.All.FollowupGenotyping.GTEXWholeBlood.SNPsonly.FinalResults.20kbWindow1eNeg4pVal.Slop100.fasta | perl -ane 'if ($F[0] =~ m/^>/) { print join("\t", @F), "\t"; } else { print join("\t", @F), "\n"; }' | perl -lane 'my $Chr = ((split(/\>/, ((split(/\:/, $F[0]))[0])))[1]); my $BP = ((split(/\-/, ((split(/\:/, $F[0]))[1])))[0]) + 100; print $Chr, "_", $BP, "\t$F[0]\t$Chr\t$BP\t$F[1]";' | sort -k 1,1) | perl -lane 'my @fasta = split(//, $F[$#F]); my $origRef = $fasta[99]; $fasta[99] = "[$F[5]/$F[6]]"; $F[$#F] = join("", @fasta); push(@F, $origRef); print join("\t", @F);' | awk '{ print $1, "\t", $2, "\t", $6, "\t", $7, "\t", "\t", $15, "\t", $16 }' | sed 's/_/ /g' | perl -lane 'print join("\t", @F);' | awk '{ print $5 }' | perl -lane 'my @vals1 = split(//, $F[0]); print $#vals1;' | sort | uniq -c
    639 0
~~~







scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.VariantLevel.100bpFlanking.Update1.txt.gz mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/.

























##Enard et al 2016 eLife, Petrov human virus selection on genome paper stuff  -- https://elifesciences.org/content/5/e12469
##20160801

















##Putting list of follow-up results/variants together for Sudhir across the multiple analyses
##20160801

























