#!/bin/sh

#Notes for interpreting this code history/'lab notebook'

#Check_# refers to notes I made to myself to return to, where _0 indicates I have not dealt with my reminder/suggestion/necessity and _1 means I have looked at/dealt with what I was indicating I should

#I do my best to provide dates and timestamps for when things are occuring -- I also get better at this, and leaving notes/comments/annotations/explanations of things as the code/labnotebook goes on

#'~~~' is commonly employed to indicate 'informational' code or results are being included and will normally have some sort of ocmment or code nearby explaining why this information is being included and/or purposely separated/emphasized

~~~~~~~

#qsub -I -l walltime=72:00:00,mem=8gb,nodes=1:ppn=6

#sffFile contains a list of every sequencing read/run and the breakdown/separation of each original .sff file into each MID tag
#ALL_MAPPING is where each individual is mapped back to the appropriate MID tag/.sff file combination from sffFile directory 

#Create appropriate main directories in ALL_MAPPING and then appropriate subdirectories in each main directory (main directory i.e. Pools/mapping_pool_1-46/, and subdirecory i.e. POOL14-01RL/). Then creating appropriate directories for each .sff file original location in sffFile/,  

20130823 - Going through past runs of code and figuring out what was done initially; collecting this information here as the first entry in this MainScript1/readme file for documentation (tends to be what I use instead of the LABDIRECTORY notebook thing I had previously tried to setup/use)

ls -lrt /data/userdata/pg/ALL_MAPPING/. | awk '{ print $9 } ' > home_pg_ALL_MAPPING_dirList.20130513.txt

bash CollectLSInfoPerDir.sh

bash DataOrganization.CollectOrigSFFFileInformation.sh

cat ../sffFiles/home_pg_ALL_MAPPING_ALL_Pools_sff_OrigLocations.Unedited.20130513.txt | sort | uniq > ../sffFiles/home_pg_ALL_MAPPING_ALL_Pools_sff_OrigLocations.Unedited.20130513.SortUniq.txt

#bash ../sffFiles/BreakUpsffFiles.sh
bash ../sffFiles/MakeDir.sh

bash DataOrganization.CombinePoolNameswsffFileIDs.sh

#Edit DataOrganization.CombinePoolNameswsffFileIDs.Thu_May_16_14:48:16_CDT_2013.output -- commenting out lines that are incorrect with '#'

bash DataOrganization.ConnectPoolsffFileswithIndvMIDRLsffFiles.sh DataOrganization.CombinePoolNameswsffFileIDs.Thu_May_16_14:48:16_CDT_2013.Edited.output

bash DataOrganization.CollectAllIndvMIDRLsffFiles.sh

bash ../sffFiles/Process_sffFile.BreakUpMIDs.Launcher.vs1.sh

bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs1.sh DataOrganization.CollectAllIndvMIDRLsffFiles.Thu_May_16_14:54:21_CDT_2013.RedundantDirectoriesRemoved.output

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs1.sh DataOrganization.CollectAllIndvMIDRLsffFiles.Thu_May_16_14:54:21_CDT_2013.RedundantDirectoriesRemoved.output 0


2nd run with pools mapping_pool_91-97 and mapping_pool_98-105, as well as edits + updating code
~~~~~~~~~~~

ls -lrt /data/userdata/pg/ALL_MAPPING/. | awk '{ print $9 } ' > home_pg_ALL_MAPPING_dirList.20130827.txt

cat home_pg_ALL_MAPPING_dirList.20130827.txt > home_pg_ALL_MAPPING_dirList.20130827.NotCreatedYet.txt
#Then manually edited to remove directories already used/processed before -- kept mapping_pool_91-97 and mapping_pool_98-105

bash DataOrganization.CollectLSInfoPerDir.vs2.sh home_pg_ALL_MAPPING_dirList.20130827.NotCreatedYet.txt #Creates subdirectory folder-list per directory

bash DataOrganization.MakeDir.vs2.sh home_pg_ALL_MAPPING_dirList.20130827.NotCreatedYet.txt #Creates the rest of the sub-subdirectories, etc. in ALL_MAPPING

bash DataOrganization.CollectOrigSFFFileInformation.vs2.sh home_pg_ALL_MAPPING_dirList.20130827.NotCreatedYet.txt #Creates home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.${date1}.txt file

cat ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130828_123042.txt | sort | uniq > ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130828_123042.SortUniq.txt

#Edit -- Directory location for sff file H37BKPH01 was changed from /data/454/raw/2013/R_2013_02_08_11_30_15_FLX08100638_Administrator_CS_KK_PG_MM5_Pool_91-2/D_2013_02_09_15_00_17_node1_altSignalProcessing/sff/H37BKPH01.sff to /data/454/raw/2013/R_2013_02_08_11_30_15_FLX08100638_Administrator_CS_KK_PG_MM5_Pool_91-2/D_2013_02_12_10_29_52_node2_signalProcessing/sff/H37BKPH01.sff (and *02.sff)
(bash DataOrganization.CorrectPool91Script.vs1.sh)
#Edit -- Change made in DataOrganization.CombinePoolNameswsffFileIDs.vs2.20130828_124947.Edited.output as well too 

#bash ../sffFiles/BreakUpsffFiles.vs2.sh ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130828_123042.SortUniq.txt
bash ../sffFiles/DataOrganization.MakeDir.vs2.sh ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130828_123042.SortUniq.Edited.txt

bash DataOrganization.CombinePoolNameswsffFileIDs.vs2.sh home_pg_ALL_MAPPING_dirList.20130827.NotCreatedYet.txt ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130828_123042.SortUniq.Edited.txt #Creates file such as DataOrganization.CombinePoolNameswsffFileIDs.vs2.${date1}.output

bash DataOrganization.ConnectPoolsffFileswithIndvMIDRLsffFiles.vs2.sh home_pg_ALL_MAPPING_dirList.20130827.NotCreatedYet.txt DataOrganization.CombinePoolNameswsffFileIDs.vs2.20130828_124947.Edited.output 

bash DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.sh home_pg_ALL_MAPPING_dirList.20130827.NotCreatedYet.txt #Creates DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.${date1}.output file 

bash ../sffFiles/DataProcessing.BreakUpMIDs.Launcher.vs2.sh ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130828_123042.SortUniq.Edited.txt

bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130828_141053.output

#cat DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130828_141053.output | grep -E 'H37BKPH|H4KITRJ|H4PUW3D' > DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130828_141053.Leftover.output

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130828_141053.output

#cp -p home_pg_ALL_MAPPING_dirList.20130827.txt home_pg_ALL_MAPPING_dirList.WorkingMainList.txt
#cat DataOrganization.CombinePoolNameswsffFileIDs.Thu_May_16_14\:48\:16_CDT_2013.Edited.output DataOrganization.CombinePoolNameswsffFileIDs.vs2.20130828_124947.output > DataOrganization.CombinePoolNameswsffFileIDs.vs2.WorkingMainFile.output #Combining all sections. Trying to use 'WorkingMainFile' as means to keep all parts together
#cat DataOrganization.CollectAllIndvMIDRLsffFiles.Thu_May_16_14\:54\:21_CDT_2013.RedundantDirectoriesRemoved.JustFirstTwoCols.output DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130828_141053.output > DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.output
#cp -p DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.output DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output 
#Made DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output  file by changing all earlier directories for mapping_pools.... to include the /Pools/... directory via the following command in vi .,$ s/ING\/mapping_pool/ING\/Pools\/mapping_pool/g 

#At this point catching any .fastq files or other downstream processed files that were lost or not properly processed in order for them to be resubmitted
bash DataProcessing.Pipeline.CheckForErroneousFiles.vs1.sh home_pg_ALL_MAPPING_dirList.WorkingMainList.txt DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output fastq #Create DataProcessing.Pipeline.CheckForErroneousFiles.vs1.DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.Redo.${SearchTerm1}.${date1}.output file

bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataProcessing.Pipeline.CheckForErroneousFiles.vs1.DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.Redo.fastq.20130830_165929.output

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataProcessing.Pipeline.CheckForErroneousFiles.vs1.DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.Redo.fastq.20130830_165929.output


3rd run with mapping_pool_106-124
~~~~~~~~~~~

ls -lrt /data/userdata/pg/ALL_MAPPING/. | awk '{ print $9 } ' > home_pg_ALL_MAPPING_dirList.20130831.txt

cat home_pg_ALL_MAPPING_dirList.20130831.txt > home_pg_ALL_MAPPING_dirList.20130831.NotCreatedYet.txt
#Then manually edited to remove directories already used/processed before -- kept mapping_pool_106-124

bash DataOrganization.CollectLSInfoPerDir.vs2.sh home_pg_ALL_MAPPING_dirList.20130831.NotCreatedYet.txt #Creates subdirectory folder-list per directory

bash DataOrganization.MakeDir.vs2.sh home_pg_ALL_MAPPING_dirList.20130831.NotCreatedYet.txt #Creates the rest of the sub-subdirectories, etc. in ALL_MAPPING

bash DataOrganization.CollectOrigSFFFileInformation.vs2.sh home_pg_ALL_MAPPING_dirList.20130831.NotCreatedYet.txt #Creates home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.${date1}.txt file

cat ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130831_103251.txt | sort | uniq > ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130831_103251.SortUniq.txt

bash ../sffFiles/DataOrganization.MakeDir.vs2.sh ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130831_103251.SortUniq.txt

bash DataOrganization.CombinePoolNameswsffFileIDs.vs2.sh home_pg_ALL_MAPPING_dirList.20130831.NotCreatedYet.txt ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130831_103251.SortUniq.txt #Creates file such as DataOrganization.CombinePoolNameswsffFileIDs.vs2.${date1}.output

bash DataOrganization.ConnectPoolsffFileswithIndvMIDRLsffFiles.vs2.sh home_pg_ALL_MAPPING_dirList.20130831.NotCreatedYet.txt DataOrganization.CombinePoolNameswsffFileIDs.vs2.20130831_103251.output 

bash DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.sh home_pg_ALL_MAPPING_dirList.20130831.NotCreatedYet.txt #Creates DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.${date1}.output file 

bash ../sffFiles/DataProcessing.BreakUpMIDs.Launcher.vs2.sh ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130831_103251.SortUniq.txt

bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130831_103352.output

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130831_103352.output

cat ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130831_103251.SortUniq.txt | grep H85C36T01 > ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130831_103251.SortUniq.Leftover.txt #Then ran ../sffFiles/DataProcessing.BreakUpMIDs.Launcher.vs2.sh

cat DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130831_103352.output | grep H85C36T01 > DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130831_103352.Leftover.output #Then ran DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh and DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh

#Also H9H1D9W01

#cat DataOrganization.CombinePoolNameswsffFileIDs.Thu_May_16_14\:48\:16_CDT_2013.Edited.output DataOrganization.CombinePoolNameswsffFileIDs.vs2.20130828_124947.output DataOrganization.CombinePoolNameswsffFileIDs.vs2.20130831_103251.output > DataOrganization.CombinePoolNameswsffFileIDs.vs2.WorkingMainFile.output
#cat DataOrganization.CollectAllIndvMIDRLsffFiles.Thu_May_16_14\:54\:21_CDT_2013.RedundantDirectoriesRemoved.JustFirstTwoCols.output DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130828_141053.output DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130831_103352.output > DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.output
#cp -p DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.output DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output 

bash DataProcessing.Pipeline.CheckForErroneousFiles.vs2.sh home_pg_ALL_MAPPING_dirList.WorkingMainList.txt DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output fastq

bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataProcessing.Pipeline.CheckForErroneousFiles.vs1.DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.Redo.fastq.20130831_235915.output

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataProcessing.Pipeline.CheckForErroneousFiles.vs1.DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.Redo.fastq.20130831_235915.output

cat home_pg_ALL_MAPPING_ALL_Pools_sff_OrigLocations.Unedited.20130513.SortUniq.txt home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130828_123042.SortUniq.Edited.txt home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130831_103251.SortUniq.txt > home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.WorkingMainList.SortUniq.txt

bash ../sffFiles/DataProcessing.BreakUpMIDs.Launcher.vs2.sh ../sffFiles/DataProcessing.Pipeline.CheckForErroneousFiles.vs1.DataProcessing.BreakUpMIDs.vs1.WorkingMainFile.Edited.Redo.fastq.20130831_235915.output


20130903 -- Created the PreMerge and PostMerge Pools/separation in the folder hierarchy; began continued development of pt3/the merging/dupl/BQSR steps in the pipeline
~~~~~~~~~~~
PreMerge

bash DataProcessing.RemoveaCertainFile.vs1.sh home_pg_ALL_MAPPING_dirList.WorkingMainList.txt PreMerge #Cleaned up old .preGATK* and *sam.gz files so can more easily locate folders where the pt2 code did not run properly (e.g. a lack of these files existing versus trying to find files with the proper timestamp signature)

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output #Redoing everything

#Note -- at this point changed original .pt3 of the pipeline scripts to .pt4 and made the 'merge file' portion of the original .pt3 into a stand alone .pt3

bash DataOrganization.AddOrRemoveRGInformation.Launcher.vs1.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output #Run of DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh included an error for the @RG information so used an extra script to correct every files. This is not intended to be normally used in the pipeline and the DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.vs2.sh script has been corrected

bash DataOrganization.CreatebamMergeList.vs1.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output #Creates file DataOrganization.CreatebamMergeList.vs1.${date1}.output

bash DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20130909_231450.output

bash DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20130909_231450.output

#Possibly was going to drop reads that did not map to expected probes but delaying this decisions since some issues came out. Will come back to figure out if this is proper after putting the rest of the pipeline together
#samtools view POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.bam | python ../../../../DataProcessing.Pipeline.DropOffTargetReads.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/OID41305_HG19_Match5_probes/OID41305_HG19_Match5_probe_coverage.bed --file2 - | vi -
#R --vanilla --slave --args < ../../../../DataProcessing.Pipeline.DropOffTargetReads.plot.R /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/OID41305_HG19_Match5_probes/OID41305_HG19_Match5_probe_coverage.chr4.bed POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.chr4.formattted POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.chr4.OverlapPlot.jpeg
#R --vanilla --slave --args < ../../../../DataProcessing.Pipeline.DropOffTargetReads.plot.R /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/OID41305_HG19_Match5_probes/OID41305_HG19_Match5_probe_coverage.chr4.bed 454Contigs.chr4.formatted 454Contigs.chr4.OverlapPlot.jpeg
#
#20131029 edited
#R --vanilla --slave --args < ../../../../DataProcessing.Pipeline.DropOffTargetReads.plot.R /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.chr4.bed POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.chr4.formattted POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.chr4.OverlapPlot.120326_HG19_SW_MM5_EZ.jpeg

#Created pt5 to be run on a single interactive node, but running ReduceReads and creating a raw .vcf file for the entire, merged .bam file at once was too long. Created support scripts to break up merged .bam file into separate chromosome .bam files so that raw .vcfs could be made and ReduceReads could be run
#bash DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged DataOrganization.CreatebamMergeList.vs1.20130916_105823.output

bash DataProcessing.Pipeline.pt5a.SplitBamIntoChromosomes.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.preGATK.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt5b.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.preGATK.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt5c.RunReduceReadsPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.preGATK.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged


20130923 - 4th run with mapping_pool_SOLID
~~~~~~~~~~~

#Manually created directories in ALL_MAPPING Pools subfolders following the excel spreadsheet POP_genomics_sample_information_2 and used 'touch' to create expected .sff files using ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130925_103251.SortUniq.txt created below to determine who got what .sff file

ls -lrt /data/454/raw/2011/*PG*/*/sff/. | grep sff | grep -E 'raw|H' | perl -sane 'if ($F[0] =~ m/raw/) { $tempVal1 = $F[0]; } else { if ($tempVal1 =~ m/(.*)(\.\:)/) { print $1, $F[8], "\n"; } }' > ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130925_103251.SortUniq.txt 

#Manually added in SOLID pools from 2012 directory

bash ../sffFiles/DataOrganization.MakeDir.vs2.sh ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130925_103251.SortUniq.txt

bash DataOrganization.CombinePoolNameswsffFileIDs.vs2.sh home_pg_ALL_MAPPING_dirList.20130831.NotCreatedYet.txt ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130925_103251.SortUniq.txt #Creates file such as DataOrganization.CombinePoolNameswsffFileIDs.vs2.${date1}.output

#Had to clean up file DataOrganization.CombinePoolNameswsffFileIDs.vs2.20130925_150954.output manually

bash DataOrganization.ConnectPoolsffFileswithIndvMIDRLsffFiles.vs2.sh home_pg_ALL_MAPPING_dirList.20130831.NotCreatedYet.txt DataOrganization.CombinePoolNameswsffFileIDs.vs2.20130925_150954.output

bash DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.sh home_pg_ALL_MAPPING_dirList.20130831.NotCreatedYet.txt #Creates DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.${date1}.output file

cp -p DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130925_152507.output DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130925_152507.Edited.output

#Manually removed some erroneous entries from DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130925_152507.Edited.output 

bash ../sffFiles/DataProcessing.BreakUpMIDs.Launcher.vs2.sh ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130925_103251.SortUniq.txt

bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130925_152507.Edited.output

#NOTE: At this point rerunning everything with QC pipeline components

DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh

bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130925_152507.Edited.output

#DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output did not include mapping_SOLID runs yet at this point. Updated the file afterwards (since there were different needs for this pt1 script in particular for each group of folder/files that would not matter with later downstream pts).
bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output

cat DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130925_152507.Edited.output > asdadad

mv asdadad DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output 

bash DataProcessing.Pipeline.CheckForPresenceOfCertainFile.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output PreMerge QCed.preGATK.QCed.bam

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataProcessing.Pipeline.CheckForPresenceOfCertainFile.vs2.QCed.preGATK.QCed.bam.20130928_134627.output

bash DataOrganization.CreatebamMergeList.vs1.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output #Creates file DataOrganization.CreatebamMergeList.vs1.${date1}.output

#Manually merged POOL39_2-... with POOL39-... lines

bash DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20130928_145300.output

bash DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20130928_145300.output

#Manually curated 20130930_POP_genomics_sample_information_2.edited.csv to look for duplicate entries and remove those that had less coverage
#Ran below code to cut down DataOrganization.CreatebamMergeList.vs1.20130928_145300.output to only include those individuals being used (and not those either missing IDs, exist as duplicates, etc.). For the moment deciding not to merge individuals who were run twice on MM5 (maybe bias with extra coverage on just a few individuals?) or run on SOLID and MM5. Also generally opted to use MM5 over SOLID unless coverage in SOLID was much better than MM5 (which was generally not the case). A few individuals were run twice on MM5, versus on SOLID and MM5
#cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20130930_POP_genomics_sample_information_2.edited.noMeta.dropIndvs.csv | grep ool | sort | wc -- to help check expected total number of entries to be kept when using DataProcessing.Pipeline.DropIndividualsBasedOnPopGenomFile.vs2.py with 20130930_POP_genomics_sample_information_2.edited.noMeta.dropIndvs.csv

python DataProcessing.Pipeline.DropIndividualsBasedOnPopGenomFile.vs2.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20130930_POP_genomics_sample_information_2.edited.noMeta.dropIndvs.csv --file2 DataOrganization.CreatebamMergeList.vs1.20130928_145300.output > DataOrganization.CreatebamMergeList.vs1.20130928_145300.dropIndvs.output

#bash DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged DataOrganization.CreatebamMergeList.vs1.20130928_145300.dropIndvs.output

bash DataProcessing.Pipeline.pt6a.SplitBamIntoChromosomes.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6b.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6c.RunReduceReadsPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

python DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20130930_POP_genomics_sample_information_2.edited.noMeta.dropIndvs.csv --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Sample_status_list_9-16-13_CT.edited.noMeta.csv > DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output
echo "macsid macsid MID# POOLID-MID# race posage hesn lesn lesc othersc vrapid rapid vslow slow timetoART timeaids" > DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output.header 
#20131004 Note -- four individuals not present in Sample_status_list file, 41478,10,POOL43-10RL
#20040,7,POOL18S-07RL
#30550,11,POOL101-11RL
#20770,3,POOL2S-03RL
#Unsure why at current point, will consult with Kevin/Sudhir after first run through of downstream analysis is done. Also, with these four individuals, total count is 756, not 757. Kevin suggested should have 757 individuals in total, so possibly missing someone? Unless POOL39 is being double counted by their count as well
#20131004 Note -- POOL57 MID10 has folder name ...-010RL for whatever reason originally, probably typo. Just fyi to be aware.

#20131004 Note -- at this point changed naming schematics to go from pt6a, pt6b, pt6c, etc... to go to pt6.All.pta, pt6.Samtools.pta, pt6.GATK.pta, etc... to delineate the different branching paths of the analysis steps for each analysis/variant-calling tool being used, with steps/code being used by all branches given the pt6.All.pt.... designation

python DataProcessing.Pipeline.CreatePhenotypeFile.vs1.py --file1 DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno

python DataProcessing.Pipeline.CreateCovariateFile.vs1.py --file1 DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar | awk '{ if ($5 == "1") { print $1, "\t", $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDIIDs

bash DataProcessing.Pipeline.pt6.All.ptb.ConvertVCFtoPEDPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools 

bash DataProcessing.Pipeline.pt6.All.ptc.CollapsePedFilesIntoSingleFile.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools

bash DataProcessing.Pipeline.pt6.All.ptd.PLINKlogisticRegression.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools

(20131012 -- time pt check)

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt --extract AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.bim.SNPsonly.snpIDs --r2 --ld-window-r2 0.8 --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r8 &

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt --extract AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.bim.SNPsonly.snpIDs --r2 --ld-window-r2 0 --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0 &

python ../../../../DataProcessing.Pipeline.pruneLDbased.py --file1 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld --r2 .1 > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt &

python ../../../../DataProcessing.Pipeline.pruneLDbased.py --file1 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld --r2 .8 > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump80.txt &

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt | awk '{ print $1 }' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt.snpList

perl /home/michaelt/Scripts/Perl/ListSubset.pl --file1 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt.snpList --subset 250000 --output AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt.snpList.250k

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt --extract AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt.snpList.250k --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k --recode --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.ped

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.pedind

ln -s AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.bim AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.pedsnp

/home/michaelt/Software/EIG5.0.1/bin/smartpca -p AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.parfile
#Note -- ran this on spudhead b/c smartpca was working already there but wasn't on olivia/miranda yet. Will ask Aleksandar to install eigenstrat if possible at some point

python ../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.Results.pca.altrd.race
#scp'ing the necessary files back and forth from work computer and spudhead

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.Results.pca.altrd.race -c 1:2 -p American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic:White_non-Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.Results.pca.altrd.race.xtxt

#/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.Results.pca.altrd.race -c 1:2 -p American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic:White_non-Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.Results.pca.altrd.race.xtxt

20131013
~~~~~~~~~~~
#At this point noticed names were off in that POOL01S, POOL02S, etc... were not included, just POOL01, POOL02, etc...so had duplicates, i.e. SOLID array names not captured properly in the .bam files and all subsequent processes. Reran SOLID arrays and then merge/post-merge scripts

cat DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output | grep SOLID > DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output.SOLID

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output.SOLID

cat DataOrganization.CreatebamMergeList.vs1.20130928_145300.dropIndvs.output | grep SOLID > DataOrganization.CreatebamMergeList.vs1.20130928_145300.dropIndvs.output.SOLID

bash DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20130928_145300.dropIndvs.output.SOLID

bash DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20130928_145300.dropIndvs.output.SOLID

#bash DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged DataOrganization.CreatebamMergeList.vs1.20130928_145300.dropIndvs.output

bash DataProcessing.Pipeline.pt6.All.pta.SplitBamIntoChromosomes.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.Samtools.pta.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.GATK.pta.RunReduceReadsPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.All.ptb.ConvertVCFtoPEDPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/Samtools AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools

bash DataProcessing.Pipeline.pt6.All.ptc.CollapsePedFilesIntoSingleFile.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/Samtools AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.bim | perl -lane 'if (($F[4] =~ m/^[AGTC]$/) && ($F[5] =~ m/^[AGTC]$/)) { print join("\t", @F);}' > Pools/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.bim.SNPsonly

cat Pools/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.bim.SNPsonly | awk '{ print $2 }' > Pools/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.bim.SNPsonly.snpIDs

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt --extract AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.bim.SNPsonly.snpIDs --r2 --ld-window-r2 0.8 --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r8 &

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt --extract AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.bim.SNPsonly.snpIDs --r2 --ld-window-r2 0 --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0 &

python ../../../../DataProcessing.Pipeline.pruneLDbased.py --file1 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld --r2 .1 > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt &

python ../../../../DataProcessing.Pipeline.pruneLDbased.py --file1 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld --r2 .8 > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump80.txt &

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt | awk '{ print $1 }' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt.snpList

perl /home/michaelt/Scripts/Perl/ListSubset.pl --file1 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt.snpList --subset 250000 --output AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt.snpList.250k

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt --extract AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt.snpList.250k --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k --recode --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.ped

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.pedind

ln -s AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.bim AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.pedsnp

/home/michaelt/Software/EIG5.0.1/bin/smartpca -p AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.parfile
#Note -- ran this on spudhead b/c smartpca was working already there but wasn't on olivia/miranda yet. Will ask Aleksandar to install eigenstrat if possible at some point

python ../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.Results.pca.altrd.race

python ../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.noOutlierRmvl.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.Results.pca.altrd.race -c 1:2 -p American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic:White_non-Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.Results.pca.altrd.race.xtxt


bash DataProcessing.Pipeline.pt6.All.ptd.PLINKlogisticRegression.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools

bash DataProcessing.Pipeline.pt6.GATK.ptb.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged 

bash ../../../../DataProcessing.Pipeline.pt6.GATK.ptc.MergeVCFsAcrossChromosomes.vs1.sh

bash DataProcessing.Pipeline.pt6.GATK.ptd.RecalAndApplyVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

vcftools --gzvcf AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz --plink --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP

plink --file AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP

bash DataProcessing.Pipeline.pt6.All.ptd.PLINKlogisticRegression.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --maf .01 --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01

perl /home/michaelt/Scripts/Perl/ListSubset.pl --file1 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.bim --subset 250000 --output AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.bim

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.bim | awk '{ print $2 }' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.bim.onlySNPids

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01 --extract AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.bim.onlySNPids --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k --recode --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.ped

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.pedind 

ln -s AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.bim AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.pedsnp

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --maf .01 --geno .95 --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95 --recode --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.ped

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.pedind

ln -s AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.bim AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.pedsnp

/home/michaelt/Software/EIG5.0.1/bin/smartpca -p AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.parfile

python ../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.Results.pca.altrd.race

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.Results.pca.altrd.race.plot.1vs2.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.Results.pca.altrd.race.plot.3vs4.xtxt

python ../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.Results.pca.altrd.race

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.Results.pca.altrd.race.plot.1vs2.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.Results.pca.altrd.race.plot.3vs4.xtxt

python ../../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.noOutlierRmvl.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race

python ../../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.noOutlierRmvl.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race.plot.1vs2.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race.plot.3vs4.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race.plot.1vs2.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race.plot.3vs4.xtxt


plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt --maf .01 --geno .95 --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.bim | perl -lane 'if (($F[4] =~ m/^[AGTC]$/) && ($F[5] =~ m/^[AGTC]$/)) { print join("\t", @F);}' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.bim.SNPsonly

perl /home/michaelt/Scripts/Perl/ListSubset.pl --file1 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.bim.SNPsonly --subset 250000 --output AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.250k.bim.SNPsonly

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.250k.bim.SNPsonly | awk '{ print $2 }' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.250k.bim.SNPsonly.onlySNPids

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95 --extract AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.250k.bim.SNPsonly.onlySNPids --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k --recode --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.ped

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.pedind

ln -s AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.bim AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.pedsnp

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/smartpca -p AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.250k.parfile

python ../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.Results.pca.altrd.race

python ../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.noOutlierRmvl.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.Results.pca.altrd.race.plot.1vs2.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.Results.pca.altrd.race.plot.3vs4.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race.plot.1vs2.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race.plot.3vs4.xtxt


python DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.Results.pca --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.5PCs.covar
#Edit code in between runs
python DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.Results.pca --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.5PCs.covar | awk '{ print $1, "\t", $2 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.5PCs.covar.FIDIIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.5PCs.covar | awk '{ if ($5 == "1") { print $1, "\t", $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.5PCs.covar.WhiteOnly.FIDIIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar | awk '{ print $1, "\t", $2 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar | awk '{ if ($5 == "1") { print $1, "\t", $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.WhiteOnly.FIDIIDs

bash DataProcessing.Pipeline.pt6.All.ptd.PLINKlogisticRegression.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP
#Edit code in between runs
bash DataProcessing.Pipeline.pt6.All.ptd.PLINKlogisticRegression.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.5PCs.covar --covar-number 1,2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.5PCs.covar.WhiteOnly.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno1.5PCsmaf01geno95.Assoc 

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.5PCs.covar --covar-number 1,2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.5PCs.covar.WhiteOnly.FIDIIDs --maf .01 --geno .95 --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.WhitesOnly.Pheno1.5PCsmaf01geno95.Assoc 

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.WhiteOnly.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno1.10PCsmaf01geno95.Assoc 

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.WhiteOnly.FIDIIDs --maf .01 --geno .95 --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.WhitesOnly.Pheno1.10PCsmaf01geno95.Assoc 

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.Assoc

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.5PCsmaf01geno05.Assoc

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.10PCsmaf01geno05.Assoc

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno1.5PCsmaf01geno95.Assoc.assoc.logistic Pheno1_Logistic_Whites_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno1.5PCsmaf01geno95.Assoc.assoc.logistic.QQplot.jpeg & 

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.WhitesOnly.Pheno1.5PCsmaf01geno95.Assoc.assoc.logistic Pheno1_Logistic_Whites_maf01geno95_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.WhitesOnly.Pheno1.5PCsmaf01geno95.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno1.10PCsmaf01geno95.Assoc.assoc.logistic Pheno1_Logistic_Whites_10PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno1.10PCsmaf01geno95.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.WhitesOnly.Pheno1.10PCsmaf01geno95.Assoc.assoc.logistic Pheno1_Logistic_Whites_maf01geno95_10PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.WhitesOnly.Pheno1.10PCsmaf01geno95.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.Assoc.assoc.logistic Pheno1_Logistic AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.5PCsmaf01geno05.Assoc.assoc.logistic Pheno1_Logistic_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.5PCsmaf01geno05.Assoc.assoc.logistic.QQPlot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.10PCsmaf01geno05.Assoc.assoc.logistic Pheno1_Logistic_10PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.10PCsmaf01geno05.Assoc.assoc.logistic.QQPlot.jpeg &

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --maf .01 --geno .95 --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno1.5PCsmaf01geno05.Assoc

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --maf .01 --geno .95 --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno1.10PCsmaf01geno05.Assoc

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno1.5PCsmaf01geno05.Assoc.assoc.logistic Pheno1_Logistic_maf01geno95_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno1.5PCsmaf01geno05.Assoc.assoc.logistic.QQplot.jpeg & 

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno1.10PCsmaf01geno05.Assoc.assoc.logistic Pheno1_Logistic_maf01geno95_10PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno1.10PCsmaf01geno05.Assoc.assoc.logistic.QQplot.jpeg & 

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.NoCovar.Assoc

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.justPOOLcovar.Assoc

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 2 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.justMIDcovar.Assoc

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.noPOOLMIDcovar.5PCsmaf01geno05.Assoc

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.NoCovar.Assoc.assoc.logistic Pheno1_Logistic_noCovar AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.NoCovar.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.justPOOLcovar.Assoc.assoc.logistic Pheno1_Logistic_POOLCovar AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.justPOOLcovar.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.justMIDcovar.Assoc.assoc.logistic Pheno1_Logistic_MIDCovar AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.justMIDcovar.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.noPOOLMIDcovar.5PCsmaf01geno05.Assoc.assoc.logistic Pheno1_Logistic_noPOOLMID_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.noPOOLMIDcovar.5PCsmaf01geno05.Assoc.assoc.logistic.QQplot.jpeg &

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 2 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno2.5PCsmaf01geno05.Assoc

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno2.5PCsmaf01geno05.Assoc.assoc.logistic Pheno2_Logistic_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno2.5PCsmaf01geno05.Assoc.assoc.logistic.QQplot.jpeg &  

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 2 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno2.Assoc

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 2 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.WhiteOnly.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno2.5PCsmaf01geno05.Assoc

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 2 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --maf .01 --geno .95 --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno2.5PCsmaf01geno05.Assoc

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno2.Assoc.assoc.logistic Pheno2_Logistic AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno2.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno2.5PCsmaf01geno05.Assoc.assoc.logistic Pheno2_Logistic_WhitesOnly_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno2.5PCsmaf01geno05.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno2.5PCsmaf01geno05.Assoc.assoc.logistic Pheno2_Logistic_maf01geno95_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno2.5PCsmaf01geno05.Assoc.assoc.logistic.QQplot.jpeg &

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz | grep PASS | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.body.gz

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz | wc
1272767 968575687 5440128566

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz | awk '{ if ($1 == "12" ) { print $0 } } ' | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr12.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz

perl /home/michaelt/Scripts/Perl/ListSubset.pl --file1 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr12.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz --subset 10000 --output AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr12.GATK.ReduceReads.UG.VQSR.SNP.vcf.ran10000

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz | head -n 123 | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.header.gz

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.header.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.body.gz | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz

rm AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.body.gz 

vcftools --gzvcf AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz --plink --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS

plink --file AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS

perl /home/michaelt/Scripts/Perl/ListSubset.pl --file1 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.bim --subset 250000 --output AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.bim.250k

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.bim.250k | awk '{ print $2 }' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.bim.250k.snpIDs

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS --extract AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.bim.250k.snpIDs --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k --recode --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.ped

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.pedind

ln -s AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.bim AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.pedsnp

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/smartpca -p AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.parfile

python ../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.Results.pca.altrd.race

python ../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.noOutlierRmvl.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.Results.pca.altrd.race.plot.1vs2.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.Results.pca.altrd.race.plot.3vs4.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race.plot.1vs2.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race.plot.3vs4.xtxt

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 3 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno3.5PCsmaf01geno05.Assoc

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno3.5PCsmaf01geno05.Assoc.assoc.logistic Pheno3_Logistic_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno3.5PCsmaf01geno05.Assoc.assoc.logistic.QQplot.jpeg &  

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 3 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno3.Assoc

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 3 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.WhiteOnly.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno3.5PCsmaf01geno05.Assoc

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 3 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --maf .01 --geno .95 --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno3.5PCsmaf01geno05.Assoc

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno3.Assoc.assoc.logistic Pheno3_Logistic AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno3.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno3.5PCsmaf01geno05.Assoc.assoc.logistic Pheno3_Logistic_WhitesOnly_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno3.5PCsmaf01geno05.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno3.5PCsmaf01geno05.Assoc.assoc.logistic Pheno3_Logistic_maf01geno95_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno3.5PCsmaf01geno05.Assoc.assoc.logistic.QQplot.jpeg &


PASS run

python DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.Results.pca --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.PASS.maf01.geno95.phenoColEdit.5PCs.covar
#Edit code in between runs
python DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.Results.pca --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.PASS.maf01.geno95.phenoColEdit.10PCs.covar


Annovar (timepoint chk: 20131029)
perl /home/michaelt/Software/annovar/annotate_variation.pl --geneanno  /home/michaelt/Software/annovar/ -buildver hg19 #annotation: 1        861283          861283          G       C
perl /mnt/lustre/home/mturchin20/Software/annovar/annotate_variation.pl --geneanno hg19_Variants4ESP6515.AnnovarFormat.txt /mnt/lustre/home/mturchin20/Software/annovar/humandb/ -buildver hg19
perl /mnt/lustre/home/mturchin20/Software/annovar/summarize_annovar.pl hg19_Variants4ESP6515.AnnovarFormat.txt /mnt/lustre/home/mturchin20/Software/annovar/humandb/ -buildver hg19

python DataPrcoessing.Pipeline.Utility.LinkGeneInfoListwTxStStpInfo.vs1.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/liquid_array.csv > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/liquid_array.wChrTrxStartStop.vs2.csv
#This script used the geneID symbol to compare the two datasets. Decided this was not adequate and used the UCSC ID and HugoID in the second version of the code below

python DataPrcoessing.Pipeline.Utility.LinkGeneInfoListwTxStStpInfo.vs2.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /home/michaelt/Data/Hugo/HGNC_GeneInformation_wUCSCIDs.txt --file3 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/liquid_array.csv > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/liquid_array.wChrTrxStartStop.vs3.csv

python DataProcessing.Pipeline.pt6.All.pte.annotateVariantsManuallyGeneID.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/liquid_array.wChrTrxStartStop.vs3.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr12.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr12.GATK.ReduceReads.UG.VQSR.SNP.vcf.selfAnnotation.gz

python DataProcessing.Pipeline.pt6.All.pte.annotateVariantsManuallyGeneID.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/liquid_array.wChrTrxStartStop.vs3.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.selfAnnotation.gz

python DataProcessing.Pipeline.pt6.All.pte.annotateVariantsManuallyGeneID.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/liquid_array.wChrTrxStartStop.vs3.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr12.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz --findClosest 1 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr12.GATK.ReduceReads.UG.VQSR.SNP.vcf.selfAnnotation.gz

python DataProcessing.Pipeline.pt6.All.pte.annotateVariantsManuallyGeneID.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/liquid_array.wChrTrxStartStop.vs3.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz --findClosest 1 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.selfAnnotation.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz | awk '{ print $1, "\t", $2, "\t", $2, "\t", $4, "\t", $5 }' | grep -v ^# > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.AnnovarFormat

#perl /home/michaelt/Software/annovar/annotate_variation.pl --downdb refGene /home/michaelt/Software/annovar -build hg19
#PWD: /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK

perl /home/michaelt/Software/annovar/annotate_variation.pl --geneanno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.AnnovarFormat /home/michaelt/Software/annovar/ -buildver hg19 

perl /home/michaelt/Software/annovar/summarize_annovar.pl /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.AnnovarFormat /home/michaelt/Software/annovar/ --ver1000g 1000g2012apr --verdbsnp 129 --veresp 6500 --buildver hg19 --remove

#perl /home/michaelt/Software/annovar/annotate_variation.pl -filter -dbtype ljb2_all -buildver hg19 -outfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.AnnovarFormat.ljb2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.AnnovarFormat /home/michaelt/Software/annovar/ -otherinfo




20131111 - 5th run fine-tuning QC setup from beginning of fastq file processing, so redoing everything from the start; still all SOLID and MM5 individuals
~~~~~~~~~~~

mv DataPrcoessing.Pipeline.Utility.LinkGeneInfoListwTxStStpInfo.vs2.py DataProcessing.Pipeline.Utility.LinkGeneInfoListwTxStStpInfo.vs2.py
mv DataPrcoessing.Pipeline.Utility.LinkGeneInfoListwTxStStpInfo.vs1.py DataProcessing.Pipeline.Utility.LinkGeneInfoListwTxStStpInfo.vs1.py

bash DataProcessing.Pipeline.Utility.GetFastqPreQCStats.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output

bash DataProcessing.Pipeline.Utility.RemoveaCertainFile.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output #To remove QCed.fastq.gz and associated stats files for new run through

bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output

mv DataProcessing.Pipeline.Utility.GetFastqPreQCStats.sh DataProcessing.Pipeline.Utility.GetFastqPreQCStats.vs1.sh

cp -p DataProcessing.Pipeline.Utility.GetFastqPreQCStats.vs1.sh DataProcessing.Pipeline.Utility.GetFastqQCStats.vs1.sh

bash DataProcessing.Pipeline.Utility.GetFastqQCStats.vs1.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output

#Notes: original sff location of /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PreMerge/mapping_pool_1-46/POOL20-12RL/. has 4 MIDs with good quality -- maybe the MID shouldn't be 12? Nah, there are 7 MIDs for POOL20 so it's unlikely to just be a MID12 thin
#Redo sff of /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PreMerge/mapping_pool_69-77/POOL75-09RL/ HX05RSL01.RL9.sff and HX05RSL02.RL9.sff

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output

cp -p DataProcessing.Pipeline.CheckForPresenceOfCertainFile.vs2.sh DataProcessing.Pipeline.CheckForPresenceOfCertainFile.vs3.sh
 
mv DataProcessing.Pipeline.CheckForPresenceOfCertainFile.vs3.sh DataProcessing.Pipeline.Utility.CheckForPresenceOfCertainFile.vs3.sh

bash DataProcessing.Pipeline.Utility.CheckForPresenceOfCertainFile.vs3.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output #searching for $baseFile1.QCed.preGATK.QCed.bam

mv DataOrganization.Utility.CreatebamMergeList.vs1.sh DataOrganization.Utility.CreatebamMergeList.vs1.sh

cp -p DataOrganization.Utility.CreatebamMergeList.vs1.sh DataOrganization.Utility.CreatebamMergeList.vs2.sh

bash DataOrganization.Utility.CreatebamMergeList.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output #Double-checking that this file was made appropriately and that all lines of the original file were used to create the eventual output. Just crossing t's and dotting i's here
#Note do the dropped individual thing to the output file from this run when I get back -- can delete this line later when I've done what I've asked in it, e.g. edit the output file to drop the individuals like was done earlier in this list/history of code

bash DataProcessing.Pipeline.Utility.RemoveaCertainFile.vs2.sh DataOrganization.CreatebamMergeList.vs1.20130928_145300.output 

python DataProcessing.Pipeline.DropIndividualsBasedOnPopGenomFile.vs2.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20130930_POP_genomics_sample_information_2.edited.noMeta.dropIndvs.csv --file2 DataOrganization.CreatebamMergeList.vs1.20131114_193315.output > DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output

bash DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output

bash DataOrganization.Utility.pt3.CheckOrigDirectoriesForFilesMissed.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output 
#Checking to see, given the directories specified in DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output and grepping all the sff files from them, are those sff files present in the single line being used to merge that particular individual's total bam file. E.g. seeing if files present in the directories are properly in the 'CreatebamMergeList' line representing that/those directories for that individual

bash DataOrganization.Utility.pt3.CheckMergedBamsForContainingProperFiles.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output
#Checking to see whether the merged bam files contain all the files we expect it to given the source line of files to combine from DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output

cp -p DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.vs1.sh DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.vs2.sh

bash DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output

bash DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs1.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output

cat 

bash DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output

cat DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output | perl -lane 'my @vals = split(/;/, $F[0]); if ($vals[0] !~ m/(POOL[a-zA-Z0-9]+-\d\dRL)/) { print join("\t", @F) ; } ' > DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output.Missing
#Realized had POOL[a-zA-Z0-9]+_\d\dRL statement in DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.vs2.sh instead of POOL[a-zA-Z0-9]+.*\d\dRL so stats file not properly formatted and needed to redo them

bash DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output.Missing

cat DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.output | perl -lane 'my @array1; push(@array1, $F[1]); foreach my $entry (@F) { if ($entry !~m/[a-zA-Z]/) { push (@array1, $entry); } } print join(",", @array1);' > DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.output.reformatted


#bash DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output

bash DataProcessing.Pipeline.pt6.All.pta.SplitBamIntoChromosomes.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.Samtools.pta.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.GATK.pta.RunReduceReadsPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.All.ptb.ConvertVCFtoPEDPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/Samtools AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools

bash DataProcessing.Pipeline.pt6.All.ptc.CollapsePedFilesIntoSingleFile.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/Samtools AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools


#Do reformatting of other statsoutput from earlier in the pipeline (the per-region/run stat file vs per-individual stat file that's already been done) and figure out how or what to graph visually via histograms and such, what's most important to look at
#Think through individual information file using to make sure it's all right and such


bash DataProcessing.Pipeline.pt6.GATK.ptb.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash ../../../../DataProcessing.Pipeline.pt6.GATK.ptc.MergeVCFsAcrossChromosomes.vs1.sh

bash DataProcessing.Pipeline.pt6.GATK.ptd.RecalAndApplyVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

vcftools --gzvcf AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz --plink --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP

plink --file AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP

#Decided on a change to how I incorporated individuals with multiple runs across Pools. Originally I was just keeping one pool and removing the other from the total dataset (demarcated using carrots, hence the 'noCarrot' version of files to preserve the original markings but remove them for downstream use), but have now decided to not manually choose which pool to use but to collect and use them all for every individual that has multiple pool runs. /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.Utility.CheckForDupEntriesPOPSampleInfoFile.vs1.py identifies individuals with multiple pools and creates a list of each pool for that individual. DataProcessing.Pipeline.Utility.CombineMultPoolsForSameIndvbamMergeList.vs1.sh then removes the individual lines for each pool from the main Createbam file, collects them into a single line and reattaches the new single-line-per-individual to the removed file (producing a list of the compeltely removed main file, a list of all the new lines, and the to-be-used list of removed main file + new list
#Going to rerun the 'new list' of individuals and then continue doing analysis with the removed maine file + new list, e.g. DataOrganization.CreatebamMergeList.vs1.20131114_193315.output.multPOOLperIndvCollected
#There may be one or two individuals being dropped in the mix that should not be so, but for the time being I will move forward with the analysis. I believe, after spot-checking most parts of the analysis and lines of code, that things are functioning more or less as they should be. When the rest of the individuals are added this will allow a second go-around to investigate if other individuals are being dropped unintentionally and to fully iron-out these portions of the code
#python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.Utility.CheckForDupEntriesPOPSampleInfoFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20131202_POP_genomics_sample_information_Solid_and_MM5.noMeta.noStar.csv

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.Utility.CheckForDupEntriesPOPSampleInfoFile.vs1.py --file1 20130930_POP_genomics_sample_information_2.edited.noMeta.noCarrot.csv | perl -F: -ane 'my @vals1 = split(/,/, $F[0]); my $id1 = $vals1[1]; print $id1, ":"; foreach my $entry1 (@F) { my @valsN = split(/,/, $entry1); my @MID = split(//, $valsN[2]); if ($#MID == "3") { print "0", $valsN[2], ","; } else { print $valsN[2], ","; } foreach my $entry2 (@valsN) { if ($entry2 =~ m/ool/) { print $entry2; } } print ":";} print "\n";' | sed "s/'//g" | sed 's/\s//g' | perl -F: -lane 'print join(":", @F[1..$#F]);' > 20130930_POP_genomics_sample_information_2.edited.noMeta.noCarrot.DupListPrepforMergeListUse.txt
#Manually delete first line of this resulting file since it somewhat not needed and adding in an arbitrary piece of code to just 'delete the first line of the output' seems too hard-coded to just put in there. Would rather do it manually and observe the need for this each time this may have to be done

bash DataProcessing.Pipeline.Utility.CombineMultPoolsForSameIndvbamMergeList.vs1.sh /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20130930_POP_genomics_sample_information_2.edited.noMeta.noCarrot.DupListPrepforMergeListUse.txt DataOrganization.CreatebamMergeList.vs1.20131114_193315.output #Creates DataOrganization.CreatebamMergeList.vs1.20131114_193315.output.multPOOLperIndvCollected* files 

cat 20130930_POP_genomics_sample_information_2.edited.noMeta.noCarrot.csv | grep -v \* > 20130930_POP_genomics_sample_information_2.edited.noMeta.noCarrot.dropIndvs.csv

python DataProcessing.Pipeline.DropIndividualsBasedOnPopGenomFile.vs2.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20130930_POP_genomics_sample_information_2.edited.noMeta.noCarrot.dropIndvs.csv --file2 DataOrganization.CreatebamMergeList.vs1.20131114_193315.output.multPOOLperIndvCollected > DataOrganization.CreatebamMergeList.vs1.20131114_193315.multPOOLperIndvCollected.dropIndvs.output 

bash DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.output.multPOOLperIndvCollected.newEntries

bash DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.output.multPOOLperIndvCollected.newEntries

#bash DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged DataOrganization.CreatebamMergeList.vs1.20131114_193315.multPOOLperIndvCollected.dropIndvs.output

bash DataProcessing.Pipeline.pt6.All.pta.SplitBamIntoChromosomes.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.Samtools.pta.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.GATK.pta.RunReduceReadsPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.GATK.ptb.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash ../../../../DataProcessing.Pipeline.pt6.GATK.ptc.MergeVCFsAcrossChromosomes.vs1.sh

bash DataProcessing.Pipeline.pt6.GATK.ptd.RecalAndApplyVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

gzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.vcf

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz | grep PASS > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.temp1

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz | grep \# > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.header

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.header AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.temp1 | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz

rm AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.temp1

vcftools --gzvcf AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz --geno .9 --recode --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.geno90.vcf

#NOTE (20131211) -- so realized when I merged multiple POOL#s into one file I kept the POOL#s the same in the .bam file's sample ID tag, so the variant calling partitioned the merged .bam file into separate 'individuals' again. So updated DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.vs3.sh to now change the SM:... ID in the merged .bam file to only contain one POOL# to account for the file only supposed to be representing a single individual now. As a result, redoing everything from the pt3 merge step onwards again

bash DataProcessing.Pipeline.Utility.RemoveaCertainFile.vs2.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.multPOOLperIndvCollected.dropIndvs.output 

bash DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.multPOOLperIndvCollected.dropIndvs.output

bash DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.multPOOLperIndvCollected.dropIndvs.output

#bash DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged DataOrganization.CreatebamMergeList.vs1.20131114_193315.multPOOLperIndvCollected.dropIndvs.output

bash DataProcessing.Pipeline.pt6.All.pta.SplitBamIntoChromosomes.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.Samtools.pta.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.GATK.pta.RunReduceReadsPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.GATK.ptb.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash ../../../../DataProcessing.Pipeline.pt6.GATK.ptc.MergeVCFsAcrossChromosomes.vs1.sh

bash DataProcessing.Pipeline.pt6.GATK.ptd.RecalAndApplyVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

gzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.vcf

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz | grep PASS > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.temp1

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz | grep \# > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.header

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.header AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.temp1 | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz

rm AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.temp1

vcftools --gzvcf AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz --geno .9 --recode --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.geno90.vcf

vcftools --gzvcf AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz --geno .85 --recode --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.geno85.vcf

vcftools --gzvcf AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz --geno .95 --recode --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.geno95.vcf

vcftools --gzvcf AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz --missing --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss | awk '{ print $6 }' | perl -lane 'printf "%.2f\n", $F[0];' | sort -g -k 1,1 | uniq -c
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss | awk '{ print $6 }' | perl -lane 'printf "%.2f\n", $F[0];' | sort -g -k 1,1 | uniq -c | perl -lane '$cumulativeSum += $F[0]; push(@F, $cumulativeSum/2292557); print join("\t", @F);'

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/OID41305_HG19_Match5_probes/OID41305_HG19_Match5_probe_coverage.bed | perl -lane 'if ($F[0] =~ m/chr([a-zA-Z0-9]*).*/) { $F[0] = $1; print join("\t", @F); } ' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/OID41305_HG19_Match5_probes/OID41305_HG19_Match5_probe_coverage.formatted.bed

samtools view  ../../mapping_pool_106-124/POOL106_01RL/POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.bam | perl -lane 'my @vals1 = split(//, $F[9]); print $F[2], "\t", $F[3], "\t", $F[3] + scalar(@vals1), "\t", scalar(@vals1);' > ../../mapping_pool_106-124/POOL106_01RL/POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.OverlapPlot.formatted

python DataProcessing.Pipeline.Utility.AddColorInfotoVCFtoolslmissFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.0kbWindow.plotReady

#R --vanilla --slave --args < ../../../../DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs1.R /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/OID41305_HG19_Match5_probes/OID41305_HG19_Match5_probe_coverage.formatted.bed /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_106-124/POOL106_01RL/POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.OverlapPlot.formatted /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs1

R --vanilla --slave --args < ../../../../DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs2.R /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_106-124/POOL106_01RL/POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.OverlapPlot.formatted /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.0kbWindow.plotReady /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs2

python ../../../../DataProcessing.Pipeline.Utility.AddColorInfotoVCFtoolslmissFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss --window 500 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.500bWindow.plotReady

python ../../../../DataProcessing.Pipeline.Utility.AddColorInfotoVCFtoolslmissFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss --window 1000 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.1kbWindow.plotReady

python ../../../../DataProcessing.Pipeline.Utility.AddColorInfotoVCFtoolslmissFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss --window 5000 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.5kbWindow.plotReady

python ../../../../DataProcessing.Pipeline.Utility.AddColorInfotoVCFtoolslmissFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss --window 10000 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.10kbWindow.plotReady

python ../../../../DataProcessing.Pipeline.Utility.AddColorInfotoVCFtoolslmissFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss --window 25000 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.25kbWindow.plotReady

python ../../../../DataProcessing.Pipeline.Utility.AddColorInfotoVCFtoolslmissFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss --window 50000 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.50kbWindow.plotReady

R --vanilla --slave --args < ../../../../DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs2.R /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_106-124/POOL106_01RL/POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.OverlapPlot.formatted /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.500bWindow.plotReady /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs2.500bWindow

R --vanilla --slave --args < ../../../../DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs2.R /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_106-124/POOL106_01RL/POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.OverlapPlot.formatted /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.1kbWindow.plotReady /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs2.1kbWindow

R --vanilla --slave --args < ../../../../DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs3.R /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_106-124/POOL106_01RL/POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.OverlapPlot.formatted /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.1kbWindow.plotReady /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs3.1kbWindow

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz | awk '{ print $1, "\t", $2, "\t", $2, "\t", $4, "\t", $5 }' | grep -v ^# > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat

perl /home/michaelt/Software/annovar/summarize_annovar.pl /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat /home/michaelt/Software/annovar/ --ver1000g 1000g2012apr --verdbsnp 129 --veresp 6500 --buildver hg19 --remove

mv DataProcessing.Pipeline.DropOffTargetReads.vs1.py DataProcessing.Pipeline.Utility.DropOff.TargetReads.vs1.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.DropOff.VCFVariants.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz --window 1000 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.DropOffTargetVariants.1kbWindow.vcf.gz

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.DropOffTargetVariants.1kbWindow.vcf.gz --geno .85 --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.DropOffTargetVariants.1kbWindow.85geno

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.DropOffTargetVariants.1kbWindow.85geno.recode.vcf &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.DropOffTargetVariants.1kbWindow.vcf.gz --geno .95 --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.DropOffTargetVariants.1kbWindow.95geno

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.DropOffTargetVariants.1kbWindow.95geno.recode.vcf &


#In directory /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles
#Using phenotype file descriptions in DataProcessing.Pipeline.CreatePhenotypeFile.vs1.py to help determine which column + values to use
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($3 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.WhiteOnly.justSeroNeg.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($3 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.WhiteOnly.justSeroPos.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($4 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.WhiteOnly.justSeroHighlyExposedNeg.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.WhiteOnly.justRapidVRapid.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.WhiteOnly.justSlowVSlow.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($6 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.WhiteOnly.justVeryRapid.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($6 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.WhiteOnly.justVerySlow.FID

#In directory /home/michaelt/Data/1000G/AncestralAlignments online sources say these reference files based on Gch37 which is the equivalent of hg19 so should be same reference bases between these files and my own 
wget ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/pilot_data/technical/reference/ancestral_alignments/human_ancestor*
wget http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/analysis_results/supporting/ancestral_alignments/human_ancestor_GRCh37_e59.tar.bz2

#Trying to get AA information for each variant called in the dataset. Following code was copied/pasted from http://seqanswers.com/forums/showthread.php?t=15026 "1000 genomes ancestral alleles?"
ls human_ancestor_*.fa.bz2 | while read IN; do
OUT=`echo $IN | sed 's,bz2$,gz,'`
CHR=`echo $IN | sed 's,human_ancestor_,, ; s,.fa.bz2,,'`
bzcat $IN | sed "s,^>.*,>$CHR," | gzip -c > $OUT
samtools faidx $OUT
done
#

#Test line
samtools faidx human_ancestor_1.fa.gz 1:1000000-1000020 #Expect to see tgggcacagcctcacccagga

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz | fill-aa -a /home/michaelt/Data/1000G/AncestralAlignments/human_ancestor_GRCh37_e59/human_ancestor_ | gzip -c > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz
#EDIT NOTE 03162014 -- when redoing this later just drop out SNPs that don't have an AA too at this point (might as well since they're not being used in our analysis, but probably double check how many are being dropped by this, probably not a lot, e.g. under 1,000?)

vcftools --gzvcf AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --plink --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped | awk '{ print $1, "\t", $2 } ' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDIIDs

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDIIDs | awk '{ print $1 } ' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.DropOff.VCFVariants.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --window 1000 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.vcf.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.DropOff.VCFVariants.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --window 0 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.0kbWindow.vcf.gz

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.vcf.gz --geno .95 --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.FID --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg.FID --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos.FID --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg.FID --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid.FID --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow.FID --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid.FID --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow.FID --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow &

#
#20140101 NOTE -- These *.pheno and *.covar files contain FID and IID information for /every/ POOL_* individual, even if the main .vcf/.ped/.whatever file does not contain duplicate entries of the same FID/IID (e.g. multiple POOL_* were combined for a single FID/IID, as should be). So 'total line counts' for some of these *.pheno and *.covar files may be 'off' but in fact this is fine since the main .vcf/.ped/.whatever file is properly delimited. This is just to allow /all/ entries to have information so if the leading POOL_* identifier for a given FID/IID group changes for whatever reason, these information files can still be used since each POOL_* in any particular FID/IID group should be annotated
#

python DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20130930_POP_genomics_sample_information_2.edited.noMeta.noCarrot.dropIndvs.csv --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Sample_status_list_9-16-13_CT.edited.noMeta.csv > DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140101_POP_genomics_sample.status_list_9-16-13.output

python DataProcessing.Pipeline.CreatePhenotypeFile.vs1.py --file1 DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140101_POP_genomics_sample.status_list_9-16-13.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno

python DataProcessing.Pipeline.CreateCovariateFile.vs1.py --file1 DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140101_POP_genomics_sample.status_list_9-16-13.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar | awk '{ if ($5 == "1") { print $1, "\t", $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDIIDs

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDIIDs | awk '{ print $1 } ' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs

#Reran creating PLINK pheno files that contain only certain sero+/- and slow/rapid individuals

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($3 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($4 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($6 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($6 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow.FID

#So order is: justSeroHighlyExposedNeg 
paste AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow.frq | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.AllPhenos.frq.gz

gzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow.frq 

#gunzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.frq.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg.frq.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg.frq.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid.frq.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos.frq.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow.frq.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid.frq.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow.frq.gz

#paste <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz | tail -n +125 | perl -lane 'print join("\t", @F[0..8]);') <(zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.AllPhenos.frq.gz | tail -n +2) | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.wFreqsRplcGenos.noHeader.vcf.gz

###cat <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz | head -n 124) <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.wFreqsRplcGenos.noHeader.vcf.gz) | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.wFreqsRplcGenos.vcf.gz
#cat <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz | awk '{ if (NR < 125) { print $0 } }' ) <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.wFreqsRplcGenos.noHeader.vcf.gz) | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.wFreqsRplcGenos.asddadadad.vcf.gz
#rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.wFreqsRplcGenos.noHeader.vcf.gz
#rm AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.*

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | grep exonic | grep -v ncRNA | head -n 10 | perl -F, -lane 'print join("\t", @F[$#F-6..$#F]);' | perl -lane 'print join("\t", @F[$#F-4..$#F]);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AnnovarTest/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.TestGroup.vs1

perl /home/michaelt/Software/annovar/summarize_annovar.pl /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AnnovarTest/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.TestGroup.vs1 /home/michaelt/Software/annovar/ --ver1000g 1000g2012apr --verdbsnp 129 --veresp 6500 --buildver hg19 --remove

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.TestGroup.vs1 | perl -lane '($F[3], $F[4]) = ($F[4], $F[3]); print join("\t", @F);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.TestGroup.vs2

perl /home/michaelt/Software/annovar/summarize_annovar.pl /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AnnovarTest/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.TestGroup.vs2 /home/michaelt/Software/annovar/ --ver1000g 1000g2012apr --verdbsnp 129 --veresp 6500 --buildver hg19 --remove

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.AllPhenos.frq.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 25; $i <= 31; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.txt.gz

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CalcMeandAF.vs1.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.txt.gz

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/DataProcessing.Pipeline.Utility.CalcMeandAF.vs1.GeneSpecificAnalyses.txt | grep PP2D | grep -v PP2DP | sort -g -k 9,9

Timepoint check: 20140106

bash DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs1.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output

cat DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs1.output | perl -lane 'my @array1; push(@array1, $F[1]); foreach my $entry (@F) { if ($entry !~ m/[a-zA-Z]/) { push (@array1, $entry); } } print join(",", @array1);' | awk 'BEGIN{ print "ID,TotalReadsPreQC,TotalReadsQ10,TotalReadsQ10Prcnt,TotalReadsF4,TotalReadsF4Prcnt,TotalReadsF256,TotalReadsF256Prcnt,TotalReadsPostQC,TotalReadsDropped,TotalReadsDroppedPrcnt,AverageReadLengthPreQC,AverageReadLengthPostQC,BedFileCoveragePostQC" } { print $0 } ' > DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs1.output.reformatted 

bash DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.multPOOLperIndvCollected.dropIndvs.output

#Total column numbers post formatting = 27
cat DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.output | perl -lane 'my @array1; push(@array1, $F[1]); foreach my $entry (@F) { if ($entry !~ m/[a-zA-Z]/) { push (@array1, $entry); } } print join(",", @array1);' | awk 'BEGIN{ print "ID,NumberOfOriginalBamFiles,TotalReadsPreQC,TotalReadsPreQCForward,TotalReadsPreQCReverse,TotalReadsPreQCForward_Reverse,PreQCMAPQmean,PreQCMAPQsd,PreQCMean,PreQCSD,PreQCMax,PreQCMin,CoveragePreQC,TotalReadsPostRmdup,TotalReadsPostRmdupDropped,TotalReadsPostRmdupDroppedPercentage,TotalReadsPostQC,TotalReadsPostQCForward,TotalReadsPostQCReverse,TotalReadsPostQCForward_Reverse,PostQCMAPQmean,PostQCMAPQsd,PostQCMean,PostQCSD,PostQCMax,PostQCMin,CoveragePostQC" } { print $0 } ' > DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.output.reformatted

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs1.output.reformatted /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.output.reformatted

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --plink --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode

plink --file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.phenoColEdit.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.phenoColEdit.pedind

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.bim /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.pedsnp

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.phenoColEdit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.phenoColEdit.pedind /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.pedsnp mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/Eigenstrat/.  

cp -p AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.parfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.parfile

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/smartpca -p AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.parfile

scp mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.Results.pca /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/.

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20140101_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.Results.pca.altrd.race

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.Results.pca.altrd.race mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/Eigenstrat/.

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.Results.pca.altrd.race.plot.1vs2.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.Results.pca.altrd.race.plot.3vs4.xtxt

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.Results.pca.altrd.race | grep White_non-Hispanic | perl lane 'my @vals1 = split(/:/, $F[0]); $F[0] = $vals1[0]; print join("\t", @F);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.Results.pca.altrd.race.FIDs 

#Directory /home/michaelt/Data/HapMap/HapMap3
wget ftp://ftp.ncbi.nlm.nih.gov/hapmap/genotypes/2009-01_phaseIII/plink_format/*
wget ftp://ftp.ncbi.nlm.nih.gov/hapmap/genotypes/2009-01_phaseIII/draft2_samples_QC+.txt

#Direcotry /home/michaelt/Software/liftOver
wget http://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64/liftOver
wget http://hgdownload.cse.ucsc.edu/goldenPath/hg18/liftOver/hg18ToHg19.over.chain.gz

cat hapmap3_r2_b36_fwd.consensus.qc.poly.map | perl -lane '$val1++; print "chr", $F[0], "\t", $F[3], "\t", $F[3]+1, "\t", $val1;' > hapmap3_r2_b36_fwd.consensus.qc.poly.map.liftOverFormat

./../../Software/liftOver/liftOver hapmap3_r2_b36_fwd.consensus.qc.poly.map.liftOverFormat ../../Software/liftOver/hg18ToHg19.over.chain.gz hapmap3_r2_b36_fwd.consensus.qc.poly.map.liftOverFormat.hg19 hapmap3_r2_b36_fwd.consensus.qc.poly.map.liftOverFormat.dropped

paste hapmap3_r2_b36_fwd.consensus.qc.poly.map hapmap3_r2_b36_fwd.consensus.qc.poly.map.liftOverFormat > hapmap3_r2_b36_fwd.consensus.qc.poly.map.liftOverFormat.ForPostProcessing

python /home/michaelt/Data/HapMap/HapMap3/VLookUp.Python.vs1.py --file1 /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.map.liftOverFormat.hg19 --file2 /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.map.liftOverFormat.ForPostProcessing > /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.liftOver.hg18tohg19.map

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/VLookUp.Python.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.map --file2 /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.liftOver.hg18tohg19.map | awk '{ print $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.rsIDs

cat /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.txt | grep -E 'CEU|CHB|YRI' | awk '{ if ($8 =="x") { print $0 } } ' > /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.CEUCHBYRI.Unrelated.txt

cat /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.CEUCHBYRI.Unrelated.txt | awk '{ print $1, "\t", $2 }' > /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.CEUCHBYRI.Unrelated.FIDIIDs

plink --file /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly --make-bed --out /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly

plink --bfile /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.rsIDs --keep /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.CEUCHBYRI.Unrelated.FIDIIDs --recode --out /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData

cat /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.map | awk '{ print $2 }' > /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.map.rsIDs 

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --extract /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.map.rsIDs --recode --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3

plink --file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3 --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3

#There are a few SNPs that do not match up between the two files. Removing them (about ~30?) and recreating the overlap .ped files once more
paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.bim /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.bim | awk '{ if (($5 == $11) || ($5 == $12) ) { print $0 } } ' | awk '{ print $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.FinalListwDrops.rsIDs

plink --bfile /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.FinalListwDrops.rsIDs --keep /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.CEUCHBYRI.Unrelated.FIDIIDs --recode --out /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.vs2

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.FinalListwDrops.rsIDs --recode --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.vs2

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.vs2.ped /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.vs2.ped > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.ped

plink --ped AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.ped --map AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.vs2.map --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.phenoColEdit.ped

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.phenoColEdit.pedind

ln -s AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.bim AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.pedsnp

scp AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.phenoColEdit.ped AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.phenoColEdit.pedind AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.pedsnp mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/Eigenstrat/.

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/smartpca -p AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.parfile

scp mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/.

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20140101_POP_genomics_sample.status_list_9-16-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt1

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.HapMap3vs.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.HapMap3vs.py --file1 /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.txt --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt2

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race

sed -i 's/American_Indian_or_Alaskan_Native/AIndian_or_Alaskan_Native/g' AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/Eigenstrat/.

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI -x -o AllPools.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.1vs2.xtxt

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race | awk '{ if ((($2 >= 0) && ($2 <= .2)) && (($3 >= -.01) && ($3 <= .03))) { print $0 } } ' | grep White_non-Hispanic > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI -x -o AllPools.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.3vs4.xtxt

ln -s AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed AllPools.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i  AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed -c 1:2 -p White_non-Hispanic -x -o  AllPools.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.xtxt

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed | perl -lane 'my @vals1 = split(/:/, $F[0]); print $vals1[0];' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs

scp mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/.

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($3 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($4 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($3 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($6 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($6 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow.FID

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs1.output.reformatted /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.output.reformatted

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Results/Graphics/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs1.*.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/Results/Graphics/.

bash DataProcessing.Pipeline.Utility.GetperfastQpostQCStats.vs1.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output 
#Deleted 4 lines with nothing in them
sed -i 's/(//g' /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperfastQpostQCStats.vs1.output
sed -i 's/)//g' /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperfastQpostQCStats.vs1.output



R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperfastQpostQCStats.vs1.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs1.output.reformatted /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.output.reformatted

python DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.5PCs.covar
#Edit code in between runs
python DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | awk '{ print $1, "\t", $2 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.FIDIIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.FIDIIDs | awk '{ print $1 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.FIDs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 1,2,3,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.5PCs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.5PCs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.FIDIIDs --maf .01 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.5PCs.maf01

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 1,2,3,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.10PCs

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.5PCs.assoc.logistic Pheno1_Logistic_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.5PCs.assoc.logsitic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.10PCs.assoc.logistic Pheno1_Logistic_10PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.10PCs.assoc.logsitic.QQplot.jpeg &

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.frq.gz | perl -lane 'my @vals1 = split(/:/, $F[4]); my @vals2 = split(/:/, $F[5]); push(@F, sprintf("%.0f", $vals1[1]*$F[3])); push(@F, sprintf("%.0f", $vals2[1]*$F[3])); print join("\t", @F); ' | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.frq.wAlleleCounts.gz

#Edited VLookUp.Python.vs1.py
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/VLookUp.Python.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.5PCs.assoc.logistic --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.frq.wAlleleCounts.gz | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.5PCs.assoc.logistic.wAC.gz

R --vanilla --slave --args  < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.ComparePValvsAC.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.5PCs.assoc.logistic.wAC.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.ComparePValvsAC.vs1.plot.jpeg

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.PlotSFS.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/Graphics/DataProcessing.Pipeline.Utility.PlotSFS.vs1.plot.jpeg

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.txt.gz | grep exonic | awk -F, '{ print $19 } ' | sort | uniq > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.txt.GeneIDList

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.txt.gz | grep exonic | awk -F, '{ print $19 } ' | sort | uniq > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.txt.GeneIDList.Exonic

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.txt.gz | grep exonic | grep -v RNA | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.txt.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs2.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.txt.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs2.ExonicVariants.All.output

#Edited code
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs2.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.txt.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs2.ExonicVariants.Nonsynonymous.output

#Edited code
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs2.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.txt.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs2.ExonicVariants.PP2_D.output

#Edited code
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs2.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.txt.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs2.ExonicVariants.PP2_DP.output

R --vanilla --slave --args < DataProcessing.Pipeline.Analysis.VariantDensity.FollowUpCalculations.vs1.R DataProcessing.Pipeline.Analysis.VariantDensity.vs2.ExonicVariants.PP2_D.output | awk '{ if (($4 != "NA") && ($2 >= 4)) { print $0 } } ' | sort -rg -k 3,3
R --vanilla --slave --args < DataProcessing.Pipeline.Analysis.VariantDensity.FollowUpCalculations.vs1.R DataProcessing.Pipeline.Analysis.VariantDensity.vs2.ExonicVariants.PP2_DP.output | awk '{ if (($4 != "NA") && ($2 >= 4)) { print $0 } } ' | sort -rg -k 3,3

#MAPK12 specifically looking at here

B 22,50691870_1,rs138582408,C,T,C,1154,PASS,AC=6;AF=3.995e-03;AN=1502;BaseQRankSum=7.966;DB;DP=13733;Dels=0.00;FS=2.273;HaplotypeScore=3.8969;InbreedingCoeff=-0.0070;MLEAC=5;MLEAF=3.329e-03;MQ=59.99;MQ0=0;MQRankSum=-0.801;QD=12.54;ReadPosRankSum=-1.530;VQSLOD=0.233;culprit=HaplotypeScore;AA=C,0.00451807,0.00319489,0.00310559,0.00569801,0.02,0.00892857,0,0,6,2,1,4,2,1,0,0,exonic,MAPK12,nonsynonymous SNV,,0.41,0.811214,N,0.22,T,0.0,B,0.921332,N,0.539884,D,0.925,22,50691870,50691870,C,T
D 22,50693697_1,rs201016559,T,C,t,221.31,PASS,AC=2;AF=1.332e-03;AN=1502;BaseQRankSum=-4.072;DB;DP=16220;Dels=0.00;FS=0.000;HaplotypeScore=1.5329;InbreedingCoeff=-0.0031;MLEAC=1;MLEAF=6.658e-04;MQ=59.99;MQ0=0;MQRankSum=0.761;QD=8.20;ReadPosRankSum=0.915;VQSLOD=0.809;culprit=FS;AA=t,0.000753012,0,0,0.0014245,0,0,0,0,1,0,0,1,0,0,0,0,exonic,MAPK12,nonsynonymous SNV,,0.03,0.99724,C,0.86,T,0.885,D,0.997985,U,0.841807,D,4.43,22,50693697,50693697,T,C
P 22,50694078_1,rs141322209,G,A,G,97.36,PASS,AC=3;AF=2.003e-03;AN=1498;BaseQRankSum=5.053;DB;DP=14056;Dels=0.00;FS=8.309;HaplotypeScore=2.3858;InbreedingCoeff=-0.0041;MLEAC=1;MLEAF=6.676e-04;MQ=59.99;MQ0=0;MQRankSum=-0.100;QD=3.14;ReadPosRankSum=-1.239;VQSLOD=0.164;culprit=QD;AA=G,0.00226586,0.00159744,0,0.00286533,0.01,0,0,0,3,1,0,2,1,0,0,0,exonic,MAPK12,nonsynonymous SNV,,0.02,0.999418,C,0.92,T,0.17,P,0.997696,U,0.241217,N,4.38,22,50694078,50694078,G,A
D 22,50694084_1,rs2066776,G,A,G,690.67,PASS,AC=3;AF=2.003e-03;AN=1498;BaseQRankSum=6.300;DB;DP=14065;Dels=0.00;FS=0.000;HaplotypeScore=1.9649;InbreedingCoeff=-0.0018;MLEAC=3;MLEAF=2.003e-03;MQ=59.99;MQ0=0;MQRankSum=0.104;QD=14.39;ReadPosRankSum=-0.306;VQSLOD=1.54;culprit=FS;AA=G,0.00226586,0,0,0.00429799,0,0.0181818,0,0.0178571,3,0,0,3,0,2,0,1,exonic,MAPK12,nonsynonymous SNV,rs2066776,0,0.999259,C,1.0,D,0.996,D,0.998205,U,0.999993,D,4.2,22,50694084,50694084,G,A
B 22,50694251_1,rs200047423,T,C,T,456.62,PASS,AC=4;AF=2.670e-03;AN=1498;BaseQRankSum=-7.928;DB;DP=14144;Dels=0.00;FS=2.627;HaplotypeScore=1.6631;InbreedingCoeff=-0.0051;MLEAC=2;MLEAF=1.335e-03;MQ=59.99;MQ0=0;MQRankSum=-0.851;QD=6.92;ReadPosRankSum=-1.241;VQSLOD=0.931;culprit=QD;AA=T,0.00226586,0.00159744,0,0.00286533,0,0,0,0,3,1,0,2,0,0,0,0,exonic,MAPK12,nonsynonymous SNV,,0,0.997358,C,1.0,D,0.087,B,0.998359,U,0.998098,D,4.49,22,50694251,50694251,T,C
D 22,50694280_1,.,G,A,G,480.31,PASS,AC=1;AF=6.676e-04;AN=1498;BaseQRankSum=5.983;DP=14170;Dels=0.00;FS=6.285;HaplotypeScore=1.2801;InbreedingCoeff=-0.0033;MLEAC=1;MLEAF=6.676e-04;MQ=59.99;MQ0=0;MQRankSum=0.507;QD=12.64;ReadPosRankSum=-1.984;VQSLOD=0.763;culprit=HaplotypeScore;AA=G,0.000755287,0,0,0.00143266,0,0,0,0,1,0,0,1,0,0,0,0,exonic,MAPK12,nonsynonymous SNV,,0,0.999458,C,1.0,D,1.0,D,0.997712,U,0.999999,D,4.44,22,50694280,50694280,G,A
D 22,50694529_1,rs2230823,G,A,G,213.68,PASS,AC=3;AF=1.997e-03;AN=1502;BaseQRankSum=5.839;DB;DP=14376;Dels=0.00;FS=9.486;HaplotypeScore=1.0269;InbreedingCoeff=-0.0038;MLEAC=1;MLEAF=6.658e-04;MQ=59.99;MQ0=0;MQRankSum=0.313;POSITIVE_TRAIN_SITE;QD=6.28;ReadPosRankSum=2.196;VQSLOD=1.17;culprit=QD;AA=G,0.00150602,0,0,0.002849,0,0,0,0,2,0,0,2,0,0,0,0,exonic,MAPK12,nonsynonymous SNV,rs2230823,0,0.986757,C,1.0,D,0.9,D,0.6867,U,0.075359,N,3.17,22,50694529,50694529,G,A
B 22,50696678_1,rs34422484,G,A,G,7444.34,PASS,AC=57;AF=0.038;AN=1488;BaseQRankSum=19.808;DB;DP=7340;Dels=0.00;FS=4.423;HaplotypeScore=0.9100;InbreedingCoeff=0.0409;MLEAC=53;MLEAF=0.036;MQ=60.00;MQ0=0;MQRankSum=-0.703;POSITIVE_TRAIN_SITE;QD=12.53;ReadPosRankSum=1.973;VQSLOD=2.50;culprit=DP;AA=G,0.0333839,0.0273312,0.03125,0.0387931,0.0306122,0.0535714,0.0454545,0.0344828,44,17,10,27,3,6,2,2,exonic,MAPK12,nonsynonymous SNV,rs34422484,0.04,0.982381,C,0.94,T,0.0,B,0.945872,N,0.506249,D,2.11,22,50696678,50696678,G,A


zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50691870 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50693697 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'
346-0/1:6,10:16:99:271,0,158    398-0/1:10,1:11:1:1,0,294
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50694078 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'
301-0/1:9,1:10:4:4,0,238        316-0/1:10,1:11:2:2,0,278       379-0/1:5,5:10:99:147,0,116
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50694084 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'
220-0/1:2,12:15:20:373,0,20     234-0/1:12,9:20:99:241,0,282    435-0/1:6,6:12:99:176,0,150
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50694251 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'
428-0/1:5,1:6:4:4,0,156 493-0/1:10,2:12:32:32,0,302     495-0/1:9,1:10:4:4,0,280        571-0/1:18,20:37:99:499,0,506
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50694280 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'
569-0/1:18,20:37:99:530,0,453
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50694529 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'
440-0/1:4,1:5:21:21,0,105       492-0/1:8,1:9:10:10,0,223       554-0/1:11,9:19:99:262,0,261
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50696678 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'
33-0/1:1,2:3:26:64,0,26 54-0/1:10,1:11:2:2,0,281        75-0/1:10,1:11:3:3,0,293        82-0/1:4,3:7:79:79,0,115        94-0/1:10,1:11:2:2,0,301        100-0/1:7,8:15:99:241,0,188     105-0/1:10,21:30:99:665,0,220   117-0/1:8,3:11:74:74,0,241      188-0/1:3,1:4:20:20,0,91        216-0/1:1,1:2:27:29,0,27        220-0/1:3,6:9:66:183,0,66  234-0/1:5,7:12:99:210,0,137     235-0/1:9,8:17:99:227,0,235     260-0/1:1,3:4:23:93,0,23        265-0/1:8,10:18:99:294,0,208    304-1/1:0,1:1:3:35,3,0  313-0/1:1,4:5:17:124,0,17       318-0/1:3,1:4:23:23,0,82        323-0/1:2,4:6:51:117,0,51       349-0/1:4,4:8:99:120,0,101      354-0/1:3,3:6:74:88,0,74   361-0/1:10,5:15:99:127,0,263    388-0/1:3,2:5:58:58,0,79        393-1/1:0,5:5:15:181,15,0       399-1/1:0,11:11:33:390,33,0     414-0/1:6,3:9:80:80,0,148       417-0/1:3,4:7:73:122,0,73       423-0/1:8,5:13:99:132,0,203     430-0/1:5,2:7:49:49,0,147       435-0/1:4,3:7:87:87,0,106       448-0/1:4,6:10:99:183,0,100        451-1/1:0,2:2:6:67,6,0  460-0/1:2,3:5:49:87,0,49        514-0/1:1,2:3:24:61,0,24        515-0/1:4,1:5:19:19,0,113       544-0/1:5,5:10:99:139,0,127     554-0/1:3,5:8:70:153,0,70       568-0/1:5,23:27:50:713,0,50     570-0/1:9,8:17:99:223,0,255     589-0/1:7,5:12:99:131,0,189     590-0/1:7,4:11:99:106,0,190        592-0/1:14,13:26:99:358,0,345   594-0/1:21,20:40:99:570,0,518   599-0/1:3,6:9:64:174,0,64       616-0/1:7,6:13:99:171,0,168     622-0/1:8,3:11:64:64,0,215      631-0/1:7,5:12:99:141,0,175     632-0/1:15,6:20:99:113,0,410    633-0/1:10,10:20:99:286,0,247   662-0/1:3,2:5:47:47,0,69        687-0/1:1,3:4:20:92,0,20   691-0/1:6,2:8:48:48,0,175       738-0/1:22,15:36:99:422,0,551


R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs3.R /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_106-124/POOL106_01RL/POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.OverlapPlot.formatted /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.0kbWindow.plotReady /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs3.0kbWindow

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.FIDIIDs --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.pheno
#Expecting 619 individuals based on plink .fam output and FIDs file -- get 619 in '...justWhite.pheno' output file

cp -p VLookUp.Python.vs1.py VLookUp.Python.vs1.vs2.py 

#Later did mv VLookUp.Python.vs1.vs2.py DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs1.py 

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { print $8, "\t", $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.SetIDFile.Exonic

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $8, "\t", $2 } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($11 == "P") || ($11 == "D") ) { print $8, "\t", $2 } } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_PD

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $8, "\t", $2 } } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_D

#Hardcoding SKAT analysis script because the tools require 'file location' as input so it seems like I'll need multiple file sets to run the code on multiple SetID sets for example

R --vanilla --slave --args < DataProcessing.Pipeline.Analysis.Skat.vs1.R

#EPACTS
#Using same 'file creation setup' instead of doing these commands within the Python script to help maintain consistency with how I'm partitioning variants into exonic, nonsynonymous, Polyphen2 categories, etc...these partitions are based on the commands I use in my CalcMeandAF.vs1.R scripts 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $0 } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($11 == "P") || ($11 == "D") ) { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "23") { $1 = "X"} print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip

/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip

/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.pheno | awk '{ print $1, "\t", $2, "\t0\t0\t0\t", $3, "\t", $4, "\t", $5, "\t", $6 } ' | sed 's/-9/NA/g' | perl -lane 'print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped

#Add line #FAM_ID IND_ID FAT_ID MOT_ID SEX HIVPROG HIVPROGHE AIDS AIDSEXTR to /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.pheno

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test b.collapse \
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.EPACTS.collapse

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test b.collapse \
--max-maf .9 \
-min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD.EPACTS.collapse

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test b.madsen \
--max-maf .9 \
-min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.EPACTS.madsen

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test b.madsen \
--max-maf .9 \
-min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.EPACTS.madsen

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test b.madsen \
--max-maf .9 \
-min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD.EPACTS.madsen

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test b.madsen \
--max-maf .9 \
-min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D.EPACTS.madsen

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test b.wcnt \
--max-maf .9 \
--min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.EPACTS.wcnt

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test b.wcnt \
--max-maf .9 \
-min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD.EPACTS.wcnt


epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test skat --skat-o --skat-adjust \
--max-maf .9 \
--min-callrate .01 \ 
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.EPACTS.Skat0

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test skat --skat-o --skat-adjust \
--max-maf .9 \
-min-maf .000000000000001 \ 
--min-callrate .01 \ 
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.EPACTS.Skat0

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test skat --skat-o --skat-adjust \
--max-maf .9 \
-min-maf .000000000000001 \ 
--min-callrate .01 \ 
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test skat --skat-o --skat-adjust \
--max-maf .9 \
-min-maf .000000000000001 \ 
--min-callrate .01 \ 
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROGHE \
--test b.madsen \
--max-maf .9 \
-min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Pheno2.Exonic.Nonsynonymous.PP2_PD.EPACTS.madsen &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROGHE \
--test b.madsen \
--max-maf .9 \
-min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Pheno2.Exonic.Nonsynonymous.PP2_D.EPACTS.madsen &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno AIDS \
--test b.madsen \
--max-maf .9 \
-min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Pheno3.Exonic.Nonsynonymous.PP2_PD.EPACTS.madsen &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno AIDS \
--test b.madsen \
--max-maf .9 \
-min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Pheno3.Exonic.Nonsynonymous.PP2_D.EPACTS.madsen &


epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROGHE \
--test skat --skat-o --skat-adjust \
--max-maf .9 \
-min-maf .000000000000001 \ 
--min-callrate .01 \ 
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Pheno2.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROGHE \
--test skat --skat-o --skat-adjust \
--max-maf .9 \
-min-maf .000000000000001 \ 
--min-callrate .01 \ 
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Pheno2.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno AIDS \
--test skat --skat-o --skat-adjust \
--max-maf .9 \
-min-maf .000000000000001 \ 
--min-callrate .01 \ 
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Pheno3.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno AIDS \
--test skat --skat-o --skat-adjust \
--max-maf .9 \
-min-maf .000000000000001 \ 
--min-callrate .01 \ 
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Pheno3.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0 &

#KIAA1429
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "8") { print $0 } } ' | grep 95501065 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.txt.gz | grep KIAA1429 | awk -F, '{ if (($28 == "D") || ($28 == "P")) { print $2, "\t", $28 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SpecificGenes/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.Exonic.KIAA1429.PP2_PD.Locations

bash DataProcessing.Pipeline.Analysis.IdentifyIndividualsContainingVariants.vs1.sh 8 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SpecificGenes/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.Exonic.KIAA1429.PP2_PD.Locations

95501065(D): 31-0/1:9,1:10:6:6,0,264    350-0/1:6,12:18:99:365,0,129    411-0/1:5,4:9:99:108,0,122      649-0/1:7,1:8:11:11,0,197
95508103(D): 102-0/1:11,7:18:99:175,0,337
95518785(D): 362-0/1:14,18:31:99:469,0,376
95522750(D): 715-0/1:11,12:22:99:296,0,326
95522824(D): 163-0/1:5,1:6:16:16,0,152  686-0/1:13,9:21:99:248,0,354
95530099(D): 107-0/1:5,1:6:17:17,0,160  192-0/1:6,1:7:3:3,0,178 243-0/1:13,13:26:99:377,0,379   
95531388(P): 721-0/1:3,4:7:69:110,0,69  
95531525(P): 473-0/1:10,8:18:99:221,0,273       705-0/1:9,1:10:4:4,0,278
95538947(P): 96-0/1:21,26:45:99:638,0,623       759-0/1:7,1:8:7:7,0,218 
95541400(P): 625-0/1:9,5:14:99:125,0,259        719-0/1:8,1:9:5:5,0,248 
95549374(P): 153-0/1:8,1:9:8:8,0,249    296-0/1:5,5:10:99:103,0,141     297-0/1:7,1:8:2:2,0,214 421-0/1:9,1:10:4:4,0,285
95550513(P): 92-0/1:8,5:13:99:131,0,242 296-0/1:4,1:5:9:9,0,95  683-0/1:6,1:7:13:13,0,175


vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.FID --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg.FID --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos.FID --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg.FID --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid.FID --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow.FID --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid.FID --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow.FID --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow &

paste AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow.frq.count | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.AllPhenos.frq.count.gz

gzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow.frq.count 

#gunzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.frq.count.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg.frq.count.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg.frq.count.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid.frq.count.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos.frq.count.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow.frq.count.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid.frq.count.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow.frq.count.gz

paste AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq.count | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.AllPhenos.frq.AndCount.gz

gzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq.count

#gunzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq.count

#20140219
#(for AWK) 26 exonic/intronic, 27 gene, 28 synon/non-synon, 29 rsID from annovari/'dbsnp129' (vs from GATK variant caller at beginning of array), 30-40 prediction stuff (30 is AVSIFT, then predic prog # / predic prog Category for PhyloP, SIFT, Polyphen2, LRT, MT)
#Now including allele counts as well as allele frequencies
#Also now outputting multiple lines for the same variant if it covers multiple genes. This is being recorded by adding a '_1', '_2', etc... to te end of the basepair position at array entry 2 for each line to designate the # of times the variant is present in the dataset to represent different loci for that same variant

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.AllPhenos.frq.AndCount.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 29; $i <= 39; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.vs2.txt.gz

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | grep exonic | awk -F, '{ print $3 }' | sort | uniq > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.AnnotatingVariantsCoveringMultTranscripts
#The manually edited /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.AnnotatingVariantsCoveringMultTranscripts comparing the output to /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv to see variants that have multiple gene-hits (e.g. overlap multiple genes).
#Decided to output multiple lines for variants that cover multiple transcripts/genes, e.g. have 3 lines for the same variant if that variant hits 3 genes at once
#So stopped this train of thought here and didn't finish annotating the above filename


#Used vs1 of DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20140101_POP_genomics_sample.status_list_9-16-13.output instead of vs2 earlier. Redoing these and downstream steps with the proper vs2 of this file
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140101_POP_genomics_sample.status_list_9-16-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt1

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race

sed -i 's/American_Indian_or_Alaskan_Native/AIndian_or_Alaskan_Native/g' AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/Eigenstrat/.

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI -x -o AllPools.AllPoolsMerged.ChrAll.GATK.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.1vs2.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI -x -o AllPools.AllPoolsMerged.ChrAll.GATK.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.3vs4.xtxt

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race | awk '{ if ((($2 >= 0) && ($2 <= .2)) && (($3 >= -.01) && ($3 <= .03))) { print $0 } } ' | grep White_non-Hispanic > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed

ln -s AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed AllPools.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i  AllPools.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed -c 1:2 -p White_non-Hispanic -x -o  AllPools.AllPoolsMerged.ChrAll.GATK.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.xtxt

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed | perl -lane 'my @vals1 = split(/:/, $F[0]); print $vals1[0];' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs

scp mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/.

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar | awk '{ print $1, "\t", $2 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar.FIDIIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar.FIDIIDs | awk '{ print $1 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.QCed.FIDs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 1,2,3,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.5PCs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.5PCs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar.FIDIIDs --maf .01 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.5PCs.maf01

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 1,2,3,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.10PCs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 2 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar.FIDIIDs --maf .01 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.5PCs.maf01.pheno2

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 3 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar.FIDIIDs --maf .01 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.5PCs.maf01.pheno3

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 4 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar.FIDIIDs --maf .01 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.5PCs.maf01.pheno4

!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.5PCs.assoc.logistic Pheno1_WhiteOnly_Logistic_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.5PCs.assoc.logsitic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.5PCs.maf01.assoc.logistic Pheno1_WhiteOnly_Logistic_maf01_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.5PCs.maf01.assoc.logsitic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.10PCs.assoc.logistic Pheno1_WhiteOnly_Logistic_10PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.10PCs.assoc.logsitic.QQplot.jpeg &

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ print $1 }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.FIDs
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($3 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.FIDs
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($4 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.FIDs
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($3 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.FIDs
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.FIDs
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.FIDs
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($6 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.FIDs
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($6 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.FIDs

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow &

paste AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq.count | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.AllPhenos.frq.AndCount.gz

gzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq.count

#gunzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq.count

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow &

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.frq.count 

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.AllPhenos.frq.AndCount.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 29; $i <= 39; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.vs2.txt.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.vs2.txt.gz | awk -F, '{ if ($26 =="exonic") { print $0 } }' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.vs2.txt.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs3.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.vs2.txt.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs3.ExonicVariants.All.output

#Edited code
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs3.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.vs2.txt.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs3.ExonicVariants.Nonsynonymous.output

#Edited code
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs3.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.vs2.txt.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs3.ExonicVariants.PP2_DP.output

#Edited code
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs3.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.vs2.txt.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs3.ExonicVariants.PP2_D.output

R --vanilla --slave --args < DataProcessing.Pipeline.Analysis.VariantDensity.FollowUpCalculations.vs1.R DataProcessing.Pipeline.Analysis.VariantDensity.vs3.ExonicVariants.PP2_DP.output | awk '{ if (($4 != "NA") && ($2 >= 3)) { print $0 } } ' | sort -rg -k 3,3
R --vanilla --slave --args < DataProcessing.Pipeline.Analysis.VariantDensity.FollowUpCalculations.vs1.R DataProcessing.Pipeline.Analysis.VariantDensity.vs3.ExonicVariants.PP2_D.output | awk '{ if (($4 != "NA") && ($2 >= 3)) { print $0 } } ' | sort -rg -k 3,3

#MAPK12 specifically looking at here
zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50691870 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50693697 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'
346-0/1:6,10:16:99:271,0,158    398-0/1:10,1:11:1:1,0,294

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar.FIDIIDs --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.vs2.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { print $8, "\t", $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.SetIDFile.Exonic

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $8, "\t", $2 } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($11 == "P") || ($11 == "D") ) { print $8, "\t", $2 } } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_PD

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $8, "\t", $2 } } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $0 } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($11 == "P") || ($11 == "D") ) { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.QCed.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.pheno

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.pheno | awk '{ print $1, "\t", $2, "\t0\t0\t0\t", $3, "\t", $4, "\t", $5, "\t", $6 } ' | sed 's/-9/NA/g' | perl -lane 'print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar | awk '{ print $4, "\t", $6, "\t", $7 "\t", $8, "\t", $9, "\t", $10, "\t", $11,  "\t", $12, "\t", $13, "\t", $14,  "\t", $15 }' | paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped.vs2

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped.vs2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped

#Add line #FAM_ID IND_ID FAT_ID MOT_ID SEX HIVPROG HIVPROGHE AIDS AIDSEXTR RMID PC1 PC2 PC3 PC4 PC5 PC6 PC7 PC8 PC9 PC10 to /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \

!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!EPACTS Stuff

#EPACTS Variables Running Through
#Pheno: HIVPROG HIVPROGHE AIDS AIDSEXTR
#Filetypes: Exonic Exonic.Nonsynonymous Exonic.Nonsynonymous.PP2_PD Exonic.Nonsynonymous.PP2_D
#Test b.collapse b.madsen b.wcnt  skat --skat-o --skat-adjust

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --test skat --skat-o --skat-adjust --run 1 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.Exonic.EPACTS.Skat0 &

#epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --test skat --skat-o --skat-adjust --max-maf .9 --min-maf .000000000000001 --min-callrate .01 --run 1 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.Exonic.EPACTS.Skat0.vs2 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --test skat --skat-o --skat-adjust --min-callrate .01 --run 1 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.Exonic.EPACTS.Skat0.vs3 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --test skat --skat-o --skat-adjust --max-maf .9 --run 1 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.Exonic.EPACTS.Skat0.vs4 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --test skat --skat-o --skat-adjust --min-maf .000000000000001 --run 1 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.Exonic.EPACTS.Skat0.vs5 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --test skat --skat-o --skat-adjust --max-maf .5 --run 1 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.Exonic.EPACTS.Skat0.vs6 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --test skat --skat-o --skat-adjust --max-maf 1 --run 1 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.Exonic.EPACTS.Skat0.vs7 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --test skat --skat-o --skat-adjust --max-maf .05 --run 1 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.Exonic.EPACTS.Skat0.vs8 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --test b.collapse --run 1 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.Exonic.EPACTS.collapse &

#epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --test b.collapse --max-maf .9 --min-maf .000000000000001 --min-callrate .01 --run 1 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.Exonic.EPACTS.collapse.vs2 &


#Note -- not really running the code, opening up an R console and copying/pasting the commands since I'm moreso playing around with things and testing them out
R --vanilla --slave --args < DataProcessing.Pipeline.Analysis.Skat.vs3.R
34: 1 SNPs with either high missing rates or no-variation are excluded!
35: The missing genotype rate is 0.004442. Imputation is applied.
36: 5 SNPs with either high missing rates or no-variation are excluded!
37: The missing genotype rate is 0.006094. Imputation is applied.
38: 5 SNPs with either high missing rates or no-variation are excluded!
39: The missing genotype rate is 0.006094. Imputation is applied.
40: 1 SNPs with either high missing rates or no-variation are excluded!
41: The missing genotype rate is 0.002703. Imputation is applied.
42: 3 SNPs with either high missing rates or no-variation are excluded!
43: The missing genotype rate is 0.003392. Imputation is applied.
44: The missing genotype rate is 0.003941. Imputation is applied.
45: The missing genotype rate is 0.003003. Imputation is applied.

30: The missing genotype rate is 0.002336. Imputation is applied.
31: 4 SNPs with either high missing rates or no-variation are excluded!
32: The missing genotype rate is 0.002336. Imputation is applied.
33: The missing genotype rate is 0.001802. Imputation is applied.
34: The missing genotype rate is 0.002252. Imputation is applied.
35: The missing genotype rate is 0.000751. Imputation is applied.
36: 1 SNPs with either high missing rates or no-variation are excluded!
37: The missing genotype rate is 0.005005. Imputation is applied.
38: The missing genotype rate is 0.004880. Imputation is applied.
39: 1 SNPs with either high missing rates or no-variation are excluded!
40: The missing genotype rate is 0.019520. Imputation is applied.
41: The missing genotype rate is 0.004692. Imputation is applied.
42: The missing genotype rate is 0.005255. Imputation is applied.
43: 1 SNPs with either high missing rates or no-variation are excluded!
44: The missing genotype rate is 0.003003. Imputation is applied.
45: 1 SNPs with either high missing rates or no-variation are excluded!
46: The missing genotype rate is 0.001802. Imputation is applied.
47: The missing genotype rate is 0.003003. Imputation is applied.
48: ALL SNPs have either high missing rates or no-variation. P-value=1
49: The missing genotype rate is 0.003003. Imputation is applied.
50: Only one SNP in the SNP set!

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.EPACTS.Skat0 &
make -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.EPACTS.Skat0.Makefile -j 6
Note: Get odd error when trying to run skat-o w/ adjustment on the above combination of parameters, e.g. pheno4 + exonic set. Does not occur elsewhere. Can run skat-o fine manually use R code and SKAT directly, so doing that for this set of results. However, none of the results for Pheno4 are coming out well due to the low sample numbers so it's a moot point.  
Error in out[i, ] <- c(nona.collapse, maf.collapse, nrow(G), nrow(genos),  :
  number of items to replace is not a multiple of replacement length
  Calls: source -> withVisible -> eval -> eval
  In addition: There were 14 warnings (use warnings() to see them)
  Execution halted
  make: *** [/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.EPACTS.Skat0.0.epacts] Error 1
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0 &


epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Collapse &

EPACTS Group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Collapse &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Collapse &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Collapse &


epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Madsen &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Madsen &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Madsen &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Madsen &


epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.wcnt &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.wcnt &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.wcnt &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.wcnt &




python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs4.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.vs2.txt.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs4.ExonicVariants.All.output

#Edited code
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs4.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.vs2.txt.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs4.ExonicVariants.Nonsynonymous.output

#Edited code
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs4.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.vs2.txt.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs4.ExonicVariants.PP2_DP.output

#Edited code
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs4.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.vs2.txt.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs4.ExonicVariants.PP2_D.output

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.FollowUpCalculations.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs4.ExonicVariants.PP2_DP.output | awk '{ if (($5 != "NA") && ($2 >= 3)) { print $0 } } ' | sort -rg -k 4,4
R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.FollowUpCalculations.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs4.ExonicVariants.PP2_D.output | awk '{ if (($4 != "NA") && ($2 >= 3)) { print $0 } } ' | sort -rg -k 4,4

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.AlleleCountComparison.Simulations.Launcher.vs1.sh 1 100

PasteCommandjustSeroNeg=`perl -e 'print "paste - "; for (my $i = 1; $i <= 100; $i++) { print "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVs", $i, ".CtDwn.frq.count "; }'`
PasteCommandjustSeroPos=`perl -e 'print "paste - "; for (my $i = 1; $i <= 100; $i++) { print "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVs", $i, ".CtDwn.frq.count "; }'`
PasteCommandjustRapidVRapid=`perl -e 'print "paste - "; for (my $i = 1; $i <= 100; $i++) { print "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVs", $i, ".CtDwn.frq.count "; }'`
PasteCommandjustSlowVSlow=`perl -e 'print "paste - "; for (my $i = 1; $i <= 100; $i++) { print "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVs", $i, ".CtDwn.frq.count "; }'`

#zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.frq.count.gz | tail -n +2 | paste - /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVs*.CtDwn.frq.count | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVsAll.frq.count.gz
#zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.frq.count.gz | tail -n +2 | paste - /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVs*.CtDwn.frq.count | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVsAll.frq.count.gz
#zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.frq.count.gz | tail -n +2 | paste - /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVs*.CtDwn.frq.count | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.gz
#zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.frq.count.gz | tail -n +2 | paste - /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVs*.CtDwn.frq.count | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVsAll.frq.count.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.frq.count.gz | tail -n +2 | $PasteCommandjustSeroNeg | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVsAll.frq.count.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.frq.count.gz | tail -n +2 | $PasteCommandjustSeroPos | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVsAll.frq.count.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.frq.count.gz | tail -n +2 | $PasteCommandjustRapidVRapid | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.frq.count.gz | tail -n +2 | $PasteCommandjustSlowVSlow | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVsAll.frq.count.gz

#Tri-Allelic and Quad-Allelic SNPs. They are eventually dropped out when including annotation information in the original, main source files since the program doesn't accept non-bi-allelic SNP setups. Could potentially divide these SNPs into multiple entries for the same variant but only including each bi-allelic state amongst the original tri- or quad-alleles, but this is currently decided as being too much work for a supposedly very low return on information gained (387 SNPs << ~60k which is around the full On-Target, 95% geno list).
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK]$zcat SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.gz | perl -lane 'if ($#F == 206) { print join("\t", @F); } ' | wc
    387   80109  377798
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK]$zcat SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.gz | perl -lane 'if ($#F == 207) { print join("\t", @F); } ' | wc
      3     624    2798

[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK]$zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ print $4, "\t", $5 }' | sort | uniq -c | grep -E 'A|G|T|C' | sort -rg -k 1,1
    15295 C        T
    15107 G        A
    9604 T        C
    8993 A        G
    3308 G        C
    3105 C        G
    2628 G        T
    2572 C        A
    2164 T        G
    2069 A        C
    1635 A        T
    1577 T        A
    60 C        G,T
    59 G        A,C
    58 G        A,T
    56 C        A,T
    37 T        C,G
    26 T        A,C
    22 G        C,T
    20 A        G,T
    20 A        C,G
    12 C        A,G
    10 A        C,T
    7 T        A,G
    3 G        A,C,T
    2 phone_home=STANDARD      gatk_key=null
    1 each     ALT
    1 REF      ALT
    1 Containing       Spanning

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.ForSimulation.AlleleCountAnalysis.vs1.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.ForSimulation.AlleleCountAnalysis.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVsAll.frq.count.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 13; $i <= 24; $i++) { if (($F[$i] eq "NA") || ($F[$i] eq "")) { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.ForSimulation.AlleleCountAnalysis.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVsAll.frq.count.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 13; $i <= 24; $i++) { if (($F[$i] eq "NA") || ($F[$i] eq "")) { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.ForSimulation.AlleleCountAnalysis.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 13; $i <= 24; $i++) { if (($F[$i] eq "NA") || ($F[$i] eq "")) { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.ForSimulation.AlleleCountAnalysis.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVsAll.frq.count.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 13; $i <= 24; $i++) { if (($F[$i] eq "NA") || ($F[$i] eq "")) { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.AlleleCountComparison.AnalyzeResults.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Analysis/AlleleCounts/DataProcessing.Pipeline.Analysis.AlleleCountComparison.AnalyzeResults.vs1.output


bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.AlleleCountComparison.Simulations.Launcher.vs1.sh 101 1000
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.AlleleCountComparison.Simulations.Launcher.vs1.sh 850 1000

PasteCommandjustSeroNeg=`perl -e 'print "paste - "; for (my $i = 1; $i <= 1000; $i++) { print "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVs", $i, ".CtDwn.frq.count "; }'`
PasteCommandjustSeroPos=`perl -e 'print "paste - "; for (my $i = 1; $i <= 1000; $i++) { print "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVs", $i, ".CtDwn.frq.count "; }'`
PasteCommandjustRapidVRapid=`perl -e 'print "paste - "; for (my $i = 1; $i <= 1000; $i++) { print "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVs", $i, ".CtDwn.frq.count "; }'`
PasteCommandjustSlowVSlow=`perl -e 'print "paste - "; for (my $i = 1; $i <= 1000; $i++) { print "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVs", $i, ".CtDwn.frq.count "; }'`

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.frq.count.gz | tail -n +2 | $PasteCommandjustSeroNeg | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVsAll.frq.count.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.frq.count.gz | tail -n +2 | $PasteCommandjustSeroPos | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVsAll.frq.count.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.frq.count.gz | tail -n +2 | $PasteCommandjustRapidVRapid | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.frq.count.gz | tail -n +2 | $PasteCommandjustSlowVSlow | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVsAll.frq.count.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.ForSimulation.AlleleCountAnalysis.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVsAll.frq.count.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 13; $i <= 24; $i++) { if (($F[$i] eq "NA") || ($F[$i] eq "")) { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.ForSimulation.AlleleCountAnalysis.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVsAll.frq.count.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 13; $i <= 24; $i++) { if (($F[$i] eq "NA") || ($F[$i] eq "")) { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.ForSimulation.AlleleCountAnalysis.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 13; $i <= 24; $i++) { if (($F[$i] eq "NA") || ($F[$i] eq "")) { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.ForSimulation.AlleleCountAnalysis.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVsAll.frq.count.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 13; $i <= 24; $i++) { if (($F[$i] eq "NA") || ($F[$i] eq "")) { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.AlleleCountComparison.AnalyzeResults.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Analysis/AlleleCounts/DataProcessing.Pipeline.Analysis.AlleleCountComparison.AnalyzeResults.vs2.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Analysis/AlleleCounts/DataProcessing.Pipeline.Analysis.AlleleCountComparison.AnalyzeResults.vs2.output | awk '{ if (($13 != "NA") && ($13 != 1)) { print $0 } } ' | sort -rg -k 14,14 -k 13,13

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($10 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Collapse &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Madsen &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.wcnt &
#Two warning messages for above run of b.wcnt (Pheno3, LRT_D)
#Warning messages:
#1: glm.fit: fitted probabilities numerically 0 or 1 occurred
#2: glm.fit: fitted probabilities numerically 0 or 1 occurred
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.wcnt &


#cat <( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
cat  <( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.All.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.All.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.All.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.All.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/All.GATK.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.justWhite.QCed.Pheno1.RMID5PCs.All.EPACTS.All.Summary.txt

cat <( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.All.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.All.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.All.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.All.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/All.GATK.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.justWhite.QCed.Pheno2.RMID5PCs.All.EPACTS.All.Summary.txt

cat <( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.All.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.All.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.All.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.All.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/All.GATK.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.justWhite.QCed.Pheno3.RMID5PCs.All.EPACTS.All.Summary.txt

cat <( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.All.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.All.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.All.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.All.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/All.GATK.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.justWhite.QCed.Pheno4.RMID5PCs.All.EPACTS.All.Summary.txt

#Accidentally ran in /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/. and converted some files that were not supposed to be. Think I fixed it
ls -lrt | awk '{ print $9 }' | perl -lane 'if ($F[0] =~ m/(.*)(\.wAA)(\.White.*)/) { my $val1 = $1 . $2 . $3; my $val2 = $1 . $2 . ".DropOffTargetVariants.1kbWindow.95geno.recode" . $3; system("mv $val1 $val2");}'
#Fix ls -lrt | grep WhiteOnly | grep -v WhiteOnly.QCed | awk '{ print $9 } ' | perl -lane 'if ($F[0] =~ m/(.*)(\.wAA)(.DropOffTargetVariants.1kbWindow.95geno.recode)(\.White.*)/) { my $val1 = $1 . $2 . $3 . $4; my $val2 = $1 . $2 . $4; system("mv $val1 $val2");}'

cp -p DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.py DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.FullVCFwAnnovar.vs1.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.AllGenesPerVariant.vcf.gz

#Edited code in /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.FullVCFwAnnovar.vs1.py
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.vcf.gz

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py DataPrcoessing.Pipeline.Utility.GetVCFQCStats.vs1.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.AllGenesPerVariant.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.AllGenesPerVariant.GetVCFQCStats.vs1.output

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.vcf.gz | grep exonic | grep -v ncRNA | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.Exonic.GetVCFQCStats.vs1.output

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py DataPrcoessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs1.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justSeroNeg 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justSeroPos 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justSeroHighlyExposedNeg 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justRapidVRapid 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justSlowVSlow 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
[1] "MeanSingletons: 20.6626826029216 MeanDoubletons: 25.2881806108898 MeanCommon: 5033.1567065073 MeanRare: 504.302788844621 MeanHeterozy: 3457.57503320053 MeanCoverage: 16.6629860920849 sdCoverage: 6.17039709917999 Exonic: 342.65604249668 Intronic: 4139.33997343958 Nonsynonymous: 127.176626826029 Synonymous: 212.610889774236 PP2_D: 9.80610889774236 PP2_P: 11.6693227091633 Sift_D: 22.0823373173971 LRT_D: 21.9189907038513"


cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justSeroNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
[1] "MeanSingletons: 16.1948881789137 MeanDoubletons: 21.1086261980831 MeanCommon: 5038.1661341853 MeanRare: 435.638977635783 MeanHeterozy: 3385.56869009585 MeanCoverage: 15.8726865928358 sdCoverage: 5.50509121070413 Exonic: 340.05750798722 Intronic: 4090.77635782748 Nonsynonymous: 126.159744408946 Synonymous: 211.268370607029 PP2_D: 9.77635782747604 PP2_P: 11.3897763578275 Sift_D: 22.0095846645367 LRT_D: 21.6932907348243"

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justSeroHighlyExposedNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
[1] "MeanSingletons: 16.3291925465839 MeanDoubletons: 21.1987577639752 MeanCommon: 5043.50310559006 MeanRare: 436.596273291925 MeanHeterozy: 3384.70186335404 MeanCoverage: 15.7976001524024 sdCoverage: 4.99886275938732 Exonic: 339.596273291925 Intronic: 4096.49689440994 Nonsynonymous: 125.944099378882 Synonymous: 210.795031055901 PP2_D: 9.64596273291925 PP2_P: 11.3416149068323 Sift_D: 21.9316770186335 LRT_D: 21.5465838509317"

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
[1] "MeanSingletons: 16.9517045454545 MeanDoubletons: 19.8977272727273 MeanCommon: 5046.28125 MeanRare: 426.855113636364 MeanHeterozy: 3405.65340909091 MeanCoverage: 17.0944802710762 sdCoverage: 6.42723639615973 Exonic: 337.542613636364 Intronic: 4091.76420454545 Nonsynonymous: 125.235795454545 Synonymous: 209.323863636364 PP2_D: 9.69886363636364 PP2_P: 11.4403409090909 Sift_D: 21.3238636363636 LRT_D: 21.9403409090909"

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justRapidVRapid | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
[1] "MeanSingletons: 16.6734693877551 MeanDoubletons: 17.3673469387755 MeanCommon: 5088.89795918367 MeanRare: 403.591836734694 MeanHeterozy: 3419.16326530612 MeanCoverage: 22.2127566842631 sdCoverage: 9.1245435877065 Exonic: 338.673469387755 Intronic: 4112.5306122449 Nonsynonymous: 125.061224489796 Synonymous: 210.69387755102 PP2_D: 9.12244897959184 PP2_P: 11.469387755102 Sift_D: 21.0408163265306 LRT_D: 21.3469387755102"

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justSlowVSlow | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
[1] "MeanSingletons: 17.3571428571429 MeanDoubletons: 20.3571428571429 MeanCommon: 5025.71428571429 MeanRare: 426.107142857143 MeanHeterozy: 3386.69642857143 MeanCoverage: 16.4965406672807 sdCoverage: 5.79915172393244 Exonic: 333.803571428571 Intronic: 4087.39285714286 Nonsynonymous: 123.196428571429 Synonymous: 207.392857142857 PP2_D: 9.80357142857143 PP2_P: 11.375 Sift_D: 21.3571428571429 LRT_D: 21.8035714285714"

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perGene.vs1.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perGene.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.AllGenesPerVariant.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perGene.vs1.output

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs3.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.AllPhenos.frq.AndCount.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 29; $i <= 39; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.vs3.txt.gz

#Including SNPs that don't have clear ancestral/derived alleles. That was the major change to the vs2 to vs3 of the CombineVCFwFreqsAnnovarInfo.vs#.py script
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.vs3.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { print $8, "\t", $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.SetIDFile.Exonic

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $8, "\t", $2 } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($11 == "P") || ($11 == "D") ) { print $8, "\t", $2 } } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_PD

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $8, "\t", $2 } } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $0 } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($11 == "P") || ($11 == "D") ) { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($10 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $0 } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($11 == "P") || ($11 == "D") ) { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($10 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.Sift_D
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.LRT_D

#TimeptChck: 20140319

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.Exonic.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.Exonic.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.Exonic.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID.Exonic.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.Exonic.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0 &


cat DataProcessing.Pipeline.Utility.GetFastqQCStats.vs1.output > DataProcessing.Pipeline.Utility.GetFastqQCStats.vs1.output.forR
#Removed empty lines with /0.00000.0000 0.0000 and dd in vi
cat DataProcessing.Pipeline.Utility.GetFastqQCStats.vs1.output.forR | R -q -e "Data1 <- read.table(file('stdin'), header=F); sum(as.numeric(Data1[,4])); sum(as.numeric(Data1[,6]));"
> Data1 <- read.table(file('stdin'), header=F); sum(as.numeric(Data1[,4])); sum(as.numeric(Data1[,6]));
[1] 156661507032
[1] 258365498
cat DataProcessing.Pipeline.Utility.GetFastqQCStats.vs1.output | wc
   2896   83858  645395

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.EPACTS.Skat0.epacts  | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); summary(qvalue(Data1[,10]));"

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.EPACTS.Skat0.epacts  | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); summary(qvalue(Data1[,10]));"


cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs3.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs4.py

#Removing '(NM_...' from 'splicing' variants so that their gene IDs can be included. Also split up gene IDs that had ;symbol;smbol' which only affectes really 3 variants that had 'exonic;splcing'
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs4.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.AllPhenos.frq.AndCount.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 29; $i <= 39; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.vs4.txt.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.vs4.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing")) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $0 } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($11 == "P") || ($11 == "D") ) { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($10 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing")) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.All.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.All.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.All.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.All.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.All.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.All.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.All.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.All.EPACTS.Collapse &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.All.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.All.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.All.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.All.EPACTS.Madsen &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.All.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.All.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.All.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.All.EPACTS.wcnt &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.All.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.All.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.All.EPACTS.Skat0 &


cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.All.EPACTS.Skat0.epacts  | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); summary(qvalue(Data1[,10]));"



plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.IBD

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | awk '{ print $1, "\t", $1 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.IBD

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --maf .05 --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.IBD

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --maf .10 --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf10.IBD

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --maf .10 --geno .01 --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf10.geno01.IBD

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.bim | awk '{ if ($1 == "23") { print $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.bim.justX

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --exclude /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.bim.justX --maf .10 --geno .01 --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.noX.maf10.geno01.IBD

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --maf .1 --indep-pairwise 50 5 0.1 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf10.r2_10

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf10.r2_10.prune.in --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf10.r2_10

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --hardy --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.FIDs --hardy --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.hwe | awk '{ if ($6 < .00001) { print $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.hwe.1eNeg5.rsIDs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --maf .05 --hwe .00001 --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.hwe1eNeg5.IBD

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --maf .05 --hwe .0001 --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.hwe1eNeg4.IBD

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --maf .05 --hwe .00001 --relatedness --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.maf05.hwe1Neg5

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --relatedness2 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.IBD.genome | grep -v POOL38_03 | grep -v POOL99_02 | grep -v POOL59_09 | grep -v POOL78_08 | sort -rg -k 8,8 | vi -

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.IBD.genome | grep -v POOL38_03 | grep -v POOL99_02 | grep -v POOL59_09 | grep -v POOL78_08 | sort -rg -k 8,8 | vi -

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --transpose --recode --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.tfam | grep -E 'POOL27_10|POOL63_08' | awk '{ print $1, "\t", $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.tfam.2Indvs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.tfam.2Indvs --transpose --recode --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.just2Indvs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.tfam.2Indvs --transpose --recode --maf .05 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.just2Indvs.maf05

#Based on cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.hwe1eNeg4.IBD.genome | grep -v POOL38_03 | grep -v POOL99_02 | grep -v POOL59_09 | grep -v POOL78_08 | sort -rg -k 9,9 | vi -
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ print $1, "\t", $2, "\t", $314, "\t", $492 }' | grep -v ^# | grep -v "\.\/\." | perl -lane 'my @vals1 = split(/:/, $F[2]); my @vals2 = split(/:/, $F[3]); my @vals1sub = split(/\//, $vals1[0]); my @vals2sub = split(/\//, $vals2[0]); my $flag1; if ($vals1[0] eq $vals2[0]) { $flag1 = 2; } elsif (($vals1sub[0] eq $vals2sub[0]) || ($vals1sub[0] eq $vals2sub[1]) || ($vals1sub[1] eq $vals2sub[0]) || ($vals1sub[1] eq $vals2sub[1])) { $flag1 = 1; } else { $flag1 = 0; } print join("\t", @F), "\t", $flag1;' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); xVals <- c(Data1[1,2]); chrm <- Data1[1,1]; adj <- 0; for (i in 2:nrow(Data1)) { if (Data1[i,1] != chrm) { chrm <- Data1[i,1]; adj <- adj + Data1[i-1,2]; }; xVals <- c(xVals, Data1[i,2] + adj); }; jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.POOL27_10RL_POOL63_08RL.IBSPlot.jpg\", width=5000, height=2000, res=300); plot(xVals, Data1[,5], main=\"Bad - POOL27_10 vs. POOL63_08\", type=\"l\"); dev.off();" &

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ print $1, "\t", $2, "\t", $496, "\t", $541 }' | grep -v ^# | grep -v "\.\/\." | perl -lane 'my @vals1 = split(/:/, $F[2]); my @vals2 = split(/:/, $F[3]); my @vals1sub = split(/\//, $vals1[0]); my @vals2sub = split(/\//, $vals2[0]); my $flag1; if ($vals1[0] eq $vals2[0]) { $flag1 = 2; } elsif (($vals1sub[0] eq $vals2sub[0]) || ($vals1sub[0] eq $vals2sub[1]) || ($vals1sub[1] eq $vals2sub[0]) || ($vals1sub[1] eq $vals2sub[1])) { $flag1 = 1; } else { $flag1 = 0; } print join("\t", @F), "\t", $flag1;' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); xVals <- c(Data1[1,2]); chrm <- Data1[1,1]; adj <- 0; for (i in 2:nrow(Data1)) { if (Data1[i,1] != chrm) { chrm <- Data1[i,1]; adj <- adj + Data1[i-1,2]; }; xVals <- c(xVals, Data1[i,2] + adj); }; jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.POOL64_05RL_POOL6S_12RL.IBSPlot.jpg\", width=5000, height=2000, res=300); plot(xVals, Data1[,5], main=\"Bad - POOL64_05 vs. POOL6S_12\", type=\"l\"); dev.off();" &

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ print $1, "\t", $2, "\t", $130, "\t", $483 }' | grep -v ^# | grep -v "\.\/\." | perl -lane 'my @vals1 = split(/:/, $F[2]); my @vals2 = split(/:/, $F[3]); my @vals1sub = split(/\//, $vals1[0]); my @vals2sub = split(/\//, $vals2[0]); my $flag1; if ($vals1[0] eq $vals2[0]) { $flag1 = 2; } elsif (($vals1sub[0] eq $vals2sub[0]) || ($vals1sub[0] eq $vals2sub[1]) || ($vals1sub[1] eq $vals2sub[0]) || ($vals1sub[1] eq $vals2sub[1])) { $flag1 = 1; } else { $flag1 = 0; } print join("\t", @F), "\t", $flag1;' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); xVals <- c(Data1[1,2]); chrm <- Data1[1,1]; adj <- 0; for (i in 2:nrow(Data1)) { if (Data1[i,1] != chrm) { chrm <- Data1[i,1]; adj <- adj + Data1[i-1,2]; }; xVals <- c(xVals, Data1[i,2] + adj); }; jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.POOL111_10RL_POOL62_04RL.IBSPlot.jpg\", width=5000, height=2000, res=300); plot(xVals, Data1[,5], main=\"Bad - POOL111_10 vs. POOL62_04\", type=\"l\"); dev.off();" &

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ print $1, "\t", $2, "\t", $181, "\t", $699 }' | grep -v ^# | grep -v "\.\/\." | perl -lane 'my @vals1 = split(/:/, $F[2]); my @vals2 = split(/:/, $F[3]); my @vals1sub = split(/\//, $vals1[0]); my @vals2sub = split(/\//, $vals2[0]); my $flag1; if ($vals1[0] eq $vals2[0]) { $flag1 = 2; } elsif (($vals1sub[0] eq $vals2sub[0]) || ($vals1sub[0] eq $vals2sub[1]) || ($vals1sub[1] eq $vals2sub[0]) || ($vals1sub[1] eq $vals2sub[1])) { $flag1 = 1; } else { $flag1 = 0; } print join("\t", @F), "\t", $flag1;' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); xVals <- c(Data1[1,2]); chrm <- Data1[1,1]; adj <- 0; for (i in 2:nrow(Data1)) { if (Data1[i,1] != chrm) { chrm <- Data1[i,1]; adj <- adj + Data1[i-1,2]; }; xVals <- c(xVals, Data1[i,2] + adj); }; jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.POOL118_06RL_POOL91_07RL.IBSPlot.jpg\", width=5000, height=2000, res=300); plot(xVals, Data1[,5], main=\"Bad - POOL118_06 vs. POOL91_07\", type=\"l\"); dev.off();" &

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ print $1, "\t", $2, "\t", $456, "\t", $524 }' | grep -v ^# | grep -v "\.\/\." | perl -lane 'my @vals1 = split(/:/, $F[2]); my @vals2 = split(/:/, $F[3]); my @vals1sub = split(/\//, $vals1[0]); my @vals2sub = split(/\//, $vals2[0]); my $flag1; if ($vals1[0] eq $vals2[0]) { $flag1 = 2; } elsif (($vals1sub[0] eq $vals2sub[0]) || ($vals1sub[0] eq $vals2sub[1]) || ($vals1sub[1] eq $vals2sub[0]) || ($vals1sub[1] eq $vals2sub[1])) { $flag1 = 1; } else { $flag1 = 0; } print join("\t", @F), "\t", $flag1;' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); xVals <- c(Data1[1,2]); chrm <- Data1[1,1]; adj <- 0; for (i in 2:nrow(Data1)) { if (Data1[i,1] != chrm) { chrm <- Data1[i,1]; adj <- adj + Data1[i-1,2]; }; xVals <- c(xVals, Data1[i,2] + adj); }; jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.POOL59_05RL_POOL68_086L.IBSPlot.jpg\", width=5000, height=2000, res=300); plot(xVals, Data1[,5], main=\"Bad - POOL59_05 vs. POOL68_08\", type=\"l\"); dev.off();" &

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ print $1, "\t", $2, "\t", $259, "\t", $308 }' | grep -v ^# | grep -v "\.\/\." | perl -lane 'my @vals1 = split(/:/, $F[2]); my @vals2 = split(/:/, $F[3]); my @vals1sub = split(/\//, $vals1[0]); my @vals2sub = split(/\//, $vals2[0]); my $flag1; if ($vals1[0] eq $vals2[0]) { $flag1 = 2; } elsif (($vals1sub[0] eq $vals2sub[0]) || ($vals1sub[0] eq $vals2sub[1]) || ($vals1sub[1] eq $vals2sub[0]) || ($vals1sub[1] eq $vals2sub[1])) { $flag1 = 1; } else { $flag1 = 0; } print join("\t", @F), "\t", $flag1;' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); xVals <- c(Data1[1,2]); chrm <- Data1[1,1]; adj <- 0; for (i in 2:nrow(Data1)) { if (Data1[i,1] != chrm) { chrm <- Data1[i,1]; adj <- adj + Data1[i-1,2]; }; xVals <- c(xVals, Data1[i,2] + adj); }; jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.POOL17S_09RL_POOL26_01RL.IBSPlot.jpg\", width=5000, height=2000, res=300); plot(xVals, Data1[,5], main=\"Good - POOL17S_09 vs. POOL26_01\", type=\"l\"); dev.off();" &

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ print $1, "\t", $2, "\t", $229, "\t", $496 }' | grep -v ^# | grep -v "\.\/\." | perl -lane 'my @vals1 = split(/:/, $F[2]); my @vals2 = split(/:/, $F[3]); my @vals1sub = split(/\//, $vals1[0]); my @vals2sub = split(/\//, $vals2[0]); my $flag1; if ($vals1[0] eq $vals2[0]) { $flag1 = 2; } elsif (($vals1sub[0] eq $vals2sub[0]) || ($vals1sub[0] eq $vals2sub[1]) || ($vals1sub[1] eq $vals2sub[0]) || ($vals1sub[1] eq $vals2sub[1])) { $flag1 = 1; } else { $flag1 = 0; } print join("\t", @F), "\t", $flag1;' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); xVals <- c(Data1[1,2]); chrm <- Data1[1,1]; adj <- 0; for (i in 2:nrow(Data1)) { if (Data1[i,1] != chrm) { chrm <- Data1[i,1]; adj <- adj + Data1[i-1,2]; }; xVals <- c(xVals, Data1[i,2] + adj); }; jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.POOL13S_04RL_POOL64_05RL.IBSPlot.jpg\", width=5000, height=2000, res=300); plot(xVals, Data1[,5], main=\"Good - POOL13S_04 vs. POOL64_05\", type=\"l\"); dev.off();" &

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ print $1, "\t", $2, "\t", $181, "\t", $708 }' | grep -v ^# | grep -v "\.\/\." | perl -lane 'my @vals1 = split(/:/, $F[2]); my @vals2 = split(/:/, $F[3]); my @vals1sub = split(/\//, $vals1[0]); my @vals2sub = split(/\//, $vals2[0]); my $flag1; if ($vals1[0] eq $vals2[0]) { $flag1 = 2; } elsif (($vals1sub[0] eq $vals2sub[0]) || ($vals1sub[0] eq $vals2sub[1]) || ($vals1sub[1] eq $vals2sub[0]) || ($vals1sub[1] eq $vals2sub[1])) { $flag1 = 1; } else { $flag1 = 0; } print join("\t", @F), "\t", $flag1;' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); xVals <- c(Data1[1,2]); chrm <- Data1[1,1]; adj <- 0; for (i in 2:nrow(Data1)) { if (Data1[i,1] != chrm) { chrm <- Data1[i,1]; adj <- adj + Data1[i-1,2]; }; xVals <- c(xVals, Data1[i,2] + adj); }; jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.POOL118_06RL_POOL92_10RL.IBSPlot.jpg\", width=5000, height=2000, res=300); plot(xVals, Data1[,5], main=\"Good - POOL18_06 vs. POOL92_10\", type=\"l\"); dev.off();" &


zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ print $1, "\t", $2, "\t", $314, "\t", $492 }' | grep -v ^# | grep -v "\.\/\." | perl -lane 'my @vals1 = split(/:/, $F[2]); my @vals2 = split(/:/, $F[3]); my @vals1sub = split(/\//, $vals1[0]); my @vals2sub = split(/\//, $vals2[0]); my $flag1; if ($vals1[0] eq $vals2[0]) { $flag1 = "BLUE"; } elsif (($vals1sub[0] eq $vals2sub[0]) || ($vals1sub[0] eq $vals2sub[1]) || ($vals1sub[1] eq $vals2sub[0]) || ($vals1sub[1] eq $vals2sub[1])) { $flag1 = "BLUE"; } else { $flag1 = "RED"; } if ($F[0] eq "X") { $F[0] = "23"; } print join("\t", @F), "\t", $flag1;' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); yAxis <- c(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, \"X\"); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.POOL27_10RL_POOL63_08RL.IBSPlot.vs2.jpg\", width=5000, height=2500, res=300); Data1.red <- Data1[Data1[,5]==\"RED\",]; plot(Data1[,2], Data1[,1], main=\"Bad - POOL27_10 vs. POOL63_08\", yaxt=\"n\", col=as.character(Data1[,5])); points(Data1.red[,2], Data1.red[,1], col=Data1.red[,5]); axis(2, at=sort(unique(Data1[,1])), labels=yAxis); dev.off();" &







###### Unsure where this all belongs or what i was doing with these steps before
#vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --maf .05 --recode --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.maf05
#
#gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.maf05.recode.vcf
#
#zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.maf05.recode.vcf.gz | awk '{ print $314, "\t", $492 }' | tail -n +124 | perl -lane
#
#plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed
######

#Outputs in directory run in, so go to directory /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/
/home/michaelt/Software/admixture_linux-1.23/admixture /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.bed --cv 2 | tee /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.Admixture.2.log &
/home/michaelt/Software/admixture_linux-1.23/admixture /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.bed --cv 3 | tee /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.Admixture.3.log &
/home/michaelt/Software/admixture_linux-1.23/admixture /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.bed --cv 4 | tee /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.Admixture.4.log &
/home/michaelt/Software/admixture_linux-1.23/admixture /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.bed --cv 5 | tee /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.Admixture.5.log &
/home/michaelt/Software/admixture_linux-1.23/admixture /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.bed --cv 6 | tee /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.Admixture.6.log &


#Manually created this file and added the entries, based on the heterzygous-issue individuals seen in above statement (the grep -v individuals) and the individuals in the .genome file with rather high Z2 values (~80%+)
vi /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.20140324.IBDDropIndvs.FIDs

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --remove /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.20140324.IBDDropIndvs.FIDs --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.gz | awk '{ if ($1 == "23") { $1 = "X"} print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip

/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.20140324.IBDDropIndvs.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.All.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.All.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.All.DropIBDProbIndvs.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.DropIBDProbIndvs.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.DropIBDProbIndvs.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.DropIBDProbIndvs.EPACTS.Skat0 &

#Complex
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.All.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.All.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.All.DropIBDProbIndvs.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.Exonic.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.Exonic.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.Exonic.DropIBDProbIndvs.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.DropIBDProbIndvs.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.DropIBDProbIndvs.EPACTS.Skat0 &


[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.pheno | grep -E 'POOL103_03|POOL105_03'
POOL103_03RL    POOL103_03RL    2       2       -9      -9
POOL105_03RL    POOL105_03RL    2       2       -9      -9
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.pheno | grep -E 'POOL47_10|POOL87_11'
POOL87_11RL     POOL87_11RL     1       -9      -9      -9
POOL47_10RL     POOL47_10RL     2       2       -9      -9
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.pheno | grep -E 'POOL07_01|POOL22_01'
POOL07_01RL     POOL07_01RL     1       -9      -9      -9
POOL22_01RL     POOL22_01RL     1       -9      -9      -9
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.pheno | grep -E 'POOL36_11|POOL50_12'
POOL50_12RL     POOL50_12RL     2       2       1       -9
POOL36_11RL     POOL36_11RL     2       2       1       -9
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.pheno | grep -E 'POOL112_10|POOL116_10'
POOL112_10RL    POOL112_10RL    1       1       -9      -9
POOL116_10RL    POOL116_10RL    2       2       -9      -9


zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50693697 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'


cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.hwe1eNeg4.IBD.genome | perl -ane 'if ($F[8] > 0) { print $F[0], "\n", $F[2], "\n"; } ' | sort | uniq -c | sort -g -k 1,1 | awk '{ if ($1 >= 100) { print $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.hwe1eNeg4.IBD.genome.AppearMoreThan100Times.FIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.hwe1eNeg4.IBD.genome | perl -ane 'if ($F[8] > 0) { print $F[0], "\n", $F[2], "\n"; } ' | sort | uniq -c | sort -g -k 1,1 | awk '{ if ($1 >= 150) { print $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.hwe1eNeg4.IBD.genome.AppearMoreThan150Times.FIDs



#Samtools
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.Samtools.ptb.MergeVCFsAcrossChromosomes.vs1.sh

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.vcf

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.vcf.gz | head -n 119 > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.vcf.header

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.vcf.gz | grep -v INDEL | perl -lane 'if (($F[3] =~ m/^[AGTC]$/) && ($F[4] =~ m/^[AGTC]$/)) { print join("\t", @F); }' | cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.vcf.header - |  gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.vcf.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.vcf.gz | perl -lane 'if ($F[5] >= 50) { print join("\t", @F); } ' | cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.vcf.header - | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.vcf.gz 

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.DropOff.VCFVariants.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.vcf.gz --window 1000 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.vcf.gz

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.vcf.gz --geno .95 --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.geno95

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.vcf.gz --geno .99 --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.geno99

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.geno95.recode.vcf

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.geno95.recode.vcf.gz --plink --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.geno95.recode

plink --file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.geno95.recode --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.geno95.recode

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.geno95.recode --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --maf .01 --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.geno95.recode.WhiteOnly.QCed

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.geno95.recode.WhiteOnly.QCed.genome | grep -v POOL116_06RL | grep -v POOL116_08RL | grep -v POOL104_06RL | grep -v POOL122_11RL | sort -rg -k 9,9 | vi - 



#Redoing tranches -- doing ts90 resulted in almost no SNPs being present after the 'DropOffTarg_1kb' step, so stepped up to ts_99
/home/shared/software/java/jre1.7.0_09/bin/java -Xmx5g -jar /home/shared/software/GATK/2.5.2/GenomeAnalysisTK.jar -T ApplyRecalibration -R /home/michaelt/Data/HumanGenome/GRCh37/Version8/human_g1k_v37.fasta -input /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.vcf -recalFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.recal -tranchesFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.tranches -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts90.vcf --ts_filter_level 90 -mode SNP

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts90.vcf

#Takes forever (with zlib 1.2.3), almost an hour
#vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts90.vcf.gz --remove-filtered-all --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts90.PASS

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts90.vcf.gz | head -n 125 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts90.vcf.header

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts90.vcf.gz | grep -v '^#' | grep -E 'PASS|VQSRTrancheSNP90.00to99.00' | cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts90.vcf.header - | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.vcf.gz 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.vcf.gz | fill-aa -a /home/michaelt/Data/1000G/AncestralAlignments/human_ancestor_GRCh37_e59/human_ancestor_ | gzip -c > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.vcf.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.DropOff.VCFVariants.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.vcf.gz --window 1000 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.DropOffTarg_1kb.vcf.gz

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.DropOffTarg_1kb.vcf.gz --geno .95 --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.DropOffTarg_1kb.95geno

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.DropOffTarg_1kb.95geno.recode.vcf

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.DropOffTarg_1kb.95geno.recode.vcf.gz --plink --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.DropOffTarg_1kb.95geno

plink --file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.DropOffTarg_1kb.95geno --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.DropOffTarg_1kb.95geno

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.DropOffTarg_1kb.95geno --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --maf .05 --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.DropOffTarg_1kb.95geno.maf05


#Doing IBD with variants covering arrays of 1000G individuals to see if get similar results, so doing this on the 4_CS cluster
scp /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/.

vi /KG/1000_Genomes/Samples/VCFs/HIVProj/DataProcessing.Pipeline.Utility.Create1000GOverlapWithArrayVCFs.sh

mv /KG/1000_Genomes/Samples/VCFs/HIVProj/DataProcessing.Pipeline.Utility.Create1000GOverlapWithArrayVCFs.sh /KG/1000_Genomes/Samples/VCFs/HIVProj/DataProcessing.Pipeline.Utility.Create1000GOverlapWithArrayVCFs.vs1.sh

cp -p /KG/1000_Genomes/Samples/VCFs/HIVProj/DataProcessing.Pipeline.Utility.Create1000GOverlapWithArrayVCFs.vs1.sh /KG/1000_Genomes/Samples/VCFs/HIVProj/DataProcessing.Pipeline.Utility.Create1000GOverlapWithArrayVCFs.Launcher.vs1.sh

bash /KG/1000_Genomes/Samples/VCFs/HIVProj/DataProcessing.Pipeline.Utility.Create1000GOverlapWithArrayVCFs.Launcher.vs1.sh

vcf-concat /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr1.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr2.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr3.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr4.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr5.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr6.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr7.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr8.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr9.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr10.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr11.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr12.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr13.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr14.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr15.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr16.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr17.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr18.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr19.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr20.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr21.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr22.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrX.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz | gzip > /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrALL.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz

vcftools --gzvcf /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrALL.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz --maf .05 --geno .95 --hwe .00001 --recode --recode-INFO-all --out /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrALL.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.maf05.geno95.hwe1eNeg5 

gzip /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrALL.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.maf05.geno95.hwe1eNeg5.recode.vcf

vcftools --gzvcf /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrALL.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.maf05.geno95.hwe1eNeg5.recode.vcf.gz --plink-tped --out /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrALL.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.maf05.geno95.hwe1eNeg5

plink --tfile /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrALL.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.maf05.geno95.hwe1eNeg5 --make-bed --out /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrALL.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.maf05.geno95.hwe1eNeg5

plink --bfile /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrALL.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.maf05.geno95.hwe1eNeg5 --genome --out /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrALL.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.maf05.geno95.hwe1eNeg5




#TimeptChck: 20140319

#For Sudhir
cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.csv | perl -F, -lane 'my $poolID; foreach my $entry (@F) { if ($entry =~ m/(.*ool)\s+([a-zA-Z0-9]*)/) { my $store2 = $2; if ($store2 =~ m/^[0-9]$/) { $poolID = "POOL0" . $store2; } else { $poolID = "POOL" . $2; } } } if ($F[2] =~ m/^[0-9]$/) { $F[2] = "0" . $F[2]; } if ($poolID) { print $F[1], "\t", $poolID, "_", $F[2], "RL"; }' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.LinkMACID_POOLID.txt

#Manually correct POOL110_04RL line -- it's wrong

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.LinkMACID_POOLID.txt | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.FIDs | sed 's/\^//g' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.justEuropean.QCed.txt

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.LinkMACID_POOLID.txt | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.hwe1eNeg4.IBD.genome.AppearMoreThan100Times.FIDs | sed 's/\^//g' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.justEuropean.PresentInMoreThan100PairsZ2.txt
cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.LinkMACID_POOLID.txt | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.hwe1eNeg4.IBD.genome.AppearMoreThan150Times.FIDs | sed 's/\^//g' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.justEuropean.PresentInMoreThan150PairsZ2.txt




#At this point began moving files without any 'Vs#' signifier in their names to Vs1/

#mapping_pool_merged directory
mkdir Vs1
ls -lrt | grep -v Vs2 | grep AllPools | awk '{ print $9 }' | perl -ane 'system("mv $F[0] Vs1/$F[0]");'

#GATK Directory
gzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.raw.vcf &	
mkdir Vs1
ls -lrt | grep -v Vs2 | grep AllPools | awk '{ print $9 }' | perl -ane 'system("mv $F[0] Vs1/$F[0]");'

#Samtools Directory
mkdir Vs1
mv AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.Chr* Vs1/.





6th run with mapping_pool_125_
~~~~~~~~~~~

#FIgure out phenotype/covar file stuff here now

scp mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/Sample_status_list_12-10-13_CT.edited.csv /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/.

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Sample_status_list_12-10-13_CT.edited.csv | sed 's/\r/\n/g' | sed 's/"//g ' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Sample_status_list_12-10-13_CT.edited.noMeta.csv

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.noMeta.noBlank.noCarrot.dropIndvs.csv --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Sample_status_list_12-10-13_CT.edited.noMeta.csv > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno

#4 individuals missing from the /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Sample_status_list_12-10-13_CT.edited.noMeta.csv file and unsure why -- emailed Sudhir about it
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk 'BEGIN{ while(getline < "/data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.POOLIDs") { POOLID[$1] = 1 }  } { if (POOLID[$1]) { print $1 } } ' > nana
cat "/data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.POOLIDs" | grep -v -f nana
#POOL43_10RL
#POOL101_11RL
#POOL18S_07RL
#POOL2S_03RL

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk 'BEGIN{ while(getline < "/data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.POOLIDs") { POOLID[$1] = 1; } } { if (POOLID[$1]) { print $0 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno.MainPOOLIDs

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateCovariateFile.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar



ls -lrt /data/userdata/pg/MM5_ALL_MAPPING/. | awk '{ print $9 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/home_pg_ALL_MAPPING_dirList.20140321.txt

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/home_pg_ALL_MAPPING_dirList.20140321.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/home_pg_ALL_MAPPING_dirList.20140321.NotCreatedYet.txt
#Then manually edited to remove directories already used/processed before -- kept mapping_pool_125_

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectLSInfoPerDir.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectLSInfoPerDir.vs3.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectLSInfoPerDir.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.NotCreatedYet.txt MM5 #Creates subdirectory folder-list per directory

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.MakeDir.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.MakeDir.vs3.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.MakeDir.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.NotCreatedYet.txt MM5 #Creates the rest of the sub-subdirectories, etc. in ALL_MAPPING

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectOrigSFFFileInformation.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectOrigSFFFileInformation.vs3.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectOrigSFFFileInformation.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.NotCreatedYet.txt MM5 #Creates home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.${date1}.txt file

cat /data/userdata/pg/michaelt/Data/sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.20140321_160330.txt | sort | uniq > /data/userdata/pg/michaelt/Data/sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.20140321_160330.SortUniq.txt

bash /data/userdata/pg/michaelt/Data/sffFiles/DataOrganization.MakeDir.vs2.sh /data/userdata/pg/michaelt/Data/sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.20140321_160330.SortUniq.txt

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CombinePoolNameswsffFileIDs.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CombinePoolNameswsffFileIDs.vs3.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CombinePoolNameswsffFileIDs.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.NotCreatedYet.txt /data/userdata/pg/michaelt/Data/sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.20140321_160330.SortUniq.txt MM5 #Creates file such as DataOrganization.CombinePoolNameswsffFileIDs.vs3.${date1}.output

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.ConnectPoolsffFileswithIndvMIDRLsffFiles.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.ConnectPoolsffFileswithIndvMIDRLsffFiles.vs3.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.ConnectPoolsffFileswithIndvMIDRLsffFiles.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.NotCreatedYet.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CombinePoolNameswsffFileIDs.vs3.20140321_161846.output MM5

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.NotCreatedYet.txt MM5 #Creates DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.${date1}.output file 

bash /data/userdata/pg/michaelt/Data/sffFiles/DataProcessing.BreakUpMIDs.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.20140321_160330.SortUniq.txt

bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.20140321_164539.output

cp -p DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.vs4.sh DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.vs5.sh

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.20140321_164539.output

#Edited DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output via :.,$ s/Pools/Pools\/MM5/g 

cat DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.20140321_164539.output > DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output

cp -p DataOrganization.Utility.CreatebamMergeList.vs2.sh DataOrganization.Utility.CreatebamMergeList.vs3.sh

bash DataOrganization.Utility.CreatebamMergeList.vs3.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output

#Remove carrots from file, not dropping individuals. Then creating a file which takes all the IDs from the second column in the POP_genomics file, hashes them, then prints out the list of multiple entries per ID plus their RMID. The extra perl stuff is formatting this because I print out multiple python arrays which have that funny [' ', ' ', ' '] formatting (joined by ;). I then grep out the multiple entries of the main 'CreatebamMergeList' file given the list of multiple entries for the single ID, add them together in a single line and reappend them to the original CreatebamMergeList file. * individuals are dropped because their ID names have the star so they don't get matched up to non-star ID# versions of that ID#. I then have to manually remove the star entries from the POP_genomics file so that its matched up to the CreatebamMergeList which future downstream steps are based on

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.csv | sed 's/\^//g' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.noCarrot.csv

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.Utility.CheckForDupEntriesPOPSampleInfoFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.noCarrot.csv | perl -F: -ane 'my @vals1 = split(/,/, $F[0]); my $id1 = $vals1[1]; print $id1, ":"; foreach my $entry1 (@F) { my @valsN = split(/,/, $entry1); my @MID = split(//, $valsN[2]); if ($#MID == "3") { print "0", $valsN[2], ","; } else { print $valsN[2], ","; } foreach my $entry2 (@valsN) { if ($entry2 =~ m/ool/) { print $entry2; } } print ":";} print "\n";' | sed "s/'//g" | sed 's/\s//g' | perl -F: -lane 'print join(":", @F[1..$#F]);' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.noCarrot.DupListPrepForMergeListUse.txt

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineMultPoolsForSameIndvbamMergeList.vs1.sh /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.noCarrot.DupListPrepForMergeListUse.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140323_234600.output #Creates DataOrganization.CreatebamMergeList.vs3.20140323_234600.output.multPOOLperIndvCollected* files

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.noCarrot.csv | grep -v \* > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.noCarrot.dropIndvs.csv 

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.DropIndividualsBasedOnPopGenomFile.vs3.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.noCarrot.dropIndvs.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140323_234600.output.multPOOLperIndvCollected > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140323_234600.output.multPOOLperIndvCollected.dropIndvs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.20140321_164539.output | awk '{ print $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.20140321_164539.output.justSFFFiles

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140323_234600.output.multPOOLperIndvCollected.dropIndvs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.20140321_164539.output.justSFFFiles > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140323_234600.output.multPOOLperIndvCollected.dropIndvs.justRecentAdditions

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs2.sh

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.vs4.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140323_234600.output.multPOOLperIndvCollected.dropIndvs.justRecentAdditions 

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs2.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140323_234600.output.multPOOLperIndvCollected.dropIndvs.justRecentAdditions

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs2.sh

#bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140323_234600.output.multPOOLperIndvCollected.dropIndvs


#Troublehsooting

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.noCarrot.DupListPrepForMergeListUse.txt | grep pool7:01 > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.noCarrot.DupListPrepForMergeListUse.01_pool7_01_Pool22.txt

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineMultPoolsForSameIndvbamMergeList.vs1.sh /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.noCarrot.DupListPrepForMergeListUse.01_pool7_01_Pool22.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140323_234600.output.multPOOLperIndvCollected.test



#At this point realized there were some differences between the earlier 20140322_POP_genomics_sample_information_2... files I was using and the most recent one (I had just been appending the most recent additions of the new files to my on-going, edited file from earlier versions. However, a few entries had been changed beyond just the new additions, so needed to more thoroughly compare IDs between my edited, on-going version and the newest file. So redoing things from the 'MergeallSampleBamFiles' point onwards. Also included individuals from POOL86 which had been skipped at some point

ls -lrt /data/userdata/pg/MM5_ALL_MAPPING/mapping_pool_78-90/. | awk '{ print $9 } ' > Pools/MM5/PreMerge/mapping_pool_78-90/home_pg_ALL_MAPPING_dirList.WorkingMainList.txt.mapping_pool_78-90.Full

cat Pools/MM5/PreMerge/mapping_pool_78-90/home_pg_ALL_MAPPING_dirList.WorkingMainList.txt.mapping_pool_78-90.Full | grep 86 > Pools/MM5/PreMerge/mapping_pool_78-90/home_pg_ALL_MAPPING_dirList.WorkingMainList.txt.mapping_pool_78-90

cp -p home_pg_ALL_MAPPING_dirList.20140321.NotCreatedYet.txt home_pg_ALL_MAPPING_dirList.20140321.86Redo.txt

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PreMerge/mapping_pool_78-90/home_pg_ALL_MAPPING_dirList.WorkingMainList.txt.mapping_pool_78-90 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PreMerge/mapping_pool_78-90/home_pg_ALL_MAPPING_dirList.20140321.86Redo.txt.mapping_pool_78-90

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PreMerge/mapping_pool_78-90/home_pg_ALL_MAPPING_dirList.WorkingMainList.txt.mapping_pool_78-90.Full home_pg_ALL_MAPPING_dirList.WorkingMainList.txt.mapping_pool_78-90

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.MakeDir.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.86Redo.txt MM5 #Creates the rest of the sub-subdirectories, etc. in ALL_MAPPING

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectOrigSFFFileInformation.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.86Redo.txt MM5 #Creates home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.${date1}.txt file

cat /data/userdata/pg/michaelt/Data/sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.20140328_154417.txt | sort | uniq > /data/userdata/pg/michaelt/Data/sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.20140328_154417.SortUniq.txt

bash /data/userdata/pg/michaelt/Data/sffFiles/DataOrganization.MakeDir.vs2.sh /data/userdata/pg/michaelt/Data/sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.20140328_154417.SortUniq.txt

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CombinePoolNameswsffFileIDs.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.86Redo.txt /data/userdata/pg/michaelt/Data/sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.20140328_154417.SortUniq.txt MM5 #Creates file such as DataOrganization.CombinePoolNameswsffFileIDs.vs3.${date1}.output

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.ConnectPoolsffFileswithIndvMIDRLsffFiles.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.86Redo.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CombinePoolNameswsffFileIDs.vs3.20140328_154612.output MM5

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.86Redo.txt MM5 #Creates DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.${date1}.output file 

bash /data/userdata/pg/michaelt/Data/sffFiles/DataProcessing.BreakUpMIDs.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.20140328_154417.SortUniq.txt

bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.20140328_173953.output

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.20140328_173953.output

cat DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.20140328_173953.output > DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output.temp

mv DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output.temp DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output

bash DataOrganization.Utility.CreatebamMergeList.vs3.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output

scp mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.csv mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.worksheet.csv /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/.

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.csv | sed 's/\r/\n/g' | grep -v '^$' | sed 's/\^//g' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.noMeta.noBlank.noCarrot.csv

#MANUALLY included *s next to POOL106-01 (there are two of them and for each MACID there is a duplicate so should be fine) and POOL112-01 because there are two entries in the POP_genomics files for this POOL#_MID# combination, so unsure which is correct MACID. But since there are duplicate entries for each MACID should be fine 

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.Utility.CheckForDupEntriesPOPSampleInfoFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.noMeta.noBlank.noCarrot.csv | perl -F: -ane 'chomp(@F); my @vals1 = split(/,/, $F[0]); my $id1 = $vals1[1]; print $id1, ":"; foreach my $entry1 (@F) { my @valsN = split(/,/, $entry1); my @MID = split(//, $valsN[2]); if ($#MID == "3") { print "0", $valsN[2], ","; } else { print $valsN[2], ","; } foreach my $entry2 (@valsN) { if ($entry2 =~ m/ool/) { print $entry2; } } print ":";} print "\n";' | sed "s/'//g" | sed 's/\s//g' | sed 's/\]//g' | perl -F: -lane 'print join(":", @F[1..$#F]);' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.noMeta.noBlank.noCarrot.DupListPrepForMergeListUse.txt

cp -p DataProcessing.Pipeline.Utility.CombineMultPoolsForSameIndvbamMergeList.vs1.sh DataProcessing.Pipeline.Utility.CombineMultPoolsForSameIndvbamMergeList.vs2.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineMultPoolsForSameIndvbamMergeList.vs2.sh /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.noMeta.noBlank.noCarrot.DupListPrepForMergeListUse.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output #Creates DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected* files

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.noMeta.noBlank.noCarrot.csv | grep -v \* > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.noMeta.noBlank.noCarrot.dropIndvs.csv 

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.DropIndividualsBasedOnPopGenomFile.vs3.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.noMeta.noBlank.noCarrot.dropIndvs.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs | perl -F\; -lane 'my @vals1 = split(/\|/, $F[0]); my @vals2 = split(/\//, $vals1[0]); if ($vals2[$#vals2] =~ m/(POOL[a-zA-Z0-9]*).*(\d\d+RL)/) { print $1 . "_" . $2 ; } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.POOLIDs

#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs | awk '{ if (NR >= 833) { print $0 } } ' > DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.Redos
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs | awk '{ if (NR >= 837) { print $0 } } ' > DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.Redos

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.Redos

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.Redos

#Did this process twice, once originally and again with the POOL106/POOL112 issue

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs | grep -E 'POOL157-01|POOL158-08|POOL158-01|POOL171-05' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.Redos2

#Once more for POOL15S-04 issue

cat DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs | grep POOL16S-02 > DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.Redos3

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.Redos2

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.Redos3

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.Redos2

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.Redos3

#bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.pta.SplitBamIntoChromosomes.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Vs1

ls -lrt | grep -v Vs2 | grep AllPools | awk '{ print $9 }' | perl -ane 'system("mv $F[0] Vs1/$F[0]");'

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.pta.RunReduceReadsPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptb.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.Samtools.pta.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptc.MergeVCFsAcrossChromosomes.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptc.MergeVCFsAcrossChromosomes.vs2.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptc.MergeVCFsAcrossChromosomes.vs2.sh

cp -p DataProcessing.Pipeline.pt6.GATK.ptd.RecalAndApplyVCFfilesPerChromosome.vs1.sh DataProcessing.Pipeline.pt6.GATK.ptd.RecalAndApplyVCFfilesPerChromosome.vs2.sh 

bash DataProcessing.Pipeline.pt6.GATK.ptd.RecalAndApplyVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

scp Pools/MM5/PostMerge/mapping_pool_merged/GATK/*tranches.pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/.

gzip -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf &

gzip -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.vcf &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz --remove-filtered VQSRTrancheSNP99.90to100.00 --remove-filtered LowQual --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9

#For zlib installing, directory /home/michaelt/Software/zlib-1.2.8
./configure --prefix=/home/michaelt/lib; make test
#Unsure how to make vcftools use this particular zlib library

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.recode.vcf

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.recode.vcf.gz | fill-aa -a /home/michaelt/Data/1000G/AncestralAlignments/human_ancestor_GRCh37_e59/human_ancestor_ | gzip -c > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.vcf.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.vcf.gz | perl -lane 'if ($F[0] =~ m/^#/) { print join("\t", @F); } else { my @alleles2 = split(/,/, $F[4]); if ($#alleles2 == 0) { print join("\t", @F); } }' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.DropOff.VCFVariants.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --window 1000 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.vcf.gz

#/home/michaelt/Software/vcftools_0.1.12a/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.vcf.gz --missing-indv --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb

gzip -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf

#Exploring what HWE cutoff to use

#vcftools --gzvcf  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz --hardy --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95

#Going with .0001 HWE cutoff. At .00001 1809 SNPs are dropped and at .0001 2470 SNPs are dropped, so not much is lost for another magnitude increase in strictness. Between .0001 and .001 not a large number of SNPs would be lost either, however see examples where obs is 900/0/1 and exp is 900/2/0 on a number of X chromosome locations and this produces a p-value between .0001 and .001. It appears that vcf file encoding gives males genotypes for both 'chromosomes' even though they only have one, so 900/0/1 is not unexpected. Therefore I feel like these SNPs should be kept and going up to .001 may start leading to a greater proportion of false positives

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz --hwe .0001 --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.recode.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf


#Note -- takes about ~2-3 days to complete even with -nt 6?
/home/shared/software/java/jre1.7.0_09/bin/java -Xmx5g -jar /home/shared/software/GATK/2.5.2/GenomeAnalysisTK.jar -T VariantAnnotator -R /home/michaelt/Data/HumanGenome/GRCh37/Version4/human_g1k_v37.fasta -I /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.bam  -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.HRun.vcf -A HomopolymerRun --variant /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf -nt 6

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.HRun.vcf  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.HRun.vcf.idx /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.idx

gzip -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | perl -lane 'my @vals1 = split(/;/, $F[7]); foreach my $SNPinfo (@vals1) { if ($SNPinfo =~ m/HRun/) { print $SNPinfo; } };' | sort | uniq -c 

[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | perl -lane 'my @vals1 = split(/;/, $F[7]); foreach my $SNPinfo (@vals1) { if ($SNPinfo =~ m/HRun/) { print $SNPinfo; } };' | sort | uniq -c
	47294 HRun=0
	18359 HRun=1
	5419 HRun=2
	1668 HRun=3
	401 HRun=4
	97 HRun=5
	15 HRun=6
	6 HRun=7
	2 HRun=8
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | grep HRun=8 | perl -lane 'print join("\t", @F[0..9]);'
2       69628089        rs12464941      A       T       6521.76 VQSRTrancheSNP99.00to99.90      AA=A;AC=197;AF=0.111;AN=1776;BaseQRankSum=-18.785;DB;DP=6423;Dels=0.00;FS=7.055;HRun=8;HaplotypeScore=4.3163;InbreedingCoeff=-0.0905;MLEAC=179;MLEAF=0.101;MQ=59.99;MQ0=0;MQRankSum=1.477;POSITIVE_TRAIN_SITE;QD=4.41;ReadPosRankSum=-0.423;VQSLOD=1.36;culprit=QD GT:AD:DP:GQ:PL  0/0:10,0:10:30:0,30,275
11      63395938        .       G       T       87.12   VQSRTrancheSNP99.00to99.90      AA=G;AC=4;AF=2.230e-03;AN=1794;BaseQRankSum=-3.920;DP=5680;Dels=0.00;FS=3.144;HRun=8;HaplotypeScore=3.2769;InbreedingCoeff=-0.0297;MLEAC=2;MLEAF=1.115e-03;MQ=59.99;MQ0=0;MQRankSum=1.000;QD=2.72;ReadPosRankSum=-1.026;VQSLOD=-1.290e+00;culprit=QD       GT:AD:DP:GQ:PL  0/0:9,0:9:27:0,27,276
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | grep HRun=7 | perl -lane 'print join("\t", @F[0..9]);'
2       182390107       rs2305580       A       T       10719.60        VQSRTrancheSNP99.00to99.90      AA=A;AC=156;AF=0.086;AN=1804;BaseQRankSum=-25.342;DB;DP=10424;Dels=0.00;FS=31.039;HRun=7;HaplotypeScore=5.5982;InbreedingCoeff=-0.0131;MLEAC=144;MLEAF=0.080;MQ=60.00;MQ0=0;MQRankSum=0.217;NEGATIVE_TRAIN_SITE;POSITIVE_TRAIN_SITE;QD=5.97;ReadPosRankSum=3.211;VQSLOD=-6.062e-01;culprit=FS      GT:AD:DP:GQ:PL  0/0:13,0:13:39:0,39,410
3       112272598       rs183474871     T       A       721.65  VQSRTrancheSNP99.00to99.90      AA=T;AC=31;AF=0.018;AN=1758;BaseQRankSum=-11.115;DB;DP=4946;Dels=0.01;FS=4.698;HRun=7;HaplotypeScore=2.4788;InbreedingCoeff=-0.0591;MLEAC=24;MLEAF=0.014;MQ=59.98;MQ0=0;MQRankSum=1.272;POSITIVE_TRAIN_SITE;QD=3.86;ReadPosRankSum=-1.012;VQSLOD=-1.082e-01;culprit=QD     GT:AD:DP:GQ:PL  0/0:2,1:3:6:0,6,62
7       24855651        rs76330454      T       A       2708.55 VQSRTrancheSNP99.00to99.90      AA=t;AC=166;AF=0.094;AN=1770;BaseQRankSum=-8.001;DB;DP=5579;Dels=0.03;FS=71.176;HRun=7;HaplotypeScore=4.9968;InbreedingCoeff=-0.0691;MLEAC=120;MLEAF=0.068;MQ=59.92;MQ0=0;MQRankSum=-0.782;NEGATIVE_TRAIN_SITE;POSITIVE_TRAIN_SITE;QD=2.58;ReadPosRankSum=2.005;VQSLOD=-1.157e+00;culprit=QD       GT:AD:DP:GQ:PL  0/0:4,0:4:12:0,12,107
12      104333866       rs1165685       T       A       24175.80        VQSRTrancheSNP99.00to99.90      AA=t;AC=560;AF=0.320;AN=1748;BaseQRankSum=2.702;DB;DP=4895;Dels=0.03;FS=3.324;HRun=7;HaplotypeScore=4.9939;InbreedingCoeff=-0.1876;MLEAC=523;MLEAF=0.299;MQ=59.97;MQ0=0;MQRankSum=-0.169;POSITIVE_TRAIN_SITE;QD=8.39;ReadPosRankSum=1.064;VQSLOD=-3.806e-01;culprit=DP     GT:AD:DP:GQ:PL  0/1:1,2:3:15:37,0,15
14      81992779        rs76653020      A       T       2523.52 VQSRTrancheSNP99.00to99.90      AA=A;AC=36;AF=0.020;AN=1808;BaseQRankSum=-12.836;DB;DP=12204;Dels=0.01;FS=4.667;HRun=7;HaplotypeScore=6.2385;InbreedingCoeff=0.0585;MLEAC=29;MLEAF=0.016;MQ=59.99;MQ0=0;MQRankSum=2.132;POSITIVE_TRAIN_SITE;QD=5.20;ReadPosRankSum=-2.150;VQSLOD=-3.809e-01;culprit=HaplotypeScore GT:AD:DP:GQ:PL  0/0:13,0:13:39:0,39,415
17      17180049        rs150128160     A       T       452.79  VQSRTrancheSNP99.00to99.90      AA=A;AC=27;AF=0.015;AN=1774;BaseQRankSum=-3.092;DB;DP=5161;Dels=0.00;FS=0.759;HRun=7;HaplotypeScore=4.9453;InbreedingCoeff=-0.0183;MLEAC=18;MLEAF=0.010;MQ=59.94;MQ0=0;MQRankSum=0.504;POSITIVE_TRAIN_SITE;QD=3.10;ReadPosRankSum=-0.415;VQSLOD=-6.715e-01;culprit=QD      GT:AD:DP:GQ:PL  0/0:3,0:3:9:0,9,85
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | grep HRun=6 | perl -lane 'print join("\t", @F[0..9]);'
1       184679716       rs149454208     C       A       274.86  VQSRTrancheSNP99.00to99.90      AA=C;AC=18;AF=9.923e-03;AN=1814;BaseQRankSum=-7.850;DB;DP=9470;Dels=0.00;FS=9.233;HRun=6;HaplotypeScore=5.5968;InbreedingCoeff=0.0145;MLEAC=9;MLEAF=4.961e-03;MQ=59.98;MQ0=0;MQRankSum=-0.037;POSITIVE_TRAIN_SITE;QD=2.10;ReadPosRankSum=-1.369;VQSLOD=-7.438e-01;culprit=QD       GT:AD:DP:GQ:PL  0/0:11,0:11:33:0,33,319
2       61760814        rs115819167     C       T       3750.38 VQSRTrancheSNP99.00to99.90      AA=C;AC=46;AF=0.025;AN=1816;BaseQRankSum=-15.429;DB;DP=12101;Dels=0.01;FS=18.682;HRun=6;HaplotypeScore=4.4872;InbreedingCoeff=-0.0312;MLEAC=42;MLEAF=0.023;MQ=59.99;MQ0=0;MQRankSum=-0.764;POSITIVE_TRAIN_SITE;QD=5.78;ReadPosRankSum=-1.032;VQSLOD=0.188;culprit=FS       GT:AD:DP:GQ:PL  0/0:6,0:6:15:0,15,156
3       51449206        .       A       T       173.08  VQSRTrancheSNP99.00to99.90      AA=A;AC=9;AF=5.039e-03;AN=1786;BaseQRankSum=-6.696;DP=5412;Dels=0.01;FS=0.839;HRun=6;HaplotypeScore=2.1658;InbreedingCoeff=-0.0358;MLEAC=4;MLEAF=2.240e-03;MQ=59.99;MQ0=0;MQRankSum=-0.333;QD=2.58;ReadPosRankSum=0.974;VQSLOD=-8.534e-01;culprit=QD       GT:AD:DP:GQ:PL  0/0:8,0:8:24:0,24,241
4       113574658       rs145663069     G       T       3963.70 VQSRTrancheSNP99.00to99.90      AA=g;AC=67;AF=0.037;AN=1812;BaseQRankSum=-12.104;DB;DP=10857;Dels=0.01;FS=3.332;HRun=6;HaplotypeScore=6.4317;InbreedingCoeff=-0.0475;MLEAC=60;MLEAF=0.033;MQ=59.92;MQ0=0;MQRankSum=-0.678;POSITIVE_TRAIN_SITE;QD=5.27;ReadPosRankSum=-0.755;VQSLOD=-9.985e-01;culprit=HaplotypeScore       GT:AD:DP:GQ:PL  0/0:10,0:10:30:0,30,281
5       52095024        rs116734477     C       T       1415.55 VQSRTrancheSNP99.00to99.90      AA=C;AC=63;AF=0.036;AN=1766;BaseQRankSum=-10.197;DB;DP=4709;Dels=0.01;FS=5.456;HRun=6;HaplotypeScore=2.1840;InbreedingCoeff=-0.0134;MLEAC=56;MLEAF=0.032;MQ=59.99;MQ0=0;MQRankSum=0.626;POSITIVE_TRAIN_SITE;QD=4.03;ReadPosRankSum=0.504;VQSLOD=0.471;culprit=QD   GT:AD:DP:GQ:PL  0/0:3,0:3:9:0,9,84
6       7894794 rs1238994       G       T       26388.40        VQSRTrancheSNP99.00to99.90      AA=G;AC=464;AF=0.260;AN=1786;BaseQRankSum=-24.753;DB;DP=6021;Dels=0.02;FS=9.077;HRun=6;HaplotypeScore=3.7333;InbreedingCoeff=0.0220;MLEAC=416;MLEAF=0.233;MQ=59.99;MQ0=0;MQRankSum=-1.277;POSITIVE_TRAIN_SITE;QD=9.70;ReadPosRankSum=1.017;VQSLOD=1.39;culprit=DP  GT:AD:DP:GQ:PL  0/0:4,0:4:12:0,12,108
6       8062437 rs9405369       T       A       17852.20        VQSRTrancheSNP99.00to99.90      AA=A;AC=360;AF=0.205;AN=1758;BaseQRankSum=-34.017;DB;DP=6008;Dels=0.03;FS=23.705;HRun=6;HaplotypeScore=4.9241;InbreedingCoeff=-0.0703;MLEAC=341;MLEAF=0.194;MQ=59.98;MQ0=0;MQRankSum=0.498;NEGATIVE_TRAIN_SITE;POSITIVE_TRAIN_SITE;QD=7.21;ReadPosRankSum=3.990;VQSLOD=-1.722e-01;culprit=FS       GT:AD:DP:GQ:PL  0/0:2,0:2:6:0,6,61
7       148451445       .       C       T       100.12  VQSRTrancheSNP99.00to99.90      AA=C;AC=7;AF=3.867e-03;AN=1810;BaseQRankSum=-3.029;DP=11028;Dels=0.00;FS=0.732;HRun=6;HaplotypeScore=4.2330;InbreedingCoeff=-0.0087;MLEAC=1;MLEAF=5.525e-04;MQ=59.98;MQ0=0;MQRankSum=-1.175;QD=1.79;ReadPosRankSum=-1.119;VQSLOD=0.023;culprit=QD  GT:AD:DP:GQ:PL  0/0:8,0:8:24:0,24,239
9       125782985       rs137999993     G       T       59.56   VQSRTrancheSNP99.00to99.90      AA=G;AC=4;AF=2.222e-03;AN=1800;BaseQRankSum=-4.567;DB;DP=7953;Dels=0.00;FS=0.000;HRun=6;HaplotypeScore=2.6845;InbreedingCoeff=-0.0210;MLEAC=2;MLEAF=1.111e-03;MQ=59.99;MQ0=0;MQRankSum=-0.096;POSITIVE_TRAIN_SITE;QD=1.65;ReadPosRankSum=-1.713;VQSLOD=0.402;culprit=QD    GT:AD:DP:GQ:PL  0/0:13,0:13:39:0,39,416
10      6138587 rs77712162      A       T       570.05  VQSRTrancheSNP99.00to99.90      AA=A;AC=31;AF=0.017;AN=1794;BaseQRankSum=-10.236;DB;DP=6688;Dels=0.00;FS=0.787;HRun=6;HaplotypeScore=3.7655;InbreedingCoeff=-0.0171;MLEAC=17;MLEAF=9.476e-03;MQ=59.99;MQ0=0;MQRankSum=-1.635;POSITIVE_TRAIN_SITE;QD=2.43;ReadPosRankSum=0.316;VQSLOD=0.453;culprit=QD      GT:AD:DP:GQ:PL  0/0:10,0:10:30:0,30,310
11      107977890       rs150382024     C       T       721.48  VQSRTrancheSNP99.00to99.90      AA=C;AC=25;AF=0.014;AN=1780;BaseQRankSum=-9.441;DB;DP=9796;Dels=0.00;FS=26.757;HRun=6;HaplotypeScore=4.1099;InbreedingCoeff=0.0079;MLEAC=11;MLEAF=6.180e-03;MQ=59.99;MQ0=0;MQRankSum=-0.181;POSITIVE_TRAIN_SITE;QD=3.21;ReadPosRankSum=-1.580;VQSLOD=-2.290e-02;culprit=FS GT:AD:DP:GQ:PL  0/0:8,0:8:24:0,24,247
14      81964559        rs112281983     C       T       730.37  VQSRTrancheSNP99.00to99.90      AA=C;AC=5;AF=2.765e-03;AN=1808;BaseQRankSum=-9.942;DB;DP=17920;Dels=0.00;FS=5.016;HRun=6;HaplotypeScore=6.8203;InbreedingCoeff=-0.0051;MLEAC=3;MLEAF=1.659e-03;MQ=59.99;MQ0=0;MQRankSum=0.284;POSITIVE_TRAIN_SITE;QD=6.46;ReadPosRankSum=-0.256;VQSLOD=-3.850e-01;culprit=HaplotypeScore   GT:AD:DP:GQ:PL  0/0:12,0:12:36:0,36,380
19      32930631        rs73567553      G       T       1706.24 VQSRTrancheSNP99.00to99.90      AA=G;AC=14;AF=7.752e-03;AN=1806;BaseQRankSum=-12.128;DB;DP=19003;Dels=0.00;FS=1.714;HRun=6;HaplotypeScore=6.7353;InbreedingCoeff=-0.0087;MLEAC=12;MLEAF=6.645e-03;MQ=59.99;MQ0=0;MQRankSum=1.297;POSITIVE_TRAIN_SITE;QD=5.88;ReadPosRankSum=-1.836;VQSLOD=-3.085e-01;culprit=HaplotypeScore        GT:AD:DP:GQ:PL  0/0:13,0:13:36:0,36,369
19      32931174        .       C       T       675.07  VQSRTrancheSNP99.00to99.90      AA=C;AC=11;AF=6.138e-03;AN=1792;BaseQRankSum=-7.756;DP=8854;Dels=0.00;FS=1.009;HRun=6;HaplotypeScore=3.5924;InbreedingCoeff=-0.0105;MLEAC=6;MLEAF=3.348e-03;MQ=59.99;MQ0=0;MQRankSum=-1.300;QD=5.49;ReadPosRankSum=-0.304;VQSLOD=0.400;culprit=QD  GT:AD:DP:GQ:PL  0/0:6,0:6:18:0,18,180
X       64950246        rs188435729     C       T       258.32  VQSRTrancheSNP99.00to99.90      AA=C;AC=20;AF=0.012;AN=1738;BaseQRankSum=-4.386;DB;DP=4701;Dels=0.00;FS=2.940;HRun=6;HaplotypeScore=2.6133;InbreedingCoeff=-0.0035;MLEAC=5;MLEAF=2.877e-03;MQ=60.00;MQ0=0;MQRankSum=0.140;POSITIVE_TRAIN_SITE;QD=2.53;ReadPosRankSum=-3.178;VQSLOD=-1.397e+00;culprit=QD   GT:AD:DP:GQ:PL  0/0:2,0:2:6:0,6,57

vcftools --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf --plink --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4

plink --file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4

###IBD pair list -- decided going to do eigenstrat first and see if 1) dropping white individuals due to QC and only looking at white individuals lowers the set of removed individuals overall 2) potentially look for clustering amongst those that are dropping or have high z1 vs z0, z2 (e.g. always at the end of the cloud of white non-hispanics)

POOL103_03RL POOL103_03RL POOL105_03RL POOL105_03RL UN    NA  0.0000  0.0480  0.9520  0.9760  -1  0.998919  1.0000      NA
POOL47_10RL  POOL47_10RL  POOL87_11RL  POOL87_11RL UN    NA  0.0038  0.1065  0.8896  0.9429  -1  0.997455  1.0000 313.0000
POOL175_11RL POOL175_11RL  POOL43_10RL  POOL43_10RL UN    NA  0.0019  0.1445  0.8535  0.9258  -1  0.996671  1.0000      NA
POOL36_11RL  POOL36_11RL  POOL50_12RL  POOL50_12RL UN    NA  0.0077  0.1560  0.8363  0.9143  -1  0.996195  1.0000 306.0000
POOL176_12RL POOL176_12RL  POOL36_12RL  POOL36_12RL UN    NA  0.0270  0.1440  0.8291  0.9010  -1  0.995736  1.0000 345.0000
POOL104_07RL POOL104_07RL  POOL33_05RL  POOL33_05RL UN    NA  0.0155  0.1616  0.8230  0.9037  -1  0.995775  1.0000 149.5000
POOL165_12RL POOL165_12RL  POOL71_11RL  POOL71_11RL UN    NA  0.0213  0.1740  0.8048  0.8918  -1  0.995276  1.0000 297.0000
POOL28_06RL  POOL28_06RL  POOL91_11RL  POOL91_11RL UN    NA  0.0194  0.1825  0.7981  0.8893  -1  0.995154  1.0000 73.7500
POOL112_10RL POOL112_10RL POOL116_10RL POOL116_10RL UN    NA  0.0275  0.1951  0.7775  0.8750  -1  0.994566  1.0000 145.0000
POOL163_12RL POOL163_12RL  POOL69_08RL  POOL69_08RL UN    NA  0.0372  0.2257  0.7371  0.8500  -1  0.993508  1.0000 96.3333

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome | sort -rg -k 9,9 | vi -

POOL103_03RL POOL103_03RL POOL105_03RL POOL105_03RL UN    NA  0.0000  0.0434  0.9566  0.9783  -1  0.998919  1.0000      NA
POOL47_10RL  POOL47_10RL  POOL87_11RL  POOL87_11RL UN    NA  0.0034  0.0963  0.9003  0.9484  -1  0.997455  1.0000 313.0000
POOL36_11RL  POOL36_11RL  POOL50_12RL  POOL50_12RL UN    NA  0.0068  0.1412  0.8521  0.9226  -1  0.996195  1.0000 306.0000
POOL112_10RL POOL112_10RL POOL116_10RL POOL116_10RL UN    NA  0.0242  0.1772  0.7985  0.8871  -1  0.994566  1.0000 145.0000
POOL163_12RL POOL163_12RL  POOL69_08RL  POOL69_08RL UN    NA  0.0328  0.2053  0.7619  0.8645  -1  0.993508  1.0000 96.3333

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -E 'POOL103_03RL|POOL105_03RL'
POOL103_03RL    POOL103_03RL    2       2       -9      -9
POOL105_03RL    POOL105_03RL    2       2       -9      -9

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -E 'POOL47_10RL|POOL87_11RL'
POOL87_11RL     POOL87_11RL     1       -9      -9      -9
POOL47_10RL     POOL47_10RL     2       2       -9      -9

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -E 'POOL36_11RL|POOL50_12RL'
POOL50_12RL     POOL50_12RL     2       2       1       -9
POOL36_11RL     POOL36_11RL     2       2       1       -9

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -E 'POOL112_10RL|POOL116_10RL'
POOL112_10RL    POOL112_10RL    1       1       -9      -9
POOL116_10RL    POOL116_10RL    2       2       -9      -9

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -E 'POOL163_12RL|POOL69_08RL'
POOL69_08RL     POOL69_08RL     1       1       -9      -9
POOL163_12RL    POOL163_12RL    1       -9      -9      -9

perl -e 'print "POOL103_03RL\nPOOL105_03RL\nPOOL87_11RL\nPOOL47_10RL\nPOOL50_12RL\nPOOL36_11RL\nPOOL112_10RL\nPOOL116_10RL\nPOOL69_08RL\nPOOL163_12RL\n";' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.FIDs


vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.FIDs --depth --missing --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs

cat AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.idepth
INDV    N_SITES MEAN_DEPTH
POOL103_03RL    73251   22.4186
POOL105_03RL    73254   22.2204
POOL112_10RL    71708   7.28291
POOL116_10RL    73100   15.5343
POOL163_12RL    71776   6.90738
POOL36_11RL     73195   14.8737
POOL47_10RL     73238   17.4978
POOL50_12RL     73127   11.7687
POOL69_08RL     73211   17.0737
POOL87_11RL     73180   13.5165

cat AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.imiss
INDV    N_DATA  N_GENOTYPES_FILTERED    N_MISS  F_MISS
POOL103_03RL    73261   0       10      0.000136498
POOL105_03RL    73261   0       7       9.55488e-05
POOL112_10RL    73261   0       1553    0.0211982
POOL116_10RL    73261   0       161     0.00219762
POOL163_12RL    73261   0       1485    0.02027
POOL36_11RL     73261   0       66      0.000900889
POOL47_10RL     73261   0       23      0.000313946
POOL50_12RL     73261   0       134     0.00182908
POOL69_08RL     73261   0       50      0.000682491
POOL87_11RL     73261   0       81      0.00110564

So choosing:

POOL105_03RL
POOL36_11RL
POOL116_10RL

perl -e 'print "POOL103_03RL\nPOOL112_10RL\nPOOL163_12RL\nPOOL47_10RL\nPOOL50_12RL\nPOOL69_08RL\nPOOL87_11RL\n";' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome | sort -rg -k 9,9 | head -n 10000 | awk '{ print $1, "\n", $3 }' | perl -lane 'chomp(@F); print join("\t", @F);' | sort | uniq -c | sort -g -k 1,1 | awk '{ if ($1 >= 100 ) { print $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ2Countgte100.FIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome | sort -rg -k 8,8 | head -n 10000 | awk '{ print $1, "\n", $3 }' | perl -lane 'chomp(@F); print join("\t", @F);' | sort | uniq -c | sort -g -k 1,1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.Z1CountList

#NOTE -- dropping these individuals because they are starkly greater than anyone else in the list. Individuals from the 'high z2' count list are not being dropped because they appear as just to be the tail of a distribution versus being such a group of outliers
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome | sort -rg -k 8,8 | head -n 10000 | awk '{ print $1, "\n", $3 }' | perl -lane 'chomp(@F); print join("\t", @F);' | sort | uniq -c | sort -g -k 1,1 | awk '{ if ($1 >= 200) { print $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.FIDs

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs

cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ2Countgte100.FIDs | perl -lane 'push(@F, "Z2"); print join("\t", @F);') <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.FIDs | perl -lane 'push(@F, "Z1"); print join("\t", @F);') > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.OddHighZ1OrZ2.FID_ZNums


#EIGENSTRAT stuff here

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/VLookUp.Python.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/VLookUp.Python.DataHapMap3Overlap.vs1.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/VLookUp.Python.DataHapMap3Overlap.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.map --file2 /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.liftOver.hg18tohg19.map | awk '{ print $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.rsIDs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.rsIDs --recode --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3

plink --file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3 --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3

#Note -- there are a few SNPs that don't match up, about ~30? Just letting them drop since troubleshooting almost got most of them back but not all of them and yeah

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.map | awk '{ print $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.map.rsIDs 

plink --bfile /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.map.rsIDs --keep  /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.CEUCHBYRI.Unrelated.FIDIIDs --recode --out  /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Vs2

plink --file /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Vs2 --make-bed --out /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Vs2

paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.bim /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Vs2.bim | awk '{ if (($5 == $11) || ($5 == $12) ) { print $0 } } ' | awk '{ print $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.FinalListwDrops.rsIDs 

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3 --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.FinalListwDrops.rsIDs --recode --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.vs2

plink --bfile /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.FinalListwDrops.rsIDs --keep /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.CEUCHBYRI.Unrelated.FIDIIDs --recode --out /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Vs2.vs2

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.vs2.ped /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Vs2.vs2.ped > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.ped

plink --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.ped --map /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.vs2.map --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.phenoColEdit.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.phenoColEdit.pedind

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.bim /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.pedsnp

#Created EigenstratFiles in GATK and moved Eigenstrat/ files into EigenstratFiles/.

scp mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.parfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/.

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.parfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.wHapMap3Data.CEUCHBYRI.parfile

/home/michaelt/Software/EIG5.0.1/bin/smartpca -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.wHapMap3Data.CEUCHBYRI.parfile

#REDO WITH 4 INDVS MISSING FROM PHENO FILE
#python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt1
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca | cat - <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca | grep -E 'POOL43_10RL|POOL101_11RL|POOL18S_07RL|POOL2S_03RL' | perl -lane '$F[$#F] = "NoPhenotype"; print join("\t", @F); ') > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt1

#Added in the following four individuals with the tag 'NoPhenotype' at the end manually in the pt1 file
#POOL43_10RL POOL101_11RL POOL18S_07RL POOL2S_03RL

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.HapMap3vs.py --file1 /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.txt --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt2

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race

sed -i 's/American_Indian_or_Alaskan_Native/AIndian_or_Alaskan_Native/g' /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.1vs2.xtxt

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.3vs4.xtxt

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race -c 5:6 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.5vs6.xtxt

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.1vs2.pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/.

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.*.pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/.

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race | awk '{ if ((($2 >= 0) && ($2 <= .22)) && (($3 >= -.01) && ($3 <= .021))) { print $0 } } ' | grep White_non-Hispanic > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed -c 1:2 -p White_non-Hispanic -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.plot.1vs2.xtxt

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed | perl -lane 'my @vals1 = split(/:/, $F[0]); print $vals1[0];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed | perl -lane 'my @vals1 = split(/:/, $F[0]); print $vals1[0], "\t", $vals1[0];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteACed.plot.1vs2.pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/.

#Need to do this on the no-HapMap3 dataset

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race | grep White_non-Hispanic | awk 'BEGIN{ while(getline < "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.OddHighZ1OrZ2.FID_ZNums") { FID = $1":"$1; FIDList[FID] = $2; }} { if (FIDList[$1]) { if (FIDList[$1] == "Z1") { $12 = "Z1High"; } else if (FIDList[$1] == "Z2") { $12 = "Z2High"; } else{ print "blah" }  } else { $12 = "NormalIBD"; } print $0; } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.IBDissuesTagged

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.IBDissuesTagged -c 1:2 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.IBDissuesTagged.plot.1vs2.xtxt 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed | awk 'BEGIN{ while(getline < "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.OddHighZ1OrZ2.FID_ZNums") { FID = $1":"$1; FIDList[FID] = $2; }} { if (FIDList[$1]) { if (FIDList[$1] == "Z1") { $12 = "Z1High"; } else if (FIDList[$1] == "Z2") { $12 = "Z2High"; } else{ print "blah" }  } else { $12 = "NormalIBD"; } print $0; } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 1:2 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.1vs2.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 3:4 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.3vs4.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 5:6 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.5vs6.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 7:8 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.7vs9.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 9:10 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.9vs10.xtxt

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/*pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/.


cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.phenoColEdit.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.phenoColEdit.pedind

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.bim /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.pedsnp

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.wHapMap3Data.CEUCHBYRI.parfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.parfile

/home/michaelt/Software/EIG5.0.1/bin/smartpca -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.parfile

#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.Results.pca.altrd.race

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race | awk 'BEGIN{ while(getline < "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.OddHighZ1OrZ2.FID_ZNums") { FID = $1":"$1; FIDList[FID] = $2; }} { if (FIDList[$1]) { if (FIDList[$1] == "Z1") { $12 = "Z1High"; } else if (FIDList[$1] == "Z2") { $12 = "Z2High"; } else{ print "blah" }  } else { $12 = "NormalIBD"; } print $0; } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race.IBDissuesTagged

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race.plot.1vs2.xtxt 

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race.plot.3vs4.xtxt 

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race.IBDissuesTagged -c 1:2 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race.IBDissuesTagged.plot.1vs2.xtxt

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race.IBDissuesTagged -c 3:4 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race.IBDissuesTagged.plot.3vs4.xtxt

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.5PCs.covar
#Edit code in between runs
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.pheno

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar | awk '{ print $1, "\t", $2 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar | awk '{ print $1 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDs

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDs --non-ref-ac 1 --recode  --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.recode.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.allSNPs

plink --bfile AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD --freq --out AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.PLINK

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --freq --out  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs.maf05

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.10PCs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.10PCs.maf05

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 2 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs.maf05.pheno2

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 3 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs.maf05.pheno3

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 4 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs.maf05.pheno4


R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs.assoc.logistic Pheno1_justWhite_Logistic_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs.maf05.assoc.logistic Pheno1_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs.maf05.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.10PCs.assoc.logistic Pheno1_justWhite_Logistic_10PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.10PCs.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.10PCs.maf05.assoc.logistic Pheno1_justWhite_Logistic_maf05_10PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.10PCs.maf05.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs.maf05.pheno2.assoc.logistic Pheno2_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs.maf05.pheno2.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs.maf05.pheno3.assoc.logistic Pheno2_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs.maf05.pheno3.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs.maf05.pheno4.assoc.logistic Pheno2_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs.maf05.pheno4.assoc.logistic.QQplot.jpeg &

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/.

#Allele Frequency stuff/collection stuff here now

#Should drop individuals from IBD stuff here
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ print $1 }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs  > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($3 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroNeg.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($4 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($3 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroPos.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justRapidVRapid.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSlowVSlow.FIDs
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSlowVSlow.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justVeryRapid.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justVerySlow.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroPos.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justRapidVRapid.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSlowVSlow.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justVeryRapid.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justVerySlow.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow &

paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq.count | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.AllPhenos.frq.AndCount.gz

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq.count

#gunzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq.count


#Annovar stuff here now
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz | awk '{ print $1, "\t", $2, "\t", $2, "\t", $4, "\t", $5 }' | grep -v ^# > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat

perl /home/michaelt/Software/annovar/annotate_variation.pl --downdb -webfrom annovar ljb23_all /home/michaelt/Software/annovar/ -build hg19

perl /home/michaelt/Software/annovar/summarize_annovar.pl /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat /home/michaelt/Software/annovar/ --ver1000g 1000g2012apr --verdbsnp 129 --veresp 6500 --buildver hg19 --remove

cat AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat | head -n 1000 > AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.test3

cp -p AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.test3 AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.test3.cpy

perl /home/michaelt/Software/annovar/summarize_annovar.pl /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.test3 /home/michaelt/Software/annovar/ --ver1000g 1000g2012apr --verdbsnp 129 --veresp 6500 --buildver hg19 --remove

#Edit file /home/michaelt/Software/annovar/summarize_annovar.pl to go from ljb_all to ljb23_all -- then edit .genome_summary.csv file to fill in extra spots in non-exonic variants and to include the below header

perl /home/michaelt/Software/annovar/summarize_annovar.pl /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.test3.cpy --ver1000g 1000g2012apr /home/michaelt/Software/annovar/ --verdbsnp 129 --veresp 6500 --buildver hg19 --remove

#LJB23_SIFT_score LJB23_SIFT_score_converted LJB23_SIFT_pred LJB23_Polyphen2_HDIV_score LJB23_Polyphen2_HDIV_pred LJB23_Polyphen2_HVAR_score LJB23_Polyphen2_HVAR_pred LJB23_LRT_score LJB23_LRT_score_converted LJB23_LRT_pred LJB23_MutationTaster_score LJB23_MutationTaster_score_converted LJB23_MutationTaster_pred LJB23_MutationAssessor_score LJB23_MutationAssessor_score_converted LJB23_MutationAssessor_pred LJB23_FATHMM_score LJB23_FATHMM_score_converted LJB23_FATHMM_pred LJB23_RadialSVM_score LJB23_RadialSVM_score_converted LJB23_RadialSVM_pred LJB23_LR_score LJB23_LR_pred LJB23_GERP++ LJB23_PhyloP LJB23_SiPhy

perl /home/michaelt/Software/annovar/summarize_annovar.pl /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat /home/michaelt/Software/annovar/ --ver1000g 1000g2012apr --verdbsnp 129 --veresp 6500 --buildver hg19 --outfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.ljb23 --remove

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.ljb23.genome_summary.csv | perl -F, -lane 'if ($#F <= 40) { splice(@F, 17, 0, ("") x 16); } print join(",", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.ljb23.genome_summary.Reformatted.csv

#Figure this part out
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.ljb23.genome_summary.Reformatted.csv | perl -F, -lane 'splice(@F, #, 1); splice( @F, #, 1);.... ; print join(",", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.ljb23.genome_summary.Reformatted.lbjColumns.csv

#Realized splicing variants were being cut up into multiple transcripts when they should really be just one -- a splicing variant is shown how it affects multiple transcripts of the same gene, but I want multiple calls of the same variant if it affects multiple genes, not multiple transcripts of the same gene
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs4.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs5.py

#Then had to fix the below issue -- note lack of first splicing 'exon' in the AnnovarAnnotation.vs5 file vs the original genome_summary.csv file
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | grep 16314427
"splicing","AP1M1(NM_001130524:exon2:c.199+1G>A,NM_032493:exon2:c.199+1G>A)",,,"556;Name=lod=242",,,,,,,,,,,,,,,,,19,16314427,16314427,G,A,
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | grep 16314427
19,16314427_1,.,G,A,G,185.60,VQSRTrancheSNP99.00to99.90,AA=G;AC=1;AF=5.543e-04;AN=1804;BaseQRankSum=4.117;DP=16480;Dels=0.00;FS=23.866;HRun=0;HaplotypeScore=3.3100;InbreedingCoeff=-0.0025;MLEAC=1;MLEAF=5.543e-04;MQ=59.99;MQ0=0;MQRankSum=0.122;QD=13.26;ReadPosRankSum=-0.212;VQSLOD=-4.493e-01;culprit=FS,0.000646831,0.0014245,0,0,0,0,0,0,1,1,0,0,0,0,0,0,splicing,AP1M1;NM_032493:exon2:c.199+1G>A),,,,,,,,,,,,,,,19,16314427,16314427,G,A

#Also realized that for entries such as "UTR5;UTR3" and "Exonic;splicing", have gene IDs such as "Gene1;Gene2" and need to fix this somehow

#Troubleshooting
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | perl -F, -lane 'if (($F[1] =~ m/\;/) || ($F[0] =~ m/\;/)) { print join(",", @F); }' | vi -
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | perl -F, -lane 'if (($F[1] =~ m/\;/) || ($F[0] =~ m/\;/)) { print join(",", @F); } ' | awk -F, '{ print $1 }' | sort | uniq -c
   8 "UTR5;UTR3"
   7 "exonic;splicing"
 895 "upstream;downstream"
#Specifically going to deal with assigning UTR5;UTR3 as two separate genes and exonic;splicing as only 'geneID' in the second column entry versus 'geneID;geneID(NM_.....)'. For the latter case, variant frequency is the same regardless of whether exonic or splicing variant and for the same gene, so leaving 'exonic;splicing' will indicate that originally there is a geneID(NM_....) portion of the gene information I'll need to look back at, but for the Skat and EPACTS purposes it'll only be encoded as a single variant for that gene. Having multiple copies of a variant for UTR5;UTR3 shouldn't matter any more than when there are multiple genes in the second column (separated by ,) and I'm cutting them up into separate variant entries anyways
#Additionally, if the first column is 'UTR5;UTR3' or 'exonic;splicing' then there should not be multiple comma-delimited entries in the second column which was the main purpose of creating that breaking up for-loop in the DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs5.py file in the first-place
#Fixed, note the use of ^\"splicing\"$ in the if/elif/else clause involving splicing, UTR5;UTR3, exonic;splicing etc. in order to properly skip 'exonic;splicing' variants when "splicing" is being searched for first, hence the 'Happening4' and 'Happneing4b' calls at first to track where these variants were falling in the clause

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs5.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.AllPhenos.frq.AndCount.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 30; $i <= 40; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz

#Some QC Checks
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | grep -v \# | perl -lane 'print join("\t", @F[0..9]);' | wc
  73261  732610 25819140
 
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | wc
  74824   82642 33131311
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | grep _2 | wc
  1540    1591  683410
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | grep _3 | wc
  21      21    9129
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | grep _4 | wc
  2       2     957
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | grep _5 | wc
  0       0       0
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$R -q -e 'val1 = 1540 + 21 + 2; print (val1);'
> val1 = 1540 + 21 + 2; print (val1);
[1] 1563
>
>
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$R -q -e 'val1 = 1540 + 21 + 2; print (val1); val2 = 74824 - 1563; print (val2);'
> val1 = 1540 + 21 + 2; print (val1); val2 = 74824 - 1563; print (val2);
[1] 1563
[1] 73261
> 
> 

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD

cat AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.allSNPs.frq | perl -lane 'my @vals1 = split(/:/, $F[4]); my @vals2 = split(/:/, $F[5]); if ($vals1[1] < $vals2[1]) { if ($vals1[1] < .05) { print join("\t", @F); } } if ($vals1[1] > $vals2[1]) { if ($vals2[1] < .05 ) { print join(",", @F); } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.allSNPs.frq.maxMaf05

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs2.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs3.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs3.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, 'BEGIN{while(getline < "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.allSNPs.frq.maxMaf05" ) { ID1 = FID = $1"_"$2; IDList[ID1] = 1; } } { ID2 = $1"_"$4; if (IDList[ID2]) { print $0; }  } ' | perl -lane 'print join(",", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.mafMax05


cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing") || ($7 == "UTR5;UTR3")) { print $8, "\t", $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.All

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.mafMax05 | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing") || ($7 == "UTR5;UTR3")) { print $8, "\t", $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.All.mafMax05

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { print $8, "\t", $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.Exonic

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $8, "\t", $2 } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($12 == "P") || ($12 == "D") ) { print $8, "\t", $2 } } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_PD

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $8, "\t", $2 } } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_D




cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing") || ($7 == "UTR5;UTR3")) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.All

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $0 } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($12 == "P") || ($12 == "D") ) { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($13 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing")) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $0 } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($12 == "P") || ($12 == "D") ) { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.Sift_D
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($13 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.LRT_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.pheno | awk '{ print $1, "\t", $2, "\t0\t0\t0\t", $3, "\t", $4, "\t", $5, "\t", $6 } ' | sed 's/-9/NA/g' | perl -lane 'print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar | awk '{ print $4, "\t", $6, "\t", $7 "\t", $8, "\t", $9, "\t", $10, "\t", $11,  "\t", $12, "\t", $13, "\t", $14,  "\t", $15 }' | paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped.vs2

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped.vs2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped

#Add line #FAM_ID IND_ID FAT_ID MOT_ID SEX HIVPROG HIVPROGHE AIDS AIDSEXTR RMID PC1 PC2 PC3 PC4 PC5 PC6 PC7 PC8 PC9 PC10 to /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz | awk '{ if ($1 == "23") { $1 = "X"} print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip

/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz

#In /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles and /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles
mv All* Vs1/.

cp -p DataProcessing.Pipeline.Analysis.Skat.vs3.R DataProcessing.Pipeline.Analysis.Skat.vs4.R

#Troubleshooting
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.All | grep ADM5 | awk '{ print $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.All.ADM5.SNPIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.All | grep ADM5 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.All.ADM5

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.All | grep ADM5 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.All.ADM5
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz --snps /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.All.ADM5.SNPIDs --recode --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.ADM5

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz | awk 'BEGIN{ while(getline < "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.All.ADM5.SNPIDs") { SNPIDList[$1] = 1; } } { if ($1 ~ /^#/) { print $0 } else { SNPID = "Null"; if ($3 !~ /\./ ) { SNPID = $3; } else { SNPID = $1":"$2; } if (SNPIDList[SNPID]) { print $0; } } } ' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.ADM5.vcf.gz   

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.ADM5.vcf.gz --plink --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.ADM5

plink --file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.ADM5 --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.ADM5

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.ADM5.vcf.gz | awk '{ if ($1 == "23") { $1 = "X"} print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.ADM5.vcf.bgzip

/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.ADM5.vcf.bgzip
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.ADM5.vcf.bgzip.gz

mkdir TroubleshootingStuff

mv *ADM5* TroubleshootingStuff/.



##Actually going over and doing the QC stuff
##20141029
#QC stuff here

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetFastqQCStats.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetFastqQCStats.vs2.sh

#Edited /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output via :.,$ s/\/home/\/data\/userdata/g

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetFastqQCStats.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output

#Problem files that threw errors and were leftover
#cat: /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PreMerge/mapping_pool_1-46/POOL20-12RL/HU5Q1CK02.RL12.fastq.stats: No such file or directory
#cat: /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PreMerge/mapping_pool_91-97/POOL94-08RL/H4KITRJ02.RL8.fastq.stats: No such file or directory
#cat: /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PreMerge/mapping_pool_91-97/POOL94-08RL/H4KITRJ01.RL8.fastq.stats: No such file or directory
#cat: /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PreMerge/mapping_pool_91-97/POOL94-10RL/H4KITRJ02.RL10.fastq.stats: No such file or directory

#Removed empty lines in /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetFastqQCStats.vs2.20141029_182644.output with /0.00000.0000 0.0000 and dd in vi
#PreMerge HU5Q1CK02.RL12 0.00000.0000 0.0000               totalDropped(%):   minDropped(%):   UpperBoundThresh:  maxDropped(%):     
#PreMerge H4KITRJ02.RL8 0.00000.0000 0.0000               totalDropped(%):   minDropped(%):   UpperBoundThresh:  maxDropped(%):
#PreMerge H4KITRJ01.RL8 0.00000.0000 0.0000               totalDropped(%):   minDropped(%):   UpperBoundThresh:  maxDropped(%):
#PreMerge H4KITRJ02.RL10 0.00000.0000 0.0000               totalDropped(%):   minDropped(%):   UpperBoundThresh:  maxDropped(%):

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs3.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output.GetperRegionpostQCStats.vs3.20141029_140243.output | perl -lane 'my @array1; push(@array1, $F[1]); foreach my $entry (@F) { if ($entry !~ m/[a-zA-Z]/) { push (@array1, $entry); } } print join(",", @array1);' | awk 'BEGIN{ print "ID,TotalReadsPreQC,TotalReadsQ10,TotalReadsQ10Prcnt,TotalReadsF4,TotalReadsF4Prcnt,TotalReadsF256,TotalReadsF256Prcnt,TotalReadsPostQC,TotalReadsDropped,TotalReadsDroppedPrcnt,AverageReadLengthPreQC,AverageReadLengthPostQC,BedFileCoveragePostQC" } { print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output.GetperRegionpostQCStats.vs3.20141029_140243.output.reformatted

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs3.sh

#Edited /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs via :.,$ s/\/home/\/data\/userdata/g 

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs

#Total column numbers post formatting = 27
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.GetperFullBampostQCStats.vs3.20141029_150137.output | perl -lane 'my @array1; push(@array1, $F[1]); foreach my $entry (@F) { if ($entry !~ m/[a-zA-Z]/) { push (@array1, $entry); } } print join(",", @array1);' | awk 'BEGIN{ print "ID,NumberOfOriginalBamFiles,TotalReadsPreQC,TotalReadsPreQCForward,TotalReadsPreQCReverse,TotalReadsPreQCForward_Reverse,PreQCMAPQmean,PreQCMAPQsd,PreQCMean,PreQCSD,PreQCMax,PreQCMin,CoveragePreQC,TotalReadsPostRmdup,TotalReadsPostRmdupDropped,TotalReadsPostRmdupDroppedPercentage,TotalReadsPostQC,TotalReadsPostQCForward,TotalReadsPostQCReverse,TotalReadsPostQCForward_Reverse,PostQCMAPQmean,PostQCMAPQsd,PostQCMean,PostQCSD,PostQCMax,PostQCMin,CoveragePostQC" } { print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.GetperFullBampostQCStats.vs3.20141029_150137.output.reformatted

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs2.R

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetFastqQCStats.vs2.20141029_182644.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output.GetperRegionpostQCStats.vs3.20141029_140243.output.reformatted /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.GetperFullBampostQCStats.vs3.20141029_150137.output.reformatted

#[1] NA
#[1] 7.09003
#[1] 0.2671718
#[1] 0.9868955
#null device 
#          1 
#	  [1] 270303.1
#	  [1] 632.9364
#	  [1] 246.6539
#	  [1] 21.70513
#	  [1] 45288.87
#	  [1] 13.76149
#	  null device 
#	            1 

#Following up why there is a NA in the above output, do the following code in R on /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output.GetperRegionpostQCStats.vs3.20141029_140243.output.reformatted and get this:
#> Data2[is.na(Data2$TotalReadsDroppedPrcnt),]
#                 ID TotalReadsPreQC TotalReadsQ10 TotalReadsQ10Prcnt
#		 150  HU5Q1CK02.RL12               0             0                  0
#		 1791  H4KITRJ02.RL8               0             0                  0
#		 1793  H4KITRJ01.RL8               0             0                  0
#		 1799 H4KITRJ02.RL10               0             0                  0
#		      TotalReadsF4 TotalReadsF4Prcnt TotalReadsF256 TotalReadsF256Prcnt
#		      150             0                 0              0                   0
#		      1791            0                 0              0                   0
#		      1793            0                 0              0                   0
#		      1799            0                 0              0                   0
#		           TotalReadsPostQC TotalReadsDropped TotalReadsDroppedPrcnt
#			   150                NA                NA                     NA
#			   1791               NA                NA                     NA
#			   1793               NA                NA                     NA
#			   1799               NA                NA                     NA
#			        AverageReadLengthPreQC AverageReadLengthPostQC BedFileCoveragePostQC
#				150                      NA                      NA                    NA
#				1791                     NA                      NA                    NA
#				1793                     NA                      NA                    NA
#				1799                     NA                      NA                    NA

#So inserted the following line into /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs2.R:
#Data2 <- Data2[!is.na(Data2$TotalReadsDroppedPrcnt),]

#[1] 7.589332
#[1] 7.097796
#[1] 0.2674644
#[1] 0.9879765
#null device 
#          1 
#	  [1] 270303.1
#	  [1] 632.9364
#	  [1] 246.6539
#	  [1] 21.70513
#	  [1] 45288.87
#	  [1] 13.76149
#	  null device 
#	            1 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetFastqQCStats.vs2.20141029_182644.output | R -q -e "Data1 <- read.table(file('stdin'), header=F); sum(as.numeric(Data1[,4])); sum(as.numeric(Data1[,6]));"
> Data1 <- read.table(file('stdin'), header=F); sum(as.numeric(Data1[,4])); sum(as.numeric(Data1[,6]));
[1] 191375813326
[1] 314267723
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetFastqQCStats.vs2.20141029_182644.output | wc
   3652  105908  801691

#20141104 -- Including the following line into vs6 below: if (re.search('^\"splicing\"$', line3[0]) and re.search('\(NM_', line3[1])) or (re.search('ncRNA_splicing', line3[0]) and re.search('\(NR_', line3[1])):
#Actually fixed this in DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs5.FullVCFwAnnovar.vs1.py first but for chronological reasons edited DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.py and renamed DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs5.FullVCFwAnnovar.vs1.py to DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py
#This is not expected to change anything since it seems like only one ncRNA_splicing variant actually remains present in the QCed vcf file being used, so the rest are dropped. However, hypothetically if they were being included, they were not being properly handled like the other 'splicing' variants were (and this presumably would have shown up in the perl -F, -lane 'print $#F;' type code I've been running to check things, e.g. this scenario would have not been fixed and I would have noticed it sooner (vs. the scenario just not being present at all and just realizing it in retrospect by looking at the code)
cp -p DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs5.py DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.py

cp -p DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.py DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py

[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | perl -F, -lane 'my @vals2 = split (/_/, $F[1]); print $vals2[1];' | sort | uniq -c
 73261 	1
  1540 	2
    21 	3
     2 	4
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | perl -F, -lane 'if ($F[1] =~ /_4/) { print join(",", @F); } '                                                                                                                                                                                                                  
16,2012009_4,rs116746316,G,A,a,5572.55,VQSRTrancheSNP90.00to99.00,AA=a;AC=66;AF=0.037;AN=1794;BaseQRankSum=18.449;DB;DP=7312;Dels=0.00;FS=1.328;HRun=0;HaplotypeScore=1.9092;InbreedingCoeff=0.0386;MLEAC=61;MLEAF=0.034;MQ=59.99;MQ0=0;MQRankSum=0.412;POSITIVE_TRAIN_SITE;QD=10.70;ReadPosRankSum=2.293;VQSLOD=2.40;culprit=DP,0.964239,0.960114,0.955357,0.967703,0.974138,0.970588,0.978261,0.986486,1483,674,321,809,113,165,45,73,downstream,SNORA64,,,,,,,,,,,,,,,,16,2012009,2012009,G,A                                                                                                                                                      
16,2012027_4,rs114278396,C,T,c,6507.78,VQSRTrancheSNP90.00to99.00,AA=c;AC=66;AF=0.037;AN=1796;BaseQRankSum=10.292;DB;DP=7458;Dels=0.00;FS=0.714;HRun=0;HaplotypeScore=1.4525;InbreedingCoeff=0.0565;MLEAC=63;MLEAF=0.035;MQ=59.98;MQ0=0;MQRankSum=-1.085;POSITIVE_TRAIN_SITE;QD=12.59;ReadPosRankSum=2.717;VQSLOD=2.14;culprit=DP,0.0357143,0.039886,0.0446429,0.0322196,0.0258621,0.0348837,0.0217391,0.0131579,55,28,15,27,3,6,1,1,downstream,SNORA64,,,,,,,,,,,,,,,,16,2012027,2012027,C,T                                                                                                                                                         

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.AllGenesPerVariant.vcf.gz

[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.AllGenesPerVariant.vcf.gz | perl -lane 'my @vals2 = split(/,/, $F[8]); print $#vals2, "\t", $F[8];' | awk '{ print $1; }' | sort | uniq -c                                                                                                                                                               100 -1                                                                                                                                                                                                                                                                                                                      28 0
  74824 25                                                                                                                                                                                                                                                                                                                 

[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$R -q -e "val1 <- 73261 + 1540 + 21 + 2; print(val1);"
> val1 <- 73261 + 1540 + 21 + 2; print(val1);                                                                                                                                                                                                                                                                              
[1] 74824
>                                                                                                                                                                                                                                                                                                                          
>

#Edited code in /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGenePerVariant.vcf.gz

[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals2 = split(/,/, $F[8]); print $#vals2, "\t", $F[8];' | awk '{ print $1; }' | sort | uniq -c
   100 	-1                                                                                                                                                                                                                                                                                                                 
    28 	0
 73261 	25

#/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.GetVCFQCStats.vs1.output too long a file name -- consider changing wHapMap3Data to wHM3Data across board later at some point....maybe?

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.vcf.gz

[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz | wc
  64015 50216707 1178830267

[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.vcf.gz | perl -lane 'my @vals2 = split(/,/, $F[8]); print $#vals2, "\t", $F[8];' | awk '{ print $1; }' | sort | uniq -c
   100 	-1                                                                                                                                                                                                                                                                                                                 
    28 	0
 65232 	25                                                                                                                                                                                                                                                                                                                 

#Edited code in /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGenePerVariant.vcf.gz

[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals2 = split(/,/, $F[8]); print $#vals2, "\t", $F[8];' | awk '{ print $1; }' | sort | uniq -c
   100 	-1                                                                                                                                                                                                                                                                                                                 
    28 	0
 63887 	25                                                                                                                                                                                                                                                                                                                 

[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$R -q -e "Val1 <- 64015 - 100 - 28; print(Val1)"
> Val1 <- 64015 - 100 - 28; print(Val1)
[1] 63887
> 
> 


#NOTE -- REDO LRT runs and ...QCed.DropIBD.bim.wGeneIDs files with correct annovar annotation stuff?
#Potentially using PP2_HVAR in place of LRT since PP2_HVAR is 2 places after PP2_HDIV (and I should be using HDIV over HVAR for my own purposes....but LRT may in fact be PP2_HVAR?)
#The ...line3[len(line3)-19:len(line3)+1]]))) line in /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.py is wrong

[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | grep nonsyn | perl -F, -lane 'print join(",", @F[$#F-25..$#F]);' | head -n 10
B,0.070,0.965,N,0.0,1,P,-2.24,0.287,N,-0.39,0.412,T,-0.957,0.261,T,0.000,T,4.28,0.578,7.695,1,888659,888659,T,C
B,0.414,0.793,N,0.0,1,P,-0.55,0.434,N,0.99,0.360,T,-1.043,0.240,T,0.000,T,2.45,0.456,2.650,1,909238,909238,G,C
B,0.001,3.155E-4,N,0.000,1.000,P,1.265,0.591,L,1.59,0.338,T,-0.903,0.275,T,0.000,T,-2.36,-0.675,9.972,1,949608,949608,G,A
P,0.000,1.000,D,0.792,0.208,N,0.345,0.511,N,3.03,0.284,T,-1.017,0.247,T,0.069,T,3.24,0.535,10.397,1,1163951,1163951,G,A
B,0.000,1.000,D,0.997,0.997,D,0.895,0.559,L,3.01,0.285,T,-0.974,0.257,T,0.009,T,2.21,0.158,6.654,1,1164015,1164015,C,A
B,0.061,0.970,N,1,0.0,N,-1.61,0.342,N,3.29,0.275,T,-0.986,0.254,T,0.000,T,-1.83,-0.628,7.783,1,1164031,1164031,A,G
B,0.004,0.998,N,1.000,0.000,N,1.445,0.607,L,3.16,0.279,T,-1.013,0.247,T,0.017,T,-1.7,-0.587,6.489,1,1164080,1164080,T,C
B,0.117,0.059,N,0.0,1,P,-0.2,0.464,N,-0.34,0.410,T,-0.961,0.261,T,0.000,T,1.41,0.096,10.110,1,1223385,1223385,G,C
B,.,.,.,0.0,1,P,0.55,0.529,N,-0.31,0.409,T,-1.104,0.225,T,0.001,T,1.92,0.219,5.198,1,1226757,1226757,G,A
D,0.128,0.064,N,0.872,0.872,D,4.18,0.844,H,0.23,0.389,T,-0.318,0.421,T,0.592,D,3.26,1.850,12.166,1,1246319,1246319,G,A

#NOTE -- Believe the above problems/NOTE/REDO was based on /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv being done with ljb23 but having the output name as normal. A new version of this file (and associated Annovar output) existed around June 6th when all the result files used, such as ...QCed.DropIBD.bim.wGeneIDs were from April 9th (when traces of another version of ...wAA.Bi.vcf.AnnovarFormat.genome_summary.csv is present too). Also, around June 6th is when P2 was being processed, therefore, I believe the ...wAA.Bi.vcf.AnnovarFormat.genome_summary.csv file was remade during this time period while figuring out P2 stuff, and when I jumped back into things /now/ I used the same filename even though the output now had the ljb23 version of things. 
#In conclusion, the EPACTS results and previous files should be fine. The only thing that's been affected is the prediction annotations sent to Kevin and this QC stuff I've started, which I've gone ahead and redone after rerunning the Annovar code without ljb23

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.AllGenesPerVariant.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.AllGenesPerVariant.GetVCFQCStats.vs1.output 

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGenePerVariant.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output 

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.GetVCFQCStats.vs1.output

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGenePerVariant.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGenePerVariant.vcf.gz | grep exonic | grep -v ncRNA | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGenePerVariant.Exonic.GetVCFQCStats.vs1.output 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGenePerVariant.vcf.gz | grep exonic | grep -v ncRNA | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGenePerVariant.Exonic.GetVCFQCStats.vs1.output

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.PerGene.vs1.py

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGenePerVariant.vcf.gz | grep exonic | grep -v ncRNA | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.PerGene.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGene.Exonic.GetVCFQCStats.PerGene.vs1.output 

#Added header to /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGene.Exonic.GetVCFQCStats.PerGene.vs1.output
#geneID,NumVar,Exonic,ExonicProp,Intronic,IntronicProp,Intergenic,UTR5,UTR3,Nonsynonymous,NonsynonymousPropEx,Synonymous,SynonymousPropEx,Stopgain,StopgainPropEx,Stoploss,StoplossPropEx,MeanDP,stdDP,meanQual,stdQual,TiTvRatio,Transitions,Transversions,meanHaplo,stdHaplo,Singletons,SingletonsProp,Doubletons,DoubletonsProp,Common,CommonProp,Rare,RareProp,PP2_D,PP2_DProp,PP2_P,PP2_PProp,Sift_D,Sift_DProp,LRT_D,LRT_DProp

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGenePerVariant.vcf.gz | grep exonic | grep -v ncRNA | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.PerGene.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.GetVCFQCStats.PerGene.vs1.output

#Added header to /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.GetVCFQCStats.PerGene.vs1.output
#geneID,NumVar,Exonic,ExonicProp,Intronic,IntronicProp,Intergenic,UTR5,UTR3,Nonsynonymous,NonsynonymousPropEx,Synonymous,SynonymousPropEx,Stopgain,StopgainPropEx,Stoploss,StoplossPropEx,MeanDP,stdDP,meanQual,stdQual,TiTvRatio,Transitions,Transversions,meanHaplo,stdHaplo,Singletons,SingletonsProp,Doubletons,DoubletonsProp,Common,CommonProp,Rare,RareProp,PP2_D,PP2_DProp,PP2_P,PP2_PProp,Sift_D,Sift_DProp,LRT_D,LRT_DProp

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.GetVCFQCStats.PerGene.vs1.output | awk -F, '{ print $1, "\t", $18, "\t", $2 }' | sort -gk 3,3 | vi -

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.GetVCFQCStats.PerGene.vs1.output | awk -F, '{ print $1, "\t", $2, "\t", $3, "\t", $10, "\t", $11, "\t", $27, "\t", $29, "\t", $31, "\t", $33, "\t", $18, "\t", $20 }' | sort -rg -k 9,9 | vi -

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs2.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGenePerVariant.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGene.perIndv.vs2.output

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGenePerVariant.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.perIndv.vs2.output

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perGene.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perGene.vs1.py.OLD1 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGenePerVariant.vcf.gz | grep -E '^#|exonic' | grep -v ncRNA | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs2.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGenePerVariant.vcf.gz | grep -E '^#|exonic' | grep -v ncRNA | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs2.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.perIndv.vs2.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.perIndv.vs2.output.justSeroNeg 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.perIndv.vs2.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.perIndv.vs2.output.justSeroHighlyExposed
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.perIndv.vs2.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroPos.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.perIndv.vs2.output.justSeroPos 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.perIndv.vs2.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justRapidVRapid.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.perIndv.vs2.output.justRapidVRapid 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.perIndv.vs2.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSlowVSlow.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.perIndv.vs2.output.justSlowVSlow 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output.justSeroNeg 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output.justSeroHighlyExposed
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroPos.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output.justSeroPos 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justRapidVRapid.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output.justRapidVRapid 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSlowVSlow.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output.justSlowVSlow 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.perIndv.vs2.output.justSeroNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 14.1133144475921 MeanDoubletons: 18.8101983002833 MeanCommon: 4446.35694050992 MeanRare: 425.130311614731 MeanHeterozy: 3053.45325779037 MeanCoverage: 16.107917375798 sdCoverage: 6.19823376737347 Exonic: 334.082152974504 Intronic: 3569.48441926346 Nonsynonymous: 122.872521246459 Synonymous: 208.801699716714 PP2_D: 9.90934844192635 PP2_P: 12.6345609065156 Sift_D: 23.0509915014164 LRT_D: 21.6345609065156"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.perIndv.vs2.output.justSeroHighlyExposed | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 14.2678571428571 MeanDoubletons: 18.9047619047619 MeanCommon: 4446.85119047619 MeanRare: 424.309523809524 MeanHeterozy: 3061.64880952381 MeanCoverage: 16.0151247331096 sdCoverage: 6.35219718685658 Exonic: 334.345238095238 Intronic: 3568.58333333333 Nonsynonymous: 122.714285714286 Synonymous: 209.404761904762 PP2_D: 10.0654761904762 PP2_P: 12.6071428571429 Sift_D: 23.0357142857143 LRT_D: 21.7261904761905"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.perIndv.vs2.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 15.0707547169811 MeanDoubletons: 17.8844339622642 MeanCommon: 4463.65094339623 MeanRare: 419.983490566038 MeanHeterozy: 3078.24528301887 MeanCoverage: 17.0877961848545 sdCoverage: 6.56096341408423 Exonic: 332.719339622641 Intronic: 3579.04716981132 Nonsynonymous: 122.200471698113 Synonymous: 207.785377358491 PP2_D: 9.78301886792453 PP2_P: 12.6768867924528 Sift_D: 22.372641509434 LRT_D: 21.8915094339623"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.perIndv.vs2.output.justRapidVRapid | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))                                                                                  
[1] "MeanSingletons: 15.0172413793103 MeanDoubletons: 15.4655172413793 MeanCommon: 4472.20689655172 MeanRare: 393.655172413793 MeanHeterozy: 3062.53448275862 MeanCoverage: 21.8443147903407 sdCoverage: 8.46501233260818 Exonic: 330.344827586207 Intronic: 3573.39655172414 Nonsynonymous: 121.275862068966 Synonymous: 206.396551724138 PP2_D: 9.39655172413793 PP2_P: 12.6724137931034 Sift_D: 22.0689655172414 LRT_D: 21.3620689655172"                                                                                                                                                                                                          
>
>                                   

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.perIndv.vs2.output.justSlowVSlow | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 15.4157303370787 MeanDoubletons: 18.1685393258427 MeanCommon: 4474 MeanRare: 423.707865168539 MeanHeterozy: 3095.23595505618 MeanCoverage: 17.0228952824631 sdCoverage: 6.77461609846057 Exonic: 333.213483146067 Intronic: 3593.01123595506 Nonsynonymous: 122.52808988764 Synonymous: 207.786516853933 PP2_D: 9.97752808988764 PP2_P: 12.7415730337079 Sift_D: 22.8539325842697 LRT_D: 22.0898876404494"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output.justSeroNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.43909348441926 MeanDoubletons: 2.86685552407932 MeanCommon: 299.28045325779 MeanRare: 35.2549575070822 MeanHeterozy: 219.773371104816 MeanCoverage: 21.9810675566897 sdCoverage: 8.58968008315038 Exonic: 334.082152974504 Intronic: 0 Nonsynonymous: 122.872521246459 Synonymous: 208.801699716714 PP2_D: 9.90934844192635 PP2_P: 12.6345609065156 Sift_D: 22.9206798866856 LRT_D: 21.6345609065156"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output.justSeroHighlyExposed | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.52380952380952 MeanDoubletons: 2.9047619047619 MeanCommon: 298.940476190476 MeanRare: 35.8452380952381 MeanHeterozy: 220.821428571429 MeanCoverage: 21.7223920798614 sdCoverage: 8.74336821131015 Exonic: 334.345238095238 Intronic: 0 Nonsynonymous: 122.714285714286 Synonymous: 209.404761904762 PP2_D: 10.0654761904762 PP2_P: 12.6071428571429 Sift_D: 22.8988095238095 LRT_D: 21.7261904761905"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.53537735849057 MeanDoubletons: 2.54009433962264 MeanCommon: 298.695754716981 MeanRare: 34.4528301886792 MeanHeterozy: 219.007075471698 MeanCoverage: 22.9559819110576 sdCoverage: 9.25012765969131 Exonic: 332.719339622641 Intronic: 0 Nonsynonymous: 122.200471698113 Synonymous: 207.785377358491 PP2_D: 9.78301886792453 PP2_P: 12.6768867924528 Sift_D: 22.2452830188679 LRT_D: 21.8915094339623"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output.justRapidVRapid | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.48275862068966 MeanDoubletons: 2.55172413793103 MeanCommon: 297.724137931034 MeanRare: 32.948275862069 MeanHeterozy: 216.293103448276 MeanCoverage: 29.945459392004 sdCoverage: 12.4184488407268 Exonic: 330.344827586207 Intronic: 0 Nonsynonymous: 121.275862068966 Synonymous: 206.396551724138 PP2_D: 9.39655172413793 PP2_P: 12.6724137931034 Sift_D: 21.9310344827586 LRT_D: 21.3620689655172"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output.justSlowVSlow | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 3.01123595505618 MeanDoubletons: 2.52808988764045 MeanCommon: 298.685393258427 MeanRare: 34.9887640449438 MeanHeterozy: 220.247191011236 MeanCoverage: 22.8268577959015 sdCoverage: 9.21476492286574 Exonic: 333.213483146067 Intronic: 0 Nonsynonymous: 122.52808988764 Synonymous: 207.786516853933 PP2_D: 9.97752808988764 PP2_P: 12.7415730337079 Sift_D: 22.685393258427 LRT_D: 22.0898876404494"
> 
> 

#20150108
#Need to look over and consider whether I need to redo these per individual/per group QC analyses because I wasn't shifting the Annovar annotated .vcf file header to include 'ANNOVAR' for the new, extra column, thus leading to individual ID names being off 

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz | perl -lane 'my @vals2 = split(/,/, $F[8]); print $#vals2, "\t", $F[8];' | awk '{ print $1; }' | sort | uniq -c

[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz | perl -lane 'my @vals2 = split(/,/, $F[8]); print $#vals2, "\t", $F[8];' | awk '{ print $1; }' | sort | uniq -c
    100 -1
    28 0
    74824 25

[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$R -q -e "val1 <- 73261 + 1540 + 21 + 2; print(val1);"
> val1 <- 73261 + 1540 + 21 + 2; print(val1);
[1] 74824
>
>

#Edited code in /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs2.py
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz | perl -lane 'my @vals2 = split(/,/, $F[8]); print $#vals2, "\t", $F[8];' | awk '{ print $1; }' | sort | uniq -c
 
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz | perl -lane 'my @vals2 = split(/,/, $F[8]); print $#vals2, "\t", $F[8];' | awk '{ print $1; }' | sort | uniq -c 
    100 -1
         28 0
	   73261 25

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz

[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz | wc
  64015 50216707 1178830267

[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz | wc  
  65360 51339110 1208205185

[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz | perl -lane 'my @vals2 = split(/,/, $F[8]); print $#vals2, "\t", $F[8];' | awk '{ print $1; }' | sort | uniq -c                                                                       
    100 -1
    28 0 
    65232 25

#Edited code in /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs2.py
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz | perl -lane 'my @vals2 = split(/,/, $F[8]); print $#vals2, "\t", $F[8];' | awk '{ print $1; }' | sort | uniq -c

[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz | perl -lane 'my @vals2 = split(/,/, $F[8]); print $#vals2, "\t", $F[8];' | awk '{ print $1; }' | sort | uniq -c
    100 -1
    28 0
    63887 25

[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$R -q -e "Val1 <- 64015 - 100 - 28; print(Val1)"
> Val1 <- 64015 - 100 - 28; print(Val1)
[1] 63887
> 

#Editing and removing +1 to 'RowLength' for '^#CHROM' block because that header now has the proper number of columns in it and doesn't need to be offset by one manually
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs2.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs3.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs3.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs3.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz | grep -E '^#|exonic' | grep -v ncRNA | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs3.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz | grep -E '^#|exonic' | grep -v ncRNA | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs3.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output.justSeroNeg 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output.justSeroHighlyExposedNeg 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroPos.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output.justSeroPos 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justRapidVRapid.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output.justRapidVRapid 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSlowVSlow.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output.justSlowVSlow 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output.justSeroNeg
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output.justHESN
#Filename too long hence HESN
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroPos.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output.justSeroPos
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justRapidVRapid.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output.justRapidVRapid
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSlowVSlow.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output.justSlowVSlow

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output.justSeroNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))                                                                                  
[1] "MeanSingletons: 14.1133144475921 MeanDoubletons: 18.8101983002833 MeanCommon: 4446.35694050992 MeanRare: 425.130311614731 MeanHeterozy: 3053.45325779037 MeanCoverage: 16.107917375798 sdCoverage: 6.19823376737347 Exonic: 334.082152974504 Intronic: 3569.48441926346 Nonsynonymous: 122.872521246459 Synonymous: 208.801699716714 PP2_D: 9.90934844192635 PP2_P: 12.6345609065156 Sift_D: 23.0509915014164 LRT_D: 21.6345609065156"                                                                                                                                                                                                           
>
>                                                                                                                                                                                                                                                                                                                          

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 15.0707547169811 MeanDoubletons: 17.8844339622642 MeanCommon: 4463.65094339623 MeanRare: 419.983490566038 MeanHeterozy: 3078.24528301887 MeanCoverage: 17.0877961848545 sdCoverage: 6.56096341408423 Exonic: 332.719339622641 Intronic: 3579.04716981132 Nonsynonymous: 122.200471698113 Synonymous: 207.785377358491 PP2_D: 9.78301886792453 PP2_P: 12.6768867924528 Sift_D: 22.372641509434 LRT_D: 21.8915094339623"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output.justSeroHighlyExposedNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 14.2678571428571 MeanDoubletons: 18.9047619047619 MeanCommon: 4446.85119047619 MeanRare: 424.309523809524 MeanHeterozy: 3061.64880952381 MeanCoverage: 16.0151247331096 sdCoverage: 6.35219718685658 Exonic: 334.345238095238 Intronic: 3568.58333333333 Nonsynonymous: 122.714285714286 Synonymous: 209.404761904762 PP2_D: 10.0654761904762 PP2_P: 12.6071428571429 Sift_D: 23.0357142857143 LRT_D: 21.7261904761905"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 15.0707547169811 MeanDoubletons: 17.8844339622642 MeanCommon: 4463.65094339623 MeanRare: 419.983490566038 MeanHeterozy: 3078.24528301887 MeanCoverage: 17.0877961848545 sdCoverage: 6.56096341408423 Exonic: 332.719339622641 Intronic: 3579.04716981132 Nonsynonymous: 122.200471698113 Synonymous: 207.785377358491 PP2_D: 9.78301886792453 PP2_P: 12.6768867924528 Sift_D: 22.372641509434 LRT_D: 21.8915094339623"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output.justRapidVRapid | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 15.0172413793103 MeanDoubletons: 15.4655172413793 MeanCommon: 4472.20689655172 MeanRare: 393.655172413793 MeanHeterozy: 3062.53448275862 MeanCoverage: 21.8443147903407 sdCoverage: 8.46501233260818 Exonic: 330.344827586207 Intronic: 3573.39655172414 Nonsynonymous: 121.275862068966 Synonymous: 206.396551724138 PP2_D: 9.39655172413793 PP2_P: 12.6724137931034 Sift_D: 22.0689655172414 LRT_D: 21.3620689655172"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output.justSlowVSlow | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 15.4157303370787 MeanDoubletons: 18.1685393258427 MeanCommon: 4474 MeanRare: 423.707865168539 MeanHeterozy: 3095.23595505618 MeanCoverage: 17.0228952824631 sdCoverage: 6.77461609846057 Exonic: 333.213483146067 Intronic: 3593.01123595506 Nonsynonymous: 122.52808988764 Synonymous: 207.786516853933 PP2_D: 9.97752808988764 PP2_P: 12.7415730337079 Sift_D: 22.8539325842697 LRT_D: 22.0898876404494"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output.justSeroNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.43909348441926 MeanDoubletons: 2.86685552407932 MeanCommon: 299.28045325779 MeanRare: 35.2549575070822 MeanHeterozy: 219.773371104816 MeanCoverage: 21.9810675566897 sdCoverage: 8.58968008315038 Exonic: 334.082152974504 Intronic: 0 Nonsynonymous: 122.872521246459 Synonymous: 208.801699716714 PP2_D: 9.90934844192635 PP2_P: 12.6345609065156 Sift_D: 22.9206798866856 LRT_D: 21.6345609065156"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.53537735849057 MeanDoubletons: 2.54009433962264 MeanCommon: 298.695754716981 MeanRare: 34.4528301886792 MeanHeterozy: 219.007075471698 MeanCoverage: 22.9559819110576 sdCoverage: 9.25012765969131 Exonic: 332.719339622641 Intronic: 0 Nonsynonymous: 122.200471698113 Synonymous: 207.785377358491 PP2_D: 9.78301886792453 PP2_P: 12.6768867924528 Sift_D: 22.2452830188679 LRT_D: 21.8915094339623"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output.justHESN | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.52380952380952 MeanDoubletons: 2.9047619047619 MeanCommon: 298.940476190476 MeanRare: 35.8452380952381 MeanHeterozy: 220.821428571429 MeanCoverage: 21.7223920798614 sdCoverage: 8.74336821131015 Exonic: 334.345238095238 Intronic: 0 Nonsynonymous: 122.714285714286 Synonymous: 209.404761904762 PP2_D: 10.0654761904762 PP2_P: 12.6071428571429 Sift_D: 22.8988095238095 LRT_D: 21.7261904761905"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.53537735849057 MeanDoubletons: 2.54009433962264 MeanCommon: 298.695754716981 MeanRare: 34.4528301886792 MeanHeterozy: 219.007075471698 MeanCoverage: 22.9559819110576 sdCoverage: 9.25012765969131 Exonic: 332.719339622641 Intronic: 0 Nonsynonymous: 122.200471698113 Synonymous: 207.785377358491 PP2_D: 9.78301886792453 PP2_P: 12.6768867924528 Sift_D: 22.2452830188679 LRT_D: 21.8915094339623"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output.justRapidVRapid | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.48275862068966 MeanDoubletons: 2.55172413793103 MeanCommon: 297.724137931034 MeanRare: 32.948275862069 MeanHeterozy: 216.293103448276 MeanCoverage: 29.945459392004 sdCoverage: 12.4184488407268 Exonic: 330.344827586207 Intronic: 0 Nonsynonymous: 121.275862068966 Synonymous: 206.396551724138 PP2_D: 9.39655172413793 PP2_P: 12.6724137931034 Sift_D: 21.9310344827586 LRT_D: 21.3620689655172"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output.justSlowVSlow | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 3.01123595505618 MeanDoubletons: 2.52808988764045 MeanCommon: 298.685393258427 MeanRare: 34.9887640449438 MeanHeterozy: 220.247191011236 MeanCoverage: 22.8268577959015 sdCoverage: 9.21476492286574 Exonic: 333.213483146067 Intronic: 0 Nonsynonymous: 122.52808988764 Synonymous: 207.786516853933 PP2_D: 9.97752808988764 PP2_P: 12.7415730337079 Sift_D: 22.685393258427 LRT_D: 22.0898876404494"
> 
> 


#20141111
#PC3 check thing, doing a couple of runs of looking at PCs with different color codings for different covariates to see if any of the top 1-5 PCs align with any of the underlying covariates/phenotype labels

vi /home/michaelt/Scripts/R/BasicScript.vs1.R

cp -p /home/michaelt/Scripts/R/BasicScript.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs1.R 

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs1.output.jpeg

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs1.output.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/.

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.parfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.withSNPWeights.parfile

#Adding in output file for 'snpweightoutname:' option and rerunning, but don't want to clobber previous files
#Also used the following command to do so :.,$ s/\/home\/pg\//\/data\/userdata\/pg\//g at the top of this file

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.bim /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.pedsnp

/home/michaelt/Software/EIG5.0.1/bin/smartpca -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.withSNPWeights.parfile

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.AddAnnovarInfoToEigenstratSNPWeights.vs1.py 

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.AddAnnovarInfoToEigenstratSNPWeights.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.withSNPWeights.snpWghts > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.withSNPWeights.snpWghts.wAnnovarAnnotation

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.AddUpAlleleCountsPerIndvGivenChrBPAndVCF.vs1.py

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.withSNPWeights.snpWghts.wAnnovarAnnotation | grep UBR4 | awk '{ print $2, "\t", $3 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.withSNPWeights.snpWghts.wAnnovarAnnotation.UBR4.ChrBP

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.AddUpAlleleCountsPerIndvGivenChrBPAndVCF.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.withSNPWeights.snpWghts.wAnnovarAnnotation.UBR4.ChrBP --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.UBR4.AlleleCounts

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.R

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.UBR4.AlleleCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.output.pt1.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.output.pt2.jpeg

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.output.pt*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/.

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.withSNPWeights.snpWghts.wAnnovarAnnotation | sort -rg -k 6,6 | awk '{ if (($6 > 8) || ($6 < -6.5)) { print $0 } } ' | grep UBR4 | awk '{ print $2, "\t", $3 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.withSNPWeights.snpWghts.wAnnovarAnnotation.UBR4.TopSNPs.ChrBP

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.AddUpAlleleCountsPerIndvGivenChrBPAndVCF.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.withSNPWeights.snpWghts.wAnnovarAnnotation.UBR4.TopSNPs.ChrBP --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.UBR4.TopSNPs.AlleleCounts

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.UBR4.TopSNPs.AlleleCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.output.pt1.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.output.pt2.vs2.jpeg

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.output.pt*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/.

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.AddUpAlleleCountsPerIndvGivenChrBPAndVCF.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.ConvertGenosToACSumPerIndvGivenChrBPAndVCF.vs1.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.ConvertGenosToACSumPerIndvGivenChrBPAndVCF.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.withSNPWeights.snpWghts.wAnnovarAnnotation.UBR4.ChrBP --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.UBR4.perIndvSumAC

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.ConvertGenosToACSumPerIndvGivenChrBPAndVCF.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.withSNPWeights.snpWghts.wAnnovarAnnotation.UBR4.TopSNPs.ChrBP --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.UBR4.TopSNPs.perIndvSumAC

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.UBR4.TopSNPs.AlleleCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.UBR4.TopSNPs.perIndvSumAC /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.output.pt1.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.output.pt2.vs2.jpeg

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.output.pt*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/.

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.ConvertGenosToACSumPerIndvGivenChrBPAndVCF.vs1.py --file1 All --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.perIndvSumAC

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs1.R

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.perIndvSumAC | sort -rg -k 2,2 | perl -lane 'my @vals1 = split(/_/, $F[0]); print join("\t", @vals1), "\t", $F[1]; ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs1.output

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.AddAnnovarInfoToEigenstratSNPWeights.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.AddAnnovarInfoToIndvSumAC.vs1.py 

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.AddAnnovarInfoToIndvSumAC.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs1.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs1.output.wAnnovarAnnotation 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs1.output.wAnnovarAnnotation | awk '{ if ($3 > .9) { print $0 } } ' | grep HEATR1 | awk '{ print $1, "\t", $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs1.output.wAnnovarAnnotation.HEATR1.TopSNPs.ChrBP

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.ConvertGenosToACSumPerIndvGivenChrBPAndVCF.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs1.output.wAnnovarAnnotation.HEATR1.TopSNPs.ChrBP --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.HEATR1.TopSNPs.perIndvSumAC

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.UBR4.TopSNPs.AlleleCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.HEATR1.TopSNPs.perIndvSumAC /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.output.pt1.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.output.pt2.vs2.HEATR1.jpeg

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.output.pt*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/.

#HEATR1 is driving the PC3 results (and UBR4 is driving the PC4 results....which is interesting enough since UBR4 SNPs came out as 'top SNPs' based on SNP-PC eigenvalues whereas HEATR1 came out by directingly looking at correlations between genotypes acorss individuals and PC3 results). Interested in redoing Eigenstrat results with downsampled white non-hispanices to see if get same PC3 stratification, just to identify whether it's the abundance of this ancestry that is producing this ancestry-specific result

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.wHapMap3Data.CEUCHBYRI.parfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.wHapMap3Data.CEUCHBYRI.WhiteNonHispSbsmpl.parfile
#Used the following command to do so :.,$ s/\/home\/pg\//\/data\/userdata\/pg\//g at the top of this file

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race | grep White_non-Hispanic | awk '{ print $1 }' | shuf | perl -lane 'my @vals1 = split(/:/, $F[0]); print $vals1[0];' | head -n 729 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.Ran729White.justFIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.phenoColEdit.ped | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.Ran729White.justFIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.phenoColEdit.WhiteNonHispSbsmpl.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.phenoColEdit.pedind | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.Ran729White.justFIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.phenoColEdit.WhiteNonHispSbsmpl.pedind

rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.pedsnp

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.bim /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.pedsnp

/home/michaelt/Software/EIG5.0.1/bin/smartpca -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.wHapMap3Data.CEUCHBYRI.WhiteNonHispSbsmpl.parfile

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca | cat - <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca | grep -E 'POOL43_10RL|POOL101_11RL|POOL18S_07RL|POOL2S_03RL' | perl -lane '$F[$#F] = "NoPhenotype"; print join("\t", @F); ') >  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race.pt1

#Added in the following four individuals with the tag 'NoPhenotype' at the end manually in the pt1 file
#POOL43_10RL POOL101_11RL POOL18S_07RL POOL2S_03RL

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.HapMap3vs.py --file1 /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.txt --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race.pt2

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race.pt1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race.pt2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race

sed -i 's/American_Indian_or_Alaskan_Native/AIndian_or_Alaskan_Native/g' /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race.plot.1vs2.xtxt

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race.plot.3vs4.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race -c 5:6 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race.plot.5vs6.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race -c 7:8 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race.plot.7vs8.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race -c 9:10 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race.plot.9vs10.xtxt

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race.plot.*pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/. 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.pheno | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.Ran729White.justFIDs | awk '{ print $1, "\t", $2, "\t0\t0\t0\t", $3, "\t", $4, "\t", $5, "\t", $6 } ' | sed 's/-9/NA/g' | perl -lane 'print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped | awk '{ print $1 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped.justFIDs

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped.justFIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.justWhite.QCed.WhiteNonHispSbsmpl.10PCs.covar

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.justWhite.QCed.WhiteNonHispSbsmpl.10PCs.covar | awk '{ print $4, "\t", $6, "\t", $7 "\t", $8, "\t", $9, "\t", $10, "\t", $11,  "\t", $12, "\t", $13, "\t", $14,  "\t", $15 }'  | paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped.vs2

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped.vs2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped

#Add line #FAM_ID IND_ID FAT_ID MOT_ID SEX HIVPROG HIVPROGHE AIDS AIDSEXTR RMID PC1 PC2 PC3 PC4 PC5 PC6 PC7 PC8 PC9 PC10 to /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped 

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz --remove /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.Ran729White.justFIDs --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.recode.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.vcf

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.vcf

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.ConvertGenosToACSumPerIndvGivenChrBPAndVCF.vs1.py --file1 All --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.perIndvSumAC

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.R

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.perIndvSumAC 11 | sort -rg -k 2,2 | perl -lane 'my @vals1 = split(/_/, $F[0]); print join("\t", @vals1), "\t", $F[1]; ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.AddAnnovarInfoToIndvSumAC.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz --file2 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.PC1.output.wAnnovarAnnotation 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.perIndvSumAC 12 | sort -rg -k 2,2 | perl -lane 'my @vals1 = split(/_/, $F[0]); print join("\t", @vals1), "\t", $F[1]; ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.AddAnnovarInfoToIndvSumAC.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz --file2 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.PC2.output.wAnnovarAnnotation 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.perIndvSumAC 13 | sort -rg -k 2,2 | perl -lane 'my @vals1 = split(/_/, $F[0]); print join("\t", @vals1), "\t", $F[1]; ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.AddAnnovarInfoToIndvSumAC.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz --file2 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.PC3.output.wAnnovarAnnotation 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.perIndvSumAC 14 | sort -rg -k 2,2 | perl -lane 'my @vals1 = split(/_/, $F[0]); print join("\t", @vals1), "\t", $F[1]; ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.AddAnnovarInfoToIndvSumAC.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz --file2 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.PC4.output.wAnnovarAnnotation 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.perIndvSumAC 15 | sort -rg -k 2,2 | perl -lane 'my @vals1 = split(/_/, $F[0]); print join("\t", @vals1), "\t", $F[1]; ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.AddAnnovarInfoToIndvSumAC.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz --file2 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.PC5.output.wAnnovarAnnotation 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.perIndvSumAC 16 | sort -rg -k 2,2 | perl -lane 'my @vals1 = split(/_/, $F[0]); print join("\t", @vals1), "\t", $F[1]; ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.AddAnnovarInfoToIndvSumAC.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz --file2 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.PC6.output.wAnnovarAnnotation 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.PC3.output.wAnnovarAnnotation | awk '{ if ($3 > .9) { print $0 } } ' | grep HEATR1 | grep -v NA | awk '{ print $1, "\t", $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.PC3.output.wAnnovarAnnotation.HEATR1.TopSNPs.ChrBP

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.ConvertGenosToACSumPerIndvGivenChrBPAndVCF.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.PC3.output.wAnnovarAnnotation.HEATR1.TopSNPs.ChrBP --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.HEATR1.TopSNPs.perIndvSumAC 

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.AddUpAlleleCountsPerIndvGivenChrBPAndVCF.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.withSNPWeights.snpWghts.wAnnovarAnnotation.UBR4.ChrBP --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.UBR4.AlleleCounts

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.UBR4.AlleleCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.HEATR1.TopSNPs.perIndvSumAC /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.output.pt1.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.output.pt2.vs2.WhiteNonHispSbsmpl.HEATR1.jpeg

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.output.pt*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/.

#20141121 moved back to Samtools mpileup stuff at this point

##20141121 Comparing GATK, Samtool and FreeBayes things
##

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.vcf.gz --min-alleles 2 --max-alleles 2 --remove-indels --mac 2 --geno .95 --hwe .0001 --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.SNP.Bi.raw.mac2.geno95.hwe1e4

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.SNP.Bi.raw.mac2.geno95.hwe1e4.recode.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.SNP.Bi.raw.mac2.geno95.hwe1e4.vcf

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.SNP.Bi.raw.mac2.geno95.hwe1e4.vcf

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.vcf.gz | awk '{ print $1, "\t", $2, "\t", $2, "\t", $4, "\t", $5 }' | grep -v ^# > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.vcf.AnnovarFormat 

perl /home/michaelt/Software/annovar/summarize_annovar.pl /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.vcf.AnnovarFormat /home/michaelt/Software/annovar/ --ver1000g 1000g2012apr --verdbsnp 129 --veresp 6500 --buildver hg19 --remove

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py /home/michaelt/Scripts/Python/DataAnalysis.All.CompareTwoVCFDatasets.vs1.py

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analysis /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons

python /home/michaelt/Scripts/Python/DataAnalysis.All.CompareTwoVCFDatasets.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.gz | grep VCF1 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.VCF1.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.gz | grep VCF2 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.VCF2.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.gz | grep VCF3a | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.VCF3a.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.gz | grep VCF3b | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.VCF3b.gz

[  michaelt@node11  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.gz | grep VCF1 | wc                                                                                                                                                                                                                                                                                          
  59736 55016856 1286393567
[  michaelt@node11  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.gz | grep VCF2 | wc                                                                                                                                                                                                                                                                                          
  10689 9844569 127227690
[  michaelt@node11  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | wc
  24334 22278124 290567514
[  michaelt@node11  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.gz | grep VCF3a | wc                                                                                                                                                                                                                                                                                         
  13525 12456525 292082160
[  michaelt@node11  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.gz | grep VCF3b | wc
  13525 12456525 163456485

python /home/michaelt/Scripts/Python/DataAnalysis.All.CompareTwoVCFDatasets.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.gz

[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.vcf.gz | wc
7634779 7023888441 88429303651

[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.vcf.gz | wc
3597614 3309696641 39947702864

[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | wc
  73389 67401779 1578124080                                                                                                                                                                                                                                                                                                

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.gz | grep VCF1 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF1.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.gz | grep VCF2 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF2.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.gz | grep VCF3a | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF3a.gz 
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.gz | grep VCF3b | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF3b.gz 

[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF1.gz | wc
  53656 49417176 1155527229
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF2.gz | wc
  3577890 3295236690 39730486110
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF3a.gz | wc
  19605 18056205 422954578
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF3b.gz | wc
  19589 18040690 235002695

python /home/michaelt/Scripts/Python/DataAnalysis.All.CompareTwoVCFDatasets.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.mac2.geno95.hwe1e4.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.gz

[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.mac2.geno95.hwe1e4.vcf.gz | wc
1536107 1413110201 17104123576

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.gz | grep VCF1 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF1.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.gz | grep VCF2 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF2.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.gz | grep VCF3a | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF3a.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.gz | grep VCF3b | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF3b.gz

[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF1.gz | wc
59548 54843708 1283788512
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Poolskb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF2.gz | wcl.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1
1522275 1402015275 16946929753
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Poolskb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF3a.gz | wc.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1
13713 12629673 294687403
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Poolskb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF3b.gz | wc.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1
13713 12629673 164869710


python /home/michaelt/Scripts/Python/DataAnalysis.All.CompareTwoVCFDatasets.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.gz

[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.vcf.gz | wc
 685929 631003829 16352675359

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.gz | grep VCF1 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF1.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.gz | grep VCF2 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF2.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.gz | grep VCF3a | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3a.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.gz | grep VCF3b | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3b.gz

[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF1.gz | wc
2031 1870551 42437025
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF2.gz | wc                                                  
614642 566085282 13944967335
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3a.gz | wc
71230 65602830 1536096407
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3b.gz | wc                                                 
71230 65602830 2411189761

python /home/michaelt/Scripts/Python/DataAnalysis.All.CompareTwoVCFDatasets.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.mac2.geno95.hwe1e4.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.gz

[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.mac2.geno95.hwe1e4.vcf.gz | wc
66408 61044509 2212902879

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.gz | grep VCF1 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF1.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.gz | grep VCF2 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF2.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.gz | grep VCF3a | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3a.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.gz | grep VCF3b | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3b.gz

[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF1.gz | wc
33797 31127037 730017829
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF2.gz | wc
26887 24762927 878150151 
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3a.gz | wc 
39464 36346344 848483837
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3b.gz | wc
39464 36346344 1335105094 

python /home/michaelt/Scripts/Python/DataAnalysis.All.CompareTwoVCFDatasets.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.gz 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.gz | grep VCF1 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF1.gz 
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.gz | grep VCF2 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF2.gz 
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.gz | grep VCF3a | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3a.gz 
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.gz | grep VCF3b | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3b.gz 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF1.gz | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF2.gz | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3a.gz | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3b.gz | wc

[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF1.gz | wc
3309948 3048462108 36712993187
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF2.gz | wc
398325 366857325 11554346684 
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3a.gz | wc 
287547 264830787 3252966933
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3b.gz | wc
287547 264830787 4802026729                                                                                                                                                                                                                                                                                               

python /home/michaelt/Scripts/Python/DataAnalysis.All.CompareTwoVCFDatasets.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.mac2.geno95.hwe1e4.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.mac2.geno95.hwe1e4.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.gz 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.gz | grep VCF1 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF1.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.gz | grep VCF2 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF2.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.gz | grep VCF3a | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3a.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.gz | grep VCF3b | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3b.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF1.gz | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF2.gz | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3a.gz | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3b.gz | wc

[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF1.gz | wc
1513182 1393640622 16838614138
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF2.gz | wc
43545 40104945 1453710147
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3a.gz | wc
22806 21004326 273194418
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3b.gz | wc
22806 21004326 759528440


python /home/michaelt/Scripts/Python/DataAnalysis.All.CompareTwoVCFDatasets.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.SNP.Bi.raw.mac2.geno95.hwe1e4.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.mac2.geno95.hwe1e4.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.gz 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.gz | grep VCF1 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF1.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.gz | grep VCF2 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF2.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.gz | grep VCF3a | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF3a.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.gz | grep VCF3b | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF3b.gz

[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF1.gz | wc
  97386 89692506 2044664843
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF2.gz | wc
  1515271 1395564591 16863362699
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF3a.gz | wc
    20717 19080357 442174357
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF3b.gz | wc
      20717 19080357 248443768


python /home/michaelt/Scripts/Python/DataAnalysis.All.CompareTwoVCFDatasets.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.SNP.Bi.raw.mac2.geno95.hwe1e4.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.mac2.geno95.hwe1e4.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.gz 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.gz | grep VCF1 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF1.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.gz | grep VCF2 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF2.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.gz | grep VCF3a | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3a.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.gz | grep VCF3b | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3b.gz

[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF1.gz | wc
  64450 59358450 1336882052
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF2.gz | wc
    12698 11694858 403174603
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3a.gz | wc
      53653 49414413 1149990084
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3b.gz | wc
        53653 49414413 1810094831


cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py

#Note -- actually did the /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF1.gz set of files before the /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.VCF1.gz set of files hence the notes about the VCF2/Annovar errors shown below with the 'second' set of files/run-through even though it's technically the first run-through. Keeping the order of the files to match physical order not necessarily 'precise' chronological order in terms of execution here

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.VCF1.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program GATK > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.VCF1.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.VCF1.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program Samtools > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.VCF2.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.VCF1.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program GATK > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.VCF3a.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.VCF1.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program Samtools > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.VCF3b.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF1.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program GATK > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF1.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF2.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program Samtools > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF2.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
#Getting the below errors for a few entries from the above code
Error2a -- ChrBP2 not found in hash1 (12_6910468)
Error2a -- ChrBP2 not found in hash1 (2_31816859)
Error2a -- ChrBP2 not found in hash1 (2_31705551)
Error2a -- ChrBP2 not found in hash1 (21_44035903)
Error2a -- ChrBP2 not found in hash1 (10_37412174)
Error2a -- ChrBP2 not found in hash1 (2_31816828)
Error2a -- ChrBP2 not found in hash1 (22_19205449)
Error2a -- ChrBP2 not found in hash1 (X_148956424)
Error2a -- ChrBP2 not found in hash1 (22_19178315)
Error2a -- ChrBP2 not found in hash1 (22_19178311)
Error2a -- ChrBP2 not found in hash1 (3_60190064)
Error2a -- ChrBP2 not found in hash1 (1_142535434)
Error2a -- ChrBP2 not found in hash1 (11_69139801)
Error2a -- ChrBP2 not found in hash1 (2_31816855)
Error2a -- ChrBP2 not found in hash1 (12_6910704)
Error2a -- ChrBP2 not found in hash1 (15_27591200)
Error2a -- ChrBP2 not found in hash1 (15_27591205)
Error2a -- ChrBP2 not found in hash1 (15_27591206)
Error2a -- ChrBP2 not found in hash1 (21_44035904)
Error2a -- ChrBP2 not found in hash1 (22_19178140)
Error2a -- ChrBP2 not found in hash1 (6_157559468)
Error2a -- ChrBP2 not found in hash1 (10_64321064)
Error2a -- ChrBP2 not found in hash1 (10_64321066)
Error2a -- ChrBP2 not found in hash1 (22_19179519)
Error2a -- ChrBP2 not found in hash1 (22_19178274)
Error2a -- ChrBP2 not found in hash1 (22_19205401)
Error2a -- ChrBP2 not found in hash1 (22_19193926)
Error2a -- ChrBP2 not found in hash1 (22_19205397)
Error2a -- ChrBP2 not found in hash1 (22_19200641)
Error2a -- ChrBP2 not found in hash1 (22_19178160)
Error2a -- ChrBP2 not found in hash1 (22_19178163)
Error2a -- ChrBP2 not found in hash1 (22_19178162)
Error2a -- ChrBP2 not found in hash1 (22_19178164)
Error2a -- ChrBP2 not found in hash1 (22_19178167)
Error2a -- ChrBP2 not found in hash1 (22_19178166)
Error2a -- ChrBP2 not found in hash1 (22_19178264)
Error2a -- ChrBP2 not found in hash1 (22_19178268)

#[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/b_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF2.gz | grep 6910468 | perl -lane 'print join("\t", @F[0..9]);'NP_PASSts99_9_wAA_Bi_DropOffTarg_1kb
VCF2    12      6910468 .       N       A       3.57    .       DP=1;AF1=1;AC1=1822;DP4=0,0,0,1;MQ=48;FQ=-20.9  GT:PL:GQ
#[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/b_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF2.gz | grep 19178311 | perl -lane 'print join("\t", @F[0..9]);'NP_PASSts99_9_wAA_Bi_DropOffTarg_1kb
VCF2    22      19178311        .       N       A       999     .       DP=21;VDB=1.273927e-02;AF1=1;AC1=1822;DP4=0,0,13,8;MQ=40;FQ=-23.5       GT:PL:GQ
#[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/b_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF2.gz | grep 37412174 | perl -lane 'print join("\t", @F[0..9]);'P_PASSts99_9_wAA_Bi_DropOffTarg_1kb
VCF2    10      37412174        .       N       G       999     .       DP=9;VDB=2.230037e-02;AF1=1;AC1=1822;DP4=0,0,6,3;MQ=49;FQ=-22.7 GT:PL:GQ
#[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/b_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF2.gz | grep 19178274 | perl -lane 'print join("\t", @F[0..9]);'P_PASSts99_9_wAA_Bi_DropOffTarg_1kb
VCF2    22      19178274        .       N       A       999     .       DP=16;VDB=4.218601e-02;AF1=1;AC1=1822;DP4=0,0,11,4;MQ=37;FQ=-23.1       GT:PL:GQ

#I think it just has to do with the ref allele being N, which is a variant that would have been dropped later in QC process steps anyways -- so shouldn't be a concern to see a few variants dropping out

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF3a.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program GATK > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF3a.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF3b.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program Samtools > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF3b.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF1.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program GATK > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF1.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF2.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program Samtools > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF2.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF3a.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program GATK > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF3a.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF3b.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program Samtools > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF3b.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF1.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program GATK > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF1.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF2.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program FreeBayes > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF2.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3a.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program GATK > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3a.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3b.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program FreeBayes > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3b.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output

/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF1.gz
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF1.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program GATK > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF1.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF2.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program FreeBayes > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF2.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3a.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program GATK > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3a.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3b.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program FreeBayes > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3b.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF1.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program Samtools > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF1.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF2.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program FreeBayes > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF2.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3a.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program Samtools > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3a.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3b.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program FreeBayes > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3b.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF1.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program Samtools > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF1.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF2.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program FreeBayes > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF2.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3a.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program Samtools > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3a.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3b.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program FreeBayes > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3b.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output


python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.vcf.AnnovarFormat.genome_summary.csv  --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF1.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program GATK > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF1.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF2.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program Samtools > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF2.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF3a.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program GATK > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF3a.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF3b.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program Samtools > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF3b.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output


python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF1.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program GATK > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF1.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF2.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program FreeBayes > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF2.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3a.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program GATK > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3a.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3b.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program FreeBayes > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3b.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output




~~~~

##EPACTS analyses

#EPACTS Variables Running Through
#Pheno: HIVPROG HIVPROGHE AIDS AIDSEXTR
#Filetypes: All Exonic Exonic.Nonsynonymous Exonic.Nonsynonymous.PP2_PD Exonic.Nonsynonymous.PP2_D Exonic.Nonsynonymous.Sift_D Exonic.Nonsynonymous.LRT_D
#Tests b.collapse b.madsen b.wcnt  skat --skat-adjust skat --skat-o --skat-adjust
#MAF cutoffs: .05 1.0 (e.g. treat all variants the same)

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/MainScript1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/MainScript1.PG_HIVArray1.sh 

vi /data/userdata/pg/michaelt/Data/ALL_MAPPING/MainScript1.PG_HIVArray1.EPACTScommands.sh #Using /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs1.sh instead

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptb.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs1.sh

#R
#install.packages("/home/michaelt/Software/SKAT_0.93.tar.gz", repos=NULL, type="source")
#R-2.0.3
#./configure; make; make check 

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.sh

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.vs2.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS

cp -p DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.sh DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.pt1.sh

cp -p DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.sh DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.pt2.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs

#WithOnlyRMIDs
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID

#Directory /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/
ls -lrt | grep preGATK | grep -v epacts$ | awk '{ print $9 }' | perl -lane 'system("mv $F[0] ExtraFiles/$F[0]");'

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.CollectResults.vs1.sh

#NOTE -- odd thing happening where I can copy and paste the commands from the file and they work, but if I run the script it doesn't work. Doesn't like the \ separating the lines
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.CollectResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs

cp -p DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.CollectResults.vs1.sh DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.CollectResults.vs1.CpyPst.sh


#install.packages("/home/michaelt/Software/SKAT_0.95.tar.gz", repos=NULL, type="source")

#cp -p /home/michaelt/Software/SKAT_0.95.tar.gz /home/michaelt/Software/EPACTS-3.2.4/share/EPACTS/.

#test statement for replacement length vector R error issue: epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --beta 1,1 --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno4.maf1.skat0flat
#epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --beta 1,25 --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno4.maf1.skat0

#Something odd happens where the method, even if calling for optimal, will go to regular SKAT davies method with the Q output instead of the rho output if the marker size is 2 (so 1 or 2 marker size leads to this behavior?)
#Fixed this by editing the code and simply throwing in a 'WeirdResult' in the rho column instead of a 'NA' when catching for rho to have length greater than 1 or equal to 0


cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.vs1.R

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs Pheno1 HIV_Acquisition_justWhite_RMID5PCs_skat0 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.RMID5PCs.Everything.Pheno1.maf1.skat0.QQplot.jpeg

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs Pheno2 HIV_AcquisitionHE_justWhite_RMID5PCs_skat0 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.RMID5PCs.Everything.Pheno2.maf1.skat0.QQplot.jpeg

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs Pheno3 AIDS_Progress_justWhite_RMID5PCs_skat0 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.RMID5PCs.Everything.Pheno3.maf1.skat0.QQplot.jpeg

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs Pheno4 AIDS_ProgressExtr_justWhite_RMID5PCs_skat0 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.RMID5PCs.Everything.Pheno4.maf1.skat0.QQplot.jpeg

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.RMID5PCs.Everything.Pheno*QQplot.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/.

/data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R

cp -p /home/michaelt/Scripts/Perl/RandomOrder.pl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs1.pl

perl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs1.pl --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --seed 10000 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped 

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno1.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno1.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno1.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno1.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno1.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno1.maf1.skat0.perm1 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno2.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno2.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno2.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno2.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno2.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno2.maf1.skat0.perm1 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno3.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno3.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno3.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno3.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno3.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno3.maf1.skat0.perm1 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno4.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno4.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno4.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno4.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno4.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno4.maf1.skat0.perm1 &




#install.packages("/home/michaelt/Software/qvalue_1.1.try2.tar.gz", repos=NULL, type="source")

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno2.maf1.skat0.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno3.maf1.skat0.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno4.maf1.skat0.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno1.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno1.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno1.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno1.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno1.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno1.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.perm1.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno2.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno2.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno2.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno2.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno2.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno2.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno2.maf1.skat0.perm1.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno3.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno3.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno3.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno3.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno3.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno3.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno3.maf1.skat0.perm1.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno4.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno4.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno4.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno4.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno4.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno4.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno4.maf1.skat0.perm1.wQvals.Summary

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes

#Complex
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.All.Pheno1.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Exonic.Pheno1.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.Pheno1.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.PP2_PD.Pheno1.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.PP2_D.Pheno1.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.Sift_D.Pheno1.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.LRT_D.Pheno1.maf1.skat0 & 

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.All.Pheno2.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Exonic.Pheno2.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.Pheno2.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.PP2_PD.Pheno2.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.PP2_D.Pheno2.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.Sift_D.Pheno2.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.LRT_D.Pheno2.maf1.skat0 & 

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.All.Pheno3.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Exonic.Pheno3.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.Pheno3.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.PP2_PD.Pheno3.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.PP2_D.Pheno3.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.Sift_D.Pheno3.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.LRT_D.Pheno3.maf1.skat0 & 

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.All.Pheno4.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Exonic.Pheno4.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.Pheno4.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.PP2_PD.Pheno4.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.PP2_D.Pheno4.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.Sift_D.Pheno4.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.LRT_D.Pheno4.maf1.skat0 & 


perl /home/michaelt/Scripts/Perl/RandomList.pl --total 10000 --seed 2938324 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.PermutationList.vs1.txt

for i in {1..1000}
do

	SEED1=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.PermutationList.vs1.txt | awk '{ if (NR == '$i') { print $0 } } '`

	perl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs1.pl --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --seed $SEED1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm${i}.ped

#	echo $SEED1

done

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptb.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.Launcher.vs1.sh 

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptb.CreateVCFfilesPerChromosome.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.vs1.sh

#Edit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.Launcher.vs1.sh to go from 1 to 100
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations Exonic.Nonsynonymous HIVPROG
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations Exonic.Nonsynonymous HIVPROGHE
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations Exonic.Nonsynonymous AIDS
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations Exonic.Nonsynonymous AIDSEXTR
#Edit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.Launcher.vs1.sh to go from 101 to 1000
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations Exonic.Nonsynonymous HIVPROG
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations Exonic.Nonsynonymous AIDS

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptb.CreateVCFfilesPerChromosome.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.vs1.sh

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.AvgPvalRanks.vs1.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.AvgPvalRanks.vs1.sh Exonic.Nonsynonymous HIVPROG &
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.AvgPvalRanks.vs1.sh Exonic.Nonsynonymous HIVPROGHE & 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.AvgPvalRanks.vs1.sh Exonic.Nonsynonymous AIDS &
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.AvgPvalRanks.vs1.sh Exonic.Nonsynonymous AIDSEXTR & 

paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | grep -v NA | grep -v CHROM | tail -n 100 | awk '{ print $10 } ') <( sort -r -k 1,1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.AvgPvalRanks.vs1.Exonic.Nonsynonymous.HIVPROG.results ) | vi -
paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | grep -v NA | grep -v CHROM | tail -n 100 | awk '{ print $10 } ') <( sort -r -k 1,1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.AvgPvalRanks.vs1.Exonic.Nonsynonymous.HIVPROGHE.results ) | vi -
paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | grep -v NA | grep -v CHROM | tail -n 100 | awk '{ print $10 } ') <( sort -r -k 1,1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.AvgPvalRanks.vs1.Exonic.Nonsynonymous.AIDS.results ) | vi -
paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | grep -v NA | grep -v CHROM | tail -n 100 | awk '{ print $10 } ') <( sort -r -k 1,1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.AvgPvalRanks.vs1.Exonic.Nonsynonymous.AIDSEXTR.results ) | vi -

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.AvgPvalRanks.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.sh
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.AvgPvalRanks.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.DirectLociPvalComp.vs1.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.sh Exonic.Nonsynonymous HIVPROG 1000 50 &
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.sh Exonic.Nonsynonymous HIVPROG 1000 100 &
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.sh Exonic.Nonsynonymous HIVPROGHE 100 50 & 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.sh Exonic.Nonsynonymous HIVPROGHE 100 100 & 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.sh Exonic.Nonsynonymous AIDS 100 50 &
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.sh Exonic.Nonsynonymous AIDS 100 100 &
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.sh Exonic.Nonsynonymous AIDSEXTR 100 50 & 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.sh Exonic.Nonsynonymous AIDSEXTR 100 100 & 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | grep -v NA | grep -v CHROM | tail -n 50 | awk '{ print $10 } ' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); print(sum(Data1[,1]));" | perl -lane 'if ($F[0] !~ /^\>/) { print join("\t", @F); } ' | awk '{ print $2 }'
1.629732
cat EPACTSFiles/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.Exonic.Nonsynonymous.HIVPROG.top50.results | sort -gr | vi -
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | grep -v NA | grep -v CHROM | tail -n 100 | awk '{ print $10 } ' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); print(sum(Data1[,1]));" | perl -lane 'if ($F[0] !~ /^\>/) { print join("\t", @F); } ' | awk '{ print $2 }'
7.918769
cat EPACTSFiles/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.Exonic.Nonsynonymous.HIVPROG.top100.results | sort -gr | vi -
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | grep -v NA | grep -v CHROM | tail -n 50 | awk '{ print $10 } ' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); print(sum(Data1[,1]));" | perl -lane 'if ($F[0] !~ /^\>/) { print join("\t", @F); } ' | awk '{ print $2 }'
2.278094
cat EPACTSFiles/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.Exonic.Nonsynonymous.HIVPROGHE.top50.results | sort -gr | vi -
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | grep -v NA | grep -v CHROM | tail -n 100 | awk '{ print $10 } ' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); print(sum(Data1[,1]));" | perl -lane 'if ($F[0] !~ /^\>/) { print join("\t", @F); } ' | awk '{ print $2 }'
10.20435
cat EPACTSFiles/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.Exonic.Nonsynonymous.HIVPROGHE.top100.results | sort -gr | vi -
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | grep -v NA | grep -v CHROM | tail -n 50 | awk '{ print $10 } ' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); print(sum(Data1[,1]));" | perl -lane 'if ($F[0] !~ /^\>/) { print join("\t", @F); } ' | awk '{ print $2 }'
3.098894
cat EPACTSFiles/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.Exonic.Nonsynonymous.AIDS.top50.results | sort -gr | vi -
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | grep -v NA | grep -v CHROM | tail -n 100 | awk '{ print $10 } ' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); print(sum(Data1[,1]));" | perl -lane 'if ($F[0] !~ /^\>/) { print join("\t", @F); } ' | awk '{ print $2 }'
14.3855
cat EPACTSFiles/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.Exonic.Nonsynonymous.AIDS.top100.results | sort -gr | vi -
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | grep -v NA | grep -v CHROM | tail -n 50 | awk '{ print $10 } ' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); print(sum(Data1[,1]));" | perl -lane 'if ($F[0] !~ /^\>/) { print join("\t", @F); } ' | awk '{ print $2 }'
4.228986
cat EPACTSFiles/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.Exonic.Nonsynonymous.AIDSEXTR.top50.results | sort -gr | vi -
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | grep -v NA | grep -v CHROM | tail -n 100 | awk '{ print $10 } ' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); print(sum(Data1[,1]));" | perl -lane 'if ($F[0] !~ /^\>/) { print join("\t", @F); } ' | awk '{ print $2 }'
19.08815
cat EPACTSFiles/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.Exonic.Nonsynonymous.AIDSEXTR.top100.results | sort -gr | vi -

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.epacts.justGeneIDs

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.epacts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.HIVPROG.maf1.skat0.epacts
ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.HIVPROGHE.maf1.skat0.epacts
ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.epacts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.AIDS.maf1.skat0.epacts
ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.epacts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.AIDSEXTR.maf1.skat0.epacts

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.DirectLociPvalComp.vs1.sh Exonic.Nonsynonymous HIVPROG

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.DirectLociPvalComp.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.DirectLociPvalComp.vs1.pt1.sh
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.DirectLociPvalComp.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.DirectLociPvalComp.vs1.pt2.sh

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/perGeneCollect

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.DirectLociPvalComp.vs1.pt1.sh Exonic.Nonsynonymous HIVPROG 1000 &
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.DirectLociPvalComp.vs1.pt1.sh Exonic.Nonsynonymous HIVPROGHE 100 &
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.DirectLociPvalComp.vs1.pt1.sh Exonic.Nonsynonymous AIDS 100 &
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.DirectLociPvalComp.vs1.pt1.sh Exonic.Nonsynonymous AIDSEXTR 100 &
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.DirectLociPvalComp.vs1.pt2.sh Exonic.Nonsynonymous HIVPROG &
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.DirectLociPvalComp.vs1.pt2.sh Exonic.Nonsynonymous HIVPROGHE &
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.DirectLociPvalComp.vs1.pt2.sh Exonic.Nonsynonymous AIDS &
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.DirectLociPvalComp.vs1.pt2.sh Exonic.Nonsynonymous AIDSEXTR &


cat all the files and grep each locus one at a time and do the p-value perm count thing there

FDR approach?

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.EmpricalValues.vs1.R

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.ForPoster.vs1.R

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.ForPoster.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs Pheno1 HIV_Acquisition_SKAT0 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.RMID5PCs.Everything.Pheno1.maf1.skat0.QQplot.ForPoster.jpeg

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.ForPoster.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs Pheno2 HIV_Acq._HighlyExposedSeroNeg_SKAT0 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.RMID5PCs.Everything.Pheno2.maf1.skat0.QQplot.ForPoster.jpeg

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.ForPoster.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs Pheno3 AIDS_Progression_SKAT0 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.RMID5PCs.Everything.Pheno3.maf1.skat0.QQplot.ForPoster.jpeg

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.ForPoster.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs Pheno4 AIDS_ProgressExtr_justWhite_RMID5PCs_skat0 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.RMID5PCs.Everything.Pheno4.maf1.skat0.QQplot.ForPoster.jpeg

#Doing multiple runs of the original analysis (100x) for each level/data-breakdown in order to get precise estimates of the p-values since there is bootstrapping going on in the small sample-size adjustment within SKAT

#20140506 Note -- did the following command
mv /data/userdata/pg/michaelt/* /data/userdata/pg/michaelt/.
#So retroactively changed all the paths in this file to the /data/userdata/pg/michaelt beginning
#Used the following command to do so :.,$ s/\/home\/pg\//\/data\/userdata\/pg\//g at the top of this file

mkdir Permutations/FakePheno
mv Permutations/* Permutations/FakePheno/.
mkdir Permutations/TruePheno

#QC on the phenotype file(s)
[  michaelt@login2  ~/LabStuff/StephensLab/HIVStuff/PG]$cat Sample_status_list_12-10-13_CT.edited.noMeta.csv | perl -F, -lane 'print join("", @F[11..18]);' | sort | uniq -c
57 00000001
49 00000011
7 00000100
1 00001100
181 00010000
6 00010001
6 00010011
20 00010100
18 00011100
10 00100000
1 00100100
140 00110000
8 00110011
16 00110100
6 00111100
200 01000000
199 10000000
1 hesnlesnlescotherscvrapidrapidvslowslow


cat EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary | grep -E 'All|Exonic' > EPACTSFiles/ForKevin.HIVAcquisition.skat0.wQvals.Summary
cat EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno2.maf1.skat0.wQvals.Summary | grep -E 'All|Exonic' > EPACTSFiles/ForKevin.HIVAcquisitionHighlyExposed.skat0.wQvals.Summary
cat EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno3.maf1.skat0.wQvals.Summary | grep -E 'All|Exonic' > EPACTSFiles/ForKevin.AIDSProgression.skat0.wQvals.Summary
#Note -- manually added the headers to each file then


#20150113
#Coming back in now and finishing up these ideas/results by finalizing the 'top' list of genes first by getting quality point estimates of p-values and then doing permutation runs to get FDR-based thresholds for 'top' genes

perl /home/michaelt/Scripts/Perl/RandomList.pl --total 10000 --seed 5830128 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.MultiRunList.vs1.txt
perl /home/michaelt/Scripts/Perl/RandomList.pl --total 10000 --seed 2938324 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.PermutationList.vs1.txt

for i in {1..1000}
do

	SEED1=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.PermutationList.vs1.txt | awk '{ if (NR == '$i') { print $0 } } '`

	perl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs1.pl --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --seed $SEED1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm${i}.ped

#	echo $SEED1

done

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.EPACTSMultiRun.Launcher.vs2.sh
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.EPACTSMultiRun.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/OLD1.DataAnalysis.GATK.EPACTSMultiRun.Launcher.vs2.sh

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.EPACTSMultiRun.vs2.sh 
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.EPACTSMultiRun.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/OLD1.DataAnalysis.GATK.EPACTSMultiRun.vs2.sh

#Decided to create these scripts using /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.pt*.sh as a base instead of /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.Launcher.vs1.sh but keeping the above files around in case I want to go back and go down that route for another reason (since I started some of the work of modulating the scripts as they would need to be in order to go that way)
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.pt2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt2.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS 

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt2.sh

tar -czvf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/ExonicNonsynonymous/OLD1.AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.try2.runAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/ExonicNonsynonymous/*

rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/ExonicNonsynonymous/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.*

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/ExonicNonsynonymous/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/

tar -cvzf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/OLD1.AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.try2.runAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/*

rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.*

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/*Exonic.Nonsynonymous.Pheno1* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/*Exonic.Nonsynonymous.Pheno2* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/*Exonic.Nonsynonymous.Pheno3* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/*Exonic.Nonsynonymous.Pheno4* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/.

rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/*ind /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/*Makefile
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/*ind /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/*Makefile
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/*ind /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/*Makefile
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/*ind /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/*Makefile

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous

rm -r /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Nonsynonymous

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/*HIVPROG.maf1* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/*HIVPROGHE.maf1* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/*AIDS.maf1* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/*AIDSEXTR.maf1* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/.

rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*Makefile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*ind
tar -cvzf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.HIVPROG.maf1.skat0.permAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*epacts

#This command wasn't needed -- these files had already been removed at some point before now
#rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*Makefile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*ind
tar -cvzf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.HIVPROGHE.maf1.skat0.permAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*epacts

rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*Makefile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*ind
tar -cvzf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.AIDS.maf1.skat0.permAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*epacts

#There were only 100 files present for this phenotype
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*Makefile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*ind
tar -cvzf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.AIDSEXTR.maf1.skat0.permAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*epacts

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/*Pheno1*perm* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/*Pheno2*perm* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/*Pheno3*perm* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/*Pheno4*perm* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/

rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*Makefile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*ind
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*Makefile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*ind
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*Makefile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*ind
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*Makefile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*ind

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run1.epacts.GeneIDList
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run1.epacts.GeneIDList
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run1.epacts.GeneIDList

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm1.epacts.GeneIDList
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm1.epacts.GeneIDList
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm1.epacts.GeneIDList

[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous | wc                                                  474    4802   70376

#Note -- *GeneIDList files for Exonic.Nonsynonymous contain 475 entries (474 genes + 1 'ID' row)

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList TruePheno Pheno1 Exonic.Nonsynonymous 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run1.epacts.GeneIDList TruePheno Pheno2 Exonic.Nonsynonymous 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run1.epacts.GeneIDList TruePheno Pheno3 Exonic.Nonsynonymous 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run1.epacts.GeneIDList TruePheno Pheno4 Exonic.Nonsynonymous 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList FakePheno Pheno1 Exonic.Nonsynonymous 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm1.epacts.GeneIDList FakePheno Pheno2 Exonic.Nonsynonymous 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm1.epacts.GeneIDList FakePheno Pheno3 Exonic.Nonsynonymous 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm1.epacts.GeneIDList FakePheno Pheno4 Exonic.Nonsynonymous 

#NOTE -- realized that for permutations I should be doing 'within-phenotype subset permutations' instead of just permuting everyone at once, so I need to update my permutation script and redo phenotypes 2, 3 and 4. Need to do permutations 'within-subsets' because if these individuals have other aspects for why they were 'chosen' to be these phenotype subsets I want to include that within the framework of the permutation, so that the only question being asked is whether the genetic information is what's causing signifiance. If I permute the entire group at large I'm also testing whether there were certain external factors on top of the genetic factors that caused those individuals to first, be that specific phenotype subset, and then second, to produce significant test results. 

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs1.pl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs2.pl

for i in {6..1000}
do

	SEED1=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.PermutationList.vs1.txt | awk '{ if (NR == '$i') { print $0 } } '`

	perl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs2.pl --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --PhenoColumn 5 --seed $SEED1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Pheno1.perm${i}.ped

#	echo $SEED1

done

~~~
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous]$head -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Pheno1.perm1.ped                                                                                                                                                                                                                    
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL98_01RL     POOL98_01RL     0       0       0       2       NA      NA      NA      1       0.0079  0.0160  0.0720  0.0296  0.0320  -0.0555 -0.0447 0.0014  0.0010  -0.0499                                                                                                                                            
POOL83_08RL     POOL83_08RL     0       0       0       2       NA      NA      NA      8       0.0157  0.0095  -0.0198 -0.0248 -0.0206 -0.0242 0.0153  0.0448  0.0243  0.0067
POOL77_08RL     POOL77_08RL     0       0       0       2       NA      NA      NA      8       0.0143  -0.0026 -0.0279 -0.0273 0.0338  -0.0194 0.0555  0.0160  -0.0053 0.0202                                                                                                                                             
POOL02_03RL     POOL02_03RL     0       0       0       1       1       NA      NA      3       0.0136  0.0057  0.0166  0.0415  -0.0098 0.0027  -0.0120 -0.0008 -0.0187 0.0112
POOL33_03RL     POOL33_03RL     0       0       0       1       2       NA      NA      3       0.0137  0.0162  -0.0260 -0.0403 -0.0116 -0.0230 -0.0308 0.0320  -0.0137 -0.0560                                                                                                                                            
POOL157_04RL    POOL157_04RL    0       0       0       2       NA      NA      NA      4       0.0128  0.0025  -0.0309 0.0374  -0.0126 0.0054  0.0047  0.0157  -0.0291 0.0530
POOL88_02RL     POOL88_02RL     0       0       0       2       2       NA      NA      2       0.0118  0.0045  -0.0299 0.0170  0.0068  -0.0149 -0.0094 -0.0493 -0.0122 -0.0149                                                                                                                                            
POOL109_11RL    POOL109_11RL    0       0       0       2       1       NA      NA      11      0.0160  0.0021  -0.0172 -0.0269 -0.0312 0.0343  0.0326  -0.0096 -0.0062 0.0452
POOL158_08RL    POOL158_08RL    0       0       0       2       2       2       NA      8       0.0106  0.0138  0.0268  -0.0544 0.0063  -0.0188 -0.0086 0.0534  -0.0106 -0.0414                                                                                                                                            
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous]$head -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped                                                                                                                                                                                                                           
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL98_01RL     POOL98_01RL     0       0       0       2       2       2       NA      1       0.0079  0.0160  0.0720  0.0296  0.0320  -0.0555 -0.0447 0.0014  0.0010  -0.0499                                                                                                                                            
POOL83_08RL     POOL83_08RL     0       0       0       2       2       1       NA      8       0.0157  0.0095  -0.0198 -0.0248 -0.0206 -0.0242 0.0153  0.0448  0.0243  0.0067
POOL77_08RL     POOL77_08RL     0       0       0       2       2       NA      NA      8       0.0143  -0.0026 -0.0279 -0.0273 0.0338  -0.0194 0.0555  0.0160  -0.0053 0.0202                                                                                                                                             
POOL02_03RL     POOL02_03RL     0       0       0       1       1       NA      NA      3       0.0136  0.0057  0.0166  0.0415  -0.0098 0.0027  -0.0120 -0.0008 -0.0187 0.0112
POOL33_03RL     POOL33_03RL     0       0       0       1       1       NA      NA      3       0.0137  0.0162  -0.0260 -0.0403 -0.0116 -0.0230 -0.0308 0.0320  -0.0137 -0.0560                                                                                                                                            
POOL157_04RL    POOL157_04RL    0       0       0       2       2       2       2       4       0.0128  0.0025  -0.0309 0.0374  -0.0126 0.0054  0.0047  0.0157  -0.0291 0.0530
POOL88_02RL     POOL88_02RL     0       0       0       2       2       NA      NA      2       0.0118  0.0045  -0.0299 0.0170  0.0068  -0.0149 -0.0094 -0.0493 -0.0122 -0.0149                                                                                                                                            
POOL109_11RL    POOL109_11RL    0       0       0       2       2       NA      NA      11      0.0160  0.0021  -0.0172 -0.0269 -0.0312 0.0343  0.0326  -0.0096 -0.0062 0.0452
POOL158_08RL    POOL158_08RL    0       0       0       2       2       1       NA      8       0.0106  0.0138  0.0268  -0.0544 0.0063  -0.0188 -0.0086 0.0534  -0.0106 -0.0414                                                                                                                                            
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous]$head -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped                                                                                                                                                                                                                                              
#FAM_ID IND_ID FAT_ID MOT_ID SEX HIVPROG HIVPROGHE AIDS AIDSEXTR RMID PC1 PC2 PC3 PC4 PC5 PC6 PC7 PC8 PC9 PC10
POOL98_01RL     POOL98_01RL     0       0       0       1       NA      NA      NA      1        0.0079          0.0160  0.0720          0.0296          0.0320          -0.0555         -0.0447         0.0014          0.0010          -0.0499                                                                           
POOL83_08RL     POOL83_08RL     0       0       0       1       NA      NA      NA      8        0.0157          0.0095  -0.0198         -0.0248         -0.0206         -0.0242         0.0153          0.0448          0.0243          0.0067
POOL77_08RL     POOL77_08RL     0       0       0       1       NA      NA      NA      8        0.0143          -0.0026         -0.0279         -0.0273         0.0338          -0.0194         0.0555          0.0160          -0.0053         0.0202                                                                    
POOL02_03RL     POOL02_03RL     0       0       0       1       1       NA      NA      3        0.0136          0.0057  0.0166          0.0415          -0.0098         0.0027          -0.0120         -0.0008         -0.0187         0.0112
POOL33_03RL     POOL33_03RL     0       0       0       2       2       NA      NA      3        0.0137          0.0162  -0.0260         -0.0403         -0.0116         -0.0230         -0.0308         0.0320          -0.0137         -0.0560                                                                           
POOL157_04RL    POOL157_04RL    0       0       0       1       NA      NA      NA      4        0.0128          0.0025  -0.0309         0.0374          -0.0126         0.0054          0.0047          0.0157          -0.0291         0.0530
POOL88_02RL     POOL88_02RL     0       0       0       2       2       NA      NA      2        0.0118          0.0045  -0.0299         0.0170          0.0068          -0.0149         -0.0094         -0.0493         -0.0122         -0.0149                                                                           
POOL109_11RL    POOL109_11RL    0       0       0       1       1       NA      NA      11       0.0160          0.0021  -0.0172         -0.0269         -0.0312         0.0343          0.0326          -0.0096         -0.0062         0.0452
POOL158_08RL    POOL158_08RL    0       0       0       2       2       2       NA      8        0.0106          0.0138  0.0268          -0.0544         0.0063          -0.0188         -0.0086         0.0534          -0.0106         -0.0414                                                                           
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous]$head -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Pheno1.perm2.ped                                                                                                                                                                                                                    
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL98_01RL     POOL98_01RL     0       0       0       2       NA      NA      NA      1       0.0079  0.0160  0.0720  0.0296  0.0320  -0.0555 -0.0447 0.0014  0.0010  -0.0499                                                                                                                                            
POOL83_08RL     POOL83_08RL     0       0       0       1       NA      NA      NA      8       0.0157  0.0095  -0.0198 -0.0248 -0.0206 -0.0242 0.0153  0.0448  0.0243  0.0067
POOL77_08RL     POOL77_08RL     0       0       0       1       NA      NA      NA      8       0.0143  -0.0026 -0.0279 -0.0273 0.0338  -0.0194 0.0555  0.0160  -0.0053 0.0202                                                                                                                                             
POOL02_03RL     POOL02_03RL     0       0       0       1       1       NA      NA      3       0.0136  0.0057  0.0166  0.0415  -0.0098 0.0027  -0.0120 -0.0008 -0.0187 0.0112
POOL33_03RL     POOL33_03RL     0       0       0       2       2       NA      NA      3       0.0137  0.0162  -0.0260 -0.0403 -0.0116 -0.0230 -0.0308 0.0320  -0.0137 -0.0560                                                                                                                                            
POOL157_04RL    POOL157_04RL    0       0       0       2       NA      NA      NA      4       0.0128  0.0025  -0.0309 0.0374  -0.0126 0.0054  0.0047  0.0157  -0.0291 0.0530
POOL88_02RL     POOL88_02RL     0       0       0       1       2       NA      NA      2       0.0118  0.0045  -0.0299 0.0170  0.0068  -0.0149 -0.0094 -0.0493 -0.0122 -0.0149                                                                                                                                            
POOL109_11RL    POOL109_11RL    0       0       0       2       1       NA      NA      11      0.0160  0.0021  -0.0172 -0.0269 -0.0312 0.0343  0.0326  -0.0096 -0.0062 0.0452
POOL158_08RL    POOL158_08RL    0       0       0       2       2       2       NA      8       0.0106  0.0138  0.0268  -0.0544 0.0063  -0.0188 -0.0086 0.0534  -0.0106 -0.0414                                                                                                                                            
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous]$head -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm2.ped                                                                                                                                                                                                                           
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL98_01RL     POOL98_01RL     0       0       0       2       2       NA      NA      1       0.0079  0.0160  0.0720  0.0296  0.0320  -0.0555 -0.0447 0.0014  0.0010  -0.0499                                                                                                                                            
POOL83_08RL     POOL83_08RL     0       0       0       1       NA      NA      NA      8       0.0157  0.0095  -0.0198 -0.0248 -0.0206 -0.0242 0.0153  0.0448  0.0243  0.0067
POOL77_08RL     POOL77_08RL     0       0       0       1       NA      NA      NA      8       0.0143  -0.0026 -0.0279 -0.0273 0.0338  -0.0194 0.0555  0.0160  -0.0053 0.0202                                                                                                                                             
POOL02_03RL     POOL02_03RL     0       0       0       1       NA      NA      NA      3       0.0136  0.0057  0.0166  0.0415  -0.0098 0.0027  -0.0120 -0.0008 -0.0187 0.0112
POOL33_03RL     POOL33_03RL     0       0       0       2       2       2       NA      3       0.0137  0.0162  -0.0260 -0.0403 -0.0116 -0.0230 -0.0308 0.0320  -0.0137 -0.0560                                                                                                                                            
POOL157_04RL    POOL157_04RL    0       0       0       2       2       1       1       4       0.0128  0.0025  -0.0309 0.0374  -0.0126 0.0054  0.0047  0.0157  -0.0291 0.0530
POOL88_02RL     POOL88_02RL     0       0       0       1       NA      NA      NA      2       0.0118  0.0045  -0.0299 0.0170  0.0068  -0.0149 -0.0094 -0.0493 -0.0122 -0.0149                                                                                                                                            
POOL109_11RL    POOL109_11RL    0       0       0       2       2       NA      NA      11      0.0160  0.0021  -0.0172 -0.0269 -0.0312 0.0343  0.0326  -0.0096 -0.0062 0.0452
POOL158_08RL    POOL158_08RL    0       0       0       2       2       NA      NA      8       0.0106  0.0138  0.0268  -0.0544 0.0063  -0.0188 -0.0086 0.0534  -0.0106 -0.0414                                                                                                                                            
~~~

for i in {1..1000}
do

	SEED1=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.PermutationList.vs1.txt | awk '{ if (NR == '$i') { print $0 } } '`

	perl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs2.pl --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --PhenoColumn 6 --seed $SEED1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Pheno2.perm${i}.ped

#	echo $SEED1

done

for i in {1..1000}
do

	SEED1=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.PermutationList.vs1.txt | awk '{ if (NR == '$i') { print $0 } } '`

	perl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs2.pl --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --PhenoColumn 7 --seed $SEED1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Pheno3.perm${i}.ped

#	echo $SEED1

done

for i in {1..1000}
do

	SEED1=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.PermutationList.vs1.txt | awk '{ if (NR == '$i') { print $0 } } '`

	perl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs2.pl --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --PhenoColumn 8 --seed $SEED1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Pheno4.perm${i}.ped

#	echo $SEED1

done

#This was wrong, I need to permute within-subsets not within specific columns, redoing the code but keeping these old commands in case they are useful at some other timepoint
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Pheno1.perm*.ped
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Pheno2.perm*.ped
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Pheno3.perm*.ped
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Pheno4.perm*.ped

tar -cvzf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.permAll.ped.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm*[!vs3].ped

rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm*.ped

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs2.pl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs3.pl 

for i in {1..1000}
do

	SEED1=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.PermutationList.vs1.txt | awk '{ if (NR == '$i') { print $0 } } '`

#	perl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs3.pl --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --PhenoColumn 5 --seed $SEED1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm${i}.vs3.ped
	perl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs3.pl --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --seed $SEED1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm${i}.vs3.ped

#	echo $SEED1

done

~~~
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous]$head -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.vs3.ped 
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL98_01RL     POOL98_01RL     0       0       0       2       NA      NA      NA      1       0.0079  0.0160  0.0720  0.0296  0.0320  -0.0555 -0.0447 0.0014  0.0010  -0.0499
POOL83_08RL     POOL83_08RL     0       0       0       2       NA      NA      NA      8       0.0157  0.0095  -0.0198 -0.0248 -0.0206 -0.0242 0.0153  0.0448  0.0243  0.0067
POOL77_08RL     POOL77_08RL     0       0       0       2       NA      NA      NA      8       0.0143  -0.0026 -0.0279 -0.0273 0.0338  -0.0194 0.0555  0.0160  -0.0053 0.0202
POOL02_03RL     POOL02_03RL     0       0       0       1       2       NA      NA      3       0.0136  0.0057  0.0166  0.0415  -0.0098 0.0027  -0.0120 -0.0008 -0.0187 0.0112
POOL33_03RL     POOL33_03RL     0       0       0       1       1       NA      NA      3       0.0137  0.0162  -0.0260 -0.0403 -0.0116 -0.0230 -0.0308 0.0320  -0.0137 -0.0560
POOL157_04RL    POOL157_04RL    0       0       0       2       NA      NA      NA      4       0.0128  0.0025  -0.0309 0.0374  -0.0126 0.0054  0.0047  0.0157  -0.0291 0.0530
POOL88_02RL     POOL88_02RL     0       0       0       2       1       NA      NA      2       0.0118  0.0045  -0.0299 0.0170  0.0068  -0.0149 -0.0094 -0.0493 -0.0122 -0.0149
POOL109_11RL    POOL109_11RL    0       0       0       2       2       NA      NA      11      0.0160  0.0021  -0.0172 -0.0269 -0.0312 0.0343  0.0326  -0.0096 -0.0062 0.0452
POOL158_08RL    POOL158_08RL    0       0       0       2       2       1       NA      8       0.0106  0.0138  0.0268  -0.0544 0.0063  -0.0188 -0.0086 0.0534  -0.0106 -0.0414
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous]$head -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm2.vs3.ped 
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL98_01RL     POOL98_01RL     0       0       0       2       NA      NA      NA      1       0.0079  0.0160  0.0720  0.0296  0.0320  -0.0555 -0.0447 0.0014  0.0010  -0.0499
POOL83_08RL     POOL83_08RL     0       0       0       1       NA      NA      NA      8       0.0157  0.0095  -0.0198 -0.0248 -0.0206 -0.0242 0.0153  0.0448  0.0243  0.0067
POOL77_08RL     POOL77_08RL     0       0       0       1       NA      NA      NA      8       0.0143  -0.0026 -0.0279 -0.0273 0.0338  -0.0194 0.0555  0.0160  -0.0053 0.0202
POOL02_03RL     POOL02_03RL     0       0       0       1       1       NA      NA      3       0.0136  0.0057  0.0166  0.0415  -0.0098 0.0027  -0.0120 -0.0008 -0.0187 0.0112
POOL33_03RL     POOL33_03RL     0       0       0       2       1       NA      NA      3       0.0137  0.0162  -0.0260 -0.0403 -0.0116 -0.0230 -0.0308 0.0320  -0.0137 -0.0560
POOL157_04RL    POOL157_04RL    0       0       0       2       NA      NA      NA      4       0.0128  0.0025  -0.0309 0.0374  -0.0126 0.0054  0.0047  0.0157  -0.0291 0.0530
POOL88_02RL     POOL88_02RL     0       0       0       1       2       NA      NA      2       0.0118  0.0045  -0.0299 0.0170  0.0068  -0.0149 -0.0094 -0.0493 -0.0122 -0.0149
POOL109_11RL    POOL109_11RL    0       0       0       2       2       NA      NA      11      0.0160  0.0021  -0.0172 -0.0269 -0.0312 0.0343  0.0326  -0.0096 -0.0062 0.0452
POOL158_08RL    POOL158_08RL    0       0       0       2       2       2       NA      8       0.0106  0.0138  0.0268  -0.0544 0.0063  -0.0188 -0.0086 0.0534  -0.0106 -0.0414
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous]$head -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm3.vs3.ped 
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL98_01RL     POOL98_01RL     0       0       0       1       NA      NA      NA      1       0.0079  0.0160  0.0720  0.0296  0.0320  -0.0555 -0.0447 0.0014  0.0010  -0.0499
POOL83_08RL     POOL83_08RL     0       0       0       1       NA      NA      NA      8       0.0157  0.0095  -0.0198 -0.0248 -0.0206 -0.0242 0.0153  0.0448  0.0243  0.0067
POOL77_08RL     POOL77_08RL     0       0       0       1       NA      NA      NA      8       0.0143  -0.0026 -0.0279 -0.0273 0.0338  -0.0194 0.0555  0.0160  -0.0053 0.0202
POOL02_03RL     POOL02_03RL     0       0       0       1       1       NA      NA      3       0.0136  0.0057  0.0166  0.0415  -0.0098 0.0027  -0.0120 -0.0008 -0.0187 0.0112
POOL33_03RL     POOL33_03RL     0       0       0       2       2       NA      NA      3       0.0137  0.0162  -0.0260 -0.0403 -0.0116 -0.0230 -0.0308 0.0320  -0.0137 -0.0560
POOL157_04RL    POOL157_04RL    0       0       0       1       NA      NA      NA      4       0.0128  0.0025  -0.0309 0.0374  -0.0126 0.0054  0.0047  0.0157  -0.0291 0.0530
POOL88_02RL     POOL88_02RL     0       0       0       1       1       NA      NA      2       0.0118  0.0045  -0.0299 0.0170  0.0068  -0.0149 -0.0094 -0.0493 -0.0122 -0.0149
POOL109_11RL    POOL109_11RL    0       0       0       1       2       NA      NA      11      0.0160  0.0021  -0.0172 -0.0269 -0.0312 0.0343  0.0326  -0.0096 -0.0062 0.0452
POOL158_08RL    POOL158_08RL    0       0       0       2       2       2       NA      8       0.0106  0.0138  0.0268  -0.0544 0.0063  -0.0188 -0.0086 0.0534  -0.0106 -0.0414
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous]$head -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm4.vs3.ped 
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL98_01RL     POOL98_01RL     0       0       0       2       NA      NA      NA      1       0.0079  0.0160  0.0720  0.0296  0.0320  -0.0555 -0.0447 0.0014  0.0010  -0.0499
POOL83_08RL     POOL83_08RL     0       0       0       2       NA      NA      NA      8       0.0157  0.0095  -0.0198 -0.0248 -0.0206 -0.0242 0.0153  0.0448  0.0243  0.0067
POOL77_08RL     POOL77_08RL     0       0       0       2       NA      NA      NA      8       0.0143  -0.0026 -0.0279 -0.0273 0.0338  -0.0194 0.0555  0.0160  -0.0053 0.0202
POOL02_03RL     POOL02_03RL     0       0       0       2       2       NA      NA      3       0.0136  0.0057  0.0166  0.0415  -0.0098 0.0027  -0.0120 -0.0008 -0.0187 0.0112
POOL33_03RL     POOL33_03RL     0       0       0       1       2       NA      NA      3       0.0137  0.0162  -0.0260 -0.0403 -0.0116 -0.0230 -0.0308 0.0320  -0.0137 -0.0560
POOL157_04RL    POOL157_04RL    0       0       0       2       NA      NA      NA      4       0.0128  0.0025  -0.0309 0.0374  -0.0126 0.0054  0.0047  0.0157  -0.0291 0.0530
POOL88_02RL     POOL88_02RL     0       0       0       2       2       NA      NA      2       0.0118  0.0045  -0.0299 0.0170  0.0068  -0.0149 -0.0094 -0.0493 -0.0122 -0.0149
POOL109_11RL    POOL109_11RL    0       0       0       1       2       NA      NA      11      0.0160  0.0021  -0.0172 -0.0269 -0.0312 0.0343  0.0326  -0.0096 -0.0062 0.0452
POOL158_08RL    POOL158_08RL    0       0       0       1       2       1       NA      8       0.0106  0.0138  0.0268  -0.0544 0.0063  -0.0188 -0.0086 0.0534  -0.0106 -0.0414
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous]$head -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm5.vs3.ped 
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL98_01RL     POOL98_01RL     0       0       0       1       NA      NA      NA      1       0.0079  0.0160  0.0720  0.0296  0.0320  -0.0555 -0.0447 0.0014  0.0010  -0.0499
POOL83_08RL     POOL83_08RL     0       0       0       2       NA      NA      NA      8       0.0157  0.0095  -0.0198 -0.0248 -0.0206 -0.0242 0.0153  0.0448  0.0243  0.0067
POOL77_08RL     POOL77_08RL     0       0       0       1       NA      NA      NA      8       0.0143  -0.0026 -0.0279 -0.0273 0.0338  -0.0194 0.0555  0.0160  -0.0053 0.0202
POOL02_03RL     POOL02_03RL     0       0       0       2       1       NA      NA      3       0.0136  0.0057  0.0166  0.0415  -0.0098 0.0027  -0.0120 -0.0008 -0.0187 0.0112
POOL33_03RL     POOL33_03RL     0       0       0       1       2       NA      NA      3       0.0137  0.0162  -0.0260 -0.0403 -0.0116 -0.0230 -0.0308 0.0320  -0.0137 -0.0560
POOL157_04RL    POOL157_04RL    0       0       0       2       NA      NA      NA      4       0.0128  0.0025  -0.0309 0.0374  -0.0126 0.0054  0.0047  0.0157  -0.0291 0.0530
POOL88_02RL     POOL88_02RL     0       0       0       1       1       NA      NA      2       0.0118  0.0045  -0.0299 0.0170  0.0068  -0.0149 -0.0094 -0.0493 -0.0122 -0.0149
POOL109_11RL    POOL109_11RL    0       0       0       1       2       NA      NA      11      0.0160  0.0021  -0.0172 -0.0269 -0.0312 0.0343  0.0326  -0.0096 -0.0062 0.0452
POOL158_08RL    POOL158_08RL    0       0       0       2       2       1       NA      8       0.0106  0.0138  0.0268  -0.0544 0.0063  -0.0188 -0.0086 0.0534  -0.0106 -0.0414
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous]$tail -n 150 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.vs3.ped | head -n 10
POOL103_11RL    POOL103_11RL    0       0       0       1       2       NA      NA      11      0.0122  0.0064  0.0138  0.0237  0.0555  -0.0011 0.0001  0.0305  -0.0344 -0.0242
POOL81_01RL     POOL81_01RL     0       0       0       2       NA      NA      NA      1       0.0130  0.0086  -0.0249 -0.0269 -0.0324 -0.0338 -0.0382 0.0230  -0.0032 -0.0139
POOL19_04RL     POOL19_04RL     0       0       0       1       2       NA      NA      4       0.0145  0.0030  -0.0206 -0.0074 0.0485  -0.0191 -0.0113 0.0336  0.0099  0.0178
POOL13_09RL     POOL13_09RL     0       0       0       2       2       NA      NA      9       0.0094  0.0054  0.0615  0.0461  -0.0008 -0.0113 0.0540  0.0077  0.0227  0.0199
POOL14S_01RL    POOL14S_01RL    0       0       0       1       1       2       1       1       0.0115  0.0025  -0.0258 0.0321  0.0069  0.0175  -0.0102 0.0213  0.0161  -0.0020
POOL95_02RL     POOL95_02RL     0       0       0       1       2       NA      NA      2       0.0183  0.0126  -0.0216 -0.0514 0.0546  -0.0125 -0.0034 0.0159  -0.0034 0.0221
POOL94_05RL     POOL94_05RL     0       0       0       1       2       NA      NA      5       0.0116  0.0147  -0.0302 -0.0024 0.0043  0.0400  0.0272  -0.0212 0.0052  0.0120
POOL64_08RL     POOL64_08RL     0       0       0       1       NA      NA      NA      8       0.0131  0.0066  0.0177  -0.0510 -0.0403 -0.0053 0.0177  0.0024  -0.0263 0.0119
POOL116_06RL    POOL116_06RL    0       0       0       1       NA      NA      NA      6       0.0141  0.0072  -0.0259 -0.0046 0.0272  0.0067  -0.0147 0.0317  0.0193  0.0015
POOL90_10RL     POOL90_10RL     0       0       0       2       1       NA      NA      10      0.0148  0.0135  0.0242  -0.0202 -0.0293 0.0450  -0.0129 0.0227  0.0005  0.0110
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous]$tail -n 150 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm2.vs3.ped | head -n 10
POOL103_11RL    POOL103_11RL    0       0       0       2       2       NA      NA      11      0.0122  0.0064  0.0138  0.0237  0.0555  -0.0011 0.0001  0.0305  -0.0344 -0.0242
POOL81_01RL     POOL81_01RL     0       0       0       1       NA      NA      NA      1       0.0130  0.0086  -0.0249 -0.0269 -0.0324 -0.0338 -0.0382 0.0230  -0.0032 -0.0139
POOL19_04RL     POOL19_04RL     0       0       0       1       2       NA      NA      4       0.0145  0.0030  -0.0206 -0.0074 0.0485  -0.0191 -0.0113 0.0336  0.0099  0.0178
POOL13_09RL     POOL13_09RL     0       0       0       1       2       NA      NA      9       0.0094  0.0054  0.0615  0.0461  -0.0008 -0.0113 0.0540  0.0077  0.0227  0.0199
POOL14S_01RL    POOL14S_01RL    0       0       0       2       1       2       1       1       0.0115  0.0025  -0.0258 0.0321  0.0069  0.0175  -0.0102 0.0213  0.0161  -0.0020
POOL95_02RL     POOL95_02RL     0       0       0       1       2       NA      NA      2       0.0183  0.0126  -0.0216 -0.0514 0.0546  -0.0125 -0.0034 0.0159  -0.0034 0.0221
POOL94_05RL     POOL94_05RL     0       0       0       2       2       NA      NA      5       0.0116  0.0147  -0.0302 -0.0024 0.0043  0.0400  0.0272  -0.0212 0.0052  0.0120
POOL64_08RL     POOL64_08RL     0       0       0       2       NA      NA      NA      8       0.0131  0.0066  0.0177  -0.0510 -0.0403 -0.0053 0.0177  0.0024  -0.0263 0.0119
POOL116_06RL    POOL116_06RL    0       0       0       2       NA      NA      NA      6       0.0141  0.0072  -0.0259 -0.0046 0.0272  0.0067  -0.0147 0.0317  0.0193  0.0015
POOL90_10RL     POOL90_10RL     0       0       0       1       1       NA      NA      10      0.0148  0.0135  0.0242  -0.0202 -0.0293 0.0450  -0.0129 0.0227  0.0005  0.0110
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous]$tail -n 150 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm3.vs3.ped | head -n 10
POOL103_11RL    POOL103_11RL    0       0       0       2       1       NA      NA      11      0.0122  0.0064  0.0138  0.0237  0.0555  -0.0011 0.0001  0.0305  -0.0344 -0.0242
POOL81_01RL     POOL81_01RL     0       0       0       1       NA      NA      NA      1       0.0130  0.0086  -0.0249 -0.0269 -0.0324 -0.0338 -0.0382 0.0230  -0.0032 -0.0139
POOL19_04RL     POOL19_04RL     0       0       0       2       2       NA      NA      4       0.0145  0.0030  -0.0206 -0.0074 0.0485  -0.0191 -0.0113 0.0336  0.0099  0.0178
POOL13_09RL     POOL13_09RL     0       0       0       1       2       NA      NA      9       0.0094  0.0054  0.0615  0.0461  -0.0008 -0.0113 0.0540  0.0077  0.0227  0.0199
POOL14S_01RL    POOL14S_01RL    0       0       0       1       2       2       1       1       0.0115  0.0025  -0.0258 0.0321  0.0069  0.0175  -0.0102 0.0213  0.0161  -0.0020
POOL95_02RL     POOL95_02RL     0       0       0       2       2       NA      NA      2       0.0183  0.0126  -0.0216 -0.0514 0.0546  -0.0125 -0.0034 0.0159  -0.0034 0.0221
POOL94_05RL     POOL94_05RL     0       0       0       1       2       NA      NA      5       0.0116  0.0147  -0.0302 -0.0024 0.0043  0.0400  0.0272  -0.0212 0.0052  0.0120
POOL64_08RL     POOL64_08RL     0       0       0       2       NA      NA      NA      8       0.0131  0.0066  0.0177  -0.0510 -0.0403 -0.0053 0.0177  0.0024  -0.0263 0.0119
POOL116_06RL    POOL116_06RL    0       0       0       1       NA      NA      NA      6       0.0141  0.0072  -0.0259 -0.0046 0.0272  0.0067  -0.0147 0.0317  0.0193  0.0015
POOL90_10RL     POOL90_10RL     0       0       0       1       2       NA      NA      10      0.0148  0.0135  0.0242  -0.0202 -0.0293 0.0450  -0.0129 0.0227  0.0005  0.0110
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous]$tail -n 150 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm4.vs3.ped | head -n 10
POOL103_11RL    POOL103_11RL    0       0       0       1       2       NA      NA      11      0.0122  0.0064  0.0138  0.0237  0.0555  -0.0011 0.0001  0.0305  -0.0344 -0.0242
POOL81_01RL     POOL81_01RL     0       0       0       1       NA      NA      NA      1       0.0130  0.0086  -0.0249 -0.0269 -0.0324 -0.0338 -0.0382 0.0230  -0.0032 -0.0139
POOL19_04RL     POOL19_04RL     0       0       0       2       2       NA      NA      4       0.0145  0.0030  -0.0206 -0.0074 0.0485  -0.0191 -0.0113 0.0336  0.0099  0.0178
POOL13_09RL     POOL13_09RL     0       0       0       2       1       NA      NA      9       0.0094  0.0054  0.0615  0.0461  -0.0008 -0.0113 0.0540  0.0077  0.0227  0.0199
POOL14S_01RL    POOL14S_01RL    0       0       0       1       1       2       2       1       0.0115  0.0025  -0.0258 0.0321  0.0069  0.0175  -0.0102 0.0213  0.0161  -0.0020
POOL95_02RL     POOL95_02RL     0       0       0       1       1       NA      NA      2       0.0183  0.0126  -0.0216 -0.0514 0.0546  -0.0125 -0.0034 0.0159  -0.0034 0.0221
POOL94_05RL     POOL94_05RL     0       0       0       2       1       NA      NA      5       0.0116  0.0147  -0.0302 -0.0024 0.0043  0.0400  0.0272  -0.0212 0.0052  0.0120
POOL64_08RL     POOL64_08RL     0       0       0       1       NA      NA      NA      8       0.0131  0.0066  0.0177  -0.0510 -0.0403 -0.0053 0.0177  0.0024  -0.0263 0.0119
POOL116_06RL    POOL116_06RL    0       0       0       2       NA      NA      NA      6       0.0141  0.0072  -0.0259 -0.0046 0.0272  0.0067  -0.0147 0.0317  0.0193  0.0015
POOL90_10RL     POOL90_10RL     0       0       0       1       1       NA      NA      10      0.0148  0.0135  0.0242  -0.0202 -0.0293 0.0450  -0.0129 0.0227  0.0005  0.0110
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous]$tail -n 150 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm5.vs3.ped | head -n 10
POOL103_11RL    POOL103_11RL    0       0       0       1       2       NA      NA      11      0.0122  0.0064  0.0138  0.0237  0.0555  -0.0011 0.0001  0.0305  -0.0344 -0.0242
POOL81_01RL     POOL81_01RL     0       0       0       1       NA      NA      NA      1       0.0130  0.0086  -0.0249 -0.0269 -0.0324 -0.0338 -0.0382 0.0230  -0.0032 -0.0139
POOL19_04RL     POOL19_04RL     0       0       0       2       2       NA      NA      4       0.0145  0.0030  -0.0206 -0.0074 0.0485  -0.0191 -0.0113 0.0336  0.0099  0.0178
POOL13_09RL     POOL13_09RL     0       0       0       2       2       NA      NA      9       0.0094  0.0054  0.0615  0.0461  -0.0008 -0.0113 0.0540  0.0077  0.0227  0.0199
POOL14S_01RL    POOL14S_01RL    0       0       0       2       2       1       1       1       0.0115  0.0025  -0.0258 0.0321  0.0069  0.0175  -0.0102 0.0213  0.0161  -0.0020
POOL95_02RL     POOL95_02RL     0       0       0       1       2       NA      NA      2       0.0183  0.0126  -0.0216 -0.0514 0.0546  -0.0125 -0.0034 0.0159  -0.0034 0.0221
POOL94_05RL     POOL94_05RL     0       0       0       2       2       NA      NA      5       0.0116  0.0147  -0.0302 -0.0024 0.0043  0.0400  0.0272  -0.0212 0.0052  0.0120
POOL64_08RL     POOL64_08RL     0       0       0       2       NA      NA      NA      8       0.0131  0.0066  0.0177  -0.0510 -0.0403 -0.0053 0.0177  0.0024  -0.0263 0.0119
POOL116_06RL    POOL116_06RL    0       0       0       1       NA      NA      NA      6       0.0141  0.0072  -0.0259 -0.0046 0.0272  0.0067  -0.0147 0.0317  0.0193  0.0015
POOL90_10RL     POOL90_10RL     0       0       0       2       2       NA      NA      10      0.0148  0.0135  0.0242  -0.0202 -0.0293 0.0450  -0.0129 0.0227  0.0005  0.0110
~~~

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS

tar -czvf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.permAll.Vs2.epacts.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm*.epacts
tar -czvf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.permAll.Vs2.epacts.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm*.epacts
tar -czvf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.permAll.Vs2.epacts.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm*.epacts
tar -czvf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.permAll.Vs2.epacts.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm*.epacts
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm*.epacts
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm*.epacts
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm*.epacts
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm*.epacts

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/*Pheno1*perm* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/*Pheno2*perm* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/*Pheno3*perm* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/*Pheno4*perm* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/.

rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*Makefile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*ind
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*Makefile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*ind
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*Makefile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*ind
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*Makefile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*ind

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm1.epacts.GeneIDList
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm1.epacts.GeneIDList
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm1.epacts.GeneIDList

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList FakePheno Pheno1 Exonic.Nonsynonymous 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm1.epacts.GeneIDList FakePheno Pheno2 Exonic.Nonsynonymous 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm1.epacts.GeneIDList FakePheno Pheno3 Exonic.Nonsynonymous 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm1.epacts.GeneIDList FakePheno Pheno4 Exonic.Nonsynonymous 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output | grep -wv NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output | grep -wv NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output | grep -wv NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output | grep -wv NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output | grep -wv NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output | grep -wv NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output | grep -wv NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output | grep -wv NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd))); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.histograms.jpg\", width=2500, height=2000, res=300); print(Data1[14,]); par(mfrow=c(2,2)); hist(as.numeric(Data1[40,])); hist(as.numeric(Data1[156,])); hist(as.numeric(Data1[261,])); hist(as.numeric(Data1[345,])); dev.off();" > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.vs2.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd))); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.histograms.jpg\", width=2500, height=2000, res=300); print(Data1[14,]); par(mfrow=c(2,2)); hist(as.numeric(Data1[40,])); hist(as.numeric(Data1[156,])); hist(as.numeric(Data1[261,])); hist(as.numeric(Data1[345,])); dev.off();" > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.vs2.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd))); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.histograms.jpg\", width=2500, height=2000, res=300); print(Data1[14,]); par(mfrow=c(2,2)); hist(as.numeric(Data1[40,])); hist(as.numeric(Data1[156,])); hist(as.numeric(Data1[261,])); hist(as.numeric(Data1[345,])); dev.off();" > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.vs2.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd))); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.histograms.jpg\", width=2500, height=2000, res=300); print(Data1[14,]); par(mfrow=c(2,2)); hist(as.numeric(Data1[40,])); hist(as.numeric(Data1[156,])); hist(as.numeric(Data1[261,])); hist(as.numeric(Data1[345,])); dev.off();" > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.vs2.Routput

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd)));" | grep -v ^\> | sort -rg -k 2,2 | head -n -1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); Data2 <- -log10(Data1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd), apply(Data2,1,mean), apply(Data2,1,sd)));" | grep -v ^\> | sort -rg -k 2,2 | head -n -1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd)));" | grep -v ^\> | sort -rg -k 2,2 | head -n -1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd)));" | grep -v ^\> | sort -rg -k 2,2 | head -n -1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd)));" | grep -v ^\> | sort -rg -k 2,2 | head -n -1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd))); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.histograms.jpg\", width=2500, height=2000, res=300); print(Data1[14,]); par(mfrow=c(2,2)); hist(as.numeric(Data1[40,])); hist(as.numeric(Data1[156,])); hist(as.numeric(Data1[261,])); hist(as.numeric(Data1[345,])); dev.off();" > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd))); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.histograms.jpg\", width=2500, height=2000, res=300); print(Data1[14,]); par(mfrow=c(2,2)); hist(as.numeric(Data1[40,])); hist(as.numeric(Data1[156,])); hist(as.numeric(Data1[261,])); hist(as.numeric(Data1[345,])); dev.off();" > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd))); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.histograms.jpg\", width=2500, height=2000, res=300); print(Data1[14,]); par(mfrow=c(2,2)); hist(as.numeric(Data1[40,])); hist(as.numeric(Data1[156,])); hist(as.numeric(Data1[261,])); hist(as.numeric(Data1[345,])); dev.off();" > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd))); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.histograms.jpg\", width=2500, height=2000, res=300); print(Data1[14,]); par(mfrow=c(2,2)); hist(as.numeric(Data1[40,])); hist(as.numeric(Data1[156,])); hist(as.numeric(Data1[261,])); hist(as.numeric(Data1[345,])); dev.off();" > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput

#Create 1..1000 loop that takes each perm# file and pastes together just the p-value for a given 'rank' after removing NAs and sorting so that 'most significant gene p-value' can be averaged across permutations, as can 2nd most significant, 3rd, 4th, etc...

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.sh FakePheno Pheno1 Exonic.Nonsynonymous 1 1000 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.sh FakePheno Pheno2 Exonic.Nonsynonymous 1 1000 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.sh FakePheno Pheno3 Exonic.Nonsynonymous 1 1000 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.sh FakePheno Pheno4 Exonic.Nonsynonymous 1 1000 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output | grep -vw NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output | grep -vw NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output | grep -vw NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output | grep -vw NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd)));" | grep -v ^\> | tail -n +2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); Data2 <- -log10(Data1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd), apply(Data2,1,mean), apply(Data2,1,sd)));" | grep -v ^\> | tail -n +2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd)));" | grep -v ^\> | tail -n +2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd)));" | grep -v ^\> | tail -n +2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd)));" | grep -v ^\> | tail -n +2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput

paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput | vi -
paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput | vi -
paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput | vi -
paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput | vi -

#<(paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput | awk '{ print $1, "\t", $2 }') /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs)

mkdir /data/userdata/pg/michaelt/Data/TempStuff

paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput | awk '{ print $1, "\t", $2 }') /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs > /data/userdata/pg/michaelt/Data/TempStuff/asdubsdbasd

#Transferring file to desktop to figure out boxplot commands in R
scp -p /data/userdata/pg/michaelt/Data/TempStuff/asdubsdbasd mturchin20@wolfy.uchicago.edu: mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/TempStuff/.

~~~
Data1 <- read.table("asdubsdbasd", header=FALSE, row.names=1)
 Data2 <- -log10(Data1)
#  boxplot(t(Data2[(nrow(Data2)-50):nrow(Data2),2:ncol(Data2)]))
 boxplot(t(Data2[(nrow(Data2)-50):nrow(Data2),2:ncol(Data2)]), ylab="-log10(p-value)", xlab="True Value Gene @ Rank", main="Boxplots of 1000 Permutations per Rank")
 legend("topleft", c("True Value"), lty=1, lwd=3, col="RED")
#  points(1:(nrow(Data2)-(nrow(Data2)-50)), Data2[(nrow(Data2)-49):nrow(Data2),1], col="RED")
  points(1:(nrow(Data2)-(nrow(Data2)-51)), Data2[(nrow(Data2)-50):nrow(Data2),1], col="RED")
#  lines(1:(nrow(Data2)-(nrow(Data2)-51)), Data2[(nrow(Data2)-48):nrow(Data2),1], col="RED", lwd=3)
  lines(1:(nrow(Data2)-(nrow(Data2)-51)), Data2[(nrow(Data2)-50):nrow(Data2),1], col="RED", lwd=3)
  lines(1:(nrow(Data2)-(nrow(Data2)-150)), Data2[(nrow(Data2)-149):nrow(Data2),1], col="RED", lwd=3)

~~~

#Coming up with basic FDR script
cp -p /home/michaelt/Scripts/R/BasicScript.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.GetFDRValsPerGeneFromPermutations.vs1.R

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.GetFDRValsPerGeneFromPermutations.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm*.epacts | awk '{ print $10 }' | grep -v -E 'NA|PVALUE' | sort -rg -k 1,1)

#CHECK_0: Check total number of permutation and 'true phenotype' runs ran and were completed
#CHECK_0: And if doing FDR calculations correctly
#CHECK_0: Try running SKAT_O using pathway annotations from msigdb as collection of variants at once instead of per gene than collapsing for GSEA
#CHECK_0: Remember to permute via phenotype and not the internal 'gene set' that occurs via what is seen here http://www.broadinstitute.org/cancer/software/genepattern/modules/docs/GSEAPreranked/1 (Permutation test: In GSEAPreranked, permutations are always done by gene set. In standard GSEA, you can choose to set the parameter Permutation type to phenotype (the default) or gene set, but GSEAPreranked does not provide this option.)...
#... and what is read here http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1239896/ (Step 2: Estimation of Significance Level of ES. We estimate the statistical significance (nominal P value) of the ES by using an empirical phenotype-based permutation test procedure that preserves the complex correlation structure of the gene expression data. Specifically, we permute the phenotype labels and recompute the ES of the gene set for the permuted data, which generates a null distribution for the ES. The empirical, nominal P value of the observed ES is then calculated relative to this null distribution. Importantly, the permutation of class labels preserves gene-gene correlations and, thus, provides a more biologically reasonable assessment of significance than would be obtained by permuting genes)
#... also from same paper but later on (Genes may be ranked based on the differences seen in a small data set, with too few samples to allow rigorous evaluation of significance levels by permuting the class labels. In these cases, a P value can be estimated by permuting the genes, with the result that genes are randomly assigned to the sets while maintaining their size. This approach is not strictly accurate: because it ignores gene-gene correlations, it will overestimate the significance levels and may lead to false positives. Nonetheless, it can be useful for hypothesis generation. The gsea-p software supports this option.)
#... I can do this by just running GSEA on my phenotype-permuted SKAT-O results and ignore the internal GSEA permutation procedure and then count up the number of times certain pathways come up as significant in the permuted data versus which come up in the observed data (or at least whatever metric ends up coming out of GSEA -- as of writing these notes 20150126 I have yet to run GSEA yet so I'm not exactly sure yet what its output looks like)

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1[,2] <- as.numeric(Data1[,2]); Data1New <- cbind(Data1, signif(qvalue(Data1[,2])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1[,2] <- as.numeric(Data1[,2]); Data1New <- cbind(Data1, signif(qvalue(Data1[,2])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" 

#Figuring out how to use/run GSEA tool GSEAPreranked
mkdir /home/michaelt/Software/GSEA2-2.1.0

#Below did not work, had to manually download and ftp this file into the desired location
#wget http://www.broadinstitute.org/gsea/msigdb/download_file.jsp?filePath=/resources/software/gsea2-2.1.0.jar 
#wget http://www.broadinstitute.org/gsea/msigdb/download_file.jsp?filePath=/resources/software/gsea2_distrib-2.1.0.zip 

scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/Software/GSEA/gsea2_distrib-2.1.0.zip mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/Software/GSEA/gsea2-2.1.0.jar /home/michaelt/Software/. 


#Below from: http://www.broadinstitute.org/gsea/doc/GSEAUserGuideTEXT.htm#_Running_GSEA_from (accessed 20150125)
~~~
To run GSEA from the command line, use a java command of the form:

java -cp full-path/gsea2.jar -Xmx512m gsea-tool  parameters

-cp

Points the CLASSPATH variable to the complete path of the gsea2.jar file. You do not need to set any other CLASSPATH variables.

For Broad internal users, the latest .jar file is available at /xchip/projects/xtools/gsea2.jar.
~~~

mkdir /home/michaelt/Data/GSEA

#Below did not work, had to manually download and ftp this file into the desired location
#wget http://www.broadinstitute.org/gsea/msigdb/download_file.jsp?filePath=/resources/msigdb/4.0/msigdb.v4.0.symbols.gmt

mkdir /home/michaelt/Data/GSEA/Testing

#Want to test that all the genes annotated exist within the database files being used

[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList | wc
    475     475    3006

for i in {2..475}
do

	gene1=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList | awk -v geneRow=$i '{ if (NR == geneRow) { print $1 } }'`
#	echo $gene1
	wc1=`cat /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt | grep $gene1 | wc`

	echo $gene1 $wc1 >> /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.TestingForPresenceIn.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList

	gene1=`cat /dev/null`
	wc1=`cat /dev/null`

done

#Very few genes are entirely not present in the msigdb pathway files
cat /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.TestingForPresenceIn.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList | sort -rg -k 2,2 | tail -n 20

~~~
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations]$cat /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.TestingForPresenceIn.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList | sort -rg -k 2,2 | tail -n 20
CKMT1A 6 1765 11599
TMEM192 5 2358 15560
PEX10 5 1584 10555
NEURL4 5 3959 25757
MITD1 5 3505 22799
ERLEC1 5 3885 25575
ATAD3B 5 3703 24402
MRPS9 4 4668 30029
LENEP 4 1999 13326
PUSL1 3 1681 11277
NRN1L 2 1563 10434
TMPRSS9 1 80 611
C14orf178 1 1841 12520
PPAN-P2RY11 0 0 0
GIMAP1-GIMAP5 0 0 0
EMC1 0 0 0
COLGALT1 0 0 0
CCDC73 0 0 0
CCAR2 0 0 0
ANKHD1-EIF4EBP3 0 0 0
~~~

#20150126
#Redoing EPACTS analyses after updating the bgzip file for the main .vcf file

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz | awk '{ if ($1 == "23") { $1 = "X"} print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip

/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz.tbi /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ORIG.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz.tbi

/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz

#20150129 -- if redoing things after this date use the .RMID5PCs ending as well....not sure why this no longer got included by technically it should be because it's true
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS 
#bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs 

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS
#bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs

tar -czvf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.runAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run*.epacts
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run*.epacts
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.histograms.jpg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.histograms.jpg
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.vs2.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.vs2.Routput
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/.
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run*ind /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run*Makefile

tar -czvf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/OLD2.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.runAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run*.epacts
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run*.epacts
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.histograms.jpg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.histograms.jpg
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.vs2.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.vs2.Routput
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run1.epacts.GeneIDList /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/OLD2.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run1.epacts.GeneIDList
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/.
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run*ind /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run*Makefile

tar -czvf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/OLD2.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.runAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run*.epacts
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run*.epacts
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.histograms.jpg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.histograms.jpg
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.vs2.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.vs2.Routput
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run1.epacts.GeneIDList /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/OLD2.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run1.epacts.GeneIDList
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/.
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run*ind /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run*Makefile

tar -czvf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/OLD2.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.runAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run*.epacts
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run*.epacts
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.histograms.jpg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.histograms.jpg
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.vs2.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.vs2.Routput
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run1.epacts.GeneIDList /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/OLD2.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run1.epacts.GeneIDList
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/.
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run*ind /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run*Makefile

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run1.epacts.GeneIDList
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run1.epacts.GeneIDList
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run1.epacts.GeneIDList

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList TruePheno Pheno1 Exonic.Nonsynonymous 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run1.epacts.GeneIDList TruePheno Pheno2 Exonic.Nonsynonymous 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run1.epacts.GeneIDList TruePheno Pheno3 Exonic.Nonsynonymous 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run1.epacts.GeneIDList TruePheno Pheno4 Exonic.Nonsynonymous 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output | grep -wv NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output | grep -wv NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output | grep -wv NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output | grep -wv NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd)));" | grep -v ^\> | sort -rg -k 2,2 | head -n -1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); Data2 <- -log10(Data1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd), apply(Data2,1,mean), apply(Data2,1,sd)));" | grep -v ^\> | sort -rg -k 2,2 | head -n -1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd)));" | grep -v ^\> | sort -rg -k 2,2 | head -n -1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd)));" | grep -v ^\> | sort -rg -k 2,2 | head -n -1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd)));" | grep -v ^\> | sort -rg -k 2,2 | head -n -1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput

~~~
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations]$tail -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput
CIAO1:           0.010419630 2.948600e-04
FUT8:            0.009499187 7.184600e-04
MRPS9:           0.009020114 9.653076e-04
EDC4:            0.008973321 8.498714e-04
CUL5:            0.007770337 1.775791e-04
NUDC:            0.007347700 0.000000e+00
SDHD:            0.006449249 5.337792e-04
VDAC3:           0.002259239 3.679158e-04
VTI1A:           0.001098320 8.732696e-05
GEMIN2:          0.000775861 3.211198e-04
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations]$tail -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput
CIAO1:           0.0104004830 3.314492e-04 1.9831653159 1.386507e-02
FUT8:            0.0092845960 8.366922e-04 2.0339736736 3.900911e-02
EDC4:            0.0090311210 8.470959e-04 2.0460940574 3.986946e-02
MRPS9:           0.0089262030 9.259414e-04 2.0516160501 4.467143e-02
CUL5:            0.0077371170 1.639625e-04 2.1115180442 9.251883e-03
NUDC:            0.0073477000 0.000000e+00 2.1338485839 0.000000e+00
SDHD:            0.0063112910 4.741037e-04 2.2011037650 3.282508e-02
VDAC3:           0.0022698400 3.582224e-04 2.6499420952 7.448913e-02
VTI1A:           0.0010969036 9.371745e-05 2.9612728538 3.485794e-02
GEMIN2:          0.0007524931 3.138343e-04 3.1780469314 2.431659e-01
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations]$tail -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput
HARS:            0.0124821300 9.241060e-04
XPO4:            0.0114074700 3.335765e-04
LDLR:            0.0108962560 1.058498e-03
SDCBP:           0.0097649280 5.540537e-04
GEMIN6:          0.0080840720 9.209877e-04
ICAM2:           0.0070759810 2.665100e-04
EIF3L:           0.0051961430 9.585082e-05
KIR3DL2:         0.0051198370 7.477929e-04
EEF1E1:          0.0027584390 5.767824e-05
GEMIN2:          0.0006154692 1.240322e-04
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations]$tail -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput
HARS:            0.0124869800 8.707598e-04
XPO4:            0.0113585900 3.650333e-04
LDLR:            0.0109963250 1.387778e-03
SDCBP:           0.0098379750 5.955183e-04
GEMIN6:          0.0081037530 1.026901e-03
ICAM2:           0.0070736290 2.271870e-04
KIR3DL2:         0.0052122210 8.001052e-04
EIF3L:           0.0051817820 1.068389e-04
EEF1E1:          0.0027681430 6.012171e-05
GEMIN2:          0.0006076614 1.173707e-04
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations]$tail -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput
NCAPD3:          0.0191322700 1.209771e-03
MCL1:            0.0129071300 8.074987e-04
CAPN1:           0.0110830750 1.064156e-03
EML3:            0.0079168660 1.032934e-04
EPPK1:           0.0054144350 8.308956e-04
SDF2L1:          0.0047018430 3.062125e-04
POLE:            0.0045066320 4.149423e-04
UBR2:            0.0034429010 1.891623e-04
BICD2:           0.0016849910 1.331283e-04
AMBRA1:          0.0008004482 1.108007e-04
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations]$tail -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput
NCAPD3:          0.0192332000 1.304343e-03
MCL1:            0.0130626600 8.144170e-04
CAPN1:           0.0111352670 1.050676e-03
EML3:            0.0079202250 1.091330e-04
EPPK1:           0.0054542040 8.477160e-04
SDF2L1:          0.0045741980 3.681954e-04
POLE:            0.0043593020 5.366147e-04
UBR2:            0.0034526600 1.712316e-04
BICD2:           0.0016938370 1.359807e-04
AMBRA1:          0.0008072145 1.082477e-04
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations]$tail -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput
NFU1:            0.011664080 2.928592e-04
SPATS2:          0.010834650 3.350323e-04
EML3:            0.010516960 1.544944e-04
AQR:             0.010495000 0.000000e+00
SDCCAG8:         0.009696784 5.895933e-07
BICD2:           0.008033618 2.265361e-04
MRPS35:          0.007225927 2.890907e-06
PDIA6:           0.006179672 5.202883e-04
CLGN:            0.005013847 7.326989e-04
SUN1:            0.002010493 8.089778e-05
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations]$tail -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput
NFU1:            0.011674090 3.114970e-04
SPATS2:          0.010817890 3.217966e-04
EML3:            0.010503600 8.600000e-05
AQR:             0.010495000 0.000000e+00
SDCCAG8:         0.009750884 5.407191e-04
BICD2:           0.008031391 2.401371e-04
MRPS35:          0.007225965 2.210826e-06
PDIA6:           0.006192917 5.281613e-04
CLGN:            0.004917945 4.046539e-05
SUN1:            0.002008964 7.120811e-05
~~~

#So it doesn't look like the 'new' bgzip file changes anything, so the differences in 'bytes' are probably negligible

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1[,2] <- as.numeric(Data1[,2]); Data1New <- cbind(Data1, signif(qvalue(Data1[,2])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1[,2] <- as.numeric(Data1[,2]); Data1New <- cbind(Data1, signif(qvalue(Data1[,2])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1[,2] <- as.numeric(Data1[,2]); Data1New <- cbind(Data1, signif(qvalue(Data1[,2])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1[,2] <- as.numeric(Data1[,2]); Data1New <- cbind(Data1, signif(qvalue(Data1[,2])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" 

#CHECK_0: Still need to check total number of runs/permutations are correct numbers for both true and fake phenotypes (fake phenotype reruns are still going as of this comment) 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.HIVPROG.maf1.skat0.permAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/OLD1.AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.HIVPROG.maf1.skat0.permAll.Files.tar.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.HIVPROG.maf1.skat0.permAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/OLD1.AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.HIVPROGHE.maf1.skat0.permAll.Files.tar.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.HIVPROG.maf1.skat0.permAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/OLD1.AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.AIDS.maf1.skat0.permAll.Files.tar.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.HIVPROG.maf1.skat0.permAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/OLD1.AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.AIDSEXTR.maf1.skat0.permAll.Files.tar.gz

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.permAll.Vs2.epacts.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/OLD1.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.permAll.Files.tar.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.permAll.Vs2.epacts.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/OLD1.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.permAll.Files.tar.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.permAll.Vs2.epacts.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/OLD1.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.permAll.Files.tar.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.permAll.Vs2.epacts.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/OLD1.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.permAll.Files.tar.gz

#20150127 -- I accidentally clobbered OLD1.AllPools......Pheno1.maf1.skat0.permAll.Files.tar.gz for Phenos 1-3 because I didn't realize i hadn't changed OLD1 to OLD2 yet on the below commands. I only noticed once I was on Pheno4.
#tar -czvf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/OLD1.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.permAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm*.epacts
tar -czvf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/OLD2.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.permAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm*.epacts
#tar -czvf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/OLD1.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.permAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm*.epacts
tar -czvf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/OLD2.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.permAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm*.epacts
#tar -czvf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/OLD1.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.permAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm*.epacts
tar -czvf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/OLD2.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.permAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm*.epacts
tar -czvf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/OLD2.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.permAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm*.epacts
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm*.epacts
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm*.epacts
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm*.epacts
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm*.epacts

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.histograms.jpg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.histograms.jpg
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.histograms.jpg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.histograms.jpg
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.histograms.jpg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.histograms.jpg
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.histograms.jpg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.histograms.jpg
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/*Pheno1*perm* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/*Pheno2*perm* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/*Pheno3*perm* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/*Pheno4*perm* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/.

rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm*Makefile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm*ind
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm*Makefile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm*ind
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm*Makefile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm*ind
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm*Makefile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm*ind

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm1.epacts.GeneIDList
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm1.epacts.GeneIDList
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm1.epacts.GeneIDList

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList FakePheno Pheno1 Exonic.Nonsynonymous 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm1.epacts.GeneIDList FakePheno Pheno2 Exonic.Nonsynonymous 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm1.epacts.GeneIDList FakePheno Pheno3 Exonic.Nonsynonymous 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm1.epacts.GeneIDList FakePheno Pheno4 Exonic.Nonsynonymous 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output | grep -wv NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output | grep -wv NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output | grep -wv NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output | grep -wv NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd))); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.histograms.jpg\", width=2500, height=2000, res=300); print(Data1[14,]); par(mfrow=c(2,2)); hist(as.numeric(Data1[40,])); hist(as.numeric(Data1[156,])); hist(as.numeric(Data1[261,])); hist(as.numeric(Data1[345,])); dev.off();" > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd))); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.histograms.jpg\", width=2500, height=2000, res=300); print(Data1[14,]); par(mfrow=c(2,2)); hist(as.numeric(Data1[40,])); hist(as.numeric(Data1[156,])); hist(as.numeric(Data1[261,])); hist(as.numeric(Data1[345,])); dev.off();" > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd))); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.histograms.jpg\", width=2500, height=2000, res=300); print(Data1[14,]); par(mfrow=c(2,2)); hist(as.numeric(Data1[40,])); hist(as.numeric(Data1[156,])); hist(as.numeric(Data1[261,])); hist(as.numeric(Data1[345,])); dev.off();" > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd))); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.histograms.jpg\", width=2500, height=2000, res=300); print(Data1[14,]); par(mfrow=c(2,2)); hist(as.numeric(Data1[40,])); hist(as.numeric(Data1[156,])); hist(as.numeric(Data1[261,])); hist(as.numeric(Data1[345,])); dev.off();" > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.sh FakePheno Pheno1 Exonic.Nonsynonymous 1 1000 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.sh FakePheno Pheno2 Exonic.Nonsynonymous 1 1000 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.sh FakePheno Pheno3 Exonic.Nonsynonymous 1 1000 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.sh FakePheno Pheno4 Exonic.Nonsynonymous 1 1000 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output | grep -vw NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output | grep -vw NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output | grep -vw NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output | grep -vw NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd)));" | grep -v ^\> | tail -n +2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); Data2 <- -log10(Data1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd), apply(Data2,1,mean), apply(Data2,1,sd)));" | grep -v ^\> | tail -n +2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd)));" | grep -v ^\> | tail -n +2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd)));" | grep -v ^\> | tail -n +2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd)));" | grep -v ^\> | tail -n +2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput

paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput | vi -
paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput | vi -
paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput | vi -
paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput | vi -

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz --missing-indv --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD

~~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.imiss | sort -rg -k 5,5 | head -n 10
POOL79_12RL     63887   0       33197   0.519621
POOL169_05RL    63887   0       28779   0.450467
POOL88_02RL     63887   0       12352   0.193341
POOL94_10RL     63887   0       6047    0.0946515
POOL116_08RL    63887   0       5812    0.0909731
POOL81_03RL     63887   0       4761    0.0745222
POOL168_10RL    63887   0       4084    0.0639254
POOL115_08RL    63887   0       3676    0.0575391
POOL95_02RL     63887   0       3579    0.0560208
POOL94_08RL     63887   0       3452    0.0540329
~~~

#20150206 Copied from before...
#CHECK_0: Check total number of permutation and 'true phenotype' runs ran and were completed
#CHECK_0: And if doing FDR calculations correctly
#CHECK_0: Try running SKAT_O using pathway annotations from msigdb as collection of variants at once instead of per gene than collapsing for GSEA
#CHECK_0: Remember to permute via phenotype and not the internal 'gene set' that occurs via what is seen here http://www.broadinstitute.org/cancer/software/genepattern/modules/docs/GSEAPreranked/1 (Permutation test: In GSEAPreranked, permutations are always done by gene set. In standard GSEA, you can choose to set the parameter Permutation type to phenotype (the default) or gene set, but GSEAPreranked does not provide this option.)...

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.GetFDRValsPerGeneFromPermutations.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm*.epacts | awk '{ print $10 }' | grep -v -E 'NA|PVALUE' | sort -rg -k 1,1) > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.GetFDRValsPerGeneFromPermutations.vs1.output

#From http://www.bioconductor.org/packages/release/bioc/vignettes/RamiGO/inst/doc/RamiGO.R
~~~
###################################################
### code chunk number 9: gseaexec (eval = FALSE)
###################################################
## ## paths to gsea jar and gmt file
## exe.path <- exe.path.string
## gmt.path <- gmt.path.string
## 
## gsea.collapse <- "false"
## ## number of permutations
## nperm <- 10000
## gsea.seed <- 54321
## gsea.out <- "out-folder"
## 
## ## build GSEA command
## gsea.report <- "report-file"
## rnk.path <- "rank-file"
## gsea.cmd <- sprintf("java -Xmx4g -cp %s xtools.gsea.GseaPreranked -gmx %s -collapse %s -nperm %i -rnk %s -scoring_scheme weighted -rpt_label %s -include_only_symbols true -make_sets true -plot_top_x 75 -rnd_seed %i -set_max 500 -set_min 15 -zip_report true -out %s -gui false", exe.path, gmt.path, gsea.collapse, nperm, rnk.path, gsea.report, gsea.seed, gsea.out)
## 
## ## execute command on the system
## system(gsea.cmd)

~~~

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput | R -q -e "Data1 <- read.table(file('stdin'), header=F); write.table(matrix(c(as.character(Data1[,1]), -log10(Data1[,2])), ncol=2), quote=FALSE, row.name=FALSE, col.name=FALSE, sep=\"\t\")" | grep -v ^\> > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.rnk
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput | R -q -e "Data1 <- read.table(file('stdin'), header=F); write.table(matrix(c(as.character(Data1[,1]), -log10(Data1[,2])), ncol=2), quote=FALSE, row.name=FALSE, col.name=FALSE, sep=\"\t\")" | grep -v ^\> > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.rnk
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput | R -q -e "Data1 <- read.table(file('stdin'), header=F); write.table(matrix(c(as.character(Data1[,1]), -log10(Data1[,2])), ncol=2), quote=FALSE, row.name=FALSE, col.name=FALSE, sep=\"\t\")" | grep -v ^\> > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.rnk
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput | R -q -e "Data1 <- read.table(file('stdin'), header=F); write.table(matrix(c(as.character(Data1[,1]), -log10(Data1[,2])), ncol=2), quote=FALSE, row.name=FALSE, col.name=FALSE, sep=\"\t\")" | grep -v ^\> > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.rnk

scp -p /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.rnk mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/Software/GSEA/. 

#java -Xmx512m -cp /home/michaelt/Software/GSEA2-2.1.0/gsea2-2.1.0.jar xtools.gsea.GseaPreranked -gmx /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt -rnk /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.rnk -nperm 0 -rnd_seed 432142 -set_min 5 -collapse false -make_sets false -include_only_symbols true -gui false -plot_top_x 0 -zip_report true -rpt_label blah2 -out /home/michaelt/Data/GSEA/Testing/blah2

java -Xmx512m -cp /home/michaelt/Software/GSEA2-2.1.0/gsea2-2.1.0.jar xtools.gsea.GseaPreranked -gmx /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt -rnk /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.rnk -nperm 0 -rnd_seed 432142 -set_min 5 -collapse false -make_sets false -include_only_symbols true -gui false -plot_top_x 0 -zip_report false -rpt_label DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput -out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/GSEA
java -Xmx512m -cp /home/michaelt/Software/GSEA2-2.1.0/gsea2-2.1.0.jar xtools.gsea.GseaPreranked -gmx /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt -rnk /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.rnk -nperm 0 -rnd_seed 432142 -set_min 5 -collapse false -make_sets false -include_only_symbols true -gui false -plot_top_x 0 -zip_report false -rpt_label DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput -out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/GSEA 
java -Xmx512m -cp /home/michaelt/Software/GSEA2-2.1.0/gsea2-2.1.0.jar xtools.gsea.GseaPreranked -gmx /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt -rnk /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.rnk -nperm 0 -rnd_seed 432142 -set_min 5 -collapse false -make_sets false -include_only_symbols true -gui false -plot_top_x 0 -zip_report false -rpt_label DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput -out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/GSEA 
java -Xmx512m -cp /home/michaelt/Software/GSEA2-2.1.0/gsea2-2.1.0.jar xtools.gsea.GseaPreranked -gmx /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt -rnk /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.rnk -nperm 0 -rnd_seed 432142 -set_min 5 -collapse false -make_sets false -include_only_symbols true -gui false -plot_top_x 0 -zip_report false -rpt_label DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput -out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/GSEA 

#rm -r /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked.1423276412751
#rm -r /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked.1423276650788/
#rm -r /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked.1423276655777/
#rm -r /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked.1423276660583/

#To easily get the path/file names for the mv commands below did these two commands to get everthing on the screen
#ls -lrt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/GSEA/.
#ls -lrt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked.1423280266219/.

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked.1423280261353/gsea_report_for_na_pos_1423280261353.xls /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked.1423280261353/gsea_report_for_na_pos.xls ; mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked.1423280261353 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked.1423280266219/gsea_report_for_na_pos_1423280266219.xls /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked.1423280266219/gsea_report_for_na_pos.xls ; mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked.1423280266219/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked.1423280271143/gsea_report_for_na_pos_1423280271143.xls /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked.1423280271143/gsea_report_for_na_pos.xls ; mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked.1423280271143/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos_1423280275943.xls /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked.1423280275943/gsea_report_for_na_pos.xls ; mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked.1423280275943/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/

for i in {1..1000}
do
	echo $i
	cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm${i}.epacts | awk '{ if ($10 != "NA") { print $0 } }' | perl -lane 'my @vals1 = split(/_/, $F[3]); $F[3] = $vals1[1]; print join("\t", @F);' | R -q -e "Data1 <- read.table(file('stdin'), header=F); write.table(matrix(c(as.character(Data1[,4]), -log10(Data1[,10])), ncol=2), quote=FALSE, row.name=FALSE, col.name=FALSE, sep=\"\t\")" | grep -v ^\> > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm${i}.GSEA.rnk
done
for i in {1..1000}
do
	echo $i
	cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm${i}.epacts | awk '{ if ($10 != "NA") { print $0 } }' | perl -lane 'my @vals1 = split(/_/, $F[3]); $F[3] = $vals1[1]; print join("\t", @F);' | R -q -e "Data1 <- read.table(file('stdin'), header=F); write.table(matrix(c(as.character(Data1[,4]), -log10(Data1[,10])), ncol=2), quote=FALSE, row.name=FALSE, col.name=FALSE, sep=\"\t\")" | grep -v ^\> > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm${i}.GSEA.rnk
done
for i in {1..1000}
do
	echo $i
	cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm${i}.epacts | awk '{ if ($10 != "NA") { print $0 } }' | perl -lane 'my @vals1 = split(/_/, $F[3]); $F[3] = $vals1[1]; print join("\t", @F);' | R -q -e "Data1 <- read.table(file('stdin'), header=F); write.table(matrix(c(as.character(Data1[,4]), -log10(Data1[,10])), ncol=2), quote=FALSE, row.name=FALSE, col.name=FALSE, sep=\"\t\")" | grep -v ^\> > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm${i}.GSEA.rnk
done
for i in {1..1000}
do
	echo $i
	cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm${i}.epacts | awk '{ if ($10 != "NA") { print $0 } }' | perl -lane 'my @vals1 = split(/_/, $F[3]); $F[3] = $vals1[1]; print join("\t", @F);' | R -q -e "Data1 <- read.table(file('stdin'), header=F); write.table(matrix(c(as.character(Data1[,4]), -log10(Data1[,10])), ncol=2), quote=FALSE, row.name=FALSE, col.name=FALSE, sep=\"\t\")" | grep -v ^\> > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm${i}.GSEA.rnk
done

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos.xls | awk '{ print $1 }' | tail -n +2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos.PathwayIDList 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos.xls | awk '{ print $1 }' | tail -n +2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos.PathwayIDList 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos.xls | awk '{ print $1 }' | tail -n +2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos.PathwayIDList 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos.xls | awk '{ print $1 }' | tail -n +2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos.PathwayIDList 

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.GSEA.PermutationPrerankAnalyses.Launcher.vs1.sh

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.GSEA.PermutationPrerankAnalyses.vs1.sh

#cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.GSEA.PermutationPrerankAnalyses.vs1.sh /home/michaelt/Scripts/Bash/SubmittedJobScript.wTime.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.GSEA.PermutationPrerankAnalyses.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ FakePheno Pheno1 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.GSEA.PermutationPrerankAnalyses.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ FakePheno Pheno2 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.GSEA.PermutationPrerankAnalyses.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ FakePheno Pheno3 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.GSEA.PermutationPrerankAnalyses.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ FakePheno Pheno4 Exonic.Nonsynonymous

ls -lrt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/GSEA/. | awk '{ print $9 }' | perl -lane 'if ($F[0] =~ m/(.*)\.(\d+$)/) { my $val1 = "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/GSEA/" . $1; my $val2 = "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/GSEA/" . $1 . "." . $2; system("mv $val2 $val1") };'
ls -lrt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/GSEA/. | awk '{ print $9 }' | perl -lane 'if ($F[0] =~ m/(.*)\.(\d+$)/) { my $val1 = "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/GSEA/" . $1; my $val2 = "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/GSEA/" . $1 . "." . $2; system("mv $val2 $val1") };'
ls -lrt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/GSEA/. | awk '{ print $9 }' | perl -lane 'if ($F[0] =~ m/(.*)\.(\d+$)/) { my $val1 = "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/GSEA/" . $1; my $val2 = "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/GSEA/" . $1 . "." . $2; system("mv $val2 $val1") };'
ls -lrt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/GSEA/. | awk '{ print $9 }' | perl -lane 'if ($F[0] =~ m/(.*)\.(\d+$)/) { my $val1 = "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/GSEA/" . $1; my $val2 = "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/GSEA/" . $1 . "." . $2; system("mv $val2 $val1") };'

#cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.sh

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.py

#bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos.PathwayIDList FakePheno Pheno1 Exonic.Nonsynonymous 
#bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos.PathwayIDList FakePheno Pheno2 Exonic.Nonsynonymous 
#bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos.PathwayIDList FakePheno Pheno3 Exonic.Nonsynonymous 
#bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos.PathwayIDList FakePheno Pheno4 Exonic.Nonsynonymous 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/GSEA/DataProcessing.Pipeline.pt6.All.ptg.GSEA.PermutationPrerankAnalyses.vs1.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm*.GSEA.GseaPreranked/gsea_report_for_na_pos_*.xls > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.gsea_report_for_na_pos_All.xls.temp
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/GSEA/DataProcessing.Pipeline.pt6.All.ptg.GSEA.PermutationPrerankAnalyses.vs1.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm*.GSEA.GseaPreranked/gsea_report_for_na_pos_*.xls > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.gsea_report_for_na_pos_All.xls.temp
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/GSEA/DataProcessing.Pipeline.pt6.All.ptg.GSEA.PermutationPrerankAnalyses.vs1.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm*.GSEA.GseaPreranked/gsea_report_for_na_pos_*.xls > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.gsea_report_for_na_pos_All.xls.temp
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/GSEA/DataProcessing.Pipeline.pt6.All.ptg.GSEA.PermutationPrerankAnalyses.vs1.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm*.GSEA.GseaPreranked/gsea_report_for_na_pos_*.xls > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.gsea_report_for_na_pos_All.xls.temp

#CHECK_0:
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.gsea_report_for_na_pos_All.xls.temp | grep -w -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos.PathwayIDList > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output  
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.gsea_report_for_na_pos_All.xls.temp | grep -w -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos.PathwayIDList > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output  
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.gsea_report_for_na_pos_All.xls.temp | grep -w -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos.PathwayIDList > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output  
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.gsea_report_for_na_pos_All.xls.temp | grep -w -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos.PathwayIDList > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output  

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.py

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/SUMSTAT
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/SUMSTAT
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/SUMSTAT

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.rnk --file2 /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.rnk.SUMSTAT 
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.rnk --file2 /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.rnk.SUMSTAT 
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.rnk --file2 /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.rnk.SUMSTAT 
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.rnk --file2 /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.rnk.SUMSTAT 

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/SUMSTAT
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/SUMSTAT
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/SUMSTAT

for i in {1..1000}
do
        echo $i
	python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm${i}.GSEA.rnk --file2 /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm${i}.GSEA.rnk.SUMSTAT
done
for i in {1..1000}
do
        echo $i
	python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm${i}.GSEA.rnk --file2 /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm${i}.GSEA.rnk.SUMSTAT
done
for i in {1..1000}
do
	echo $i
	python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm${i}.GSEA.rnk --file2 /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm${i}.GSEA.rnk.SUMSTAT
done
for i in {1..1000}
do
	echo $i
	python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm${i}.GSEA.rnk --file2 /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm${i}.GSEA.rnk.SUMSTAT
done

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm*.GSEA.rnk.SUMSTAT > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.permAll.GSEA.rnk.SUMSTAT.temp
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm*.GSEA.rnk.SUMSTAT > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.permAll.GSEA.rnk.SUMSTAT.temp
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm*.GSEA.rnk.SUMSTAT > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.permAll.GSEA.rnk.SUMSTAT.temp
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm*.GSEA.rnk.SUMSTAT > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.permAll.GSEA.rnk.SUMSTAT.temp

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.vs1.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.permAll.GSEA.rnk.SUMSTAT.temp > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output 
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.permAll.GSEA.rnk.SUMSTAT.temp > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output 
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.permAll.GSEA.rnk.SUMSTAT.temp > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output 
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.permAll.GSEA.rnk.SUMSTAT.temp > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output 

cp -p /home/michaelt/Scripts/R/BasicScript.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.CalcPermBasedPVal.vs1.R

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.CalcPermBasedPVal.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.rnk.SUMSTAT /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.CalcPermBasedPVal.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output
R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.CalcPermBasedPVal.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.rnk.SUMSTAT /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.CalcPermBasedPVal.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output
R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.CalcPermBasedPVal.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.rnk.SUMSTAT /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.CalcPermBasedPVal.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output
R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.CalcPermBasedPVal.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.rnk.SUMSTAT /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.CalcPermBasedPVal.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output


#20150210
#7th run -- investigating applying a more stringent imiss threshold (just to see how it may or may not change results....this run may not be used, just investigating)
#Using .Vs3. in titles

#SNP and Indv thresholds, PCA -- get vcf file; all indvs and just white
#PLINK logistic regression stuff
#Freq and Annovar file creation; all indvs and just white
#QC stuff
#VC caller overlap?
#EPACTS skat0 runs
#permutation runs
#pathway analyses; GSEA and SUMSTAT?

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --missing --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4  

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss | awk '{ if ($5 >= .2) { print $1 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.pheno

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar | awk '{ print $1, "\t", $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar | awk '{ print $1 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDs

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDs --non-ref-ac 1 --recode  --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.recode.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80 

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.allSNPs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs 
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.10PCs
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.10PCs.maf05
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 2 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno2
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 3 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno3
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 4 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno4

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.assoc.logistic Pheno1_justWhite_Logistic_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.assoc.logistic.QQplot.jpeg & 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.assoc.logistic Pheno1_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.assoc.logistic.QQplot.jpeg & 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.10PCs.assoc.logistic Pheno1_justWhite_Logistic_10PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.10PCs.assoc.logistic.QQplot.jpeg & 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.10PCs.maf05.assoc.logistic Pheno1_justWhite_Logistic_maf05_10PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.10PCs.maf05.assoc.logistic.QQplot.jpeg & 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.assoc.logistic Pheno2_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno2.assoc.logistic.QQplot.jpeg & 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.assoc.logistic Pheno3_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno3.assoc.logistic.QQplot.jpeg & 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.assoc.logistic Pheno4_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno4.assoc.logistic.QQplot.jpeg & 

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/.

#Allele Frequency stuff/collection stuff here now

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ print $1 }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.FIDs 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($3 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroNeg.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($4 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroHighlyExposedNeg.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($3 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroPos.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justRapidVRapid.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSlowVSlow.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($6 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVeryRapid.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($6 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVerySlow.FIDs

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80 & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroNeg.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroHighlyExposedNeg.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroPos.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justRapidVRapid.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSlowVSlow.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVeryRapid.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVerySlow.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow & 

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80 & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroHighlyExposedNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroPos.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justRapidVRapid.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSlowVSlow.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVeryRapid.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVerySlow.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow & 

paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow.frq.count | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.AllPhenos.frq.AndCount.gz
gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow.frq.count
#gunzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow.frq.count.gz













##Sometime in April/May 2014?
##Putting together list of variants for West coast people from the first pass through of results

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary | grep Nonsy | tail -n 25 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary.NonSyn.top25

#Added the following header
#CATEGORY	CHROM	BEGIN	END	MARKER_ID	NS	FRAC_WITH_RARE	#Variants	#Passed	#Singletons	PVALUE	QVALUE	RHO

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary.NonSyn.top25 | perl -lane 'my @vals1 = split(/_/, $F[4]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary.NonSyn.top25.geneIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary.NonSyn.top25.geneIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.top25

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/VLookUp.Python.PullOutEPACTSVariants.vs1.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/VLookUp.Python.PullOutEPACTSVariants.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.top25 --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | cut -d, -f 1-6,18-29,35,37,39,41 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary.NonSyn.top25.Variants.csv

#Added the following header
#Chr,BP,rsID,refAllele,otherAllele,AncAllele,AC_AllIndvs,AC_SeroNeg,AC_SeroNegHE,AC_SeroPos,AC_RapidVRapid,AC_SlowVSlow,AC_VeryRapid,AC_VerySlow,VarType,GeneID,SNVType,AAChange,Sift,Polyphen2,LRT,MutationTaster

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary.NonSyn.top25 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/ForSumit.Top25.NonSynGenes
ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary.NonSyn.top25.Variants.csv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/ForSumit.Top25.NonSynGenes.Variants.csv

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/ForSumit.Top25.NonSynGenes /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/ForSumit.Top25.NonSynGenes.Variants.csv mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/. 


##20140925
##Getting variants present in specific group of invidiuals for Kevin

#Going to create a utility script that takes a list of patientID#s, converts them to poolID#s via a linkerfile and thne pulls those columns of genotypes from a specified .vcf file. Then going to create a separate utility script that adds (hard-coded) annotation information to a .vcf file 

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.ExtractPatientIDsFromVCF.vs1.py 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | head -n 150 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.top150

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.ExtractPatientIDsFromVCF.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.top150 --IDs 20572,20306,20219,20170,20123,20093,20501,20698,20132,20661,20471,20575

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.ExtractPatientIDsFromVCF.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --IDs 20572,20306,20219,20170,20123,20093,20501,20698,20132,20661,20471,20575 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf

[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf | wc                                                                                                                                                                                                                     
  73389 1539241 44180927

[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | wc                                                                                                                                                                                                                                                              
  73389 67401779 1578124080

#python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.ExtractPatientIDsFromVCF.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --IDs All | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wPatientIDs.vcf.gz 

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.vs1.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.wAnnot.vcf  

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf --AC 5 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.wAnnot.AClte5.vcf  

vcftools --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf --max-mac 5 --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcfToolsmaxMAC5.vcf 

#Made edits to DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.vs1.py and renamed asdouasdiubasdibasd, so moved file to DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.vs2.py. Basically changed AC cutoff so that only print variants that have the same number of allele copies within these 12 individuals as expected for the variant itself, i.e. keep only variants whose allele copies are present in just these 12 individuals
mv asdouasdiubasdibasd DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.vs2.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.UtilityAddAnnotationsToVCF.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf --AC 5 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.wAnnot.AClte5.vs2.vcf

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.wAnnot.AClte5.vs2.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.20140925KevinIndvRequest.wAnnot.AClte5.vcf

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.20140925KevinIndvRequest.wAnnot.AClte5.vcf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/.

#20141010 -- Finding the individual that has the single alternative allele for a CCNT1 variant, 12_49089559

[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz | grep 49089559 | perl -lane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "\t", $F[$i] } } ;'         662     0/1:12,11:23:99:304,0,282
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz | grep 49089559 | awk '{ print $663 }'                                                                                                                          0/1:12,11:23:99:304,0,282
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz | awk '{ print $2, "\t", $663 }' | perl -lane 'my @vals1 = split(/:/, $F[1]); if ($vals1[0] =~ m/POOL|\/1/) { print join("\t", @F); }' | grep -E '49089559|POOL'
POS     POOL81_11RL 
49089559        0/1:12,11:23:99:304,0,282
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | grep POOL81
41493,41493,4,POOL81-04RL,White non-Hispanic,0,0,1,0,0,0,0,0,0,0
40457,40457,10,POOL81-10RL,White non-Hispanic,0,1,0,0,0,0,0,0,0,0
41574,41574,11,POOL81-11RL,White non-Hispanic,0,0,1,0,0,0,0,0,0,0
30751,30751,3,POOL81-03RL,White non-Hispanic,0,0,1,0,0,0,0,0,0,0
41628,41628,9,POOL81-09RL,White non-Hispanic,33.3,0,0,0,1,0,0,0,0,0.53
40270,40270,8,POOL81-08RL,White non-Hispanic,0,1,0,0,0,0,0,0,0,0
41625,41625,1,POOL81-01RL,White non-Hispanic,0,0,1,0,0,0,0,0,0,0
40063,40063,7,POOL81-07RL,White non-Hispanic,0,0,1,0,0,0,0,0,0,0

#20150127 -- continuing this specific HESN-subset group analysis, now interested in just seeing whether a specific group of genes that they have identified have any type of clustering of variants among these 12 individuals

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf.gz | perl -lane 'if ($F[0] =~ m/^#/) { print join("\t", @F); } else { my $printFlag1 = 0; for (my $i = 9; $i <= $#F; $i++) { my @indvQuals = split(/:/, $F[$i]); my @genos = split(/\//, $indvQuals[0]); if (($genos[0] eq "1") || ($genos[1] eq "1")) { $printFlag1 = 1; } } if ($printFlag1 == 1) { print join("\t", @F); }  } ' | gzip >  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.vcf.gz

[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.vcf.gz | wc
  13488  281320 8212108

[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.vcf.gz | tail -n 10
X       53644586        .       T       C       148.38  VQSRTrancheSNP99.00to99.90      AA=T;AC=9;AF=5.017e-03;AN=1794;BaseQRankSum=0.268;DP=7898;Dels=0.00;FS=6.510;HRun=1;HaplotypeScore=0.9294;InbreedingCoeff=0.0388;MLEAC=1;MLEAF=5.574e-04;MQ=60.00;MQ0=0;MQRankSum=-0.141;QD=2.51;ReadPosRankSum=0.660;VQSLOD=1.31;culprit=QD       GT:AD:DP:GQ:PL  0/0:8,0:8:24:0,24,273   1/1:0,6:6:18:214,18,0   0/0:9,0:9:27:0,27,311   0/0:2,0:2:6:0,6,71      0/0:3,0:3:9:0,9,97      0/0:2,0:2:6:0,6,69      0/0:12,0:12:36:0,36,416 0/0:3,0:3:9:0,9,104     0/0:7,0:7:21:0,21,241   0/0:5,0:5:15:0,15,168   0/0:10,0:10:30:0,30,347 0/0:4,0:4:12:0,12,139
X       53674550        .       T       A       340.33  VQSRTrancheSNP99.00to99.90      AA=T;AC=3;AF=1.667e-03;AN=1800;BaseQRankSum=-5.388;DP=11244;Dels=0.00;FS=3.554;HRun=2;HaplotypeScore=1.5743;InbreedingCoeff=0.1312;MLEAC=2;MLEAF=1.111e-03;MQ=59.99;MQ0=0;MQRankSum=-0.287;QD=17.02;ReadPosRankSum=-0.717;VQSLOD=1.48;culprit=HaplotypeScore       GT:AD:DP:GQ:PL  0/0:10,0:10:30:0,30,326 0/0:7,0:7:21:0,21,236   0/0:7,0:7:21:0,21,232   0/0:16,0:16:48:0,48,551 0/0:5,0:5:15:0,15,171   0/0:6,0:6:18:0,18,209   0/0:16,0:16:48:0,48,527 0/1:4,1:5:17:17,0,122   0/0:12,0:12:36:0,36,407 0/0:16,0:16:48:0,48,532 0/0:12,0:12:36:0,36,413 0/0:9,0:9:27:0,27,286
X       64948279        .       T       C       55.25   VQSRTrancheSNP99.00to99.90      AA=T;AC=26;AF=0.015;AN=1788;BaseQRankSum=-7.262;DP=6334;Dels=0.00;FS=3.996;HRun=1;HaplotypeScore=0.8907;InbreedingCoeff=-0.0180;MLEAC=10;MLEAF=5.593e-03;MQ=59.76;MQ0=0;MQRankSum=-1.467;QD=0.38;ReadPosRankSum=-0.311;VQSLOD=-6.838e-01;culprit=DP        GT:AD:DP:GQ:PL  0/0:3,0:3:9:0,9,103     0/0:3,0:3:9:0,9,108     0/0:9,0:9:27:0,27,314   0/0:9,0:9:27:0,27,314   0/0:1,0:1:3:0,3,34      0/0:5,0:5:15:0,15,178   0/1:8,1:9:9:9,0,210     0/0:1,0:1:3:0,3,34      0/0:13,0:13:39:0,39,437 0/0:10,0:10:30:0,30,347 0/0:5,0:5:15:0,15,176   0/0:5,0:5:15:0,15,177
X       77113219        rs146752003     C       T       124.94  VQSRTrancheSNP99.00to99.90      AA=C;AC=3;AF=1.669e-03;AN=1798;BaseQRankSum=4.969;DB;DP=8111;Dels=0.00;FS=8.932;HRun=1;HaplotypeScore=0.7404;InbreedingCoeff=-0.0150;MLEAC=1;MLEAF=5.562e-04;MQ=60.00;MQ0=0;MQRankSum=0.113;QD=3.67;ReadPosRankSum=-0.898;VQSLOD=0.475;culprit=QD  GT:AD:DP:GQ:PL  0/1:8,6:14:99:175,0,207 0/0:6,0:6:18:0,18,204   0/0:6,0:6:18:0,18,196   0/0:7,0:7:21:0,21,238   0/0:3,0:3:9:0,9,90      0/0:6,0:6:18:0,18,197   0/0:10,0:10:30:0,30,318 0/0:3,0:3:9:0,9,95      0/0:5,0:5:15:0,15,154   0/0:19,0:19:57:0,57,589 0/0:8,0:8:24:0,24,256   0/0:8,0:8:24:0,24,252
X       77150634        .       G       A       266.01  VQSRTrancheSNP99.00to99.90      AA=G;AC=5;AF=2.775e-03;AN=1802;BaseQRankSum=-3.715;DP=9491;Dels=0.00;FS=0.000;HRun=2;HaplotypeScore=1.6921;InbreedingCoeff=0.0737;MLEAC=2;MLEAF=1.110e-03;MQ=60.00;MQ0=0;MQRankSum=-0.303;QD=10.64;ReadPosRankSum=-1.345;VQSLOD=1.55;culprit=DP    GT:AD:DP:GQ:PL  0/0:8,0:8:24:0,24,268   0/0:12,0:12:36:0,36,435 0/0:8,0:8:24:0,24,280   0/0:12,0:12:36:0,36,415 0/0:4,0:4:12:0,12,135   0/0:6,0:6:18:0,18,213   0/0:11,0:11:33:0,33,366 0/0:1,0:1:3:0,3,34      0/0:22,0:22:66:0,66,732 0/0:13,0:13:39:0,39,448 0/0:10,0:10:30:0,30,344 0/1:3,1:4:24:24,0,86
X       153061748       .       C       T       761.06  VQSRTrancheSNP99.00to99.90      AA=C;AC=6;AF=3.326e-03;AN=1804;BaseQRankSum=6.545;DP=11331;Dels=0.00;FS=6.414;HRun=1;HaplotypeScore=1.4674;InbreedingCoeff=0.1070;MLEAC=2;MLEAF=1.109e-03;MQ=59.99;MQ0=0;MQRankSum=-0.139;QD=12.48;ReadPosRankSum=0.991;VQSLOD=1.68;culprit=HaplotypeScore GT:AD:DP:GQ:PL  0/0:11,0:11:33:0,33,347 0/0:21,0:21:63:0,63,705 0/0:11,0:11:33:0,33,361 0/0:20,0:20:60:0,60,674 0/0:3,0:3:9:0,9,94      0/0:10,0:10:30:0,30,333 0/0:13,0:13:39:0,39,415 0/0:7,0:7:21:0,21,216   0/0:11,0:11:33:0,33,357 0/0:8,0:8:24:0,24,243   0/0:10,0:10:30:0,30,317 0/1:5,1:6:17:17,0,140
X       153713382       .       T       C       268.37  VQSRTrancheSNP99.00to99.90      AA=T;AC=9;AF=5.000e-03;AN=1800;BaseQRankSum=-5.440;DP=13441;Dels=0.00;FS=0.500;HRun=1;HaplotypeScore=3.8832;InbreedingCoeff=0.1192;MLEAC=1;MLEAF=5.556e-04;MQ=59.98;MQ0=0;MQRankSum=1.064;QD=3.78;ReadPosRankSum=1.520;VQSLOD=0.131;culprit=QD     GT:AD:DP:GQ:PL  0/0:12,0:12:36:0,36,396 0/0:31,0:31:93:0,93,1030        0/0:13,0:13:39:0,39,456 0/0:18,0:18:54:0,54,613 0/0:3,0:3:9:0,9,96      0/0:10,0:10:30:0,30,339 0/0:12,0:12:36:0,36,404 0/0:2,0:2:6:0,6,66      0/0:17,0:17:51:0,51,569 0/1:9,1:10:3:3,0,266    0/0:7,0:7:21:0,21,241   0/0:8,0:8:24:0,24,271
X       153713641       rs147530071     G       A       338.36  VQSRTrancheSNP90.00to99.00      AA=G;AC=5;AF=2.784e-03;AN=1796;BaseQRankSum=4.788;DB;DP=12508;Dels=0.00;FS=2.813;HRun=0;HaplotypeScore=2.5272;InbreedingCoeff=0.1407;MLEAC=2;MLEAF=1.114e-03;MQ=59.99;MQ0=0;MQRankSum=0.648;POSITIVE_TRAIN_SITE;QD=9.40;ReadPosRankSum=0.718;VQSLOD=3.02;culprit=HaplotypeScore    GT:AD:DP:GQ:PL  0/0:9,0:9:27:0,27,285   0/0:26,0:26:78:0,78,854 0/0:12,0:12:36:0,36,396 0/0:13,0:13:39:0,39,412 ./.     0/0:10,0:10:30:0,30,338 0/0:15,0:15:45:0,45,472 0/1:3,1:4:22:22,0,86    0/0:11,0:11:33:0,33,357 0/0:7,0:7:21:0,21,223   0/0:5,0:5:15:0,15,165      0/0:6,0:6:18:0,18,190
X       154456747       rs572013        A       G       346432  VQSRTrancheSNP99.00to99.90      AA=G;AC=1748;AF=0.969;AN=1804;BaseQRankSum=18.300;DB;DP=10832;Dels=0.00;FS=2.667;HRun=0;HaplotypeScore=1.6675;InbreedingCoeff=-0.0339;MLEAC=1797;MLEAF=0.996;MQ=59.99;MQ0=0;MQRankSum=0.253;POSITIVE_TRAIN_SITE;QD=31.98;ReadPosRankSum=6.371;VQSLOD=0.370;culprit=ReadPosRankSum  GT:AD:DP:GQ:PL  1/1:0,7:7:21:241,21,0   1/1:0,2:2:6:71,6,0      1/1:0,7:7:21:243,21,0   1/1:0,16:16:48:556,48,0 1/1:0,1:1:3:32,3,0      1/1:0,7:7:21:243,21,0   1/1:0,13:13:39:434,39,0 1/1:0,5:5:15:168,15,0   1/1:0,7:7:21:230,21,0   1/1:0,16:16:48:530,48,0 1/1:0,8:8:24:260,24,0      1/1:0,6:6:18:197,18,0
X       154464466       .       C       T       471.87  VQSRTrancheSNP99.00to99.90      AA=C;AC=7;AF=3.911e-03;AN=1790;BaseQRankSum=2.203;DP=7087;Dels=0.00;FS=0.000;HRun=1;HaplotypeScore=0.6094;InbreedingCoeff=0.0261;MLEAC=3;MLEAF=1.676e-03;MQ=59.99;MQ0=0;MQRankSum=2.281;QD=10.04;ReadPosRankSum=-1.245;VQSLOD=0.733;culprit=DP     GT:AD:DP:GQ:PL  0/0:4,0:4:12:0,12,134   0/0:4,0:4:12:0,12,144   0/0:2,0:2:6:0,6,71      0/1:6,1:7:9:9,0,186     0/0:4,0:4:12:0,12,134   0/0:6,0:6:18:0,18,214   0/0:5,0:5:15:0,15,171   0/0:4,0:4:12:0,12,131   0/0:4,0:4:12:0,12,138   0/0:10,0:10:30:0,30,337 0/0:3,0:3:9:0,9,100     0/0:9,0:9:27:0,27,301

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.wAnnot.vs2.vcf.gz

#Genes of interest:
#ARFGEF1	8_68200276	20219
#NUP98	11_3735127	20219
#IPO4	14_24654486	20219
#IFT122	3_129207102	20219
#FUCA1	1_24175283	20501
#TPR	1_186303511	20501
#RNF213	17_78301691	20501

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.wAnnot.vs2.vcf.gz | grep -E '^#CHROM|ARFGEF1|NUP98|IPO4|IFT122|FUCA1|TPR|RNF213' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.wAnnot.vs2.KevGenesOfInterest1.vcf.gz

#zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.wAnnot.vs2.KevGenesOfInterest1.vcf.gz | perl -lane 'my @snpQuals = split(/;/, $F[9]); if (!$snpQuals[2]) { $snpQuals[2] = $snpQuals[0]; } my @printVals = ($F[0], $F[1], $F[2], $snpQuals[0], $snpQuals[1], $snpQuals[2]); for (my $i = 10; $i <= $#F; $i++) { my @indvQuals = split(/:/, $F[$i]); my @genos = split(/\//, $indvQuals[0]); my $AC1 = 0; if ($genos[0] eq "1") { $AC1++; } if ($genos[1] eq "1") { $AC1++; } push(@printVals, $AC1); } print join("\t", @printVals);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.wAnnot.vs2.KevGenesOfInterest1.Rinput.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.wAnnot.vs2.KevGenesOfInterest1.vcf.gz | perl -lane 'if ($F[0] =~ m/^#CHROM/) { print "CHROM\t", $F[1], "\t", $F[2], "\tExonicity\tGene\tAAEffect\t", join("\t", @F[10..$#F]); } else { my @snpQuals = split(/;/, $F[9]); if (!$snpQuals[2]) { $snpQuals[2] = $snpQuals[0]; } my @printVals = ($F[0], $F[1], $F[2], $snpQuals[0], $snpQuals[1], $snpQuals[2]); for (my $i = 10; $i <= $#F; $i++) { my @indvQuals = split(/:/, $F[$i]); my @genos = split(/\//, $indvQuals[0]); my $AC1 = 0; if ($genos[0] eq "1") { $AC1++; } if ($genos[1] eq "1") { $AC1++; } push(@printVals, $AC1); } print join("\t", @printVals); }' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.wAnnot.vs2.KevGenesOfInterest1.Rinput.gz

#zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.wAnnot.vs2.KevGenesOfInterest1.vcf.gz | perl -lane 'my @snpQuals = split(/;/, $F[9]); if (!$snpQuals[2]) { $snpQuals[2] = $snpQuals[0]; } my $AC1 = 0; for (my $i = 10; $i <= $#F; $i++) { my @indvQuals = split(/:/, $F[$i]); my @genos = split(/\//, $indvQuals[0]); if ($genos[0] eq "1") { $AC1++; } if ($genos[1] eq "1") { $AC1++; } } print $F[0], "\t", $F[1], "\t", $F[2], "\t", $snpQuals[0], "\t", $snpQuals[1], "\t", $snpQuals[2], "\t", $AC1; ' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.wAnnot.vs2.KevGenesOfInterest1.Rinput.Collapsed.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.wAnnot.vs2.KevGenesOfInterest1.vcf.gz | perl -lane 'if ($F[0] =~ m/^#CHROM/) { print "CHROM\t", $F[1], "\t", $F[2], "\tExonicity\tGene\tAAEffect\tACTotal"; } else { my @snpQuals = split(/;/, $F[9]); if (!$snpQuals[2]) { $snpQuals[2] = $snpQuals[0]; } my $AC1 = 0; for (my $i = 10; $i <= $#F; $i++) { my @indvQuals = split(/:/, $F[$i]); my @genos = split(/\//, $indvQuals[0]); if ($genos[0] eq "1") { $AC1++; } if ($genos[1] eq "1") { $AC1++; } } print $F[0], "\t", $F[1], "\t", $F[2], "\t", $snpQuals[0], "\t", $snpQuals[1], "\t", $snpQuals[2], "\t", $AC1; }' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.wAnnot.vs2.KevGenesOfInterest1.Rinput.Collapsed.gz

[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.wAnnot.vs2.KevGenesOfInterest1.Rinput.Collapsed.gz | awk '{ print $7 }' | sort -rg | uniq -c
     2 	24
     3 	21
     7 	19
     8 	18
     2	17
     7 	16
     3 	15
     3 	14
     2 	12
     3	11
     8 	10
     4 	9
     2 	8
     4 	7
    11 	6
    23 	5
    35 	4
    32 	3
    57 	2
   163 	1

cp -p /home/michaelt/Scripts/R/BasicScript.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.PlotVarsAlongGenes.Collapsed.vs1.R

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.PlotVarsAlongGenes.Collapsed.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.wAnnot.vs2.KevGenesOfInterest1.Rinput.Collapsed.gz

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.PlotVarsAlongGenes.Collapse.vs1.*jpg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/.

#cp -p /home/michaelt/Scripts/R/BasicScript.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.PlotVarsAlongGenes.vs1.R
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.PlotVarsAlongGenes.Collapsed.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.PlotVarsAlongGenes.vs1.R

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.PlotVarsAlongGenes.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.wAnnot.vs2.KevGenesOfInterest1.Rinput.gz 

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.PlotVarsAlongGenes.vs1.*jpg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/.

scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/Data/UCSC/UCSC_hg19_CanonicalTranscripts_wSymbols.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/Data/UCSC/UCSC_hg19_ExonLocations.txt.gz /home/michaelt/Data/UCSC/.

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py /home/michaelt/Data/UCSC/UCSC_hg19_CreateCanonicalExonList.vs1.py

python /home/michaelt/Data/UCSC/UCSC_hg19_CreateCanonicalExonList.vs1.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_ExonLocations.txt.gz --file2 /home/michaelt/Data/UCSC/UCSC_hg19_CanonicalTranscripts_wSymbols.txt > /home/michaelt/Data/UCSC/UCSC_hg19_CanonicalExonList.txt 

~~~
#Some notes on the editing of /home/michaelt/Data/UCSC/UCSC_hg19_CreateCanonicalExonList.vs1.py
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles]$cat /home/michaelt/Data/UCSC/UCSC_hg19_Canonic
alExonList.txt | perl -lane 'print $#F;' | sort | uniq -c
15 10
7 11
3 13
2 14
239634 8
12 9

[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles]$cat /home/michaelt/Data/UCSC/UCSC_hg19_CanonicalExonList.txt | perl -lane 'if ($#F == 10) { print join("\t", @F) };'
chrX    155255322       155257848       155255322       155256502       uc011nad.1      0       uc011nad.1      DDX11L  family  member
chrX    155255322       155257848       155257000       155257109       uc011nad.1      1       uc011nad.1      DDX11L  family  member
chrX    155255322       155257848       155257494       155257848       uc011nad.1      2       uc011nad.1      DDX11L  family  member
chrY    59358328        59360854        59358328        59359508        uc011ncc.1      0       uc011ncc.1      DDX11L  family  member
chrY    59358328        59360854        59360006        59360115        uc011ncc.1      1       uc011ncc.1      DDX11L  family  member
chrY    59358328        59360854        59360500        59360854        uc011ncc.1      2       uc011ncc.1      DDX11L  family  member

python /home/michaelt/Data/UCSC/UCSC_hg19_CreateCanonicalExonList.vs1.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_ExonLocations.txt.gz --file2 /home/michaelt/Data/UCSC/UCSC_hg19_CanonicalTranscripts_wSymbols.txt > /home/michaelt/Data/UCSC/UCSC_hg19_CanonicalExonList.txt

[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles]$cat /home/michaelt/Data/UCSC/UCSC_hg19_CanonicalExonList.txt | perl -lane 'print $#F;' | sort | uniq -c                                                                                                                239673 8

#Hence the 'joining' of line2 at the beginning of the processing for the second file
~~~

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.PlotVarsAlongGenes.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.PlotVarsAlongGenes.vs2.R

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.PlotVarsAlongGenes.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.wAnnot.vs2.KevGenesOfInterest1.Rinput.gz /home/michaelt/Data/UCSC/UCSC_hg19_CanonicalExonList.txt

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.PlotVarsAlongGenes.vs2.*jpg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/.

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts | grep -E 'ARFGEF1|NUP98|IPO4|IFT122|FUCA1|TPR|RNF213' | perl -lane 'my @vals1 = split("_", $F[3]); $F[3] = $vals1[1]; print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts.20150502KevGeneRequest
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno2.maf1.skat0.epacts | grep -E 'ARFGEF1|NUP98|IPO4|IFT122|FUCA1|TPR|RNF213' | perl -lane 'my @vals1 = split("_", $F[3]); $F[3] = $vals1[1]; print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno2.maf1.skat0.epacts.20150502KevGeneRequest









##SKAT0 Behavior Tests
##20140930 -- Setting up files to test the behavior of SKAT0 when purposely change the case/control designation of the different variant/allele copies used in the analysis, e.g. to give us a better idea of how the test statistic is behaving

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/CovariateTests

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.ExtractVCFVarsBasedOnAnnoFile.vs1.py 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | grep VDAC3 | grep exonic | grep nonsyn > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/VDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.VDAC3.Exonic.Nonsynonymous.txt

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.ExtractVCFVarsBasedOnAnnoFile.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/VDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.VDAC3.Exonic.Nonsynonymous.txt --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Toy.Exonic.Nonsynonymous.vcf.bgzip

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped | sort -k 1,1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous | grep VDAC3 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3

#20141002 -- Editing /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped so that the first handful of individuals are HIV+ and I can easily alter genotypes in the associated VDAC3 toy examples. I will be exchanging phenotype designations with other individuals in the .ped file so that the total number of HIVPROG + vs HIVPROG - is the same 

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.Orig.ped

#20141002 -- Scrubbing clean the .vcf file so have a version where everyone is 0/0 first, then creating copies where I manually include 1s among the first five individuals. Currently only going to do heterozygotes, and then will start moving some of the 1 allele copies from the first 5 individuals to the next set of individuals to look at behavior when mutation is among HIV+ and HIV- individuals. The file coding is 'XXXXXlYYYYY_XXXXXlYYYYY....' to explain how many copies of the 1 allele there are per variant and which individual(s) they are in, where X = number of 1 copies in the first 5 individuals, whom are HIV+, and where Y = the number of 1 copies in the next 5 individuals, whom are HIV-  

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.vcf.bgzip | perl -lane 'if ($F[0] !~ /^#/) { for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/\:/, $F[$i]); $vals1[0] = "0/0"; $F[$i] = join(":", @vals1); } } print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.All0s.vcf.bgzip

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.All0s.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_00000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_00000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip 
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_00000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip.gz
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.All0s.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip.gz
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.All0s.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00000l00000_00000l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00000l00000_00000l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00000l00000_00000l00000.vcf.bgzip.gz
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.All0s.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00000l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00000l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00000l00000.vcf.bgzip.gz
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.All0s.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip.gz

#QC check line
tail -n 5 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10* | perl -lane 'print join("\t", @F[0..14]);'

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.All0s.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.All0s.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_00000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_00000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip 
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_00000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip.gz
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.All0s.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l11111.vcf.bgzip
#Manually edit file to include expected 1 genotypes now                                                                                                                                                                                                                                                                    
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l11111.vcf.bgzip
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l11111.vcf.bgzip.gz
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now                                                                                                                                                                                                                                                                    
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip.gz

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now                                                                                                                                                                                                                                                                    
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00000_00010l00000_00001l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now                                                                                                                                                                                                                                                                    
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00100_00010l00000_00001l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now                                                                                                                                                                                                                                                                    
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00100_00010l00010_00001l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now                                                                                                                                                                                                                                                                    
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00100_00010l00010_00001l00001.vcf.bgzip
#Manually edit file to include expected 1 genotypes now                                                                                                                                                                                                                                                                    
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00000_00010l00000_00001l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00100_00010l00000_00001l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00100_00010l00010_00001l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00100_00010l00010_00001l00001.vcf.bgzip
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip.gz
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00000_00010l00000_00001l00000.vcf.bgzip.gz
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00100_00010l00000_00001l00000.vcf.bgzip.gz
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00100_00010l00010_00001l00000.vcf.bgzip.gz
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00100_00010l00010_00001l00001.vcf.bgzip.gz

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_00000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_00000l00000_00000l00000_00000l00000_00000l00000.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00000l00000_00000l00000_00000l00000.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00000l00000_00000l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00000l00000_00000l00000.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00000l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00000l00000.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.Pheno1.maf1.skat0.noAdj

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_00000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_00000l00000_00000l00000_00000l00000_00000l00000.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_01000l00000_00100l00000_00010l00000_00001l00000.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l11111.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l11111.Pheno1.maf1.skat0.noAdj

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l00000_00100l00000_00010l00000_00001l00000.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00000_00010l00000_00001l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00000_00010l00000_00001l00000.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00100_00010l00000_00001l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00100_00010l00000_00001l00000.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00100_00010l00010_00001l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00100_00010l00010_00001l00000.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00100_00010l00010_00001l00001.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00100_00010l00010_00001l00001.Pheno1.maf1.skat0.noAdj

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3.OneVar
#Manually edit  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 to contain just the first variant

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_00000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3.OneVar --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_00000l00000_00000l00000_00000l00000_00000l00000.Pheno1.maf1.skat0.noAdj.oneVar
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_00000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3.OneVar --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_00000l00000_00000l00000_00000l00000_00000l00000.Pheno1.maf1.skat0.noAdj.oneVar


cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.All0s.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_00000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now                                                                                                                                                                                                                                                                    
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.All0s.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_00000l02000_00000l00000_00000l00000_00000l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now                                                                                                                                                                                                                                                                    
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now                                                                                                                                                                                                                                                                    
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_02000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now                                                                                                                                                                                                                                                                    
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.All0s.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_00000l11111_11111l00000_00000l11111_11111l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now                                                                                                                                                                                                                                                                    
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.All0s.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.22222l00000_00000l22222_22222l00000_00000l22222_22222l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now                                                                                                                                                                                                                                                                    
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_00000l01000_00100l00000_00000l00010_00001l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now                                                                                                                                                                                                                                                                    
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_00000l02000_00200l00000_00000l00020_00002l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now                                                                                                                                                                                                                                                                    

/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_00000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_00000l02000_00000l00000_00000l00000_00000l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_02000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_00000l11111_11111l00000_00000l11111_11111l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.22222l00000_00000l22222_22222l00000_00000l22222_22222l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_00000l01000_00100l00000_00000l00010_00001l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_00000l02000_00200l00000_00000l00020_00002l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_00000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip.gz
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_00000l02000_00000l00000_00000l00000_00000l00000.vcf.bgzip.gz
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip.gz
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_02000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip.gz
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_00000l11111_11111l00000_00000l11111_11111l00000.vcf.bgzip.gz
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.22222l00000_00000l22222_22222l00000_00000l22222_22222l00000.vcf.bgzip.gz
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_00000l01000_00100l00000_00000l00010_00001l00000.vcf.bgzip.gz
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_00000l02000_00200l00000_00000l00020_00002l00000.vcf.bgzip.gz

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_00000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_00000l00000_00000l00000_00000l00000_00000l00000.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_02000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_02000l00000_00000l00000_00000l00000_00000l00000.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_01000l00000_00100l00000_00010l00000_00001l00000.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_02000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_02000l00000_00100l00000_00010l00000_00001l00000.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_00000l11111_11111l00000_00000l11111_11111l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_00000l11111_11111l00000_00000l11111_11111l00000.vcf.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.22222l00000_00000l22222_22222l00000_00000l22222_22222l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.22222l00000_00000l22222_22222l00000_00000l22222_22222l00000.vcf.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_00000l01000_00100l00000_00000l00010_00001l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_00000l01000_00100l00000_00000l00010_00001l00000.vcf.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_00000l02000_00200l00000_00000l00020_00002l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_00000l02000_00200l00000_00000l00020_00002l00000.vcf.Pheno1.maf1.skat0.noAdj

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --skat-flat --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.Pheno1.maf1.skat0.flatPrior.noAdj

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/MultipleRuns/
for i in {11..100}
do
	epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/MultipleRuns/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.Pheno1.maf1.skat0.Adj.run$i
done

#20141009
#going to run 100 versions of the real data EPACTS SKAT-O analysis using Pheno1 ExonicNonsynonymous in order to get a feel for the variation present in the smal sample-size adjustment permutation process 
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/ExonicNonsynonymous
for i in {1..100}
do
	epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/ExonicNonsynonymous/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.try2.run$i
done

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous
for i in {1..100}
do
	epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.try2.run$i
done

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.try2.run1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.try2.run1.epacts.GeneIDList

#Made a default bash shell script to start other bash shell scripts off with
vi /home/michaelt/Scripts/Bash/BashScript.InputVarsPlusTime.vs1.sh

cp -p  /home/michaelt/Scripts/Bash/BashScript.InputVarsPlusTime.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh

#Going to cat all *epacts files from the multiple runs then grep the GeneID of importance one at a time to print out a table that has 'GeneID' then all that GeneID's p-values in a single row. May have to create an auxilary scripts or extra perl code one-liner to double-check every row has the same number of entries in case certain GeneIDs occasionally drop out 

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.try2.run1.epacts.GeneIDList TruePheno Pheno3 ExonicNonsynonymous

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.ExonicNonsynonymous.output | grep -v NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.ExonicNonsynonymous.output.noNAs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.ExonicNonsynonymous.output.noNAs | R -q -e "Data1 <- read.table(file('stdin'), header=F, row.names=1); print(paste(row.names(Data1), mean(Data1[,1]), var(Data1[,1]), sep=\" \"))" 

#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.ExonicNonsynonymous.output.noNAs | R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,var))); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.ExonicNonsynonymous.output.noNAs.histograms.jpg\", width=2500, height=2000, res=300); print(Data1[14,]); par(mfrow=c(2,2)); hist(as.numeric(Data1[40,])); hist(as.numeric(Data1[156,])); hist(as.numeric(Data1[261,])); hist(as.numeric(Data1[345,])); dev.off();" > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.ExonicNonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.ExonicNonsynonymous.output.noNAs | R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd))); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.ExonicNonsynonymous.output.noNAs.histograms.jpg\", width=2500, height=2000, res=300); par(mfrow=c(2,2)); hist(as.numeric(Data1[40,])); hist(as.numeric(Data1[156,])); hist(as.numeric(Data1[261,])); hist(as.numeric(Data1[345,])); dev.off();" > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.ExonicNonsynonymous.output.noNAs.Routput

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.ExonicNonsynonymous.output.noNAs.histograms.jpg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/.

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.ExonicNonsynonymous.output.noNAs | awk '{ print $1 }' | sed 's/://g' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.ExonicNonsynonymous.output.noNAs.GeneIDList

for gene in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.ExonicNonsynonymous.output.noNAs.GeneIDList`
do
	paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.ExonicNonsynonymous.output.noNAs.Routput | grep $gene) <(cat *epacts | grep $gene | tail -n 1 | awk '{ print $7, "\t", $8, "\t", $9 }')  >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.ExonicNonsynonymous.output.noNAs.Routput.wAlleleCountData 
done

#20150114
#I did a bunch of work trying to figure out the behavior of EPACTS and SKAT in relation to how EPACTS runs SKAT and how I could run SKAT independently, both through R, to make sure what EPACTS was doing with SKAT made sense (and I could replicate it independently). However, as of 20150114 there was no record of the work I had done (I think anyways) but one of my screens that is still open happen to contains the history of a lot of that work. So momentarily saving that history into a file and logging it here so that I have some sort of record of this work

history > /data/userdata/pg/michaelt/Data/ALL_MAPPING/MainScript1.PG_HIVArray1.EPACTSSKATWorkLog.addition1.output

#On another screen with a bit of information at the top
history > /data/userdata/pg/michaelt/Data/ALL_MAPPING/MainScript1.PG_HIVArray1.EPACTSSKATWorkLog.addition2.output

PWD: /home/michaelt/Software/EPACTS-3.2.4/bin
[  michaelt@login2  ~/Software/EPACTS-3.2.4/bin]$cmp epacts-group epacts-group.orig
epacts-group epacts-group.orig differ: byte 811, line 43

#I added code in epacts-group to try and include skatCommonRare as an option but it never really went anywhere. Nothing else appears to be altered

PWD: /home/michaelt/Software/EPACTS-3.2.4/data
#In /home/michaelt/Software/EPACTS-3.2.4/data/group.skat.R I added the following (these are instances where rho estimates are weird and I just wanted to print more info to help diagnose the situation -- as you'll see even the in-house code included mentions of 'temporary solutions'. My additions are only the print commands in both blocks)

~~~

#	print(r)
#	print(pheno)
#	print(cov)
#	print(lm((pheno~cov-1)))
#	print(.libPaths())
#	print(sessionInfo())
#	print(betas)

      p <- r$p.value
      if ( m == 1 ) {
        rho <- NA
      }
      else {
        if ( skatCommonRare ) {
#		rho <- paste(r$param$, , , sep=",")
	} 
	else {
		rho <- r$param$rho_est
	        if ( length(rho) > 1 ) {
	          rho <- "NA"  ## temporary solution
	        	print("Here1")
			print(r)
			print(r$param)
	        }
		if ( length(rho) == 0 ) {
	          rho <- "-9"  ## temporary solution
	        	print("Here2")
			print(r)
			print(r$param)
		}
      	}

~~~

#It looks like any 'changes' I made to both epactsGroup.R or group.skat.R have been commented out since (except for those print statements above), so there's a log of some sort of what I originally included. Most of these additions were just print statements to understand what the code (EPACTS' code) was doing, as well as an attempt to get the program to run skatCommonRare (which I ended up demuring from because skatCommonRare didn't do the rho parameter space search, as well as the belief that ultimately it wouldn't add that much power in exchange for using and having to defend a much more novel program implementation at the time)




##20141006
##HESeroNegative Fix

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs2.py

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno.OLDvs1

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.pheno.OLDvs1

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.pheno

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped.OLDvs1

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.pheno | awk '{ print $1, "\t", $2, "\t0\t0\t0\t", $3, "\t", $4, "\t", $5, "\t", $6 } ' | sed 's/-9/NA/g' | perl -lane 'print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar | awk '{ print $4, "\t", $6, "\t", $7 "\t", $8, "\t", $9, "\t", $10, "\t", $11,  "\t", $12, "\t", $13, "\t", $14,  "\t", $15 }' | paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped.vs2

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped.vs2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped

#Add line #FAM_ID IND_ID FAT_ID MOT_ID SEX HIVPROG HIVPROGHE AIDS AIDSEXTR RMID PC1 PC2 PC3 PC4 PC5 PC6 PC7 PC8 PC9 PC10 to /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped

paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped.OLDvs1 | perl -lane 'print join("\t", @F[0..6]);') <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped | perl -lane 'print join("\t", @F[0..6]);')

#Go through running epacts bash commands, mv all the old ones with Pheno2 or HIVPROGHE into a separate folder and redo those runs
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 2 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs.maf05.pheno2

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs.maf05.pheno2.assoc.logistic Pheno2_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs.maf05.pheno2.assoc.logistic.QQplot.jpeg &

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs.OLDvs1

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($4 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count.OLDvs1.gz

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.OLDvs1.gz

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.log /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.OLDvs1.log 

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg &

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.AllPhenos.frq.AndCount.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.AllPhenos.frq.AndCount.OLDvs1.gz

paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq.count | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.AllPhenos.frq.AndCount.gz

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq.count

#gunzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq.count.gz

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.OLDvs1.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs5.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.AllPhenos.frq.AndCount.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 30; $i <= 40; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary.NonSyn.top25.Variants.csv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary.NonSyn.top25.Variants.OLDvs1.csv

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/VLookUp.Python.PullOutEPACTSVariants.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.top25 --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | cut -d, -f 1-6,18-29,35,37,39,41 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary.NonSyn.top25.Variants.csv

#Added the following header
#Chr,BP,rsID,refAllele,otherAllele,AncAllele,AC_AllIndvs,AC_SeroNeg,AC_SeroNegHE,AC_SeroPos,AC_RapidVRapid,AC_SlowVSlow,AC_VeryRapid,AC_VerySlow,VarType,GeneID,SNVType,AAChange,Sift,Polyphen2,LRT,MutationTaster

#Just for checking things
paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary.NonSyn.top25.Variants.OLDvs1.csv | perl -F, -lane 'print join(",", @F[6..13]);') <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary.NonSyn.top25.Variants.csv | perl -F, -lane 'print join(",", @F[6..13]);') | vi -

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary.NonSyn.top25 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/ForSumit.Top25.NonSynGenes
ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary.NonSyn.top25.Variants.csv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/ForSumit.Top25.NonSynGenes.Variants.csv

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/ForSumit.Top25.NonSynGenes /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/ForSumit.Top25.NonSynGenes.Variants.csv mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/.

##20141114
##Getting Malim gene results for Steve from P2

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.epacts | grep -f /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Second_capture_P2_gene-list.MalimSpecific.noMeta.txt | awk '{ if ($10 != "NA") { print $0 } } ' | sort -rg -k 10,10 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.epacts.MalimGenes 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts | grep -f /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Second_capture_P2_gene-list.MalimSpecific.noMeta.txt | awk '{ if ($10 != "NA") { print $0 } } ' | sort -rg -k 10,10 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts.MalimGenes 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.epacts | grep -f /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Second_capture_P2_gene-list.MalimSpecific.noMeta.txt | awk '{ if ($10 != "NA") { print $0 } } ' | sort -rg -k 10,10 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.epacts.MalimGenes 

##20141129
##Doing some extra coding things to compare results of the mapping/calling pipeline with GATK to results in FreeBayes and Samtools
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.DropOff.VCFVariants.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz --window 1000 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.DropOffTarg_1kb.vcf.gz 


##20150106
##Creating code/scripts to get list of individuals per variant that contain the alternative allele for that variant, given a list of specific variants

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SpecificGeneVCFs

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.vcf.gz | grep -E '^#|VDAC3' | grep -E '^#|nonsynonymous|stop' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SpecificGeneVCFs/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVar.VDAC3.Exonic.Nonsyn.vcf.gz

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.vs1.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SpecificGeneVCFs/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVar.VDAC3.Exonic.Nonsyn.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.vs1.VDAC3.Exonic.Nonsyn.output

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs1.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.vs1.VDAC3.Exonic.Nonsyn.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs1.VDAC3.Exonic.Nonsyn.output

#Need to add a line to #CHROM header of VCF file for 'ANNOVAR' column, otherwise the individual IDs are all off by one, e.g. there's an extra column in the variant rows but not the header row leading to name errors
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs2.py

##20150109 
##Getting allele counts/frequencies for specific variants Sudhir requested based on other groups' work involving them

exonic ARFGEF1 nonsynonymous_SNV NM_006421 c.A941G p.Y314C
exonic NUP98 nonsynonymous_SNV NM_005387 c.G2549A p.G850E
exonic IPO4 nonsynonymous_SNV NM_024658 c.G1311A p.M437I
exonic IFT122 stoploss_SNV NM_052990 c.A1521T p.X507C
exonic FUCA1 nonsynonymous_SNV NM_000147 c.T1016C p.I339T
exonic TPR nonsynonymous_SNV NM_003292 c.A5128G p.T1710A
exonic RNF213 nonsynonymous_SNV NM_001256071 c.C3269T p.T1090M

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | grep c.A941G 
8,68200276_1,rs76811394,T,C,T,134.63,VQSRTrancheSNP90.00to99.00,AA=T;AC=1;AF=5.531e-04;AN=1808;BaseQRankSum=3.147;DB;DP=13747;Dels=0.00;FS=5.328;HRun=0;HaplotypeScore=1.0303;InbreedingCoeff=-0.0043;MLEAC=1;MLEAF=5.531e-04;MQ=60.00;MQ0=0;MQRankSum=0.938;POSITIVE_TRAIN_SITE;QD=11.22;ReadPosRankSum=1.713;VQSLOD=2.63;culprit=HaplotypeScore,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,exonic,ARFGEF1,nonsynonymous SNV,NM_006421:c.A941G:p.Y314C,,0.18,0.969028,C,0.83,T,0.0,B,0.876934,N,0.754805,D,4.21,8,68200276,68200276,T,C

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | grep c.G2549A
11,3735127_1,.,C,T,C,265.57,VQSRTrancheSNP99.00to99.90,AA=C;AC=1;AF=5.525e-04;AN=1810;BaseQRankSum=0.621;DP=23164;Dels=0.00;FS=2.205;HRun=2;HaplotypeScore=2.1553;InbreedingCoeff=-0.0013;MLEAC=1;MLEAF=5.525e-04;MQ=60.00;MQ0=0;MQRankSum=-0.092;QD=13.98;ReadPosRankSum=-1.186;VQSLOD=1.06;culprit=HaplotypeScore,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,exonic,NUP98,nonsynonymous SNV,NM_005387:c.G2549A:p.G850E,,0,0.998911,C,0.99,D,0.321,P,1.0,D,0.999879,D,5.28,11,3735127,3735127,C,T

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | grep c.G1311A
14,24654486_1,.,C,T,C,363.55,VQSRTrancheSNP99.00to99.90,AA=C;AC=1;AF=5.525e-04;AN=1810;BaseQRankSum=1.320;DP=28335;Dels=0.00;FS=4.522;HRun=0;HaplotypeScore=3.0813;InbreedingCoeff=-0.0009;MLEAC=1;MLEAF=5.525e-04;MQ=59.99;MQ0=0;MQRankSum=-0.109;QD=7.57;ReadPosRankSum=1.683;VQSLOD=0.573;culprit=HaplotypeScore,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,exonic,IPO4,nonsynonymous SNV,NM_024658:c.G1311A:p.M437I,,0.04,0.999236,C,0.85,T,0.834,P,1.0,D,0.834784,D,5.61,14,24654486,24654486,C,T

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | grep c.A1521T
3,129207102_1,rs146874343,C,T,C,472.56,VQSRTrancheSNP99.00to99.90,AA=C;AC=1;AF=5.501e-04;AN=1818;BaseQRankSum=9.069;DB;DP=24880;Dels=0.00;FS=3.168;HRun=0;HaplotypeScore=3.6113;InbreedingCoeff=-0.0014;MLEAC=1;MLEAF=5.501e-04;MQ=43.11;MQ0=0;MQRankSum=-1.967;QD=8.75;ReadPosRankSum=-0.348;VQSLOD=0.327;culprit=HaplotypeScore,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,exonic,IFT122,stoploss SNV,NM_052990:c.A1521T:p.X507C,,,,,,,,,,,,,,3,129207102,129207102,C,T 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | grep c.T1016C
1,24175283_1,.,A,G,A,227.53,VQSRTrancheSNP99.00to99.90,AA=A;AC=1;AF=5.488e-04;AN=1822;BaseQRankSum=-1.461;DP=23298;Dels=0.00;FS=3.295;HRun=0;HaplotypeScore=2.5523;InbreedingCoeff=-0.0011;MLEAC=1;MLEAF=5.488e-04;MQ=60.00;MQ0=0;MQRankSum=-0.967;QD=10.83;ReadPosRankSum=-1.099;VQSLOD=0.951;culprit=HaplotypeScore,0.000643501,0.00141643,0.00297619,0,0,0,0,0,1,1,1,0,0,0,0,0,exonic,FUCA1,nonsynonymous SNV,NM_000147:c.T1016C:p.I339T,,0.02,0.998825,C,0.725216,-9,0.0050,B,0.888728,N,0.485564,N,4.81,1,24175283,24175283,A,G
#HESN

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | grep c.A5128G
1,186303511_1,.,T,C,T,250.54,VQSRTrancheSNP99.00to99.90,AA=T;AC=1;AF=5.488e-04;AN=1822;BaseQRankSum=-6.471;DP=21581;Dels=0.00;FS=0.690;HRun=0;HaplotypeScore=2.4101;InbreedingCoeff=-0.0016;MLEAC=1;MLEAF=5.488e-04;MQ=60.00;MQ0=0;MQRankSum=0.822;QD=13.19;ReadPosRankSum=0.100;VQSLOD=1.41;culprit=HaplotypeScore,0.000643501,0.00141643,0.00297619,0,0,0,0,0,1,1,1,0,0,0,0,0,exonic,TPR,nonsynonymous SNV,NM_003292:c.A5128G:p.T1710A,,0.18,0.997877,C,0.97,D,0.640678,-9,0.999996,D,0.42172,N,5.25,1,186303511,186303511,T,C
#HESN

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | grep c.C3269T
17,78301691_1,.,C,T,C,412.60,VQSRTrancheSNP99.00to99.90,AA=C;AC=1;AF=5.593e-04;AN=1788;BaseQRankSum=6.238;DP=31467;Dels=0.00;FS=0.625;HRun=0;HaplotypeScore=3.8645;InbreedingCoeff=-0.0007;MLEAC=1;MLEAF=5.593e-04;MQ=60.00;MQ0=0;MQRankSum=-0.686;QD=11.79;ReadPosRankSum=-1.594;VQSLOD=0.488;culprit=HaplotypeScore,0.000649351,0.00142045,0.00297619,0,0,0,0,0,1,1,1,0,0,0,0,0,exonic,RNF213,nonsynonymous SNV,NM_001256071:c.C3269T:p.T1090M,,0.25,,,,,,,,,,,,17,78301691,78301691,C,T
#HESN

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.vcf.gz | grep -E '^#|68200276|3735127|24654486|129207102|24175283|186303511|78301691' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SpecificGeneVCFs/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVar.SudhirRequest20150109.vcf.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz | grep -E '^#|68200276|3735127|24654486|129207102|24175283|186303511|78301691' | gzip >  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SpecificGeneVCFs/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.AllGenesPerVar.vs2.SudhirRequest20150109.vcf.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SpecificGeneVCFs/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.AllGenesPerVar.vs2.SudhirRequest20150109.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.vs1.SudhirRequest20150109.output

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.vs1.SudhirRequest20150109.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs1.SudhirRequest20150109.output

Gene	Location	MACID
ARFGEF1	8_68200276      20219
NUP98	11_3735127      20219
IPO4	14_24654486     20219
IFT122	3_129207102     20219
FUCA1	1_24175283      20501
TPR	1_186303511     20501
RNF213	17_78301691     20501

##20150122
##Unsure where this file came from or when it was created? appears to use Samtools sort to create all-individual merge .bam file but picard-merge is what should be used, and what is used, in vs2 of the file, which is the only record of this file being used in the most recent run (e.g. no record of vs3 being used anywhere or in P2 record file)

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs3._20150122_NORECORDOFUSE_probablyusedjusttocomparesortsmaybe.sh

##20150126
##.vcf file and .pheno file for Sudhir to check and compare against GWAS results (for whatever collection of individuals overlap between our study and the GWAS one)

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/ForSudhir.P1.GATK.QCedVariants.QCedIndvs.justWhite.vcf.gz

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/ForSudhir.Phenotypes.txt

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/ForSudhir.P1.GATK.QCedVariants.QCedIndvs.justWhite.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/ForSudhir.Phenotypes.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/. 



##20150126
##Noticed the timestamp on -rw-rw-r-- 1 michaelt pg        177396566 Nov  1 19:52 AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz was later than the timestamp on -rw-rw-r-- 1 michaelt pg        182849697 Apr 11  2014 AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz but I think this was most likely because I had to recreate AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz and not because it had been updatd
##However, to check that the two files are the same, recreating the .bgzip.gz version and going to run cmp on the two files to make sure they are equivalent

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ORIG.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz | awk '{ if ($1 == "23") { $1 = "X"} print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip

/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip

 1115  cmp ORIG.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz
  1116  head -n 10 AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz
   1117  /home/shared/software/tabix-0.2.5/bgzip
    1118  cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ORIG.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ORIG.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.vs2.bgzip.gz
     1119  /home/shared/software/tabix-0.2.5/bgzip -d /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ORIG.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.vs2.bgzip.gz
      1120  ls -lrt
       1121  head -n 10 ORIG.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.vs2.bgzip
        1122  /home/shared/software/tabix-0.2.5/bgzip -d AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz
	 1123  ls -lrt
	  1124  wc ORIG.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.vs2.bgzip
	   1125  wc AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip

[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$wc ORIG.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.vs2.bgzip
     64015   50216707 1162570014 ORIG.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.vs2.bgzip
     [  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$wc AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip
          64015   50216707 1178830267 AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip

#So the files look pretty much the same but because there's a slight difference in the 'byte' count between the two, I'm going to update the bgzip file and rerun the EPACTS analyses (TruePheno and FakePhenos) just to be thorough. I will compare the output of the new run to the previous one to make sure nothing's really changed, and if for some reason something has I'll look further into it

rm ORIG.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.vs2.bgzip AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip


##20150209
##Running fastQC over fastq.gz files to doublecheck everything looked fine, just to make sure

wget http://www.bioinformatics.babraham.ac.uk/projects/fastqc/fastqc_v0.11.2.zip /home/michaelt/Software/. 

unzip /home/michaelt/Software/fastqc_v0.11.2.zip

#Adding in a command to run fastQC -- going to rerun launcher with all other steps commented out
#Going to edit launcher to include vs3 of this file, but going to keep the version of the launcher as vs3 since this is just a single digit replacement in that file

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.vs3.sh

head -n 1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output.top1
head -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output.top10

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output.top1
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output.top10
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output | perl -ane 'my $baseFile1; if ($F[1] =~ m/(.*.RL\d+).sff/) { $baseFile1 = $1; } print $F[0], "/", $1, ".QCed.fastq.gz ";' | awk '{ print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.AllfastqFilesSingleLine.output

#Below command would take too long it seems like
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.AllfastqFilesSingleLine.output | xargs /home/michaelt/Software/FastQC/fastqc

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py /home/michaelt/Scripts/Python/Python.Utility.CutFileIntoNSizedLineChunks.vs1.py

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/FileChunks

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/FileChunks/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output

python /home/michaelt/Scripts/Python/Python.Utility.CutFileIntoNSizedLineChunks.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/FileChunks/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output --chunkSize 100

for i in {1..37}
do

	cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/FileChunks/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.chunk${i}.output | perl -ane 'my $baseFile1; if ($F[1] =~ m/(.*.RL\d+).sff/) { $baseFile1 = $1; } print $F[0], "/", $1, ".QCed.fastq.gz ";' | awk '{ print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/FileChunks/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.AllfastqFilesSingleLine.chunk${i}.output

done

cp -p /home/michaelt/Scripts/Bash/BashScript.InputVarsPlusTime.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.fastQCgroupOfFiles.vs1.sh

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.fastQCgroupOfFiles.Launcher.vs1.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.fastQCgroupOfFiles.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/FileChunks/ DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.AllfastqFilesSingleLine.output




http://genomemedicine.com/content/pdf/gm432.pdf
http://www.ncbi.nlm.nih.gov/pubmed/25078893
http://www.ncbi.nlm.nih.gov/pubmed/24558117
http://www.ncbi.nlm.nih.gov/pubmed/24086590



#CHECK_O: Actually setup and do the following below
##20150209
##The following is the standard git commands I should run when updating and caching the important files/scripts/results/etc...etc... for this project



