#!/bin/sh

#Notes for interpreting this code history/'lab notebook'

#Check_# refers to notes I made to myself to return to, where _0 indicates I have not dealt with my reminder/suggestion/necessity and _1 means I have looked at/dealt with what I was indicating I should

#I do my best to provide dates and timestamps for when things are occuring -- I also get better at this, and leaving notes/comments/annotations/explanations of things as the code/labnotebook goes on

#'~~~' is commonly employed to indicate 'informational' code or results are being included and will normally have some sort of ocmment or code nearby explaining why this information is being included and/or purposely separated/emphasized

#CHECK_O: Actually setup and do the following below
##20150209
##The following is the standard git commands I should run when updating and caching the important files/scripts/results/etc...etc... for this project
##20150921 Note -- setting this up at below 
#20150921 Note -- Main location of git repository is currently /data/userdata/pg/michaelt



~~~~~~~

#qsub -I -l walltime=72:00:00,mem=8gb,nodes=1:ppn=6

#sffFile contains a list of every sequencing read/run and the breakdown/separation of each original .sff file into each MID tag
#ALL_MAPPING is where each individual is mapped back to the appropriate MID tag/.sff file combination from sffFile directory 

#Create appropriate main directories in ALL_MAPPING and then appropriate subdirectories in each main directory (main directory i.e. Pools/mapping_pool_1-46/, and subdirecory i.e. POOL14-01RL/). Then creating appropriate directories for each .sff file original location in sffFile/,  

20130823 - Going through past runs of code and figuring out what was done initially; collecting this information here as the first entry in this MainScript1/readme file for documentation (tends to be what I use instead of the LABDIRECTORY notebook thing I had previously tried to setup/use)

ls -lrt /data/userdata/pg/ALL_MAPPING/. | awk '{ print $9 } ' > home_pg_ALL_MAPPING_dirList.20130513.txt

bash CollectLSInfoPerDir.sh

bash DataOrganization.CollectOrigSFFFileInformation.sh

cat ../sffFiles/home_pg_ALL_MAPPING_ALL_Pools_sff_OrigLocations.Unedited.20130513.txt | sort | uniq > ../sffFiles/home_pg_ALL_MAPPING_ALL_Pools_sff_OrigLocations.Unedited.20130513.SortUniq.txt

#bash ../sffFiles/BreakUpsffFiles.sh
bash ../sffFiles/MakeDir.sh

bash DataOrganization.CombinePoolNameswsffFileIDs.sh

#Edit DataOrganization.CombinePoolNameswsffFileIDs.Thu_May_16_14:48:16_CDT_2013.output -- commenting out lines that are incorrect with '#'

bash DataOrganization.ConnectPoolsffFileswithIndvMIDRLsffFiles.sh DataOrganization.CombinePoolNameswsffFileIDs.Thu_May_16_14:48:16_CDT_2013.Edited.output

bash DataOrganization.CollectAllIndvMIDRLsffFiles.sh

bash ../sffFiles/Process_sffFile.BreakUpMIDs.Launcher.vs1.sh

bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs1.sh DataOrganization.CollectAllIndvMIDRLsffFiles.Thu_May_16_14:54:21_CDT_2013.RedundantDirectoriesRemoved.output

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs1.sh DataOrganization.CollectAllIndvMIDRLsffFiles.Thu_May_16_14:54:21_CDT_2013.RedundantDirectoriesRemoved.output 0


2nd run with pools mapping_pool_91-97 and mapping_pool_98-105, as well as edits + updating code
~~~~~~~~~~~

ls -lrt /data/userdata/pg/ALL_MAPPING/. | awk '{ print $9 } ' > home_pg_ALL_MAPPING_dirList.20130827.txt

cat home_pg_ALL_MAPPING_dirList.20130827.txt > home_pg_ALL_MAPPING_dirList.20130827.NotCreatedYet.txt
#Then manually edited to remove directories already used/processed before -- kept mapping_pool_91-97 and mapping_pool_98-105

bash DataOrganization.CollectLSInfoPerDir.vs2.sh home_pg_ALL_MAPPING_dirList.20130827.NotCreatedYet.txt #Creates subdirectory folder-list per directory

bash DataOrganization.MakeDir.vs2.sh home_pg_ALL_MAPPING_dirList.20130827.NotCreatedYet.txt #Creates the rest of the sub-subdirectories, etc. in ALL_MAPPING

bash DataOrganization.CollectOrigSFFFileInformation.vs2.sh home_pg_ALL_MAPPING_dirList.20130827.NotCreatedYet.txt #Creates home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.${date1}.txt file

cat ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130828_123042.txt | sort | uniq > ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130828_123042.SortUniq.txt

#Edit -- Directory location for sff file H37BKPH01 was changed from /data/454/raw/2013/R_2013_02_08_11_30_15_FLX08100638_Administrator_CS_KK_PG_MM5_Pool_91-2/D_2013_02_09_15_00_17_node1_altSignalProcessing/sff/H37BKPH01.sff to /data/454/raw/2013/R_2013_02_08_11_30_15_FLX08100638_Administrator_CS_KK_PG_MM5_Pool_91-2/D_2013_02_12_10_29_52_node2_signalProcessing/sff/H37BKPH01.sff (and *02.sff)
(bash DataOrganization.CorrectPool91Script.vs1.sh)
#Edit -- Change made in DataOrganization.CombinePoolNameswsffFileIDs.vs2.20130828_124947.Edited.output as well too 

#bash ../sffFiles/BreakUpsffFiles.vs2.sh ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130828_123042.SortUniq.txt
bash ../sffFiles/DataOrganization.MakeDir.vs2.sh ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130828_123042.SortUniq.Edited.txt

bash DataOrganization.CombinePoolNameswsffFileIDs.vs2.sh home_pg_ALL_MAPPING_dirList.20130827.NotCreatedYet.txt ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130828_123042.SortUniq.Edited.txt #Creates file such as DataOrganization.CombinePoolNameswsffFileIDs.vs2.${date1}.output

bash DataOrganization.ConnectPoolsffFileswithIndvMIDRLsffFiles.vs2.sh home_pg_ALL_MAPPING_dirList.20130827.NotCreatedYet.txt DataOrganization.CombinePoolNameswsffFileIDs.vs2.20130828_124947.Edited.output 

bash DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.sh home_pg_ALL_MAPPING_dirList.20130827.NotCreatedYet.txt #Creates DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.${date1}.output file 

bash ../sffFiles/DataProcessing.BreakUpMIDs.Launcher.vs2.sh ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130828_123042.SortUniq.Edited.txt

bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130828_141053.output

#cat DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130828_141053.output | grep -E 'H37BKPH|H4KITRJ|H4PUW3D' > DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130828_141053.Leftover.output

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130828_141053.output

#cp -p home_pg_ALL_MAPPING_dirList.20130827.txt home_pg_ALL_MAPPING_dirList.WorkingMainList.txt
#cat DataOrganization.CombinePoolNameswsffFileIDs.Thu_May_16_14\:48\:16_CDT_2013.Edited.output DataOrganization.CombinePoolNameswsffFileIDs.vs2.20130828_124947.output > DataOrganization.CombinePoolNameswsffFileIDs.vs2.WorkingMainFile.output #Combining all sections. Trying to use 'WorkingMainFile' as means to keep all parts together
#cat DataOrganization.CollectAllIndvMIDRLsffFiles.Thu_May_16_14\:54\:21_CDT_2013.RedundantDirectoriesRemoved.JustFirstTwoCols.output DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130828_141053.output > DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.output
#cp -p DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.output DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output 
#Made DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output  file by changing all earlier directories for mapping_pools.... to include the /Pools/... directory via the following command in vi .,$ s/ING\/mapping_pool/ING\/Pools\/mapping_pool/g 

#At this point catching any .fastq files or other downstream processed files that were lost or not properly processed in order for them to be resubmitted
bash DataProcessing.Pipeline.CheckForErroneousFiles.vs1.sh home_pg_ALL_MAPPING_dirList.WorkingMainList.txt DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output fastq #Create DataProcessing.Pipeline.CheckForErroneousFiles.vs1.DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.Redo.${SearchTerm1}.${date1}.output file

bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataProcessing.Pipeline.CheckForErroneousFiles.vs1.DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.Redo.fastq.20130830_165929.output

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataProcessing.Pipeline.CheckForErroneousFiles.vs1.DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.Redo.fastq.20130830_165929.output


3rd run with mapping_pool_106-124
~~~~~~~~~~~

ls -lrt /data/userdata/pg/ALL_MAPPING/. | awk '{ print $9 } ' > home_pg_ALL_MAPPING_dirList.20130831.txt

cat home_pg_ALL_MAPPING_dirList.20130831.txt > home_pg_ALL_MAPPING_dirList.20130831.NotCreatedYet.txt
#Then manually edited to remove directories already used/processed before -- kept mapping_pool_106-124

bash DataOrganization.CollectLSInfoPerDir.vs2.sh home_pg_ALL_MAPPING_dirList.20130831.NotCreatedYet.txt #Creates subdirectory folder-list per directory

bash DataOrganization.MakeDir.vs2.sh home_pg_ALL_MAPPING_dirList.20130831.NotCreatedYet.txt #Creates the rest of the sub-subdirectories, etc. in ALL_MAPPING

bash DataOrganization.CollectOrigSFFFileInformation.vs2.sh home_pg_ALL_MAPPING_dirList.20130831.NotCreatedYet.txt #Creates home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.${date1}.txt file

cat ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130831_103251.txt | sort | uniq > ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130831_103251.SortUniq.txt

bash ../sffFiles/DataOrganization.MakeDir.vs2.sh ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130831_103251.SortUniq.txt

bash DataOrganization.CombinePoolNameswsffFileIDs.vs2.sh home_pg_ALL_MAPPING_dirList.20130831.NotCreatedYet.txt ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130831_103251.SortUniq.txt #Creates file such as DataOrganization.CombinePoolNameswsffFileIDs.vs2.${date1}.output

bash DataOrganization.ConnectPoolsffFileswithIndvMIDRLsffFiles.vs2.sh home_pg_ALL_MAPPING_dirList.20130831.NotCreatedYet.txt DataOrganization.CombinePoolNameswsffFileIDs.vs2.20130831_103251.output 

bash DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.sh home_pg_ALL_MAPPING_dirList.20130831.NotCreatedYet.txt #Creates DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.${date1}.output file 

bash ../sffFiles/DataProcessing.BreakUpMIDs.Launcher.vs2.sh ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130831_103251.SortUniq.txt

bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130831_103352.output

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130831_103352.output

cat ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130831_103251.SortUniq.txt | grep H85C36T01 > ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130831_103251.SortUniq.Leftover.txt #Then ran ../sffFiles/DataProcessing.BreakUpMIDs.Launcher.vs2.sh

cat DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130831_103352.output | grep H85C36T01 > DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130831_103352.Leftover.output #Then ran DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh and DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh

#Also H9H1D9W01

#cat DataOrganization.CombinePoolNameswsffFileIDs.Thu_May_16_14\:48\:16_CDT_2013.Edited.output DataOrganization.CombinePoolNameswsffFileIDs.vs2.20130828_124947.output DataOrganization.CombinePoolNameswsffFileIDs.vs2.20130831_103251.output > DataOrganization.CombinePoolNameswsffFileIDs.vs2.WorkingMainFile.output
#cat DataOrganization.CollectAllIndvMIDRLsffFiles.Thu_May_16_14\:54\:21_CDT_2013.RedundantDirectoriesRemoved.JustFirstTwoCols.output DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130828_141053.output DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130831_103352.output > DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.output
#cp -p DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.output DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output 

bash DataProcessing.Pipeline.CheckForErroneousFiles.vs2.sh home_pg_ALL_MAPPING_dirList.WorkingMainList.txt DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output fastq

bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataProcessing.Pipeline.CheckForErroneousFiles.vs1.DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.Redo.fastq.20130831_235915.output

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataProcessing.Pipeline.CheckForErroneousFiles.vs1.DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.Redo.fastq.20130831_235915.output

cat home_pg_ALL_MAPPING_ALL_Pools_sff_OrigLocations.Unedited.20130513.SortUniq.txt home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130828_123042.SortUniq.Edited.txt home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130831_103251.SortUniq.txt > home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.WorkingMainList.SortUniq.txt

bash ../sffFiles/DataProcessing.BreakUpMIDs.Launcher.vs2.sh ../sffFiles/DataProcessing.Pipeline.CheckForErroneousFiles.vs1.DataProcessing.BreakUpMIDs.vs1.WorkingMainFile.Edited.Redo.fastq.20130831_235915.output


20130903 -- Created the PreMerge and PostMerge Pools/separation in the folder hierarchy; began continued development of pt3/the merging/dupl/BQSR steps in the pipeline
~~~~~~~~~~~
PreMerge

bash DataProcessing.RemoveaCertainFile.vs1.sh home_pg_ALL_MAPPING_dirList.WorkingMainList.txt PreMerge #Cleaned up old .preGATK* and *sam.gz files so can more easily locate folders where the pt2 code did not run properly (e.g. a lack of these files existing versus trying to find files with the proper timestamp signature)

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output #Redoing everything

#Note -- at this point changed original .pt3 of the pipeline scripts to .pt4 and made the 'merge file' portion of the original .pt3 into a stand alone .pt3

bash DataOrganization.AddOrRemoveRGInformation.Launcher.vs1.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output #Run of DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh included an error for the @RG information so used an extra script to correct every files. This is not intended to be normally used in the pipeline and the DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.vs2.sh script has been corrected

bash DataOrganization.CreatebamMergeList.vs1.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output #Creates file DataOrganization.CreatebamMergeList.vs1.${date1}.output

bash DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20130909_231450.output

bash DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20130909_231450.output

#Possibly was going to drop reads that did not map to expected probes but delaying this decisions since some issues came out. Will come back to figure out if this is proper after putting the rest of the pipeline together
#samtools view POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.bam | python ../../../../DataProcessing.Pipeline.DropOffTargetReads.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/OID41305_HG19_Match5_probes/OID41305_HG19_Match5_probe_coverage.bed --file2 - | vi -
#R --vanilla --slave --args < ../../../../DataProcessing.Pipeline.DropOffTargetReads.plot.R /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/OID41305_HG19_Match5_probes/OID41305_HG19_Match5_probe_coverage.chr4.bed POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.chr4.formattted POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.chr4.OverlapPlot.jpeg
#R --vanilla --slave --args < ../../../../DataProcessing.Pipeline.DropOffTargetReads.plot.R /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/OID41305_HG19_Match5_probes/OID41305_HG19_Match5_probe_coverage.chr4.bed 454Contigs.chr4.formatted 454Contigs.chr4.OverlapPlot.jpeg
#
#20131029 edited
#R --vanilla --slave --args < ../../../../DataProcessing.Pipeline.DropOffTargetReads.plot.R /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.chr4.bed POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.chr4.formattted POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.chr4.OverlapPlot.120326_HG19_SW_MM5_EZ.jpeg

#Created pt5 to be run on a single interactive node, but running ReduceReads and creating a raw .vcf file for the entire, merged .bam file at once was too long. Created support scripts to break up merged .bam file into separate chromosome .bam files so that raw .vcfs could be made and ReduceReads could be run
#bash DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged DataOrganization.CreatebamMergeList.vs1.20130916_105823.output

bash DataProcessing.Pipeline.pt5a.SplitBamIntoChromosomes.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.preGATK.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt5b.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.preGATK.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt5c.RunReduceReadsPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.preGATK.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged


20130923 - 4th run with mapping_pool_SOLID
~~~~~~~~~~~

#Manually created directories in ALL_MAPPING Pools subfolders following the excel spreadsheet POP_genomics_sample_information_2 and used 'touch' to create expected .sff files using ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130925_103251.SortUniq.txt created below to determine who got what .sff file

ls -lrt /data/454/raw/2011/*PG*/*/sff/. | grep sff | grep -E 'raw|H' | perl -sane 'if ($F[0] =~ m/raw/) { $tempVal1 = $F[0]; } else { if ($tempVal1 =~ m/(.*)(\.\:)/) { print $1, $F[8], "\n"; } }' > ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130925_103251.SortUniq.txt 

#Manually added in SOLID pools from 2012 directory

bash ../sffFiles/DataOrganization.MakeDir.vs2.sh ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130925_103251.SortUniq.txt

bash DataOrganization.CombinePoolNameswsffFileIDs.vs2.sh home_pg_ALL_MAPPING_dirList.20130831.NotCreatedYet.txt ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130925_103251.SortUniq.txt #Creates file such as DataOrganization.CombinePoolNameswsffFileIDs.vs2.${date1}.output

#Had to clean up file DataOrganization.CombinePoolNameswsffFileIDs.vs2.20130925_150954.output manually

bash DataOrganization.ConnectPoolsffFileswithIndvMIDRLsffFiles.vs2.sh home_pg_ALL_MAPPING_dirList.20130831.NotCreatedYet.txt DataOrganization.CombinePoolNameswsffFileIDs.vs2.20130925_150954.output

bash DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.sh home_pg_ALL_MAPPING_dirList.20130831.NotCreatedYet.txt #Creates DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.${date1}.output file

cp -p DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130925_152507.output DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130925_152507.Edited.output

#Manually removed some erroneous entries from DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130925_152507.Edited.output 

bash ../sffFiles/DataProcessing.BreakUpMIDs.Launcher.vs2.sh ../sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs2.20130925_103251.SortUniq.txt

bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130925_152507.Edited.output

#NOTE: At this point rerunning everything with QC pipeline components

DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh

bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130925_152507.Edited.output

#DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output did not include mapping_SOLID runs yet at this point. Updated the file afterwards (since there were different needs for this pt1 script in particular for each group of folder/files that would not matter with later downstream pts).
bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output

cat DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.20130925_152507.Edited.output > asdadad

mv asdadad DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output 

bash DataProcessing.Pipeline.CheckForPresenceOfCertainFile.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output PreMerge QCed.preGATK.QCed.bam

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataProcessing.Pipeline.CheckForPresenceOfCertainFile.vs2.QCed.preGATK.QCed.bam.20130928_134627.output

bash DataOrganization.CreatebamMergeList.vs1.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output #Creates file DataOrganization.CreatebamMergeList.vs1.${date1}.output

#Manually merged POOL39_2-... with POOL39-... lines

bash DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20130928_145300.output

bash DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20130928_145300.output

#Manually curated 20130930_POP_genomics_sample_information_2.edited.csv to look for duplicate entries and remove those that had less coverage
#Ran below code to cut down DataOrganization.CreatebamMergeList.vs1.20130928_145300.output to only include those individuals being used (and not those either missing IDs, exist as duplicates, etc.). For the moment deciding not to merge individuals who were run twice on MM5 (maybe bias with extra coverage on just a few individuals?) or run on SOLID and MM5. Also generally opted to use MM5 over SOLID unless coverage in SOLID was much better than MM5 (which was generally not the case). A few individuals were run twice on MM5, versus on SOLID and MM5
#cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20130930_POP_genomics_sample_information_2.edited.noMeta.dropIndvs.csv | grep ool | sort | wc -- to help check expected total number of entries to be kept when using DataProcessing.Pipeline.DropIndividualsBasedOnPopGenomFile.vs2.py with 20130930_POP_genomics_sample_information_2.edited.noMeta.dropIndvs.csv

python DataProcessing.Pipeline.DropIndividualsBasedOnPopGenomFile.vs2.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20130930_POP_genomics_sample_information_2.edited.noMeta.dropIndvs.csv --file2 DataOrganization.CreatebamMergeList.vs1.20130928_145300.output > DataOrganization.CreatebamMergeList.vs1.20130928_145300.dropIndvs.output

#bash DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged DataOrganization.CreatebamMergeList.vs1.20130928_145300.dropIndvs.output

bash DataProcessing.Pipeline.pt6a.SplitBamIntoChromosomes.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6b.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6c.RunReduceReadsPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

python DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20130930_POP_genomics_sample_information_2.edited.noMeta.dropIndvs.csv --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Sample_status_list_9-16-13_CT.edited.noMeta.csv > DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output
echo "macsid macsid MID# POOLID-MID# race posage hesn lesn lesc othersc vrapid rapid vslow slow timetoART timeaids" > DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output.header 
#20131004 Note -- four individuals not present in Sample_status_list file, 41478,10,POOL43-10RL
#20040,7,POOL18S-07RL
#30550,11,POOL101-11RL
#20770,3,POOL2S-03RL
#Unsure why at current point, will consult with Kevin/Sudhir after first run through of downstream analysis is done. Also, with these four individuals, total count is 756, not 757. Kevin suggested should have 757 individuals in total, so possibly missing someone? Unless POOL39 is being double counted by their count as well
#20131004 Note -- POOL57 MID10 has folder name ...-010RL for whatever reason originally, probably typo. Just fyi to be aware.

#20131004 Note -- at this point changed naming schematics to go from pt6a, pt6b, pt6c, etc... to go to pt6.All.pta, pt6.Samtools.pta, pt6.GATK.pta, etc... to delineate the different branching paths of the analysis steps for each analysis/variant-calling tool being used, with steps/code being used by all branches given the pt6.All.pt.... designation

python DataProcessing.Pipeline.CreatePhenotypeFile.vs1.py --file1 DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno

python DataProcessing.Pipeline.CreateCovariateFile.vs1.py --file1 DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar | awk '{ if ($5 == "1") { print $1, "\t", $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDIIDs

bash DataProcessing.Pipeline.pt6.All.ptb.ConvertVCFtoPEDPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools 

bash DataProcessing.Pipeline.pt6.All.ptc.CollapsePedFilesIntoSingleFile.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools

bash DataProcessing.Pipeline.pt6.All.ptd.PLINKlogisticRegression.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools

(20131012 -- time pt check)

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt --extract AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.bim.SNPsonly.snpIDs --r2 --ld-window-r2 0.8 --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r8 &

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt --extract AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.bim.SNPsonly.snpIDs --r2 --ld-window-r2 0 --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0 &

python ../../../../DataProcessing.Pipeline.pruneLDbased.py --file1 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld --r2 .1 > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt &

python ../../../../DataProcessing.Pipeline.pruneLDbased.py --file1 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld --r2 .8 > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump80.txt &

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt | awk '{ print $1 }' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt.snpList

perl /home/michaelt/Scripts/Perl/ListSubset.pl --file1 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt.snpList --subset 250000 --output AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt.snpList.250k

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt --extract AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt.snpList.250k --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k --recode --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.ped

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.pedind

ln -s AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.bim AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.pedsnp

/home/michaelt/Software/EIG5.0.1/bin/smartpca -p AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.parfile
#Note -- ran this on spudhead b/c smartpca was working already there but wasn't on olivia/miranda yet. Will ask Aleksandar to install eigenstrat if possible at some point

python ../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.Results.pca.altrd.race
#scp'ing the necessary files back and forth from work computer and spudhead

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.Results.pca.altrd.race -c 1:2 -p American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic:White_non-Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.Results.pca.altrd.race.xtxt

#/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.Results.pca.altrd.race -c 1:2 -p American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic:White_non-Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.Results.pca.altrd.race.xtxt

20131013
~~~~~~~~~~~
#At this point noticed names were off in that POOL01S, POOL02S, etc... were not included, just POOL01, POOL02, etc...so had duplicates, i.e. SOLID array names not captured properly in the .bam files and all subsequent processes. Reran SOLID arrays and then merge/post-merge scripts

cat DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output | grep SOLID > DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output.SOLID

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output.SOLID

cat DataOrganization.CreatebamMergeList.vs1.20130928_145300.dropIndvs.output | grep SOLID > DataOrganization.CreatebamMergeList.vs1.20130928_145300.dropIndvs.output.SOLID

bash DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20130928_145300.dropIndvs.output.SOLID

bash DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20130928_145300.dropIndvs.output.SOLID

#bash DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged DataOrganization.CreatebamMergeList.vs1.20130928_145300.dropIndvs.output

bash DataProcessing.Pipeline.pt6.All.pta.SplitBamIntoChromosomes.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.Samtools.pta.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.GATK.pta.RunReduceReadsPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.All.ptb.ConvertVCFtoPEDPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/Samtools AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools

bash DataProcessing.Pipeline.pt6.All.ptc.CollapsePedFilesIntoSingleFile.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/Samtools AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.bim | perl -lane 'if (($F[4] =~ m/^[AGTC]$/) && ($F[5] =~ m/^[AGTC]$/)) { print join("\t", @F);}' > Pools/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.bim.SNPsonly

cat Pools/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.bim.SNPsonly | awk '{ print $2 }' > Pools/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.bim.SNPsonly.snpIDs

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt --extract AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.bim.SNPsonly.snpIDs --r2 --ld-window-r2 0.8 --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r8 &

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt --extract AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.bim.SNPsonly.snpIDs --r2 --ld-window-r2 0 --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0 &

python ../../../../DataProcessing.Pipeline.pruneLDbased.py --file1 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld --r2 .1 > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt &

python ../../../../DataProcessing.Pipeline.pruneLDbased.py --file1 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld --r2 .8 > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump80.txt &

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt | awk '{ print $1 }' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt.snpList

perl /home/michaelt/Scripts/Perl/ListSubset.pl --file1 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt.snpList --subset 250000 --output AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt.snpList.250k

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt --extract AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNPsonly.r0.ld.r2clump10.txt.snpList.250k --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k --recode --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.ped

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.pedind

ln -s AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.bim AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.pedsnp

/home/michaelt/Software/EIG5.0.1/bin/smartpca -p AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.parfile
#Note -- ran this on spudhead b/c smartpca was working already there but wasn't on olivia/miranda yet. Will ask Aleksandar to install eigenstrat if possible at some point

python ../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.Results.pca.altrd.race

python ../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.noOutlierRmvl.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.Results.pca.altrd.race -c 1:2 -p American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic:White_non-Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.250k.phenoColEdit.Results.pca.altrd.race.xtxt


bash DataProcessing.Pipeline.pt6.All.ptd.PLINKlogisticRegression.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools

bash DataProcessing.Pipeline.pt6.GATK.ptb.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged 

bash ../../../../DataProcessing.Pipeline.pt6.GATK.ptc.MergeVCFsAcrossChromosomes.vs1.sh

bash DataProcessing.Pipeline.pt6.GATK.ptd.RecalAndApplyVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

vcftools --gzvcf AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz --plink --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP

plink --file AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP

bash DataProcessing.Pipeline.pt6.All.ptd.PLINKlogisticRegression.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --maf .01 --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01

perl /home/michaelt/Scripts/Perl/ListSubset.pl --file1 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.bim --subset 250000 --output AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.bim

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.bim | awk '{ print $2 }' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.bim.onlySNPids

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01 --extract AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.bim.onlySNPids --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k --recode --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.ped

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.pedind 

ln -s AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.bim AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.pedsnp

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --maf .01 --geno .95 --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95 --recode --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.ped

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.pedind

ln -s AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.bim AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.pedsnp

/home/michaelt/Software/EIG5.0.1/bin/smartpca -p AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.parfile

python ../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.Results.pca.altrd.race

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.Results.pca.altrd.race.plot.1vs2.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.Results.pca.altrd.race.plot.3vs4.xtxt

python ../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.Results.pca.altrd.race

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.Results.pca.altrd.race.plot.1vs2.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.Results.pca.altrd.race.plot.3vs4.xtxt

python ../../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.noOutlierRmvl.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race

python ../../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.noOutlierRmvl.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race.plot.1vs2.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race.plot.3vs4.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race.plot.1vs2.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race.plot.3vs4.xtxt


plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt --maf .01 --geno .95 --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.bim | perl -lane 'if (($F[4] =~ m/^[AGTC]$/) && ($F[5] =~ m/^[AGTC]$/)) { print join("\t", @F);}' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.bim.SNPsonly

perl /home/michaelt/Scripts/Perl/ListSubset.pl --file1 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.bim.SNPsonly --subset 250000 --output AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.250k.bim.SNPsonly

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.250k.bim.SNPsonly | awk '{ print $2 }' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.250k.bim.SNPsonly.onlySNPids

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95 --extract AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.250k.bim.SNPsonly.onlySNPids --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k --recode --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.ped

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.pedind

ln -s AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.bim AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.pedsnp

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/smartpca -p AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.250k.parfile

python ../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.Results.pca.altrd.race

python ../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.noOutlierRmvl.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.Results.pca.altrd.race.plot.1vs2.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.Results.pca.altrd.race.plot.3vs4.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race.plot.1vs2.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.maf01.geno95.SNPsonly.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race.plot.3vs4.xtxt


python DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.Results.pca --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.5PCs.covar
#Edit code in between runs
python DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.Results.pca --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.5PCs.covar | awk '{ print $1, "\t", $2 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.5PCs.covar.FIDIIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.5PCs.covar | awk '{ if ($5 == "1") { print $1, "\t", $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.5PCs.covar.WhiteOnly.FIDIIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar | awk '{ print $1, "\t", $2 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar | awk '{ if ($5 == "1") { print $1, "\t", $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.WhiteOnly.FIDIIDs

bash DataProcessing.Pipeline.pt6.All.ptd.PLINKlogisticRegression.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP
#Edit code in between runs
bash DataProcessing.Pipeline.pt6.All.ptd.PLINKlogisticRegression.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.5PCs.covar --covar-number 1,2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.5PCs.covar.WhiteOnly.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno1.5PCsmaf01geno95.Assoc 

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.5PCs.covar --covar-number 1,2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.5PCs.covar.WhiteOnly.FIDIIDs --maf .01 --geno .95 --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.WhitesOnly.Pheno1.5PCsmaf01geno95.Assoc 

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.WhiteOnly.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno1.10PCsmaf01geno95.Assoc 

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.WhiteOnly.FIDIIDs --maf .01 --geno .95 --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.WhitesOnly.Pheno1.10PCsmaf01geno95.Assoc 

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.Assoc

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.5PCsmaf01geno05.Assoc

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.10PCsmaf01geno05.Assoc

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno1.5PCsmaf01geno95.Assoc.assoc.logistic Pheno1_Logistic_Whites_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno1.5PCsmaf01geno95.Assoc.assoc.logistic.QQplot.jpeg & 

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.WhitesOnly.Pheno1.5PCsmaf01geno95.Assoc.assoc.logistic Pheno1_Logistic_Whites_maf01geno95_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.WhitesOnly.Pheno1.5PCsmaf01geno95.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno1.10PCsmaf01geno95.Assoc.assoc.logistic Pheno1_Logistic_Whites_10PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno1.10PCsmaf01geno95.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.WhitesOnly.Pheno1.10PCsmaf01geno95.Assoc.assoc.logistic Pheno1_Logistic_Whites_maf01geno95_10PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.WhitesOnly.Pheno1.10PCsmaf01geno95.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.Assoc.assoc.logistic Pheno1_Logistic AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.5PCsmaf01geno05.Assoc.assoc.logistic Pheno1_Logistic_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.5PCsmaf01geno05.Assoc.assoc.logistic.QQPlot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.10PCsmaf01geno05.Assoc.assoc.logistic Pheno1_Logistic_10PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.10PCsmaf01geno05.Assoc.assoc.logistic.QQPlot.jpeg &

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --maf .01 --geno .95 --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno1.5PCsmaf01geno05.Assoc

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --maf .01 --geno .95 --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno1.10PCsmaf01geno05.Assoc

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno1.5PCsmaf01geno05.Assoc.assoc.logistic Pheno1_Logistic_maf01geno95_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno1.5PCsmaf01geno05.Assoc.assoc.logistic.QQplot.jpeg & 

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno1.10PCsmaf01geno05.Assoc.assoc.logistic Pheno1_Logistic_maf01geno95_10PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno1.10PCsmaf01geno05.Assoc.assoc.logistic.QQplot.jpeg & 

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.NoCovar.Assoc

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.justPOOLcovar.Assoc

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 2 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.justMIDcovar.Assoc

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.noPOOLMIDcovar.5PCsmaf01geno05.Assoc

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.NoCovar.Assoc.assoc.logistic Pheno1_Logistic_noCovar AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.NoCovar.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.justPOOLcovar.Assoc.assoc.logistic Pheno1_Logistic_POOLCovar AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.justPOOLcovar.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.justMIDcovar.Assoc.assoc.logistic Pheno1_Logistic_MIDCovar AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.justMIDcovar.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.noPOOLMIDcovar.5PCsmaf01geno05.Assoc.assoc.logistic Pheno1_Logistic_noPOOLMID_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno1.noPOOLMIDcovar.5PCsmaf01geno05.Assoc.assoc.logistic.QQplot.jpeg &

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 2 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno2.5PCsmaf01geno05.Assoc

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno2.5PCsmaf01geno05.Assoc.assoc.logistic Pheno2_Logistic_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno2.5PCsmaf01geno05.Assoc.assoc.logistic.QQplot.jpeg &  

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 2 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno2.Assoc

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 2 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.WhiteOnly.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno2.5PCsmaf01geno05.Assoc

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 2 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --maf .01 --geno .95 --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno2.5PCsmaf01geno05.Assoc

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno2.Assoc.assoc.logistic Pheno2_Logistic AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno2.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno2.5PCsmaf01geno05.Assoc.assoc.logistic Pheno2_Logistic_WhitesOnly_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno2.5PCsmaf01geno05.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno2.5PCsmaf01geno05.Assoc.assoc.logistic Pheno2_Logistic_maf01geno95_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno2.5PCsmaf01geno05.Assoc.assoc.logistic.QQplot.jpeg &

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz | grep PASS | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.body.gz

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz | wc
1272767 968575687 5440128566

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz | awk '{ if ($1 == "12" ) { print $0 } } ' | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr12.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz

perl /home/michaelt/Scripts/Perl/ListSubset.pl --file1 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr12.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz --subset 10000 --output AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr12.GATK.ReduceReads.UG.VQSR.SNP.vcf.ran10000

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz | head -n 123 | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.header.gz

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.header.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.body.gz | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz

rm AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.body.gz 

vcftools --gzvcf AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz --plink --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS

plink --file AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS

perl /home/michaelt/Scripts/Perl/ListSubset.pl --file1 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.bim --subset 250000 --output AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.bim.250k

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.bim.250k | awk '{ print $2 }' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.bim.250k.snpIDs

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS --extract AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.bim.250k.snpIDs --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k --recode --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.ped

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.pedind

ln -s AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.bim AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.pedsnp

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/smartpca -p AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.parfile

python ../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.Results.pca.altrd.race

python ../../../../DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 ../../../../DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20130930_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.noOutlierRmvl.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.Results.pca.altrd.race.plot.1vs2.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.Results.pca.altrd.race.plot.3vs4.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race.plot.1vs2.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Black_Hispanic:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.phenoColEdit.noOutlierRmvl.Results.pca.altrd.race.plot.3vs4.xtxt

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 3 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno3.5PCsmaf01geno05.Assoc

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno3.5PCsmaf01geno05.Assoc.assoc.logistic Pheno3_Logistic_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno3.5PCsmaf01geno05.Assoc.assoc.logistic.QQplot.jpeg &  

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 3 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno3.Assoc

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 3 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.WhiteOnly.FIDIIDs --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno3.5PCsmaf01geno05.Assoc

plink --bfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 3 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar --covar-number 1,2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.10PCs.covar.FIDIIDs --maf .01 --geno .95 --logistic --allow-no-sex --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno3.5PCsmaf01geno05.Assoc

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno3.Assoc.assoc.logistic Pheno3_Logistic AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.Pheno3.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno3.5PCsmaf01geno05.Assoc.assoc.logistic Pheno3_Logistic_WhitesOnly_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.WhitesOnly.Pheno3.5PCsmaf01geno05.Assoc.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno3.5PCsmaf01geno05.Assoc.assoc.logistic Pheno3_Logistic_maf01geno95_5PCsmaf01geno95 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.Pheno3.5PCsmaf01geno05.Assoc.assoc.logistic.QQplot.jpeg &


PASS run

python DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.Results.pca --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.PASS.maf01.geno95.phenoColEdit.5PCs.covar
#Edit code in between runs
python DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.maf01.geno95.phenoColEdit.Results.pca --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.GATK.ReduceReads.UG.VQSR.SNP.PASS.maf01.geno95.phenoColEdit.10PCs.covar


Annovar (timepoint chk: 20131029)
perl /home/michaelt/Software/annovar/annotate_variation.pl --geneanno  /home/michaelt/Software/annovar/ -buildver hg19 #annotation: 1        861283          861283          G       C
perl /mnt/lustre/home/mturchin20/Software/annovar/annotate_variation.pl --geneanno hg19_Variants4ESP6515.AnnovarFormat.txt /mnt/lustre/home/mturchin20/Software/annovar/humandb/ -buildver hg19
perl /mnt/lustre/home/mturchin20/Software/annovar/summarize_annovar.pl hg19_Variants4ESP6515.AnnovarFormat.txt /mnt/lustre/home/mturchin20/Software/annovar/humandb/ -buildver hg19

python DataPrcoessing.Pipeline.Utility.LinkGeneInfoListwTxStStpInfo.vs1.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/liquid_array.csv > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/liquid_array.wChrTrxStartStop.vs2.csv
#This script used the geneID symbol to compare the two datasets. Decided this was not adequate and used the UCSC ID and HugoID in the second version of the code below

python DataPrcoessing.Pipeline.Utility.LinkGeneInfoListwTxStStpInfo.vs2.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /home/michaelt/Data/Hugo/HGNC_GeneInformation_wUCSCIDs.txt --file3 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/liquid_array.csv > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/liquid_array.wChrTrxStartStop.vs3.csv

python DataProcessing.Pipeline.pt6.All.pte.annotateVariantsManuallyGeneID.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/liquid_array.wChrTrxStartStop.vs3.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr12.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr12.GATK.ReduceReads.UG.VQSR.SNP.vcf.selfAnnotation.gz

python DataProcessing.Pipeline.pt6.All.pte.annotateVariantsManuallyGeneID.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/liquid_array.wChrTrxStartStop.vs3.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.selfAnnotation.gz

python DataProcessing.Pipeline.pt6.All.pte.annotateVariantsManuallyGeneID.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/liquid_array.wChrTrxStartStop.vs3.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr12.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz --findClosest 1 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr12.GATK.ReduceReads.UG.VQSR.SNP.vcf.selfAnnotation.gz

python DataProcessing.Pipeline.pt6.All.pte.annotateVariantsManuallyGeneID.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/liquid_array.wChrTrxStartStop.vs3.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz --findClosest 1 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.selfAnnotation.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz | awk '{ print $1, "\t", $2, "\t", $2, "\t", $4, "\t", $5 }' | grep -v ^# > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.AnnovarFormat

#perl /home/michaelt/Software/annovar/annotate_variation.pl --downdb refGene /home/michaelt/Software/annovar -build hg19
#PWD: /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK

perl /home/michaelt/Software/annovar/annotate_variation.pl --geneanno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.AnnovarFormat /home/michaelt/Software/annovar/ -buildver hg19 

perl /home/michaelt/Software/annovar/summarize_annovar.pl /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.AnnovarFormat /home/michaelt/Software/annovar/ --ver1000g 1000g2012apr --verdbsnp 129 --veresp 6500 --buildver hg19 --remove

#perl /home/michaelt/Software/annovar/annotate_variation.pl -filter -dbtype ljb2_all -buildver hg19 -outfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.AnnovarFormat.ljb2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.AnnovarFormat /home/michaelt/Software/annovar/ -otherinfo




20131111 - 5th run fine-tuning QC setup from beginning of fastq file processing, so redoing everything from the start; still all SOLID and MM5 individuals
~~~~~~~~~~~

mv DataPrcoessing.Pipeline.Utility.LinkGeneInfoListwTxStStpInfo.vs2.py DataProcessing.Pipeline.Utility.LinkGeneInfoListwTxStStpInfo.vs2.py
mv DataPrcoessing.Pipeline.Utility.LinkGeneInfoListwTxStStpInfo.vs1.py DataProcessing.Pipeline.Utility.LinkGeneInfoListwTxStStpInfo.vs1.py

bash DataProcessing.Pipeline.Utility.GetFastqPreQCStats.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output

bash DataProcessing.Pipeline.Utility.RemoveaCertainFile.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output #To remove QCed.fastq.gz and associated stats files for new run through

bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output

mv DataProcessing.Pipeline.Utility.GetFastqPreQCStats.sh DataProcessing.Pipeline.Utility.GetFastqPreQCStats.vs1.sh

cp -p DataProcessing.Pipeline.Utility.GetFastqPreQCStats.vs1.sh DataProcessing.Pipeline.Utility.GetFastqQCStats.vs1.sh

bash DataProcessing.Pipeline.Utility.GetFastqQCStats.vs1.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output

#Notes: original sff location of /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PreMerge/mapping_pool_1-46/POOL20-12RL/. has 4 MIDs with good quality -- maybe the MID shouldn't be 12? Nah, there are 7 MIDs for POOL20 so it's unlikely to just be a MID12 thin
#Redo sff of /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PreMerge/mapping_pool_69-77/POOL75-09RL/ HX05RSL01.RL9.sff and HX05RSL02.RL9.sff

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output

cp -p DataProcessing.Pipeline.CheckForPresenceOfCertainFile.vs2.sh DataProcessing.Pipeline.CheckForPresenceOfCertainFile.vs3.sh
 
mv DataProcessing.Pipeline.CheckForPresenceOfCertainFile.vs3.sh DataProcessing.Pipeline.Utility.CheckForPresenceOfCertainFile.vs3.sh

bash DataProcessing.Pipeline.Utility.CheckForPresenceOfCertainFile.vs3.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output #searching for $baseFile1.QCed.preGATK.QCed.bam

mv DataOrganization.Utility.CreatebamMergeList.vs1.sh DataOrganization.Utility.CreatebamMergeList.vs1.sh

cp -p DataOrganization.Utility.CreatebamMergeList.vs1.sh DataOrganization.Utility.CreatebamMergeList.vs2.sh

bash DataOrganization.Utility.CreatebamMergeList.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output #Double-checking that this file was made appropriately and that all lines of the original file were used to create the eventual output. Just crossing t's and dotting i's here
#Note do the dropped individual thing to the output file from this run when I get back -- can delete this line later when I've done what I've asked in it, e.g. edit the output file to drop the individuals like was done earlier in this list/history of code

bash DataProcessing.Pipeline.Utility.RemoveaCertainFile.vs2.sh DataOrganization.CreatebamMergeList.vs1.20130928_145300.output 

python DataProcessing.Pipeline.DropIndividualsBasedOnPopGenomFile.vs2.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20130930_POP_genomics_sample_information_2.edited.noMeta.dropIndvs.csv --file2 DataOrganization.CreatebamMergeList.vs1.20131114_193315.output > DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output

bash DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output

bash DataOrganization.Utility.pt3.CheckOrigDirectoriesForFilesMissed.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output 
#Checking to see, given the directories specified in DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output and grepping all the sff files from them, are those sff files present in the single line being used to merge that particular individual's total bam file. E.g. seeing if files present in the directories are properly in the 'CreatebamMergeList' line representing that/those directories for that individual

bash DataOrganization.Utility.pt3.CheckMergedBamsForContainingProperFiles.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output
#Checking to see whether the merged bam files contain all the files we expect it to given the source line of files to combine from DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output

cp -p DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.vs1.sh DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.vs2.sh

bash DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output

bash DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs1.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output

cat 

bash DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output

cat DataOrganization.[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSumit/MM5.skatO.nonsyn.qVals.ACs.Orig.txt | perl -lane 'if (($F[3] != $F[4]) || ($F[3] != $F[5]) || ($F[4] != $F[5])) { print join("\t", @F); } '     
[  michaelt@node1  ~]$CreatebamMergeList.vs1.20131114_193315.dropIndvs.output | perl -lane 'my @vals = [  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSumit/MM5.skatO.nonsyn.qVals.ACs.Orig.txt | perl -lane 'if (($F[3] != $F[4]) || ($F[3] != $F[5]) || ($F[4] != $F[5])) { print join("\t", @F); } '     
[  michaelt@node1  ~]$split(/;/, $F[0]); if ($vals[0] !~ m/(POOL[a-zA-Z0-9]+-\d\dRL)/) { print join("\t", @F) ; } ' > DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output.Missing
#Realized had POOL[a-zA-Z0-9]+_\d\dRL statement in DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.vs2.sh instead of POOL[a-zA-Z0-9]+.*\d\dRL so stats file not properly formatted and needed to redo them

bash DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output.Missing

cat DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.output | perl -lane 'my @array1; push(@array1, $F[1]); foreach my $entry (@F) { if ($entry !~m/[a-zA-Z]/) { push (@array1, $entry); } } print join(",", @array1);' > DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.output.reformatted


#bash DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged DataOrganization.CreatebamMergeList.vs1.20131114_193315.dropIndvs.output

bash DataProcessing.Pipeline.pt6.All.pta.SplitBamIntoChromosomes.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.Samtools.pta.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.GATK.pta.RunReduceReadsPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.All.ptb.ConvertVCFtoPEDPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/Samtools AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools

bash DataProcessing.Pipeline.pt6.All.ptc.CollapsePedFilesIntoSingleFile.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/Samtools AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools


#Do reformatting of other statsoutput from earlier in the pipeline (the per-region/run stat file vs per-individual stat file that's already been done) and figure out how or what to graph visually via histograms and such, what's most important to look at
#Think through individual information file using to make sure it's all right and such


bash DataProcessing.Pipeline.pt6.GATK.ptb.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash ../../../../DataProcessing.Pipeline.pt6.GATK.ptc.MergeVCFsAcrossChromosomes.vs1.sh

bash DataProcessing.Pipeline.pt6.GATK.ptd.RecalAndApplyVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

vcftools --gzvcf AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz --plink --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP

plink --file AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP

#Decided on a change to how I incorporated individuals with multiple runs across Pools. Originally I was just keeping one pool and removing the other from the total dataset (demarcated using carrots, hence the 'noCarrot' version of files to preserve the original markings but remove them for downstream use), but have now decided to not manually choose which pool to use but to collect and use them all for every individual that has multiple pool runs. /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.Utility.CheckForDupEntriesPOPSampleInfoFile.vs1.py identifies individuals with multiple pools and creates a list of each pool for that individual. DataProcessing.Pipeline.Utility.CombineMultPoolsForSameIndvbamMergeList.vs1.sh then removes the individual lines for each pool from the main Createbam file, collects them into a single line and reattaches the new single-line-per-individual to the removed file (producing a list of the compeltely removed main file, a list of all the new lines, and the to-be-used list of removed main file + new list
#Going to rerun the 'new list' of individuals and then continue doing analysis with the removed maine file + new list, e.g. DataOrganization.CreatebamMergeList.vs1.20131114_193315.output.multPOOLperIndvCollected
#There may be one or two individuals being dropped in the mix that should not be so, but for the time being I will move forward with the analysis. I believe, after spot-checking most parts of the analysis and lines of code, that things are functioning more or less as they should be. When the rest of the individuals are added this will allow a second go-around to investigate if other individuals are being dropped unintentionally and to fully iron-out these portions of the code
#python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.Utility.CheckForDupEntriesPOPSampleInfoFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20131202_POP_genomics_sample_information_Solid_and_MM5.noMeta.noStar.csv

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.Utility.CheckForDupEntriesPOPSampleInfoFile.vs1.py --file1 20130930_POP_genomics_sample_information_2.edited.noMeta.noCarrot.csv | perl -F: -ane 'my @vals1 = split(/,/, $F[0]); my $id1 = $vals1[1]; print $id1, ":"; foreach my $entry1 (@F) { my @valsN = split(/,/, $entry1); my @MID = split(//, $valsN[2]); if ($#MID == "3") { print "0", $valsN[2], ","; } else { print $valsN[2], ","; } foreach my $entry2 (@valsN) { if ($entry2 =~ m/ool/) { print $entry2; } } print ":";} print "\n";' | sed "s/'//g" | sed 's/\s//g' | perl -F: -lane 'print join(":", @F[1..$#F]);' > 20130930_POP_genomics_sample_information_2.edited.noMeta.noCarrot.DupListPrepforMergeListUse.txt
#Manually delete first line of this resulting file since it somewhat not needed and adding in an arbitrary piece of code to just 'delete the first line of the output' seems too hard-coded to just put in there. Would rather do it manually and observe the need for this each time this may have to be done

bash DataProcessing.Pipeline.Utility.CombineMultPoolsForSameIndvbamMergeList.vs1.sh /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20130930_POP_genomics_sample_information_2.edited.noMeta.noCarrot.DupListPrepforMergeListUse.txt DataOrganization.CreatebamMergeList.vs1.20131114_193315.output #Creates DataOrganization.CreatebamMergeList.vs1.20131114_193315.output.multPOOLperIndvCollected* files 

cat 20130930_POP_genomics_sample_information_2.edited.noMeta.noCarrot.csv | grep -v \* > 20130930_POP_genomics_sample_information_2.edited.noMeta.noCarrot.dropIndvs.csv

python DataProcessing.Pipeline.DropIndividualsBasedOnPopGenomFile.vs2.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20130930_POP_genomics_sample_information_2.edited.noMeta.noCarrot.dropIndvs.csv --file2 DataOrganization.CreatebamMergeList.vs1.20131114_193315.output.multPOOLperIndvCollected > DataOrganization.CreatebamMergeList.vs1.20131114_193315.multPOOLperIndvCollected.dropIndvs.output 

bash DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.output.multPOOLperIndvCollected.newEntries

bash DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.output.multPOOLperIndvCollected.newEntries

#bash DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged DataOrganization.CreatebamMergeList.vs1.20131114_193315.multPOOLperIndvCollected.dropIndvs.output

bash DataProcessing.Pipeline.pt6.All.pta.SplitBamIntoChromosomes.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.Samtools.pta.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.GATK.pta.RunReduceReadsPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.GATK.ptb.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash ../../../../DataProcessing.Pipeline.pt6.GATK.ptc.MergeVCFsAcrossChromosomes.vs1.sh

bash DataProcessing.Pipeline.pt6.GATK.ptd.RecalAndApplyVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

gzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.vcf

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz | grep PASS > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.temp1

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz | grep \# > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.header

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.header AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.temp1 | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz

rm AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.temp1

vcftools --gzvcf AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz --geno .9 --recode --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.geno90.vcf

#NOTE (20131211) -- so realized when I merged multiple POOL#s into one file I kept the POOL#s the same in the .bam file's sample ID tag, so the variant calling partitioned the merged .bam file into separate 'individuals' again. So updated DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.vs3.sh to now change the SM:... ID in the merged .bam file to only contain one POOL# to account for the file only supposed to be representing a single individual now. As a result, redoing everything from the pt3 merge step onwards again

bash DataProcessing.Pipeline.Utility.RemoveaCertainFile.vs2.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.multPOOLperIndvCollected.dropIndvs.output 

bash DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.multPOOLperIndvCollected.dropIndvs.output

bash DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.multPOOLperIndvCollected.dropIndvs.output

#bash DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged DataOrganization.CreatebamMergeList.vs1.20131114_193315.multPOOLperIndvCollected.dropIndvs.output

bash DataProcessing.Pipeline.pt6.All.pta.SplitBamIntoChromosomes.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.Samtools.pta.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.GATK.pta.RunReduceReadsPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash DataProcessing.Pipeline.pt6.GATK.ptb.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash ../../../../DataProcessing.Pipeline.pt6.GATK.ptc.MergeVCFsAcrossChromosomes.vs1.sh

bash DataProcessing.Pipeline.pt6.GATK.ptd.RecalAndApplyVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

gzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.vcf

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz | grep PASS > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.temp1

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz | grep \# > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.header

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.header AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.temp1 | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz

rm AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.temp1

vcftools --gzvcf AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz --geno .9 --recode --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.geno90.vcf

vcftools --gzvcf AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz --geno .85 --recode --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.geno85.vcf

vcftools --gzvcf AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz --geno .95 --recode --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.geno95.vcf

vcftools --gzvcf AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz --missing --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss | awk '{ print $6 }' | perl -lane 'printf "%.2f\n", $F[0];' | sort -g -k 1,1 | uniq -c
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss | awk '{ print $6 }' | perl -lane 'printf "%.2f\n", $F[0];' | sort -g -k 1,1 | uniq -c | perl -lane '$cumulativeSum += $F[0]; push(@F, $cumulativeSum/2292557); print join("\t", @F);'

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/OID41305_HG19_Match5_probes/OID41305_HG19_Match5_probe_coverage.bed | perl -lane 'if ($F[0] =~ m/chr([a-zA-Z0-9]*).*/) { $F[0] = $1; print join("\t", @F); } ' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/OID41305_HG19_Match5_probes/OID41305_HG19_Match5_probe_coverage.formatted.bed

samtools view  ../../mapping_pool_106-124/POOL106_01RL/POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.bam | perl -lane 'my @vals1 = split(//, $F[9]); print $F[2], "\t", $F[3], "\t", $F[3] + scalar(@vals1), "\t", scalar(@vals1);' > ../../mapping_pool_106-124/POOL106_01RL/POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.OverlapPlot.formatted

python DataProcessing.Pipeline.Utility.AddColorInfotoVCFtoolslmissFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.0kbWindow.plotReady

#R --vanilla --slave --args < ../../../../DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs1.R /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/OID41305_HG19_Match5_probes/OID41305_HG19_Match5_probe_coverage.formatted.bed /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_106-124/POOL106_01RL/POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.OverlapPlot.formatted /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs1

R --vanilla --slave --args < ../../../../DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs2.R /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_106-124/POOL106_01RL/POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.OverlapPlot.formatted /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.0kbWindow.plotReady /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs2

python ../../../../DataProcessing.Pipeline.Utility.AddColorInfotoVCFtoolslmissFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss --window 500 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.500bWindow.plotReady

python ../../../../DataProcessing.Pipeline.Utility.AddColorInfotoVCFtoolslmissFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss --window 1000 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.1kbWindow.plotReady

python ../../../../DataProcessing.Pipeline.Utility.AddColorInfotoVCFtoolslmissFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss --window 5000 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.5kbWindow.plotReady

python ../../../../DataProcessing.Pipeline.Utility.AddColorInfotoVCFtoolslmissFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss --window 10000 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.10kbWindow.plotReady

python ../../../../DataProcessing.Pipeline.Utility.AddColorInfotoVCFtoolslmissFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss --window 25000 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.25kbWindow.plotReady

python ../../../../DataProcessing.Pipeline.Utility.AddColorInfotoVCFtoolslmissFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss --window 50000 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.50kbWindow.plotReady

R --vanilla --slave --args < ../../../../DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs2.R /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_106-124/POOL106_01RL/POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.OverlapPlot.formatted /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.500bWindow.plotReady /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs2.500bWindow

R --vanilla --slave --args < ../../../../DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs2.R /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_106-124/POOL106_01RL/POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.OverlapPlot.formatted /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.1kbWindow.plotReady /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs2.1kbWindow

R --vanilla --slave --args < ../../../../DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs3.R /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_106-124/POOL106_01RL/POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.OverlapPlot.formatted /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.1kbWindow.plotReady /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs3.1kbWindow

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz | awk '{ print $1, "\t", $2, "\t", $2, "\t", $4, "\t", $5 }' | grep -v ^# > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat

perl /home/michaelt/Software/annovar/summarize_annovar.pl /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat /home/michaelt/Software/annovar/ --ver1000g 1000g2012apr --verdbsnp 129 --veresp 6500 --buildver hg19 --remove

mv DataProcessing.Pipeline.DropOffTargetReads.vs1.py DataProcessing.Pipeline.Utility.DropOff.TargetReads.vs1.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.DropOff.VCFVariants.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz --window 1000 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.DropOffTargetVariants.1kbWindow.vcf.gz

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.DropOffTargetVariants.1kbWindow.vcf.gz --geno .85 --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.DropOffTargetVariants.1kbWindow.85geno

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.DropOffTargetVariants.1kbWindow.85geno.recode.vcf &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.DropOffTargetVariants.1kbWindow.vcf.gz --geno .95 --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.DropOffTargetVariants.1kbWindow.95geno

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.DropOffTargetVariants.1kbWindow.95geno.recode.vcf &


#In directory /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles
#Using phenotype file descriptions in DataProcessing.Pipeline.CreatePhenotypeFile.vs1.py to help determine which column + values to use
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($3 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.WhiteOnly.justSeroNeg.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($3 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.WhiteOnly.justSeroPos.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($4 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.WhiteOnly.justSeroHighlyExposedNeg.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.WhiteOnly.justRapidVRapid.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.WhiteOnly.justSlowVSlow.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($6 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.WhiteOnly.justVeryRapid.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($6 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.WhiteOnly.justVerySlow.FID

#In directory /home/michaelt/Data/1000G/AncestralAlignments online sources say these reference files based on Gch37 which is the equivalent of hg19 so should be same reference bases between these files and my own 
wget ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/pilot_data/technical/reference/ancestral_alignments/human_ancestor*
wget http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/analysis_results/supporting/ancestral_alignments/human_ancestor_GRCh37_e59.tar.bz2

#Trying to get AA information for each variant called in the dataset. Following code was copied/pasted from http://seqanswers.com/forums/showthread.php?t=15026 "1000 genomes ancestral alleles?"
ls human_ancestor_*.fa.bz2 | while read IN; do
OUT=`echo $IN | sed 's,bz2$,gz,'`
CHR=`echo $IN | sed 's,human_ancestor_,, ; s,.fa.bz2,,'`
bzcat $IN | sed "s,^>.*,>$CHR," | gzip -c > $OUT
samtools faidx $OUT
done
#

#Test line
samtools faidx human_ancestor_1.fa.gz 1:1000000-1000020 #Expect to see tgggcacagcctcacccagga

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.gz | fill-aa -a /home/michaelt/Data/1000G/AncestralAlignments/human_ancestor_GRCh37_e59/human_ancestor_ | gzip -c > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz
#EDIT NOTE 03162014 -- when redoing this later just drop out SNPs that don't have an AA too at this point (might as well since they're not being used in our analysis, but probably double check how many are being dropped by this, probably not a lot, e.g. under 1,000?)

vcftools --gzvcf AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --plink --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped | awk '{ print $1, "\t", $2 } ' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDIIDs

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDIIDs | awk '{ print $1 } ' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.DropOff.VCFVariants.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --window 1000 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.vcf.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.DropOff.VCFVariants.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --window 0 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.0kbWindow.vcf.gz

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.vcf.gz --geno .95 --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.FID --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg.FID --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos.FID --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg.FID --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid.FID --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow.FID --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid.FID --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow.FID --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow &

#
#20140101 NOTE -- These *.pheno and *.covar files contain FID and IID information for /every/ POOL_* individual, even if the main .vcf/.ped/.whatever file does not contain duplicate entries of the same FID/IID (e.g. multiple POOL_* were combined for a single FID/IID, as should be). So 'total line counts' for some of these *.pheno and *.covar files may be 'off' but in fact this is fine since the main .vcf/.ped/.whatever file is properly delimited. This is just to allow /all/ entries to have information so if the leading POOL_* identifier for a given FID/IID group changes for whatever reason, these information files can still be used since each POOL_* in any particular FID/IID group should be annotated
#

python DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20130930_POP_genomics_sample_information_2.edited.noMeta.noCarrot.dropIndvs.csv --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Sample_status_list_9-16-13_CT.edited.noMeta.csv > DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140101_POP_genomics_sample.status_list_9-16-13.output

python DataProcessing.Pipeline.CreatePhenotypeFile.vs1.py --file1 DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140101_POP_genomics_sample.status_list_9-16-13.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno

python DataProcessing.Pipeline.CreateCovariateFile.vs1.py --file1 DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140101_POP_genomics_sample.status_list_9-16-13.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar | awk '{ if ($5 == "1") { print $1, "\t", $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDIIDs

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDIIDs | awk '{ print $1 } ' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs

#Reran creating PLINK pheno files that contain only certain sero+/- and slow/rapid individuals

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($3 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($4 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($6 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($6 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow.FID

#So order is: justSeroHighlyExposedNeg 
paste AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow.frq | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.AllPhenos.frq.gz

gzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow.frq 

#gunzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.frq.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg.frq.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg.frq.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid.frq.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos.frq.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow.frq.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid.frq.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow.frq.gz

#paste <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz | tail -n +125 | perl -lane 'print join("\t", @F[0..8]);') <(zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.AllPhenos.frq.gz | tail -n +2) | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.wFreqsRplcGenos.noHeader.vcf.gz

###cat <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz | head -n 124) <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.wFreqsRplcGenos.noHeader.vcf.gz) | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.wFreqsRplcGenos.vcf.gz
#cat <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz | awk '{ if (NR < 125) { print $0 } }' ) <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.wFreqsRplcGenos.noHeader.vcf.gz) | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.wFreqsRplcGenos.asddadadad.vcf.gz
#rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.wFreqsRplcGenos.noHeader.vcf.gz
#rm AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.*

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | grep exonic | grep -v ncRNA | head -n 10 | perl -F, -lane 'print join("\t", @F[$#F-6..$#F]);' | perl -lane 'print join("\t", @F[$#F-4..$#F]);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AnnovarTest/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.TestGroup.vs1

perl /home/michaelt/Software/annovar/summarize_annovar.pl /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AnnovarTest/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.TestGroup.vs1 /home/michaelt/Software/annovar/ --ver1000g 1000g2012apr --verdbsnp 129 --veresp 6500 --buildver hg19 --remove

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.TestGroup.vs1 | perl -lane '($F[3], $F[4]) = ($F[4], $F[3]); print join("\t", @F);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.TestGroup.vs2

perl /home/michaelt/Software/annovar/summarize_annovar.pl /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AnnovarTest/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.TestGroup.vs2 /home/michaelt/Software/annovar/ --ver1000g 1000g2012apr --verdbsnp 129 --veresp 6500 --buildver hg19 --remove

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.AllPhenos.frq.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 25; $i <= 31; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.txt.gz

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CalcMeandAF.vs1.R AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.txt.gz

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/DataProcessing.Pipeline.Utility.CalcMeandAF.vs1.GeneSpecificAnalyses.txt | grep PP2D | grep -v PP2DP | sort -g -k 9,9

Timepoint check: 20140106

bash DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs1.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output

cat DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs1.output | perl -lane 'my @array1; push(@array1, $F[1]); foreach my $entry (@F) { if ($entry !~ m/[a-zA-Z]/) { push (@array1, $entry); } } print join(",", @array1);' | awk 'BEGIN{ print "ID,TotalReadsPreQC,TotalReadsQ10,TotalReadsQ10Prcnt,TotalReadsF4,TotalReadsF4Prcnt,TotalReadsF256,TotalReadsF256Prcnt,TotalReadsPostQC,TotalReadsDropped,TotalReadsDroppedPrcnt,AverageReadLengthPreQC,AverageReadLengthPostQC,BedFileCoveragePostQC" } { print $0 } ' > DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs1.output.reformatted 

bash DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.sh DataOrganization.CreatebamMergeList.vs1.20131114_193315.multPOOLperIndvCollected.dropIndvs.output

#Total column numbers post formatting = 27
cat DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.output | perl -lane 'my @array1; push(@array1, $F[1]); foreach my $entry (@F) { if ($entry !~ m/[a-zA-Z]/) { push (@array1, $entry); } } print join(",", @array1);' | awk 'BEGIN{ print "ID,NumberOfOriginalBamFiles,TotalReadsPreQC,TotalReadsPreQCForward,TotalReadsPreQCReverse,TotalReadsPreQCForward_Reverse,PreQCMAPQmean,PreQCMAPQsd,PreQCMean,PreQCSD,PreQCMax,PreQCMin,CoveragePreQC,TotalReadsPostRmdup,TotalReadsPostRmdupDropped,TotalReadsPostRmdupDroppedPercentage,TotalReadsPostQC,TotalReadsPostQCForward,TotalReadsPostQCReverse,TotalReadsPostQCForward_Reverse,PostQCMAPQmean,PostQCMAPQsd,PostQCMean,PostQCSD,PostQCMax,PostQCMin,CoveragePostQC" } { print $0 } ' > DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.output.reformatted

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs1.output.reformatted /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.output.reformatted

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --plink --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode

plink --file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.phenoColEdit.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.phenoColEdit.pedind

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.bim /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.pedsnp

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.phenoColEdit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.phenoColEdit.pedind /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.pedsnp mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/Eigenstrat/.  

cp -p AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.250k.parfile AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.parfile

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/smartpca -p AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.parfile

scp mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.Results.pca /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/.

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20140101_POP_genomics_sample.status_list_9-16-13.output --file2 AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.Results.pca > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.Results.pca.altrd.race

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.Results.pca.altrd.race mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/Eigenstrat/.

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.Results.pca.altrd.race.plot.1vs2.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:American_Indian_or_Alaskan_Native:Black_non-Hispanic:Other:White_Hispanic -x -o AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.Results.pca.altrd.race.plot.3vs4.xtxt

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.Results.pca.altrd.race | grep White_non-Hispanic | perl lane 'my @vals1 = split(/:/, $F[0]); $F[0] = $vals1[0]; print join("\t", @F);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.Results.pca.altrd.race.FIDs 

#Directory /home/michaelt/Data/HapMap/HapMap3
wget ftp://ftp.ncbi.nlm.nih.gov/hapmap/genotypes/2009-01_phaseIII/plink_format/*
wget ftp://ftp.ncbi.nlm.nih.gov/hapmap/genotypes/2009-01_phaseIII/draft2_samples_QC+.txt

#Direcotry /home/michaelt/Software/liftOver
wget http://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64/liftOver
wget http://hgdownload.cse.ucsc.edu/goldenPath/hg18/liftOver/hg18ToHg19.over.chain.gz

cat hapmap3_r2_b36_fwd.consensus.qc.poly.map | perl -lane '$val1++; print "chr", $F[0], "\t", $F[3], "\t", $F[3]+1, "\t", $val1;' > hapmap3_r2_b36_fwd.consensus.qc.poly.map.liftOverFormat

./../../Software/liftOver/liftOver hapmap3_r2_b36_fwd.consensus.qc.poly.map.liftOverFormat ../../Software/liftOver/hg18ToHg19.over.chain.gz hapmap3_r2_b36_fwd.consensus.qc.poly.map.liftOverFormat.hg19 hapmap3_r2_b36_fwd.consensus.qc.poly.map.liftOverFormat.dropped

paste hapmap3_r2_b36_fwd.consensus.qc.poly.map hapmap3_r2_b36_fwd.consensus.qc.poly.map.liftOverFormat > hapmap3_r2_b36_fwd.consensus.qc.poly.map.liftOverFormat.ForPostProcessing

python /home/michaelt/Data/HapMap/HapMap3/VLookUp.Python.vs1.py --file1 /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.map.liftOverFormat.hg19 --file2 /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.map.liftOverFormat.ForPostProcessing > /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.liftOver.hg18tohg19.map

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/VLookUp.Python.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.map --file2 /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.liftOver.hg18tohg19.map | awk '{ print $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.rsIDs

cat /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.txt | grep -E 'CEU|CHB|YRI' | awk '{ if ($8 =="x") { print $0 } } ' > /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.CEUCHBYRI.Unrelated.txt

cat /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.CEUCHBYRI.Unrelated.txt | awk '{ print $1, "\t", $2 }' > /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.CEUCHBYRI.Unrelated.FIDIIDs

plink --file /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly --make-bed --out /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly

plink --bfile /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.rsIDs --keep /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.CEUCHBYRI.Unrelated.FIDIIDs --recode --out /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData

cat /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.map | awk '{ print $2 }' > /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.map.rsIDs 

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --extract /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.map.rsIDs --recode --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3

plink --file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3 --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3

#There are a few SNPs that do not match up between the two files. Removing them (about ~30?) and recreating the overlap .ped files once more
paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.bim /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.bim | awk '{ if (($5 == $11) || ($5 == $12) ) { print $0 } } ' | awk '{ print $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.FinalListwDrops.rsIDs

plink --bfile /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.FinalListwDrops.rsIDs --keep /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.CEUCHBYRI.Unrelated.FIDIIDs --recode --out /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.vs2

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.FinalListwDrops.rsIDs --recode --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.vs2

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.vs2.ped /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.vs2.ped > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.ped

plink --ped AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.ped --map AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.vs2.map --make-bed --out AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.phenoColEdit.ped

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.phenoColEdit.pedind

ln -s AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.bim AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.pedsnp

scp AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.phenoColEdit.ped AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.phenoColEdit.pedind AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.pedsnp mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/Eigenstrat/.

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/smartpca -p AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.parfile

scp mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/.

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20140101_POP_genomics_sample.status_list_9-16-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt1

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.HapMap3vs.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.HapMap3vs.py --file1 /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.txt --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt2

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race

sed -i 's/American_Indian_or_Alaskan_Native/AIndian_or_Alaskan_Native/g' AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/Eigenstrat/.

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI -x -o AllPools.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.1vs2.xtxt

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race | awk '{ if ((($2 >= 0) && ($2 <= .2)) && (($3 >= -.01) && ($3 <= .03))) { print $0 } } ' | grep White_non-Hispanic > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI -x -o AllPools.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.3vs4.xtxt

ln -s AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed AllPools.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i  AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed -c 1:2 -p White_non-Hispanic -x -o  AllPools.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.xtxt

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed | perl -lane 'my @vals1 = split(/:/, $F[0]); print $vals1[0];' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs

scp mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/.

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($3 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($4 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($3 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($6 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid.FID
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($6 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow.FID

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs1.output.reformatted /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.output.reformatted

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Results/Graphics/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs1.*.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/Results/Graphics/.

bash DataProcessing.Pipeline.Utility.GetperfastQpostQCStats.vs1.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output 
#Deleted 4 lines with nothing in them
sed -i 's/(//g' /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperfastQpostQCStats.vs1.output
sed -i 's/)//g' /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperfastQpostQCStats.vs1.output



R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperfastQpostQCStats.vs1.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs1.output.reformatted /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.output.reformatted

python DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.5PCs.covar
#Edit code in between runs
python DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | awk '{ print $1, "\t", $2 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.FIDIIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.FIDIIDs | awk '{ print $1 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.FIDs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 1,2,3,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.5PCs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.5PCs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.FIDIIDs --maf .01 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.5PCs.maf01

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 1,2,3,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.10PCs

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.5PCs.assoc.logistic Pheno1_Logistic_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.5PCs.assoc.logsitic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.10PCs.assoc.logistic Pheno1_Logistic_10PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.10PCs.assoc.logsitic.QQplot.jpeg &

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.frq.gz | perl -lane 'my @vals1 = split(/:/, $F[4]); my @vals2 = split(/:/, $F[5]); push(@F, sprintf("%.0f", $vals1[1]*$F[3])); push(@F, sprintf("%.0f", $vals2[1]*$F[3])); print join("\t", @F); ' | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.frq.wAlleleCounts.gz

#Edited VLookUp.Python.vs1.py
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/VLookUp.Python.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.5PCs.assoc.logistic --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.frq.wAlleleCounts.gz | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.5PCs.assoc.logistic.wAC.gz

R --vanilla --slave --args  < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.ComparePValvsAC.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.5PCs.assoc.logistic.wAC.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.ComparePValvsAC.vs1.plot.jpeg

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.PlotSFS.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/Graphics/DataProcessing.Pipeline.Utility.PlotSFS.vs1.plot.jpeg

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.txt.gz | grep exonic | awk -F, '{ print $19 } ' | sort | uniq > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.txt.GeneIDList

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.txt.gz | grep exonic | awk -F, '{ print $19 } ' | sort | uniq > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.txt.GeneIDList.Exonic

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.txt.gz | grep exonic | grep -v RNA | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.txt.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs2.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.txt.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs2.ExonicVariants.All.output

#Edited code
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs2.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.txt.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs2.ExonicVariants.Nonsynonymous.output

#Edited code
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs2.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.txt.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs2.ExonicVariants.PP2_D.output

#Edited code
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs2.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.txt.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs2.ExonicVariants.PP2_DP.output

R --vanilla --slave --args < DataProcessing.Pipeline.Analysis.VariantDensity.FollowUpCalculations.vs1.R DataProcessing.Pipeline.Analysis.VariantDensity.vs2.ExonicVariants.PP2_D.output | awk '{ if (($4 != "NA") && ($2 >= 4)) { print $0 } } ' | sort -rg -k 3,3
R --vanilla --slave --args < DataProcessing.Pipeline.Analysis.VariantDensity.FollowUpCalculations.vs1.R DataProcessing.Pipeline.Analysis.VariantDensity.vs2.ExonicVariants.PP2_DP.output | awk '{ if (($4 != "NA") && ($2 >= 4)) { print $0 } } ' | sort -rg -k 3,3

#MAPK12 specifically looking at here

B 22,50691870_1,rs138582408,C,T,C,1154,PASS,AC=6;AF=3.995e-03;AN=1502;BaseQRankSum=7.966;DB;DP=13733;Dels=0.00;FS=2.273;HaplotypeScore=3.8969;InbreedingCoeff=-0.0070;MLEAC=5;MLEAF=3.329e-03;MQ=59.99;MQ0=0;MQRankSum=-0.801;QD=12.54;ReadPosRankSum=-1.530;VQSLOD=0.233;culprit=HaplotypeScore;AA=C,0.00451807,0.00319489,0.00310559,0.00569801,0.02,0.00892857,0,0,6,2,1,4,2,1,0,0,exonic,MAPK12,nonsynonymous SNV,,0.41,0.811214,N,0.22,T,0.0,B,0.921332,N,0.539884,D,0.925,22,50691870,50691870,C,T
D 22,50693697_1,rs201016559,T,C,t,221.31,PASS,AC=2;AF=1.332e-03;AN=1502;BaseQRankSum=-4.072;DB;DP=16220;Dels=0.00;FS=0.000;HaplotypeScore=1.5329;InbreedingCoeff=-0.0031;MLEAC=1;MLEAF=6.658e-04;MQ=59.99;MQ0=0;MQRankSum=0.761;QD=8.20;ReadPosRankSum=0.915;VQSLOD=0.809;culprit=FS;AA=t,0.000753012,0,0,0.0014245,0,0,0,0,1,0,0,1,0,0,0,0,exonic,MAPK12,nonsynonymous SNV,,0.03,0.99724,C,0.86,T,0.885,D,0.997985,U,0.841807,D,4.43,22,50693697,50693697,T,C
P 22,50694078_1,rs141322209,G,A,G,97.36,PASS,AC=3;AF=2.003e-03;AN=1498;BaseQRankSum=5.053;DB;DP=14056;Dels=0.00;FS=8.309;HaplotypeScore=2.3858;InbreedingCoeff=-0.0041;MLEAC=1;MLEAF=6.676e-04;MQ=59.99;MQ0=0;MQRankSum=-0.100;QD=3.14;ReadPosRankSum=-1.239;VQSLOD=0.164;culprit=QD;AA=G,0.00226586,0.00159744,0,0.00286533,0.01,0,0,0,3,1,0,2,1,0,0,0,exonic,MAPK12,nonsynonymous SNV,,0.02,0.999418,C,0.92,T,0.17,P,0.997696,U,0.241217,N,4.38,22,50694078,50694078,G,A
D 22,50694084_1,rs2066776,G,A,G,690.67,PASS,AC=3;AF=2.003e-03;AN=1498;BaseQRankSum=6.300;DB;DP=14065;Dels=0.00;FS=0.000;HaplotypeScore=1.9649;InbreedingCoeff=-0.0018;MLEAC=3;MLEAF=2.003e-03;MQ=59.99;MQ0=0;MQRankSum=0.104;QD=14.39;ReadPosRankSum=-0.306;VQSLOD=1.54;culprit=FS;AA=G,0.00226586,0,0,0.00429799,0,0.0181818,0,0.0178571,3,0,0,3,0,2,0,1,exonic,MAPK12,nonsynonymous SNV,rs2066776,0,0.999259,C,1.0,D,0.996,D,0.998205,U,0.999993,D,4.2,22,50694084,50694084,G,A
B 22,50694251_1,rs200047423,T,C,T,456.62,PASS,AC=4;AF=2.670e-03;AN=1498;BaseQRankSum=-7.928;DB;DP=14144;Dels=0.00;FS=2.627;HaplotypeScore=1.6631;InbreedingCoeff=-0.0051;MLEAC=2;MLEAF=1.335e-03;MQ=59.99;MQ0=0;MQRankSum=-0.851;QD=6.92;ReadPosRankSum=-1.241;VQSLOD=0.931;culprit=QD;AA=T,0.00226586,0.00159744,0,0.00286533,0,0,0,0,3,1,0,2,0,0,0,0,exonic,MAPK12,nonsynonymous SNV,,0,0.997358,C,1.0,D,0.087,B,0.998359,U,0.998098,D,4.49,22,50694251,50694251,T,C
D 22,50694280_1,.,G,A,G,480.31,PASS,AC=1;AF=6.676e-04;AN=1498;BaseQRankSum=5.983;DP=14170;Dels=0.00;FS=6.285;HaplotypeScore=1.2801;InbreedingCoeff=-0.0033;MLEAC=1;MLEAF=6.676e-04;MQ=59.99;MQ0=0;MQRankSum=0.507;QD=12.64;ReadPosRankSum=-1.984;VQSLOD=0.763;culprit=HaplotypeScore;AA=G,0.000755287,0,0,0.00143266,0,0,0,0,1,0,0,1,0,0,0,0,exonic,MAPK12,nonsynonymous SNV,,0,0.999458,C,1.0,D,1.0,D,0.997712,U,0.999999,D,4.44,22,50694280,50694280,G,A
D 22,50694529_1,rs2230823,G,A,G,213.68,PASS,AC=3;AF=1.997e-03;AN=1502;BaseQRankSum=5.839;DB;DP=14376;Dels=0.00;FS=9.486;HaplotypeScore=1.0269;InbreedingCoeff=-0.0038;MLEAC=1;MLEAF=6.658e-04;MQ=59.99;MQ0=0;MQRankSum=0.313;POSITIVE_TRAIN_SITE;QD=6.28;ReadPosRankSum=2.196;VQSLOD=1.17;culprit=QD;AA=G,0.00150602,0,0,0.002849,0,0,0,0,2,0,0,2,0,0,0,0,exonic,MAPK12,nonsynonymous SNV,rs2230823,0,0.986757,C,1.0,D,0.9,D,0.6867,U,0.075359,N,3.17,22,50694529,50694529,G,A
B 22,50696678_1,rs34422484,G,A,G,7444.34,PASS,AC=57;AF=0.038;AN=1488;BaseQRankSum=19.808;DB;DP=7340;Dels=0.00;FS=4.423;HaplotypeScore=0.9100;InbreedingCoeff=0.0409;MLEAC=53;MLEAF=0.036;MQ=60.00;MQ0=0;MQRankSum=-0.703;POSITIVE_TRAIN_SITE;QD=12.53;ReadPosRankSum=1.973;VQSLOD=2.50;culprit=DP;AA=G,0.0333839,0.0273312,0.03125,0.0387931,0.0306122,0.0535714,0.0454545,0.0344828,44,17,10,27,3,6,2,2,exonic,MAPK12,nonsynonymous SNV,rs34422484,0.04,0.982381,C,0.94,T,0.0,B,0.945872,N,0.506249,D,2.11,22,50696678,50696678,G,A


zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50691870 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50693697 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'
346-0/1:6,10:16:99:271,0,158    398-0/1:10,1:11:1:1,0,294
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50694078 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'
301-0/1:9,1:10:4:4,0,238        316-0/1:10,1:11:2:2,0,278       379-0/1:5,5:10:99:147,0,116
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50694084 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'
220-0/1:2,12:15:20:373,0,20     234-0/1:12,9:20:99:241,0,282    435-0/1:6,6:12:99:176,0,150
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50694251 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'
428-0/1:5,1:6:4:4,0,156 493-0/1:10,2:12:32:32,0,302     495-0/1:9,1:10:4:4,0,280        571-0/1:18,20:37:99:499,0,506
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50694280 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'
569-0/1:18,20:37:99:530,0,453
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50694529 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'
440-0/1:4,1:5:21:21,0,105       492-0/1:8,1:9:10:10,0,223       554-0/1:11,9:19:99:262,0,261
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50696678 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'
33-0/1:1,2:3:26:64,0,26 54-0/1:10,1:11:2:2,0,281        75-0/1:10,1:11:3:3,0,293        82-0/1:4,3:7:79:79,0,115        94-0/1:10,1:11:2:2,0,301        100-0/1:7,8:15:99:241,0,188     105-0/1:10,21:30:99:665,0,220   117-0/1:8,3:11:74:74,0,241      188-0/1:3,1:4:20:20,0,91        216-0/1:1,1:2:27:29,0,27        220-0/1:3,6:9:66:183,0,66  234-0/1:5,7:12:99:210,0,137     235-0/1:9,8:17:99:227,0,235     260-0/1:1,3:4:23:93,0,23        265-0/1:8,10:18:99:294,0,208    304-1/1:0,1:1:3:35,3,0  313-0/1:1,4:5:17:124,0,17       318-0/1:3,1:4:23:23,0,82        323-0/1:2,4:6:51:117,0,51       349-0/1:4,4:8:99:120,0,101      354-0/1:3,3:6:74:88,0,74   361-0/1:10,5:15:99:127,0,263    388-0/1:3,2:5:58:58,0,79        393-1/1:0,5:5:15:181,15,0       399-1/1:0,11:11:33:390,33,0     414-0/1:6,3:9:80:80,0,148       417-0/1:3,4:7:73:122,0,73       423-0/1:8,5:13:99:132,0,203     430-0/1:5,2:7:49:49,0,147       435-0/1:4,3:7:87:87,0,106       448-0/1:4,6:10:99:183,0,100        451-1/1:0,2:2:6:67,6,0  460-0/1:2,3:5:49:87,0,49        514-0/1:1,2:3:24:61,0,24        515-0/1:4,1:5:19:19,0,113       544-0/1:5,5:10:99:139,0,127     554-0/1:3,5:8:70:153,0,70       568-0/1:5,23:27:50:713,0,50     570-0/1:9,8:17:99:223,0,255     589-0/1:7,5:12:99:131,0,189     590-0/1:7,4:11:99:106,0,190        592-0/1:14,13:26:99:358,0,345   594-0/1:21,20:40:99:570,0,518   599-0/1:3,6:9:64:174,0,64       616-0/1:7,6:13:99:171,0,168     622-0/1:8,3:11:64:64,0,215      631-0/1:7,5:12:99:141,0,175     632-0/1:15,6:20:99:113,0,410    633-0/1:10,10:20:99:286,0,247   662-0/1:3,2:5:47:47,0,69        687-0/1:1,3:4:20:92,0,20   691-0/1:6,2:8:48:48,0,175       738-0/1:22,15:36:99:422,0,551


R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs3.R /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_106-124/POOL106_01RL/POOL106_01RL.preGATK.samplesMerged.rmdup.BQSR.calmd.OverlapPlot.formatted /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.lmiss.0kbWindow.plotReady /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/DataProcessing.Pipeline.Utility.PlotArrayReads_SeqReads.wVariants.vs3.0kbWindow

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.FIDIIDs --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.pheno
#Expecting 619 individuals based on plink .fam output and FIDs file -- get 619 in '...justWhite.pheno' output file

cp -p VLookUp.Python.vs1.py VLookUp.Python.vs1.vs2.py 

#Later did mv VLookUp.Python.vs1.vs2.py DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs1.py 

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { print $8, "\t", $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.SetIDFile.Exonic

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $8, "\t", $2 } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($11 == "P") || ($11 == "D") ) { print $8, "\t", $2 } } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_PD

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $8, "\t", $2 } } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_D

#Hardcoding SKAT analysis script because the tools require 'file location' as input so it seems like I'll need multiple file sets to run the code on multiple SetID sets for example

R --vanilla --slave --args < DataProcessing.Pipeline.Analysis.Skat.vs1.R

#EPACTS
#Using same 'file creation setup' instead of doing these commands within the Python script to help maintain consistency with how I'm partitioning variants into exonic, nonsynonymous, Polyphen2 categories, etc...these partitions are based on the commands I use in my CalcMeandAF.vs1.R scripts 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $0 } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($11 == "P") || ($11 == "D") ) { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "23") { $1 = "X"} print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip

/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip

/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.pheno | awk '{ print $1, "\t", $2, "\t0\t0\t0\t", $3, "\t", $4, "\t", $5, "\t", $6 } ' | sed 's/-9/NA/g' | perl -lane 'print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped

#Add line #FAM_ID IND_ID FAT_ID MOT_ID SEX HIVPROG HIVPROGHE AIDS AIDSEXTR to /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.pheno

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test b.collapse \
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.EPACTS.collapse

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test b.collapse \
--max-maf .9 \
-min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD.EPACTS.collapse

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test b.madsen \
--max-maf .9 \
-min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.EPACTS.madsen

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test b.madsen \
--max-maf .9 \
-min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.EPACTS.madsen

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test b.madsen \
--max-maf .9 \
-min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD.EPACTS.madsen

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test b.madsen \
--max-maf .9 \
-min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D.EPACTS.madsen

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test b.wcnt \
--max-maf .9 \
--min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.EPACTS.wcnt

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test b.wcnt \
--max-maf .9 \
-min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD.EPACTS.wcnt


epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test skat --skat-o --skat-adjust \
--max-maf .9 \
--min-callrate .01 \ 
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.EPACTS.Skat0

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test skat --skat-o --skat-adjust \
--max-maf .9 \
-min-maf .000000000000001 \ 
--min-callrate .01 \ 
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.EPACTS.Skat0

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test skat --skat-o --skat-adjust \
--max-maf .9 \
-min-maf .000000000000001 \ 
--min-callrate .01 \ 
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROG \
--test skat --skat-o --skat-adjust \
--max-maf .9 \
-min-maf .000000000000001 \ 
--min-callrate .01 \ 
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROGHE \
--test b.madsen \
--max-maf .9 \
-min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Pheno2.Exonic.Nonsynonymous.PP2_PD.EPACTS.madsen &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROGHE \
--test b.madsen \
--max-maf .9 \
-min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Pheno2.Exonic.Nonsynonymous.PP2_D.EPACTS.madsen &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno AIDS \
--test b.madsen \
--max-maf .9 \
-min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Pheno3.Exonic.Nonsynonymous.PP2_PD.EPACTS.madsen &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno AIDS \
--test b.madsen \
--max-maf .9 \
-min-maf .000000000000001 \
--min-callrate .01 \
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Pheno3.Exonic.Nonsynonymous.PP2_D.EPACTS.madsen &


epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROGHE \
--test skat --skat-o --skat-adjust \
--max-maf .9 \
-min-maf .000000000000001 \ 
--min-callrate .01 \ 
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Pheno2.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno HIVPROGHE \
--test skat --skat-o --skat-adjust \
--max-maf .9 \
-min-maf .000000000000001 \ 
--min-callrate .01 \ 
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Pheno2.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno AIDS \
--test skat --skat-o --skat-adjust \
--max-maf .9 \
-min-maf .000000000000001 \ 
--min-callrate .01 \ 
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Pheno3.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \
--groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D \
--ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.EPACTSedit.ped \
--pheno AIDS \
--test skat --skat-o --skat-adjust \
--max-maf .9 \
-min-maf .000000000000001 \ 
--min-callrate .01 \ 
--run 2 \
--out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.bim.wGeneIDs.GroupFile.Pheno3.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0 &

#KIAA1429
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "8") { print $0 } } ' | grep 95501065 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.txt.gz | grep KIAA1429 | awk -F, '{ if (($28 == "D") || ($28 == "P")) { print $2, "\t", $28 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SpecificGenes/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.Exonic.KIAA1429.PP2_PD.Locations

bash DataProcessing.Pipeline.Analysis.IdentifyIndividualsContainingVariants.vs1.sh 8 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SpecificGenes/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.Exonic.KIAA1429.PP2_PD.Locations

95501065(D): 31-0/1:9,1:10:6:6,0,264    350-0/1:6,12:18:99:365,0,129    411-0/1:5,4:9:99:108,0,122      649-0/1:7,1:8:11:11,0,197
95508103(D): 102-0/1:11,7:18:99:175,0,337
95518785(D): 362-0/1:14,18:31:99:469,0,376
95522750(D): 715-0/1:11,12:22:99:296,0,326
95522824(D): 163-0/1:5,1:6:16:16,0,152  686-0/1:13,9:21:99:248,0,354
95530099(D): 107-0/1:5,1:6:17:17,0,160  192-0/1:6,1:7:3:3,0,178 243-0/1:13,13:26:99:377,0,379   
95531388(P): 721-0/1:3,4:7:69:110,0,69  
95531525(P): 473-0/1:10,8:18:99:221,0,273       705-0/1:9,1:10:4:4,0,278
95538947(P): 96-0/1:21,26:45:99:638,0,623       759-0/1:7,1:8:7:7,0,218 
95541400(P): 625-0/1:9,5:14:99:125,0,259        719-0/1:8,1:9:5:5,0,248 
95549374(P): 153-0/1:8,1:9:8:8,0,249    296-0/1:5,5:10:99:103,0,141     297-0/1:7,1:8:2:2,0,214 421-0/1:9,1:10:4:4,0,285
95550513(P): 92-0/1:8,5:13:99:131,0,242 296-0/1:4,1:5:9:9,0,95  683-0/1:6,1:7:13:13,0,175


vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.FID --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg.FID --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos.FID --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg.FID --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid.FID --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow.FID --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid.FID --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow.FID --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow &

paste AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow.frq.count | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.AllPhenos.frq.count.gz

gzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow.frq.count 

#gunzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.frq.count.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroNeg.frq.count.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroHighlyExposedNeg.frq.count.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justRapidVRapid.frq.count.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSeroPos.frq.count.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justSlowVSlow.frq.count.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVeryRapid.frq.count.gz AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.justVerySlow.frq.count.gz

paste AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq.count | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.AllPhenos.frq.AndCount.gz

gzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq.count

#gunzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq.count

#20140219
#(for AWK) 26 exonic/intronic, 27 gene, 28 synon/non-synon, 29 rsID from annovari/'dbsnp129' (vs from GATK variant caller at beginning of array), 30-40 prediction stuff (30 is AVSIFT, then predic prog # / predic prog Category for PhyloP, SIFT, Polyphen2, LRT, MT)
#Now including allele counts as well as allele frequencies
#Also now outputting multiple lines for the same variant if it covers multiple genes. This is being recorded by adding a '_1', '_2', etc... to te end of the basepair position at array entry 2 for each line to designate the # of times the variant is present in the dataset to represent different loci for that same variant

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.AllPhenos.frq.AndCount.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 29; $i <= 39; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.vs2.txt.gz

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | grep exonic | awk -F, '{ print $3 }' | sort | uniq > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.AnnotatingVariantsCoveringMultTranscripts
#The manually edited /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.AnnotatingVariantsCoveringMultTranscripts comparing the output to /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv to see variants that have multiple gene-hits (e.g. overlap multiple genes).
#Decided to output multiple lines for variants that cover multiple transcripts/genes, e.g. have 3 lines for the same variant if that variant hits 3 genes at once
#So stopped this train of thought here and didn't finish annotating the above filename


#Used vs1 of DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs1.20140101_POP_genomics_sample.status_list_9-16-13.output instead of vs2 earlier. Redoing these and downstream steps with the proper vs2 of this file
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140101_POP_genomics_sample.status_list_9-16-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt1

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race

sed -i 's/American_Indian_or_Alaskan_Native/AIndian_or_Alaskan_Native/g' AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/Eigenstrat/.

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI -x -o AllPools.AllPoolsMerged.ChrAll.GATK.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.1vs2.xtxt

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI -x -o AllPools.AllPoolsMerged.ChrAll.GATK.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.3vs4.xtxt

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race | awk '{ if ((($2 >= 0) && ($2 <= .2)) && (($3 >= -.01) && ($3 <= .03))) { print $0 } } ' | grep White_non-Hispanic > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed

ln -s AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed AllPools.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed

/mnt/lustre/home/mturchin20/Software/EigenSoft/bin/ploteig -i  AllPools.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed -c 1:2 -p White_non-Hispanic -x -o  AllPools.AllPoolsMerged.ChrAll.GATK.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.xtxt

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed | perl -lane 'my @vals1 = split(/:/, $F[0]); print $vals1[0];' > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs

scp mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/.

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar | awk '{ print $1, "\t", $2 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar.FIDIIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar.FIDIIDs | awk '{ print $1 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.QCed.FIDs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 1,2,3,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.5PCs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.5PCs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar.FIDIIDs --maf .01 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.5PCs.maf01

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 1,2,3,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.10PCs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 2 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar.FIDIIDs --maf .01 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.5PCs.maf01.pheno2

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 3 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar.FIDIIDs --maf .01 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.5PCs.maf01.pheno3

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 4 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar.FIDIIDs --maf .01 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.5PCs.maf01.pheno4

!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.5PCs.assoc.logistic Pheno1_WhiteOnly_Logistic_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.5PCs.assoc.logsitic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.5PCs.maf01.assoc.logistic Pheno1_WhiteOnly_Logistic_maf01_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.5PCs.maf01.assoc.logsitic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.10PCs.assoc.logistic Pheno1_WhiteOnly_Logistic_10PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/OutputFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.whiteOnly.QCed.10PCs.assoc.logsitic.QQplot.jpeg &

cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ print $1 }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.FIDs
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($3 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.FIDs
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($4 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.FIDs
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($3 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.FIDs
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.FIDs
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.FIDs
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($6 == "1") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.FIDs
cat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($6 == "2") { print $1 } }' | grep -f AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar.WhiteOnly.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.ped.FIDs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.FIDs

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow &

paste AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq.count | gzip > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.AllPhenos.frq.AndCount.gz

gzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq.count

#gunzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVeryRapid.frq.count AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justVerySlow.frq.count

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow &

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.frq.count 

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.AllPhenos.frq.AndCount.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 29; $i <= 39; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.vs2.txt.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.vs2.txt.gz | awk -F, '{ if ($26 =="exonic") { print $0 } }' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.vs2.txt.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs3.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.vs2.txt.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs3.ExonicVariants.All.output

#Edited code
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs3.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.vs2.txt.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs3.ExonicVariants.Nonsynonymous.output

#Edited code
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs3.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.vs2.txt.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs3.ExonicVariants.PP2_DP.output

#Edited code
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs3.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.vs2.txt.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs3.ExonicVariants.PP2_D.output

R --vanilla --slave --args < DataProcessing.Pipeline.Analysis.VariantDensity.FollowUpCalculations.vs1.R DataProcessing.Pipeline.Analysis.VariantDensity.vs3.ExonicVariants.PP2_DP.output | awk '{ if (($4 != "NA") && ($2 >= 3)) { print $0 } } ' | sort -rg -k 3,3
R --vanilla --slave --args < DataProcessing.Pipeline.Analysis.VariantDensity.FollowUpCalculations.vs1.R DataProcessing.Pipeline.Analysis.VariantDensity.vs3.ExonicVariants.PP2_D.output | awk '{ if (($4 != "NA") && ($2 >= 3)) { print $0 } } ' | sort -rg -k 3,3

#MAPK12 specifically looking at here
zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50691870 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50693697 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'
346-0/1:6,10:16:99:271,0,158    398-0/1:10,1:11:1:1,0,294

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar.FIDIIDs --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.vs2.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { print $8, "\t", $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.SetIDFile.Exonic

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $8, "\t", $2 } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($11 == "P") || ($11 == "D") ) { print $8, "\t", $2 } } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_PD

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $8, "\t", $2 } } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $0 } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($11 == "P") || ($11 == "D") ) { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.covar.justWhite.QCed.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.pheno

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.pheno | awk '{ print $1, "\t", $2, "\t0\t0\t0\t", $3, "\t", $4, "\t", $5, "\t", $6 } ' | sed 's/-9/NA/g' | perl -lane 'print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.covar | awk '{ print $4, "\t", $6, "\t", $7 "\t", $8, "\t", $9, "\t", $10, "\t", $11,  "\t", $12, "\t", $13, "\t", $14,  "\t", $15 }' | paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped.vs2

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped.vs2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped

#Add line #FAM_ID IND_ID FAT_ID MOT_ID SEX HIVPROG HIVPROGHE AIDS AIDSEXTR RMID PC1 PC2 PC3 PC4 PC5 PC6 PC7 PC8 PC9 PC10 to /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz \

!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!EPACTS Stuff

#EPACTS Variables Running Through
#Pheno: HIVPROG HIVPROGHE AIDS AIDSEXTR
#Filetypes: Exonic Exonic.Nonsynonymous Exonic.Nonsynonymous.PP2_PD Exonic.Nonsynonymous.PP2_D
#Test b.collapse b.madsen b.wcnt  skat --skat-o --skat-adjust

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --test skat --skat-o --skat-adjust --run 1 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.Exonic.EPACTS.Skat0 &

#epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --test skat --skat-o --skat-adjust --max-maf .9 --min-maf .000000000000001 --min-callrate .01 --run 1 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.Exonic.EPACTS.Skat0.vs2 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --test skat --skat-o --skat-adjust --min-callrate .01 --run 1 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.Exonic.EPACTS.Skat0.vs3 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --test skat --skat-o --skat-adjust --max-maf .9 --run 1 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.Exonic.EPACTS.Skat0.vs4 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --test skat --skat-o --skat-adjust --min-maf .000000000000001 --run 1 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.Exonic.EPACTS.Skat0.vs5 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --test skat --skat-o --skat-adjust --max-maf .5 --run 1 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.Exonic.EPACTS.Skat0.vs6 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --test skat --skat-o --skat-adjust --max-maf 1 --run 1 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.Exonic.EPACTS.Skat0.vs7 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --test skat --skat-o --skat-adjust --max-maf .05 --run 1 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.Exonic.EPACTS.Skat0.vs8 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --test b.collapse --run 1 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.Exonic.EPACTS.collapse &

#epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --test b.collapse --max-maf .9 --min-maf .000000000000001 --min-callrate .01 --run 1 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.Exonic.EPACTS.collapse.vs2 &


#Note -- not really running the code, opening up an R console and copying/pasting the commands since I'm moreso playing around with things and testing them out
R --vanilla --slave --args < DataProcessing.Pipeline.Analysis.Skat.vs3.R
34: 1 SNPs with either high missing rates or no-variation are excluded!
35: The missing genotype rate is 0.004442. Imputation is applied.
36: 5 SNPs with either high missing rates or no-variation are excluded!
37: The missing genotype rate is 0.006094. Imputation is applied.
38: 5 SNPs with either high missing rates or no-variation are excluded!
39: The missing genotype rate is 0.006094. Imputation is applied.
40: 1 SNPs with either high missing rates or no-variation are excluded!
41: The missing genotype rate is 0.002703. Imputation is applied.
42: 3 SNPs with either high missing rates or no-variation are excluded!
43: The missing genotype rate is 0.003392. Imputation is applied.
44: The missing genotype rate is 0.003941. Imputation is applied.
45: The missing genotype rate is 0.003003. Imputation is applied.

30: The missing genotype rate is 0.002336. Imputation is applied.
31: 4 SNPs with either high missing rates or no-variation are excluded!
32: The missing genotype rate is 0.002336. Imputation is applied.
33: The missing genotype rate is 0.001802. Imputation is applied.
34: The missing genotype rate is 0.002252. Imputation is applied.
35: The missing genotype rate is 0.000751. Imputation is applied.
36: 1 SNPs with either high missing rates or no-variation are excluded!
37: The missing genotype rate is 0.005005. Imputation is applied.
38: The missing genotype rate is 0.004880. Imputation is applied.
39: 1 SNPs with either high missing rates or no-variation are excluded!
40: The missing genotype rate is 0.019520. Imputation is applied.
41: The missing genotype rate is 0.004692. Imputation is applied.
42: The missing genotype rate is 0.005255. Imputation is applied.
43: 1 SNPs with either high missing rates or no-variation are excluded!
44: The missing genotype rate is 0.003003. Imputation is applied.
45: 1 SNPs with either high missing rates or no-variation are excluded!
46: The missing genotype rate is 0.001802. Imputation is applied.
47: The missing genotype rate is 0.003003. Imputation is applied.
48: ALL SNPs have either high missing rates or no-variation. P-value=1
49: The missing genotype rate is 0.003003. Imputation is applied.
50: Only one SNP in the SNP set!

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.EPACTS.Skat0 &
make -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.EPACTS.Skat0.Makefile -j 6
Note: Get odd error when trying to run skat-o w/ adjustment on the above combination of parameters, e.g. pheno4 + exonic set. Does not occur elsewhere. Can run skat-o fine manually use R code and SKAT directly, so doing that for this set of results. However, none of the results for Pheno4 are coming out well due to the low sample numbers so it's a moot point.  
Error in out[i, ] <- c(nona.collapse, maf.collapse, nrow(G), nrow(genos),  :
  number of items to replace is not a multiple of replacement length
  Calls: source -> withVisible -> eval -> eval
  In addition: There were 14 warnings (use warnings() to see them)
  Execution halted
  make: *** [/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.EPACTS.Skat0.0.epacts] Error 1
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0 &


epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Collapse &

EPACTS Group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Collapse &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Collapse &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Collapse &


epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Madsen &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Madsen &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Madsen &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Madsen &


epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.wcnt &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.wcnt &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.wcnt &




python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs4.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.vs2.txt.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs4.ExonicVariants.All.output

#Edited code
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs4.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.vs2.txt.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs4.ExonicVariants.Nonsynonymous.output

#Edited code
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs4.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.vs2.txt.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs4.ExonicVariants.PP2_DP.output

#Edited code
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs4.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_GeneInformation_wSymbols.vs2.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.Exonic.vs2.txt.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs4.ExonicVariants.PP2_D.output

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.FollowUpCalculations.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs4.ExonicVariants.PP2_DP.output | awk '{ if (($5 != "NA") && ($2 >= 3)) { print $0 } } ' | sort -rg -k 4,4
R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.FollowUpCalculations.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.VariantDensity.vs4.ExonicVariants.PP2_D.output | awk '{ if (($4 != "NA") && ($2 >= 3)) { print $0 } } ' | sort -rg -k 4,4

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.AlleleCountComparison.Simulations.Launcher.vs1.sh 1 100

PasteCommandjustSeroNeg=`perl -e 'print "paste - "; for (my $i = 1; $i <= 100; $i++) { print "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVs", $i, ".CtDwn.frq.count "; }'`
PasteCommandjustSeroPos=`perl -e 'print "paste - "; for (my $i = 1; $i <= 100; $i++) { print "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVs", $i, ".CtDwn.frq.count "; }'`
PasteCommandjustRapidVRapid=`perl -e 'print "paste - "; for (my $i = 1; $i <= 100; $i++) { print "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVs", $i, ".CtDwn.frq.count "; }'`
PasteCommandjustSlowVSlow=`perl -e 'print "paste - "; for (my $i = 1; $i <= 100; $i++) { print "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVs", $i, ".CtDwn.frq.count "; }'`

#zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.frq.count.gz | tail -n +2 | paste - /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVs*.CtDwn.frq.count | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVsAll.frq.count.gz
#zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.frq.count.gz | tail -n +2 | paste - /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVs*.CtDwn.frq.count | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVsAll.frq.count.gz
#zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.frq.count.gz | tail -n +2 | paste - /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVs*.CtDwn.frq.count | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.gz
#zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.frq.count.gz | tail -n +2 | paste - /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVs*.CtDwn.frq.count | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVsAll.frq.count.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.frq.count.gz | tail -n +2 | $PasteCommandjustSeroNeg | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVsAll.frq.count.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.frq.count.gz | tail -n +2 | $PasteCommandjustSeroPos | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVsAll.frq.count.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.frq.count.gz | tail -n +2 | $PasteCommandjustRapidVRapid | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.frq.count.gz | tail -n +2 | $PasteCommandjustSlowVSlow | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVsAll.frq.count.gz

#Tri-Allelic and Quad-Allelic SNPs. They are eventually dropped out when including annotation information in the original, main source files since the program doesn't accept non-bi-allelic SNP setups. Could potentially divide these SNPs into multiple entries for the same variant but only including each bi-allelic state amongst the original tri- or quad-alleles, but this is currently decided as being too much work for a supposedly very low return on information gained (387 SNPs << ~60k which is around the full On-Target, 95% geno list).
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK]$zcat SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.gz | perl -lane 'if ($#F == 206) { print join("\t", @F); } ' | wc
    387   80109  377798
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK]$zcat SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.gz | perl -lane 'if ($#F == 207) { print join("\t", @F); } ' | wc
      3     624    2798

[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK]$zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ print $4, "\t", $5 }' | sort | uniq -c | grep -E 'A|G|T|C' | sort -rg -k 1,1
    15295 C        T
    15107 G        A
    9604 T        C
    8993 A        G
    3308 G        C
    3105 C        G
    2628 G        T
    2572 C        A
    2164 T        G
    2069 A        C
    1635 A        T
    1577 T        A
    60 C        G,T
    59 G        A,C
    58 G        A,T
    56 C        A,T
    37 T        C,G
    26 T        A,C
    22 G        C,T
    20 A        G,T
    20 A        C,G
    12 C        A,G
    10 A        C,T
    7 T        A,G
    3 G        A,C,T
    2 phone_home=STANDARD      gatk_key=null
    1 each     ALT
    1 REF      ALT
    1 Containing       Spanning

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.ForSimulation.AlleleCountAnalysis.vs1.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.ForSimulation.AlleleCountAnalysis.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVsAll.frq.count.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 13; $i <= 24; $i++) { if (($F[$i] eq "NA") || ($F[$i] eq "")) { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.ForSimulation.AlleleCountAnalysis.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVsAll.frq.count.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 13; $i <= 24; $i++) { if (($F[$i] eq "NA") || ($F[$i] eq "")) { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.ForSimulation.AlleleCountAnalysis.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 13; $i <= 24; $i++) { if (($F[$i] eq "NA") || ($F[$i] eq "")) { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.ForSimulation.AlleleCountAnalysis.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVsAll.frq.count.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 13; $i <= 24; $i++) { if (($F[$i] eq "NA") || ($F[$i] eq "")) { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.AlleleCountComparison.AnalyzeResults.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Analysis/AlleleCounts/DataProcessing.Pipeline.Analysis.AlleleCountComparison.AnalyzeResults.vs1.output


bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.AlleleCountComparison.Simulations.Launcher.vs1.sh 101 1000
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.AlleleCountComparison.Simulations.Launcher.vs1.sh 850 1000

PasteCommandjustSeroNeg=`perl -e 'print "paste - "; for (my $i = 1; $i <= 1000; $i++) { print "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVs", $i, ".CtDwn.frq.count "; }'`
PasteCommandjustSeroPos=`perl -e 'print "paste - "; for (my $i = 1; $i <= 1000; $i++) { print "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVs", $i, ".CtDwn.frq.count "; }'`
PasteCommandjustRapidVRapid=`perl -e 'print "paste - "; for (my $i = 1; $i <= 1000; $i++) { print "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVs", $i, ".CtDwn.frq.count "; }'`
PasteCommandjustSlowVSlow=`perl -e 'print "paste - "; for (my $i = 1; $i <= 1000; $i++) { print "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVs", $i, ".CtDwn.frq.count "; }'`

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.frq.count.gz | tail -n +2 | $PasteCommandjustSeroNeg | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVsAll.frq.count.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.frq.count.gz | tail -n +2 | $PasteCommandjustSeroPos | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVsAll.frq.count.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.frq.count.gz | tail -n +2 | $PasteCommandjustRapidVRapid | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.frq.count.gz | tail -n +2 | $PasteCommandjustSlowVSlow | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVsAll.frq.count.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.ForSimulation.AlleleCountAnalysis.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVsAll.frq.count.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 13; $i <= 24; $i++) { if (($F[$i] eq "NA") || ($F[$i] eq "")) { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.ForSimulation.AlleleCountAnalysis.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVsAll.frq.count.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 13; $i <= 24; $i++) { if (($F[$i] eq "NA") || ($F[$i] eq "")) { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.ForSimulation.AlleleCountAnalysis.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 13; $i <= 24; $i++) { if (($F[$i] eq "NA") || ($F[$i] eq "")) { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.ForSimulation.AlleleCountAnalysis.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVsAll.frq.count.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 13; $i <= 24; $i++) { if (($F[$i] eq "NA") || ($F[$i] eq "")) { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Analysis.AlleleCountComparison.AnalyzeResults.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroNeg.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSeroPos.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justSlowVSlow.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SimulationFiles/AlleleCounts/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.justRapidVRapid.permuteVsAll.frq.count.VariantInfo.AnnovarAnnotation.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Analysis/AlleleCounts/DataProcessing.Pipeline.Analysis.AlleleCountComparison.AnalyzeResults.vs2.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Analysis/AlleleCounts/DataProcessing.Pipeline.Analysis.AlleleCountComparison.AnalyzeResults.vs2.output | awk '{ if (($13 != "NA") && ($13 != 1)) { print $0 } } ' | sort -rg -k 14,14 -k 13,13

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($10 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Collapse &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Madsen &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.wcnt &
#Two warning messages for above run of b.wcnt (Pheno3, LRT_D)
#Warning messages:
#1: glm.fit: fitted probabilities numerically 0 or 1 occurred
#2: glm.fit: fitted probabilities numerically 0 or 1 occurred
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.wcnt &


#cat <( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
cat  <( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.All.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.All.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.All.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.All.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/All.GATK.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.justWhite.QCed.Pheno1.RMID5PCs.All.EPACTS.All.Summary.txt

cat <( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.All.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.All.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.All.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.All.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/All.GATK.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.justWhite.QCed.Pheno2.RMID5PCs.All.EPACTS.All.Summary.txt

cat <( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.All.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.All.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.All.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.All.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/All.GATK.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.justWhite.QCed.Pheno3.RMID5PCs.All.EPACTS.All.Summary.txt

cat <( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.All.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.All.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.All.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.All.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
<( paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Collapse.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Madsen.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.wcnt.epacts | sort -rg -k 10,10 | grep -v NA | head -n -1 | tail -n 25) | awk 'BEGIN{ print ""} { print $0 } END{ print ""}' ) \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/All.GATK.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.justWhite.QCed.Pheno4.RMID5PCs.All.EPACTS.All.Summary.txt

#Accidentally ran in /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/. and converted some files that were not supposed to be. Think I fixed it
ls -lrt | awk '{ print $9 }' | perl -lane 'if ($F[0] =~ m/(.*)(\.wAA)(\.White.*)/) { my $val1 = $1 . $2 . $3; my $val2 = $1 . $2 . ".DropOffTargetVariants.1kbWindow.95geno.recode" . $3; system("mv $val1 $val2");}'
#Fix ls -lrt | grep WhiteOnly | grep -v WhiteOnly.QCed | awk '{ print $9 } ' | perl -lane 'if ($F[0] =~ m/(.*)(\.wAA)(.DropOffTargetVariants.1kbWindow.95geno.recode)(\.White.*)/) { my $val1 = $1 . $2 . $3 . $4; my $val2 = $1 . $2 . $4; system("mv $val1 $val2");}'

cp -p DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.py DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.FullVCFwAnnovar.vs1.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.AllGenesPerVariant.vcf.gz

#Edited code in /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.FullVCFwAnnovar.vs1.py
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs2.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.vcf.gz

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py DataPrcoessing.Pipeline.Utility.GetVCFQCStats.vs1.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.AllGenesPerVariant.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.AllGenesPerVariant.GetVCFQCStats.vs1.output

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.vcf.gz | grep exonic | grep -v ncRNA | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.Exonic.GetVCFQCStats.vs1.output

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py DataPrcoessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs1.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justSeroNeg 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroPos.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justSeroPos 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSeroHighlyExposedNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justSeroHighlyExposedNeg 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justRapidVRapid.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justRapidVRapid 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.justSlowVSlow.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justSlowVSlow 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
[1] "MeanSingletons: 20.6626826029216 MeanDoubletons: 25.2881806108898 MeanCommon: 5033.1567065073 MeanRare: 504.302788844621 MeanHeterozy: 3457.57503320053 MeanCoverage: 16.6629860920849 sdCoverage: 6.17039709917999 Exonic: 342.65604249668 Intronic: 4139.33997343958 Nonsynonymous: 127.176626826029 Synonymous: 212.610889774236 PP2_D: 9.80610889774236 PP2_P: 11.6693227091633 Sift_D: 22.0823373173971 LRT_D: 21.9189907038513"


cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justSeroNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
[1] "MeanSingletons: 16.1948881789137 MeanDoubletons: 21.1086261980831 MeanCommon: 5038.1661341853 MeanRare: 435.638977635783 MeanHeterozy: 3385.56869009585 MeanCoverage: 15.8726865928358 sdCoverage: 5.50509121070413 Exonic: 340.05750798722 Intronic: 4090.77635782748 Nonsynonymous: 126.159744408946 Synonymous: 211.268370607029 PP2_D: 9.77635782747604 PP2_P: 11.3897763578275 Sift_D: 22.0095846645367 LRT_D: 21.6932907348243"

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justSeroHighlyExposedNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
[1] "MeanSingletons: 16.3291925465839 MeanDoubletons: 21.1987577639752 MeanCommon: 5043.50310559006 MeanRare: 436.596273291925 MeanHeterozy: 3384.70186335404 MeanCoverage: 15.7976001524024 sdCoverage: 4.99886275938732 Exonic: 339.596273291925 Intronic: 4096.49689440994 Nonsynonymous: 125.944099378882 Synonymous: 210.795031055901 PP2_D: 9.64596273291925 PP2_P: 11.3416149068323 Sift_D: 21.9316770186335 LRT_D: 21.5465838509317"

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
[1] "MeanSingletons: 16.9517045454545 MeanDoubletons: 19.8977272727273 MeanCommon: 5046.28125 MeanRare: 426.855113636364 MeanHeterozy: 3405.65340909091 MeanCoverage: 17.0944802710762 sdCoverage: 6.42723639615973 Exonic: 337.542613636364 Intronic: 4091.76420454545 Nonsynonymous: 125.235795454545 Synonymous: 209.323863636364 PP2_D: 9.69886363636364 PP2_P: 11.4403409090909 Sift_D: 21.3238636363636 LRT_D: 21.9403409090909"

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justRapidVRapid | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
[1] "MeanSingletons: 16.6734693877551 MeanDoubletons: 17.3673469387755 MeanCommon: 5088.89795918367 MeanRare: 403.591836734694 MeanHeterozy: 3419.16326530612 MeanCoverage: 22.2127566842631 sdCoverage: 9.1245435877065 Exonic: 338.673469387755 Intronic: 4112.5306122449 Nonsynonymous: 125.061224489796 Synonymous: 210.69387755102 PP2_D: 9.12244897959184 PP2_P: 11.469387755102 Sift_D: 21.0408163265306 LRT_D: 21.3469387755102"

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perIndv.vs1.output.justSlowVSlow | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
[1] "MeanSingletons: 17.3571428571429 MeanDoubletons: 20.3571428571429 MeanCommon: 5025.71428571429 MeanRare: 426.107142857143 MeanHeterozy: 3386.69642857143 MeanCoverage: 16.4965406672807 sdCoverage: 5.79915172393244 Exonic: 333.803571428571 Intronic: 4087.39285714286 Nonsynonymous: 123.196428571429 Synonymous: 207.392857142857 PP2_D: 9.80357142857143 PP2_P: 11.375 Sift_D: 21.3571428571429 LRT_D: 21.8035714285714"

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perGene.vs1.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perGene.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.AllGenesPerVariant.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.perGene.vs1.output

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs3.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.AllPhenos.frq.AndCount.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 29; $i <= 39; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.vs3.txt.gz

#Including SNPs that don't have clear ancestral/derived alleles. That was the major change to the vs2 to vs3 of the CombineVCFwFreqsAnnovarInfo.vs#.py script
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.vs3.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { print $8, "\t", $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.SetIDFile.Exonic

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $8, "\t", $2 } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($11 == "P") || ($11 == "D") ) { print $8, "\t", $2 } } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_PD

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $8, "\t", $2 } } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $0 } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($11 == "P") || ($11 == "D") ) { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($10 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $0 } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($11 == "P") || ($11 == "D") ) { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($10 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.Sift_D
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.LRT_D

#TimeptChck: 20140319

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.Exonic.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.Exonic.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.Exonic.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID.Exonic.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.Exonic.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_D.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.Sift_D.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.LRT_D.EPACTS.Skat0 &


cat DataProcessing.Pipeline.Utility.GetFastqQCStats.vs1.output > DataProcessing.Pipeline.Utility.GetFastqQCStats.vs1.output.forR
#Removed empty lines with /0.00000.0000 0.0000 and dd in vi
cat DataProcessing.Pipeline.Utility.GetFastqQCStats.vs1.output.forR | R -q -e "Data1 <- read.table(file('stdin'), header=F); sum(as.numeric(Data1[,4])); sum(as.numeric(Data1[,6]));"
> Data1 <- read.table(file('stdin'), header=F); sum(as.numeric(Data1[,4])); sum(as.numeric(Data1[,6]));
[1] 156661507032
[1] 258365498
cat DataProcessing.Pipeline.Utility.GetFastqQCStats.vs1.output | wc
   2896   83858  645395

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.EPACTS.Skat0.epacts  | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); summary(qvalue(Data1[,10]));"

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.EPACTS.Skat0.epacts  | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); summary(qvalue(Data1[,10]));"


cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs3.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs4.py

#Removing '(NM_...' from 'splicing' variants so that their gene IDs can be included. Also split up gene IDs that had ;symbol;smbol' which only affectes really 3 variants that had 'exonic;splcing'
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs4.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.AllPhenos.frq.AndCount.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 29; $i <= 39; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.vs4.txt.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.FreqInfo.AnnovarAnnotation.vs4.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing")) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $0 } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($11 == "P") || ($11 == "D") ) { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($10 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing")) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.All.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.All.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.All.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.All.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.All.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.All.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.All.EPACTS.Collapse &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.collapse --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.All.EPACTS.Collapse &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.All.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.All.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.All.EPACTS.Madsen &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.madsen --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.All.EPACTS.Madsen &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.All.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.All.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.All.EPACTS.wcnt &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test b.wcnt --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno4.RMID5PCs.All.EPACTS.wcnt &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.All.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.All.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.All.EPACTS.Skat0 &


cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.All.EPACTS.Skat0.epacts  | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); summary(qvalue(Data1[,10]));"



plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.IBD

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | awk '{ print $1, "\t", $1 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.IBD

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --maf .05 --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.IBD

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --maf .10 --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf10.IBD

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --maf .10 --geno .01 --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf10.geno01.IBD

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.bim | awk '{ if ($1 == "23") { print $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.bim.justX

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --exclude /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.bim.justX --maf .10 --geno .01 --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.noX.maf10.geno01.IBD

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --maf .1 --indep-pairwise 50 5 0.1 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf10.r2_10

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf10.r2_10.prune.in --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf10.r2_10

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --hardy --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.FIDs --hardy --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.hwe | awk '{ if ($6 < .00001) { print $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.hwe.1eNeg5.rsIDs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --maf .05 --hwe .00001 --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.hwe1eNeg5.IBD

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --maf .05 --hwe .0001 --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.hwe1eNeg4.IBD

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --maf .05 --hwe .00001 --relatedness --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.maf05.hwe1Neg5

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --relatedness2 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.IBD.genome | grep -v POOL38_03 | grep -v POOL99_02 | grep -v POOL59_09 | grep -v POOL78_08 | sort -rg -k 8,8 | vi -

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.IBD.genome | grep -v POOL38_03 | grep -v POOL99_02 | grep -v POOL59_09 | grep -v POOL78_08 | sort -rg -k 8,8 | vi -

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --transpose --recode --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.tfam | grep -E 'POOL27_10|POOL63_08' | awk '{ print $1, "\t", $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.tfam.2Indvs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.tfam.2Indvs --transpose --recode --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.just2Indvs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.tfam.2Indvs --transpose --recode --maf .05 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.just2Indvs.maf05

#Based on cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.hwe1eNeg4.IBD.genome | grep -v POOL38_03 | grep -v POOL99_02 | grep -v POOL59_09 | grep -v POOL78_08 | sort -rg -k 9,9 | vi -
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ print $1, "\t", $2, "\t", $314, "\t", $492 }' | grep -v ^# | grep -v "\.\/\." | perl -lane 'my @vals1 = split(/:/, $F[2]); my @vals2 = split(/:/, $F[3]); my @vals1sub = split(/\//, $vals1[0]); my @vals2sub = split(/\//, $vals2[0]); my $flag1; if ($vals1[0] eq $vals2[0]) { $flag1 = 2; } elsif (($vals1sub[0] eq $vals2sub[0]) || ($vals1sub[0] eq $vals2sub[1]) || ($vals1sub[1] eq $vals2sub[0]) || ($vals1sub[1] eq $vals2sub[1])) { $flag1 = 1; } else { $flag1 = 0; } print join("\t", @F), "\t", $flag1;' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); xVals <- c(Data1[1,2]); chrm <- Data1[1,1]; adj <- 0; for (i in 2:nrow(Data1)) { if (Data1[i,1] != chrm) { chrm <- Data1[i,1]; adj <- adj + Data1[i-1,2]; }; xVals <- c(xVals, Data1[i,2] + adj); }; jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.POOL27_10RL_POOL63_08RL.IBSPlot.jpg\", width=5000, height=2000, res=300); plot(xVals, Data1[,5], main=\"Bad - POOL27_10 vs. POOL63_08\", type=\"l\"); dev.off();" &

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ print $1, "\t", $2, "\t", $496, "\t", $541 }' | grep -v ^# | grep -v "\.\/\." | perl -lane 'my @vals1 = split(/:/, $F[2]); my @vals2 = split(/:/, $F[3]); my @vals1sub = split(/\//, $vals1[0]); my @vals2sub = split(/\//, $vals2[0]); my $flag1; if ($vals1[0] eq $vals2[0]) { $flag1 = 2; } elsif (($vals1sub[0] eq $vals2sub[0]) || ($vals1sub[0] eq $vals2sub[1]) || ($vals1sub[1] eq $vals2sub[0]) || ($vals1sub[1] eq $vals2sub[1])) { $flag1 = 1; } else { $flag1 = 0; } print join("\t", @F), "\t", $flag1;' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); xVals <- c(Data1[1,2]); chrm <- Data1[1,1]; adj <- 0; for (i in 2:nrow(Data1)) { if (Data1[i,1] != chrm) { chrm <- Data1[i,1]; adj <- adj + Data1[i-1,2]; }; xVals <- c(xVals, Data1[i,2] + adj); }; jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.POOL64_05RL_POOL6S_12RL.IBSPlot.jpg\", width=5000, height=2000, res=300); plot(xVals, Data1[,5], main=\"Bad - POOL64_05 vs. POOL6S_12\", type=\"l\"); dev.off();" &

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ print $1, "\t", $2, "\t", $130, "\t", $483 }' | grep -v ^# | grep -v "\.\/\." | perl -lane 'my @vals1 = split(/:/, $F[2]); my @vals2 = split(/:/, $F[3]); my @vals1sub = split(/\//, $vals1[0]); my @vals2sub = split(/\//, $vals2[0]); my $flag1; if ($vals1[0] eq $vals2[0]) { $flag1 = 2; } elsif (($vals1sub[0] eq $vals2sub[0]) || ($vals1sub[0] eq $vals2sub[1]) || ($vals1sub[1] eq $vals2sub[0]) || ($vals1sub[1] eq $vals2sub[1])) { $flag1 = 1; } else { $flag1 = 0; } print join("\t", @F), "\t", $flag1;' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); xVals <- c(Data1[1,2]); chrm <- Data1[1,1]; adj <- 0; for (i in 2:nrow(Data1)) { if (Data1[i,1] != chrm) { chrm <- Data1[i,1]; adj <- adj + Data1[i-1,2]; }; xVals <- c(xVals, Data1[i,2] + adj); }; jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.POOL111_10RL_POOL62_04RL.IBSPlot.jpg\", width=5000, height=2000, res=300); plot(xVals, Data1[,5], main=\"Bad - POOL111_10 vs. POOL62_04\", type=\"l\"); dev.off();" &

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ print $1, "\t", $2, "\t", $181, "\t", $699 }' | grep -v ^# | grep -v "\.\/\." | perl -lane 'my @vals1 = split(/:/, $F[2]); my @vals2 = split(/:/, $F[3]); my @vals1sub = split(/\//, $vals1[0]); my @vals2sub = split(/\//, $vals2[0]); my $flag1; if ($vals1[0] eq $vals2[0]) { $flag1 = 2; } elsif (($vals1sub[0] eq $vals2sub[0]) || ($vals1sub[0] eq $vals2sub[1]) || ($vals1sub[1] eq $vals2sub[0]) || ($vals1sub[1] eq $vals2sub[1])) { $flag1 = 1; } else { $flag1 = 0; } print join("\t", @F), "\t", $flag1;' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); xVals <- c(Data1[1,2]); chrm <- Data1[1,1]; adj <- 0; for (i in 2:nrow(Data1)) { if (Data1[i,1] != chrm) { chrm <- Data1[i,1]; adj <- adj + Data1[i-1,2]; }; xVals <- c(xVals, Data1[i,2] + adj); }; jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.POOL118_06RL_POOL91_07RL.IBSPlot.jpg\", width=5000, height=2000, res=300); plot(xVals, Data1[,5], main=\"Bad - POOL118_06 vs. POOL91_07\", type=\"l\"); dev.off();" &

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ print $1, "\t", $2, "\t", $456, "\t", $524 }' | grep -v ^# | grep -v "\.\/\." | perl -lane 'my @vals1 = split(/:/, $F[2]); my @vals2 = split(/:/, $F[3]); my @vals1sub = split(/\//, $vals1[0]); my @vals2sub = split(/\//, $vals2[0]); my $flag1; if ($vals1[0] eq $vals2[0]) { $flag1 = 2; } elsif (($vals1sub[0] eq $vals2sub[0]) || ($vals1sub[0] eq $vals2sub[1]) || ($vals1sub[1] eq $vals2sub[0]) || ($vals1sub[1] eq $vals2sub[1])) { $flag1 = 1; } else { $flag1 = 0; } print join("\t", @F), "\t", $flag1;' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); xVals <- c(Data1[1,2]); chrm <- Data1[1,1]; adj <- 0; for (i in 2:nrow(Data1)) { if (Data1[i,1] != chrm) { chrm <- Data1[i,1]; adj <- adj + Data1[i-1,2]; }; xVals <- c(xVals, Data1[i,2] + adj); }; jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.POOL59_05RL_POOL68_086L.IBSPlot.jpg\", width=5000, height=2000, res=300); plot(xVals, Data1[,5], main=\"Bad - POOL59_05 vs. POOL68_08\", type=\"l\"); dev.off();" &

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ print $1, "\t", $2, "\t", $259, "\t", $308 }' | grep -v ^# | grep -v "\.\/\." | perl -lane 'my @vals1 = split(/:/, $F[2]); my @vals2 = split(/:/, $F[3]); my @vals1sub = split(/\//, $vals1[0]); my @vals2sub = split(/\//, $vals2[0]); my $flag1; if ($vals1[0] eq $vals2[0]) { $flag1 = 2; } elsif (($vals1sub[0] eq $vals2sub[0]) || ($vals1sub[0] eq $vals2sub[1]) || ($vals1sub[1] eq $vals2sub[0]) || ($vals1sub[1] eq $vals2sub[1])) { $flag1 = 1; } else { $flag1 = 0; } print join("\t", @F), "\t", $flag1;' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); xVals <- c(Data1[1,2]); chrm <- Data1[1,1]; adj <- 0; for (i in 2:nrow(Data1)) { if (Data1[i,1] != chrm) { chrm <- Data1[i,1]; adj <- adj + Data1[i-1,2]; }; xVals <- c(xVals, Data1[i,2] + adj); }; jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.POOL17S_09RL_POOL26_01RL.IBSPlot.jpg\", width=5000, height=2000, res=300); plot(xVals, Data1[,5], main=\"Good - POOL17S_09 vs. POOL26_01\", type=\"l\"); dev.off();" &

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ print $1, "\t", $2, "\t", $229, "\t", $496 }' | grep -v ^# | grep -v "\.\/\." | perl -lane 'my @vals1 = split(/:/, $F[2]); my @vals2 = split(/:/, $F[3]); my @vals1sub = split(/\//, $vals1[0]); my @vals2sub = split(/\//, $vals2[0]); my $flag1; if ($vals1[0] eq $vals2[0]) { $flag1 = 2; } elsif (($vals1sub[0] eq $vals2sub[0]) || ($vals1sub[0] eq $vals2sub[1]) || ($vals1sub[1] eq $vals2sub[0]) || ($vals1sub[1] eq $vals2sub[1])) { $flag1 = 1; } else { $flag1 = 0; } print join("\t", @F), "\t", $flag1;' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); xVals <- c(Data1[1,2]); chrm <- Data1[1,1]; adj <- 0; for (i in 2:nrow(Data1)) { if (Data1[i,1] != chrm) { chrm <- Data1[i,1]; adj <- adj + Data1[i-1,2]; }; xVals <- c(xVals, Data1[i,2] + adj); }; jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.POOL13S_04RL_POOL64_05RL.IBSPlot.jpg\", width=5000, height=2000, res=300); plot(xVals, Data1[,5], main=\"Good - POOL13S_04 vs. POOL64_05\", type=\"l\"); dev.off();" &

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ print $1, "\t", $2, "\t", $181, "\t", $708 }' | grep -v ^# | grep -v "\.\/\." | perl -lane 'my @vals1 = split(/:/, $F[2]); my @vals2 = split(/:/, $F[3]); my @vals1sub = split(/\//, $vals1[0]); my @vals2sub = split(/\//, $vals2[0]); my $flag1; if ($vals1[0] eq $vals2[0]) { $flag1 = 2; } elsif (($vals1sub[0] eq $vals2sub[0]) || ($vals1sub[0] eq $vals2sub[1]) || ($vals1sub[1] eq $vals2sub[0]) || ($vals1sub[1] eq $vals2sub[1])) { $flag1 = 1; } else { $flag1 = 0; } print join("\t", @F), "\t", $flag1;' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); xVals <- c(Data1[1,2]); chrm <- Data1[1,1]; adj <- 0; for (i in 2:nrow(Data1)) { if (Data1[i,1] != chrm) { chrm <- Data1[i,1]; adj <- adj + Data1[i-1,2]; }; xVals <- c(xVals, Data1[i,2] + adj); }; jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.POOL118_06RL_POOL92_10RL.IBSPlot.jpg\", width=5000, height=2000, res=300); plot(xVals, Data1[,5], main=\"Good - POOL18_06 vs. POOL92_10\", type=\"l\"); dev.off();" &


zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ print $1, "\t", $2, "\t", $314, "\t", $492 }' | grep -v ^# | grep -v "\.\/\." | perl -lane 'my @vals1 = split(/:/, $F[2]); my @vals2 = split(/:/, $F[3]); my @vals1sub = split(/\//, $vals1[0]); my @vals2sub = split(/\//, $vals2[0]); my $flag1; if ($vals1[0] eq $vals2[0]) { $flag1 = "BLUE"; } elsif (($vals1sub[0] eq $vals2sub[0]) || ($vals1sub[0] eq $vals2sub[1]) || ($vals1sub[1] eq $vals2sub[0]) || ($vals1sub[1] eq $vals2sub[1])) { $flag1 = "BLUE"; } else { $flag1 = "RED"; } if ($F[0] eq "X") { $F[0] = "23"; } print join("\t", @F), "\t", $flag1;' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); yAxis <- c(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, \"X\"); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.POOL27_10RL_POOL63_08RL.IBSPlot.vs2.jpg\", width=5000, height=2500, res=300); Data1.red <- Data1[Data1[,5]==\"RED\",]; plot(Data1[,2], Data1[,1], main=\"Bad - POOL27_10 vs. POOL63_08\", yaxt=\"n\", col=as.character(Data1[,5])); points(Data1.red[,2], Data1.red[,1], col=Data1.red[,5]); axis(2, at=sort(unique(Data1[,1])), labels=yAxis); dev.off();" &







###### Unsure where this all belongs or what i was doing with these steps before
#vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --maf .05 --recode --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.maf05
#
#gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.maf05.recode.vcf
#
#zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.maf05.recode.vcf.gz | awk '{ print $314, "\t", $492 }' | tail -n +124 | perl -lane
#
#plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed
######

#Outputs in directory run in, so go to directory /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/
/home/michaelt/Software/admixture_linux-1.23/admixture /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.bed --cv 2 | tee /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.Admixture.2.log &
/home/michaelt/Software/admixture_linux-1.23/admixture /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.bed --cv 3 | tee /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.Admixture.3.log &
/home/michaelt/Software/admixture_linux-1.23/admixture /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.bed --cv 4 | tee /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.Admixture.4.log &
/home/michaelt/Software/admixture_linux-1.23/admixture /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.bed --cv 5 | tee /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.Admixture.5.log &
/home/michaelt/Software/admixture_linux-1.23/admixture /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.bed --cv 6 | tee /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.Admixture.6.log &


#Manually created this file and added the entries, based on the heterzygous-issue individuals seen in above statement (the grep -v individuals) and the individuals in the .genome file with rather high Z2 values (~80%+)
vi /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.20140324.IBDDropIndvs.FIDs

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz --remove /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.20140324.IBDDropIndvs.FIDs --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.gz | awk '{ if ($1 == "23") { $1 = "X"} print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip

/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.ped | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.20140324.IBDDropIndvs.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.All.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.All.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.All.DropIBDProbIndvs.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.DropIBDProbIndvs.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.DropIBDProbIndvs.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.DropIBDProbIndvs.EPACTS.Skat0 &

#Complex
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.All.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.All.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.All.DropIBDProbIndvs.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.Exonic.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.Exonic.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.Exonic.DropIBDProbIndvs.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.DropIBDProbIndvs.EPACTS.Skat0 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno1.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno2.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.DropIBDProbIndvs.EPACTS.Skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.DropIBDProbIndvs.recode.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.EPACTSedit.DropIBDProbIndvs.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DropIBDProbIndvs/AllPools.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.justWhite.QCed.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Pheno3.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.DropIBDProbIndvs.EPACTS.Skat0 &


[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.pheno | grep -E 'POOL103_03|POOL105_03'
POOL103_03RL    POOL103_03RL    2       2       -9      -9
POOL105_03RL    POOL105_03RL    2       2       -9      -9
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.pheno | grep -E 'POOL47_10|POOL87_11'
POOL87_11RL     POOL87_11RL     1       -9      -9      -9
POOL47_10RL     POOL47_10RL     2       2       -9      -9
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.pheno | grep -E 'POOL07_01|POOL22_01'
POOL07_01RL     POOL07_01RL     1       -9      -9      -9
POOL22_01RL     POOL22_01RL     1       -9      -9      -9
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.pheno | grep -E 'POOL36_11|POOL50_12'
POOL50_12RL     POOL50_12RL     2       2       1       -9
POOL36_11RL     POOL36_11RL     2       2       1       -9
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.pheno | grep -E 'POOL112_10|POOL116_10'
POOL112_10RL    POOL112_10RL    1       1       -9      -9
POOL116_10RL    POOL116_10RL    2       2       -9      -9


zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.vcf.gz | awk '{ if ($1 == "22") { print $0 } } ' | grep 50693697 | perl -ane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "-", $F[$i], "\t"; } } print "\n";'


cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.hwe1eNeg4.IBD.genome | perl -ane 'if ($F[8] > 0) { print $F[0], "\n", $F[2], "\n"; } ' | sort | uniq -c | sort -g -k 1,1 | awk '{ if ($1 >= 100) { print $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.hwe1eNeg4.IBD.genome.AppearMoreThan100Times.FIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.hwe1eNeg4.IBD.genome | perl -ane 'if ($F[8] > 0) { print $F[0], "\n", $F[2], "\n"; } ' | sort | uniq -c | sort -g -k 1,1 | awk '{ if ($1 >= 150) { print $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.hwe1eNeg4.IBD.genome.AppearMoreThan150Times.FIDs



#Samtools
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.Samtools.ptb.MergeVCFsAcrossChromosomes.vs1.sh

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.vcf

zcat AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.vcf.gz | head -n 119 > AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.vcf.header

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.vcf.gz | grep -v INDEL | perl -lane 'if (($F[3] =~ m/^[AGTC]$/) && ($F[4] =~ m/^[AGTC]$/)) { print join("\t", @F); }' | cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.vcf.header - |  gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.vcf.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.vcf.gz | perl -lane 'if ($F[5] >= 50) { print join("\t", @F); } ' | cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.vcf.header - | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.vcf.gz 

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.DropOff.VCFVariants.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.vcf.gz --window 1000 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.vcf.gz

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.vcf.gz --geno .95 --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.geno95

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.vcf.gz --geno .99 --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.geno99

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.geno95.recode.vcf

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.geno95.recode.vcf.gz --plink --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.geno95.recode

plink --file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.geno95.recode --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.geno95.recode

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.geno95.recode --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --maf .01 --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.geno95.recode.WhiteOnly.QCed

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.SNP.Q50.DropOffTarget_1kb.geno95.recode.WhiteOnly.QCed.genome | grep -v POOL116_06RL | grep -v POOL116_08RL | grep -v POOL104_06RL | grep -v POOL122_11RL | sort -rg -k 9,9 | vi - 



#Redoing tranches -- doing ts90 resulted in almost no SNPs being present after the 'DropOffTarg_1kb' step, so stepped up to ts_99
/home/shared/software/java/jre1.7.0_09/bin/java -Xmx5g -jar /home/shared/software/GATK/2.5.2/GenomeAnalysisTK.jar -T ApplyRecalibration -R /home/michaelt/Data/HumanGenome/GRCh37/Version8/human_g1k_v37.fasta -input /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.vcf -recalFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.recal -tranchesFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.tranches -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts90.vcf --ts_filter_level 90 -mode SNP

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts90.vcf

#Takes forever (with zlib 1.2.3), almost an hour
#vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts90.vcf.gz --remove-filtered-all --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts90.PASS

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts90.vcf.gz | head -n 125 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts90.vcf.header

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts90.vcf.gz | grep -v '^#' | grep -E 'PASS|VQSRTrancheSNP90.00to99.00' | cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts90.vcf.header - | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.vcf.gz 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.vcf.gz | fill-aa -a /home/michaelt/Data/1000G/AncestralAlignments/human_ancestor_GRCh37_e59/human_ancestor_ | gzip -c > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.vcf.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.DropOff.VCFVariants.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.vcf.gz --window 1000 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.DropOffTarg_1kb.vcf.gz

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.DropOffTarg_1kb.vcf.gz --geno .95 --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.DropOffTarg_1kb.95geno

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.DropOffTarg_1kb.95geno.recode.vcf

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.DropOffTarg_1kb.95geno.recode.vcf.gz --plink --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.DropOffTarg_1kb.95geno

plink --file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.DropOffTarg_1kb.95geno --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.DropOffTarg_1kb.95geno

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.DropOffTarg_1kb.95geno --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --maf .05 --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.ts99.PASS.wAA.DropOffTarg_1kb.95geno.maf05


#Doing IBD with variants covering arrays of 1000G individuals to see if get similar results, so doing this on the 4_CS cluster
scp /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/.

vi /KG/1000_Genomes/Samples/VCFs/HIVProj/DataProcessing.Pipeline.Utility.Create1000GOverlapWithArrayVCFs.sh

mv /KG/1000_Genomes/Samples/VCFs/HIVProj/DataProcessing.Pipeline.Utility.Create1000GOverlapWithArrayVCFs.sh /KG/1000_Genomes/Samples/VCFs/HIVProj/DataProcessing.Pipeline.Utility.Create1000GOverlapWithArrayVCFs.vs1.sh

cp -p /KG/1000_Genomes/Samples/VCFs/HIVProj/DataProcessing.Pipeline.Utility.Create1000GOverlapWithArrayVCFs.vs1.sh /KG/1000_Genomes/Samples/VCFs/HIVProj/DataProcessing.Pipeline.Utility.Create1000GOverlapWithArrayVCFs.Launcher.vs1.sh

bash /KG/1000_Genomes/Samples/VCFs/HIVProj/DataProcessing.Pipeline.Utility.Create1000GOverlapWithArrayVCFs.Launcher.vs1.sh

vcf-concat /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr1.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr2.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr3.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr4.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr5.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr6.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr7.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr8.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr9.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr10.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr11.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr12.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr13.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr14.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr15.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr16.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr17.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr18.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr19.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr20.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr21.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chr22.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrX.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz | gzip > /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrALL.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz

vcftools --gzvcf /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrALL.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.vcf.gz --maf .05 --geno .95 --hwe .00001 --recode --recode-INFO-all --out /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrALL.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.maf05.geno95.hwe1eNeg5 

gzip /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrALL.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.maf05.geno95.hwe1eNeg5.recode.vcf

vcftools --gzvcf /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrALL.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.maf05.geno95.hwe1eNeg5.recode.vcf.gz --plink-tped --out /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrALL.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.maf05.geno95.hwe1eNeg5

plink --tfile /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrALL.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.maf05.geno95.hwe1eNeg5 --make-bed --out /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrALL.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.maf05.geno95.hwe1eNeg5

plink --bfile /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrALL.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.maf05.geno95.hwe1eNeg5 --genome --out /KG/1000_Genomes/Samples/VCFs/HIVProj/ALL.chrALL.phase1_release_v3.20101123.snps_indels_svs.genotypes.DropOffTarg_1kb.maf05.geno95.hwe1eNeg5




#TimeptChck: 20140319

#For Sudhir
cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.csv | perl -F, -lane 'my $poolID; foreach my $entry (@F) { if ($entry =~ m/(.*ool)\s+([a-zA-Z0-9]*)/) { my $store2 = $2; if ($store2 =~ m/^[0-9]$/) { $poolID = "POOL0" . $store2; } else { $poolID = "POOL" . $2; } } } if ($F[2] =~ m/^[0-9]$/) { $F[2] = "0" . $F[2]; } if ($poolID) { print $F[1], "\t", $poolID, "_", $F[2], "RL"; }' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.LinkMACID_POOLID.txt

#Manually correct POOL110_04RL line -- it's wrong

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.LinkMACID_POOLID.txt | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.WhiteOnly.QCed.FIDs | sed 's/\^//g' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.justEuropean.QCed.txt

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.LinkMACID_POOLID.txt | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.hwe1eNeg4.IBD.genome.AppearMoreThan100Times.FIDs | sed 's/\^//g' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.justEuropean.PresentInMoreThan100PairsZ2.txt
cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.LinkMACID_POOLID.txt | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.WhiteOnly.QCed.maf05.hwe1eNeg4.IBD.genome.AppearMoreThan150Times.FIDs | sed 's/\^//g' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.justEuropean.PresentInMoreThan150PairsZ2.txt




#At this point began moving files without any 'Vs#' signifier in their names to Vs1/

#mapping_pool_merged directory
mkdir Vs1
ls -lrt | grep -v Vs2 | grep AllPools | awk '{ print $9 }' | perl -ane 'system("mv $F[0] Vs1/$F[0]");'

#GATK Directory
gzip AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.raw.vcf &	
mkdir Vs1
ls -lrt | grep -v Vs2 | grep AllPools | awk '{ print $9 }' | perl -ane 'system("mv $F[0] Vs1/$F[0]");'

#Samtools Directory
mkdir Vs1
mv AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.Chr* Vs1/.





6th run with mapping_pool_125_
~~~~~~~~~~~

#FIgure out phenotype/covar file stuff here now

scp mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/Sample_status_list_12-10-13_CT.edited.csv /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/.

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Sample_status_list_12-10-13_CT.edited.csv | sed 's/\r/\n/g' | sed 's/"//g ' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Sample_status_list_12-10-13_CT.edited.noMeta.csv

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.noMeta.noBlank.noCarrot.dropIndvs.csv --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Sample_status_list_12-10-13_CT.edited.noMeta.csv > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno

#4 individuals missing from the /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Sample_status_list_12-10-13_CT.edited.noMeta.csv file and unsure why -- emailed Sudhir about it
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk 'BEGIN{ while(getline < "/data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.POOLIDs") { POOLID[$1] = 1 }  } { if (POOLID[$1]) { print $1 } } ' > nana
cat "/data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.POOLIDs" | grep -v -f nana
#POOL43_10RL
#POOL101_11RL
#POOL18S_07RL
#POOL2S_03RL

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk 'BEGIN{ while(getline < "/data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.POOLIDs") { POOLID[$1] = 1; } } { if (POOLID[$1]) { print $0 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno.MainPOOLIDs

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateCovariateFile.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar



ls -lrt /data/userdata/pg/MM5_ALL_MAPPING/. | awk '{ print $9 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/home_pg_ALL_MAPPING_dirList.20140321.txt

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/home_pg_ALL_MAPPING_dirList.20140321.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/home_pg_ALL_MAPPING_dirList.20140321.NotCreatedYet.txt
#Then manually edited to remove directories already used/processed before -- kept mapping_pool_125_

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectLSInfoPerDir.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectLSInfoPerDir.vs3.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectLSInfoPerDir.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.NotCreatedYet.txt MM5 #Creates subdirectory folder-list per directory

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.MakeDir.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.MakeDir.vs3.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.MakeDir.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.NotCreatedYet.txt MM5 #Creates the rest of the sub-subdirectories, etc. in ALL_MAPPING

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectOrigSFFFileInformation.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectOrigSFFFileInformation.vs3.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectOrigSFFFileInformation.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.NotCreatedYet.txt MM5 #Creates home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.${date1}.txt file

cat /data/userdata/pg/michaelt/Data/sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.20140321_160330.txt | sort | uniq > /data/userdata/pg/michaelt/Data/sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.20140321_160330.SortUniq.txt

bash /data/userdata/pg/michaelt/Data/sffFiles/DataOrganization.MakeDir.vs2.sh /data/userdata/pg/michaelt/Data/sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.20140321_160330.SortUniq.txt

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CombinePoolNameswsffFileIDs.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CombinePoolNameswsffFileIDs.vs3.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CombinePoolNameswsffFileIDs.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.NotCreatedYet.txt /data/userdata/pg/michaelt/Data/sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.20140321_160330.SortUniq.txt MM5 #Creates file such as DataOrganization.CombinePoolNameswsffFileIDs.vs3.${date1}.output

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.ConnectPoolsffFileswithIndvMIDRLsffFiles.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.ConnectPoolsffFileswithIndvMIDRLsffFiles.vs3.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.ConnectPoolsffFileswithIndvMIDRLsffFiles.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.NotCreatedYet.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CombinePoolNameswsffFileIDs.vs3.20140321_161846.output MM5

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.NotCreatedYet.txt MM5 #Creates DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.${date1}.output file 

bash /data/userdata/pg/michaelt/Data/sffFiles/DataProcessing.BreakUpMIDs.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.20140321_160330.SortUniq.txt

bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.20140321_164539.output

cp -p DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.vs4.sh DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.vs5.sh

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.20140321_164539.output

#Edited DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output via :.,$ s/Pools/Pools\/MM5/g 

cat DataOrganization.CollectAllIndvMIDRLsffFiles.vs2.WorkingMainFile.Edited.output DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.20140321_164539.output > DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output

cp -p DataOrganization.Utility.CreatebamMergeList.vs2.sh DataOrganization.Utility.CreatebamMergeList.vs3.sh

bash DataOrganization.Utility.CreatebamMergeList.vs3.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output

#Remove carrots from file, not dropping individuals. Then creating a file which takes all the IDs from the second column in the POP_genomics file, hashes them, then prints out the list of multiple entries per ID plus their RMID. The extra perl stuff is formatting this because I print out multiple python arrays which have that funny [' ', ' ', ' '] formatting (joined by ;). I then grep out the multiple entries of the main 'CreatebamMergeList' file given the list of multiple entries for the single ID, add them together in a single line and reappend them to the original CreatebamMergeList file. * individuals are dropped because their ID names have the star so they don't get matched up to non-star ID# versions of that ID#. I then have to manually remove the star entries from the POP_genomics file so that its matched up to the CreatebamMergeList which future downstream steps are based on

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.csv | sed 's/\^//g' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.noCarrot.csv

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.Utility.CheckForDupEntriesPOPSampleInfoFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.noCarrot.csv | perl -F: -ane 'my @vals1 = split(/,/, $F[0]); my $id1 = $vals1[1]; print $id1, ":"; foreach my $entry1 (@F) { my @valsN = split(/,/, $entry1); my @MID = split(//, $valsN[2]); if ($#MID == "3") { print "0", $valsN[2], ","; } else { print $valsN[2], ","; } foreach my $entry2 (@valsN) { if ($entry2 =~ m/ool/) { print $entry2; } } print ":";} print "\n";' | sed "s/'//g" | sed 's/\s//g' | perl -F: -lane 'print join(":", @F[1..$#F]);' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.noCarrot.DupListPrepForMergeListUse.txt

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineMultPoolsForSameIndvbamMergeList.vs1.sh /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.noCarrot.DupListPrepForMergeListUse.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140323_234600.output #Creates DataOrganization.CreatebamMergeList.vs3.20140323_234600.output.multPOOLperIndvCollected* files

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.noCarrot.csv | grep -v \* > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.noCarrot.dropIndvs.csv 

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.DropIndividualsBasedOnPopGenomFile.vs3.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.noCarrot.dropIndvs.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140323_234600.output.multPOOLperIndvCollected > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140323_234600.output.multPOOLperIndvCollected.dropIndvs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.20140321_164539.output | awk '{ print $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.20140321_164539.output.justSFFFiles

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140323_234600.output.multPOOLperIndvCollected.dropIndvs | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.20140321_164539.output.justSFFFiles > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140323_234600.output.multPOOLperIndvCollected.dropIndvs.justRecentAdditions

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs2.sh

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.vs4.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140323_234600.output.multPOOLperIndvCollected.dropIndvs.justRecentAdditions 

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs2.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140323_234600.output.multPOOLperIndvCollected.dropIndvs.justRecentAdditions

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs2.sh

#bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140323_234600.output.multPOOLperIndvCollected.dropIndvs


#Troublehsooting

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.noCarrot.DupListPrepForMergeListUse.txt | grep pool7:01 > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.noCarrot.DupListPrepForMergeListUse.01_pool7_01_Pool22.txt

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineMultPoolsForSameIndvbamMergeList.vs1.sh /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_2.edited.noCarrot.DupListPrepForMergeListUse.01_pool7_01_Pool22.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140323_234600.output.multPOOLperIndvCollected.test



#At this point realized there were some differences between the earlier 20140322_POP_genomics_sample_information_2... files I was using and the most recent one (I had just been appending the most recent additions of the new files to my on-going, edited file from earlier versions. However, a few entries had been changed beyond just the new additions, so needed to more thoroughly compare IDs between my edited, on-going version and the newest file. So redoing things from the 'MergeallSampleBamFiles' point onwards. Also included individuals from POOL86 which had been skipped at some point

ls -lrt /data/userdata/pg/MM5_ALL_MAPPING/mapping_pool_78-90/. | awk '{ print $9 } ' > Pools/MM5/PreMerge/mapping_pool_78-90/home_pg_ALL_MAPPING_dirList.WorkingMainList.txt.mapping_pool_78-90.Full

cat Pools/MM5/PreMerge/mapping_pool_78-90/home_pg_ALL_MAPPING_dirList.WorkingMainList.txt.mapping_pool_78-90.Full | grep 86 > Pools/MM5/PreMerge/mapping_pool_78-90/home_pg_ALL_MAPPING_dirList.WorkingMainList.txt.mapping_pool_78-90

cp -p home_pg_ALL_MAPPING_dirList.20140321.NotCreatedYet.txt home_pg_ALL_MAPPING_dirList.20140321.86Redo.txt

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PreMerge/mapping_pool_78-90/home_pg_ALL_MAPPING_dirList.WorkingMainList.txt.mapping_pool_78-90 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PreMerge/mapping_pool_78-90/home_pg_ALL_MAPPING_dirList.20140321.86Redo.txt.mapping_pool_78-90

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PreMerge/mapping_pool_78-90/home_pg_ALL_MAPPING_dirList.WorkingMainList.txt.mapping_pool_78-90.Full home_pg_ALL_MAPPING_dirList.WorkingMainList.txt.mapping_pool_78-90

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.MakeDir.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.86Redo.txt MM5 #Creates the rest of the sub-subdirectories, etc. in ALL_MAPPING

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectOrigSFFFileInformation.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.86Redo.txt MM5 #Creates home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.${date1}.txt file

cat /data/userdata/pg/michaelt/Data/sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.20140328_154417.txt | sort | uniq > /data/userdata/pg/michaelt/Data/sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.20140328_154417.SortUniq.txt

bash /data/userdata/pg/michaelt/Data/sffFiles/DataOrganization.MakeDir.vs2.sh /data/userdata/pg/michaelt/Data/sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.20140328_154417.SortUniq.txt

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CombinePoolNameswsffFileIDs.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.86Redo.txt /data/userdata/pg/michaelt/Data/sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.20140328_154417.SortUniq.txt MM5 #Creates file such as DataOrganization.CombinePoolNameswsffFileIDs.vs3.${date1}.output

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.ConnectPoolsffFileswithIndvMIDRLsffFiles.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.86Redo.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CombinePoolNameswsffFileIDs.vs3.20140328_154612.output MM5

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.sh home_pg_ALL_MAPPING_dirList.20140321.86Redo.txt MM5 #Creates DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.${date1}.output file 

bash /data/userdata/pg/michaelt/Data/sffFiles/DataProcessing.BreakUpMIDs.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/sffFiles/home_pg_ALL_Mapping_ALL_Pools_sff_OrigLocations.Unedited.vs3.20140328_154417.SortUniq.txt

bash DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.20140328_173953.output

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.20140328_173953.output

cat DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.20140328_173953.output > DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output.temp

mv DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output.temp DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output

bash DataOrganization.Utility.CreatebamMergeList.vs3.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output

scp mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.csv mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.worksheet.csv /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/.

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.csv | sed 's/\r/\n/g' | grep -v '^$' | sed 's/\^//g' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.noMeta.noBlank.noCarrot.csv

#MANUALLY included *s next to POOL106-01 (there are two of them and for each MACID there is a duplicate so should be fine) and POOL112-01 because there are two entries in the POP_genomics files for this POOL#_MID# combination, so unsure which is correct MACID. But since there are duplicate entries for each MACID should be fine 

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.Utility.CheckForDupEntriesPOPSampleInfoFile.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.noMeta.noBlank.noCarrot.csv | perl -F: -ane 'chomp(@F); my @vals1 = split(/,/, $F[0]); my $id1 = $vals1[1]; print $id1, ":"; foreach my $entry1 (@F) { my @valsN = split(/,/, $entry1); my @MID = split(//, $valsN[2]); if ($#MID == "3") { print "0", $valsN[2], ","; } else { print $valsN[2], ","; } foreach my $entry2 (@valsN) { if ($entry2 =~ m/ool/) { print $entry2; } } print ":";} print "\n";' | sed "s/'//g" | sed 's/\s//g' | sed 's/\]//g' | perl -F: -lane 'print join(":", @F[1..$#F]);' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.noMeta.noBlank.noCarrot.DupListPrepForMergeListUse.txt

cp -p DataProcessing.Pipeline.Utility.CombineMultPoolsForSameIndvbamMergeList.vs1.sh DataProcessing.Pipeline.Utility.CombineMultPoolsForSameIndvbamMergeList.vs2.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineMultPoolsForSameIndvbamMergeList.vs2.sh /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.noMeta.noBlank.noCarrot.DupListPrepForMergeListUse.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output #Creates DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected* files

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.noMeta.noBlank.noCarrot.csv | grep -v \* > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.noMeta.noBlank.noCarrot.dropIndvs.csv 

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.DropIndividualsBasedOnPopGenomFile.vs3.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20140322_POP_genomics_sample_information_3.edited.noMeta.noBlank.noCarrot.dropIndvs.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs | perl -F\; -lane 'my @vals1 = split(/\|/, $F[0]); my @vals2 = split(/\//, $vals1[0]); if ($vals2[$#vals2] =~ m/(POOL[a-zA-Z0-9]*).*(\d\d+RL)/) { print $1 . "_" . $2 ; } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.POOLIDs

#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs | awk '{ if (NR >= 833) { print $0 } } ' > DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.Redos
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs | awk '{ if (NR >= 837) { print $0 } } ' > DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.Redos

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.Redos

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.Redos

#Did this process twice, once originally and again with the POOL106/POOL112 issue

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs | grep -E 'POOL157-01|POOL158-08|POOL158-01|POOL171-05' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.Redos2

#Once more for POOL15S-04 issue

cat DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs | grep POOL16S-02 > DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.Redos3

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.Redos2

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.Redos3

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.Redos2

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.Redos3

#bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.pta.SplitBamIntoChromosomes.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Vs1

ls -lrt | grep -v Vs2 | grep AllPools | awk '{ print $9 }' | perl -ane 'system("mv $F[0] Vs1/$F[0]");'

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.pta.RunReduceReadsPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptb.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.Samtools.pta.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptc.MergeVCFsAcrossChromosomes.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptc.MergeVCFsAcrossChromosomes.vs2.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptc.MergeVCFsAcrossChromosomes.vs2.sh

cp -p DataProcessing.Pipeline.pt6.GATK.ptd.RecalAndApplyVCFfilesPerChromosome.vs1.sh DataProcessing.Pipeline.pt6.GATK.ptd.RecalAndApplyVCFfilesPerChromosome.vs2.sh 

bash DataProcessing.Pipeline.pt6.GATK.ptd.RecalAndApplyVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged

scp Pools/MM5/PostMerge/mapping_pool_merged/GATK/*tranches.pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/.

gzip -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf &

gzip -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.vcf &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz --remove-filtered VQSRTrancheSNP99.90to100.00 --remove-filtered LowQual --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9

#For zlib installing, directory /home/michaelt/Software/zlib-1.2.8
./configure --prefix=/home/michaelt/lib; make test
#Unsure how to make vcftools use this particular zlib library

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.recode.vcf

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.recode.vcf.gz | fill-aa -a /home/michaelt/Data/1000G/AncestralAlignments/human_ancestor_GRCh37_e59/human_ancestor_ | gzip -c > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.vcf.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.vcf.gz | perl -lane 'if ($F[0] =~ m/^#/) { print join("\t", @F); } else { my @alleles2 = split(/,/, $F[4]); if ($#alleles2 == 0) { print join("\t", @F); } }' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.DropOff.VCFVariants.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --window 1000 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.vcf.gz

#/home/michaelt/Software/vcftools_0.1.12a/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.vcf.gz --missing-indv --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb

gzip -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf

#Exploring what HWE cutoff to use

#vcftools --gzvcf  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz --hardy --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95

#Going with .0001 HWE cutoff. At .00001 1809 SNPs are dropped and at .0001 2470 SNPs are dropped, so not much is lost for another magnitude increase in strictness. Between .0001 and .001 not a large number of SNPs would be lost either, however see examples where obs is 900/0/1 and exp is 900/2/0 on a number of X chromosome locations and this produces a p-value between .0001 and .001. It appears that vcf file encoding gives males genotypes for both 'chromosomes' even though they only have one, so 900/0/1 is not unexpected. Therefore I feel like these SNPs should be kept and going up to .001 may start leading to a greater proportion of false positives
#20151217 NOTE -- going back and thinking/addressing this again, there is also the case that by at this point the total list of SNPs before HWE filtering is 75856. Other 'sequencing' studies generally have less conservative HWE cutoffs (in the 1e6 to 1e7 range), but I believe this is in part to the much larger number of variants they are filtering in their particular projects. Despite this also being a 'sequencing' study, because it is only targetting a subset of gene exomes, the total number of variants is still rather modest. This further suggests, aside from just empirically looking as above, that using the more common HWE cutoff of 1e4 is fine ('more common' in the sense that SNP genotyping studies tend to have this cutoff, whereas 'sequencing' studies tend to have a lower cutoff).
#20151217 NOTE -- As 'full sequencing study' examples with HWE cutoffs between 1e6 and 1e7 http://www.nature.com/articles/sdata201511 & http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3852935/ 

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz --hwe .0001 --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.recode.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf


#Note -- takes about ~2-3 days to complete even with -nt 6?
/home/shared/software/java/jre1.7.0_09/bin/java -Xmx5g -jar /home/shared/software/GATK/2.5.2/GenomeAnalysisTK.jar -T VariantAnnotator -R /home/michaelt/Data/HumanGenome/GRCh37/Version4/human_g1k_v37.fasta -I /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.bam  -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.HRun.vcf -A HomopolymerRun --variant /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf -nt 6

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.HRun.vcf  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.HRun.vcf.idx /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.idx

gzip -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | perl -lane 'my @vals1 = split(/;/, $F[7]); foreach my $SNPinfo (@vals1) { if ($SNPinfo =~ m/HRun/) { print $SNPinfo; } };' | sort | uniq -c 

[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | perl -lane 'my @vals1 = split(/;/, $F[7]); foreach my $SNPinfo (@vals1) { if ($SNPinfo =~ m/HRun/) { print $SNPinfo; } };' | sort | uniq -c
	47294 HRun=0
	18359 HRun=1
	5419 HRun=2
	1668 HRun=3
	401 HRun=4
	97 HRun=5
	15 HRun=6
	6 HRun=7
	2 HRun=8
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | grep HRun=8 | perl -lane 'print join("\t", @F[0..9]);'
2       69628089        rs12464941      A       T       6521.76 VQSRTrancheSNP99.00to99.90      AA=A;AC=197;AF=0.111;AN=1776;BaseQRankSum=-18.785;DB;DP=6423;Dels=0.00;FS=7.055;HRun=8;HaplotypeScore=4.3163;InbreedingCoeff=-0.0905;MLEAC=179;MLEAF=0.101;MQ=59.99;MQ0=0;MQRankSum=1.477;POSITIVE_TRAIN_SITE;QD=4.41;ReadPosRankSum=-0.423;VQSLOD=1.36;culprit=QD GT:AD:DP:GQ:PL  0/0:10,0:10:30:0,30,275
11      63395938        .       G       T       87.12   VQSRTrancheSNP99.00to99.90      AA=G;AC=4;AF=2.230e-03;AN=1794;BaseQRankSum=-3.920;DP=5680;Dels=0.00;FS=3.144;HRun=8;HaplotypeScore=3.2769;InbreedingCoeff=-0.0297;MLEAC=2;MLEAF=1.115e-03;MQ=59.99;MQ0=0;MQRankSum=1.000;QD=2.72;ReadPosRankSum=-1.026;VQSLOD=-1.290e+00;culprit=QD       GT:AD:DP:GQ:PL  0/0:9,0:9:27:0,27,276
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | grep HRun=7 | perl -lane 'print join("\t", @F[0..9]);'
2       182390107       rs2305580       A       T       10719.60        VQSRTrancheSNP99.00to99.90      AA=A;AC=156;AF=0.086;AN=1804;BaseQRankSum=-25.342;DB;DP=10424;Dels=0.00;FS=31.039;HRun=7;HaplotypeScore=5.5982;InbreedingCoeff=-0.0131;MLEAC=144;MLEAF=0.080;MQ=60.00;MQ0=0;MQRankSum=0.217;NEGATIVE_TRAIN_SITE;POSITIVE_TRAIN_SITE;QD=5.97;ReadPosRankSum=3.211;VQSLOD=-6.062e-01;culprit=FS      GT:AD:DP:GQ:PL  0/0:13,0:13:39:0,39,410
3       112272598       rs183474871     T       A       721.65  VQSRTrancheSNP99.00to99.90      AA=T;AC=31;AF=0.018;AN=1758;BaseQRankSum=-11.115;DB;DP=4946;Dels=0.01;FS=4.698;HRun=7;HaplotypeScore=2.4788;InbreedingCoeff=-0.0591;MLEAC=24;MLEAF=0.014;MQ=59.98;MQ0=0;MQRankSum=1.272;POSITIVE_TRAIN_SITE;QD=3.86;ReadPosRankSum=-1.012;VQSLOD=-1.082e-01;culprit=QD     GT:AD:DP:GQ:PL  0/0:2,1:3:6:0,6,62
7       24855651        rs76330454      T       A       2708.55 VQSRTrancheSNP99.00to99.90      AA=t;AC=166;AF=0.094;AN=1770;BaseQRankSum=-8.001;DB;DP=5579;Dels=0.03;FS=71.176;HRun=7;HaplotypeScore=4.9968;InbreedingCoeff=-0.0691;MLEAC=120;MLEAF=0.068;MQ=59.92;MQ0=0;MQRankSum=-0.782;NEGATIVE_TRAIN_SITE;POSITIVE_TRAIN_SITE;QD=2.58;ReadPosRankSum=2.005;VQSLOD=-1.157e+00;culprit=QD       GT:AD:DP:GQ:PL  0/0:4,0:4:12:0,12,107
12      104333866       rs1165685       T       A       24175.80        VQSRTrancheSNP99.00to99.90      AA=t;AC=560;AF=0.320;AN=1748;BaseQRankSum=2.702;DB;DP=4895;Dels=0.03;FS=3.324;HRun=7;HaplotypeScore=4.9939;InbreedingCoeff=-0.1876;MLEAC=523;MLEAF=0.299;MQ=59.97;MQ0=0;MQRankSum=-0.169;POSITIVE_TRAIN_SITE;QD=8.39;ReadPosRankSum=1.064;VQSLOD=-3.806e-01;culprit=DP     GT:AD:DP:GQ:PL  0/1:1,2:3:15:37,0,15
14      81992779        rs76653020      A       T       2523.52 VQSRTrancheSNP99.00to99.90      AA=A;AC=36;AF=0.020;AN=1808;BaseQRankSum=-12.836;DB;DP=12204;Dels=0.01;FS=4.667;HRun=7;HaplotypeScore=6.2385;InbreedingCoeff=0.0585;MLEAC=29;MLEAF=0.016;MQ=59.99;MQ0=0;MQRankSum=2.132;POSITIVE_TRAIN_SITE;QD=5.20;ReadPosRankSum=-2.150;VQSLOD=-3.809e-01;culprit=HaplotypeScore GT:AD:DP:GQ:PL  0/0:13,0:13:39:0,39,415
17      17180049        rs150128160     A       T       452.79  VQSRTrancheSNP99.00to99.90      AA=A;AC=27;AF=0.015;AN=1774;BaseQRankSum=-3.092;DB;DP=5161;Dels=0.00;FS=0.759;HRun=7;HaplotypeScore=4.9453;InbreedingCoeff=-0.0183;MLEAC=18;MLEAF=0.010;MQ=59.94;MQ0=0;MQRankSum=0.504;POSITIVE_TRAIN_SITE;QD=3.10;ReadPosRankSum=-0.415;VQSLOD=-6.715e-01;culprit=QD      GT:AD:DP:GQ:PL  0/0:3,0:3:9:0,9,85
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | grep HRun=6 | perl -lane 'print join("\t", @F[0..9]);'
1       184679716       rs149454208     C       A       274.86  VQSRTrancheSNP99.00to99.90      AA=C;AC=18;AF=9.923e-03;AN=1814;BaseQRankSum=-7.850;DB;DP=9470;Dels=0.00;FS=9.233;HRun=6;HaplotypeScore=5.5968;InbreedingCoeff=0.0145;MLEAC=9;MLEAF=4.961e-03;MQ=59.98;MQ0=0;MQRankSum=-0.037;POSITIVE_TRAIN_SITE;QD=2.10;ReadPosRankSum=-1.369;VQSLOD=-7.438e-01;culprit=QD       GT:AD:DP:GQ:PL  0/0:11,0:11:33:0,33,319
2       61760814        rs115819167     C       T       3750.38 VQSRTrancheSNP99.00to99.90      AA=C;AC=46;AF=0.025;AN=1816;BaseQRankSum=-15.429;DB;DP=12101;Dels=0.01;FS=18.682;HRun=6;HaplotypeScore=4.4872;InbreedingCoeff=-0.0312;MLEAC=42;MLEAF=0.023;MQ=59.99;MQ0=0;MQRankSum=-0.764;POSITIVE_TRAIN_SITE;QD=5.78;ReadPosRankSum=-1.032;VQSLOD=0.188;culprit=FS       GT:AD:DP:GQ:PL  0/0:6,0:6:15:0,15,156
3       51449206        .       A       T       173.08  VQSRTrancheSNP99.00to99.90      AA=A;AC=9;AF=5.039e-03;AN=1786;BaseQRankSum=-6.696;DP=5412;Dels=0.01;FS=0.839;HRun=6;HaplotypeScore=2.1658;InbreedingCoeff=-0.0358;MLEAC=4;MLEAF=2.240e-03;MQ=59.99;MQ0=0;MQRankSum=-0.333;QD=2.58;ReadPosRankSum=0.974;VQSLOD=-8.534e-01;culprit=QD       GT:AD:DP:GQ:PL  0/0:8,0:8:24:0,24,241
4       113574658       rs145663069     G       T       3963.70 VQSRTrancheSNP99.00to99.90      AA=g;AC=67;AF=0.037;AN=1812;BaseQRankSum=-12.104;DB;DP=10857;Dels=0.01;FS=3.332;HRun=6;HaplotypeScore=6.4317;InbreedingCoeff=-0.0475;MLEAC=60;MLEAF=0.033;MQ=59.92;MQ0=0;MQRankSum=-0.678;POSITIVE_TRAIN_SITE;QD=5.27;ReadPosRankSum=-0.755;VQSLOD=-9.985e-01;culprit=HaplotypeScore       GT:AD:DP:GQ:PL  0/0:10,0:10:30:0,30,281
5       52095024        rs116734477     C       T       1415.55 VQSRTrancheSNP99.00to99.90      AA=C;AC=63;AF=0.036;AN=1766;BaseQRankSum=-10.197;DB;DP=4709;Dels=0.01;FS=5.456;HRun=6;HaplotypeScore=2.1840;InbreedingCoeff=-0.0134;MLEAC=56;MLEAF=0.032;MQ=59.99;MQ0=0;MQRankSum=0.626;POSITIVE_TRAIN_SITE;QD=4.03;ReadPosRankSum=0.504;VQSLOD=0.471;culprit=QD   GT:AD:DP:GQ:PL  0/0:3,0:3:9:0,9,84
6       7894794 rs1238994       G       T       26388.40        VQSRTrancheSNP99.00to99.90      AA=G;AC=464;AF=0.260;AN=1786;BaseQRankSum=-24.753;DB;DP=6021;Dels=0.02;FS=9.077;HRun=6;HaplotypeScore=3.7333;InbreedingCoeff=0.0220;MLEAC=416;MLEAF=0.233;MQ=59.99;MQ0=0;MQRankSum=-1.277;POSITIVE_TRAIN_SITE;QD=9.70;ReadPosRankSum=1.017;VQSLOD=1.39;culprit=DP  GT:AD:DP:GQ:PL  0/0:4,0:4:12:0,12,108
6       8062437 rs9405369       T       A       17852.20        VQSRTrancheSNP99.00to99.90      AA=A;AC=360;AF=0.205;AN=1758;BaseQRankSum=-34.017;DB;DP=6008;Dels=0.03;FS=23.705;HRun=6;HaplotypeScore=4.9241;InbreedingCoeff=-0.0703;MLEAC=341;MLEAF=0.194;MQ=59.98;MQ0=0;MQRankSum=0.498;NEGATIVE_TRAIN_SITE;POSITIVE_TRAIN_SITE;QD=7.21;ReadPosRankSum=3.990;VQSLOD=-1.722e-01;culprit=FS       GT:AD:DP:GQ:PL  0/0:2,0:2:6:0,6,61
7       148451445       .       C       T       100.12  VQSRTrancheSNP99.00to99.90      AA=C;AC=7;AF=3.867e-03;AN=1810;BaseQRankSum=-3.029;DP=11028;Dels=0.00;FS=0.732;HRun=6;HaplotypeScore=4.2330;InbreedingCoeff=-0.0087;MLEAC=1;MLEAF=5.525e-04;MQ=59.98;MQ0=0;MQRankSum=-1.175;QD=1.79;ReadPosRankSum=-1.119;VQSLOD=0.023;culprit=QD  GT:AD:DP:GQ:PL  0/0:8,0:8:24:0,24,239
9       125782985       rs137999993     G       T       59.56   VQSRTrancheSNP99.00to99.90      AA=G;AC=4;AF=2.222e-03;AN=1800;BaseQRankSum=-4.567;DB;DP=7953;Dels=0.00;FS=0.000;HRun=6;HaplotypeScore=2.6845;InbreedingCoeff=-0.0210;MLEAC=2;MLEAF=1.111e-03;MQ=59.99;MQ0=0;MQRankSum=-0.096;POSITIVE_TRAIN_SITE;QD=1.65;ReadPosRankSum=-1.713;VQSLOD=0.402;culprit=QD    GT:AD:DP:GQ:PL  0/0:13,0:13:39:0,39,416
10      6138587 rs77712162      A       T       570.05  VQSRTrancheSNP99.00to99.90      AA=A;AC=31;AF=0.017;AN=1794;BaseQRankSum=-10.236;DB;DP=6688;Dels=0.00;FS=0.787;HRun=6;HaplotypeScore=3.7655;InbreedingCoeff=-0.0171;MLEAC=17;MLEAF=9.476e-03;MQ=59.99;MQ0=0;MQRankSum=-1.635;POSITIVE_TRAIN_SITE;QD=2.43;ReadPosRankSum=0.316;VQSLOD=0.453;culprit=QD      GT:AD:DP:GQ:PL  0/0:10,0:10:30:0,30,310
11      107977890       rs150382024     C       T       721.48  VQSRTrancheSNP99.00to99.90      AA=C;AC=25;AF=0.014;AN=1780;BaseQRankSum=-9.441;DB;DP=9796;Dels=0.00;FS=26.757;HRun=6;HaplotypeScore=4.1099;InbreedingCoeff=0.0079;MLEAC=11;MLEAF=6.180e-03;MQ=59.99;MQ0=0;MQRankSum=-0.181;POSITIVE_TRAIN_SITE;QD=3.21;ReadPosRankSum=-1.580;VQSLOD=-2.290e-02;culprit=FS GT:AD:DP:GQ:PL  0/0:8,0:8:24:0,24,247
14      81964559        rs112281983     C       T       730.37  VQSRTrancheSNP99.00to99.90      AA=C;AC=5;AF=2.765e-03;AN=1808;BaseQRankSum=-9.942;DB;DP=17920;Dels=0.00;FS=5.016;HRun=6;HaplotypeScore=6.8203;InbreedingCoeff=-0.0051;MLEAC=3;MLEAF=1.659e-03;MQ=59.99;MQ0=0;MQRankSum=0.284;POSITIVE_TRAIN_SITE;QD=6.46;ReadPosRankSum=-0.256;VQSLOD=-3.850e-01;culprit=HaplotypeScore   GT:AD:DP:GQ:PL  0/0:12,0:12:36:0,36,380
19      32930631        rs73567553      G       T       1706.24 VQSRTrancheSNP99.00to99.90      AA=G;AC=14;AF=7.752e-03;AN=1806;BaseQRankSum=-12.128;DB;DP=19003;Dels=0.00;FS=1.714;HRun=6;HaplotypeScore=6.7353;InbreedingCoeff=-0.0087;MLEAC=12;MLEAF=6.645e-03;MQ=59.99;MQ0=0;MQRankSum=1.297;POSITIVE_TRAIN_SITE;QD=5.88;ReadPosRankSum=-1.836;VQSLOD=-3.085e-01;culprit=HaplotypeScore        GT:AD:DP:GQ:PL  0/0:13,0:13:36:0,36,369
19      32931174        .       C       T       675.07  VQSRTrancheSNP99.00to99.90      AA=C;AC=11;AF=6.138e-03;AN=1792;BaseQRankSum=-7.756;DP=8854;Dels=0.00;FS=1.009;HRun=6;HaplotypeScore=3.5924;InbreedingCoeff=-0.0105;MLEAC=6;MLEAF=3.348e-03;MQ=59.99;MQ0=0;MQRankSum=-1.300;QD=5.49;ReadPosRankSum=-0.304;VQSLOD=0.400;culprit=QD  GT:AD:DP:GQ:PL  0/0:6,0:6:18:0,18,180
X       64950246        rs188435729     C       T       258.32  VQSRTrancheSNP99.00to99.90      AA=C;AC=20;AF=0.012;AN=1738;BaseQRankSum=-4.386;DB;DP=4701;Dels=0.00;FS=2.940;HRun=6;HaplotypeScore=2.6133;InbreedingCoeff=-0.0035;MLEAC=5;MLEAF=2.877e-03;MQ=60.00;MQ0=0;MQRankSum=0.140;POSITIVE_TRAIN_SITE;QD=2.53;ReadPosRankSum=-3.178;VQSLOD=-1.397e+00;culprit=QD   GT:AD:DP:GQ:PL  0/0:2,0:2:6:0,6,57

vcftools --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf --plink --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4

plink --file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4

###IBD pair list -- decided going to do eigenstrat first and see if 1) dropping white individuals due to QC and only looking at white individuals lowers the set of removed individuals overall 2) potentially look for clustering amongst those that are dropping or have high z1 vs z0, z2 (e.g. always at the end of the cloud of white non-hispanics)

POOL103_03RL POOL103_03RL POOL105_03RL POOL105_03RL UN    NA  0.0000  0.0480  0.9520  0.9760  -1  0.998919  1.0000      NA
POOL47_10RL  POOL47_10RL  POOL87_11RL  POOL87_11RL UN    NA  0.0038  0.1065  0.8896  0.9429  -1  0.997455  1.0000 313.0000
POOL175_11RL POOL175_11RL  POOL43_10RL  POOL43_10RL UN    NA  0.0019  0.1445  0.8535  0.9258  -1  0.996671  1.0000      NA
POOL36_11RL  POOL36_11RL  POOL50_12RL  POOL50_12RL UN    NA  0.0077  0.1560  0.8363  0.9143  -1  0.996195  1.0000 306.0000
POOL176_12RL POOL176_12RL  POOL36_12RL  POOL36_12RL UN    NA  0.0270  0.1440  0.8291  0.9010  -1  0.995736  1.0000 345.0000
POOL104_07RL POOL104_07RL  POOL33_05RL  POOL33_05RL UN    NA  0.0155  0.1616  0.8230  0.9037  -1  0.995775  1.0000 149.5000
POOL165_12RL POOL165_12RL  POOL71_11RL  POOL71_11RL UN    NA  0.0213  0.1740  0.8048  0.8918  -1  0.995276  1.0000 297.0000
POOL28_06RL  POOL28_06RL  POOL91_11RL  POOL91_11RL UN    NA  0.0194  0.1825  0.7981  0.8893  -1  0.995154  1.0000 73.7500
POOL112_10RL POOL112_10RL POOL116_10RL POOL116_10RL UN    NA  0.0275  0.1951  0.7775  0.8750  -1  0.994566  1.0000 145.0000
POOL163_12RL POOL163_12RL  POOL69_08RL  POOL69_08RL UN    NA  0.0372  0.2257  0.7371  0.8500  -1  0.993508  1.0000 96.3333

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome | sort -rg -k 9,9 | vi -

POOL103_03RL POOL103_03RL POOL105_03RL POOL105_03RL UN    NA  0.0000  0.0434  0.9566  0.9783  -1  0.998919  1.0000      NA
POOL47_10RL  POOL47_10RL  POOL87_11RL  POOL87_11RL UN    NA  0.0034  0.0963  0.9003  0.9484  -1  0.997455  1.0000 313.0000
POOL36_11RL  POOL36_11RL  POOL50_12RL  POOL50_12RL UN    NA  0.0068  0.1412  0.8521  0.9226  -1  0.996195  1.0000 306.0000
POOL112_10RL POOL112_10RL POOL116_10RL POOL116_10RL UN    NA  0.0242  0.1772  0.7985  0.8871  -1  0.994566  1.0000 145.0000
POOL163_12RL POOL163_12RL  POOL69_08RL  POOL69_08RL UN    NA  0.0328  0.2053  0.7619  0.8645  -1  0.993508  1.0000 96.3333

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -E 'POOL103_03RL|POOL105_03RL'
POOL103_03RL    POOL103_03RL    2       2       -9      -9
POOL105_03RL    POOL105_03RL    2       2       -9      -9

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -E 'POOL47_10RL|POOL87_11RL'
POOL87_11RL     POOL87_11RL     1       -9      -9      -9
POOL47_10RL     POOL47_10RL     2       2       -9      -9

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -E 'POOL36_11RL|POOL50_12RL'
POOL50_12RL     POOL50_12RL     2       2       1       -9
POOL36_11RL     POOL36_11RL     2       2       1       -9

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -E 'POOL112_10RL|POOL116_10RL'
POOL112_10RL    POOL112_10RL    1       1       -9      -9
POOL116_10RL    POOL116_10RL    2       2       -9      -9

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -E 'POOL163_12RL|POOL69_08RL'
POOL69_08RL     POOL69_08RL     1       1       -9      -9
POOL163_12RL    POOL163_12RL    1       -9      -9      -9

perl -e 'print "POOL103_03RL\nPOOL105_03RL\nPOOL87_11RL\nPOOL47_10RL\nPOOL50_12RL\nPOOL36_11RL\nPOOL112_10RL\nPOOL116_10RL\nPOOL69_08RL\nPOOL163_12RL\n";' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.FIDs


vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.FIDs --depth --missing --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs

cat AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.idepth
INDV    N_SITES MEAN_DEPTH
POOL103_03RL    73251   22.4186
POOL105_03RL    73254   22.2204
POOL112_10RL    71708   7.28291
POOL116_10RL    73100   15.5343
POOL163_12RL    71776   6.90738
POOL36_11RL     73195   14.8737
POOL47_10RL     73238   17.4978
POOL50_12RL     73127   11.7687
POOL69_08RL     73211   17.0737
POOL87_11RL     73180   13.5165

cat AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.imiss
INDV    N_DATA  N_GENOTYPES_FILTERED    N_MISS  F_MISS
POOL103_03RL    73261   0       10      0.000136498
POOL105_03RL    73261   0       7       9.55488e-05
POOL112_10RL    73261   0       1553    0.0211982
POOL116_10RL    73261   0       161     0.00219762
POOL163_12RL    73261   0       1485    0.02027
POOL36_11RL     73261   0       66      0.000900889
POOL47_10RL     73261   0       23      0.000313946
POOL50_12RL     73261   0       134     0.00182908
POOL69_08RL     73261   0       50      0.000682491
POOL87_11RL     73261   0       81      0.00110564

So choosing:

POOL105_03RL
POOL36_11RL
POOL116_10RL

perl -e 'print "POOL103_03RL\nPOOL112_10RL\nPOOL163_12RL\nPOOL47_10RL\nPOOL50_12RL\nPOOL69_08RL\nPOOL87_11RL\n";' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome | sort -rg -k 9,9 | head -n 10000 | awk '{ print $1, "\n", $3 }' | perl -lane 'chomp(@F); print join("\t", @F);' | sort | uniq -c | sort -g -k 1,1 | awk '{ if ($1 >= 100 ) { print $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ2Countgte100.FIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome | sort -rg -k 8,8 | head -n 10000 | awk '{ print $1, "\n", $3 }' | perl -lane 'chomp(@F); print join("\t", @F);' | sort | uniq -c | sort -g -k 1,1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.Z1CountList

#NOTE -- dropping these individuals because they are starkly greater than anyone else in the list. Individuals from the 'high z2' count list are not being dropped because they appear as just to be the tail of a distribution versus being such a group of outliers
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome | sort -rg -k 8,8 | head -n 10000 | awk '{ print $1, "\n", $3 }' | perl -lane 'chomp(@F); print join("\t", @F);' | sort | uniq -c | sort -g -k 1,1 | awk '{ if ($1 >= 200) { print $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.FIDs

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs

cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ2Countgte100.FIDs | perl -lane 'push(@F, "Z2"); print join("\t", @F);') <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.FIDs | perl -lane 'push(@F, "Z1"); print join("\t", @F);') > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.OddHighZ1OrZ2.FID_ZNums


#EIGENSTRAT stuff here

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/VLookUp.Python.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/VLookUp.Python.DataHapMap3Overlap.vs1.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/VLookUp.Python.DataHapMap3Overlap.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.map --file2 /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.liftOver.hg18tohg19.map | awk '{ print $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.rsIDs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.rsIDs --recode --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3

plink --file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3 --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3

#Note -- there are a few SNPs that don't match up, about ~30? Just letting them drop since troubleshooting almost got most of them back but not all of them and yeah

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.map | awk '{ print $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.map.rsIDs 

plink --bfile /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.map.rsIDs --keep  /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.CEUCHBYRI.Unrelated.FIDIIDs --recode --out  /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Vs2

plink --file /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Vs2 --make-bed --out /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Vs2

paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.bim /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Vs2.bim | awk '{ if (($5 == $11) || ($5 == $12) ) { print $0 } } ' | awk '{ print $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.FinalListwDrops.rsIDs 

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3 --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.FinalListwDrops.rsIDs --recode --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.vs2

plink --bfile /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.FinalListwDrops.rsIDs --keep /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.CEUCHBYRI.Unrelated.FIDIIDs --recode --out /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Vs2.vs2

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.vs2.ped /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Vs2.vs2.ped > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.ped

plink --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.ped --map /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.vs2.map --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.phenoColEdit.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.phenoColEdit.pedind

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.bim /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.pedsnp

#Created EigenstratFiles in GATK and moved Eigenstrat/ files into EigenstratFiles/.

scp mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.parfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/.

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.parfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.wHapMap3Data.CEUCHBYRI.parfile

/home/michaelt/Software/EIG5.0.1/bin/smartpca -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.wHapMap3Data.CEUCHBYRI.parfile

#REDO WITH 4 INDVS MISSING FROM PHENO FILE
#python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt1
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca | cat - <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca | grep -E 'POOL43_10RL|POOL101_11RL|POOL18S_07RL|POOL2S_03RL' | perl -lane '$F[$#F] = "NoPhenotype"; print join("\t", @F); ') > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt1

#Added in the following four individuals with the tag 'NoPhenotype' at the end manually in the pt1 file
#POOL43_10RL POOL101_11RL POOL18S_07RL POOL2S_03RL

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.HapMap3vs.py --file1 /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.txt --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt2

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race

sed -i 's/American_Indian_or_Alaskan_Native/AIndian_or_Alaskan_Native/g' /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.1vs2.xtxt

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.3vs4.xtxt

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race -c 5:6 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.5vs6.xtxt

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.1vs2.pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/.

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.*.pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/.

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race | awk '{ if ((($2 >= 0) && ($2 <= .22)) && (($3 >= -.01) && ($3 <= .021))) { print $0 } } ' | grep White_non-Hispanic > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed -c 1:2 -p White_non-Hispanic -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.plot.1vs2.xtxt

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed | perl -lane 'my @vals1 = split(/:/, $F[0]); print $vals1[0];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed | perl -lane 'my @vals1 = split(/:/, $F[0]); print $vals1[0], "\t", $vals1[0];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteACed.plot.1vs2.pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/.

#Need to do this on the no-HapMap3 dataset

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race | grep White_non-Hispanic | awk 'BEGIN{ while(getline < "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.OddHighZ1OrZ2.FID_ZNums") { FID = $1":"$1; FIDList[FID] = $2; }} { if (FIDList[$1]) { if (FIDList[$1] == "Z1") { $12 = "Z1High"; } else if (FIDList[$1] == "Z2") { $12 = "Z2High"; } else{ print "blah" }  } else { $12 = "NormalIBD"; } print $0; } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.IBDissuesTagged

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.IBDissuesTagged -c 1:2 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.IBDissuesTagged.plot.1vs2.xtxt 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed | awk 'BEGIN{ while(getline < "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.OddHighZ1OrZ2.FID_ZNums") { FID = $1":"$1; FIDList[FID] = $2; }} { if (FIDList[$1]) { if (FIDList[$1] == "Z1") { $12 = "Z1High"; } else if (FIDList[$1] == "Z2") { $12 = "Z2High"; } else{ print "blah" }  } else { $12 = "NormalIBD"; } print $0; } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 1:2 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.1vs2.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 3:4 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.3vs4.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 5:6 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.5vs6.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 7:8 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.7vs9.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 9:10 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.9vs10.xtxt

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/*pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/.


cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.phenoColEdit.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.phenoColEdit.pedind

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.bim /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.pedsnp

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.wHapMap3Data.CEUCHBYRI.parfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.parfile

/home/michaelt/Software/EIG5.0.1/bin/smartpca -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.parfile

#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.Results.pca.altrd.race

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race | awk 'BEGIN{ while(getline < "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.OddHighZ1OrZ2.FID_ZNums") { FID = $1":"$1; FIDList[FID] = $2; }} { if (FIDList[$1]) { if (FIDList[$1] == "Z1") { $12 = "Z1High"; } else if (FIDList[$1] == "Z2") { $12 = "Z2High"; } else{ print "blah" }  } else { $12 = "NormalIBD"; } print $0; } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race.IBDissuesTagged

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race.plot.1vs2.xtxt 

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race.plot.3vs4.xtxt 

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race.IBDissuesTagged -c 1:2 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race.IBDissuesTagged.plot.1vs2.xtxt

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race.IBDissuesTagged -c 3:4 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race.IBDissuesTagged.plot.3vs4.xtxt

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.5PCs.covar
#Edit code in between runs
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.pheno

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar | awk '{ print $1, "\t", $2 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar | awk '{ print $1 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDs

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDs --non-ref-ac 1 --recode  --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.recode.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.allSNPs

plink --bfile AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD --freq --out AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.PLINK

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --freq --out  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs.maf05

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.10PCs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.10PCs.maf05

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 2 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs.maf05.pheno2

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 3 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs.maf05.pheno3

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 4 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs.maf05.pheno4


R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs.assoc.logistic Pheno1_justWhite_Logistic_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs.maf05.assoc.logistic Pheno1_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs.maf05.assoc.logistic.QQplot.jpeg &

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.10PCs.assoc.logistic Pheno1_justWhite_Logistic_10PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.10PCs.assoc.logistic.QQplot.jpeg &

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($4 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($3 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroPos.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justRapidVRapid.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSlowVSlow.FIDs
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSlowVSlow.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justVeryRapid.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justVerySlow.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroPos.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justRapidVRapid.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSlowVSlow.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justVeryRapid.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justVerySlow.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow &

paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq.count | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.AllPhenos.frq.AndCount.gz

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq.count

#gunzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq.count


#Annovar stuff here now
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz | awk '{ print $1, "\t", $2, "\t", $2, "\t", $4, "\t", $5 }' | grep -v ^# > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat

perl /home/michaelt/Software/annovar/annotate_variation.pl --downdb -webfrom annovar ljb23_all /home/michaelt/Software/annovar/ -build hg19

perl /home/michaelt/Software/annovar/summarize_annovar.pl /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat /home/michaelt/Software/annovar/ --ver1000g 1000g2012apr --verdbsnp 129 --veresp 6500 --buildver hg19 --remove

cat AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat | head -n 1000 > AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.test3

cp -p AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.test3 AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.test3.cpy

perl /home/michaelt/Software/annovar/summarize_annovar.pl /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.test3 /home/michaelt/Software/annovar/ --ver1000g 1000g2012apr --verdbsnp 129 --veresp 6500 --buildver hg19 --remove

#Edit file /home/michaelt/Software/annovar/summarize_annovar.pl to go from ljb_all to ljb23_all -- then edit .genome_summary.csv file to fill in extra spots in non-exonic variants and to include the below header

perl /home/michaelt/Software/annovar/summarize_annovar.pl /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.test3.cpy --ver1000g 1000g2012apr /home/michaelt/Software/annovar/ --verdbsnp 129 --veresp 6500 --buildver hg19 --remove

#LJB23_SIFT_score LJB23_SIFT_score_converted LJB23_SIFT_pred LJB23_Polyphen2_HDIV_score LJB23_Polyphen2_HDIV_pred LJB23_Polyphen2_HVAR_score LJB23_Polyphen2_HVAR_pred LJB23_LRT_score LJB23_LRT_score_converted LJB23_LRT_pred LJB23_MutationTaster_score LJB23_MutationTaster_score_converted LJB23_MutationTaster_pred LJB23_MutationAssessor_score LJB23_MutationAssessor_score_converted LJB23_MutationAssessor_pred LJB23_FATHMM_score LJB23_FATHMM_score_converted LJB23_FATHMM_pred LJB23_RadialSVM_score LJB23_RadialSVM_score_converted LJB23_RadialSVM_pred LJB23_LR_score LJB23_LR_pred LJB23_GERP++ LJB23_PhyloP LJB23_SiPhy

perl /home/michaelt/Software/annovar/summarize_annovar.pl /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat /home/michaelt/Software/annovar/ --ver1000g 1000g2012apr --verdbsnp 129 --veresp 6500 --buildver hg19 --outfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.ljb23 --remove

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.ljb23.genome_summary.csv | perl -F, -lane 'if ($#F <= 40) { splice(@F, 17, 0, ("") x 16); } print join(",", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.ljb23.genome_summary.Reformatted.csv

#Figure this part out
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.ljb23.genome_summary.Reformatted.csv | perl -F, -lane 'splice(@F, #, 1); splice( @F, #, 1);.... ; print join(",", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.ljb23.genome_summary.Reformatted.lbjColumns.csv

#Realized splicing variants were being cut up into multiple transcripts when they should really be just one -- a splicing variant is shown how it affects multiple transcripts of the same gene, but I want multiple calls of the same variant if it affects multiple genes, not multiple transcripts of the same gene
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs4.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs5.py

#Then had to fix the below issue -- note lack of first splicing 'exon' in the AnnovarAnnotation.vs5 file vs the original genome_summary.csv file
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | grep 16314427
"splicing","AP1M1(NM_001130524:exon2:c.199+1G>A,NM_032493:exon2:c.199+1G>A)",,,"556;Name=lod=242",,,,,,,,,,,,,,,,,19,16314427,16314427,G,A,
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | grep 16314427
19,16314427_1,.,G,A,G,185.60,VQSRTrancheSNP99.00to99.90,AA=G;AC=1;AF=5.543e-04;AN=1804;BaseQRankSum=4.117;DP=16480;Dels=0.00;FS=23.866;HRun=0;HaplotypeScore=3.3100;InbreedingCoeff=-0.0025;MLEAC=1;MLEAF=5.543e-04;MQ=59.99;MQ0=0;MQRankSum=0.122;QD=13.26;ReadPosRankSum=-0.212;VQSLOD=-4.493e-01;culprit=FS,0.000646831,0.0014245,0,0,0,0,0,0,1,1,0,0,0,0,0,0,splicing,AP1M1;NM_032493:exon2:c.199+1G>A),,,,,,,,,,,,,,,19,16314427,16314427,G,A

#Also realized that for entries such as "UTR5;UTR3" and "Exonic;splicing", have gene IDs such as "Gene1;Gene2" and need to fix this somehow

#Troubleshooting
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | perl -F, -lane 'if (($F[1] =~ m/\;/) || ($F[0] =~ m/\;/)) { print join(",", @F); }' | vi -
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | perl -F, -lane 'if (($F[1] =~ m/\;/) || ($F[0] =~ m/\;/)) { print join(",", @F); } ' | awk -F, '{ print $1 }' | sort | uniq -c
   8 "UTR5;UTR3"
   7 "exonic;splicing"
 895 "upstream;downstream"
#Specifically going to deal with assigning UTR5;UTR3 as two separate genes and exonic;splicing as only 'geneID' in the second column entry versus 'geneID;geneID(NM_.....)'. For the latter case, variant frequency is the same regardless of whether exonic or splicing variant and for the same gene, so leaving 'exonic;splicing' will indicate that originally there is a geneID(NM_....) portion of the gene information I'll need to look back at, but for the Skat and EPACTS purposes it'll only be encoded as a single variant for that gene. Having multiple copies of a variant for UTR5;UTR3 shouldn't matter any more than when there are multiple genes in the second column (separated by ,) and I'm cutting them up into separate variant entries anyways
#Additionally, if the first column is 'UTR5;UTR3' or 'exonic;splicing' then there should not be multiple comma-delimited entries in the second column which was the main purpose of creating that breaking up for-loop in the DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs5.py file in the first-place
#Fixed, note the use of ^\"splicing\"$ in the if/elif/else clause involving splicing, UTR5;UTR3, exonic;splicing etc. in order to properly skip 'exonic;splicing' variants when "splicing" is being searched for first, hence the 'Happening4' and 'Happneing4b' calls at first to track where these variants were falling in the clause

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs5.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.AllPhenos.frq.AndCount.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 30; $i <= 40; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz

#Some QC Checks
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | grep -v \# | perl -lane 'print join("\t", @F[0..9]);' | wc
  73261  732610 25819140
 
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | wc
  74824   82642 33131311
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | grep _2 | wc
  1540    1591  683410
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | grep _3 | wc
  21      21    9129
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | grep _4 | wc
  2       2     957
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | grep _5 | wc
  0       0       0
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$R -q -e 'val1 = 1540 + 21 + 2; print (val1);'
> val1 = 1540 + 21 + 2; print (val1);
[1] 1563
>
>
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$R -q -e 'val1 = 1540 + 21 + 2; print (val1); val2 = 74824 - 1563; print (val2);'
> val1 = 1540 + 21 + 2; print (val1); val2 = 74824 - 1563; print (val2);
[1] 1563
[1] 73261
> 
> 

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD

cat AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.allSNPs.frq | perl -lane 'my @vals1 = split(/:/, $F[4]); my @vals2 = split(/:/, $F[5]); if ($vals1[1] < $vals2[1]) { if ($vals1[1] < .05) { print join("\t", @F); } } if ($vals1[1] > $vals2[1]) { if ($vals2[1] < .05 ) { print join(",", @F); } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.allSNPs.frq.maxMaf05

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs2.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs3.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs3.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, 'BEGIN{while(getline < "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.allSNPs.frq.maxMaf05" ) { ID1 = FID = $1"_"$2; IDList[ID1] = 1; } } { ID2 = $1"_"$4; if (IDList[ID2]) { print $0; }  } ' | perl -lane 'print join(",", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.mafMax05


cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing") || ($7 == "UTR5;UTR3")) { print $8, "\t", $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.All

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.mafMax05 | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing") || ($7 == "UTR5;UTR3")) { print $8, "\t", $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.All.mafMax05

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { print $8, "\t", $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.Exonic

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $8, "\t", $2 } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($12 == "P") || ($12 == "D") ) { print $8, "\t", $2 } } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_PD

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $8, "\t", $2 } } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_D




cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing") || ($7 == "UTR5;UTR3")) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.All

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $0 } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($12 == "P") || ($12 == "D") ) { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($13 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing")) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $0 } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($12 == "P") || ($12 == "D") ) { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.Sift_D
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($13 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.LRT_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.pheno | awk '{ print $1, "\t", $2, "\t0\t0\t0\t", $3, "\t", $4, "\t", $5, "\t", $6 } ' | sed 's/-9/NA/g' | perl -lane 'print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar | awk '{ print $4, "\t", $6, "\t", $7 "\t", $8, "\t", $9, "\t", $10, "\t", $11,  "\t", $12, "\t", $13, "\t", $14,  "\t", $15 }' | paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped.vs2

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped.vs2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped

#Add line #FAM_ID IND_ID FAT_ID MOT_ID SEX HIVPROG HIVPROGHE AIDS AIDSEXTR RMID PC1 PC2 PC3 PC4 PC5 PC6 PC7 PC8 PC9 PC10 to /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz | awk '{ if ($1 == "23") { $1 = "X"} print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip

/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz

#In /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles and /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles
mv All* Vs1/.

cp -p DataProcessing.Pipeline.Analysis.Skat.vs3.R DataProcessing.Pipeline.Analysis.Skat.vs4.R

#Troubleshooting
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.All | grep ADM5 | awk '{ print $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.All.ADM5.SNPIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.All | grep ADM5 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.All.ADM5

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.All | grep ADM5 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.All.ADM5
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz --snps /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.All.ADM5.SNPIDs --recode --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.ADM5

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz | awk 'BEGIN{ while(getline < "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.All.ADM5.SNPIDs") { SNPIDList[$1] = 1; } } { if ($1 ~ /^#/) { print $0 } else { SNPID = "Null"; if ($3 !~ /\./ ) { SNPID = $3; } else { SNPID = $1":"$2; } if (SNPIDList[SNPID]) { print $0; } } } ' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.ADM5.vcf.gz   

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.ADM5.vcf.gz --plink --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.ADM5

plink --file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.ADM5 --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.ADM5

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.ADM5.vcf.gz | awk '{ if ($1 == "23") { $1 = "X"} print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.ADM5.vcf.bgzip

/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.ADM5.vcf.bgzip
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.ADM5.vcf.bgzip.gz

mkdir TroubleshootingStuff

mv *ADM5* TroubleshootingStuff/.



##Actually going over and doing the QC stuff
##20141029
#QC stuff here

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetFastqQCStats.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetFastqQCStats.vs2.sh

#Edited /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output via :.,$ s/\/home/\/data\/userdata/g

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetFastqQCStats.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output

#Problem files that threw errors and were leftover
#cat: /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PreMerge/mapping_pool_1-46/POOL20-12RL/HU5Q1CK02.RL12.fastq.stats: No such file or directory
#cat: /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PreMerge/mapping_pool_91-97/POOL94-08RL/H4KITRJ02.RL8.fastq.stats: No such file or directory
#cat: /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PreMerge/mapping_pool_91-97/POOL94-08RL/H4KITRJ01.RL8.fastq.stats: No such file or directory
#cat: /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PreMerge/mapping_pool_91-97/POOL94-10RL/H4KITRJ02.RL10.fastq.stats: No such file or directory

#Removed empty lines in /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetFastqQCStats.vs2.20141029_182644.output with /0.00000.0000 0.0000 and dd in vi
#PreMerge HU5Q1CK02.RL12 0.00000.0000 0.0000               totalDropped(%):   minDropped(%):   UpperBoundThresh:  maxDropped(%):     
#PreMerge H4KITRJ02.RL8 0.00000.0000 0.0000               totalDropped(%):   minDropped(%):   UpperBoundThresh:  maxDropped(%):
#PreMerge H4KITRJ01.RL8 0.00000.0000 0.0000               totalDropped(%):   minDropped(%):   UpperBoundThresh:  maxDropped(%):
#PreMerge H4KITRJ02.RL10 0.00000.0000 0.0000               totalDropped(%):   minDropped(%):   UpperBoundThresh:  maxDropped(%):

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs3.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output.GetperRegionpostQCStats.vs3.20141029_140243.output | perl -lane 'my @array1; push(@array1, $F[1]); foreach my $entry (@F) { if ($entry !~ m/[a-zA-Z]/) { push (@array1, $entry); } } print join(",", @array1);' | awk 'BEGIN{ print "ID,TotalReadsPreQC,TotalReadsQ10,TotalReadsQ10Prcnt,TotalReadsF4,TotalReadsF4Prcnt,TotalReadsF256,TotalReadsF256Prcnt,TotalReadsPostQC,TotalReadsDropped,TotalReadsDroppedPrcnt,AverageReadLengthPreQC,AverageReadLengthPostQC,BedFileCoveragePostQC" } { print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output.GetperRegionpostQCStats.vs3.20141029_140243.output.reformatted

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs3.sh

#Edited /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs via :.,$ s/\/home/\/data\/userdata/g 

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs

#Total column numbers post formatting = 27
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.GetperFullBampostQCStats.vs3.20141029_150137.output | perl -lane 'my @array1; push(@array1, $F[1]); foreach my $entry (@F) { if ($entry !~ m/[a-zA-Z]/) { push (@array1, $entry); } } print join(",", @array1);' | awk 'BEGIN{ print "ID,NumberOfOriginalBamFiles,TotalReadsPreQC,TotalReadsPreQCForward,TotalReadsPreQCReverse,TotalReadsPreQCForward_Reverse,PreQCMAPQmean,PreQCMAPQsd,PreQCMean,PreQCSD,PreQCMax,PreQCMin,CoveragePreQC,TotalReadsPostRmdup,TotalReadsPostRmdupDropped,TotalReadsPostRmdupDroppedPercentage,TotalReadsPostQC,TotalReadsPostQCForward,TotalReadsPostQCReverse,TotalReadsPostQCForward_Reverse,PostQCMAPQmean,PostQCMAPQsd,PostQCMean,PostQCSD,PostQCMax,PostQCMin,CoveragePostQC" } { print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.GetperFullBampostQCStats.vs3.20141029_150137.output.reformatted

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs2.R

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetFastqQCStats.vs2.20141029_182644.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output.GetperRegionpostQCStats.vs3.20141029_140243.output.reformatted /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs.GetperFullBampostQCStats.vs3.20141029_150137.output.reformatted

#[1] NA
#[1] 7.09003
#[1] 0.2671718
#[1] 0.9868955
#null device 
#          1 
#	  [1] 270303.1
#	  [1] 632.9364
#	  [1] 246.6539
#	  [1] 21.70513
#	  [1] 45288.87
#	  [1] 13.76149
#	  null device 
#	            1 

#Following up why there is a NA in the above output, do the following code in R on /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output.GetperRegionpostQCStats.vs3.20141029_140243.output.reformatted and get this:
#> Data2[is.na(Data2$TotalReadsDroppedPrcnt),]
#                 ID TotalReadsPreQC TotalReadsQ10 TotalReadsQ10Prcnt
#		 150  HU5Q1CK02.RL12               0             0                  0
#		 1791  H4KITRJ02.RL8               0             0                  0
#		 1793  H4KITRJ01.RL8               0             0                  0
#		 1799 H4KITRJ02.RL10               0             0                  0
#		      TotalReadsF4 TotalReadsF4Prcnt TotalReadsF256 TotalReadsF256Prcnt
#		      150             0                 0              0                   0
#		      1791            0                 0              0                   0
#		      1793            0                 0              0                   0
#		      1799            0                 0              0                   0
#		           TotalReadsPostQC TotalReadsDropped TotalReadsDroppedPrcnt
#			   150                NA                NA                     NA
#			   1791               NA                NA                     NA
#			   1793               NA                NA                     NA
#			   1799               NA                NA                     NA
#			        AverageReadLengthPreQC AverageReadLengthPostQC BedFileCoveragePostQC
#				150                      NA                      NA                    NA
#				1791                     NA                      NA                    NA
#				1793                     NA                      NA                    NA
#				1799                     NA                      NA                    NA

#So inserted the following line into /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs2.R:
#Data2 <- Data2[!is.na(Data2$TotalReadsDroppedPrcnt),]

#[1] 7.589332
#[1] 7.097796
#[1] 0.2674644
#[1] 0.9879765
#null device 
#          1 
#	  [1] 270303.1
#	  [1] 632.9364
#	  [1] 246.6539
#	  [1] 21.70513
#	  [1] 45288.87
#	  [1] 13.76149
#	  null device 
#	            1 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetFastqQCStats.vs2.20141029_182644.output | R -q -e "Data1 <- read.table(file('stdin'), header=F); sum(as.numeric(Data1[,4])); sum(as.numeric(Data1[,6]));"
> Data1 <- read.table(file('stdin'), header=F); sum(as.numeric(Data1[,4])); sum(as.numeric(Data1[,6]));
[1] 191375813326
[1] 314267723
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetFastqQCStats.vs2.20141029_182644.output | wc
   3652  105908  801691

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/OutputFiles/Graphics/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs2.2014-10-29_19:20:14.SummaryRegionPlots.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/OutputFiles/Graphics/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/OutputFiles/Graphics/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs1.vs2.2014-10-29_19:20:19.SummaryIndvPlots.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/OutputFiles/Graphics/.

#20141104 -- Including the following line into vs6 below: if (re.search('^\"splicing\"$', line3[0]) and re.search('\(NM_', line3[1])) or (re.search('ncRNA_splicing', line3[0]) and re.search('\(NR_', line3[1])):
#Actually fixed this in DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs5.FullVCFwAnnovar.vs1.py first but for chronological reasons edited DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.py and renamed DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs5.FullVCFwAnnovar.vs1.py to DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py
#This is not expected to change anything since it seems like only one ncRNA_splicing variant actually remains present in the QCed vcf file being used, so the rest are dropped. However, hypothetically if they were being included, they were not being properly handled like the other 'splicing' variants were (and this presumably would have shown up in the perl -F, -lane 'print $#F;' type code I've been running to check things, e.g. this scenario would have not been fixed and I would have noticed it sooner (vs. the scenario just not being present at all and just realizing it in retrospect by looking at the code)
cp -p DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs5.py DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.py

cp -p DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.py DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py

[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | perl -F, -lane 'my @vals2 = split (/_/, $F[1]); print $vals2[1];' | sort | uniq -c
 73261 	1
  1540 	2
    21 	3
     2 	4
[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | perl -F, -lane 'if ($F[1] =~ /_4/) { print join(",", @F); } '                                                                                                                                                                                                                  
16,2012009_4,rs116746316,G,A,a,5572.55,VQSRTrancheSNP90.00to99.00,AA=a;AC=66;AF=0.037;AN=1794;BaseQRankSum=18.449;DB;DP=7312;Dels=0.00;FS=1.328;HRun=0;HaplotypeScore=1.9092;InbreedingCoeff=0.0386;MLEAC=61;MLEAF=0.034;MQ=59.99;MQ0=0;MQRankSum=0.412;POSITIVE_TRAIN_SITE;QD=10.70;ReadPosRankSum=2.293;VQSLOD=2.40;culprit=DP,0.964239,0.960114,0.955357,0.967703,0.974138,0.970588,0.978261,0.986486,1483,674,321,809,113,165,45,73,downstream,SNORA64,,,,,,,,,,,,,,,,16,2012009,2012009,G,A                                                                                                                                                      
16,2012027_4,rs114278396,C,T,c,6507.78,VQSRTrancheSNP90.00to99.00,AA=c;AC=66;AF=0.037;AN=1796;BaseQRankSum=10.292;DB;DP=7458;Dels=0.00;FS=0.714;HRun=0;HaplotypeScore=1.4525;InbreedingCoeff=0.0565;MLEAC=63;MLEAF=0.035;MQ=59.98;MQ0=0;MQRankSum=-1.085;POSITIVE_TRAIN_SITE;QD=12.59;ReadPosRankSum=2.717;VQSLOD=2.14;culprit=DP,0.0357143,0.039886,0.0446429,0.0322196,0.0258621,0.0348837,0.0217391,0.0131579,55,28,15,27,3,6,1,1,downstream,SNORA64,,,,,,,,,,,,,,,,16,2012027,2012027,C,T                                                                                                                                                         

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.AllGenesPerVariant.vcf.gz

[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.AllGenesPerVariant.vcf.gz | perl -lane 'my @vals2 = split(/,/, $F[8]); print $#vals2, "\t", $F[8];' | awk '{ print $1; }' | sort | uniq -c                                                                                                                                                               100 -1                                                                                                                                                                                                                                                                                                                      28 0
  74824 25                                                                                                                                                                                                                                                                                                                 

[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$R -q -e "val1 <- 73261 + 1540 + 21 + 2; print(val1);"
> val1 <- 73261 + 1540 + 21 + 2; print(val1);                                                                                                                                                                                                                                                                              
[1] 74824
>                                                                                                                                                                                                                                                                                                                          
>

#Edited code in /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGenePerVariant.vcf.gz

[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals2 = split(/,/, $F[8]); print $#vals2, "\t", $F[8];' | awk '{ print $1; }' | sort | uniq -c
   100 	-1                                                                                                                                                                                                                                                                                                                 
    28 	0
 73261 	25

#/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.GetVCFQCStats.vs1.output too long a file name -- consider changing wHapMap3Data to wHM3Data across board later at some point....maybe?

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.vcf.gz

[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz | wc
  64015 50216707 1178830267

[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.vcf.gz | perl -lane 'my @vals2 = split(/,/, $F[8]); print $#vals2, "\t", $F[8];' | awk '{ print $1; }' | sort | uniq -c
   100 	-1                                                                                                                                                                                                                                                                                                                 
    28 	0
 65232 	25                                                                                                                                                                                                                                                                                                                 

#Edited code in /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGenePerVariant.vcf.gz

[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals2 = split(/,/, $F[8]); print $#vals2, "\t", $F[8];' | awk '{ print $1; }' | sort | uniq -c
   100 	-1                                                                                                                                                                                                                                                                                                                 
    28 	0
 63887 	25                                                                                                                                                                                                                                                                                                                 

[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$R -q -e "Val1 <- 64015 - 100 - 28; print(Val1)"
> Val1 <- 64015 - 100 - 28; print(Val1)
[1] 63887
> 
> 


#NOTE -- REDO LRT runs and ...QCed.DropIBD.bim.wGeneIDs files with correct annovar annotation stuff?
#Potentially using PP2_HVAR in place of LRT since PP2_HVAR is 2 places after PP2_HDIV (and I should be using HDIV over HVAR for my own purposes....but LRT may in fact be PP2_HVAR?)
#The ...line3[len(line3)-19:len(line3)+1]]))) line in /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.py is wrong

[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | grep nonsyn | perl -F, -lane 'print join(",", @F[$#F-25..$#F]);' | head -n 10
B,0.070,0.965,N,0.0,1,P,-2.24,0.287,N,-0.39,0.412,T,-0.957,0.261,T,0.000,T,4.28,0.578,7.695,1,888659,888659,T,C
B,0.414,0.793,N,0.0,1,P,-0.55,0.434,N,0.99,0.360,T,-1.043,0.240,T,0.000,T,2.45,0.456,2.650,1,909238,909238,G,C
B,0.001,3.155E-4,N,0.000,1.000,P,1.265,0.591,L,1.59,0.338,T,-0.903,0.275,T,0.000,T,-2.36,-0.675,9.972,1,949608,949608,G,A
P,0.000,1.000,D,0.792,0.208,N,0.345,0.511,N,3.03,0.284,T,-1.017,0.247,T,0.069,T,3.24,0.535,10.397,1,1163951,1163951,G,A
B,0.000,1.000,D,0.997,0.997,D,0.895,0.559,L,3.01,0.285,T,-0.974,0.257,T,0.009,T,2.21,0.158,6.654,1,1164015,1164015,C,A
B,0.061,0.970,N,1,0.0,N,-1.61,0.342,N,3.29,0.275,T,-0.986,0.254,T,0.000,T,-1.83,-0.628,7.783,1,1164031,1164031,A,G
B,0.004,0.998,N,1.000,0.000,N,1.445,0.607,L,3.16,0.279,T,-1.013,0.247,T,0.017,T,-1.7,-0.587,6.489,1,1164080,1164080,T,C
B,0.117,0.059,N,0.0,1,P,-0.2,0.464,N,-0.34,0.410,T,-0.961,0.261,T,0.000,T,1.41,0.096,10.110,1,1223385,1223385,G,C
B,.,.,.,0.0,1,P,0.55,0.529,N,-0.31,0.409,T,-1.104,0.225,T,0.001,T,1.92,0.219,5.198,1,1226757,1226757,G,A
D,0.128,0.064,N,0.872,0.872,D,4.18,0.844,H,0.23,0.389,T,-0.318,0.421,T,0.592,D,3.26,1.850,12.166,1,1246319,1246319,G,A

#NOTE -- Believe the above problems/NOTE/REDO was based on /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv being done with ljb23 but having the output name as normal. A new version of this file (and associated Annovar output) existed around June 6th when all the result files used, such as ...QCed.DropIBD.bim.wGeneIDs were from April 9th (when traces of another version of ...wAA.Bi.vcf.AnnovarFormat.genome_summary.csv is present too). Also, around June 6th is when P2 was being processed, therefore, I believe the ...wAA.Bi.vcf.AnnovarFormat.genome_summary.csv file was remade during this time period while figuring out P2 stuff, and when I jumped back into things /now/ I used the same filename even though the output now had the ljb23 version of things. 
#In conclusion, the EPACTS results and previous files should be fine. The only thing that's been affected is the prediction annotations sent to Kevin and this QC stuff I've started, which I've gone ahead and redone after rerunning the Annovar code without ljb23

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.AllGenesPerVariant.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.AllGenesPerVariant.GetVCFQCStats.vs1.output 

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGenePerVariant.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output 

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.GetVCFQCStats.vs1.output

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGenePerVariant.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGenePerVariant.vcf.gz | grep exonic | grep -v ncRNA | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGenePerVariant.Exonic.GetVCFQCStats.vs1.output 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGenePerVariant.vcf.gz | grep exonic | grep -v ncRNA | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGenePerVariant.Exonic.GetVCFQCStats.vs1.output

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.PerGene.vs1.py

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGenePerVariant.vcf.gz | grep exonic | grep -v ncRNA | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.PerGene.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGene.Exonic.GetVCFQCStats.PerGene.vs1.output 

#Added header to /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGene.Exonic.GetVCFQCStats.PerGene.vs1.output
#geneID,NumVar,Exonic,ExonicProp,Intronic,IntronicProp,Intergenic,UTR5,UTR3,Nonsynonymous,NonsynonymousPropEx,Synonymous,SynonymousPropEx,Stopgain,StopgainPropEx,Stoploss,StoplossPropEx,MeanDP,stdDP,meanQual,stdQual,TiTvRatio,Transitions,Transversions,meanHaplo,stdHaplo,Singletons,SingletonsProp,Doubletons,DoubletonsProp,Common,CommonProp,Rare,RareProp,PP2_D,PP2_DProp,PP2_P,PP2_PProp,Sift_D,Sift_DProp,LRT_D,LRT_DProp

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGenePerVariant.vcf.gz | grep exonic | grep -v ncRNA | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.PerGene.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.GetVCFQCStats.PerGene.vs1.output

#Added header to /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.GetVCFQCStats.PerGene.vs1.output
#geneID,NumVar,Exonic,ExonicProp,Intronic,IntronicProp,Intergenic,UTR5,UTR3,Nonsynonymous,NonsynonymousPropEx,Synonymous,SynonymousPropEx,Stopgain,StopgainPropEx,Stoploss,StoplossPropEx,MeanDP,stdDP,meanQual,stdQual,TiTvRatio,Transitions,Transversions,meanHaplo,stdHaplo,Singletons,SingletonsProp,Doubletons,DoubletonsProp,Common,CommonProp,Rare,RareProp,PP2_D,PP2_DProp,PP2_P,PP2_PProp,Sift_D,Sift_DProp,LRT_D,LRT_DProp

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.GetVCFQCStats.PerGene.vs1.output | awk -F, '{ print $1, "\t", $18, "\t", $2 }' | sort -gk 3,3 | vi -

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.GetVCFQCStats.PerGene.vs1.output | awk -F, '{ print $1, "\t", $2, "\t", $3, "\t", $10, "\t", $11, "\t", $27, "\t", $29, "\t", $31, "\t", $33, "\t", $18, "\t", $20 }' | sort -rg -k 9,9 | vi -

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs2.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGenePerVariant.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGene.perIndv.vs2.output

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGenePerVariant.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.perIndv.vs2.output

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perGene.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perGene.vs1.py.OLD1 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGenePerVariant.vcf.gz | grep -E '^#|exonic' | grep -v ncRNA | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs2.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGenePerVariant.vcf.gz | grep -E '^#|exonic' | grep -v ncRNA | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs2.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.perIndv.vs2.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.perIndv.vs2.output.justSeroNeg 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.perIndv.vs2.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.perIndv.vs2.output.justSeroHighlyExposed
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.perIndv.vs2.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroPos.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.perIndv.vs2.output.justSeroPos 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.perIndv.vs2.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justRapidVRapid.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.perIndv.vs2.output.justRapidVRapid 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.perIndv.vs2.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSlowVSlow.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.perIndv.vs2.output.justSlowVSlow 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output.justSeroNeg 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output.justSeroHighlyExposed
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroPos.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output.justSeroPos 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justRapidVRapid.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output.justRapidVRapid 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSlowVSlow.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output.justSlowVSlow 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.perIndv.vs2.output.justSeroNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 14.1133144475921 MeanDoubletons: 18.8101983002833 MeanCommon: 4446.35694050992 MeanRare: 425.130311614731 MeanHeterozy: 3053.45325779037 MeanCoverage: 16.107917375798 sdCoverage: 6.19823376737347 Exonic: 334.082152974504 Intronic: 3569.48441926346 Nonsynonymous: 122.872521246459 Synonymous: 208.801699716714 PP2_D: 9.90934844192635 PP2_P: 12.6345609065156 Sift_D: 23.0509915014164 LRT_D: 21.6345609065156"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.perIndv.vs2.output.justSeroHighlyExposed | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 14.2678571428571 MeanDoubletons: 18.9047619047619 MeanCommon: 4446.85119047619 MeanRare: 424.309523809524 MeanHeterozy: 3061.64880952381 MeanCoverage: 16.0151247331096 sdCoverage: 6.35219718685658 Exonic: 334.345238095238 Intronic: 3568.58333333333 Nonsynonymous: 122.714285714286 Synonymous: 209.404761904762 PP2_D: 10.0654761904762 PP2_P: 12.6071428571429 Sift_D: 23.0357142857143 LRT_D: 21.7261904761905"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.perIndv.vs2.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 15.0707547169811 MeanDoubletons: 17.8844339622642 MeanCommon: 4463.65094339623 MeanRare: 419.983490566038 MeanHeterozy: 3078.24528301887 MeanCoverage: 17.0877961848545 sdCoverage: 6.56096341408423 Exonic: 332.719339622641 Intronic: 3579.04716981132 Nonsynonymous: 122.200471698113 Synonymous: 207.785377358491 PP2_D: 9.78301886792453 PP2_P: 12.6768867924528 Sift_D: 22.372641509434 LRT_D: 21.8915094339623"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.perIndv.vs2.output.justRapidVRapid | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))                                                                                  
[1] "MeanSingletons: 15.0172413793103 MeanDoubletons: 15.4655172413793 MeanCommon: 4472.20689655172 MeanRare: 393.655172413793 MeanHeterozy: 3062.53448275862 MeanCoverage: 21.8443147903407 sdCoverage: 8.46501233260818 Exonic: 330.344827586207 Intronic: 3573.39655172414 Nonsynonymous: 121.275862068966 Synonymous: 206.396551724138 PP2_D: 9.39655172413793 PP2_P: 12.6724137931034 Sift_D: 22.0689655172414 LRT_D: 21.3620689655172"                                                                                                                                                                                                          
>
>                                   

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.perIndv.vs2.output.justSlowVSlow | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 15.4157303370787 MeanDoubletons: 18.1685393258427 MeanCommon: 4474 MeanRare: 423.707865168539 MeanHeterozy: 3095.23595505618 MeanCoverage: 17.0228952824631 sdCoverage: 6.77461609846057 Exonic: 333.213483146067 Intronic: 3593.01123595506 Nonsynonymous: 122.52808988764 Synonymous: 207.786516853933 PP2_D: 9.97752808988764 PP2_P: 12.7415730337079 Sift_D: 22.8539325842697 LRT_D: 22.0898876404494"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output.justSeroNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.43909348441926 MeanDoubletons: 2.86685552407932 MeanCommon: 299.28045325779 MeanRare: 35.2549575070822 MeanHeterozy: 219.773371104816 MeanCoverage: 21.9810675566897 sdCoverage: 8.58968008315038 Exonic: 334.082152974504 Intronic: 0 Nonsynonymous: 122.872521246459 Synonymous: 208.801699716714 PP2_D: 9.90934844192635 PP2_P: 12.6345609065156 Sift_D: 22.9206798866856 LRT_D: 21.6345609065156"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output.justSeroHighlyExposed | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.52380952380952 MeanDoubletons: 2.9047619047619 MeanCommon: 298.940476190476 MeanRare: 35.8452380952381 MeanHeterozy: 220.821428571429 MeanCoverage: 21.7223920798614 sdCoverage: 8.74336821131015 Exonic: 334.345238095238 Intronic: 0 Nonsynonymous: 122.714285714286 Synonymous: 209.404761904762 PP2_D: 10.0654761904762 PP2_P: 12.6071428571429 Sift_D: 22.8988095238095 LRT_D: 21.7261904761905"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.53537735849057 MeanDoubletons: 2.54009433962264 MeanCommon: 298.695754716981 MeanRare: 34.4528301886792 MeanHeterozy: 219.007075471698 MeanCoverage: 22.9559819110576 sdCoverage: 9.25012765969131 Exonic: 332.719339622641 Intronic: 0 Nonsynonymous: 122.200471698113 Synonymous: 207.785377358491 PP2_D: 9.78301886792453 PP2_P: 12.6768867924528 Sift_D: 22.2452830188679 LRT_D: 21.8915094339623"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output.justRapidVRapid | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.48275862068966 MeanDoubletons: 2.55172413793103 MeanCommon: 297.724137931034 MeanRare: 32.948275862069 MeanHeterozy: 216.293103448276 MeanCoverage: 29.945459392004 sdCoverage: 12.4184488407268 Exonic: 330.344827586207 Intronic: 0 Nonsynonymous: 121.275862068966 Synonymous: 206.396551724138 PP2_D: 9.39655172413793 PP2_P: 12.6724137931034 Sift_D: 21.9310344827586 LRT_D: 21.3620689655172"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.Exonic.perIndv.vs2.output.justSlowVSlow | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 3.01123595505618 MeanDoubletons: 2.52808988764045 MeanCommon: 298.685393258427 MeanRare: 34.9887640449438 MeanHeterozy: 220.247191011236 MeanCoverage: 22.8268577959015 sdCoverage: 9.21476492286574 Exonic: 333.213483146067 Intronic: 0 Nonsynonymous: 122.52808988764 Synonymous: 207.786516853933 PP2_D: 9.97752808988764 PP2_P: 12.7415730337079 Sift_D: 22.685393258427 LRT_D: 22.0898876404494"
> 
> 

#20150108
#Need to look over and consider whether I need to redo these per individual/per group QC analyses because I wasn't shifting the Annovar annotated .vcf file header to include 'ANNOVAR' for the new, extra column, thus leading to individual ID names being off 

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz | perl -lane 'my @vals2 = split(/,/, $F[8]); print $#vals2, "\t", $F[8];' | awk '{ print $1; }' | sort | uniq -c

[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz | perl -lane 'my @vals2 = split(/,/, $F[8]); print $#vals2, "\t", $F[8];' | awk '{ print $1; }' | sort | uniq -c
    100 -1
    28 0
    74824 25

[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$R -q -e "val1 <- 73261 + 1540 + 21 + 2; print(val1);"
> val1 <- 73261 + 1540 + 21 + 2; print(val1);
[1] 74824
>
>

#Edited code in /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs2.py
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz | perl -lane 'my @vals2 = split(/,/, $F[8]); print $#vals2, "\t", $F[8];' | awk '{ print $1; }' | sort | uniq -c
 
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz | perl -lane 'my @vals2 = split(/,/, $F[8]); print $#vals2, "\t", $F[8];' | awk '{ print $1; }' | sort | uniq -c 
    100 -1
         28 0
	   73261 25

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz

[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz | wc
  64015 50216707 1178830267

[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz | wc  
  65360 51339110 1208205185

[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz | perl -lane 'my @vals2 = split(/,/, $F[8]); print $#vals2, "\t", $F[8];' | awk '{ print $1; }' | sort | uniq -c                                                                       
    100 -1
    28 0 
    65232 25

#Edited code in /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs2.py
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz | perl -lane 'my @vals2 = split(/,/, $F[8]); print $#vals2, "\t", $F[8];' | awk '{ print $1; }' | sort | uniq -c

[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz | perl -lane 'my @vals2 = split(/,/, $F[8]); print $#vals2, "\t", $F[8];' | awk '{ print $1; }' | sort | uniq -c
    100 -1
    28 0
    63887 25

[  michaelt@node4  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$R -q -e "Val1 <- 64015 - 100 - 28; print(Val1)"
> Val1 <- 64015 - 100 - 28; print(Val1)
[1] 63887
> 

#Editing and removing +1 to 'RowLength' for '^#CHROM' block because that header now has the proper number of columns in it and doesn't need to be offset by one manually
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs2.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs3.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs3.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs3.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz | grep -E '^#|exonic' | grep -v ncRNA | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs3.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz | grep -E '^#|exonic' | grep -v ncRNA | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs3.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output.justSeroNeg 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output.justSeroHighlyExposedNeg 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroPos.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output.justSeroPos 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justRapidVRapid.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output.justRapidVRapid 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSlowVSlow.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output.justSlowVSlow 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output.justSeroNeg
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output.justHESN
#Filename too long hence HESN
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroPos.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output.justSeroPos
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justRapidVRapid.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output.justRapidVRapid
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSlowVSlow.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output.justSlowVSlow

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output.justSeroNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))                                                                                  
[1] "MeanSingletons: 14.1133144475921 MeanDoubletons: 18.8101983002833 MeanCommon: 4446.35694050992 MeanRare: 425.130311614731 MeanHeterozy: 3053.45325779037 MeanCoverage: 16.107917375798 sdCoverage: 6.19823376737347 Exonic: 334.082152974504 Intronic: 3569.48441926346 Nonsynonymous: 122.872521246459 Synonymous: 208.801699716714 PP2_D: 9.90934844192635 PP2_P: 12.6345609065156 Sift_D: 23.0509915014164 LRT_D: 21.6345609065156"                                                                                                                                                                                                           
>
>                                                                                                                                                                                                                                                                                                                          

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 15.0707547169811 MeanDoubletons: 17.8844339622642 MeanCommon: 4463.65094339623 MeanRare: 419.983490566038 MeanHeterozy: 3078.24528301887 MeanCoverage: 17.0877961848545 sdCoverage: 6.56096341408423 Exonic: 332.719339622641 Intronic: 3579.04716981132 Nonsynonymous: 122.200471698113 Synonymous: 207.785377358491 PP2_D: 9.78301886792453 PP2_P: 12.6768867924528 Sift_D: 22.372641509434 LRT_D: 21.8915094339623"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output.justSeroHighlyExposedNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 14.2678571428571 MeanDoubletons: 18.9047619047619 MeanCommon: 4446.85119047619 MeanRare: 424.309523809524 MeanHeterozy: 3061.64880952381 MeanCoverage: 16.0151247331096 sdCoverage: 6.35219718685658 Exonic: 334.345238095238 Intronic: 3568.58333333333 Nonsynonymous: 122.714285714286 Synonymous: 209.404761904762 PP2_D: 10.0654761904762 PP2_P: 12.6071428571429 Sift_D: 23.0357142857143 LRT_D: 21.7261904761905"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 15.0707547169811 MeanDoubletons: 17.8844339622642 MeanCommon: 4463.65094339623 MeanRare: 419.983490566038 MeanHeterozy: 3078.24528301887 MeanCoverage: 17.0877961848545 sdCoverage: 6.56096341408423 Exonic: 332.719339622641 Intronic: 3579.04716981132 Nonsynonymous: 122.200471698113 Synonymous: 207.785377358491 PP2_D: 9.78301886792453 PP2_P: 12.6768867924528 Sift_D: 22.372641509434 LRT_D: 21.8915094339623"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output.justRapidVRapid | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 15.0172413793103 MeanDoubletons: 15.4655172413793 MeanCommon: 4472.20689655172 MeanRare: 393.655172413793 MeanHeterozy: 3062.53448275862 MeanCoverage: 21.8443147903407 sdCoverage: 8.46501233260818 Exonic: 330.344827586207 Intronic: 3573.39655172414 Nonsynonymous: 121.275862068966 Synonymous: 206.396551724138 PP2_D: 9.39655172413793 PP2_P: 12.6724137931034 Sift_D: 22.0689655172414 LRT_D: 21.3620689655172"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output.justSlowVSlow | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 15.4157303370787 MeanDoubletons: 18.1685393258427 MeanCommon: 4474 MeanRare: 423.707865168539 MeanHeterozy: 3095.23595505618 MeanCoverage: 17.0228952824631 sdCoverage: 6.77461609846057 Exonic: 333.213483146067 Intronic: 3593.01123595506 Nonsynonymous: 122.52808988764 Synonymous: 207.786516853933 PP2_D: 9.97752808988764 PP2_P: 12.7415730337079 Sift_D: 22.8539325842697 LRT_D: 22.0898876404494"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output.justSeroNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.43909348441926 MeanDoubletons: 2.86685552407932 MeanCommon: 299.28045325779 MeanRare: 35.2549575070822 MeanHeterozy: 219.773371104816 MeanCoverage: 21.9810675566897 sdCoverage: 8.58968008315038 Exonic: 334.082152974504 Intronic: 0 Nonsynonymous: 122.872521246459 Synonymous: 208.801699716714 PP2_D: 9.90934844192635 PP2_P: 12.6345609065156 Sift_D: 22.9206798866856 LRT_D: 21.6345609065156"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.53537735849057 MeanDoubletons: 2.54009433962264 MeanCommon: 298.695754716981 MeanRare: 34.4528301886792 MeanHeterozy: 219.007075471698 MeanCoverage: 22.9559819110576 sdCoverage: 9.25012765969131 Exonic: 332.719339622641 Intronic: 0 Nonsynonymous: 122.200471698113 Synonymous: 207.785377358491 PP2_D: 9.78301886792453 PP2_P: 12.6768867924528 Sift_D: 22.2452830188679 LRT_D: 21.8915094339623"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output.justHESN | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.52380952380952 MeanDoubletons: 2.9047619047619 MeanCommon: 298.940476190476 MeanRare: 35.8452380952381 MeanHeterozy: 220.821428571429 MeanCoverage: 21.7223920798614 sdCoverage: 8.74336821131015 Exonic: 334.345238095238 Intronic: 0 Nonsynonymous: 122.714285714286 Synonymous: 209.404761904762 PP2_D: 10.0654761904762 PP2_P: 12.6071428571429 Sift_D: 22.8988095238095 LRT_D: 21.7261904761905"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.53537735849057 MeanDoubletons: 2.54009433962264 MeanCommon: 298.695754716981 MeanRare: 34.4528301886792 MeanHeterozy: 219.007075471698 MeanCoverage: 22.9559819110576 sdCoverage: 9.25012765969131 Exonic: 332.719339622641 Intronic: 0 Nonsynonymous: 122.200471698113 Synonymous: 207.785377358491 PP2_D: 9.78301886792453 PP2_P: 12.6768867924528 Sift_D: 22.2452830188679 LRT_D: 21.8915094339623"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output.justRapidVRapid | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.48275862068966 MeanDoubletons: 2.55172413793103 MeanCommon: 297.724137931034 MeanRare: 32.948275862069 MeanHeterozy: 216.293103448276 MeanCoverage: 29.945459392004 sdCoverage: 12.4184488407268 Exonic: 330.344827586207 Intronic: 0 Nonsynonymous: 121.275862068966 Synonymous: 206.396551724138 PP2_D: 9.39655172413793 PP2_P: 12.6724137931034 Sift_D: 21.9310344827586 LRT_D: 21.3620689655172"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output.justSlowVSlow | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 3.01123595505618 MeanDoubletons: 2.52808988764045 MeanCommon: 298.685393258427 MeanRare: 34.9887640449438 MeanHeterozy: 220.247191011236 MeanCoverage: 22.8268577959015 sdCoverage: 9.21476492286574 Exonic: 333.213483146067 Intronic: 0 Nonsynonymous: 122.52808988764 Synonymous: 207.786516853933 PP2_D: 9.97752808988764 PP2_P: 12.7415730337079 Sift_D: 22.685393258427 LRT_D: 22.0898876404494"
> 
> 


#20141111
#PC3 check thing, doing a couple of runs of looking at PCs with different color codings for different covariates to see if any of the top 1-5 PCs align with any of the underlying covariates/phenotype labels

vi /home/michaelt/Scripts/R/BasicScript.vs1.R

cp -p /home/michaelt/Scripts/R/BasicScript.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs1.R 

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs1.output.jpeg

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs1.output.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/.

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.parfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.withSNPWeights.parfile

#Adding in output file for 'snpweightoutname:' option and rerunning, but don't want to clobber previous files
#Also used the following command to do so :.,$ s/\/home\/pg\//\/data\/userdata\/pg\//g at the top of this file

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.bim /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.pedsnp

/home/michaelt/Software/EIG5.0.1/bin/smartpca -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.withSNPWeights.parfile

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.AddAnnovarInfoToEigenstratSNPWeights.vs1.py 

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.AddAnnovarInfoToEigenstratSNPWeights.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.withSNPWeights.snpWghts > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.withSNPWeights.snpWghts.wAnnovarAnnotation

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.AddUpAlleleCountsPerIndvGivenChrBPAndVCF.vs1.py

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.withSNPWeights.snpWghts.wAnnovarAnnotation | grep UBR4 | awk '{ print $2, "\t", $3 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.withSNPWeights.snpWghts.wAnnovarAnnotation.UBR4.ChrBP

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.AddUpAlleleCountsPerIndvGivenChrBPAndVCF.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.withSNPWeights.snpWghts.wAnnovarAnnotation.UBR4.ChrBP --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.UBR4.AlleleCounts

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.R

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.UBR4.AlleleCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.output.pt1.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.output.pt2.jpeg

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.output.pt*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/.

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.withSNPWeights.snpWghts.wAnnovarAnnotation | sort -rg -k 6,6 | awk '{ if (($6 > 8) || ($6 < -6.5)) { print $0 } } ' | grep UBR4 | awk '{ print $2, "\t", $3 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.withSNPWeights.snpWghts.wAnnovarAnnotation.UBR4.TopSNPs.ChrBP

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.AddUpAlleleCountsPerIndvGivenChrBPAndVCF.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.withSNPWeights.snpWghts.wAnnovarAnnotation.UBR4.TopSNPs.ChrBP --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.UBR4.TopSNPs.AlleleCounts

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.UBR4.TopSNPs.AlleleCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.output.pt1.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.output.pt2.vs2.jpeg

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.output.pt*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/.

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.AddUpAlleleCountsPerIndvGivenChrBPAndVCF.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.ConvertGenosToACSumPerIndvGivenChrBPAndVCF.vs1.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.ConvertGenosToACSumPerIndvGivenChrBPAndVCF.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.withSNPWeights.snpWghts.wAnnovarAnnotation.UBR4.ChrBP --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.UBR4.perIndvSumAC

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.ConvertGenosToACSumPerIndvGivenChrBPAndVCF.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.withSNPWeights.snpWghts.wAnnovarAnnotation.UBR4.TopSNPs.ChrBP --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.UBR4.TopSNPs.perIndvSumAC

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.UBR4.TopSNPs.AlleleCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.UBR4.TopSNPs.perIndvSumAC /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.output.pt1.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.output.pt2.vs2.jpeg

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.output.pt*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/.

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.ConvertGenosToACSumPerIndvGivenChrBPAndVCF.vs1.py --file1 All --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.perIndvSumAC

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs1.R

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.perIndvSumAC | sort -rg -k 2,2 | perl -lane 'my @vals1 = split(/_/, $F[0]); print join("\t", @vals1), "\t", $F[1]; ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs1.output

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.AddAnnovarInfoToEigenstratSNPWeights.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.AddAnnovarInfoToIndvSumAC.vs1.py 

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.AddAnnovarInfoToIndvSumAC.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs1.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs1.output.wAnnovarAnnotation 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs1.output.wAnnovarAnnotation | awk '{ if ($3 > .9) { print $0 } } ' | grep HEATR1 | awk '{ print $1, "\t", $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs1.output.wAnnovarAnnotation.HEATR1.TopSNPs.ChrBP

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.ConvertGenosToACSumPerIndvGivenChrBPAndVCF.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs1.output.wAnnovarAnnotation.HEATR1.TopSNPs.ChrBP --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.HEATR1.TopSNPs.perIndvSumAC

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.UBR4.TopSNPs.AlleleCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.HEATR1.TopSNPs.perIndvSumAC /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.output.pt1.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.output.pt2.vs2.HEATR1.jpeg

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.output.pt*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/.

#HEATR1 is driving the PC3 results (and UBR4 is driving the PC4 results....which is interesting enough since UBR4 SNPs came out as 'top SNPs' based on SNP-PC eigenvalues whereas HEATR1 came out by directingly looking at correlations between genotypes acorss individuals and PC3 results). Interested in redoing Eigenstrat results with downsampled white non-hispanices to see if get same PC3 stratification, just to identify whether it's the abundance of this ancestry that is producing this ancestry-specific result

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.wHapMap3Data.CEUCHBYRI.parfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.wHapMap3Data.CEUCHBYRI.WhiteNonHispSbsmpl.parfile
#Used the following command to do so :.,$ s/\/home\/pg\//\/data\/userdata\/pg\//g at the top of this file

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race | grep White_non-Hispanic | awk '{ print $1 }' | shuf | perl -lane 'my @vals1 = split(/:/, $F[0]); print $vals1[0];' | head -n 729 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.Ran729White.justFIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.phenoColEdit.ped | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.Ran729White.justFIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.phenoColEdit.WhiteNonHispSbsmpl.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.phenoColEdit.pedind | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.Ran729White.justFIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.phenoColEdit.WhiteNonHispSbsmpl.pedind

rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.pedsnp

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.bim /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.pedsnp

/home/michaelt/Software/EIG5.0.1/bin/smartpca -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.wHapMap3Data.CEUCHBYRI.WhiteNonHispSbsmpl.parfile

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca | cat - <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca | grep -E 'POOL43_10RL|POOL101_11RL|POOL18S_07RL|POOL2S_03RL' | perl -lane '$F[$#F] = "NoPhenotype"; print join("\t", @F); ') >  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race.pt1

#Added in the following four individuals with the tag 'NoPhenotype' at the end manually in the pt1 file
#POOL43_10RL POOL101_11RL POOL18S_07RL POOL2S_03RL

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.HapMap3vs.py --file1 /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.txt --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race.pt2

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race.pt1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race.pt2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race

sed -i 's/American_Indian_or_Alaskan_Native/AIndian_or_Alaskan_Native/g' /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race.plot.1vs2.xtxt

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race.plot.3vs4.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race -c 5:6 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race.plot.5vs6.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race -c 7:8 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race.plot.7vs8.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race -c 9:10 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race.plot.9vs10.xtxt

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca.altrd.race.plot.*pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/. 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.pheno | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.Ran729White.justFIDs | awk '{ print $1, "\t", $2, "\t0\t0\t0\t", $3, "\t", $4, "\t", $5, "\t", $6 } ' | sed 's/-9/NA/g' | perl -lane 'print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped | awk '{ print $1 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped.justFIDs

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.WhiteNonHispSbsmpl.Results.pca --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped.justFIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.justWhite.QCed.WhiteNonHispSbsmpl.10PCs.covar

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.justWhite.QCed.WhiteNonHispSbsmpl.10PCs.covar | awk '{ print $4, "\t", $6, "\t", $7 "\t", $8, "\t", $9, "\t", $10, "\t", $11,  "\t", $12, "\t", $13, "\t", $14,  "\t", $15 }'  | paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped.vs2

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped.vs2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped

#Add line #FAM_ID IND_ID FAT_ID MOT_ID SEX HIVPROG HIVPROGHE AIDS AIDSEXTR RMID PC1 PC2 PC3 PC4 PC5 PC6 PC7 PC8 PC9 PC10 to /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped 

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz --remove /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.Ran729White.justFIDs --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.recode.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.vcf

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.vcf

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.ConvertGenosToACSumPerIndvGivenChrBPAndVCF.vs1.py --file1 All --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.perIndvSumAC

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.R

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.perIndvSumAC 11 | sort -rg -k 2,2 | perl -lane 'my @vals1 = split(/_/, $F[0]); print join("\t", @vals1), "\t", $F[1]; ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.AddAnnovarInfoToIndvSumAC.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz --file2 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.PC1.output.wAnnovarAnnotation 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.perIndvSumAC 12 | sort -rg -k 2,2 | perl -lane 'my @vals1 = split(/_/, $F[0]); print join("\t", @vals1), "\t", $F[1]; ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.AddAnnovarInfoToIndvSumAC.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz --file2 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.PC2.output.wAnnovarAnnotation 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.perIndvSumAC 13 | sort -rg -k 2,2 | perl -lane 'my @vals1 = split(/_/, $F[0]); print join("\t", @vals1), "\t", $F[1]; ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.AddAnnovarInfoToIndvSumAC.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz --file2 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.PC3.output.wAnnovarAnnotation 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.perIndvSumAC 14 | sort -rg -k 2,2 | perl -lane 'my @vals1 = split(/_/, $F[0]); print join("\t", @vals1), "\t", $F[1]; ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.AddAnnovarInfoToIndvSumAC.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz --file2 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.PC4.output.wAnnovarAnnotation 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.perIndvSumAC 15 | sort -rg -k 2,2 | perl -lane 'my @vals1 = split(/_/, $F[0]); print join("\t", @vals1), "\t", $F[1]; ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.AddAnnovarInfoToIndvSumAC.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz --file2 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.PC5.output.wAnnovarAnnotation 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.perIndvSumAC 16 | sort -rg -k 2,2 | perl -lane 'my @vals1 = split(/_/, $F[0]); print join("\t", @vals1), "\t", $F[1]; ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.AddAnnovarInfoToIndvSumAC.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz --file2 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.PC6.output.wAnnovarAnnotation 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.PC3.output.wAnnovarAnnotation | awk '{ if ($3 > .9) { print $0 } } ' | grep HEATR1 | grep -v NA | awk '{ print $1, "\t", $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.PC3.output.wAnnovarAnnotation.HEATR1.TopSNPs.ChrBP

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.ConvertGenosToACSumPerIndvGivenChrBPAndVCF.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.CorrelatePCsWithIndvSumAC.vs2.PC3.output.wAnnovarAnnotation.HEATR1.TopSNPs.ChrBP --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.HEATR1.TopSNPs.perIndvSumAC 

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.AddUpAlleleCountsPerIndvGivenChrBPAndVCF.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.withSNPWeights.snpWghts.wAnnovarAnnotation.UBR4.ChrBP --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.UBR4.AlleleCounts

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.WhiteNonHispSbsmpl.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.UBR4.AlleleCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.WhiteNonHispSbsmpl.HEATR1.TopSNPs.perIndvSumAC /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.output.pt1.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.output.pt2.vs2.WhiteNonHispSbsmpl.HEATR1.jpeg

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PlotPCsWithPhenoCovars.vs2.output.pt*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/.

#20141121 moved back to Samtools mpileup stuff at this point

##20141121 Comparing GATK, Samtool and FreeBayes things
##

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.vcf.gz --min-alleles 2 --max-alleles 2 --remove-indels --mac 2 --geno .95 --hwe .0001 --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.SNP.Bi.raw.mac2.geno95.hwe1e4

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.SNP.Bi.raw.mac2.geno95.hwe1e4.recode.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.SNP.Bi.raw.mac2.geno95.hwe1e4.vcf

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.SNP.Bi.raw.mac2.geno95.hwe1e4.vcf

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.vcf.gz | awk '{ print $1, "\t", $2, "\t", $2, "\t", $4, "\t", $5 }' | grep -v ^# > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.vcf.AnnovarFormat 

perl /home/michaelt/Software/annovar/summarize_annovar.pl /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.vcf.AnnovarFormat /home/michaelt/Software/annovar/ --ver1000g 1000g2012apr --verdbsnp 129 --veresp 6500 --buildver hg19 --remove

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py /home/michaelt/Scripts/Python/DataAnalysis.All.CompareTwoVCFDatasets.vs1.py

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analysis /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons

python /home/michaelt/Scripts/Python/DataAnalysis.All.CompareTwoVCFDatasets.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.gz | grep VCF1 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.VCF1.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.gz | grep VCF2 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.VCF2.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.gz | grep VCF3a | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.VCF3a.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.gz | grep VCF3b | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.VCF3b.gz

[  michaelt@node11  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.gz | grep VCF1 | wc                                                                                                                                                                                                                                                                                          
  59736 55016856 1286393567
[  michaelt@node11  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.gz | grep VCF2 | wc                                                                                                                                                                                                                                                                                          
  10689 9844569 127227690
[  michaelt@node11  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.flt.wAA.SNP.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | wc
  24334 22278124 290567514
[  michaelt@node11  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.gz | grep VCF3a | wc                                                                                                                                                                                                                                                                                         
  13525 12456525 292082160
[  michaelt@node11  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.gz | grep VCF3b | wc
  13525 12456525 163456485

python /home/michaelt/Scripts/Python/DataAnalysis.All.CompareTwoVCFDatasets.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.gz

[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.vcf.gz | wc
7634779 7023888441 88429303651

[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.vcf.gz | wc
3597614 3309696641 39947702864

[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | wc
  73389 67401779 1578124080                                                                                                                                                                                                                                                                                                

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.gz | grep VCF1 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF1.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.gz | grep VCF2 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF2.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.gz | grep VCF3a | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF3a.gz 
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.gz | grep VCF3b | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF3b.gz 

[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF1.gz | wc
  53656 49417176 1155527229
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF2.gz | wc
  3577890 3295236690 39730486110
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF3a.gz | wc
  19605 18056205 422954578
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF3b.gz | wc
  19589 18040690 235002695

python /home/michaelt/Scripts/Python/DataAnalysis.All.CompareTwoVCFDatasets.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.mac2.geno95.hwe1e4.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.gz

[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.mac2.geno95.hwe1e4.vcf.gz | wc
1536107 1413110201 17104123576

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.gz | grep VCF1 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF1.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.gz | grep VCF2 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF2.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.gz | grep VCF3a | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF3a.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.gz | grep VCF3b | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF3b.gz

[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF1.gz | wc
59548 54843708 1283788512
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Poolskb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF2.gz | wcl.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1
1522275 1402015275 16946929753
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Poolskb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF3a.gz | wc.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1
13713 12629673 294687403
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Poolskb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF3b.gz | wc.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1
13713 12629673 164869710


python /home/michaelt/Scripts/Python/DataAnalysis.All.CompareTwoVCFDatasets.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.gz

[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.vcf.gz | wc
 685929 631003829 16352675359

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.gz | grep VCF1 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF1.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.gz | grep VCF2 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF2.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.gz | grep VCF3a | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3a.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.gz | grep VCF3b | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3b.gz

[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF1.gz | wc
2031 1870551 42437025
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF2.gz | wc                                                  
614642 566085282 13944967335
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3a.gz | wc
71230 65602830 1536096407
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3b.gz | wc                                                 
71230 65602830 2411189761

python /home/michaelt/Scripts/Python/DataAnalysis.All.CompareTwoVCFDatasets.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.mac2.geno95.hwe1e4.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.gz

[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.mac2.geno95.hwe1e4.vcf.gz | wc
66408 61044509 2212902879

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.gz | grep VCF1 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF1.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.gz | grep VCF2 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF2.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.gz | grep VCF3a | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3a.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.gz | grep VCF3b | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3b.gz

[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF1.gz | wc
33797 31127037 730017829
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF2.gz | wc
26887 24762927 878150151 
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3a.gz | wc 
39464 36346344 848483837
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3b.gz | wc
39464 36346344 1335105094 

python /home/michaelt/Scripts/Python/DataAnalysis.All.CompareTwoVCFDatasets.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.gz 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.gz | grep VCF1 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF1.gz 
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.gz | grep VCF2 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF2.gz 
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.gz | grep VCF3a | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3a.gz 
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.gz | grep VCF3b | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3b.gz 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF1.gz | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF2.gz | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3a.gz | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3b.gz | wc

[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF1.gz | wc
3309948 3048462108 36712993187
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF2.gz | wc
398325 366857325 11554346684 
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3a.gz | wc 
287547 264830787 3252966933
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3b.gz | wc
287547 264830787 4802026729                                                                                                                                                                                                                                                                                               

python /home/michaelt/Scripts/Python/DataAnalysis.All.CompareTwoVCFDatasets.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.mac2.geno95.hwe1e4.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.mac2.geno95.hwe1e4.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.gz 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.gz | grep VCF1 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF1.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.gz | grep VCF2 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF2.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.gz | grep VCF3a | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3a.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.gz | grep VCF3b | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3b.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF1.gz | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF2.gz | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3a.gz | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3b.gz | wc

[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF1.gz | wc
1513182 1393640622 16838614138
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF2.gz | wc
43545 40104945 1453710147
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3a.gz | wc
22806 21004326 273194418
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3b.gz | wc
22806 21004326 759528440


python /home/michaelt/Scripts/Python/DataAnalysis.All.CompareTwoVCFDatasets.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.SNP.Bi.raw.mac2.geno95.hwe1e4.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.mac2.geno95.hwe1e4.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.gz 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.gz | grep VCF1 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF1.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.gz | grep VCF2 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF2.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.gz | grep VCF3a | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF3a.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.gz | grep VCF3b | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF3b.gz

[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF1.gz | wc
  97386 89692506 2044664843
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF2.gz | wc
  1515271 1395564591 16863362699
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF3a.gz | wc
    20717 19080357 442174357
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF3b.gz | wc
      20717 19080357 248443768


python /home/michaelt/Scripts/Python/DataAnalysis.All.CompareTwoVCFDatasets.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.SNP.Bi.raw.mac2.geno95.hwe1e4.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.mac2.geno95.hwe1e4.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.gz 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.gz | grep VCF1 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF1.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.gz | grep VCF2 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF2.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.gz | grep VCF3a | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3a.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.gz | grep VCF3b | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3b.gz

[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF1.gz | wc
  64450 59358450 1336882052
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF2.gz | wc
    12698 11694858 403174603
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3a.gz | wc
      53653 49414413 1149990084
[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3b.gz | wc
        53653 49414413 1810094831


cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py

#Note -- actually did the /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF1.gz set of files before the /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.VCF1.gz set of files hence the notes about the VCF2/Annovar errors shown below with the 'second' set of files/run-through even though it's technically the first run-through. Keeping the order of the files to match physical order not necessarily 'precise' chronological order in terms of execution here

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.VCF1.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program GATK > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.VCF1.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.VCF1.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program Samtools > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.VCF2.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.VCF1.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program GATK > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.VCF3a.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.VCF1.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program Samtools > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.GATK.Samtools.output.VCF3b.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF1.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program GATK > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF1.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF2.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program Samtools > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF2.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
#Getting the below errors for a few entries from the above code
Error2a -- ChrBP2 not found in hash1 (12_6910468)
Error2a -- ChrBP2 not found in hash1 (2_31816859)
Error2a -- ChrBP2 not found in hash1 (2_31705551)
Error2a -- ChrBP2 not found in hash1 (21_44035903)
Error2a -- ChrBP2 not found in hash1 (10_37412174)
Error2a -- ChrBP2 not found in hash1 (2_31816828)
Error2a -- ChrBP2 not found in hash1 (22_19205449)
Error2a -- ChrBP2 not found in hash1 (X_148956424)
Error2a -- ChrBP2 not found in hash1 (22_19178315)
Error2a -- ChrBP2 not found in hash1 (22_19178311)
Error2a -- ChrBP2 not found in hash1 (3_60190064)
Error2a -- ChrBP2 not found in hash1 (1_142535434)
Error2a -- ChrBP2 not found in hash1 (11_69139801)
Error2a -- ChrBP2 not found in hash1 (2_31816855)
Error2a -- ChrBP2 not found in hash1 (12_6910704)
Error2a -- ChrBP2 not found in hash1 (15_27591200)
Error2a -- ChrBP2 not found in hash1 (15_27591205)
Error2a -- ChrBP2 not found in hash1 (15_27591206)
Error2a -- ChrBP2 not found in hash1 (21_44035904)
Error2a -- ChrBP2 not found in hash1 (22_19178140)
Error2a -- ChrBP2 not found in hash1 (6_157559468)
Error2a -- ChrBP2 not found in hash1 (10_64321064)
Error2a -- ChrBP2 not found in hash1 (10_64321066)
Error2a -- ChrBP2 not found in hash1 (22_19179519)
Error2a -- ChrBP2 not found in hash1 (22_19178274)
Error2a -- ChrBP2 not found in hash1 (22_19205401)
Error2a -- ChrBP2 not found in hash1 (22_19193926)
Error2a -- ChrBP2 not found in hash1 (22_19205397)
Error2a -- ChrBP2 not found in hash1 (22_19200641)
Error2a -- ChrBP2 not found in hash1 (22_19178160)
Error2a -- ChrBP2 not found in hash1 (22_19178163)
Error2a -- ChrBP2 not found in hash1 (22_19178162)
Error2a -- ChrBP2 not found in hash1 (22_19178164)
Error2a -- ChrBP2 not found in hash1 (22_19178167)
Error2a -- ChrBP2 not found in hash1 (22_19178166)
Error2a -- ChrBP2 not found in hash1 (22_19178264)
Error2a -- ChrBP2 not found in hash1 (22_19178268)

#[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/b_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF2.gz | grep 6910468 | perl -lane 'print join("\t", @F[0..9]);'NP_PASSts99_9_wAA_Bi_DropOffTarg_1kb
VCF2    12      6910468 .       N       A       3.57    .       DP=1;AF1=1;AC1=1822;DP4=0,0,0,1;MQ=48;FQ=-20.9  GT:PL:GQ
#[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/b_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF2.gz | grep 19178311 | perl -lane 'print join("\t", @F[0..9]);'NP_PASSts99_9_wAA_Bi_DropOffTarg_1kb
VCF2    22      19178311        .       N       A       999     .       DP=21;VDB=1.273927e-02;AF1=1;AC1=1822;DP4=0,0,13,8;MQ=40;FQ=-23.5       GT:PL:GQ
#[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/b_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF2.gz | grep 37412174 | perl -lane 'print join("\t", @F[0..9]);'P_PASSts99_9_wAA_Bi_DropOffTarg_1kb
VCF2    10      37412174        .       N       G       999     .       DP=9;VDB=2.230037e-02;AF1=1;AC1=1822;DP4=0,0,6,3;MQ=49;FQ=-22.7 GT:PL:GQ
#[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/b_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF2.gz | grep 19178274 | perl -lane 'print join("\t", @F[0..9]);'P_PASSts99_9_wAA_Bi_DropOffTarg_1kb
VCF2    22      19178274        .       N       A       999     .       DP=16;VDB=4.218601e-02;AF1=1;AC1=1822;DP4=0,0,11,4;MQ=37;FQ=-23.1       GT:PL:GQ

#I think it just has to do with the ref allele being N, which is a variant that would have been dropped later in QC process steps anyways -- so shouldn't be a concern to see a few variants dropping out

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF3a.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program GATK > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF3a.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF3b.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program Samtools > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi.output.VCF3b.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF1.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program GATK > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF1.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF2.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program Samtools > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF2.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF3a.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program GATK > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF3a.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF3b.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program Samtools > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF3b.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF1.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program GATK > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF1.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF2.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program FreeBayes > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF2.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3a.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program GATK > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3a.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3b.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program FreeBayes > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3b.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output

/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF1.gz
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF1.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program GATK > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF1.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF2.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program FreeBayes > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF2.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3a.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program GATK > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3a.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3b.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program FreeBayes > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_SNP_PASSts99_9_wAA_Bi_DropOffTarg_1kb_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3b.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF1.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program Samtools > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF1.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF2.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program FreeBayes > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF2.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3a.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program Samtools > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3a.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3b.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program FreeBayes > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi.FreeBayes_w0Flag_SNP_Bi_raw.output.VCF3b.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF1.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program Samtools > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF1.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF2.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program FreeBayes > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF2.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3a.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program Samtools > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3a.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3b.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program FreeBayes > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3b.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output


python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.vcf.AnnovarFormat.genome_summary.csv  --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF1.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program GATK > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF1.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF2.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program Samtools > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF2.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF3a.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program GATK > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF3a.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/Samtools/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Samtools.ChrAll.C50.raw.SNP.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF3b.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program Samtools > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.Samtools_C50_raw_SNP_Bi_mac2_geno95_hwe1e4.output.VCF3b.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output


python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF1.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program GATK > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF1.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF2.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program FreeBayes > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF2.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3a.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program GATK > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3a.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/FreeBayes/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.FB.w0Flag.ChrAll.SNP.Bi.raw.vcf.AnnovarFormat.genome_summary.csv --file2 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3b.gz | perl -lane 'if ($F[0] =~ m/^VCF/) { shift(@F); print join("\t", @F); }') | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.ForVCFComparisons.vs1.py --file1 - --program FreeBayes > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_raw_SNP_Bi_mac2_geno95_hwe1e4.FreeBayes_w0Flag_SNP_Bi_raw_mac2_geno95_hwe1e4.output.VCF3b.wAnnovarForQC.RanGenePerVariant.GetVCFQCStats.vs1.output




~~~~

##EPACTS analyses

#EPACTS Variables Running Through
#Pheno: HIVPROG HIVPROGHE AIDS AIDSEXTR
#Filetypes: All Exonic Exonic.Nonsynonymous Exonic.Nonsynonymous.PP2_PD Exonic.Nonsynonymous.PP2_D Exonic.Nonsynonymous.Sift_D Exonic.Nonsynonymous.LRT_D
#Tests b.collapse b.madsen b.wcnt  skat --skat-adjust skat --skat-o --skat-adjust
#MAF cutoffs: .05 1.0 (e.g. treat all variants the same)

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/MainScript1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/MainScript1.PG_HIVArray1.sh 

vi /data/userdata/pg/michaelt/Data/ALL_MAPPING/MainScript1.PG_HIVArray1.EPACTScommands.sh #Using /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs1.sh instead

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptb.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs1.sh

#R
#install.packages("/home/michaelt/Software/SKAT_0.93.tar.gz", repos=NULL, type="source")
#R-2.0.3
#./configure; make; make check 

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.sh

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.vs2.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS

cp -p DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.sh DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.pt1.sh

cp -p DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.sh DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.pt2.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs

#WithOnlyRMIDs
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID

#Directory /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/
ls -lrt | grep preGATK | grep -v epacts$ | awk '{ print $9 }' | perl -lane 'system("mv $F[0] ExtraFiles/$F[0]");'

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.CollectResults.vs1.sh

#NOTE -- odd thing happening where I can copy and paste the commands from the file and they work, but if I run the script it doesn't work. Doesn't like the \ separating the lines
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.CollectResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs

cp -p DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.CollectResults.vs1.sh DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.CollectResults.vs1.CpyPst.sh


#install.packages("/home/michaelt/Software/SKAT_0.95.tar.gz", repos=NULL, type="source")

#cp -p /home/michaelt/Software/SKAT_0.95.tar.gz /home/michaelt/Software/EPACTS-3.2.4/share/EPACTS/.

#test statement for replacement length vector R error issue: epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --beta 1,1 --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno4.maf1.skat0flat
#epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --beta 1,25 --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno4.maf1.skat0

#Something odd happens where the method, even if calling for optimal, will go to regular SKAT davies method with the Q output instead of the rho output if the marker size is 2 (so 1 or 2 marker size leads to this behavior?)
#Fixed this by editing the code and simply throwing in a 'WeirdResult' in the rho column instead of a 'NA' when catching for rho to have length greater than 1 or equal to 0


cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.vs1.R

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs Pheno1 HIV_Acquisition_justWhite_RMID5PCs_skat0 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.RMID5PCs.Everything.Pheno1.maf1.skat0.QQplot.jpeg

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs Pheno2 HIV_AcquisitionHE_justWhite_RMID5PCs_skat0 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.RMID5PCs.Everything.Pheno2.maf1.skat0.QQplot.jpeg

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs Pheno3 AIDS_Progress_justWhite_RMID5PCs_skat0 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.RMID5PCs.Everything.Pheno3.maf1.skat0.QQplot.jpeg

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs Pheno4 AIDS_ProgressExtr_justWhite_RMID5PCs_skat0 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.RMID5PCs.Everything.Pheno4.maf1.skat0.QQplot.jpeg

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.RMID5PCs.Everything.Pheno*QQplot.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/.

/data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R

cp -p /home/michaelt/Scripts/Perl/RandomOrder.pl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs1.pl

perl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs1.pl --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --seed 10000 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped 

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno1.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno1.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno1.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno1.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno1.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno1.maf1.skat0.perm1 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno2.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno2.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno2.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno2.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno2.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno2.maf1.skat0.perm1 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno3.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno3.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno3.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno3.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno3.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno3.maf1.skat0.perm1 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno4.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno4.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno4.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno4.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno4.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno4.maf1.skat0.perm1 &




#install.packages("/home/michaelt/Software/qvalue_1.1.try2.tar.gz", repos=NULL, type="source")
#mkdir /home/michaelt/Software/R-3.0.1
#mkdir /home/michaelt/Software/R-3.0.1/library
#install.packages("/home/michaelt/Software/qvalue_1.1.try2.tar.gz", repos=NULL, type="source", lib="/home/michaelt/Software/R-3.0.1/library")

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno2.maf1.skat0.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno3.maf1.skat0.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno4.maf1.skat0.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno1.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno1.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno1.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno1.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno1.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno1.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.perm1.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno2.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno2.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno2.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno2.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno2.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno2.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno2.maf1.skat0.perm1.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno3.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno3.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno3.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno3.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno3.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno3.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno3.maf1.skat0.perm1.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno4.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno4.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno4.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno4.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno4.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno4.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno4.maf1.skat0.perm1.wQvals.Summary

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes

#Complex
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.All.Pheno1.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Exonic.Pheno1.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.PP2_D.Pheno2.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.Sift_D.Pheno2.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.LRT_D.Pheno2.maf1.skat0 & 

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.All.Pheno3.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Exonic.Pheno3.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.Pheno3.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.PP2_PD.Pheno3.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.PP2_D.Pheno3.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.Sift_D.Pheno3.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.LRT_D.Pheno3.maf1.skat0 & 

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.All.Pheno4.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Exonic.Pheno4.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.Pheno4.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.PP2_PD.Pheno4.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.PP2_D.Pheno4.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.Sift_D.Pheno4.maf1.skat0 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.ComplexesArray1.RMID5PCs.Nonsynonymous.LRT_D.Pheno4.maf1.skat0 & 


perl /home/michaelt/Scripts/Perl/RandomList.pl --total 10000 --seed 2938324 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.PermutationList.vs1.txt

for i in {1..1000}
do

	SEED1=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.PermutationList.vs1.txt | awk '{ if (NR == '$i') { print $0 } } '`

	perl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs1.pl --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --seed $SEED1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm${i}.ped

#	echo $SEED1

done

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptb.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.Launcher.vs1.sh 

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptb.CreateVCFfilesPerChromosome.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.vs1.sh

#Edit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.Launcher.vs1.sh to go from 1 to 100
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations Exonic.Nonsynonymous HIVPROG
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations Exonic.Nonsynonymous HIVPROGHE
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations Exonic.Nonsynonymous AIDS
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations Exonic.Nonsynonymous AIDSEXTR
#Edit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.Launcher.vs1.sh to go from 101 to 1000
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations Exonic.Nonsynonymous HIVPROG
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations Exonic.Nonsynonymous AIDS

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptb.CreateVCFfilesPerChromosome.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.vs1.sh

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.AvgPvalRanks.vs1.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.AvgPvalRanks.vs1.sh Exonic.Nonsynonymous HIVPROG &
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.AvgPvalRanks.vs1.sh Exonic.Nonsynonymous HIVPROGHE & 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.AvgPvalRanks.vs1.sh Exonic.Nonsynonymous AIDS &
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.AvgPvalRanks.vs1.sh Exonic.Nonsynonymous AIDSEXTR & 

paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | grep -v NA | grep -v CHROM | tail -n 100 | awk '{ print $10 } ') <( sort -r -k 1,1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.AvgPvalRanks.vs1.Exonic.Nonsynonymous.HIVPROG.results ) | vi -
paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | grep -v NA | grep -v CHROM | tail -n 100 | awk '{ print $10 } ') <( sort -r -k 1,1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.AvgPvalRanks.vs1.Exonic.Nonsynonymous.HIVPROGHE.results ) | vi -
paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | grep -v NA | grep -v CHROM | tail -n 100 | awk '{ print $10 } ') <( sort -r -k 1,1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.AvgPvalRanks.vs1.Exonic.Nonsynonymous.AIDS.results ) | vi -
paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | grep -v NA | grep -v CHROM | tail -n 100 | awk '{ print $10 } ') <( sort -r -k 1,1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.AvgPvalRanks.vs1.Exonic.Nonsynonymous.AIDSEXTR.results ) | vi -

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.AvgPvalRanks.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.sh
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.AvgPvalRanks.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.DirectLociPvalComp.vs1.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.sh Exonic.Nonsynonymous HIVPROG 1000 50 &
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.sh Exonic.Nonsynonymous HIVPROG 1000 100 &
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.sh Exonic.Nonsynonymous HIVPROGHE 100 50 & 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.sh Exonic.Nonsynonymous HIVPROGHE 100 100 & 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.sh Exonic.Nonsynonymous AIDS 100 50 &
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.sh Exonic.Nonsynonymous AIDS 100 100 &
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.sh Exonic.Nonsynonymous AIDSEXTR 100 50 & 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.sh Exonic.Nonsynonymous AIDSEXTR 100 100 & 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | grep -v NA | grep -v CHROM | tail -n 50 | awk '{ print $10 } ' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); print(sum(Data1[,1]));" | perl -lane 'if ($F[0] !~ /^\>/) { print join("\t", @F); } ' | awk '{ print $2 }'
1.629732
cat EPACTSFiles/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.Exonic.Nonsynonymous.HIVPROG.top50.results | sort -gr | vi -
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | grep -v NA | grep -v CHROM | tail -n 100 | awk '{ print $10 } ' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); print(sum(Data1[,1]));" | perl -lane 'if ($F[0] !~ /^\>/) { print join("\t", @F); } ' | awk '{ print $2 }'
7.918769
cat EPACTSFiles/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.Exonic.Nonsynonymous.HIVPROG.top100.results | sort -gr | vi -
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | grep -v NA | grep -v CHROM | tail -n 50 | awk '{ print $10 } ' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); print(sum(Data1[,1]));" | perl -lane 'if ($F[0] !~ /^\>/) { print join("\t", @F); } ' | awk '{ print $2 }'
2.278094
cat EPACTSFiles/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.Exonic.Nonsynonymous.HIVPROGHE.top50.results | sort -gr | vi -
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | grep -v NA | grep -v CHROM | tail -n 100 | awk '{ print $10 } ' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); print(sum(Data1[,1]));" | perl -lane 'if ($F[0] !~ /^\>/) { print join("\t", @F); } ' | awk '{ print $2 }'
10.20435
cat EPACTSFiles/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.Exonic.Nonsynonymous.HIVPROGHE.top100.results | sort -gr | vi -
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | grep -v NA | grep -v CHROM | tail -n 50 | awk '{ print $10 } ' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); print(sum(Data1[,1]));" | perl -lane 'if ($F[0] !~ /^\>/) { print join("\t", @F); } ' | awk '{ print $2 }'
3.098894
cat EPACTSFiles/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.Exonic.Nonsynonymous.AIDS.top50.results | sort -gr | vi -
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | grep -v NA | grep -v CHROM | tail -n 100 | awk '{ print $10 } ' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); print(sum(Data1[,1]));" | perl -lane 'if ($F[0] !~ /^\>/) { print join("\t", @F); } ' | awk '{ print $2 }'
14.3855
cat EPACTSFiles/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.Exonic.Nonsynonymous.AIDS.top100.results | sort -gr | vi -
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | grep -v NA | grep -v CHROM | tail -n 50 | awk '{ print $10 } ' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); print(sum(Data1[,1]));" | perl -lane 'if ($F[0] !~ /^\>/) { print join("\t", @F); } ' | awk '{ print $2 }'
4.228986
cat EPACTSFiles/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.Exonic.Nonsynonymous.AIDSEXTR.top50.results | sort -gr | vi -
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | grep -v NA | grep -v CHROM | tail -n 100 | awk '{ print $10 } ' | R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); print(sum(Data1[,1]));" | perl -lane 'if ($F[0] !~ /^\>/) { print join("\t", @F); } ' | awk '{ print $2 }'
19.08815
cat EPACTSFiles/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.CummPvalComp.vs1.Exonic.Nonsynonymous.AIDSEXTR.top100.results | sort -gr | vi -

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.epacts.justGeneIDs

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.epacts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.HIVPROG.maf1.skat0.epacts
ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.HIVPROGHE.maf1.skat0.epacts
ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.epacts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.AIDS.maf1.skat0.epacts
ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.epacts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.AIDSEXTR.maf1.skat0.epacts

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.DirectLociPvalComp.vs1.sh Exonic.Nonsynonymous HIVPROG

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.DirectLociPvalComp.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.DirectLociPvalComp.vs1.pt1.sh
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.DirectLociPvalComp.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.DirectLociPvalComp.vs1.pt2.sh

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/perGeneCollect

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.DirectLociPvalComp.vs1.pt1.sh Exonic.Nonsynonymous HIVPROG 1000 &
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.DirectLociPvalComp.vs1.pt1.sh Exonic.Nonsynonymous HIVPROGHE 100 &
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.DirectLociPvalComp.vs1.pt1.sh Exonic.Nonsynonymous AIDS 100 &
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.DirectLociPvalComp.vs1.pt1.sh Exonic.Nonsynonymous AIDSEXTR 100 &
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.DirectLociPvalComp.vs1.pt2.sh Exonic.Nonsynonymous HIVPROG &
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.DirectLociPvalComp.vs1.pt2.sh Exonic.Nonsynonymous HIVPROGHE &
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.DirectLociPvalComp.vs1.pt2.sh Exonic.Nonsynonymous AIDS &
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.CollectResults.DirectLociPvalComp.vs1.pt2.sh Exonic.Nonsynonymous AIDSEXTR &


cat all the files and grep each locus one at a time and do the p-value perm count thing there

FDR approach?

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.EmpricalValues.vs1.R

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.ForPoster.vs1.R

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.ForPoster.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs Pheno1 HIV_Acquisition_SKAT0 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.RMID5PCs.Everything.Pheno1.maf1.skat0.QQplot.ForPoster.jpeg

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.ForPoster.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs Pheno2 HIV_Acq._HighlyExposedSeroNeg_SKAT0 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.RMID5PCs.Everything.Pheno2.maf1.skat0.QQplot.ForPoster.jpeg

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.ForPoster.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs Pheno3 AIDS_Progression_SKAT0 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.RMID5PCs.Everything.Pheno3.maf1.skat0.QQplot.ForPoster.jpeg

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.ForPoster.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs Pheno4 AIDS_ProgressExtr_justWhite_RMID5PCs_skat0 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.RMID5PCs.Everything.Pheno4.maf1.skat0.QQplot.ForPoster.jpeg

#Doing multiple runs of the original analysis (100x) for each level/data-breakdown in order to get precise estimates of the p-values since there is bootstrapping going on in the small sample-size adjustment within SKAT

#20140506 Note -- did the following command
mv /data/userdata/pg/michaelt/* /data/userdata/pg/michaelt/.
#So retroactively changed all the paths in this file to the /data/userdata/pg/michaelt beginning
#Used the following command to do so :.,$ s/\/home\/pg\//\/data\/userdata\/pg\//g at the top of this file

mkdir Permutations/FakePheno
mv Permutations/* Permutations/FakePheno/.
mkdir Permutations/TruePheno

#QC on the phenotype file(s)
[  michaelt@login2  ~/LabStuff/StephensLab/HIVStuff/PG]$cat Sample_status_list_12-10-13_CT.edited.noMeta.csv | perl -F, -lane 'print join("", @F[11..18]);' | sort | uniq -c
57 00000001
49 00000011
7 00000100
1 00001100
181 00010000
6 00010001
6 00010011
20 00010100
18 00011100
10 00100000
1 00100100
140 00110000
8 00110011
16 00110100
6 00111100
200 01000000
199 10000000
1 hesnlesnlescotherscvrapidrapidvslowslow


cat EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary | grep -E 'All|Exonic' > EPACTSFiles/ForKevin.HIVAcquisition.skat0.wQvals.Summary
cat EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno2.maf1.skat0.wQvals.Summary | grep -E 'All|Exonic' > EPACTSFiles/ForKevin.HIVAcquisitionHighlyExposed.skat0.wQvals.Summary
cat EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno3.maf1.skat0.wQvals.Summary | grep -E 'All|Exonic' > EPACTSFiles/ForKevin.AIDSProgression.skat0.wQvals.Summary
#Note -- manually added the headers to each file then


#20150113
#Coming back in now and finishing up these ideas/results by finalizing the 'top' list of genes first by getting quality point estimates of p-values and then doing permutation runs to get FDR-based thresholds for 'top' genes

perl /home/michaelt/Scripts/Perl/RandomList.pl --total 10000 --seed 5830128 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.MultiRunList.vs1.txt
perl /home/michaelt/Scripts/Perl/RandomList.pl --total 10000 --seed 2938324 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.PermutationList.vs1.txt

for i in {1..1000}
do

	SEED1=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.PermutationList.vs1.txt | awk '{ if (NR == '$i') { print $0 } } '`

	perl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs1.pl --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --seed $SEED1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm${i}.ped

#	echo $SEED1

done

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.EPACTSMultiRun.Launcher.vs2.sh
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.EPACTSMultiRun.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/OLD1.DataAnalysis.GATK.EPACTSMultiRun.Launcher.vs2.sh

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.EPACTSMultiRun.vs2.sh 
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.EPACTSMultiRun.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/OLD1.DataAnalysis.GATK.EPACTSMultiRun.vs2.sh

#Decided to create these scripts using /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.pt*.sh as a base instead of /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.Launcher.vs1.sh but keeping the above files around in case I want to go back and go down that route for another reason (since I started some of the work of modulating the scripts as they would need to be in order to go that way)
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.pt2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt2.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS 

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt2.sh

tar -czvf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/ExonicNonsynonymous/OLD1.AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.try2.runAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/ExonicNonsynonymous/*

rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/ExonicNonsynonymous/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.*

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/ExonicNonsynonymous/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/

tar -cvzf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/OLD1.AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.try2.runAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/*

rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.*

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/*Exonic.Nonsynonymous.Pheno1* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/*Exonic.Nonsynonymous.Pheno2* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/*Exonic.Nonsynonymous.Pheno3* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/*Exonic.Nonsynonymous.Pheno4* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/.

rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/*ind /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/*Makefile
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/*ind /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/*Makefile
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/*ind /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/*Makefile
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/*ind /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/*Makefile

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous

rm -r /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Nonsynonymous

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/*HIVPROG.maf1* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/*HIVPROGHE.maf1* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/*AIDS.maf1* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/*AIDSEXTR.maf1* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/.

rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*Makefile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*ind
tar -cvzf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.HIVPROG.maf1.skat0.permAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*epacts

#This command wasn't needed -- these files had already been removed at some point before now
#rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*Makefile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*ind
tar -cvzf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.HIVPROGHE.maf1.skat0.permAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*epacts

rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*Makefile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*ind
tar -cvzf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.AIDS.maf1.skat0.permAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*epacts

#There were only 100 files present for this phenotype
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*Makefile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*ind
tar -cvzf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.AIDSEXTR.maf1.skat0.permAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*epacts

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/*Pheno1*perm* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/*Pheno2*perm* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/*Pheno3*perm* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/*Pheno4*perm* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/

rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*Makefile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*ind
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*Makefile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*ind
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*Makefile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*ind
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*Makefile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*ind

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run1.epacts.GeneIDList
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run1.epacts.GeneIDList
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run1.epacts.GeneIDList

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm1.epacts.GeneIDList
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm1.epacts.GeneIDList
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm1.epacts.GeneIDList

[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous | wc                                                  474    4802   70376

#Note -- *GeneIDList files for Exonic.Nonsynonymous contain 475 entries (474 genes + 1 'ID' row)

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList TruePheno Pheno1 Exonic.Nonsynonymous 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run1.epacts.GeneIDList TruePheno Pheno2 Exonic.Nonsynonymous 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run1.epacts.GeneIDList TruePheno Pheno3 Exonic.Nonsynonymous 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run1.epacts.GeneIDList TruePheno Pheno4 Exonic.Nonsynonymous 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList FakePheno Pheno1 Exonic.Nonsynonymous 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm1.epacts.GeneIDList FakePheno Pheno2 Exonic.Nonsynonymous 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm1.epacts.GeneIDList FakePheno Pheno3 Exonic.Nonsynonymous 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm1.epacts.GeneIDList FakePheno Pheno4 Exonic.Nonsynonymous 

#NOTE -- realized that for permutations I should be doing 'within-phenotype subset permutations' instead of just permuting everyone at once, so I need to update my permutation script and redo phenotypes 2, 3 and 4. Need to do permutations 'within-subsets' because if these individuals have other aspects for why they were 'chosen' to be these phenotype subsets I want to include that within the framework of the permutation, so that the only question being asked is whether the genetic information is what's causing signifiance. If I permute the entire group at large I'm also testing whether there were certain external factors on top of the genetic factors that caused those individuals to first, be that specific phenotype subset, and then second, to produce significant test results. 

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs1.pl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs2.pl

for i in {6..1000}
do

	SEED1=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.PermutationList.vs1.txt | awk '{ if (NR == '$i') { print $0 } } '`

	perl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs2.pl --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --PhenoColumn 5 --seed $SEED1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Pheno1.perm${i}.ped

#	echo $SEED1

done

~~~
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous]$head -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Pheno1.perm1.ped                                                                                                                                                                                                                    
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL98_01RL     POOL98_01RL     0       0       0       2       NA      NA      NA      1       0.0079  0.0160  0.0720  0.0296  0.0320  -0.0555 -0.0447 0.0014  0.0010  -0.0499                                                                                                                                            
POOL83_08RL     POOL83_08RL     0       0       0       2       NA      NA      NA      8       0.0157  0.0095  -0.0198 -0.0248 -0.0206 -0.0242 0.0153  0.0448  0.0243  0.0067
POOL77_08RL     POOL77_08RL     0       0       0       2       NA      NA      NA      8       0.0143  -0.0026 -0.0279 -0.0273 0.0338  -0.0194 0.0555  0.0160  -0.0053 0.0202                                                                                                                                             
POOL02_03RL     POOL02_03RL     0       0       0       1       1       NA      NA      3       0.0136  0.0057  0.0166  0.0415  -0.0098 0.0027  -0.0120 -0.0008 -0.0187 0.0112
POOL33_03RL     POOL33_03RL     0       0       0       1       2       NA      NA      3       0.0137  0.0162  -0.0260 -0.0403 -0.0116 -0.0230 -0.0308 0.0320  -0.0137 -0.0560                                                                                                                                            
POOL157_04RL    POOL157_04RL    0       0       0       2       NA      NA      NA      4       0.0128  0.0025  -0.0309 0.0374  -0.0126 0.0054  0.0047  0.0157  -0.0291 0.0530
POOL88_02RL     POOL88_02RL     0       0       0       2       2       NA      NA      2       0.0118  0.0045  -0.0299 0.0170  0.0068  -0.0149 -0.0094 -0.0493 -0.0122 -0.0149                                                                                                                                            
POOL109_11RL    POOL109_11RL    0       0       0       2       1       NA      NA      11      0.0160  0.0021  -0.0172 -0.0269 -0.0312 0.0343  0.0326  -0.0096 -0.0062 0.0452
POOL158_08RL    POOL158_08RL    0       0       0       2       2       2       NA      8       0.0106  0.0138  0.0268  -0.0544 0.0063  -0.0188 -0.0086 0.0534  -0.0106 -0.0414                                                                                                                                            
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous]$head -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped                                                                                                                                                                                                                           
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL98_01RL     POOL98_01RL     0       0       0       2       2       2       NA      1       0.0079  0.0160  0.0720  0.0296  0.0320  -0.0555 -0.0447 0.0014  0.0010  -0.0499                                                                                                                                            
POOL83_08RL     POOL83_08RL     0       0       0       2       2       1       NA      8       0.0157  0.0095  -0.0198 -0.0248 -0.0206 -0.0242 0.0153  0.0448  0.0243  0.0067
POOL77_08RL     POOL77_08RL     0       0       0       2       2       NA      NA      8       0.0143  -0.0026 -0.0279 -0.0273 0.0338  -0.0194 0.0555  0.0160  -0.0053 0.0202                                                                                                                                             
POOL02_03RL     POOL02_03RL     0       0       0       1       1       NA      NA      3       0.0136  0.0057  0.0166  0.0415  -0.0098 0.0027  -0.0120 -0.0008 -0.0187 0.0112
POOL33_03RL     POOL33_03RL     0       0       0       1       1       NA      NA      3       0.0137  0.0162  -0.0260 -0.0403 -0.0116 -0.0230 -0.0308 0.0320  -0.0137 -0.0560                                                                                                                                            
POOL157_04RL    POOL157_04RL    0       0       0       2       2       2       2       4       0.0128  0.0025  -0.0309 0.0374  -0.0126 0.0054  0.0047  0.0157  -0.0291 0.0530
POOL88_02RL     POOL88_02RL     0       0       0       2       2       NA      NA      2       0.0118  0.0045  -0.0299 0.0170  0.0068  -0.0149 -0.0094 -0.0493 -0.0122 -0.0149                                                                                                                                            
POOL109_11RL    POOL109_11RL    0       0       0       2       2       NA      NA      11      0.0160  0.0021  -0.0172 -0.0269 -0.0312 0.0343  0.0326  -0.0096 -0.0062 0.0452
POOL158_08RL    POOL158_08RL    0       0       0       2       2       1       NA      8       0.0106  0.0138  0.0268  -0.0544 0.0063  -0.0188 -0.0086 0.0534  -0.0106 -0.0414                                                                                                                                            
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous]$head -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped                                                                                                                                                                                                                                              
#FAM_ID IND_ID FAT_ID MOT_ID SEX HIVPROG HIVPROGHE AIDS AIDSEXTR RMID PC1 PC2 PC3 PC4 PC5 PC6 PC7 PC8 PC9 PC10
POOL98_01RL     POOL98_01RL     0       0       0       1       NA      NA      NA      1        0.0079          0.0160  0.0720          0.0296          0.0320          -0.0555         -0.0447         0.0014          0.0010          -0.0499                                                                           
POOL83_08RL     POOL83_08RL     0       0       0       1       NA      NA      NA      8        0.0157          0.0095  -0.0198         -0.0248         -0.0206         -0.0242         0.0153          0.0448          0.0243          0.0067
POOL77_08RL     POOL77_08RL     0       0       0       1       NA      NA      NA      8        0.0143          -0.0026         -0.0279         -0.0273         0.0338          -0.0194         0.0555          0.0160          -0.0053         0.0202                                                                    
POOL02_03RL     POOL02_03RL     0       0       0       1       1       NA      NA      3        0.0136          0.0057  0.0166          0.0415          -0.0098         0.0027          -0.0120         -0.0008         -0.0187         0.0112
POOL33_03RL     POOL33_03RL     0       0       0       2       2       NA      NA      3        0.0137          0.0162  -0.0260         -0.0403         -0.0116         -0.0230         -0.0308         0.0320          -0.0137         -0.0560                                                                           
POOL157_04RL    POOL157_04RL    0       0       0       1       NA      NA      NA      4        0.0128          0.0025  -0.0309         0.0374          -0.0126         0.0054          0.0047          0.0157          -0.0291         0.0530
POOL88_02RL     POOL88_02RL     0       0       0       2       2       NA      NA      2        0.0118          0.0045  -0.0299         0.0170          0.0068          -0.0149         -0.0094         -0.0493         -0.0122         -0.0149                                                                           
POOL109_11RL    POOL109_11RL    0       0       0       1       1       NA      NA      11       0.0160          0.0021  -0.0172         -0.0269         -0.0312         0.0343          0.0326          -0.0096         -0.0062         0.0452
POOL158_08RL    POOL158_08RL    0       0       0       2       2       2       NA      8        0.0106          0.0138  0.0268          -0.0544         0.0063          -0.0188         -0.0086         0.0534          -0.0106         -0.0414                                                                           
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous]$head -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Pheno1.perm2.ped                                                                                                                                                                                                                    
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL98_01RL     POOL98_01RL     0       0       0       2       NA      NA      NA      1       0.0079  0.0160  0.0720  0.0296  0.0320  -0.0555 -0.0447 0.0014  0.0010  -0.0499                                                                                                                                            
POOL83_08RL     POOL83_08RL     0       0       0       1       NA      NA      NA      8       0.0157  0.0095  -0.0198 -0.0248 -0.0206 -0.0242 0.0153  0.0448  0.0243  0.0067
POOL77_08RL     POOL77_08RL     0       0       0       1       NA      NA      NA      8       0.0143  -0.0026 -0.0279 -0.0273 0.0338  -0.0194 0.0555  0.0160  -0.0053 0.0202                                                                                                                                             
POOL02_03RL     POOL02_03RL     0       0       0       1       1       NA      NA      3       0.0136  0.0057  0.0166  0.0415  -0.0098 0.0027  -0.0120 -0.0008 -0.0187 0.0112
POOL33_03RL     POOL33_03RL     0       0       0       2       2       NA      NA      3       0.0137  0.0162  -0.0260 -0.0403 -0.0116 -0.0230 -0.0308 0.0320  -0.0137 -0.0560                                                                                                                                            
POOL157_04RL    POOL157_04RL    0       0       0       2       NA      NA      NA      4       0.0128  0.0025  -0.0309 0.0374  -0.0126 0.0054  0.0047  0.0157  -0.0291 0.0530
POOL88_02RL     POOL88_02RL     0       0       0       1       2       NA      NA      2       0.0118  0.0045  -0.0299 0.0170  0.0068  -0.0149 -0.0094 -0.0493 -0.0122 -0.0149                                                                                                                                            
POOL109_11RL    POOL109_11RL    0       0       0       2       1       NA      NA      11      0.0160  0.0021  -0.0172 -0.0269 -0.0312 0.0343  0.0326  -0.0096 -0.0062 0.0452
POOL158_08RL    POOL158_08RL    0       0       0       2       2       2       NA      8       0.0106  0.0138  0.0268  -0.0544 0.0063  -0.0188 -0.0086 0.0534  -0.0106 -0.0414                                                                                                                                            
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous]$head -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm2.ped                                                                                                                                                                                                                           
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL98_01RL     POOL98_01RL     0       0       0       2       2       NA      NA      1       0.0079  0.0160  0.0720  0.0296  0.0320  -0.0555 -0.0447 0.0014  0.0010  -0.0499                                                                                                                                            
POOL83_08RL     POOL83_08RL     0       0       0       1       NA      NA      NA      8       0.0157  0.0095  -0.0198 -0.0248 -0.0206 -0.0242 0.0153  0.0448  0.0243  0.0067
POOL77_08RL     POOL77_08RL     0       0       0       1       NA      NA      NA      8       0.0143  -0.0026 -0.0279 -0.0273 0.0338  -0.0194 0.0555  0.0160  -0.0053 0.0202                                                                                                                                             
POOL02_03RL     POOL02_03RL     0       0       0       1       NA      NA      NA      3       0.0136  0.0057  0.0166  0.0415  -0.0098 0.0027  -0.0120 -0.0008 -0.0187 0.0112
POOL33_03RL     POOL33_03RL     0       0       0       2       2       2       NA      3       0.0137  0.0162  -0.0260 -0.0403 -0.0116 -0.0230 -0.0308 0.0320  -0.0137 -0.0560                                                                                                                                            
POOL157_04RL    POOL157_04RL    0       0       0       2       2       1       1       4       0.0128  0.0025  -0.0309 0.0374  -0.0126 0.0054  0.0047  0.0157  -0.0291 0.0530
POOL88_02RL     POOL88_02RL     0       0       0       1       NA      NA      NA      2       0.0118  0.0045  -0.0299 0.0170  0.0068  -0.0149 -0.0094 -0.0493 -0.0122 -0.0149                                                                                                                                            
POOL109_11RL    POOL109_11RL    0       0       0       2       2       NA      NA      11      0.0160  0.0021  -0.0172 -0.0269 -0.0312 0.0343  0.0326  -0.0096 -0.0062 0.0452
POOL158_08RL    POOL158_08RL    0       0       0       2       2       NA      NA      8       0.0106  0.0138  0.0268  -0.0544 0.0063  -0.0188 -0.0086 0.0534  -0.0106 -0.0414                                                                                                                                            
~~~

for i in {1..1000}
do

	SEED1=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.PermutationList.vs1.txt | awk '{ if (NR == '$i') { print $0 } } '`

	perl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs2.pl --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --PhenoColumn 6 --seed $SEED1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Pheno2.perm${i}.ped

#	echo $SEED1

done

for i in {1..1000}
do

	SEED1=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.PermutationList.vs1.txt | awk '{ if (NR == '$i') { print $0 } } '`

	perl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs2.pl --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --PhenoColumn 7 --seed $SEED1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Pheno3.perm${i}.ped

#	echo $SEED1

done

for i in {1..1000}
do

	SEED1=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.PermutationList.vs1.txt | awk '{ if (NR == '$i') { print $0 } } '`

	perl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs2.pl --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --PhenoColumn 8 --seed $SEED1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Pheno4.perm${i}.ped

#	echo $SEED1

done

#This was wrong, I need to permute within-subsets not within specific columns, redoing the code but keeping these old commands in case they are useful at some other timepoint
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Pheno1.perm*.ped
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Pheno2.perm*.ped
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Pheno3.perm*.ped
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Pheno4.perm*.ped

tar -cvzf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.permAll.ped.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm*[!vs3].ped

rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm*.ped

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs2.pl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs3.pl 

for i in {1..1000}
do

	SEED1=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.PermutationList.vs1.txt | awk '{ if (NR == '$i') { print $0 } } '`

#	perl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs3.pl --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --PhenoColumn 5 --seed $SEED1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm${i}.vs3.ped
	perl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs3.pl --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --seed $SEED1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm${i}.vs3.ped

#	echo $SEED1

done

~~~
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous]$head -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.vs3.ped 
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL98_01RL     POOL98_01RL     0       0       0       2       NA      NA      NA      1       0.0079  0.0160  0.0720  0.0296  0.0320  -0.0555 -0.0447 0.0014  0.0010  -0.0499
POOL83_08RL     POOL83_08RL     0       0       0       2       NA      NA      NA      8       0.0157  0.0095  -0.0198 -0.0248 -0.0206 -0.0242 0.0153  0.0448  0.0243  0.0067
POOL77_08RL     POOL77_08RL     0       0       0       2       NA      NA      NA      8       0.0143  -0.0026 -0.0279 -0.0273 0.0338  -0.0194 0.0555  0.0160  -0.0053 0.0202
POOL02_03RL     POOL02_03RL     0       0       0       1       2       NA      NA      3       0.0136  0.0057  0.0166  0.0415  -0.0098 0.0027  -0.0120 -0.0008 -0.0187 0.0112
POOL33_03RL     POOL33_03RL     0       0       0       1       1       NA      NA      3       0.0137  0.0162  -0.0260 -0.0403 -0.0116 -0.0230 -0.0308 0.0320  -0.0137 -0.0560
POOL157_04RL    POOL157_04RL    0       0       0       2       NA      NA      NA      4       0.0128  0.0025  -0.0309 0.0374  -0.0126 0.0054  0.0047  0.0157  -0.0291 0.0530
POOL88_02RL     POOL88_02RL     0       0       0       2       1       NA      NA      2       0.0118  0.0045  -0.0299 0.0170  0.0068  -0.0149 -0.0094 -0.0493 -0.0122 -0.0149
POOL109_11RL    POOL109_11RL    0       0       0       2       2       NA      NA      11      0.0160  0.0021  -0.0172 -0.0269 -0.0312 0.0343  0.0326  -0.0096 -0.0062 0.0452
POOL158_08RL    POOL158_08RL    0       0       0       2       2       1       NA      8       0.0106  0.0138  0.0268  -0.0544 0.0063  -0.0188 -0.0086 0.0534  -0.0106 -0.0414
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous]$head -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm2.vs3.ped 
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL98_01RL     POOL98_01RL     0       0       0       2       NA      NA      NA      1       0.0079  0.0160  0.0720  0.0296  0.0320  -0.0555 -0.0447 0.0014  0.0010  -0.0499
POOL83_08RL     POOL83_08RL     0       0       0       1       NA      NA      NA      8       0.0157  0.0095  -0.0198 -0.0248 -0.0206 -0.0242 0.0153  0.0448  0.0243  0.0067
POOL77_08RL     POOL77_08RL     0       0       0       1       NA      NA      NA      8       0.0143  -0.0026 -0.0279 -0.0273 0.0338  -0.0194 0.0555  0.0160  -0.0053 0.0202
POOL02_03RL     POOL02_03RL     0       0       0       1       1       NA      NA      3       0.0136  0.0057  0.0166  0.0415  -0.0098 0.0027  -0.0120 -0.0008 -0.0187 0.0112
POOL33_03RL     POOL33_03RL     0       0       0       2       1       NA      NA      3       0.0137  0.0162  -0.0260 -0.0403 -0.0116 -0.0230 -0.0308 0.0320  -0.0137 -0.0560
POOL157_04RL    POOL157_04RL    0       0       0       2       NA      NA      NA      4       0.0128  0.0025  -0.0309 0.0374  -0.0126 0.0054  0.0047  0.0157  -0.0291 0.0530
POOL88_02RL     POOL88_02RL     0       0       0       1       2       NA      NA      2       0.0118  0.0045  -0.0299 0.0170  0.0068  -0.0149 -0.0094 -0.0493 -0.0122 -0.0149
POOL109_11RL    POOL109_11RL    0       0       0       2       2       NA      NA      11      0.0160  0.0021  -0.0172 -0.0269 -0.0312 0.0343  0.0326  -0.0096 -0.0062 0.0452
POOL158_08RL    POOL158_08RL    0       0       0       2       2       2       NA      8       0.0106  0.0138  0.0268  -0.0544 0.0063  -0.0188 -0.0086 0.0534  -0.0106 -0.0414
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous]$head -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm3.vs3.ped 
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL98_01RL     POOL98_01RL     0       0       0       1       NA      NA      NA      1       0.0079  0.0160  0.0720  0.0296  0.0320  -0.0555 -0.0447 0.0014  0.0010  -0.0499
POOL83_08RL     POOL83_08RL     0       0       0       1       NA      NA      NA      8       0.0157  0.0095  -0.0198 -0.0248 -0.0206 -0.0242 0.0153  0.0448  0.0243  0.0067
POOL77_08RL     POOL77_08RL     0       0       0       1       NA      NA      NA      8       0.0143  -0.0026 -0.0279 -0.0273 0.0338  -0.0194 0.0555  0.0160  -0.0053 0.0202
POOL02_03RL     POOL02_03RL     0       0       0       1       1       NA      NA      3       0.0136  0.0057  0.0166  0.0415  -0.0098 0.0027  -0.0120 -0.0008 -0.0187 0.0112
POOL33_03RL     POOL33_03RL     0       0       0       2       2       NA      NA      3       0.0137  0.0162  -0.0260 -0.0403 -0.0116 -0.0230 -0.0308 0.0320  -0.0137 -0.0560
POOL157_04RL    POOL157_04RL    0       0       0       1       NA      NA      NA      4       0.0128  0.0025  -0.0309 0.0374  -0.0126 0.0054  0.0047  0.0157  -0.0291 0.0530
POOL88_02RL     POOL88_02RL     0       0       0       1       1       NA      NA      2       0.0118  0.0045  -0.0299 0.0170  0.0068  -0.0149 -0.0094 -0.0493 -0.0122 -0.0149
POOL109_11RL    POOL109_11RL    0       0       0       1       2       NA      NA      11      0.0160  0.0021  -0.0172 -0.0269 -0.0312 0.0343  0.0326  -0.0096 -0.0062 0.0452
POOL158_08RL    POOL158_08RL    0       0       0       2       2       2       NA      8       0.0106  0.0138  0.0268  -0.0544 0.0063  -0.0188 -0.0086 0.0534  -0.0106 -0.0414
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous]$head -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm4.vs3.ped 
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL98_01RL     POOL98_01RL     0       0       0       2       NA      NA      NA      1       0.0079  0.0160  0.0720  0.0296  0.0320  -0.0555 -0.0447 0.0014  0.0010  -0.0499
POOL83_08RL     POOL83_08RL     0       0       0       2       NA      NA      NA      8       0.0157  0.0095  -0.0198 -0.0248 -0.0206 -0.0242 0.0153  0.0448  0.0243  0.0067
POOL77_08RL     POOL77_08RL     0       0       0       2       NA      NA      NA      8       0.0143  -0.0026 -0.0279 -0.0273 0.0338  -0.0194 0.0555  0.0160  -0.0053 0.0202
POOL02_03RL     POOL02_03RL     0       0       0       2       2       NA      NA      3       0.0136  0.0057  0.0166  0.0415  -0.0098 0.0027  -0.0120 -0.0008 -0.0187 0.0112
POOL33_03RL     POOL33_03RL     0       0       0       1       2       NA      NA      3       0.0137  0.0162  -0.0260 -0.0403 -0.0116 -0.0230 -0.0308 0.0320  -0.0137 -0.0560
POOL157_04RL    POOL157_04RL    0       0       0       2       NA      NA      NA      4       0.0128  0.0025  -0.0309 0.0374  -0.0126 0.0054  0.0047  0.0157  -0.0291 0.0530
POOL88_02RL     POOL88_02RL     0       0       0       2       2       NA      NA      2       0.0118  0.0045  -0.0299 0.0170  0.0068  -0.0149 -0.0094 -0.0493 -0.0122 -0.0149
POOL109_11RL    POOL109_11RL    0       0       0       1       2       NA      NA      11      0.0160  0.0021  -0.0172 -0.0269 -0.0312 0.0343  0.0326  -0.0096 -0.0062 0.0452
POOL158_08RL    POOL158_08RL    0       0       0       1       2       1       NA      8       0.0106  0.0138  0.0268  -0.0544 0.0063  -0.0188 -0.0086 0.0534  -0.0106 -0.0414
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous]$head -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm5.vs3.ped 
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL98_01RL     POOL98_01RL     0       0       0       1       NA      NA      NA      1       0.0079  0.0160  0.0720  0.0296  0.0320  -0.0555 -0.0447 0.0014  0.0010  -0.0499
POOL83_08RL     POOL83_08RL     0       0       0       2       NA      NA      NA      8       0.0157  0.0095  -0.0198 -0.0248 -0.0206 -0.0242 0.0153  0.0448  0.0243  0.0067
POOL77_08RL     POOL77_08RL     0       0       0       1       NA      NA      NA      8       0.0143  -0.0026 -0.0279 -0.0273 0.0338  -0.0194 0.0555  0.0160  -0.0053 0.0202
POOL02_03RL     POOL02_03RL     0       0       0       2       1       NA      NA      3       0.0136  0.0057  0.0166  0.0415  -0.0098 0.0027  -0.0120 -0.0008 -0.0187 0.0112
POOL33_03RL     POOL33_03RL     0       0       0       1       2       NA      NA      3       0.0137  0.0162  -0.0260 -0.0403 -0.0116 -0.0230 -0.0308 0.0320  -0.0137 -0.0560
POOL157_04RL    POOL157_04RL    0       0       0       2       NA      NA      NA      4       0.0128  0.0025  -0.0309 0.0374  -0.0126 0.0054  0.0047  0.0157  -0.0291 0.0530
POOL88_02RL     POOL88_02RL     0       0       0       1       1       NA      NA      2       0.0118  0.0045  -0.0299 0.0170  0.0068  -0.0149 -0.0094 -0.0493 -0.0122 -0.0149
POOL109_11RL    POOL109_11RL    0       0       0       1       2       NA      NA      11      0.0160  0.0021  -0.0172 -0.0269 -0.0312 0.0343  0.0326  -0.0096 -0.0062 0.0452
POOL158_08RL    POOL158_08RL    0       0       0       2       2       1       NA      8       0.0106  0.0138  0.0268  -0.0544 0.0063  -0.0188 -0.0086 0.0534  -0.0106 -0.0414
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous]$tail -n 150 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.vs3.ped | head -n 10
POOL103_11RL    POOL103_11RL    0       0       0       1       2       NA      NA      11      0.0122  0.0064  0.0138  0.0237  0.0555  -0.0011 0.0001  0.0305  -0.0344 -0.0242
POOL81_01RL     POOL81_01RL     0       0       0       2       NA      NA      NA      1       0.0130  0.0086  -0.0249 -0.0269 -0.0324 -0.0338 -0.0382 0.0230  -0.0032 -0.0139
POOL19_04RL     POOL19_04RL     0       0       0       1       2       NA      NA      4       0.0145  0.0030  -0.0206 -0.0074 0.0485  -0.0191 -0.0113 0.0336  0.0099  0.0178
POOL13_09RL     POOL13_09RL     0       0       0       2       2       NA      NA      9       0.0094  0.0054  0.0615  0.0461  -0.0008 -0.0113 0.0540  0.0077  0.0227  0.0199
POOL14S_01RL    POOL14S_01RL    0       0       0       1       1       2       1       1       0.0115  0.0025  -0.0258 0.0321  0.0069  0.0175  -0.0102 0.0213  0.0161  -0.0020
POOL95_02RL     POOL95_02RL     0       0       0       1       2       NA      NA      2       0.0183  0.0126  -0.0216 -0.0514 0.0546  -0.0125 -0.0034 0.0159  -0.0034 0.0221
POOL94_05RL     POOL94_05RL     0       0       0       1       2       NA      NA      5       0.0116  0.0147  -0.0302 -0.0024 0.0043  0.0400  0.0272  -0.0212 0.0052  0.0120
POOL64_08RL     POOL64_08RL     0       0       0       1       NA      NA      NA      8       0.0131  0.0066  0.0177  -0.0510 -0.0403 -0.0053 0.0177  0.0024  -0.0263 0.0119
POOL116_06RL    POOL116_06RL    0       0       0       1       NA      NA      NA      6       0.0141  0.0072  -0.0259 -0.0046 0.0272  0.0067  -0.0147 0.0317  0.0193  0.0015
POOL90_10RL     POOL90_10RL     0       0       0       2       1       NA      NA      10      0.0148  0.0135  0.0242  -0.0202 -0.0293 0.0450  -0.0129 0.0227  0.0005  0.0110
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous]$tail -n 150 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm2.vs3.ped | head -n 10
POOL103_11RL    POOL103_11RL    0       0       0       2       2       NA      NA      11      0.0122  0.0064  0.0138  0.0237  0.0555  -0.0011 0.0001  0.0305  -0.0344 -0.0242
POOL81_01RL     POOL81_01RL     0       0       0       1       NA      NA      NA      1       0.0130  0.0086  -0.0249 -0.0269 -0.0324 -0.0338 -0.0382 0.0230  -0.0032 -0.0139
POOL19_04RL     POOL19_04RL     0       0       0       1       2       NA      NA      4       0.0145  0.0030  -0.0206 -0.0074 0.0485  -0.0191 -0.0113 0.0336  0.0099  0.0178
POOL13_09RL     POOL13_09RL     0       0       0       1       2       NA      NA      9       0.0094  0.0054  0.0615  0.0461  -0.0008 -0.0113 0.0540  0.0077  0.0227  0.0199
POOL14S_01RL    POOL14S_01RL    0       0       0       2       1       2       1       1       0.0115  0.0025  -0.0258 0.0321  0.0069  0.0175  -0.0102 0.0213  0.0161  -0.0020
POOL95_02RL     POOL95_02RL     0       0       0       1       2       NA      NA      2       0.0183  0.0126  -0.0216 -0.0514 0.0546  -0.0125 -0.0034 0.0159  -0.0034 0.0221
POOL94_05RL     POOL94_05RL     0       0       0       2       2       NA      NA      5       0.0116  0.0147  -0.0302 -0.0024 0.0043  0.0400  0.0272  -0.0212 0.0052  0.0120
POOL64_08RL     POOL64_08RL     0       0       0       2       NA      NA      NA      8       0.0131  0.0066  0.0177  -0.0510 -0.0403 -0.0053 0.0177  0.0024  -0.0263 0.0119
POOL116_06RL    POOL116_06RL    0       0       0       2       NA      NA      NA      6       0.0141  0.0072  -0.0259 -0.0046 0.0272  0.0067  -0.0147 0.0317  0.0193  0.0015
POOL90_10RL     POOL90_10RL     0       0       0       1       1       NA      NA      10      0.0148  0.0135  0.0242  -0.0202 -0.0293 0.0450  -0.0129 0.0227  0.0005  0.0110
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous]$tail -n 150 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm3.vs3.ped | head -n 10
POOL103_11RL    POOL103_11RL    0       0       0       2       1       NA      NA      11      0.0122  0.0064  0.0138  0.0237  0.0555  -0.0011 0.0001  0.0305  -0.0344 -0.0242
POOL81_01RL     POOL81_01RL     0       0       0       1       NA      NA      NA      1       0.0130  0.0086  -0.0249 -0.0269 -0.0324 -0.0338 -0.0382 0.0230  -0.0032 -0.0139
POOL19_04RL     POOL19_04RL     0       0       0       2       2       NA      NA      4       0.0145  0.0030  -0.0206 -0.0074 0.0485  -0.0191 -0.0113 0.0336  0.0099  0.0178
POOL13_09RL     POOL13_09RL     0       0       0       1       2       NA      NA      9       0.0094  0.0054  0.0615  0.0461  -0.0008 -0.0113 0.0540  0.0077  0.0227  0.0199
POOL14S_01RL    POOL14S_01RL    0       0       0       1       2       2       1       1       0.0115  0.0025  -0.0258 0.0321  0.0069  0.0175  -0.0102 0.0213  0.0161  -0.0020
POOL95_02RL     POOL95_02RL     0       0       0       2       2       NA      NA      2       0.0183  0.0126  -0.0216 -0.0514 0.0546  -0.0125 -0.0034 0.0159  -0.0034 0.0221
POOL94_05RL     POOL94_05RL     0       0       0       1       2       NA      NA      5       0.0116  0.0147  -0.0302 -0.0024 0.0043  0.0400  0.0272  -0.0212 0.0052  0.0120
POOL64_08RL     POOL64_08RL     0       0       0       2       NA      NA      NA      8       0.0131  0.0066  0.0177  -0.0510 -0.0403 -0.0053 0.0177  0.0024  -0.0263 0.0119
POOL116_06RL    POOL116_06RL    0       0       0       1       NA      NA      NA      6       0.0141  0.0072  -0.0259 -0.0046 0.0272  0.0067  -0.0147 0.0317  0.0193  0.0015
POOL90_10RL     POOL90_10RL     0       0       0       1       2       NA      NA      10      0.0148  0.0135  0.0242  -0.0202 -0.0293 0.0450  -0.0129 0.0227  0.0005  0.0110
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous]$tail -n 150 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm4.vs3.ped | head -n 10
POOL103_11RL    POOL103_11RL    0       0       0       1       2       NA      NA      11      0.0122  0.0064  0.0138  0.0237  0.0555  -0.0011 0.0001  0.0305  -0.0344 -0.0242
POOL81_01RL     POOL81_01RL     0       0       0       1       NA      NA      NA      1       0.0130  0.0086  -0.0249 -0.0269 -0.0324 -0.0338 -0.0382 0.0230  -0.0032 -0.0139
POOL19_04RL     POOL19_04RL     0       0       0       2       2       NA      NA      4       0.0145  0.0030  -0.0206 -0.0074 0.0485  -0.0191 -0.0113 0.0336  0.0099  0.0178
POOL13_09RL     POOL13_09RL     0       0       0       2       1       NA      NA      9       0.0094  0.0054  0.0615  0.0461  -0.0008 -0.0113 0.0540  0.0077  0.0227  0.0199
POOL14S_01RL    POOL14S_01RL    0       0       0       1       1       2       2       1       0.0115  0.0025  -0.0258 0.0321  0.0069  0.0175  -0.0102 0.0213  0.0161  -0.0020
POOL95_02RL     POOL95_02RL     0       0       0       1       1       NA      NA      2       0.0183  0.0126  -0.0216 -0.0514 0.0546  -0.0125 -0.0034 0.0159  -0.0034 0.0221
POOL94_05RL     POOL94_05RL     0       0       0       2       1       NA      NA      5       0.0116  0.0147  -0.0302 -0.0024 0.0043  0.0400  0.0272  -0.0212 0.0052  0.0120
POOL64_08RL     POOL64_08RL     0       0       0       1       NA      NA      NA      8       0.0131  0.0066  0.0177  -0.0510 -0.0403 -0.0053 0.0177  0.0024  -0.0263 0.0119
POOL116_06RL    POOL116_06RL    0       0       0       2       NA      NA      NA      6       0.0141  0.0072  -0.0259 -0.0046 0.0272  0.0067  -0.0147 0.0317  0.0193  0.0015
POOL90_10RL     POOL90_10RL     0       0       0       1       1       NA      NA      10      0.0148  0.0135  0.0242  -0.0202 -0.0293 0.0450  -0.0129 0.0227  0.0005  0.0110
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous]$tail -n 150 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm5.vs3.ped | head -n 10
POOL103_11RL    POOL103_11RL    0       0       0       1       2       NA      NA      11      0.0122  0.0064  0.0138  0.0237  0.0555  -0.0011 0.0001  0.0305  -0.0344 -0.0242
POOL81_01RL     POOL81_01RL     0       0       0       1       NA      NA      NA      1       0.0130  0.0086  -0.0249 -0.0269 -0.0324 -0.0338 -0.0382 0.0230  -0.0032 -0.0139
POOL19_04RL     POOL19_04RL     0       0       0       2       2       NA      NA      4       0.0145  0.0030  -0.0206 -0.0074 0.0485  -0.0191 -0.0113 0.0336  0.0099  0.0178
POOL13_09RL     POOL13_09RL     0       0       0       2       2       NA      NA      9       0.0094  0.0054  0.0615  0.0461  -0.0008 -0.0113 0.0540  0.0077  0.0227  0.0199
POOL14S_01RL    POOL14S_01RL    0       0       0       2       2       1       1       1       0.0115  0.0025  -0.0258 0.0321  0.0069  0.0175  -0.0102 0.0213  0.0161  -0.0020
POOL95_02RL     POOL95_02RL     0       0       0       1       2       NA      NA      2       0.0183  0.0126  -0.0216 -0.0514 0.0546  -0.0125 -0.0034 0.0159  -0.0034 0.0221
POOL94_05RL     POOL94_05RL     0       0       0       2       2       NA      NA      5       0.0116  0.0147  -0.0302 -0.0024 0.0043  0.0400  0.0272  -0.0212 0.0052  0.0120
POOL64_08RL     POOL64_08RL     0       0       0       2       NA      NA      NA      8       0.0131  0.0066  0.0177  -0.0510 -0.0403 -0.0053 0.0177  0.0024  -0.0263 0.0119
POOL116_06RL    POOL116_06RL    0       0       0       1       NA      NA      NA      6       0.0141  0.0072  -0.0259 -0.0046 0.0272  0.0067  -0.0147 0.0317  0.0193  0.0015
POOL90_10RL     POOL90_10RL     0       0       0       2       2       NA      NA      10      0.0148  0.0135  0.0242  -0.0202 -0.0293 0.0450  -0.0129 0.0227  0.0005  0.0110
~~~

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS

tar -czvf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.permAll.Vs2.epacts.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm*.epacts
tar -czvf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.permAll.Vs2.epacts.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm*.epacts
tar -czvf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.permAll.Vs2.epacts.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm*.epacts
tar -czvf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.permAll.Vs2.epacts.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm*.epacts
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm*.epacts
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm*.epacts
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm*.epacts
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm*.epacts

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/*Pheno1*perm* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/*Pheno2*perm* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/*Pheno3*perm* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/*Pheno4*perm* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/.

rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*Makefile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/*ind
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*Makefile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/*ind
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*Makefile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/*ind
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*Makefile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/*ind

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm1.epacts.GeneIDList
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm1.epacts.GeneIDList
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm1.epacts.GeneIDList

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList FakePheno Pheno1 Exonic.Nonsynonymous 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm1.epacts.GeneIDList FakePheno Pheno2 Exonic.Nonsynonymous 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm1.epacts.GeneIDList FakePheno Pheno3 Exonic.Nonsynonymous 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm1.epacts.GeneIDList FakePheno Pheno4 Exonic.Nonsynonymous 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output | grep -wv NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output | grep -wv NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output | grep -wv NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output | grep -wv NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output | grep -wv NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output | grep -wv NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output | grep -wv NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output | grep -wv NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd))); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.histograms.jpg\", width=2500, height=2000, res=300); print(Data1[14,]); par(mfrow=c(2,2)); hist(as.numeric(Data1[40,])); hist(as.numeric(Data1[156,])); hist(as.numeric(Data1[261,])); hist(as.numeric(Data1[345,])); dev.off();" > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.vs2.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd))); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.histograms.jpg\", width=2500, height=2000, res=300); print(Data1[14,]); par(mfrow=c(2,2)); hist(as.numeric(Data1[40,])); hist(as.numeric(Data1[156,])); hist(as.numeric(Data1[261,])); hist(as.numeric(Data1[345,])); dev.off();" > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.vs2.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd))); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.histograms.jpg\", width=2500, height=2000, res=300); print(Data1[14,]); par(mfrow=c(2,2)); hist(as.numeric(Data1[40,])); hist(as.numeric(Data1[156,])); hist(as.numeric(Data1[261,])); hist(as.numeric(Data1[345,])); dev.off();" > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.vs2.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd))); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.histograms.jpg\", width=2500, height=2000, res=300); print(Data1[14,]); par(mfrow=c(2,2)); hist(as.numeric(Data1[40,])); hist(as.numeric(Data1[156,])); hist(as.numeric(Data1[261,])); hist(as.numeric(Data1[345,])); dev.off();" > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.vs2.Routput

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd)));" | grep -v ^\> | sort -rg -k 2,2 | head -n -1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); Data2 <- -log10(Data1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd), apply(Data2,1,mean), apply(Data2,1,sd)));" | grep -v ^\> | sort -rg -k 2,2 | head -n -1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd)));" | grep -v ^\> | sort -rg -k 2,2 | head -n -1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd)));" | grep -v ^\> | sort -rg -k 2,2 | head -n -1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd)));" | grep -v ^\> | sort -rg -k 2,2 | head -n -1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd))); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.histograms.jpg\", width=2500, height=2000, res=300); print(Data1[14,]); par(mfrow=c(2,2)); hist(as.numeric(Data1[40,])); hist(as.numeric(Data1[156,])); hist(as.numeric(Data1[261,])); hist(as.numeric(Data1[345,])); dev.off();" > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd))); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.histograms.jpg\", width=2500, height=2000, res=300); print(Data1[14,]); par(mfrow=c(2,2)); hist(as.numeric(Data1[40,])); hist(as.numeric(Data1[156,])); hist(as.numeric(Data1[261,])); hist(as.numeric(Data1[345,])); dev.off();" > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd))); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.histograms.jpg\", width=2500, height=2000, res=300); print(Data1[14,]); par(mfrow=c(2,2)); hist(as.numeric(Data1[40,])); hist(as.numeric(Data1[156,])); hist(as.numeric(Data1[261,])); hist(as.numeric(Data1[345,])); dev.off();" > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd))); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.histograms.jpg\", width=2500, height=2000, res=300); print(Data1[14,]); par(mfrow=c(2,2)); hist(as.numeric(Data1[40,])); hist(as.numeric(Data1[156,])); hist(as.numeric(Data1[261,])); hist(as.numeric(Data1[345,])); dev.off();" > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput

#Create 1..1000 loop that takes each perm# file and pastes together just the p-value for a given 'rank' after removing NAs and sorting so that 'most significant gene p-value' can be averaged across permutations, as can 2nd most significant, 3rd, 4th, etc...

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.sh FakePheno Pheno1 Exonic.Nonsynonymous 1 1000 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.sh FakePheno Pheno2 Exonic.Nonsynonymous 1 1000 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.sh FakePheno Pheno3 Exonic.Nonsynonymous 1 1000 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.sh FakePheno Pheno4 Exonic.Nonsynonymous 1 1000 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output | grep -vw NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output | grep -vw NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output | grep -vw NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output | grep -vw NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd)));" | grep -v ^\> | tail -n +2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); Data2 <- -log10(Data1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd), apply(Data2,1,mean), apply(Data2,1,sd)));" | grep -v ^\> | tail -n +2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd)));" | grep -v ^\> | tail -n +2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd)));" | grep -v ^\> | tail -n +2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd)));" | grep -v ^\> | tail -n +2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput

paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput | vi -
paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput | vi -
paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput | vi -
paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput | vi -

#<(paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput | awk '{ print $1, "\t", $2 }') /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs)

mkdir /data/userdata/pg/michaelt/Data/TempStuff

paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput | awk '{ print $1, "\t", $2 }') /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs > /data/userdata/pg/michaelt/Data/TempStuff/asdubsdbasd

#Transferring file to desktop to figure out boxplot commands in R
scp -p /data/userdata/pg/michaelt/Data/TempStuff/asdubsdbasd mturchin20@wolfy.uchicago.edu: mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/TempStuff/.

~~~
Data1 <- read.table("asdubsdbasd", header=FALSE, row.names=1)
 Data2 <- -log10(Data1)
#  boxplot(t(Data2[(nrow(Data2)-50):nrow(Data2),2:ncol(Data2)]))
 boxplot(t(Data2[(nrow(Data2)-50):nrow(Data2),2:ncol(Data2)]), ylab="-log10(p-value)", xlab="True Value Gene @ Rank", main="Boxplots of 1000 Permutations per Rank")
 legend("topleft", c("True Value"), lty=1, lwd=3, col="RED")
#  points(1:(nrow(Data2)-(nrow(Data2)-50)), Data2[(nrow(Data2)-49):nrow(Data2),1], col="RED")
  points(1:(nrow(Data2)-(nrow(Data2)-51)), Data2[(nrow(Data2)-50):nrow(Data2),1], col="RED")
#  lines(1:(nrow(Data2)-(nrow(Data2)-51)), Data2[(nrow(Data2)-48):nrow(Data2),1], col="RED", lwd=3)
  lines(1:(nrow(Data2)-(nrow(Data2)-51)), Data2[(nrow(Data2)-50):nrow(Data2),1], col="RED", lwd=3)
  lines(1:(nrow(Data2)-(nrow(Data2)-150)), Data2[(nrow(Data2)-149):nrow(Data2),1], col="RED", lwd=3)

~~~

#Coming up with basic FDR script
cp -p /home/michaelt/Scripts/R/BasicScript.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.GetFDRValsPerGeneFromPermutations.vs1.R

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.GetFDRValsPerGeneFromPermutations.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm*.epacts | awk '{ print $10 }' | grep -v -E 'NA|PVALUE' | sort -rg -k 1,1)

#CHECK_1: Check total number of permutation and 'true phenotype' runs ran and were completed
#CHECK_1: And if doing FDR calculations correctly
#CHECK_1: Try running SKAT_O using pathway annotations from msigdb as collection of variants at once instead of per gene than collapsing for GSEA
#CHECK_1: Remember to permute via phenotype and not the internal 'gene set' that occurs via what is seen here http://www.broadinstitute.org/cancer/software/genepattern/modules/docs/GSEAPreranked/1 (Permutation test: In GSEAPreranked, permutations are always done by gene set. In standard GSEA, you can choose to set the parameter Permutation type to phenotype (the default) or gene set, but GSEAPreranked does not provide this option.)...
#... and what is read here http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1239896/ (Step 2: Estimation of Significance Level of ES. We estimate the statistical significance (nominal P value) of the ES by using an empirical phenotype-based permutation test procedure that preserves the complex correlation structure of the gene expression data. Specifically, we permute the phenotype labels and recompute the ES of the gene set for the permuted data, which generates a null distribution for the ES. The empirical, nominal P value of the observed ES is then calculated relative to this null distribution. Importantly, the permutation of class labels preserves gene-gene correlations and, thus, provides a more biologically reasonable assessment of significance than would be obtained by permuting genes)
#... also from same paper but later on (Genes may be ranked based on the differences seen in a small data set, with too few samples to allow rigorous evaluation of significance levels by permuting the class labels. In these cases, a P value can be estimated by permuting the genes, with the result that genes are randomly assigned to the sets while maintaining their size. This approach is not strictly accurate: because it ignores gene-gene correlations, it will overestimate the significance levels and may lead to false positives. Nonetheless, it can be useful for hypothesis generation. The gsea-p software supports this option.)
#... I can do this by just running GSEA on my phenotype-permuted SKAT-O results and ignore the internal GSEA permutation procedure and then count up the number of times certain pathways come up as significant in the permuted data versus which come up in the observed data (or at least whatever metric ends up coming out of GSEA -- as of writing these notes 20150126 I have yet to run GSEA yet so I'm not exactly sure yet what its output looks like)

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1[,2] <- as.numeric(Data1[,2]); Data1New <- cbind(Data1, signif(qvalue(Data1[,2])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1[,2] <- as.numeric(Data1[,2]); Data1New <- cbind(Data1, signif(qvalue(Data1[,2])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" 

#Figuring out how to use/run GSEA tool GSEAPreranked
mkdir /home/michaelt/Software/GSEA2-2.1.0

#Below did not work, had to manually download and ftp this file into the desired location
#wget http://www.broadinstitute.org/gsea/msigdb/download_file.jsp?filePath=/resources/software/gsea2-2.1.0.jar 
#wget http://www.broadinstitute.org/gsea/msigdb/download_file.jsp?filePath=/resources/software/gsea2_distrib-2.1.0.zip 

scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/Software/GSEA/gsea2_distrib-2.1.0.zip mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/Software/GSEA/gsea2-2.1.0.jar /home/michaelt/Software/. 


#Below from: http://www.broadinstitute.org/gsea/doc/GSEAUserGuideTEXT.htm#_Running_GSEA_from (accessed 20150125)
~~~
To run GSEA from the command line, use a java command of the form:

java -cp full-path/gsea2.jar -Xmx512m gsea-tool  parameters

-cp

Points the CLASSPATH variable to the complete path of the gsea2.jar file. You do not need to set any other CLASSPATH variables.

For Broad internal users, the latest .jar file is available at /xchip/projects/xtools/gsea2.jar.
~~~

mkdir /home/michaelt/Data/GSEA

#Below did not work, had to manually download and ftp this file into the desired location
#wget http://www.broadinstitute.org/gsea/msigdb/download_file.jsp?filePath=/resources/msigdb/4.0/msigdb.v4.0.symbols.gmt

mkdir /home/michaelt/Data/GSEA/Testing

#Want to test that all the genes annotated exist within the database files being used

[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList | wc
    475     475    3006

for i in {2..475}
do

	gene1=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList | awk -v geneRow=$i '{ if (NR == geneRow) { print $1 } }'`
#	echo $gene1
	wc1=`cat /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt | grep $gene1 | wc`

	echo $gene1 $wc1 >> /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.TestingForPresenceIn.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList

	gene1=`cat /dev/null`
	wc1=`cat /dev/null`

done

#Very few genes are entirely not present in the msigdb pathway files
cat /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.TestingForPresenceIn.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList | sort -rg -k 2,2 | tail -n 20

~~~
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations]$cat /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.TestingForPresenceIn.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList | sort -rg -k 2,2 | tail -n 20
CKMT1A 6 1765 11599
TMEM192 5 2358 15560
PEX10 5 1584 10555
NEURL4 5 3959 25757
MITD1 5 3505 22799
ERLEC1 5 3885 25575
ATAD3B 5 3703 24402
MRPS9 4 4668 30029
LENEP 4 1999 13326
PUSL1 3 1681 11277
NRN1L 2 1563 10434
TMPRSS9 1 80 611
C14orf178 1 1841 12520
PPAN-P2RY11 0 0 0
GIMAP1-GIMAP5 0 0 0
EMC1 0 0 0
COLGALT1 0 0 0
CCDC73 0 0 0
CCAR2 0 0 0
ANKHD1-EIF4EBP3 0 0 0
~~~

#20150126
#Redoing EPACTS analyses after updating the bgzip file for the main .vcf file

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz | awk '{ if ($1 == "23") { $1 = "X"} print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip

/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz.tbi /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ORIG.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz.tbi

/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz

#20150129 -- if redoing things after this date use the .RMID5PCs ending as well....not sure why this no longer got included by technically it should be because it's true
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS 
#bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs 

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS
#bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs

tar -czvf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.runAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run*.epacts
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run*.epacts
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.histograms.jpg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.histograms.jpg
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.vs2.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.vs2.Routput
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/.
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run*ind /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run*Makefile

tar -czvf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/OLD2.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.runAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run*.epacts
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run*.epacts
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.histograms.jpg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.histograms.jpg
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.vs2.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.vs2.Routput
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run1.epacts.GeneIDList /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/OLD2.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run1.epacts.GeneIDList
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/.
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run*ind /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run*Makefile

tar -czvf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/OLD2.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.runAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run*.epacts
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run*.epacts
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.histograms.jpg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.histograms.jpg
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.vs2.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.vs2.Routput
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run1.epacts.GeneIDList /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/OLD2.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run1.epacts.GeneIDList
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/.
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run*ind /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run*Makefile

tar -czvf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/OLD2.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.runAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run*.epacts
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run*.epacts
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.histograms.jpg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.histograms.jpg
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.vs2.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.vs2.Routput
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run1.epacts.GeneIDList /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/OLD2.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run1.epacts.GeneIDList
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/.
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run*ind /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run*Makefile

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run1.epacts.GeneIDList
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run1.epacts.GeneIDList
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run1.epacts.GeneIDList

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList TruePheno Pheno1 Exonic.Nonsynonymous 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run1.epacts.GeneIDList TruePheno Pheno2 Exonic.Nonsynonymous 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run1.epacts.GeneIDList TruePheno Pheno3 Exonic.Nonsynonymous 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run1.epacts.GeneIDList TruePheno Pheno4 Exonic.Nonsynonymous 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output | grep -wv NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output | grep -wv NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output | grep -wv NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output | grep -wv NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd)));" | grep -v ^\> | sort -rg -k 2,2 | head -n -1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); Data2 <- -log10(Data1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd), apply(Data2,1,mean), apply(Data2,1,sd)));" | grep -v ^\> | sort -rg -k 2,2 | head -n -1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd)));" | grep -v ^\> | sort -rg -k 2,2 | head -n -1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd)));" | grep -v ^\> | sort -rg -k 2,2 | head -n -1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd)));" | grep -v ^\> | sort -rg -k 2,2 | head -n -1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput

~~~
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations]$tail -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput
CIAO1:           0.010419630 2.948600e-04
FUT8:            0.009499187 7.184600e-04
MRPS9:           0.009020114 9.653076e-04
EDC4:            0.008973321 8.498714e-04
CUL5:            0.007770337 1.775791e-04
NUDC:            0.007347700 0.000000e+00
SDHD:            0.006449249 5.337792e-04
VDAC3:           0.002259239 3.679158e-04
VTI1A:           0.001098320 8.732696e-05
GEMIN2:          0.000775861 3.211198e-04
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations]$tail -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput
CIAO1:           0.0104004830 3.314492e-04 1.9831653159 1.386507e-02
FUT8:            0.0092845960 8.366922e-04 2.0339736736 3.900911e-02
EDC4:            0.0090311210 8.470959e-04 2.0460940574 3.986946e-02
MRPS9:           0.0089262030 9.259414e-04 2.0516160501 4.467143e-02
CUL5:            0.0077371170 1.639625e-04 2.1115180442 9.251883e-03
NUDC:            0.0073477000 0.000000e+00 2.1338485839 0.000000e+00
SDHD:            0.0063112910 4.741037e-04 2.2011037650 3.282508e-02
VDAC3:           0.0022698400 3.582224e-04 2.6499420952 7.448913e-02
VTI1A:           0.0010969036 9.371745e-05 2.9612728538 3.485794e-02
GEMIN2:          0.0007524931 3.138343e-04 3.1780469314 2.431659e-01
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations]$tail -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput
HARS:            0.0124821300 9.241060e-04
XPO4:            0.0114074700 3.335765e-04
LDLR:            0.0108962560 1.058498e-03
SDCBP:           0.0097649280 5.540537e-04
GEMIN6:          0.0080840720 9.209877e-04
ICAM2:           0.0070759810 2.665100e-04
EIF3L:           0.0051961430 9.585082e-05
KIR3DL2:         0.0051198370 7.477929e-04
EEF1E1:          0.0027584390 5.767824e-05
GEMIN2:          0.0006154692 1.240322e-04
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations]$tail -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput
HARS:            0.0124869800 8.707598e-04
XPO4:            0.0113585900 3.650333e-04
LDLR:            0.0109963250 1.387778e-03
SDCBP:           0.0098379750 5.955183e-04
GEMIN6:          0.0081037530 1.026901e-03
ICAM2:           0.0070736290 2.271870e-04
KIR3DL2:         0.0052122210 8.001052e-04
EIF3L:           0.0051817820 1.068389e-04
EEF1E1:          0.0027681430 6.012171e-05
GEMIN2:          0.0006076614 1.173707e-04
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations]$tail -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput
NCAPD3:          0.0191322700 1.209771e-03
MCL1:            0.0129071300 8.074987e-04
CAPN1:           0.0110830750 1.064156e-03
EML3:            0.0079168660 1.032934e-04
EPPK1:           0.0054144350 8.308956e-04
SDF2L1:          0.0047018430 3.062125e-04
POLE:            0.0045066320 4.149423e-04
UBR2:            0.0034429010 1.891623e-04
BICD2:           0.0016849910 1.331283e-04
AMBRA1:          0.0008004482 1.108007e-04
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations]$tail -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput
NCAPD3:          0.0192332000 1.304343e-03
MCL1:            0.0130626600 8.144170e-04
CAPN1:           0.0111352670 1.050676e-03
EML3:            0.0079202250 1.091330e-04
EPPK1:           0.0054542040 8.477160e-04
SDF2L1:          0.0045741980 3.681954e-04
POLE:            0.0043593020 5.366147e-04
UBR2:            0.0034526600 1.712316e-04
BICD2:           0.0016938370 1.359807e-04
AMBRA1:          0.0008072145 1.082477e-04
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations]$tail -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput
NFU1:            0.011664080 2.928592e-04
SPATS2:          0.010834650 3.350323e-04
EML3:            0.010516960 1.544944e-04
AQR:             0.010495000 0.000000e+00
SDCCAG8:         0.009696784 5.895933e-07
BICD2:           0.008033618 2.265361e-04
MRPS35:          0.007225927 2.890907e-06
PDIA6:           0.006179672 5.202883e-04
CLGN:            0.005013847 7.326989e-04
SUN1:            0.002010493 8.089778e-05
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations]$tail -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput
NFU1:            0.011674090 3.114970e-04
SPATS2:          0.010817890 3.217966e-04
EML3:            0.010503600 8.600000e-05
AQR:             0.010495000 0.000000e+00
SDCCAG8:         0.009750884 5.407191e-04
BICD2:           0.008031391 2.401371e-04
MRPS35:          0.007225965 2.210826e-06
PDIA6:           0.006192917 5.281613e-04
CLGN:            0.004917945 4.046539e-05
SUN1:            0.002008964 7.120811e-05
~~~

#So it doesn't look like the 'new' bgzip file changes anything, so the differences in 'bytes' are probably negligible

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1[,2] <- as.numeric(Data1[,2]); Data1New <- cbind(Data1, signif(qvalue(Data1[,2])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1[,2] <- as.numeric(Data1[,2]); Data1New <- cbind(Data1, signif(qvalue(Data1[,2])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1[,2] <- as.numeric(Data1[,2]); Data1New <- cbind(Data1, signif(qvalue(Data1[,2])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1[,2] <- as.numeric(Data1[,2]); Data1New <- cbind(Data1, signif(qvalue(Data1[,2])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" 

#CHECK_0: Still need to check total number of runs/permutations are correct numbers for both true and fake phenotypes (fake phenotype reruns are still going as of this comment) 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.HIVPROG.maf1.skat0.permAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/OLD1.AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.HIVPROG.maf1.skat0.permAll.Files.tar.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.HIVPROG.maf1.skat0.permAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/OLD1.AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.HIVPROGHE.maf1.skat0.permAll.Files.tar.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.HIVPROG.maf1.skat0.permAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/OLD1.AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.AIDS.maf1.skat0.permAll.Files.tar.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.HIVPROG.maf1.skat0.permAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/OLD1.AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.AIDSEXTR.maf1.skat0.permAll.Files.tar.gz

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.permAll.Vs2.epacts.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/OLD1.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.permAll.Files.tar.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.permAll.Vs2.epacts.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/OLD1.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.permAll.Files.tar.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.permAll.Vs2.epacts.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/OLD1.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.permAll.Files.tar.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.permAll.Vs2.epacts.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/OLD1.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.permAll.Files.tar.gz

#20150127 -- I accidentally clobbered OLD1.AllPools......Pheno1.maf1.skat0.permAll.Files.tar.gz for Phenos 1-3 because I didn't realize i hadn't changed OLD1 to OLD2 yet on the below commands. I only noticed once I was on Pheno4.
#tar -czvf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/OLD1.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.permAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm*.epacts
tar -czvf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/OLD2.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.permAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm*.epacts
#tar -czvf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/OLD1.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.permAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm*.epacts
tar -czvf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/OLD2.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.permAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm*.epacts
#tar -czvf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/OLD1.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.permAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm*.epacts
tar -czvf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/OLD2.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.permAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm*.epacts
tar -czvf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/OLD2.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.permAll.Files.tar.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm*.epacts
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm*.epacts
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm*.epacts
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm*.epacts
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm*.epacts

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.histograms.jpg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.histograms.jpg
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.histograms.jpg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.histograms.jpg
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.histograms.jpg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.histograms.jpg
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.histograms.jpg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.histograms.jpg
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/*Pheno1*perm* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/*Pheno2*perm* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/*Pheno3*perm* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/*Pheno4*perm* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/.

rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm*Makefile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm*ind
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm*Makefile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm*ind
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm*Makefile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm*ind
rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm*epacts.OK /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm*cov /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm*phe /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm*Makefile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm*ind

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm1.epacts.GeneIDList
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm1.epacts.GeneIDList
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm1.epacts.GeneIDList

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList FakePheno Pheno1 Exonic.Nonsynonymous 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm1.epacts.GeneIDList FakePheno Pheno2 Exonic.Nonsynonymous 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm1.epacts.GeneIDList FakePheno Pheno3 Exonic.Nonsynonymous 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm1.epacts.GeneIDList FakePheno Pheno4 Exonic.Nonsynonymous 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output | grep -wv NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output | grep -wv NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output | grep -wv NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output | grep -wv NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd))); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.histograms.jpg\", width=2500, height=2000, res=300); print(Data1[14,]); par(mfrow=c(2,2)); hist(as.numeric(Data1[40,])); hist(as.numeric(Data1[156,])); hist(as.numeric(Data1[261,])); hist(as.numeric(Data1[345,])); dev.off();" > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd))); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.histograms.jpg\", width=2500, height=2000, res=300); print(Data1[14,]); par(mfrow=c(2,2)); hist(as.numeric(Data1[40,])); hist(as.numeric(Data1[156,])); hist(as.numeric(Data1[261,])); hist(as.numeric(Data1[345,])); dev.off();" > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd))); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.histograms.jpg\", width=2500, height=2000, res=300); print(Data1[14,]); par(mfrow=c(2,2)); hist(as.numeric(Data1[40,])); hist(as.numeric(Data1[156,])); hist(as.numeric(Data1[261,])); hist(as.numeric(Data1[345,])); dev.off();" > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd))); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.histograms.jpg\", width=2500, height=2000, res=300); print(Data1[14,]); par(mfrow=c(2,2)); hist(as.numeric(Data1[40,])); hist(as.numeric(Data1[156,])); hist(as.numeric(Data1[261,])); hist(as.numeric(Data1[345,])); dev.off();" > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.sh FakePheno Pheno1 Exonic.Nonsynonymous 1 1000 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.sh FakePheno Pheno2 Exonic.Nonsynonymous 1 1000 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.sh FakePheno Pheno3 Exonic.Nonsynonymous 1 1000 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.sh FakePheno Pheno4 Exonic.Nonsynonymous 1 1000 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output | grep -vw NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output | grep -vw NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output | grep -vw NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output | grep -vw NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd)));" | grep -v ^\> | tail -n +2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); Data2 <- -log10(Data1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd), apply(Data2,1,mean), apply(Data2,1,sd)));" | grep -v ^\> | tail -n +2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd)));" | grep -v ^\> | tail -n +2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd)));" | grep -v ^\> | tail -n +2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd)));" | grep -v ^\> | tail -n +2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput

paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput | vi -
paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput | vi -
paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput | vi -
paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput | vi -

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz --missing-indv --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD

~~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.imiss | sort -rg -k 5,5 | head -n 10
POOL79_12RL     63887   0       33197   0.519621
POOL169_05RL    63887   0       28779   0.450467
POOL88_02RL     63887   0       12352   0.193341
POOL94_10RL     63887   0       6047    0.0946515
POOL116_08RL    63887   0       5812    0.0909731
POOL81_03RL     63887   0       4761    0.0745222
POOL168_10RL    63887   0       4084    0.0639254
POOL115_08RL    63887   0       3676    0.0575391
POOL95_02RL     63887   0       3579    0.0560208
POOL94_08RL     63887   0       3452    0.0540329
~~~

#20150206 Copied from before...
#CHECK_0: Check total number of permutation and 'true phenotype' runs ran and were completed
#CHECK_1: And if doing FDR calculations correctly
#CHECK_1: Try running SKAT_O using pathway annotations from msigdb as collection of variants at once instead of per gene than collapsing for GSEA
#CHECK_1: Remember to permute via phenotype and not the internal 'gene set' that occurs via what is seen here http://www.broadinstitute.org/cancer/software/genepattern/modules/docs/GSEAPreranked/1 (Permutation test: In GSEAPreranked, permutations are always done by gene set. In standard GSEA, you can choose to set the parameter Permutation type to phenotype (the default) or gene set, but GSEAPreranked does not provide this option.)...

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.GetFDRValsPerGeneFromPermutations.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm*.epacts | awk '{ print $10 }' | grep -v -E 'NA|PVALUE' | sort -rg -k 1,1) > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.GetFDRValsPerGeneFromPermutations.vs1.output

#From http://www.bioconductor.org/packages/release/bioc/vignettes/RamiGO/inst/doc/RamiGO.R
~~~
###################################################
### code chunk number 9: gseaexec (eval = FALSE)
###################################################
## ## paths to gsea jar and gmt file
## exe.path <- exe.path.string
## gmt.path <- gmt.path.string
## 
## gsea.collapse <- "false"
## ## number of permutations
## nperm <- 10000
## gsea.seed <- 54321
## gsea.out <- "out-folder"
## 
## ## build GSEA command
## gsea.report <- "report-file"
## rnk.path <- "rank-file"
## gsea.cmd <- sprintf("java -Xmx4g -cp %s xtools.gsea.GseaPreranked -gmx %s -collapse %s -nperm %i -rnk %s -scoring_scheme weighted -rpt_label %s -include_only_symbols true -make_sets true -plot_top_x 75 -rnd_seed %i -set_max 500 -set_min 15 -zip_report true -out %s -gui false", exe.path, gmt.path, gsea.collapse, nperm, rnk.path, gsea.report, gsea.seed, gsea.out)
## 
## ## execute command on the system
## system(gsea.cmd)

~~~

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput | R -q -e "Data1 <- read.table(file('stdin'), header=F); write.table(matrix(c(as.character(Data1[,1]), -log10(Data1[,2])), ncol=2), quote=FALSE, row.name=FALSE, col.name=FALSE, sep=\"\t\")" | grep -v ^\> > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.rnk
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput | R -q -e "Data1 <- read.table(file('stdin'), header=F); write.table(matrix(c(as.character(Data1[,1]), -log10(Data1[,2])), ncol=2), quote=FALSE, row.name=FALSE, col.name=FALSE, sep=\"\t\")" | grep -v ^\> > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.rnk
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput | R -q -e "Data1 <- read.table(file('stdin'), header=F); write.table(matrix(c(as.character(Data1[,1]), -log10(Data1[,2])), ncol=2), quote=FALSE, row.name=FALSE, col.name=FALSE, sep=\"\t\")" | grep -v ^\> > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.rnk
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput | R -q -e "Data1 <- read.table(file('stdin'), header=F); write.table(matrix(c(as.character(Data1[,1]), -log10(Data1[,2])), ncol=2), quote=FALSE, row.name=FALSE, col.name=FALSE, sep=\"\t\")" | grep -v ^\> > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.rnk

scp -p /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.rnk mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/Software/GSEA/. 

#java -Xmx512m -cp /home/michaelt/Software/GSEA2-2.1.0/gsea2-2.1.0.jar xtools.gsea.GseaPreranked -gmx /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt -rnk /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.rnk -nperm 0 -rnd_seed 432142 -set_min 5 -collapse false -make_sets false -include_only_symbols true -gui false -plot_top_x 0 -zip_report true -rpt_label blah2 -out /home/michaelt/Data/GSEA/Testing/blah2

java -Xmx512m -cp /home/michaelt/Software/GSEA2-2.1.0/gsea2-2.1.0.jar xtools.gsea.GseaPreranked -gmx /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt -rnk /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.rnk -nperm 0 -rnd_seed 432142 -set_min 5 -collapse false -make_sets false -include_only_symbols true -gui false -plot_top_x 0 -zip_report false -rpt_label DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput -out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/GSEA
java -Xmx512m -cp /home/michaelt/Software/GSEA2-2.1.0/gsea2-2.1.0.jar xtools.gsea.GseaPreranked -gmx /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt -rnk /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.rnk -nperm 0 -rnd_seed 432142 -set_min 5 -collapse false -make_sets false -include_only_symbols true -gui false -plot_top_x 0 -zip_report false -rpt_label DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput -out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/GSEA 
java -Xmx512m -cp /home/michaelt/Software/GSEA2-2.1.0/gsea2-2.1.0.jar xtools.gsea.GseaPreranked -gmx /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt -rnk /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.rnk -nperm 0 -rnd_seed 432142 -set_min 5 -collapse false -make_sets false -include_only_symbols true -gui false -plot_top_x 0 -zip_report false -rpt_label DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput -out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/GSEA 
java -Xmx512m -cp /home/michaelt/Software/GSEA2-2.1.0/gsea2-2.1.0.jar xtools.gsea.GseaPreranked -gmx /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt -rnk /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.rnk -nperm 0 -rnd_seed 432142 -set_min 5 -collapse false -make_sets false -include_only_symbols true -gui false -plot_top_x 0 -zip_report false -rpt_label DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput -out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/GSEA 

#rm -r /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked.1423276412751
#rm -r /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked.1423276650788/
#rm -r /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked.1423276655777/
#rm -r /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked.1423276660583/

#To easily get the path/file names for the mv commands below did these two commands to get everthing on the screen
#ls -lrt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/GSEA/.
#ls -lrt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked.1423280266219/.

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked.1423280261353/gsea_report_for_na_pos_1423280261353.xls /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked.1423280261353/gsea_report_for_na_pos.xls ; mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked.1423280261353 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked.1423280266219/gsea_report_for_na_pos_1423280266219.xls /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked.1423280266219/gsea_report_for_na_pos.xls ; mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked.1423280266219/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked.1423280271143/gsea_report_for_na_pos_1423280271143.xls /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked.1423280271143/gsea_report_for_na_pos.xls ; mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked.1423280271143/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos_1423280275943.xls /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked.1423280275943/gsea_report_for_na_pos.xls ; mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked.1423280275943/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/

for i in {1..1000}
do
	echo $i
	cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm${i}.epacts | awk '{ if ($10 != "NA") { print $0 } }' | perl -lane 'my @vals1 = split(/_/, $F[3]); $F[3] = $vals1[1]; print join("\t", @F);' | R -q -e "Data1 <- read.table(file('stdin'), header=F); write.table(matrix(c(as.character(Data1[,4]), -log10(Data1[,10])), ncol=2), quote=FALSE, row.name=FALSE, col.name=FALSE, sep=\"\t\")" | grep -v ^\> > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm${i}.GSEA.rnk
done
for i in {1..1000}
do
	echo $i
	cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm${i}.epacts | awk '{ if ($10 != "NA") { print $0 } }' | perl -lane 'my @vals1 = split(/_/, $F[3]); $F[3] = $vals1[1]; print join("\t", @F);' | R -q -e "Data1 <- read.table(file('stdin'), header=F); write.table(matrix(c(as.character(Data1[,4]), -log10(Data1[,10])), ncol=2), quote=FALSE, row.name=FALSE, col.name=FALSE, sep=\"\t\")" | grep -v ^\> > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm${i}.GSEA.rnk
done
for i in {1..1000}
do
	echo $i
	cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm${i}.epacts | awk '{ if ($10 != "NA") { print $0 } }' | perl -lane 'my @vals1 = split(/_/, $F[3]); $F[3] = $vals1[1]; print join("\t", @F);' | R -q -e "Data1 <- read.table(file('stdin'), header=F); write.table(matrix(c(as.character(Data1[,4]), -log10(Data1[,10])), ncol=2), quote=FALSE, row.name=FALSE, col.name=FALSE, sep=\"\t\")" | grep -v ^\> > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm${i}.GSEA.rnk
done
for i in {1..1000}
do
	echo $i
	cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm${i}.epacts | awk '{ if ($10 != "NA") { print $0 } }' | perl -lane 'my @vals1 = split(/_/, $F[3]); $F[3] = $vals1[1]; print join("\t", @F);' | R -q -e "Data1 <- read.table(file('stdin'), header=F); write.table(matrix(c(as.character(Data1[,4]), -log10(Data1[,10])), ncol=2), quote=FALSE, row.name=FALSE, col.name=FALSE, sep=\"\t\")" | grep -v ^\> > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm${i}.GSEA.rnk
done

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos.xls | awk '{ print $1 }' | tail -n +2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos.PathwayIDList 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos.xls | awk '{ print $1 }' | tail -n +2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos.PathwayIDList 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos.xls | awk '{ print $1 }' | tail -n +2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos.PathwayIDList 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos.xls | awk '{ print $1 }' | tail -n +2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos.PathwayIDList 

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.GSEA.PermutationPrerankAnalyses.Launcher.vs1.sh

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataAnalysis.GATK.PermutationEPACTSRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.GSEA.PermutationPrerankAnalyses.vs1.sh

#cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.GSEA.PermutationPrerankAnalyses.vs1.sh /home/michaelt/Scripts/Bash/SubmittedJobScript.wTime.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.GSEA.PermutationPrerankAnalyses.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ FakePheno Pheno1 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.GSEA.PermutationPrerankAnalyses.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ FakePheno Pheno2 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.GSEA.PermutationPrerankAnalyses.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ FakePheno Pheno3 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.GSEA.PermutationPrerankAnalyses.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ FakePheno Pheno4 Exonic.Nonsynonymous

ls -lrt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/GSEA/. | awk '{ print $9 }' | perl -lane 'if ($F[0] =~ m/(.*)\.(\d+$)/) { my $val1 = "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/GSEA/" . $1; my $val2 = "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/GSEA/" . $1 . "." . $2; system("mv $val2 $val1") };'
ls -lrt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/GSEA/. | awk '{ print $9 }' | perl -lane 'if ($F[0] =~ m/(.*)\.(\d+$)/) { my $val1 = "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/GSEA/" . $1; my $val2 = "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/GSEA/" . $1 . "." . $2; system("mv $val2 $val1") };'
ls -lrt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/GSEA/. | awk '{ print $9 }' | perl -lane 'if ($F[0] =~ m/(.*)\.(\d+$)/) { my $val1 = "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/GSEA/" . $1; my $val2 = "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/GSEA/" . $1 . "." . $2; system("mv $val2 $val1") };'
ls -lrt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/GSEA/. | awk '{ print $9 }' | perl -lane 'if ($F[0] =~ m/(.*)\.(\d+$)/) { my $val1 = "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/GSEA/" . $1; my $val2 = "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/GSEA/" . $1 . "." . $2; system("mv $val2 $val1") };'

#cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.sh

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.py

#bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos.PathwayIDList FakePheno Pheno1 Exonic.Nonsynonymous 
#bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos.PathwayIDList FakePheno Pheno2 Exonic.Nonsynonymous 
#bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos.PathwayIDList FakePheno Pheno3 Exonic.Nonsynonymous 
#bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos.PathwayIDList FakePheno Pheno4 Exonic.Nonsynonymous 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/GSEA/DataProcessing.Pipeline.pt6.All.ptg.GSEA.PermutationPrerankAnalyses.vs1.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm*.GSEA.GseaPreranked/gsea_report_for_na_pos_*.xls > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.gsea_report_for_na_pos_All.xls.temp
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/GSEA/DataProcessing.Pipeline.pt6.All.ptg.GSEA.PermutationPrerankAnalyses.vs1.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm*.GSEA.GseaPreranked/gsea_report_for_na_pos_*.xls > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.gsea_report_for_na_pos_All.xls.temp
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/GSEA/DataProcessing.Pipeline.pt6.All.ptg.GSEA.PermutationPrerankAnalyses.vs1.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm*.GSEA.GseaPreranked/gsea_report_for_na_pos_*.xls > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.gsea_report_for_na_pos_All.xls.temp
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/GSEA/DataProcessing.Pipeline.pt6.All.ptg.GSEA.PermutationPrerankAnalyses.vs1.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm*.GSEA.GseaPreranked/gsea_report_for_na_pos_*.xls > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.gsea_report_for_na_pos_All.xls.temp

#CHECK_0:
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.gsea_report_for_na_pos_All.xls.temp | grep -w -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos.PathwayIDList > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output  
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.gsea_report_for_na_pos_All.xls.temp | grep -w -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos.PathwayIDList > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output  
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.gsea_report_for_na_pos_All.xls.temp | grep -w -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos.PathwayIDList > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output  
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.gsea_report_for_na_pos_All.xls.temp | grep -w -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.GseaPreranked/gsea_report_for_na_pos.PathwayIDList > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/GSEA/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output  

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.py

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/SUMSTAT
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/SUMSTAT
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/SUMSTAT

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.rnk --file2 /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.rnk.SUMSTAT 
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.rnk --file2 /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.rnk.SUMSTAT 
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.rnk --file2 /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.rnk.SUMSTAT 
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.rnk --file2 /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.rnk.SUMSTAT 

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/SUMSTAT
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/SUMSTAT
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/SUMSTAT

for i in {1..1000}
do
        echo $i
	python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm${i}.GSEA.rnk --file2 /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm${i}.GSEA.rnk.SUMSTAT
done
for i in {1..1000}
do
        echo $i
	python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm${i}.GSEA.rnk --file2 /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm${i}.GSEA.rnk.SUMSTAT
done
for i in {1..1000}
do
	echo $i
	python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm${i}.GSEA.rnk --file2 /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm${i}.GSEA.rnk.SUMSTAT
done
for i in {1..1000}
do
	echo $i
	python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm${i}.GSEA.rnk --file2 /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm${i}.GSEA.rnk.SUMSTAT
done

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm*.GSEA.rnk.SUMSTAT > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.permAll.GSEA.rnk.SUMSTAT.temp
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm*.GSEA.rnk.SUMSTAT > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.permAll.GSEA.rnk.SUMSTAT.temp
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm*.GSEA.rnk.SUMSTAT > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.permAll.GSEA.rnk.SUMSTAT.temp
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm*.GSEA.rnk.SUMSTAT > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.permAll.GSEA.rnk.SUMSTAT.temp

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.GSEA.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.vs1.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.Exonic.Nonsynonymous.Pheno1.maf1.skat0.permAll.GSEA.rnk.SUMSTAT.temp > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output 
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.Exonic.Nonsynonymous.Pheno2.maf1.skat0.permAll.GSEA.rnk.SUMSTAT.temp > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output 
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.Exonic.Nonsynonymous.Pheno3.maf1.skat0.permAll.GSEA.rnk.SUMSTAT.temp > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output 
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.Exonic.Nonsynonymous.Pheno4.maf1.skat0.permAll.GSEA.rnk.SUMSTAT.temp > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output 

cp -p /home/michaelt/Scripts/R/BasicScript.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.CalcPermBasedPVal.vs1.R

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.CalcPermBasedPVal.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.rnk.SUMSTAT /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.CalcPermBasedPVal.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output
R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.CalcPermBasedPVal.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.rnk.SUMSTAT /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.vs1.FakePheno.Pheno2.Exonic.Nonsynonymous.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.CalcPermBasedPVal.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output
R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.CalcPermBasedPVal.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.rnk.SUMSTAT /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.vs1.FakePheno.Pheno3.Exonic.Nonsynonymous.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.CalcPermBasedPVal.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output
R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.CalcPermBasedPVal.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.rnk.SUMSTAT /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.vs1.FakePheno.Pheno4.Exonic.Nonsynonymous.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.CalcPermBasedPVal.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output









##20150210
##7th run -- investigating applying a more stringent imiss threshold (just to see how it may or may not change results....this run may not be used, just investigating)
##Using .Vs3. in titles
##20150825 -- this run was used (adding in comment much later since it is true)

#SNP and Indv thresholds, PCA -- get vcf file; all indvs and just white
#PLINK logistic regression stuff
#Freq and Annovar file creation; all indvs and just white
#QC stuff
#VC caller overlap?
#EPACTS skat0 runs
#permutation runs
#pathway analyses; GSEA and SUMSTAT?








vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --missing --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4  

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss | awk '{ if ($5 >= .2) { print $1 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.pheno

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar 

#20151227 NOTE -- This is where the ...AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs... notation begins due to the use of indv80 imiss threshold, e.g. the 'Vs3' at the beginning of the filenames here
#ComeHere1

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar | awk '{ print $1, "\t", $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar | awk '{ print $1 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDs

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDs --non-ref-ac 1 --recode  --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.recode.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80 

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.allSNPs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs 
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.10PCs
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.10PCs.maf05
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 2 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno2
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 3 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno3
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 4 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno4

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.assoc.logistic Pheno1_justWhite_Logistic_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.assoc.logistic.QQplot.jpeg & 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.assoc.logistic Pheno1_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.assoc.logistic.QQplot.jpeg & 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.10PCs.assoc.logistic Pheno1_justWhite_Logistic_10PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.10PCs.assoc.logistic.QQplot.jpeg & 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.10PCs.maf05.assoc.logistic Pheno1_justWhite_Logistic_maf05_10PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.10PCs.maf05.assoc.logistic.QQplot.jpeg & 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.assoc.logistic Pheno2_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno2.assoc.logistic.QQplot.jpeg & 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.assoc.logistic Pheno3_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno3.assoc.logistic.QQplot.jpeg & 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.assoc.logistic Pheno4_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno4.assoc.logistic.QQplot.jpeg & 

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/.

#Allele Frequency stuff/collection stuff here now
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ print $1 }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.FIDs 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($3 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroNeg.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($4 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroHighlyExposedNeg.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($3 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroPos.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justRapidVRapid.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSlowVSlow.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($6 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVeryRapid.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($6 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVerySlow.FIDs

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80 & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroNeg.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroHighlyExposedNeg.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroPos.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justRapidVRapid.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSlowVSlow.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVeryRapid.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVerySlow.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80 & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroHighlyExposedNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroPos.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justRapidVRapid.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSlowVSlow.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVeryRapid.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVerySlow.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow & 

paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow.frq.count | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.AllPhenos.frq.AndCount.gz
gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow.frq.count
#gunzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow.frq.count.gz

##python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.AllPhenos.frq.AndCount.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 30; $i <= 40; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs6.txt.gz
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.AllPhenos.frq.AndCount.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 30; $i <= 40; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz 

#Some QC Checks
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | grep -v \# | perl -lane 'print join("\t", @F[0..9]);' | wc
  73261  732610 25819140

[  michaelt@node8  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs6.txt.gz | wc
74824   82642 33402353
[  michaelt@node8  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/_2/) { print join(",", @F); } ' | wc
1540    1699  712744
[  michaelt@node8  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/_3/) { print join(",", @F); } ' | wc
21      21    9170
[  michaelt@node8  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/_4/) { print join(",", @F); } ' | wc
2       2     959
[  michaelt@node8  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/_5/) { print join(",", @F); } ' | wc
0       0       0
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$R -q -e 'val1 = 1540 + 21 + 2; print (val1);'
> val1 = 1540 + 21 + 2; print (val1);
[1] 1563
>
>
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$R -q -e 'val1 = 1540 + 21 + 2; print (val1); val2 = 74824 - 1563; print (val2);'
> val1 = 1540 + 21 + 2; print (val1); val2 = 74824 - 1563; print (val2);
[1] 1563
[1] 73261
> 
> 

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80 

##python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs3.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs6.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs 
#mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/WRONG1.AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs3.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs 

#20120226
#Technically correct number now is 65208 for both of them. This annotated .bim file was including variants that were not present in white non-hispanic individuals based on the ...hwe1e4.FreqInfo.AnnovarAnnotation.vs6.txt.gz file. However, because the .vcf used in the analyses only had variants present among this subset, any variant that was listed in the below 'gene-variant grouping' files that was outside this ancestry was simply not included so nothing wrong happened. The numbers just originally didn't match with how many variants were included in the skat-O analyse and the number of variants per gene in those files. Using the correct ....hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz file to create the annotated .bim file corrects this and makes the numbers in the 'gene-variant grouping' files match up to the skat-O analyses 
~~~
[  michaelt@node7  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | wc
74824   74824 4098962
[  michaelt@node7  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs | wc
74824   74824 4098962
~~~

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing") || ($7 == "UTR5;UTR3")) { print $8, "\t", $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.All 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "splicing") || ($7 == "exonic;splicing")) { print $8, "\t", $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.Exonic
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") )  { print $8, "\t", $2 } } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if ((($7 == "exonic") && (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV"))) || (($7 == "splicing") || ($7 == "exonic;splicing"))) { print $8, "\t", $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.wSplice
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($12 == "P") || ($12 == "D") ) { print $8, "\t", $2 } } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_PD
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $8, "\t", $2 } } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing") || ($7 == "UTR5;UTR3")) { print $0 } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.All
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "splicing") || ($7 == "exonic;splicing")) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") )  { print $0 } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if ((($7 == "exonic") && (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV"))) || (($7 == "splicing") || ($7 == "exonic;splicing"))) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($12 == "P") || ($12 == "D") ) { print $0 } } } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($13 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing") || ($7 == "UTR5;UTR3")) { print $0 } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "splicing") || ($7 == "exonic;splicing")) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") )  { print $0 } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if ((($7 == "exonic") && (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV"))) || (($7 == "splicing") || ($7 == "exonic;splicing"))) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.wSplice
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($12 == "P") || ($12 == "D") ) { print $0 } } } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_D
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.Sift_D
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($13 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.LRT_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.pheno | awk '{ print $1, "\t", $2, "\t0\t0\t0\t", $3, "\t", $4, "\t", $5, "\t", $6 } ' | sed 's/-9/NA/g' | perl -lane 'print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit.ped

#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar | awk '{ print $4, "\t", $6, "\t", $7 "\t", $8, "\t", $9, "\t", $10, "\t", $11,  "\t", $12, "\t", $13, "\t", $14,  "\t", $15 }' | paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit.ped - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit.ped.vs3
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar | awk '{ print $4, "\t", $6, "\t", $7 "\t", $8, "\t", $9, "\t", $10, "\t", $11,  "\t", $12, "\t", $13, "\t", $14,  "\t", $15 }' | paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit.ped - | perl -lane 'print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit.ped.vs3
join /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar | perl -lane 'splice(@F, 9, 2); splice(@F, 10, 1); print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit.ped.vs2

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit.ped.vs2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit.ped

~~
[  michaelt@node7  ~]$cmp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit.ped.vs3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit.ped                                         [  michaelt@node7  ~]$
~~

#Add line #FAM_ID IND_ID FAT_ID MOT_ID SEX HIVPROG HIVPROGHE AIDS AIDSEXTR RMID PC1 PC2 PC3 PC4 PC5 PC6 PC7 PC8 PC9 PC10 to /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit.ped

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | awk '{ if ($1 == "23") { $1 = "X"} print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip

/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz

for i in {1..1000}
do
	SEED1=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.PermutationList.vs1.txt | awk '{ if (NR == '$i') { print $0 } } '`
#	perl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs3.pl --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --seed $SEED1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm${i}.vs3.ped
	perl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs3.pl --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit.ped --seed $SEED1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit.perm${i}.vs3.ped 

#       echo $SEED1

done

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs3.pt2.sh
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs3.pt2.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs 

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs

#20151227 NOTE -- This is where the ...AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASS... notation begins to appear. Before this, ...AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs... was being used 

~~~
[  michaelt@node8  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit.perm1.vs3.ped | head -n 10
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9    PC10
POOL98_01RL     POOL98_01RL     0       0       0       1       NA      NA      NA      1       0.0079  0.0160  0.0720  0.0296  0.0320  -0.0555 -0.0447 0.0014  0.0010 -0.0499
POOL83_08RL     POOL83_08RL     0       0       0       2       NA      NA      NA      8       0.0157  0.0095  -0.0198 -0.0248 -0.0206 -0.0242 0.0153  0.0448  0.0243 0.0067
POOL77_08RL     POOL77_08RL     0       0       0       1       NA      NA      NA      8       0.0143  -0.0026 -0.0279 -0.0273 0.0338  -0.0194 0.0555  0.0160  -0.00530.0202
POOL02_03RL     POOL02_03RL     0       0       0       2       2       NA      NA      3       0.0136  0.0057  0.0166  0.0415  -0.0098 0.0027  -0.0120 -0.0008 -0.01870.0112
POOL33_03RL     POOL33_03RL     0       0       0       2       2       NA      NA      3       0.0137  0.0162  -0.0260 -0.0403 -0.0116 -0.0230 -0.0308 0.0320  -0.0137-0.0560
POOL157_04RL    POOL157_04RL    0       0       0       2       NA      NA      NA      4       0.0128  0.0025  -0.0309 0.0374  -0.0126 0.0054  0.0047  0.0157  -0.02910.0530
POOL88_02RL     POOL88_02RL     0       0       0       2       1       NA      NA      2       0.0118  0.0045  -0.0299 0.0170  0.0068  -0.0149 -0.0094 -0.0493 -0.0122-0.0149
POOL109_11RL    POOL109_11RL    0       0       0       1       1       NA      NA      11      0.0160  0.0021  -0.0172 -0.0269 -0.0312 0.0343  0.0326  -0.0096 -0.00620.0452
POOL158_08RL    POOL158_08RL    0       0       0       2       2       1       NA      8       0.0106  0.0138  0.0268  -0.0544 0.0063  -0.0188 -0.0086 0.0534  -0.0106-0.0414
[  michaelt@node8  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit.perm2.vs3.ped | head -n 10
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9    PC10
POOL98_01RL     POOL98_01RL     0       0       0       1       NA      NA      NA      1       0.0079  0.0160  0.0720  0.0296  0.0320  -0.0555 -0.0447 0.0014  0.0010 -0.0499
POOL83_08RL     POOL83_08RL     0       0       0       2       NA      NA      NA      8       0.0157  0.0095  -0.0198 -0.0248 -0.0206 -0.0242 0.0153  0.0448  0.0243 0.0067
POOL77_08RL     POOL77_08RL     0       0       0       2       NA      NA      NA      8       0.0143  -0.0026 -0.0279 -0.0273 0.0338  -0.0194 0.0555  0.0160  -0.00530.0202
POOL02_03RL     POOL02_03RL     0       0       0       1       2       NA      NA      3       0.0136  0.0057  0.0166  0.0415  -0.0098 0.0027  -0.0120 -0.0008 -0.01870.0112
POOL33_03RL     POOL33_03RL     0       0       0       2       2       NA      NA      3       0.0137  0.0162  -0.0260 -0.0403 -0.0116 -0.0230 -0.0308 0.0320  -0.0137-0.0560
POOL157_04RL    POOL157_04RL    0       0       0       1       NA      NA      NA      4       0.0128  0.0025  -0.0309 0.0374  -0.0126 0.0054  0.0047  0.0157  -0.02910.0530
POOL88_02RL     POOL88_02RL     0       0       0       2       2       NA      NA      2       0.0118  0.0045  -0.0299 0.0170  0.0068  -0.0149 -0.0094 -0.0493 -0.0122-0.0149
POOL109_11RL    POOL109_11RL    0       0       0       2       2       NA      NA      11      0.0160  0.0021  -0.0172 -0.0269 -0.0312 0.0343  0.0326  -0.0096 -0.00620.0452
POOL158_08RL    POOL158_08RL    0       0       0       1       1       1       NA      8       0.0106  0.0138  0.0268  -0.0544 0.0063  -0.0188 -0.0086 0.0534  -0.0106-
[  michaelt@node8  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit.perm3.vs3.ped | head -n 10
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9    PC10
POOL98_01RL     POOL98_01RL     0       0       0       1       NA      NA      NA      1       0.0079  0.0160  0.0720  0.0296  0.0320  -0.0555 -0.0447 0.0014  0.0010 -0.0499
POOL83_08RL     POOL83_08RL     0       0       0       1       NA      NA      NA      8       0.0157  0.0095  -0.0198 -0.0248 -0.0206 -0.0242 0.0153  0.0448  0.0243 0.0067
POOL77_08RL     POOL77_08RL     0       0       0       2       NA      NA      NA      8       0.0143  -0.0026 -0.0279 -0.0273 0.0338  -0.0194 0.0555  0.0160  -0.00530.0202
POOL02_03RL     POOL02_03RL     0       0       0       2       2       NA      NA      3       0.0136  0.0057  0.0166  0.0415  -0.0098 0.0027  -0.0120 -0.0008 -0.01870.0112
POOL33_03RL     POOL33_03RL     0       0       0       1       2       NA      NA      3       0.0137  0.0162  -0.0260 -0.0403 -0.0116 -0.0230 -0.0308 0.0320  -0.0137-0.0560
POOL157_04RL    POOL157_04RL    0       0       0       2       NA      NA      NA      4       0.0128  0.0025  -0.0309 0.0374  -0.0126 0.0054  0.0047  0.0157  -0.02910.0530
POOL88_02RL     POOL88_02RL     0       0       0       2       2       NA      NA      2       0.0118  0.0045  -0.0299 0.0170  0.0068  -0.0149 -0.0094 -0.0493 -0.0122-0.0149
POOL109_11RL    POOL109_11RL    0       0       0       2       1       NA      NA      11      0.0160  0.0021  -0.0172 -0.0269 -0.0312 0.0343  0.0326  -0.0096 -0.00620.0452
POOL158_08RL    POOL158_08RL    0       0       0       2       2       1       NA      8       0.0106  0.0138  0.0268  -0.0544 0.0063  -0.0188 -0.0086 0.0534  -0.0106-0.0414
[  michaelt@node8  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit.perm4.vs3.ped | head -n 10
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9    PC10
POOL98_01RL     POOL98_01RL     0       0       0       2       NA      NA      NA      1       0.0079  0.0160  0.0720  0.0296  0.0320  -0.0555 -0.0447 0.0014  0.0010 -0.0499
POOL83_08RL     POOL83_08RL     0       0       0       2       NA      NA      NA      8       0.0157  0.0095  -0.0198 -0.0248 -0.0206 -0.0242 0.0153  0.0448  0.0243 0.0067
POOL77_08RL     POOL77_08RL     0       0       0       2       NA      NA      NA      8       0.0143  -0.0026 -0.0279 -0.0273 0.0338  -0.0194 0.0555  0.0160  -0.00530.0202
POOL02_03RL     POOL02_03RL     0       0       0       1       1       NA      NA      3       0.0136  0.0057  0.0166  0.0415  -0.0098 0.0027  -0.0120 -0.0008 -0.01870.0112
POOL33_03RL     POOL33_03RL     0       0       0       2       2       NA      NA      3       0.0137  0.0162  -0.0260 -0.0403 -0.0116 -0.0230 -0.0308 0.0320  -0.0137-0.0560
POOL157_04RL    POOL157_04RL    0       0       0       1       NA      NA      NA      4       0.0128  0.0025  -0.0309 0.0374  -0.0126 0.0054  0.0047  0.0157  -0.02910.0530
POOL88_02RL     POOL88_02RL     0       0       0       1       2       NA      NA      2       0.0118  0.0045  -0.0299 0.0170  0.0068  -0.0149 -0.0094 -0.0493 -0.0122-0.0149
POOL109_11RL    POOL109_11RL    0       0       0       1       1       NA      NA      11      0.0160  0.0021  -0.0172 -0.0269 -0.0312 0.0343  0.0326  -0.0096 -0.00620.0452
POOL158_08RL    POOL158_08RL    0       0       0       1       1       1       NA      8       0.0106  0.0138  0.0268  -0.0544 0.0063  -0.0188 -0.0086 0.0534  -0.0106-0.0414
[  michaelt@node8  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit.perm5.vs3.ped | head -n 10
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9    PC10
POOL98_01RL     POOL98_01RL     0       0       0       2       NA      NA      NA      1       0.0079  0.0160  0.0720  0.0296  0.0320  -0.0555 -0.0447 0.0014  0.0010 -0.0499
POOL83_08RL     POOL83_08RL     0       0       0       1       NA      NA      NA      8       0.0157  0.0095  -0.0198 -0.0248 -0.0206 -0.0242 0.0153  0.0448  0.0243 0.0067
POOL77_08RL     POOL77_08RL     0       0       0       1       NA      NA      NA      8       0.0143  -0.0026 -0.0279 -0.0273 0.0338  -0.0194 0.0555  0.0160  -0.00530.0202
POOL02_03RL     POOL02_03RL     0       0       0       1       2       NA      NA      3       0.0136  0.0057  0.0166  0.0415  -0.0098 0.0027  -0.0120 -0.0008 -0.01870.0112
POOL33_03RL     POOL33_03RL     0       0       0       2       1       NA      NA      3       0.0137  0.0162  -0.0260 -0.0403 -0.0116 -0.0230 -0.0308 0.0320  -0.0137-0.0560
POOL157_04RL    POOL157_04RL    0       0       0       2       NA      NA      NA      4       0.0128  0.0025  -0.0309 0.0374  -0.0126 0.0054  0.0047  0.0157  -0.02910.0530
POOL88_02RL     POOL88_02RL     0       0       0       2       1       NA      NA      2       0.0118  0.0045  -0.0299 0.0170  0.0068  -0.0149 -0.0094 -0.0493 -0.0122-0.0149
POOL109_11RL    POOL109_11RL    0       0       0       2       1       NA      NA      11      0.0160  0.0021  -0.0172 -0.0269 -0.0312 0.0343  0.0326  -0.0096 -0.00620.0452
POOL158_08RL    POOL158_08RL    0       0       0       2       2       1       NA      8       0.0106  0.0138  0.0268  -0.0544 0.0063  -0.0188 -0.0086 0.0534  -0.0106-0.0414
[  michaelt@node8  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit.perm1.vs3.ped | head -n 150 | tail -n 10
POOL169_06RL    POOL169_06RL    0       0       0       2       2       NA      NA      6       0.0152  0.0020  0.0289  -0.0016 0.0194  0.0148  -0.0678 0.0267  -0.02180.0006
POOL54_05RL     POOL54_05RL     0       0       0       2       1       NA      NA      5       0.0100  0.0130  -0.0148 0.0023  0.0079  0.0065  0.0110  -0.0392 0.0006 0.0416
POOL70_07RL     POOL70_07RL     0       0       0       2       NA      NA      NA      7       0.0053  0.0011  0.0204  0.0040  0.0478  -0.0230 -0.0075 0.0162  0.0133 0.0037
POOL99_11RL     POOL99_11RL     0       0       0       2       2       NA      NA      11      0.0088  0.0084  -0.0249 0.0309  0.0547  -0.0064 -0.0122 -0.0504 0.0408 -0.0194
POOL09_06RL     POOL09_06RL     0       0       0       2       2       NA      NA      6       0.0122  0.0110  -0.0257 -0.0438 0.0198  0.0818  -0.0100 0.0120  -0.01520.0244
POOL125_05RL    POOL125_05RL    0       0       0       1       2       2       NA      5       0.0093  0.0019  0.0155  0.0372  0.0251  0.0403  0.0090  0.0138  -0.01300.0100
POOL157_11RL    POOL157_11RL    0       0       0       1       2       1       2       11      0.0142  0.0123  0.0236  -0.0230 0.0025  0.0099  0.0349  0.0111  0.0092 -0.0064
POOL80_01RL     POOL80_01RL     0       0       0       1       NA      NA      NA      1       0.0119  0.0110  -0.0337 -0.0046 -0.0187 0.0664  0.0401  0.0072  0.0304 -0.0416
POOL23_06RL     POOL23_06RL     0       0       0       2       2       NA      NA      6       0.0118  0.0045  0.0168  0.0124  -0.0158 0.0022  0.0210  -0.0111 0.0668 -0.0002
POOL173_01RL    POOL173_01RL    0       0       0       1       2       NA      NA      1       0.0104  0.0069  0.0257  -0.0448 0.0234  0.0036  0.0458  0.0089  0.0125 -0.0064
[  michaelt@node8  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit.perm2.vs3.ped | head -n 150 | tail -n 10
POOL169_06RL    POOL169_06RL    0       0       0       2       2       NA      NA      6       0.0152  0.0020  0.0289  -0.0016 0.0194  0.0148  -0.0678 0.0267  -0.02180.0006
POOL54_05RL     POOL54_05RL     0       0       0       1       2       NA      NA      5       0.0100  0.0130  -0.0148 0.0023  0.0079  0.0065  0.0110  -0.0392 0.0006 0.0416
POOL70_07RL     POOL70_07RL     0       0       0       1       NA      NA      NA      7       0.0053  0.0011  0.0204  0.0040  0.0478  -0.0230 -0.0075 0.0162  0.0133 0.0037
POOL99_11RL     POOL99_11RL     0       0       0       2       2       NA      NA      11      0.0088  0.0084  -0.0249 0.0309  0.0547  -0.0064 -0.0122 -0.0504 0.0408 -0.0194
POOL09_06RL     POOL09_06RL     0       0       0       1       2       NA      NA      6       0.0122  0.0110  -0.0257 -0.0438 0.0198  0.0818  -0.0100 0.0120  -0.01520.0244
POOL125_05RL    POOL125_05RL    0       0       0       1       2       2       NA      5       0.0093  0.0019  0.0155  0.0372  0.0251  0.0403  0.0090  0.0138  -0.01300.0100
POOL157_11RL    POOL157_11RL    0       0       0       1       2       1       2       11      0.0142  0.0123  0.0236  -0.0230 0.0025  0.0099  0.0349  0.0111  0.0092 -0.0064
POOL80_01RL     POOL80_01RL     0       0       0       1       NA      NA      NA      1       0.0119  0.0110  -0.0337 -0.0046 -0.0187 0.0664  0.0401  0.0072  0.0304 -0.0416
POOL23_06RL     POOL23_06RL     0       0       0       2       2       NA      NA      6       0.0118  0.0045  0.0168  0.0124  -0.0158 0.0022  0.0210  -0.0111 0.0668 -0.0002
POOL173_01RL    POOL173_01RL    0       0       0       2       1       NA      NA      1       0.0104  0.0069  0.0257  -0.0448 0.0234  0.0036  0.0458  0.0089  0.0125 -0.0064
[  michaelt@node8  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit.perm3.vs3.ped | head -n 150 | tail -n 10
POOL169_06RL    POOL169_06RL    0       0       0       1       2       NA      NA      6       0.0152  0.0020  0.0289  -0.0016 0.0194  0.0148  -0.0678 0.0267  -0.02180.0006
POOL54_05RL     POOL54_05RL     0       0       0       2       1       NA      NA      5       0.0100  0.0130  -0.0148 0.0023  0.0079  0.0065  0.0110  -0.0392 0.0006 0.0416
POOL70_07RL     POOL70_07RL     0       0       0       1       NA      NA      NA      7       0.0053  0.0011  0.0204  0.0040  0.0478  -0.0230 -0.0075 0.0162  0.0133 0.0037
POOL99_11RL     POOL99_11RL     0       0       0       1       1       NA      NA      11      0.0088  0.0084  -0.0249 0.0309  0.0547  -0.0064 -0.0122 -0.0504 0.0408 -0.0194
POOL09_06RL     POOL09_06RL     0       0       0       1       2       NA      NA      6       0.0122  0.0110  -0.0257 -0.0438 0.0198  0.0818  -0.0100 0.0120  -0.01520.0244
POOL125_05RL    POOL125_05RL    0       0       0       2       1       2       NA      5       0.0093  0.0019  0.0155  0.0372  0.0251  0.0403  0.0090  0.0138  -0.01300.0100
POOL157_11RL    POOL157_11RL    0       0       0       2       2       1       2       11      0.0142  0.0123  0.0236  -0.0230 0.0025  0.0099  0.0349  0.0111  0.0092 -0.0064
POOL80_01RL     POOL80_01RL     0       0       0       2       NA      NA      NA      1       0.0119  0.0110  -0.0337 -0.0046 -0.0187 0.0664  0.0401  0.0072  0.0304 -0.0416
POOL23_06RL     POOL23_06RL     0       0       0       2       1       NA      NA      6       0.0118  0.0045  0.0168  0.0124  -0.0158 0.0022  0.0210  -0.0111 0.0668 -0.0002
POOL173_01RL    POOL173_01RL    0       0       0       1       2       NA      NA      1       0.0104  0.0069  0.0257  -0.0448 0.0234  0.0036  0.0458  0.0089  0.0125 -0.0064
[  michaelt@node8  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit.perm4.vs3.ped | head -n 150 | tail -n 10
POOL169_06RL    POOL169_06RL    0       0       0       2       2       NA      NA      6       0.0152  0.0020  0.0289  -0.0016 0.0194  0.0148  -0.0678 0.0267  -0.02180.0006
POOL54_05RL     POOL54_05RL     0       0       0       2       2       NA      NA      5       0.0100  0.0130  -0.0148 0.0023  0.0079  0.0065  0.0110  -0.0392 0.0006 0.0416
POOL70_07RL     POOL70_07RL     0       0       0       2       NA      NA      NA      7       0.0053  0.0011  0.0204  0.0040  0.0478  -0.0230 -0.0075 0.0162  0.0133 0.0037
POOL99_11RL     POOL99_11RL     0       0       0       2       1       NA      NA      11      0.0088  0.0084  -0.0249 0.0309  0.0547  -0.0064 -0.0122 -0.0504 0.0408 -0.0194
POOL09_06RL     POOL09_06RL     0       0       0       2       2       NA      NA      6       0.0122  0.0110  -0.0257 -0.0438 0.0198  0.0818  -0.0100 0.0120  -0.01520.0244
POOL125_05RL    POOL125_05RL    0       0       0       1       2       1       NA      5       0.0093  0.0019  0.0155  0.0372  0.0251  0.0403  0.0090  0.0138  -0.01300.0100
POOL157_11RL    POOL157_11RL    0       0       0       1       1       1       2       11      0.0142  0.0123  0.0236  -0.0230 0.0025  0.0099  0.0349  0.0111  0.0092 -0.0064
POOL80_01RL     POOL80_01RL     0       0       0       2       NA      NA      NA      1       0.0119  0.0110  -0.0337 -0.0046 -0.0187 0.0664  0.0401  0.0072  0.0304 -0.0416
POOL23_06RL     POOL23_06RL     0       0       0       1       2       NA      NA      6       0.0118  0.0045  0.0168  0.0124  -0.0158 0.0022  0.0210  -0.0111 0.0668 -0.0002
POOL173_01RL    POOL173_01RL    0       0       0       1       2       NA      NA      1       0.0104  0.0069  0.0257  -0.0448 0.0234  0.0036  0.0458  0.0089  0.0125 -0.0064
[  michaelt@node8  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit.perm5.vs3.ped | head -n 150 | tail -n 10
POOL169_06RL    POOL169_06RL    0       0       0       2       2       NA      NA      6       0.0152  0.0020  0.0289  -0.0016 0.0194  0.0148  -0.0678 0.0267  -0.02180.0006
POOL54_05RL     POOL54_05RL     0       0       0       2       2       NA      NA      5       0.0100  0.0130  -0.0148 0.0023  0.0079  0.0065  0.0110  -0.0392 0.0006 0.0416
POOL70_07RL     POOL70_07RL     0       0       0       1       NA      NA      NA      7       0.0053  0.0011  0.0204  0.0040  0.0478  -0.0230 -0.0075 0.0162  0.0133 0.0037
POOL99_11RL     POOL99_11RL     0       0       0       2       1       NA      NA      11      0.0088  0.0084  -0.0249 0.0309  0.0547  -0.0064 -0.0122 -0.0504 0.0408 -0.0194
POOL09_06RL     POOL09_06RL     0       0       0       1       1       NA      NA      6       0.0122  0.0110  -0.0257 -0.0438 0.0198  0.0818  -0.0100 0.0120  -0.01520.0244
POOL125_05RL    POOL125_05RL    0       0       0       1       1       2       NA      5       0.0093  0.0019  0.0155  0.0372  0.0251  0.0403  0.0090  0.0138  -0.01300.0100
POOL157_11RL    POOL157_11RL    0       0       0       1       1       1       1       11      0.0142  0.0123  0.0236  -0.0230 0.0025  0.0099  0.0349  0.0111  0.0092 -0.0064
POOL80_01RL     POOL80_01RL     0       0       0       2       NA      NA      NA      1       0.0119  0.0110  -0.0337 -0.0046 -0.0187 0.0664  0.0401  0.0072  0.0304 -0.0416
POOL23_06RL     POOL23_06RL     0       0       0       2       2       NA      NA      6       0.0118  0.0045  0.0168  0.0124  -0.0158 0.0022  0.0210  -0.0111 0.0668 -0.0002
POOL173_01RL    POOL173_01RL    0       0       0       1       1       NA      NA      1       0.0104  0.0069  0.0257  -0.0448 0.0234  0.0036  0.0458  0.0089  0.0125 -0.0064
~~~

cp -p /home/michaelt/Scripts/Bash/BashScript.InputVarsPlusTime.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs1.sh 
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs1.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS TruePheno Pheno1 Exonic.Nonsynonymous OLD3
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS TruePheno Pheno2 Exonic.Nonsynonymous OLD3
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS TruePheno Pheno3 Exonic.Nonsynonymous OLD3
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS TruePheno Pheno4 Exonic.Nonsynonymous OLD3

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS FakePheno Pheno1 Exonic.Nonsynonymous OLD3
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS FakePheno Pheno2 Exonic.Nonsynonymous OLD3
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS FakePheno Pheno3 Exonic.Nonsynonymous OLD3
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS FakePheno Pheno4 Exonic.Nonsynonymous OLD3

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs1.sh

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs1.sh

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.sh

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs2.sh


bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno1 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno2 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno3 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno4 Exonic.Nonsynonymous

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno1 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno2 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno3 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno4 Exonic.Nonsynonymous

for i in {1..1000}
do
	if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm${i}.epacts ] ; then
		echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno1.Exonic.Nonsynonymous
	fi
	if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm${i}.epacts ] ; then
		echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno2.Exonic.Nonsynonymous
	fi
	if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm${i}.epacts ] ; then
		echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno3.Exonic.Nonsynonymous
	fi
	if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm${i}.epacts ] ; then
		echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno4.Exonic.Nonsynonymous
	fi
done

#Only /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno1.Exonic.Nonsynonymous has entries -- going to rerun /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh with the following line of code
#for i in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno1.Exonic.Nonsynonymous`

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.GetFDRValsPerGeneFromPermutations.vs1.output | vi -
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.GetFDRValsPerGeneFromPermutations.vs1.output | vi -
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.GetFDRValsPerGeneFromPermutations.vs1.output | vi -
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.GetFDRValsPerGeneFromPermutations.vs1.output | vi -

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.SUMSTAT.CalcPermBasedPVal.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output | sort -t, -rg -k 8,8 | vi -
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.SUMSTAT.CalcPermBasedPVal.vs1.TruePheno.Pheno2.Exonic.Nonsynonymous.output | sort -t, -rg -k 8,8 | vi -
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.SUMSTAT.CalcPermBasedPVal.vs1.TruePheno.Pheno3.Exonic.Nonsynonymous.output | sort -t, -rg -k 8,8 | vi -
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.SUMSTAT.CalcPermBasedPVal.vs1.TruePheno.Pheno4.Exonic.Nonsynonymous.output | sort -t, -rg -k 8,8 | vi -

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.QQPlotUsingPermutations.vs1.R

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.QQPlotUsingPermutations.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs2.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg HIVAcquisition_Nonsyn_SkatO
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.QQPlotUsingPermutations.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs2.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg HIVAcquisitionHE_Nonsyn_SkatO
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.QQPlotUsingPermutations.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs2.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg AIDSProgression_Nonsyn_SkatO
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.QQPlotUsingPermutations.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs2.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg AIDSProgressionExtr_Nonsyn_SkatO

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/.

scp -p  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/.
scp -p  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/.
scp -p  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/.
scp -p  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/.

#TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs2.sh MM5 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno1 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs2.sh MM5 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno2 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs2.sh MM5 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno3 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs2.sh MM5 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno4 Exonic.Nonsynonymous

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs2.sh MM5 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno1 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs2.sh MM5 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno2 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs2.sh MM5 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno3 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs2.sh MM5 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno4 Exonic.Nonsynonymous

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.GetFDRValsPerGeneFromPermutations.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.TrueVsFakePerRankCmSum.vs1.R

#Because of the 'non-White subset Annovar .bim.wGenes' files being used originally issue
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/WRONG1.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/WRONG1.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/WRONG1.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/WRONG1.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Results/Checkpoints
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Results/Checkpoints/7thRun
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Results/Checkpoints/7thRun/20150302

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno*/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno*.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno*/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno*.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno*/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno*.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Results/Checkpoints/7thRun/20150302/. 

##cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm*.epacts | awk '{ print $10 }' | grep -v -E 'NA|PVALUE' | sort -rg -k 1,1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.permAll.epacts.justpVals
##cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm*.epacts | awk '{ print $10 }' | grep -v -E 'NA|PVALUE' | sort -rg -k 1,1 | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.permAll.epacts.NullDistrQQPlotCheck.vs1.jpeg\", width=2500, height=2000, res=300); xVals1 <- c(1:nrow(Data1))/(nrow(Data1)); plot(rev(-log10(xVals1)), -log10(Data1[,1]), xlab=\"-log10 Expected p-Values\", ylim=c(0,6), ylab=\"-log10 Observed p-Values\", main=\"HIVAcquisition -- All Permuted p-Values\", col=\"RED\"); abline(a=0,b=1); dev.off();"

#CHECK_1 -- plot all permuted p-vals at oncee against 'expected' in a QQ-plot to see that things look correct

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.permAll.epacts.NullDistrQQPlotCheck.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.permAll.epacts.NullDistrQQPlotCheck.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.permAll.epacts.NullDistrQQPlotCheck.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.permAll.epacts.NullDistrQQPlotCheck.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/.

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/.

#cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.QQPlotUsingPermutations.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.QQPlotUsingPermutations.vs2.R

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justpVals /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/. 
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justpVals /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/. 
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justpVals /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/. 
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justpVals /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/. 

#doesn't work using local installation of ImageMagick -- need Aleksandr to install a shared version
#convert +append /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.obspValHist.QQPlotUsingPermutations.jpeg 

#20150330
#Retroactively including code for these 'gene/variant level result summary outputs', following the logic of excel sheet exploration, here

echo ".3" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .3) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .3) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".5" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".6" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".65" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .65) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .65) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo "1" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= 1) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= 1) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`

~~~
[  michaelt@node1  ~]$echo ".3" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .3) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .3) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.3 23 157
[  michaelt@node1  ~]$echo ".5" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.5 69 529
[  michaelt@node1  ~]$echo ".6" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.6 172 1430
[  michaelt@node1  ~]$echo ".65" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .65) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .65) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.65 212 1789
[  michaelt@node1  ~]$echo "1" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= 1) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= 1) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
1 473 3703
~~~



#20150311
#Want to test whether the skewed distribution of p-values from skat-O is occuring because of the 'minimum p-value' search across rho values, so running epacts with a) skat w/ adjustment b) skat w/ no adjustment and c) wcnt method which I'm currently approximating is equivalent to skat-O's burden test (since I can't directly pass/force rho values to skat through epacts, though with the R skat function you can do this)
#goal is to see if the p-value distributions, both for observed and permuted statistics, look more 'normal'
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno1
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno2
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/FakePheno
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/FakePheno/Pheno1
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/FakePheno/Pheno2
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs3.pt2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs3.pt2.ForNullTests.sh

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs3.pt2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs3.pt2.ForNullTests.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests

#In pwd: /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations 
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations]$mv *Pheno1*run* TruePheno/Pheno1/Exonic.Nonsynonymous/.
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations]$mv *Pheno2*run* TruePheno/Pheno2/Exonic.Nonsynonymous/.
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations]$mv *Pheno1*perm* FakePheno/Pheno1/Exonic.Nonsynonymous/.
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations]$mv *Pheno2*perm* FakePheno/Pheno2/Exonic.Nonsynonymous/.

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno1.maf1.skat.wAdjust.run1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print join("_", @vals1[1..$#vals1]);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno1.maf1.skat.wAdjust.run1.epacts.GeneIDList 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno1.maf1.skat.wAdjust.run1.epacts.GeneIDList /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno1.maf1.skat.wAdjust TruePheno Pheno1 Exonic.Nonsynonymous
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.skat.wAdjust.TruePheno.Pheno1.Exonic.Nonsynonymous.output
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.skat.wAdjust.TruePheno.Pheno1.Exonic.Nonsynonymous.output | grep -vw NA | /home/shared/software/R-3.0.1/bin/R -q -e "library(qvalue, lib=\"/home/michaelt/Software/R-3.0.1/library\"); Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd), signif(qvalue(apply(Data1,1,mean))\$qvalues, 5)));" | grep -v ^\> | sort -rg -k 2,2 | head -n -1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.skat.wAdjust.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.skat.wAdjust.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput | awk '{ print $2 }' | sort -g | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno1.maf1.skat.wAdjust.runAll.epacts.pValPlots.vs1.jpeg\", width=5000, height=2000, res=300); par(mfrow=c(1,2)); xVals1 <- c(1:nrow(Data1))/(nrow(Data1)); plot(rev(-log10(xVals1)), rev(-log10(Data1[,1])), xlab=\"-log10 Expected p-Values\", ylim=c(0,6), ylab=\"-log10 Observed p-Values\", main=\"HIVAcquisition skat wAdj -- All Observed p-Values\", col=\"RED\"); abline(a=0,b=1); hist(Data1[,1], xlab=\"p-Values\", main=\"HIVAcquisition skat wAdjust -- All Observed p-Values\", breaks=50); dev.off(); print(dim(Data1));"
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno1.maf1.skat.noAdjust.run1.epacts | awk '{ print $10 }' | grep -v -E 'NA|PVALUE' | sort -g | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno1.maf1.skat.noAdjust.run1.epacts.pValPlots.vs1.jpeg\", width=5000, height=2000, res=300); par(mfrow=c(1,2)); xVals1 <- c(1:nrow(Data1))/(nrow(Data1)); plot(rev(-log10(xVals1)), rev(-log10(Data1[,1])), xlab=\"-log10 Expected p-Values\", ylim=c(0,6), ylab=\"-log10 Observed p-Values\", main=\"HIVAcquisition skat noAdj -- All Observed p-Values\", col=\"RED\"); abline(a=0,b=1); hist(Data1[,1], xlab=\"p-Values\", main=\"HIVAcquisition skat noAdjust -- All Observed p-Values\", breaks=50); dev.off(); print(dim(Data1));" 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno1.maf1.wcnt.run1.epacts | awk '{ print $10 }' | grep -v -E 'NA|PVALUE' | sort -g | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno1.maf1.wcnt.run1.epacts.pValPlots.vs1.jpeg\", width=5000, height=2000, res=300); par(mfrow=c(1,2)); xVals1 <- c(1:nrow(Data1))/(nrow(Data1)); plot(rev(-log10(xVals1)), rev(-log10(Data1[,1])), xlab=\"-log10 Expected p-Values\", ylim=c(0,6), ylab=\"-log10 Observed p-Values\", main=\"HIVAcquisition wcnt -- All Observed p-Values\", col=\"RED\"); abline(a=0,b=1); hist(Data1[,1], xlab=\"p-Values\", main=\"HIVAcquisition wcnt -- All Observed p-Values\", breaks=50); dev.off(); print(dim(Data1));" 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno1.maf1.skat.wAdjust.perm*.epacts | awk '{ print $10 }' | grep -v -E 'NA|PVALUE' | sort -g | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno1.maf1.skat.wAdjust.permAll.epacts.pValPlots.vs1.jpeg\", width=5000, height=2000, res=300); par(mfrow=c(1,2)); xVals1 <- c(1:nrow(Data1))/(nrow(Data1)); plot(rev(-log10(xVals1)), rev(-log10(Data1[,1])), xlab=\"-log10 Expected p-Values\", ylim=c(0,6), ylab=\"-log10 Observed p-Values\", main=\"HIVAcquisition skat wAdj -- All Observed p-Values\", col=\"RED\"); abline(a=0,b=1); hist(Data1[,1], xlab=\"p-Values\", main=\"HIVAcquisition skat wAdj -- All Permuted p-Values\", breaks=50); dev.off(); print(dim(Data1));"
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno1.maf1.skat.noAdjust.perm*.epacts | awk '{ print $10 }' | grep -v -E 'NA|PVALUE' | sort -g | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno1.maf1.skat.noAdjust.permAll.epacts.pValPlots.vs1.jpeg\", width=5000, height=2000, res=300); par(mfrow=c(1,2)); xVals1 <- c(1:nrow(Data1))/(nrow(Data1)); plot(rev(-log10(xVals1)), rev(-log10(Data1[,1])), xlab=\"-log10 Expected p-Values\", ylim=c(0,6), ylab=\"-log10 Observed p-Values\", main=\"HIVAcquisition skat noAdj -- All Observed p-Values\", col=\"RED\"); abline(a=0,b=1); hist(Data1[,1], xlab=\"p-Values\", main=\"HIVAcquisition skat noAdj -- All Permuted p-Values\", breaks=50); dev.off(); print(dim(Data1));"
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno1.maf1.wcnt.perm*.epacts | awk '{ print $10 }' | grep -v -E 'NA|PVALUE' | sort -g | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno1.maf1.wcnt.permAll.epacts.pValPlots.vs1.jpeg\", width=5000, height=2000, res=300); par(mfrow=c(1,2)); xVals1 <- c(1:nrow(Data1))/(nrow(Data1)); plot(rev(-log10(xVals1)), rev(-log10(Data1[,1])), xlab=\"-log10 Expected p-Values\", ylim=c(0,6), ylab=\"-log10 Observed p-Values\", main=\"HIVAcquisition wcnt -- All Observed p-Values\", col=\"RED\"); abline(a=0,b=1); hist(Data1[,1], xlab=\"p-Values\", main=\"HIVAcquisition wcnt -- All Permuted p-Values\", breaks=50); dev.off(); print(dim(Data1));"

#dim calls were all either 473 or 473000 for all runs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno2.maf1.skat.wAdjust.run1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print join("_", @vals1[1..$#vals1]);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno2.maf1.skat.wAdjust.run1.epacts.GeneIDList 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno2.maf1.skat.wAdjust.run1.epacts.GeneIDList /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno2.maf1.skat.wAdjust TruePheno Pheno2 Exonic.Nonsynonymous
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.skat.wAdjust.TruePheno.Pheno2.Exonic.Nonsynonymous.output
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.skat.wAdjust.TruePheno.Pheno2.Exonic.Nonsynonymous.output | grep -vw NA | /home/shared/software/R-3.0.1/bin/R -q -e "library(qvalue, lib=\"/home/michaelt/Software/R-3.0.1/library\"); Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd), signif(qvalue(apply(Data1,1,mean))\$qvalues, 5)));" | grep -v ^\> | sort -rg -k 2,2 | head -n -1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.skat.wAdjust.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.skat.wAdjust.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput | awk '{ print $2 }' | sort -g | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno2.maf1.skat.wAdjust.runAll.epacts.pValPlots.vs1.jpeg\", width=5000, height=2000, res=300); par(mfrow=c(1,2)); xVals1 <- c(1:nrow(Data1))/(nrow(Data1)); plot(rev(-log10(xVals1)), rev(-log10(Data1[,1])), xlab=\"-log10 Expected p-Values\", ylim=c(0,6), ylab=\"-log10 Observed p-Values\", main=\"HIVAcquisitionHE skat wAdj -- All Observed p-Values\", col=\"RED\"); abline(a=0,b=1); hist(Data1[,1], xlab=\"p-Values\", main=\"HIVAcquisitionHE skat wAdjust -- All Observed p-Values\", breaks=50); dev.off(); print(dim(Data1));"
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno2.maf1.skat.noAdjust.run1.epacts | awk '{ print $10 }' | grep -v -E 'NA|PVALUE' | sort -g | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno2.maf1.skat.noAdjust.run1.epacts.pValPlots.vs1.jpeg\", width=5000, height=2000, res=300); par(mfrow=c(1,2)); xVals1 <- c(1:nrow(Data1))/(nrow(Data1)); plot(rev(-log10(xVals1)), rev(-log10(Data1[,1])), xlab=\"-log10 Expected p-Values\", ylim=c(0,6), ylab=\"-log10 Observed p-Values\", main=\"HIVAcquisitionHE skat noAdj -- All Observed p-Values\", col=\"RED\"); abline(a=0,b=1); hist(Data1[,1], xlab=\"p-Values\", main=\"HIVAcquisitionHE skat noAdjust -- All Observed p-Values\", breaks=50); dev.off(); print(dim(Data1));" 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno2.maf1.wcnt.run1.epacts | awk '{ print $10 }' | grep -v -E 'NA|PVALUE' | sort -g | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno2.maf1.wcnt.run1.epacts.pValPlots.vs1.jpeg\", width=5000, height=2000, res=300); par(mfrow=c(1,2)); xVals1 <- c(1:nrow(Data1))/(nrow(Data1)); plot(rev(-log10(xVals1)), rev(-log10(Data1[,1])), xlab=\"-log10 Expected p-Values\", ylim=c(0,6), ylab=\"-log10 Observed p-Values\", main=\"HIVAcquisitionHE wcnt -- All Observed p-Values\", col=\"RED\"); abline(a=0,b=1); hist(Data1[,1], xlab=\"p-Values\", main=\"HIVAcquisitionHE wcnt -- All Observed p-Values\", breaks=50); dev.off(); print(dim(Data1));" 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno2.maf1.skat.wAdjust.perm*.epacts | awk '{ print $10 }' | grep -v -E 'NA|PVALUE' | sort -g | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno2.maf1.skat.wAdjust.permAll.epacts.pValPlots.vs1.jpeg\", width=5000, height=2000, res=300); par(mfrow=c(1,2)); xVals1 <- c(1:nrow(Data1))/(nrow(Data1)); plot(rev(-log10(xVals1)), rev(-log10(Data1[,1])), xlab=\"-log10 Expected p-Values\", ylim=c(0,6), ylab=\"-log10 Observed p-Values\", main=\"HIVAcquisitionHE skat wAdj -- All Observed p-Values\", col=\"RED\"); abline(a=0,b=1); hist(Data1[,1], xlab=\"p-Values\", main=\"HIVAcquisitionHE skat wAdj -- All Permuted p-Values\", breaks=50); dev.off(); print(dim(Data1));"
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno2.maf1.skat.noAdjust.perm*.epacts | awk '{ print $10 }' | grep -v -E 'NA|PVALUE' | sort -g | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno2.maf1.skat.noAdjust.permAll.epacts.pValPlots.vs1.jpeg\", width=5000, height=2000, res=300); par(mfrow=c(1,2)); xVals1 <- c(1:nrow(Data1))/(nrow(Data1)); plot(rev(-log10(xVals1)), rev(-log10(Data1[,1])), xlab=\"-log10 Expected p-Values\", ylim=c(0,6), ylab=\"-log10 Observed p-Values\", main=\"HIVAcquisitionHE skat noAdj -- All Observed p-Values\", col=\"RED\"); abline(a=0,b=1); hist(Data1[,1], xlab=\"p-Values\", main=\"HIVAcquisitionHE skat noAdj -- All Permuted p-Values\", breaks=50); dev.off(); print(dim(Data1));"
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno2.maf1.wcnt.perm*.epacts | awk '{ print $10 }' | grep -v -E 'NA|PVALUE' | sort -g | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno2.maf1.wcnt.permAll.epacts.pValPlots.vs1.jpeg\", width=5000, height=2000, res=300); par(mfrow=c(1,2)); xVals1 <- c(1:nrow(Data1))/(nrow(Data1)); plot(rev(-log10(xVals1)), rev(-log10(Data1[,1])), xlab=\"-log10 Expected p-Values\", ylim=c(0,6), ylab=\"-log10 Observed p-Values\", main=\"HIVAcquisitionHE wcnt -- All Observed p-Values\", col=\"RED\"); abline(a=0,b=1); hist(Data1[,1], xlab=\"p-Values\", main=\"HIVAcquisitionHE wcnt -- All Permuted p-Values\", breaks=50); dev.off(); print(dim(Data1));"

# 458 and 458000

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno1.maf1.skat.wAdjust.runAll.epacts.pValPlots.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno1.maf1.skat.noAdjust.run1.epacts.pValPlots.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno1.maf1.wcnt.run1.epacts.pValPlots.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno1.maf1.skat.wAdjust.permAll.epacts.pValPlots.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno1.maf1.skat.noAdjust.permAll.epacts.pValPlots.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno1.maf1.wcnt.permAll.epacts.pValPlots.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno2.maf1.skat.wAdjust.runAll.epacts.pValPlots.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno2.maf1.skat.noAdjust.run1.epacts.pValPlots.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno2.maf1.wcnt.run1.epacts.pValPlots.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno2.maf1.skat.wAdjust.permAll.epacts.pValPlots.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno2.maf1.skat.noAdjust.permAll.epacts.pValPlots.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.NullTests.Exonic.Nonsynonymous.Pheno2.maf1.wcnt.permAll.epacts.pValPlots.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/NullTests/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/.

#CHECK_1: Redo code for pheno2s since $pheno1 was accidentally used  for them instead of $pheno2 in the multirun/permutation launcher code

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts 

#20150325
#Adding new annotated .vcf format option now using vcf-annotate (http://vcftools.sourceforge.net/perl_module.html#vcf-annotate)
#Currently mainly being used to help pass on information to Sudhir for evaluation cost/design-strategy for variant follow-up genotyping
#CHECK_1: Getting vcf-annotate to work properly for the AC_... columns if this setup/format is desired to be continually used

#Using the ...FreqInfo.AnnovarAnnotation.vs6.txt.gz format since already did a certain level of work properly parsing the ANNOVAR output (versus going straight from ANNOVAR output to do this)
#Difference between /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs2.py and /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py is that ...FullVCFwAnnovar.vs2.py was explicitly adding another column into the vcf file whereas /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py aims to take that same information and reformat it to be used by vcf-annotate, wherein vcf-annotate can include the information to the INFO column versus another, different stand-alone column

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py 

#python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py --file1 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | head -n 10) 

~~~
[  michaelt@node1  ~]$join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^\# | perl -lane 'print join("_", @F[0..1]), "\t", $F[3], "\t", $F[4];' | sort -k 1,1) <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1)  | wc
63863  319315 1722550
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^\# | wc
63863 50068592 1176055055
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.frq.count.gz | wc
2643484 15860903 67821551
~~~

#join <( python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | sort -k 1,1) \
#<(join <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^\# | perl -lane 'print join("_", @F[0..1]), "\t", $F[3], "\t", $F[4];' | sort -k 1,1) \
#<(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | sort -k 1,1)

#10      103190521       103190521       intronic        BTRC    .       .       G       A       1528    A:5     A:1     A:1     A:4     A:0     A:1

cat <(echo "#CHROM	FROM	TO	LOC	GENE	VARTYPE	AACHANGE	REF	ALT	AC_ALL	AC_TOTAL	AC_SERONEG	AC_HESERONEG	AC_SEROPOS	AC_RAPID	AC_SLOW") \
<(join <( python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | sort -k 1,1) <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^\# | perl -lane 'print join("_", @F[0..1]), "\t", $F[3], "\t", $F[4];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
perl -lane 'splice(@F, 0, 1, (((split(/_/, $F[0]))[0]), ((split(/_/, $F[0]))[1]), ((split(/_/, $F[0]))[1]))); print join("\t", @F);' | 
perl -ane 'print join("\t", @F[0..8]); my $alt = $F[8]; my $totAC = join("", ((split(/:/, $F[9]))[1])) + join("", ((split(/:/, $F[10]))[1])); print "\t", $totAC; for (my $i=9; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", $F[$i]; } } print "\n";' | sort -g -k 1,1 -k 2,2 ) | 
#awk '{ print $1, "\t", $2, "\t", $3, "\t", $8, "\t", $9, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $10, "\t", $11, "\t", $12, "\t", $13, "\t", $14, "\t", $15, "\t", $16 }' | 
#awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $10, "\t", $11, "\t", $12, "\t", $13, "\t", $14, "\t", $15, "\t", $16 }' | 
#awk '{ if ($1 !~ /^#/) { print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $10, "\t", $11, "\t", $12, "\t", $13, "\t", $14, "\t", $15, "\t", $16, "\t", $8, "\t", $9 } else { print $0 } }' | 
bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs1.wACs.txt.gz

#CHECK_1: Include pvalue and qvalue as another form of annotation in INFO -- maybe alter /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py code to do this, include two input files maybe?
#perl -lane 'if ($F[0] =~ m/^#CHROM/) { splice(@F, 7, 0, ("PVAL", "QVAL")); print join("\t", @F); } else { my $pVal = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep $F[4] | awk "{ print $2 }"`; my $qVal = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep $F[4] | awk "{ print $4 }"`; splice(@F, 7, 0, ($pVal, $qVal)); print join("\t", @F); }' | 

#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        REF     ALT     AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW
#X       16863126        16863126        UTR3    RBBP7   .       .       C       T       1546    T:3     T:0     T:0     T:3     T:0     T:0
#X       16863135        16863135        UTR3    RBBP7   .       .       G       A       1546    A:2     A:1     A:1     A:1     A:0     A:0

#From vcf-annotate -h
#Compress the file (using bgzip annotations), index (using tabix -s 1 -b 2 -e 3 annotations.gz) and run
tabix -s 1 -b 2 -e 3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs1.wACs.txt.gz

perl -e 'print  
"key=INFO,ID=LOC,Number=1,Type=String,Description='\''Gene-oriented genomic location(s) of the variant (e.g. exonic, intronic, intergenic, etc...)'\''
key=INFO,ID=GENE,Number=1,Type=String,Description='\''Gene(s) associated with the variant'\''
key=INFO,ID=VARTYPE,Number=1,Type=String,Description='\''If exonic, type of coding-change variant'\''
key=INFO,ID=AACHANGE,Number=1,Type=String,Description='\''If coding-change variant, related protein- and transcript-change'\''
key=INFO,ID=AC_ALL,Number=1,Type=Integer,Description='\''Total number of Ref and Alt allele copies (among white non-hispanics)'\''
key=INFO,ID=AC_TOTAL,Number=1,Type=Integer,Description='\''Total number of Alt allele copies (among white non-hispanics)'\''
key=INFO,ID=AC_SERONEG,Number=1,Type=Integer,Description='\''Seronegative number of Alt allele copies (among white non-hispanics)'\''
key=INFO,ID=AC_HESERONEG,Number=1,Type=Integer,Description='\''Highly-exposed seronegative number of Alt allele copies (among white non-hispanics)'\''
key=INFO,ID=AC_SEROPOS,Number=1,Type=Integer,Description='\''Seropositive number of Alt allele copies (among white non-hispanics)'\''
key=INFO,ID=AC_RAPID,Number=1,Type=Integer,Description='\''Rapid and very rapid AIDS progressor Alt allele copies (among white non-hispanics)'\''
key=INFO,ID=AC_SLOW,Number=1,Type=Integer,Description='\''Slow and very slow AIDS progressor Alt allele copies (among white non-hispanics)'\''\n"' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.PrepFor_vcfannotate.vs1.desc.txt

#From vcf-annotate -h
#cat in.vcf | vcf-annotate -a annotations.gz \
#   -d key=INFO,ID=ANN,Number=1,Type=Integer,Description='My custom annotation' \
#         -c CHROM,FROM,TO,INFO/ANN > out.vcf
#Examples:
#	zcat in.vcf.gz | vcf-annotate -a annotations.gz -d descriptions.txt -c FROM,TO,CHROM,ID,INFO/DP | bgzip -c >out.vcf.gz              
#	Where descriptions.txt contains:
#	key=INFO,ID=GN,Number=1,Type=String,Description='Gene Name'
#	key=INFO,ID=STR,Number=1,Type=Integer,Description='Strand'
#

#zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | vcf-annotate -a /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs1.wACs.txt.gz -d /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.PrepFor_vcfannotate.vs1.desc.txt -c CHROM,FROM,TO,REF,ALT,INFO/LOC,INFO/GENE,INFO/VARTYPE,INFO/AACHANGE,INFO/AC_ALL,INFO/AC_TOTAL,INFO/AC_SERONEG,INFO/AC_HESERONEG,INFO/AC_SEROPOS,INFO/AC_RAPID,INFO/AC_SLOW | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs1.wACs.vcf.gz 
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | vcf-annotate -a /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs1.wACs.txt.gz -d /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.PrepFor_vcfannotate.vs1.desc.txt -c CHROM,FROM,TO,INFO/LOC,INFO/GENE,INFO/VARTYPE,INFO/AACHANGE,INFO/AC_ALL,INFO/AC_TOTAL,INFO/AC_SERONEG,INFO/AC_HESERONEG,INFO/AC_SEROPOS,INFO/AC_RAPID,INFO/AC_SLOW | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs1.wACs.vcf.gz 

cat <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs1.wACs.vcf.gz | grep ^# | perl -lane 'if ($F[0] =~ /#CHROM/) { print join("\t", @F[0..7]); } else { print join(" ", @F); }' ) \
<(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs1.wACs.vcf.gz | perl -lane 'print join("\t", @F[0..7]);' | grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .65) { print $1; } }' | grep -vw NA ) | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq )) | \
gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.AllPools.Vs3.qVal_lte65.annotated.vcf.gz

~~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .65) { print $1; } }'i | grep -vw NA | wc
    212     212    1353
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs1.wACs.vcf.gz | perl -lane 'print join("\t", @F[0..7]);' | grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .65) { print $1; } }' | grep -vw NA ) | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq ) | wc
   1789   14312  881012
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.AllPools.Vs3.qVal_lte65.annotated.vcf.gz | awk '{ print $1, "\t", $2}' | grep -v ^# | wc
   1789    3578   24381
~~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.AllPools.Vs3.qVal_lte65.annotated.vcf.gz mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/. 

cat <(echo "#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        PVAL    QVAL    REF     ALT     AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW") \
<(join <( python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | sort -k 1,1) <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^\# | perl -lane 'print join("_", @F[0..1]), "\t", $F[3], "\t", $F[4];' | sort -k 1,1) | \
perl -lane '$F[1] = join("|", split(/[\;,\,]/, $F[1])); $F[2] = join("|", split(/[\;,\,]/, $F[2])); print join("\t", @F);' | \
perl -lane 'my @geneEntries = split(/\|/, $F[2]); my @pVals; my @qVals; foreach my $gene (@geneEntries) { $gene = ((split(/\(/, $gene))[0]); my $geneEntry = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w "$gene"`; if ($geneEntry) { my @geneVals1 = split(/\s+/, $geneEntry); push(@pVals, $geneVals1[1]); push(@qVals, $geneVals1[3]); } else { push(@pVals, "NA"); push(@qVals, "NA"); } } splice(@F, 5, 0, (join("|", @pVals), join("|", @qVals))); print join("\t", @F);' | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
perl -lane 'splice(@F, 0, 1, (((split(/_/, $F[0]))[0]), ((split(/_/, $F[0]))[1]), ((split(/_/, $F[0]))[1]))); print join("\t", @F);' |
perl -ane 'print join("\t", @F[0..10]); my $alt = $F[10]; my $totAC = join("", ((split(/:/, $F[11]))[1])) + join("", ((split(/:/, $F[12]))[1])); print "\t", $totAC; for (my $i=11; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", ((split(/\:/, $F[$i]))[1]); } } print "\n";' | sort -g -k 1,1 -k 2,2 ) |
bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz

~~~
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | head -n 10
#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        PVAL    QVAL    REF     ALT     AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW
X       16863126        16863126        UTR3    RBBP7   .       .       0.8921100000    0.70177 C       T       1546    3       0       0       3       0       0
X       16863135        16863135        UTR3    RBBP7   .       .       0.8921100000    0.70177 G       A       1546    2       1       1       1       0       0
X       16863691        16863691        intronic        RBBP7   .       .       0.8921100000    0.70177 A       G       1546    2       2       2       0       0       0
X       16863783        16863783        intronic        RBBP7   .       .       0.8921100000    0.70177 T       C       1544    2       0       0       2       0       0
X       16863871        16863871        intronic        RBBP7   .       .       0.8921100000    0.70177 C       T       1542    5       3       1       2       0       0
X       16864025        16864025        exonic  RBBP7   nonsynonymous_SNV       NM_001198719:c.T1267C:p.F423L   0.8921100000    0.70177 A       G       1542    1       0       0       1       0       1
X       16864106        16864106        intronic        RBBP7   .       .       0.8921100000    0.70177 T       C       1538    6       0       0       6       0       2
X       16864144        16864144        intronic        RBBP7   .       .       0.8921100000    0.70177 T       C       1532    7       5       1       2       0       2
X       16866829        16866829        intronic        RBBP7   .       .       0.8921100000    0.70177 C       T       1512    2       1       1       1       0       0
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | perl -lane 'print $#F;' | sort | uniq -c
  63864 17
~~~

#20150925 Note -- realized the below statement, which I included here as the 'finalized' version of code to produce the vcf-annotate prep file, is in fact incorrect. The grep statement associated with the '...wVarCounts' file of grep -w "$gene" is wrong; it needs to be grep ^"$gene"[^-0-9a-zA-Z_]. See later in log file during the rerun of everything for the 'rephenotyping of one individual' where the new code starts to get used
cat <(echo "#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        PVAL    QVAL    REF     ALT     AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW") <(join <( python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | sort -k 1,1) <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^\# | perl -lane 'print join("_", @F[0..1]), "\t", $F[3], "\t", $F[4];' | sort -k 1,1) | perl -lane '$F[1] = join("|", split(/[\;,\,]/, $F[1])); $F[2] = join("|", split(/[\;,\,]/, $F[2])); print join("\t", @F);' | perl -lane 'my @geneEntries = split(/\|/, $F[2]); my @pVals; my @qVals; foreach my $gene (@geneEntries) { $gene = ((split(/\(/, $gene))[0]); my $geneEntry = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w "$gene"`; if ($geneEntry) { my @geneVals1 = split(/\s+/, $geneEntry); push(@pVals, $geneVals1[1]); push(@qVals, $geneVals1[3]); } else { push(@pVals, "NA"); push(@qVals, "NA"); } } splice(@F, 5, 0, (join("|", @pVals), join("|", @qVals))); print join("\t", @F);' | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | perl -lane 'splice(@F, 0, 1, (((split(/_/, $F[0]))[0]), ((split(/_/, $F[0]))[1]), ((split(/_/, $F[0]))[1]))); print join("\t", @F);' |
perl -ane 'print join("\t", @F[0..10]); my $alt = $F[10]; my $totAC = join("", ((split(/:/, $F[11]))[1])) + join("", ((split(/:/, $F[12]))[1])); print "\t", $totAC; for (my $i=11; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", ((split(/\:/, $F[$i]))[1]); } } print "\n";' | sort -g -k 1,1 -k 2,2 ) | /home/shared/software/tabix-0.2.5/bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz
perl -ane 'print join("\t", @F[0..10]); my $alt = $F[10]; my $totAC = join("", ((split(/:/, $F[11]))[1])) + join("", ((split(/:/, $F[12]))[1])); print "\t", $totAC; for (my $i=11; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", ((split(/\:/, $F[$i]))[1]); } } print "\n";' | sort -g -k 1,1 -k 2,2 ) | perl -lane 'splice(@F, 3, 0, ($F[9], $F[10])); splice(@F, 11, 2); print join("\t", @F);' | /home/shared/software/tabix-0.2.5/bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz

tabix -s 1 -b 2 -e 3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz

perl -e 'print
"key=INFO,ID=LOC,Number=1,Type=String,Description='\''Gene-oriented genomic location(s) of the variant (e.g. exonic, intronic, intergenic, etc...)'\''
key=INFO,ID=GENE,Number=1,Type=String,Description='\''Gene(s) associated with the variant'\''
key=INFO,ID=VARTYPE,Number=1,Type=String,Description='\''If exonic, type of coding-change variant'\''
key=INFO,ID=AACHANGE,Number=1,Type=String,Description='\''If coding-change variant, related protein- and transcript-change'\''
key=INFO,ID=PVAL,Number=1,Type=Integer,Description='\''p-value(s) associated with gene(s) in GENE field. NOTE -- p-value is gene-level based on using SKAT-O, only nonsynonymous variants within the specific gene and comparing seronegatives vs. seropositives. It is /NOT/ a variant-specific or variant-level p-value. NA indicates no p-value is available'\''
key=INFO,ID=QVAL,Number=1,Type=Integer,Description='\''q-value(s) associated with gene(s) in GENE field. Derived from the p-value distribution of all genes from the SKAT-O, nonsynonymous variants only, seronegative vs. seropositive analysis. It is /NOT/ a variant-specific or variant-level q-value. NA indicates no q-value is available'\''
key=INFO,ID=AC_ALL,Number=1,Type=Integer,Description='\''Total number of Ref and Alt allele copies (among white non-hispanics)'\''
key=INFO,ID=AC_TOTAL,Number=1,Type=Integer,Description='\''Total number of Alt allele copies (among white non-hispanics)'\''
key=INFO,ID=AC_SERONEG,Number=1,Type=Integer,Description='\''Seronegative number of Alt allele copies (among white non-hispanics)'\''
key=INFO,ID=AC_HESERONEG,Number=1,Type=Integer,Description='\''Highly-exposed seronegative number of Alt allele copies (among white non-hispanics)'\''
key=INFO,ID=AC_SEROPOS,Number=1,Type=Integer,Description='\''Seropositive number of Alt allele copies (among white non-hispanics)'\''
key=INFO,ID=AC_RAPID,Number=1,Type=Integer,Description='\''Rapid and very rapid AIDS progressor Alt allele copies (among white non-hispanics)'\''
key=INFO,ID=AC_SLOW,Number=1,Type=Integer,Description='\''Slow and very slow AIDS progressor Alt allele copies (among white non-hispanics)'\''\n"' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.PrepFor_vcfannotate.vs2.desc.txt

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | vcf-annotate -a /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz -d /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.PrepFor_vcfannotate.vs2.desc.txt -c CHROM,FROM,TO,INFO/LOC,INFO/GENE,INFO/VARTYPE,INFO/AACHANGE,INFO/PVAL,INFO/QVAL,REF,ALT,INFO/AC_ALL,INFO/AC_TOTAL,INFO/AC_SERONEG,INFO/AC_HESERONEG,INFO/AC_SEROPOS,INFO/AC_RAPID,INFO/AC_SLOW | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.AllPools.Vs3.annotated.vcf.gz

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz

#Difference between 141 and 142 due to '#CHROM...' header in /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz
~~~
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VarLevel.txt.gz | wc
  63864 1149552 5545452
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | wc
  63864 1149552 5545452
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.AllPools.Vs3.annotated.vcf.gz | wc
  64005 50070304 1185515553
[  michaelt@node2  ~]$R -q -e "64005 - 63864"
> 64005 - 63864
[1] 141
> 
> 
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.AllPools.Vs3.annotated.vcf.gz | grep ^# | wc
 142    1712   29465
~~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.AllPools.Vs3.annotated.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.ForSudhir.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/. 

#20150825 20150930 CHECK_1: Redo certain analyses because of the single "30430   POOL87_02RL" individual. Check subset side analysis below for details regarding these revisions and what this is referring to specifically
#20150907 201510__ CHECK_1: Redo all this in P2 as well

#Put pheno redo correction here?
#
#Have to redo group-pheno level QC stats
#Perm files
#Pathway analyses
#Mktest?
#20150825
#Current to-do list
#
#Redo MultinRun TarAndClean and PermutationRun TarAndClean

#20150907 CHECK_0: Redo all this in P2 as well
#20150825 CHECK_0: Also figure out how to check alignments of reads/bam file stuff around specific/certain variants, such as those that look interesting on the lollipop charts, to make sure misaligning reads are not the cause of multiple variants popping up in localized places (and aberrantly causing a region to look interesting)
#20150825 CHECK_0: Really need to redo those subsampling-to-P2 analyses, but on the individual .bam level files first?
#20150825 CHECK_0: Do those p-value comparisons or rank comparison between top hits in my analyses and the other analyses the molecular people have done?
#20150914 CEHCK_0: Look at variants that are not present in ExAC or 1000G, but are present in my dataset, to see if there are any systematic qualities to them (poorly mapped reads, certain regions of the genome, correlation or overabundance in significant-trending genes?..correlations with gene p-values perhaps?)
#20150825 CHECK_0: Regarding the fastqc stuff -- Check this stuff?

#20150909 CHECK_0: Ask Sudhir about timetoART (i.e. is it 'time to Anti-Retroviral Therapy'?)

~~~
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | head -n 10
20446,20446,6,POOL27-06RL,Black non-Hispanic,35.2,0,0,0,1,0,0,0,0,3.42
21159,21159,1,POOL98-01RL,White non-Hispanic,0,0,1,0,0,0,0,0,0,0
41011,41011,8,POOL83-08RL,White non-Hispanic,0,0,1,0,0,0,0,0,0,0
41013,41013,8,POOL77-08RL,White non-Hispanic,0,0,1,0,0,0,0,0,0,0
40157,40157,9,POOL85-09RL,White Hispanic,0,1,0,0,0,0,0,0,0,0
10290,10290,2,POOL16S-02RL,Black non-Hispanic,39.3,0,0,0,1,1,1,0,0,1.62
10290,10290,6,POOL41-06RL,Black non-Hispanic,39.3,0,0,0,1,1,1,0,0,1.62
10292,10292,3,POOL02-03RL,White non-Hispanic,0,1,0,0,0,0,0,0,0,0
10453,10453,3,POOL33-03RL,White non-Hispanic,28.3,0,0,0,1,0,0,0,0,7.91
10519,10519,4,POOL157-04RL,White non-Hispanic,0,0,1,0,0,0,0,0,0,0
[  michaelt@login2  ~]$cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Sample_status_list_12-10-13_CT.edited.noMeta.csv | head -n 10 | perl -F\, -lane 'print join("\t", @F[10..19]);'
posage  hesn    lesn    lesc    othersc vrapid  rapid   vslow   slow    timetoART
36.4    0       0       0       1       0       0       0       0       2.13
40.2    0       0       1       1       0       1       0       0       1.25
0       0       1       0       0       0       0       0       0       0
35.6    0       0       0       0       0       0       1       1       24.03
34      0       0       0       1       0       0       0       0       2.42
45.7    0       0       1       1       0       0       0       0       1.24
0       0       1       0       0       0       0       0       0       0
47.9    0       0       0       1       1       1       0       0       3.15
0       1       0       0       0       0       0       0       0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | perl -F\, -lane 'print join("\t", @F[5..14]);' | head -n 10
35.2    0       0       0       1       0       0       0       0       3.42
0       0       1       0       0       0       0       0       0       0
0       0       1       0       0       0       0       0       0       0
0       0       1       0       0       0       0       0       0       0
0       1       0       0       0       0       0       0       0       0
39.3    0       0       0       1       1       1       0       0       1.62
39.3    0       0       0       1       1       1       0       0       1.62
0       1       0       0       0       0       0       0       0       0
28.3    0       0       0       1       0       0       0       0       7.91
0       0       1       0       0       0       0       0       0       0

[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4 }' | sort | uniq -c
582 0        0
205 0        1
201 1        0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $5 }' | sort | uniq -c
388 0        0       0
194 0        0       1
205 0        1       0
201 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $6 }' | sort | uniq -c
140 0        0       0
442 0        0       1
205 0        1       0
201 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $5, "\t", $6 }' | sort | uniq -c
129 0        0       0       0
259 0        0       0       1
11 0        0       1       0
183 0        0       1       1
205 0        1       0       0
201 1        0       0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $7 }' | sort | uniq -c
533 0        0       0
49 0        0       1
205 0        1       0
201 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $8 }' | sort | uniq -c
488 0        0       0
94 0        0       1
205 0        1       0
201 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $9 }' | sort | uniq -c
496 0        0       0
86 0        0       1
205 0        1       0
201 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $10 }' | sort | uniq -c
436 0        0       0
146 0        0       1
205 0        1       0
201 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4 } }' | sort | uniq -c
582 0        0
1 1        0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $5 } }' | sort | uniq -c
388 0        0       0
194 0        0       1
1 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $6 } }' | sort | uniq -c
140 0        0       0
442 0        0       1
1 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $5, "\t", $6 } }' | sort | uniq -c
129 0        0       0       0
259 0        0       0       1
11 0        0       1       0
183 0        0       1       1
1 1        0       0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $7 } }' | sort | uniq -c
533 0        0       0
49 0        0       1
1 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $8 } }' | sort | uniq -c
488 0        0       0
94 0        0       1
1 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $9 } }' | sort | uniq -c
496 0        0       0
86 0        0       1
1 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $10 } }' | sort | uniq -c
436 0        0       0
146 0        0       1
1 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4 } }' | sort | uniq -c
580 0        0
1 1        0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $5 } }' | sort | uniq -c
386 0        0       0
194 0        0       1
1 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $6 } }' | sort | uniq -c
140 0        0       0
440 0        0       1
1 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $5, "\t", $6 } }' | sort | uniq -c
129 0        0       0       0
257 0        0       0       1
11 0        0       1       0
183 0        0       1       1
1 1        0       0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $7 } }' | sort | uniq -c
531 0        0       0
49 0        0       1
1 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $8 } }' | sort | uniq -c
486 0        0       0
94 0        0       1
1 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $9 } }' | sort | uniq -c
494 0        0       0
86 0        0       1
1 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $10 } }' | sort | uniq -c
434 0        0       0
146 0        0       1
1 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{print $3, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $8, "\t", $9, "\t", $10 }' | sort | uniq -c
54 0        0       0       0       0       0       0       1
66 0        0       0       0       0       0       1       1
8 0        0       0       0       0       1       0       0
1 0        0       0       0       1       1       0       0
189 0        0       0       1       0       0       0       0
6 0        0       0       1       0       0       0       1
8 0        0       0       1       0       0       1       1
20 0        0       0       1       0       1       0       0
36 0        0       0       1       1       1       0       0
10 0        0       1       0       0       0       0       0
1 0        0       1       0       0       1       0       0
143 0        0       1       1       0       0       0       0
12 0        0       1       1       0       0       1       1
16 0        0       1       1       0       1       0       0
12 0        0       1       1       1       1       0       0
205 0        1       0       0       0       0       0       0
201 1        0       0       0       0       0       0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $8, "\t", $9, "\t", $10 } }' | sort | uniq -c
54 0        0       0       0       0       0       0       1
66 0        0       0       0       0       0       1       1
8 0        0       0       0       0       1       0       0
1 0        0       0       0       1       1       0       0
189 0        0       0       1       0       0       0       0
6 0        0       0       1       0       0       0       1
8 0        0       0       1       0       0       1       1
20 0        0       0       1       0       1       0       0
36 0        0       0       1       1       1       0       0
10 0        0       1       0       0       0       0       0
1 0        0       1       0       0       1       0       0
143 0        0       1       1       0       0       0       0
12 0        0       1       1       0       0       1       1
16 0        0       1       1       0       1       0       0
12 0        0       1       1       1       1       0       0
1 1        0       0       0       0       0       0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $8, "\t", $9, "\t", $10 } }' | sort | uniq -c
54 0        0       0       0       0       0       0       1
66 0        0       0       0       0       0       1       1
8 0        0       0       0       0       1       0       0
1 0        0       0       0       1       1       0       0
187 0        0       0       1       0       0       0       0
6 0        0       0       1       0       0       0       1
8 0        0       0       1       0       0       1       1
20 0        0       0       1       0       1       0       0
36 0        0       0       1       1       1       0       0
10 0        0       1       0       0       0       0       0
1 0        0       1       0       0       1       0       0
143 0        0       1       1       0       0       0       0
12 0        0       1       1       0       0       1       1
16 0        0       1       1       0       1       0       0
12 0        0       1       1       1       1       0       0
1 1        0       0       0       0       0       0       0
[  michaelt@login2  ~]$cmp <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $8, "\t", $9, "\t", $10 } }' | sort | uniq -c) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $8, "\t", $9, "\t", $10 } }' | sort | uniq -c)
/dev/fd/63 /dev/fd/62 differ: byte 159, line 5
~

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs2.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs3.py

#20150909 -- Changing the ...DataProcessing.Pipeline.CreatePhenotypeFile.vs3.py file to now check whether posage is >0 to determine seropositive status, and I will be doing this as the initial determining step. I will also now add an extra 'else' step that define a phenotype as 'NA' if none of the conditions are met anywhere in the rules provided

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs3.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.vs3.pheno

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.vs2.pheno

~~~
[  michaelt@login2  ~]$python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs3.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.vs3.pheno
[  michaelt@login2  ~]$
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.vs3.pheno | awk '{ print $3 }' | sort | uniq -c                                                                     
405 1
583 2
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.vs3.pheno | awk '{ print $4 }' | sort | uniq -c
200 1
583 2
205 -9
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.vs3.pheno | awk '{ print $5 }' | sort | uniq -c
94 1
146 2
748 -9
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.vs3.pheno | awk '{ print $6 }' | sort | uniq -c
49 1
86 2
853 -9
~~~

#Note -- above matches the below entry from the previous QC, e.g. there are 406 indvs that have 1 in column 3 or 4 (plus that one individual with >0 pos but also a column 3 entry) and 583 individuals with no entry in column 3 or 4 
#[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4 }' | sort | uniq -c
#582 0        0
#205 0        1
#201 1        0

#20150930 Note -- the below code is copy/pasted from the end of the rerun done at ##20150825 as a point of reference

~~~
.
.
.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.AllPools.Vs3.annotated.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/OLD3.MM5.AllPools.Vs3.annotated.vcf.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/OLD3.MM5.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.ForSudhir.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/OLD3.MM5.skatO.nonsyn.qVals.ACs.GeneLevel.ForSudhir.txt

for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs | head -n 10`
do

	paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane 'if ($F[0] =~ m/^'$gene1'$/) { print join("\t", @F); }' | awk '{ print $1, "\t", $2, "\t", $4, "\t", $5, "\t", $8, "\t", $9, "\t", $11 }' ) \
	<(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | \
	grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'if ($F[0] =~ m/^'$gene1'$/) { print join("\t", @F); }' | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }') | \
	awk '{ print $12, "\t", $13, "\t", $14, "\t", $15, "\t", $16, "\t", $17, "\t", $18 }' | \
	R -q -e "Data1 <- read.table(file('stdin'), header=F); print(paste(apply(Data1, 2, sum)), quote=FALSE);" | grep -v ^\> | perl -lane 'splice(@F, 0, 1); print join("\t", @F);') \
	>> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt
	
done

#	paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane 'if ($F[0] =~ m/^'$gene1'$/) { print join("\t", @F); }' | awk '{ print $1, "\t", $2, "\t", $4, "\t", $5, "\t", $8, "\t", $9, "\t", $11 }' ) <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'if ($F[0] =~ m/^'$gene1'$/) { print join("\t", @F); }' | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }') | awk '{ print $12, "\t", $13, "\t", $14, "\t", $15, "\t", $16, "\t", $17, "\t", $18 }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); print(paste(apply(Data1, 2, sum)), quote=FALSE);" | grep -v ^\> | perl -lane 'splice(@F, 0, 1); print join("\t", @F);') >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt | perl -lane 'splice(@F, 4, 3); print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt 

#Included the below header in file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt
#GENE   PVAL    QVAL    NUMVAR    AC_ALL        AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW

#QC Checks on file
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt | wc
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt | perl -lane 'if (($F[3] == $F[4]) && ($F[3] == $F[5]) && ($F[4] == $F[5])) { print join("\t", @F); } '  | wc
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt | perl -lane 'if (($F[3] != $F[4]) || ($F[3] != $F[5]) || ($F[4] != $F[5])) { print join("\t", @F); } '
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[5] != $val2) { print join("\t", @F); }'
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[5] == $val2) { print join("\t", @F); }' | wc
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[7] > $F[6]) { print join("\t", @F); }'
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[7] <= $F[6]) { print join("\t", @F); }' | wc
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[9] + $F[10]; if ($F[8] < $val2) { print join("\t", @F); }'
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[9] + $F[10]; if ($F[8] >= $val2) { print join("\t", @F); }' | wc

~~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt | wc                           
474    5207   23994
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt | perl -lane 'if (($F[3] == $F[4]) && ($F[3] == $F[5]) && ($F[4] == $F[5])) { print join("\t", @F); } '  | wc
474    6629   29855
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt | perl -lane 'if (($F[3] != $F[4]) || ($F[3] != $F[5]) || ($F[4] != $F[5])) { print join("\t", @F); } '                       
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[5] != $val2) { print join("\t", @F); }'
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[5] == $val2) { print join("\t", @F); }' | wc                                      
474    5207   23994
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[7] > $F[6]) { print join("\t", @F); }'
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[7] <= $F[6]) { print join("\t", @F); }' | wc                               
474    5207   23994
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[9] + $F[10]; if ($F[8] < $val2) { print join("\t", @F); }'
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[9] + $F[10]; if ($F[8] >= $val2) { print join("\t", @F); }' | wc                  
474    5207   23994
[  michaelt@node1  ~]$
~~~

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.vcfannotate.vs3.wACs.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.AllPools.Vs3.annotated.vcf.gz

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz 

#Difference between 141 and 142 due to '#CHROM...' header in /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz
~~~
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz | wc
  63864 1149552 5545409                                                                                                                                                                                                                                                                                                    
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | wc
  63864 1149552 5545409
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.AllPools.Vs3.annotated.vcf.gz | wc
  64005 50070304 1185515553
[  michaelt@node2  ~]$R -q -e "64005 - 63864"
> 64005 - 63864
[1] 141
> 
> 
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.AllPools.Vs3.annotated.vcf.gz | grep ^# | wc
 142    1712   29465
~~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/OLD3.MM5.AllPools.Vs3.annotated.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/OLD3.MM5.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/OLD3.MM5.skatO.nonsyn.qVals.ACs.GeneLevel.ForSudhir.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/.

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.AllPools.Vs3.annotated.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/. 

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/msigdb_c2/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt.MM5.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.indv3.EPACTS.RMID5PCs.msigdb_c2

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/msigdb_c2/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt.MM5.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs.msigdb_c2

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/Permutations /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20150427Array1KevinGeneGroupings.array_gene_groupings_03_31_2014.semicolons.SpaceToUnderscores.withHGNCdata.SplitIntoCategories.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/Permutations/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs.20150427Array1KevinGeneGroupings

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/Permutations /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20150427Array1KevinGeneGroupings.array_gene_groupings_03_31_2014.semicolons.SpaceToUnderscores.withHGNCdata.SplitIntoCategories.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/Permutations/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs.20150427Array1KevinGeneGroupings


#DO QC STUFF
~~~




















###
### Current end of main analyses
### CURRENTEND
###
### Below begin specific subset side analyses
###

#20150825 CHECK_0: Do glossary thing below?

### Glossary of below subset side analyses




###




















##Sometime in April/May 2014?
##Putting together list of variants for West coast people from the first pass through of results

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary | grep Nonsy | tail -n 25 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary.NonSyn.top25

#Added the following header
#CATEGORY	CHROM	BEGIN	END	MARKER_ID	NS	FRAC_WITH_RARE	#Variants	#Passed	#Singletons	PVALUE	QVALUE	RHO

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary.NonSyn.top25 | perl -lane 'my @vals1 = split(/_/, $F[4]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary.NonSyn.top25.geneIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary.NonSyn.top25.geneIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.top25

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/VLookUp.Python.PullOutEPACTSVariants.vs1.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/VLookUp.Python.PullOutEPACTSVariants.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.top25 --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | cut -d, -f 1-6,18-29,35,37,39,41 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary.NonSyn.top25.Variants.csv

#Added the following header
#Chr,BP,rsID,refAllele,otherAllele,AncAllele,AC_AllIndvs,AC_SeroNeg,AC_SeroNegHE,AC_SeroPos,AC_RapidVRapid,AC_SlowVSlow,AC_VeryRapid,AC_VerySlow,VarType,GeneID,SNVType,AAChange,Sift,Polyphen2,LRT,MutationTaster

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary.NonSyn.top25 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/ForSumit.Top25.NonSynGenes
ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary.NonSyn.top25.Variants.csv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/ForSumit.Top25.NonSynGenes.Variants.csv

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/ForSumit.Top25.NonSynGenes /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/ForSumit.Top25.NonSynGenes.Variants.csv mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/. 


##20140925
##Getting variants present in specific group of invidiuals for Kevin

#Going to create a utility script that takes a list of patientID#s, converts them to poolID#s via a linkerfile and thne pulls those columns of genotypes from a specified .vcf file. Then going to create a separate utility script that adds (hard-coded) annotation information to a .vcf file 

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.ExtractPatientIDsFromVCF.vs1.py 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | head -n 150 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.top150

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.ExtractPatientIDsFromVCF.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.top150 --IDs 20572,20306,20219,20170,20123,20093,20501,20698,20132,20661,20471,20575

#20572
#20471
#20093
#20170
#20123
#20306
#20575
#20661
#20219
#20132
#20501
#20698

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.ExtractPatientIDsFromVCF.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --IDs 20572,20306,20219,20170,20123,20093,20501,20698,20132,20661,20471,20575 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf

[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf | wc                                                                                                                                                                                                                     
  73389 1539241 44180927

[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | wc                                                                                                                                                                                                                                                              
  73389 67401779 1578124080

#python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.ExtractPatientIDsFromVCF.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --IDs All | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wPatientIDs.vcf.gz 

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.vs1.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.wAnnot.vcf  

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf --AC 5 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.wAnnot.AClte5.vcf  

vcftools --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf --max-mac 5 --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcfToolsmaxMAC5.vcf 

#Made edits to DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.vs1.py and renamed asdouasdiubasdibasd, so moved file to DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.vs2.py. Basically changed AC cutoff so that only print variants that have the same number of allele copies within these 12 individuals as expected for the variant itself, i.e. keep only variants whose allele copies are present in just these 12 individuals
mv asdouasdiubasdibasd DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.vs2.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf --AC 5 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.wAnnot.AClte5.vs2.vcf

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.wAnnot.AClte5.vs2.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.20140925KevinIndvRequest.wAnnot.AClte5.vcf

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.20140925KevinIndvRequest.wAnnot.AClte5.vcf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/.

#20141010 -- Finding the individual that has the single alternative allele for a CCNT1 variant, 12_49089559

[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz | grep 49089559 | perl -lane 'for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/:/, $F[$i]); if ($vals1[0] =~ m/1/) { print $i, "\t", $F[$i] } } ;'         662     0/1:12,11:23:99:304,0,282
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz | grep 49089559 | awk '{ print $663 }'                                                                                                                          0/1:12,11:23:99:304,0,282
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz | awk '{ print $2, "\t", $663 }' | perl -lane 'my @vals1 = split(/:/, $F[1]); if ($vals1[0] =~ m/POOL|\/1/) { print join("\t", @F); }' | grep -E '49089559|POOL'
POS     POOL81_11RL 
49089559        0/1:12,11:23:99:304,0,282
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | grep POOL81
41493,41493,4,POOL81-04RL,White non-Hispanic,0,0,1,0,0,0,0,0,0,0
40457,40457,10,POOL81-10RL,White non-Hispanic,0,1,0,0,0,0,0,0,0,0
41574,41574,11,POOL81-11RL,White non-Hispanic,0,0,1,0,0,0,0,0,0,0
30751,30751,3,POOL81-03RL,White non-Hispanic,0,0,1,0,0,0,0,0,0,0
41628,41628,9,POOL81-09RL,White non-Hispanic,33.3,0,0,0,1,0,0,0,0,0.53
40270,40270,8,POOL81-08RL,White non-Hispanic,0,1,0,0,0,0,0,0,0,0
41625,41625,1,POOL81-01RL,White non-Hispanic,0,0,1,0,0,0,0,0,0,0
40063,40063,7,POOL81-07RL,White non-Hispanic,0,0,1,0,0,0,0,0,0,0

#20150127 -- continuing this specific HESN-subset group analysis, now interested in just seeing whether a specific group of genes that they have identified have any type of clustering of variants among these 12 individuals

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf.gz | perl -lane 'if ($F[0] =~ m/^#/) { print join("\t", @F); } else { my $printFlag1 = 0; for (my $i = 9; $i <= $#F; $i++) { my @indvQuals = split(/:/, $F[$i]); my @genos = split(/\//, $indvQuals[0]); if (($genos[0] eq "1") || ($genos[1] eq "1")) { $printFlag1 = 1; } } if ($printFlag1 == 1) { print join("\t", @F); }  } ' | gzip >  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.vcf.gz

[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.vcf.gz | wc
  13488  281320 8212108

[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.vcf.gz | tail -n 10
X       53644586        .       T       C       148.38  VQSRTrancheSNP99.00to99.90      AA=T;AC=9;AF=5.017e-03;AN=1794;BaseQRankSum=0.268;DP=7898;Dels=0.00;FS=6.510;HRun=1;HaplotypeScore=0.9294;InbreedingCoeff=0.0388;MLEAC=1;MLEAF=5.574e-04;MQ=60.00;MQ0=0;MQRankSum=-0.141;QD=2.51;ReadPosRankSum=0.660;VQSLOD=1.31;culprit=QD       GT:AD:DP:GQ:PL  0/0:8,0:8:24:0,24,273   1/1:0,6:6:18:214,18,0   0/0:9,0:9:27:0,27,311   0/0:2,0:2:6:0,6,71      0/0:3,0:3:9:0,9,97      0/0:2,0:2:6:0,6,69      0/0:12,0:12:36:0,36,416 0/0:3,0:3:9:0,9,104     0/0:7,0:7:21:0,21,241   0/0:5,0:5:15:0,15,168   0/0:10,0:10:30:0,30,347 0/0:4,0:4:12:0,12,139
X       53674550        .       T       A       340.33  VQSRTrancheSNP99.00to99.90      AA=T;AC=3;AF=1.667e-03;AN=1800;BaseQRankSum=-5.388;DP=11244;Dels=0.00;FS=3.554;HRun=2;HaplotypeScore=1.5743;InbreedingCoeff=0.1312;MLEAC=2;MLEAF=1.111e-03;MQ=59.99;MQ0=0;MQRankSum=-0.287;QD=17.02;ReadPosRankSum=-0.717;VQSLOD=1.48;culprit=HaplotypeScore       GT:AD:DP:GQ:PL  0/0:10,0:10:30:0,30,326 0/0:7,0:7:21:0,21,236   0/0:7,0:7:21:0,21,232   0/0:16,0:16:48:0,48,551 0/0:5,0:5:15:0,15,171   0/0:6,0:6:18:0,18,209   0/0:16,0:16:48:0,48,527 0/1:4,1:5:17:17,0,122   0/0:12,0:12:36:0,36,407 0/0:16,0:16:48:0,48,532 0/0:12,0:12:36:0,36,413 0/0:9,0:9:27:0,27,286
X       64948279        .       T       C       55.25   VQSRTrancheSNP99.00to99.90      AA=T;AC=26;AF=0.015;AN=1788;BaseQRankSum=-7.262;DP=6334;Dels=0.00;FS=3.996;HRun=1;HaplotypeScore=0.8907;InbreedingCoeff=-0.0180;MLEAC=10;MLEAF=5.593e-03;MQ=59.76;MQ0=0;MQRankSum=-1.467;QD=0.38;ReadPosRankSum=-0.311;VQSLOD=-6.838e-01;culprit=DP        GT:AD:DP:GQ:PL  0/0:3,0:3:9:0,9,103     0/0:3,0:3:9:0,9,108     0/0:9,0:9:27:0,27,314   0/0:9,0:9:27:0,27,314   0/0:1,0:1:3:0,3,34      0/0:5,0:5:15:0,15,178   0/1:8,1:9:9:9,0,210     0/0:1,0:1:3:0,3,34      0/0:13,0:13:39:0,39,437 0/0:10,0:10:30:0,30,347 0/0:5,0:5:15:0,15,176   0/0:5,0:5:15:0,15,177
X       77113219        rs146752003     C       T       124.94  VQSRTrancheSNP99.00to99.90      AA=C;AC=3;AF=1.669e-03;AN=1798;BaseQRankSum=4.969;DB;DP=8111;Dels=0.00;FS=8.932;HRun=1;HaplotypeScore=0.7404;InbreedingCoeff=-0.0150;MLEAC=1;MLEAF=5.562e-04;MQ=60.00;MQ0=0;MQRankSum=0.113;QD=3.67;ReadPosRankSum=-0.898;VQSLOD=0.475;culprit=QD  GT:AD:DP:GQ:PL  0/1:8,6:14:99:175,0,207 0/0:6,0:6:18:0,18,204   0/0:6,0:6:18:0,18,196   0/0:7,0:7:21:0,21,238   0/0:3,0:3:9:0,9,90      0/0:6,0:6:18:0,18,197   0/0:10,0:10:30:0,30,318 0/0:3,0:3:9:0,9,95      0/0:5,0:5:15:0,15,154   0/0:19,0:19:57:0,57,589 0/0:8,0:8:24:0,24,256   0/0:8,0:8:24:0,24,252
X       77150634        .       G       A       266.01  VQSRTrancheSNP99.00to99.90      AA=G;AC=5;AF=2.775e-03;AN=1802;BaseQRankSum=-3.715;DP=9491;Dels=0.00;FS=0.000;HRun=2;HaplotypeScore=1.6921;InbreedingCoeff=0.0737;MLEAC=2;MLEAF=1.110e-03;MQ=60.00;MQ0=0;MQRankSum=-0.303;QD=10.64;ReadPosRankSum=-1.345;VQSLOD=1.55;culprit=DP    GT:AD:DP:GQ:PL  0/0:8,0:8:24:0,24,268   0/0:12,0:12:36:0,36,435 0/0:8,0:8:24:0,24,280   0/0:12,0:12:36:0,36,415 0/0:4,0:4:12:0,12,135   0/0:6,0:6:18:0,18,213   0/0:11,0:11:33:0,33,366 0/0:1,0:1:3:0,3,34      0/0:22,0:22:66:0,66,732 0/0:13,0:13:39:0,39,448 0/0:10,0:10:30:0,30,344 0/1:3,1:4:24:24,0,86
X       153061748       .       C       T       761.06  VQSRTrancheSNP99.00to99.90      AA=C;AC=6;AF=3.326e-03;AN=1804;BaseQRankSum=6.545;DP=11331;Dels=0.00;FS=6.414;HRun=1;HaplotypeScore=1.4674;InbreedingCoeff=0.1070;MLEAC=2;MLEAF=1.109e-03;MQ=59.99;MQ0=0;MQRankSum=-0.139;QD=12.48;ReadPosRankSum=0.991;VQSLOD=1.68;culprit=HaplotypeScore GT:AD:DP:GQ:PL  0/0:11,0:11:33:0,33,347 0/0:21,0:21:63:0,63,705 0/0:11,0:11:33:0,33,361 0/0:20,0:20:60:0,60,674 0/0:3,0:3:9:0,9,94      0/0:10,0:10:30:0,30,333 0/0:13,0:13:39:0,39,415 0/0:7,0:7:21:0,21,216   0/0:11,0:11:33:0,33,357 0/0:8,0:8:24:0,24,243   0/0:10,0:10:30:0,30,317 0/1:5,1:6:17:17,0,140
X       153713382       .       T       C       268.37  VQSRTrancheSNP99.00to99.90      AA=T;AC=9;AF=5.000e-03;AN=1800;BaseQRankSum=-5.440;DP=13441;Dels=0.00;FS=0.500;HRun=1;HaplotypeScore=3.8832;InbreedingCoeff=0.1192;MLEAC=1;MLEAF=5.556e-04;MQ=59.98;MQ0=0;MQRankSum=1.064;QD=3.78;ReadPosRankSum=1.520;VQSLOD=0.131;culprit=QD     GT:AD:DP:GQ:PL  0/0:12,0:12:36:0,36,396 0/0:31,0:31:93:0,93,1030        0/0:13,0:13:39:0,39,456 0/0:18,0:18:54:0,54,613 0/0:3,0:3:9:0,9,96      0/0:10,0:10:30:0,30,339 0/0:12,0:12:36:0,36,404 0/0:2,0:2:6:0,6,66      0/0:17,0:17:51:0,51,569 0/1:9,1:10:3:3,0,266    0/0:7,0:7:21:0,21,241   0/0:8,0:8:24:0,24,271
X       153713641       rs147530071     G       A       338.36  VQSRTrancheSNP90.00to99.00      AA=G;AC=5;AF=2.784e-03;AN=1796;BaseQRankSum=4.788;DB;DP=12508;Dels=0.00;FS=2.813;HRun=0;HaplotypeScore=2.5272;InbreedingCoeff=0.1407;MLEAC=2;MLEAF=1.114e-03;MQ=59.99;MQ0=0;MQRankSum=0.648;POSITIVE_TRAIN_SITE;QD=9.40;ReadPosRankSum=0.718;VQSLOD=3.02;culprit=HaplotypeScore    GT:AD:DP:GQ:PL  0/0:9,0:9:27:0,27,285   0/0:26,0:26:78:0,78,854 0/0:12,0:12:36:0,36,396 0/0:13,0:13:39:0,39,412 ./.     0/0:10,0:10:30:0,30,338 0/0:15,0:15:45:0,45,472 0/1:3,1:4:22:22,0,86    0/0:11,0:11:33:0,33,357 0/0:7,0:7:21:0,21,223   0/0:5,0:5:15:0,15,165      0/0:6,0:6:18:0,18,190
X       154456747       rs572013        A       G       346432  VQSRTrancheSNP99.00to99.90      AA=G;AC=1748;AF=0.969;AN=1804;BaseQRankSum=18.300;DB;DP=10832;Dels=0.00;FS=2.667;HRun=0;HaplotypeScore=1.6675;InbreedingCoeff=-0.0339;MLEAC=1797;MLEAF=0.996;MQ=59.99;MQ0=0;MQRankSum=0.253;POSITIVE_TRAIN_SITE;QD=31.98;ReadPosRankSum=6.371;VQSLOD=0.370;culprit=ReadPosRankSum  GT:AD:DP:GQ:PL  1/1:0,7:7:21:241,21,0   1/1:0,2:2:6:71,6,0      1/1:0,7:7:21:243,21,0   1/1:0,16:16:48:556,48,0 1/1:0,1:1:3:32,3,0      1/1:0,7:7:21:243,21,0   1/1:0,13:13:39:434,39,0 1/1:0,5:5:15:168,15,0   1/1:0,7:7:21:230,21,0   1/1:0,16:16:48:530,48,0 1/1:0,8:8:24:260,24,0      1/1:0,6:6:18:197,18,0
X       154464466       .       C       T       471.87  VQSRTrancheSNP99.00to99.90      AA=C;AC=7;AF=3.911e-03;AN=1790;BaseQRankSum=2.203;DP=7087;Dels=0.00;FS=0.000;HRun=1;HaplotypeScore=0.6094;InbreedingCoeff=0.0261;MLEAC=3;MLEAF=1.676e-03;MQ=59.99;MQ0=0;MQRankSum=2.281;QD=10.04;ReadPosRankSum=-1.245;VQSLOD=0.733;culprit=DP     GT:AD:DP:GQ:PL  0/0:4,0:4:12:0,12,134   0/0:4,0:4:12:0,12,144   0/0:2,0:2:6:0,6,71      0/1:6,1:7:9:9,0,186     0/0:4,0:4:12:0,12,134   0/0:6,0:6:18:0,18,214   0/0:5,0:5:15:0,15,171   0/0:4,0:4:12:0,12,131   0/0:4,0:4:12:0,12,138   0/0:10,0:10:30:0,30,337 0/0:3,0:3:9:0,9,100     0/0:9,0:9:27:0,27,301

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.wAnnot.vs2.vcf.gz

#Genes of interest:
#ARFGEF1	8_68200276	20219
#NUP98	11_3735127	20219
#IPO4	14_24654486	20219
#IFT122	3_129207102	20219
#FUCA1	1_24175283	20501
#TPR	1_186303511	20501
#RNF213	17_78301691	20501

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.wAnnot.vs2.vcf.gz | grep -E '^#CHROM|ARFGEF1|NUP98|IPO4|IFT122|FUCA1|TPR|RNF213' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.wAnnot.vs2.KevGenesOfInterest1.vcf.gz

#zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.wAnnot.vs2.KevGenesOfInterest1.vcf.gz | perl -lane 'my @snpQuals = split(/;/, $F[9]); if (!$snpQuals[2]) { $snpQuals[2] = $snpQuals[0]; } my @printVals = ($F[0], $F[1], $F[2], $snpQuals[0], $snpQuals[1], $snpQuals[2]); for (my $i = 10; $i <= $#F; $i++) { my @indvQuals = split(/:/, $F[$i]); my @genos = split(/\//, $indvQuals[0]); my $AC1 = 0; if ($genos[0] eq "1") { $AC1++; } if ($genos[1] eq "1") { $AC1++; } push(@printVals, $AC1); } print join("\t", @printVals);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.wAnnot.vs2.KevGenesOfInterest1.Rinput.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.wAnnot.vs2.KevGenesOfInterest1.vcf.gz | perl -lane 'if ($F[0] =~ m/^#CHROM/) { print "CHROM\t", $F[1], "\t", $F[2], "\tExonicity\tGene\tAAEffect\t", join("\t", @F[10..$#F]); } else { my @snpQuals = split(/;/, $F[9]); if (!$snpQuals[2]) { $snpQuals[2] = $snpQuals[0]; } my @printVals = ($F[0], $F[1], $F[2], $snpQuals[0], $snpQuals[1], $snpQuals[2]); for (my $i = 10; $i <= $#F; $i++) { my @indvQuals = split(/:/, $F[$i]); my @genos = split(/\//, $indvQuals[0]); my $AC1 = 0; if ($genos[0] eq "1") { $AC1++; } if ($genos[1] eq "1") { $AC1++; } push(@printVals, $AC1); } print join("\t", @printVals); }' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.wAnnot.vs2.KevGenesOfInterest1.Rinput.gz

#zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.wAnnot.vs2.KevGenesOfInterest1.vcf.gz | perl -lane 'my @snpQuals = split(/;/, $F[9]); if (!$snpQuals[2]) { $snpQuals[2] = $snpQuals[0]; } my $AC1 = 0; for (my $i = 10; $i <= $#F; $i++) { my @indvQuals = split(/:/, $F[$i]); my @genos = split(/\//, $indvQuals[0]); if ($genos[0] eq "1") { $AC1++; } if ($genos[1] eq "1") { $AC1++; } } print $F[0], "\t", $F[1], "\t", $F[2], "\t", $snpQuals[0], "\t", $snpQuals[1], "\t", $snpQuals[2], "\t", $AC1; ' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.wAnnot.vs2.KevGenesOfInterest1.Rinput.Collapsed.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.wAnnot.vs2.KevGenesOfInterest1.vcf.gz | perl -lane 'if ($F[0] =~ m/^#CHROM/) { print "CHROM\t", $F[1], "\t", $F[2], "\tExonicity\tGene\tAAEffect\tACTotal"; } else { my @snpQuals = split(/;/, $F[9]); if (!$snpQuals[2]) { $snpQuals[2] = $snpQuals[0]; } my $AC1 = 0; for (my $i = 10; $i <= $#F; $i++) { my @indvQuals = split(/:/, $F[$i]); my @genos = split(/\//, $indvQuals[0]); if ($genos[0] eq "1") { $AC1++; } if ($genos[1] eq "1") { $AC1++; } } print $F[0], "\t", $F[1], "\t", $F[2], "\t", $snpQuals[0], "\t", $snpQuals[1], "\t", $snpQuals[2], "\t", $AC1; }' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.wAnnot.vs2.KevGenesOfInterest1.Rinput.Collapsed.gz

[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.wAnnot.vs2.KevGenesOfInterest1.Rinput.Collapsed.gz | awk '{ print $7 }' | sort -rg | uniq -c
     2 	24
     3 	21
     7 	19
     8 	18
     2	17
     7 	16
     3 	15
     3 	14
     2 	12
     3	11
     8 	10
     4 	9
     2 	8
     4 	7
    11 	6
    23 	5
    35 	4
    32 	3
    57 	2
   163 	1

cp -p /home/michaelt/Scripts/R/BasicScript.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.PlotVarsAlongGenes.Collapsed.vs1.R

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.PlotVarsAlongGenes.Collapsed.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.wAnnot.vs2.KevGenesOfInterest1.Rinput.Collapsed.gz

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.PlotVarsAlongGenes.Collapse.vs1.*jpg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/.

#cp -p /home/michaelt/Scripts/R/BasicScript.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.PlotVarsAlongGenes.vs1.R
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.PlotVarsAlongGenes.Collapsed.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.PlotVarsAlongGenes.vs1.R

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.PlotVarsAlongGenes.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.wAnnot.vs2.KevGenesOfInterest1.Rinput.gz 

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.PlotVarsAlongGenes.vs1.*jpg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/.

scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/Data/UCSC/UCSC_hg19_CanonicalTranscripts_wSymbols.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/Data/UCSC/UCSC_hg19_ExonLocations.txt.gz /home/michaelt/Data/UCSC/.

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py /home/michaelt/Data/UCSC/UCSC_hg19_CreateCanonicalExonList.vs1.py

python /home/michaelt/Data/UCSC/UCSC_hg19_CreateCanonicalExonList.vs1.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_ExonLocations.txt.gz --file2 /home/michaelt/Data/UCSC/UCSC_hg19_CanonicalTranscripts_wSymbols.txt > /home/michaelt/Data/UCSC/UCSC_hg19_CanonicalExonList.txt 

~~~
#Some notes on the editing of /home/michaelt/Data/UCSC/UCSC_hg19_CreateCanonicalExonList.vs1.py
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles]$cat /home/michaelt/Data/UCSC/UCSC_hg19_Canonic
alExonList.txt | perl -lane 'print $#F;' | sort | uniq -c
15 10
7 11
3 13
2 14
239634 8
12 9

[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles]$cat /home/michaelt/Data/UCSC/UCSC_hg19_CanonicalExonList.txt | perl -lane 'if ($#F == 10) { print join("\t", @F) };'
chrX    155255322       155257848       155255322       155256502       uc011nad.1      0       uc011nad.1      DDX11L  family  member
chrX    155255322       155257848       155257000       155257109       uc011nad.1      1       uc011nad.1      DDX11L  family  member
chrX    155255322       155257848       155257494       155257848       uc011nad.1      2       uc011nad.1      DDX11L  family  member
chrY    59358328        59360854        59358328        59359508        uc011ncc.1      0       uc011ncc.1      DDX11L  family  member
chrY    59358328        59360854        59360006        59360115        uc011ncc.1      1       uc011ncc.1      DDX11L  family  member
chrY    59358328        59360854        59360500        59360854        uc011ncc.1      2       uc011ncc.1      DDX11L  family  member

python /home/michaelt/Data/UCSC/UCSC_hg19_CreateCanonicalExonList.vs1.py --file1 /home/michaelt/Data/UCSC/UCSC_hg19_ExonLocations.txt.gz --file2 /home/michaelt/Data/UCSC/UCSC_hg19_CanonicalTranscripts_wSymbols.txt > /home/michaelt/Data/UCSC/UCSC_hg19_CanonicalExonList.txt

[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles]$cat /home/michaelt/Data/UCSC/UCSC_hg19_CanonicalExonList.txt | perl -lane 'print $#F;' | sort | uniq -c                                                                                                                239673 8

#Hence the 'joining' of line2 at the beginning of the processing for the second file
~~~

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.PlotVarsAlongGenes.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.PlotVarsAlongGenes.vs2.R

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.PlotVarsAlongGenes.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVar.wAnnot.vs2.KevGenesOfInterest1.Rinput.gz /home/michaelt/Data/UCSC/UCSC_hg19_CanonicalExonList.txt

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.PlotVarsAlongGenes.vs2.*jpg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/.

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts | grep -E 'ARFGEF1|NUP98|IPO4|IFT122|FUCA1|TPR|RNF213' | perl -lane 'my @vals1 = split("_", $F[3]); $F[3] = $vals1[1]; print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts.20150502KevGeneRequest
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno2.maf1.skat0.epacts | grep -E 'ARFGEF1|NUP98|IPO4|IFT122|FUCA1|TPR|RNF213' | perl -lane 'my @vals1 = split("_", $F[3]); $F[3] = $vals1[1]; print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno2.maf1.skat0.epacts.20150502KevGeneRequest


#20150418 
#Getting any variant whose alternative allele copies only exist in these subset of individuals. By definition then only including variants whose alternative ACs are equal to or less than 24 (b/c 12 individuals are specifically being looked at here)
#Still using /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz file since geno80 step was done post subsetting to QCed white non-hispanic individuals
#E.g. there is no /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz -- only a ...Vs2... exists

#Troubleshooting code below (hence STDERR statements in commented out section of code here)
#zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf.gz | perl -lane 'my @SNPinfo = split(/;/, $F[7]); my $AC; foreach my $info1 (@SNPinfo) { if ($info1 =~ /^AC\=/) { $AC = ((split(/=/, $info1))[1]); print STDERR $info1, "\t", $AC;} } my $genoCount = 0; foreach (my $i = 9; $i <= $#F; $i++) { my @indvInfo = split(/\:/, $F[$i]); my @genoInfo = split(/\//, $indvInfo[0]); if ($genoInfo[0] == 1) { 
#$genoCount++; } if ($genoInfo[1] == 1) { print $genoCount++; } } if ($genoCount == $AC) { print STDERR $F[0], "\t", $F[1], "\t", $genoCount, "\t", $AC; print join("\t", @F); } ' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.vcf.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf.gz | perl -lane 'my @SNPinfo = split(/;/, $F[7]); my $AC; foreach my $info1 (@SNPinfo) { if ($info1 =~ /^AC\=/) { $AC = ((split(/=/, $info1))[1]); } } my $genoCount = 0; foreach (my $i = 9; $i <= $#F; $i++) { my @indvInfo = split(/\:/, $F[$i]); my @genoInfo = split(/\//, $indvInfo[0]); if ($genoInfo[0] == 1) { $genoCount++; } if ($genoInfo[1] == 1) { $genoCount++; } } if ($genoCount == $AC) { print join("\t", @F); } ' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.vcf.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs6.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.wAnnotVs6.vs2.vcf.gz

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | grep -E '20572|20306|20219|20170|20123|20093|20501|20698|20132|20661|20471|20575'

~~~
20219,20219,7,POOL109-07RL,White Hispanic,0,1,0,0,0,0,0,0,0,0
20132,20132,9,POOL30-09RL,White non-Hispanic,0,1,0,0,0,0,0,0,0,0
20123,20123,6,POOL102-06RL,White non-Hispanic,0,1,0,0,0,0,0,0,0,0
20306,20306,7,POOL13-07RL,White Hispanic,0,1,0,0,0,0,0,0,0,0
20170,20170,2,POOL174-02RL,White non-Hispanic,0,1,0,0,0,0,0,0,0,0
20501,20501,5,POOL15-05RL,White non-Hispanic,0,1,0,0,0,0,0,0,0,0
20698,20698,11,POOL92-11RL,White non-Hispanic,0,1,0,0,0,0,0,0,0,0
20093,20093,11,POOL87-11RL,White non-Hispanic,0,1,0,0,0,0,0,0,0,0
20572,20572,9,POOL109-09RL,White non-Hispanic,0,1,0,0,0,0,0,0,0,0
20575,20575,2,POOL114-02RL,White non-Hispanic,0,1,0,0,0,0,0,0,0,0
20471,20471,1,POOL159-01RL,White non-Hispanic,0,1,0,0,0,0,0,0,0,0
20661,20661,12,POOL163-12RL,White non-Hispanic,0,1,0,0,0,0,0,0,0,0
~~~

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .65 ) { print $1 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.qVal_lte65.GeneIDs

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.wAnnotVs6.vs2.vcf.gz | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.qVal_lte65.GeneIDs | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.wAnnotVs6.vs2.TopGenes_qVallte65.vcf.gz 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.wAnnotVs6.vs2.vcf.gz | perl -lane 'my @snpInfo2 = split(/;/, $F[9]); push(@F, $snpInfo2[1]); print join("\t", @F);' | grep -w -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.qVal_lte65.GeneIDs | perl -lane 'pop(@F); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.wAnnotVs6.vs2.TopGenes_qVallte65.vcf.gz

cat <(echo "CHR BP SNPID REF ALT LOC GENE VARTYPE AACHANGE NUMAC 20123   20219   20572   20575   20306   20471   20501   20661   20170   20132   20093   20698") <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.wAnnotVs6.vs2.TopGenes_qVallte65.vcf.gz | grep -v ^# | perl -ane 'print join("\t", @F[0..4]); 
my @SNPinfo2 = split(/;/, $F[9]); for (my $j = 0; $j <= 3; $j++) { if ($SNPinfo2[$j]) { print "\t", $SNPinfo2[$j]; } else { print "\t", "."; } }
my @SNPinfo1 = split(/;/, $F[7]); my $AC; foreach my $info1 (@SNPinfo1) { if ($info1 =~ /^AC\=/) { $AC = ((split(/=/, $info1))[1]); } } print "\t", $AC;
for (my $i = 10; $i <= $#F; $i++) { my @indvInfo1 = split(/\:/, $F[$i]); print "\t", $indvInfo1[0]; }
print "\n";') > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.wAnnotVs6.vs2.TopGenes_qVallte65.ForKevin

~~~
[  michaelt@node11  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.wAnnotVs6.vs2.TopGenes_qVallte65.ForKevin | perl -lane 'print $#F;' | sort | uniq -c
     80 21
~~~

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf.gz | perl -lane 'my @SNPinfo1 = split(/;/, $F[7]); my $AC; foreach my $info1 (@SNPinfo1) { if ($info1 =~ /^AC\=/) { $AC = ((split(/=/, $info1))[1]); } } if ($AC == 2) { print join("\t", @F); }' | grep \/1 | perl -lane 'my $genoCount = 0; foreach (my $i = 9; $i <= $#F; $i++) { my @indvInfo = split(/\:/, $F[$i]); my @genoInfo = split(/\//, $indvInfo[0]); if ($genoInfo[0] == 1) { $genoCount++; } if ($genoInfo[1] == 1) { $genoCount++; } } print $genoCount; ' | sort | uniq -c

~~~
[  michaelt@node11  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf.gz | perl -lane 'my @SNPinfo1 = split(/;/, $F[7]); my $AC; foreach my $info1 (@SNPinfo1) { if ($info1 =~ /^AC\=/) { $AC = ((split(/=/, $info1))[1]); } } if ($AC == 2) { print join("\t", @F); }' | grep \/1 | perl -lane 'my $genoCount = 0; foreach (my $i = 9; $i <= $#F; $i++) { my @indvInfo = split(/\:/, $F[$i]); my @genoInfo = split(/\//, $indvInfo[0]); if ($genoInfo[0] == 1) { $genoCount++; } if ($genoInfo[1] == 1) { $genoCount++; } } print $genoCount; ' | sort | uniq -c
	270 1
	5 2
[  michaelt@node11  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf.gz | perl -lane 'my @SNPinfo1 = split(/;/, $F[7]); my $AC; foreach my $info1 (@SNPinfo1) { if ($info1 =~ /^AC\=/) { $AC = ((split(/=/, $info1))[1]); } } if ($AC == 2) { print join("\t", @F); }' | grep 1\/1 | wc
	5     105    2923
~~~


ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.wAnnotVs6.vs2.TopGenes_qVallte65.ForKevin /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/HESNIndvs.MM5.PrivateVariants.AllVariants.TopGenes_qVal65.txt

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/HESNIndvs.MM5.PrivateVariants.AllVariants.TopGenes_qVal65.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForKevinO/.

#20150427 
#Specifically pulling out variant level information for SULT1C3 based on Kevin's request
#Done in P2












##SKAT0 Behavior Tests
##20140930 -- Setting up files to test the behavior of SKAT0 when purposely change the case/control designation of the different variant/allele copies used in the analysis, e.g. to give us a better idea of how the test statistic is behaving

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/CovariateTests

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.ExtractVCFVarsBasedOnAnnoFile.vs1.py 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | grep VDAC3 | grep exonic | grep nonsyn > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/VDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.VDAC3.Exonic.Nonsynonymous.txt

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.ExtractVCFVarsBasedOnAnnoFile.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/VDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.VDAC3.Exonic.Nonsynonymous.txt --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Toy.Exonic.Nonsynonymous.vcf.bgzip

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped | sort -k 1,1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous | grep VDAC3 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3

#20141002 -- Editing /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped so that the first handful of individuals are HIV+ and I can easily alter genotypes in the associated VDAC3 toy examples. I will be exchanging phenotype designations with other individuals in the .ped file so that the total number of HIVPROG + vs HIVPROG - is the same 

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.Orig.ped

#20141002 -- Scrubbing clean the .vcf file so have a version where everyone is 0/0 first, then creating copies where I manually include 1s among the first five individuals. Currently only going to do heterozygotes, and then will start moving some of the 1 allele copies from the first 5 individuals to the next set of individuals to look at behavior when mutation is among HIV+ and HIV- individuals. The file coding is 'XXXXXlYYYYY_XXXXXlYYYYY....' to explain how many copies of the 1 allele there are per variant and which individual(s) they are in, where X = number of 1 copies in the first 5 individuals, whom are HIV+, and where Y = the number of 1 copies in the next 5 individuals, whom are HIV-  

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.vcf.bgzip | perl -lane 'if ($F[0] !~ /^#/) { for (my $i = 9; $i <= $#F; $i++) { my @vals1 = split(/\:/, $F[$i]); $vals1[0] = "0/0"; $F[$i] = join(":", @vals1); } } print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.All0s.vcf.bgzip

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.All0s.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_00000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_00000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip 
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_00000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip.gz
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.All0s.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip.gz
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.All0s.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00000l00000_00000l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00000l00000_00000l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00000l00000_00000l00000.vcf.bgzip.gz
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.All0s.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00000l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00000l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00000l00000.vcf.bgzip.gz
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.All0s.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip.gz

#QC check line
tail -n 5 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10* | perl -lane 'print join("\t", @F[0..14]);'

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.All0s.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.All0s.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_00000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_00000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip 
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_00000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip.gz
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.All0s.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l11111.vcf.bgzip
#Manually edit file to include expected 1 genotypes now                                                                                                                                                                                                                                                                    
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l11111.vcf.bgzip
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l11111.vcf.bgzip.gz
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now                                                                                                                                                                                                                                                                    
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip.gz

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now                                                                                                                                                                                                                                                                    
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00000_00010l00000_00001l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now                                                                                                                                                                                                                                                                    
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00100_00010l00000_00001l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now                                                                                                                                                                                                                                                                    
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00100_00010l00010_00001l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now                                                                                                                                                                                                                                                                    
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00100_00010l00010_00001l00001.vcf.bgzip
#Manually edit file to include expected 1 genotypes now                                                                                                                                                                                                                                                                    
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00000_00010l00000_00001l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00100_00010l00000_00001l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00100_00010l00010_00001l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00100_00010l00010_00001l00001.vcf.bgzip
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip.gz
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00000_00010l00000_00001l00000.vcf.bgzip.gz
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00100_00010l00000_00001l00000.vcf.bgzip.gz
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00100_00010l00010_00001l00000.vcf.bgzip.gz
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00100_00010l00010_00001l00001.vcf.bgzip.gz

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_00000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_00000l00000_00000l00000_00000l00000_00000l00000.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00000l00000_00000l00000_00000l00000.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00000l00000_00000l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00000l00000_00000l00000.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00000l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00000l00000.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.Pheno1.maf1.skat0.noAdj

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_00000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_00000l00000_00000l00000_00000l00000_00000l00000.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_01000l00000_00100l00000_00010l00000_00001l00000.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l11111.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l11111.Pheno1.maf1.skat0.noAdj

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l00000_00100l00000_00010l00000_00001l00000.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00000_00010l00000_00001l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00000_00010l00000_00001l00000.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00100_00010l00000_00001l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00100_00010l00000_00001l00000.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00100_00010l00010_00001l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00100_00010l00010_00001l00000.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00100_00010l00010_00001l00001.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l10000_01000l01000_00100l00100_00010l00010_00001l00001.Pheno1.maf1.skat0.noAdj

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3.OneVar
#Manually edit  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 to contain just the first variant

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_00000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3.OneVar --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_00000l00000_00000l00000_00000l00000_00000l00000.Pheno1.maf1.skat0.noAdj.oneVar
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_00000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3.OneVar --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_00000l00000_00000l00000_00000l00000_00000l00000.Pheno1.maf1.skat0.noAdj.oneVar


cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.All0s.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_00000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now                                                                                                                                                                                                                                                                    
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.All0s.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_00000l02000_00000l00000_00000l00000_00000l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now                                                                                                                                                                                                                                                                    
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now                                                                                                                                                                                                                                                                    
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_02000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now                                                                                                                                                                                                                                                                    
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.All0s.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_00000l11111_11111l00000_00000l11111_11111l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now                                                                                                                                                                                                                                                                    
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.All0s.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.22222l00000_00000l22222_22222l00000_00000l22222_22222l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now                                                                                                                                                                                                                                                                    
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_00000l01000_00100l00000_00000l00010_00001l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now                                                                                                                                                                                                                                                                    
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_00000l02000_00200l00000_00000l00020_00002l00000.vcf.bgzip
#Manually edit file to include expected 1 genotypes now                                                                                                                                                                                                                                                                    

/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_00000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_00000l02000_00000l00000_00000l00000_00000l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_02000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_00000l11111_11111l00000_00000l11111_11111l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.22222l00000_00000l22222_22222l00000_00000l22222_22222l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_00000l01000_00100l00000_00000l00010_00001l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_00000l02000_00200l00000_00000l00020_00002l00000.vcf.bgzip
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_00000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip.gz
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_00000l02000_00000l00000_00000l00000_00000l00000.vcf.bgzip.gz
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip.gz
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_02000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip.gz
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_00000l11111_11111l00000_00000l11111_11111l00000.vcf.bgzip.gz
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.22222l00000_00000l22222_22222l00000_00000l22222_22222l00000.vcf.bgzip.gz
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_00000l01000_00100l00000_00000l00010_00001l00000.vcf.bgzip.gz
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_00000l02000_00200l00000_00000l00020_00002l00000.vcf.bgzip.gz

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_00000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_00000l00000_00000l00000_00000l00000_00000l00000.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_02000l00000_00000l00000_00000l00000_00000l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_02000l00000_00000l00000_00000l00000_00000l00000.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_01000l00000_00100l00000_00010l00000_00001l00000.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_02000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_02000l00000_00100l00000_00010l00000_00001l00000.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_00000l11111_11111l00000_00000l11111_11111l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.11111l00000_00000l11111_11111l00000_00000l11111_11111l00000.vcf.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.22222l00000_00000l22222_22222l00000_00000l22222_22222l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.22222l00000_00000l22222_22222l00000_00000l22222_22222l00000.vcf.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_00000l01000_00100l00000_00000l00010_00001l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_00000l01000_00100l00000_00000l00010_00001l00000.vcf.Pheno1.maf1.skat0.noAdj
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_00000l02000_00200l00000_00000l00020_00002l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.20000l00000_00000l02000_00200l00000_00000l00020_00002l00000.vcf.Pheno1.maf1.skat0.noAdj

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --skat-flat --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.Pheno1.maf1.skat0.flatPrior.noAdj

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/MultipleRuns/
for i in {11..100}
do
	epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.VDAC3 --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.Toy.ped --pheno HIVPROG --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/SKAT0BehaviorTests/ToyVDAC3/MultipleRuns/AllPools.Vs2.wHapMap3Data.justWhite.QCed.DropIBD.VDAC3.Exonic.Nonsynonymous.10000l00000_01000l00000_00100l00000_00010l00000_00001l00000.Pheno1.maf1.skat0.Adj.run$i
done

#20141009
#going to run 100 versions of the real data EPACTS SKAT-O analysis using Pheno1 ExonicNonsynonymous in order to get a feel for the variation present in the smal sample-size adjustment permutation process 
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/ExonicNonsynonymous
for i in {1..100}
do
	epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/ExonicNonsynonymous/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.try2.run$i
done

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous
for i in {1..100}
do
	epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.try2.run$i
done

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.try2.run1.epacts | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.try2.run1.epacts.GeneIDList

#Made a default bash shell script to start other bash shell scripts off with
vi /home/michaelt/Scripts/Bash/BashScript.InputVarsPlusTime.vs1.sh

cp -p  /home/michaelt/Scripts/Bash/BashScript.InputVarsPlusTime.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh

#Going to cat all *epacts files from the multiple runs then grep the GeneID of importance one at a time to print out a table that has 'GeneID' then all that GeneID's p-values in a single row. May have to create an auxilary scripts or extra perl code one-liner to double-check every row has the same number of entries in case certain GeneIDs occasionally drop out 

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.try2.run1.epacts.GeneIDList TruePheno Pheno3 ExonicNonsynonymous

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.ExonicNonsynonymous.output | grep -v NA > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.ExonicNonsynonymous.output.noNAs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.ExonicNonsynonymous.output.noNAs | R -q -e "Data1 <- read.table(file('stdin'), header=F, row.names=1); print(paste(row.names(Data1), mean(Data1[,1]), var(Data1[,1]), sep=\" \"))" 

#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.ExonicNonsynonymous.output.noNAs | R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,var))); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.ExonicNonsynonymous.output.noNAs.histograms.jpg\", width=2500, height=2000, res=300); print(Data1[14,]); par(mfrow=c(2,2)); hist(as.numeric(Data1[40,])); hist(as.numeric(Data1[156,])); hist(as.numeric(Data1[261,])); hist(as.numeric(Data1[345,])); dev.off();" > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.ExonicNonsynonymous.output.noNAs.Routput
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.ExonicNonsynonymous.output.noNAs | R -q -e "Data1 <- read.table(file('stdin'), header=T, row.names=1); print(cbind(apply(Data1,1,mean), apply(Data1,1,sd))); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.ExonicNonsynonymous.output.noNAs.histograms.jpg\", width=2500, height=2000, res=300); par(mfrow=c(2,2)); hist(as.numeric(Data1[40,])); hist(as.numeric(Data1[156,])); hist(as.numeric(Data1[261,])); hist(as.numeric(Data1[345,])); dev.off();" > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.ExonicNonsynonymous.output.noNAs.Routput

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.ExonicNonsynonymous.output.noNAs.histograms.jpg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/.

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.ExonicNonsynonymous.output.noNAs | awk '{ print $1 }' | sed 's/://g' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.ExonicNonsynonymous.output.noNAs.GeneIDList

for gene in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.ExonicNonsynonymous.output.noNAs.GeneIDList`
do
	paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.ExonicNonsynonymous.output.noNAs.Routput | grep $gene) <(cat *epacts | grep $gene | tail -n 1 | awk '{ print $7, "\t", $8, "\t", $9 }')  >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/ExonicNonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.TruePheno.Pheno3.ExonicNonsynonymous.output.noNAs.Routput.wAlleleCountData 
done

#20150114
#I did a bunch of work trying to figure out the behavior of EPACTS and SKAT in relation to how EPACTS runs SKAT and how I could run SKAT independently, both through R, to make sure what EPACTS was doing with SKAT made sense (and I could replicate it independently). However, as of 20150114 there was no record of the work I had done (I think anyways) but one of my screens that is still open happen to contains the history of a lot of that work. So momentarily saving that history into a file and logging it here so that I have some sort of record of this work

history > /data/userdata/pg/michaelt/Data/ALL_MAPPING/MainScript1.PG_HIVArray1.EPACTSSKATWorkLog.addition1.output

#On another screen with a bit of information at the top
history > /data/userdata/pg/michaelt/Data/ALL_MAPPING/MainScript1.PG_HIVArray1.EPACTSSKATWorkLog.addition2.output

PWD: /home/michaelt/Software/EPACTS-3.2.4/bin
[  michaelt@login2  ~/Software/EPACTS-3.2.4/bin]$cmp epacts-group epacts-group.orig
epacts-group epacts-group.orig differ: byte 811, line 43

#I added code in epacts-group to try and include skatCommonRare as an option but it never really went anywhere. Nothing else appears to be altered

PWD: /home/michaelt/Software/EPACTS-3.2.4/data
#In /home/michaelt/Software/EPACTS-3.2.4/data/group.skat.R I added the following (these are instances where rho estimates are weird and I just wanted to print more info to help diagnose the situation -- as you'll see even the in-house code included mentions of 'temporary solutions'. My additions are only the print commands in both blocks)

~~~

#	print(r)
#	print(pheno)
#	print(cov)
#	print(lm((pheno~cov-1)))
#	print(.libPaths())
#	print(sessionInfo())
#	print(betas)

      p <- r$p.value
      if ( m == 1 ) {
        rho <- NA
      }
      else {
        if ( skatCommonRare ) {
#		rho <- paste(r$param$, , , sep=",")
	} 
	else {
		rho <- r$param$rho_est
	        if ( length(rho) > 1 ) {
	          rho <- "NA"  ## temporary solution
	        	print("Here1")
			print(r)
			print(r$param)
	        }
		if ( length(rho) == 0 ) {
	          rho <- "-9"  ## temporary solution
	        	print("Here2")
			print(r)
			print(r$param)
		}
      	}

~~~

#It looks like any 'changes' I made to both epactsGroup.R or group.skat.R have been commented out since (except for those print statements above), so there's a log of some sort of what I originally included. Most of these additions were just print statements to understand what the code (EPACTS' code) was doing, as well as an attempt to get the program to run skatCommonRare (which I ended up demuring from because skatCommonRare didn't do the rho parameter space search, as well as the belief that ultimately it wouldn't add that much power in exchange for using and having to defend a much more novel program implementation at the time)




##20141006
##HESeroNegative Fix

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs2.py

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno.OLDvs1

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.pheno.OLDvs1

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.pheno

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped.OLDvs1

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.pheno | awk '{ print $1, "\t", $2, "\t0\t0\t0\t", $3, "\t", $4, "\t", $5, "\t", $6 } ' | sed 's/-9/NA/g' | perl -lane 'print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar | awk '{ print $4, "\t", $6, "\t", $7 "\t", $8, "\t", $9, "\t", $10, "\t", $11,  "\t", $12, "\t", $13, "\t", $14,  "\t", $15 }' | paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped.vs2

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped.vs2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped

#Add line #FAM_ID IND_ID FAT_ID MOT_ID SEX HIVPROG HIVPROGHE AIDS AIDSEXTR RMID PC1 PC2 PC3 PC4 PC5 PC6 PC7 PC8 PC9 PC10 to /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped

paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped.OLDvs1 | perl -lane 'print join("\t", @F[0..6]);') <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped | perl -lane 'print join("\t", @F[0..6]);')

#Go through running epacts bash commands, mv all the old ones with Pheno2 or HIVPROGHE into a separate folder and redo those runs
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 2 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs.maf05.pheno2

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs.maf05.pheno2.assoc.logistic Pheno2_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs.maf05.pheno2.assoc.logistic.QQplot.jpeg &

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs.OLDvs1

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($4 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count.OLDvs1.gz

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.OLDvs1.gz

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.log /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.OLDvs1.log 

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg &

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.AllPhenos.frq.AndCount.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.AllPhenos.frq.AndCount.OLDvs1.gz

paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq.count | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.AllPhenos.frq.AndCount.gz

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq.count

#gunzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq.count.gz

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.OLDvs1.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs5.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHapMap3Data.justWhite.QCed.DropIBD.AllPhenos.frq.AndCount.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 30; $i <= 40; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary.NonSyn.top25.Variants.csv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary.NonSyn.top25.Variants.OLDvs1.csv

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/VLookUp.Python.PullOutEPACTSVariants.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.top25 --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | cut -d, -f 1-6,18-29,35,37,39,41 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary.NonSyn.top25.Variants.csv

#Added the following header
#Chr,BP,rsID,refAllele,otherAllele,AncAllele,AC_AllIndvs,AC_SeroNeg,AC_SeroNegHE,AC_SeroPos,AC_RapidVRapid,AC_SlowVSlow,AC_VeryRapid,AC_VerySlow,VarType,GeneID,SNVType,AAChange,Sift,Polyphen2,LRT,MutationTaster

#Just for checking things
paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary.NonSyn.top25.Variants.OLDvs1.csv | perl -F, -lane 'print join(",", @F[6..13]);') <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary.NonSyn.top25.Variants.csv | perl -F, -lane 'print join(",", @F[6..13]);') | vi -

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary.NonSyn.top25 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/ForSumit.Top25.NonSynGenes
ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/6thRun.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary.NonSyn.top25.Variants.csv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/ForSumit.Top25.NonSynGenes.Variants.csv

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/ForSumit.Top25.NonSynGenes /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/ForSumit.Top25.NonSynGenes.Variants.csv mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/.

##20141114
##Getting Malim gene results for Steve from P2

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.epacts | grep -f /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Second_capture_P2_gene-list.MalimSpecific.noMeta.txt | awk '{ if ($10 != "NA") { print $0 } } ' | sort -rg -k 10,10 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.epacts.MalimGenes 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts | grep -f /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Second_capture_P2_gene-list.MalimSpecific.noMeta.txt | awk '{ if ($10 != "NA") { print $0 } } ' | sort -rg -k 10,10 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts.MalimGenes 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.epacts | grep -f /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Second_capture_P2_gene-list.MalimSpecific.noMeta.txt | awk '{ if ($10 != "NA") { print $0 } } ' | sort -rg -k 10,10 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.epacts.MalimGenes 

##20141129
##Doing some extra coding things to compare results of the mapping/calling pipeline with GATK to results in FreeBayes and Samtools
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.DropOff.VCFVariants.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz --window 1000 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.DropOffTarg_1kb.vcf.gz 


##20150106
##Creating code/scripts to get list of individuals per variant that contain the alternative allele for that variant, given a list of specific variants

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SpecificGeneVCFs

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.vcf.gz | grep -E '^#|VDAC3' | grep -E '^#|nonsynonymous|stop' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SpecificGeneVCFs/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVar.VDAC3.Exonic.Nonsyn.vcf.gz

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.vs1.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SpecificGeneVCFs/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVar.VDAC3.Exonic.Nonsyn.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.vs1.VDAC3.Exonic.Nonsyn.output

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs1.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.vs1.VDAC3.Exonic.Nonsyn.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs1.VDAC3.Exonic.Nonsyn.output

#Need to add a line to #CHROM header of VCF file for 'ANNOVAR' column, otherwise the individual IDs are all off by one, e.g. there's an extra column in the variant rows but not the header row leading to name errors
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs2.py

##20150109 
##Getting allele counts/frequencies for specific variants Sudhir requested based on other groups' work involving them

exonic ARFGEF1 nonsynonymous_SNV NM_006421 c.A941G p.Y314C
exonic NUP98 nonsynonymous_SNV NM_005387 c.G2549A p.G850E
exonic IPO4 nonsynonymous_SNV NM_024658 c.G1311A p.M437I
exonic IFT122 stoploss_SNV NM_052990 c.A1521T p.X507C
exonic FUCA1 nonsynonymous_SNV NM_000147 c.T1016C p.I339T
exonic TPR nonsynonymous_SNV NM_003292 c.A5128G p.T1710A
exonic RNF213 nonsynonymous_SNV NM_001256071 c.C3269T p.T1090M

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | grep c.A941G 
8,68200276_1,rs76811394,T,C,T,134.63,VQSRTrancheSNP90.00to99.00,AA=T;AC=1;AF=5.531e-04;AN=1808;BaseQRankSum=3.147;DB;DP=13747;Dels=0.00;FS=5.328;HRun=0;HaplotypeScore=1.0303;InbreedingCoeff=-0.0043;MLEAC=1;MLEAF=5.531e-04;MQ=60.00;MQ0=0;MQRankSum=0.938;POSITIVE_TRAIN_SITE;QD=11.22;ReadPosRankSum=1.713;VQSLOD=2.63;culprit=HaplotypeScore,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,exonic,ARFGEF1,nonsynonymous SNV,NM_006421:c.A941G:p.Y314C,,0.18,0.969028,C,0.83,T,0.0,B,0.876934,N,0.754805,D,4.21,8,68200276,68200276,T,C

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | grep c.G2549A
11,3735127_1,.,C,T,C,265.57,VQSRTrancheSNP99.00to99.90,AA=C;AC=1;AF=5.525e-04;AN=1810;BaseQRankSum=0.621;DP=23164;Dels=0.00;FS=2.205;HRun=2;HaplotypeScore=2.1553;InbreedingCoeff=-0.0013;MLEAC=1;MLEAF=5.525e-04;MQ=60.00;MQ0=0;MQRankSum=-0.092;QD=13.98;ReadPosRankSum=-1.186;VQSLOD=1.06;culprit=HaplotypeScore,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,exonic,NUP98,nonsynonymous SNV,NM_005387:c.G2549A:p.G850E,,0,0.998911,C,0.99,D,0.321,P,1.0,D,0.999879,D,5.28,11,3735127,3735127,C,T

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | grep c.G1311A
14,24654486_1,.,C,T,C,363.55,VQSRTrancheSNP99.00to99.90,AA=C;AC=1;AF=5.525e-04;AN=1810;BaseQRankSum=1.320;DP=28335;Dels=0.00;FS=4.522;HRun=0;HaplotypeScore=3.0813;InbreedingCoeff=-0.0009;MLEAC=1;MLEAF=5.525e-04;MQ=59.99;MQ0=0;MQRankSum=-0.109;QD=7.57;ReadPosRankSum=1.683;VQSLOD=0.573;culprit=HaplotypeScore,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,exonic,IPO4,nonsynonymous SNV,NM_024658:c.G1311A:p.M437I,,0.04,0.999236,C,0.85,T,0.834,P,1.0,D,0.834784,D,5.61,14,24654486,24654486,C,T

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | grep c.A1521T
3,129207102_1,rs146874343,C,T,C,472.56,VQSRTrancheSNP99.00to99.90,AA=C;AC=1;AF=5.501e-04;AN=1818;BaseQRankSum=9.069;DB;DP=24880;Dels=0.00;FS=3.168;HRun=0;HaplotypeScore=3.6113;InbreedingCoeff=-0.0014;MLEAC=1;MLEAF=5.501e-04;MQ=43.11;MQ0=0;MQRankSum=-1.967;QD=8.75;ReadPosRankSum=-0.348;VQSLOD=0.327;culprit=HaplotypeScore,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,exonic,IFT122,stoploss SNV,NM_052990:c.A1521T:p.X507C,,,,,,,,,,,,,,3,129207102,129207102,C,T 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | grep c.T1016C
1,24175283_1,.,A,G,A,227.53,VQSRTrancheSNP99.00to99.90,AA=A;AC=1;AF=5.488e-04;AN=1822;BaseQRankSum=-1.461;DP=23298;Dels=0.00;FS=3.295;HRun=0;HaplotypeScore=2.5523;InbreedingCoeff=-0.0011;MLEAC=1;MLEAF=5.488e-04;MQ=60.00;MQ0=0;MQRankSum=-0.967;QD=10.83;ReadPosRankSum=-1.099;VQSLOD=0.951;culprit=HaplotypeScore,0.000643501,0.00141643,0.00297619,0,0,0,0,0,1,1,1,0,0,0,0,0,exonic,FUCA1,nonsynonymous SNV,NM_000147:c.T1016C:p.I339T,,0.02,0.998825,C,0.725216,-9,0.0050,B,0.888728,N,0.485564,N,4.81,1,24175283,24175283,A,G
#HESN

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | grep c.A5128G
1,186303511_1,.,T,C,T,250.54,VQSRTrancheSNP99.00to99.90,AA=T;AC=1;AF=5.488e-04;AN=1822;BaseQRankSum=-6.471;DP=21581;Dels=0.00;FS=0.690;HRun=0;HaplotypeScore=2.4101;InbreedingCoeff=-0.0016;MLEAC=1;MLEAF=5.488e-04;MQ=60.00;MQ0=0;MQRankSum=0.822;QD=13.19;ReadPosRankSum=0.100;VQSLOD=1.41;culprit=HaplotypeScore,0.000643501,0.00141643,0.00297619,0,0,0,0,0,1,1,1,0,0,0,0,0,exonic,TPR,nonsynonymous SNV,NM_003292:c.A5128G:p.T1710A,,0.18,0.997877,C,0.97,D,0.640678,-9,0.999996,D,0.42172,N,5.25,1,186303511,186303511,T,C
#HESN

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | grep c.C3269T
17,78301691_1,.,C,T,C,412.60,VQSRTrancheSNP99.00to99.90,AA=C;AC=1;AF=5.593e-04;AN=1788;BaseQRankSum=6.238;DP=31467;Dels=0.00;FS=0.625;HRun=0;HaplotypeScore=3.8645;InbreedingCoeff=-0.0007;MLEAC=1;MLEAF=5.593e-04;MQ=60.00;MQ0=0;MQRankSum=-0.686;QD=11.79;ReadPosRankSum=-1.594;VQSLOD=0.488;culprit=HaplotypeScore,0.000649351,0.00142045,0.00297619,0,0,0,0,0,1,1,1,0,0,0,0,0,exonic,RNF213,nonsynonymous SNV,NM_001256071:c.C3269T:p.T1090M,,0.25,,,,,,,,,,,,17,78301691,78301691,C,T
#HESN

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.vcf.gz | grep -E '^#|68200276|3735127|24654486|129207102|24175283|186303511|78301691' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SpecificGeneVCFs/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVar.SudhirRequest20150109.vcf.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz | grep -E '^#|68200276|3735127|24654486|129207102|24175283|186303511|78301691' | gzip >  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SpecificGeneVCFs/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.AllGenesPerVar.vs2.SudhirRequest20150109.vcf.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SpecificGeneVCFs/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.AllGenesPerVar.vs2.SudhirRequest20150109.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.vs1.SudhirRequest20150109.output

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.vs1.SudhirRequest20150109.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs1.SudhirRequest20150109.output

Gene	Location	MACID
ARFGEF1	8_68200276      20219
NUP98	11_3735127      20219
IPO4	14_24654486     20219
IFT122	3_129207102     20219
FUCA1	1_24175283      20501
TPR	1_186303511     20501
RNF213	17_78301691     20501

##20150122
##Unsure where this file came from or when it was created? appears to use Samtools sort to create all-individual merge .bam file but picard-merge is what should be used, and what is used, in vs2 of the file, which is the only record of this file being used in the most recent run (e.g. no record of vs3 being used anywhere or in P2 record file)

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs3._20150122_NORECORDOFUSE_probablyusedjusttocomparesortsmaybe.sh

##20150126
##.vcf file and .pheno file for Sudhir to check and compare against GWAS results (for whatever collection of individuals overlap between our study and the GWAS one)

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/ForSudhir.P1.GATK.QCedVariants.QCedIndvs.justWhite.vcf.gz

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/ForSudhir.Phenotypes.txt

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/ForSudhir.P1.GATK.QCedVariants.QCedIndvs.justWhite.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/ForSudhir.Phenotypes.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/. 



##20150126
##Noticed the timestamp on -rw-rw-r-- 1 michaelt pg        177396566 Nov  1 19:52 AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz was later than the timestamp on -rw-rw-r-- 1 michaelt pg        182849697 Apr 11  2014 AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz but I think this was most likely because I had to recreate AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz and not because it had been updatd
##However, to check that the two files are the same, recreating the .bgzip.gz version and going to run cmp on the two files to make sure they are equivalent

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ORIG.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz | awk '{ if ($1 == "23") { $1 = "X"} print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip

/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip

 1115  cmp ORIG.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz
  1116  head -n 10 AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz
   1117  /home/shared/software/tabix-0.2.5/bgzip
    1118  cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ORIG.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ORIG.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.vs2.bgzip.gz
     1119  /home/shared/software/tabix-0.2.5/bgzip -d /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ORIG.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.vs2.bgzip.gz
      1120  ls -lrt
       1121  head -n 10 ORIG.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.vs2.bgzip
        1122  /home/shared/software/tabix-0.2.5/bgzip -d AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip.gz
	 1123  ls -lrt
	  1124  wc ORIG.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.vs2.bgzip
	   1125  wc AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip

[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$wc ORIG.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.vs2.bgzip
     64015   50216707 1162570014 ORIG.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.vs2.bgzip
     [  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$wc AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip
          64015   50216707 1178830267 AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip

#So the files look pretty much the same but because there's a slight difference in the 'byte' count between the two, I'm going to update the bgzip file and rerun the EPACTS analyses (TruePheno and FakePhenos) just to be thorough. I will compare the output of the new run to the previous one to make sure nothing's really changed, and if for some reason something has I'll look further into it

rm ORIG.AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.vs2.bgzip AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.bgzip



##20150209
##Running fastQC over fastq.gz files to doublecheck everything looked fine, just to make sure
##20150825 CHECK_0: Check this stuff?

wget http://www.bioinformatics.babraham.ac.uk/projects/fastqc/fastqc_v0.11.2.zip /home/michaelt/Software/. 

unzip /home/michaelt/Software/fastqc_v0.11.2.zip

#Adding in a command to run fastQC -- going to rerun launcher with all other steps commented out
#Going to edit launcher to include vs3 of this file, but going to keep the version of the launcher as vs3 since this is just a single digit replacement in that file

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.vs3.sh

head -n 1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output.top1
head -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output.top10

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output.top1
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output.top10
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output | perl -ane 'my $baseFile1; if ($F[1] =~ m/(.*.RL\d+).sff/) { $baseFile1 = $1; } print $F[0], "/", $1, ".QCed.fastq.gz ";' | awk '{ print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.AllfastqFilesSingleLine.output

#Below command would take too long it seems like
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.AllfastqFilesSingleLine.output | xargs /home/michaelt/Software/FastQC/fastqc

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py /home/michaelt/Scripts/Python/Python.Utility.CutFileIntoNSizedLineChunks.vs1.py

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/FileChunks

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/FileChunks/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output

python /home/michaelt/Scripts/Python/Python.Utility.CutFileIntoNSizedLineChunks.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/FileChunks/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output --chunkSize 100

for i in {1..37}
do

	cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/FileChunks/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.chunk${i}.output | perl -ane 'my $baseFile1; if ($F[1] =~ m/(.*.RL\d+).sff/) { $baseFile1 = $1; } print $F[0], "/", $1, ".QCed.fastq.gz ";' | awk '{ print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/FileChunks/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.AllfastqFilesSingleLine.chunk${i}.output

done

cp -p /home/michaelt/Scripts/Bash/BashScript.InputVarsPlusTime.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.fastQCgroupOfFiles.vs1.sh

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.fastQCgroupOfFiles.Launcher.vs1.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.fastQCgroupOfFiles.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/FileChunks/ DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.AllfastqFilesSingleLine.output




http://genomemedicine.com/content/pdf/gm432.pdf
http://www.ncbi.nlm.nih.gov/pubmed/25078893
http://www.ncbi.nlm.nih.gov/pubmed/24558117
http://www.ncbi.nlm.nih.gov/pubmed/24086590
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3605113/
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3912837/
http://www.nature.com/nature/journal/v511/n7510/full/nature13595.html
http://www.nature.com/ng/journal/v44/n4/full/ng.2213.html
http://r.789695.n4.nabble.com/How-to-get-R-to-compile-with-PNG-support-td3457938.html



##20150315
##Downloading and comparing overlap between ExAC data and my own for QC sake

mkdir /home/michaelt/Data/ExAC

cd /home/michaelt/Data/ExAC

wget ftp://ftp.broadinstitute.org/pub/ExAC_release/release0.3/*  

python /home/michaelt/Scripts/Python/DataAnalysis.All.CompareTwoVCFDatasets.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --file2 /home/michaelt/Data/ExAC/ExAC.r0.3.sites.vep.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.gz

cp -p /home/michaelt/Scripts/Python/DataAnalysis.All.CompareTwoVCFDatasets.vs1.py /home/michaelt/Scripts/Python/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.py

python /home/michaelt/Scripts/Python/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --file2 /home/michaelt/Data/ExAC/ExAC.r0.3.sites.vep.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.gz | grep VCF1 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF1.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.gz | grep VCF3a | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.gz | grep VCF3b | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3b.gz

[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF1.gz | wc
52524 41231340 962454798
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a.gz | wc
11339 8901115 213930911
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3b.gz | wc
11339  102051 33860693

cat /home/michaelt/Data/ExAC/exome_calling_regions.v1.interval_list | awk '{ print $1, "\t", $2, "\t", $3 }' | grep -v \@ > /home/michaelt/Data/ExAC/exome_calling_regions.v1.interval_list.bed

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --bed /home/michaelt/Data/ExAC/exome_calling_regions.v1.interval_list.bed --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.ExACOverlapBed 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --exclude-bed /home/michaelt/Data/ExAC/exome_calling_regions.v1.interval_list.bed --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.ExACNonOverlapBed

[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.ExACOverlapBed.recode.vcf | wc                                                                                                                                                                                          
13279 10311907 247927019
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.ExACNonOverlapBed.recode.vcf | wc
50840 39759731 928181016

python /home/michaelt/Scripts/Python/DataAnalysis.All.CompareTwoVCFDatasets.vs1.py --file1 <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a.gz | perl -lane 'print join("\t", @F[1..$#F]);') --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.ExACOverlapBed.recode.vcf | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_Final_OverlapExACBed.GATK_Final_OverlapWithExACVCF.output.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_Final_OverlapExACBed.GATK_Final_OverlapWithExACVCF.output.gz | grep VCF1 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_Final_OverlapExACBed.GATK_Final_OverlapWithExACVCF.output.VCF1.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_Final_OverlapExACBed.GATK_Final_OverlapWithExACVCF.output.gz | grep VCF2 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_Final_OverlapExACBed.GATK_Final_OverlapWithExACVCF.output.VCF2.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_Final_OverlapExACBed.GATK_Final_OverlapWithExACVCF.output.gz | grep VCF3a | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_Final_OverlapExACBed.GATK_Final_OverlapWithExACVCF.output.VCF3a.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_Final_OverlapExACBed.GATK_Final_OverlapWithExACVCF.output.gz | grep VCF3b | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_Final_OverlapExACBed.GATK_Final_OverlapWithExACVCF.output.VCF3b.gz 

[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_Final_OverlapExACBed.GATK_Final_OverlapWithExACVCF.output.VCF1.gz | wc
43   33755  809736
#A few variants are present in the VCF file of ExAC that are suppose to be outside the bed file
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_Final_OverlapExACBed.GATK_Final_OverlapWithExACVCF.output.VCF2.gz | wc
1855 1456175 34856448
#Variants that are in the bed file overlap with ExAC that are not in the ExAC vcf file
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_Final_OverlapExACBed.GATK_Final_OverlapWithExACVCF.output.VCF3a.gz | wc
11296 8867360 213121132
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.GATK_Final_OverlapExACBed.GATK_Final_OverlapWithExACVCF.output.VCF3b.gz | wc
11296 8867360 213121132
#Variants that are both in the 'bed file overlap' from ExAC and the 'ExAC vcf'

paste <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a.gz | perl -ane 'print $F[1], "_", $F[2], "\t"; my @vals1 = split(/\;/, $F[8]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { print $entry1; } } print "\n";') <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3b.gz | perl -ane 'print $F[1], "_", $F[2], "\t"; my @vals1 = split(/\;/, $F[8]); my $AC_NFE; my $AN_NFE; foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AC_NFE=/) { $AC_NFE = (split(/,/, (split(/=/, $entry1))[1]))[0]; } if ($entry1 =~ m/^AN_NFE=/) { $AN_NFE = (split(/=/, $entry1))[1]; } } if (($AC_NFE) && ($AN_NFE)) { print "AF_NFE=", $AC_NFE/$AN_NFE; } else { print "NA"} print "\n";') > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs

##cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs | grep -v NA | perl -lane '$F[1] = (split(/=/, $F[1]))[1]; $F[3] = (split(/=/, $F[3]))[1]; print join("\t", @F);' | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs.AFplots.jpeg\", width=4000, height=5000, res=300); par(mfrow=c(3,2)); Data2 <- Data1[,1:2]; head(Data1); head(Data2); Data3a <- Data1[,3:4]; head(Data3a); Data3b <- merge(Data2, Data3a, by.x='V1', by.y='V3'); colnames(Data3b) <- c(\"Pos\", \"HIV\", \"ExAC\"); head(Data3b); plot(Data3b[,2], Data3b[,3], xlab=\"HIV AF\", ylab=\"ExAC_NFE AF\", main=\"HIV vs. ExAC_NFE AF\"); abline(a=0,b=1); plot(-log10(Data3b[,2]), -log10(Data3b[,3]), xlab=\"HIV -log10(AF)\", ylab=\"ExAC_NFE -log10(AF)\", main=\"HIV vs. ExAC_NFE -log10(AF)\"); abline(a=0,b=1); hist(Data3b[,2], xlab=\"HIV AF\", main=\"HIV AF Histogram\", breaks=100); hist(-log10(Data3b[,2]), xlab=\"HIV -log10(AF)\", main=\"HIV -log10(AF) Histogram\", breaks=100); hist(Data3b[,3], xlab=\"ExAC_NFE AF\", main=\"ExAC_NFE AF Histogram\", breaks=100); hist(-log10(Data3b[,3]), xlab=\"ExAC_NFE -log10(AF)\", main=\"ExAC_NFE -log10(AF) Histogram\", breaks=100); dev.off()"

~~~
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs | grep -v NA | perl -lane '$F[1] = (split(/=/, $F[1]))[1]; $F[3] = (split(/=/, $F[3]))[1]; print join("\t", @F);' | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs.AFplots.jpeg\", width=4000, height=5000, res=300); par(mfrow=c(3,2)); Data2 <- Data1[,1:2]; head(Data1); head(Data2); Data3a <- Data1[,3:4]; head(Data3a); Data3b <- merge(Data2, Data3a, by.x='V1', by.y='V3'); colnames(Data3b) <- c(\"Pos\", \"HIV\", \"ExAC\"); head(Data3b); plot(Data3b[,2], Data3b[,3], xlab=\"HIV AF\", ylab=\"ExAC_NFE AF\", main=\"HIV vs. ExAC_NFE AF\"); abline(a=0,b=1); plot(-log10(Data3b[,2]), -log10(Data3b[,3]), xlab=\"HIV -log10(AF)\", ylab=\"ExAC_NFE -log10(AF)\", main=\"HIV vs. ExAC_NFE -log10(AF)\"); abline(a=0,b=1); hist(Data3b[,2], xlab=\"HIV AF\", main=\"HIV AF Histogram\", breaks=100); hist(-log10(Data3b[,2]), xlab=\"HIV -log10(AF)\", main=\"HIV -log10(AF) Histogram\", breaks=100); hist(Data3b[,3], xlab=\"ExAC_NFE AF\", main=\"ExAC_NFE AF Histogram\", breaks=100); hist(-log10(Data3b[,3]), xlab=\"ExAC_NFE -log10(AF)\", main=\"ExAC_NFE -log10(AF) Histogram\", breaks=100); dev.off()"
> Data1 <- read.table(file('stdin'), header=F); jpeg("/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs.AFplots.jpeg", width=4000, height=5000, res=300); par(mfrow=c(3,2)); Data2 <- Data1[,1:2]; head(Data1); head(Data2); Data3a <- Data1[,3:4]; head(Data3a); Data3b <- merge(Data2, Data3a, by.x='V1', by.y='V3'); colnames(Data3b) <- c("Pos", "HIV", "ExAC"); head(Data3b); plot(Data3b[,2], Data3b[,3], xlab="HIV AF", ylab="ExAC_NFE AF", main="HIV vs. ExAC_NFE AF"); abline(a=0,b=1); plot(-log10(Data3b[,2]), -log10(Data3b[,3]), xlab="HIV -log10(AF)", ylab="ExAC_NFE -log10(AF)", main="HIV vs. ExAC_NFE -log10(AF)"); abline(a=0,b=1); hist(Data3b[,2], xlab="HIV AF", main="HIV AF Histogram", breaks=100); hist(-log10(Data3b[,2]), xlab="HIV -log10(AF)", main="HIV -log10(AF) Histogram", breaks=100); hist(Data3b[,3], xlab="ExAC_NFE AF", main="ExAC_NFE AF Histogram", breaks=100); hist(-log10(Data3b[,3]), xlab="ExAC_NFE -log10(AF)", main="ExAC_NFE -log10(AF) Histogram", breaks=100); dev.off()
V1       V2          V3           V4
1 11_62374525 0.002762 11_62374525 1.139977e-03                                                                                                                                                                                                                                                                            
2  7_91630628 0.001101 17_80015878 2.675657e-05
3 17_80015878 0.001669 2_109380157 4.156164e-03                                                                                                                                                                                                                                                                            
4 2_109380157 0.002747  19_2810714 1.575998e-03
5  19_2810714 0.002775 2_109380406 1.079674e-02                                                                                                                                                                                                                                                                            
6 2_109380406 0.007692  3_45637230 1.387802e-02
	V1       V2                                                                                                                                                                                                                                                                                                     
1 11_62374525 0.002762
2  7_91630628 0.001101                                                                                                                                                                                                                                                                                                     
3 17_80015878 0.001669
4 2_109380157 0.002747                                                                                                                                                                                                                                                                                                     
5  19_2810714 0.002775
6 2_109380406 0.007692                                                                                                                                                                                                                                                                                                     
	V3           V4
1 11_62374525 1.139977e-03                                                                                                                                                                                                                                                                                                 
2 17_80015878 2.675657e-05
3 2_109380157 4.156164e-03                                                                                                                                                                                                                                                                                                 
4  19_2810714 1.575998e-03
5 2_109380406 1.079674e-02                                                                                                                                                                                                                                                                                                 
6  3_45637230 1.387802e-02
	Pos       HIV         ExAC                                                                                                                                                                                                                                                                                      
1 10_103239146 0.0005525 0.0007347649
2 10_103281634 0.2040000 0.1815701542                                                                                                                                                                                                                                                                                      
3 10_103281670 0.0011060 0.0006755616
4 10_103281676 0.2130000 0.1924391361                                                                                                                                                                                                                                                                                      
5 10_103285900 0.0680000 0.0694523545
6 10_103296278 0.0093920 0.0095648518                                                                                                                                                                                                                                                                                      
null device
	1                                                                                                                                                                                                                                                                                                                
>
>                                                                                                                                                                                 
~~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs.AFplots.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/.

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs6.txt.gz | grep exonic | perl -F, -lane 'print $F[0], "\t", (split(/_/, $F[1]))[0];' | sort | uniq > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs6.Exonic.justChrBP

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --positions /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs6.Exonic.justChrBP --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.Exonic 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.Exonic.recode.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.Exonic.vcf

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.Exonic.vcf

python /home/michaelt/Scripts/Python/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.Exonic.vcf.gz --file2 /home/michaelt/Data/ExAC/ExAC.r0.3.sites.vep.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.gz | grep VCF1 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.gz | grep VCF3a | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.gz | grep VCF3b | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3b.gz 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.gz | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a.gz | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3b.gz | wc

[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.gz | wc                                    
1255  985175 23530790
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a.gz | wc
5763 4523955 109173601
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3b.gz | wc                                   
5763   51867 17654939

[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.gz | grep rs | wc
    153  120105 2825181

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.gz | perl -lane 'my @vals1 = split(/\;/, $F[8]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { print ((split(/=/, $entry1))[1]); } }' | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.AFplots.jpeg\", width=4000, height=1750, res=300); par(mfrow=c(1,2)); hist(Data1[,1], xlab=\"HIV AF\", main=\"HIV Exonic AF non-ExAC Overlap\", breaks=50); hist(-log10(Data1[,1]), xlab=\"HIV -log10(AF)\", main=\"HIV Exonic -log10(AF) non-ExAC Overlap\", breaks=50); dev.off();"
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.gz | perl -lane 'my @vals1 = split(/\;/, $F[8]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AC=/) { print ((split(/=/, $entry1))[1]); } }' | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.ACplots.jpeg\", width=4000, height=1750, res=300); par(mfrow=c(1,2)); hist(Data1[,1], xlab=\"HIV AC\", main=\"HIV Exonic AC non-ExAC Overlap\", breaks=50); hist(-log10(Data1[,1]), xlab=\"HIV -log10(AC)\", main=\"HIV Exonic -log10(AC) non-ExAC Overlap\", breaks=50); dev.off();"

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.AFplots.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.ACplots.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/.

paste <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a.gz | perl -ane 'print $F[1], "_", $F[2], "\t"; my @vals1 = split(/\;/, $F[8]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { print $entry1; } } print "\n";') <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3b.gz | perl -ane 'print $F[1], "_", $F[2], "\t"; my @vals1 = split(/\;/, $F[8]); my $AC_NFE; my $AN_NFE; foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AC_NFE=/) { $AC_NFE = (split(/,/, (split(/=/, $entry1))[1]))[0]; } if ($entry1 =~ m/^AN_NFE=/) { $AN_NFE = (split(/=/, $entry1))[1]; } } if (($AC_NFE) && ($AN_NFE)) { print "AF_NFE=", $AC_NFE/$AN_NFE; } else { print "NA"} print "\n";') > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs

##cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs | grep -v NA | perl -lane '$F[1] = (split(/=/, $F[1]))[1]; $F[3] = (split(/=/, $F[3]))[1]; print join("\t", @F);' | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs.AFplots.jpeg\", width=4000, height=5000, res=300); par(mfrow=c(3,2)); Data2 <- Data1[,1:2]; head(Data1); head(Data2); Data3a <- Data1[,3:4]; head(Data3a); Data3b <- merge(Data2, Data3a, by.x='V1', by.y='V3'); colnames(Data3b) <- c(\"Pos\", \"HIV\", \"ExAC\"); head(Data3b); plot(Data3b[,2], Data3b[,3], xlab=\"HIV AF\", ylab=\"ExAC_NFE AF\", main=\"HIV vs. ExAC_NFE AF\"); abline(a=0,b=1); plot(-log10(Data3b[,2]), -log10(Data3b[,3]), xlab=\"HIV -log10(AF)\", ylab=\"ExAC_NFE -log10(AF)\", main=\"HIV vs. ExAC_NFE -log10(AF)\"); abline(a=0,b=1); hist(Data3b[,2], xlab=\"HIV AF\", main=\"HIV AF Histogram\", breaks=100); hist(-log10(Data3b[,2]), xlab=\"HIV -log10(AF)\", main=\"HIV -log10(AF) Histogram\", breaks=100); hist(Data3b[,3], xlab=\"ExAC_NFE AF\", main=\"ExAC_NFE AF Histogram\", breaks=100); hist(-log10(Data3b[,3]), xlab=\"ExAC_NFE -log10(AF)\", main=\"ExAC_NFE -log10(AF) Histogram\", breaks=100); dev.off()"

~~~
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs | grep -v NA | perl -lane '$F[1] = (split(/=/, $F[1]))[1]; $F[3] = (split(/=/, $F[3]))[1]; print join("\t", @F);' | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs.AFplots.jpeg\", width=4000, height=5000, res=300); par(mfrow=c(3,2)); Data2 <- Data1[,1:2]; head(Data1); head(Data2); Data3a <- Data1[,3:4]; head(Data3a); Data3b <- merge(Data2, Data3a, by.x='V1', by.y='V3'); colnames(Data3b) <- c(\"Pos\", \"HIV\", \"ExAC\"); head(Data3b); plot(Data3b[,2], Data3b[,3], xlab=\"HIV AF\", ylab=\"ExAC_NFE AF\", main=\"HIV vs. ExAC_NFE AF\"); abline(a=0,b=1); plot(-log10(Data3b[,2]), -log10(Data3b[,3]), xlab=\"HIV -log10(AF)\", ylab=\"ExAC_NFE -log10(AF)\", main=\"HIV vs. ExAC_NFE -log10(AF)\"); abline(a=0,b=1); hist(Data3b[,2], xlab=\"HIV AF\", main=\"HIV AF Histogram\", breaks=100); hist(-log10(Data3b[,2]), xlab=\"HIV -log10(AF)\", main=\"HIV -log10(AF) Histogram\", breaks=100); hist(Data3b[,3], xlab=\"ExAC_NFE AF\", main=\"ExAC_NFE AF Histogram\", breaks=100); hist(-log10(Data3b[,3]), xlab=\"ExAC_NFE -log10(AF)\", main=\"ExAC_NFE -log10(AF) Histogram\", breaks=100); dev.off()"
> Data1 <- read.table(file('stdin'), header=F); jpeg("/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs.AFplots.jpeg", width=4000, height=5000, res=300); par(mfrow=c(3,2)); Data2 <- Data1[,1:2]; head(Data1); head(Data2); Data3a <- Data1[,3:4]; head(Data3a); Data3b <- merge(Data2, Data3a, by.x='V1', by.y='V3'); colnames(Data3b) <- c("Pos", "HIV", "ExAC"); head(Data3b); plot(Data3b[,2], Data3b[,3], xlab="HIV AF", ylab="ExAC_NFE AF", main="HIV vs. ExAC_NFE AF"); abline(a=0,b=1); plot(-log10(Data3b[,2]), -log10(Data3b[,3]), xlab="HIV -log10(AF)", ylab="ExAC_NFE -log10(AF)", main="HIV vs. ExAC_NFE -log10(AF)"); abline(a=0,b=1); hist(Data3b[,2], xlab="HIV AF", main="HIV AF Histogram", breaks=100); hist(-log10(Data3b[,2]), xlab="HIV -log10(AF)", main="HIV -log10(AF) Histogram", breaks=100); hist(Data3b[,3], xlab="ExAC_NFE AF", main="ExAC_NFE AF Histogram", breaks=100); hist(-log10(Data3b[,3]), xlab="ExAC_NFE -log10(AF)", main="ExAC_NFE -log10(AF) Histogram", breaks=100); dev.off()
	V1       V2          V3           V4                                                                                                                                                                                                                                                                            
1 11_62374525 0.002762 11_62374525 1.139977e-03
2 2_109380157 0.002747 2_109380157 4.156164e-03                                                                                                                                                                                                                                                                            
3  19_2810714 0.002775  19_2810714 1.575998e-03
4 2_109380406 0.007692 2_109380406 1.079674e-02                                                                                                                                                                                                                                                                            
5  3_45637230 0.008251  3_45637230 1.387802e-02
6 9_127693630 0.002208 9_127693630 1.498711e-05                                                                                                                                                                                                                                                                            
	V1       V2
1 11_62374525 0.002762                                                                                                                                                                                                                                                                                                     
2 2_109380157 0.002747
3  19_2810714 0.002775                                                                                                                                                                                                                                                                                                     
4 2_109380406 0.007692
5  3_45637230 0.008251                                                                                                                                                                                                                                                                                                     
6 9_127693630 0.002208
	V3           V4                                                                                                                                                                                                                                                                                                 
1 11_62374525 1.139977e-03
2 2_109380157 4.156164e-03                                                                                                                                                                                                                                                                                                 
3  19_2810714 1.575998e-03
4 2_109380406 1.079674e-02                                                                                                                                                                                                                                                                                                 
5  3_45637230 1.387802e-02
6 9_127693630 1.498711e-05                                                                                                                                                                                                                                                                                                 
	Pos       HIV         ExAC
1 10_103239146 0.0005525 7.347649e-04                                                                                                                                                                                                                                                                                      
2 10_103285900 0.0680000 6.945235e-02
3 10_103310604 0.0033150 5.999340e-05                                                                                                                                                                                                                                                                                      
4 10_103542667 0.0005525 1.534731e-05
5 10_103543127 0.0022100 2.845238e-03                                                                                                                                                                                                                                                                                      
6 10_103543141 0.0016570 1.308441e-03
null device                                                                                                                                                                                                                                                                                                                
	1
>                                                                                                                                                                                                                                                                                                                          
>
~~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs.AFplots.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/. 

#20151231 NOTE -- Noticed that in the .justAFs files the SNPs were not automatically aligned with one another row-by-row. Occasionally SNPs would be off by a row or so. This looks to be because I just pasted together VCF3a and VCF3b assuming the two would match, but I should have used join (I don't think I knew about join at the time I wrote this exact code). However, doing join produces the same number of lines as the original justAFs file, so at least all SNPs are present in both columns it appears (if a bit out of order)? See below code and the redo of the AF vs. AF plots
#20151231 NOTE -- Apparently I did a 'merge' of sorts in R that was supposed to align the SNPs together by ChrBP. Still, some of the plot points showed differences in AF that the join --> AF1 - AF2 did not show. So redoing the plots with the justAF based on the linux join command and seeing if it looks different or not.
#20151231 NOTE -- The two plots don't look different actually, at least for the ExAC setup. 1000G might be different?

#join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) |

join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) | grep -v NA | perl -lane 'my $AF1 = (split(/\=/, $F[1]))[1]; my $AF2 = (split(/\=/, $F[2]))[1]; print $AF1, "\t", $AF2, "\t", $AF1 - $AF2;' | sort -rg -k 3,3 | vi -

#use table code on VCF1 AFs to get bin #s?
# table(cut(Data1[,2], seq(0,200,10), right=FALSE));"

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF1.gz | perl -lane 'splice(@F, 0, 1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 - | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a.gz | perl -lane 'splice(@F, 0, 1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 - | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz

#python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz 
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.gz | perl -lane 'splice(@F, 0, 1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 - | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz 

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a.gz | perl -lane 'splice(@F, 0, 1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 - | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz

join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) | grep -v NA | perl -lane 'my $AF1 = (split(/\=/, $F[1]))[1]; my $AF2 = (split(/\=/, $F[2]))[1]; print $AF1, "\t", $AF2, "\t", $AF1 - $AF2;' | grep -v NA | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs.AFplots.joinVers.jpeg\", width=4000, height=5000, res=300); par(mfrow=c(3,2)); plot(Data1[,1], Data1[,2], xlab=\"HIV AF\", ylab=\"ExAC_NFE AF\", main=\"HIV vs. ExAC_NFE AF\"); abline(a=0,b=1); plot(-log10(Data1[,1]), -log10(Data1[,2]), xlab=\"HIV -log10(AF)\", ylab=\"ExAC_NFE -log10(AF)\", main=\"HIV vs. ExAC_NFE -log10(AF)\"); abline(a=0,b=1); hist(Data1[,1], xlab=\"HIV AF\", main=\"HIV AF Histogram\", breaks=100); hist(-log10(Data1[,1]), xlab=\"HIV -log10(AF)\", main=\"HIV -log10(AF) Histogram\", breaks=100); hist(Data1[,2], xlab=\"ExAC_NFE AF\", main=\"ExAC_NFE AF Histogram\", breaks=100); hist(-log10(Data1[,2]), xlab=\"ExAC_NFE -log10(AF)\", main=\"ExAC_NFE -log10(AF) Histogram\", breaks=100); dev.off()"

join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) | grep -v NA | perl -lane 'my $AF1 = (split(/\=/, $F[1]))[1]; my $AF2 = (split(/\=/, $F[2]))[1]; print $AF1, "\t", $AF2, "\t", $AF1 - $AF2;' | grep -v NA | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs.AFplots.joinVers.jpeg\", width=4000, height=5000, res=300); par(mfrow=c(3,2)); plot(Data1[,1], Data1[,2], xlab=\"HIV AF\", ylab=\"ExAC_NFE AF\", main=\"HIV vs. ExAC_NFE AF\"); abline(a=0,b=1); plot(-log10(Data1[,1]), -log10(Data1[,2]), xlab=\"HIV -log10(AF)\", ylab=\"ExAC_NFE -log10(AF)\", main=\"HIV vs. ExAC_NFE -log10(AF)\"); abline(a=0,b=1); hist(Data1[,1], xlab=\"HIV AF\", main=\"HIV AF Histogram\", breaks=100); hist(-log10(Data1[,1]), xlab=\"HIV -log10(AF)\", main=\"HIV -log10(AF) Histogram\", breaks=100); hist(Data1[,2], xlab=\"ExAC_NFE AF\", main=\"ExAC_NFE AF Histogram\", breaks=100); hist(-log10(Data1[,2]), xlab=\"ExAC_NFE -log10(AF)\", main=\"ExAC_NFE -log10(AF) Histogram\", breaks=100); dev.off()"

~~~
[  michaelt@node1  ~]$python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz
NumVar: 52524 Exonic: 1006 Intronic: 39760 Intergenic: 231 UTR5: 835 UTR3: 5440
Synonymous: 315 Nonsynonymous: 648 Stopgain: 19 Stoploss: 2
PP2_D: 232 PP2_P: 111 PP2_B: 203 PP2_-9: 61 Sift_D: 280 Sift_T: 278 Sift_-9: 49 LRT_D: 376 LRT_N: 185 LRT_U: 23 LRT_-9: 23
Common: 7457 Rare: 45067 Singletons: 8981 Doubletons: 6823
MeanQuality: 21655.7797491 QualityStd: 78667.9078092 MeanDepth: 15.765874244 DepthStd: 8.14652964444 TiTvRatio: 2.90164908632 Transitions: 39062 Transversions: 13462
MeanHapScore: 2.2505812619 HapScoreStd: 1.54003401477 HapScore0Quart: 0.1447 HapScore25Quart: 1.1516 HapScore50Quart: 1.8319 HapScore75Quart: 2.916325 HapScore100Quart: 14.7663
[  michaelt@node1  ~]$python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz
NumVar: 11339 Exonic: 5712 Intronic: 4791 Intergenic: 34 UTR5: 215 UTR3: 360
Synonymous: 2627 Nonsynonymous: 2961 Stopgain: 34 Stoploss: 9
PP2_D: 776 PP2_P: 482 PP2_B: 1158 PP2_-9: 301 Sift_D: 1071 Sift_T: 1427 Sift_-9: 219 LRT_D: 1262 LRT_N: 1217 LRT_U: 119 LRT_-9: 119
Common: 1582 Rare: 9757 Singletons: 2372 Doubletons: 1847
MeanQuality: 29303.1804392 QualityStd: 105470.136094 MeanDepth: 21.5523891017 DepthStd: 8.55594910131 TiTvRatio: 3.3796832754 Transitions: 8750 Transversions: 2589
MeanHapScore: 2.79193538231 HapScoreStd: 1.69309941492 HapScore0Quart: 0.2255 HapScore25Quart: 1.6029 HapScore50Quart: 2.3858 HapScore75Quart: 3.5246 HapScore100Quart: 14.6402
[  michaelt@node1  ~]$python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz
NumVar: 1255 Exonic: 1006 Intronic: 0 Intergenic: 0 UTR5: 0 UTR3: 0
Synonymous: 315 Nonsynonymous: 648 Stopgain: 19 Stoploss: 2
PP2_D: 232 PP2_P: 111 PP2_B: 203 PP2_-9: 61 Sift_D: 280 Sift_T: 278 Sift_-9: 49 LRT_D: 376 LRT_N: 185 LRT_U: 23 LRT_-9: 23
Common: 42 Rare: 1213 Singletons: 594 Doubletons: 256
MeanQuality: 3440.1972988 QualityStd: 27278.6441189 MeanDepth: 21.188898926 DepthStd: 9.50183827057 TiTvRatio: 2.52528089888 Transitions: 899 Transversions: 356
MeanHapScore: 2.55335697211 HapScoreStd: 1.58802319993 HapScore0Quart: 0.2267 HapScore25Quart: 1.3872 HapScore50Quart: 2.1967 HapScore75Quart: 3.3025 HapScore100Quart: 12.3627
[  michaelt@node1  ~]$python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz
NumVar: 5763 Exonic: 5712 Intronic: 0 Intergenic: 0 UTR5: 0 UTR3: 0
Synonymous: 2627 Nonsynonymous: 2961 Stopgain: 34 Stoploss: 9
PP2_D: 776 PP2_P: 482 PP2_B: 1158 PP2_-9: 297 Sift_D: 1068 Sift_T: 1427 Sift_-9: 218 LRT_D: 1262 LRT_N: 1217 LRT_U: 119 LRT_-9: 115
Common: 628 Rare: 5135 Singletons: 1401 Doubletons: 1039
MeanQuality: 23618.0054156 QualityStd: 93507.4899224 MeanDepth: 22.830176612 DepthStd: 8.88336047996 TiTvRatio: 4.08201058201 Transitions: 4629 Transversions: 1134
MeanHapScore: 2.72138254381 HapScoreStd: 1.67640875918 HapScore0Quart: 0.2255 HapScore25Quart: 1.5705 HapScore50Quart: 2.3312 HapScore75Quart: 3.4077 HapScore100Quart: 14.6402
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ if ($8 >= .05) { print $0 } } ' | perl -lane 'my @vals1 = split(/,/, $F[8]); print $vals1[0];' | sort | uniq -c
    736 UTR3
     92 UTR5
    332 downstream
     36 intergenic
   5797 intronic
     15 ncRNA_UTR3
     42 ncRNA_exonic
    207 ncRNA_intronic
    177 upstream
     23 upstream;downstream
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ if ($8 < .05) { print $0 } } ' | perl -lane 'my @vals1 = split(/,/, $F[8]); print $vals1[0];' | sort | uniq -c
   4704 UTR3
    743 UTR5
   1831 downstream
   1006 exonic
    195 intergenic
  33963 intronic
    151 ncRNA_UTR3
     17 ncRNA_UTR5
    207 ncRNA_exonic
   1154 ncRNA_intronic
      1 ncRNA_splicing
     14 splicing
    904 upstream
    177 upstream;downstream
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ if ($8 >= .05) { print $0 } } ' | perl -lane 'my @vals1 = split(/,/, $F[8]); print $vals1[0];' | sort | uniq -c
     59 UTR3
     29 UTR5
    621 exonic
      1 exonic;splicing
      7 intergenic
    833 intronic
      1 ncRNA_UTR3
      6 ncRNA_exonic
     21 ncRNA_intronic
      3 upstream
      1 upstream;downstream
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ if ($8 < .05) { print $0 } } ' | perl -lane 'my @vals1 = split(/,/, $F[8]); print $vals1[0];' | sort | uniq -c
    301 UTR3
    186 UTR5
      8 downstream
   5091 exonic
      1 exonic;splicing
     27 intergenic
   3958 intronic
      3 ncRNA_UTR3
      2 ncRNA_UTR5
     43 ncRNA_exonic
     89 ncRNA_intronic
     16 splicing
     29 upstream
      3 upstream;downstream
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ if ($8 >= .05) { print $0 } } ' | perl -lane 'my @vals1 = split(/,/, $F[8]); print $vals1[0];' | sort | uniq -c
     42 ncRNA_exonic
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ if ($8 < .05) { print $0 } } ' | perl -lane 'my @vals1 = split(/,/, $F[8]); print $vals1[0];' | sort | uniq -c
   1006 exonic
    207 ncRNA_exonic
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ if ($8 >= .05) { print $0 } } ' | perl -lane 'my @vals1 = split(/,/, $F[8]); print $vals1[0];' | sort | uniq -c
    621 exonic
      1 exonic;splicing
      6 ncRNA_exonic
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ if ($8 < .05) { print $0 } } ' | perl -lane 'my @vals1 = split(/,/, $F[8]); print $vals1[0];' | sort | uniq -c
   5091 exonic
      1 exonic;splicing
     43 ncRNA_exonic
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ print $8 } ' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.05), right=FALSE));"
> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.05), right=FALSE));

  [0,0.05) [0.05,0.1) [0.1,0.15) [0.15,0.2) [0.2,0.25) [0.25,0.3) [0.3,0.35)
     45067       1401       1004        701        528        486        424
[0.35,0.4) [0.4,0.45) [0.45,0.5) [0.5,0.55) [0.55,0.6) [0.6,0.65) [0.65,0.7)
       286        347        242        203        333        227        239
[0.7,0.75) [0.75,0.8) [0.8,0.85) [0.85,0.9) [0.9,0.95)   [0.95,1)
       153        183        124        126        119        328
>
>
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ print $8 } ' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.05), right=FALSE));"
> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.05), right=FALSE));

  [0,0.05) [0.05,0.1) [0.1,0.15) [0.15,0.2) [0.2,0.25) [0.25,0.3) [0.3,0.35)
      9757        311        204        189        108         94         83
[0.35,0.4) [0.4,0.45) [0.45,0.5) [0.5,0.55) [0.55,0.6) [0.6,0.65) [0.65,0.7)
        66         81         44         31         58         49         51
[0.7,0.75) [0.75,0.8) [0.8,0.85) [0.85,0.9) [0.9,0.95)   [0.95,1)
        31         42         30         16         18         76
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ print $8 } ' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.05), right=FALSE));"
> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.05), right=FALSE));

  [0,0.05) [0.05,0.1) [0.1,0.15) [0.15,0.2) [0.2,0.25) [0.25,0.3) [0.3,0.35)
      1213          7         22          0          1          4          1
[0.35,0.4) [0.4,0.45) [0.45,0.5) [0.5,0.55) [0.55,0.6) [0.6,0.65) [0.65,0.7)
         1          2          0          0          0          1          1
[0.7,0.75) [0.75,0.8) [0.8,0.85) [0.85,0.9) [0.9,0.95)   [0.95,1)
         0          0          2          0          0          0
>
>
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ print $8 } ' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.05), right=FALSE));"
> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.05), right=FALSE));

  [0,0.05) [0.05,0.1) [0.1,0.15) [0.15,0.2) [0.2,0.25) [0.25,0.3) [0.3,0.35)
      5135        132         80         74         38         35         34
[0.35,0.4) [0.4,0.45) [0.45,0.5) [0.5,0.55) [0.55,0.6) [0.6,0.65) [0.65,0.7)
        32         25         19         12         25         22         29
[0.7,0.75) [0.75,0.8) [0.8,0.85) [0.85,0.9) [0.9,0.95)   [0.95,1)
        10         14          9          6          6         26
[  michaelt@node1  ~]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) | grep -v NA | perl -lane 'my $AF1 = (split(/\=/, $F[1]))[1]; my $AF2 = (split(/\=/, $F[2]))[1]; print $AF1, "\t", $AF2, "\t", $AF1 - $AF2;' | grep -v NA | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs.AFplots.joinVers.jpeg\", width=4000, height=5000, res=300); par(mfrow=c(3,2)); plot(Data1[,1], Data1[,2], xlab=\"HIV AF\", ylab=\"ExAC_NFE AF\", main=\"HIV vs. ExAC_NFE AF\"); abline(a=0,b=1); plot(-log10(Data1[,1]), -log10(Data1[,2]), xlab=\"HIV -log10(AF)\", ylab=\"ExAC_NFE -log10(AF)\", main=\"HIV vs. ExAC_NFE -log10(AF)\"); abline(a=0,b=1); hist(Data1[,1], xlab=\"HIV AF\", main=\"HIV AF Histogram\", breaks=100); hist(-log10(Data1[,1]), xlab=\"HIV -log10(AF)\", main=\"HIV -log10(AF) Histogram\", breaks=100); hist(Data1[,2], xlab=\"ExAC_NFE AF\", main=\"ExAC_NFE AF Histogram\", breaks=100); hist(-log10(Data1[,2]), xlab=\"ExAC_NFE -log10(AF)\", main=\"ExAC_NFE -log10(AF) Histogram\", breaks=100); dev.off()"
> Data1 <- read.table(file('stdin'), header=F); jpeg("/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs.AFplots.joinVers.jpeg", width=4000, height=5000, res=300); par(mfrow=c(3,2)); plot(Data1[,1], Data1[,2], xlab="HIV AF", ylab="ExAC_NFE AF", main="HIV vs. ExAC_NFE AF"); abline(a=0,b=1); plot(-log10(Data1[,1]), -log10(Data1[,2]), xlab="HIV -log10(AF)", ylab="ExAC_NFE -log10(AF)", main="HIV vs. ExAC_NFE -log10(AF)"); abline(a=0,b=1); hist(Data1[,1], xlab="HIV AF", main="HIV AF Histogram", breaks=100); hist(-log10(Data1[,1]), xlab="HIV -log10(AF)", main="HIV -log10(AF) Histogram", breaks=100); hist(Data1[,2], xlab="ExAC_NFE AF", main="ExAC_NFE AF Histogram", breaks=100); hist(-log10(Data1[,2]), xlab="ExAC_NFE -log10(AF)", main="ExAC_NFE -log10(AF) Histogram", breaks=100); dev.off()
null device
          1
>
>
[  michaelt@node1  ~]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) | grep -v NA | perl -lane 'my $AF1 = (split(/\=/, $F[1]))[1]; my $AF2 = (split(/\=/, $F[2]))[1]; print $AF1, "\t", $AF2, "\t", $AF1 - $AF2;' | grep -v NA | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs.AFplots.joinVers.jpeg\", width=4000, height=5000, res=300); par(mfrow=c(3,2)); plot(Data1[,1], Data1[,2], xlab=\"HIV AF\", ylab=\"ExAC_NFE AF\", main=\"HIV vs. ExAC_NFE AF\"); abline(a=0,b=1); plot(-log10(Data1[,1]), -log10(Data1[,2]), xlab=\"HIV -log10(AF)\", ylab=\"ExAC_NFE -log10(AF)\", main=\"HIV vs. ExAC_NFE -log10(AF)\"); abline(a=0,b=1); hist(Data1[,1], xlab=\"HIV AF\", main=\"HIV AF Histogram\", breaks=100); hist(-log10(Data1[,1]), xlab=\"HIV -log10(AF)\", main=\"HIV -log10(AF) Histogram\", breaks=100); hist(Data1[,2], xlab=\"ExAC_NFE AF\", main=\"ExAC_NFE AF Histogram\", breaks=100); hist(-log10(Data1[,2]), xlab=\"ExAC_NFE -log10(AF)\", main=\"ExAC_NFE -log10(AF) Histogram\", breaks=100); dev.off()"
> Data1 <- read.table(file('stdin'), header=F); jpeg("/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs.AFplots.joinVers.jpeg", width=4000, height=5000, res=300); par(mfrow=c(3,2)); plot(Data1[,1], Data1[,2], xlab="HIV AF", ylab="ExAC_NFE AF", main="HIV vs. ExAC_NFE AF"); abline(a=0,b=1); plot(-log10(Data1[,1]), -log10(Data1[,2]), xlab="HIV -log10(AF)", ylab="ExAC_NFE -log10(AF)", main="HIV vs. ExAC_NFE -log10(AF)"); abline(a=0,b=1); hist(Data1[,1], xlab="HIV AF", main="HIV AF Histogram", breaks=100); hist(-log10(Data1[,1]), xlab="HIV -log10(AF)", main="HIV -log10(AF) Histogram", breaks=100); hist(Data1[,2], xlab="ExAC_NFE AF", main="ExAC_NFE AF Histogram", breaks=100); hist(-log10(Data1[,2]), xlab="ExAC_NFE -log10(AF)", main="ExAC_NFE -log10(AF) Histogram", breaks=100); dev.off()
null device
          1
>
>
~~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs.AFplots.joinVers.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs.AFplots.joinVers.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/.

#From Windows netbook via cmder
scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/*AFplots* .
#(C:\Users\PortMCT2\Desktop\Lab_Stuff\StephensLab\NUWrk\HIVStff\PG\ALL_MAPPING\Pools\MM5\PostMerge\mapping_pool_merged\GATK\Analyses\VCFComparisons)








##20150315
##Downloading and comparing overlap between 1000G data and my own for QC sake

mkdir /home/michaelt/Data/1000G/VCFs 

cd /home/michaelt/Data/1000G/VCFs

wget ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502/*

vcf-concat /home/michaelt/Data/1000G/VCFs/ALL.chr1.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz \
/home/michaelt/Data/1000G/VCFs/ALL.chr2.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz \
/home/michaelt/Data/1000G/VCFs/ALL.chr3.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz \
/home/michaelt/Data/1000G/VCFs/ALL.chr4.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz \
/home/michaelt/Data/1000G/VCFs/ALL.chr5.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz \
/home/michaelt/Data/1000G/VCFs/ALL.chr6.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz \
/home/michaelt/Data/1000G/VCFs/ALL.chr7.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz \
/home/michaelt/Data/1000G/VCFs/ALL.chr8.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz \
/home/michaelt/Data/1000G/VCFs/ALL.chr9.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz \
/home/michaelt/Data/1000G/VCFs/ALL.chr10.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz \
/home/michaelt/Data/1000G/VCFs/ALL.chr11.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz \
/home/michaelt/Data/1000G/VCFs/ALL.chr12.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz \
/home/michaelt/Data/1000G/VCFs/ALL.chr13.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz \
/home/michaelt/Data/1000G/VCFs/ALL.chr14.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz \
/home/michaelt/Data/1000G/VCFs/ALL.chr15.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz \
/home/michaelt/Data/1000G/VCFs/ALL.chr16.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz \
/home/michaelt/Data/1000G/VCFs/ALL.chr17.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz \
/home/michaelt/Data/1000G/VCFs/ALL.chr18.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz \
/home/michaelt/Data/1000G/VCFs/ALL.chr19.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz \
/home/michaelt/Data/1000G/VCFs/ALL.chr20.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz \
/home/michaelt/Data/1000G/VCFs/ALL.chr21.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz \
/home/michaelt/Data/1000G/VCFs/ALL.chr22.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz \
/home/michaelt/Data/1000G/VCFs/ALL.chrX.phase3_shapeit2_mvncall_integrated_v1a.20130502.genotypes.vcf.gz | gzip > /home/michaelt/Data/1000G/VCFs/ALL.chrAll.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz
#vcf-concat /home/michaelt/Data/1000G/VCFs/ALL.chr1.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz /home/michaelt/Data/1000G/VCFs/ALL.chr2.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz /home/michaelt/Data/1000G/VCFs/ALL.chr3.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz /home/michaelt/Data/1000G/VCFs/ALL.chr4.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz /home/michaelt/Data/1000G/VCFs/ALL.chr5.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz /home/michaelt/Data/1000G/VCFs/ALL.chr6.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz /home/michaelt/Data/1000G/VCFs/ALL.chr7.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz /home/michaelt/Data/1000G/VCFs/ALL.chr8.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz /home/michaelt/Data/1000G/VCFs/ALL.chr9.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz /home/michaelt/Data/1000G/VCFs/ALL.chr10.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz /home/michaelt/Data/1000G/VCFs/ALL.chr11.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz /home/michaelt/Data/1000G/VCFs/ALL.chr12.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz /home/michaelt/Data/1000G/VCFs/ALL.chr13.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz /home/michaelt/Data/1000G/VCFs/ALL.chr14.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz /home/michaelt/Data/1000G/VCFs/ALL.chr15.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz /home/michaelt/Data/1000G/VCFs/ALL.chr16.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz /home/michaelt/Data/1000G/VCFs/ALL.chr17.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz /home/michaelt/Data/1000G/VCFs/ALL.chr18.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz /home/michaelt/Data/1000G/VCFs/ALL.chr19.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz /home/michaelt/Data/1000G/VCFs/ALL.chr20.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz /home/michaelt/Data/1000G/VCFs/ALL.chr21.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz /home/michaelt/Data/1000G/VCFs/ALL.chr22.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz /home/michaelt/Data/1000G/VCFs/ALL.chrX.phase3_shapeit2_mvncall_integrated_v1a.20130502.genotypes.vcf.gz | gzip > /home/michaelt/Data/1000G/VCFs/ALL.chrAll.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz

python /home/michaelt/Scripts/Python/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --file2 /home/michaelt/Data/1000G/VCFs/ALL.chrAll.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.gz | grep VCF1 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.gz | grep VCF3a | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.gz | grep VCF3b | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3b.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.gz | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a.gz | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3b.gz | wc

[  michaelt@node6  ~/Data/1000G/VCFs]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysisgrated_v5a_20130502.output.VCF1.gz | wcomp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integ
24814 19478990 458039224
[  michaelt@node6  ~/Data/1000G/VCFs]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysisgrated_v5a_20130502.output.VCF3a.gz | wcmp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_inte
39049 30653465 718374195
[  michaelt@node6  ~/Data/1000G/VCFs]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysisgrated_v5a_20130502.output.VCF3b.gz | wcmp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_inte
39049 98169186 397119503

paste <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a.gz | perl -ane 'print $F[1], "_", $F[2], "\t"; my @vals1 = split(/\;/, $F[8]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { print $entry1; } } print "\n";') <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3b.gz | perl -ane 'print $F[1], "_", $F[2], "\t"; my @vals1 = split(/\;/, $F[8]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^EUR_AF=/) { print $entry1; } } print "\n";') > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | grep -v NA | perl -lane '$F[1] = (split(/=/, $F[1]))[1]; $F[3] = (split(/,/, (split(/=/, $F[3]))[1]))[0]; print join("\t", @F);' | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs.AFplots.jpeg\", width=4000, height=5000, res=300); par(mfrow=c(3,2)); Data2 <- Data1[,1:2]; head(Data1); head(Data2); Data3a <- Data1[,3:4]; head(Data3a); Data3b <- merge(Data2, Data3a, by.x='V1', by.y='V3'); colnames(Data3b) <- c(\"Pos\", \"HIV\", \"1000G\"); head(Data3b); plot(Data3b[,2], Data3b[,3], xlab=\"HIV AF\", ylab=\"1000G_EUR AF\", main=\"HIV vs. 1000G_EUR AF\"); abline(a=0,b=1); plot(-log10(Data3b[,2]), -log10(Data3b[,3]), xlab=\"HIV -log10(AF)\", ylab=\"1000G_EUR -log10(AF)\", main=\"HIV vs. 1000G_EUR -log10(AF)\"); abline(a=0,b=1); hist(Data3b[,2], xlab=\"HIV AF\", main=\"HIV AF Histogram\", breaks=100); hist(-log10(Data3b[,2]), xlab=\"HIV -log10(AF)\", main=\"HIV -log10(AF) Histogram\", breaks=100); hist(Data3b[,3], xlab=\"1000G_EUR AF\", main=\"1000G_EUR AF Histogram\", breaks=100); hist(-log10(Data3b[,3]), xlab=\"1000G_EUR -log10(AF)\", main=\"1000G_EUR -log10(AF) Histogram\", breaks=100); dev.off()"

~~~
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample]$cat /data/userdata/pg/michaelt/Data/AL
L_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4
_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | grep -v NA | perl -lane '$F[1] = (split(/=/, $F[1]))[
1]; $F[3] = (split(/,/, (split(/=/, $F[3]))[1]))[0]; print join("\t", @F);' | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); j
peg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExA
CComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs.AFplots.jpeg\",
width=4000, height=5000, res=300); par(mfrow=c(3,2)); Data2 <- Data1[,1:2]; head(Data1); head(Data2); Data3a <- Data1[,3:4]; head(Data3a); Data3b <- merge(Data2, Data3
a, by.x='V1', by.y='V3'); colnames(Data3b) <- c(\"Pos\", \"HIV\", \"1000G\"); head(Data3b); plot(Data3b[,2], Data3b[,3], xlab=\"HIV AF\", ylab=\"1000G_EUR AF\", main=\"
HIV vs. 1000G_EUR AF\"); abline(a=0,b=1); plot(-log10(Data3b[,2]), -log10(Data3b[,3]), xlab=\"HIV -log10(AF)\", ylab=\"1000G_EUR -log10(AF)\", main=\"HIV vs. 1000G_EUR
-log10(AF)\"); abline(a=0,b=1); hist(Data3b[,2], xlab=\"HIV AF\", main=\"HIV AF Histogram\", breaks=100); hist(-log10(Data3b[,2]), xlab=\"HIV -log10(AF)\", main=\"HIV -log10(AF) Histogram\", breaks=100); hist(Data3b[,3], xlab=\"1000G_EUR AF\", main=\"1000G_EUR AF Histogram\", breaks=100); hist(-log10(Data3b[,3]), xlab=\"1000G_EUR -log10(AF)\", main=\"1000G_EUR -log10(AF) Histogram\", breaks=100); dev.off()"
> Data1 <- read.table(file('stdin'), header=F); jpeg("/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs.AFplots.jpeg", width=4000, height=5000, res=300); par(mfrow=c(3,2)); Data2 <- Data1[,1:2]; head(Data1); head(Data2); Data3a <- Data1[,3:4]; head(Data3a); Data3b <- merge(Data2, Data3a, by.x='V1', by.y='V3'); colnames(Data3b) <- c("Pos", "HIV", "1000G"); head(Data3b); plot(Data3b[,2], Data3b[,3], xlab="HIV AF", ylab="1000G_EUR AF", main="HIV vs. 1000G_EUR AF"); abline(a=0,b=1); plot(-log10(Data3b[,2]), -log10(Data3b[,3]), xlab="HIV -log10(AF)", ylab="1000G_EUR -log10(AF)", main="HIV vs. 1000G_EUR -log10(AF)"); abline(a=0,b=1); hist(Data3b[,2], xlab="HIV AF", main="HIV AF Histogram", breaks=100); hist(-log10(Data3b[,2]), xlab="HIV -log10(AF)", main="HIV -log10(AF) Histogram", breaks=100); hist(Data3b[,3], xlab="1000G_EUR AF", main="1000G_EUR AF Histogram", breaks=100); hist(-log10(Data3b[,3]), xlab="1000G_EUR -log10(AF)", main="1000G_EUR -log10(AF) Histogram", breaks=100); dev.off()
	V1       V2           V3     V4
1 10_114427718 0.005006 10_114427718 0.0099
2  1_201843076 0.001098  1_201843076 0.0010
3  2_109380406 0.007692  2_109380406 0.0060
4  17_73038000 0.055000  17_73038000 0.0318
5   3_45637230 0.008251   3_45637230 0.0149
6  9_127693630 0.002208  9_127693630 0.0000
	V1       V2
1 10_114427718 0.005006
2  1_201843076 0.001098
3  2_109380406 0.007692
4  17_73038000 0.055000
5   3_45637230 0.008251
6  9_127693630 0.002208
	V3     V4
1 10_114427718 0.0099
2  1_201843076 0.0010
3  2_109380406 0.0060
4  17_73038000 0.0318
5   3_45637230 0.0149
6  9_127693630 0.0000
	Pos      HIV  1000G
1 10_103190011 0.001670 0.0030
2 10_103190521 0.003367 0.0020
3 10_103190533 0.011000 0.0099
4 10_103221377 0.006857 0.0010
5 10_103221969 0.001117 0.0000
6 10_103238858 0.068000 0.0755
null device
	1
>
>
~~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs.AFplots.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/. 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.gz | perl -lane 'my @vals1 = split(/\;/, $F[8]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { print ((split(/=/, $entry1))[1]); } }' | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.AFplots.jpeg\", width=4000, height=1750, res=300); par(mfrow=c(1,2)); hist(Data1[,1], xlab=\"HIV AF\", main=\"HIV AF non-1000G Overlap\", breaks=50); hist(-log10(Data1[,1]), xlab=\"HIV -log10(AF)\", main=\"HIV -log10(AF) non-1000G Overlap\", breaks=50); dev.off();"
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.gz | perl -lane 'my @vals1 = split(/\;/, $F[8]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AC=/) { print ((split(/=/, $entry1))[1]); } }' | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.ACplots.jpeg\", width=4000, height=1750, res=300); par(mfrow=c(1,2)); hist(Data1[,1], xlab=\"HIV AC\", main=\"HIV AC non-1000G Overlap\", breaks=50); hist(-log10(Data1[,1]), xlab=\"HIV -log10(AC)\", main=\"HIV -log10(AC) non-1000G Overlap\", breaks=50); dev.off(); print(c(mean(Data1[,1]),sd(Data1[,1]),min(Data1[,1]),max(Data1[,1])));"

~~~
null device
	1
[1]    3.663779   31.769742    1.000000 1739.000000
>
>
~~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.AFplots.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.ACplots.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/.

#20151231 NOTE -- Continuing from the ExAC notes of this same time period...
#20151231 NOTE -- It would appear there are some SNPs that are encoded as tri-allelic in the 1000G vcf so, given the two AF possibilities present in the 1000G column of AFs, sometimes the 'wrong' AF was chosen (whichever one was 'first' was taken by default). This is why some SNPs appeared to have AF differences >.5 or something like that; just going to drop SNPs that have greater than two alleles in the 1000G data for the purposes of this analysis

#join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) |
join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) | grep -v NA | perl -lane 'my $AF1 = (split(/\=/, $F[1]))[1]; my $AF2 = (split(/\=/, $F[2]))[1]; print $AF1, "\t", $AF2, "\t", $AF1 - $AF2;' | sort -rg -k 3,3 | vi -

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.gz | perl -lane 'splice(@F, 0, 1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 - | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a.gz | perl -lane 'splice(@F, 0, 1); print join("\t", @F);' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 - | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz

join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) | grep -v NA | perl -lane 'my $AF1 = (split(/\=/, $F[1]))[1]; my $AF2 = (split(/\=/, $F[2]))[1]; print $AF1, "\t", $AF2, "\t", $AF1 - $AF2;' | grep -v \, | grep -v NA | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs.AFplots.joinVers.jpeg\", width=4000, height=5000, res=300); par(mfrow=c(3,2)); plot(Data1[,1], Data1[,2], xlab=\"HIV AF\", ylab=\"1000G_EUR AF\", main=\"HIV vs. 1000G_EUR AF\"); abline(a=0,b=1); plot(-log10(Data1[,1]), -log10(Data1[,2]), xlab=\"HIV -log10(AF)\", ylab=\"1000G_EUR -log10(AF)\", main=\"HIV vs. 1000G_EUR -log10(AF)\"); abline(a=0,b=1); hist(Data1[,1], xlab=\"HIV AF\", main=\"HIV AF Histogram\", breaks=100); hist(-log10(Data1[,1]), xlab=\"HIV -log10(AF)\", main=\"HIV -log10(AF) Histogram\", breaks=100); hist(Data1[,2], xlab=\"1000G_EUR AF\", main=\"1000G_EUR AF Histogram\", breaks=100); hist(-log10(Data1[,2]), xlab=\"1000G_EUR -log10(AF)\", main=\"1000G_EUR -log10(AF) Histogram\", breaks=100); dev.off()"


~~~
[  michaelt@node1  ~]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) | grep -v NA | perl -lane 'my $AF1 = (split(/\=/, $F[1]))[1]; my $AF2 = (split(/\=/, $F[2]))[1]; print $AF1, "\t", $AF2, "\t", $AF1 - $AF2;' | sort -rg -k 3,3 | head -n 10
0.994   0,0.998 0.994
0.970   0,0.9791        0.97
0.691   0,0.7087        0.691
0.622   0,0.661 0.622
0.324   0,0.2584        0.324
0.267   0,0.2505        0.267
0.263   0,0.2396        0.263
0.253   0,0.2624        0.253
0.723   0.4811  0.2419
0.999   0.7614  0.2376
[  michaelt@node1  ~]$python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz
NumVar: 24814 Exonic: 3226 Intronic: 16498 Intergenic: 103 UTR5: 470 UTR3: 2457
Synonymous: 1145 Nonsynonymous: 1985 Stopgain: 37 Stoploss: 4
PP2_D: 669 PP2_P: 341 PP2_B: 652 PP2_-9: 186 Sift_D: 809 Sift_T: 886 Sift_-9: 153 LRT_D: 1068 LRT_N: 639 LRT_U: 75 LRT_-9: 66
Common: 32 Rare: 24782 Singletons: 9172 Doubletons: 5593
MeanQuality: 405.516359313 QualityStd: 6406.99526978 MeanDepth: 17.4149027868 DepthStd: 8.60518372828 TiTvRatio: 2.62142440163 Transitions: 17962 Transversions: 6852
MeanHapScore: 2.31370625856 HapScoreStd: 1.4757546393 HapScore0Quart: 0.1694 HapScore25Quart: 1.2297 HapScore50Quart: 1.94465 HapScore75Quart: 3.020975 HapScore100Quart: 12.6246
[  michaelt@node1  ~]$python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz
NumVar: 39049 Exonic: 3492 Intronic: 28053 Intergenic: 162 UTR5: 580 UTR3: 3343
Synonymous: 1797 Nonsynonymous: 1624 Stopgain: 16 Stoploss: 7
PP2_D: 339 PP2_P: 252 PP2_B: 709 PP2_-9: 176 Sift_D: 542 Sift_T: 819 Sift_-9: 115 LRT_D: 570 LRT_N: 763 LRT_U: 67 LRT_-9: 76
Common: 9007 Rare: 30042 Singletons: 2181 Doubletons: 3077
MeanQuality: 37380.0726165 QualityStd: 104933.583337 MeanDepth: 16.3982668202 DepthStd: 8.4299657817 TiTvRatio: 3.24491792586 Transitions: 29850 Transversions: 9199
MeanHapScore: 2.36766571231 HapScoreStd: 1.64543362889 HapScore0Quart: 0.1447 HapScore25Quart: 1.2104 HapScore50Quart: 1.925 HapScore75Quart: 3.046 HapScore100Quart: 14.7663
[  michaelt@node1  ~]$python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz^C                          [  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ if ($8 < .05) { print $0 } } ' | perl -lane 'my @vals1 = split(/,/, $F[8]); print $vals1[0];' | sort | uniq -c                                                                                                                            2457 UTR3
    467 UTR5
    795 downstream
   3221 exonic
    103 intergenic
  16477 intronic
     75 ncRNA_UTR3
      7 ncRNA_UTR5
    128 ncRNA_exonic
    516 ncRNA_intronic
     22 splicing
    435 upstream
     79 upstream;downstream
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ if ($8 >= .05) { print $0 } } ' | perl -lane 'my @vals1 = split(/,/, $F[8]); print $vals1[0];' | sort | uniq -c
      3 UTR5
      1 downstream
      5 exonic
     21 intronic
      2 upstream
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ if ($8 >= .05) { print $0 } } ' | perl -lane 'my @vals1 = split(/,/, $F[8]); if ($vals1[0] eq "exonic") { print join("\t", @F[0..9]); }'
6       31238995        rs41553316      G       A       5374.66 VQSRTrancheSNP99.00to99.90      0.051   exonic,HLA-C,synonymous_SNV,NM_001243042:c.C474T:p.T158T       GT:AD:DP:GQ:PL
6       31238880        rs1050357       C       T       17465.6 VQSRTrancheSNP99.00to99.90      0.121   exonic,HLA-C,nonsynonymous_SNV,NM_001243042:c.G589A:p.E197K    GT:AD:DP:GQ:PL
6       31238875        rs1131104       G       A       17127.7 VQSRTrancheSNP99.00to99.90      0.121   exonic,HLA-C,synonymous_SNV,NM_001243042:c.C594T:p.N198N       GT:AD:DP:GQ:PL
6       31324128        rs1050642       C       T       17267.9 VQSRTrancheSNP90.00to99.00      0.077   exonic,HLA-B,synonymous_SNV,NM_005514:c.G435A:p.K145K   GT:AD:DP:GQ:PL
3       129214302       rs111401191     G       A       47524.7 VQSRTrancheSNP99.00to99.90      0.078   exonic,IFT122,stopgain_SNV,NM_052990:c.G1727A:p.W576X   GT:AD:DP:GQ:PL
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ if ($8 < .05) { print $0 } } ' | perl -lane 'my @vals1 = split(/,/, $F[8]); print $vals1[0];' | sort | uniq -c
   2548 UTR3
    462 UTR5
   1044 downstream
   2876 exonic
      1 exonic;splicing
    119 intergenic
  21444 intronic
     79 ncRNA_UTR3
     12 ncRNA_UTR5
    122 ncRNA_exonic
    727 ncRNA_intronic
      1 ncRNA_splicing
      8 splicing
    498 upstream
    101 upstream;downstream
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ if ($8 >= .05) { print $0 } } ' | perl -lane 'my @vals1 = split(/,/, $F[8]); print $vals1[0];' | sort | uniq -c
    795 UTR3
    118 UTR5
    331 downstream
    616 exonic
      1 exonic;splicing
     43 intergenic
   6609 intronic
     16 ncRNA_UTR3
     48 ncRNA_exonic
    228 ncRNA_intronic
    178 upstream
     24 upstream;downstream
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ print $8 } ' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.05), right=FALSE));"
> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.05), right=FALSE));

  [0,0.05) [0.05,0.1) [0.1,0.15) [0.15,0.2) [0.2,0.25) [0.25,0.3) [0.3,0.35)
     24782          7          7          4          2          1          0
[0.35,0.4) [0.4,0.45) [0.45,0.5) [0.5,0.55) [0.55,0.6) [0.6,0.65) [0.65,0.7)
         1          0          0          1          1          0          0
[0.7,0.75) [0.75,0.8) [0.8,0.85) [0.85,0.9) [0.9,0.95)   [0.95,1)
         1          1          1          0          0          5
>
>
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a.wAnnovarForQC.RanGenePerVariant.vcf.gz | perl -lane 'my @vals1 = split(/\;/, $F[7]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { splice(@F, 7, 1, ((split(/\=/, $entry1))[1])); } } $F[8] = join(",", ((split(/,/, $F[8]))[0..3])); print join("\t", @F[0..9]);' | awk '{ print $8 } ' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.05), right=FALSE));"
> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.05), right=FALSE));

  [0,0.05) [0.05,0.1) [0.1,0.15) [0.15,0.2) [0.2,0.25) [0.25,0.3) [0.3,0.35)
     30042       1705       1201        886        634        579        507
[0.35,0.4) [0.4,0.45) [0.45,0.5) [0.5,0.55) [0.55,0.6) [0.6,0.65) [0.65,0.7)
       351        428        286        233        390        276        290
[0.7,0.75) [0.75,0.8) [0.8,0.85) [0.85,0.9) [0.9,0.95)   [0.95,1)
       183        224        153        142        137        399
>
>
[  michaelt@node1  ~]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) | grep -v NA | perl -lane 'my $AF1 = (split(/\=/, $F[1]))[1]; my $AF2 = (split(/\=/, $F[2]))[1]; print $AF1, "\t", $AF2, "\t", $AF1 - $AF2;' | wc
  39049  117147  899714
[  michaelt@node1  ~]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) | grep -v NA | perl -lane 'my $AF1 = (split(/\=/, $F[1]))[1]; my $AF2 = (split(/\=/, $F[2]))[1]; print $AF1, "\t", $AF2, "\t", $AF1 - $AF2;' | grep -v \, | wc
  38859  116577  894990
[  michaelt@node1  ~]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) | grep -v NA | perl -lane 'my $AF1 = (split(/\=/, $F[1]))[1]; my $AF2 = (split(/\=/, $F[2]))[1]; print $AF1, "\t", $AF2, "\t", $AF1 - $AF2;' | grep -v \, | grep -v NA | wc
[  michaelt@node1  ~]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | awk '{ print $1, "\t", $2 }' | sort ) <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | awk '{ print $3, "\t", $4 }' | sort) | grep -v NA | perl -lane 'my $AF1 = (split(/\=/, $F[1]))[1]; my $AF2 = (split(/\=/, $F[2]))[1]; print $AF1, "\t", $AF2, "\t", $AF1 - $AF2;' | grep -v \, | grep -v NA | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs.AFplots.joinVers.jpeg\", width=4000, height=5000, res=300); par(mfrow=c(3,2)); plot(Data1[,1], Data1[,2], xlab=\"HIV AF\", ylab=\"1000G_EUR AF\", main=\"HIV vs. 1000G_EUR AF\"); abline(a=0,b=1); plot(-log10(Data1[,1]), -log10(Data1[,2]), xlab=\"HIV -log10(AF)\", ylab=\"1000G_EUR -log10(AF)\", main=\"HIV vs. 1000G_EUR -log10(AF)\"); abline(a=0,b=1); hist(Data1[,1], xlab=\"HIV AF\", main=\"HIV AF Histogram\", breaks=100); hist(-log10(Data1[,1]), xlab=\"HIV -log10(AF)\", main=\"HIV -log10(AF) Histogram\", breaks=100); hist(Data1[,2], xlab=\"1000G_EUR AF\", main=\"1000G_EUR AF Histogram\", breaks=100); hist(-log10(Data1[,2]), xlab=\"1000G_EUR -log10(AF)\", main=\"1000G_EUR -log10(AF) Histogram\", breaks=100); dev.off()"
> Data1 <- read.table(file('stdin'), header=F); jpeg("/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs.AFplots.joinVers.jpeg", width=4000, height=5000, res=300); par(mfrow=c(3,2)); plot(Data1[,1], Data1[,2], xlab="HIV AF", ylab="1000G_EUR AF", main="HIV vs. 1000G_EUR AF"); abline(a=0,b=1); plot(-log10(Data1[,1]), -log10(Data1[,2]), xlab="HIV -log10(AF)", ylab="1000G_EUR -log10(AF)", main="HIV vs. 1000G_EUR -log10(AF)"); abline(a=0,b=1); hist(Data1[,1], xlab="HIV AF", main="HIV AF Histogram", breaks=100); hist(-log10(Data1[,1]), xlab="HIV -log10(AF)", main="HIV -log10(AF) Histogram", breaks=100); hist(Data1[,2], xlab="1000G_EUR AF", main="1000G_EUR AF Histogram", breaks=100); hist(-log10(Data1[,2]), xlab="1000G_EUR -log10(AF)", main="1000G_EUR -log10(AF) Histogram", breaks=100); dev.off()
null device
          1
>
>
~~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs.AFplots.joinVers.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/.

#From Windows netbook via cmder
scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/*AFplots* .
#(C:\Users\PortMCT2\Desktop\Lab_Stuff\StephensLab\NUWrk\HIVStff\PG\ALL_MAPPING\Pools\MM5\PostMerge\mapping_pool_merged\GATK\Analyses\VCFComparisons)





















##20150316
##Trying to use /home/michaelt/Software/lollipops-v0.96-linux64 on top20/35 or so genes to see how things look and how they come out 

#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | sort -rg -k 2,2 | grep -vw NA | grep -vw NS | awk '{ print $1 }' | tail -n 35 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top35
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | sort -rg -k 2,2 | head -n -3 | awk '{ print $1 }' | tail -n 35 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top35

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/LollipopPlots 

#Below Works
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top35 | perl -ane 'print $F[0], "\t"; for (my $i = 1; $i<=$#F; $i++) { my $chrBP1 = (split(/_/, $F[$i]))[0]; $chrBP1 =~ s/:/,/g; my @annoInfo1 = split(/,/, `zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | grep $F[0] | grep $chrBP1`); chomp(@annoInfo1); print ((split(/\./, ((split(/:/, $annoInfo1[28]))[2])))[1]); print "\t";} print "\n";'

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top35 | perl -ane 'print $F[0], "\t"; for (my $i = 1; $i<=$#F; $i++) { my $chrBP1 = (split(/_/, $F[$i]))[0]; $chrBP1 =~ s/:/,/g; my @annoInfo1 = split(/,/, `zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | grep $F[0] | grep $chrBP1`); chomp(@annoInfo1); print ((split(/\./, ((split(/:/, $annoInfo1[28]))[2])))[1]); print "\t";} print "\n";' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/LollipopPlots/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top35.LollipopFormat

OLD_IFS=$IFS
IFS=$'\n'

for command1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/LollipopPlots/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top35.LollipopFormat`
do

	IFS=$OLD_IFS

	geneID1=`echo $command1 | perl -ane 'print $F[0];'`

#	echo $geneID1 $command1

	/home/michaelt/Software/lollipops-v0.96-linux64/lollipops -o=/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/LollipopPlots/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top35.${geneID1}.LollipopPlot.svg $command1

#	cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/LollipopPlots/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top35.${geneID1}.LollipopPlot.svg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/LollipopPlots/MM5.LollipopPlot.${geneID1}.Nonsyn.svg 

done

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/LollipopPlots/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top35.*.LollipopPlot.svg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/LollipopPlots/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/LollipopPlots/MM5.LollipopPlot.*.Nonsyn.svg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/LollipopPlots/.

#20150825 20150930 CHECK_1: Figure out if I was correct with my below explanation for this code here bere under the 'P2Subsample' part?
#20150825 -- think the below code was copy/pasted to follow with the P2Subsample stuff but when I realized that my initial version of approaching this, e.g. just by subsampling the main .bam file versus the original per individual .bam files, produced an odd result, I just stopped? The plan at that point was to go back and do the proper subsampling of the per individual .bam files to do it the more stringent way
#20150930 Note -- Yeah but I don't see another version of this code anywhere else in the file...so where was it copy/pasted from?
#20150930 Note -- I believe this code was originally before the P2 subsample section and after the lollipop code. This would make sense since the below files were sent out at the same time as the lollipop plots. Also, I don't believe this code would have been relevant to the P2subsampling of MM5. I think this code just got pushed down when I began the P2subsampling. As of 20150930 I'm going to bring this code back above the P2Subsampling section and below the lollipop code

for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSumit/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs`
do

#	paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w $gene1 | awk '{ print $1, "\t", $2, "\t", $4, "\t", $5, "\t", $8, "\t", $9, "\t", $11 }' ) \
#	grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w $gene1 | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }') | \
#	<(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs1.wACs.txt.gz | \
#	grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'if ($F[0] =~ m/^'$gene1'$/) { print join("\t", @F); }' | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }') | \
#	perl -lane 'if ($hash1{$F[5]}) { $hash1{$F[5]}++; } else { $hash1{$F[5]} = 1; } if (eof()) { if ($hash1{"
#	)

	paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane 'if ($F[0] =~ m/^'$gene1'$/) { print join("\t", @F); }' | awk '{ print $1, "\t", $2, "\t", $4, "\t", $5, "\t", $8, "\t", $9, "\t", $11 }' ) \
	<(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs1.wACs.txt.gz | \
	grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'if ($F[0] =~ m/^'$gene1'$/) { print join("\t", @F); }' | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }') | \
	perl -lane 'print join("", ((split(/:/, $F[10]))[1])), "\t", ((split(/:/, $F[11]))[1]), "\t", ((split(/:/, $F[12]))[1]), "\t", ((split(/:/, $F[13]))[1]), "\t", ((split(/:/, $F[14]))[1]), "\t", ((split(/:/, $F[15]))[1]);' | \
	R -q -e "Data1 <- read.table(file('stdin'), header=F); print(paste(apply(Data1, 2, sum)), quote=FALSE);" | grep -v ^\> | perl -lane 'splice(@F, 0, 1); print join("\t", @F);') \
	>> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSumit/MM5.skatO.nonsyn.qVals.ACs.Orig.txt
	
done

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSumit/MM5.skatO.nonsyn.qVals.ACs.Orig.txt | perl -lane 'splice(@F, 4, 2); print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.ForSudhir.txt 

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.ForSudhir.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/. 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSumit/MM5.skatO.nonsyn.qVals.ACs.Orig.txt | perl -lane 'splice(@F, 4, 3); print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSumit/MM5.skatO.nonsyn.qVals.ACs.txt

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSumit/MM5.skatO.nonsyn.qVals.ACs.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSumit/.

#QC Checks on file

~~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSumit/MM5.skatO.nonsyn.qVals.ACs.txt | wc
    474    4740   21473

[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSumit/MM5.skatO.nonsyn.qVals.ACs.Orig.txt | perl -lane 'if (($F[3] == $F[4]) && ($F[3] == $F[5]) && ($F[4] == $F[5])) { print join("\t", @F); } '  | wc
    474    6162   24741
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSumit/MM5.skatO.nonsyn.qVals.ACs.Orig.txt | perl -lane 'if (($F[3] != $F[4]) || ($F[3] != $F[5]) || ($F[4] != $F[5])) { print join("\t", @F); } '     
[  michaelt@node1  ~]$

[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSumit/MM5.skatO.nonsyn.qVals.ACs.txt | perl -lane 'my $val2 = $F[5] + $F[7]; if ($F[4] != $val2) { print join("\t", @F); }'
[  michaelt@node1  ~]$
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSumit/MM5.skatO.nonsyn.qVals.ACs.txt | perl -lane 'my $val2 = $F[5] + $F[7]; if ($F[4] == $val2) { print join("\t", @F); }' | wc
    474    4740   21473

[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSumit/MM5.skatO.nonsyn.qVals.ACs.txt | perl -lane 'my $val2 = $F[5] + $F[7]; if ($F[6] > $F[5]) { print join("\t", @F); }'
[  michaelt@node1  ~]$
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSumit/MM5.skatO.nonsyn.qVals.ACs.txt | perl -lane 'my $val2 = $F[5] + $F[7]; if ($F[6] <= $F[5]) { print join("\t", @F); }' | wc
    474    4740   21473

[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSumit/MM5.skatO.nonsyn.qVals.ACs.txt | perl -lane 'my $val2 = $F[8] + $F[9]; if ($F[7] < $val2) { print join("\t", @F); }'     
[  michaelt@node1  ~]$
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSumit/MM5.skatO.nonsyn.qVals.ACs.txt | perl -lane 'my $val2 = $F[8] + $F[9]; if ($F[7] >= $val2) { print join("\t", @F); }' | wc
    474    4740   21473
~~~

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.ForSudhir.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.ForSudhir.txt




##20150315
##Downsampling MM5 array to allSample.bam wc level comparable to P2 and rerunning GATK protocol as a real rough way to see if 'lower coverage' would affect the results we see in MM5 vs. P2
##Should also do QC level things on P2 to see singleton, doubleton calls, SFS, compared to MM5 results
##Downsampling randomly from allSample.bam file as a real, real rough first-attempt at this. Making assumption that 'all reads across all individuals' are more or less uniformly distributed so can just remove a 'proportion' of them from the overall .bam file to recapitulate a similar level of coverage between MM5 and P2

samtools view /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.bam | wc 

samtools flagstat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.bam

[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged]$246246156 + 0 in total (QC-passed reads + QC-failed reads)
0 + 0 duplicates                                                                                                                                                                                                                                                                                                           
246246156 + 0 mapped (100.00%:-nan%)
0 + 0 paired in sequencing                                                                                                                                                                                                                                                                                                 
0 + 0 read1
0 + 0 read2                                                                                                                                                                                                                                                                                                                
0 + 0 properly paired (-nan%:-nan%)
0 + 0 with itself and mate mapped                                                                                                                                                                                                                                                                                          
0 + 0 singletons (-nan%:-nan%)
0 + 0 with mate mapped to a different chr                                                                                                                                                                                                                                                                                  
0 + 0 with mate mapped to a different chr (mapQ>=5)
246246156 4680205654 661321756462                                                                                                                                                                                                                                                                                          

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptb.CreateVCFfilesPerChromosome.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptb.CreateVCFfilesPerChromosome.vs1.Tests.ForSubsample.sh

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptb.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptb.CreateVCFfilesPerChromosome.Launcher.vs1.Tests.ForSubsample.sh

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptb.CreateVCFfilesPerChromosome.Launcher.vs1.Tests.ForSubsample.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged 

vcf-concat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr1.GATK.ReduceReads.P2Subsample.UG.raw.vcf.gz \ 
/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr2.GATK.ReduceReads.P2Subsample.UG.raw.vcf.gz \
/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr3.GATK.ReduceReads.P2Subsample.UG.raw.vcf.gz \
/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr4.GATK.ReduceReads.P2Subsample.UG.raw.vcf.gz \
/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr5.GATK.ReduceReads.P2Subsample.UG.raw.vcf.gz \
/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr6.GATK.ReduceReads.P2Subsample.UG.raw.vcf.gz \
/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr7.GATK.ReduceReads.P2Subsample.UG.raw.vcf.gz \
/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr8.GATK.ReduceReads.P2Subsample.UG.raw.vcf.gz \
/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr9.GATK.ReduceReads.P2Subsample.UG.raw.vcf.gz \
/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr10.GATK.ReduceReads.P2Subsample.UG.raw.vcf.gz \
/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr11.GATK.ReduceReads.P2Subsample.UG.raw.vcf.gz \
/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr12.GATK.ReduceReads.P2Subsample.UG.raw.vcf.gz \
/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr13.GATK.ReduceReads.P2Subsample.UG.raw.vcf.gz \
/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr14.GATK.ReduceReads.P2Subsample.UG.raw.vcf.gz \
/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr15.GATK.ReduceReads.P2Subsample.UG.raw.vcf.gz \
/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr16.GATK.ReduceReads.P2Subsample.UG.raw.vcf.gz \
/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr17.GATK.ReduceReads.P2Subsample.UG.raw.vcf.gz \
/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr18.GATK.ReduceReads.P2Subsample.UG.raw.vcf.gz \
/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr19.GATK.ReduceReads.P2Subsample.UG.raw.vcf.gz \
/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr20.GATK.ReduceReads.P2Subsample.UG.raw.vcf.gz \
/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr21.GATK.ReduceReads.P2Subsample.UG.raw.vcf.gz \
/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.Chr22.GATK.ReduceReads.P2Subsample.UG.raw.vcf.gz \
/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrX.GATK.ReduceReads.P2Subsample.UG.raw.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.P2Subsample.UG.raw.vcf

[  michaelt@node6  ~/Data/1000G/VCFs]$cat ALL.chrAll.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf | wc
84740090 212951216435 853449167680

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptd.RecalAndApplyVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptd.RecalAndApplyVCFfilesPerChromosome.Launcher.vs1.Tests.ForSubsample.sh

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptd.RecalAndApplyVCFfilesPerChromosome.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptd.RecalAndApplyVCFfilesPerChromosome.vs2.Tests.ForSubsample.sh

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.P2Subsample.UG.raw.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.P2Subsample.ChrAll.GATK.ReduceReads.UG.raw.vcf

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptd.RecalAndApplyVCFfilesPerChromosome.Launcher.vs1.Tests.ForSubsample.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/ AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.P2Subsample

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/*tranches.pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/.

gzip -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.P2Subsample.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf
gzip -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.P2Subsample.UG.raw.vcf

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.P2Subsample.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz --remove-filtered VQSRTrancheSNP99.90to100.00 --remove-filtered LowQual --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.P2Subsample.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.P2Subsample.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.recode.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.P2Subsample.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.vcf

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.P2Subsample.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.vcf

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.P2Subsample.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.vcf.gz | perl -lane 'if ($F[0] =~ m/^#/) { print join("\t", @F); } else { my @alleles2 = split(/,/, $F[4]); if ($#alleles2 == 0) { print join("\t", @F); } }' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.P2Subsample.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.Bi.vcf.gz

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.P2Subsample.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.vcf.gz --min-alleles 2 --max-alleles 2 --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.P2Subsample.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.Bi.vs2

#So the two approaches are equivalent but I should probably do the 'vcftools' way when possible since it's a more 'uniform/consistent/documented' approach vs. an 'in-house one-liner' piece of code
~~~
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.P2Subsample.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.Bi.vcf.gz | wc
 684073 629233726 8020769475
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.P2Subsample.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.Bi.vs2.recode.vcf | wc
 684073 629233726 8020769475                                                                                                                                                                                                                                                                                               
~~~

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.P2Subsample.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.Bi.vs2.recode.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.P2Subsample.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.Bi.vs2.vcf

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.P2Subsample.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.Bi.vs2.vcf

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.DropOff.VCFVariants.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.P2Subsample.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.Bi.vcf.gz --window 1000 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.P2Subsample.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.Bi.DropOffTarg_1kb.vcf.gz

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.P2Subsample.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.Bi.DropOffTarg_1kb.vcf.gz --geno .95 --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.P2Subsample.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.Bi.DropOffTarg_1kb.geno95

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.P2Subsample.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.Bi.DropOffTarg_1kb.geno95.recode.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.P2Subsample.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.Bi.DropOffTarg_1kb.geno95.vcf

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.P2Subsample.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.Bi.DropOffTarg_1kb.geno95.vcf

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.P2Subsample.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.Bi.DropOffTarg_1kb.geno95.vcf.gz --hwe .0001 --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.P2Subsample.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.Bi.DropOffTarg_1kb.geno95.hwe1e4

#20151228 NOTE -- Starting back here to do the second round of this analysis. The goal here is to do a more 'precise' version of subsampling by starting at the post-merge, per-individual level .bam files first, ie subsampling them first directly and then doing the full-bam file genotype calling afterwards

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.vs2.P2Subsample.vs1.sh

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs2.P2Subsample.vs1.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs2.P2Subsample.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs

~~~
[  michaelt@login2  ~/Software/ruby-2.2.4/ext/openssl]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_106-124/POOL106_08RL/POOL106_08RL.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.bam | wc
1046623 5961234 275415951
[  michaelt@login2  ~/Software/ruby-2.2.4/ext/openssl]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_106-124/POOL106_08RL/P2Subsample/POOL106_08RL.QCed.preGATK.QCed.samplesMerged.P2Subsample25.rmdup.BQSR.calmd.bam | wc
 782489 4469149 206248104
[  michaelt@login2  ~/Software/ruby-2.2.4/ext/openssl]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_106-124/POOL106_04RL/POOL106_04RL.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.bam | wc
1127198 6494934 298383163
[  michaelt@login2  ~/Software/ruby-2.2.4/ext/openssl]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_106-124/POOL106_04RL/P2Subsample/POOL106_04RL.QCed.preGATK.QCed.samplesMerged.P2Subsample25.rmdup.BQSR.calmd.bam | wc
 779611 4475398 207055459
[  michaelt@login2  ~/Software/ruby-2.2.4/ext/openssl]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_106-124/POOL106_05RL/POOL106_05RL.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.bam | wc
 977239 5596280 256548464
[  michaelt@login2  ~/Software/ruby-2.2.4/ext/openssl]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_106-124/POOL106_05RL/P2Subsample/POOL106_05RL.QCed.preGATK.QCed.samplesMerged.P2Subsample25.rmdup.BQSR.calmd.bam | wc
 592440 3369853 154721848
 ~~~

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs2.P2Subsample.sh

#20151228 CHECK_0 -- Prob: Change /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs2.P2Subsample.vs1.sh to P2Subsample75

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs | perl -F/\;/ -lane 'my @vals1 = split(/\|/, $F[0]); my @vals2 = split(/\//, $vals1[0]); $vals2[9] = "PostMerge"; if ($vals2[11] =~ m/(POOL\w+).*(\d\dRL)/) { $vals2[$#vals2] = $1 . "_" . $2;  print "/", join("/", @vals2[1..11]), "/P2Subsample/", $vals2[11], ".QCed.preGATK.QCed.samplesMerged.P2Subsample25.rmdup.BQSR.calmd.bam\t";}' | head -n 5 | perl -lane 'my @vals1 = split(/P2Subsample25/, $F[0]); my $val1 = $vals1[0] . "P2Subsample25" . $vals1[1]; my $val2 = $vals1[0] . "P2Subsample75" . $vals1[1]; print $val1, "\n", $val2, "\n";'

#bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs2.P2Subsample.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.20140328_174052.output.multPOOLperIndvCollected.dropIndvs
#Like ~16.6hrs to run?

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.pta.SplitBamIntoChromosomes.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.P2Subsample75.rmdup.BQSR.calmd.AllPoolsMerged

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.pta.RunReduceReadsPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.P2Subsample75.rmdup.BQSR.calmd.AllPoolsMerged

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptb.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.P2Subsample75.rmdup.BQSR.calmd.AllPoolsMerged

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.P2Subsample75.rmdup.BQSR.calmd.AllPoolsMerged*vcf* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/.

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptc.MergeVCFsAcrossChromosomes.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptc.MergeVCFsAcrossChromosomes.vs3.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptc.MergeVCFsAcrossChromosomes.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.P2Subsample75.rmdup.BQSR.calmd.AllPoolsMerged

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptd.RecalAndApplyVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.P2Subsample75.rmdup.BQSR.calmd.AllPoolsMerged

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/*tranches.pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/.

#On Apple MacBook Air
scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/*tranches.pdf /Users/mturchin20/Documents/Work/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/.

gzip -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.P2Subsample75.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf &

gzip -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.P2Subsample75.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.vcf &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.P2Subsample75.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz --remove-filtered VQSRTrancheSNP99.90to100.00 --remove-filtered LowQual --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.P2Subsample75.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9 

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.P2Subsample75.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.recode.vcf






#Compare if same variants have been discovered/chosen? then do cutoffs? join? 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.P2Subsample75.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.recode.vcf.gz | grep -v ^# | awk '{ print $1, "\t", $2 }' | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.recode.vcf.gz | grep -v ^# | awk '{ print $1, "\t", $2 }' | wc
join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/P2Subsample/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.P2Subsample75.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.recode.vcf.gz | grep -v ^# | awk '{ print $1, "_", $2 }' ) <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.recode.vcf.gz | grep -v ^# | awk '{ print $1, "_", $2 }' ) | wc



zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.vcf.gz










zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.vcf.gz | perl -lane 'if ($F[0] =~ m/^#/) { print join("\t", @F); } else { my @alleles2 = split(/,/, $F[4]); if ($#alleles2 == 0) { print join("\t", @F); } }' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.DropOff.VCFVariants.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/120326_HG19_SW_MM5_EZ_HX3/120326_HG19_SW_MM5_EZ.altrd.bed --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --window 1000 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.vcf.gz

#/home/michaelt/Software/vcftools_0.1.12a/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.vcf.gz --missing-indv --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb

gzip -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf

#Exploring what HWE cutoff to use

#vcftools --gzvcf  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz --hardy --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95

#Going with .0001 HWE cutoff. At .00001 1809 SNPs are dropped and at .0001 2470 SNPs are dropped, so not much is lost for another magnitude increase in strictness. Between .0001 and .001 not a large number of SNPs would be lost either, however see examples where obs is 900/0/1 and exp is 900/2/0 on a number of X chromosome locations and this produces a p-value between .0001 and .001. It appears that vcf file encoding gives males genotypes for both 'chromosomes' even though they only have one, so 900/0/1 is not unexpected. Therefore I feel like these SNPs should be kept and going up to .001 may start leading to a greater proportion of false positives
#20151217 NOTE -- going back and thinking/addressing this again, there is also the case that by at this point the total list of SNPs before HWE filtering is 75856. Other 'sequencing' studies generally have less conservative HWE cutoffs (in the 1e6 to 1e7 range), but I believe this is in part to the much larger number of variants they are filtering in their particular projects. Despite this also being a 'sequencing' study, because it is only targetting a subset of gene exomes, the total number of variants is still rather modest. This further suggests, aside from just empirically looking as above, that using the more common HWE cutoff of 1e4 is fine ('more common' in the sense that SNP genotyping studies tend to have this cutoff, whereas 'sequencing' studies tend to have a lower cutoff).
#20151217 NOTE -- As 'full sequencing study' examples with HWE cutoffs between 1e6 and 1e7 http://www.nature.com/articles/sdata201511 & http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3852935/ 

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz --hwe .0001 --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.recode.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf


#Note -- takes about ~2-3 days to complete even with -nt 6?
/home/shared/software/java/jre1.7.0_09/bin/java -Xmx5g -jar /home/shared/software/GATK/2.5.2/GenomeAnalysisTK.jar -T VariantAnnotator -R /home/michaelt/Data/HumanGenome/GRCh37/Version4/human_g1k_v37.fasta -I /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.bam  -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.HRun.vcf -A HomopolymerRun --variant /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf -nt 6

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.HRun.vcf  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.HRun.vcf.idx /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.idx

gzip -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | perl -lane 'my @vals1 = split(/;/, $F[7]); foreach my $SNPinfo (@vals1) { if ($SNPinfo =~ m/HRun/) { print $SNPinfo; } };' | sort | uniq -c 

[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | perl -lane 'my @vals1 = split(/;/, $F[7]); foreach my $SNPinfo (@vals1) { if ($SNPinfo =~ m/HRun/) { print $SNPinfo; } };' | sort | uniq -c
	47294 HRun=0
	18359 HRun=1
	5419 HRun=2
	1668 HRun=3
	401 HRun=4
	97 HRun=5
	15 HRun=6
	6 HRun=7
	2 HRun=8
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | grep HRun=8 | perl -lane 'print join("\t", @F[0..9]);'
2       69628089        rs12464941      A       T       6521.76 VQSRTrancheSNP99.00to99.90      AA=A;AC=197;AF=0.111;AN=1776;BaseQRankSum=-18.785;DB;DP=6423;Dels=0.00;FS=7.055;HRun=8;HaplotypeScore=4.3163;InbreedingCoeff=-0.0905;MLEAC=179;MLEAF=0.101;MQ=59.99;MQ0=0;MQRankSum=1.477;POSITIVE_TRAIN_SITE;QD=4.41;ReadPosRankSum=-0.423;VQSLOD=1.36;culprit=QD GT:AD:DP:GQ:PL  0/0:10,0:10:30:0,30,275
11      63395938        .       G       T       87.12   VQSRTrancheSNP99.00to99.90      AA=G;AC=4;AF=2.230e-03;AN=1794;BaseQRankSum=-3.920;DP=5680;Dels=0.00;FS=3.144;HRun=8;HaplotypeScore=3.2769;InbreedingCoeff=-0.0297;MLEAC=2;MLEAF=1.115e-03;MQ=59.99;MQ0=0;MQRankSum=1.000;QD=2.72;ReadPosRankSum=-1.026;VQSLOD=-1.290e+00;culprit=QD       GT:AD:DP:GQ:PL  0/0:9,0:9:27:0,27,276
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | grep HRun=7 | perl -lane 'print join("\t", @F[0..9]);'
2       182390107       rs2305580       A       T       10719.60        VQSRTrancheSNP99.00to99.90      AA=A;AC=156;AF=0.086;AN=1804;BaseQRankSum=-25.342;DB;DP=10424;Dels=0.00;FS=31.039;HRun=7;HaplotypeScore=5.5982;InbreedingCoeff=-0.0131;MLEAC=144;MLEAF=0.080;MQ=60.00;MQ0=0;MQRankSum=0.217;NEGATIVE_TRAIN_SITE;POSITIVE_TRAIN_SITE;QD=5.97;ReadPosRankSum=3.211;VQSLOD=-6.062e-01;culprit=FS      GT:AD:DP:GQ:PL  0/0:13,0:13:39:0,39,410
3       112272598       rs183474871     T       A       721.65  VQSRTrancheSNP99.00to99.90      AA=T;AC=31;AF=0.018;AN=1758;BaseQRankSum=-11.115;DB;DP=4946;Dels=0.01;FS=4.698;HRun=7;HaplotypeScore=2.4788;InbreedingCoeff=-0.0591;MLEAC=24;MLEAF=0.014;MQ=59.98;MQ0=0;MQRankSum=1.272;POSITIVE_TRAIN_SITE;QD=3.86;ReadPosRankSum=-1.012;VQSLOD=-1.082e-01;culprit=QD     GT:AD:DP:GQ:PL  0/0:2,1:3:6:0,6,62
7       24855651        rs76330454      T       A       2708.55 VQSRTrancheSNP99.00to99.90      AA=t;AC=166;AF=0.094;AN=1770;BaseQRankSum=-8.001;DB;DP=5579;Dels=0.03;FS=71.176;HRun=7;HaplotypeScore=4.9968;InbreedingCoeff=-0.0691;MLEAC=120;MLEAF=0.068;MQ=59.92;MQ0=0;MQRankSum=-0.782;NEGATIVE_TRAIN_SITE;POSITIVE_TRAIN_SITE;QD=2.58;ReadPosRankSum=2.005;VQSLOD=-1.157e+00;culprit=QD       GT:AD:DP:GQ:PL  0/0:4,0:4:12:0,12,107
12      104333866       rs1165685       T       A       24175.80        VQSRTrancheSNP99.00to99.90      AA=t;AC=560;AF=0.320;AN=1748;BaseQRankSum=2.702;DB;DP=4895;Dels=0.03;FS=3.324;HRun=7;HaplotypeScore=4.9939;InbreedingCoeff=-0.1876;MLEAC=523;MLEAF=0.299;MQ=59.97;MQ0=0;MQRankSum=-0.169;POSITIVE_TRAIN_SITE;QD=8.39;ReadPosRankSum=1.064;VQSLOD=-3.806e-01;culprit=DP     GT:AD:DP:GQ:PL  0/1:1,2:3:15:37,0,15
14      81992779        rs76653020      A       T       2523.52 VQSRTrancheSNP99.00to99.90      AA=A;AC=36;AF=0.020;AN=1808;BaseQRankSum=-12.836;DB;DP=12204;Dels=0.01;FS=4.667;HRun=7;HaplotypeScore=6.2385;InbreedingCoeff=0.0585;MLEAC=29;MLEAF=0.016;MQ=59.99;MQ0=0;MQRankSum=2.132;POSITIVE_TRAIN_SITE;QD=5.20;ReadPosRankSum=-2.150;VQSLOD=-3.809e-01;culprit=HaplotypeScore GT:AD:DP:GQ:PL  0/0:13,0:13:39:0,39,415
17      17180049        rs150128160     A       T       452.79  VQSRTrancheSNP99.00to99.90      AA=A;AC=27;AF=0.015;AN=1774;BaseQRankSum=-3.092;DB;DP=5161;Dels=0.00;FS=0.759;HRun=7;HaplotypeScore=4.9453;InbreedingCoeff=-0.0183;MLEAC=18;MLEAF=0.010;MQ=59.94;MQ0=0;MQRankSum=0.504;POSITIVE_TRAIN_SITE;QD=3.10;ReadPosRankSum=-0.415;VQSLOD=-6.715e-01;culprit=QD      GT:AD:DP:GQ:PL  0/0:3,0:3:9:0,9,85
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | grep HRun=6 | perl -lane 'print join("\t", @F[0..9]);'
1       184679716       rs149454208     C       A       274.86  VQSRTrancheSNP99.00to99.90      AA=C;AC=18;AF=9.923e-03;AN=1814;BaseQRankSum=-7.850;DB;DP=9470;Dels=0.00;FS=9.233;HRun=6;HaplotypeScore=5.5968;InbreedingCoeff=0.0145;MLEAC=9;MLEAF=4.961e-03;MQ=59.98;MQ0=0;MQRankSum=-0.037;POSITIVE_TRAIN_SITE;QD=2.10;ReadPosRankSum=-1.369;VQSLOD=-7.438e-01;culprit=QD       GT:AD:DP:GQ:PL  0/0:11,0:11:33:0,33,319
2       61760814        rs115819167     C       T       3750.38 VQSRTrancheSNP99.00to99.90      AA=C;AC=46;AF=0.025;AN=1816;BaseQRankSum=-15.429;DB;DP=12101;Dels=0.01;FS=18.682;HRun=6;HaplotypeScore=4.4872;InbreedingCoeff=-0.0312;MLEAC=42;MLEAF=0.023;MQ=59.99;MQ0=0;MQRankSum=-0.764;POSITIVE_TRAIN_SITE;QD=5.78;ReadPosRankSum=-1.032;VQSLOD=0.188;culprit=FS       GT:AD:DP:GQ:PL  0/0:6,0:6:15:0,15,156
3       51449206        .       A       T       173.08  VQSRTrancheSNP99.00to99.90      AA=A;AC=9;AF=5.039e-03;AN=1786;BaseQRankSum=-6.696;DP=5412;Dels=0.01;FS=0.839;HRun=6;HaplotypeScore=2.1658;InbreedingCoeff=-0.0358;MLEAC=4;MLEAF=2.240e-03;MQ=59.99;MQ0=0;MQRankSum=-0.333;QD=2.58;ReadPosRankSum=0.974;VQSLOD=-8.534e-01;culprit=QD       GT:AD:DP:GQ:PL  0/0:8,0:8:24:0,24,241
4       113574658       rs145663069     G       T       3963.70 VQSRTrancheSNP99.00to99.90      AA=g;AC=67;AF=0.037;AN=1812;BaseQRankSum=-12.104;DB;DP=10857;Dels=0.01;FS=3.332;HRun=6;HaplotypeScore=6.4317;InbreedingCoeff=-0.0475;MLEAC=60;MLEAF=0.033;MQ=59.92;MQ0=0;MQRankSum=-0.678;POSITIVE_TRAIN_SITE;QD=5.27;ReadPosRankSum=-0.755;VQSLOD=-9.985e-01;culprit=HaplotypeScore       GT:AD:DP:GQ:PL  0/0:10,0:10:30:0,30,281
5       52095024        rs116734477     C       T       1415.55 VQSRTrancheSNP99.00to99.90      AA=C;AC=63;AF=0.036;AN=1766;BaseQRankSum=-10.197;DB;DP=4709;Dels=0.01;FS=5.456;HRun=6;HaplotypeScore=2.1840;InbreedingCoeff=-0.0134;MLEAC=56;MLEAF=0.032;MQ=59.99;MQ0=0;MQRankSum=0.626;POSITIVE_TRAIN_SITE;QD=4.03;ReadPosRankSum=0.504;VQSLOD=0.471;culprit=QD   GT:AD:DP:GQ:PL  0/0:3,0:3:9:0,9,84
6       7894794 rs1238994       G       T       26388.40        VQSRTrancheSNP99.00to99.90      AA=G;AC=464;AF=0.260;AN=1786;BaseQRankSum=-24.753;DB;DP=6021;Dels=0.02;FS=9.077;HRun=6;HaplotypeScore=3.7333;InbreedingCoeff=0.0220;MLEAC=416;MLEAF=0.233;MQ=59.99;MQ0=0;MQRankSum=-1.277;POSITIVE_TRAIN_SITE;QD=9.70;ReadPosRankSum=1.017;VQSLOD=1.39;culprit=DP  GT:AD:DP:GQ:PL  0/0:4,0:4:12:0,12,108
6       8062437 rs9405369       T       A       17852.20        VQSRTrancheSNP99.00to99.90      AA=A;AC=360;AF=0.205;AN=1758;BaseQRankSum=-34.017;DB;DP=6008;Dels=0.03;FS=23.705;HRun=6;HaplotypeScore=4.9241;InbreedingCoeff=-0.0703;MLEAC=341;MLEAF=0.194;MQ=59.98;MQ0=0;MQRankSum=0.498;NEGATIVE_TRAIN_SITE;POSITIVE_TRAIN_SITE;QD=7.21;ReadPosRankSum=3.990;VQSLOD=-1.722e-01;culprit=FS       GT:AD:DP:GQ:PL  0/0:2,0:2:6:0,6,61
7       148451445       .       C       T       100.12  VQSRTrancheSNP99.00to99.90      AA=C;AC=7;AF=3.867e-03;AN=1810;BaseQRankSum=-3.029;DP=11028;Dels=0.00;FS=0.732;HRun=6;HaplotypeScore=4.2330;InbreedingCoeff=-0.0087;MLEAC=1;MLEAF=5.525e-04;MQ=59.98;MQ0=0;MQRankSum=-1.175;QD=1.79;ReadPosRankSum=-1.119;VQSLOD=0.023;culprit=QD  GT:AD:DP:GQ:PL  0/0:8,0:8:24:0,24,239
9       125782985       rs137999993     G       T       59.56   VQSRTrancheSNP99.00to99.90      AA=G;AC=4;AF=2.222e-03;AN=1800;BaseQRankSum=-4.567;DB;DP=7953;Dels=0.00;FS=0.000;HRun=6;HaplotypeScore=2.6845;InbreedingCoeff=-0.0210;MLEAC=2;MLEAF=1.111e-03;MQ=59.99;MQ0=0;MQRankSum=-0.096;POSITIVE_TRAIN_SITE;QD=1.65;ReadPosRankSum=-1.713;VQSLOD=0.402;culprit=QD    GT:AD:DP:GQ:PL  0/0:13,0:13:39:0,39,416

vcftools --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf --plink --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4

plink --file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4

###IBD pair list -- decided going to do eigenstrat first and see if 1) dropping white individuals due to QC and only looking at white individuals lowers the set of removed individuals overall 2) potentially look for clustering amongst those that are dropping or have high z1 vs z0, z2 (e.g. always at the end of the cloud of white non-hispanics)

POOL103_03RL POOL103_03RL POOL105_03RL POOL105_03RL UN    NA  0.0000  0.0480  0.9520  0.9760  -1  0.998919  1.0000      NA
POOL47_10RL  POOL47_10RL  POOL87_11RL  POOL87_11RL UN    NA  0.0038  0.1065  0.8896  0.9429  -1  0.997455  1.0000 313.0000
POOL175_11RL POOL175_11RL  POOL43_10RL  POOL43_10RL UN    NA  0.0019  0.1445  0.8535  0.9258  -1  0.996671  1.0000      NA
POOL36_11RL  POOL36_11RL  POOL50_12RL  POOL50_12RL UN    NA  0.0077  0.1560  0.8363  0.9143  -1  0.996195  1.0000 306.0000
POOL176_12RL POOL176_12RL  POOL36_12RL  POOL36_12RL UN    NA  0.0270  0.1440  0.8291  0.9010  -1  0.995736  1.0000 345.0000
POOL104_07RL POOL104_07RL  POOL33_05RL  POOL33_05RL UN    NA  0.0155  0.1616  0.8230  0.9037  -1  0.995775  1.0000 149.5000
POOL165_12RL POOL165_12RL  POOL71_11RL  POOL71_11RL UN    NA  0.0213  0.1740  0.8048  0.8918  -1  0.995276  1.0000 297.0000
POOL28_06RL  POOL28_06RL  POOL91_11RL  POOL91_11RL UN    NA  0.0194  0.1825  0.7981  0.8893  -1  0.995154  1.0000 73.7500
POOL112_10RL POOL112_10RL POOL116_10RL POOL116_10RL UN    NA  0.0275  0.1951  0.7775  0.8750  -1  0.994566  1.0000 145.0000
POOL163_12RL POOL163_12RL  POOL69_08RL  POOL69_08RL UN    NA  0.0372  0.2257  0.7371  0.8500  -1  0.993508  1.0000 96.3333

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome | sort -rg -k 9,9 | vi -

POOL103_03RL POOL103_03RL POOL105_03RL POOL105_03RL UN    NA  0.0000  0.0434  0.9566  0.9783  -1  0.998919  1.0000      NA
POOL47_10RL  POOL47_10RL  POOL87_11RL  POOL87_11RL UN    NA  0.0034  0.0963  0.9003  0.9484  -1  0.997455  1.0000 313.0000
POOL36_11RL  POOL36_11RL  POOL50_12RL  POOL50_12RL UN    NA  0.0068  0.1412  0.8521  0.9226  -1  0.996195  1.0000 306.0000
POOL112_10RL POOL112_10RL POOL116_10RL POOL116_10RL UN    NA  0.0242  0.1772  0.7985  0.8871  -1  0.994566  1.0000 145.0000
POOL163_12RL POOL163_12RL  POOL69_08RL  POOL69_08RL UN    NA  0.0328  0.2053  0.7619  0.8645  -1  0.993508  1.0000 96.3333

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -E 'POOL103_03RL|POOL105_03RL'
POOL103_03RL    POOL103_03RL    2       2       -9      -9
POOL105_03RL    POOL105_03RL    2       2       -9      -9

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -E 'POOL47_10RL|POOL87_11RL'
POOL87_11RL     POOL87_11RL     1       -9      -9      -9
POOL47_10RL     POOL47_10RL     2       2       -9      -9

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -E 'POOL36_11RL|POOL50_12RL'
POOL50_12RL     POOL50_12RL     2       2       1       -9
POOL36_11RL     POOL36_11RL     2       2       1       -9

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -E 'POOL112_10RL|POOL116_10RL'
POOL112_10RL    POOL112_10RL    1       1       -9      -9
POOL116_10RL    POOL116_10RL    2       2       -9      -9

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -E 'POOL163_12RL|POOL69_08RL'
POOL69_08RL     POOL69_08RL     1       1       -9      -9
POOL163_12RL    POOL163_12RL    1       -9      -9      -9

perl -e 'print "POOL103_03RL\nPOOL105_03RL\nPOOL87_11RL\nPOOL47_10RL\nPOOL50_12RL\nPOOL36_11RL\nPOOL112_10RL\nPOOL116_10RL\nPOOL69_08RL\nPOOL163_12RL\n";' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.FIDs

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.FIDs --depth --missing --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs

cat AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.idepth
INDV    N_SITES MEAN_DEPTH
POOL103_03RL    73251   22.4186
POOL105_03RL    73254   22.2204
POOL112_10RL    71708   7.28291
POOL116_10RL    73100   15.5343
POOL163_12RL    71776   6.90738
POOL36_11RL     73195   14.8737
POOL47_10RL     73238   17.4978
POOL50_12RL     73127   11.7687
POOL69_08RL     73211   17.0737
POOL87_11RL     73180   13.5165

cat AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.imiss
INDV    N_DATA  N_GENOTYPES_FILTERED    N_MISS  F_MISS
POOL103_03RL    73261   0       10      0.000136498
POOL105_03RL    73261   0       7       9.55488e-05
POOL112_10RL    73261   0       1553    0.0211982
POOL116_10RL    73261   0       161     0.00219762
POOL163_12RL    73261   0       1485    0.02027
POOL36_11RL     73261   0       66      0.000900889
POOL47_10RL     73261   0       23      0.000313946
POOL50_12RL     73261   0       134     0.00182908
POOL69_08RL     73261   0       50      0.000682491
POOL87_11RL     73261   0       81      0.00110564

So choosing:

POOL105_03RL
POOL36_11RL
POOL116_10RL

perl -e 'print "POOL103_03RL\nPOOL112_10RL\nPOOL163_12RL\nPOOL47_10RL\nPOOL50_12RL\nPOOL69_08RL\nPOOL87_11RL\n";' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome | sort -rg -k 9,9 | head -n 10000 | awk '{ print $1, "\n", $3 }' | perl -lane 'chomp(@F); print join("\t", @F);' | sort | uniq -c | sort -g -k 1,1 | awk '{ if ($1 >= 100 ) { print $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ2Countgte100.FIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome | sort -rg -k 8,8 | head -n 10000 | awk '{ print $1, "\n", $3 }' | perl -lane 'chomp(@F); print join("\t", @F);' | sort | uniq -c | sort -g -k 1,1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.Z1CountList

#NOTE -- dropping these individuals because they are starkly greater than anyone else in the list. Individuals from the 'high z2' count list are not being dropped because they appear as just to be the tail of a distribution versus being such a group of outliers
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome | sort -rg -k 8,8 | head -n 10000 | awk '{ print $1, "\n", $3 }' | perl -lane 'chomp(@F); print join("\t", @F);' | sort | uniq -c | sort -g -k 1,1 | awk '{ if ($1 >= 200) { print $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.FIDs

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs

cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ2Countgte100.FIDs | perl -lane 'push(@F, "Z2"); print join("\t", @F);') <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.FIDs | perl -lane 'push(@F, "Z1"); print join("\t", @F);') > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.OddHighZ1OrZ2.FID_ZNums


#EIGENSTRAT stuff here

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/VLookUp.Python.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/VLookUp.Python.DataHapMap3Overlap.vs1.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/VLookUp.Python.DataHapMap3Overlap.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.map --file2 /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.liftOver.hg18tohg19.map | awk '{ print $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.rsIDs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.rsIDs --recode --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3

plink --file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3 --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3

#Note -- there are a few SNPs that don't match up, about ~30? Just letting them drop since troubleshooting almost got most of them back but not all of them and yeah

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.map | awk '{ print $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.map.rsIDs 

plink --bfile /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.map.rsIDs --keep  /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.CEUCHBYRI.Unrelated.FIDIIDs --recode --out  /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Vs2

plink --file /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Vs2 --make-bed --out /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Vs2

paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.bim /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Vs2.bim | awk '{ if (($5 == $11) || ($5 == $12) ) { print $0 } } ' | awk '{ print $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.FinalListwDrops.rsIDs 

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3 --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.FinalListwDrops.rsIDs --recode --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.vs2

plink --bfile /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.FinalListwDrops.rsIDs --keep /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.CEUCHBYRI.Unrelated.FIDIIDs --recode --out /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Vs2.vs2

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.vs2.ped /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Vs2.vs2.ped > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.ped

plink --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.ped --map /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.vs2.map --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.phenoColEdit.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.phenoColEdit.pedind

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.bim /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.pedsnp

#Created EigenstratFiles in GATK and moved Eigenstrat/ files into EigenstratFiles/.

scp mturchin20@wolfy.uchicago.edu:/Users/mturchin20/clstrHme/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/Eigenstrat/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.parfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/.

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASS.wAA.DropOffTargetVariants.1kbWindow.95geno.recode.overlapHapMap3.wHapMap3Data.CEUCHBYRI.parfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.wHapMap3Data.CEUCHBYRI.parfile

/home/michaelt/Software/EIG5.0.1/bin/smartpca -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.wHapMap3Data.CEUCHBYRI.parfile

#REDO WITH 4 INDVS MISSING FROM PHENO FILE
#python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt1
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca | cat - <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca | grep -E 'POOL43_10RL|POOL101_11RL|POOL18S_07RL|POOL2S_03RL' | perl -lane '$F[$#F] = "NoPhenotype"; print join("\t", @F); ') > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt1

#Added in the following four individuals with the tag 'NoPhenotype' at the end manually in the pt1 file
#POOL43_10RL POOL101_11RL POOL18S_07RL POOL2S_03RL

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.HapMap3vs.py --file1 /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.txt --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt2

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race

sed -i 's/American_Indian_or_Alaskan_Native/AIndian_or_Alaskan_Native/g' /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.1vs2.xtxt

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.3vs4.xtxt

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race -c 5:6 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.5vs6.xtxt

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.1vs2.pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/.

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.*.pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/.

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race | awk '{ if ((($2 >= 0) && ($2 <= .22)) && (($3 >= -.01) && ($3 <= .021))) { print $0 } } ' | grep White_non-Hispanic > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed -c 1:2 -p White_non-Hispanic -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.plot.1vs2.xtxt

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed | perl -lane 'my @vals1 = split(/:/, $F[0]); print $vals1[0];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed | perl -lane 'my @vals1 = split(/:/, $F[0]); print $vals1[0], "\t", $vals1[0];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteACed.plot.1vs2.pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/.

#Need to do this on the no-HapMap3 dataset

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race | grep White_non-Hispanic | awk 'BEGIN{ while(getline < "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.OddHighZ1OrZ2.FID_ZNums") { FID = $1":"$1; FIDList[FID] = $2; }} { if (FIDList[$1]) { if (FIDList[$1] == "Z1") { $12 = "Z1High"; } else if (FIDList[$1] == "Z2") { $12 = "Z2High"; } else{ print "blah" }  } else { $12 = "NormalIBD"; } print $0; } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.IBDissuesTagged

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.IBDissuesTagged -c 1:2 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.IBDissuesTagged.plot.1vs2.xtxt 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed | awk 'BEGIN{ while(getline < "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.OddHighZ1OrZ2.FID_ZNums") { FID = $1":"$1; FIDList[FID] = $2; }} { if (FIDList[$1]) { if (FIDList[$1] == "Z1") { $12 = "Z1High"; } else if (FIDList[$1] == "Z2") { $12 = "Z2High"; } else{ print "blah" }  } else { $12 = "NormalIBD"; } print $0; } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 1:2 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.1vs2.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 3:4 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.3vs4.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 5:6 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.5vs6.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 7:8 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.7vs9.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 9:10 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.9vs10.xtxt

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/*pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/.


cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.phenoColEdit.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.phenoColEdit.pedind

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.bim /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.pedsnp

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.wHapMap3Data.CEUCHBYRI.parfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.parfile

/home/michaelt/Software/EIG5.0.1/bin/smartpca -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.parfile

#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.Results.pca.altrd.race

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race | awk 'BEGIN{ while(getline < "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.OddHighZ1OrZ2.FID_ZNums") { FID = $1":"$1; FIDList[FID] = $2; }} { if (FIDList[$1]) { if (FIDList[$1] == "Z1") { $12 = "Z1High"; } else if (FIDList[$1] == "Z2") { $12 = "Z2High"; } else{ print "blah" }  } else { $12 = "NormalIBD"; } print $0; } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race.IBDissuesTagged

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race.plot.1vs2.xtxt 

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race.plot.3vs4.xtxt 

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race.IBDissuesTagged -c 1:2 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race.IBDissuesTagged.plot.1vs2.xtxt

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race.IBDissuesTagged -c 3:4 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Results.pca.altrd.race.IBDissuesTagged.plot.3vs4.xtxt

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.5PCs.covar
#Edit code in between runs
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.covar > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.pheno

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar | awk '{ print $1, "\t", $2 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar | awk '{ print $1 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDs

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDs --non-ref-ac 1 --recode  --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.recode.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.allSNPs

plink --bfile AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD --freq --out AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.PLINK

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --freq --out  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs.maf05

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.10PCs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.10PCs.maf05

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 2 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.DropIBD.5PCs.maf05.pheno2





vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --missing --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4  

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss | awk '{ if ($5 >= .2) { print $1 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.pheno

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar 

#20151227 NOTE -- This is where the ...AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs... notation begins due to the use of indv80 imiss threshold, e.g. the 'Vs3' at the beginning of the filenames here
#ComeHere1

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar | awk '{ print $1, "\t", $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar | awk '{ print $1 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDs

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDs --non-ref-ac 1 --recode  --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.recode.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80 

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.allSNPs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs 
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.10PCs
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.10PCs.maf05
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 2 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno2
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 3 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno3
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno --mpheno 4 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno4

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.assoc.logistic Pheno1_justWhite_Logistic_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.assoc.logistic.QQplot.jpeg & 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.assoc.logistic Pheno1_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.assoc.logistic.QQplot.jpeg & 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.10PCs.assoc.logistic Pheno1_justWhite_Logistic_10PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.10PCs.assoc.logistic.QQplot.jpeg & 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.10PCs.maf05.assoc.logistic Pheno1_justWhite_Logistic_maf05_10PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.10PCs.maf05.assoc.logistic.QQplot.jpeg & 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.assoc.logistic Pheno2_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno2.assoc.logistic.QQplot.jpeg & 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.assoc.logistic Pheno3_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno3.assoc.logistic.QQplot.jpeg & 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.assoc.logistic Pheno4_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno4.assoc.logistic.QQplot.jpeg & 

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/.

#Allele Frequency stuff/collection stuff here now
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ print $1 }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.FIDs 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($3 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroNeg.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($4 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroHighlyExposedNeg.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($3 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroPos.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justRapidVRapid.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno | awk '{ if ($5 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSlowVSlow.FIDs










##20150422
##Further working on properly setting up annotation code stuff

~~~
[  michaelt@node11  ~]$perl -e 'my $val1 = `echo "nanana"`; print $val1;'
nanana
[  michaelt@node11  ~]$perl -e 'my $val2 = "shana"; my $val1 = `echo $val2 "nanana"`; print $val1;'
shana nanana

[  michaelt@node11  ~]$join <( python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | sort -k 1,1) <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^\# | perl -lane 'print join("_", @F[0..1]), "\t", $F[3], "\t", $F[4];' | sort -k 1,1 ) | head -n 10
10_103190011 intronic BTRC . .  A G                                                                                                                                                                                                                                                                                        
10_103190439 intronic BTRC . .  A G
10_103190521 intronic BTRC . .  G A                                                                                                                                                                                                                                                                                        
10_103190533 intronic BTRC . .  C T
10_103221343 intronic BTRC . .  A G                                                                                                                                                                                                                                                                                        
10_103221377 intronic BTRC . .  G A
10_103221446 intronic BTRC . .  T C                                                                                                                                                                                                                                                                                        
10_103221659 intronic BTRC . .  A T
10_103221969 intronic BTRC . .  A G                                                                                                                                                                                                                                                                                        
10_103238644 intronic BTRC . .  C T
sort: write failed: standard output: Broken pipe                                                                                                                                                                                                                                                                           
sort: write error
sort: write failed: standard output: Broken pipe                                                                                                                                                                                                                                                                           
sort: write error
~~~

#perl -lane 'my $pAndqVals = `zcat ' | \
#perl -lane 'if ($F[0] =~ m/^#CHROM/) { splice(@F, 7, 0, ("PVAL", "QVAL")); print join("\t", @F); } else { my $pVal = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w $F[2] | awk "{ print $2 }"`; my $qVal = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w $F[2] | awk "{ print $4 }"`; splice(@F, 7, 0, ($pVal, $qVal)); print join("\t", @F); }' | \
#10      103190521       103190521       intronic        BTRC    .       .       G       A       1528    A:5     A:1     A:1     A:4     A:0     A:1

cat <(echo "#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        REF     ALT     AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW") 
<(join <( python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | sort -k 1,1) <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^\# | perl -lane 'print join("_", @F[0..1]), "\t", $F[3], "\t", $F[4];' | sort -k 1,1) | \
perl -lane 'if ($F[0] =~ m/^#CHROM/) { splice(@F, 7, 0, ("PVAL", "QVAL")); print join("\t", @F); } else { my $pVal = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w $F[2] | sed -r "s/\\s+/ /g" | cut -d " " -f 2 `; my $qVal = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w $F[2] | sed -r "s/\\s+/ /g" | cut -d " " -f 4 `; splice(@F, 5, 0, ($pVal, $qVal)); chomp(@F); print join("\t", @F); }' | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
perl -lane 'splice(@F, 0, 1, (((split(/_/, $F[0]))[0]), ((split(/_/, $F[0]))[1]), ((split(/_/, $F[0]))[1]))); print join("\t", @F);' |
perl -ane 'print join("\t", @F[0..8]); my $alt = $F[8]; my $totAC = join("", ((split(/:/, $F[9]))[1])) + join("", ((split(/:/, $F[10]))[1])); print "\t", $totAC; for (my $i=9; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", $F[$i]; } } print "\n";' | sort -g -k 1,1 -k 2,2 ) |
#awk '{ print $1, "\t", $2, "\t", $3, "\t", $8, "\t", $9, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $10, "\t", $11, "\t", $12, "\t", $13, "\t", $14, "\t", $15, "\t", $16 }' |
#awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $10, "\t", $11, "\t", $12, "\t", $13, "\t", $14, "\t", $15, "\t", $16 }' |
#awk '{ if ($1 !~ /^#/) { print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $10, "\t", $11, "\t", $12, "\t", $13, "\t", $14, "\t", $15, "\t", $16, "\t", $8, "\t", $9 } else { print $0 } }' |
bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs2.wACs.txt.gz

cat <(echo "#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        REF     ALT     AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW") \
<(join <( python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | sort -k 1,1) <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^\# | perl -lane 'print join("_", @F[0..1]), "\t", $F[3], "\t", $F[4];' | sort -k 1,1) | \
perl -lane 'if ($F[0] =~ m/^#CHROM/) { splice(@F, 7, 0, ("PVAL", "QVAL")); print join("\t", @F); } else { my $pVal = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w $F[2] | sed -r "s/\\s+/ /g" | cut -d " " -f 2 `; my $qVal = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w $F[2] | sed -r "s/\\s+/ /g" | cut -d " " -f 4 `; splice(@F, 5, 0, ($pVal, $qVal)); chomp(@F); print join("\t", @F); }' | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count.gz | perl 
-lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count.gz | perl -lane 'print 
join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
perl -lane 'splice(@F, 0, 1, (((split(/_/, $F[0]))[0]), ((split(/_/, $F[0]))[1]), ((split(/_/, $F[0]))[1]))); print join("\t", @F);' |
perl -ane 'print join("\t", @F[0..8]); my $alt = $F[8]; my $totAC = join("", ((split(/:/, $F[9]))[1])) + join("", ((split(/:/, $F[10]))[1])); print "\t", $totAC; for (my $i=9; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", $F[$i]; } } print "\n";' | sort -g -k 1,1 -k 2,2 ) |
#awk '{ print $1, "\t", $2, "\t", $3, "\t", $8, "\t", $9, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $10, "\t", $11, "\t", $12, "\t", $13, "\t", $14, "\t", $15, "\t", $16 }' |
#awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $10, "\t", $11, "\t", $12, "\t", $13, "\t", $14, "\t", $15, "\t", $16 }' |
#awk '{ if ($1 !~ /^#/) { print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $10, "\t", $11, "\t", $12, "\t", $13, "\t", $14, "\t", $15, "\t", $16, "\t", $8, "\t", $9 } else { print $0 } }' |
bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs1.wACs.txt.gz

join <( python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | sort -k 1,1) <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^\# | perl -lane 'print join("_", @F[0..1]), "\t", $F[3], "\t", $F[4];' | sort -k 1,1) | \
perl -lane 'if ($F[0] =~ m/^#CHROM/) { splice(@F, 7, 0, ("PVAL", "QVAL")); print join("\t", @F); } else { my $pVal = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w $F[2] | sed -r "s/\\s+/ /g" | cut -d " " -f 2 `; my $qVal = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w $F[2] | sed -r "s/\\s+/ /g" | cut -d " " -f 4 `; splice(@F, 5, 0, ($pVal, $qVal)); chomp(@F); print join("\t", @F); }'

join <( python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | sort -k 1,1) <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^\# | perl -lane 'print join("_", @F[0..1]), "\t", $F[3], "\t", $F[4];' | sort -k 1,1) | \
perl -lane 'if ($F[0] =~ m/^#CHROM/) { splice(@F, 7, 0, ("PVAL", "QVAL")); print join("\t", @F); } else { my $geneEntry = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w "$F[2]"`; if ($geneEntry) { my @geneVals1 = split(/\s+/, $geneEntry); splice(@F, 5, 0, ($geneVals1[1], $geneVals1[3])); print join("\t", @F); } else { splice(@F, 5, 0, ("NA", "NA")); print join("\t", @F);} } '  

join <( python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | sort -k 1,1) <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^\# | perl -lane 'print join("_", @F[0..1]), "\t", $F[3], "\t", $F[4];' | sort -k 1,1) | \
perl -lane '$F[1] = join("|", split(/[\;,\,]/, $F[1])); $F[2] = join("|", split(/[\;,\,]/, $F[2])); print join("\t", @F);' | \
perl -lane 'my @geneEntries = split(/\|/, $F[2]); my @pVals; my @qVals; foreach my $gene (@geneEntries) { my $geneEntry = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w "$gene"`; if ($geneEntry) { my @geneVals1 = split(/\s+/, $geneEntry); push(@pVals, $geneVals1[1]); push(@qVals, $geneVals1[3]); } else { push(@pVals, "NA"); push(@qVals, "NA"); } } splice(@F, 5, 0, (join("|", @pVals), join("|", @qVals))); print join("\t", @F);' > ondfgoundfg 
my $geneEntry = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w "$F[2]"`; if ($geneEntry) { my @geneVals1 = split(/\s+/, $geneEntry); splice(@F, 5, 0, ($geneVals1[1], $geneVals1[3])); print join("\t", @F); } else { splice(@F, 5, 0, ("NA", "NA")); print join("\t", @F);} } '  

~~~
[  michaelt@node11  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs2.wACs.txt.gz | grep dist | head -n 10
X       80185258        80185258        intergenic,intergenic   BRWD3(dist=120025),HMGN5(dist=183942)   .,.     .,.     NA      NA      0
X       80185294        80185294        intergenic,intergenic   BRWD3(dist=120061),HMGN5(dist=183906)   .,.     .,.     NA      NA      0
X       80185612        80185612        intergenic,intergenic   BRWD3(dist=120379),HMGN5(dist=183588)   .,.     .,.     NA      NA      0
X       80185646        80185646        intergenic,intergenic   BRWD3(dist=120413),HMGN5(dist=183554)   .,.     .,.     NA      NA      0
X       80185930        80185930        intergenic,intergenic   BRWD3(dist=120697),HMGN5(dist=183270)   .,.     .,.     NA      NA      0
X       80185988        80185988        intergenic,intergenic   BRWD3(dist=120755),HMGN5(dist=183212)   .,.     .,.     NA      NA      0
X       80186103        80186103        intergenic,intergenic   BRWD3(dist=120870),HMGN5(dist=183097)   .,.     .,.     NA      NA      0
X       80186551        80186551        intergenic,intergenic   BRWD3(dist=121318),HMGN5(dist=182649)   .,.     .,.     NA      NA      0
1       28420855        28420855        intergenic,intergenic   EYA3(dist=5707),PTAFR(dist=52822)       .,.     .,.     NA      NA      0
1       28420882        28420882        intergenic,intergenic   EYA3(dist=5734),PTAFR(dist=52795)       .,.     .,.     NA      NA      0
~~~

#The quotes around $gene in ...ut.wVarCounts | grep -w "$gene"`; if ($geneEntry) { my @gen... appears to be necessary. Without the quotes, entries such as the above intergenic genes, which contain parentheses, seem to cause bash problems
#Please note that intergenic entries, separated by a comma, are actually just single variant entries showing the closest two genes to this intergenic mutation. Variants that have more than one entry are separated by semi-colons, hence why only splitting variants that have ';' in the 'LOC' category and not ',' as well. However a p-value is being given to any circumstances a gene appears that has a p-value from the association analyses
#CHECK_1: Take into account the ...(dist=... entries as well for linking the gene p-/q-value association information

cat <(echo "#CHROM	FROM	TO	LOC	GENE	VARTYPE	AACHANGE	PVAL	QVAL	REF	ALT	AC_ALL	AC_TOTAL	AC_SERONEG	AC_HESERONEG	AC_SEROPOS	AC_RAPID	AC_SLOW") \
<(join <( python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | sort -k 1,1) <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^\# | perl -lane 'print join("_", @F[0..1]), "\t", $F[3], "\t", $F[4];' | sort -k 1,1) | \
perl -lane '$F[1] = join("|", split(/[\;,\,]/, $F[1])); $F[2] = join("|", split(/[\;,\,]/, $F[2])); print join("\t", @F);' | \
perl -lane 'my @geneEntries = split(/\|/, $F[2]); my @pVals; my @qVals; foreach my $gene (@geneEntries) { my $geneEntry = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w "$gene"`; if ($geneEntry) { my @geneVals1 = split(/\s+/, $geneEntry); push(@pVals, $geneVals1[1]); push(@qVals, $geneVals1[3]); } else { push(@pVals, "NA"); push(@qVals, "NA"); } } splice(@F, 5, 0, (join("|", @pVals), join("|", @qVals))); print join("\t", @F);' | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
perl -lane 'splice(@F, 0, 1, (((split(/_/, $F[0]))[0]), ((split(/_/, $F[0]))[1]), ((split(/_/, $F[0]))[1]))); print join("\t", @F);' | 
perl -ane 'print join("\t", @F[0..10]); my $alt = $F[10]; my $totAC = join("", ((split(/:/, $F[11]))[1])) + join("", ((split(/:/, $F[12]))[1])); print "\t", $totAC; for (my $i=11; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", $F[$i]; } } print "\n";' | sort -g -k 1,1 -k 2,2 ) | 
#awk '{ print $1, "\t", $2, "\t", $3, "\t", $8, "\t", $9, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $10, "\t", $11, "\t", $12, "\t", $13, "\t", $14, "\t", $15, "\t", $16 }' | 
#awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $10, "\t", $11, "\t", $12, "\t", $13, "\t", $14, "\t", $15, "\t", $16 }' | 
#awk '{ if ($1 !~ /^#/) { print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $10, "\t", $11, "\t", $12, "\t", $13, "\t", $14, "\t", $15, "\t", $16, "\t", $8, "\t", $9 } else { print $0 } }' | 
bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs2.wACs.txt.gz

#CHECK_1: Change multi-gene inputs from ';' to '|' to account for .vcf INFO fields being separated by ';'s

~~~
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs2.wACs.txt.gz | head -n 20                                                                                                                                                                                                                                                           
#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        PVAL    QVAL    REF     ALT     AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW
X       16863126        16863126        UTR3    RBBP7   .       .       0.8921100000    0.70177 C       T       1546    T:3     T:0     T:0     T:3     T:0     T:0                                                                                                                                                        
X       16863135        16863135        UTR3    RBBP7   .       .       0.8921100000    0.70177 G       A       1546    A:2     A:1     A:1     A:1     A:0     A:0
X       16863691        16863691        intronic        RBBP7   .       .       0.8921100000    0.70177 A       G       1546    G:2     G:2     G:2     G:0     G:0     G:0                                                                                                                                                
X       16863783        16863783        intronic        RBBP7   .       .       0.8921100000    0.70177 T       C       1544    C:2     C:0     C:0     C:2     C:0     C:0
X       16863871        16863871        intronic        RBBP7   .       .       0.8921100000    0.70177 C       T       1542    T:5     T:3     T:1     T:2     T:0     T:0                                                                                                                                                
X       16864025        16864025        exonic  RBBP7   nonsynonymous_SNV       NM_001198719:c.T1267C:p.F423L   0.8921100000    0.70177 A       G       1542    G:1     G:0     G:0     G:1     G:0     G:1
~~~

cat <(echo "#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        PVAL    QVAL    REF     ALT     AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW") <(join <( python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | sort -k 1,1) <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^\# | perl -lane 'print join("_", @F[0..1]), "\t", $F[3], "\t", $F[4];' | sort -k 1,1) | perl -lane '$F[1] = join("|", split(/[\;,\,]/, $F[1])); $F[2] = join("|", split(/[\;,\,]/, $F[2])); print join("\t", @F);' | perl -lane 'my @geneEntries = split(/\|/, $F[2]); my @pVals; my @qVals; foreach my $gene (@geneEntries) { my $geneEntry = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w "$gene"`; if ($geneEntry) { my @geneVals1 = split(/\s+/, $geneEntry); push(@pVals, $geneVals1[1]); push(@qVals, $geneVals1[3]); } else { push(@pVals, "NA"); push(@qVals, "NA"); } } splice(@F, 5, 0, (join("|", @pVals), join("|", @qVals))); print join("\t", @F);' | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | perl -lane 'splice(@F, 0, 1, (((split(/_/, $F[0]))[0]), ((split(/_/, $F[0]))[1]), ((split(/_/, $F[0]))[1]))); print join("\t", @F);' | perl -ane 'print join("\t", @F[0..10]); my $alt = $F[10]; my $totAC = join("", ((split(/:/, $F[11]))[1])) + join("", ((split(/:/, $F[12]))[1])); print "\t", $totAC; for (my $i=11; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", $F[$i]; } } print "\n";' | sort -g -k 1,1 -k 2,2 ) | /home/shared/software/tabix-0.2.5/bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs2.wACs.txt.gz


#All of this is QC checks to infer/imply that my ...arCounts | grep -w "$gene"`; if ($geneEntry) { my @geneVal... line in the 4th piece of code for the above chunk is actually working and that genes not being caught, e.g. being given a p-val/q-val of 'NA' are in fact no present in /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts, which appears to be the case
#Also ran a version where I did not split the multiple entries (delineated by a '|' in the p-val/q-val entries for the variant) prior to splitting them and accounting for every instance of 'NA' available
~~~
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs2.wACs.txt.gz | perl -lane 'if ($F[7] =~ m/NA/) { print $F[4], "\t", $F[7]; } ' | sort | uniq -c | head -n 20
2 ACOT8|SNX21     1.0000000000|NA
3 ACOT8|ZSWIM3    1.0000000000|NA
7 ACSBG1  NA
1 ACSBG1|IDH3A    NA|0.8102417000
3 ADAMTSL4-AS1    NA
9 ADAP1|GET4      NA|0.0419389700
3 ADM5    NA
2 ADM5|PRMT1      NA|NA
85 ADRBK1  NA
5 AFF4|LEAP2      0.3738752000|NA
141 AGPS    NA
138 AHCYL1  NA
6 AIMP1|TBCK      0.8119655000|NA
2 AIMP2|EIF2AK1   0.8721100000|NA
2 AKAP9|CYP51A1   0.2298240000|NA
3 AKT3    NA
28 AKT3|SDCCAG8    NA|0.4503489000
1 ALDOC   NA
2 ALDOC|SPAG5     NA|0.1191984000
6 AOC2|PSME3      NA|0.6338704000
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs2.wACs.txt.gz | perl -lane 'if ($F[7] =~ m/NA/) { print $F[4], "\t", $F[7]; } ' | sort | uniq -c | wc        
588    1764   22050
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs2.wACs.txt.gz | perl -lane 'if ($F[7] =~ m/^NA$/) { print $F[4], "\t", $F[7]; } ' | sort | uniq -c | head -n 10
7 ACSBG1  NA
3 ADAMTSL4-AS1    NA
3 ADM5    NA
85 ADRBK1  NA
141 AGPS    NA
138 AHCYL1  NA
3 AKT3    NA
1 ALDOC   NA
4 AP4M1   NA
2 APEX1   NA
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs2.wACs.txt.gz | perl -lane 'if ($F[7] =~ m/^NA$/) { print $F[4], "\t", $F[7]; } ' | sort | uniq -c | wc        
184     552    3336
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs2.wACs.txt.gz | perl -lane 'if ($F[7] =~ m/^NA$/) { print $F[4], "\t", $F[7]; } ' | sort | uniq -c | awk '{ print $2 }' | perl -lane 'system("cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w $F[0]");'
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs2.wACs.txt.gz | perl -lane 'if ($F[7] =~ m/^NA$/) { print $F[4], "\t", $F[7]; } ' | sort | uniq -c | awk '{ print $2 }' | perl -lane 'system("cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep $F[0]");'
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs2.wACs.txt.gz | perl -lane 'if ($F[7] =~ m/^NA$/) { print $F[4], "\t", $F[7]; } ' | sort | uniq -c | awk '{ print $2 }' | perl -lane 'system("cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep $F[0]"); print $F[0];' | head -n 10
ACSBG1
ADAMTSL4-AS1
ADM5
ADRBK1
AGPS
AHCYL1
AKT3
ALDOC
AP4M1
APEX1
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs2.wACs.txt.gz | perl -lane 'if ($F[7] =~ m/NA/) { print $F[4], "\t", $F[7]; } ' | sort | uniq -c | perl -lane 'my @vals1 = split(/\|/, $F[1]); my @vals2 = split(/\|/, $F[2]); for (my $i = 0; $i <= $#vals2; $i++) { if ($vals2[$i] =~ m/NA/) { print $vals1[$i]; } }' | sort | uniq -c | head -n 20
2 ACSBG1
1 ADAMTSL4-AS1
1 ADAP1
2 ADM5
1 ADRBK1
1 AGPS
1 AHCYL1
2 AKT3
1 ALDH3B1
2 ALDOC
1 AOC2
2 AP4M1
2 APEX1
1 ARPC5L
1 ARVCF
1 ASB3
2 ATP6V1C2
1 B3GALTL(dist=36824)
1 B3GALTL(dist=36967)
1 B4GALT7(dist=2969)
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs2.wACs.txt.gz | perl -lane 'if ($F[7] =~ m/NA/) { print $F[4], "\t", $F[7]; } ' | sort | uniq -c | perl -lane 'my @vals1 = split(/\|/, $F[1]); my @vals2 = split(/\|/, $F[2]); for (my $i = 0; $i <= $#vals2; $i++) { if ($vals2[$i] =~ m/NA/) { print $vals1[$i]; } }' | sort | uniq -c | wc
780    1560   17834
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs2.wACs.txt.gz | perl -lane 'if ($F[7] =~ m/NA/) { print $F[4], "\t", $F[7]; } ' | sort | uniq -c | perl -lane 'my @vals1 = split(/\|/, $F[1]); my @vals2 = split(/\|/, $F[2]); for (my $i = 0; $i <= $#vals2; $i++) { if ($vals2[$i] =~ m/NA/) { print $vals1[$i]; } }' | sort | uniq -c | grep -v dist | wc
250     500    3771
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs2.wACs.txt.gz | perl -lane 'if ($F[7] =~ m/NA/) { print $F[4], "\t", $F[7]; } ' | sort | uniq -c | perl -lane 'my @vals1 = split(/\|/, $F[1]); my @vals2 = split(/\|/, $F[2]); for (my $i = 0; $i <= $#vals2; $i++) { if ($vals2[$i] =~ m/NA/) { print $vals1[$i]; } }' | sort | uniq -c | grep -v dist | head -n 20
2 ACSBG1
1 ADAMTSL4-AS1
1 ADAP1
2 ADM5
1 ADRBK1
1 AGPS
1 AHCYL1
2 AKT3
1 ALDH3B1
2 ALDOC
1 AOC2
2 AP4M1
2 APEX1
1 ARPC5L
1 ARVCF
1 ASB3
2 ATP6V1C2
1 BECN1
1 BIN3
1 BLOC1S5
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs2.wACs.txt.gz | perl -lane 'if ($F[7] =~ m/NA/) { print $F[4], "\t", $F[7]; } ' | sort | uniq -c | perl -lane 'my @vals1 = split(/\|/, $F[1]); my @vals2 = split(/\|/, $F[2]); for (my $i = 0; $i <= $#vals2; $i++) { if ($vals2[$i] =~ m/NA/) { print $vals1[$i]; } }' | sort | uniq -c | grep -v dist | awk '{ print $2 }' | perl -lane 'system("cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep $F[0]");'
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs2.wACs.txt.gz | perl -lane 'if ($F[7] =~ m/NA/) { print $F[4], "\t", $F[7]; } ' | sort | uniq -c | perl -lane 'my @vals1 = split(/\|/, $F[1]); my @vals2 = split(/\|/, $F[2]); for (my $i = 0; $i <= $#vals2; $i++) { if ($vals2[$i] =~ m/NA/) { print $vals1[$i]; } }' | sort | uniq -c | grep -v dist | awk '{ print $2 }' | perl -lane 'system("cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep $F[0]"); print $F[0];' | head -n 10 
ACSBG1
ADAMTSL4-AS1
ADAP1
ADM5
ADRBK1
AGPS
AHCYL1
AKT3
ALDH3B1
ALDOC
~~~

cat <(echo "#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        PVAL    QVAL    REF     ALT     AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW") \
<(join <( python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | sort -k 1,1) <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^\# | perl -lane 'print join("_", @F[0..1]), "\t", $F[3], "\t", $F[4];' | sort -k 1,1) | \
perl -lane '$F[1] = join("|", split(/[\;,\,]/, $F[1])); $F[2] = join("|", split(/[\;,\,]/, $F[2])); print join("\t", @F);' | \
perl -lane 'my @geneEntries = split(/\|/, $F[2]); my @pVals; my @qVals; foreach my $gene (@geneEntries) { $gene = ((split(/\(/, $gene))[0]); my $geneEntry = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w "$gene"`; if ($geneEntry) { my @geneVals1 = split(/\s+/, $geneEntry); push(@pVals, $geneVals1[1]); push(@qVals, $geneVals1[3]); } else { push(@pVals, "NA"); push(@qVals, "NA"); } } splice(@F, 5, 0, (join("|", @pVals), join("|", @qVals))); print join("\t", @F);' | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
perl -lane 'splice(@F, 0, 1, (((split(/_/, $F[0]))[0]), ((split(/_/, $F[0]))[1]), ((split(/_/, $F[0]))[1]))); print join("\t", @F);' |
perl -ane 'print join("\t", @F[0..10]); my $alt = $F[10]; my $totAC = join("", ((split(/:/, $F[11]))[1])) + join("", ((split(/:/, $F[12]))[1])); print "\t", $totAC; for (my $i=11; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", $F[$i]; } } print "\n";' | sort -g -k 1,1 -k 2,2 ) |
#awk '{ print $1, "\t", $2, "\t", $3, "\t", $8, "\t", $9, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $10, "\t", $11, "\t", $12, "\t", $13, "\t", $14, "\t", $15, "\t", $16 }' |
#awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $10, "\t", $11, "\t", $12, "\t", $13, "\t", $14, "\t", $15, "\t", $16 }' |
#awk '{ if ($1 !~ /^#/) { print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $10, "\t", $11, "\t", $12, "\t", $13, "\t", $14, "\t", $15, "\t", $16, "\t", $8, "\t", $9 } else { print $0 } }' |
bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs2.wACs.txt.gz

~~~
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs2.wACs.txt.gz | perl -lane 'if ($F[7] =~ m/NA/) { print $F[4], "\t", $F[7]; } ' | sort | uniq -c | perl -lane 'my @vals1 = split(/\|/, $F[1]); my @vals2 = split(/\|/, $F[2]); for (my $i = 0; $i <= $#vals2; $i++) { if ($vals2[$i] =~ m/NA/) { print ((split(/\(/, $vals1[$i]))[0]); } }' | sort | uniq -c | wc  
    273     546    4116
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs2.wACs.txt.gz | perl -lane 'if ($F[7] =~ m/NA/) { print $F[4], "\t", $F[7]; } ' | sort | uniq -c | perl -lane 'my @vals1 = split(/\|/, $F[1]); my @vals2 = split(/\|/, $F[2]); for (my $i = 0; $i <= $#vals2; $i++) { if ($vals2[$i] =~ m/NA/) { print ((split(/\(/, $vals1[$i]))[0]); } }' | sort | uniq -c | awk '{ print $2 }' | perl -lane 'system("cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep $F[0]");'
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs2.wACs.txt.gz | perl -lane 'if ($F[7] =~ m/NA/) { print $F[4], "\t", $F[7]; } ' | sort | uniq -c | perl -lane 'my @vals1 = split(/\|/, $F[1]); my @vals2 = split(/\|/, $F[2]); for (my $i = 0; $i <= $#vals2; $i++) { if ($vals2[$i] =~ m/NA/) { print ((split(/\(/, $vals1[$i]))[0]); } }' | sort | uniq -c | awk '{ print $2 }' | perl -lane 'system("cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep $F[0]"); print $F[0];' | head -n 10
ACSBG1
ADAMTSL4-AS1
ADAP1
ADM5
ADRBK1
AGPS
AHCYL1
AKT3
ALDH3B1
ALDOC
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs2.wACs.txt.gz | perl -lane 'if ($F[7] =~ m/NA/) { print $F[4], "\t", $F[7]; } ' | sort | uniq -c | grep "(dist=" | perl -lane 'my @vals1 = split(/\|/, $F[1]); my @vals2 = split(/\|/, $F[2]); for (my $i = 0; $i <= $#vals2; $i++) { if ($vals2[$i] !~ m/NA/) { print ((split(/=/, $vals1[$i]))[0]), "\t", $vals2[$i]; } }' | sort | uniq -c | wc        
15      30     284
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs2.wACs.txt.gz | perl -lane 'if ($F[7] =~ m/NA/) { print $F[4], "\t", $F[7]; } ' | sort | uniq -c | grep "(dist=" | perl -lane 'my @vals1 = split(/\|/, $F[1]); my @vals2 = split(/\|/, $F[2]); for (my $i = 0; $i <= $#vals2; $i++) { if ($vals2[$i] !~ m/NA/) { print ((split(/=/, $vals1[$i]))[0]), "\t", $vals2[$i]; } }' | sort | uniq -c           
11 CLN6(dist
7 CUL1(dist
2 EIF3F(dist
2 EIF3H(dist
9 EVL(dist
25 GEMIN6(dist
7 HIST1H3E(dist
2 HSPH1(dist
5 MESDC2(dist
16 MSH2(dist
12 PSME3(dist
11 RPN1(dist
11 SCAF11(dist
19 TAB2(dist
1 TMED9(dist
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs2.wACs.txt.gz | perl -lane 'if ($F[7] =~ m/NA/) { print $F[4], "\t", $F[7]; } ' | sort | uniq -c | grep "(dist=" | perl -lane 'my @vals1 = split(/\|/, $F[1]); my @vals2 = split(/\|/, $F[2]); for (my $i = 0; $i <= $#vals2; $i++) { if ($vals2[$i] =~ m/NA/) { print ((split(/=/, $vals1[$i]))[0]), "\t", $vals2[$i]; } }' | sort | uniq -c | wc
28      56     559
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs2.wACs.txt.gz | perl -lane 'if ($F[7] =~ m/NA/) { print $F[4], "\t", $F[7]; } ' | sort | uniq -c | grep "(dist=" | perl -lane 'my @vals1 = split(/\|/, $F[1]); my @vals2 = split(/\|/, $F[2]); for (my $i = 0; $i <= $#vals2; $i++) { if ($vals2[$i] =~ m/NA/) { print ((split(/=/, $vals1[$i]))[0]), "\t", $vals2[$i]; } }' | sort | uniq -c | head -n 10
2 B3GALTL(dist
1 B4GALT7(dist
12 BECN1(dist
8 BRWD3(dist
7 C7orf33(dist
9 EML1(dist
32 EPHB6(dist
21 EYA3(dist
11 FEM1B(dist
7 HIST1H2AE(dist
~~~

cat <(echo "#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        PVAL    QVAL    REF     ALT     AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW") \
<(join <( python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | sort -k 1,1) <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^\# | perl -lane 'print join("_", @F[0..1]), "\t", $F[3], "\t", $F[4];' | sort -k 1,1) | \
perl -lane '$F[1] = join("|", split(/[\;,\,]/, $F[1])); $F[2] = join("|", split(/[\;,\,]/, $F[2])); print join("\t", @F);' | \
perl -lane 'my @geneEntries = split(/\|/, $F[2]); my @pVals; my @qVals; foreach my $gene (@geneEntries) { $gene = ((split(/\(/, $gene))[0]); my $geneEntry = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w "$gene"`; if ($geneEntry) { my @geneVals1 = split(/\s+/, $geneEntry); push(@pVals, $geneVals1[1]); push(@qVals, $geneVals1[3]); } else { push(@pVals, "NA"); push(@qVals, "NA"); } } splice(@F, 5, 0, (join("|", @pVals), join("|", @qVals))); print join("\t", @F);' | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
perl -lane 'splice(@F, 0, 1, (((split(/_/, $F[0]))[0]), ((split(/_/, $F[0]))[1]), ((split(/_/, $F[0]))[1]))); print join("\t", @F);' |
perl -ane 'print join("\t", @F[0..10]); my $alt = $F[10]; my $totAC = join("", ((split(/:/, $F[11]))[1])) + join("", ((split(/:/, $F[12]))[1])); print "\t", $totAC; for (my $i=11; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", ((split(/\:/, $F[$i]))[1]); } } print "\n";' | sort -g -k 1,1 -k 2,2 ) |
bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz

~~~
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | head -n 10
#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        PVAL    QVAL    REF     ALT     AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW
X       16863126        16863126        UTR3    RBBP7   .       .       0.8921100000    0.70177 C       T       1546    3       0       0       3       0       0
X       16863135        16863135        UTR3    RBBP7   .       .       0.8921100000    0.70177 G       A       1546    2       1       1       1       0       0
X       16863691        16863691        intronic        RBBP7   .       .       0.8921100000    0.70177 A       G       1546    2       2       2       0       0       0
X       16863783        16863783        intronic        RBBP7   .       .       0.8921100000    0.70177 T       C       1544    2       0       0       2       0       0
X       16863871        16863871        intronic        RBBP7   .       .       0.8921100000    0.70177 C       T       1542    5       3       1       2       0       0
X       16864025        16864025        exonic  RBBP7   nonsynonymous_SNV       NM_001198719:c.T1267C:p.F423L   0.8921100000    0.70177 A       G       1542    1       0       0       1       0       1
X       16864106        16864106        intronic        RBBP7   .       .       0.8921100000    0.70177 T       C       1538    6       0       0       6       0       2
X       16864144        16864144        intronic        RBBP7   .       .       0.8921100000    0.70177 T       C       1532    7       5       1       2       0       2
X       16866829        16866829        intronic        RBBP7   .       .       0.8921100000    0.70177 C       T       1512    2       1       1       1       0       0
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | perl -lane 'print $#F;' | sort | uniq -c
  63864 17
~~~

cat <(echo "#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        PVAL    QVAL    REF     ALT     AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW") <(join <( python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | sort -k 1,1) <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^\# | perl -lane 'print join("_", @F[0..1]), "\t", $F[3], "\t", $F[4];' | sort -k 1,1) | perl -lane '$F[1] = join("|", split(/[\;,\,]/, $F[1])); $F[2] = join("|", split(/[\;,\,]/, $F[2])); print join("\t", @F);' | perl -lane 'my @geneEntries = split(/\|/, $F[2]); my @pVals; my @qVals; foreach my $gene (@geneEntries) { $gene = ((split(/\(/, $gene))[0]); my $geneEntry = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w "$gene"`; if ($geneEntry) { my @geneVals1 = split(/\s+/, $geneEntry); push(@pVals, $geneVals1[1]); push(@qVals, $geneVals1[3]); } else { push(@pVals, "NA"); push(@qVals, "NA"); } } splice(@F, 5, 0, (join("|", @pVals), join("|", @qVals))); print join("\t", @F);' | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | perl -lane 'splice(@F, 0, 1, (((split(/_/, $F[0]))[0]), ((split(/_/, $F[0]))[1]), ((split(/_/, $F[0]))[1]))); print join("\t", @F);' |
perl -ane 'print join("\t", @F[0..10]); my $alt = $F[10]; my $totAC = join("", ((split(/:/, $F[11]))[1])) + join("", ((split(/:/, $F[12]))[1])); print "\t", $totAC; for (my $i=11; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", ((split(/\:/, $F[$i]))[1]); } } print "\n";' | sort -g -k 1,1 -k 2,2 ) | /home/shared/software/tabix-0.2.5/bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz
perl -ane 'print join("\t", @F[0..10]); my $alt = $F[10]; my $totAC = join("", ((split(/:/, $F[11]))[1])) + join("", ((split(/:/, $F[12]))[1])); print "\t", $totAC; for (my $i=11; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", ((split(/\:/, $F[$i]))[1]); } } print "\n";' | sort -g -k 1,1 -k 2,2 ) | perl -lane 'splice(@F, 3, 0, ($F[9], $F[10])); splice(@F, 11, 2); print join("\t", @F);' | /home/shared/software/tabix-0.2.5/bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz

tabix -s 1 -b 2 -e 3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz

perl -e 'print
"key=INFO,ID=LOC,Number=1,Type=String,Description='\''Gene-oriented genomic location(s) of the variant (e.g. exonic, intronic, intergenic, etc...)'\''
key=INFO,ID=GENE,Number=1,Type=String,Description='\''Gene(s) associated with the variant'\''
key=INFO,ID=VARTYPE,Number=1,Type=String,Description='\''If exonic, type of coding-change variant'\''
key=INFO,ID=AACHANGE,Number=1,Type=String,Description='\''If coding-change variant, related protein- and transcript-change'\''
key=INFO,ID=PVAL,Number=1,Type=Integer,Description='\''p-value(s) associated with gene(s) in GENE field. NOTE -- p-value is gene-level based on using SKAT-O, only nonsynonymous variants within the specific gene and comparing seronegatives vs. seropositives. It is /NOT/ a variant-specific or variant-level p-value. NA indicates no p-value is available'\''
key=INFO,ID=QVAL,Number=1,Type=Integer,Description='\''q-value(s) associated with gene(s) in GENE field. Derived from the p-value distribution of all genes from the SKAT-O, nonsynonymous variants only, seronegative vs. seropositive analysis. It is /NOT/ a variant-specific or variant-level q-value. NA indicates no q-value is available'\''
key=INFO,ID=AC_ALL,Number=1,Type=Integer,Description='\''Total number of Ref and Alt allele copies (among white non-hispanics)'\''
key=INFO,ID=AC_TOTAL,Number=1,Type=Integer,Description='\''Total number of Alt allele copies (among white non-hispanics)'\''
key=INFO,ID=AC_SERONEG,Number=1,Type=Integer,Description='\''Seronegative number of Alt allele copies (among white non-hispanics)'\''
key=INFO,ID=AC_HESERONEG,Number=1,Type=Integer,Description='\''Highly-exposed seronegative number of Alt allele copies (among white non-hispanics)'\''
key=INFO,ID=AC_SEROPOS,Number=1,Type=Integer,Description='\''Seropositive number of Alt allele copies (among white non-hispanics)'\''
key=INFO,ID=AC_RAPID,Number=1,Type=Integer,Description='\''Rapid and very rapid AIDS progressor Alt allele copies (among white non-hispanics)'\''
key=INFO,ID=AC_SLOW,Number=1,Type=Integer,Description='\''Slow and very slow AIDS progressor Alt allele copies (among white non-hispanics)'\''\n"' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.PrepFor_vcfannotate.vs2.desc.txt

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | vcf-annotate -a /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz -d /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.PrepFor_vcfannotate.vs2.desc.txt -c CHROM,FROM,TO,INFO/LOC,INFO/GENE,INFO/VARTYPE,INFO/AACHANGE,INFO/PVAL,INFO/QVAL,REF,ALT,INFO/AC_ALL,INFO/AC_TOTAL,INFO/AC_SERONEG,INFO/AC_HESERONEG,INFO/AC_SEROPOS,INFO/AC_RAPID,INFO/AC_SLOW | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz




##20150422
##Working up code to run SKAT-O on msigdb pathways as a form of pathway analysis

~~~
[  michaelt@node2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.SUMSTAT.CalcPermBasedPVal.vs1.TruePheno.Pheno1.Exonic.Nonsynonymous.output | head -n 5 
KEGG_GLYCOLYSIS_GLUCONEOGENESIS,62,0,,ACSS2|GCK|PGK2|PGK1|PDHB|PDHA1|PDHA2|PGM2|TPI1|ACSS1|FBP1|ADH1B|HK2|ADH1C|HK1|HK3|ADH4|PGAM2|ADH5|PGAM1|ADH1A|ALDOC|ALDH7A1|LDHAL6B|PKLR|LDHAL6A|ENO1|PKM2|PFKP|BPGM|PCK2|PCK1|ALDH1B1|ALDH2|ALDH3A1|AKR1A1|FBP2|PFKM|PFKL|LDHC|GAPDH|ENO3|ENO2|PGAM4|ADH7|ADH6|LDHB|ALDH1A3|ALDH3B1|ALDH3B2|ALDH9A1|ALDH3A2|GALM|ALDOA|DLD|DLAT|ALDOB|G6PC2|LDHA|G6PC|PGM1|GPI,0,0,1000,0,0,0
KEGG_CITRATE_CYCLE_TCA_CYCLE,32,4,SDHD|IDH3A|SDHA|SUCLA2,IDH3B|DLST|PCK2|CS|PDHB|PCK1|PDHA1|LOC642502|PDHA2|LOC283398|FH|OGDH|SDHB|SDHC|IDH2|IDH1|ACO1|ACLY|MDH2|DLD|MDH1|DLAT|OGDHL|PC|SUCLG1|SUCLG2|IDH3G|ACO2,3.16139156955,4,70,6.18994224979,1.7571054227207,0.863115370908038
KEGG_PENTOSE_PHOSPHATE_PATHWAY,27,0,,RPE|RPIA|PGM2|PGLS|PRPS2|FBP2|PFKM|PFKL|TALDO1|TKT|FBP1|TKTL2|PGD|RBKS|ALDOA|ALDOC|ALDOB|H6PD|LOC729020|PRPS1L1|PRPS1|DERA|G6PD|PGM1|TKTL1|PFKP|GPI,0,0,1000,0,0,0
KEGG_PENTOSE_AND_GLUCURONATE_INTERCONVERSIONS,28,0,,UGT1A10|UGT1A8|RPE|UGT1A7|UGT1A6|UGT2B28|UGT1A5|CRYL1|UGDH|UGT2A1|GUSB|UGT1A9|DCXR|LOC729020|DHDH|UGT2B11|UGP2|XYLB|UGT2B10|AKR1B1|UGT2B7|UGT2B4|UGT2A3|UGT1A4|UGT2B17|UGT1A1|UGT1A3|UGT2B15,0,0,1000,0,0,0
KEGG_FRUCTOSE_AND_MANNOSE_METABOLISM,34,0,,MPI|PMM2|PMM1|FBP2|PFKM|GMDS|PFKFB4|PFKL|MTMR6|TPI1|PHPT1|PFKFB3|FUK|PFKFB2|MTMR1|PFKFB1|AKR1B10|FPGT|KHK|FBP1|MTMR2|HK2|HK3|HK1|ALDOA|ALDOC|ALDOB|MTMR7|TSTA3|AKR1B1|SORD|GMPPA|PFKP|GMPPB,0,0,1000,0,0,0
~~~

#SUMSTAT code from /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs2.sh 
~~~
...
	python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.py --file1 $mainDir1/$TrueFake/$Pheno/$Subset/${File1}.${Subset}.${Pheno}.maf1.skat0.perm${i}.epacts.rnk --file2 /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt > $mainDir1/$TrueFake/$Pheno/$Subset/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.${TrueFake}.${Subset}.${Pheno}.maf1.skat0.perm${i}.rnk.SUMSTAT

...

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.vs1.py --file1 $mainDir1/$TrueFake/$Pheno/$Subset/SUMSTAT/DataProcessing.Pipeline.pt6.All.ptg.HomeCodeSUMSTAT.Analysis.vs1.EPACTS.${TrueFake}.${Subset}.${Pheno}.maf1.skat0.permAll.rnk.SUMSTAT.temp > $mainDir1/$TrueFake/$Pheno/$Subset/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.vs1.${TrueFake}.${Pheno}.${Subset}.output

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.CalcPermBasedPVal.vs1.R  $mainDir1/TruePheno/$Pheno/$Subset/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.${Pheno}.${Subset}.output.noNAs.Routput.rnk.SUMSTAT $mainDir1/$TrueFake/$Pheno/$Subset/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.SUMSTAT.vs1.${TrueFake}.${Pheno}.${Subset}.output > $mainDir1/TruePheno/$Pheno/$Subset/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.SUMSTAT.CalcPermBasedPVal.vs1.TruePheno.${Pheno}.${Subset}.output
...
~~~

~~~
[  michaelt@node2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | head -n 10
PGAP2   11:3819108_G/C
HSPA6   1:161494743_G/T 1:161495003_T/G 1:161495040_C/T 1:161495148_G/A 1:161495275_T/C 1:161495463_G/A 1:161495497_A/G 1:161495757_C/T 1:161496010_T/G 1:161496039_G/C
AP1M1   19:16314427_G/A 19:16339645_A/G 19:16339672_C/T
P4HA2   5:131530706_C/T 5:131531112_G/A 5:131533954_C/T 5:131543430_C/T 5:131543541_G/A 5:131546032_G/C
SGPL1   10:72604246_A/G 10:72604263_G/T 10:72617424_A/G 10:72628170_G/C 10:72629578_T/A 10:72629610_A/G 10:72633127_G/T 10:72633216_C/T 10:72633291_G/A 10:72633336_C/T 10:72637078_C/T
SDF4    1:1163951_G/A   1:1164031_A/G   1:1164080_T/C
SDF2    17:26976071_A/G 17:26976108_T/C 17:26982321_G/A 17:26988852_C/T 17:26988885_C/G
ALMS1   2:73613108_G/A  2:73613176_C/A  2:73613177_G/C  2:73650009_C/A  2:73651920_C/G  2:73651937_A/G  2:73651967_C/T  2:73653610_G/A  2:73675110_A/G  2:73675263_C/G  2:73675492_G/A  2:73675525_A/G  2:73675630_C/T  2:73675669_T/G  2:73675850_C/G  2:73676131_T/C  2:73676547_T/C  2:73676623_A/G  2:73676730_G/C  2:73676797_G/A     2:73676961_C/G  2:73676965_G/A  2:73677033_C/T  2:73677066_A/G  2:73677466_C/A  2:73677553_C/T  2:73677808_C/G  2:73677853_A/G  2:73677876_G/A  2:73678360_C/T  2:73678540_G/A  2:73678616_G/T  2:73678641_A/G  2:73678642_C/T  2:73678843_A/G  2:73679013_A/G  2:73679107_G/T  2:73679280_A/G  2:73679440_G/A     2:73679617_A/G  2:73679687_T/G  2:73679866_T/C  2:73679990_T/A  2:73680118_A/G  2:73680207_C/T  2:73680226_A/G  2:73680508_G/C  2:73680580_C/T  2:73680922_A/G  2:73680942_A/C  2:73681090_T/C  2:73716810_G/A  2:73717103_G/C  2:73717169_G/A  2:73717343_C/A  2:73717497_G/A  2:73717532_G/C  2:73717567_G/T  2:73717656_A/G     2:73717921_C/G  2:73718012_G/A  2:73718546_A/T  2:73746979_C/T  2:73747021_T/C  2:73747074_C/T  2:73777400_A/G  2:73786188_A/G  2:73799632_C/G  2:73800058_G/A  2:73800109_G/A  2:73800259_A/G  2:73800264_C/G  2:73800270_G/A  2:73800357_A/G  2:73800396_A/G  2:73800496_T/G  2:73826621_C/T  2:73827844_G/A     2:73828443_G/C  2:73828453_C/T  2:73828538_G/A
TCEB2   16:2821525_G/A  16:2821573_C/T  16:2821589_G/T
ABCD3   1:94930338_G/T  1:94939335_C/T  1:94946030_G/C  1:94948746_A/G  1:94955337_A/G
[  michaelt@node2  ~]$cat /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt | head -n 10                                                                                                                                                                                                                                    
KEGG_GLYCOLYSIS_GLUCONEOGENESIS http://www.broadinstitute.org/gsea/msigdb/cards/KEGG_GLYCOLYSIS_GLUCONEOGENESIS ACSS2   GCK     PGK2    PGK1    PDHB    PDHA1   PDHA2   PGM2    TPI1    ACSS1   FBP1    ADH1B   HK2     ADH1C   HK1     HK3     ADH4    PGAM2   ADH5    PGAM1   ADH1A   ALDOC   ALDH7A1 LDHAL6B PKLR    LDHAL6A    ENO1    PKM2    PFKP    BPGM    PCK2    PCK1    ALDH1B1 ALDH2   ALDH3A1 AKR1A1  FBP2    PFKM    PFKL    LDHC    GAPDH   ENO3    ENO2    PGAM4   ADH7    ADH6    LDHB    ALDH1A3 ALDH3B1 ALDH3B2 ALDH9A1 ALDH3A2 GALM    ALDOA   DLD     DLAT    ALDOB   G6PC2   LDHA    G6PC    PGM1    GPI
KEGG_CITRATE_CYCLE_TCA_CYCLE    http://www.broadinstitute.org/gsea/msigdb/cards/KEGG_CITRATE_CYCLE_TCA_CYCLE    IDH3B   DLST    PCK2    CS      PDHB    PCK1    PDHA1   LOC642502       PDHA2   LOC283398       FH      SDHD    OGDH    SDHB    IDH3A   SDHC    IDH2    IDH1    ACO1    ACLY    MDH2    DLD     MDH1    DLAT       OGDHL   PC      SDHA    SUCLG1  SUCLA2  SUCLG2  IDH3G   ACO2
KEGG_PENTOSE_PHOSPHATE_PATHWAY  http://www.broadinstitute.org/gsea/msigdb/cards/KEGG_PENTOSE_PHOSPHATE_PATHWAY  RPE     RPIA    PGM2    PGLS    PRPS2   FBP2    PFKM    PFKL    TALDO1  TKT     FBP1    TKTL2   PGD     RBKS    ALDOA   ALDOC   ALDOB   H6PD    LOC729020       PRPS1L1 PRPS1   DERA    G6PD    PGM1    TKTL1      PFKP    GPI
KEGG_PENTOSE_AND_GLUCURONATE_INTERCONVERSIONS   http://www.broadinstitute.org/gsea/msigdb/cards/KEGG_PENTOSE_AND_GLUCURONATE_INTERCONVERSIONS   UGT1A10 UGT1A8  RPE     UGT1A7  UGT1A6  UGT2B28 UGT1A5  CRYL1   UGDH    UGT2A1  GUSB    UGT1A9  DCXR    LOC729020       DHDH    UGT2B11 UGP2    XYLB    UGT2B10 AKR1B1  UGT2B7     UGT2B4  UGT2A3  UGT1A4  UGT2B17 UGT1A1  UGT1A3  UGT2B15
KEGG_FRUCTOSE_AND_MANNOSE_METABOLISM    http://www.broadinstitute.org/gsea/msigdb/cards/KEGG_FRUCTOSE_AND_MANNOSE_METABOLISM    MPI     PMM2    PMM1    FBP2    PFKM    GMDS    PFKFB4  PFKL    MTMR6   TPI1    PHPT1   PFKFB3  FUK     PFKFB2  MTMR1   PFKFB1  AKR1B10 FPGT    KHK     FBP1    MTMR2   HK2     HK3     HK1
ALDOA   ALDOC   ALDOB   MTMR7   TSTA3   AKR1B1  SORD    GMPPA   PFKP    GMPPB
KEGG_GALACTOSE_METABOLISM       http://www.broadinstitute.org/gsea/msigdb/cards/KEGG_GALACTOSE_METABOLISM       GCK     GALK1   GLB1    GALE    B4GALT1 PGM2    LALBA   PFKM    PFKL    MGAM    HK2     HK1     HK3     GALT    G6PC2   GLA     GANC    LCT     GALK2   G6PC    UGP2    PGM1    AKR1B1  B4GALT2 GAA     PFKP
KEGG_ASCORBATE_AND_ALDARATE_METABOLISM  http://www.broadinstitute.org/gsea/msigdb/cards/KEGG_ASCORBATE_AND_ALDARATE_METABOLISM  UGT1A10 UGT1A8  UGT1A7  UGT1A6  ALDH1B1 UGT2B28 ALDH2   UGT1A5  MIOX    UGDH    UGT2A1  ALDH9A1 ALDH3A2 UGT1A9  ALDH7A1 UGT2B11 UGT2B10 UGT2B7  UGT2B4  UGT2A3  UGT1A4  UGT1A1  UGT2B17 UGT1A3     UGT2B15
KEGG_FATTY_ACID_METABOLISM      http://www.broadinstitute.org/gsea/msigdb/cards/KEGG_FATTY_ACID_METABOLISM      CPT1A   CPT1C   ACADS   ALDH1B1 ACADSB  ACADL   ALDH2   ACADM   CYP4A11 ACAT2   ACADVL  ACAT1   ACAA2   HADH    HADHB   HADHA   CYP4A22 ADH7    ADH6    ACSL6   ADH1B   ADH1C   ADH4    ECHS1   ADH5    ALDH9A1    ALDH3A2 ACSL5   ADH1A   EHHADH  GCDH    ALDH7A1 ACOX3   ACSL1   ACAA1   CPT2    CPT1B   ACOX1   ECI2    ECI1    ACSL3   ACSL4
KEGG_STEROID_BIOSYNTHESIS       http://www.broadinstitute.org/gsea/msigdb/cards/KEGG_STEROID_BIOSYNTHESIS       SOAT1   LSS     SQLE    EBP     CYP51A1 DHCR7   CYP27B1 DHCR24  HSD17B7 MSMO1   FDFT1   SC5DL   LIPA    CEL     TM7SF2  NSDHL   SOAT2
KEGG_PRIMARY_BILE_ACID_BIOSYNTHESIS     http://www.broadinstitute.org/gsea/msigdb/cards/KEGG_PRIMARY_BILE_ACID_BIOSYNTHESIS     CYP46A1 SLC27A5 BAAT    CYP7B1  AKR1C4  HSD17B4 SCP2    AKR1D1  ACOX2   HSD3B7  CYP27A1 AMACR   CYP7A1  CYP8B1  CYP39A1 CH25H
~~~

cat /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt | perl -ane 'print $F[0], "\t"; my @genes; my $geneCount = 0; my @vars; my @varsPerGene; for (my $i = 2; $i <= $#F; $i++) {my $geneGrep = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -P "^$F[$i]\\s"`; if ($geneGrep) { my @geneGrepArray = split(/\s+/, $geneGrep); push(@genes, $F[$i]); $geneCount++; push(@varsPerGene, $#geneGrepArray); push(@vars, @geneGrepArray[1..$#geneGrepArray]); } } if ($geneCount == 0) { print $geneCount, "\t.\t.\t.\t.\n"; } else { print $geneCount, "\t", scalar(@vars), "\t", join(",", @varsPerGene), "\t", join(",", @genes), "\t", join("\t", @vars), "\n"; }' > /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt.MM5.GroupFile.Exonic.Nonsynonymous.wSplice.pre2

cat /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt.MM5.GroupFile.Exonic.Nonsynonymous.wSplice.pre | perl -lane 'if ($F[1] >= 2) { print $F[0], "\t", join("\t", @F[5..$#F]); }' > /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt.MM5.GroupFile.Exonic.Nonsynonymous.wSplice 


mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/msigdb_c2/
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/msigdb_c2/Permutations/
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/msigdb_c2/Permutations/TruePheno/
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/msigdb_c2/Permutations/TruePheno/Pheno1/
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/msigdb_c2/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/msigdb_c2/Permutations/TruePheno/Pheno2/
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/msigdb_c2/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/msigdb_c2/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt.MM5.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.msigdb_c2

#CHECK_0: Do 100 FakePheno/Simulations to see the p-value distribution from them? Just to see if there's more 'power' in these pathway analyses or if their p-values are not that distinguishable from random chance?

MM5TopPathways=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.msigdb_c2.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | sort -rg -k 10,10 | tail -n 11 | head -n 10 | awk '{ print $4 }' | perl -ane 'my @pathwayArray = split(/_/, $F[0]); if (eof()) { print join("_", @pathwayArray[1..$#pathwayArray]), "\n"; } else { print join("_", @pathwayArray[1..$#pathwayArray]), ";"; }'`

#rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.Vs3.EPACTS.RMID5PCs.msigdb_c2.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneLevel.AllPathways

OLD_IFS=$IFS
IFS=$';'

for MM5Pathway in $MM5TopPathways
do
	IFS=$OLD_IFS
	echo $MM5Pathway

#	rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.Vs3.EPACTS.RMID5PCs.msigdb_c2.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneLevel.$MM5Pathway

	echo $MM5Pathway `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.msigdb_c2.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep $MM5Pathway | awk '{ print $10 }'` >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.Vs3.EPACTS.RMID5PCs.msigdb_c2.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneLevel.AllPathways

#	local GenesInPathway=`cat /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt.MM5.GroupFile.Exonic.Nonsynonymous.wSplice.pre | grep -w $MM5Pathway | awk '{ print $4 }'`
	GenesInPathway=`cat /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt.MM5.GroupFile.Exonic.Nonsynonymous.wSplice.pre | grep -w $MM5Pathway | awk '{ print $5 }' | perl -lane 'print join(";", split(/,/, $F[0]));'`

	OLD_IFS=$IFS
	IFS=$';'

	for GenePathway in $GenesInPathway
	do
		IFS=$OLD_IFS

		echo $GenePathway

#		cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.ForSudhir.txt | grep -w -P "^$GenePathway\\s" >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.Vs3.EPACTS.RMID5PCs.msigdb_c2.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneLevel.$MM5Pathway
		cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.ForSudhir.txt | grep -w -P "^$GenePathway\\s" >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.Vs3.EPACTS.RMID5PCs.msigdb_c2.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneLevel.AllPathways

	done

	echo "" >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.Vs3.EPACTS.RMID5PCs.msigdb_c2.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneLevel.AllPathways

done

~~~
[  michaelt@node2  ~/Data/GSEA]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.Vs3.EPACTS.RMID5PCs.msigdb_c2.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneLevel.* | awk '{ print $1 }' | sort | uniq -c | sort -k 1,1 | tail -n 25
1 TBL1XR1
1 UNG
1 VDAC1
1 WDR77
1 XPO4
1 ZFP91
2 AP1M1
2 BTRC
2 CDK9
2 CUL5
2 GEMIN2
2 HLA-A
2 KPNB1
2 NMT1
2 PSMD7
2 RANBP1
2 TCEB2
3 CCNT1
3 PSIP1
3 PSMB6
3 XPO1
4 SEH1L
5 NUP153
5 RANBP2
5 TPR
~~~

#Specific pathway/gene-combination considerations
#An * indicates considering running a group SKAT-O analysis on using these genes

~~~
WANG_CISPLATIN_RESPONSE_AND_XPC_DN
MCM7    0.3491130000    0.63739 13      1       973     454     216     519     60      103
CNTNAP1 0.6132043000    0.69767 10      1       18      10      6       8       0       4
PARP4   0.1275589000    0.52173 40      1       6517    2981    1430    3536    467     745
NMT1    0.8658447000    0.69812 2       1       3       2       0       1       0       0
EDC4    0.0090346330    0.29264 7       0.5     18      14      7       4       0       1
SDCBP   0.0914164000    0.48019 3       0.5     5       4       4       1       0       1
GEMIN2  0.0007225926    0.19813 4       0.09    7       7       4       0       0       0
RNH1    0.7232442000    0.69767 14      1       324     136     64      188     24      46

NMT1    0.8658447000    0.69812 2       1       3       2       0       1       0       0
EDC4    0.0090346330    0.29264 7       0.5     18      14      7       4       0       1
SDCBP   0.0914164000    0.48019 3       0.5     5       4       4       1       0       1
GEMIN2  0.0007225926    0.19813 4       0.09    7       7       4       0       0       0
~~~

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/msigdb_c2/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt.MM5.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.msigdb_c2


#DOINGTHISAGAIN

/data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs2.sh
/data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs2.sh





##20150425
##Working up code to run SKAT-O on Kevin O's code/groupings as a form of pathway analysis

mkdir /home/michaelt/Data/HGNC

#PWD -- /home/michaelt/Data/HGNC
wget ftp://ftp.ebi.ac.uk/pub/databases/genenames/new/tsv/locus_groups/protein-coding_gene.txt

#Custom columns selected from http://www.genenames.org/cgi-bin/download and copy/pasted into protein-coding_gene.GeneSymbol_AltSymbols_EntrezID.txt, specifically "Approved Symbol	Previous Symbols	Entrez Gene ID(supplied by NCBI)"

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | awk '{ print $1 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice.GeneIDs

~~~
[  michaelt@node2  ~/Data/HGNC]$cat /home/michaelt/Data/HGNC/protein-coding_gene.GeneSymbol_AltSymbols_EntrezID.txt | grep -w -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice.GeneIDs | wc^C
[  michaelt@node2  ~/Data/HGNC]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice.GeneIDs | wc
    473     473    2992
[  michaelt@node2  ~/Data/HGNC]$cat /home/michaelt/Data/HGNC/protein-coding_gene.GeneSymbol_AltSymbols_EntrezID.txt | grep -w -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice.GeneIDs | wc
    471    1140    7115
[  michaelt@node2  ~/Data/HGNC]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice.GeneIDs | grep -w -v -f <(cat /home/michaelt/Data/HGNC/protein-coding_gene.GeneSymbol_AltSymbols_EntrezID.GroupFile.Exonic.Nonsynonymous.wSplice | awk '{ print $1 }' )
PTPLAD1
DAK
~~~

#CHECK_0: See if PTPLAD1 and DAK have alternative gene names or entrez id#s that would work for this?

cat /home/michaelt/Data/HGNC/protein-coding_gene.GeneSymbol_AltSymbols_EntrezID.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'print $F[0], "\t", $F[$#F];' > /home/michaelt/Data/HGNC/protein-coding_gene.GeneSymbol__EntrezID.GroupFile.Exonic.Nonsynonymous.wSplice

#Original "array gene groupings 03 31 2014.xlsx" file taken from 20140314 e-mail with the below info
#email from: Kevin Olivieri <kolivieri@sanfordburnham.org>
#email subj: "Gene groupings for HiNT data sets" 
#Edited on 20150427 to reformat it for use here; file was transferred via scp -p
#Note -- the 'Complexes from Array 1' filesets were precursors to this fileset. The later 'HINT db...' filesets are for the full list of genes present across all arrays (?), but at least are clearly more than just the first set of genes from array 1, which these files mainly represent
#Replaced all commas in file with semicolons
#PWD -- /home/michaelt/LabStuff/StephensLab/HIVStuff/PG

cd /home/michaelt/LabStuff/StephensLab/HIVStuff/PG
scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/20150427Array1KevinGeneGroupings.array_gene_groupings_03_31_2014.semicolons.csv .

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20150427Array1KevinGeneGroupings.array_gene_groupings_03_31_2014.semicolons.csv | perl -F, -lane 'for (my $i = 0; $i <= $#F; $i++) { $F[$i] = join("_", split(/\s+/, $F[$i])); } print join(",", @F); ' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20150427Array1KevinGeneGroupings.array_gene_groupings_03_31_2014.semicolons.SpaceToUnderscores.csv

~~~
[  michaelt@node2  ~/Data/HGNC]$cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20150427Array1KevinGeneGroupings.array_gene_groupings_03_31_2014.semicolons.SpaceToUnderscores.csv | perl -F, -lane 'print $#F;' | sort | uniq -c
    585 29
~~~

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20150427Array1KevinGeneGroupings.array_gene_groupings_03_31_2014.semicolons.SpaceToUnderscores.csv | perl -F, -lane 'my $geneSymbol =`cat /home/michaelt/Data/HGNC/protein-coding_gene.GeneSymbol__EntrezID.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w $F[0]`; if ($geneSymbol) { push(@F, split(/\s+/, $geneSymbol)) } else { push(@F, ("NA", "NA"))}; print join(",", @F);' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20150427Array1KevinGeneGroupings.array_gene_groupings_03_31_2014.semicolons.SpaceToUnderscores.withHGNCdata.csv

~~~
[  michaelt@node2  ~/Data/HGNC]$cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20150427Array1KevinGeneGroupings.array_gene_groupings_03_31_2014.semicolons.SpaceToUnderscores.withHGNCdata.csv | perl -F, -lane 'print $#F;' | sort | uniq -c
    585 31
~~~

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20150427Array1KevinGeneGroupings.array_gene_groupings_03_31_2014.semicolons.SpaceToUnderscores.withHGNCdata.csv | perl -F, -lane 'if($bindingPartner{$F[1]}) { push(@{$bindingPartner{$F[1]}}, $F[$#F-1]); } else {$bindingPartner{$F[1]} = [($F[$#F-1])]; }if (eof()) { foreach my $partner (keys %bindingPartner) { print $partner, "\t", join(",", @{$bindingPartner{$partner}}); }}' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20150427Array1KevinGeneGroupings.array_gene_groupings_03_31_2014.semicolons.SpaceToUnderscores.withHGNCdata.GroupFile.Exonic.Nonsynonymous.wSplice.pre

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20150427Array1KevinGeneGroupings.array_gene_groupings_03_31_2014.semicolons.SpaceToUnderscores.withHGNCdata.GroupFile.Exonic.Nonsynonymous.wSplice.pre | perl -lane 'my @vars; if ($F[1]) { my @genes = split(/,/, $F[1]); for (my $i = 0; $i <= $#genes; $i++) { if ($genes[$i] ne "NA") {my $geneGrep = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -P "^$genes[$i]\\s"`; if ($geneGrep) { my @geneGrepArray = split(/\s+/, $geneGrep); push(@vars, @geneGrepArray[1..$#geneGrepArray]); } else { print STDERR "Error1a -- Gene $genes[$i] not located in file";}}} print $F[0], "\t", join("\t", @vars);}' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20150427Array1KevinGeneGroupings.array_gene_groupings_03_31_2014.semicolons.SpaceToUnderscores.withHGNCdata.GroupFile.Exonic.Nonsynonymous.wSplice

#HACD3 not located in results file/original GroupFile
~~~
[  michaelt@node2  ~/Data/HGNC]$cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20150427Array1KevinGeneGroupings.arr...GroupFile.Exonic.Nonsynonymous.wSplice
Error1a -- Gene HACD3 not located in file
~~~

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/Permutations
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/Permutations/TruePheno
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/Permutations/TruePheno/Pheno1
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/Permutations/TruePheno/Pheno2
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/Permutations /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20150427Array1KevinGeneGroupings.array_gene_groupings_03_31_2014.semicolons.SpaceToUnderscores.withHGNCdata.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/Permutations/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Array1Groupings


REVgenes=`cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20150427Array1KevinGeneGroupings.array_gene_groupings_03_31_2014.semicolons.SpaceToUnderscores.withHGNCdata.GroupFile.Exonic.Nonsynonymous.wSplice.pre | grep REV | perl -lane 'print join(";", split(",", $F[1]));'`

OLD_IFS=$IFS
IFS=$';'

for revGene in $REVgenes
do
	IFS=$OLD_IFS

	echo $revGene

	cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.ForSudhir.txt | grep -w -P "^$revGene\\s" >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/Permutations/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Array1Groupings.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneLevel.REV

done
	
~~~
[  michaelt@node2  ~/Data/HGNC]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/Permutations/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Array1Groupings.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneLevel.REV
IK      0.2533898000    0.57925 4       1       27      9       3       18      0       2
TMED10  0.0563339200    0.43885 2       1       4       0       0       4       0       0
KIF21B  0.2219503000    0.55075 12      0.09    55      29      18      26      1       4
MRPS28  0.5392356000    0.69767 6       0       38      17      5       21      2       9
MRPL42  1.0000000000    0.73970 2       1       7       3       3       4       2       0
TMED9   0.0652776600    0.44009 3       0       61      21      11      40      4       9
GPATCH4 0.0160863200    0.29264 4       1       6       0       0       6       1       3
SMU1    0.8416136000    0.69767 2       1       3       2       0       1       0       0
MRS2    0.0784893900    0.45769 14      0.25    373     180     78      193     31      41
MRPS9   0.0093243610    0.29264 9       0.04    46      27      8       19      2       6
SPATS2  0.5272844000    0.69767 12      0       39      20      12      19      2       3
CHCHD1  0.7253000000    0.69767 2       0       2       1       0       1       0       0
MITD1   0.7665800000    0.69767 1       NA      1       0       0       1       0       0
TMED4   0.8206400000    0.69767 1       NA      2       1       0       1       0       1
~~~

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20150427Array1KevinGeneGroupings.array_gene_groupings_03_31_2014.semicolons.SpaceToUnderscores.withHGNCdata.csv | perl -F, -lane 'for (my $i = 1; $i <= $#F-2; $i++) { if ($F[0] =~ m/Array/) { $titles1{$i} = $F[$i]; } else { if ($F[$i] ne "0") { $pathwayKey1 = $titles1{$i} . "_" . $F[$i]; if ($hash1{$pathwayKey1}) { push(@{$hash1{$pathwayKey1}}, $F[$#F-1]); } else { $hash1{$pathwayKey1} = [($F[$#F-1])]; } } } } if (eof()) { foreach my $pathwayKey2 (keys %hash1) {print $pathwayKey2, "\t", join(",", @{$hash1{$pathwayKey2}}); }}' | grep -v Binding_Partner | perl -lane 'if ($F[0] =~ m/Functional_Data/) { my @data1 = split(/,/, $F[1]); my @uniqueData1 = do { my %seen; grep { !$seen{$_}++ } @data1 }; $F[1] = join(",", @uniqueData1); } print join("\t", @F); ' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20150427Array1KevinGeneGroupings.array_gene_groupings_03_31_2014.semicolons.SpaceToUnderscores.withHGNCdata.SplitIntoCategories.GroupFile.Exonic.Nonsynonymous.wSplice.pre

~~~
my @unique = do { my %seen; grep { !$seen{$_}++ } @data };

http://perlmaven.com/unique-values-in-an-array-in-perl

Shortening the home made unique function
First of all we replace the assignment of 1 $seen{$value} = 1; by the post-increment operator $seen{$value}++. This does not change the behavior of the previous solution - any positive number is going to be evaluated as TRUE, but it will allow us to include the setting of the "seen flag" within the if condition. It is important that this is a postfix increment (and not a prefix increment) as this means the increment only takes place after the boolean expression was evaluated. The first time we encounter a value the expression will be TRUE and the rest of the times it will be FALSE.

my @unique;
my %seen;

foreach my $value (@data) {
	if (! $seen{$value}++ ) {
		push @unique, $value;
	}
}
This is shorter, but we can do even better.


Filtering duplicate values using grep
The grep function in Perl is a generalized form of the well known grep command of Unix.

It is basically a filter. You provide an array on the right hand side and an expression in the block. The grep function will take each value of the array one-by-one, put it in $_, the default scalar variable of Perl and then execute the block. If the block evaluates to TRUE, the value can pass. If the block evaluates to FALSE the current value is filtered out.

That's how we got to this expression:
my %seen;
my @unique = grep { !$seen{$_}++ } @words;
~~~

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20150427Array1KevinGeneGroupings.array_gene_groupings_03_31_2014.semicolons.SpaceToUnderscores.withHGNCdata.SplitIntoCategories.GroupFile.Exonic.Nonsynonymous.wSplice.pre | perl -lane 'my @vars; if ($F[1]) { my @genes = split(/,/, $F[1]); for (my $i = 0; $i <= $#genes; $i++) { if ($genes[$i] ne "NA") {my $geneGrep = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -P "^$genes[$i]\\s"`; if ($geneGrep) { my @geneGrepArray = split(/\s+/, $geneGrep); push(@vars, @geneGrepArray[1..$#geneGrepArray]); } else { print STDERR "Error1a -- Gene $genes[$i] not located in file";}}} print $F[0], "\t", join("\t", @vars);}' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20150427Array1KevinGeneGroupings.array_gene_groupings_03_31_2014.semicolons.SpaceToUnderscores.withHGNCdata.SplitIntoCategories.GroupFile.Exonic.Nonsynonymous.wSplice

#HACD3 & TKFC not located in results file/original GroupFile
~~~
[  michaelt@node2  ~/Data/HGNC]$cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20150427Array1KevinGeneGroupings.arr...GroupFile.Exonic.Nonsynonymous.wSplice
Error1a -- Gene TKFC not located in file
Error1a -- Gene HACD3 not located in file                                                                                                                                                                                                                                                                                  
Error1a -- Gene HACD3 not located in file
Error1a -- Gene HACD3 not located in file                                                                                                                                                                                                                                                                                  
Error1a -- Gene TKFC not located in file
Error1a -- Gene HACD3 not located in file                                                                                                                                                                                                                                                                                  
Error1a -- Gene TKFC not located in file
Error1a -- Gene TKFC not located in file                                                                                                                                                                                                                                                                                   
Error1a -- Gene HACD3 not located in file
Error1a -- Gene HACD3 not located in file                                    
~~~

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/Permutations /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20150427Array1KevinGeneGroupings.array_gene_groupings_03_31_2014.semicolons.SpaceToUnderscores.withHGNCdata.SplitIntoCategories.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/Permutations/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.20150427Array1KevinGeneGroupings

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/Permutations /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20150427Array1KevinGeneGroupings.array_gene_groupings_03_31_2014.semicolons.SpaceToUnderscores.withHGNCdata.SplitIntoCategories.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/Permutations/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.20150427Array1KevinGeneGroupings




#DOINGTHISAGAIN















#Original "HINT db  05 14 2014.xlsx" file taken from 20140514 e-mail with the below info
#email from: Kevin Olivieri <kolivieri@sanfordburnham.org>
#mail subj: "Gene groupings/pathways"
#Replaced all commas in file with semicolons
#PWD -- /home/michaelt/LabStuff/StephensLab/HIVStuff/PG

cd /home/michaelt/LabStuff/StephensLab/HIVStuff/PG
scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/20150427AllGenesKevinGeneGroupings.HINT_db_05_14_2014.semicolons.csv .

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20150427AllGenesKevinGeneGroupings.HINT_db_05_14_2014.semicolons.csv | perl -F, -lane 'for (my $i = 0; $i <= $#F; $i++) { $F[$i] = join("_", split(/\s+/, $F[$i])); } print join(",", @F); ' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20150427AllGenesKevinGeneGroupings.HINT_db_05_14_2014.semicolons.SpaceToUnderscores.csv

cp -p /home/michaelt/Scripts/Python/2fileTemplate.vs1.py /home/michaelt/Scripts/Python/Basic2FileVLookUp.vs1.py

/home/michaelt/Scripts/Python/Basic2FileVLookUp.vs1.py --file1 /home/michaelt/Data/HGNC/protein-coding_gene.txt 




##20150509
##Trying to develop code to perform a McDonald-Kreitman test per gene
##Need to figure out how to identify 'fixed differenes' in the human lineage vs. chimpanzee since a 'fixed difference' across all individuals in my dataset, while still different from chimpanzee, won't show up in the .vcf file

pwd: /home/michaelt/Data/UCSC/Chimpanzee/panTro4
wget -i ftp://hgdownload.soe.ucsc.edu/goldenPath/panTro4/vsHg19/panTro4.hg19.*

pwd: /home/michaelt/Data/UCSC/Human/hg19
wget -i ftp://hgdownload.soe.ucsc.edu/goldenPath/hg19/vsPanTro4/hg19.panTro4.*
wget -i ftp://hgdownload.soe.ucsc.edu/goldenPath/hg19/vsPanTro4/axtNet/*.hg19.panTro4.net.axt.gz

zcat /home/michaelt/Data/UCSC/Human/hg19/chr*.hg19.panTro4.net.axt.gz | perl -lane 'if ($F[1] =~ m/chr(\d+)/) { $alignFlag1 = 1; $chr = $1; $start = $F[2]; } else { if ($alignFlag1 == 1) { @humanSeq = split(//, $F[0]); $alignFlag1++; } elsif ($alignFlag1 == 2) { @chimpSeq = split(//, $F[0]); for (my $i = 0; $i <= $#humanSeq; $i++) { if (lc($humanSeq[$i]) ne lc($chimpSeq[$i])) { print $chr, "\t", $start, "\t", $humanSeq[$i], "\t", $chimpSeq[$i]; } $start++; } $alignFlag1 = 0; $start = NULL; @humanSeq = NULL; @chimpSeq = NULL; } elsif ($alignFlag1 == 0) { $PH = 1; } else { print STDERR "\$alignFlag1 neither 0, 1 or 2 ($alignFlag1)"; }}' | gzip > /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.txt.gz

~~~
[  michaelt@login2  ~/Data/UCSC/Human/hg19]$zcat chrAll.hg19.panTro4.net.axt.AlignmentDifferences.txt.gz | wc
52395838 209583352 818377004
~~~

zcat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.txt.gz | perl -lane 'if (($F[2] ne "-") && ($F[3] ne "-")) { print $F[0], "\t", $F[1], "\t", $F[1], "\t", uc($F[2]), "\t", uc($F[3]); }' > /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.DropGaps.AnnovarFormat

~~~
[  michaelt@login2  ~/Data/UCSC/Human/hg19]$cat chrAll.hg19.panTro4.net.axt.AlignmentDifferences.DropGaps.AnnovarFormat | wc
35601707 178008535 883839858
~~~

perl /home/michaelt/Software/annovar/summarize_annovar.pl /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.DropGaps.AnnovarFormat /home/michaelt/Software/annovar/ --ver1000g 1000g2012apr --verdbsnp 129 --veresp 6500 --buildver hg19 --remove

~~~
[  michaelt@login2  ~/Data/UCSC/Human/hg19]$perl /home/michaelt/Software/annovar/summarize_annovar.pl /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.DropGaps.AnnovarFormat /home/michaelt/Software/annovar/ --ver1000g 1000g2012apr --verdbsnp 129 --veresp 6500 --buildver hg19 --remove
.
.
.
---------------------------------------------------------------------------------------
WARNING: 26348 exonic SNPs have WRONG reference alleles specified in your input file!
WARNING: An example input line is <10   122618160       122618160       T       C>
WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!
WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!
---------------------------------------------------------------------------------------
.
.
.
[  michaelt@node4  ~]$cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.DropGaps.AnnovarFormat.genome_summary.csv | wc
35512786 35776031 3075906858
[  michaelt@node4  ~]$cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.DropGaps.AnnovarFormat.genome_summary.csv | grep exonic | grep -v ncRNA | wc
270084  533329 37649749
[  michaelt@node4  ~]$cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.DropGaps.AnnovarFormat.genome_summary.csv | perl -F, -lane 'print $F[0];' | sort | uniq -c
250837 	"UTR3"
 53611 	"UTR5"
    75 	"UTR5;UTR3"
226309 	"downstream"
270024 	"exonic"
    60 	"exonic;splicing"
20583362 "intergenic"
12525407 "intronic"
  4787 	"ncRNA_UTR3"
  1132 	"ncRNA_UTR5"
     4 	"ncRNA_UTR5;ncRNA_UTR3"
100050 	"ncRNA_exonic"
1264509 "ncRNA_intronic"
   509 	"ncRNA_splicing"
  5410 	"splicing"
218832 	"upstream"
  7867 	"upstream;downstream"
     1 	Func
~~~

cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.DropGaps.AnnovarFormat.genome_summary.csv | grep -E 'exonic|splicing' | grep -v ncRNA > /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.DropGaps.AnnovarFormat.genome_summary.Exonic.csv

~~~
[  michaelt@node4  ~]$cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.DropGaps.AnnovarFormat.genome_summary.Exonic.csv | perl -F, -lane 'print $F[0];' | sort | uniq -c
 270024 "exonic"
    60 	"exonic;splicing"
  5410 	"splicing"

[  michaelt@node4  ~]$cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.DropGaps.AnnovarFormat.genome_summary.Exonic.csv | perl -F, -lane 'print $F[$#F-23];' | sort | uniq -c
5548
143896 "nonsynonymous SNV"
6765 "stopgain SNV"
590 "stoploss SNV"
111994 "synonymous SNV"
6701 "unknown"
[  michaelt@node4  ~]$cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.DropGaps.AnnovarFormat.genome_summary.Exonic.csv | perl -F, -lane 'print $#F' | sort | uniq -c
266274 25
5829 26
1104 27
309 28
192 29
855 30
181 31
707 32
8 33
10 34
5 35
5 36
2 37
1 38
6 39
1 40
1 41
1 44
2 46
1 47
[  michaelt@node4  ~]$cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.DropGaps.AnnovarFormat.genome_summary.Exonic.csv | perl -F, -lane 'if ($#F == 47) { print join(",", @F); }'
"splicing","CACNA1C(NM_001129840:exon3:c.372-1G>A,NM_001129833:exon3:c.372-1G>A,NM_001129827:exon3:c.372-1G>A,NM_001129839:exon3:c.372-1G>A,NM_001129830:exon3:c.372-1G>A,NM_001129832:exon3:c.372-1G>A,NM_001129831:exon3:c.372-1G>A,NM_001129829:exon3:c.372-1G>A,NM_001129844:exon3:c.372-1G>A,NM_001129835:exon3:c.372-1G>A,NM_001129846:exon3:c.372-1G>A,NM_199460:exon3:c.372-1G>A,NM_001167624:exon3:c.372-1G>A,NM_001129834:exon3:c.372-1G>A,NM_001129841:exon3:c.372-1G>A,NM_001129843:exon3:c.372-1G>A,NM_001129836:exon3:c.372-1G>A,NM_000719:exon3:c.372-1G>A,NM_001129842:exon3:c.372-1G>A,NM_001129838:exon3:c.372-1G>A,NM_001129837:exon3:c.372-1G>A,NM_001167623:exon3:c.372-1G>A,NM_001167625:exon3:c.372-1G>A)",,,"710;Name=lod=1028",,,,,,,,,,,,,,,,,12,2229490,2229490,G,A
[  michaelt@node2  ~]$cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.DropGaps.AnnovarFormat.genome_summary.Exonic.csv | perl -F, -lane 'print $F[0], "|", join(",", @F[1..$#F-24]), "|", join("|", @F[$#F-23..$#F]);' | perl -F'\|' -lane 'print $#F;' | sort | uniq -c
 275494 25
~~~

cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.DropGaps.AnnovarFormat.genome_summary.Exonic.csv | perl -F, -lane 'print $F[0], "|", join(",", @F[1..$#F-24]), "|", join("|", @F[$#F-23..$#F]);' | perl -F'\|' -lane 'print join("|", @F[0..2]), "|", join("|", @F[$#F-4..$#F]);' > /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top35 | tail -n 5 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top5

#Deciding to drop splicing variants since they are 'predicted'? A 'exonic;splicing' variant includes both the nonsynonymous/synonymous call based on just the variant's exonic location as well as the potential splicing change, so the nonsyn/syn information is there. A 'splicing' variant is on the intronic side of the junction so there's no hard nonsyn/syn call there
#Also deciding to change 'stopgain' or 'stoploss' to 'nonsynonymous' for the purposes of this analysis?
#cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep $gene1 | perl -F'\|' -lane 'if (($F[0] =~ m/\"splicing\"/) || ($F[2] =~ m/stop/)) { $F[2] = "\"nonsynonymous SNV\""; } print $F[2];' | sort | uniq -c

for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top5`
do

#	echo $gene1
	cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep $gene1 | perl -F'\|' -lane 'if ($F[2] =~ m/stop/) { $F[2] = "\"nonsynonymous SNV\""; } print $F[2];' | sort | uniq -c | perl -ane 'print join("_", @F), "\t"; if (eof()) { print "\n"; }' | \
	perl -lane 'my $nonsynonymous = 0; my $synonymous = 0; foreach my $geneInfo1 (@F) { if ($geneInfo1 =~ m/\"nonsynonymous/) { $nonsynonymous = ((split(/_/, $geneInfo1))[0]); } elsif ($geneInfo1 =~ m/\"synonymous/) { $synonymous = ((split(/_/, $geneInfo1))[0]); } else { } } print "synonymous: $synonymous\tnonsynonymous: $nonsynonymous\n"; ' | xargs echo $gene1

done

~~~
[  michaelt@node2  ~]$for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top5`
> do
>
> #       echo $gene1
>         cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep $gene1 | perl -F'\|' -lane 'if ($F[2] =~ m/stop/) { $F[2] = "\"nonsynonymous SNV\""; } print $F[2];' | sort | uniq -c | perl -ane 'print join("_", @F), "\t"; if (eof()) { print "\n"; }' | \
>         perl -lane 'my $nonsynonymous = 0; my $synonymous = 0; foreach my $geneInfo1 (@F) { if ($geneInfo1 =~ m/\"nonsynonymous/) { $nonsynonymous = ((split(/_/, $geneInfo1))[0]); } elsif ($geneInfo1 =~ m/\"synonymous/) { $synonymous = ((split(/_/, $geneInfo1))[0]); } else { } } print "synonymous: $synonymous\tnonsynonymous: $nonsynonymous\n"; ' | xargs echo $gene1
>
> done

NUDC synonymous: 8 nonsynonymous: 10
SDHD synonymous: 6 nonsynonymous: 0
VDAC3 synonymous: 2 nonsynonymous: 2
VTI1A synonymous: 1 nonsynonymous: 2
GEMIN2 synonymous: 1 nonsynonymous: 4
~~~

#CHECK_0: If a variant is fixed for the alternative allele in the sample population is it still considered 'polymorphic' (since it is at least still different from the reference human genome)?

for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top5`
do
	
#       echo $gene1
#	cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w $gene1 | perl -F'\|' -lane 'if ($F[2] =~ m/stop/) { $F[2] = "\"nonsynonymous SNV\""; } print $F[2];' | sort | uniq -c | perl -ane 'print join("_", @F), "\t"; if (eof()) { print "\n"; }' | \
#	perl -lane 'my $nonsynonymous = 0; my $synonymous = 0; foreach my $geneInfo1 (@F) { if ($geneInfo1 =~ m/\"nonsynonymous/) { $nonsynonymous = ((split(/_/, $geneInfo1))[0]); } elsif ($geneInfo1 =~ m/\"synonymous/) { $synonymous = ((split(/_/, $geneInfo1))[0]); } else { } } print "synonymous: $synonymous\tnonsynonymous: $nonsynonymous\n"; ' | xargs echo $gene1

	comm <(cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w $gene1 | perl -F'\|' -lane 'if ($F[2] =~ m/stop/) { $F[2] = "\"nonsynonymous SNV\""; } print $F[3], "_", $F[4];') <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz | grep -w $gene1 | perl -lane 'print $F[0], "_", $F[1];') > test1.$gene1.txt

done

#CHECK_1: Noticed that for the same variant position in NUDC I was getting a 'synonymous SNV' in the more recent results but it's actually a 'nonsynonmous SNV' in my previous AnnovarFormat output (as well as the latter annotation being found online), so starting to dig into what's going on
#~~~
#[  michaelt@login2  ~/Data/UCSC/Cow/bosTau7]$cat  /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.DropGaps.AnnovarFormat.genome_summary.csv | grep 27269413 
#"exonic","NUDC","synonymous SNV","NM_006600:c.G598G:p.V200V","612;Name=lod=409",,,,,,,,,,,,,,,,,1,27269413,27269413,A,G,
#[  michaelt@login2  ~/Data/UCSC/Cow/bosTau7]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | grep 27269413
#"exonic","NUDC","nonsynonymous SNV","NM_006600:c.G598A:p.V200M","612;Name=lod=409",,0.000923,0.0009,,0,0.999556,C,1.0,D,0.965,D,1.0,D,0.999988,D,5.12,1,27269413,27269413,G,A,
#~~~
#Note the change in reference/alternative allele information here
#However, upon looking at other information down-below, it's starting to appear that in general there may be issues with my current calls of human/chimp differences and that this example just happens to be a random overlap. Flipping the reference/alternative allele across all results just so this case 'matches up' causes just as many errors to be thrown as before in terms of the 'WARNING:...' I mention earlier and once again below
#Recommended to start with just a few blocks, if not two two or three, of human/chimp FASTA and really work out that I'm doing things correctly with calling basepair position of differences

zcat /home/michaelt/Data/UCSC/Human/hg19/chr*.hg19.panTro4.net.axt.gz | perl -lane 'if ($F[1] =~ m/chr(\d+)/) { $alignFlag1 = 1; $chr = $1; $start = $F[2]; } else { if ($alignFlag1 == 1) { @humanSeq = split(//, $F[0]); $alignFlag1++; } elsif ($alignFlag1 == 2) { @chimpSeq = split(//, $F[0]); for (my $i = 0; $i <= $#humanSeq; $i++) { if (lc($humanSeq[$i]) ne lc($chimpSeq[$i])) { print $chr, "\t", $start, "\t", $chimpSeq[$i], "\t", $humanSeq[$i]; } $start++; } $alignFlag1 = 0; $start = NULL; @humanSeq = NULL; @chimpSeq = NULL; } elsif ($alignFlag1 == 0) { $PH = 1; } else { print STDERR "\$alignFlag1 neither 0, 1 or 2 ($alignFlag1)"; }}' | gzip > /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.try2.txt.gz

zcat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.try2.txt.gz | head -n 100000 | perl -lane 'if (($F[2] ne "-") && ($F[3] ne "-")) { print $F[0], "\t", $F[1], "\t", $F[1], "\t", uc($F[2]), "\t", uc($F[3]); }' > /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.try2.First100000.DropGaps.AnnovarFormat

perl /home/michaelt/Software/annovar/summarize_annovar.pl /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.try2.First100000.DropGaps.AnnovarFormat /home/michaelt/Software/annovar/ --ver1000g 1000g2012apr --verdbsnp 129 --veresp 6500 --buildver hg19 --remove 

zcat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.try2.txt.gz | head -n 26206030 | perl -lane 'if (($F[2] ne "-") && ($F[3] ne "-")) { print $F[0], "\t", $F[1], "\t", $F[1], "\t", uc($F[2]), "\t", uc($F[3]); }' > /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.try2.First26206030.DropGaps.AnnovarFormat

perl /home/michaelt/Software/annovar/summarize_annovar.pl /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.try2.First26206030.DropGaps.AnnovarFormat /home/michaelt/Software/annovar/ --ver1000g 1000g2012apr --verdbsnp 129 --veresp 6500 --buildver hg19 --remove

~~~
.
.
.
NOTICE: Begin processing batch 2 (each batch contains 5000000 variants)
NOTICE: Finished analyzing 1000000 query variants
NOTICE: Finished analyzing 2000000 query variants
NOTICE: Finished analyzing 3000000 query variants
NOTICE: Finished analyzing 4000000 query variants
NOTICE: Finished analyzing 5000000 query variants
NOTICE: Reading FASTA sequences from /home/michaelt/Software/annovar/hg19_refGeneMrna.fa ... Done with 4545 sequences
WARNING: A total of 277 sequences cannot be found in /home/michaelt/Software/annovar/hg19_refGeneMrna.fa
(example: NM_001278556 NM_001145339 NM_001283044)
WARNING: A total of 279 sequences will be ignored due to lack of correct ORF annotation
WARNING: Potential database annotation error seqid=NM_178570, refvarstart=1480, fs=1, seqlength=1263 refcdsstart=330, with inputline=11 57244272        57244272        C       T
WARNING: Potential database annotation error seqid=NM_178570, refvarstart=1486, fs=1, seqlength=1263 refcdsstart=330, with inputline=11 57244278        57244278        T       C
WARNING: Potential database annotation error seqid=NM_178570, refvarstart=1534, fs=1, seqlength=1263 refcdsstart=330, with inputline=11 57244326        57244326        T       C
WARNING: Potential database annotation error seqid=NM_178570, refvarstart=1535, fs=2, seqlength=1263 refcdsstart=330, with inputline=11 57244327        57244327        A       T
WARNING: Potential database annotation error seqid=NM_178570, refvarstart=1554, fs=0, seqlength=1263 refcdsstart=330, with inputline=11 57244346        57244346        A       C
WARNING: Potential database annotation error seqid=NM_178570, refvarstart=1556, fs=2, seqlength=1263 refcdsstart=330, with inputline=11 57244348        57244348        G       C
WARNING: Potential database annotation error seqid=NM_016638, refvarstart=1336, fs=2, seqlength=1318 refcdsstart=146, with inputline=12 123467205       123467205       G       T
WARNING: Potential database annotation error seqid=NM_022782, refvarstart=3793, fs=2, seqlength=3611 refcdsstart=323, with inputline=12 123641435       123641435       C       T
WARNING: Potential database annotation error seqid=NM_018694, refvarstart=1393, fs=2, seqlength=1375 refcdsstart=146, with inputline=12 123467205       123467205       G       T
WARNING: Potential database annotation error seqid=NM_014619, refvarstart=2951, fs=2, seqlength=2871 refcdsstart=291, with inputline=11 120856759       120856759       C       G
WARNING: Potential database annotation error seqid=NM_001002251, refvarstart=1369, fs=2, seqlength=1351 refcdsstart=146, with inputline=12      123467205       123467205       G       T
WARNING: Potential database annotation error seqid=NM_016404, refvarstart=882, fs=2, seqlength=581 refcdsstart=559, with inputline=11   64084387        64084387        T       C
WARNING: Potential database annotation error seqid=NM_016404, refvarstart=872, fs=1, seqlength=581 refcdsstart=559, with inputline=11   64084397        64084397        G       A
WARNING: Potential database annotation error seqid=NM_016404, refvarstart=691, fs=0, seqlength=581 refcdsstart=559, with inputline=11   64084783        64084783        A       C
WARNING: Potential database annotation error seqid=NM_016404, refvarstart=676, fs=0, seqlength=581 refcdsstart=559, with inputline=11   64084798        64084798        G       A
WARNING: Potential database annotation error seqid=NM_016404, refvarstart=657, fs=2, seqlength=581 refcdsstart=559, with inputline=11   64084817        64084817        A       G
WARNING: Potential database annotation error seqid=NM_016404, refvarstart=654, fs=2, seqlength=581 refcdsstart=559, with inputline=11   64084820        64084820        G       A

---------------------------------------------------------------------------------------
WARNING: 25243 exonic SNPs have WRONG reference alleles specified in your input file!
WARNING: An example input line is <11   86161014        86161014        C       T>
WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!
WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!
---------------------------------------------------------------------------------------
.
.
.
~~~

#So systemically getting wrong reference alleles currently -- it is not simply the reference/alternative alleles being flipped between human and chimp references in my code. Doing it either way produces 'WARNING: ##### exonic SNPs have WRONG reference alleles...'


mkdir /home/michaelt/Data/UCSC/Cow/
mkdir /home/michaelt/Data/UCSC/Cow/bosTau7
#PWD /home/michaelt/Data/UCSC/Cow/bosTau7
wget http://hgdownload.soe.ucsc.edu/goldenPath/bosTau7/vsHg19/bosTau7.hg19.net.axt.gz

~~~
[  michaelt@login2  ~/Data/UCSC/Cow/bosTau7]$zcat bosTau7.hg19.net.axt.gz | grep chr | grep -v chrUn | awk '{ print $2 }' | sort | uniq -c
104255 chr1
73768 chr10
75106 chr11
59317 chr12
62043 chr13
57462 chr14
52850 chr15
52160 chr16
52200 chr17
45778 chr18
49479 chr19
95073 chr2
51247 chr20
45640 chr21
41777 chr22
38619 chr23
45609 chr24
34364 chr25
35260 chr26
34312 chr27
29725 chr28
32377 chr29
81272 chr3
82227 chr4
85034 chr5
79073 chr6
70504 chr7
75350 chr8
71852 chr9
7 chrM
52229 chrX
8837 chrY
[  michaelt@login2  ~/Data/UCSC/Cow/bosTau7]$zcat bosTau7.hg19.net.axt.gz | grep chr | grep -v chrUn | awk '{ print $5 }' | sort | uniq -c
149311 chr1
82673 chr10
85077 chr11
9 chr11_gl000202_random
90879 chr12
59114 chr13
58763 chr14
52504 chr15
49523 chr16
50148 chr17
24 chr17_gl000204_random
7 chr17_gl000205_random
1 chr17_gl000206_random
45792 chr18
30332 chr19
7 chr19_gl000209_random
4 chr1_gl000191_random
17 chr1_gl000192_random
153095 chr2
37028 chr20
23975 chr21
3 chr21_gl000210_random
19699 chr22
127133 chr3
121898 chr4
2 chr4_gl000193_random
1 chr4_gl000194_random
114841 chr5
109907 chr6
95453 chr7
11 chr7_gl000195_random
86870 chr8
70806 chr9
13 chr9_gl000200_random
2 chr9_gl000201_random
88 chrM
55257 chrX
4539 chrY
~~~

#So it looks like the first column is the 'main/reference alignment' and the second column is the 'alternative alignment'. So a pairwise alignment taken from a certain organism's directory in UCSC will always have that organism as the 'main/reference aligment'. Therefore any alignment from hg19 should have the human reference as the first set of columns and the secondary reference genome as the last set of columns


zcat /home/michaelt/Data/UCSC/Human/hg19/chr*.hg19.panTro4.net.axt.gz | head -n 98 | tail -n 8

zcat /home/michaelt/Data/UCSC/Human/hg19/chr*.hg19.panTro4.net.axt.gz | head -n 98 | tail -n 8 | perl -lane 'if ($F[1] =~ m/chr(\d+)/) { $alignFlag1 = 1; $chr = $1; $start = $F[2]; } else { if ($alignFlag1 == 1) { @humanSeq = split(//, $F[0]); $alignFlag1++; } elsif ($alignFlag1 == 2) { @chimpSeq = split(//, $F[0]); for (my $i = 0; $i <= $#humanSeq; $i++) { if ($humanSeq[$i] eq "-") { $PH2 = 1; } else { if (lc($humanSeq[$i]) ne lc($chimpSeq[$i])) { print $chr, "\t", $start, "\t", $humanSeq[$i], "\t", $chimpSeq[$i]; } $start++; } } $alignFlag1 = 0; $start = NULL; @humanSeq = NULL; @chimpSeq = NULL; } elsif ($alignFlag1 == 0) { $PH = 1; } else { print STDERR "\$alignFlag1 neither 0, 1 or 2 ($alignFlag1)"; }}'

~~~
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$zcat /home/michaelt/Data/UCSC/Human/hg19/chr*.hg19.panTro4.net.axt.gz | head -n 98 | tail -n 8
21 chr1 64454 65616 chrUn_GL393541 145756 146958 + 104160
caatgagaacacatggacacaggaaggggaacatcacacaccggggcctgttgtggggtggggggaagggggagggatagcattaggagatataactagtgttaaatgacgagttaatgggtgcagcacacccacatggcacatgtatacatatgtaactaacctgcacattgtgcacatgtaccctagaacttaaagtataataa--aaaaaaaTAGACTCTAGTACTCTGTATTATGCAAAATTTGTCTATGTTACACTTTTTTAACAACACAATCCTATTGCCCTTGAAATCTTCTTCAAAGCATTTCTCGAGTCACTCTTAAAAAGCATCTACAACCTAAAAGTATAGGAAGAGATTTATTTCCTGGAGAAGAGACCCCATTGAGATCTTAAAAGCACATTTAATGTGCCTGTGCTTAACTTAAGGT----GCTTAGGACAAAGAAGGCGATTGACATCTTTCAGGTAAAACCTGGTAAGTTTGGTGGTCAAGGAACACAACTGAGACATCACTTGGATGTATTCCTATGACTATTTTAAGAAACATAAATTGTGGTGACTCACTCAGCTCACTTTTAACTACTGCATGGTAATTAAAGATGCAAAATAAAATAAGTTACAAGAAGTGAGGTTTTTTATTGGTTAAAGCAATTTTTCTATATTTTCTCCGCAAGTTGGTCATAAAAGTTCTAAGCATTCCTCTTTTTATAAAATCGAAGCATTATTACTTACTCTCTTGTTAACCTATCTGGATTTTAATTTTGTAACTTTATTATATTTGTTTTGCTGTGATTCTTTAAAAAGCACCTTTAGACTCAGTGAGATAGCAAAAATATCCAAATAGGCCAAAAAATTGTGGCAATGTCCTCTCACTCAGGAAAATTCTGTGTGTTTTCTCTAATGGCCAAGGGAAAACTTGTGAGACTATAAAAGTTAGTCTCAGTACACAAAGCTCAGACTGGCTATTCCCAGATCTCTTCAGGTACATCTAGTCCATTCATAAAGGGCTTTTAATTAACCAAGTGGTTTACTAAAAAGGACAATTCACTACATATTATTCTCTTACAGTTTTTATG-----------------------------------CCTCATTCTGTGAAAATTGCTGTAGTCTCTTCCAGTTATGAAGAAGGTAGGTGGAAACAAAGACAAAACACATATATTAGAAGAATGAA
caatgagaacacatggacacaggaaggggaacatcacacaccggggcctgttgtggggtgggggaaggggggagggatagcattaggagatatacctaatgttaaatgacgagttaatgggtgcagcacacccacatggcacatgtatacatatgtaacaaacctacacattgtgcacatgtaccctagaacttaaagtataacaacaaaaaaaaaaGACTCCAGTACTCTGTATTATGCAAAATTTGTCTATGTTACACTTTTTTAACAACACAATCCTATTGCCCTTGAAATCTTCTTCAAAGCATTTCTCGAGTCACTCTTAAAAAGCATCTACAACCTAAAAGTATAGGAAGAGATTTATTTCCTGGAGAAGAGACTCCATTGAGATCTTAAAAGCACATTTAATGTGCCCGTGCTTAACTTAAGGTAGGTGCTTAGGACAAAGAAGACGATTGACATCTTTCAGGTAAAACCTGGTAAGTTTGGTGGTCAAGGAACACAACTGAGACATCACTTGGATGTATTTCTATGACTATTTTAAGAAACATAAATTGTGGTGACTCACTCAGCTCACTTTTAACTACTGCATGGTAATTAAAGATGCAAAATAAAATAAGTTACAAGAAGTGAGGTTTTTTATTGGTTAAAGCAATTTTTCTATATTTT-TCCGCAAGTTGGTCCTAAAAGTTCTAAGCATTCCTCTTTTTATAAAATCGAAGCATTATTACTTACTCTCTTGTTAACCTATCTGGATTTTAATTTTGTAACTTTATTATATTTGTTTTGCTGTGATTCTTTAAAAAGCACCTTTAGACTCAGTGAGATAGCAAAAATATCCAAATAGGCCAAAAAATTGTGGCAATGTCCTCTCACTCAGGAAAATTCTGTGTGTTTTCTCTAATGGCCAAGGGAAAACTTGTGAGACTATAAAAGTTAGTCTCAGTACACAAAGCTCAGACTGGCTATTCCCAGATCTCTTCAGGTACATCTAGTCCATTCATAAAGGGCTTTTAATTAACCAAGTGGTTTACTAAAAAGGACAATTCACTACATATTATTCTCTTACAGTTTTTATGCTTCAAGGCAGTGCAAATAAAGGGAAATTTGTACTCCTCATTCTGTGAAAATTGCTGTAGTCTCTTCCAGTTATGAAGAAGGTAGGTGGAAACAAAGATAAAACAAATATATTAGAAGAATGAA

22 chr1 65878 66552 chrUn_GL393541 146987 147690 + 43935
TCTTTCCCCAGGTCCGGTGTTTTCTTACCCACCTCCTTCCCTCCTTTTTATAATACCAGTGAAACTTGGTTTGGAGCATTTCTTTCACATAAAGGTACAaatcatactgctagagttgtgaggatttttacagcttttgaaagaataaactcattttaaaaacaggaaagctaaggcccagagatttttaaatgatattcccatgatcacactgtgaatttgtgccagaacccaaatgcctactc-----------------------------------ccatctcactgaGACTTACTATAAGGACATAAGGCatttatatatatatatattatatatactatatatttatatatattacatattatatatataatatatattatataatatatattatat-tatataatatataatataaatataatataaattatattatataatatataatataaatataatataaattatataaatataatatatattttattatataatataatatatattatataaatataatatata---aattatat-aatataatatatattatataatata--atatattttattatataaatatatattatatt------atataatatatattttattatataatatatattatatatttatagaatataatatatattttattatataatatatattatataatatatattatatttatatataacat
TCTTTCCTCAGGTCCGGTGTTTTCTTACCCACCTCCTTCCCTCCTTTTTATAATACCAGTGAAACTTGGTTTGGAGCATTTCTTTCACATAAAGGTACAaatcatactgctagagttgtgaggatttttagagcttttgaaagaataaactcattttaaaaacaggaaagctaaggcccagagatttttaaatgatattcccatgatcacactgtgaatttctgccagaacccaaatgcctactcccaatatatattatataaatataacatatattttaCCATCTCACTGAGACTTACTATAAAAACATAATACATTTTtatatatatatataatatatataata----taaatataatacatattat-tatattatatatattatataa---atattatatatatttattatat--tatatatattatataa---atataatatattttattatataaatataatatatattatctaaatataatatatattttattatataatgt--tatatattatctaaatataatatatattttattatataaatataatatattatcta-aatataatatatattttattatataaatataatatatattatctaaatataatatatattttattatata--atataatatatattatctaaatataatatatattttattatatga-atataatatatattatatattatatttatatttattat
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$zcat /home/michaelt/Data/UCSC/Human/hg19/chr*.hg19.panTro4.net.axt.gz | head -n 98 | tail -n 8 | perl -lane 'if ($F[1] =~ m/chr(\d+)/) { $alignFlag1 = 1; $chr = $1; $start = $F[2]; } else { if ($alignFlag1 == 1) { @humanSeq = split(//, $F[0]); $alignFlag1++; } elsif ($alignFlag1 == 2) { @chimpSeq = split(//, $F[0]); for (my $i = 0; $i <= $#humanSeq; $i++) { if (lc($humanSeq[$i]) ne lc($chimpSeq[$i])) { print $chr, "\t", $start, "\t", $humanSeq[$i], "\t", $chimpSeq[$i]; } $start++; } $alignFlag1 = 0; $start = NULL; @humanSeq = NULL; @chimpSeq = NULL; } elsif ($alignFlag1 == 0) { $PH = 1; } else { print STDERR "\$alignFlag1 neither 0, 1 or 2 ($alignFlag1)"; }}' | tail -n 10
1       66533   t       c
1       66534   a       t
1       66535   g       a
1       66560   a       g
1       66562   t       -
1       66568   t       a
1       66575   a       t
1       66594   a       t
1       66597   a       t
1       66598   c       t
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$zcat /home/michaelt/Data/UCSC/Human/hg19/chr*.hg19.panTro4.net.axt.gz | head -n 98 | tail -n 8 | perl -lane 'if ($F[1] =~ m/chr(\d+)/) { $alignFlag1 = 1; $chr = $1; $start = $F[2]; } else { if ($alignFlag1 == 1) { @humanSeq = split(//, $F[0]); $alignFlag1++; } elsif ($alignFlag1 == 2) { @chimpSeq = split(//, $F[0]); for (my $i = 0; $i <= $#humanSeq; $i++) { if ($humanSeq[$i] eq "-") { $PH2 = 1; } else { if (lc($humanSeq[$i]) ne lc($chimpSeq[$i])) { print $chr, "\t", $start, "\t", $humanSeq[$i], "\t", $chimpSeq[$i]; } $start++; } } $alignFlag1 = 0; $start = NULL; @humanSeq = NULL; @chimpSeq = NULL; } elsif ($alignFlag1 == 0) { $PH = 1; } else { print STDERR "\$alignFlag1 neither 0, 1 or 2 ($alignFlag1)"; }}' | tail -n 10
1       66485   t       c
1       66486   a       t
1       66487   g       a
1       66512   a       g
1       66514   t       -
1       66520   t       a
1       66527   a       t
1       66546   a       t
1       66549   a       t
1       66550   c       t
~~~

#Needed to include '...if ($humanSeq[$i] eq "-") { $PH2 = 1; } else...' (PH is just 'placeholder' in case I want to keep the case exception for use later) which takes into account that gaps in the primary reference do not increment the basepair position at all. So what was occuring was that whenever the primary alignment had a gap, I incremented by one the basepair position but in fact the FASTA file did not. So any situation where there were no gaps yet I would be matching up with the genome browser but in instances where at least one gap had occured (if not more) I would begin to have issues

#Rerunning the original code now with a 'Vs2' tag to indicate a newer version of the results exist 

#CHECK_0: May want to redo MK-tests eventually too using the panTro4 as-primary-alignments to come up with the list of 'fixed differences' just to see if use of 'primary/reference genome' impacts results.....if results are interesting in the first place to look at after the first round of results?

zcat /home/michaelt/Data/UCSC/Human/hg19/chr*.hg19.panTro4.net.axt.gz | perl -lane 'if ($F[1] =~ m/chr(\d+)/) { $alignFlag1 = 1; $chr = $1; $start = $F[2]; } else { if ($alignFlag1 == 1) { @humanSeq = split(//, $F[0]); $alignFlag1++; } elsif ($alignFlag1 == 2) { @chimpSeq = split(//, $F[0]); for (my $i = 0; $i <= $#humanSeq; $i++) { if ($humanSeq[$i] eq "-") { $PH2 = 1; } else { if (lc($humanSeq[$i]) ne lc($chimpSeq[$i])) { print $chr, "\t", $start, "\t", $humanSeq[$i], "\t", $chimpSeq[$i]; } $start++; } } $alignFlag1 = 0; $start = NULL; @humanSeq = NULL; @chimpSeq = NULL; } elsif ($alignFlag1 == 0) { $PH = 1; } else { print STDERR "\$alignFlag1 neither 0, 1 or 2 ($alignFlag1)"; }}' | gzip > /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.txt.gz 

~~~
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$zcat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.txt.gz | wc
44161774 176647096 689703307
~~~

zcat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.txt.gz | perl -lane 'if (($F[2] ne "-") && ($F[3] ne "-")) { print $F[0], "\t", $F[1], "\t", $F[1], "\t", uc($F[2]), "\t", uc($F[3]); }' > /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat 

~~~
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat | wc
35601707 178008535 883839722
~~~

perl /home/michaelt/Software/annovar/summarize_annovar.pl /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat /home/michaelt/Software/annovar/ --ver1000g 1000g2012apr --verdbsnp 129 --veresp 6500 --buildver hg19 --remove

~~~
.
.
.
---------------------------------------------------------------------------------------
WARNING: 1459 exonic SNPs have WRONG reference alleles specified in your input file!
WARNING: An example input line is <11   17809724        17809724        T       C>
WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!
WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!
---------------------------------------------------------------------------------------
.
.
.
~~~

#Still getting this error but much less than 20000 SNPs per set?

cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.csv | grep -E 'exonic|splicing' | grep -v ncRNA > /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.csv 

cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.csv | perl -F, -lane 'print $F[0], "|", join(",", @F[1..$#F-24]), "|", join("|", @F[$#F-23..$#F]);' | perl -F'\|' -lane 'print join("|", @F[0..2]), "|", join("|", @F[$#F-4..$#F]);' > /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt

~~~
[  michaelt@node1  ~]$cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | wc
 275494  538739 17029300
[  michaelt@node1  ~]$cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | wc
 234358  461239 14305422
~~~

for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top5`
do

#       echo $gene1
       cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w $gene1 | perl -F'\|' -lane 'if ($F[2] =~ m/stop/) { $F[2] = "\"nonsynonymous SNV\""; } print $F[2];' | sort | uniq -c | perl -ane 'print join("_", @F), "\t"; if (eof()) { print "\n"; }' | \
       perl -lane 'my $nonsynonymous = 0; my $synonymous = 0; foreach my $geneInfo1 (@F) { if ($geneInfo1 =~ m/\"nonsynonymous/) { $nonsynonymous = ((split(/_/, $geneInfo1))[0]); } elsif ($geneInfo1 =~ m/\"synonymous/) { $synonymous = ((split(/_/, $geneInfo1))[0]); } else { } } print "synonymous: $synonymous\tnonsynonymous: $nonsynonymous\n"; ' | xargs echo $gene1

#        comm <(cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w $gene1 | perl -F'\|' -lane 'if ($F[2] =~ m/stop/) { $F[2] = "\"nonsynonymous SNV\""; } print $F[3], "_", $F[4];') <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz | grep -w $gene1 | perl -lane 'print $F[0], "_", $F[1];') > test1.$gene1.txt

done

~~~
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top5`line.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynon
> do
>
> #       echo $gene1
>        cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | gre"_", @F), "\t"; if (eof()) { print "\n"; }' | \/stop/) { $F[2] = "\"nonsynonymous SNV\""; } print $F[2];' | sort | uniq -c | perl -ane 'print join(
>        perl -lane 'my $nonsynonymous = 0; my $synonymous = 0; foreach my $geneInfo1 (@F) { if ($geneInfo1 =~ m/\"nonsynonymous/) { $nonsynonymous = ((split(/_/, $genensynonymous: $nonsynonymous\n"; ' | xargs echo $gene1{ $synonymous = ((split(/_/, $geneInfo1))[0]); } else { } } print "synonymous: $synonymous\tno
>
> #        comm <(cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.ta/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRl -lane 'print $F[0], "_", $F[1];') > test1.$gene1.txt.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz | grep -w $gene1 | per
>
> done
NUDC synonymous: 5 nonsynonymous: 2
SDHD synonymous: 6 nonsynonymous: 0
VDAC3 synonymous: 2 nonsynonymous: 2
VTI1A synonymous: 1 nonsynonymous: 2
GEMIN2 synonymous: 1 nonsynonymous: 4
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top5`
> do
>
> #       echo $gene1
>        cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w $gene1 | perl -F'\|' -lane 'if ($F[2] =~ m/stop/) { $F[2] = "\"nonsynonymous SNV\""; } print $F[2];' | sort | uniq -c | perl -ane 'print join("_", @F), "\t"; if (eof()) { print "\n"; }' | \
>        perl -lane 'my $nonsynonymous = 0; my $synonymous = 0; foreach my $geneInfo1 (@F) { if ($geneInfo1 =~ m/\"nonsynonymous/) { $nonsynonymous = ((split(/_/, $geneInfo1))[0]); } elsif ($geneInfo1 =~ m/\"synonymous/) { $synonymous = ((split(/_/, $geneInfo1))[0]); } else { } } print "synonymous: $synonymous\tnonsynonymous: $nonsynonymous\n"; ' | xargs echo $gene1
>
> #        comm <(cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w $gene1 | perl -F'\|' -lane 'if ($F[2] =~ m/stop/) { $F[2] = "\"nonsynonymous SNV\""; } print $F[3], "_", $F[4];') <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz | grep -w $gene1 | perl -lane 'print $F[0], "_", $F[1];') > test1.$gene1.txt
>
> done

NUDC synonymous: 3 nonsynonymous: 2
SDHD synonymous: 3 nonsynonymous: 4
VDAC3 synonymous: 2 nonsynonymous: 0
VTI1A
GEMIN2
~~~

for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top5`
do

#       echo $gene1
#	results1=`cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w $gene1 | perl -F'\|' -lane 'if ($F[2] =~ m/stop/) { $F[2] = "\"nonsynonymous SNV\""; } print $F[2];' | sort | uniq -c | perl -ane 'print join("_", @F), "\t"; if (eof()) { print "\n"; }' | \
#	perl -lane 'my $nonsynonymous = 0; my $synonymous = 0; foreach my $geneInfo1 (@F) { if ($geneInfo1 =~ m/\"nonsynonymous/) { $nonsynonymous = ((split(/_/, $geneInfo1))[0]); } elsif ($geneInfo1 =~ m/\"synonymous/) { $synonymous = ((split(/_/, $geneInfo1))[0]); } else { } } print "synonymous: $synonymous\tnonsynonymous: $nonsynonymous\n"; '` 
	
#	if [ ! -z "$results1" ] ; then
#		echo $gene1 $results1
#	else
#		echo $gene1 "synonymous: 0 nonsynonymous: 0"
#	fi

        comm -12 <(cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w $gene1 | perl -F'\|' -lane 'if ($F[2] =~ m/stop/) { $F[2] = "\"nonsynonymous SNV\""; } print $F[3], "_", $F[4];') <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz | grep -w $gene1 | perl -lane 'print $F[0], "_", $F[1];') > test1.$gene1.txt

done

~~~
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top5`
> do
>
> #       echo $gene1
>         results1=`cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w $gene1 | perl -F'\|' -lane 'if ($F[2] =~ m/stop/) { $F[2] = "\"nonsynonymous SNV\""; } print $F[2];' | sort | uniq -c | perl -ane 'print join("_", @F), "\t"; if (eof()) { print "\n"; }' | \
>         perl -lane 'my $nonsynonymous = 0; my $synonymous = 0; foreach my $geneInfo1 (@F) { if ($geneInfo1 =~ m/\"nonsynonymous/) { $nonsynonymous = ((split(/_/, $geneInfo1))[0]); } elsif ($geneInfo1 =~ m/\"synonymous/) { $synonymous = ((split(/_/, $geneInfo1))[0]); } else { } } print "synonymous: $synonymous\tnonsynonymous: $nonsynonymous\n"; '`
>
>         if [ ! -z "$results1" ] ; then
>                 echo $gene1 $results1
>         else
>                 echo $gene1 "synonymous: 0 nonsynonymous: 0"
>         fi
>
> #        comm <(cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w $gene1 | perl -F'\|' -lane 'if ($F[2] =~ m/stop/) { $F[2] = "\"nonsynonymous SNV\""; } print $F[3], "_", $F[4];') <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz | grep -w $gene1 | perl -lane 'print $F[0], "_", $F[1];') > test1.$gene1.txt
>
> done


NUDC synonymous: 3 nonsynonymous: 2
SDHD synonymous: 3 nonsynonymous: 4
VDAC3 synonymous: 2 nonsynonymous: 0
VTI1A synonymous: 0 nonsynonymous: 0
GEMIN2 synonymous: 0 nonsynonymous: 0

[  michaelt@node1  ~/Data/UCSC/Human/hg19]$for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top5`
> do
>
.
.
.
>
>         comm <(cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w $gene1 | perl -F'\|' -lane 'if ($F[2] =~ m/stop/) { $F[2] = "\"nonsynonymous SNV\""; } print $F[3], "_", $F[4];') <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz | grep -w $gene1 | perl -lane 'print $F[0], "_", $F[1];') > test1.$gene1.txt
>
> done
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$ls -lrt | tail -n 10
-rw-rw-r-- 1 michaelt michaelt       3470 May 12 22:29 chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.log
-rw-rw-r-- 1 michaelt michaelt 3100823711 May 12 22:46 chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.csv
-rw-rw-r-- 1 michaelt michaelt   31251146 May 12 22:46 chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.exome_summary.csv
-rw-rw-r-- 1 michaelt michaelt   31250864 May 13 00:02 chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.csv
-rw-rw-r-- 1 michaelt michaelt   14305422 May 13 00:10 chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt
-rw-rw-r-- 1 michaelt michaelt        979 May 13 01:00 test1.NUDC.txt
-rw-rw-r-- 1 michaelt michaelt        189 May 13 01:00 test1.SDHD.txt
-rw-rw-r-- 1 michaelt michaelt        802 May 13 01:00 test1.VDAC3.txt
-rw-rw-r-- 1 michaelt michaelt       2198 May 13 01:01 test1.VTI1A.txt
-rw-rw-r-- 1 michaelt michaelt       1456 May 13 01:01 test1.GEMIN2.txt
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top5`
> do
>
.
.
.
>
>         comm -12 <(cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w $gene1 | perl -F'\|' -lane 'if ($F[2] =~ m/stop/) { $F[2] = "\"nonsynonymous SNV\""; } print $F[3], "_", $F[4];') <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz | grep -w $gene1 | perl -lane 'print $F[0], "_", $F[1];') > test1.$gene1.txt
>
> done
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$ls -lrt | tail -n 10
-rw-rw-r-- 1 michaelt michaelt       3470 May 12 22:29 chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.log
-rw-rw-r-- 1 michaelt michaelt 3100823711 May 12 22:46 chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.csv
-rw-rw-r-- 1 michaelt michaelt   31251146 May 12 22:46 chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.exome_summary.csv
-rw-rw-r-- 1 michaelt michaelt   31250864 May 13 00:02 chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.csv
-rw-rw-r-- 1 michaelt michaelt   14305422 May 13 00:10 chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt
-rw-rw-r-- 1 michaelt michaelt          0 May 13 01:02 test1.NUDC.txt
-rw-rw-r-- 1 michaelt michaelt          0 May 13 01:02 test1.SDHD.txt
-rw-rw-r-- 1 michaelt michaelt          0 May 13 01:03 test1.VDAC3.txt
-rw-rw-r-- 1 michaelt michaelt          0 May 13 01:03 test1.VTI1A.txt
-rw-rw-r-- 1 michaelt michaelt          0 May 13 01:03 test1.GEMIN2.txt
~~~

#In new setup, it looks like there are no sites that are both polymorphic in my HIV dataset as well as a 'fixed difference' (as I am calling it) in the hg19/panTro4 pairwise alignment 

#CHECK_0: Manually double-check that these changes/mutations are in fact cause nonsynonymous/synonymous changes?
~~~
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w SDHD
"exonic"|"SDHD"|"nonsynonymous SNV"|11|111958646|111958646|A|G
"exonic"|"SDHD"|"nonsynonymous SNV"|11|111958652|111958652|G|C
"exonic"|"SDHD"|"synonymous SNV"|11|111963834|111963834|G|T
"exonic"|"SDHD"|"synonymous SNV"|11|111963849|111963849|A|G
"exonic"|"SDHD"|"nonsynonymous SNV"|11|111963860|111963860|A|G
"exonic"|"SDHD"|"nonsynonymous SNV"|11|111963881|111963881|T|C
"exonic"|"SDHD"|"synonymous SNV"|11|111963894|111963894|G|C
~~~

cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.csv | perl -F, -lane 'print $F[0], "|", join(",", @F[1..$#F-24]), "|", join("|", @F[$#F-23..$#F]);' | perl -F'\|' -lane 'print join("|", @F[0..3]), "|", join("|", @F[$#F-4..$#F]);' > /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.Vs2.txt

~~~
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.Vs2.txt | grep -w SDHD
"exonic"|"SDHD"|"nonsynonymous SNV"|"NM_001276503:c.A118G:p.I40V"|11|111958646|111958646|A|G
"exonic"|"SDHD"|"nonsynonymous SNV"|"NM_001276503:c.G124C:p.E42Q"|11|111958652|111958652|G|C
"exonic"|"SDHD"|"synonymous SNV"|"NM_001276506:c.G345T:p.R115R"|11|111963834|111963834|G|T
"exonic"|"SDHD"|"synonymous SNV"|"NM_001276506:c.A360G:p.S120S"|11|111963849|111963849|A|G
"exonic"|"SDHD"|"nonsynonymous SNV"|"NM_001276506:c.A371G:p.Q124R"|11|111963860|111963860|A|G
"exonic"|"SDHD"|"nonsynonymous SNV"|"NM_001276506:c.T392C:p.F131S"|11|111963881|111963881|T|C
"exonic"|"SDHD"|"synonymous SNV"|"NM_001276506:c.G405C:p.A135A"|11|111963894|111963894|G|C
~~~


for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top5`
do

#       echo $gene1
	FixedDifferences1=`cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w $gene1 | perl -F'\|' -lane 'if ($F[2] =~ m/stop/) { $F[2] = "\"nonsynonymous SNV\""; } print $F[2];' | sort | uniq -c | perl -ane 'print join("_", @F), "\t"; if (eof()) { print "\n"; }' | \
	perl -lane 'my $nonsynonymous = 0; my $synonymous = 0; foreach my $geneInfo1 (@F) { if ($geneInfo1 =~ m/\"nonsynonymous/) { $nonsynonymous = ((split(/_/, $geneInfo1))[0]); } elsif ($geneInfo1 =~ m/\"synonymous/) { $synonymous = ((split(/_/, $geneInfo1))[0]); } else { } } print "fixed_syn: $synonymous\tfixed_nonsyn: $nonsynonymous\n"; '`

	Polymorphisms1=`zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz | grep -w $gene1 | grep exonic | perl -lane 'my @snpInfo = split(/;/, $F[7]); foreach my $snpInfoEntry (@snpInfo) { if ($snpInfoEntry =~ m/^VARTYPE=(.*)/) { my $AAChange = $1; if ($AAChange =~ m/stop/) { $AAChange = "nonsynonymous SNV"; } print $AAChange; } } ' | sort | uniq -c | perl -ane 'print join("_", @F), "\t"; if (eof()) { print "\n"; }' | \
	 perl -lane 'my $nonsynonymous = 0; my $synonymous = 0; foreach my $geneInfo1 (@F) { if ($geneInfo1 =~ m/nonsynonymous/) { $nonsynonymous = ((split(/_/, $geneInfo1))[0]); } elsif ($geneInfo1 =~ m/synonymous/) { $synonymous = ((split(/_/, $geneInfo1))[0]); } else { } } print "poly_syn: $synonymous\tpoly_nonsyn: $nonsynonymous\n"; '`

	if [ ! -z "$FixedDifferences1" ] && [ ! -z "$Polymorphisms1" ] ; then
		echo $gene1 $Polymorphisms1 $FixedDifferences1
	elif [ -z "$FixedDifferences1" ] ; then
		echo $gene1 $Polymorphisms1 "fixed_syn: 0 poly_syn: 0"
	elif [ -z "$Polymorphisms1" ] ; then
		echo $gene1 "poly_syn: 0 poly_nonsyn: 0" $FixedDifferences1
	else
		echo $gene1 "poly_syn: 0 poly_nonsyn: 0 fixed_syn: 0 fixed_nonsyn: 0"
	fi

done

~~~
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top5`
> do
>
> #       echo $gene1
>         FixedDifferences1=`cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w $gene1 | perl -F'\|' -lane 'if ($F[2] =~ m/stop/) { $F[2] = "\"nonsynonymous SNV\""; } print $F[2];' | sort | uniq -c | perl -ane 'print join("_", @F), "\t"; if (eof()) { print "\n"; }' | \
>         perl -lane 'my $nonsynonymous = 0; my $synonymous = 0; foreach my $geneInfo1 (@F) { if ($geneInfo1 =~ m/\"nonsynonymous/) { $nonsynonymous = ((split(/_/, $geneInfo1))[0]); } elsif ($geneInfo1 =~ m/\"synonymous/) { $synonymous = ((split(/_/, $geneInfo1))[0]); } else { } } print "fixed_syn: $synonymous\tfixed_nonsyn: $nonsynonymous\n"; '`
>
>         Polymorphisms1=`zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz | grep -w $gene1 | grep exonic | perl -lane 'my @snpInfo = split(/;/, $F[7]); foreach my $snpInfoEntry (@snpInfo) { if ($snpInfoEntry =~ m/^VARTYPE=(.*)/) { my $AAChange = $1; if ($AAChange =~ m/stop/) { $AAChange = "nonsynonymous SNV"; } print $AAChange; } } ' | sort | uniq -c | perl -ane 'print join("_", @F), "\t"; if (eof()) { print "\n"; }' | \
>          perl -lane 'my $nonsynonymous = 0; my $synonymous = 0; foreach my $geneInfo1 (@F) { if ($geneInfo1 =~ m/nonsynonymous/) { $nonsynonymous = ((split(/_/, $geneInfo1))[0]); } elsif ($geneInfo1 =~ m/synonymous/) { $synonymous = ((split(/_/, $geneInfo1))[0]); } else { } } print "poly_syn: $synonymous\tpoly_nonsyn: $nonsynonymous\n"; '`
>
>         if [ ! -z "$FixedDifferences1" ] && [ ! -z "$Polymorphisms1" ] ; then
>                 echo $gene1 $Polymorphisms1 $FixedDifferences1
>         elif [ -z "$FixedDifferences1" ] ; then
>                 echo $gene1 $Polymorphisms1 "fixed_syn: 0 poly_syn: 0"
>         elif [ -z "$Polymorphisms1" ] ; then
>                 echo $gene1 "poly_syn: 0 poly_nonsyn: 0" $FixedDifferences1
>         else
>                 echo $gene1 "poly_syn: 0 poly_nonsyn: 0 fixed_syn: 0 fixed_nonsyn: 0"
>         fi
>
> done


NUDC poly_syn: 2 poly_nonsyn: 4 fixed_syn: 3 fixed_nonsyn: 2
SDHD poly_syn: 0 poly_nonsyn: 1 fixed_syn: 3 fixed_nonsyn: 4
VDAC3 poly_syn: 1 poly_nonsyn: 5 fixed_syn: 2 fixed_nonsyn: 0
VTI1A poly_syn: 1 poly_nonsyn: 4 fixed_syn: 0 poly_syn: 0
GEMIN2 poly_syn: 1 poly_nonsyn: 3 fixed_syn: 0 poly_syn: 0
~~

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | sort -rg -k 2,2 | head -n -3 | awk '{ print $1 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs

MKTestResults1=""

#for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top5`
for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs`
do

	FixedDifferences1=`cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w $gene1 | perl -F'\|' -lane 'if ($F[2] =~ m/stop/) { $F[2] = "\"nonsynonymous SNV\""; } print $F[2];' | sort | uniq -c | perl -ane 'print join("_", @F), "\t"; if (eof()) { print "\n"; }' | perl -lane 'my $nonsynonymous = 0; my $synonymous = 0; foreach my $geneInfo1 (@F) { if ($geneInfo1 =~ m/\"nonsynonymous/) { $nonsynonymous = ((split(/_/, $geneInfo1))[0]); } elsif ($geneInfo1 =~ m/\"synonymous/) { $synonymous = ((split(/_/, $geneInfo1))[0]); } else { } } print "fixed_syn: $synonymous\tfixed_nonsyn: $nonsynonymous\n"; '`
	Polymorphisms1=`zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz | grep -w $gene1 | grep exonic | perl -lane 'my @snpInfo = split(/;/, $F[7]); foreach my $snpInfoEntry (@snpInfo) { if ($snpInfoEntry =~ m/^VARTYPE=(.*)/) { my $AAChange = $1; if ($AAChange =~ m/stop/) { $AAChange = "nonsynonymous SNV"; } print $AAChange; } } ' | sort | uniq -c | perl -ane 'print join("_", @F), "\t"; if (eof()) { print "\n"; }' | perl -lane 'my $nonsynonymous = 0; my $synonymous = 0; foreach my $geneInfo1 (@F) { if ($geneInfo1 =~ m/nonsynonymous/) { $nonsynonymous = ((split(/_/, $geneInfo1))[0]); } elsif ($geneInfo1 =~ m/synonymous/) { $synonymous = ((split(/_/, $geneInfo1))[0]); } else { } } print "poly_syn: $synonymous\tpoly_nonsyn: $nonsynonymous\n"; '`
	pValue=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w $gene1 | awk '{ print $2 }'`

	if [ ! -z "$FixedDifferences1" ] && [ ! -z "$Polymorphisms1" ] ; then
		MKTestResults1+=`echo $gene1 $pValue $Polymorphisms1 $FixedDifferences1 "\n"`
	elif [ -z "$FixedDifferences1" ] ; then 
		MKTestResults1+=`echo $gene1 $pValue $Polymorphisms1 "fixed_syn: 0 poly_syn: 0\n"`
	elif [ -z "$Polymorphisms1" ] ; then 
		MKTestResults1+=`echo $gene1 $pValue "poly_syn: 0 poly_nonsyn: 0" $FixedDifferences1 "\n"`
	else
		MKTestResults1+=`echo $gene1 $pValue "poly_syn: 0 poly_nonsyn: 0 fixed_syn: 0 fixed_nonsyn: 0\n"`
	fi

done

#echo -e $MKTestResults1top5 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.top5.txt
echo -e $MKTestResults1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.txt

~~~
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$echo $MKTestResults1top5
NUDC poly_syn: 2 poly_nonsyn: 4 fixed_syn: 3 fixed_nonsyn: 2 \nSDHD poly_syn: 0 poly_nonsyn: 1 fixed_syn: 3 fixed_nonsyn: 4 \nVDAC3 poly_syn: 1 poly_nonsyn: 5 fixed_syn: 2 fixed_nonsyn: 0 \nVTI1A poly_syn: 1 poly_nonsyn: 4 fixed_syn: 0 poly_syn: 0\nGEMIN2 poly_syn: 1 poly_nonsyn: 3 fixed_syn: 0 poly_syn: 0\n
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$echo -e $MKTestResults1top5
NUDC poly_syn: 2 poly_nonsyn: 4 fixed_syn: 3 fixed_nonsyn: 2
SDHD poly_syn: 0 poly_nonsyn: 1 fixed_syn: 3 fixed_nonsyn: 4
VDAC3 poly_syn: 1 poly_nonsyn: 5 fixed_syn: 2 fixed_nonsyn: 0
VTI1A poly_syn: 1 poly_nonsyn: 4 fixed_syn: 0 poly_syn: 0
GEMIN2 poly_syn: 1 poly_nonsyn: 3 fixed_syn: 0 poly_syn: 0
~~~

#CHECK_1: Need to do comm -12 for entire dataset
#CHECK_1: Check if comm col 3 output overlaps with fixed differences based on chr_basepair position and remove those? If a site isn't fixed in the sample for the reference allele then it should be considered polymorphic, even if the alternative allele is the same/different as the Chimpanzee allele
#CHECK_1: Do redo of  MKTestResults1 with new overlap sites once done; then do NI stuff, top hit NI stuff, pathway stuff, seroneg vs sero pos?; then fix grep stuff with GIMAP5-GIMAP10...
#CHECK_1: GIMAP5 situation?
#CHECK_1: Do MK tests comparing polymorphism between seronegatives and seropositive subsets? Same with Tajima's D/Pi? Just need to create two new .vcfs subsetting those two sets of individuals, since the code below 
#CHECK_1: Do Neutrality Index for each gene, then do permutation and look at collapsed NI for top 5, 10, 25, 50, 100, see if significant compared to permutations?
#CHECK_1: Look at Neutrality Index over pathways?.....shouldn't be too hard to analyze/collect since have baseline results already

#CHECK_0: Seroneg vs. seropos stuff
#CHECK_0: Do in other array?
#CHECK_0: Handle zero entries during NI calculation?
#CHECK_0: Still do Tajima's D and Pi estimates through vcftools?
#CHECK_0: And redo everything, or at least the basic level results, using a different outgroup? Such as Gorilla or Rhesus?

rm -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.txt

for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs`
do

#        comm -12 <(cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w $gene1 | perl -F'\|' -lane 'if ($F[2] =~ m/stop/) { $F[2] = "\"nonsynonymous SNV\""; } print $F[3], "_", $F[4];') <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz | grep -w $gene1 | perl -lane 'print $F[0], "_", $F[1];') | perl -ane 'print join("\t", @F), "\t"; if (eof()) { print "\n"; }' | xargs echo $gene1 >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.txt
        OverlapSites1=`comm -12 <(cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep [^-a-zA-Z0-9]$gene1[^-a-zA-Z0-9] | perl -F'\|' -lane 'if ($F[2] =~ m/stop/) { $F[2] = "\"nonsynonymous SNV\""; } print $F[3], "_", $F[4];') <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz | grep [^-a-zA-Z0-9]$gene1[^-a-zA_Z0-9] | perl -lane 'print $F[0], "_", $F[1];') | perl -ane 'print join("\t", @F), "\t"; if (eof()) { print "\n"; }'`
	echo $gene1 $OverlapSites1 >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.try2.txt

done

#CHECK_1: Compare try2 to original with use of [^-]$gene1[^-] versus -w $gene1; specifically check GIMAP5 or GIMAP1 as potential examples

~~~
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.txt | grep CAND2                                                                                                                                                                                                                                         
CAND2 3_12848822 3_12848946 3_12858028 3_12858395 3_12859004 3_12867067
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep 12848822                                                                                                   
"exonic"|"CAND2"|"nonsynonymous SNV"|3|12848822|12848822|T|C
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz | perl -lane 'print join("\t", @F[0..9]);' | grep 12848822
3       12848822        rs11718898      T       C       438879  VQSRTrancheSNP99.00to99.90      AA=C;AC=1236;AF=0.680;AN=1818;BaseQRankSum=-46.405;DB;DP=21883;Dels=0.00;FS=4.142;HRun=1;HaplotypeScore=2.4830;InbreedingCoeff=0.0176;MLEAC=1235;MLEAF=0.679;MQ=60.00;MQ0=0;MQRankSum=0.852;POSITIVE_TRAIN_SITE;QD=22.42;ReadPosRankSum=0.178;VQSLOD=2.08;culprit=HaplotypeScore;AC_SLOW=117;AACHANGE=NM_001162499:c.T230C:p.V77A;AC_ALL=1550;AC_HESERONEG=220;AC_SERONEG=480;LOC=exonic;AC_SEROPOS=563;AC_RAPID=81;PVAL=0.8945759000;QVAL=0.70177;GENE=CAND2;VARTYPE=nonsynonymous_SNV;AC_TOTAL=1043      GT:AD:DP:GQ:PL  1/1:0,17:17:51:544,51,0   
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep 12867067
"exonic"|"CAND2"|"nonsynonymous SNV"|3|12867067|12867067|C|T
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz | perl -lane 'print join("\t", @F[0..9]);' | grep 12867067
3       12867067        .       C       T       214.59  VQSRTrancheSNP99.00to99.90      AA=c;AC=3;AF=1.650e-03;AN=1818;BaseQRankSum=7.933;DP=20999;Dels=0.00;FS=0.590;HRun=0;HaplotypeScore=2.5359;InbreedingCoeff=-0.0022;MLEAC=1;MLEAF=5.501e-04;MQ=59.99;MQ0=0;MQRankSum=0.526;QD=5.65;ReadPosRankSum=-0.588;VQSLOD=1.23;culprit=QD;AC_SLOW=0;AACHANGE=NM_012298:c.C2788T:p.R930W;AC_ALL=1550;AC_HESERONEG=1;AC_SERONEG=2;LOC=exonic;AC_SEROPOS=1;AC_RAPID=0;PVAL=0.8945759000;QVAL=0.70177;GENE=CAND2;VARTYPE=nonsynonymous_SNV;AC_TOTAL=3     GT:AD:DP:GQ:PL  0/0:23,0:23:69:0,69,684

[  michaelt@node1  ~/Data/UCSC/Human/hg19]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.txt | grep NCAPD3
NCAPD3 11_134047243 11_134055275
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep 134047243
"exonic"|"NCAPD3"|"synonymous SNV"|11|134047243|134047243|T|C
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz | perl -lane 'print join("\t", @F[0..9]);' | grep 134047243
11      134047243       rs2276265       T       C       273007  VQSRTrancheSNP99.00to99.90      AA=C;AC=603;AF=0.333;AN=1810;BaseQRankSum=45.882;DB;DP=26323;Dels=0.00;FS=1.086;HRun=0;HaplotypeScore=1.1857;InbreedingCoeff=0.0210;MLEAC=603;MLEAF=0.333;MQ=59.99;MQ0=0;MQRankSum=-0.800;POSITIVE_TRAIN_SITE;QD=18.28;ReadPosRankSum=-1.084;VQSLOD=1.30;culprit=HaplotypeScore;AC_SLOW=55;AACHANGE=NM_015261:c.A2892G:p.V964V;AC_ALL=1550;AC_HESERONEG=118;AC_SERONEG=223;LOC=exonic;AC_SEROPOS=254;AC_RAPID=42;PVAL=0.8288377000;QVAL=0.69767;GENE=NCAPD3;VARTYPE=synonymous_SNV;AC_TOTAL=477    GT:AD:DP:GQ:PL  0/1:10,7:17:99:189,0,292
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep 134055275
"exonic"|"NCAPD3"|"nonsynonymous SNV"|11|134055275|134055275|C|T
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz | perl -lane 'print join("\t", @F[0..9]);' | grep 134055275
11      134055275       rs61744119      C       T       526.31  VQSRTrancheSNP99.00to99.90      AA=c;AC=4;AF=2.210e-03;AN=1810;BaseQRankSum=0.590;DB;DP=19728;Dels=0.00;FS=14.631;HRun=1;HaplotypeScore=1.5755;InbreedingCoeff=-0.0034;MLEAC=3;MLEAF=1.657e-03;MQ=59.99;MQ0=0;MQRankSum=0.554;QD=8.35;ReadPosRankSum=-0.540;VQSLOD=-7.518e-01;culprit=HaplotypeScore;AC_SLOW=0;AACHANGE=NM_015261:c.G2192A:p.R731K;AC_ALL=1550;AC_HESERONEG=0;AC_SERONEG=0;LOC=exonic;AC_SEROPOS=1;AC_RAPID=1;PVAL=0.8288377000;QVAL=0.69767;GENE=NCAPD3;VARTYPE=nonsynonymous_SNV;AC_TOTAL=1      GT:AD:DP:GQ:PL  0/0:17,0:17:51:0,51,553
~~~

~~~
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.try2.txt | grep -w CAND2 | perl -ane 'print join("\n", @F[1..$#F]), "\n";' | perl -lane 'print ((split(/_/, $F[0]))[1]);'
12848822
12848946
12858028
12858395
12859004
12867067
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w CAND2 | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.try2.txt | grep -w CAND2 | perl -ane 'print join("\n", @F[1..$#F]), "\n";' | perl -lane 'print ((split(/_/, $F[0]))[1]);')
"exonic"|"CAND2"|"synonymous SNV"|3|12845014|12845014|G|A
"exonic"|"CAND2"|"synonymous SNV"|3|12845035|12845035|C|T
"exonic"|"CAND2"|"synonymous SNV"|3|12848853|12848853|T|C
"exonic"|"CAND2"|"synonymous SNV"|3|12848907|12848907|T|C
"exonic"|"CAND2"|"synonymous SNV"|3|12856665|12856665|C|T
"exonic"|"CAND2"|"synonymous SNV"|3|12856821|12856821|C|T
"exonic"|"CAND2"|"nonsynonymous SNV"|3|12856856|12856856|A|G
"exonic"|"CAND2"|"synonymous SNV"|3|12857449|12857449|C|T
"exonic"|"CAND2"|"synonymous SNV"|3|12857458|12857458|G|A
"exonic"|"CAND2"|"synonymous SNV"|3|12858147|12858147|T|G
"exonic"|"CAND2"|"nonsynonymous SNV"|3|12858166|12858166|C|T
"exonic"|"CAND2"|"synonymous SNV"|3|12858258|12858258|C|T
"exonic"|"CAND2"|"synonymous SNV"|3|12858330|12858330|G|T
"exonic"|"CAND2"|"synonymous SNV"|3|12858630|12858630|T|C
"exonic"|"CAND2"|"nonsynonymous SNV"|3|12858631|12858631|G|A
"exonic"|"CAND2"|"synonymous SNV"|3|12858888|12858888|G|A
"exonic"|"CAND2"|"synonymous SNV"|3|12859092|12859092|T|C
"exonic"|"CAND2"|"nonsynonymous SNV"|3|12872982|12872982|G|C
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w CAND2 | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.try2.txt | grep -w CAND2 | perl -ane 'print join("\n", @F[1..$#F]), "\n";' | perl -lane 'print ((split(/_/, $F[0]))[1]);') | wc
	18      36    1056
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w CAND2 | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.try2.txt | grep -w CAND2 | perl -ane 'print join("\n", @F[1..$#F]), "\n";') | wc
	24      48    1419
~~~

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.try2.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.txt

~~~
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.txt | perl -lane 'print $#F;' | sort | uniq -c
288 0
115 1
1 13
1 15
40 2
12 3
6 4
4 5
4 6
1 8
1 9
~~~

MKTestResults1=""
#Note -- the $MKTestResults1 routine doesn't work if submitted as a background process -- the result does not get stored outside of the loop/job if it doesn't stay in the foreground
FileOutput1="/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.wOverlapCorrection.txt"
rm -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.wOverlapCorrection.txt

#for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top5`
for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs`
do

#	gene1="CAND2"

	FixedDifferences1=`cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep [^-0-9a-zA-Z]$gene1[^-0-9a-zA-Z] | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.txt | grep ^$gene1[^-0-9a-zA-Z] | perl -ane 'splice(@F, 1, 0, "-9_-9"); print join("\n", @F[1..$#F]), "\n";' | perl -lane 'print ((split(/_/, $F[0]))[1]);') | perl -F'\|' -lane 'if ($F[2] =~ m/stop/) { $F[2] = "\"nonsynonymous SNV\""; } print $F[2];' | sort | uniq -c | perl -ane 'print join("_", @F), "\t"; if (eof()) { print "\n"; }' | perl -lane 'my $nonsynonymous = 0; my $synonymous = 0; foreach my $geneInfo1 (@F) { if ($geneInfo1 =~ m/\"nonsynonymous/) { $nonsynonymous = ((split(/_/, $geneInfo1))[0]); } elsif ($geneInfo1 =~ m/\"synonymous/) { $synonymous = ((split(/_/, $geneInfo1))[0]); } else { } } print "fixed_syn: $synonymous\tfixed_nonsyn: $nonsynonymous\n"; '`
	Polymorphisms1=`zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz | grep [^-0-9a-zA-Z]$gene1[^-0-9a-zA-Z] | grep exonic | perl -lane 'my @snpInfo = split(/;/, $F[7]); foreach my $snpInfoEntry (@snpInfo) { if ($snpInfoEntry =~ m/^VARTYPE=(.*)/) { my $AAChange = $1; if ($AAChange =~ m/stop/) { $AAChange = "nonsynonymous SNV"; } print $AAChange; } } ' | sort | uniq -c | perl -ane 'print join("_", @F), "\t"; if (eof()) { print "\n"; }' | perl -lane 'my $nonsynonymous = 0; my $synonymous = 0; foreach my $geneInfo1 (@F) { if ($geneInfo1 =~ m/nonsynonymous/) { $nonsynonymous = ((split(/_/, $geneInfo1))[0]); } elsif ($geneInfo1 =~ m/synonymous/) { $synonymous = ((split(/_/, $geneInfo1))[0]); } else { } } print "poly_syn: $synonymous\tpoly_nonsyn: $nonsynonymous\n"; '`
	pValue=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep ^$gene1[^-0-9a-zA-Z] | awk '{ print $2 }'`

	if [ ! -z "$FixedDifferences1" ] && [ ! -z "$Polymorphisms1" ] ; then
#		MKTestResults1+=`echo $gene1 $pValue $Polymorphisms1 $FixedDifferences1 "\n"`
		echo $gene1 $pValue $Polymorphisms1 $FixedDifferences1 >> $FileOutput1
	elif [ -z "$FixedDifferences1" ] ; then 
#		MKTestResults1+=`echo $gene1 $pValue $Polymorphisms1 "fixed_syn: 0 fixed_nonsyn: 0\n"`
		echo $gene1 $pValue $Polymorphisms1 "fixed_syn: 0 fixed_nonsyn: 0" >> $FileOutput1
	elif [ -z "$Polymorphisms1" ] ; then 
#		MKTestResults1+=`echo $gene1 $pValue "poly_syn: 0 poly_nonsyn: 0" $FixedDifferences1`
		echo $gene1 $pValue "poly_syn: 0 poly_nonsyn: 0" $FixedDifferences1 >> $FileOutput1
	else
#		MKTestResults1+=`echo $gene1 $pValue "poly_syn: 0 poly_nonsyn: 0 fixed_syn: 0 fixed_nonsyn: 0\n"`
		echo $gene1 $pValue "poly_syn: 0 poly_nonsyn: 0 fixed_syn: 0 fixed_nonsyn: 0" >> $FileOutput1
	fi

#	echo $MKTestResults1 $FixedDifferences1 $Polymorphisms1 $pValue

done

#echo -e $MKTestResults1top5 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.top5.txt
#echo -e $MKTestResults1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.wOverlapCorrection.txt

~~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.wOverlapCorrection.txt | perl -lane 'print $#F;' | sort | uniq -c
    473 9
~~~

~~~
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$echo -e $MKTestResults1 
CAND2 0.8945759000 poly_syn: 22 poly_nonsyn: 31 fixed_syn: 15 fixed_nonsyn: 9 
CAND2 0.8945759000 poly_syn: 22 poly_nonsyn: 31 fixed_syn: 15 fixed_nonsyn: 9 
CAND2 0.8945759000 poly_syn: 22 poly_nonsyn: 31 fixed_syn: 15 fixed_nonsyn: 9 
CAND2 0.8945759000 poly_syn: 22 poly_nonsyn: 31 fixed_syn: 15 fixed_nonsyn: 9 
CAND2 0.8945759000 poly_syn: 22 poly_nonsyn: 31 fixed_syn: 15 fixed_nonsyn: 9 
.
.
.

[  michaelt@node1  ~/Data/UCSC/Human/hg19]$echo -e $MKTestResults1 
CAND2 0.8945759000 poly_syn: 22 poly_nonsyn: 31 fixed_syn: 14 fixed_nonsyn: 4 
CAND2 0.8945759000 poly_syn: 22 poly_nonsyn: 31 fixed_syn: 14 fixed_nonsyn: 4 
CAND2 0.8945759000 poly_syn: 22 poly_nonsyn: 31 fixed_syn: 14 fixed_nonsyn: 4 
CAND2 0.8945759000 poly_syn: 22 poly_nonsyn: 31 fixed_syn: 14 fixed_nonsyn: 4 
CAND2 0.8945759000 poly_syn: 22 poly_nonsyn: 31 fixed_syn: 14 fixed_nonsyn: 4 
~~~

#Looked at how many loci have variants that overlap between files, the sort | uniq -c below; doesn't seem to be that many really, and there appears to be a correlation between 'longer genes' (here assumed based on a larger total number of variants) 
#Ran into problem where if gene had no entries in ...FD_Poly_OverlapSites.txt would get a "0       0       0" from wordcount because presumably perl then fails the 'join("\n", @F[1..$#F])' call. So including a placeholder '-9_-9' value at the beginning of every entry so that ontop of the basepair positions being included, technically grep is also removing -w "-9"....which shouldn't affect anything?
~~~
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.txt | perl -lane 'print $#F;' | sort | uniq -c
288 0
115 1
1 13
1 15
40 2
12 3
6 4
4 5
4 6
1 8
1 9
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.txt | perl -lane 'if ($#F == 9) { print join("\t", @F); } ' 
EPPK1   8_144940777     8_144941181     8_144941611     8_144941903     8_144942903     8_144943196     8_144944176     8_144945792     8_144945943
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.txt | grep EPPK1
EPPK1 0.0901366700 poly_syn: 63 poly_nonsyn: 76 fixed_syn: 55 fixed_nonsyn: 33
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.txt | perl -lane 'if ($#F == 8) { print join("\t", @F); } '
PARP4   13_25009099     13_25009485     13_25027744     13_25027745     13_25029295     13_25033227     13_25068808     13_25075859
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.txt | perl -lane 'if ($#F == 13) { print join("\t", @F); } '
RNF213  17_78247068     17_78261805     17_78262161     17_78269406     17_78280264     17_78298938     17_78306280     17_78311508     17_78319136     17_78327358     17_78338302     17_78354661     17_78354772
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.txt | perl -lane 'if ($#F == 15) { print join("\t", @F); } '
ALMS1   2_73651967      2_73675669      2_73677808      2_73678616      2_73679013      2_73679280      2_73679990      2_73680508      2_73717103      2_73717497      2_73717567      2_73717656      2_73800270      2_73826590      2_73828538
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.txt | grep PARP4
PARP4 0.1275589000 poly_syn: 12 poly_nonsyn: 39 fixed_syn: 18 fixed_nonsyn: 37
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.txt | grep RNF213
RNF213 0.1942127000 poly_syn: 76 poly_nonsyn: 107 fixed_syn: 65 fixed_nonsyn: 106
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.txt | grep ALMS1
ALMS1 0.6375027000 poly_syn: 36 poly_nonsyn: 80 fixed_syn: 43 fixed_nonsyn: 79

[  michaelt@node1  ~/Data/UCSC/Human/hg19]$cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w EDC4 | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.try2.txt | grep -w $gene1 | perl -ane 'print join("\n", @F[1..$#F]), "\n";' | perl -lane 'print ((split(/_/, $F[0]))[1]);') | wc
25      50    1465
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w EDC4 | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.try2.txt | grep -w EDC4 | perl -ane 'print join("\n", @F[1..$#F]), "\n";' | perl -lane 'print ((split(/_/, $F[0]))[1]);') | wc
24      48    1407
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w EDC4 | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.try2.txt | grep -w VDAC3 | perl -ane 'print join("\n", @F[1..$#F]), "\n";' | perl -lane 'print ((split(/_/, $F[0]))[1]);') | wc
0       0       0
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$echo $gene1
CAND2
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w EDC4 | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.try2.txt | grep -w CAND2 | perl -ane 'print join("\n", @F[1..$#F]), "\n";' | perl -lane 'print ((split(/_/, $F[0]))[1]);') | wc
25      50    1465
[  michaelt@node1  ~/Data/UCSC/Human/hg19]$cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w EDC4 | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.txt | grep -w VDAC3 | perl -ane 'splice(@F, 1, 0, "-9_-9"); print join("\n", @F[1..$#F]), "\n";' | perl -lane 'print ((split(/_/, $F[0]))[1]);') | wc
25      50    1465
~~~

#Grep correction stuff
#Showing the work done to figure out which is the correct form of grep to use in order to only get the geneID I particularly want. There is an ongoing issue because of the dash ("-") in certain gene/transcripts such as GIMAP1-GIMAP5 since the -w flag considers the dash as a 'word boundary' but in my case I'd like for it to be considered just as much as a letter as a-zA-Z. Also putting the delimiters on the beginning and end doesn't always work since in certain files the geneID begins the file and having this delimiter in the beginning forces there to be '/some/ character at the beginning before $gene1' so a line that immediately begins with $gene1 is precluded
~~~
947  cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep GIMAP5
948  cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w GIMAP5
949  cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w [^-]GIMAP5[^-]
950  cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep [^-]GIMAP5[^-]
951  cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w [^-]GIMAP5[^-]
952  cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w [^-]GIMAP5
953  cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w GIMAP5[^-]
954  cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w [^-]GIMAP5[!-]
959  cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w -E ([^-]|$)GIMAP5([^-]|$)
960  cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w -E '([^-]|$)GIMAP5([^-]|$)'
961  cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w -E '([^-]|$)GIMAP5'
962  cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w -E 'GIMAP5([^-]|$)'
963  cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep '\bGIMAP5\b'
964  cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep [^-]GIMAP5[^-]
965  cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep [^-]GIMAP5
966  cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep [^-]GIMAP1[^-]
967  cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep -w [^-]GIMAP1[^-]
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz | perl -lane 'print join("\t", @F[0..9]);' | grep [^-]AHCY[^-] | wc
417    4170  205213
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz | perl -lane 'print join("\t", @F[0..9]);' | grep [^-]AHCY[^-] | grep AHCYL1 | wc
138    1380   66479
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz | perl -lane 'print join("\t", @F[0..9]);' | grep [^-]AHCY[^-] | grep AHCYL2 | wc
    182    1820   90294
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz | perl -lane 'print join("\t", @F[0..9]);' | grep [^-0-9a-zA-Z]AHCY[^-0-9a-zA-Z] | wc
97     970   48440
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz | perl -lane 'print join("\t", @F[0..9]);' | grep [^-0-9a-zA-Z]AHCY[^-0-9a-zA-Z] | grep AHCYL1 | wc
0       0       0
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz | perl -lane 'print join("\t", @F[0..9]);' | grep [^\-0-9a-zA-Z]AHCY[^\-0-9a-zA-Z] | wc
97     970   48440
[  michaelt@node1  ~]$cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep AHCY
"exonic"|"AHCYL1"|"synonymous SNV"|1|110561699|110561699|C|A
"exonic"|"AHCYL1"|"synonymous SNV"|1|110562202|110562202|C|T
"exonic"|"AHCYL1"|"synonymous SNV"|1|110562208|110562208|G|A
"exonic"|"AHCY"|"synonymous SNV"|20|32878165|32878165|G|A
"exonic"|"AHCY"|"synonymous SNV"|20|32878168|32878168|G|A
"exonic"|"AHCY"|"synonymous SNV"|20|32880216|32880216|G|A
"exonic"|"AHCY"|"synonymous SNV"|20|32883326|32883326|G|A
"exonic"|"AHCYL2"|"synonymous SNV"|7|128865034|128865034|C|T
"exonic"|"AHCYL2"|"nonsynonymous SNV"|7|128865102|128865102|T|C
"exonic"|"AHCYL2"|"synonymous SNV"|7|128865118|128865118|G|A
"exonic"|"AHCYL2"|"synonymous SNV"|7|128865121|128865121|G|A
"exonic"|"AHCYL2"|"synonymous SNV"|7|129028952|129028952|A|G
"exonic"|"AHCYL2"|"synonymous SNV"|7|129040186|129040186|C|T
[  michaelt@node1  ~]$cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep AHCY | wc
13      26     784
[  michaelt@node1  ~]$cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep [^-0-9a-zA-Z]AHCY[^-0-9a-zA-Z] | wc
4       8     232
[  michaelt@node1  ~]$cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep [^-0-9a-zA-Z]GIMAP5[^-0-9a-zA-Z] | wc
6      12     465
[  michaelt@node1  ~]$cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep [^-0-9a-zA-Z]GIMAP5[^-0-9a-zA-Z]
"exonic"|"GIMAP1-GIMAP5,GIMAP5"|"nonsynonymous SNV"|7|150437989|150437989|G|A
"exonic"|"GIMAP1-GIMAP5,GIMAP5"|"nonsynonymous SNV"|7|150439297|150439297|A|G
"exonic"|"GIMAP1-GIMAP5,GIMAP5"|"synonymous SNV"|7|150439500|150439500|C|T
"exonic"|"GIMAP1-GIMAP5,GIMAP5"|"nonsynonymous SNV"|7|150439610|150439610|C|A
"exonic"|"GIMAP1-GIMAP5,GIMAP5"|"nonsynonymous SNV"|7|150439645|150439645|A|G
"exonic"|"GIMAP1-GIMAP5,GIMAP5"|"nonsynonymous SNV"|7|150439943|150439943|A|G
[  michaelt@node1  ~]$cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep [^-0-9a-zA-Z]GIMAP1[^-0-9a-zA-Z]
"exonic"|"GIMAP1,GIMAP1-GIMAP5"|"nonsynonymous SNV"|7|150417150|150417150|G|A
"exonic"|"GIMAP1,GIMAP1-GIMAP5"|"synonymous SNV"|7|150417185|150417185|C|T
"exonic"|"GIMAP1,GIMAP1-GIMAP5"|"synonymous SNV"|7|150417287|150417287|C|T
"exonic"|"GIMAP1,GIMAP1-GIMAP5"|"nonsynonymous SNV"|7|150417342|150417342|C|G
"exonic"|"GIMAP1"|"synonymous SNV"|7|150417509|150417509|C|T
"exonic"|"GIMAP1"|"synonymous SNV"|7|150417527|150417527|A|G
"exonic"|"GIMAP1"|"nonsynonymous SNV"|7|150417709|150417709|G|A
"exonic"|"GIMAP1"|"synonymous SNV"|7|150417716|150417716|C|G
"exonic"|"GIMAP1"|"synonymous SNV"|7|150417723|150417723|T|C
"exonic"|"GIMAP1"|"nonsynonymous SNV"|7|150417730|150417730|T|G
"exonic"|"GIMAP1"|"nonsynonymous SNV"|7|150417847|150417847|A|G
"exonic"|"GIMAP1"|"nonsynonymous SNV"|7|150417991|150417991|T|C
"exonic"|"GIMAP1"|"synonymous SNV"|7|150418012|150418012|G|A
[  michaelt@node1  ~]$cat /home/michaelt/Data/UCSC/Human/hg19/chrAll.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.AnnovarFormat.genome_summary.Exonic.ForMKTestFormat.txt | grep [^-0-9a-zA-Z]GIMAP1-GIMAP5[^-0-9a-zA-Z]
"exonic"|"GIMAP1,GIMAP1-GIMAP5"|"nonsynonymous SNV"|7|150417150|150417150|G|A
"exonic"|"GIMAP1,GIMAP1-GIMAP5"|"synonymous SNV"|7|150417185|150417185|C|T
"exonic"|"GIMAP1,GIMAP1-GIMAP5"|"synonymous SNV"|7|150417287|150417287|C|T
"exonic"|"GIMAP1,GIMAP1-GIMAP5"|"nonsynonymous SNV"|7|150417342|150417342|C|G
"exonic"|"GIMAP1-GIMAP5"|"synonymous SNV"|7|150434601|150434601|T|C
"exonic"|"GIMAP1-GIMAP5"|"synonymous SNV"|7|150434700|150434700|A|G
"exonic"|"GIMAP1-GIMAP5"|"synonymous SNV"|7|150434787|150434787|A|G
"exonic"|"GIMAP1-GIMAP5,GIMAP5"|"nonsynonymous SNV"|7|150437989|150437989|G|A
"exonic"|"GIMAP1-GIMAP5,GIMAP5"|"nonsynonymous SNV"|7|150439297|150439297|A|G
"exonic"|"GIMAP1-GIMAP5,GIMAP5"|"synonymous SNV"|7|150439500|150439500|C|T
"exonic"|"GIMAP1-GIMAP5,GIMAP5"|"nonsynonymous SNV"|7|150439610|150439610|C|A
"exonic"|"GIMAP1-GIMAP5,GIMAP5"|"nonsynonymous SNV"|7|150439645|150439645|A|G
"exonic"|"GIMAP1-GIMAP5,GIMAP5"|"nonsynonymous SNV"|7|150439943|150439943|A|G
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.txt | grep [^-a-zA-Z0-9]GIMAP5[^-a-zA-Z0-9]                                                                                                                                                                                                                                   
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.txt | grep GIMAP5[^-a-zA-Z0-9]                                                                                                                                                                                                                                                
GIMAP1-GIMAP5 7_150434700 7_150439500
GIMAP5 7_150439500                                                                                                                                                                                                                                                                                                         
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.txt | grep ^GIMAP5[^-a-zA-Z0-9]                                                                                                                                                                                                                                               
GIMAP5 7_150439500
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep [^-a-zA-Z0-9]GIMAP5[^-a-zA-Z0-9]
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep GIMAP5[^-a-zA-Z0-9]
GIMAP1-GIMAP5   0.2016256000 3.675602e-03 0.54826       11      775      0.068387        11      11      0.20668         0.25   775      0.068387        11      11      0.31419         1      775      0.068387        11      11      0.059376        1      775      0.068387        11      11      0.082922        0.09
GIMAP5          0.0225139900 8.590028e-04 0.30297       8       775      0.023226        8       8       0.02232         0.04   775      0.023226        8       8       0.11654         0.09   775      0.023226        8       8       0.20179         1      775      0.023226        8       8       0.41482         0 
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep ^GIMAP5[^-a-zA-Z0-9]                                                                                                                                                                                                                                                                                             
GIMAP5          0.0225139900 8.590028e-04 0.30297       8       775      0.023226        8       8       0.02232         0.04   775      0.023226        8       8       0.11654         0.09   775      0.023226        8       8       0.20179         1      775      0.023226        8       8       0.41482         0
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep ^GIMAP1
GIMAP1-GIMAP5   0.2016256000 3.675602e-03 0.54826       11      775      0.068387        11      11      0.20668         0.25   775      0.068387        11      11      0.31419         1      775      0.068387        11      11      0.059376        1      775      0.068387        11      11      0.082922        0.09
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep ^ACHY
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep ^AHCY
AHCY            0.8046793000 4.888746e-03 0.69767       7       775      0.069677        7       7       0.80258         0      775      0.069677        7       7       0.33972         0      775      0.069677        7       7       1       1      775      0.069677        7       7       0.02476         0
AHCYL2          0.4530100000 0.000000e+00 0.67543       2       775      0.0025806       2       2       0.45301         1      775      0.0025806       2       2       0.75939         0      775      0.0025806       2       2       0.36224         0      775      0.0025806       2       2       0.38378         1
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w ^AHCY
AHCY            0.8046793000 4.888746e-03 0.69767       7       775      0.069677        7       7       0.80258         0      775      0.069677        7       7       0.33972         0      775      0.069677        7       7       1       1      775      0.069677        7       7       0.02476         0
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w ^GIMAP5
GIMAP5          0.0225139900 8.590028e-04 0.30297       8       775      0.023226        8       8       0.02232         0.04   775      0.023226        8       8       0.11654         0.09   775      0.023226        8       8       0.20179         1      775      0.023226        8       8       0.41482         0
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w ^GIMAP1
GIMAP1-GIMAP5   0.2016256000 3.675602e-03 0.54826       11      775      0.068387        11      11      0.20668         0.25   775      0.068387        11      11      0.31419         1      775      0.068387        11      11      0.059376        1      775      0.068387        11      11      0.082922        0.09
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep ^GIMAP1[^-a-zA-Z0-9]
~~~

#Calculating NI

#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.txt | perl -lane 'if (($F[3] == 0) || ($F[9] == 0) || ($F[7] == 0)) { push(@F, ("NI:", "-9")); } else { push(@F, ("NI:", (($F[5]/$F[3])/($F[9]/$F[7])))); } print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.wNI.txt
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.txt | perl -lane 'if (($F[3] == 0) || ($F[9] == 0) || ($F[7] == 0)) { push(@F, ("NI:", "-9")); } else { push(@F, ("NI:", (($F[5]/$F[3])/($F[9]/$F[7])))); } push(@F, ("NI_Plus1:", ((($F[5]+1)/($F[3]+1))/(($F[9]+1)/($F[7]+1))))); print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.wNI.txt
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.wOverlapCorrection.txt | perl -lane 'if (($F[3] == 0) || ($F[9] == 0) || ($F[7] == 0)) { push(@F, ("NI:", "-9")); } else { push(@F, ("NI:", (($F[5]/$F[3])/($F[9]/$F[7])))); } push(@F, ("NI_Plus1:", ((($F[5]+1)/($F[3]+1))/(($F[9]+1)/($F[7]+1))))); my $polySum = $F[3] + $F[5]; my $fixedSum = $F[7] + $F[9]; if (($polySum == 0) || ($fixedSum == 0)) { push(@F, ("SumRatio:", "-9")); } else { push(@F, ("SumRatio:", $polySum/$fixedSum)); } push(@F, ("SumRatio_Plus1:", ($polySum+1)/($fixedSum+1))); print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.wOverlapCorrection.wNIwSums.txt

~~~
EIF3I   0.7461500000    poly_syn:       2       poly_nonsyn:    1       fixed_syn:      14      fixed_nonsyn:   20      NI:     0.35    NI_Plus1:       0.476190476190476       SumRatio:       0.0882352941176471      SumRatio_Plus1: 0.114285714285714                                                                  
CCDC73  0.8142138000    poly_syn:       0       poly_nonsyn:    1       fixed_syn:      7       fixed_nonsyn:   10      NI:     -9      NI_Plus1:       1.45454545454545        SumRatio:       0.0588235294117647      SumRatio_Plus1: 0.111111111111111
TMPRSS9 0.1714198000    poly_syn:       0       poly_nonsyn:    2       fixed_syn:      26      fixed_nonsyn:   17      NI:     -9      NI_Plus1:       4.5     SumRatio:       0.0465116279069767      SumRatio_Plus1: 0.0681818181818182                                                                                 
KIR3DL1 0.0132004500    poly_syn:       0       poly_nonsyn:    2       fixed_syn:      15      fixed_nonsyn:   43      NI:     -9      NI_Plus1:       1.09090909090909        SumRatio:       0.0344827586206897      SumRatio_Plus1: 0.0508474576271186
KIR3DL2 0.1626752000    poly_syn:       2       poly_nonsyn:    4       fixed_syn:      42      fixed_nonsyn:   95      NI:     0.884210526315789       NI_Plus1:       0.746527777777778       SumRatio:       0.0437956204379562      SumRatio_Plus1: 0.0507246376811594                                                 
PRG4    0.7285340000    poly_syn:       2       poly_nonsyn:    2       fixed_syn:      46      fixed_nonsyn:   75      NI:     0.613333333333333       NI_Plus1:       0.618421052631579       SumRatio:       0.0330578512396694      SumRatio_Plus1: 0.040983606557377
~~~

#NOTE -- note the excess of fixed differences in PRG4, TMPRSS9, CCDC73 and EIF3I. KIR* are noted as 'rapidly evolving' already but no mention of TMPRSS9 in literature, PRG4 has been mentioned

#Permutations for calculating NI among top sets of genes

#Observed values
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.wOverlapCorrection.txt | sort -g -k 2,2 | R -q -e "Data1 <- read.table(file('stdin'), header=F); Top5NI <- c(sum(Data1[1:5,6]),sum(Data1[1:5,4]),sum(Data1[1:5,10]),sum(Data1[1:5,8])); Top10NI <- c(sum(Data1[1:10,6]),sum(Data1[1:10,4]),sum(Data1[1:10,10]),sum(Data1[1:10,8])); Top25NI <- c(sum(Data1[1:25,6]),sum(Data1[1:25,4]),sum(Data1[1:25,10]),sum(Data1[1:25,8])); Top50NI <- c(sum(Data1[1:50,6]),sum(Data1[1:50,4]),sum(Data1[1:50,10]),sum(Data1[1:50,8])); Top100NI <- c(sum(Data1[1:100,6]),sum(Data1[1:100,4]),sum(Data1[1:100,10]),sum(Data1[1:100,8])); print(rbind(Top5NI, Top10NI, Top25NI, Top50NI, Top100NI));" | head -n -2 | tail -n +3 | perl -ane 'print join(",", @F[1..$#F]), "\t"; if (eof()) { print "\n"; }'

~~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.wOverlapCorrection.txt | sort -g -k 2,2 | R -q -e "Data1 <- read.table(file('stdin'), header=F); Top5NI <- c(sum(Data1[1:5,6]),sum(Data1[1:5,4]),sum(Data1[1:5,10]),sum(Data1[1:5,8])); Top10NI <- c(sum(Data1[1:10,6]),sum(Data1[1:10,4]),sum(Data1[1:10,10]),sum(Data1[1:10,8])); Top25NI <- c(sum(Data1[1:25,6]),sum(Data1[1:25,4]),sum(Data1[1:25,10]),sum(Data1[1:25,8])); Top50NI <- c(sum(Data1[1:50,6]),sum(Data1[1:50,4]),sum(Data1[1:50,10]),sum(Data1[1:50,8])); Top100NI <- c(sum(Data1[1:100,6]),sum(Data1[1:100,4]),sum(Data1[1:100,10]),sum(Data1[1:100,8])); print(rbind(Top5NI, Top10NI, Top25NI, Top50NI, Top100NI));" | head -n -2 | tail -n +3 | perl -ane 'print join(",", @F[1..$#F]), "\t"; if (eof()) { print "\n"; }'
17,5,6,8        42,34,13,37     158,142,120,154 313,271,203,283 729,602,506,678 
~~~

perl /home/michaelt/Scripts/Perl/RandomList.pl --seed 345809345 --total 100000 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/RandomList.pl.Seed345809345.total100000.Output

#CHECK_1: Fix line length != 9 thing
#Should be fixed now due to grep [^-]$gene1[^-] correction?...
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.txt | sort -rg -k 2,2 | perl -lane 'if ($#F == 9) { print join("\t", @F); } ' | perl /home/michaelt/Scripts/Perl/RandomOrder.pl --file1 - --seed $SEED1 | R -q -e "Data1 <- read.table(file('stdin'), header=F); Top5NI <- c(sum(Data1[1:5,6]),sum(Data1[1:5,4]),sum(Data1[1:5,10]),sum(Data1[1:5,8])); Top10NI <- c(sum(Data1[1:10,6]),sum(Data1[1:10,4]),sum(Data1[1:10,10]),sum(Data1[1:10,8])); Top25NI <- c(sum(Data1[1:25,6]),sum(Data1[1:25,4]),sum(Data1[1:25,10]),sum(Data1[1:25,8])); Top50NI <- c(sum(Data1[1:50,6]),sum(Data1[1:50,4]),sum(Data1[1:50,10]),sum(Data1[1:50,8])); Top100NI <- c(sum(Data1[1:100,6]),sum(Data1[1:100,4]),sum(Data1[1:100,10]),sum(Data1[1:100,8])); print(rbind(Top5NI, Top10NI, Top25NI, Top50NI, Top100NI));" | head -n -2 | tail -n +3 | perl -ane 'print join(",", @F[1..$#F]), "\t"; if (eof()) { print "\n"; }' >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.Permutations.PreNIs.txt

rm -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.wOverlapCorrection.Permutations.txt

for i in {1..1000}
do
	
	SEED1=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/RandomList.pl.Seed345809345.total100000.Output | awk '{ if (NR == '$i') { print $0 } } '`

	cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.wOverlapCorrection.txt | sort -rg -k 2,2 | perl /home/michaelt/Scripts/Perl/RandomOrder.pl --file1 - --seed $SEED1 | R -q -e "Data1 <- read.table(file('stdin'), header=F); Top5NI <- c(sum(Data1[1:5,6]),sum(Data1[1:5,4]),sum(Data1[1:5,10]),sum(Data1[1:5,8])); Top10NI <- c(sum(Data1[1:10,6]),sum(Data1[1:10,4]),sum(Data1[1:10,10]),sum(Data1[1:10,8])); Top25NI <- c(sum(Data1[1:25,6]),sum(Data1[1:25,4]),sum(Data1[1:25,10]),sum(Data1[1:25,8])); Top50NI <- c(sum(Data1[1:50,6]),sum(Data1[1:50,4]),sum(Data1[1:50,10]),sum(Data1[1:50,8])); Top100NI <- c(sum(Data1[1:100,6]),sum(Data1[1:100,4]),sum(Data1[1:100,10]),sum(Data1[1:100,8])); print(rbind(Top5NI, Top10NI, Top25NI, Top50NI, Top100NI));" | head -n -2 | tail -n +3 | perl -ane 'print join(",", @F[1..$#F]), "\t"; if (eof()) { print "\n"; }' >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.wOverlapCorrection.Permutations.PreNIs.txt

done

#CHECK_0: Compare permutations to observed values somehow, both NI, sumRatio...total variant counts too?

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.txt.OLD1
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.try2.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.txt
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.txt.OLD2
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.try2.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.txt
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.txt.OLD3
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.try2.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.FD_Poly_OverlapSites.txt



#Pathway analysis based on 'pathway gene1 gene2 gene3...' type format of files

cat /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt | head -n 10 > /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt.top10

rm -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.wOverlapCorrection.c2.all.v4.0.symbols.gmt.wNIswSums.txt


OLD_IFS=$IFS
IFS=$'\n'

#for pathway1 in `cat /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt.top10`
for pathway1 in `cat /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt`
do

        IFS=$OLD_IFS
	pathwayCounts1=""

	for gene1 in `cat <(echo $pathway1 | perl -ane 'print join("\n", @F[2..$#F]);')`
	do
		pathwayCounts1+=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.wOverlapCorrection.txt | grep ^$gene1[^-a-zA-Z0-9] | perl -ane 'print join(",", ($F[3], $F[5], $F[7], $F[9])), "\\\n";'` 
	done

	if [ -z "$pathwayCounts1" ] ; 
	then
		echo -e "poly_syn: NA\tpoly_nonsyn: NA\tfixed_syn: NA\tfixed_nonsyn: NA\tNI: NA\tNI_plus1: NA\tSumRatio: NA\tSumRatio_plus1: NA" | paste <(echo $pathway1 | awk '{ print $1 }') - >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.wOverlapCorrection.c2.all.v4.0.symbols.gmt.wNIswSums.txt
	else
		echo -e $pathwayCounts1 | perl -F, -lane 'print join("\t", @F);' | R -q -e "Data1 <- read.table(file('stdin'), header=F); print(c(sum(Data1[,1]), sum(Data1[,2]), sum(Data1[,3]), sum(Data1[,4])));" | head -n -2 | tail -n +2 | perl -lane 'my $NI; my $sumRatio; if (($F[1] == 0) || ($F[4] == 0) || ($F[3] == 0)) { $NI = "-9"; } else { $NI = ($F[2]/$F[1]) / ($F[4]/$F[3]) ; } if (($F[1] + $F[2] == 0) || ($F[3] + $F[4] == 0)) { $sumRatio = "-9"; } else { $sumRatio = ($F[1] + $F[2]) / ($F[3] + $F[4]); } my $NI_plus1 = (($F[2]+1)/($F[1]+1)) / (($F[4]+1)/($F[3]+1)); my $sumRatio_plus1 = ($F[1] + $F[2] + 1) / ($F[3] + $F[4] + 1) ; print "poly_syn: ", $F[1], "\tpoly_nonsyn: ", $F[2], "\tfixed_syn: ", $F[3], "\tfixed_nonsyn: ", $F[4], "\tNI: ", $NI, "\tNI_plus1: ", $NI_plus1, "\tSumRatio: ", $sumRatio, "\tSumRatio_plus1: ", $sumRatio_plus1;' | paste <(echo $pathway1 | awk '{ print $1 }') - >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.wOverlapCorrection.c2.all.v4.0.symbols.gmt.wNIswSums.txt 
	fi

done

rm -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.wOverlapCorrection.20150427Array1KevinGeneGroupings.array_gene_groupings_03_31_2014.wNIswSums.txt 

OLD_IFS=$IFS
IFS=$'\n'

#for pathway1 in `cat /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt.top10`
for pathway2 in `cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20150427Array1KevinGeneGroupings.array_gene_groupings_03_31_2014.semicolons.SpaceToUnderscores.withHGNCdata.SplitIntoCategories.GroupFile.Exonic.Nonsynonymous.wSplice.pre`
do

        IFS=$OLD_IFS
	pathwayCounts2=""

	for gene2 in `cat <(echo $pathway2 | perl -ane 'print join("\n", split(/,/, $F[1]));')`
	do
#		echo $gene2
		pathwayCounts2+=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.wOverlapCorrection.txt | grep ^$gene2[^-a-zA-Z0-9] | perl -ane 'print join(",", ($F[3], $F[5], $F[7], $F[9])), "\\\n";'` 
	done

	if [ -z "$pathwayCounts2" ] ; 
	then
		echo -e "poly_syn: NA\tpoly_nonsyn: NA\tfixed_syn: NA\tfixed_nonsyn: NA\tNI: NA\tNI_plus1: NA\tSumRatio: NA\tSumRatio_plus1: NA" | paste <(echo $pathway2 | awk '{ print $1 }') - >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.wOverlapCorrection.20150427Array1KevinGeneGroupings.array_gene_groupings_03_31_2014.wNIswSums.txt
	else
		echo -e $pathwayCounts2 | perl -F, -lane 'print join("\t", @F);' | R -q -e "Data1 <- read.table(file('stdin'), header=F); print(c(sum(Data1[,1]), sum(Data1[,2]), sum(Data1[,3]), sum(Data1[,4])));" | head -n -2 | tail -n +2 | perl -lane 'my $NI; my $sumRatio; if (($F[1] == 0) || ($F[4] == 0) || ($F[3] == 0)) { $NI = "-9"; } else { $NI = ($F[2]/$F[1]) / ($F[4]/$F[3]) ; } if (($F[1] + $F[2] == 0) || ($F[3] + $F[4] == 0)) { $sumRatio = "-9"; } else { $sumRatio = ($F[1] + $F[2]) / ($F[3] + $F[4]); } my $NI_plus1 = (($F[2]+1)/($F[1]+1)) / (($F[4]+1)/($F[3]+1)); my $sumRatio_plus1 = ($F[1] + $F[2] + 1) / ($F[3] + $F[4] + 1) ; print "poly_syn: ", $F[1], "\tpoly_nonsyn: ", $F[2], "\tfixed_syn: ", $F[3], "\tfixed_nonsyn: ", $F[4], "\tNI: ", $NI, "\tNI_plus1: ", $NI_plus1, "\tSumRatio: ", $sumRatio, "\tSumRatio_plus1: ", $sumRatio_plus1;' | paste <(echo $pathway2 | awk '{ print $1 }') - >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MKTests/AllPools.Vs3.ChrAll.GATK.vcfannotate.vs3.wACs.hg19.panTro4.net.axt.AlignmentDifferences.Vs2.DropGaps.MKTest.wOverlapCorrection.20150427Array1KevinGeneGroupings.array_gene_groupings_03_31_2014.wNIswSums.txt 
	fi
done

#NOTE -- note the RING_ubiquitin_ligase_complex group potentially being very conserved? But also concerns about using ultra-rare variants biasing the MK test somehow?

~~~
Group_1_23      poly_syn: 7     poly_nonsyn: 13 fixed_syn: 16   fixed_nonsyn: 2 NI: 14.8571428571429    NI_plus1: 9.91666666666667      SumRatio: 1.11111111111111      SumRatio_plus1: 1.10526315789474
Best_name_for_group_cullin-RING_ubiquitin_ligase_complex        poly_syn: 7     poly_nonsyn: 13 fixed_syn: 16   fixed_nonsyn: 2 NI: 14.8571428571429    NI_plus1: 9.91666666666667      SumRatio: 1.11111111111111      SumRatio_plus1: 1.10526315789474
Group_1_4       poly_syn: 21    poly_nonsyn: 12 fixed_syn: 20   fixed_nonsyn: 1 NI: 11.4285714285714    NI_plus1: 6.20454545454546      SumRatio: 1.57142857142857      SumRatio_plus1: 1.54545454545455
Best_name_for_group_establishment_of_protein_localization       poly_syn: 21    poly_nonsyn: 12 fixed_syn: 20   fixed_nonsyn: 1 NI: 11.4285714285714    NI_plus1: 6.20454545454546      SumRatio: 1.57142857142857      SumRatio_plus1: 1.54545454545455
Group_1_9       poly_syn: 17    poly_nonsyn: 27 fixed_syn: 16   fixed_nonsyn: 3 NI: 8.47058823529412    NI_plus1: 6.61111111111111      SumRatio: 2.31578947368421      SumRatio_plus1: 2.25
Group_1_25      poly_syn: 21    poly_nonsyn: 27 fixed_syn: 18   fixed_nonsyn: 3 NI: 7.71428571428572    NI_plus1: 6.04545454545455      SumRatio: 2.28571428571429      SumRatio_plus1: 2.22727272727273
.
.
.
NK_Cell_mediated_cytotoxicity_1 poly_syn: 55    poly_nonsyn: 71 fixed_syn: 68   fixed_nonsyn: 95        NI: 0.924019138755981   NI_plus1: 0.924107142857143     SumRatio: 0.773006134969325     SumRatio_plus1: 0.774390243902439
Prot_Abund_Change_by_HIV_2      poly_syn: 75    poly_nonsyn: 90 fixed_syn: 65   fixed_nonsyn: 85        NI: 0.917647058823529   NI_plus1: 0.91891064871481      SumRatio: 1.1   SumRatio_plus1: 1.09933774834437
Vpu_degradation_1       poly_syn: 19    poly_nonsyn: 32 fixed_syn: 31   fixed_nonsyn: 62        NI: 0.842105263157895   NI_plus1: 0.838095238095238     SumRatio: 0.548387096774194     SumRatio_plus1: 0.553191489361702
_Network_Propagation_Complex_Histone_H3.3_complex       poly_syn: 20    poly_nonsyn: 12 fixed_syn: 14   fixed_nonsyn: 10        NI: 0.84        NI_plus1: 0.844155844155844     SumRatio: 1.33333333333333      SumRatio_plus1: 1.32
Linde_Virion_Incorporation_1    poly_syn: 32    poly_nonsyn: 57 fixed_syn: 37   fixed_nonsyn: 83        NI: 0.794051204819277   NI_plus1: 0.795093795093795     SumRatio: 0.741666666666667     SumRatio_plus1: 0.743801652892562
In_Virions_4    poly_syn: 12    poly_nonsyn: 14 fixed_syn: 12   fixed_nonsyn: 28        NI: 0.5 NI_plus1: 0.517241379310345     SumRatio: 0.65  SumRatio_plus1: 0.658536585365854
~~~

#Seroneg & Seropos stuff....probably don't need to do pathway stuff??










##20150514
##Tajima's D and Pi estimates per gene using vcftools

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries 

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --TajimaD 100 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.TajimaD100 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --TajimaD 1000 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.TajimaD1000 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --TajimaD 10000 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.TajimaD10000 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --TajimaD 100000 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.TajimaD100000 

~~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.TajimaD1000.Tajima.D | awk '{ if ($3 != 0) { print $0 } } ' | sort -rg -k 4,4 | awk '{ print $4 }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); quantile(Data1[,1]);"
> Data1 <- read.table(file('stdin'), header=F); quantile(Data1[,1]);
Error in quantile.default(Data1[, 1]) : factors are not allowed
Calls: quantile -> quantile.default
Execution halted
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.TajimaD1000.Tajima.D | awk '{ if ($3 != 0) { print $0 } } ' | sort -g -k 4,4 | awk '{ print $4 }' | R -q -e "Data1 <- read.table(file('stdin'), header=T); quantile(Data1[,1]);"
> Data1 <- read.table(file('stdin'), header=T); quantile(Data1[,1]);
0%        25%        50%        75%       100%
-2.4013300 -1.4753725 -1.1289850 -0.5087797  3.0772500
>
>
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.TajimaD100.Tajima.D | awk '{ if ($3 != 0) { print $0 } } ' | sort -g -k 4,4 | awk '{ print $4 }' | R -q -e "Data1 <- read.table(file('stdin'), header=T); quantile(Data1[,1]);"
> Data1 <- read.table(file('stdin'), header=T); quantile(Data1[,1]);
0%       25%       50%       75%      100%
-2.066840 -0.955893  0.000000  0.000000  3.643340
>
>
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.TajimaD10000.Tajima.D | awk '{ if ($3 != 0) { print $0 } } ' | sort -g -k 4,4 | awk '{ print $4 }' | R -q -e "Data1 <- read.table(file('stdin'), header=T); quantile(Data1[,1]);"
> Data1 <- read.table(file('stdin'), header=T); quantile(Data1[,1]);
0%       25%       50%       75%      100%
-2.630190 -1.859078 -1.493435 -1.080775  3.077250
>
>
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.TajimaD100000.Tajima.D | awk '{ if ($3 != 0) { print $0 } } ' | sort -g -k 4,4 | awk '{ print $4 }' | R -q -e "Data1 <- read.table(file('stdin'), header=T); quantile(Data1[,1]);"
> Data1 <- read.table(file('stdin'), header=T); quantile(Data1[,1]);
0%       25%       50%       75%      100%
-2.646720 -2.013553 -1.666365 -1.281275  1.082160
>
>
~~~

#SGPL1   10:72604246_A/G 10:72604263_G/T 10:72617424_A/G 10:72628170_G/C 10:72629578_T/A 10:72629610_A/G 10:72633127_G/T 10:72633216_C/T 10:72633291_G/A 10:72633336_C/T 10:72637078_C/T                                                                                                                                    
#SDF4    1:1163951_G/A   1:1164031_A/G   1:1164080_T/C

rm -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.TajimaD.AllGenes.All.output 
rm -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.TajimaD.AllGenes.Exonic.output 

for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs`
do

	vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --TajimaD 1000000000 --positions <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.All | grep ^$gene1[^-a-zA-Z0-9] | perl -lane 'foreach my $position (@F[1..$#F]) { print join("\t", (((split(/:/, ((split(/_/, $position))[0])))[0]), ((split(/:/, ((split(/_/, $position))[0])))[1])));; }') --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.TajimaD.AllGenes.All.temp
	cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.TajimaD.AllGenes.All.temp.Tajima.D | tail -n +2 | xargs echo $gene1 >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.TajimaD.AllGenes.All.output
	rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.TajimaD.AllGenes.All.temp.Tajima.D

done

~~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.TajimaD.AllGenes.All.output | awk '{ if ($4 != 0) { print $0 } } ' | sort -g -k 5,5 | awk '{ print $5 }' | R -q -e "Data1 <- read.table(file('stdin'), header=T); quantile(Data1[,1]);"
> Data1 <- read.table(file('stdin'), header=T); quantile(Data1[,1]);
0%       25%       50%       75%      100%
-2.592150 -2.039657 -1.693560 -1.333090  0.915266
>
>
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.TajimaD.AllGenes.Exonic.output | awk '{ if ($4 != 0) { print $0 } } ' | sort -g -k 5,5 | awk '{ print $5 }' | R -q -e "Data1 <- read.table(file('stdin'), header=T); quantile(Data1[,1]);"
> Data1 <- read.table(file('stdin'), header=T); quantile(Data1[,1]);
0%      25%      50%      75%     100%
-2.45554 -1.80618 -1.46423 -1.06769  0.77059
>
>
~~~

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --window-pi 100 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.WindowPi100
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --window-pi 1000 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.WindowPi1000
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --window-pi 10000 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.WindowPi10000
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --window-pi 100000 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.WindowPi100000

~~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.WindowPi100.windowed.pi | awk '{ if ($4 != 0) { print $0 } } ' | sort -rg -k 5,5 | head -n -1 | awk '{ print $5 }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); quantile(Data1[,1]);"
> Data1 <- read.table(file('stdin'), header=F); quantile(Data1[,1]);
0%         25%         50%         75%        100%
1.22655e-05 2.57898e-05 6.44495e-05 4.04640e-04 1.96397e-02
>
>
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.WindowPi1000.windowed.pi | awk '{ if ($4 != 0) { print $0 } } ' | sort -rg -k 5,5 | head -n -1 | awk '{ print $5 }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); quantile(Data1[,1]);"
> Data1 <- read.table(file('stdin'), header=F); quantile(Data1[,1]);
0%          25%          50%          75%         100%
1.227130e-06 2.433920e-05 1.379105e-04 5.011708e-04 7.152920e-03
>
>
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.WindowPi10000.windowed.pi | awk '{ if ($4 != 0) { print $0 } } ' | sort -rg -k 5,5 | head -n -1 | awk '{ print $5 }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); quantile(Data1[,1]);"
> Data1 <- read.table(file('stdin'), header=F); quantile(Data1[,1]);
0%          25%          50%          75%         100%
1.227030e-07 2.882072e-05 9.108220e-05 1.943298e-04 2.357070e-03
>
>
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.WindowPi100000.windowed.pi | awk '{ if ($4 != 0) { print $0 } } ' | sort -rg -k 5,5 | head -n -1 | awk '{ print $5 }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); quantile(Data1[,1]);"
> Data1 <- read.table(file('stdin'), header=F); quantile(Data1[,1]);
0%          25%          50%          75%         100%
2.555660e-08 1.076982e-05 2.861220e-05 5.652567e-05 5.580160e-04
>
>
~~~

rm -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.WindowPi.AllGenes.All.output
rm -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.WindowPi.AllGenes.Exonic.output

for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs`
do
	vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --window-pi 1000000000 --positions <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.All | grep ^$gene1[^-a-zA-Z0-9] | perl -lane 'foreach my $position (@F[1..$#F]) { print join("\t", (((split(/:/, ((split(/_/, $position))[0])))[0]), ((split(/:/, ((split(/_/, $position))[0])))[1])));; }') --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.WindowPi.AllGenes.All.temp
	cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.WindowPi.AllGenes.All.temp/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.WindowPi.AllGenes.All.temp.windowed.pi | tail -n +2 | xargs echo $gene1 >> 
done







        vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --TajimaD 1000000000 --positions <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.All | grep ^$gene1[^-a-zA-Z0-9] | perl -lane 'foreach my $position (@F[1..$#F]) { print join("\t", (((split(/:/, ((split(/_/, $position))[0])))[0]), ((split(/:/, ((split(/_/, $position))[0])))[1])));; }') --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.TajimaD.AllGenes.All.temp
        cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.TajimaD.AllGenes.All.temp.Tajima.D | tail -n +2 | xargs echo $gene1 >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.TajimaD.AllGenes.All.output
        rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.TajimaD.AllGenes.All.temp.Tajima.D




--window-pi




vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --TajimaD 100 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/PopgenSummaries/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.TajimaD100



glossary for me later
pathways in multiple routes mostly just doing the skat-O way, not the sumstat way
tajima D / pi at end


##20150825
##Redoing .pheno file and specific, associated analyses, or at least creating a log here and editting them back in the main file above, because of the single indinvidual mix-up Sudhir identified
##Also going to be creating a specific script here, or at least a one-liner, to check various aspects of the file on which the phenotype codes are based on for QC/cross-checking purposes
##Going to be copying/pasting all the lines of code I am rerunning from above, so there may a large number of duplicate entries in terms of 'logging' specifically-run code. But this section will provide lines of code I did specifically rerun even if naming is same as before

#20150825 CHECK_0: Do this stuff

#From Sudhir 20150403 email chain -- redoing/fixing/correcting "30430   POOL87_02RL" indivdual -- only individual found with issue
~~
...
I checked the CAMACS_phenotypes against the master status list and found one discrepancy.

        (1)     MACSID  30430   POOL87_02RL     is listed in the CAMACS dataset as HESN and is seropositive.  I moved to seropositive in my reconstruction.

	In deidentifying the sequencing pool-midtag IDÃ¢ÂÂ<80><99>s from MichaelÃ¢ÂÂ<80><99>s phenotype data, I could not resolve three samples.
	...
~~

#20150825? 20150913 CHECK_1: Create phenotype file QC check script #Ran a bunch of a one-liners, which can be seen in the main log portion around this time frame, that looks at and QCs the entries both for the original phenotype file and the coded phenotype file

#Put pheno redo correction here?
#
#Have to redo group-pheno level QC stats
#Perm files
#Pathway analyses
#Mktest?
#20150825
#Current to-do list
#
#Redo MultinRun TarAndClean and PermutationRun TarAndClean

#CHECK_0: Really need to redo those subsampling-to-P2 analyses, but on the individual .bam level files first?

#20150907 -- First is to just redo all the prep code and prep files, worry about redoing analyses once all the prep work/steps have been redone properly
#Then run the base-level SKAT-O analyses, run the scripts that collect that information, then the analyses that operate on top of those results
#Note -- all the code below was determined by copying/pasting a large portion of the log from '7th run...' onwards and determining which lines needed to be rerun. Additionaly lines of code were also included, generally referring to renaming previous '...pheno' files to '...vs2.pheno' files to begin this organization system of 'vs2', 'vs3', etc...

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs2.pheno

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.vs3.pheno | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.pheno

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.vs3.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs & 
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.vs3.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05 &
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.vs3.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.10PCs &
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.vs3.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.10PCs.maf05 &
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.vs3.pheno --mpheno 2 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno2 &
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.vs3.pheno --mpheno 3 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno3 &
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.vs3.pheno --mpheno 4 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno4 &

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.assoc.logistic Pheno1_justWhite_Logistic_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.assoc.logistic.QQplot.jpeg & 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.assoc.logistic Pheno1_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.assoc.logistic.QQplot.jpeg & 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.10PCs.assoc.logistic Pheno1_justWhite_Logistic_10PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.10PCs.assoc.logistic.QQplot.jpeg & 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.10PCs.maf05.assoc.logistic Pheno1_justWhite_Logistic_maf05_10PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.10PCs.maf05.assoc.logistic.QQplot.jpeg & 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.assoc.logistic Pheno2_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno2.assoc.logistic.QQplot.jpeg & 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.assoc.logistic Pheno3_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno3.assoc.logistic.QQplot.jpeg & 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.assoc.logistic Pheno4_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno4.assoc.logistic.QQplot.jpeg & 

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/.

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.FIDs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroNeg.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroNeg.FIDs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroHighlyExposedNeg.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroHighlyExposedNeg.FIDs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroPos.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroPos.FIDs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justRapidVRapid.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justRapidVRapid.FIDs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSlowVSlow.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSlowVSlow.FIDs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVeryRapid.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justVeryRapid.FIDs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVerySlow.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justVerySlow.FIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.vs3.pheno | awk '{ print $1 }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.FIDs 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.vs3.pheno | awk '{ if ($3 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justSeroNeg.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.vs3.pheno | awk '{ if ($4 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justSeroHighlyExposedNeg.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.vs3.pheno | awk '{ if ($3 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justSeroPos.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.vs3.pheno | awk '{ if ($5 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justRapidVRapid.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.vs3.pheno | awk '{ if ($5 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justSlowVSlow.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.vs3.pheno | awk '{ if ($6 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justVeryRapid.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.vs3.pheno | awk '{ if ($6 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte200.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justVerySlow.FIDs

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.frq.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.frq.count.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroNeg.frq.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroNeg.frq.count.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroHighlyExposedNeg.frq.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroHighlyExposedNeg.frq.count.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroPos.frq.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroPos.frq.count.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justRapidVRapid.frq.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justRapidVRapid.frq.count.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSlowVSlow.frq.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSlowVSlow.frq.count.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVeryRapid.frq.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVeryRapid.frq.count.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVerySlow.frq.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVerySlow.frq.count.gz

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3 & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justSeroNeg.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSeroNeg & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justSeroHighlyExposedNeg.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSeroHighlyExposedNeg & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justSeroPos.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSeroPos & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justRapidVRapid.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justRapidVRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justSlowVSlow.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSlowVSlow & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justVeryRapid.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justVeryRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justVerySlow.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justVerySlow & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3 & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justSeroNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSeroNeg & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justSeroHighlyExposedNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSeroHighlyExposedNeg & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justSeroPos.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSeroPos & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justRapidVRapid.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justRapidVRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justSlowVSlow.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSlowVSlow & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justVeryRapid.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justVeryRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justVerySlow.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justVerySlow & 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.AllPhenos.frq.AndCount.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs2.AllPhenos.frq.AndCount.gz

paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justVerySlow.frq.count | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.AllPhenos.frq.AndCount.gz
gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justVerySlow.frq.count
#gunzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSeroNeg.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSeroHighlyExposedNeg.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSeroPos.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justRapidVRapid.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSlowVSlow.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justVeryRapid.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justVerySlow.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSeroNeg.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSeroHighlyExposedNeg.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSeroPos.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justRapidVRapid.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSlowVSlow.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justVeryRapid.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justVerySlow.frq.count.gz

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.AllPhenos.frq.AndCount.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 30; $i <= 40; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.txt.gz 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.txt.gz | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/_2/) { print join(",", @F); } ' | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/_3/) { print join(",", @F); } ' | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/_4/) { print join(",", @F); } ' | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/_5/) { print join(",", @F); } ' | wc
R -q -e 'val1 = 1540 + 21 + 2; print (val1);'
R -q -e 'val1 = 1540 + 21 + 2; print (val1); val2 = 74824 - 1563; print (val2);'
~~~
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.txt.gz | wc                                                                                                                                                                                                                                                                                          65208   71864 29345920
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/_2/) { print join(",", @F); } ' | wc                                                                                                                                                                                                                       1328    1369  596287
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/_3/) { print join(",", @F); } ' | wc                                                                                                                                                                                                                       15      15    6635
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/_4/) { print join(",", @F); } ' | wc                                                                                                                                                                                      
2       2     949
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/_5/) { print join(",", @F); } ' | wc
0       0       0
[  michaelt@node1  ~]$R -q -e 'val1 = 1540 + 21 + 2; print (val1);'
> val1 = 1540 + 21 + 2; print (val1);
[1] 1563
> 
> 
[  michaelt@node1  ~]$R -q -e 'val1 = 1540 + 21 + 2; print (val1); val2 = 74824 - 1563; print (val2);'                                            
> val1 = 1540 + 21 + 2; print (val1); val2 = 74824 - 1563; print (val2);
[1] 1563
[1] 73261
> 
> 
~~~

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs3.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.All /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.SetIDFile.All
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.Exonic /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.SetIDFile.Exonic
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.wSplice /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.wSplice
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_PD /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_PD
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_D /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_D

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.All /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.All
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice.GeneIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice.GeneIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing") || ($7 == "UTR5;UTR3")) { print $8, "\t", $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.SetIDFile.All 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "splicing") || ($7 == "exonic;splicing")) { print $8, "\t", $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.SetIDFile.Exonic
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") )  { print $8, "\t", $2 } } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs | awk -F, '{ if ((($7 == "exonic") && (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV"))) || (($7 == "splicing") || ($7 == "exonic;splicing"))) { print $8, "\t", $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.wSplice
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($12 == "P") || ($12 == "D") ) { print $8, "\t", $2 } } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_PD
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $8, "\t", $2 } } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/SKATFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing") || ($7 == "UTR5;UTR3")) { print $0 } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.All
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "splicing") || ($7 == "exonic;splicing")) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") )  { print $0 } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs | awk -F, '{ if ((($7 == "exonic") && (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV"))) || (($7 == "splicing") || ($7 == "exonic;splicing"))) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($12 == "P") || ($12 == "D") ) { print $0 } } } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($13 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.wSplice /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.wSplice
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_D /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_D
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.Sift_D /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.Sift_D
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.LRT_D /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.LRT_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing") || ($7 == "UTR5;UTR3")) { print $0 } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "splicing") || ($7 == "exonic;splicing")) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") )  { print $0 } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs | awk -F, '{ if ((($7 == "exonic") && (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV"))) || (($7 == "splicing") || ($7 == "exonic;splicing"))) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.wSplice
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($12 == "P") || ($12 == "D") ) { print $0 } } } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_D
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.Sift_D
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($13 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.LRT_D

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs2.EPACTSedit.ped
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.EPACTSedit.ped.vs3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs2.EPACTSedit.ped.vs3

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.pheno | awk '{ print $1, "\t", $2, "\t0\t0\t0\t", $3, "\t", $4, "\t", $5, "\t", $6 } ' | sed 's/-9/NA/g' | perl -lane 'print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit.ped

join /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar | perl -lane 'splice(@F, 9, 2); splice(@F, 10, 1); print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit.ped.vs2

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit.ped.vs2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit.ped

#Add line #FAM_ID IND_ID FAT_ID MOT_ID SEX HIVPROG HIVPROGHE AIDS AIDSEXTR RMID PC1 PC2 PC3 PC4 PC5 PC6 PC7 PC8 PC9 PC10 to /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit.ped

ls -lrt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/. | awk '{ print $9 }' | perl -lane 'if ($F[0] =~ m/(AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.)(EPACTSedit.perm\d+.vs3.ped)/) { my $val1 = "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/" . $1 . $2; my $val2 = "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/" . $1 . "vs2." . $2; system("mv $val1 $val2"); }'

for i in {1..1500}
do
	SEED1=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.PermutationList.vs1.txt | awk '{ if (NR == '$i') { print $0 } } '`
	perl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs3.pl --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit.ped --seed $SEED1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit.perm${i}.vs3.ped 

#       echo $SEED1

done

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs 

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs

~~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit.perm1.vs3.ped | head -n 10                                 
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL98_01RL     POOL98_01RL     0       0       0       1       NA      NA      NA      1       0.0079  0.0160  0.0720  0.0296  0.0320  -0.0555 -0.0447 0.0014  0.0010  -0.0499
POOL83_08RL     POOL83_08RL     0       0       0       2       NA      NA      NA      8       0.0157  0.0095  -0.0198 -0.0248 -0.0206 -0.0242 0.0153  0.0448  0.0243  0.0067
POOL77_08RL     POOL77_08RL     0       0       0       1       NA      NA      NA      8       0.0143  -0.0026 -0.0279 -0.0273 0.0338  -0.0194 0.0555  0.0160  -0.0053 0.0202
POOL02_03RL     POOL02_03RL     0       0       0       2       2       NA      NA      3       0.0136  0.0057  0.0166  0.0415  -0.0098 0.0027  -0.0120 -0.0008 -0.0187 0.0112
POOL33_03RL     POOL33_03RL     0       0       0       2       2       NA      NA      3       0.0137  0.0162  -0.0260 -0.0403 -0.0116 -0.0230 -0.0308 0.0320  -0.0137 -0.0560
POOL157_04RL    POOL157_04RL    0       0       0       2       NA      NA      NA      4       0.0128  0.0025  -0.0309 0.0374  -0.0126 0.0054  0.0047  0.0157  -0.0291 0.0530
POOL88_02RL     POOL88_02RL     0       0       0       2       1       NA      NA      2       0.0118  0.0045  -0.0299 0.0170  0.0068  -0.0149 -0.0094 -0.0493 -0.0122 -0.0149
POOL109_11RL    POOL109_11RL    0       0       0       1       1       NA      NA      11      0.0160  0.0021  -0.0172 -0.0269 -0.0312 0.0343  0.0326  -0.0096 -0.0062 0.0452
POOL158_08RL    POOL158_08RL    0       0       0       2       2       1       NA      8       0.0106  0.0138  0.0268  -0.0544 0.0063  -0.0188 -0.0086 0.0534  -0.0106 -0.0414
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit.perm2.vs3.ped | head -n 10
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL98_01RL     POOL98_01RL     0       0       0       1       NA      NA      NA      1       0.0079  0.0160  0.0720  0.0296  0.0320  -0.0555 -0.0447 0.0014  0.0010  -0.0499
POOL83_08RL     POOL83_08RL     0       0       0       2       NA      NA      NA      8       0.0157  0.0095  -0.0198 -0.0248 -0.0206 -0.0242 0.0153  0.0448  0.0243  0.0067
POOL77_08RL     POOL77_08RL     0       0       0       2       NA      NA      NA      8       0.0143  -0.0026 -0.0279 -0.0273 0.0338  -0.0194 0.0555  0.0160  -0.0053 0.0202
POOL02_03RL     POOL02_03RL     0       0       0       1       2       NA      NA      3       0.0136  0.0057  0.0166  0.0415  -0.0098 0.0027  -0.0120 -0.0008 -0.0187 0.0112
POOL33_03RL     POOL33_03RL     0       0       0       2       2       NA      NA      3       0.0137  0.0162  -0.0260 -0.0403 -0.0116 -0.0230 -0.0308 0.0320  -0.0137 -0.0560
POOL157_04RL    POOL157_04RL    0       0       0       1       NA      NA      NA      4       0.0128  0.0025  -0.0309 0.0374  -0.0126 0.0054  0.0047  0.0157  -0.0291 0.0530
POOL88_02RL     POOL88_02RL     0       0       0       2       2       NA      NA      2       0.0118  0.0045  -0.0299 0.0170  0.0068  -0.0149 -0.0094 -0.0493 -0.0122 -0.0149
POOL109_11RL    POOL109_11RL    0       0       0       2       2       NA      NA      11      0.0160  0.0021  -0.0172 -0.0269 -0.0312 0.0343  0.0326  -0.0096 -0.0062 0.0452
POOL158_08RL    POOL158_08RL    0       0       0       1       1       1       NA      8       0.0106  0.0138  0.0268  -0.0544 0.0063  -0.0188 -0.0086 0.0534  -0.0106 -0.0414
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit.perm3.vs3.ped | head -n 10
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL98_01RL     POOL98_01RL     0       0       0       1       NA      NA      NA      1       0.0079  0.0160  0.0720  0.0296  0.0320  -0.0555 -0.0447 0.0014  0.0010  -0.0499
POOL83_08RL     POOL83_08RL     0       0       0       1       NA      NA      NA      8       0.0157  0.0095  -0.0198 -0.0248 -0.0206 -0.0242 0.0153  0.0448  0.0243  0.0067
POOL77_08RL     POOL77_08RL     0       0       0       2       NA      NA      NA      8       0.0143  -0.0026 -0.0279 -0.0273 0.0338  -0.0194 0.0555  0.0160  -0.0053 0.0202
POOL02_03RL     POOL02_03RL     0       0       0       2       2       NA      NA      3       0.0136  0.0057  0.0166  0.0415  -0.0098 0.0027  -0.0120 -0.0008 -0.0187 0.0112
POOL33_03RL     POOL33_03RL     0       0       0       1       2       NA      NA      3       0.0137  0.0162  -0.0260 -0.0403 -0.0116 -0.0230 -0.0308 0.0320  -0.0137 -0.0560
POOL157_04RL    POOL157_04RL    0       0       0       2       NA      NA      NA      4       0.0128  0.0025  -0.0309 0.0374  -0.0126 0.0054  0.0047  0.0157  -0.0291 0.0530
POOL88_02RL     POOL88_02RL     0       0       0       2       2       NA      NA      2       0.0118  0.0045  -0.0299 0.0170  0.0068  -0.0149 -0.0094 -0.0493 -0.0122 -0.0149
POOL109_11RL    POOL109_11RL    0       0       0       2       1       NA      NA      11      0.0160  0.0021  -0.0172 -0.0269 -0.0312 0.0343  0.0326  -0.0096 -0.0062 0.0452
POOL158_08RL    POOL158_08RL    0       0       0       2       2       1       NA      8       0.0106  0.0138  0.0268  -0.0544 0.0063  -0.0188 -0.0086 0.0534  -0.0106 -0.0414
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit.perm4.vs3.ped | head -n 10
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL98_01RL     POOL98_01RL     0       0       0       2       NA      NA      NA      1       0.0079  0.0160  0.0720  0.0296  0.0320  -0.0555 -0.0447 0.0014  0.0010  -0.0499
POOL83_08RL     POOL83_08RL     0       0       0       2       NA      NA      NA      8       0.0157  0.0095  -0.0198 -0.0248 -0.0206 -0.0242 0.0153  0.0448  0.0243  0.0067
POOL77_08RL     POOL77_08RL     0       0       0       2       NA      NA      NA      8       0.0143  -0.0026 -0.0279 -0.0273 0.0338  -0.0194 0.0555  0.0160  -0.0053 0.0202
POOL02_03RL     POOL02_03RL     0       0       0       1       1       NA      NA      3       0.0136  0.0057  0.0166  0.0415  -0.0098 0.0027  -0.0120 -0.0008 -0.0187 0.0112
POOL33_03RL     POOL33_03RL     0       0       0       2       2       NA      NA      3       0.0137  0.0162  -0.0260 -0.0403 -0.0116 -0.0230 -0.0308 0.0320  -0.0137 -0.0560
POOL157_04RL    POOL157_04RL    0       0       0       1       NA      NA      NA      4       0.0128  0.0025  -0.0309 0.0374  -0.0126 0.0054  0.0047  0.0157  -0.0291 0.0530
POOL88_02RL     POOL88_02RL     0       0       0       1       2       NA      NA      2       0.0118  0.0045  -0.0299 0.0170  0.0068  -0.0149 -0.0094 -0.0493 -0.0122 -0.0149
POOL109_11RL    POOL109_11RL    0       0       0       1       1       NA      NA      11      0.0160  0.0021  -0.0172 -0.0269 -0.0312 0.0343  0.0326  -0.0096 -0.0062 0.0452
POOL158_08RL    POOL158_08RL    0       0       0       1       1       1       NA      8       0.0106  0.0138  0.0268  -0.0544 0.0063  -0.0188 -0.0086 0.0534  -0.0106 -0.0414
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit.perm5.vs3.ped | head -n 10
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL98_01RL     POOL98_01RL     0       0       0       2       NA      NA      NA      1       0.0079  0.0160  0.0720  0.0296  0.0320  -0.0555 -0.0447 0.0014  0.0010  -0.0499
POOL83_08RL     POOL83_08RL     0       0       0       1       NA      NA      NA      8       0.0157  0.0095  -0.0198 -0.0248 -0.0206 -0.0242 0.0153  0.0448  0.0243  0.0067
POOL77_08RL     POOL77_08RL     0       0       0       1       NA      NA      NA      8       0.0143  -0.0026 -0.0279 -0.0273 0.0338  -0.0194 0.0555  0.0160  -0.0053 0.0202
POOL02_03RL     POOL02_03RL     0       0       0       1       2       NA      NA      3       0.0136  0.0057  0.0166  0.0415  -0.0098 0.0027  -0.0120 -0.0008 -0.0187 0.0112
POOL33_03RL     POOL33_03RL     0       0       0       2       1       NA      NA      3       0.0137  0.0162  -0.0260 -0.0403 -0.0116 -0.0230 -0.0308 0.0320  -0.0137 -0.0560
POOL157_04RL    POOL157_04RL    0       0       0       2       NA      NA      NA      4       0.0128  0.0025  -0.0309 0.0374  -0.0126 0.0054  0.0047  0.0157  -0.0291 0.0530
POOL88_02RL     POOL88_02RL     0       0       0       2       1       NA      NA      2       0.0118  0.0045  -0.0299 0.0170  0.0068  -0.0149 -0.0094 -0.0493 -0.0122 -0.0149
POOL109_11RL    POOL109_11RL    0       0       0       2       1       NA      NA      11      0.0160  0.0021  -0.0172 -0.0269 -0.0312 0.0343  0.0326  -0.0096 -0.0062 0.0452
POOL158_08RL    POOL158_08RL    0       0       0       2       2       1       NA      8       0.0106  0.0138  0.0268  -0.0544 0.0063  -0.0188 -0.0086 0.0534  -0.0106 -0.0414
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit.perm1.vs3.ped | head -n 150 | tail -n 10                   
POOL169_06RL    POOL169_06RL    0       0       0       2       2       NA      NA      6       0.0152  0.0020  0.0289  -0.0016 0.0194  0.0148  -0.0678 0.0267  -0.0218 0.0006
POOL54_05RL     POOL54_05RL     0       0       0       2       1       NA      NA      5       0.0100  0.0130  -0.0148 0.0023  0.0079  0.0065  0.0110  -0.0392 0.0006  0.0416
POOL70_07RL     POOL70_07RL     0       0       0       2       NA      NA      NA      7       0.0053  0.0011  0.0204  0.0040  0.0478  -0.0230 -0.0075 0.0162  0.0133  0.0037
POOL99_11RL     POOL99_11RL     0       0       0       2       2       NA      NA      11      0.0088  0.0084  -0.0249 0.0309  0.0547  -0.0064 -0.0122 -0.0504 0.0408  -0.0194
POOL09_06RL     POOL09_06RL     0       0       0       2       2       NA      NA      6       0.0122  0.0110  -0.0257 -0.0438 0.0198  0.0818  -0.0100 0.0120  -0.0152 0.0244
POOL125_05RL    POOL125_05RL    0       0       0       1       2       2       NA      5       0.0093  0.0019  0.0155  0.0372  0.0251  0.0403  0.0090  0.0138  -0.0130 0.0100
POOL157_11RL    POOL157_11RL    0       0       0       1       2       1       2       11      0.0142  0.0123  0.0236  -0.0230 0.0025  0.0099  0.0349  0.0111  0.0092  -0.0064
POOL80_01RL     POOL80_01RL     0       0       0       1       NA      NA      NA      1       0.0119  0.0110  -0.0337 -0.0046 -0.0187 0.0664  0.0401  0.0072  0.0304  -0.0416
POOL23_06RL     POOL23_06RL     0       0       0       2       2       NA      NA      6       0.0118  0.0045  0.0168  0.0124  -0.0158 0.0022  0.0210  -0.0111 0.0668  -0.0002
POOL173_01RL    POOL173_01RL    0       0       0       1       2       NA      NA      1       0.0104  0.0069  0.0257  -0.0448 0.0234  0.0036  0.0458  0.0089  0.0125  -0.0064
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit.perm2.vs3.ped | head -n 150 | tail -n 10
POOL169_06RL    POOL169_06RL    0       0       0       2       2       NA      NA      6       0.0152  0.0020  0.0289  -0.0016 0.0194  0.0148  -0.0678 0.0267  -0.0218 0.0006
POOL54_05RL     POOL54_05RL     0       0       0       1       2       NA      NA      5       0.0100  0.0130  -0.0148 0.0023  0.0079  0.0065  0.0110  -0.0392 0.0006  0.0416
POOL70_07RL     POOL70_07RL     0       0       0       1       NA      NA      NA      7       0.0053  0.0011  0.0204  0.0040  0.0478  -0.0230 -0.0075 0.0162  0.0133  0.0037
POOL99_11RL     POOL99_11RL     0       0       0       2       2       NA      NA      11      0.0088  0.0084  -0.0249 0.0309  0.0547  -0.0064 -0.0122 -0.0504 0.0408  -0.0194
POOL09_06RL     POOL09_06RL     0       0       0       1       2       NA      NA      6       0.0122  0.0110  -0.0257 -0.0438 0.0198  0.0818  -0.0100 0.0120  -0.0152 0.0244
POOL125_05RL    POOL125_05RL    0       0       0       1       2       2       NA      5       0.0093  0.0019  0.0155  0.0372  0.0251  0.0403  0.0090  0.0138  -0.0130 0.0100
POOL157_11RL    POOL157_11RL    0       0       0       1       2       1       2       11      0.0142  0.0123  0.0236  -0.0230 0.0025  0.0099  0.0349  0.0111  0.0092  -0.0064
POOL80_01RL     POOL80_01RL     0       0       0       1       NA      NA      NA      1       0.0119  0.0110  -0.0337 -0.0046 -0.0187 0.0664  0.0401  0.0072  0.0304  -0.0416
POOL23_06RL     POOL23_06RL     0       0       0       2       2       NA      NA      6       0.0118  0.0045  0.0168  0.0124  -0.0158 0.0022  0.0210  -0.0111 0.0668  -0.0002
POOL173_01RL    POOL173_01RL    0       0       0       2       1       NA      NA      1       0.0104  0.0069  0.0257  -0.0448 0.0234  0.0036  0.0458  0.0089  0.0125  -0.0064
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit.perm3.vs3.ped | head -n 150 | tail -n 10
POOL169_06RL    POOL169_06RL    0       0       0       1       2       NA      NA      6       0.0152  0.0020  0.0289  -0.0016 0.0194  0.0148  -0.0678 0.0267  -0.0218 0.0006
POOL54_05RL     POOL54_05RL     0       0       0       2       1       NA      NA      5       0.0100  0.0130  -0.0148 0.0023  0.0079  0.0065  0.0110  -0.0392 0.0006  0.0416
POOL70_07RL     POOL70_07RL     0       0       0       1       NA      NA      NA      7       0.0053  0.0011  0.0204  0.0040  0.0478  -0.0230 -0.0075 0.0162  0.0133  0.0037
POOL99_11RL     POOL99_11RL     0       0       0       1       1       NA      NA      11      0.0088  0.0084  -0.0249 0.0309  0.0547  -0.0064 -0.0122 -0.0504 0.0408  -0.0194
POOL09_06RL     POOL09_06RL     0       0       0       1       2       NA      NA      6       0.0122  0.0110  -0.0257 -0.0438 0.0198  0.0818  -0.0100 0.0120  -0.0152 0.0244
POOL125_05RL    POOL125_05RL    0       0       0       2       1       2       NA      5       0.0093  0.0019  0.0155  0.0372  0.0251  0.0403  0.0090  0.0138  -0.0130 0.0100
POOL157_11RL    POOL157_11RL    0       0       0       2       2       1       2       11      0.0142  0.0123  0.0236  -0.0230 0.0025  0.0099  0.0349  0.0111  0.0092  -0.0064
POOL80_01RL     POOL80_01RL     0       0       0       2       NA      NA      NA      1       0.0119  0.0110  -0.0337 -0.0046 -0.0187 0.0664  0.0401  0.0072  0.0304  -0.0416
POOL23_06RL     POOL23_06RL     0       0       0       2       1       NA      NA      6       0.0118  0.0045  0.0168  0.0124  -0.0158 0.0022  0.0210  -0.0111 0.0668  -0.0002
POOL173_01RL    POOL173_01RL    0       0       0       1       2       NA      NA      1       0.0104  0.0069  0.0257  -0.0448 0.0234  0.0036  0.0458  0.0089  0.0125  -0.0064
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit.perm4.vs3.ped | head -n 150 | tail -n 10
POOL169_06RL    POOL169_06RL    0       0       0       2       2       NA      NA      6       0.0152  0.0020  0.0289  -0.0016 0.0194  0.0148  -0.0678 0.0267  -0.0218 0.0006
POOL54_05RL     POOL54_05RL     0       0       0       2       2       NA      NA      5       0.0100  0.0130  -0.0148 0.0023  0.0079  0.0065  0.0110  -0.0392 0.0006  0.0416
POOL70_07RL     POOL70_07RL     0       0       0       2       NA      NA      NA      7       0.0053  0.0011  0.0204  0.0040  0.0478  -0.0230 -0.0075 0.0162  0.0133  0.0037
POOL99_11RL     POOL99_11RL     0       0       0       2       1       NA      NA      11      0.0088  0.0084  -0.0249 0.0309  0.0547  -0.0064 -0.0122 -0.0504 0.0408  -0.0194
POOL09_06RL     POOL09_06RL     0       0       0       2       2       NA      NA      6       0.0122  0.0110  -0.0257 -0.0438 0.0198  0.0818  -0.0100 0.0120  -0.0152 0.0244
POOL125_05RL    POOL125_05RL    0       0       0       1       2       1       NA      5       0.0093  0.0019  0.0155  0.0372  0.0251  0.0403  0.0090  0.0138  -0.0130 0.0100
POOL157_11RL    POOL157_11RL    0       0       0       1       1       1       2       11      0.0142  0.0123  0.0236  -0.0230 0.0025  0.0099  0.0349  0.0111  0.0092  -0.0064
POOL80_01RL     POOL80_01RL     0       0       0       2       NA      NA      NA      1       0.0119  0.0110  -0.0337 -0.0046 -0.0187 0.0664  0.0401  0.0072  0.0304  -0.0416
POOL23_06RL     POOL23_06RL     0       0       0       1       2       NA      NA      6       0.0118  0.0045  0.0168  0.0124  -0.0158 0.0022  0.0210  -0.0111 0.0668  -0.0002
POOL173_01RL    POOL173_01RL    0       0       0       1       2       NA      NA      1       0.0104  0.0069  0.0257  -0.0448 0.0234  0.0036  0.0458  0.0089  0.0125  -0.0064
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit.perm5.vs3.ped | head -n 150 | tail -n 10
POOL169_06RL    POOL169_06RL    0       0       0       2       2       NA      NA      6       0.0152  0.0020  0.0289  -0.0016 0.0194  0.0148  -0.0678 0.0267  -0.0218 0.0006
POOL54_05RL     POOL54_05RL     0       0       0       2       2       NA      NA      5       0.0100  0.0130  -0.0148 0.0023  0.0079  0.0065  0.0110  -0.0392 0.0006  0.0416
POOL70_07RL     POOL70_07RL     0       0       0       1       NA      NA      NA      7       0.0053  0.0011  0.0204  0.0040  0.0478  -0.0230 -0.0075 0.0162  0.0133  0.0037
POOL99_11RL     POOL99_11RL     0       0       0       2       1       NA      NA      11      0.0088  0.0084  -0.0249 0.0309  0.0547  -0.0064 -0.0122 -0.0504 0.0408  -0.0194
POOL09_06RL     POOL09_06RL     0       0       0       1       1       NA      NA      6       0.0122  0.0110  -0.0257 -0.0438 0.0198  0.0818  -0.0100 0.0120  -0.0152 0.0244
POOL125_05RL    POOL125_05RL    0       0       0       1       1       2       NA      5       0.0093  0.0019  0.0155  0.0372  0.0251  0.0403  0.0090  0.0138  -0.0130 0.0100
POOL157_11RL    POOL157_11RL    0       0       0       1       1       1       1       11      0.0142  0.0123  0.0236  -0.0230 0.0025  0.0099  0.0349  0.0111  0.0092  -0.0064
POOL80_01RL     POOL80_01RL     0       0       0       2       NA      NA      NA      1       0.0119  0.0110  -0.0337 -0.0046 -0.0187 0.0664  0.0401  0.0072  0.0304  -0.0416
POOL23_06RL     POOL23_06RL     0       0       0       2       2       NA      NA      6       0.0118  0.0045  0.0168  0.0124  -0.0158 0.0022  0.0210  -0.0111 0.0668  -0.0002
POOL173_01RL    POOL173_01RL    0       0       0       1       1       NA      NA      1       0.0104  0.0069  0.0257  -0.0448 0.0234  0.0036  0.0458  0.0089  0.0125  -0.0064
~~~

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs2.sh
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs3.sh

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/WRONG1.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.WRONG1.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts
ls -lrt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/. | awk '{ print $9 }' | grep ^DataProcessing | perl -lane 'my $val1 = "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/" . $F[0]; my $val2 = "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4." . $F[0]; system("mv $val1 $val2");'
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/WRONG1.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/OLD4.WRONG1.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/WRONG1.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/OLD4.WRONG1.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.jpg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.jpg
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/WRONG1.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/OLD4.WRONG1.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno1 Exonic.Nonsynonymous OLD4
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno2 Exonic.Nonsynonymous OLD4
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno3 Exonic.Nonsynonymous OLD4
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno4 Exonic.Nonsynonymous OLD4

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.jpg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.jpg
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.rnk.SUMSTAT /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs1.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.rnk.SUMSTAT
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.jpg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.jpg
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.jpg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.jpg
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.jpg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.jpg

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno1 Exonic.Nonsynonymous OLD4 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno2 Exonic.Nonsynonymous OLD4 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno3 Exonic.Nonsynonymous OLD4 
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno4 Exonic.Nonsynonymous OLD4 

#20150918 Note -- Mistake, clobbered old files that already existed
####cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs2.sh
####cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs2.sh

#20150921 Note -- So I messed up and clobbered those files as I mentioned above. However, I think there wasn't a lot of differences between them and the last version of the 'vs1' files. Also, I can mostly use the differences between what files are produced by the last 'vs1' file and what's in the directory from the previous run of the 'vs2' files to figure out what's missing. So going to create a 'vs3' of this file now and just leave the 'vs2' stuff as 'nonexistent' for now. 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs3.sh

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs3.sh

#Code/scratch-notes used to help fix/diagnose/recover things
~~~~~~
~~~
[  michaelt@login2  ~]$scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Results/Checkpoints/7thRun/20150302/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/TempStuff/.
[  michaelt@login2  ~]$scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/OLD4*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/TempStuff/.
Password:
OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.histograms.jpeg                                                100%  237KB 237.0KB/s   00:00    
OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg                                               100%  224KB 223.8KB/s   00:00    
OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg                       100%  285KB 285.0KB/s   00:00    
[  michaelt@login2  ~]$scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/OLD4*jp*g mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/TempStuff/.
Password:
OLD4.AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm 100%  354KB 353.5KB/s   00:00    
OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.jpg                                                            100%  255KB 255.4KB/s   00:00    
#On MacBookAir
Michaels-Air:TempStuff mturchin20$ scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/TempStuff/OLD4*jp*g .
Password:
OLD4.AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm 100%  354KB 353.5KB/s   00:01    
OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.jpg                                                            100%  255KB 255.4KB/s   00:01    
OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg                       100%  285KB 285.0KB/s   00:00    
OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.histograms.jpeg                                                100%  237KB 237.0KB/s   00:00    
OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg                                               100%  224KB 223.8KB/s   00:00    
~~~

4 plots are taken care of by new vs2 qq plut w/ permutations file
rnkcmsm gets taken care of by R file and by prmutation file
.jpg file was same as histograms.jpeg
Array1 not really used in MultiRun file? ...don't think so unless something obvious comes from starting to look at P2 stuff next
track down where the multicolumn .wVarCounts things began/started or just how to go about adding a few 3-4 of them/those?

#Notes
the Runs.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts has like 3-4 extra pasted entries from other runs just to see/compare that things are consistent in the runs/permutations
looks like it's maybe only done in the 'Run' section not the multi permutation part

scp -p the missing jpeg files over to current laptop to just see/view them and see what's different
Array1 variable used mainly for plotting purposes

I think I added the following R code to the permutation run file:

##cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm*.epacts | awk '{ print $10 }' | grep -v -E 'NA|PVALUE' | sort -rg -k 1,1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.permAll.epacts.justpVals
##cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm*.epacts | awk '{ print $10 }' | grep -v -E 'NA|PVALUE' | sort -rg -k 1,1 | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.permAll.epacts.NullDistrQQPlotCheck.vs1.jpeg\", width=2500, height=2000, res=300); xVals1 <- c(1:nrow(Data1))/(nrow(Data1)); plot(rev(-log10(xVals1)), -log10(Data1[,1]), xlab=\"-log10 Expected p-Values\", ylim=c(0,6), ylab=\"-log10 Observed p-Values\", main=\"HIVAcquisition -- All Permuted p-Values\", col=\"RED\"); abline(a=0,b=1); dev.off();"

#CHECK_1 -- plot all permuted p-vals at oncee against 'expected' in a QQ-plot to see that things look correct

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.permAll.epacts.NullDistrQQPlotCheck.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.permAll.epacts.NullDistrQQPlotCheck.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.permAll.epacts.NullDistrQQPlotCheck.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.permAll.epacts.NullDistrQQPlotCheck.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/.

~~~~~~


bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs3.sh MM5 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs TruePheno Pheno1 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs3.sh MM5 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs TruePheno Pheno2 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs3.sh MM5 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs TruePheno Pheno3 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs3.sh MM5 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs TruePheno Pheno4 Exonic.Nonsynonymous

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.QQPlotUsingPermutations.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.QQPlotUsingPermutations.vs2.R.NotUsed

cp -p  /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.TrueVsFakePerRankCmSum.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.TrueVsFakePerRankCmSum.vs2.R

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs3.sh MM5 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs FakePheno Pheno1 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs3.sh MM5 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs FakePheno Pheno2 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs3.sh MM5 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs FakePheno Pheno3 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs3.sh MM5 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs FakePheno Pheno4 Exonic.Nonsynonymous

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcess*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/TempStuff/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcess*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/TempStuff/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcess*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/TempStuff/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcess*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/TempStuff/.

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcess*jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/All*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/TempStuff/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcess*jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/All*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/TempStuff/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcess*jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/All*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/TempStuff/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcess*jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/All*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/TempStuff/.

#From MacBook Air
scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/TempStuff/DataProcess*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/TempStuff/All*jpeg /Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/TempStuff/.

#rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno1.Exonic.Nonsynonymous /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno2.Exonic.Nonsynonymous /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno3.Exonic.Nonsynonymous /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno4.Exonic.Nonsynonymous

~~~
[  michaelt@login2  /data/userdata/pg/michaelt]$ls -lrt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno*/Exonic.Nonsynonymous/. | grep RedoList                                     
[  michaelt@login2  /data/userdata/pg/michaelt]$                                                                        
~~~

for i in {1..100}
do
	if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run${i}.epacts ] ; then
		echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.RedoList.TruePheno.Pheno1.Exonic.Nonsynonymous
	fi
	if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run${i}.epacts ] ; then
		echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.RedoList.TruePheno.Pheno2.Exonic.Nonsynonymous
	fi
	if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run${i}.epacts ] ; then
		echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.RedoList.TruePheno.Pheno3.Exonic.Nonsynonymous
	fi
	if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run${i}.epacts ] ; then
		echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.RedoList.TruePheno.Pheno4.Exonic.Nonsynonymous
	fi
done

~~~
[  michaelt@node1  ~]$for i in {1..100}
> do
>         if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonym.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run${i}.epacts ] ; thenDropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs
>                 echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.RedoList.TruePheno.Pheno1.Exonic.Nonsynonymous
>         fi
>         if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonym.Exonic.Nonsynonymous.Pheno2.maf1.skat0.run${i}.epacts ] ; thenDropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs
>                 echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.RedoList.TruePheno.Pheno2.Exonic.Nonsynonymous
>         fi
>         if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonym.Exonic.Nonsynonymous.Pheno3.maf1.skat0.run${i}.epacts ] ; thenDropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs
>                 echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.RedoList.TruePheno.Pheno3.Exonic.Nonsynonymous
>         fi
>         if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonym.Exonic.Nonsynonymous.Pheno4.maf1.skat0.run${i}.epacts ] ; thenDropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs
>                 echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.RedoList.TruePheno.Pheno4.Exonic.Nonsynonymous
>         fi
> done
[  michaelt@node1  ~]$ls -lrt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno*/Exonic.Nonsynonymous/. | grep RedoList
[  michaelt@node1  ~]$
~~~

~~~
[  michaelt@login2  /data/userdata/pg/michaelt]$ls -lrt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno*/Exonic.Nonsynonymous/. | grep RedoList                                     
-r~~~w-rw-r-- 1 michaelt pg      377 Feb 18  2015 DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno1.Exonic.Nonsynonymous                                               
[  michaelt@login2  /data/userdata/pg/michaelt]$                                                                        
~~~

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno1.Exonic.Nonsynonymous /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno1.Exonic.Nonsynonymous

for i in {1..1000}
do
	if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm${i}.epacts ] ; then
		echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno1.Exonic.Nonsynonymous
	fi
	if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm${i}.epacts ] ; then
		echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno2.Exonic.Nonsynonymous
	fi
	if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm${i}.epacts ] ; then
		echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno3.Exonic.Nonsynonymous
	fi
	if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm${i}.epacts ] ; then
		echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno4.Exonic.Nonsynonymous
	fi
done

~~~
[  michaelt@login2  /data/userdata/pg/michaelt]$for i in {1..1000}
> do
>         if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm${i}.epacts ] ; then
>                 echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno1.Exonic.Nonsynonymous
>         fi
>         if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm${i}.epacts ] ; then
>                 echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno2.Exonic.Nonsynonymous
>         fi
>         if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm${i}.epacts ] ; then
>                 echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno3.Exonic.Nonsynonymous
>         fi
>         if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm${i}.epacts ] ; then
>                 echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno4.Exonic.Nonsynonymous
>         fi
> done
[  michaelt@login2  /data/userdata/pg/michaelt]$
[  michaelt@login2  /data/userdata/pg/michaelt]$ls -lrt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno*/Exonic.Nonsynonymous/. | grep RedoList                                     
-rw-rw-r-- 1 michaelt pg      377 Feb 18  2015 OLD4.DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno1.Exonic.Nonsynonymous                                               
[  michaelt@login2  /data/userdata/pg/michaelt]$                                                                        
~~~

#Include null distribution in cp -p / scp -p backup statements

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Results/Checkpoints/7thRun/20150923

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno*/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno*.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno*/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno*.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno*/Exonic.Nonsynonymous/Data*jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno*/Exonic.Nonsynonymous/Data*jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno*/Exonic.Nonsynonymous/All*jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Results/Checkpoints/7thRun/20150923/.

#20150925 CHECK_0: Look at/over work desktop to make sure proper files/folders were overwritten with the below system/commands. Maybe just use the 'checkpoint' file/folder system instead for the desktop specifically?? 
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/Data*jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/Data*jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/All*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/Data*jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/Data*jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/All*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/Data*jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/Data*jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/All*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/Data*jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/Data*jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/All*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/.

#20150925 Note -- things vary a little as the p-values become closer but the same groups of IDs exist in the same portions of the p-value distribution. Encouragingly, the top set of genes are essentially the same (like top 5-10 or so)
#paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput) | vi -

#20150330
#Retroactively including code for these 'gene/variant level result summary outputs', following the logic of excel sheet exploration, here

echo ".3" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .3) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .3) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".5" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".6" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".65" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .65) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .65) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".7" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".75" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .75) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .75) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".8" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo "1" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= 1) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= 1) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`

~~~
[  michaelt@login2  /data/userdata/pg/michaelt]$echo ".3" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .3) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .3) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.3 3 13
[  michaelt@login2  /data/userdata/pg/michaelt]$echo ".5" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.5 61 397
[  michaelt@login2  /data/userdata/pg/michaelt]$echo ".6" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.6 152 1301
[  michaelt@login2  /data/userdata/pg/michaelt]$echo ".65" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .65) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .65) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.65 175 1472
[  michaelt@login2  /data/userdata/pg/michaelt]$echo ".7" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.7 214 1817
[  michaelt@node1  ~]$echo ".75" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.Cput.wVarCounts | awk '{ if ($4 <= .75) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.pr9_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_ng.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .75) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @[1]; }' | sort | uniq | wc | awk '{ print $1 }'`/, $vals1[0]); print $vals2[0], "\t", $vals2 
.75 451 3552
[  michaelt@login2  /data/userdata/pg/michaelt]$echo ".8" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.8 473 3703
[  michaelt@login2  /data/userdata/pg/michaelt]$echo "1" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= 1) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= 1) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
1 473 3703
~~~

#20150325
#Adding new annotated .vcf format option now using vcf-annotate (http://vcftools.sourceforge.net/perl_module.html#vcf-annotate)
#Currently mainly being used to help pass on information to Sudhir for evaluation cost/design-strategy for variant follow-up genotyping
#CHECK_1: Getting vcf-annotate to work properly for the AC_... columns if this setup/format is desired to be continually used

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz
#mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz.tbi /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz.tbi

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.vcfannotate.vs3.wACs.vcf.gz


#20150925 Note -- I started using a new form of the grep statement associated with the "...wVarCounts" file here, e.g. grep ^"$gene"[^-0-9A-Za-z_] instead of grep -w. This is the correct way to go about things.
#20150930 Note -- Changed filename beginning from 'AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs...' to 'AllPools.Vs3.ChrAll.GATK.RRs...' due to filename length problems
cat <(echo "#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        PVAL    QVAL    REF     ALT     AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW") <(join <( python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.txt.gz | sort -k 1,1) <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^\# | perl -lane 'print join("_", @F[0..1]), "\t", $F[3], "\t", $F[4];' | sort -k 1,1) | perl -lane '$F[1] = join("|", split(/[\;,\,]/, $F[1])); $F[2] = join("|", split(/[\;,\,]/, $F[2])); print join("\t", @F);' | perl -lane 'my @geneEntries = split(/\|/, $F[2]); my @pVals; my @qVals; foreach my $gene (@geneEntries) { $gene = ((split(/\(/, $gene))[0]); my $geneEntry = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep ^"$gene"[^-0-9A-Za-z_]`; if ($geneEntry) { my @geneVals1 = split(/\s+/, $geneEntry); push(@pVals, $geneVals1[1]); push(@qVals, $geneVals1[3]); } else { push(@pVals, "NA"); push(@qVals, "NA"); } } splice(@F, 5, 0, (join("|", @pVals), join("|", @qVals))); print join("\t", @F);' | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSeroNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSeroHighlyExposedNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSeroPos.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justRapidVRapid.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSlowVSlow.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | perl -lane 'splice(@F, 0, 1, (((split(/_/, $F[0]))[0]), ((split(/_/, $F[0]))[1]), ((split(/_/, $F[0]))[1]))); print join("\t", @F);' |
perl -ane 'print join("\t", @F[0..10]); my $alt = $F[10]; my $totAC = join("", ((split(/:/, $F[11]))[1])) + join("", ((split(/:/, $F[12]))[1])); print "\t", $totAC; for (my $i=11; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", ((split(/\:/, $F[$i]))[1]); } } print "\n";' | sort -g -k 1,1 -k 2,2 ) | /home/shared/software/tabix-0.2.5/bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz
###perl -ane 'print join("\t", @F[0..10]); my $alt = $F[10]; my $totAC = join("", ((split(/:/, $F[11]))[1])) + join("", ((split(/:/, $F[12]))[1])); print "\t", $totAC; for (my $i=11; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", ((split(/\:/, $F[$i]))[1]); } } print "\n";' | sort -g -k 1,1 -k 2,2 ) | perl -lane 'splice(@F, 3, 0, ($F[9], $F[10])); splice(@F, 11, 2); print join("\t", @F);' | /home/shared/software/tabix-0.2.5/bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz

#Note -- Notice in the first version, produced by using 'grep -w', the p-/q-values for ANKHD1 and ANKHD1-EIF4EBP3 are the same, whereas in the second version ANKHD1 has a different (and the correct) p-/q-values. This is because if you just 'grep -w' with ANKHD1 you get ANKHD1-EIF4EBP3 first as well as ANKHD1, and so the perl code takes the ANKHD1-EIF4EBP3 line/values first 
~~~
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PA
Sts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | grep ANKHD1 | tail -n 10
5       139917800       139917800       exonic|exonic   ANKHD1|ANKHD1-EIF4EBP3  nonsynonymous_SNV,nonsynonymous_SNV     NM_017747:c.A7358G:p.N2453S,NM_017747:c.A7358G:p.N2453S 0.1394595000|0.1394595000       0.54483|0.54483
A       G       1550    1       0       0       1       0       1
5       139918155       139918155       intronic|intronic       ANKHD1|ANKHD1-EIF4EBP3  .,.     .,.     0.1394595000|0.1394595000       0.54483|0.54483 G       A       1550    3       2       1       1       0       0
5       139918390       139918390       intronic|intronic       ANKHD1|ANKHD1-EIF4EBP3  .,.     .,.     0.1394595000|0.1394595000       0.54483|0.54483 T       C       1532    2       0       0       2       0       0
5       139918410       139918410       intronic|intronic       ANKHD1|ANKHD1-EIF4EBP3  .,.     .,.     0.1394595000|0.1394595000       0.54483|0.54483 A       G       1524    2       1       1       1       0       1
5       139918421       139918421       intronic|intronic       ANKHD1|ANKHD1-EIF4EBP3  .,.     .,.     0.1394595000|0.1394595000       0.54483|0.54483 G       T       1526    1       0       0       1       0       0
5       139918554       139918554       exonic|exonic   ANKHD1|ANKHD1-EIF4EBP3  synonymous_SNV,synonymous_SNV   NM_017747:c.G7455A:p.Q2485Q,NM_017747:c.G7455A:p.Q2485Q 0.1394595000|0.1394595000       0.54483|0.54483 G     A        1544    1       1       1       0       0       0                                                                                                     
5       139918774       139918774       intronic|intronic       ANKHD1|ANKHD1-EIF4EBP3  .,.     .,.     0.1394595000|0.1394595000       0.54483|0.54483 C       T       1550    7       3       1       4       0       3      
5       139918923       139918923       intronic|intronic       ANKHD1|ANKHD1-EIF4EBP3  .,.     .,.     0.1394595000|0.1394595000       0.54483|0.54483 A       G       1550    1       0       0       1       0       0      
5       139919233       139919233       UTR3    ANKHD1  .       .       0.1394595000    0.54483 C       A       1550    1       0       0       1       0       1                                                              
5       139919537       139919537       intronic        ANKHD1-EIF4EBP3 .       .       0.1394595000    0.54483 C       T       1548    1       0       0       1       1       0                                              
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | grep ANKHD1 | tail -n 100
5       139917800       139917800       exonic|exonic   ANKHD1|ANKHD1-EIF4EBP3  nonsynonymous_SNV,nonsynonymous_SNV     NM_017747:c.A7358G:p.N2453S,NM_017747:c.A7358G:p.N2453S 0.1394952000|0.1394595000       0.54483|0.54483
A       G       1550    1       0       0       1       0       1
5       139918155       139918155       intronic|intronic       ANKHD1|ANKHD1-EIF4EBP3  .,.     .,.     0.1394952000|0.1394595000       0.54483|0.54483 G       A       1550    3       2       1       1       0       0
5       139918390       139918390       intronic|intronic       ANKHD1|ANKHD1-EIF4EBP3  .,.     .,.     0.1394952000|0.1394595000       0.54483|0.54483 T       C       1532    2       0       0       2       0       0
5       139918410       139918410       intronic|intronic       ANKHD1|ANKHD1-EIF4EBP3  .,.     .,.     0.1394952000|0.1394595000       0.54483|0.54483 A       G       1524    2       1       1       1       0       1
5       139918421       139918421       intronic|intronic       ANKHD1|ANKHD1-EIF4EBP3  .,.     .,.     0.1394952000|0.1394595000       0.54483|0.54483 G       T       1526    1       0       0       1       0       0
5       139918554       139918554       exonic|exonic   ANKHD1|ANKHD1-EIF4EBP3  synonymous_SNV,synonymous_SNV   NM_017747:c.G7455A:p.Q2485Q,NM_017747:c.G7455A:p.Q2485Q 0.1394952000|0.1394595000       0.54483|0.54483 G     A
1544    1       1       1       0       0       0
5       139918774       139918774       intronic|intronic       ANKHD1|ANKHD1-EIF4EBP3  .,.     .,.     0.1394952000|0.1394595000       0.54483|0.54483 C       T       1550    7       3       1       4       0       3
5       139918923       139918923       intronic|intronic       ANKHD1|ANKHD1-EIF4EBP3  .,.     .,.     0.1394952000|0.1394595000       0.54483|0.54483 A       G       1550    1       0       0       1       0       0
5       139919233       139919233       UTR3    ANKHD1  .       .       0.1394952000    0.54483 C       A       1550    1       0       0       1       0       1
5       139919537       139919537       intronic        ANKHD1-EIF4EBP3 .       .       0.1394595000    0.54483 C       T       1548    1       0       0       1       1       0
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | head -n 10
#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        PVAL    QVAL    REF     ALT     AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW
X       16863126        16863126        UTR3    RBBP7   .       .       0.8955400000    0.74690 C       T       1546    3       0       0       3       0       0
X       16863135        16863135        UTR3    RBBP7   .       .       0.8955400000    0.74690 G       A       1546    2       1       1       1       0       0
X       16863691        16863691        intronic        RBBP7   .       .       0.8955400000    0.74690 A       G       1546    2       2       2       0       0       0
X       16863783        16863783        intronic        RBBP7   .       .       0.8955400000    0.74690 T       C       1544    2       0       0       2       0       0
X       16863871        16863871        intronic        RBBP7   .       .       0.8955400000    0.74690 C       T       1542    5       3       1       2       0       0
X       16864025        16864025        exonic  RBBP7   nonsynonymous_SNV       NM_001198719:c.T1267C:p.F423L   0.8955400000    0.74690 A       G       1542    1       0       0       1       0       1
X       16864106        16864106        intronic        RBBP7   .       .       0.8955400000    0.74690 T       C       1538    6       0       0       6       0       2
X       16864144        16864144        intronic        RBBP7   .       .       0.8955400000    0.74690 T       C       1532    7       5       1       2       0       2
X       16866829        16866829        intronic        RBBP7   .       .       0.8955400000    0.74690 C       T       1512    2       1       1       1       0       0
[  michaelt@node1  ~]$mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz.tbi /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz.tbi
mv: accessing `/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz.tbi': File name too long
~~~

tabix -s 1 -b 2 -e 3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.PrepFor_vcfannotate.vs2.desc.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.PrepFor_vcfannotate.vs2.desc.txt

perl -e 'print
"key=INFO,ID=LOC,Number=1,Type=String,Description='\''Gene-oriented genomic location(s) of the variant (e.g. exonic, intronic, intergenic, etc...)'\''
key=INFO,ID=GENE,Number=1,Type=String,Description='\''Gene(s) associated with the variant'\''
key=INFO,ID=VARTYPE,Number=1,Type=String,Description='\''If exonic, type of coding-change variant'\''
key=INFO,ID=AACHANGE,Number=1,Type=String,Description='\''If coding-change variant, related protein- and transcript-change'\''
key=INFO,ID=PVAL,Number=1,Type=Integer,Description='\''p-value(s) associated with gene(s) in GENE field. NOTE -- p-value is gene-level based on using SKAT-O, only nonsynonymous variants within the specific gene and comparing seronegatives vs. seropositives. It is /NOT/ a variant-specific or variant-level p-value. NA indicates no p-value is available'\''
key=INFO,ID=QVAL,Number=1,Type=Integer,Description='\''q-value(s) associated with gene(s) in GENE field. Derived from the p-value distribution of all genes from the SKAT-O, nonsynonymous variants only, seronegative vs. seropositive analysis. It is /NOT/ a variant-specific or variant-level q-value. NA indicates no q-value is available'\''
key=INFO,ID=AC_ALL,Number=1,Type=Integer,Description='\''Total number of Ref and Alt allele copies (among white non-hispanics)'\''
key=INFO,ID=AC_TOTAL,Number=1,Type=Integer,Description='\''Total number of Alt allele copies (among white non-hispanics)'\''
key=INFO,ID=AC_SERONEG,Number=1,Type=Integer,Description='\''Seronegative number of Alt allele copies (among white non-hispanics)'\''
key=INFO,ID=AC_HESERONEG,Number=1,Type=Integer,Description='\''Highly-exposed seronegative number of Alt allele copies (among white non-hispanics)'\''
key=INFO,ID=AC_SEROPOS,Number=1,Type=Integer,Description='\''Seropositive number of Alt allele copies (among white non-hispanics)'\''
key=INFO,ID=AC_RAPID,Number=1,Type=Integer,Description='\''Rapid and very rapid AIDS progressor Alt allele copies (among white non-hispanics)'\''
key=INFO,ID=AC_SLOW,Number=1,Type=Integer,Description='\''Slow and very slow AIDS progressor Alt allele copies (among white non-hispanics)'\''\n"' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.PrepFor_vcfannotate.vs2.desc.txt

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | vcf-annotate -a /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz -d /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.PrepFor_vcfannotate.vs2.desc.txt -c CHROM,FROM,TO,INFO/LOC,INFO/GENE,INFO/VARTYPE,INFO/AACHANGE,INFO/PVAL,INFO/QVAL,REF,ALT,INFO/AC_ALL,INFO/AC_TOTAL,INFO/AC_SERONEG,INFO/AC_HESERONEG,INFO/AC_SEROPOS,INFO/AC_RAPID,INFO/AC_SLOW | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.vcfannotate.vs3.wACs.vcf.gz

#cd /home/michaelt/Software
#wget http://sourceforge.net/projects/vcftools/files/vcftools_0.1.13.tar.gz/download
#mv download vcftools_0.1.13.tar.gz
#tar -xvzf vcftools_0.1.13.tar.gz
#make

#QC Scripts
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.vcfannotate.vs3.wACs.vcf.gz | grep -v ^# | perl -lane 'print $#F;' | sort | uniq -c 
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.vcfannotate.vs3.wACs.vcf.gz | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_TOTAL=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | head -n 10
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.vcfannotate.vs3.wACs.vcf.gz | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_TOTAL=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | perl -lane 'if ($F[1] ne $F[2]) { print join("\t", @F); }' | wc
/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justSeroNeg.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'print $#F;' | sort | uniq -c 
/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justSeroNeg.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_SERONEG=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | head -n 10
/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justSeroNeg.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_SERONEG=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | perl -lane 'if ($F[1] ne $F[2]) { print join("\t", @F); }' | wc
/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justSeroHighlyExposedNeg.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'print $#F;' | sort | uniq -c 
/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justSeroHighlyExposedNeg.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_HESERONEG=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | head -n 10
/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justSeroHighlyExposedNeg.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_HESERONEG=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | perl -lane 'if ($F[1] ne $F[2]) { print join("\t", @F); }' | wc
/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justSeroPos.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'print $#F;' | sort | uniq -c 
/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justSeroPos.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_SEROPOS=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | head -n 10
/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justSeroPos.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_SEROPOS=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | perl -lane 'if ($F[1] ne $F[2]) { print join("\t", @F); }' | wc
/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justRapidVRapid.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'print $#F;' | sort | uniq -c 
/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justRapidVRapid.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_RAPID=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | head -n 10
/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justRapidVRapid.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_RAPID=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | perl -lane 'if ($F[1] ne $F[2]) { print join("\t", @F); }' | wc
/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justSlowVSlow.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'print $#F;' | sort | uniq -c 
/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justSlowVSlow.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_SLOW=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | head -n 10
/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.vcfannotate.vs3.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.pheno.justSlowVSlow.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_SLOW=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | perl -lane 'if ($F[1] ne $F[2]) { print join("\t", @F); }' | wc

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.vcfannotate.vs3.wACs.vcf.gz | grep -v ^# | perl -ane 'my $GENE1 = "NA"; $PVAL1 = "NA"; $QVAL1 = "NA"; $GENE2 = "NA"; if ($F[7] =~ m/.*GENE=([0-9a-zA-Z]+)(;|\|).*/) { $GENE1 = $1; } if ($F[7] =~ m/.*PVAL=(\d\.\d+)(;|\|).*/) { $PVAL1 = $1; } if ($F[7] =~ m/.*QVAL=(\d\.\d+)(;|\|).*/) { $QVAL1 = $1; } if ($PVAL1 ne "NA") { $GENE2=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep ^'"'"'$GENE1'"'"'[^-0-9a-zA-Z_] | awk '"'"'{ print \$1, "\t", \$2, "\t", \$4 } '"'"'`; print $GENE1, "\t", $PVAL1, "\t", $QVAL1, "\t", $GENE2; } else { print $., "\t", $GENE1, "\t", $PVAL1, "\t", $QVAL1, "\t", $GENE2, "\n";} ' | grep -v NA | awk '{ if ($1 != $4) { print $0 } } ' | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.vcfannotate.vs3.wACs.vcf.gz | grep -v ^# | perl -ane 'my $GENE1 = "NA"; $PVAL1 = "NA"; $QVAL1 = "NA"; $GENE2 = "NA"; if ($F[7] =~ m/.*GENE=([0-9a-zA-Z]+)(;|\|).*/) { $GENE1 = $1; } if ($F[7] =~ m/.*PVAL=(\d\.\d+)(;|\|).*/) { $PVAL1 = $1; } if ($F[7] =~ m/.*QVAL=(\d\.\d+)(;|\|).*/) { $QVAL1 = $1; } if ($PVAL1 ne "NA") { $GENE2=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep ^'"'"'$GENE1'"'"'[^-0-9a-zA-Z_] | awk '"'"'{ print \$1, "\t", \$2, "\t", \$4 } '"'"'`; print $GENE1, "\t", $PVAL1, "\t", $QVAL1, "\t", $GENE2; } else { print $., "\t", $GENE1, "\t", $PVAL1, "\t", $QVAL1, "\t", $GENE2, "\n";} ' | grep -v NA | awk '{ if ($2 != $5) { print $0 } } ' | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.vcfannotate.vs3.wACs.vcf.gz | grep -v ^# | perl -ane 'my $GENE1 = "NA"; $PVAL1 = "NA"; $QVAL1 = "NA"; $GENE2 = "NA"; if ($F[7] =~ m/.*GENE=([0-9a-zA-Z]+)(;|\|).*/) { $GENE1 = $1; } if ($F[7] =~ m/.*PVAL=(\d\.\d+)(;|\|).*/) { $PVAL1 = $1; } if ($F[7] =~ m/.*QVAL=(\d\.\d+)(;|\|).*/) { $QVAL1 = $1; } if ($PVAL1 ne "NA") { $GENE2=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep ^'"'"'$GENE1'"'"'[^-0-9a-zA-Z_] | awk '"'"'{ print \$1, "\t", \$2, "\t", \$4 } '"'"'`; print $GENE1, "\t", $PVAL1, "\t", $QVAL1, "\t", $GENE2; } else { print $., "\t", $GENE1, "\t", $PVAL1, "\t", $QVAL1, "\t", $GENE2, "\n";} ' | grep -v NA | awk '{ if ($3 != $6) { print $0 } } ' | wc

~~~
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.vcfannotate.vs3.wACs.vcf.gz | grep -v ^# | perl -lane 'print $#F;' | sort | uniq -c  
  63863 783
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.vcfannotate.vs3.wACs.vcf.gz | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_TOTAL=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | head -n 10
1       2       2
2       26      26
3       3       3
4       56      56
5       2       2
6       225     225
7       25      25
8       4       4
9       2       2
10      6       6
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.vcfannotate.vs3.wACs.vcf.gz | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_TOTAL=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = split(/:/, $F[$i]); my @alleleVals = split(/\//, $indvVals[0]); if ($alleleVals[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | perl -lane 'if ($F[1] ne $F[2]) { print join("\t", @F); }' | wc
    0       0       0
[  michaelt@node1  ~]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASstSeroNeg.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'print $#F;' | sort | uniq -c
  63863 359
[  michaelt@node1  ~]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASstSeroNeg.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_SERONEG=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals =  $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | head -n 10
1 0     0
2 13    13
3 2     2
4 27    27
5 0     0
6 109   109
7 14    14
8 2     2
9 1     1
10      2       2
[  michaelt@node1  ~]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASstSeroNeg.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_SERONEG=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals =  $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | perl -lane 'if ($F[1] ne $F[2]) { print join("\t", @F); }' | wc
   0       0       0
[  michaelt@node1  ~]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASstSeroHighlyExposedNeg.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'print $#F;' | sort | uniq -c
  63863 175
[  michaelt@node1  ~]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASstSeroHighlyExposedNeg.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_HESERONEG=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { ls[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | head -n 10
1       0       0
2       5       5
3       2       2
4       15      15
5       0       0
6       51      51
7       5       5
8       0       0
9       1       1
10      0       0
[  michaelt@node1  ~]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASstSeroHighlyExposedNeg.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_HESERONEG=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { ls[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | perl -lane 'if ($F[1] ne $F[2]) { print join("\t", @F); }' | wc
   0       0       0
[  michaelt@node1  ~]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASstSeroHighlyExposedNeg.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'print $#F;' | sort | uniq -c
  63863 175
[  michaelt@node1  ~]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASstSeroHighlyExposedNeg.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_HESERONEG=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { ls[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | head -n 10
1       0       0
2       5       5
3       2       2
4       15      15
5       0       0
6       51      51
7       5       5
8       0       0
9       1       1
10      0       0
[  michaelt@node1  ~]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASstSeroHighlyExposedNeg.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_HESERONEG=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { ls[0] eq "1") { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | perl -lane 'if ($F[1] ne $F[2]) { print join("\t", @F); }' | wc
   0       0       0
[  michaelt@node1  ~]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASstSeroPos.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'print $#F;' | sort | uniq -c
  63863 432
[  michaelt@node1  ~]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASstSeroPos.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_SEROPOS=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals =  $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | head -n 10
1       2       2
2       13      13
3       1       1
4       29      29
5       2       2
6       116     116
7       11      11
8       2       2
9       1       1
10      4       4
[  michaelt@node1  ~]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASstSeroPos.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_SEROPOS=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals =  $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | perl -lane 'if ($F[1] ne $F[2]) { print join("\t", @F); }' | wc
   0       0       0
[  michaelt@node1  ~]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASstRapidVRapid.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'print $#F;' | sort | uniq -c
63863 66
  [  michaelt@node1  ~]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASstRapidVRapid.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_RAPID=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals  { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | head -n 10
1       0       0
2       2       2
3       0       0
4       7       7
5       0       0
6       19      19
7       1       1
8       0       0
9       0       0
10      0       0
  [  michaelt@node1  ~]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASstRapidVRapid.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_RAPID=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals  { $ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | perl -lane 'if ($F[1] ne $F[2]) { print join("\t", @F); }' | wc
   0       0       0
[  michaelt@node1  ~]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASstSlowVSlow.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'print $#F;' | sort | uniq -c
63863 97
[  michaelt@node1  ~]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASstSlowVSlow.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_SLOW=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = s$ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | head -n 10
1       2       2
2       2       2
3       0       0
4       3       3
5       0       0
6       22      22
7       3       3
8       0       0
9       0       0
10      0       0
[  michaelt@node1  ~]$/home/michaelt/Software/vcftools_0.1.13/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wACs.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASstSlowVSlow.FIDs --recode --recode-INFO-all --stdout | grep -v ^# | perl -lane 'my $ACTOT1 = "NA"; my $ACTOT2 = "0"; if ($F[7] =~ m/.*(AC_SLOW=)(\d+).*/) { $ACTOT1 = $2; } for (my $i = 9; $i <= $#F; $i++) { my @indvVals = s$ACTOT2++; } if ($alleleVals[1] eq "1") { $ACTOT2++; } } print $., "\t", $ACTOT1, "\t", $ACTOT2; ' | perl -lane 'if ($F[1] ne $F[2]) { print join("\t", @F); }' | wc
   0       0       0

[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.vcfannotate.vs3.wACs.vcf.gz | grep -v ^# | perl -ane 'my $GENE1 = "NA"; $PVAL1 = "NA"; $QVAL1 = "NA"; $GENE2 = "NA"; if ($F[7] =~ m/.*GENE=([0-9a-zA-Z]+)(;|\|).*/) { $GENE1 = $1; } if ($F[7] =~ m/.*PVAL=(\d\.\d+)(;|\|).*/) { $PVAL1 = $1; } if ($F[7] =~ m/.*QVAL=(\d\.\d+)(;|\|).*/) { $QVAL1 = $1; } if ($PVAL1 ne "NA") { $GENE2=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep ^'"'"'$GENE1'"'"'[^-0-9a-zA-Z_] | awk '"'"'{ print \$1, "\t", \$2, "\t", \$4 } '"'"'`; print $GENE1, "\t", $PVAL1, "\t", $QVAL1, "\t", $GENE2; } else { print $., "\t", $GENE1, "\t", $PVAL1, "\t", $QVAL1, "\t", $GENE2, "\n";} ' | grep -v NA | awk '{ if ($1 != $4) { print $0 } } ' | wc
    0       0       0
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.vcfannotate.vs3.wACs.vcf.gz | grep -v ^# | perl -ane 'my $GENE1 = "NA"; $PVAL1 = "NA"; $QVAL1 = "NA"; $GENE2 = "NA"; if ($F[7] =~ m/.*GENE=([0-9a-zA-Z]+)(;|\|).*/) { $GENE1 = $1; } if ($F[7] =~ m/.*PVAL=(\d\.\d+)(;|\|).*/) { $PVAL1 = $1; } if ($F[7] =~ m/.*QVAL=(\d\.\d+)(;|\|).*/) { $QVAL1 = $1; } if ($PVAL1 ne "NA") { $GENE2=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep ^'"'"'$GENE1'"'"'[^-0-9a-zA-Z_] | awk '"'"'{ print \$1, "\t", \$2, "\t", \$4 } '"'"'`; print $GENE1, "\t", $PVAL1, "\t", $QVAL1, "\t", $GENE2; } else { print $., "\t", $GENE1, "\t", $PVAL1, "\t", $QVAL1, "\t", $GENE2, "\n";} ' | grep -v NA | awk '{ if ($2 != $5) { print $0 } } ' | wc                                                                                                     
    0       0       0
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.vcfannotate.vs3.wACs.vcf.gz | grep -v ^# | perl -ane 'my $GENE1 = "NA"; $PVAL1 = "NA"; $QVAL1 = "NA"; $GENE2 = "NA"; if ($F[7] =~ m/.*GENE=([0-9a-zA-Z]+)(;|\|).*/) { $GENE1 = $1; } if ($F[7] =~ m/.*PVAL=(\d\.\d+)(;|\|).*/) { $PVAL1 = $1; } if ($F[7] =~ m/.*QVAL=(\d\.\d+)(;|\|).*/) { $QVAL1 = $1; } if ($PVAL1 ne "NA") { $GENE2=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep ^'"'"'$GENE1'"'"'[^-0-9a-zA-Z_] | awk '"'"'{ print \$1, "\t", \$2, "\t", \$4 } '"'"'`; print $GENE1, "\t", $PVAL1, "\t", $QVAL1, "\t", $GENE2; } else { print $., "\t", $GENE1, "\t", $PVAL1, "\t", $QVAL1, "\t", $GENE2, "\n";} ' | grep -v NA | awk '{ if ($3 != $6) { print $0 } } ' | wc
   0       0       0
~~~

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.AllPools.Vs3.annotated.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/OLD3.MM5.AllPools.Vs3.annotated.vcf.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/OLD3.MM5.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.ForSudhir.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/OLD3.MM5.skatO.nonsyn.qVals.ACs.GeneLevel.ForSudhir.txt

for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs | head -n 10`
do

	paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane 'if ($F[0] =~ m/^'$gene1'$/) { print join("\t", @F); }' | awk '{ print $1, "\t", $2, "\t", $4, "\t", $5, "\t", $8, "\t", $9, "\t", $11 }' ) \
	<(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | \
	grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'if ($F[0] =~ m/^'$gene1'$/) { print join("\t", @F); }' | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }') | \
	awk '{ print $12, "\t", $13, "\t", $14, "\t", $15, "\t", $16, "\t", $17, "\t", $18 }' | \
	R -q -e "Data1 <- read.table(file('stdin'), header=F); print(paste(apply(Data1, 2, sum)), quote=FALSE);" | grep -v ^\> | perl -lane 'splice(@F, 0, 1); print join("\t", @F);') \
	>> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt
	
done

#	paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane 'if ($F[0] =~ m/^'$gene1'$/) { print join("\t", @F); }' | awk '{ print $1, "\t", $2, "\t", $4, "\t", $5, "\t", $8, "\t", $9, "\t", $11 }' ) <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'if ($F[0] =~ m/^'$gene1'$/) { print join("\t", @F); }' | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }') | awk '{ print $12, "\t", $13, "\t", $14, "\t", $15, "\t", $16, "\t", $17, "\t", $18 }' | R -q -e "Data1 <- read.table(file('stdin'), header=F); print(paste(apply(Data1, 2, sum)), quote=FALSE);" | grep -v ^\> | perl -lane 'splice(@F, 0, 1); print join("\t", @F);') >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt | perl -lane 'splice(@F, 4, 3); print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt 

#Included the below header in file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt
#GENE   PVAL    QVAL    NUMVAR    AC_ALL        AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW

#QC Checks on file
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt | wc
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt | perl -lane 'if (($F[3] == $F[4]) && ($F[3] == $F[5]) && ($F[4] == $F[5])) { print join("\t", @F); } '  | wc
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt | perl -lane 'if (($F[3] != $F[4]) || ($F[3] != $F[5]) || ($F[4] != $F[5])) { print join("\t", @F); } '
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[5] != $val2) { print join("\t", @F); }'
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[5] == $val2) { print join("\t", @F); }' | wc
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[7] > $F[6]) { print join("\t", @F); }'
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[7] <= $F[6]) { print join("\t", @F); }' | wc
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[9] + $F[10]; if ($F[8] < $val2) { print join("\t", @F); }'
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[9] + $F[10]; if ($F[8] >= $val2) { print join("\t", @F); }' | wc

~~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt | wc                           
474    5207   23994
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt | perl -lane 'if (($F[3] == $F[4]) && ($F[3] == $F[5]) && ($F[4] == $F[5])) { print join("\t", @F); } '  | wc
474    6629   29855
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt | perl -lane 'if (($F[3] != $F[4]) || ($F[3] != $F[5]) || ($F[4] != $F[5])) { print join("\t", @F); } '                       
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[5] != $val2) { print join("\t", @F); }'
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[5] == $val2) { print join("\t", @F); }' | wc                                      
474    5207   23994
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[7] > $F[6]) { print join("\t", @F); }'
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[7] <= $F[6]) { print join("\t", @F); }' | wc                               
474    5207   23994
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[9] + $F[10]; if ($F[8] < $val2) { print join("\t", @F); }'
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt | perl -lane 'my $val2 = $F[9] + $F[10]; if ($F[8] >= $val2) { print join("\t", @F); }' | wc                  
474    5207   23994
[  michaelt@node1  ~]$
~~~

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.vcfannotate.vs3.wACs.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.AllPools.Vs3.annotated.vcf.gz

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz 

#Difference between 141 and 142 due to '#CHROM...' header in /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz
~~~
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz | wc
  63864 1149552 5545409                                                                                                                                                                                                                                                                                                    
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | wc
  63864 1149552 5545409
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.AllPools.Vs3.annotated.vcf.gz | wc
  64005 50070304 1185515553
[  michaelt@node2  ~]$R -q -e "64005 - 63864"
> 64005 - 63864
[1] 141
> 
> 
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.AllPools.Vs3.annotated.vcf.gz | grep ^# | wc
 142    1712   29465
~~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/OLD3.MM5.AllPools.Vs3.annotated.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/OLD3.MM5.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/OLD3.MM5.skatO.nonsyn.qVals.ACs.GeneLevel.ForSudhir.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/.

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.AllPools.Vs3.annotated.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/. 

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/msigdb_c2/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt.MM5.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.indv3.EPACTS.RMID5PCs.msigdb_c2

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/msigdb_c2/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt.MM5.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs.msigdb_c2

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/Permutations /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20150427Array1KevinGeneGroupings.array_gene_groupings_03_31_2014.semicolons.SpaceToUnderscores.withHGNCdata.SplitIntoCategories.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/Permutations/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs.20150427Array1KevinGeneGroupings

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/Permutations /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/20150427Array1KevinGeneGroupings.array_gene_groupings_03_31_2014.semicolons.SpaceToUnderscores.withHGNCdata.SplitIntoCategories.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/KevinComplexes/Permutations/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs.20150427Array1KevinGeneGroupings

#then do P2 finishing up too




##20151005 Note -- Comparing old results to new results some more
##Specifically looking at what genes are being included/not included given comparable q-value cutoffs between current results and past results

paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^NA | awk '{ print $4 }' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts } | awk '{ print $4 }' | sort -g) | vi -

comm <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^NA | awk '{ print $1 }' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts } | awk '{ print $1 }' | sort -g) | vi -


for i in {1..450..100}; do
	BEGIN=$i
	END=$(($i + 99))

	echo $i

	comm -23 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^NA | sort -g -k 4,4 | awk '{ if ((NR>='$BEGIN') && (NR<='$END')) { print $1 } } ' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | sort -g -k 4,4 | awk '{ if ((NR>='$BEGIN') && (NR<='$END')) { print $1 } } ' | sort -g) | wc
	comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^NA | sort -g -k 4,4 | awk '{ if ((NR>='$BEGIN') && (NR<='$END')) { print $1 } } ' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | sort -g -k 4,4 | awk '{ if ((NR>='$BEGIN') && (NR<='$END')) { print $1 } } ' | sort -g) | wc

done


comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .3) { print $1; } }' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .3) { print $1; } }' | sort -g) | wc
paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .3) { print $1; } }' | wc) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .3) { print $1; } }' | wc)
paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .3) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .3) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10)

comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }' | sort -g) | wc
paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }' | wc) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }' | wc)
paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10)

comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }' | sort -g) | wc
paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }' | wc) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }' | wc)
paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10)

comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .75) { print $1; } }' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .75) { print $1; } }' | sort -g) | wc
paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .75) { print $1; } }' | wc) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .75) { print $1; } }' | wc)
paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .75) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .75) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10)

comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }' | sort -g) | wc
paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }' | wc) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }' | wc)
paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10)


~~~
[  michaelt@login2  /data/userdata/pg/michaelt]$for i in {1..450..25}; do                                                                                                                                                                                                                                                   
>         BEGIN=$i                                                                                                                                                                                                                                                                                                          
>         END=$(($i + 24))                                                                                                                                                                                                                                                                                                  
>                                                                                                                                                                                                                                                                                                                           
>         echo $i                                                                                                                                                                                                                                                                                                           
>                                                                                                                                                                                                                                                                                                                           
>         comm -23 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^NA | sort -g -k 4,4 | awk '{ if ((NR>='$BEGIN') && (NR<='$END')) { print $1 } } ' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | sort -g -k 4,4 | awk '{ if ((NR>='$BEGIN') && (NR<='$END')) { print $1 } } ' | sort -g) | wc                                                                                                                                  
>         comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^NA | sort -g -k 4,4 | awk '{ if ((NR>='$BEGIN') && (NR<='$END')) { print $1 } } ' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | sort -g -k 4,4 | awk '{ if ((NR>='$BEGIN') && (NR<='$END')) { print $1 } } ' | sort -g) | wc                                                                                                                                  
>                                                                                                                                                                                                                                                                                                                           
> done                                                                                                                                                                                                                                                                                                                      
1                                                                                                                                                                                                                                                                                                                           
2       2       6                                                                                                                                                                                                                                                                                                     
2       2      13                                                                                                                                                                                                                                                                                                     
26                                                                                                                                                                                                                                                                                                                          
4       4      26                                                                                                                                                                                                                                                                                                     
4       4      24                                                                                                                                                                                                                                                                                                     
51                                                                                                                                                                                                                                                                                                                          
6       6      35                                                                                                                                                                                                                                                                                                     
6       6      38                                                                                                                                                                                                                                                                                                     
76                                                                                                                                                                                                                                                                                                                          
6       6      36                                                                                                                                                                                                                                                                                                     
6       6      34                                                                                                                                                                                                                                                                                                     
101                                                                                                                                                                                                                                                                                                                         
6       6      48                                                                                                                                                                                                                                                                                                     
6       6      36                                                                                                                                                                                                                                                                                                     
126                                                                                                                                                                                                                                                                                                                         
7       7      41                                                                                                                                                                                                                                                                                                     
7       7      51                                                                                                                                                                                                                                                                                                     
151                                                                                                                                                                                                                                                                                                                         
4       4      23                                                                                                                                                                                                                                                                                                     
4       4      24                                                                                                                                                                                                                                                                                                     
176                                                                                                                                                                                                                                                                                                                         
9       9      56                                                                                                                                                                                                                                                                                                     
9       9      54                                                                                                                                                                                                                                                                                                     
201                                                                                                                                                                                                                                                                                                                         
9       9      52                                                                                                                                                                                                                                                                                                     
9       9      58                                                                                                                                                                                                                                                                                                     
226                                                                                                                                                                                                                                                                                                                         
9       9      57                                                                                                                                                                                                                                                                                                     
9       9      54                                                                                                                                                                                                                                                                                                     
251                                                                                                                                                                                                                                                                                                                         
9       9      56                                                                                                                                                                                                                                                                                                     
9       9      60                                                                                                                                                                                                                                                                                                     
276                                                                                                                                                                                                                                                                                                                         
11      11      72                                                                                                                                                                                                                                                                                                     
11      11      72                                                                                                                                                                                                                                                                                                     
301                                                                                                                                                                                                                                                                                                                         
10      10      64                                                                                                                                                                                                                                                                                                     
10      10      78                                                                                                                                                                                                                                                                                                     
326                                                                                                                                                                                                                                                                                                                         
10      10      79                                                                                                                                                                                                                                                                                                     
10      10      63                                                                                                                                                                                                                                                                                                     
351                                                                                                                                                                                                                                                                                                                         
8       8      51                                                                                                                                                                                                                                                                                                     
8       8      52                                                                                                                                                                                                                                                                                                     
376                                                                                                                                                                                                                                                                                                                         
5       5      34                                                                                                                                                                                                                                                                                                     
5       5      31                                                                                                                                                                                                                                                                                                     
401                                                                                                                                                                                                                                                                                                                         
7       7      41                                                                                                                                                                                                                                                                                                     
7       7      36                                                                                                                                                                                                                                                                                                     
426                                                                                                                                                                                                                                                                                                                         
9       9      48                                                                                                                                                                                                                                                                                                     
9       9      55                                                                               
[  michaelt@login2  /data/userdata/pg/michaelt]$for i in {1..450..50}; do
>         BEGIN=$i
>         END=$(($i + 49))
> 
>         echo $i
> 
>         comm -23 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^NA | sort -g -k 4,4 | awk '{ if ((NR>='$BEGIN') && (NR<='$END')) { print $1 } } ' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | sort -g -k 4,4 | awk '{ if ((NR>='$BEGIN') && (NR<='$END')) { print $1 } } ' | sort -g) | wc
>         comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^NA | sort -g -k 4,4 | awk '{ if ((NR>='$BEGIN') && (NR<='$END')) { print $1 } } ' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | sort -g -k 4,4 | awk '{ if ((NR>='$BEGIN') && (NR<='$END')) { print $1 } } ' | sort -g) | wc
> 
> done
1
4       4      19
4       4      24
51
6       6      35
6       6      36
101
4       4      25
4       4      23
151
8       8      47
8       8      46
201
13      13      77
13      13      80
251
11      11      68
11      11      72
301
11      11      71
11      11      69
351
8       8      51
8       8      49
401
9       9      54
9       9      56
[  michaelt@login2  /data/userdata/pg/michaelt]$for i in {1..450..100}; do
>         BEGIN=$i
>         END=$(($i + 99))
> 
>         echo $i
> 
>         comm -23 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^NA | sort -g -k 4,4 | awk '{ if ((NR>='$BEGIN') && (NR<='$END')) { print $1 } } ' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | sort -g -k 4,4 | awk '{ if ((NR>='$BEGIN') && (NR<='$END')) { print $1 } } ' | sort -g) | wc
>         comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -v ^NA | sort -g -k 4,4 | awk '{ if ((NR>='$BEGIN') && (NR<='$END')) { print $1 } } ' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | sort -g -k 4,4 | awk '{ if ((NR>='$BEGIN') && (NR<='$END')) { print $1 } } ' | sort -g) | wc
> 
> done
1
4       4      17
4       4      23
101
9       9      55
9       9      52
201
15      15      89
15      15      96
301
11      11      71
11      11      67
401
5       5      31
5       5      32
[  michaelt@login2  /data/userdata/pg/michaelt]$comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .3) { print $1; } }' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .3) { print $1; } }' | sort -g) | wc
0       0       0
[  michaelt@login2  /data/userdata/pg/michaelt]$paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .3) { print $1; } }' | wc) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .3) { print $1; } }' | wc)
25      25     143       3       3      19
[  michaelt@login2  /data/userdata/pg/michaelt]$paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .3) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .3) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10)
TPR      0.0179784600    1.402779e-03    0.29550        VDAC3    0.0021478040    3.558253e-04    0.26496
OSBPL6   0.0191018300    1.393998e-03    0.29550        VTI1A    0.0011255423    1.964403e-04    0.20828
COPB1    0.0194252700    8.904565e-05    0.29550        GEMIN2   0.0007637044    2.849703e-04    0.20828
SDHD     0.0062799510    5.324625e-04    0.29264
PHKG1    0.0136560500    7.209239e-04    0.29264
NUDC     0.0074898000    0.000000e+00    0.29264
MRPS9    0.0093243610    1.003811e-03    0.29264
MLLT1    0.0161103700    2.402246e-04    0.29264
KIR3DL1          0.0132004500    7.649175e-04    0.29264
ICAM2    0.0127430600    4.528685e-04    0.29264
[  michaelt@login2  /data/userdata/pg/michaelt]$comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }' | sort -g) | wc
0       0       0
[  michaelt@login2  /data/userdata/pg/michaelt]$paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }' | wc) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }' | wc)
71      71     437      61      61     383
[  michaelt@login2  /data/userdata/pg/michaelt]$paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10)
LDLR     0.0954660500    1.388695e-03    0.48408        XPO4     0.0704735400    4.661435e-04    0.45134
SDCBP    0.0914164000    2.638715e-04    0.48019        WDR61    0.0567350000    0.000000e+00    0.45134
HUWE1    0.0921905500    5.082060e-04    0.48019        TOR1AIP2         0.0730810000    0.000000e+00    0.45134
EPPK1    0.0901366700    1.798588e-03    0.48019        TMED9    0.0725844300    4.781201e-04    0.45134
COPS4    0.0933273800    3.313783e-04    0.48019        TMED10   0.0585758200    3.318390e-04    0.45134
SEPSECS          0.0850065800    6.079834e-04    0.46768        SERPINH1         0.0627336000    1.375285e-03    0.45134
FAM133B          0.0854473000    6.001718e-04    0.46768        PON2     0.0713547500    5.750000e-05    0.45134
CNTF     0.0855490000    0.000000e+00    0.46768        NPM3     0.0658032000    2.981737e-04    0.45134
AP1M1    0.0848860100    3.427674e-04    0.46768        NFAT5    0.0677065600    6.550603e-04    0.45134
MRS2     0.0784893900    1.383277e-03    0.45769        MRS2     0.0741817000    1.247554e-03    0.45134
[  michaelt@login2  /data/userdata/pg/michaelt]$comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }' | sort -g) | wc
0       0       0
[  michaelt@login2  /data/userdata/pg/michaelt]$paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }' | wc) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }' | wc)
441     441    2786     214     214    1368
[  michaelt@login2  /data/userdata/pg/michaelt]$paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10)                                                        
EPRS     0.8765946000    2.449386e-02    0.69863        EIF2AK2          0.4007059000    3.835295e-03    0.69298
XPO5     0.8714748000    2.582549e-02    0.69812        ITGA4    0.3968260000    5.756581e-03    0.68949
TLK2     0.8739594000    5.768424e-04    0.69812        HLA-B    0.3947561000    5.274871e-03    0.68913
SIRT1    0.8700947000    5.115029e-04    0.69812        CERS2    0.3875643000    5.055291e-03    0.68527
NMT1     0.8658447000    1.976939e-03    0.69812        ATAD3B   0.3859445000    4.762373e-03    0.68527
DARS     0.8701861000    6.819993e-03    0.69812        ARHGEF12         0.3906947000    7.641625e-03    0.68527
AIMP2    0.8721100000    0.000000e+00    0.69812        AQR      0.3902804000    1.055832e-02    0.68527
ZFP91    0.8307363000    1.012223e-02    0.69767        TM9SF1   0.3806165000    2.184987e-03    0.68050
ZEB2     0.5186180000    6.535995e-03    0.69767        VDAC2    0.3536500000    0.000000e+00    0.67601
YWHAB    0.6652781000    6.463693e-04    0.69767        TXNDC5   0.3579552000    9.478907e-03    0.67601
[  michaelt@login2  /data/userdata/pg/michaelt]$comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .75) { print $1; } }' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .75) { print $1; } }' | sort -g) | wc
0       0       0
[  michaelt@login2  /data/userdata/pg/michaelt]$paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .75) { print $1; } }' | wc) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .75) { print $1; } }' | wc)
475     475    2998     451     451    2855
[  michaelt@login2  /data/userdata/pg/michaelt]$paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .75) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .75) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10)
WDR77    1.0000000000    0.000000e+00    0.73970        CUL2     0.9111782000    3.617588e-04    0.74772
TYSND1   0.9892165000    3.082824e-02    0.73970        NDUFA2   0.9082518000    2.825618e-04    0.74697
PSMD7    1.0000000000    0.000000e+00    0.73970        XPO5     0.8824160000    3.147410e-02    0.74690
PARP9    0.9981395000    1.309007e-02    0.73970        TYSND1   0.9015964000    5.153199e-04    0.74690
MRPS35   1.0000000000    0.000000e+00    0.73970        TMEM11   0.9046219000    9.152862e-04    0.74690
MRPL42   1.0000000000    0.000000e+00    0.73970        TLK2     0.8788912000    6.366679e-04    0.74690
MEPCE    1.0000000000    0.000000e+00    0.73970        THOP1    0.9019140000    2.853862e-02    0.74690
MAGED1   1.0000000000    0.000000e+00    0.73970        SDF4     0.9061500000    0.000000e+00    0.74690
ITGAL    1.0000000000    0.000000e+00    0.73970        RHOC     0.9023125000    1.931466e-03    0.74690
HSPA6    1.0000000000    0.000000e+00    0.73970        RBBP7    0.8955400000    0.000000e+00    0.74690
[  michaelt@login2  /data/userdata/pg/michaelt]$comm -13 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }' | sort -g) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }' | sort -g) | wc
0       0       0
[  michaelt@login2  /data/userdata/pg/michaelt]$paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }' | wc) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }' | wc)
475     475    2998     473     473    2992
[  michaelt@login2  /data/userdata/pg/michaelt]$paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD4.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1, "\t", $2, "\t", $3, "\t", $4 } }' | sort -rg -k 4,4 | head -n 10)                                        
WDR77    1.0000000000    0.000000e+00    0.73970        WDR77    1.0000000000    0.000000e+00    0.78244
TYSND1   0.9892165000    3.082824e-02    0.73970        SURF2    0.9833349000    4.152537e-02    0.78244
PSMD7    1.0000000000    0.000000e+00    0.73970        PSMD7    1.0000000000    0.000000e+00    0.78244
PARP9    0.9981395000    1.309007e-02    0.73970        PARP9    0.9980905000    1.343671e-02    0.78244
MRPS35   1.0000000000    0.000000e+00    0.73970        NOLC1    1.0000000000    0.000000e+00    0.78244
MRPL42   1.0000000000    0.000000e+00    0.73970        MRPS35   1.0000000000    0.000000e+00    0.78244
MEPCE    1.0000000000    0.000000e+00    0.73970        MRPL42   1.0000000000    0.000000e+00    0.78244
MAGED1   1.0000000000    0.000000e+00    0.73970        MEPCE    1.0000000000    0.000000e+00    0.78244
ITGAL    1.0000000000    0.000000e+00    0.73970        MAGED1   1.0000000000    0.000000e+00    0.78244
HSPA6    1.0000000000    0.000000e+00    0.73970        LRRC47   0.9953638000    2.283511e-02    0.78244
~~~




#DO QC STUFF

##20150930
##Collecting runs of QC scripts on main dataset here

#20150930
#20150930 CHECK_0: Update QC results




cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.perIndv.vs3.output.justSeroNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 14.1133144475921 MeanDoubletons: 18.8101983002833 MeanCommon: 4446.35694050992 MeanRare: 425.130311614731 MeanHeterozy: 3053.45325779037 MeanCoverage: 16.107917375798 sdCoverage: 6.19823376737347 Exonic: 334.082152974504 Intronic: 3569.48441926346 Nonsynonymous: 122.872521246459 Synonymous: 208.801699716714 PP2_D: 9.90934844192635 PP2_P: 12.6345609065156 Sift_D: 23.0509915014164 LRT_D: 21.6345609065156"
>
>







##201511
##Creating github project page

#Following directions from https://help.github.com/articles/creating-project-pages-manually/

mkdir /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Webpage

cd /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Webpage

git clone https://github.com/mturchin20/Wolinsky_TargetedExome

cd /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome

git checkout --orphan gh-pages

~~~
[  michaelt@login2  ~/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome]$git checkout --orphan gh-pages
error: unknown option `orphan'
usage: git checkout [options] <branch>
   or: git checkout [options] [<branch>] -- <file>...
.
.
.
[  michaelt@login2  ~/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome]$git version
git version 1.7.1
~~~

#20151201 20151209 CHECK_1: Need to install a newer version of git?; do a local install from https://github.com/git/git/releases & using https://git-scm.com/book/en/v2/Getting-Started-Installing-Git for help/guidance too? Use /home/michaelt/PackageInstallationLog.vs1.sh

/home/michaelt/lib/bin/git checkout --orphan gh-pages

/home/michaelt/lib/bin/git rm -rf .

#Updated PATH to include /home/michaelt/lib/bin/ at the beginning of the PATH now versus the end where it was before

echo "My Page" > index.html
git add index.html
git commit -a -m "First pages commit"
git push origin gh-pages

~~~
[  michaelt@login2  ~/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome]$git push origin gh-pages
fatal: Unable to find remote helper for 'https'
~~~

#According to http://stackoverflow.com/questions/8329485/git-clone-fatal-unable-to-find-remote-helper-for-https I need libcurl-devel installed when I make git for this error not to occur

~~~
.
.
.
115
down vote
accepted
It looks like not having (lib)curl-devel installed when you compile git can cause this.

If you install (lib)curl-devel, and then rebuild/install git, this should solve the problem:

$ yum install curl-devel
$ # cd to wherever the source for git is
$ cd /usr/local/src/git-1.7.9  
$ ./configure
$ make
$ make install
This worked for me on Centos 6.3.

If you don't have yum, you can download the source to curl-devel here:

http://curl.haxx.se/dlwiz/?type=devel
.
.
.
~~~

#Aleksandar did the local install for me and now things work

#20151209 20151211 CHECK_1: Now work on moving the template over from John's/Rmarkdown's examples

#vi /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome/_output.yaml
#
##Copy/pased from https://github.com/jdblischak/singleCellSeq/blob/gh-pages/analysis/_output.yaml

#In /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome/ 
#Figured out how to clone John's repo then switch to gh-pages branch through this stack overflow thread -- http://stackoverflow.com/questions/67699/clone-all-remote-branches-with-git?rq=1
#Can't just directly clone a gh-pages branch it appears
git clone https://github.com/jdblischak/singleCellSeq/
git branch -a
git checkout remotes/origin/gh-pages
mkdir /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome/JohnBackUp
cp -rp * /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome/JohnBackUp/.
cd /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome/
rm -r /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome/singleCellSeq/
echo "JohnBackUp/" > .gitignore
echo "JohnBackUp_Scratch/" >> .gitignore
cp -rp /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome/JohnBackUp/libs/. /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome/.
cp -rp /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome/JohnBackUp/include/. /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome/.
mkdir /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome/Figures
cp -p /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome/JohnBackUp/_output.yaml /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome/.
cp -p /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome/JohnBackUp/about.Rmd /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome/JohnBackUp/license.Rmd /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome/JohnBackUp/template.Rmd /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome/JohnBackUp/Makefile /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome/.
cp -p /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome/JohnBackUp/index.Rmd /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome/index.John.Rmd
echo "index.John.Rmd" >> .gitignore

git add *
git commit -a -m "Adding main components copied from https://github.com/jdblischak/singleCellSeq/tree/gh-pages/analysis"
git push origin gh-pages

#Working on installing Rmarkdown

cd /home/michaelt/Software 
wget https://cran.r-project.org/src/contrib/rmarkdown_0.8.1.tar.gz
wget https://cran.r-project.org/src/contrib/shiny_0.12.2.tar.gz
wget https://cran.r-project.org/src/contrib/testthat_0.11.0.tar.gz
wget https://cran.r-project.org/src/contrib/digest_0.6.8.tar.gz
wget https://cran.r-project.org/src/contrib/ggplot2_1.0.1.tar.gz
wget https://cran.r-project.org/src/contrib/knitr_1.11.tar.gz
wget https://cran.r-project.org/src/contrib/markdown_0.7.7.tar.gz
wget https://cran.r-project.org/src/contrib/Cairo_1.5-9.tar.gz

R -q -e "install.packages(\"/home/michaelt/Software/Cairo_1.5-9.tar.gz\", repos=NULL, type=\"source\");"
R -q -e "install.packages(\"/home/michaelt/Software/knitr_1.11.tar.gz\", repos=NULL, type=\"source\");"
#R -q -e "install.packages(\"/home/michaelt/Software/testthat_0.11.0.tar.gz\", repos=NULL, type=\"source\");"
R -q -e "install.packages(\"/home/michaelt/Software/markdown_0.7.7.tar.gz\", repos=NULL, type=\"source\");"
#R -q -e "install.packages(\"/home/michaelt/Software/digest_0.6.8.tar.gz\", repos=NULL, type=\"source\");"
#R -q -e "install.packages(\"/home/michaelt/Software/ggplot2_1.0.1.tar.gz\", repos=NULL, type=\"source\");"
R -q -e "install.packages(\"/home/michaelt/Software/shiny_0.12.2.tar.gz\", repos=NULL, type=\"source\");"
R -q -e "install.packages(\"/home/michaelt/Software/rmarkdown_0.8.1.tar.gz\", repos=NULL, type=\"source\");"

mkdir /home/michaelt/Software/Rpackages
cd /home/michaelt/Software/Rpackages

#For markdown -- need mime
wget https://cran.r-project.org/src/contrib/mime_0.4.tar.gz
R -q -e "install.packages(\"/home/michaelt/Software/Rpackages/mime_0.4.tar.gz\", repos=NULL, type=\"source\");"
R -q -e "install.packages(\"/home/michaelt/Software/markdown_0.7.7.tar.gz\", repos=NULL, type=\"source\");"

#For knitr
~~~
ERROR: dependencies ?evaluate?, ?formatR?, ?highr?, ?yaml? are not available for package ?knitr?
~~~

#R -q -e "download.packages(c(\"evaluate\", \"formatR\", \"highr\", \"yaml\"), destdir=\"/home/michaelt/Software/Rpackages\", type=\"source\")"
#Ran in an R session since need to choose a mirror

~~~
     [,1]       [,2]
     [1,] "evaluate" "/home/michaelt/Software/Rpackages/evaluate_0.8.tar.gz"
     [2,] "formatR"  "/home/michaelt/Software/Rpackages/formatR_1.2.1.tar.gz"
     [3,] "highr"    "/home/michaelt/Software/Rpackages/highr_0.5.1.tar.gz"
     [4,] "yaml"     "/home/michaelt/Software/Rpackages/yaml_2.1.13.tar.gz"
~~~

R -q -e "install.packages(\"/home/michaelt/Software/Rpackages/evaluate_0.8.tar.gz\", repos=NULL, type=\"source\");"
R -q -e "install.packages(\"/home/michaelt/Software/Rpackages/formatR_1.2.1.tar.gz\", repos=NULL, type=\"source\");"
R -q -e "install.packages(\"/home/michaelt/Software/Rpackages/highr_0.5.1.tar.gz\", repos=NULL, type=\"source\");"
R -q -e "install.packages(\"/home/michaelt/Software/Rpackages/yaml_2.1.13.tar.gz\", repos=NULL, type=\"source\");"


##For digest
#R; download.packages(c("digest"), destdir="/home/michaelt/Software/Rpackages", type="source"); install.packages("/home/michaelt/Software/Rpackages/digest_0.6.8.tar.gz", repos=NULL, type="source", INSTALL_opts = c('--no-lock'));

mkdir /home/michaelt/tmp
mkdir /home/michaelt/tmp/Rpackages

R -q -e "install.packages(\"/home/michaelt/Software/digest_0.6.8.tar.gz\", repos=NULL, type=\"source\", lib=\"/home/michaelt/tmp/Rpackages\");"

#NOTE -- packages 'digest' and 'ggplot2' already exist in my local library; they were installed previously (sometime in 2014?) by root. Current installs were 'failing' because they were attempting to overwrite files that had been created by root in the install locations


#For rmarkdown
~~~
ERROR: dependencies ?htmltools?, ?caTools? are not available for package ?rmarkdown?
~~~

R; download.packages(c("htmltools", "caTools"), destdir="/home/michaelt/Software/Rpackages", type="source")

~~~
     [,1]        [,2]
     [1,] "htmltools" "/home/michaelt/Software/Rpackages/htmltools_0.2.6.tar.gz"
     [2,] "caTools"   "/home/michaelt/Software/Rpackages/caTools_1.17.1.tar.gz"
~~~

R -q -e "install.packages(\"/home/michaelt/Software/Rpackages/htmltools_0.2.6.tar.gz\", repos=NULL, type=\"source\");"
R -q -e "install.packages(\"/home/michaelt/Software/Rpackages/caTools_1.17.1.tar.gz\", repos=NULL, type=\"source\");"

~~~
ERROR: dependency ?bitops? is not available for package ?caTools?
~~~
R; download.packages(c("bitops"), destdir="/home/michaelt/Software/Rpackages", type="source")
~~~
[1,] "bitops" "/home/michaelt/Software/Rpackages/bitops_1.0-6.tar.gz"
~~~
R -q -e "install.packages(\"/home/michaelt/Software/Rpackages/bitops_1.0-6.tar.gz\", repos=NULL, type=\"source\");"


#For shiny
~~~
ERROR: dependencies ?httpuv?, ?jsonlite?, ?xtable?, ?R6? are not available for package ?shiny?
~~~

R; download.packages(c("httpuv", "jsonlite", "xtable", "R6"), destdir="/home/michaelt/Software/Rpackages", type="source")

~~~
     [,1]       [,2]
     [1,] "httpuv"   "/home/michaelt/Software/Rpackages/httpuv_1.3.3.tar.gz"
     [2,] "jsonlite" "/home/michaelt/Software/Rpackages/jsonlite_0.9.19.tar.gz"
     [3,] "xtable"   "/home/michaelt/Software/Rpackages/xtable_1.8-0.tar.gz"
     [4,] "R6"       "/home/michaelt/Software/Rpackages/R6_2.1.1.tar.gz"
~~~

R -q -e "install.packages(\"/home/michaelt/Software/Rpackages/httpuv_1.3.3.tar.gz\", repos=NULL, type=\"source\");"
R -q -e "install.packages(\"/home/michaelt/Software/Rpackages/jsonlite_0.9.19.tar.gz\", repos=NULL, type=\"source\");"
R -q -e "install.packages(\"/home/michaelt/Software/Rpackages/xtable_1.8-0.tar.gz\", repos=NULL, type=\"source\");"
R -q -e "install.packages(\"/home/michaelt/Software/Rpackages/R6_2.1.1.tar.gz\", repos=NULL, type=\"source\");"


##Running into dependency issues mostly revolving around needing newer version of R. Thinking of downloading and installing the most recent version of R and using R markdown through just this newest version. Should be mostly fine?
##Didn't end up needing to do this -- couldn't install 'testthat' because of this reason, but it wasn't actually a dependency of anything; I misunderstood the CRAN breakdown. It is recommended for a particular package but not needed 

cd /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome

R -q -e "rmarkdown::render('index.John.Rmd');"

#For pandoc
~~~
[  michaelt@login2  ~/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome]$R -q -e "rmarkdown::render('index.John.Rmd');"
> rmarkdown::render('index.John.Rmd');
Error: pandoc version 1.12.3 or higher is required and was not found.
Execution halted
~~~

#Download pandoc from wget https://github.com/jgm/pandoc/archive/1.15.2.tar.gz

#Trying to install a local rpm source based on the instructions in http://www.linuxquestions.org/questions/linux-newbie-8/rpm-installation-having-no-root-access-762363/
~~~
It can be done but requires a number of steps:

Setup a private RPM database

mkdir -p /home/username/local/lib/rpm
rpm --initdb --root /home/username/local --dbpath /home/username/local/lib/rpm

If you do not like ~/local/lib/rpm, you can pick a different subdirectory structure.

Check for package dependancies

rpm -ivh package.rpm

If dependancies found, install the dependant RPMs before installing the package RPM.

Finally install the package

rpm --root /home/username/local --dbpath /home/username/local/lib/rpm \
--relocate /usr=/home/username/local --nodeps -ivh package.rpm

To use the package you will have to play around with LD_LIBRARY_PATH to let the
loader know where the required shared labraries are and modify your PATH to
include any required scripts and executables.
~~~

mkdir -p /home/michaelt/local/lib/rpm
rpm --initdb --root /home/michaelt/local --dbpath /home/michaelt/local/lib/rpm
rpm -ivh /home/michaelt/Software/gmp-devel-6.1.0-1.fc24.x86_64.rpm

#20151210 -- was able to get pandoc installed eventualyl by using the haskell stack setup, and also manually installing a copy of the gmp library + realizing I needed to put the library path in LIBRARY_PATH and not LD_LIBRARY_PATJ

cp -p /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome/index.John.Rmd /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome/index.Rmd

#Edit /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome/about.Rmd, /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome/license.Rmd, /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome/include/before_body.html

#20151216 NOTE -- northwestern system currently on/off for maintenance and was getting some odd issues on a variety of commands. Stopping here and trying again later 
#20151223 CHECK_0: Consider running the following code on file to convert all ~~~ to ``` for markdown purposes?
#:.,$ s/^\~\~\~$/\`\`\`/g
#20151223 NOTE -- I needed to include /home/michaelt/.local/bin/ in my $PATH directory in order to access pandoc (where a few other things are installed too apparently)
#20151223 20151223 CHECK_1 -- Prob: Running R through a makefile doesn't appear to find 'rmarkdown' though running it through the command line outside of make does.  Resp: Need to include this statement (...; .libPaths(c( .libPaths(), \"/home/michaelt/Software/R-3.0.2/library\"));...) in my Makefile R calls in order to access the local library R is normally using when it runs from the commandline directly

~~~
[  michaelt@login2  ~/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome]$make
R --slave -e "set.seed(100); rmarkdown::render('index.Rmd')"
Error in loadNamespace(name) : there is no package called 'rmarkdown'
Calls: :: ... tryCatch -> tryCatchList -> tryCatchOne -> <Anonymous>
Execution halted
make: *** [index.html] Error 1
[  michaelt@login2  ~/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome]$R -q -e "rmarkdown::render('index.John.Rmd');"
> rmarkdown::render('index.John.Rmd');


processing file: index.John.Rmd
  |.................................................................| 100%
     inline R code fragments


output file: index.John.knit.md

/home/michaelt/.local/bin/pandoc +RTS -K512m -RTS index.John.utf8.md --to html --from markdown+autolink_bare_uris+ascii_identifiers+tex_math_single_backslash-implicit_figures --output index.John.html --smart --email-obfuscation none +RTS -K64m -RTS --standalone --section-divs --table-of-contents --toc-depth 3 --template /home/michaelt/Software/R-3.0.2/library/rmarkdown/rmd/h/default.html --variable 'theme:cerulean' --include-in-header /tmp/RtmptxRBBm/rmarkdown-str3a36141af6ac.html --no-highlight --variable highlightjs=libs/highlight --variable highlightjs-theme=textmate --include-in-header include/in_header.html --include-before-body include/before_body.html --include-after-body include/after_body.html 

Output created: index.John.html
> 
> 
~~~

#Installed Ruby localled -- see /home/michaelt/PackageInstallationLog.vs1.sh 

gem install bundler

~~~
[  michaelt@login2  ~/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome]$gem install bundler
ERROR:  While executing gem ... (Gem::Exception)
    Unable to require openssl, install OpenSSL and rebuild ruby (preferred) or use non-HTTPS sources
~~~    

#Need to install OpenSSL apparently?...so install/installing it (see once again /home/michaelt/PackageInstallationLog.vs1.sh)

mkdir /home/michaelt/lib/openssl/include
cp -p /home/michaelt/lib/include/openssl/* /home/michaelt/lib/openssl/include/.

gem install bundler -- --with-cppflags=-I/home/michaelt/lib/openssl/include

#20151223 NOTE -- Doing a local install of ruby + OpenSSL was not working, so ultimately asked Aleksandar if he could do a global install of OpenSSL + Ruby to /home/shared/software/

echo "source 'https://rubygems.org'\ngem 'github-pages'" > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome/Gemfile

bundle install

bundle exec jekyll serve

##http://askubuntu.com/questions/414930/access-webpage-through-ssh

~~~
First Method:

Launch an SSH tunnel

To initiate your SSH tunnel, simply open terminal and connect to your remote server via SSH with the following flags:

ssh -D 8080 -C -N username@example.com
Browse the Web with Your SSH Tunnel (Chrome)

Now, let’s start browsing the web using our new SSH tunnel.

Open Google Chrome
Select the wrench icon on the top right
Select ‘Settings’
Select ‘Show advanced settings…’
Select ‘Change proxy settings…’
Select ‘SOCKS Proxy’
Enter ’127.0.0.1′
Enter port ’8080′
Save changes by selecting ‘OK’
Search Google for ‘my ip’ and take a look at what your IP address is now.

This will launch our SSH tunnel on port 8080 and route all traffic (securely) through the server at example.com.

Exiting the SSH Tunnel

To exit the SSH tunnel, simply disable the SOCKS proxy within your browser.

source (http://drewsymo.com/2013/11/ssh-tunnel-in-30-seconds-mac-osx-linux/)
~~~







##201511
##Including analysis run for intronic regions/variants

~~~
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs | perl -F, -lane 'print $F[6];' | sort | uniq -c
5932 UTR3
1073 UTR5
2409 downstream
6759 exonic
2 exonic;splicing
530 intergenic
45045 intronic
171 ncRNA_UTR3
19 ncRNA_UTR5
298 ncRNA_exonic
1508 ncRNA_intronic
1 ncRNA_splicing
30 splicing
1227 upstream
204 upstream;downstream
~~~

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs | awk -F, '{ if ($7 == "intronic") { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Intronic.wSplice

#Including '.wSplice' at end due to /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh//data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs3.pt2.sh naming convention, ie code inserts '.wSplice' at end of a variable expecting it to say 'Exonic.Nonsynonymous'. So rather than edit a random single line in a somewhat important file, I'm just going to name the 'Intronic' file here with '.wSplice' knowing that, in fact, there is now 'splice' in this file (though I could there coudl have been possibly? just hypothetically speaking)

~~~
[  michaelt@node1  ~/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Intronic.wSplice | perl -lane 'print $#F;' | sort -g | uniq -c | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,2], seq(0,1000,25), right=FALSE));"
> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,2], seq(0,1000,25), right=FALSE));

[0,25)     [25,50)     [50,75)    [75,100)   [100,125)   [125,150)
24          25          24          24          21          20
[150,175)   [175,200)   [200,225)   [225,250)   [250,275)   [275,300)
17          11          10           6           9           3
[300,325)   [325,350)   [350,375)   [375,400)   [400,425)   [425,450)
2           0           6           1           1           1
[450,475)   [475,500)   [500,525)   [525,550)   [550,575)   [575,600)
0           2           0           0           0           0
[600,625)   [625,650)   [650,675)   [675,700)   [700,725)   [725,750)
1           0           0           0           0           1
[750,775)   [775,800)   [800,825)   [825,850)   [850,875)   [875,900)
0           0           0           0           0           0
[900,925)   [925,950)   [950,975) [975,1e+03)
0           0           0           0
>
>
[  michaelt@node1  ~/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic | perl -lane 'print $#F;' | sort -g | uniq -c | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,2], seq(0,200,10), right=FALSE));"
> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,2], seq(0,200,10), right=FALSE));

[0,10)   [10,20)   [20,30)   [30,40)   [40,50)   [50,60)   [60,70)   [70,80)
9        10        10         7         5         5         3         4
[80,90)  [90,100) [100,110) [110,120) [120,130) [130,140) [140,150) [150,160)
1         1         0         1         0         0         1         0
[160,170) [170,180) [180,190) [190,200)
0         0         1         0
>
>
~~~

#Edit /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs

#Edit /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs

#20151223 NOTE -- Code takes about ~2.5hrs to run

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Intronic
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Intronic

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs3.sh MM5 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs TruePheno Pheno1 Intronic

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs3.sh MM5 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs FakePheno Pheno1 Intronic





bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs3.sh MM5 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs TruePheno Pheno1 Intronic

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs3.sh MM5 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs FakePheno Pheno1 Intronic


scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcess*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/TempStuff/.

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcess*jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/All*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/TempStuff/.










##20151216
##Doing some followup work on SNPs that failed HWE to make sure/see nothing was really lost from them

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/CheckHWESNPs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | wc

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe | awk '{ if ($6 < .0001) { print $0 } } ' | awk '{ print $1 "," $2 }' ) > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/CheckHWESNPs/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.genome_summary.csv

~~~
[  michaelt@login2  ~/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/CheckHWESNPs/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.genome_summary.csv | perl -F, -lane 'print $F[0];' | sort | uniq -c
80 "UTR3"
25 "UTR5"
103 "downstream"
87 "exonic"
48 "intergenic"
2031 "intronic"
1 "ncRNA_UTR3"
11 "ncRNA_exonic"
62 "ncRNA_intronic"
56 "upstream"
8 "upstream;downstream"
[  michaelt@login2  ~/LabStuff/StephensLab/HIVStuff/PG/Webpage/Wolinsky_TargetedExome]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/CheckHWESNPs/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.genome_summary.csv | perl -F, -lane 'foreach my $entry1 (@F) { if ($entry1 =~ /synonymous/) { print $entry1; break; } } ' | sort | uniq -c
31 "nonsynonymous SNV"
55 "synonymous SNV"
~~~

paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe | awk '{ if ($6 < .0001) { print $0 } } ' | awk '{ print $1 "," $2 }' )) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe | grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe | awk '{ if ($6 < .0001) { print $0 } } ' | awk '{ print $1 "\t" $2 }' )) > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/CheckHWESNPs/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.genome_summary.wHardyPLINK.txt

#20151217 CHECK_0 -- Prob: Redo analyses on X chromosome








##20151216
##Flanking sequence of SNPs for Sudhir custom genotyping array

#On NHS 
mkdir /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork
mkdir /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence
#dl'ed MM5.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz and P2.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz from e-mail to NHS to get files ready & prepped

#/home/shared/software/bedtools/BEDTools-Version-2.16.2/bin/.
mkdir /mnt/lustre/home/mturchin20/Data/HumanGenome/GRCh37
mkdir /mnt/lustre/home/mturchin20/Data/HumanGenome/GRCh37/Version1
cd /mnt/lustre/home/mturchin20/Data/HumanGenome/GRCh37/Version1
wget ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/reference/human_g1k_v37.fasta.gz
wget ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/reference/human_g1k_v37.fasta.fai

cd /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence
[  mturchin20@spudhead  ~/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence]$zcat /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz | awk '{ print "chr" $1, "\t", $2, "\t", $3 }' | tail -n +2 | perl -lane 'print join("\t", @F);' > /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.bed
[  mturchin20@spudhead  ~/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence]$zcat /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz | awk '{ print "chr" $1, "\t", $2, "\t", $3 }' | tail -n +2 | perl -lane 'print join("\t", @F);' > /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.bed

wget http://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64/fetchChromSizes
#From https://www.biostars.org/p/70795/
bash fetchChromSizes hg19 > hg19.chrom.sizes

[  mturchin20@spudling77  ~/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence]$bedtools flank -i /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.bed -g hg19.chrom.sizes -b 100 > /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Flank100.bed
[  mturchin20@spudling77  ~/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence]$bedtools flank -i /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.bed -g hg19.chrom.sizes -b 100 > /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Flank100.bed

~~~
[  mturchin20@spudling77  ~/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence]$cat /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Flank100.bed | head -n 10
chrX    16863025        16863125
chrX    16863127        16863227
chrX    16863034        16863134
chrX    16863136        16863236
chrX    16863590        16863690
chrX    16863692        16863792
chrX    16863682        16863782
chrX    16863784        16863884
chrX    16863770        16863870
chrX    16863872        16863972
[  mturchin20@spudling77  ~/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence]$cat /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Flank100.bed | head -n 10
chrX    2821483 2821583
chrX    2821585 2821685
chrX    2821681 2821781
chrX    2821783 2821883
chrX    2821747 2821847
chrX    2821849 2821949
chrX    2822002 2822102
chrX    2822104 2822204
chrX    2823032 2823132
chrX    2823134 2823234
~~~

bedtools getfasta -fi /mnt/lustre/home/mturchin20/Data/HumanGenome/GRCh37/Version1/human_g1k_v37.fasta.gz -bed /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Flank100.bed -fo /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Flank100.fasta
[  mturchin20@bigmem03  ~]$bedtools getfasta -fi /mnt/lustre/home/mturchin20/Data/HumanGenome/GRCh37/Version1/human_g1k_v37.fasta -bed <( cat /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Flank100.bed | sed 's/chr//g') -fo /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Flank100.fasta

#Realized flank creates two outputs per single entry, where it increases the left and right side of the given bed sequence separately. Using slop instead should increase both edges and keep the ouptut as a single entry, i.e. both sides should be increased on the same single line of output

bedtools slop -i /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.bed -g /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/hg19.chrom.sizes -b 100 > /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Slop100.bed
bedtools slop -i /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.bed -g /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/hg19.chrom.sizes -b 100 > /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Slop100.bed

bedtools getfasta -fi /mnt/lustre/home/mturchin20/Data/HumanGenome/GRCh37/Version1/human_g1k_v37.fasta -bed <(cat /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Slop100.bed | sed 's/chr//g') -fo /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Slop100.fasta
bedtools getfasta -fi /mnt/lustre/home/mturchin20/Data/HumanGenome/GRCh37/Version1/human_g1k_v37.fasta -bed <(cat /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Slop100.bed | sed 's/chr//g') -fo /mnt/lustre/home/mturchin20/Lab_Stuff/StephensLab/NUWrk/HIVStff/PG/ScratchWork/SudhirGenotyping_FlankingSequence/P2.skatO.nonsyn.qVals.ACs.VariantLevel.Slop100.fasta

bedtools getfasta [OPTIONS] -fi <input FASTA> -bed <BED/GFF/VCF> -fo <output FASTA>

#20160117 -- Picking this back up here now on the Miranda servers since they are back online

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | awk '{ print "chr" $1, "\t", $2, "\t", $3 }' | tail -n +2 | perl -lane 'print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.bed

#in /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence

wget http://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64/fetchChromSizes
#From https://www.biostars.org/p/70795/
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence/fetchChromSizes hg19 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence/hg19.chrom.sizes
#20160117 NOTE -- Used GRCh37 from 1000G, which is hg19 (see https://www.biostars.org/p/13879/ & human_g1k_v37.fasta)

~~~
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence]$cat hg19.chrom.sizes | head -n 24
chr1    249250621
chr2    243199373
chr3    198022430
chr4    191154276
chr5    180915260
chr6    171115067
chr7    159138663
chrX    155270560
chr8    146364022
chr9    141213431
chr10   135534747
chr11   135006516
chr12   133851895
chr13   115169878
chr14   107349540
chr15   102531392
chr16   90354753
chr17   81195210
chr18   78077248
chr20   63025520
chrY    59373566
chr19   59128983
chr22   51304566
chr21   48129895
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence]$cat hg19.chrom.sizes | wc        
     93     186    1971
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence]$cat hg19.chrom.sizes | grep "_" | wc
     68     136    1595
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence]$cat hg19.chrom.sizes | grep -v "_" | wc
     25      50     376
~~~

bedtools flank -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.bed -g /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence/hg19.chrom.sizes -b 100 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Flank100.bed
bedtools slop -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.bed -g /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence/hg19.chrom.sizes -b 100 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.bed

~~~
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Flank100.bed | head -n 10
chrX    16862414        16862514
chrX    16862516        16862616
chrX    16863025        16863125
chrX    16863127        16863227
chrX    16863034        16863134
chrX    16863136        16863236
chrX    16863488        16863588
chrX    16863590        16863690
chrX    16863583        16863683
chrX    16863685        16863785
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.bed | head -n 10
chrX    16862415        16862615
chrX    16863026        16863226
chrX    16863035        16863235
chrX    16863489        16863689
chrX    16863584        16863784
chrX    16863591        16863791
chrX    16863683        16863883
chrX    16863771        16863971
chrX    16863925        16864125
chrX    16864006        16864206
~~~

bedtools getfasta -fi /home/michaelt/Data/HumanGenome/GRCh37/Version1/human_g1k_v37.fasta -bed <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.bed | sed 's/chr//g') -fo /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.fasta

join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'print $F[0], "_", $F[1], "\t$F[0]\t$F[1]\t$F[3]\t$F[4]\t$F[9]\t$F[10]";' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.fasta | perl -ane 'if ($F[0] =~ m/^>/) { print join("\t", @F), "\t"; } else { print join("\t", @F), "\n"; }' | perl -lane 'my $Chr = ((split(/\>/, ((split(/\:/, $F[0]))[0])))[1]); my $BP = ((split(/\-/, ((split(/\:/, $F[0]))[1])))[0]) + 100; print $Chr, "_", $BP, "\t$F[0]\t$Chr\t$BP\t$F[1]";' | sort) | perl -lane 'my @fasta = split(//, $F[$#F]); my $origRef = $fasta[99]; $fasta[99] = "[$F[5]/$F[6]]"; $F[$#F] = join("", @fasta); push(@F, $origRef); print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.FullAnnotation.vs1.txt

~~~
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.fasta | head -n 10
>X:16862415-16862615
ACTGTAGATGGAAATAACTGTGAAAAAAATGATGCAGATATCTTCTAGTTTGTGCTAAACACACTGCTTTATTTTTACAGCCCACGTCTTTCATGAGGATACGAATTGTTAAGAGGCAGTCTCGTTTTGCTTTTCAAAGACATTTTGTAGAGATTTTTCACTAATGTGAATCTGATTTTATCTACTTGATTCTGTATAGA
>X:16863026-16863226
AGAATCAGAGGATAAAGAAGCCATAAGCCACCCCACTTACATTTCCTACTATACAATGCCTTTTTGGCGCTTGATAAATCAAGCATTCATGTAGCATTACATTCAACAGAAACATTTCTCGTACTTTGGGTTTAAGATCCTTGTCCCTCCAGTTCGGATGTCGTGACATCTGACTCTTCATCATTGTAAATATTTTCAGC
>X:16863035-16863235
GGATAAAGAAGCCATAAGCCACCCCACTTACATTTCCTACTATACAATGCCTTTTTGGCGCTTGATAAATCAAGCATTCATGTAGCATTACATTCAACAGAAACATTTCTCGTACTTTGGGTTTAAGATCCTTGTCCCTCCAGTTCGGATGTCGTGACATCTGACTCTTCATCATTGTAAATATTTTCAGCCTGGAGATA
>X:16863489-16863689
GCATCGTTTAGAGGAAGTGGTAGCCCAGATGTTATGGAAGTCTCGGTTAACAACAACTTAGAAATGTCAGCTACTTAGTGCTCATCCTCAGAGAACTTGCTGTCATTTGAATGCCTTGACCTCTCCTTCCTGTATCCCTCATGCCTACCACACAGAACAGACACTGGAGACAGAGTAGCGACGCCAAGACTCCAGTGTAT
>X:16863584-16863784
CTTGCTGTCATTTGAATGCCTTGACCTCTCCTTCCTGTATCCCTCATGCCTACCACACAGAACAGACACTGGAGACAGAGTAGCGACGCCAAGACTCCAGTGTATGAAGAGCACAACAACAATGAAAATCAGTGCAGATGCAGAGATTTCTCCTGACCTGACAGGTGCTACAAATAGCAAACAGTGGACGAAGCACCATA
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.fasta | perl -ane 'if ($F[0] =~ m/^>/) { print join("\t", @F), "\t"; } else { print join("\t", @F), "\n"; }' | perl -lane 'my $Chr = ((split(/\>/, ((split(/\:/, $F[0]))[0])))[1]); my $BP = ((split(/\-/, ((split(/\:/, $F[0]))[1])))[0]) + 100; print $Chr, "_", $BP, "\t", join("\t", @F);' | head -n 10
X_16862515      >X:16862415-16862615    ACTGTAGATGGAAATAACTGTGAAAAAAATGATGCAGATATCTTCTAGTTTGTGCTAAACACACTGCTTTATTTTTACAGCCCACGTCTTTCATGAGGATACGAATTGTTAAGAGGCAGTCTCGTTTTGCTTTTCAAAGACATTTTGTAGAGATTTTTCACTAATGTGAATCTGATTTTATCTACTTGATTCTGTATAGA
X_16863126      >X:16863026-16863226    AGAATCAGAGGATAAAGAAGCCATAAGCCACCCCACTTACATTTCCTACTATACAATGCCTTTTTGGCGCTTGATAAATCAAGCATTCATGTAGCATTACATTCAACAGAAACATTTCTCGTACTTTGGGTTTAAGATCCTTGTCCCTCCAGTTCGGATGTCGTGACATCTGACTCTTCATCATTGTAAATATTTTCAGC
X_16863135      >X:16863035-16863235    GGATAAAGAAGCCATAAGCCACCCCACTTACATTTCCTACTATACAATGCCTTTTTGGCGCTTGATAAATCAAGCATTCATGTAGCATTACATTCAACAGAAACATTTCTCGTACTTTGGGTTTAAGATCCTTGTCCCTCCAGTTCGGATGTCGTGACATCTGACTCTTCATCATTGTAAATATTTTCAGCCTGGAGATA
X_16863589      >X:16863489-16863689    GCATCGTTTAGAGGAAGTGGTAGCCCAGATGTTATGGAAGTCTCGGTTAACAACAACTTAGAAATGTCAGCTACTTAGTGCTCATCCTCAGAGAACTTGCTGTCATTTGAATGCCTTGACCTCTCCTTCCTGTATCCCTCATGCCTACCACACAGAACAGACACTGGAGACAGAGTAGCGACGCCAAGACTCCAGTGTAT
X_16863684      >X:16863584-16863784    CTTGCTGTCATTTGAATGCCTTGACCTCTCCTTCCTGTATCCCTCATGCCTACCACACAGAACAGACACTGGAGACAGAGTAGCGACGCCAAGACTCCAGTGTATGAAGAGCACAACAACAATGAAAATCAGTGCAGATGCAGAGATTTCTCCTGACCTGACAGGTGCTACAAATAGCAAACAGTGGACGAAGCACCATA
X_16863691      >X:16863591-16863791    TCATTTGAATGCCTTGACCTCTCCTTCCTGTATCCCTCATGCCTACCACACAGAACAGACACTGGAGACAGAGTAGCGACGCCAAGACTCCAGTGTATGAAGAGCACAACAACAATGAAAATCAGTGCAGATGCAGAGATTTCTCCTGACCTGACAGGTGCTACAAATAGCAAACAGTGGACGAAGCACCATATGGGACT
X_16863783      >X:16863683-16863883    GTGTATGAAGAGCACAACAACAATGAAAATCAGTGCAGATGCAGAGATTTCTCCTGACCTGACAGGTGCTACAAATAGCAAACAGTGGACGAAGCACCATATGGGACTAAAGATTGTATTGTTGAAATGGAAAAAGTTCTTTTAAACGGTTTCTGTAAATGTTTGGCATAATATGCACCTACATATTCGAAATAACTTTA
X_16863871      >X:16863771-16863971    ACGAAGCACCATATGGGACTAAAGATTGTATTGTTGAAATGGAAAAAGTTCTTTTAAACGGTTTCTGTAAATGTTTGGCATAATATGCACCTACATATTCGAAATAACTTTATTACATTATGTAAGTAGTAAATCTCTCCTTTACAAGACAGTCATTCAAGAAAAACATTTTAACTCACCATTTGCCATATCTGCATGAT
X_16864025      >X:16863925-16864125    ATTCAAGAAAAACATTTTAACTCACCATTTGCCATATCTGCATGATGTTATCCTCAGACACTGAGCAAATGACCCAAGGCTCATTGGGGTTCCAGCTAAAATCTGAAATCTTAGCAGTGTGTCCTCCATGAATAAACTGTTACAAGAACAAAAAAAGCTTTGATTTCATATACTCTCCTGTTAATCCTAACAAAGGTTTT
X_16864106      >X:16864006-16864206    CATTGGGGTTCCAGCTAAAATCTGAAATCTTAGCAGTGTGTCCTCCATGAATAAACTGTTACAAGAACAAAAAAAGCTTTGATTTCATATACTCTCCTGTTAATCCTAACAAAGGTTTTACTAAGTTTTTAGAGCTATCAGTATTTCAACTGGGGGTGGGTGCTGTCTGGTTTTAGAAGGTGGTACAGGCTCAGTAATCT
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence]$join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'print $F[0], "_", $F[1], "\t$F[0]\t$F[1]\t$F[3]\t$F[4]\t$F[9]\t$F[10]";' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.fasta | perl -ane 'if ($F[0] =~ m/^>/) { print join("\t", @F), "\t"; } else { print join("\t", @F), "\n"; }' | perl -lane 'my $Chr = ((split(/\>/, ((split(/\:/, $F[0]))[0])))[1]); my $BP = ((split(/\-/, ((split(/\:/, $F[0]))[1])))[0]) + 100; print $Chr, "_", $BP, "\t$F[0]\t$Chr\t$BP\t$F[1]";' | sort) | head -n 10
10_103190011 10 103190011 intronic BTRC A G >10:103189911-103190111 10 103190011 TGGCCACTACTGCCCCTTTTGGGATTCTCTCTGGGGTTAAAGCCAGTAAGTAGTGTGTTTCTTAAAGATGTTTCTATCATAAAGGTCAGCTGCAACTGGAATATTGCGTTCAGCCAAGCCTTAAAAATTCCTGTAATCTGTGCCATCCTGTCTCATACTGTCCCATCTCATAGTTGTCCTCTCTCTGCAGTGCTCTATGC
10_103190439 10 103190439 intronic BTRC A G >10:103190339-103190539 10 103190439 TCACTCAGTACTCCTTTAATGATTAGTTTATAATGTAGACCCAGACTGATCCACTCTTAAGACAATGTCATCATGATTGGTTACAAAAATTGACATAGTACAGCTTGACTGTTGGCTTTCAGATCAAGTAGAAGTATGATAGTAGATTTTAAAGTAATCTCAATAAATAGCAGTTCAATACGTAATATTTGCTCTGGACC
10_103190521 10 103190521 intronic BTRC G A >10:103190421-103190621 10 103190521 ACAAAAATTGACATAGTACAGCTTGACTGTTGGCTTTCAGATCAAGTAGAAGTATGATAGTAGATTTTAAAGTAATCTCAATAAATAGCAGTTCAATACGTAATATTTGCTCTGGACCTACATAAAGCCAATTAGTATAATCAATCAATTAATTACAATCAGTTTCCTGAACATTTTCTTATTTTTTTTAAATGATTACT
10_103190533 10 103190533 intronic BTRC C T >10:103190433-103190633 10 103190533 ATAGTACAGCTTGACTGTTGGCTTTCAGATCAAGTAGAAGTATGATAGTAGATTTTAAAGTAATCTCAATAAATAGCAGTTCAATACGTAATATTTGCTCTGGACCTACATAAAGCCAATTAGTATAATCAATCAATTAATTACAATCAGTTTCCTGAACATTTTCTTATTTTTTTTAAATGATTACTCAAAGTATAAAG
10_103221343 10 103221343 intronic BTRC A G >10:103221243-103221443 10 103221343 AATGTTTTTTTAAGTCCTATGCTACTTTTATCAAAGAAAATAAAATCAAATCTGAGAAGAGGGGGAAGAGAAATAACTATTTTTAGGACCCAGACTTGGAGATCTGTGCCAACCAAACATCTACTATAATCGAGAGAAGGTAATAATCATGGTTTTAGTTGTCATCAGCTAAGTGTGGGCTGGAACTTTCTTGTATCCAG
10_103221377 10 103221377 intronic BTRC G A >10:103221277-103221477 10 103221377 AGAAAATAAAATCAAATCTGAGAAGAGGGGGAAGAGAAATAACTATTTTTAGGACCCAGACTTGGAGATCTGTGCCAACCAAACATCTACTATAATCGAGAGAAGGTAATAATCATGGTTTTAGTTGTCATCAGCTAAGTGTGGGCTGGAACTTTCTTGTATCCAGTGTCAACCTGTGTCAAAGTGCAAGCTAGAGATCT
10_103221446 10 103221446 intronic BTRC T C >10:103221346-103221546 10 103221446 CTGTGCCAACCAAACATCTACTATAATCGAGAGAAGGTAATAATCATGGTTTTAGTTGTCATCAGCTAAGTGTGGGCTGGAACTTTCTTGTATCCAGTGTCAACCTGTGTCAAAGTGCAAGCTAGAGATCTAATGAATATGATCTCCTACACAATGCTTAAAGATACATGAAATACTTCCCATCTAAATCCTTACATAAT
10_103221659 10 103221659 intronic BTRC A T >10:103221559-103221759 10 103221659 TGGGCATATGTTTTCTTTTACATATGTGTATATCTGTATGTATACAAATCTTCAAGTGAATAATTGTATAAATATATGTATGTTCTAACTTACAGAATTAGAAATGTAATTCAGTTTACTCCCAAAGGATAAGATTGAAGATAATGAGAACTGAATTAAAGCTTACTTTCTTTCACAGAATTCCTCAGAGAGAGAAGACT
10_103221969 10 103221969 intronic BTRC A G >10:103221869-103222069 10 103221969 CAGGAGGAAACCCTTGATTTGCCATTCTTTAAGCACTGGAGCTTATATTAAGAGTACTCGGACGTTGGCCTGGCCATAGCCATCACTTTCAATCAGGTGACATGGTTTTCCCAGTTATTTAAAAAACATTCTAGGTATACATGTACAGGACAGCTGGTTGGCATTTTTAAAAAAGTATAAAAACAGATCTGCCTGTAATA
10_103238644 10 103238644 intronic BTRC C T >10:103238544-103238744 10 103238644 AAAAAAAAAGAAAAGAAAAAAAGAGAAAGAAACATGTCCTCTGTGGGAGGCTGAGGCAGGAGAATCGCTTGAACCCCGGAGGCGGAGGTTGCAGTGAGCCGAGATCACGCCACTGCACCCCAGCCTGGGTGACAGAGCAAGACTCCATCTGGGGAAAAAAAAAAAAAAAAAAGAAACATGCCCTCTAACTACCTTCAGAT
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence]$join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'print $F[0], "_", $F[1], "\t$F[0]\t$F[1]\t$F[3]\t$F[4]\t$F[9]\t$F[10]";' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.fasta | perl -ane 'if ($F[0] =~ m/^>/) { print join("\t", @F), "\t"; } else { print join("\t", @F), "\n"; }' | perl -lane 'my $Chr = ((split(/\>/, ((split(/\:/, $F[0]))[0])))[1]); my $BP = ((split(/\-/, ((split(/\:/, $F[0]))[1])))[0]) + 100; print $Chr, "_", $BP, "\t$F[0]\t$Chr\t$BP\t$F[1]";' | sort) | perl -lane 'my @fasta = split(//, $F[$#F]); my $origRef = $fasta[99]; $fasta[99] = "[$F[5]/$F[6]]"; $F[$#F] = join("", @fasta); push(@F, $origRef); print join("\t", @F);' | head -n 10
10_103190011    10      103190011       intronic        BTRC    A       G       >10:103189911-103190111 10      103190011       TGGCCACTACTGCCCCTTTTGGGATTCTCTCTGGGGTTAAAGCCAGTAAGTAGTGTGTTTCTTAAAGATGTTTCTATCATAAAGGTCAGCTGCAACTGG[A/G]ATATTGCGTTCAGCCAAGCCTTAAAAATTCCTGTAATCTGTGCCATCCTGTCTCATACTGTCCCATCTCATAGTTGTCCTCTCTCTGCAGTGCTCTATGC       A
10_103190439    10      103190439       intronic        BTRC    A       G       >10:103190339-103190539 10      103190439       TCACTCAGTACTCCTTTAATGATTAGTTTATAATGTAGACCCAGACTGATCCACTCTTAAGACAATGTCATCATGATTGGTTACAAAAATTGACATAGT[A/G]CAGCTTGACTGTTGGCTTTCAGATCAAGTAGAAGTATGATAGTAGATTTTAAAGTAATCTCAATAAATAGCAGTTCAATACGTAATATTTGCTCTGGACC       A
10_103190521    10      103190521       intronic        BTRC    G       A       >10:103190421-103190621 10      103190521       ACAAAAATTGACATAGTACAGCTTGACTGTTGGCTTTCAGATCAAGTAGAAGTATGATAGTAGATTTTAAAGTAATCTCAATAAATAGCAGTTCAATAC[G/A]TAATATTTGCTCTGGACCTACATAAAGCCAATTAGTATAATCAATCAATTAATTACAATCAGTTTCCTGAACATTTTCTTATTTTTTTTAAATGATTACT       G
10_103190533    10      103190533       intronic        BTRC    C       T       >10:103190433-103190633 10      103190533       ATAGTACAGCTTGACTGTTGGCTTTCAGATCAAGTAGAAGTATGATAGTAGATTTTAAAGTAATCTCAATAAATAGCAGTTCAATACGTAATATTTGCT[C/T]TGGACCTACATAAAGCCAATTAGTATAATCAATCAATTAATTACAATCAGTTTCCTGAACATTTTCTTATTTTTTTTAAATGATTACTCAAAGTATAAAG       C
10_103221343    10      103221343       intronic        BTRC    A       G       >10:103221243-103221443 10      103221343       AATGTTTTTTTAAGTCCTATGCTACTTTTATCAAAGAAAATAAAATCAAATCTGAGAAGAGGGGGAAGAGAAATAACTATTTTTAGGACCCAGACTTGG[A/G]GATCTGTGCCAACCAAACATCTACTATAATCGAGAGAAGGTAATAATCATGGTTTTAGTTGTCATCAGCTAAGTGTGGGCTGGAACTTTCTTGTATCCAG       A
10_103221377    10      103221377       intronic        BTRC    G       A       >10:103221277-103221477 10      103221377       AGAAAATAAAATCAAATCTGAGAAGAGGGGGAAGAGAAATAACTATTTTTAGGACCCAGACTTGGAGATCTGTGCCAACCAAACATCTACTATAATCGA[G/A]AGAAGGTAATAATCATGGTTTTAGTTGTCATCAGCTAAGTGTGGGCTGGAACTTTCTTGTATCCAGTGTCAACCTGTGTCAAAGTGCAAGCTAGAGATCT       G
10_103221446    10      103221446       intronic        BTRC    T       C       >10:103221346-103221546 10      103221446       CTGTGCCAACCAAACATCTACTATAATCGAGAGAAGGTAATAATCATGGTTTTAGTTGTCATCAGCTAAGTGTGGGCTGGAACTTTCTTGTATCCAGTG[T/C]CAACCTGTGTCAAAGTGCAAGCTAGAGATCTAATGAATATGATCTCCTACACAATGCTTAAAGATACATGAAATACTTCCCATCTAAATCCTTACATAAT       T
10_103221659    10      103221659       intronic        BTRC    A       T       >10:103221559-103221759 10      103221659       TGGGCATATGTTTTCTTTTACATATGTGTATATCTGTATGTATACAAATCTTCAAGTGAATAATTGTATAAATATATGTATGTTCTAACTTACAGAATT[A/T]GAAATGTAATTCAGTTTACTCCCAAAGGATAAGATTGAAGATAATGAGAACTGAATTAAAGCTTACTTTCTTTCACAGAATTCCTCAGAGAGAGAAGACT       A
10_103221969    10      103221969       intronic        BTRC    A       G       >10:103221869-103222069 10      103221969       CAGGAGGAAACCCTTGATTTGCCATTCTTTAAGCACTGGAGCTTATATTAAGAGTACTCGGACGTTGGCCTGGCCATAGCCATCACTTTCAATCAGGTG[A/G]CATGGTTTTCCCAGTTATTTAAAAAACATTCTAGGTATACATGTACAGGACAGCTGGTTGGCATTTTTAAAAAAGTATAAAAACAGATCTGCCTGTAATA       A
10_103238644    10      103238644       intronic        BTRC    C       T       >10:103238544-103238744 10      103238644       AAAAAAAAAGAAAAGAAAAAAAGAGAAAGAAACATGTCCTCTGTGGGAGGCTGAGGCAGGAGAATCGCTTGAACCCCGGAGGCGGAGGTTGCAGTGAGC[C/T]GAGATCACGCCACTGCACCCCAGCCTGGGTGACAGAGCAAGACTCCATCTGGGGAAAAAAAAAAAAAAAAAAGAAACATGCCCTCTAACTACCTTCAGAT       C
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.FullAnnotation.vs1.txt | head -n 10
10_103190011    10      103190011       intronic        BTRC    A       G       >10:103189911-103190111 10      103190011       TGGCCACTACTGCCCCTTTTGGGATTCTCTCTGGGGTTAAAGCCAGTAAGTAGTGTGTTTCTTAAAGATGTTTCTATCATAAAGGTCAGCTGCAACTGG[A/G]ATATTGCGTTCAGCCAAGCCTTAAAAATTCCTGTAATCTGTGCCATCCTGTCTCATACTGTCCCATCTCATAGTTGTCCTCTCTCTGCAGTGCTCTATGC       A
10_103190439    10      103190439       intronic        BTRC    A       G       >10:103190339-103190539 10      103190439       TCACTCAGTACTCCTTTAATGATTAGTTTATAATGTAGACCCAGACTGATCCACTCTTAAGACAATGTCATCATGATTGGTTACAAAAATTGACATAGT[A/G]CAGCTTGACTGTTGGCTTTCAGATCAAGTAGAAGTATGATAGTAGATTTTAAAGTAATCTCAATAAATAGCAGTTCAATACGTAATATTTGCTCTGGACC       A
10_103190521    10      103190521       intronic        BTRC    G       A       >10:103190421-103190621 10      103190521       ACAAAAATTGACATAGTACAGCTTGACTGTTGGCTTTCAGATCAAGTAGAAGTATGATAGTAGATTTTAAAGTAATCTCAATAAATAGCAGTTCAATAC[G/A]TAATATTTGCTCTGGACCTACATAAAGCCAATTAGTATAATCAATCAATTAATTACAATCAGTTTCCTGAACATTTTCTTATTTTTTTTAAATGATTACT       G
10_103190533    10      103190533       intronic        BTRC    C       T       >10:103190433-103190633 10      103190533       ATAGTACAGCTTGACTGTTGGCTTTCAGATCAAGTAGAAGTATGATAGTAGATTTTAAAGTAATCTCAATAAATAGCAGTTCAATACGTAATATTTGCT[C/T]TGGACCTACATAAAGCCAATTAGTATAATCAATCAATTAATTACAATCAGTTTCCTGAACATTTTCTTATTTTTTTTAAATGATTACTCAAAGTATAAAG       C
10_103221343    10      103221343       intronic        BTRC    A       G       >10:103221243-103221443 10      103221343       AATGTTTTTTTAAGTCCTATGCTACTTTTATCAAAGAAAATAAAATCAAATCTGAGAAGAGGGGGAAGAGAAATAACTATTTTTAGGACCCAGACTTGG[A/G]GATCTGTGCCAACCAAACATCTACTATAATCGAGAGAAGGTAATAATCATGGTTTTAGTTGTCATCAGCTAAGTGTGGGCTGGAACTTTCTTGTATCCAG       A
10_103221377    10      103221377       intronic        BTRC    G       A       >10:103221277-103221477 10      103221377       AGAAAATAAAATCAAATCTGAGAAGAGGGGGAAGAGAAATAACTATTTTTAGGACCCAGACTTGGAGATCTGTGCCAACCAAACATCTACTATAATCGA[G/A]AGAAGGTAATAATCATGGTTTTAGTTGTCATCAGCTAAGTGTGGGCTGGAACTTTCTTGTATCCAGTGTCAACCTGTGTCAAAGTGCAAGCTAGAGATCT       G
10_103221446    10      103221446       intronic        BTRC    T       C       >10:103221346-103221546 10      103221446       CTGTGCCAACCAAACATCTACTATAATCGAGAGAAGGTAATAATCATGGTTTTAGTTGTCATCAGCTAAGTGTGGGCTGGAACTTTCTTGTATCCAGTG[T/C]CAACCTGTGTCAAAGTGCAAGCTAGAGATCTAATGAATATGATCTCCTACACAATGCTTAAAGATACATGAAATACTTCCCATCTAAATCCTTACATAAT       T
10_103221659    10      103221659       intronic        BTRC    A       T       >10:103221559-103221759 10      103221659       TGGGCATATGTTTTCTTTTACATATGTGTATATCTGTATGTATACAAATCTTCAAGTGAATAATTGTATAAATATATGTATGTTCTAACTTACAGAATT[A/T]GAAATGTAATTCAGTTTACTCCCAAAGGATAAGATTGAAGATAATGAGAACTGAATTAAAGCTTACTTTCTTTCACAGAATTCCTCAGAGAGAGAAGACT       A
10_103221969    10      103221969       intronic        BTRC    A       G       >10:103221869-103222069 10      103221969       CAGGAGGAAACCCTTGATTTGCCATTCTTTAAGCACTGGAGCTTATATTAAGAGTACTCGGACGTTGGCCTGGCCATAGCCATCACTTTCAATCAGGTG[A/G]CATGGTTTTCCCAGTTATTTAAAAAACATTCTAGGTATACATGTACAGGACAGCTGGTTGGCATTTTTAAAAAAGTATAAAAACAGATCTGCCTGTAATA       A
10_103238644    10      103238644       intronic        BTRC    C       T       >10:103238544-103238744 10      103238644       AAAAAAAAAGAAAAGAAAAAAAGAGAAAGAAACATGTCCTCTGTGGGAGGCTGAGGCAGGAGAATCGCTTGAACCCCGGAGGCGGAGGTTGCAGTGAGC[C/T]GAGATCACGCCACTGCACCCCAGCCTGGGTGACAGAGCAAGACTCCATCTGGGGAAAAAAAAAAAAAAAAAAGAAACATGCCCTCTAACTACCTTCAGAT       C
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.FullAnnotation.vs1.txt | awk '{ print $2, "\t", $3, "\t", $6, "\t", $7, "\t", $11, "\t", $12 }' | head -n 10
10       103190011       A       G       TGGCCACTACTGCCCCTTTTGGGATTCTCTCTGGGGTTAAAGCCAGTAAGTAGTGTGTTTCTTAAAGATGTTTCTATCATAAAGGTCAGCTGCAACTGG[A/G]ATATTGCGTTCAGCCAAGCCTTAAAAATTCCTGTAATCTGTGCCATCCTGTCTCATACTGTCCCATCTCATAGTTGTCCTCTCTCTGCAGTGCTCTATGC    A
10       103190439       A       G       TCACTCAGTACTCCTTTAATGATTAGTTTATAATGTAGACCCAGACTGATCCACTCTTAAGACAATGTCATCATGATTGGTTACAAAAATTGACATAGT[A/G]CAGCTTGACTGTTGGCTTTCAGATCAAGTAGAAGTATGATAGTAGATTTTAAAGTAATCTCAATAAATAGCAGTTCAATACGTAATATTTGCTCTGGACC    A
10       103190521       G       A       ACAAAAATTGACATAGTACAGCTTGACTGTTGGCTTTCAGATCAAGTAGAAGTATGATAGTAGATTTTAAAGTAATCTCAATAAATAGCAGTTCAATAC[G/A]TAATATTTGCTCTGGACCTACATAAAGCCAATTAGTATAATCAATCAATTAATTACAATCAGTTTCCTGAACATTTTCTTATTTTTTTTAAATGATTACT    G
10       103190533       C       T       ATAGTACAGCTTGACTGTTGGCTTTCAGATCAAGTAGAAGTATGATAGTAGATTTTAAAGTAATCTCAATAAATAGCAGTTCAATACGTAATATTTGCT[C/T]TGGACCTACATAAAGCCAATTAGTATAATCAATCAATTAATTACAATCAGTTTCCTGAACATTTTCTTATTTTTTTTAAATGATTACTCAAAGTATAAAG    C
10       103221343       A       G       AATGTTTTTTTAAGTCCTATGCTACTTTTATCAAAGAAAATAAAATCAAATCTGAGAAGAGGGGGAAGAGAAATAACTATTTTTAGGACCCAGACTTGG[A/G]GATCTGTGCCAACCAAACATCTACTATAATCGAGAGAAGGTAATAATCATGGTTTTAGTTGTCATCAGCTAAGTGTGGGCTGGAACTTTCTTGTATCCAG    A
10       103221377       G       A       AGAAAATAAAATCAAATCTGAGAAGAGGGGGAAGAGAAATAACTATTTTTAGGACCCAGACTTGGAGATCTGTGCCAACCAAACATCTACTATAATCGA[G/A]AGAAGGTAATAATCATGGTTTTAGTTGTCATCAGCTAAGTGTGGGCTGGAACTTTCTTGTATCCAGTGTCAACCTGTGTCAAAGTGCAAGCTAGAGATCT    G
10       103221446       T       C       CTGTGCCAACCAAACATCTACTATAATCGAGAGAAGGTAATAATCATGGTTTTAGTTGTCATCAGCTAAGTGTGGGCTGGAACTTTCTTGTATCCAGTG[T/C]CAACCTGTGTCAAAGTGCAAGCTAGAGATCTAATGAATATGATCTCCTACACAATGCTTAAAGATACATGAAATACTTCCCATCTAAATCCTTACATAAT    T
10       103221659       A       T       TGGGCATATGTTTTCTTTTACATATGTGTATATCTGTATGTATACAAATCTTCAAGTGAATAATTGTATAAATATATGTATGTTCTAACTTACAGAATT[A/T]GAAATGTAATTCAGTTTACTCCCAAAGGATAAGATTGAAGATAATGAGAACTGAATTAAAGCTTACTTTCTTTCACAGAATTCCTCAGAGAGAGAAGACT    A
10       103221969       A       G       CAGGAGGAAACCCTTGATTTGCCATTCTTTAAGCACTGGAGCTTATATTAAGAGTACTCGGACGTTGGCCTGGCCATAGCCATCACTTTCAATCAGGTG[A/G]CATGGTTTTCCCAGTTATTTAAAAAACATTCTAGGTATACATGTACAGGACAGCTGGTTGGCATTTTTAAAAAAGTATAAAAACAGATCTGCCTGTAATA    A
10       103238644       C       T       AAAAAAAAAGAAAAGAAAAAAAGAGAAAGAAACATGTCCTCTGTGGGAGGCTGAGGCAGGAGAATCGCTTGAACCCCGGAGGCGGAGGTTGCAGTGAGC[C/T]GAGATCACGCCACTGCACCCCAGCCTGGGTGACAGAGCAAGACTCCATCTGGGGAAAAAAAAAAAAAAAAAAGAAACATGCCCTCTAACTACCTTCAGAT    C
~~~

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.FullAnnotation.vs1.txt | awk '{ print $2, "\t", $3, "\t", $6, "\t", $7, "\t", $11, "\t", $12 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.VariantLevel.100bpFlanking.Update1.txt 

#Included the below header in file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.VariantLevel.100bpFlanking.Update1.txt
#CHR	BP	REF	ALT	SEQUENCE	ORIG_POS_100

~~~
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.VariantLevel.100bpFlanking.Update1.txt | wc
  64225  385350 14939978
~~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.VariantLevel.100bpFlanking.Update1.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/.

join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'print $F[0], "_", $F[1], "\t$F[0]\t$F[1]\t$F[3]\t$F[4]\t$F[9]\t$F[10]";' | sort) <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.frq.gz | perl -lane 'print $F[0], "_", $F[1], "\t", $F[0], "_", $F[1], "\t", join("\t", split(/\:/, $F[4])), "\t", join("\t", split(/\:/, $F[5]));' | sort ) | join - <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.fasta | perl -ane 'if ($F[0] =~ m/^>/) { print join("\t", @F), "\t"; } else { print join("\t", @F), "\n"; }' | perl -lane 'my $Chr = ((split(/\>/, ((split(/\:/, $F[0]))[0])))[1]); my $BP = ((split(/\-/, ((split(/\:/, $F[0]))[1])))[0]) + 100; print $Chr, "_", $BP, "\t$F[0]\t$Chr\t$BP\t$F[1]";' | sort) | perl -lane 'my @fasta = split(//, $F[$#F]); my $origRef = $fasta[99]; $fasta[99] = "[$F[5]/$F[6]]"; $F[$#F] = join("", @fasta); push(@F, $origRef); print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.FullAnnotation.vs2.txt

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.FullAnnotation.vs2.txt | perl -lane 'if ($F[5] eq $F[8]) { splice (@F, 10, 2); splice(@F, 7, 2); } elsif ($F[5] eq $F[10]) { splice(@F, 7, 4); } else { print STDERR "Error1a -- $F[0] does not have either ref or alt allele matching with allelef requency"; } print join("\t", @F);' | awk '{ print $2, "\t", $3, "\t", $6, "\t", $7, "\t", $8, "\t", $12, "\t", $13 }' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.VariantLevel.100bpFlanking.Update1.txt.gz

~~~
[  michaelt@node7  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.FullAnnotation.vs2.txt  | perl -lane 'if ($F[5] eq $F[8]) { print "REF"; } elsif ($F[5] eq $F[10]) { print "ALT"; } else { print "NA"; }' | sort | uniq -c
  64224 REF
[  michaelt@node7  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.Slop100.FullAnnotation.vs2.txt | perl -lane 'if ($F[5] eq $F[8]) { splice (@F, 10, 2); splice(@F, 7, 2); } elsif ($F[5] eq $F[10]) { splice(@F, 7, 4); } else { print STDERR "Error1a -- $F[0] does not have either ref or alt allele matching with allelef requency"; } print join("\t", @F);' | awk '{ print $2, "\t", $3, "\t", $6, "\t", $7, "\t", $8, "\t", $12, "\t", $13 }' | head -n 10
10       103190011       A       G       0.998049        TGGCCACTACTGCCCCTTTTGGGATTCTCTCTGGGGTTAAAGCCAGTAAGTAGTGTGTTTCTTAAAGATGTTTCTATCATAAAGGTCAGCTGCAACTGG[A/G]ATATTGCGTTCAGCCAAGCCTTAAAAATTCCTGTAATCTGTGCCATCCTGTCTCATACTGTCCCATCTCATAGTTGTCCTCTCTCTGCAGTGCTCTATGC    A
10       103190439       A       G       0.996753        TCACTCAGTACTCCTTTAATGATTAGTTTATAATGTAGACCCAGACTGATCCACTCTTAAGACAATGTCATCATGATTGGTTACAAAAATTGACATAGT[A/G]CAGCTTGACTGTTGGCTTTCAGATCAAGTAGAAGTATGATAGTAGATTTTAAAGTAATCTCAATAAATAGCAGTTCAATACGTAATATTTGCTCTGGACC    A
10       103190521       G       A       0.996728        ACAAAAATTGACATAGTACAGCTTGACTGTTGGCTTTCAGATCAAGTAGAAGTATGATAGTAGATTTTAAAGTAATCTCAATAAATAGCAGTTCAATAC[G/A]TAATATTTGCTCTGGACCTACATAAAGCCAATTAGTATAATCAATCAATTAATTACAATCAGTTTCCTGAACATTTTCTTATTTTTTTTAAATGATTACT    G
10       103190533       C       T       0.99017         ATAGTACAGCTTGACTGTTGGCTTTCAGATCAAGTAGAAGTATGATAGTAGATTTTAAAGTAATCTCAATAAATAGCAGTTCAATACGTAATATTTGCT[C/T]TGGACCTACATAAAGCCAATTAGTATAATCAATCAATTAATTACAATCAGTTTCCTGAACATTTTCTTATTTTTTTTAAATGATTACTCAAAGTATAAAG    C
10       103221343       A       G       0.99664         AATGTTTTTTTAAGTCCTATGCTACTTTTATCAAAGAAAATAAAATCAAATCTGAGAAGAGGGGGAAGAGAAATAACTATTTTTAGGACCCAGACTTGG[A/G]GATCTGTGCCAACCAAACATCTACTATAATCGAGAGAAGGTAATAATCATGGTTTTAGTTGTCATCAGCTAAGTGTGGGCTGGAACTTTCTTGTATCCAG    A
10       103221377       G       A       0.998661        AGAAAATAAAATCAAATCTGAGAAGAGGGGGAAGAGAAATAACTATTTTTAGGACCCAGACTTGGAGATCTGTGCCAACCAAACATCTACTATAATCGA[G/A]AGAAGGTAATAATCATGGTTTTAGTTGTCATCAGCTAAGTGTGGGCTGGAACTTTCTTGTATCCAGTGTCAACCTGTGTCAAAGTGCAAGCTAGAGATCT    G
10       103221446       T       C       0.996021        CTGTGCCAACCAAACATCTACTATAATCGAGAGAAGGTAATAATCATGGTTTTAGTTGTCATCAGCTAAGTGTGGGCTGGAACTTTCTTGTATCCAGTG[T/C]CAACCTGTGTCAAAGTGCAAGCTAGAGATCTAATGAATATGATCTCCTACACAATGCTTAAAGATACATGAAATACTTCCCATCTAAATCCTTACATAAT    T
10       103221659       A       T       0.998052        TGGGCATATGTTTTCTTTTACATATGTGTATATCTGTATGTATACAAATCTTCAAGTGAATAATTGTATAAATATATGTATGTTCTAACTTACAGAATT[A/T]GAAATGTAATTCAGTTTACTCCCAAAGGATAAGATTGAAGATAATGAGAACTGAATTAAAGCTTACTTTCTTTCACAGAATTCCTCAGAGAGAGAAGACT    A
10       103221969       A       G       0.999346        CAGGAGGAAACCCTTGATTTGCCATTCTTTAAGCACTGGAGCTTATATTAAGAGTACTCGGACGTTGGCCTGGCCATAGCCATCACTTTCAATCAGGTG[A/G]CATGGTTTTCCCAGTTATTTAAAAAACATTCTAGGTATACATGTACAGGACAGCTGGTTGGCATTTTTAAAAAAGTATAAAAACAGATCTGCCTGTAATA    A
10       103238644       C       T       0.998663        AAAAAAAAAGAAAAGAAAAAAAGAGAAAGAAACATGTCCTCTGTGGGAGGCTGAGGCAGGAGAATCGCTTGAACCCCGGAGGCGGAGGTTGCAGTGAGC[C/T]GAGATCACGCCACTGCACCCCAGCCTGGGTGACAGAGCAAGACTCCATCTGGGGAAAAAAAAAAAAAAAAAAGAAACATGCCCTCTAACTACCTTCAGAT    C
[  michaelt@node7  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.VariantLevel.100bpFlanking.Update1.txt.gz | perl -lane 'if ($F[2] eq $F[$#F]) { print "REF"; } elsif ($F[3] eq $F[$#F]) { print "ALT"; } else { print "NA"; }' | sort | uniq -c
      1 NA
  64224 REF
~~~

#Included the below header in file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.VariantLevel.100bpFlanking.Update1.txt.gz
#CHR    BP      REF     ALT     AF_REF  SEQUENCE        ORIG_POS_100

~~~
[  michaelt@node7  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.VariantLevel.100bpFlanking.Update1.txt.gz | wc
  64225  449575 15638559
~~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.VariantLevel.100bpFlanking.Update1.txt.gz mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/.


















##2015121226
##Redoing X-chromosome analyses -- or all analyses using proper X-chromosome SNPs now?

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz | grep ^X | perl -lane 'print join("\t", @F[0..9]);' | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz | grep ^X | perl -lane 'print join("\t", @F[0..9]);' | head -n 10

~~~
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz | grep ^X | perl -lane 'print join("\t", @F[0..9]);' | wc
   1066   10660  362480
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz | grep ^X | perl -lane 'print join("\t", @F[0..9]);' | head -n 10
X       16862515        rs57547802      T       C       2557.4  VQSRTrancheSNP99.00to99.90      AC=25;AF=0.014;AN=1792;BaseQRankSum=0.648;DB;DP=7174;Dels=0.00;FS=3.826;HaplotypeScore=1.0990;InbreedingCoeff=0.3029;MLEAC=16;MLEAF=8.929e-03;MQ=58.48;MQ0=0;MQRankSum=2.699;POSITIVE_TRAIN_SITE;QD=19.37;ReadPosRankSum=1.308;VQSLOD=1.24;culprit=DP;AA=T      GT:AD:DP:GQ:PL  0/0:3,0:3:9:0,9,102
X       16863126        .       C       T       302.19  VQSRTrancheSNP99.00to99.90      AC=3;AF=1.663e-03;AN=1804;BaseQRankSum=-0.178;DP=13196;Dels=0.00;FS=4.950;HaplotypeScore=0.9367;InbreedingCoeff=0.1785;MLEAC=2;MLEAF=1.109e-03;MQ=59.99;MQ0=0;MQRankSum=-0.264;QD=21.59;ReadPosRankSum=0.839;VQSLOD=0.764;culprit=HaplotypeScore;AA=C  GT:AD:DP:GQ:PL   0/0:11,0:11:33:0,33,373
X       16863135        .       G       A       196.77  VQSRTrancheSNP99.00to99.90      AC=4;AF=2.217e-03;AN=1804;BaseQRankSum=-3.402;DP=13233;Dels=0.00;FS=0.000;HaplotypeScore=1.0355;InbreedingCoeff=0.1874;MLEAC=1;MLEAF=5.543e-04;MQ=60.00;MQ0=0;MQRankSum=1.004;QD=6.79;ReadPosRankSum=-0.105;VQSLOD=1.47;culprit=HaplotypeScore;AA=G    GT:AD:DP:GQ:PL   0/0:11,0:11:33:0,33,356
X       16863409        rs140423015     C       A       2121.87 VQSRTrancheSNP99.00to99.90      AC=4;AF=2.215e-03;AN=1806;BaseQRankSum=12.295;DB;DP=12938;Dels=0.00;FS=4.024;HaplotypeScore=1.1471;InbreedingCoeff=0.3435;MLEAC=4;MLEAF=2.215e-03;MQ=60.00;MQ0=0;MQRankSum=-0.049;POSITIVE_TRAIN_SITE;QD=33.68;ReadPosRankSum=-1.930;VQSLOD=0.781;culprit=HaplotypeScore;AA=C   GT:AD:DP:GQ:PL  0/0:8,0:8:24:0,24,239
X       16863589        rs4828535       C       G       177230  VQSRTrancheSNP99.00to99.90      AC=772;AF=0.428;AN=1804;BaseQRankSum=-11.394;DB;DP=13068;Dels=0.00;FS=1.409;HaplotypeScore=1.3697;InbreedingCoeff=0.9539;MLEAC=758;MLEAF=0.420;MQ=59.99;MQ0=0;MQRankSum=1.878;POSITIVE_TRAIN_SITE;QD=30.71;ReadPosRankSum=1.540;VQSLOD=1.88;culprit=HaplotypeScore;AA=g GT:AD:DP:GQ:PL  1/1:0,10:10:30:322,30,0
X       16863684        rs142880774     G       C       5104.19 VQSRTrancheSNP99.00to99.90      AC=19;AF=0.011;AN=1804;BaseQRankSum=-1.453;DB;DP=12950;Dels=0.00;FS=0.000;HaplotypeScore=1.1131;InbreedingCoeff=0.6822;MLEAC=18;MLEAF=9.978e-03;MQ=60.00;MQ0=0;MQRankSum=-1.308;POSITIVE_TRAIN_SITE;QD=29.33;ReadPosRankSum=2.614;VQSLOD=0.852;culprit=HaplotypeScore;AA=G      GT:AD:DP:GQ:PL  0/0:10,0:10:30:0,30,332
X       16863691        .       A       G       133.8   VQSRTrancheSNP99.00to99.90      AC=2;AF=1.109e-03;AN=1804;BaseQRankSum=-4.714;DP=12901;Dels=0.00;FS=3.781;HaplotypeScore=1.2887;InbreedingCoeff=0.1985;MLEAC=1;MLEAF=5.543e-04;MQ=59.99;MQ0=0;MQRankSum=-0.199;QD=22.30;ReadPosRankSum=-1.479;VQSLOD=0.245;culprit=HaplotypeScore;AA=A GT:AD:DP:GQ:PL   0/0:10,0:10:30:0,30,339
X       16863783        rs189523407     T       C       415.41  VQSRTrancheSNP99.00to99.90      AC=2;AF=1.110e-03;AN=1802;BaseQRankSum=-1.324;DB;DP=12183;Dels=0.00;FS=0.000;HaplotypeScore=1.3856;InbreedingCoeff=0.1726;MLEAC=2;MLEAF=1.110e-03;MQ=59.99;MQ0=0;MQRankSum=0.092;POSITIVE_TRAIN_SITE;QD=27.69;ReadPosRankSum=-1.622;VQSLOD=0.869;culprit=HaplotypeScore;AA=T    GT:AD:DP:GQ:PL  0/0:9,0:9:27:0,27,298
X       16863871        .       C       T       192.53  VQSRTrancheSNP99.00to99.90      AC=6;AF=3.337e-03;AN=1798;BaseQRankSum=3.176;DP=11514;Dels=0.00;FS=0.821;HaplotypeScore=1.1719;InbreedingCoeff=0.1145;MLEAC=1;MLEAF=5.562e-04;MQ=59.99;MQ0=0;MQRankSum=-1.365;QD=4.94;ReadPosRankSum=1.461;VQSLOD=0.931;culprit=QD;AA=C GT:AD:DP:GQ:PL 0/0:8,0:8:24:0,24,235
X       16864025        .       A       G       110.75  VQSRTrancheSNP99.00to99.90      AC=1;AF=5.562e-04;AN=1798;BaseQRankSum=0.195;DP=10072;Dels=0.00;FS=3.262;HaplotypeScore=1.3899;InbreedingCoeff=-0.0087;MLEAC=1;MLEAF=5.562e-04;MQ=59.99;MQ0=0;MQRankSum=-0.284;QD=7.38;ReadPosRankSum=-3.435;VQSLOD=-7.560e-02;culprit=HaplotypeScore;AA=A      GT:AD:DP:GQ:PL  0/0:7,0:7:21:0,21,228

~~~

#From http://vcftools.sourceforge.net/man_latest.html
~~~
--positions <filename> 
--exclude-positions <filename>

Include or exclude a set of sites on the basis of a list of positions in a file. Each line of the input file should contain a (tab-separated) chromosome and position. The file can have comment lines that start with a "#", they will be ignored.
~~~

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz | grep ^X | awk '{ print $1, "\t", $2}' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.XChr.ChrBPs

~~~
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.XChr.ChrBPs | wc
   1066    2132   14047
~~~   

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz --positions /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.XChr.ChrBPs --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr 

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.recode.vcf

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.recode.vcf.gz | grep -v ^# | perl -lane 'my $hetCounts = 0; for (my $i=9; $i <= $#F; $i++) { my $altCounts = 0; my @indvInfo1 = split(/\:/, $F[$i]); my @genoInfo1 = split(/\//, $indvInfo1[0]); if ($genoInfo1[0] eq "1") { $altCounts++; } if ($genoInfo1[1] eq "1") { $altCounts++; } if ($altCounts eq "1") { $hetCounts++; } } print join("\t", @F[0..8]), "\t", $hetCounts; ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.perSNPhetCounts.txt  

~~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.perSNPhetCounts.txt | awk '{ print $10 }' | sort | uniq -c | sort -g -k 2,2
    211 0
    194 1
    175 2
    149 3
     98 4
     77 5
     44 6
     26 7
     19 8
     12 9
     13 10
     10 11
      2 12
      3 13
      7 14
      4 15
      1 17
      6 18
      2 19
      1 21
      1 22
      1 23
      1 24
      2 25
      2 31
      1 37
      1 53
      1 56
      1 121
      1 273
~~~

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDs --positions /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.XChr.ChrBPs  --non-ref-ac 1 --recode  --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3 

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.recode.vcf

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.recode.vcf.gz | grep -v ^# | perl -lane 'my $hetCounts = 0; for (my $i=9; $i <= $#F; $i++) { my $altCounts = 0; my @indvInfo1 = split(/\:/, $F[$i]); my @genoInfo1 = split(/\//, $indvInfo1[0]); if ($genoInfo1[0] eq "1") { $altCounts++; } if ($genoInfo1[1] eq "1") { $altCounts++; } if ($altCounts eq "1") { $hetCounts++; } } print join("\t", @F[0..8]), "\t", $hetCounts; ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.perSNPhetCounts.txt

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.perSNPhetCounts.txt | awk '{ print $10 }' | sort | uniq -c | sort -g -k 2,2

~~~
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.perSNPhetCounts.txt | awk '{ print $10 }' | sort | uniq -c | sort -g -k 2,2
    180 0
    212 1
    187 2
    137 3
     96 4
     50 5
     34 6
     23 7
     14 8
     11 9
      6 10
      1 11
      1 12
      3 13
      1 14
      2 15
      3 16
      2 20
      1 23
      1 31
      1 43
      1 50
      1 103
      1 230
~~~

#So looks like maybe cutting off variants with het calls above 10 or something like that? Probably a nice balance of 'conservative but also not removing too many variants by doing so'

#20151227 NOTE -- I was considering identifying whether any individuals were particularly causative of heterozygous calls among this subset of SNPs (ie for further QC'ing), but even if I did that would require going back and redoing the main analysis as well. The individuals used here should match the main analysis.

#20151227 CHECK_0 -- Prob: come up with way to easily plot variants along chromosomes/x-axis given a list of SNPs, consider creating a single R -q -e '...' line to do this?

#Just looking at some basic SNP quality gradients, etc...
~~~
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz | grep -v ^# | awk '{ print $6 }' | sort -rg | tail -n 10
50.1
50.09
50.08
50.05
50.05
50.03
50.02
50.01
50
50
~~~~

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.recode.vcf.gz | grep -v ^# | perl -lane 'BEGIN { my %indvCounts; } my $hetCounts = 0; for (my $i=9; $i <= $#F; $i++) { my $altCounts = 0; my @indvInfo1 = split(/\:/, $F[$i]); my @genoInfo1 = split(/\//, $indvInfo1[0]); if ($genoInfo1[0] eq "1") { $altCounts++; } if ($genoInfo1[1] eq "1") { $altCounts++; } if ($altCounts eq "1") { $hetCounts++; if (!$indvCounts{$i}) { $indvCounts{$i} = 1; } else { $indvCounts{$i}++; } } } END { foreach my $indv1 (keys %indvCounts) { print $indv1, "\t", $indvCounts{$indv1};} } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.perIndvhetCounts.txt 

~~~
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.perIndvhetCounts.txt | wc
    741    1482    4375
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.perIndvhetCounts.txt | head -n 10               559     6
127     6
32      8
206     2
118     7
443     2
71      5
331     1
358     1
560     7
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.perIndvhetCounts.txt | awk '{ print $2 }' | sort | uniq -c
     82 1
      7 10
      3 11
      2 13
      1 14
    129 2
      1 20
      1 28
    133 3
    152 4
     95 5
     60 6
     46 7
     15 8
     14 9
~~~

#So looks like only one or two individuals have a 'larger than normal # of het calls' but 28 or 20 out of say ~990 is still a low percentage. Doesn't seem to warrant considering how to not include one or two individuals at this stage and mesh it back up with the full autosomal analysis done earlier on.

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.perSNPhetCounts.txt | awk '{ if ($10 > 10) { print $1, "\t", $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.perSNPhetCounts.gt10Hets.ChrBPs 

~~~
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.perSNPhetCounts.gt10Hets.ChrBPs | wc
     19      38     251
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup]$R -q -e '1 + 1 + 3 + 1 + 2 + 3 + 2 + 1 + 1 + 1 + 1 + 1 + 1'
> 1 + 1 + 3 + 1 + 2 + 3 + 2 + 1 + 1 + 1 + 1 + 1 + 1
[1] 19
~~~

#This matches the # of SNPs with > 10 het calls expected as from above

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.recode.vcf.gz --exclude-positions /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.perSNPhetCounts.gt10Hets.ChrBPs --recode  --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het 

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.recode.vcf

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.recode.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.AllPhenos.frq.AndCount.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 30; $i <= 40; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.FreqInfo.AnnovarAnnotation.vs6.txt.gz

~~~
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.recode.vcf.gz | grep -v \# | perl -lane 'print join("\t", @F[0..9]);' | wc
    950    9500  322269
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.FreqInfo.AnnovarAnnotation.vs6.txt.gz | wc
    967    1065  420380
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/_2/) { print join(",", @F); } ' | wc
     17      17    7757
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/_3/) { print join(",", @F); } ' | wc
      0       0       0
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/_4/) { print join(",", @F); } ' | wc
      0       0       0
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/_5/) { print join(",", @F); } ' | wc
      0       0       0
~~~

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.recode.vcf.gz --plink --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het 

plink --file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs3.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.FreqInfo.AnnovarAnnotation.vs6.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.bim > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.bim.wGeneIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.bim.wGeneIDs | awk -F, '{ if ((($7 == "exonic") && (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV"))) || (($7 == "splicing") || ($7 == "exonic;splicing"))) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice 

~~~
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.bim.wGeneIDs | grep -E 'nonsynonymous|stop' | wc
     52      52    4905
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'print $#F;'
2
1
3
21
2
2
3
1
2
7
8
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'print $#F;' | paste -s -d+ -          2+1+3+21+2+2+3+1+2+7+8
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'print $#F;' | paste -s -d+ - | bc
52
~~~

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.recode.vcf.gz | awk '{ if ($1 == "23") { $1 = "X"} print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.recode.vcf.bgzip 

/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.recode.vcf.bgzip
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.recode.vcf.bgzip.gz

~~~
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup]$ls -lrt | grep indv80 | awk '{ print $9 }' | perl -lane 'my @vals1 = split(/indv80/, $F[0]); my $val1 = $vals1[0] . "indv80" . $vals1[1]; my $val2 = $vals1[0] . "indv80.vs3" . $vals1[1]; system("mv $val1 $val2");'
mv: accessing `AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.FreqInfo.AnnovarAnnotation.vs6.txt.gz': File name too long
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup]$mv AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.gt10Het.FreqInfo.AnnovarAnnotation.vs6.txt.gz AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.FreqInfo.AnnovarAnnotation.vs6.txt.gz
~~~

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.recode.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.recode.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/Permutations/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.justWhite.QCed.DropIBD.indv80.vs3.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/XChr
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/XChr

#20151230 NOTE -- So that I don't have to change my core code, I'm going to create /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous to run the analysis but then I'm going to move all files into /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/XChr once finished. If I need to zip/log/save files now in the ...XChr location I'll just mv * back to the ...TruePheno_XChr... location. Doing this because I don't think the ...TruePheno_XChr... organization is intuitive whereas ....Exonic.Nonsynonymous/XChr is. But I also don't want to actively edit or make a 'subfile' version of ...MultiRuns.MvAndProcessResults.vs3.sh (etc...) to suddenly begin maintaining as well

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno_XChr
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1/Exonic.Nonsynonymous

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs3.sh MM5 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs TruePheno_XChr Pheno1 Exonic.Nonsynonymous

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs3.sh MM5 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs FakePheno_XChr Pheno1 Exonic.Nonsynonymous

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | wc

#20160111

~~~
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | perl -lane 'print join("\t", @F[0..9]);' | tail -n 10
X       153713641       rs147530071     G       A       338.36  VQSRTrancheSNP90.00to99.00      AA=G;AC=5;AF=2.784e-03;AN=1796;BaseQRankSum=4.788;DB;DP=12508;Dels=0.00;FS=2.813;HRun=0;HaplotypeScore=2.5272;InbreedingCoeff=0.1407;MLEAC=2;MLEAF=1.114e-03;MQ=59.99;MQ0=0;MQRankSum=0.648;POSITIVE_TRAIN_SITE;QD=9.40;ReadPosRankSum=0.718;VQSLOD=3.02;culprit=HaplotypeScore    GT:AD:DP:GQ:PL  0/0:13,0:13:39:0,39,411
X       154456747       rs572013        A       G       346432  VQSRTrancheSNP99.00to99.90      AA=G;AC=1748;AF=0.969;AN=1804;BaseQRankSum=18.300;DB;DP=10832;Dels=0.00;FS=2.667;HRun=0;HaplotypeScore=1.6675;InbreedingCoeff=-0.0339;MLEAC=1797;MLEAF=0.996;MQ=59.99;MQ0=0;MQRankSum=0.253;POSITIVE_TRAIN_SITE;QD=31.98;ReadPosRankSum=6.371;VQSLOD=0.370;culprit=ReadPosRankSum  GT:AD:DP:GQ:PL  1/1:0,12:12:36:394,36,0
X       154456805       .       C       T       236.9   VQSRTrancheSNP99.00to99.90      AA=C;AC=5;AF=2.775e-03;AN=1802;BaseQRankSum=1.367;DP=10443;Dels=0.00;FS=0.000;HRun=0;HaplotypeScore=1.8156;InbreedingCoeff=0.1151;MLEAC=1;MLEAF=5.549e-04;MQ=60.00;MQ0=0;MQRankSum=0.672;QD=6.07;ReadPosRankSum=-1.012;VQSLOD=1.63;culprit=QD      GT:AD:DP:GQ:PL  0/0:11,0:11:33:0,33,337
X       154457155       .       A       C       78.2    VQSRTrancheSNP99.00to99.90      AA=A;AC=2;AF=1.125e-03;AN=1778;BaseQRankSum=1.931;DP=5390;Dels=0.00;FS=0.000;HRun=0;HaplotypeScore=0.5798;InbreedingCoeff=-0.0003;MLEAC=1;MLEAF=5.624e-04;MQ=60.00;MQ0=0;MQRankSum=-1.161;QD=19.55;ReadPosRankSum=0.030;VQSLOD=0.871;culprit=DP    GT:AD:DP:GQ:PL  0/0:8,0:8:24:0,24,265
X       154464412       .       C       T       140.72  VQSRTrancheSNP99.00to99.90      AA=C;AC=8;AF=4.479e-03;AN=1786;BaseQRankSum=1.820;DP=6729;Dels=0.00;FS=2.016;HRun=1;HaplotypeScore=0.6796;InbreedingCoeff=0.0184;MLEAC=1;MLEAF=5.599e-04;MQ=60.00;MQ0=0;MQRankSum=-0.335;QD=2.47;ReadPosRankSum=1.332;VQSLOD=1.20;culprit=QD       GT:AD:DP:GQ:PL  0/0:2,0:2:6:0,6,64
X       154464466       .       C       T       471.87  VQSRTrancheSNP99.00to99.90      AA=C;AC=7;AF=3.911e-03;AN=1790;BaseQRankSum=2.203;DP=7087;Dels=0.00;FS=0.000;HRun=1;HaplotypeScore=0.6094;InbreedingCoeff=0.0261;MLEAC=3;MLEAF=1.676e-03;MQ=59.99;MQ0=0;MQRankSum=2.281;QD=10.04;ReadPosRankSum=-1.245;VQSLOD=0.733;culprit=DP     GT:AD:DP:GQ:PL  0/0:3,0:3:9:0,9,98
X       154467046       .       C       G       865.03  VQSRTrancheSNP99.00to99.90      AA=C;AC=2;AF=1.139e-03;AN=1756;BaseQRankSum=0.181;DP=6067;Dels=0.00;FS=0.000;HRun=0;HaplotypeScore=0.4960;InbreedingCoeff=0.0050;MLEAC=2;MLEAF=1.139e-03;MQ=60.00;MQ0=0;MQRankSum=-0.055;QD=30.89;ReadPosRankSum=1.072;VQSLOD=0.728;culprit=DP     GT:AD:DP:GQ:PL  0/0:4,0:4:12:0,12,131
X       154467426       .       G       A       98.14   VQSRTrancheSNP90.00to99.00      AA=G;AC=3;AF=1.670e-03;AN=1796;BaseQRankSum=3.213;DP=9762;Dels=0.00;FS=0.000;HRun=0;HaplotypeScore=1.6544;InbreedingCoeff=0.0787;MLEAC=1;MLEAF=5.568e-04;MQ=60.00;MQ0=0;MQRankSum=-0.162;QD=8.18;ReadPosRankSum=0.662;VQSLOD=2.09;culprit=DP       GT:AD:DP:GQ:PL  0/0:9,0:9:27:0,27,275
X       154467678       .       A       G       172.05  VQSRTrancheSNP99.00to99.90      AA=A;AC=4;AF=2.217e-03;AN=1804;BaseQRankSum=-1.112;DP=11079;Dels=0.00;FS=2.805;HRun=0;HaplotypeScore=1.2644;InbreedingCoeff=0.1093;MLEAC=1;MLEAF=5.543e-04;MQ=59.99;MQ0=0;MQRankSum=0.784;QD=9.06;ReadPosRankSum=2.162;VQSLOD=1.42;culprit=HaplotypeScore  GT:AD:DP:GQ:PL  0/0:15,0:15:45:0,45,513
X       154468046       rs143206182     T       C       373.31  VQSRTrancheSNP90.00to99.00      AA=T;AC=3;AF=1.667e-03;AN=1800;BaseQRankSum=2.275;DB;DP=11469;Dels=0.00;FS=3.170;HRun=0;HaplotypeScore=1.9134;InbreedingCoeff=0.1243;MLEAC=2;MLEAF=1.111e-03;MQ=59.96;MQ0=0;MQRankSum=0.286;POSITIVE_TRAIN_SITE;QD=20.74;ReadPosRankSum=0.459;VQSLOD=2.45;culprit=HaplotypeScore   GT:AD:DP:GQ:PL  0/0:18,0:18:54:0,54,615
~~~

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^# | perl -lane 'print $F[0], "\t", $F[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.ChrBP.txt





#20160216 CHECK_0 -- Problem: redo vcf concat by doing auto only from original main file and including all XChr from redone X vcf version








zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.recode.vcf.gz | grep -v ^# | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.ChrBP.txt | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.recode.NonOverlapWithMain.vcf.gz

~~~
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.recode.vcf.gz | grep -v ^# | wc
    950  744800 16941991
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.recode.NonOverlapWithMain.vcf.gz | wc
    361  283024 6434275
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.recode.vcf.gz | grep -v ^# | awk '{ print $1, "\t", $2 }' | sort | uniq -c | head -n 10
      1 1        101361178
      1 1        101361385
      1 1        101361609
      1 1        101361764
      1 1        101361784
      1 1        101361808
      1 1        101361856
      1 1        101361940
      1 1        101361964
      1 1        101362323
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.recode.vcf.gz | grep -v ^# | awk '{ print $1, "\t", $2 }' | sort | uniq -c | awk '{ if ($1 > 1) { print $0 } } ' | wc
    589    1767   12459
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.recode.NonOverlapWithMain.vcf.gz | grep -v ^# | awk '{ print $1, "\t", $2 }' | sort | uniq -c | awk '{ if ($1 > 1) { print $0 } } ' | wc
      0       0       0
[  michaelt@login2  ~]$R -q -e '361 + 589'
> 361 + 589
[1] 950
>
>
~~~

#zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.recode.NonOverlapWithMain.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wProperXChr.vcf.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.recode.NonOverlapWithMain.vcf.gz | /home/michaelt/Software/vcftools_0.1.11/bin/vcf-sort | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wProperXChr.vcf.gz

~~~
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wProperXChr.vcf.gz | wc
  64352 50353139 1182515820
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | wc
  63991 50070115 1176081545
[  michaelt@login2  ~]$R -q -e "64352 - 63991"
> 64352 - 63991
[1] 361
>
>
~~~

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'print $F[0], ",", $F[1];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.ChrBP.txt

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.FreqInfo.AnnovarAnnotation.vs6.txt.gz | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.ChrBP.txt | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.FreqInfo.AnnovarAnnotation.vs6.NonOverlapWithMain.txt.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.FreqInfo.AnnovarAnnotation.vs6.NonOverlapWithMain.txt.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.txt.gz

~~~
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.txt.gz  | wc
  65208   71864 29345920
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.txt.gz | wc
  65208   71864 29345920
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.FreqInfo.AnnovarAnnotation.vs6.NonOverlapWithMain.txt.gz | grep _1 | wc
    361     386  163369
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.FreqInfo.AnnovarAnnotation.vs6.NonOverlapWithMain.txt.gz | grep _2 | wc
      9       9    4200
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.FreqInfo.AnnovarAnnotation.vs6.NonOverlapWithMain.txt.gz | wc
    370     395  167569
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.txt.gz | wc
  65578   72259 29513489
[  michaelt@login2  ~]$R -q -e "65208 + 370"
> 65208 + 370
[1] 65578
>
>
~~~

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ print $1 }' | grep -v ^NS$ > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.NonOverlapWithProperXChr

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.NonOverlapWithProperXChr <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w -v NS) > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | tail -n +2 | awk '{ print $1, "\t", $2, "\t", $4 }' | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); print(cbind(Data1, signif(qvalue(Data1[,2])\$qvalues, 5)));" | head -n 10

~~~
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs | wc
    11      11      65
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | wc
    474   12319   66986
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs | wc
     10     260    1393
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs | wc
    464   12059   65593
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | wc
    475   12334   66998
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | tail -n +2 | awk '{ print $1, "\t", $2, "\t", $4 }' | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); print(cbind(Data1, signif(qvalue(Data1[,2])\$qvalues, 5)));" | head -n 10
> library(qvalue); Data1 <- read.table(file('stdin'), header=F); print(cbind(Data1, signif(qvalue(Data1[,2])$qvalues, 5)));
                 V1           V2      V3 signif(qvalue(Data1[, 2])$qvalues, 5)
		 1             PGAP2 0.7536400000 0.73967                               0.73982
		 2             HSPA6 1.0000000000 0.78244                               0.78278
		 3             AP1M1 0.0953249600 0.52655                               0.52013
		 4             P4HA2 0.2212796000 0.57458                               0.58449
		 5             SGPL1 0.1711647000 0.54981                               0.56056
		 6              SDF4 0.9061500000 0.74690                               0.74549
		 7              SDF2 0.1754747000 0.54981                               0.56056
		 8             ALMS1 0.6064754000 0.73967                               0.73982
~~~

join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | tail -n +2 | sort ) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | tail -n +2 | awk '{ print $1, "\t", $2, "\t", $4 }' | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); print(cbind(Data1, signif(qvalue(Data1[,2])\$qvalues, 5)));" | awk '{ print $2, "\t", $5 }' | sort -k 1,1 ) | perl -lane 'splice(@F, 3, 1, $F[$#F]); splice(@F, $#F, 1); print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals

~~~
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | wc
    474   12313   62277
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | wc
    475   12334   66998
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | tail -n +2 | wc
    474   12313   66774
[  michaelt@login2  ~]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | tail -n +2 | awk '{ print $1, "\t", $4 }' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ print $1, "\t", $4 }') | wc
    474    1422   10583
[  michaelt@login2  ~]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | tail -n +2 | awk '{ print $1, "\t", $4 }' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ print $1, "\t", $4 }') | head -n 10
ABCB10 0.45134 0.44520
ABCD3 0.73967 0.73982
ABHD1 0.45134 0.44520
ACBD3 0.73967 0.73982
ACBD5 0.45134 0.44520
ACOT8 0.78244 0.78278
ACSL3 0.73967 0.73982
AFF1 0.73967 0.73982
AFF4 0.67601 0.68438
AHCY 0.73967 0.73982
[  michaelt@login2  ~]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | tail -n +2 | awk '{ print $1, "\t", $4 }' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ print $1, "\t", $4 }') | perl -lane 'my $val1 = $F[2] - $F[1]; if (abs($val1) > .01) { print join("\t", @F), "\t", $val1; }'
BCAP31  0.63681 0.58449 -0.05232
CKMT1A  0.54981 0.56056 0.0107499999999999
DSC2    0.54981 0.56056 0.0107499999999999
FLAD1   0.54981 0.56056 0.0107499999999999
HADHB   0.54981 0.56056 0.0107499999999999
HUWE1   0.89554 0.73982 -0.15572
MAGED1  0.89554 0.74100 -0.15454
MAGT1   0.63681 0.44520 -0.19161
MCL1    0.57024 0.58130 0.0110600000000001
MSN     0.63681 0.58449 -0.05232
PDIA6   0.54981 0.56056 0.0107499999999999
PGRMC1  0.89554 0.73982 -0.15572
PREB    0.54981 0.56056 0.0107499999999999
PRKCSH  0.54981 0.56056 0.0107499999999999
RBBP4   0.54981 0.56056 0.0107499999999999
RBBP7   0.89554 0.74549 -0.15005
RBM10   0.89554 0.73982 -0.15572
RTN4    0.54981 0.56056 0.0107499999999999
SDF2    0.54981 0.56056 0.0107499999999999
SGPL1   0.54981 0.56056 0.0107499999999999
SLC27A4 0.54981 0.56056 0.0107499999999999
SSR4    0.63681 0.54482 -0.09199
TMPRSS9 0.54981 0.56056 0.0107499999999999
UBA1    0.89554 0.73982 -0.15572
USP11   0.89554 0.73982 -0.15572
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | tail -n +2 | awk '{ print $1, "\t", $4 }' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ print $1, "\t", $4 }') | perl -lane 'my $val1 = $F[2] - $F[1]; if (abs($val1) > .01) { print join("\t", @F), "\t", $val1; }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs
BCAP31  0.63681 0.58449 -0.05232
HUWE1   0.89554 0.73982 -0.15572
MAGED1  0.89554 0.74100 -0.15454
MAGT1   0.63681 0.44520 -0.19161
MSN     0.63681 0.58449 -0.05232
PGRMC1  0.89554 0.73982 -0.15572
RBBP7   0.89554 0.74549 -0.15005
RBM10   0.89554 0.73982 -0.15572
SSR4    0.63681 0.54482 -0.09199
UBA1    0.89554 0.73982 -0.15572
USP11   0.89554 0.73982 -0.15572
[  michaelt@login2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/SudhirSNPFlankingSequence]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr | tail -n +2 | awk '{ print $1, "\t", $4 }' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | awk '{ print $1, "\t", $4 }') | perl -lane 'my $val1 = $F[2] - $F[1]; if (abs($val1) > .01) { print join("\t", @F), "\t", $val1; }' | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs
CKMT1A  0.54981 0.56056 0.0107499999999999
DSC2    0.54981 0.56056 0.0107499999999999
FLAD1   0.54981 0.56056 0.0107499999999999
HADHB   0.54981 0.56056 0.0107499999999999
MCL1    0.57024 0.58130 0.0110600000000001
PDIA6   0.54981 0.56056 0.0107499999999999
PREB    0.54981 0.56056 0.0107499999999999
PRKCSH  0.54981 0.56056 0.0107499999999999
RBBP4   0.54981 0.56056 0.0107499999999999
RTN4    0.54981 0.56056 0.0107499999999999
SDF2    0.54981 0.56056 0.0107499999999999
SGPL1   0.54981 0.56056 0.0107499999999999
SLC27A4 0.54981 0.56056 0.0107499999999999
TMPRSS9 0.54981 0.56056 0.0107499999999999
~~~

cat <(echo "#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        PVAL    QVAL    REF     ALT     AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW") <(join <( python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.txt.gz | sort -k 1,1) <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wProperXChr.vcf.gz | grep -v ^\# | perl -lane 'print join("_", @F[0..1]), "\t", $F[3], "\t", $F[4];' | sort -k 1,1) | perl -lane '$F[1] = join("|", split(/[\;,\,]/, $F[1])); $F[2] = join("|", split(/[\;,\,]/, $F[2])); print join("\t", @F);' | perl -lane 'my @geneEntries = split(/\|/, $F[2]); my @pVals; my @qVals; foreach my $gene (@geneEntries) { $gene = ((split(/\(/, $gene))[0]); my $geneEntry = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | grep ^"$gene"[^-0-9A-Za-z_]`; if ($geneEntry) { my @geneVals1 = split(/\s+/, $geneEntry); push(@pVals, $geneVals1[1]); push(@qVals, $geneVals1[3]); } else { push(@pVals, "NA"); push(@qVals, "NA"); } } splice(@F, 5, 0, (join("|", @pVals), join("|", @qVals))); print join("\t", @F);' | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSeroNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSeroHighlyExposedNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSeroPos.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justRapidVRapid.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/FreqInfo/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.wHM3.justWhite.QCed.DropIBD.indv80.vs3.justSlowVSlow.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | perl -lane 'splice(@F, 0, 1, (((split(/_/, $F[0]))[0]), ((split(/_/, $F[0]))[1]), ((split(/_/, $F[0]))[1]))); print join("\t", @F);' |
perl -ane 'print join("\t", @F[0..10]); my $alt = $F[10]; my $totAC = join("", ((split(/:/, $F[11]))[1])) + join("", ((split(/:/, $F[12]))[1])); print "\t", $totAC; for (my $i=11; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", ((split(/\:/, $F[$i]))[1]); } } print "\n";' | sort -g -k 1,1 -k 2,2 ) | /home/shared/software/tabix-0.2.5/bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz
###perl -ane 'print join("\t", @F[0..10]); my $alt = $F[10]; my $totAC = join("", ((split(/:/, $F[11]))[1])) + join("", ((split(/:/, $F[12]))[1])); print "\t", $totAC; for (my $i=11; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", ((split(/\:/, $F[$i]))[1]); } } print "\n";' | sort -g -k 1,1 -k 2,2 ) | perl -lane 'splice(@F, 3, 0, ($F[9], $F[10])); splice(@F, 11, 2); print join("\t", @F);' | /home/shared/software/tabix-0.2.5/bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz

tabix -s 1 -b 2 -e 3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wProperXChr.vcf.gz | vcf-annotate -a /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz -d /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.PrepFor_vcfannotate.vs2.desc.txt -c CHROM,FROM,TO,INFO/LOC,INFO/GENE,INFO/VARTYPE,INFO/AACHANGE,INFO/PVAL,INFO/QVAL,REF,ALT,INFO/AC_ALL,INFO/AC_TOTAL,INFO/AC_SERONEG,INFO/AC_HESERONEG,INFO/AC_SEROPOS,INFO/AC_RAPID,INFO/AC_SLOW | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.wProperXChr.vcfannotate.vs3.wACs.vcf.gz 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.AllPools.Vs3.annotated.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/OLD4.MM5.AllPools.Vs3.annotated.vcf.gz
ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.wProperXChr.vcfannotate.vs3.wACs.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.AllPools.Vs3.annotated.Update1.vcf.gz

#20160113 NOTE -- No differences between two files except for QVAL, which is the $F[8] column being removed via splice in the perl statements below. This then allows the cmp command to pass without any differences being found (whereas 'char 252, line 2' is quickly found as differing when the splice command isn't conducted, e.g. $F[8]/QVAL column remains intact)
~~~
[  michaelt@node4  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | grep -v ^X | wc
  63270 1138860 5494987
[  michaelt@node4  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | grep -v ^X | wc              63270 1138860 5494987
[  michaelt@node4  ~]$cmp <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | grep -v ^X | perl -lane 'splice(@F, 8, 1); print join("\t", @F);') <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | grep -v ^X | perl -lane 'splice(@F, 8, 1); print join("\t", @F);')
[  michaelt@node4  ~]$
[  michaelt@node4  ~]$cmp <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | grep -v ^X) <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | grep -v ^X)
/dev/fd/63 /dev/fd/62 differ: char 252, line 2
[  michaelt@node4  ~]$
~~~

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs | sort | uniq > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.GeneIDList

~~~
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.GeneIDList | wc
    475     475    3002
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs | wc
    474     474    2995
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | wc
     12      12      68
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs | wc
      1       1       7
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs
PGRMC1
~~~

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | awk '{ print $1 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice.justGeneIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | awk '{ print $1 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice.justGeneIDs

cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -v -w -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice.justGeneIDs) /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice

~~~
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | wc
    473    4205   61068
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | wc
     11      63     849
[  michaelt@node4  ~]$cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -v -w -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice.justGeneIDs) /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice.justGeneIDs | wc
    474    4166   60472
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -v -w -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice.justGeneIDs | wc
    463    4155   60407
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice.justGeneIDs | grep -w -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice.justGeneIDs
PGRMC1
~~~

for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.GeneIDList`
do
	echo $gene1;
	paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | perl -lane 'if ($F[0] =~ m/^'$gene1'$/) { print join("\t", @F); }' | awk '{ print $1, "\t", $2, "\t", $4, "\t", $5, "\t", $8, "\t", $9, "\t", $11 }' ) \ 
	<(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | \
	grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'if ($F[0] =~ m/^'$gene1'$/) { print join("\t", @F); }' | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }') | \
	awk '{ print $12, "\t", $13, "\t", $14, "\t", $15, "\t", $16, "\t", $17, "\t", $18 }' | \
	R -q -e "Data1 <- read.table(file('stdin'), header=F); print(paste(apply(Data1, 2, sum)), quote=FALSE);" | grep -v ^\> | perl -lane 'splice(@F, 0, 1); print join("\t", @F);') \
	>> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.Orig.txt

done

~~~
.
.
.
NMT1
NOL12
NOLC1
NPM3
NRN1L
NS
Error in read.table(file("stdin"), header = F) :
  no lines available in input
Execution halted
NUDC
NUP153
NUP98
OLA1
.
.
.
[  michaelt@node4  ~]$cmp <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt | sort -k 1,1 | grep -v ^NS | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs | perl -lane 'splice(@F, 2, 1); print join("\t", @F);') <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.Orig.txt | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs | perl -lane 'splice(@F, 2, 1); print join("\t", @F);')
[  michaelt@node4  ~]$
~~~

#20160114 NOTE -- Excluding (redone) qvalues and the 11 XChr genes, the new updated MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.Orig.txt file and the original are the same (re: cmp statement above)

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/OLD4.MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Orig.txt

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/OLD4.MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.Orig.txt | perl -lane 'splice(@F, 4, 3); print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt 

#Included the below header in file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt
#GENE   PVAL    QVAL    NUMVAR    AC_ALL        AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW

#QC Checks on file
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | wc
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.Orig.txt | perl -lane 'if (($F[3] == $F[4]) && ($F[3] == $F[5]) && ($F[4] == $F[5])) { print join("\t", @F); } '  | wc
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.Orig.txt | perl -lane 'if (($F[3] != $F[4]) || ($F[3] != $F[5]) || ($F[4] != $F[5])) { print join("\t", @F); } '
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[5] != $val2) { print join("\t", @F); }'
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[5] == $val2) { print join("\t", @F); }' | wc
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[7] > $F[6]) { print join("\t", @F); }'
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[7] <= $F[6]) { print join("\t", @F); }' | wc
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | perl -lane 'my $val2 = $F[9] + $F[10]; if ($F[8] < $val2) { print join("\t", @F); }'
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | perl -lane 'my $val2 = $F[9] + $F[10]; if ($F[8] >= $val2) { print join("\t", @F); }' | wc

~~
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | wc
    475    5225   24122
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.Orig.txt | perl -lane 'if (($F[3] == $F[4]) && ($F[3] == $F[5]) && ($F[4] == $F[5])) { print join("\t", @F); } '  | wc
    474    6636   29835
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.Orig.txt | perl -lane 'if (($F[3] != $F[4]) || ($F[3] != $F[5]) || ($F[4] != $F[5])) { print join("\t", @F); } '
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[5] != $val2) { print join("\t", @F); }'
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[5] == $val2) { print join("\t", @F); }' | wc
    475    5225   24077
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[7] > $F[6]) { print join("\t", @F); }'
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | perl -lane 'my $val2 = $F[6] + $F[8]; if ($F[7] <= $F[6]) { print join("\t", @F); }' | wc
    475    5225   24077
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | perl -lane 'my $val2 = $F[9] + $F[10]; if ($F[8] < $val2) { print join("\t", @F); }'
[  michaelt@node4  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | perl -lane 'my $val2 = $F[9] + $F[10]; if ($F[8] >= $val2) { print join("\t", @F); }' | wc
    475    5225   24077
~~

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/OLD4.MM5.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz 

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | wc

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | wc

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.AllPools.Vs3.annotated.Update1.vcf.gz | wc

#Difference between 141 and 142 due to '#CHROM...' header in /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz
~~~
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | wc
  64225 1156050 5574574
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | wc
  64225 1156050 5574574
[  michaelt@login2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.AllPools.Vs3.annotated.Update1.vcf.gz | wc
  64366 50353328 1192000729
[  michaelt@login2  ~]$R -q -e "64366 - 64225"
> 64366 - 64225
[1] 141
> 
> 
~~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/OLD4.MM5.AllPools.Vs3.annotated.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/OLD4.MM5.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/OLD4.MM5.skatO.nonsyn.qVals.ACs.GeneLevel.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/.

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.AllPools.Vs3.annotated.Update1.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/.

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1/Exonic.Nonsynonymous/* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/.

~~~
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | wc
    474    5214   36778
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -v ID) | wc
    464    5104   36021
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | wc
     12     132     944
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList
ID
BCAP31
PGRMC1
MAGED1
HUWE1
RBBP7
MSN
RBM10
SSR4
MAGT1
UBA1
USP11
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -v ID | wc
     11      11      65
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | wc
     12      12      68
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs | wc
     11      11      65
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep -v \#CHROM | wc
     11     121     843
[  michaelt@login2  ~]$cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -v ID)) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep -v \#CHROM) | wc
    475    5225   36864
[  michaelt@login2  ~]$cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -v ID)) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep -v \#CHROM) | head -n 10
#CHROM  BEGIN   END     MARKER_ID       NS      FRAC_WITH_RARE  NUM_ALL_VARS    NUM_PASS_VARS   NUM_SING_VARS   PVALUE  STATRHO
11      3819108 3819108 11:3819108-3819108_PGAP2        775     0.0012903       1       1       1       0.75364 NA
1       161494743       161496039       1:161494743-161496039_HSPA6     775     0.26452 10      10      2       1       1
19      16314427        16339672        19:16314427-16339672_AP1M1      775     0.0064516       3       3       1       0.095287        0
5       131530706       131546032       5:131530706-131546032_P4HA2     775     0.018065        6       6       0       0.22483 0.09
10      72604246        72637078        10:72604246-72637078_SGPL1      775     0.27097 11      11      8       0.16986 0.25
1       1163951 1164080 1:1163951-1164080_SDF4  775     0.003871        3       3       3       0.90615 0
17      26976071        26988885        17:26976071-26988885_SDF2       775     0.0077419       5       5       4       0.17536 0
2       73613108        73828538        2:73613108-73828538_ALMS1       775     0.73419 81      81      34      0.59389 1
16      2821525 2821589 16:2821525-2821589_TCEB2        775     0.74968 3       3       1       0.42265 1
[  michaelt@login2  ~]$cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -v ID)) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep -v \#CHROM) | awk '{ print $4 }' | sort | uniq -c | head -n 10
      1 10:103239146-103310604_BTRC
      1 10:103542667-103543141_NPM3
      1 10:103917282-103921973_NOLC1
      1 10:114207210-114575114_VTI1A
      1 10:120795717-120830555_EIF3A
      1 10:22830778-22898567_PIP4K2A
      1 10:27497191-27529313_ACBD5
      1 10:35320510-35324143_CUL2
      1 10:6143298-6148136_RBM17
      1 10:67862956-69281701_CTNNA3
[  michaelt@login2  ~]$cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -v ID)) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep -v \#CHROM) | awk '{ print $4 }' | sort | uniq -c | awk '{ if ($1 > 1) { print $0 } } '
[  michaelt@login2  ~]$
[  michaelt@login2  ~]$cmp <(cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -v ID)) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep -v \#CHROM) | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -v ID)) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -v ID))
[  michaelt@login2  ~]$
~~~

for i in {1..100}
do                                                                                                                                                                              cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run${i}.epacts | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -v ID)) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run${i}.epacts | grep -v \#CHROM | perl -lane 'push (@F, "XChr"); print join("\t", @F);') > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run${i}.epacts
done

~~~
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts  | grep XChr | wc
     11     132     898
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | tail -n 20
6       149699946       149700516       6:149699946-149700516_TAB2      775     0.003871        2       2       1       0.34731 0
15      91479631        91496789        15:91479631-91496789_UNC45A     775     0.08    17      17      9       0.1185  0
19      11210979        11240278        19:11210979-11240278_LDLR       775     0.11613 16      16      6       0.10328 0
18      55912699        56056404        18:55912699-56056404_NEDD4L     775     0.027097        6       6       4       0.62009 1
5       151175052       151179472       5:151175052-151179472_G3BP1     775     0.003871        3       3       3       0.50229 0
4       76570777        76582803        4:76570777-76582803_G3BP2       775     0.0064516       3       3       2       0.022371        0
5       223646  235329  5:223646-235329_SDHA    775     0.12645 8       8       4       0.57877 1
11      75277486        75282991        11:75277486-75282991_SERPINH1   775     0.018065        7       7       4       0.059552        0
11      111957665       111957665       11:111957665-111957665_SDHD     775     0.028387        1       1       0       0.0065964       NA
X       152969508       152989100       X:152969508-152989100_BCAP31    775     0.010323        2       2       0       0.22688 1       XChr
X       118377123       118377123       X:118377123-118377123_PGRMC1    775     0.003871        1       1       0       0.81056 NA      XChr
X       51637846        51639709        X:51637846-51639709_MAGED1      775     0.012903        3       3       0       0.85906 1       XChr
X       53562421        53675200        X:53562421-53675200_HUWE1       775     0.082581        21      21      3       0.74699 1       XChr
X       16864025        16881212        X:16864025-16881212_RBBP7       775     0.003871        2       2       1       0.89554 1       XChr
X       64957060        64958913        X:64957060-64958913_MSN 775     0.0077419       2       2       0       0.23119 1       XChr
X       47006844        47041666        X:47006844-47041666_RBM10       775     0.036129        3       3       0       0.55246 0       XChr
X       153063794       153063794       X:153063794-153063794_SSR4      775     0.0012903       1       1       1       0.13815 NA      XChr
X       77150892        77150921        X:77150892-77150921_MAGT1       775     0.0064516       2       2       0       0.066947        0       XChr
X       47058735        47073975        X:47058735-47073975_UBA1        775     0.12774 7       7       0       0.70646 0       XChr
X       47092327        47099731        X:47092327-47099731_USP11       775     0.037419        8       8       0       0.79691 0       XChr
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep  -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -v ID)
X       47006844        47040915        X:47006844-47040915_RBM10       775     0.003871        2       2       0       0.15236 0
X       53562421        53644327        X:53562421-53644327_HUWE1       775     0.056774        16      16      2       0.14173 1
X       77150921        77150921        X:77150921-77150921_MAGT1       775     0.003871        1       1       0       0.16112 NA
X       16864025        16881212        X:16864025-16881212_RBBP7       775     0.003871        2       2       1       0.89554 1
X       47058735        47073975        X:47058735-47073975_UBA1        775     0.020645        6       6       0       0.73687 0
X       51637846        51639644        X:51637846-51639644_MAGED1      775     0.0077419       2       2       0       1       1
X       152969508       152989100       X:152969508-152989100_BCAP31    775     0.010323        2       2       0       0.22935 1
X       64957060        64958913        X:64957060-64958913_MSN 775     0.0077419       2       2       0       0.22928 1
X       153063794       153063794       X:153063794-153063794_SSR4      775     0.0012903       1       1       1       0.13815 NA
X       47092327        47099731        X:47092327-47099731_USP11       775     0.030968        6       6       0       0.50941 0
~~~

for i in {1..1000}
do                                                                                                                                                                              cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm${i}.epacts | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList | grep -v ID)) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm${i}.epacts | grep -v \#CHROM | perl -lane 'push (@F, "XChr"); print join("\t", @F);') > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm${i}.epacts
done

~~~
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts | wc
    475    5236   36853
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts | grep XChr | wc
     11     132     898
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1/Exonic.Nonsynonymous/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts | tail -n 20
6       149699946       149700516       6:149699946-149700516_TAB2      775     0.003871        2       2       1       0.062423        0
15      91479631        91496789        15:91479631-91496789_UNC45A     775     0.08    17      17      9       0.6123  1
19      11210979        11240278        19:11210979-11240278_LDLR       775     0.11613 16      16      6       0.47599 0
18      55912699        56056404        18:55912699-56056404_NEDD4L     775     0.027097        6       6       4       0.81897 1
5       151175052       151179472       5:151175052-151179472_G3BP1     775     0.003871        3       3       3       0.13601 0
4       76570777        76582803        4:76570777-76582803_G3BP2       775     0.0064516       3       3       2       0.87989 1
5       223646  235329  5:223646-235329_SDHA    775     0.12645 8       8       4       0.42953 0
11      75277486        75282991        11:75277486-75282991_SERPINH1   775     0.018065        7       7       4       0.68599 1
11      111957665       111957665       11:111957665-111957665_SDHD     775     0.028387        1       1       0       0.70414 NA
X       152969508       152989100       X:152969508-152989100_BCAP31    775     0.010323        2       2       0       0.15107 0       XChr
X       118377123       118377123       X:118377123-118377123_PGRMC1    775     0.003871        1       1       0       0.1035  NA      XChr
X       51637846        51639709        X:51637846-51639709_MAGED1      775     0.012903        3       3       0       0.78337 0       XChr
X       53562421        53675200        X:53562421-53675200_HUWE1       775     0.082581        21      21      3       0.0033857       0       XChr
X       16864025        16881212        X:16864025-16881212_RBBP7       775     0.003871        2       2       1       0.19957 0       XChr
X       64957060        64958913        X:64957060-64958913_MSN 775     0.0077419       2       2       0       0.15633 0       XChr
X       47006844        47041666        X:47006844-47041666_RBM10       775     0.036129        3       3       0       0.01508 1       XChr
X       153063794       153063794       X:153063794-153063794_SSR4      775     0.0012903       1       1       1       0.49034 NA      XChr
X       77150892        77150921        X:77150892-77150921_MAGT1       775     0.0064516       2       2       0       0.86712 1       XChr
X       47058735        47073975        X:47058735-47073975_UBA1        775     0.12774 7       7       0       0.58944 0       XChr
X       47092327        47099731        X:47092327-47099731_USP11       775     0.037419        8       8       0       0.89741 0       XChr
~~~

cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList | grep -v ID)) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice) > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice

~~~
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | wc
    473    4205   61068
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList | grep -v ID) | wc
    463    4155   60407
[  michaelt@login2  ~]$cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList | grep -v ID)) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice) | wc
    474    4218   61256
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep  -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList | grep -v ID) | perl -lane 'print "$F[0]\t$#F";' | sort -k 1,1    BCAP31  2
HUWE1   16
MAGED1  2
MAGT1   1
MSN     2
RBBP7   2
RBM10   2
SSR4    1
UBA1    6
USP11   6
[  michaelt@login2  ~]$cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -v -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList | grep -v ID)) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice) | grep  -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1/Exonic.Nonsynonymous/20160118_Orig1/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts.GeneIDList | grep -v ID) | perl -lane 'print "$F[0]\t$#F";' | sort -k 1,1
BCAP31  2
HUWE1   21
MAGED1  3
MAGT1   2
MSN     2
PGRMC1  1
RBBP7   2
RBM10   3
SSR4    1
UBA1    7
USP11   8
~~~

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/SUMSTAT/ 
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/FakePheno_XChr/Pheno1/Exonic.Nonsynonymous/SUMSTAT/ 

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs3.sh MM5 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.EPACTS.RMID5PCs TruePheno_XChr Pheno1 Exonic.Nonsynonymous

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs3.sh MM5 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/ AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.EPACTS.RMID5PCs FakePheno_XChr Pheno1 Exonic.Nonsynonymous

#20160121 NOTE -- The '.rnk' and 'SUMSTAT' files are failing because I am including an extra column, the 'XChr', in the appended genes to the original file. I can remove this or do something about this later if I need to redo the pathway analyses at this stage with the finalized data. At the moment I'm not going t
o change around /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs3.sh since I don't need these files. It may be more helpful to let the need drive how the changes should be made if they are eventually needed. 

#20160215 NOTE -- doing per qval threshold gene/numvar counting with updated XChr data now, also more fine-grain to help finalize follow-up genoptyping results

echo ".3" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .3) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .3) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".5" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".6" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".65" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .65) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .65) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".7" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".71" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .71) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .71) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".72" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .72) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .72) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".73" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .73) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .73) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".735" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .735) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .735) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".74" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .74) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .74) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".75" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .75) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .75) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".8" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".85" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".9" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .9) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .9) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo "1" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= 1) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= 1) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`

~~~
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir]$echo ".3" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .3) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .3) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.3 3 13
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir]$echo ".5" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.5 62 399
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir]$echo ".6" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.6 150 1284
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir]$echo ".65" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .65) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .65) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.65 172 1454
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir]$echo ".7" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.7 211 1797
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir]$echo ".71" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .71) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .71) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.71 215 1830
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir]$echo ".72" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .72) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .72) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.72 229 1909
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir]$echo ".73" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .73) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .73) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.73 231 1933
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir]$echo ".735" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .735) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .735) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.735 232 1938
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir]$echo ".74" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .74) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .74) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.74 427 3399
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir]$echo ".75" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .75) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .75) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.75 455 3585
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir]$echo ".8" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.8 474 3715
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir]$echo ".85" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.85 474 3715
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir]$echo ".9" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .9) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .9) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.9 474 3715
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir]$echo "1" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= 1) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.XChr.wHM3.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.indv80.vs3.gt10Het.AutoAndX.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno_XChr.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= 1) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
1 474 3715
~~~

#.3 3 13
#.5 62 399
#.6 150 1284
#.65 172 1454
#.7 211 1797
#.71 215 1830
#.72 229 1909
#.73 231 1933
#.735 232 1938
#.74 427 3399
#.75 455 3585
#.8 474 3715
#.85 474 3715
#.9 474 3715
#1 474 3715














#20160216 CHECK_0 -- Problem: redo vcf concat by doing auto only from original main file and including all XChr from redone X vcf version
#20151230 CHECK_0 -- Prob: Create a ...MvAndProcessResults.vs4.sh version of file that includes the X-chr analysis grep/replace action?
#20151230 CHECK_0 -- Prob: Create a ...Vs4... set of files with the additional X-chr SNPs in .vcf format. Don't think I should need to redo the main analysis, just need to grep/replace the X-chr genes into the results file. Keep a copy of the old one with '_Autosomal' or something like that? Do this for the '.Routput' as well as 'noNAs' set files too (so can do perm analyses as well)
#20160111 NOTE -- I currently do not feel I should actually create a 'vs4' of these files, as of yet at least. For the moment I will now be creating files that extend the results from 'vs3', adding a suffix such as 'XChr' to the pre-existing files. I will leave the original files from 'vs3' untouched which means if I automatically use them down-the-line, I will in fact not be using the updated, additional XChr information. I will have to remember to use the file versions with the suffix I choose. However, I think for the moment this is ultimately better since it's unclear how much extra work I'll be doing on these files versus moving towards follow-up genotyping when that data becomes available. If in fact I don't really end up redoing the latest round of these analyses, recreating a new naming and organization scheme would not be a great investment. If, however, I end up having to redo these analyses extensively, I'll consider revamping the organization process at that point to then include a 'Vs4' that more handily/stream-wised deals with the XChr version of results
#20160113 CHECK_0 -- Prob: Go back and create now a ...Vs4... set of files interjecting the 'hetChr' step somewhere, figure out where. As a complement to hwe1e4 makes sense -- include in filename too? Check to see if indvs dropped based on 80% missingness and IBD params still be same? If so then yeah. 
#20160113 CHECK_0 -- Prob: Do on P2 as well.
#20160113 CHECK_0 -- Prob: Put corrected wVarCounts results final in analysis? Replace/Edit .epacts files to include new XChr from XChr files? So if run 'results collection' scripts get correct things then? Should be able to create a bash loop scrip from 1..100 or 1..1000 that can switch out/in the proper XChr gene information from the runs in this section into the correct filenames present in the analysis files then. Also cp -p orig results file and then add in new file?

##Files for Sudhir == priorty
##.vcf, annotated gene level, annotated variant level
#
##Subsampling (MM5)
##ExAC/1000G (MM5/P2)
##Intron (MM5/P2)
##XChr (MM5/P2)
##Coverage (MM5/P2)

#20151228 CHECK_0 -- Prob: Do P2 for these too
#20151228 CHECK_0 -- Prob: Do Pheno2, Pheno3 & Pheno4 for these









##2015121228
##Matthew p-val vs. coverage analysis

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vcf.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.PerGene.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenes.GetVCFQCStats.PerGene.vs2.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenes.GetVCFQCStats.PerGene.vs2.output | awk -F, '{ print $1, "\t", $18, "\t", $2 }' | sort -gk 3,3 | vi -

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vcf.gz | grep -E 'nonsynonymous|splicing' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.PerGene.vs2.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenes.Exonic.NonSyn_Splicing.GetVCFQCStats.PerGene.vs2.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenes.Exonic.NonSyn_Splicing.GetVCFQCStats.PerGene.vs2.output | awk -F, '{ print $1, "\t", $18, "\t", $2 }' | sort -gk 3,3 | vi -

#join w/ gene pvals

join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenes.Exonic.NonSyn_Splicing.GetVCFQCStats.PerGene.vs2.output | awk -F, '{ print $1, "\t", $18, "\t", $2 }' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ print $1, "\t", $2 }' | sort) | head -n 10
#join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenes.Exonic.NonSyn_Splicing.GetVCFQCStats.PerGene.vs2.output | awk -F, '{ print $1, "\t", $18, "\t", $2 }' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ print $1, "\t", $2 }' | sort) > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MatthewPvalVsCoverage/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenes.Exonic.NonSyn_Splicing.PvalVsCoverage.txt

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MatthewPvalVsCoverage

join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenes.Exonic.NonSyn_Splicing.GetVCFQCStats.PerGene.vs2.output | awk -F, '{ print $1, "\t", $18, "\t", $2 }' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ print $1, "\t", $2 }' | sort) | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); cor(Data1[,2], -log10(Data1[,4])); summary(lm(-log10(Data1[,4])~Data1[,2])); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MatthewPvalVsCoverage/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenes.Exonic.NonSyn_Splicing.PvalVsCoverage.jpeg\", height=2000, width=4000, res=300); par(mfrow=c(1,2)); plot(Data1[,2], Data1[,4], main=\"MM5 -- per gene Coverage vs. p-Value\", xlab=\"Coverage\", ylab=\"p-Value\"); abline(lm(Data1[,4]~Data1[,2]), col=\"RED\"); plot(Data1[,2], -log10(Data1[,4]), main=\"MM5 -- per gene Coverage vs. -log10 p-Value\", xlab=\"Coverage\", ylab=\"-log10 p-Value\"); abline(lm(-log10(Data1[,4])~Data1[,2]), col=\"RED\"); dev.off();"

#On wolfy.uchicago.edu
#mkdir /Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MatthewPvalVsCoverage/

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MatthewPvalVsCoverage/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenes.Exonic.NonSyn_Splicing.PvalVsCoverage.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MatthewPvalVsCoverage/

#20151230 CHECK_0 -- Prob: Do this analysis on P2 as well

~~~
[  michaelt@node1  ~]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenes.Exonic.NonSyn_Splicing.GetVCFQCStats.PerGene.vs2.output | awk -F, '{ print $1, "\t", $18, "\t", $2 }' | sort) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ print $1, "\t", $2 }' | sort) | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); cor(Data1[,2], -log10(Data1[,4])); summary(lm(-log10(Data1[,4])~Data1[,2])); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MatthewPvalVsCoverage/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenes.Exonic.NonSyn_Splicing.PvalVsCoverage.jpeg\", height=2000, width=4000, res=300); par(mfrow=c(1,2)); plot(Data1[,2], Data1[,4], main=\"MM5 -- per gene Coverage vs. p-Value\", xlab=\"Coverage\", ylab=\"p-Value\"); abline(lm(Data1[,4]~Data1[,2]), col=\"RED\"); plot(Data1[,2], -log10(Data1[,4]), main=\"MM5 -- per gene Coverage vs. -log10 p-Value\", xlab=\"Coverage\", ylab=\"-log10 p-Value\"); abline(lm(-log10(Data1[,4])~Data1[,2]), col=\"RED\"); dev.off();"
> Data1 <- read.table(file('stdin'), header=F); cor(Data1[,2], -log10(Data1[,4])); summary(lm(-log10(Data1[,4])~Data1[,2])); jpeg("/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/MatthewPvalVsCoverage/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenes.Exonic.NonSyn_Splicing.PvalVsCoverage.jpeg", height=2000, width=4000, res=300); par(mfrow=c(1,2)); plot(Data1[,2], Data1[,4], main="MM5 -- per gene Coverage vs. p-Value", xlab="Coverage", ylab="p-Value"); abline(lm(Data1[,4]~Data1[,2]), col="RED"); plot(Data1[,2], -log10(Data1[,4]), main="MM5 -- per gene Coverage vs. -log10 p-Value", xlab="Coverage", ylab="-log10 p-Value"); abline(lm(-log10(Data1[,4])~Data1[,2]), col="RED"); dev.off();
[1] 0.03483261

Call:
lm(formula = -log10(Data1[, 4]) ~ Data1[, 2])

Residuals:
    Min      1Q  Median      3Q     Max
-0.5538 -0.3853 -0.1892  0.2224  2.6072

Coefficients:
            Estimate Std. Error t value Pr(>|t|)
(Intercept) 0.461007   0.074300   6.205 1.21e-09 ***
Data1[, 2]  0.002458   0.003257   0.755    0.451
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Residual standard error: 0.5216 on 469 degrees of freedom
Multiple R-squared:  0.001213,  Adjusted R-squared:  -0.0009163
F-statistic: 0.5697 on 1 and 469 DF,  p-value: 0.4507

null device
          1
>
>
~~~






##20160117
##Cleaning up some scripts to show to Aleksandar (as requested)

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForAleksandar

#/data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs2.sh
#/data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.vs4.sh
#/data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.vs2.sh
#/data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs2.sh
#cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForAleksandar/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh
#cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs3.pt2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForAleksandar/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs3.pt2.sh
#cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForAleksandar/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.vs2.sh
#cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs2.sh
#cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs3.sh

#Did not make separate cp -p copies of any of these files. Just copied them to the office desktop 'ForAleksandar' location and then e-mailed Aleksandar the files (along with some explanations). Also copy/pasted the for loop below as another example of coding-style types.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.vs4.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs3.pt2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs3.sh mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForAleksandar/.



for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno_XChr/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.GeneIDList`
do
        echo $gene1;
        paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.wProperXChr.wUpdtdQVals | perl -lane 'if ($F[0] =~ m/^'$gene1'$/) { print join("\t", @F); }' | awk '{ print $1, "\t", $2, "\t", $4, "\t", $5, "\t", $8, "\t", $9, "\t", $11 }' ) \
        <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.wProperXChr.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | \
        grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/XChrFollowup/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs3.wProperXChr.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'if ($F[0] =~ m/^'$gene1'$/) { print join("\t", @F); }' | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }') | \
        awk '{ print $12, "\t", $13, "\t", $14, "\t", $15, "\t", $16, "\t", $17, "\t", $18 }' | \
        R -q -e "Data1 <- read.table(file('stdin'), header=F); print(paste(apply(Data1, 2, sum)), quote=FALSE);" | grep -v ^\> | perl -lane 'splice(@F, 0, 1); print join("\t", @F);') \
        >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.Orig.txt

done







##20160205
##Looking at genotype array prediction scores for later phone call with Sudhir

#join <(cat MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | perl -lane 'print $F[$#F];' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.1), right=FALSE));"
join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | perl -lane 'print $F[$#F];' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.1), right=FALSE));"

genotypeScoreThresh=".9"
#join <(cat MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 > '$genotypeScoreThresh') { print $3 } }' | head -n 10
join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 > '$genotypeScoreThresh') { print $3 } }' | head -n 10

join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ print $3 }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $12 }' | awk '{ if ($4 != $5) { print $0 } } '

#cat MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt | grep \| | head -n 10 | perl -lane 'for (my $i = 0; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine = @F; $printLine[3] = (split(/\|/, $F[3]))[$i]; $printLine[4] = (split(/\|/, $F[4]))[$i]; $printLine[5] = (split(/\,/, $F[5]))[$i]; print join("\t", @printLine); }'
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | grep \| | head -n 10 | perl -lane 'for (my $i = 0; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine = @F; $printLine[3] = (split(/\|/, $F[3]))[$i]; $printLine[4] = (split(/\|/, $F[4]))[$i]; $printLine[5] = (split(/\,/, $F[5]))[$i]; print join("\t", @printLine); }'

join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ print $3 }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $12 }' | awk '{ if ($4 != $5) { print $0 } } '
join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ print $3 }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $12 }' | wc
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | wc
join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ print $3 }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $12 }' | wc

~~~
Michaels-MacBook-Air:PG mturchin20$ join <(cat MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | perl -lane 'print $F[$#F];' | R -q -e "Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.1), right=FALSE));"
> Data1 <- read.table(file('stdin'), header=F); table(cut(Data1[,1], seq(0,1,.1), right=FALSE));

  [0,0.1) [0.1,0.2) [0.2,0.3) [0.3,0.4) [0.4,0.5) [0.5,0.6) [0.6,0.7) [0.7,0.8) 
        0         6        33        55        69       167       244       430 
[0.8,0.9)   [0.9,1) 
      547      1179 
> 
> 
Michaels-MacBook-Air:PG mturchin20$ join <(cat MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(cat MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ print $3 }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $12 }' | awk '{ if ($4 != $5) { print $0 } } '
GIMAP1-GIMAP5 	 0.1942587000 	 0.58449 	 11 	 3
Michaels-MacBook-Air:PG mturchin20$ join <(cat MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ print $3 }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | grep GIMAP1
GIMAP1-GIMAP5 	 3
GIMAP1-GIMAP5|GIMAP5 	 8
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | grep \| | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | head -n 10
X       80184962        80184962        intergenic      BRWD3(dist=119729)      .       .,.     NA|NA   NA|NA   C       T       1528    61      36      15      25      7       1
X       80184962        80184962        intergenic      HMGN5(dist=184238)      .       .,.     NA|NA   NA|NA   C       T       1528    61      36      15      25      7       1
X       80185102        80185102        intergenic      BRWD3(dist=119869)      .       .,.     NA|NA   NA|NA   A       G       1540    99      55      22      44      8       2
X       80185102        80185102        intergenic      HMGN5(dist=184098)      .       .,.     NA|NA   NA|NA   A       G       1540    99      55      22      44      8       2
X       80185183        80185183        intergenic      BRWD3(dist=119950)      .       .,.     NA|NA   NA|NA   A       C       1540    6       2       0       4       2       2
X       80185183        80185183        intergenic      HMGN5(dist=184017)      .       .,.     NA|NA   NA|NA   A       C       1540    6       2       0       4       2       2
X       80185258        80185258        intergenic      BRWD3(dist=120025)      .       .,.     NA|NA   NA|NA   G       C       1544    2       0       0       2       0       0
X       80185258        80185258        intergenic      HMGN5(dist=183942)      .       .,.     NA|NA   NA|NA   G       C       1544    2       0       0       2       0       0
X       80185266        80185266        intergenic      BRWD3(dist=120033)      .       .,.     NA|NA   NA|NA   G       T       1544    26      12      4       14      0       2
X       80185266        80185266        intergenic      HMGN5(dist=183934)      .       .,.     NA|NA   NA|NA   G       T       1544    26      12      4       14      0       2
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | grep exonic | grep \| | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | head -n 10
5       139781734       139781734       exonic  ANKHD1  nonsynonymous_SNV       NM_001197030:c.G182A:p.G61D,NM_001197030:c.G182A:p.G61D 0.1394952000|0.1394595000       0.54482|0.54482 G       A       1524    4       2       1       2       0       0
5       139781734       139781734       exonic  ANKHD1-EIF4EBP3 nonsynonymous_SNV       NM_001197030:c.G182A:p.G61D,NM_001197030:c.G182A:p.G61D 0.1394952000|0.1394595000       0.54482|0.54482 G       A       1524    4       2       1       2       0       0
5       139815746       139815746       exonic  ANKHD1  nonsynonymous_SNV       NM_001197030:c.T364G:p.S122A,NM_001197030:c.T364G:p.S122A       0.1394952000|0.1394595000       0.54482|0.54482 T       G       1546    1       0       0       1       0       0
5       139815746       139815746       exonic  ANKHD1-EIF4EBP3 nonsynonymous_SNV       NM_001197030:c.T364G:p.S122A,NM_001197030:c.T364G:p.S122A       0.1394952000|0.1394595000       0.54482|0.54482 T       G       1546    1       0       0       1       0       0
5       139815809       139815809       exonic  ANKHD1  nonsynonymous_SNV       NM_001197030:c.C427G:p.Q143E,NM_001197030:c.C427G:p.Q143E       0.1394952000|0.1394595000       0.54482|0.54482 C       G       1546    7       4       2       3       2       1
5       139815809       139815809       exonic  ANKHD1-EIF4EBP3 nonsynonymous_SNV       NM_001197030:c.C427G:p.Q143E,NM_001197030:c.C427G:p.Q143E       0.1394952000|0.1394595000       0.54482|0.54482 C       G       1546    7       4       2       3       2       1
5       139818103       139818103       exonic  ANKHD1  nonsynonymous_SNV       NM_001197030:c.G518A:p.R173Q,NM_001197030:c.G518A:p.R173Q       0.1394952000|0.1394595000       0.54482|0.54482 G       A       1550    2       1       0       1       0       1
5       139818103       139818103       exonic  ANKHD1-EIF4EBP3 nonsynonymous_SNV       NM_001197030:c.G518A:p.R173Q,NM_001197030:c.G518A:p.R173Q       0.1394952000|0.1394595000       0.54482|0.54482 G       A       1550    2       1       0       1       0       1
5       139828886       139828886       exonic  ANKHD1  nonsynonymous_SNV       NM_001197030:c.G1238C:p.C413S,NM_001197030:c.G1238C:p.C413S     0.1394952000|0.1394595000       0.54482|0.54482 G       C       1548    1       0       0       1       1       0
5       139828886       139828886       exonic  ANKHD1-EIF4EBP3 nonsynonymous_SNV       NM_001197030:c.G1238C:p.C413S,NM_001197030:c.G1238C:p.C413S     0.1394952000|0.1394595000       0.54482|0.54482 G       C       1548    1       0       0       1       1       0
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ print $3 }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $12 }' | awk '{ if ($4 != $5) { print $0 } } '
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir]$
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ print $3 }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $12 }' | wc
    474    2370   18871
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | wc
    475    5225   24122
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir]$join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ print $3 }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $12 }' | wc
    469    2345   18652
~~~

join -a 1 -a 2 -e 0 -o 0 1.2 1.3 1.4 1.5 1.6 1.7 1.8 1.9 1.10 1.11 2.2 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 >= .9) { print $3 } }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $12, "\t", $4-$12 }' | sort -k 1,1 | \
join -a 1 -a 2 -e NA -o 0 1.2 1.3 1.4 1.5 1.6 2.2 2.3 - <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 >= .8) { print $3 } }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | sort -k 1,1 | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | head -n 10

join -a 1 -a 2 -e 0 -o 0 1.2 1.3 1.4 1.5 1.6 1.7 1.8 1.9 1.10 1.11 2.2 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 == 1) { print $3 } }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $12, "\t", $4-$12 }' | sort -k 1,1 | \
join -a 1 -a 2 -e NA -o 0 1.2 1.3 1.4 1.5 1.6 2.2 2.3 - <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 >= .9) { print $3 } }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | sort -k 1,1 | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | \
join -a 1 -a 2 -e NA -o 0 1.2 1.3 1.4 1.5 1.6 1.7 1.8 2.2 2.3 - <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 >= .8) { print $3 } }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | sort -k 1,1 | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | \
join -a 1 -a 2 -e NA -o 0 1.2 1.3 1.4 1.5 1.6 1.7 1.8 1.9 1.10 2.2 2.3 - <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 >= .7) { print $3 } }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | sort -k 1,1 | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | \
join -a 1 -a 2 -e NA -o 0 1.2 1.3 1.4 1.5 1.6 1.7 1.8 1.9 1.10 1.11 1.12 2.2 2.3 - <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 >= .6) { print $3 } }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | sort -k 1,1 | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | \
join -a 1 -a 2 -e NA -o 0 1.2 1.3 1.4 1.5 1.6 1.7 1.8 1.9 1.10 1.11 1.12 1.13 1.14 2.2 2.3 - <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 >= .5) { print $3 } }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | sort -k 1,1 | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | \
join -a 1 -a 2 -e NA -o 0 1.2 1.3 1.4 1.5 1.6 1.7 1.8 1.9 1.10 1.11 1.12 1.13 1.14 1.15 1.16 2.2 2.3 - <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 >= .4) { print $3 } }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | sort -k 1,1 | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | \
join -a 1 -a 2 -e NA -o 0 1.2 1.3 1.4 1.5 1.6 1.7 1.8 1.9 1.10 1.11 1.12 1.13 1.14 1.15 1.16 1.17 1.18 2.2 2.3 - <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 >= .3) { print $3 } }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | sort -k 1,1 | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | \
join -a 1 -a 2 -e NA -o 0 1.2 1.3 1.4 1.5 1.6 1.7 1.8 1.9 1.10 1.11 1.12 1.13 1.14 1.15 1.16 1.17 1.18 1.19 1.20 2.2 2.3 - <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 >= .2) { print $3 } }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | sort -k 1,1 | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | \
join -a 1 -a 2 -e NA -o 0 1.2 1.3 1.4 1.5 1.6 1.7 1.8 1.9 1.10 1.11 1.12 1.13 1.14 1.15 1.16 1.17 1.18 1.19 1.20 1.21 1.22 2.2 2.3 - <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 >= .1) { print $3 } }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | sort -k 1,1 | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | \
join -a 1 -a 2 -e NA -o 0 1.2 1.3 1.4 1.5 1.6 1.7 1.8 1.9 1.10 1.11 1.12 1.13 1.14 1.15 1.16 1.17 1.18 1.19 1.20 1.21 1.22 1.23 1.24 2.2 2.3 - <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ print $3 }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | sort -k 2,2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/WGGTPrelimScoreResults_MM5.GeneAnnotations.Update1.txt 
#join -a 1 -a 2 -e NA -o 0 1.2 1.3 1.4 1.5 1.6 1.7 1.8 1.9 1.10 1.11 1.12 1.13 1.14 1.15 1.16 1.17 1.18 1.19 1.20 1.21 1.22 1.23 1.24 2.2 2.3 - <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ print $3 }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | sort -k 2,2 | head -n 10 

#Deleted following lines from /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/WGGTPrelimScoreResults_MM5.GeneAnnotations.Update1.txt
#MIR548N 0       0       0       1       -1      0       0       0       0       0       0       0       0       0       0       0       0       0       0       0       0       0       0       0       0
#GENE   PVAL    QVAL    NUMVAR  0       0       0       NUMVAR  0       NUMVAR  0       NUMVAR  0       NUMVAR  0       NUMVAR  0       NUMVAR  0       NUMVAR  0       NUMVAR  0       NUMVAR  0       NUMVAR

#Added the following header to /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/WGGTPrelimScoreResults_MM5.GeneAnnotations.Update1.txt
#Gene pVal qVal Tot_numVar 1_numVar 1_diff .9_numVar .9_diff .8_numVar .8_diff .7_numVar .7_diff .6_numVar .6_diff .5_numVar .5_diff .4_numVar .4_diff .3_numVar .3_diff .2_numVar .2_diff .1_numVar .1_diff 0_numVar 0_diff

join <(cat MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(cat MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 >= .8) { print $3 } }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $12, "\t", $4-$12 }' | sort -k 2,2

R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); SummaryScoreThreshMissingSNPs <- c();
SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[1:25,6]), sum(Data1[1:25,8]), sum(Data1[1:25,10]), sum(Data1[1:25,12]), sum(Data1[1:25,14]), sum(Data1[1:25,16]), sum(Data1[1:25,18]), sum(Data1[1:25,20]), sum(Data1[1:25,22]), sum(Data1[1:25,24]), sum(Data1[1:25,26]), sum(Data1[1:25,25]))); 
SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[1:50,6]), sum(Data1[1:50,8]), sum(Data1[1:50,10]), sum(Data1[1:50,12]), sum(Data1[1:50,14]), sum(Data1[1:50,16]), sum(Data1[1:50,18]), sum(Data1[1:50,20]), sum(Data1[1:50,22]), sum(Data1[1:50,24]), sum(Data1[1:50,26]), sum(Data1[1:50,25]))); 
SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[1:100,6]), sum(Data1[1:100,8]), sum(Data1[1:100,10]), sum(Data1[1:100,12]), sum(Data1[1:100,14]), sum(Data1[1:100,16]), sum(Data1[1:100,18]), sum(Data1[1:100,20]), sum(Data1[1:100,22]), sum(Data1[1:100,24]), sum(Data1[1:100,26]), sum(Data1[1:100,25]))); 
SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[1:200,6]), sum(Data1[1:200,8]), sum(Data1[1:200,10]), sum(Data1[1:200,12]), sum(Data1[1:200,14]), sum(Data1[1:200,16]), sum(Data1[1:200,18]), sum(Data1[1:200,20]), sum(Data1[1:200,22]), sum(Data1[1:200,24]), sum(Data1[1:200,26]), sum(Data1[1:200,25]))); 
SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[,6]), sum(Data1[,8]), sum(Data1[,10]), sum(Data1[,12]), sum(Data1[,14]), sum(Data1[,16]), sum(Data1[,18]), sum(Data1[,20]), sum(Data1[,22]), sum(Data1[,24]), sum(Data1[,26]), sum(Data1[,25])));
colnames(SummaryScoreThreshMissingSNPs) <- c(1, .9, .8, .7, .6 , .5, .4, .3, .2, .1, 0, \"(NumVarCheck)\"); rownames(SummaryScoreThreshMissingSNPs) <- c(\"Top 25\", \"Top 50\", \"Top 100\", \"Top 200\", \"All\");
print(SummaryScoreThreshMissingSNPs);"

R -q -e "Data1 <- read.table(file('stdin'), header=FALSE); SummaryScoreThreshMissingSNPs <- c(); SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[1:25,6]), sum(Data1[1:25,8]), sum(Data1[1:25,10]), sum(Data1[1:25,12]), sum(Data1[1:25,14]), sum(Data1[1:25,16]), sum(Data1[1:25,18]), sum(Data1[1:25,20]), sum(Data1[1:25,22]), sum(Data1[1:25,24]), sum(Data1[1:25,26]), sum(Data1[1:25,25]))); SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[1:50,6]), sum(Data1[1:50,8]), sum(Data1[1:50,10]), sum(Data1[1:50,12]), sum(Data1[1:50,14]), sum(Data1[1:50,16]), sum(Data1[1:50,18]), sum(Data1[1:50,20]), sum(Data1[1:50,22]), sum(Data1[1:50,24]), sum(Data1[1:50,26]), sum(Data1[1:50,25]))); SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[1:100,6]), sum(Data1[1:100,8]), sum(Data1[1:100,10]), sum(Data1[1:100,12]), sum(Data1[1:100,14]), sum(Data1[1:100,16]), sum(Data1[1:100,18]), sum(Data1[1:100,20]), sum(Data1[1:100,22]), sum(Data1[1:100,24]), sum(Data1[1:100,26]), sum(Data1[1:100,25]))); SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[1:200,6]), sum(Data1[1:200,8]), sum(Data1[1:200,10]), sum(Data1[1:200,12]), sum(Data1[1:200,14]), sum(Data1[1:200,16]), sum(Data1[1:200,18]), sum(Data1[1:200,20]), sum(Data1[1:200,22]), sum(Data1[1:200,24]), sum(Data1[1:200,26]), sum(Data1[1:200,25]))); SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[,6]), sum(Data1[,8]), sum(Data1[,10]), sum(Data1[,12]), sum(Data1[,14]), sum(Data1[,16]), sum(Data1[,18]), sum(Data1[,20]), sum(Data1[,22]), sum(Data1[,24]), sum(Data1[,26]), sum(Data1[,25]))); colnames(SummaryScoreThreshMissingSNPs) <- c(1, .9, .8, .7, .6 , .5, .4, .3, .2, .1, 0, \"(NumVarCheck)\"); rownames(SummaryScoreThreshMissingSNPs) <- c(\"Top 25\", \"Top 50\", \"Top 100\", \"Top 200\", \"All\"); print(SummaryScoreThreshMissingSNPs);"


~~~
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir]$join -a 1 -a 2 -e 0 -o 0 1.2 1.3 1.4 1.5 1.6 1.7 1.8 1.9 1.10 1.11 2.2 <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 >= .9) { print $3 } }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, "\t", $2, "\t", $3, "\t", $4, "\t", $12, "\t", $4-$12 }' | sort -k 1,1 | \
> join -a 1 -a 2 -e NA -o 0 1.2 1.3 1.4 1.5 1.6 2.2 2.3 - <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ if ($9 >= .8) { print $3 } }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | sort -k 1,1 | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | head -n 10
#GENE   PVAL    QVAL    NUMVAR  0       0       0       NUMVAR
ABCB10  0.0705404100    0.44520 7       3       4       6       1
ABCD3   0.5214367000    0.73982 5       4       1       5       0
ABHD1   0.0650166000    0.44520 6       2       4       4       2
ACBD3   0.7921847000    0.73982 6       6       0       6       0
ACBD5   0.0590341400    0.44520 5       4       1       5       0
ACOT8   0.9979598000    0.78278 8       1       7       4       4
ACSL3   0.5363287000    0.73982 8       5       3       7       1
AFF1    0.7559117000    0.73982 18      6       12      13      5
AFF4    0.3670351000    0.68438 7       5       2       6       1
.
.
.
 "nonsynonymous|stop|splicing" | awk '{ if ($9 >= .1) { print $3 } }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | sort -k 1,1 | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | join -a 1 -a 2 -e NA -o 0 1.2 1.3 1.4 1.5 1.6 1.7 1.8 1.9 1.10 1.11 1.12 1.13 1.14 1.15 1.16 1.17 1.18 1.19 1.20 1.21 1.22 1.23 1.24 2.2 2.3 - <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ print $3 }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | sort -k 2,2 | head -n 10
MIR548N 0       0       0       1       -1      0       0       0       0       0       0       0       0       0       0       0       0       0       0       0       0       0       0       0       0
GEMIN2  0.0007637044    0.20881 4       0       4       2       2       4       0       4       0       4       0       4       0       4       0       4       0       4       0       4       0       4       0
VTI1A   0.0011255423    0.20881 4       2       2       3       1       3       1       4       0       4       0       4       0       4       0       4       0       4       0       4       0       4       0
VDAC3   0.0021478040    0.26564 5       0       5       0       5       5       0       5       0       5       0       5       0       5       0       5       0       5       0       5       0       5       0
SDHD    0.0061396500    0.31125 1       0       1       0       1       0       1       0       1       1       0       1       0       1       0       1       0       1       0       1       0       1       0
NUDC    0.0076396000    0.31125 4       1       3       1       3       2       2       4       0       4       0       4       0       4       0       4       0       4       0       4       0       4       0
CUL5    0.0081515040    0.31125 2       1       1       1       1       1       1       2       0       2       0       2       0       2       0       2       0       2       0       2       0       2       0
EDC4    0.0083883740    0.31125 7       0       7       1       6       3       4       6       1       7       0       7       0       7       0       7       0       7       0       7       0       7       0
FUT8    0.0086799650    0.31125 6       5       1       6       0       6       0       6       0       6       0       6       0       6       0       6       0       6       0       6       0       6       0
MRPS9   0.0088281470    0.31125 9       1       8       6       3       8       1       9       0       9       0       9       0       9       0       9       0       9       0       9       0       9       0
.
.
.
") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | join -a 1 -a 2 -e NA -o 0 1.2 1.3 1.4 1.5 1.6 1.7 1.8 1.9 1.10 1.11 1.12 1.13 1.14 1.15 1.16 1.17 1.18 1.19 1.20 1.21 1.22 1.23 1.24 2.2 2.3 - <(join <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ print $3 }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | sort -k 2,2 | grep -w NA
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir]$
.
.
ta/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.GeneLevel.Update1.txt | sort -k 1,1) <(join <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/MM5.skatO.nonsyn.qVals.ACs.VariantLevel.Update1.txt.gz | perl -lane 'for (my $i = 0 ; $i < scalar(split(/\|/, $F[3])); $i++) { my @printLine1 = @F; $printLine1[3] = (split(/\|/, $F[3]))[$i]; $printLine1[4] = (split(/\|/, $F[4]))[$i]; $printLine1[5] = (split(/,/, $F[5]))[$i]; print join("\t", @printLine1); }'  | perl -lane 'print "$F[0]_$F[1]\t", join("\t", @F);' | awk '{ print $1, "\t", $5, "\t", $6, "\t", $7, "\t", $9, "\t", $10, "\t", $13, "\t", $14 }' | sort -k 1,1) <(cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/WGGTPrelimScoreResults_MM5.csv | perl -F, -lane 'print $F[0], "\t", $F[9];' | sort -k 1,1 ) | grep -E "nonsynonymous|stop|splicing" | awk '{ print $3 }' | sort | uniq -c | awk '{ print $2, "\t", $1 }' | sort -k 1,1) | awk '{ print $1, $12, "\t", $4-$12 }' ) | perl -lane 'if ($F[$#F-1] eq "NA") { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | sort -k 2,2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/WGGTPrelimScoreResults_MM5.GeneAnnotat { $F[$#F-1] = 0; $F[$#F] = $F[3]; } print join("\t", @F);' | sort -k 2,2 | wc        
    476   12376   36296
.
.
> Data1 <- read.table(file('stdin'), header=FALSE); SummaryScoreThreshMissingSNPs <- c(); SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[1:25,6]), sum(Data1[1:25,8]), sum(Data1[1:25,10]), sum(Data1[1:25,12]), sum(Data1[1:25,14]), sum(Data1[1:25,16]), sum(Data1[1:25,18]), sum(Data1[1:25,20]), sum(Data1[1:25,22]), sum(Data1[1:25,24]), sum(Data1[1:25,26]), sum(Data1[1:25,25]))); SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[1:50,6]), sum(Data1[1:50,8]), sum(Data1[1:50,10]), sum(Data1[1:50,12]), sum(Data1[1:50,14]), sum(Data1[1:50,16]), sum(Data1[1:50,18]), sum(Data1[1:50,20]), sum(Data1[1:50,22]), sum(Data1[1:50,24]), sum(Data1[1:50,26]), sum(Data1[1:50,25]))); SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[1:100,6]), sum(Data1[1:100,8]), sum(Data1[1:100,10]), sum(Data1[1:100,12]), sum(Data1[1:100,14]), sum(Data1[1:100,16]), sum(Data1[1:100,18]), sum(Data1[1:100,20]), sum(Data1[1:100,22]), sum(Data1[1:100,24]), sum(Data1[1:100,26]), sum(Data1[1:100,25]))); SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[1:200,6]), sum(Data1[1:200,8]), sum(Data1[1:200,10]), sum(Data1[1:200,12]), sum(Data1[1:200,14]), sum(Data1[1:200,16]), sum(Data1[1:200,18]), sum(Data1[1:200,20]), sum(Data1[1:200,22]), sum(Data1[1:200,24]), sum(Data1[1:200,26]), sum(Data1[1:200,25]))); SummaryScoreThreshMissingSNPs <- rbind(SummaryScoreThreshMissingSNPs, c(sum(Data1[,6]), sum(Data1[,8]), sum(Data1[,10]), sum(Data1[,12]), sum(Data1[,14]), sum(Data1[,16]), sum(Data1[,18]), sum(Data1[,20]), sum(Data1[,22]), sum(Data1[,24]), sum(Data1[,26]), sum(Data1[,25]))); colnames(SummaryScoreThreshMissingSNPs) <- c(1, .9, .8, .7, .6 , .5, .4, .3, .2, .1, 0, "(NumVarCheck)"); rownames(SummaryScoreThreshMissingSNPs) <- c("Top 25", "Top 50", "Top 100", "Top 200", "All"); print(SummaryScoreThreshMissingSNPs);
           1  0.9  0.8 0.7 0.6 0.5 0.4 0.3 0.2 0.1 0 (NumVarCheck)
Top 25   107   52   25  11   4   3   2   1   0   0 0           163
Top 50   218  113   68  41  23  14  10   8   0   0 0           323
Top 100  554  342  227 138  79  34  20  11   1   0 0           738
Top 200 1233  715  455 257 148  68  37  16   3   0 0          1692
All     2751 1564 1012 577 326 164  95  37   6   0 0          3744
> 
> 
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir]$head -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/WGGTPrelimScoreResults_MM5.GeneAnnotations.Update1.txt
#Gene pVal qVal Tot_numVar 1_numVar 1_diff .9_numVar .9_diff .8_numVar .8_diff .7_numVar .7_diff .6_numVar .6_diff .5_numVar .5_diff .4_numVar .4_diff .3_numVar .3_diff .2_numVar .2_diff .1_numVar .1_diff 0_numVar 0_diff
GEMIN2  0.0007637044    0.20881 4       0       4       2       2       4       0       4       0       4       0       4       0       4       0       4       0       4       0       4       0       4       0
VTI1A   0.0011255423    0.20881 4       2       2       3       1       3       1       4       0       4       0       4       0       4       0       4       0       4       0       4       0       4       0
VDAC3   0.0021478040    0.26564 5       0       5       0       5       5       0       5       0       5       0       5       0       5       0       5       0       5       0       5       0       5       0
SDHD    0.0061396500    0.31125 1       0       1       0       1       0       1       0       1       1       0       1       0       1       0       1       0       1       0       1       0       1       0
NUDC    0.0076396000    0.31125 4       1       3       1       3       2       2       4       0       4       0       4       0       4       0       4       0       4       0       4       0       4       0
CUL5    0.0081515040    0.31125 2       1       1       1       1       1       1       2       0       2       0       2       0       2       0       2       0       2       0       2       0       2       0
EDC4    0.0083883740    0.31125 7       0       7       1       6       3       4       6       1       7       0       7       0       7       0       7       0       7       0       7       0       7       0
FUT8    0.0086799650    0.31125 6       5       1       6       0       6       0       6       0       6       0       6       0       6       0       6       0       6       0       6       0       6       0
MRPS9   0.0088281470    0.31125 9       1       8       6       3       8       1       9       0       9       0       9       0       9       0       9       0       9       0       9       0       9       0
~~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/WGGTPrelimScoreResults_MM5.GeneAnnotations.Update1.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/.
#On MacBook Air
#scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForSudhir/WGGTPrelimScoreResults_MM5.GeneAnnotations.Update1.txt /Users/mturchin20/Documents/Work/StephensLab/HIVStuff/PG/. 









##20150427
##Email from Kevin on 20140324 explaining the breakdown of "array gene groupings 03 31 2014.xlsx" -- logged here for consistency 

~~~
#From: Kevin Olivieri <kolivieri@sanfordburnham.org>
#Subj: "Gene groupings for HiNT data sets"

Hi Michael,

ItÃ¢ÂÂs kind of a long message here, but I wanted to try to tell you in as much detail as I could what and why I was sending you. If you want to just start trying out the data before reading all below, feel free. Well, here it goes:

I put together some sets of genes for you to try out in your analysis. I have asked for help from others, particularly to map the genes to all known pathways, but that is in progress. What I have done is grouped the genes by how they have fared in our HiNT related assays and in pathways related to innate immunity. On the first tab, you will see much of the same data I sent you last time with the mass spec determined HIV interactions and the results of our siRNA screen of that data set. After that are the genes that fell into significant complexes, determined by a network diffusion method. After that I listed three broad summary categories:  Immune Signaling, Abundance Change, and Virion Incorporation. The numbers in each row represent the number of studies the gene was found int. The tabs following have each of these broad summary categories broken down into their constituent data sets.

I chose these three categories for the following reasons. Immune signaling is an important category to consider because it may represent genes that help to counter viral replication or it may include genes that create an inflammatory condition in which HIV replicates more efficiently. Protein abundance change may reflect the degradation of cellular antiviral proteins by the virus. Lastly, I included a category that includes proteins that have been found in virions. These may be cellular proteins that are important for viral replication or are part of a system the virus needs to overcome to bud.

There are 7 immune signaling pathways represented and one list of genes induced by type 1 interferons. There are two extremes listed and several in-betweens. I mean that the NFkB pathways most likely lead to an inflammatory state that promotes viral replication. At the other extreme is Type 1 IFN signaling which leads to upregulation of antiviral genes. The other pathways represented have a mix of both inflammatory and antiviral components. These pathways I consider between the two extremes of antiviral or proviral signaling.

Protein abundance changes were estimated in 5 different assays. Vpu mediated degradation of Interferon stimulated genes was done to estimate how HIV may counter interferon-mediated restriction of  viral replication. The amount of ubiquitinated proteins underwent was estimated by mass spec. These proteins may include antiviral proteins that the virus degrades to overcome their negative influence on replication. Protein abundance was also measured after HIV infection in two different assays. One involved infection and the other involved induction from cells already harboring an HIV genome. When protein abundance is lost, that too may reflect the degradation of an antiviral protein by the virus. Lastly, there was a screen run to determine if the HIV protease directly degrades antiviral proteins within the cell.

Virion incorporation studies have been carried out by three different groups, each using different techniques. Each technique has its own strengths and weaknesses. Therefore, it was hard to rule out any one study. The proteins found in virions may be important for viral replication or are part of a system the virus needs to overcome to bud.

On all pages, I listed the HIV binding data and the results of our functional screen. I thought, as things went alone, perhaps you would be able to break down the above categories into functional or not, which just multiplies things by two. I guess that is not so bad. You may find it interesting to break down the gene groups, where large enough, by HIV interacting partner. There are 18, but I donÃ¢ÂÂt think there are many groups large enough for this analysis to cover each HIV  gene.

I know it is a lot of info, so be sure to let me know if you have any questions.

Best,

Kevin
------------------------------------------------------
Kevin Olivieri, PhD
Director HiNT Virology Core
Chanda Lab
Sanford-Burnham Medical Research Insitute
Building 10, Rm 2608
10901 N Torrey Pines Rd.
San Diego, CA 92037
Ph: (858)646-3100 x3574
~~~




##20150921
##Collection of backup statements and protocols to be used

#20150921 CHECK_0: Do this for PP2 as well
#20150921 CHECK_0: Put github commands up together
#20150921 CHECK_0: Aleksandar these are the directories to be backed up? /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Backup/michaelt/Data/ALL_MAPPING and all subdirectories of /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Backup/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/* ?
#20150921 CHECK_0: Consider whether there are other files other than scripts that should be backed up? Go through them 

##Github

#20150921
cd /data/userdata/pg/michaelt/Data/ALL_MAPPING
git add /data/userdata/pg/michaelt/Data/ALL_MAPPING/*sh
git add /data/userdata/pg/michaelt/Data/ALL_MAPPING/*R
git add /data/userdata/pg/michaelt/Data/ALL_MAPPING/*py
git add /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/*sh
git add /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/*R
git add /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/*py

git commit -m "20150921 -- updating"
git push origin master


##From /userdata/ to work desktop

#On wolfy.uchicago.edu
mkdir /Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/Backup 
mkdir /Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/Backup/michaelt
mkdir /Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/Backup/michaelt/Data
mkdir /Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/Backup/michaelt/Data/ALL_MAPPING
mkdir /Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/Backup/michaelt/Data/ALL_MAPPING/Pools
mkdir /Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/Backup/michaelt/Data/ALL_MAPPING/Pools/MM5
mkdir /Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/Backup/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge
mkdir /Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/Backup/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged
mkdir /Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/Backup/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/*sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/*R  /data/userdata/pg/michaelt/Data/ALL_MAPPING/*py mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/Backup/michaelt/Data/ALL_MAPPING/.

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/*sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/*R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/*py mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/Backup/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/.

##From /userdata/ to /home/

mkdir /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Backup 
mkdir /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Backup/michaelt
mkdir /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Backup/michaelt/Data
mkdir /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Backup/michaelt/Data/ALL_MAPPING
mkdir /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Backup/michaelt/Data/ALL_MAPPING/Pools
mkdir /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Backup/michaelt/Data/ALL_MAPPING/Pools/MM5
mkdir /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Backup/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge
mkdir /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Backup/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged
mkdir /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Backup/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/*sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/*R  /data/userdata/pg/michaelt/Data/ALL_MAPPING/*py /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Backup/michaelt/Data/ALL_MAPPING/. 

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/*sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/*R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/*py /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Backup/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/. 


##20150327
##collection of Cygwin64 scp -p commands being used

wd: /home/PortMCT2/TempStuff
scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts .
scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts .

cp -p DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.edited
cp -p DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.edited


wd: /home/PortMCT2/TempStuff/P2
scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts P2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts
scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts P2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts

cp -p P2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts P2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.edited
cp -p P2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts P2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.edited

wd: /home/PortMCT2/TempStuff
scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs.AFplots.jpeg .
scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.AFplots.jpeg .
scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.ACplots.jpeg .

scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs.AFplots.jpeg .
scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF3a3b.justAFs.AFplots.jpeg .
scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.AFplots.jpeg .
scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80_Exonic.ExAC_r0.3.output.VCF1.ACplots.jpeg .
\

scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/LollipopPlots/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top35.DCAF11.LollipopPlot.svg .
scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/LollipopPlots/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top35.OSBPL6.LollipopPlot.svg .
scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/Analyses/LollipopPlots/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top35.TPR.LollipopPlot.svg .

#20150427
scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/P2.ForKevin.SULT1C3.ACs.txt .


##20160112
##Collection of commands for cmder

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.vs5.sh mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForAleksandar/.
scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabMisc/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/ForPeople/ForAleksandar/*pt2.ProcessQCedfastqFiles* .


