#!/bin/sh

20140517 - 1st run
~~~~~~~~~~~

ls -lrt /data/userdata/kevin/P2_ALL_MAPPING/FINAL/. | awk '{ print $9 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/data_kevin_P2_ALL_MAPPING_FINAL_dirList.20140519.txt

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.MakeDir.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.MakeDir.vs3.ForP2.vs1.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.MakeDir.vs3.ForP2.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/data_kevin_P2_ALL_MAPPING_FINAL_dirList.20140519.txt P2

#date "+%F_%k%M%S" | sed 's/\-//g' 
#20140519_154626
ls -lrt /data/454/raw/2013/*/*node*/sff/. | grep P2 | sed 's/\://g' | sed 's/\.//g' > /data/userdata/pg/michaelt/Data/sffFiles/data_kevin_P2_ALL_MAPPING_FINAL_Pools_sff_OrigLocations.Unedited.vs3.20140519_154626.pt1.txt

ls -lrt /data/454/raw/2013/*/*node*/sff/. | grep P2 | sed 's/\://g' | perl -lane 'print "ls -lrt ", $F[0];' | bash - | grep sff | perl -lane 'if ($F[8] =~ m/(.*)(01)(\.sff)/) { print $F[8]; }' > /data/userdata/pg/michaelt/Data/sffFiles/data_kevin_P2_ALL_MAPPING_FINAL_Pools_sff_OrigLocations.Unedited.vs3.20140519_154626.pt2.txt

paste /data/userdata/pg/michaelt/Data/sffFiles/data_kevin_P2_ALL_MAPPING_FINAL_Pools_sff_OrigLocations.Unedited.vs3.20140519_154626.pt1.txt /data/userdata/pg/michaelt/Data/sffFiles/data_kevin_P2_ALL_MAPPING_FINAL_Pools_sff_OrigLocations.Unedited.vs3.20140519_154626.pt2.txt | perl -lane 'if ($F[1] =~ m/(.*)(01)(\.sff)/) { print $F[0], $1, $2, $3; print $F[0], $1, "02", $3; }' > /data/userdata/pg/michaelt/Data/sffFiles/data_kevin_P2_ALL_MAPPING_FINAL_Pools_sff_OrigLocations.Unedited.vs3.20140519_154626.ptAll.txt 

cp -p /data/userdata/pg/michaelt/Data/sffFiles/DataOrganization.MakeDir.vs2.sh /data/userdata/pg/michaelt/Data/sffFiles/DataOrganization.MakeDir.vs3.sh

bash /data/userdata/pg/michaelt/Data/sffFiles/DataOrganization.MakeDir.vs3.sh /data/userdata/pg/michaelt/Data/sffFiles/data_kevin_P2_ALL_MAPPING_FINAL_Pools_sff_OrigLocations.Unedited.vs3.20140519_154626.ptAll.txt

cat /data/userdata/pg/michaelt/Data/sffFiles/data_kevin_P2_ALL_MAPPING_FINAL_Pools_sff_OrigLocations.Unedited.vs3.20140519_154626.ptAll.txt | perl -F/ -lane 'if ($F[5] =~ m/(.*)(P2)(\D*)(\d+)(-|_)(.*)/) { my $poolVal1 = "POOL_P2_" . $4; print join("/", @F), "\t", $F[8];}' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CombinePoolNameswsffFileIDs.vs3.ForP2.vs1.20140519_171823.pt1.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/data_kevin_P2_ALL_MAPPING_FINAL_dirList.20140519.txt | perl -F- -ane 'print join("-", @F); system("cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CombinePoolNameswsffFileIDs.vs3.ForP2.vs1.20140519_171823.pt1.output | grep $F[0] ");' | perl -lane 'if (!$F[1]) { $usePool1 = $F[0]; } else { print $usePool1, "\t", join("\t", @F);}' | perl -lane 'if ($F[2] =~ m/(.*)(\.sff)/) { print $F[0], "\t", $1, "\t", $F[3];}' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CombinePoolNameswsffFileIDs.vs3.ForP2.vs1.20140519_171823.ptAll.output

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.ConnectPoolsffFileswithIndvMIDRLsffFiles.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.ConnectPoolsffFileswithIndvMIDRLsffFiles.vs3.ForP2.vs1.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.ConnectPoolsffFileswithIndvMIDRLsffFiles.vs3.ForP2.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/data_kevin_P2_ALL_MAPPING_FINAL_dirList.20140519.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CombinePoolNameswsffFileIDs.vs3.ForP2.vs1.20140519_171823.ptAll.output P2 #Creates  

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForP2.vs1.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForP2.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/data_kevin_P2_ALL_MAPPING_FINAL_dirList.20140519.txt P2 #Creates DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG.vs1.${date1}.output file

cp -p /data/userdata/pg/michaelt/Data/sffFiles/DataProcessing.BreakUpMIDs.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/sffFiles/DataProcessing.BreakUpMIDs.Launcher.vs3.sh

cp -p /data/userdata/pg/michaelt/Data/sffFiles/DataProcessing.BreakUpMIDs.vs1.sh /data/userdata/pg/michaelt/Data/sffFiles/DataProcessing.BreakUpMIDs.vs2.sh 

bash /data/userdata/pg/michaelt/Data/sffFiles/DataProcessing.BreakUpMIDs.Launcher.vs3.sh /data/userdata/pg/michaelt/Data/sffFiles/data_kevin_P2_ALL_MAPPING_FINAL_Pools_sff_OrigLocations.Unedited.vs3.20140519_154626.ptAll.txt

cat DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.20140519_181702.output | head -n 10 > DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.20140519_181702.output.top10

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.20140519_181702.output

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/OID41305_HG19_Match5_probes/OID41305_HG19_Match5_probe_coverage.bed | perl -lane 'if ($F[0] =~ m/chr(.*)/) { $F[0] = $1; } print join("\t", @F);' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/OID41305_HG19_Match5_probes/OID41305_HG19_Match5_probe_coverage.altrd.bed

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.vs5.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.vs5.ForP2.vs1.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.20140519_181702.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.20140519_181702.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output 

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CheckForPresenceOfCertainFile.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CheckForPresenceOfCertainFile.vs3.ForP2.vs1.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CheckForPresenceOfCertainFile.vs3.ForP2.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output QCed.fastq.gz

mv DataProcessing.Pipeline.Utility.CheckForPresenceOfCertainFile.vs3.output DataProcessing.Pipeline.Utility.CheckForPresenceOfCertainFile.vs3.output.orig1

cat DataProcessing.Pipeline.Utility.CheckForPresenceOfCertainFile.vs3.output.orig1 | awk '{ print $2, "\t", $3 }' > DataProcessing.Pipeline.Utility.CheckForPresenceOfCertainFile.vs3.output.orig1.forSubmission

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh DataProcessing.Pipeline.Utility.CheckForPresenceOfCertainFile.vs3.output.orig1.forSubmission

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataProcessing.Pipeline.Utility.CheckForPresenceOfCertainFile.vs3.output.orig1.forSubmission

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CheckForPresenceOfCertainFile.vs3.ForP2.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output QCed.preGATK.QCed.bam

bash DataProcessing.Pipeline.pt2.ProcessQCedfastqFiles.Launcher.vs2.sh DataProcessing.Pipeline.Utility.CheckForPresenceOfCertainFile.vs3.ForP2.vs1.20140521_101415.output

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.Utility.CreatebamMergeList.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.Utility.CreatebamMergeList.vs3.ForP2.vs1.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.Utility.CreatebamMergeList.vs3.ForP2.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output

#Got P2_master_pool_sheet_12-10-13.xls from Wolinsky lab wiki http://swlab.virology.northwestern.edu/groups/454_sequencing_group/
#Note -- starred out POOL 112 01RL because it has two entries and both are resequenced elsewhere

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/P2_master_pool_sheet_12-10-13.edited.csv | perl -F, -lane 'if ($F[4] =~ m/ool/) { $poolVal = $F[4]; } else { if ($poolVal) { $F[4] = $poolVal; } } print join(",", @F);' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/P2_master_pool_sheet_12-10-13.edited.PoolFilledOut.csv

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/P2_master_pool_sheet_12-10-13.edited.PoolFilledOut.csv | grep -v \* > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/P2_master_pool_sheet_12-10-13.edited.PoolFilledOut.DropIndv.csv

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.Utility.CheckForDupEntriesPOPSampleInfoFile.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.Utility.CheckForDupEntriesPOPSampleInfoFile.vs1.ForP2.vs1.py

#Remove carrots from file, not dropping individuals. Then creating a file which takes all the IDs from the second column in the POP_genomics file, hashes them, then prints out the list of multiple entries per ID plus their RMID. The extra perl stuff is formatting this because I print out multiple python arrays which have that funny [' ', ' ', ' '] formatting (joined by ;). I then grep out the multiple entries of the main 'CreatebamMergeList' file given the list of multiple entries for the single ID, add them together in a single line and reappend them to the original CreatebamMergeList file.

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.Utility.CheckForDupEntriesPOPSampleInfoFile.vs1.ForP2.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/P2_master_pool_sheet_12-10-13.edited.PoolFilledOut.DropIndv.csv | perl -F: -ane 'my @vals1 = split(/,/, $F[0]); my $id1 = $vals1[0]; print $id1, ":"; foreach my $entry1 (@F) { my @valsN = split(/,/, $entry1); my @MID = split(//, $valsN[1]); if ($#MID == "3") { print "0", $valsN[1], ","; } else { print $valsN[1], ","; } foreach my $entry2 (@valsN) { if ($entry2 =~ m/ool/) { my @POOL = split(/\s+/, $entry2); print uc($POOL[1]), "_P2_", $POOL[2]; } } print ":";} print "\n";' | sed "s/'//g" | sed 's/\s//g' | perl -F: -lane 'print join(":", @F[1..$#F]);' > /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/P2_master_pool_sheet_12-10-13.edited.PoolFilledOut.DropIndv.DupListPrepForMergeListUse.txt

scp -p P2_master_pool_sheet_12-10-13_Original.xlsx P2_master_pool_sheet_12-10-13.edited.xlsx P2_master_pool_sheet_12-10-13.edited.csv P2_master_pool_sheet_12-10-13.edited.PoolFilledOut.csv mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/.

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineMultPoolsForSameIndvbamMergeList.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineMultPoolsForSameIndvbamMergeList.vs2.ForP2.vs1.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineMultPoolsForSameIndvbamMergeList.vs2.ForP2.vs1.sh /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/P2_master_pool_sheet_12-10-13.edited.PoolFilledOut.DropIndv.DupListPrepForMergeListUse.txt DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output #Creates DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected*

#Line to check the new entries at the bottom contain only the individuals they should
cat DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected | tail -n 30 | perl -F\; -ane 'my $poolID; foreach my $entry1 (@F) { if ($entry1 =~ m/.*(POOL_P2.*)\|\D+.*.sff/) { if (!$poolID) { $poolID = $1; } if ($1 ne $poolID) { print $poolID, "\t"; $poolID = $1;} } } print $poolID, "\n";'

#Line to check that every entry in this file is unique so the right number of individuals are exepcted to be found in final .vcf file
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected | perl -F\; -lane 'if ($F[0] =~ m/.*(POOL_P2.*)\|\D+.*/) { print $1; } ' | sort | uniq -d

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs2.ForP2.vs1.sh

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.vs4.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.vs4.ForP2.vs1.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs2.ForP2.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs2.ForP2.Vs1.sh

cp -p DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.vs2.sh DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.vs2.ForP2.vs1.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs2.ForP2.Vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected

#Forgot that 112 01RL needs to be dropped so redoing these two POOLs that were originally both grouped with 112 01RL independently
cat DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected | grep -E 'POOL_P2_158-08|POOL_P2_157-01' > DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected.redos

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt3.MergeallSampleBamFiles.Launcher.vs2.ForP2.vs1.sh DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected.redos

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt4.ProcessSampleLvlbamFiles.Launcher.vs2.ForP2.Vs1.sh DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected.redos

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs2.ForP2.vs1.sh

#bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt5.MergebamFilesVariantCall.vs2.ForP2.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.pta.SplitBamIntoChromosomes.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1 AllPools.P2.Vs1.AllPoolsMerged

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/Samtools
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.pta.RunReduceReadsPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1 AllPools.P2.Vs1.AllPoolsMerged

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptb.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1 AllPools.P2.Vs1.AllPoolsMerged

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.Samtools.pta.CreateVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1 AllPools.P2.Vs1.AllPoolsMerged

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptc.MergeVCFsAcrossChromosomes.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptc.MergeVCFsAcrossChromosomes.vs2.ForP2.Vs1.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptc.MergeVCFsAcrossChromosomes.vs2.ForP2.Vs1.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.GATK.ptd.RecalAndApplyVCFfilesPerChromosome.Launcher.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK AllPools.P2.Vs1.AllPoolsMerged

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/*tranches.pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/.

gzip -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf &

gzip -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.raw.vcf &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.vcf.gz --remove-filtered VQSRTrancheSNP99.90to100.00 --remove-filtered LowQual --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.recode.vcf

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.re
code.vcf.gz | fill-aa -a /home/michaelt/Data/1000G/AncestralAlignments/human_ancestor_GRCh37_e59/human_ancestor_ | gzip -c > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.vcf.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.vcf.gz | perl -lane 'if ($F[0] =~ m/^#/) { print join("\t", @F); } else { my @alleles2 = split(/,/, $F[4]); if ($#alleles2 == 0) { print join("\t", @F); } }' | gzip >
 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.DropOff.VCFVariants.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/OID41305_HG19_Match5_probes/OID41305_HG19_Match5_probe_coverage.altrd.bed --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz --window 1000 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.vcf.gz

/home/michaelt/Software/vcftools_0.1.12a/bin/vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.vcf.gz --missing-indv --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.imiss | awk '{ if ($5 > .99) { print $1 } } ' | grep -v INDV > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.imiss.Above99Miss.justIIDs

######HERE######

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.vcf.gz --remove /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.imiss.Above99Miss.justIIDs --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99

gzip -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.recode.vcf

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.recode.vcf.gz --geno .95 --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95

gzip -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.recode.vcf

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.recode.vcf.gz --hwe .0001 --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.recode.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf

#Note -- takes about ~2-3 days to complete even with -nt 6? With -nt 32 it takes
/home/shared/software/java/jre1.7.0_09/bin/java -Xmx10g -jar /home/shared/software/GATK/2.5.2/GenomeAnalysisTK.jar -T VariantAnnotator -R /home/michaelt/Data/HumanGenome/GRCh37/Version4/human_g1k_v37.fasta -I /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/AllPools.P2.Vs1.AllPoolsMerged.bam -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.HRun.vcf -A HomopolymerRun --variant /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf -nt 32 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.HRun.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.HRun.vcf.idx /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.idx

gzip -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz | perl -lane 'my @vals1 = split(/;/, $F[7]); foreach my $SNPinfo (@vals1) { if ($SNPinfo =~ m/HRun/) { print $SNPinfo; } };' | sort | uniq -c

61165 HRun=0                  
24643 HRun=1
7165 HRun=2
2342 HRun=3
633 HRun=4
156 HRun=5
30 HRun=6
4 HRun=7
1 HRun=8
1 HRun=9

vcftools --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf --plink --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4

plink --file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4 --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4 --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --genome --out  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome | sort -rg -k 9,9 | vi -

POOL_P2_103_03RL  POOL_P2_103_03RL  POOL_P2_105_03RL  POOL_P2_105_03RL UN    NA  0.0000  0.0575  0.9425  0.9713  -1  0.998376  1.0000      NA
POOL_P2_101_01RL  POOL_P2_101_01RL  POOL_P2_104_01RL  POOL_P2_104_01RL UN    NA  0.0249  0.1971  0.7780  0.8765  -1  0.993254  1.0000 77.6667
POOL_P2_163_12RL  POOL_P2_163_12RL   POOL_P2_69_08RL   POOL_P2_69_08RL UN    NA  0.0442  0.2246  0.7312  0.8435  -1  0.991569  1.0000 67.0000
POOL_P2_140_10RL  POOL_P2_140_10RL   POOL_P2_87_11RL   POOL_P2_87_11RL UN    NA  0.0252  0.2551  0.7197  0.8473  -1  0.991603  1.0000 64.5714
POOL_P2_145_11RL  POOL_P2_145_11RL  POOL_P2_150_12RL  POOL_P2_150_12RL UN    NA  0.0455  0.2412  0.7133  0.8339  -1  0.991037  1.0000 67.5714
#Not considering the below two pairs to be 'identical', and the last two are provided just to show the dropoff
POOL_P2_164_05RL  POOL_P2_164_05RL   POOL_P2_86_04RL   POOL_P2_86_04RL UN    NA  0.0011  0.6718  0.3271  0.6630  -1  0.980962  1.0000      NA
POOL_P2_59_09RL   POOL_P2_59_09RL   POOL_P2_61_12RL   POOL_P2_61_12RL UN    NA  0.0112  0.6874  0.3014  0.6451  -1  0.980046  1.0000 164.6667
POOL_P2_155_08RL  POOL_P2_155_08RL   POOL_P2_66_03RL   POOL_P2_66_03RL UN    NA  0.7293  0.0000  0.2707  0.2707  -1  0.965164  0.1030  1.3684
POOL_P2_66_03RL   POOL_P2_66_03RL   POOL_P2_93_07RL   POOL_P2_93_07RL UN    NA  0.7300  0.0000  0.2700  0.2700  -1  0.964634  0.1792  1.5263

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | grep -E 'POOL_P2_103_03RL|POOL_P2_105_03RL'
POOL_P2_103_03RL        POOL_P2_103_03RL        2       2       -9      -9
POOL_P2_105_03RL        POOL_P2_105_03RL        2       2       -9      -9

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | grep -E 'POOL_P2_101_01RL|POOL_P2_104_01RL'
POOL_P2_101_01RL        POOL_P2_101_01RL        1       1       -9      -9
POOL_P2_104_01RL        POOL_P2_104_01RL        1       1       -9      -9

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | grep -E 'POOL_P2_163_12RL|POOL_P2_69_08RL'
POOL_P2_69_08RL POOL_P2_69_08RL 1       1       -9      -9
POOL_P2_163_12RL        POOL_P2_163_12RL        1       -9      -9      -9

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | grep -E 'POOL_P2_140_10RL|POOL_P2_87_11RL'
POOL_P2_87_11RL POOL_P2_87_11RL 1       -9      -9      -9
POOL_P2_140_10RL        POOL_P2_140_10RL        2       2       -9      -9

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | grep -E 'POOL_P2_145_11RL|POOL_P2_150_12RL'
POOL_P2_150_12RL        POOL_P2_150_12RL        2       2       1       -9
POOL_P2_145_11RL        POOL_P2_145_11RL        2       2       1       -9

perl -e 'print "POOL_P2_103_03RL\nPOOL_P2_105_03RL\nPOOL_P2_101_01RL\nPOOL_P2_104_01RL\nPOOL_P2_163_12RL\nPOOL_P2_69_08RL\nPOOL_P2_140_10RL\nPOOL_P2_87_11RL\nPOOL_P2_145_11RL\nPOOL_P2_150_12RL\n";' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.FIDs

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.FIDs --depth --missing --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs


cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.idepth

INDV    N_SITES MEAN_DEPTH
POOL_P2_101_01RL        96013   11.6469
POOL_P2_103_03RL        96129   18.5583
POOL_P2_104_01RL        94656   6.22495
POOL_P2_105_03RL        96130   18.1951
POOL_P2_140_10RL        93623   5.21629
POOL_P2_145_11RL        95675   9.08535
POOL_P2_150_12RL        94502   5.87342
POOL_P2_163_12RL        95352   7.56006
POOL_P2_69_08RL 95232   8.13951
POOL_P2_87_11RL 96087   13.8943


cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.imiss

INDV    N_DATA  N_GENOTYPES_FILTERED    N_MISS  F_MISS
POOL_P2_101_01RL        96140   0       127     0.00132099
POOL_P2_103_03RL        96140   0       11      0.000114416
POOL_P2_104_01RL        96140   0       1484    0.0154358
POOL_P2_105_03RL        96140   0       10      0.000104015
POOL_P2_140_10RL        96140   0       2517    0.0261806
POOL_P2_145_11RL        96140   0       465     0.0048367
POOL_P2_150_12RL        96140   0       1638    0.0170377
POOL_P2_163_12RL        96140   0       788     0.00819638
POOL_P2_69_08RL 96140   0       908     0.00944456
POOL_P2_87_11RL 96140   0       53      0.000551279

So choosing:
POOL_P2_105_03RL
POOL_P2_101_01RL
POOL_P2_145_11RL

perl -e 'print "POOL_P2_103_03RL\nPOOL_P2_104_01RL\nPOOL_P2_140_10RL\nPOOL_P2_150_12RL\nPOOL_P2_163_12RL\nPOOL_P2_69_08RL\nPOOL_P2_87_11RL\n";' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome | sort -rg -k 9,9 | head -n 10000 | awk '{ print $1, "\n", $3 }' | perl -lane 'chomp(@F); print join("\t", @F);' | sort | uniq -c | sort -g -k 1,1 | awk '{ if ($1 >= 110 ) { print $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ2Countgte110.FIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome | sort -rg -k 8,8 | head -n 10000 | awk '{ print $1, "\n", $3 }' | perl -lane 'chomp(@F); print join("\t", @F);' | sort | uniq -c | sort -g -k 1,1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.Z1CountList

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome | sort -rg -k 8,8 | head -n 10000 | awk '{ print $1, "\n", $3 }' | perl -lane 'chomp(@F); print join("\t", @F);' | sort | uniq -c | sort -g -k 1,1 | awk '{ if ($1 >= 100) { print $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.FIDs

#NOTE -- dropping these individuals because they are starkly greater than anyone else in the list. Individuals from the 'high z2' count list are not being dropped because they appear as just to be the tail of a distribution versus being such a group of outliers

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs

cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ2Countgte110.FIDs | perl -lane 'push(@F, "Z2"); print join("\t", @F);') <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.FIDs | perl -lane 'push(@F, "Z1"); print join("\t", @F);') > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.OddHighZ1OrZ2.FID_ZNums 

#EIGENSTRAT stuff here

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/VLookUp.Python.DataHapMap3Overlap.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.map --file2 /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.liftOver.hg18tohg19.map | awk '{ print $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.overlapHapMap3.rsIDs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4 --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.overlapHapMap3.rsIDs --recode --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.overlapHapMap3

plink --file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.overlapHapMap3 --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.overlapHapMap3

#Note -- there are a few SNPs that don't match up, about ~30? Just letting them drop since troubleshooting almost got most of them back but not all of them and yeah

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.overlapHapMap3.map | awk '{ print $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.overlapHapMap3.map.rsIDs

plink --bfile /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.overlapHapMap3.map.rsIDs --keep /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.CEUCHBYRI.Unrelated.FIDIIDs --recode --out /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Array2.Vs1

plink --file /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Array2.Vs1 --make-bed --out /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Array2.Vs1

paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.overlapHapMap3.bim /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Array2.Vs1.bim | awk '{ if (($5 == $11) || ($5 == $12) ) { print $0 } } ' | awk '{ print $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.overlapHapMap3.FinalListwDrops.rsIDs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.overlapHapMap3 --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.overlapHapMap3.FinalListwDrops.rsIDs --recode --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.overlapHapMap3.vs2

plink --bfile /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.overlapHapMap3.FinalListwDrops.rsIDs --keep /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.CEUCHBYRI.Unrelated.FIDIIDs --recode --out /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Array2.Vs1.vs2

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.overlapHapMap3.vs2.ped /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Array2.Vs1.vs2.ped > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.ped

plink --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.ped --map /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.overlapHapMap3.vs2.map --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.phenoColEdit.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.phenoColEdit.pedind

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.bim /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.pedsnp

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.95geno.hwe1e4.wHapMap3Data.CEUCHBYRI.parfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.parfile

/home/michaelt/Software/EIG5.0.1/bin/smartpca -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.parfile

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.py --file1 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/P2_master_pool_sheet_12-10-13.edited.PoolFilledOut.DropIndv.csv --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Sample_status_list_12-10-13_CT.edited.noMeta.csv > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs1.ForP2.vs1.py

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs1.ForP2.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateCovariateFile.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateCovariateFile.vs1.ForP2.vs1.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateCovariateFile.vs1.ForP2.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.covar

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.vs1.ForP2.vs1.py

#Troubleshooting
#cat EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca | grep POOL > EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.justArray2
#python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.vs1.ForP2.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.justArray2 > asdouasdoui

#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_100_06RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_101_08RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_101_11RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_112_01RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_112_08RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_119_02RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_149_10RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_154_02RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_154_12RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_67_10RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_71_03RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_72_06RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_73_09RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_87_09RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_87_12RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_88_10RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_89_11RL)

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.vs1.ForP2.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca | cat - <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca | grep -E 'POOL_P2_100_06RL|POOL_P2_101_08RL|POOL_P2_101_11RL|POOL_P2_112_01RL|POOL_P2_112_08RL|POOL_P2_119_02RL|POOL_P2_149_10RL|POOL_P2_154_02RL|POOL_P2_154_12RL|POOL_P2_67_10RL|POOL_P2_71_03RL|POOL_P2_72_06RL|POOL_P2_73_09RL|POOL_P2_87_09RL|POOL_P2_87_12RL|POOL_P2_88_10RL|POOL_P2_89_11RL' | perl -lane '$F[$#F] = "NoPhenotype"; print join("\t", @F); ') > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt1

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.HapMap3vs.py --file1 /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.txt --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt2

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.pt2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race

sed -i 's/American_Indian_or_Alaskan_Native/AIndian_or_Alaskan_Native/g' /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.1vs2.xtxt

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.3vs4.xtxt

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race -c 5:6 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.5vs6.xtxt

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.plot.*.pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/.

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race | awk '{ if ((($2 >= -.001) && ($2 <= .22)) && (($3 >= -.005) && ($3 <= .021))) { print $0 } } ' | grep White_non-Hispanic > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed -c 1:2 -p White_non-Hispanic -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.plot.1vs2.xtxt

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.plot.1vs2.pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/.

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed | perl -lane 'my @vals1 = split(/:/, $F[0]); print $vals1[0];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed | perl -lane 'my @vals1 = split(/:/, $F[0]); print $vals1[0], "\t", $vals1[0];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race | awk 'BEGIN{ while(getline < "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.OddHighZ1OrZ2.FID_ZNums") { FID = $1":"$1; FIDList[FID] = $2; }} { if (FIDList[$1]) { if (FIDList[$1] == "Z1") { $12 = "Z1High"; } else if (FIDList[$1] == "Z2") { $12 = "Z2High"; } else{ print "blah" }  } else { $12 = "NormalIBD"; } print $0; } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.IBDissuesTagged

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.IBDissuesTagged -c 1:2 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.IBDissuesTagged.plot.1vs2.xtxt
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt1.ConvrtsffFilestoFastq.Launcher.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.WorkingMainFile.Edited.output.top10

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed | awk 'BEGIN{ while(getline < "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.OddHighZ1OrZ2.FID_ZNums") { FID = $1":"$1; FIDList[FID] = $2; }} { if (FIDList[$1]) { if (FIDList[$1] == "Z1") { $12 = "Z1High"; } else if (FIDList[$1] == "Z2") { $12 = "Z2High"; } else{ print "blah" }  } else { $12 = "NormalIBD"; } print $0; } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 1:2 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.1vs2.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 3:4 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.3vs4.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 5:6 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.5vs6.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 7:8 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.7vs8.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 9:10 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.9vs10.xtxt

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/*pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/.

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.phenoColEdit.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.phenoColEdit.pedind

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.bim /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.pedsnp

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.parfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.parfile 

/home/michaelt/Software/EIG5.0.1/bin/smartpca -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.parfile

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.vs1.ForP2.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.Results.pca > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.Results.pca.altrd.race 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.Results.pca.altrd.race | awk 'BEGIN{ while(getline < "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.OddHighZ1OrZ2.FID_ZNums") { FID = $1":"$1; FIDList[FID] = $2; }} { if (FIDList[$1]) { if (FIDList[$1] == "Z1") { $12 = "Z1High"; } else if (FIDList[$1] == "Z2") { $12 = "Z2High"; } else{ print "blah" }  } else { $12 = "NormalIBD"; } print $0; } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.Results.pca.altrd.race.IBDissuesTagged

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.Results.pca.altrd.race.plot.1vs2.xtxt

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.Results.pca.altrd.race.plot.3vs4.xtxt

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.Results.pca.altrd.race.IBDissuesTagged -c 1:2 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.Results.pca.altrd.race.IBDissuesTagged.plot.1vs2.xtxt

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.Results.pca.altrd.race.IBDissuesTagged -c 3:4 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.Results.pca.altrd.race.IBDissuesTagged.plot.3vs4.xtxt

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/*pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/GATK/EigenstratFiles/.

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.covar > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.5PCs.covar
#Edit code in between runs
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.covar > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.pheno 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar | awk '{ print $1, "\t", $2 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar | awk '{ print $1 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDs

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDs --non-ref-ac 1 --recode --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.recode.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.gz --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.allSNPs

#plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD --freq --out 

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs & 

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05 &

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.10PCs &

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.10PCs.maf05 &

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 2 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno2 &

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 3 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno3 &

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 4 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno4 &

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.assoc.logistic Pheno1_justWhite_Logistic_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.assoc.logistic.QQplot.jpeg & 

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.assoc.logistic Pheno1_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.assoc.logistic.QQplot.jpeg & 

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.10PCs.assoc.logistic Pheno1_justWhite_Logistic_10PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.10PCs.assoc.logistic.QQplot.jpeg & 

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.10PCs.maf05.assoc.logistic Pheno1_justWhite_Logistic_maf05_10PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.10PCs.maf05.assoc.logistic.QQplot.jpeg & 

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno2.assoc.logistic Pheno2_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno2.assoc.logistic.QQplot.jpeg & 

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno3.assoc.logistic Pheno3_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno3.assoc.logistic.QQplot.jpeg & 

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno4.assoc.logistic Pheno4_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno4.assoc.logistic.QQplot.jpeg & 

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/GATK/PLINKfiles/. 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ print $1 }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ if ($3 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroNeg.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ if ($4 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ if ($3 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroPos.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ if ($5 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justRapidVRapid.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ if ($5 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSlowVSlow.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ if ($6 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justVeryRapid.FIDs
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ if ($6 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justVerySlow.FIDs

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroNeg.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroPos.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justRapidVRapid.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSlowVSlow.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justVeryRapid.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justVerySlow.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroPos.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justRapidVRapid.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSlowVSlow.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justVeryRapid.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justVerySlow.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow &

paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq.count | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.AllPhenos.frq.AndCount.gz

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq.count

#gunzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq.count

#Annovar stuff here now

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.gz | awk '{ print $1, "\t", $2, "\t", $2, "\t", $4, "\t", $5 }' | grep -v ^# > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat

perl /home/michaelt/Software/annovar/summarize_annovar.pl /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat /home/michaelt/Software/annovar/ --ver1000g 1000g2012apr --verdbsnp 129 --veresp 6500 --buildver hg19 --remove

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs5.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.AllPhenos.frq.AndCount.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 30; $i <= 40; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz | grep -v \# | wc
96140 88738882 2030932886

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | wc
98735  109719 44353043

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | perl -F, -lane 'if ($F[1] =~ m/\_2/) { print join("\t", @F); }' | wc
 2551   83674 1149243

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | perl -F, -lane 'if ($F[1] =~ m/\_3/) { print join("\t", @F); }' | wc
 41      41   18201

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | perl -F, -lane 'if ($F[1] =~ m/\_4/) { print join("\t", @F); }' | wc
 2       2     858

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz | perl -F, -lane 'if ($F[1] =~ m/\_5/) { print join("\t", @F); }' | wc
 1       1     419


R -q -e 'val1 = 2551 + 41 + 2 + 1; print (val1);'
> val1 = 2551 + 41 + 2 + 1; print (val1);
[1] 2595
> 
> 

R -q -e 'val1 = 2551 + 41 + 2 + 1; print (val1); val2 = 98735 - 2595; print (val2)'
> val1 = 2551 + 41 + 2 + 1; print (val1); val2 = 98735 - 2595; print (val2)
[1] 2595
[1] 96140
> 
> 

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs3.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs

mkdir SKATFiles

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing") || ($7 == "UTR5;UTR3")) { print $8, "\t", $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.All

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { print $8, "\t", $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.Exonic

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $8, "\t", $2 } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($12 == "P") || ($12 == "D") ) { print $8, "\t", $2 } } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_PD

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $8, "\t", $2 } } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_D



mkdir EPACTSFiles

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing") || ($7 == "UTR5;UTR3")) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.All

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $0 } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($12 == "P") || ($12 == "D") ) { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 == "exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($13 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D

##These are based on array1 -- don't expect them to work here
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing")) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All

#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic

#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $0 } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous

#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($12 == "P") || ($12 == "D") ) { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD

#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { ($12 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_D

#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.Sift_D
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs | awk -F, '{ if ($7 =="exonic") { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($13 == "D") { print $0 } } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.LRT_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.pheno | awk '{ print $1, "\t", $2, "\t0\t0\t0\t", $3, "\t", $4, "\t", $5, "\t", $6 } ' | sed 's/-9/NA/g' | perl -lane 'print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar | awk '{ print $4, "\t", $6, "\t", $7 "\t", $8, "\t", $9, "\t", $10, "\t", $11,  "\t", $12, "\t", $13, "\t", $14,  "\t", $15 }' | paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped.vs2

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped.vs2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped

#Add line #FAM_ID IND_ID FAT_ID MOT_ID SEX HIVPROG HIVPROGHE AIDS AIDSEXTR RMID PC1 PC2 PC3 PC4 PC5 PC6 PC7 PC8 PC9 PC10 to /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.gz | awk '{ if ($1 == "23") { $1 = "X"} print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip

/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip 
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz

#EPACTS analyses

#EPACTS Variables Running Through
#Pheno: HIVPROG HIVPROGHE AIDS AIDSEXTR
#Filetypes: All Exonic Exonic.Nonsynonymous Exonic.Nonsynonymous.PP2_PD Exonic.Nonsynonymous.PP2_D Exonic.Nonsynonymous.Sift_D Exonic.Nonsynonymous.LRT_D
#Tests b.collapse b.madsen b.wcnt  skat --skat-adjust skat --skat-o --skat-adjust
#MAF cutoffs: .05 1.0 (e.g. treat all variants the same)

#Ran .,$ s/\/home\/pg\//\/data\/userdata\/pg\//g in DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.pt1.sh and /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.pt2.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs 

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/ExtraFiles

#Directory /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/
ls -lrt | grep RMID5PCs | grep -v epacts$ | awk '{ print $9 }' | perl -lane 'system("mv $F[0] ExtraFiles/$F[0]");'

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.CollectResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.CollectResults.vs2.sh

#NOTE -- odd thing happening where I can copy and paste the commands from the file and they work, but if I run the script it doesn't work. Doesn't like the \ separating the lines
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.CollectResults.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/ 1

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.CollectResults.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.CollectResults.vs2.CpyPst.sh

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs Pheno1 HIV_Acquisition_justWhite_RMID5PCs_skat0 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.RMID5PCs.Everything.Pheno1.maf1.skat0.QQplot.jpeg

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs Pheno2 HIV_AcquisitionHE_justWhite_RMID5PCs_skat0 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.RMID5PCs.Everything.Pheno2.maf1.skat0.QQplot.jpeg

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs Pheno3 AIDS_Progress_justWhite_RMID5PCs_skat0 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.RMID5PCs.Everything.Pheno3.maf1.skat0.QQplot.jpeg

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.EPACTSruns.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs Pheno4 AIDS_ProgressExtr_justWhite_RMID5PCs_skat0 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.RMID5PCs.Everything.Pheno4.maf1.skat0.QQplot.jpeg

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.RMID5PCs.Everything.Pheno*QQplot.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/.

perl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs1.pl --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --seed 10000 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped 

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno1.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno1.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno1.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno1.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno1.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROG --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno1.maf1.skat0.perm1 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno2.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno2.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno2.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno2.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno2.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno HIVPROGHE --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno2.maf1.skat0.perm1 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno3.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno3.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno3.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno3.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno3.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDS --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno3.maf1.skat0.perm1 &

epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.All --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno4.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno4.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno4.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno4.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno4.maf1.skat0.perm1 &
epacts group --vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz --groupf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.ped --pheno AIDSEXTR --cov RMID --cov PC1 --cov PC2 --cov PC3 --cov PC4 --cov PC5 --max-maf 1.0 --test skat --skat-o --skat-adjust --run 6 --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno4.maf1.skat0.perm1 &

#install.packages("/home/michaelt/Software/qvalue_1.1.try2.tar.gz", repos=NULL, type="source")

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno1.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno2.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.EPACTS.RMID5PCs.Every.Pheno2.maf1.skat0.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno3.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.EPACTS.RMID5PCs.Every.Pheno3.maf1.skat0.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno4.maf1.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.EPACTS.RMID5PCs.Every.Pheno4.maf1.skat0.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno1.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno1.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno1.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno1.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno1.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno1.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.EPACTS.RMID5PCs.Every.Pheno1.maf1.skat0.perm1.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno2.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno2.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno2.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno2.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno2.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno2.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.EPACTS.RMID5PCs.Every.Pheno2.maf1.skat0.perm1.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno3.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno3.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno3.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno3.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno3.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno3.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.EPACTS.RMID5PCs.Every.Pheno3.maf1.skat0.perm1.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno4.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno4.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno4.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno4.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno4.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno4.maf1.skat0.perm1.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.EPACTS.RMID5PCs.Every.Pheno4.maf1.skat0.perm1.wQvals.Summary



cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno1.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno1.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno1.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno1.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno1.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno1.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.EPACTS.RMID5PCs.Every.Pheno1.maf05.skat0.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno2.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno2.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno2.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno2.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno2.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno2.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.EPACTS.RMID5PCs.Every.Pheno2.maf05.skat0.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno3.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno3.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno3.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno3.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno3.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno3.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.EPACTS.RMID5PCs.Every.Pheno3.maf05.skat0.wQvals.Summary

cat <( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno4.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "All"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno4.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Exonic"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Nonsynonymous"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno4.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_PD"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno4.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "PP2_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno4.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "Sift_D"); print join("\t", @F);} ') \
<( cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno4.maf05.skat0.epacts | sort -rg -k 10,10 | awk '{ if ($10 != "NA") { print $0 } } ' | head -n -1 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); splice(@F, 0, 0, "LRT_D"); print join("\t", @F);} ') \
> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Version1.EPACTS.RMID5PCs.Every.Pheno4.maf05.skat0.wQvals.Summary



mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses















#QC Stuff

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs2.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output.GetperRegionpostQCStats.vs2.output | perl -lane 'my @array1; push(@array1, $F[1]); foreach my $entry (@F) { if ($entry !~ m/[a-zA-Z]/) { push (@array1, $entry); } } print join(",", @array1);' | awk 'BEGIN{ print "ID,TotalReadsPreQC,TotalReadsQ10,TotalReadsQ10Prcnt,TotalReadsF4,TotalReadsF4Prcnt,TotalReadsF256,TotalReadsF256Prcnt,TotalReadsPostQC,TotalReadsDropped,TotalReadsDroppedPrcnt,AverageReadLengthPreQC,AverageReadLengthPostQC,BedFileCoveragePostQC" } { print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output.GetperRegionpostQCStats.vs2.output.reformatted

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs2.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected



bash DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.sh DataOrganization.CreatebamMergeList.vs3.20140328_174052.multPOOLperIndvCollected.dropIndvs.output

#Total column numbers post formatting = 27
cat DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.vs2.output | perl -lane 'my @array1; push(@array1, $F[1]); foreach my $entry (@F) { if ($entry !~ m/[a-zA-Z]/) { push (@array1, $entry); } } print join(",", @array1);' | awk 'BEGIN{ print "ID,NumberOfOriginalBamFiles,TotalReadsPreQC,TotalReadsPreQCForward,TotalReadsPreQCReverse,TotalReadsPreQCForward_Reverse,PreQCMAPQmean,PreQCMAPQsd,PreQCMean,PreQCSD,PreQCMax,PreQCMin,CoveragePreQC,TotalReadsPostRmdup,TotalReadsPostRmdupDropped,TotalReadsPostRmdupDroppedPercentage,TotalReadsPostQC,TotalReadsPostQCForward,TotalReadsPostQCReverse,TotalReadsPostQCForward_Reverse,PostQCMAPQmean,PostQCMAPQsd,PostQCMean,PostQCSD,PostQCMax,PostQCMin,CoveragePostQC" } { print $0 } ' > DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.vs2.output.reformatted

R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperfastQpostQCStats.vs1.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs1.output.reformatted /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs1.output.reformatted





#20140808 For F31 Grant
vcftools --gzvcf Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.gz --freq --out Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.vcf.ex1

#20150129
#Jumping back in to start running true value multiple runs and fake value permutations to get final p-values/FDRs/results 

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.gz --missing --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD

~~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.imiss | sort -rg -k 5,5 | head -n 10
POOL_P2_66_03RL 86483   0       75173   0.869223
POOL_P2_175_01RL        86483   0       70385   0.813859
POOL_P2_139_08RL        86483   0       61414   0.710128
POOL_P2_155_12RL        86483   0       51092   0.590775
POOL_P2_105_08RL        86483   0       40091   0.463571
POOL_P2_133_05RL        86483   0       40029   0.462854
POOL_P2_138_08RL        86483   0       34303   0.396644
POOL_P2_170_11RL        86483   0       23038   0.266388
POOL_P2_133_04RL        86483   0       22483   0.25997
POOL_P2_113_09RL        86483   0       14051   0.162471
~~~

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/*Pheno1*perm1* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/*Pheno2*perm1* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/*Pheno3*perm1* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/*Pheno4*perm1* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/.

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/*Pheno1* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/*Pheno2* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/*Pheno3* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/.
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/*Pheno4* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/.

perl /home/michaelt/Scripts/Perl/RandomList.pl --total 10000 --seed 5697324 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.MultiRunList.vs1.txt
perl /home/michaelt/Scripts/Perl/RandomList.pl --total 10000 --seed 2138074 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.PermutationList.vs1.txt

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations

for i in {1..1000}
do

        SEED1=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.PermutationList.vs1.txt | awk '{ if (NR == '$i') { print $0 } } '`

	perl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs3.pl --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped --seed $SEED1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm${i}.vs3.ped 

#	echo $SEED1

done

~~~
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.vs3.ped | head -n 10                                                                                                                                                                                                                                                                                                       
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL_P2_83_08RL POOL_P2_83_08RL 0       0       0       2       NA      NA      NA      8       0.0156  0.0127  -0.0161 -0.0306 -0.0460 0.0286  0.0071  -0.0276 0.0150  -0.0125                                                                                                                                            
POOL_P2_156_08RL        POOL_P2_156_08RL        0       0       0       2       NA      NA      NA      8       0.0126  0.0086  -0.0303 0.0166  -0.0120 -0.0239 -0.0156 -0.0252 -0.0331 -0.0006
POOL_P2_135_03RL        POOL_P2_135_03RL        0       0       0       1       2       NA      NA      3       0.0088  0.0041  0.0147  0.0344  0.0045  -0.0063 0.0018  -0.0247 0.0196  -0.0216                                                                                                                            
POOL_P2_141_03RL        POOL_P2_141_03RL        0       0       0       1       2       NA      NA      3       0.0128  0.0106  -0.0328 0.0322  -0.0089 -0.0540 -0.0393 0.0170  0.0503  0.0290
POOL_P2_157_04RL        POOL_P2_157_04RL        0       0       0       1       NA      NA      NA      4       0.0124  0.0006  -0.0328 0.0068  0.0043  0.0227  -0.0153 -0.0068 -0.0462 -0.0168                                                                                                                            
POOL_P2_131_05RL        POOL_P2_131_05RL        0       0       0       1       2       NA      NA      5       0.0134  0.0059  0.0123  0.0390  0.0145  -0.0268 0.0012  0.0261  -0.0256 0.0273
POOL_P2_88_02RL POOL_P2_88_02RL 0       0       0       2       2       NA      NA      2       0.0116  0.0033  -0.0360 0.0206  0.0383  -0.0404 -0.0117 -0.0159 0.0066  0.0251                                                                                                                                             
POOL_P2_158_08RL        POOL_P2_158_08RL        0       0       0       2       1       2       NA      8       0.0088  0.0044  0.0137  0.0112  0.0499  0.0253  -0.0095 0.0353  -0.0019 0.0059
POOL_P2_169_01RL        POOL_P2_169_01RL        0       0       0       1       NA      NA      NA      1       0.0123  0.0039  0.0122  0.0155  0.0095  0.0183  0.0229  -0.0000 0.0239  0.0068                                                                                                                             
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm2.vs3.ped | head -n 10                                                                                                                                                                                                                                                                                                       
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL_P2_83_08RL POOL_P2_83_08RL 0       0       0       1       NA      NA      NA      8       0.0156  0.0127  -0.0161 -0.0306 -0.0460 0.0286  0.0071  -0.0276 0.0150  -0.0125                                                                                                                                            
POOL_P2_156_08RL        POOL_P2_156_08RL        0       0       0       1       NA      NA      NA      8       0.0126  0.0086  -0.0303 0.0166  -0.0120 -0.0239 -0.0156 -0.0252 -0.0331 -0.0006
POOL_P2_135_03RL        POOL_P2_135_03RL        0       0       0       2       2       NA      NA      3       0.0088  0.0041  0.0147  0.0344  0.0045  -0.0063 0.0018  -0.0247 0.0196  -0.0216                                                                                                                            
POOL_P2_141_03RL        POOL_P2_141_03RL        0       0       0       2       1       NA      NA      3       0.0128  0.0106  -0.0328 0.0322  -0.0089 -0.0540 -0.0393 0.0170  0.0503  0.0290
POOL_P2_157_04RL        POOL_P2_157_04RL        0       0       0       1       NA      NA      NA      4       0.0124  0.0006  -0.0328 0.0068  0.0043  0.0227  -0.0153 -0.0068 -0.0462 -0.0168                                                                                                                            
POOL_P2_131_05RL        POOL_P2_131_05RL        0       0       0       2       2       NA      NA      5       0.0134  0.0059  0.0123  0.0390  0.0145  -0.0268 0.0012  0.0261  -0.0256 0.0273
POOL_P2_88_02RL POOL_P2_88_02RL 0       0       0       1       2       NA      NA      2       0.0116  0.0033  -0.0360 0.0206  0.0383  -0.0404 -0.0117 -0.0159 0.0066  0.0251                                                                                                                                             
POOL_P2_158_08RL        POOL_P2_158_08RL        0       0       0       2       2       1       NA      8       0.0088  0.0044  0.0137  0.0112  0.0499  0.0253  -0.0095 0.0353  -0.0019 0.0059
POOL_P2_169_01RL        POOL_P2_169_01RL        0       0       0       2       NA      NA      NA      1       0.0123  0.0039  0.0122  0.0155  0.0095  0.0183  0.0229  -0.0000 0.0239  0.0068                                                                                                                             
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm3.vs3.ped | head -n 10                                                                                                                                                                                                                                                                                                       
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL_P2_83_08RL POOL_P2_83_08RL 0       0       0       2       NA      NA      NA      8       0.0156  0.0127  -0.0161 -0.0306 -0.0460 0.0286  0.0071  -0.0276 0.0150  -0.0125                                                                                                                                            
POOL_P2_156_08RL        POOL_P2_156_08RL        0       0       0       1       NA      NA      NA      8       0.0126  0.0086  -0.0303 0.0166  -0.0120 -0.0239 -0.0156 -0.0252 -0.0331 -0.0006
POOL_P2_135_03RL        POOL_P2_135_03RL        0       0       0       2       1       NA      NA      3       0.0088  0.0041  0.0147  0.0344  0.0045  -0.0063 0.0018  -0.0247 0.0196  -0.0216                                                                                                                            
POOL_P2_141_03RL        POOL_P2_141_03RL        0       0       0       1       2       NA      NA      3       0.0128  0.0106  -0.0328 0.0322  -0.0089 -0.0540 -0.0393 0.0170  0.0503  0.0290
POOL_P2_157_04RL        POOL_P2_157_04RL        0       0       0       2       NA      NA      NA      4       0.0124  0.0006  -0.0328 0.0068  0.0043  0.0227  -0.0153 -0.0068 -0.0462 -0.0168                                                                                                                            
POOL_P2_131_05RL        POOL_P2_131_05RL        0       0       0       2       2       NA      NA      5       0.0134  0.0059  0.0123  0.0390  0.0145  -0.0268 0.0012  0.0261  -0.0256 0.0273
POOL_P2_88_02RL POOL_P2_88_02RL 0       0       0       1       2       NA      NA      2       0.0116  0.0033  -0.0360 0.0206  0.0383  -0.0404 -0.0117 -0.0159 0.0066  0.0251                                                                                                                                             
POOL_P2_158_08RL        POOL_P2_158_08RL        0       0       0       2       1       2       NA      8       0.0088  0.0044  0.0137  0.0112  0.0499  0.0253  -0.0095 0.0353  -0.0019 0.0059
POOL_P2_169_01RL        POOL_P2_169_01RL        0       0       0       1       NA      NA      NA      1       0.0123  0.0039  0.0122  0.0155  0.0095  0.0183  0.0229  -0.0000 0.0239  0.0068                                                                                                                             
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm4.vs3.ped | head -n 10                                                                                                                                                                                                                                                                                                       
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL_P2_83_08RL POOL_P2_83_08RL 0       0       0       2       NA      NA      NA      8       0.0156  0.0127  -0.0161 -0.0306 -0.0460 0.0286  0.0071  -0.0276 0.0150  -0.0125                                                                                                                                            
POOL_P2_156_08RL        POOL_P2_156_08RL        0       0       0       2       NA      NA      NA      8       0.0126  0.0086  -0.0303 0.0166  -0.0120 -0.0239 -0.0156 -0.0252 -0.0331 -0.0006
POOL_P2_135_03RL        POOL_P2_135_03RL        0       0       0       2       2       NA      NA      3       0.0088  0.0041  0.0147  0.0344  0.0045  -0.0063 0.0018  -0.0247 0.0196  -0.0216                                                                                                                            
POOL_P2_141_03RL        POOL_P2_141_03RL        0       0       0       2       1       NA      NA      3       0.0128  0.0106  -0.0328 0.0322  -0.0089 -0.0540 -0.0393 0.0170  0.0503  0.0290
POOL_P2_157_04RL        POOL_P2_157_04RL        0       0       0       2       NA      NA      NA      4       0.0124  0.0006  -0.0328 0.0068  0.0043  0.0227  -0.0153 -0.0068 -0.0462 -0.0168                                                                                                                            
POOL_P2_131_05RL        POOL_P2_131_05RL        0       0       0       2       1       NA      NA      5       0.0134  0.0059  0.0123  0.0390  0.0145  -0.0268 0.0012  0.0261  -0.0256 0.0273
POOL_P2_88_02RL POOL_P2_88_02RL 0       0       0       1       2       NA      NA      2       0.0116  0.0033  -0.0360 0.0206  0.0383  -0.0404 -0.0117 -0.0159 0.0066  0.0251                                                                                                                                             
POOL_P2_158_08RL        POOL_P2_158_08RL        0       0       0       1       2       2       NA      8       0.0088  0.0044  0.0137  0.0112  0.0499  0.0253  -0.0095 0.0353  -0.0019 0.0059
POOL_P2_169_01RL        POOL_P2_169_01RL        0       0       0       2       NA      NA      NA      1       0.0123  0.0039  0.0122  0.0155  0.0095  0.0183  0.0229  -0.0000 0.0239  0.0068                                                                                                                             
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm5.vs3.ped | head -n 10                                                                                                                                                                                                                                                                                                       
#FAM_ID IND_ID  FAT_ID  MOT_ID  SEX     HIVPROG HIVPROGHE       AIDS    AIDSEXTR        RMID    PC1     PC2     PC3     PC4     PC5     PC6     PC7     PC8     PC9     PC10
POOL_P2_83_08RL POOL_P2_83_08RL 0       0       0       2       NA      NA      NA      8       0.0156  0.0127  -0.0161 -0.0306 -0.0460 0.0286  0.0071  -0.0276 0.0150  -0.0125                                                                                                                                            
POOL_P2_156_08RL        POOL_P2_156_08RL        0       0       0       1       NA      NA      NA      8       0.0126  0.0086  -0.0303 0.0166  -0.0120 -0.0239 -0.0156 -0.0252 -0.0331 -0.0006
POOL_P2_135_03RL        POOL_P2_135_03RL        0       0       0       2       2       NA      NA      3       0.0088  0.0041  0.0147  0.0344  0.0045  -0.0063 0.0018  -0.0247 0.0196  -0.0216                                                                                                                            
POOL_P2_141_03RL        POOL_P2_141_03RL        0       0       0       2       2       NA      NA      3       0.0128  0.0106  -0.0328 0.0322  -0.0089 -0.0540 -0.0393 0.0170  0.0503  0.0290
POOL_P2_157_04RL        POOL_P2_157_04RL        0       0       0       1       NA      NA      NA      4       0.0124  0.0006  -0.0328 0.0068  0.0043  0.0227  -0.0153 -0.0068 -0.0462 -0.0168                                                                                                                            
POOL_P2_131_05RL        POOL_P2_131_05RL        0       0       0       2       2       NA      NA      5       0.0134  0.0059  0.0123  0.0390  0.0145  -0.0268 0.0012  0.0261  -0.0256 0.0273
POOL_P2_88_02RL POOL_P2_88_02RL 0       0       0       1       1       NA      NA      2       0.0116  0.0033  -0.0360 0.0206  0.0383  -0.0404 -0.0117 -0.0159 0.0066  0.0251                                                                                                                                             
POOL_P2_158_08RL        POOL_P2_158_08RL        0       0       0       1       2       2       NA      8       0.0088  0.0044  0.0137  0.0112  0.0499  0.0253  -0.0095 0.0353  -0.0019 0.0059
POOL_P2_169_01RL        POOL_P2_169_01RL        0       0       0       1       NA      NA      NA      1       0.0123  0.0039  0.0122  0.0155  0.0095  0.0183  0.0229  -0.0000 0.0239  0.0068                                                                                                                            
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm1.vs3.ped | head -n 150 | tail -n 10
POOL_P2_80_01RL POOL_P2_80_01RL 0       0       0       2       NA      NA      NA      1       0.0148  0.0059  -0.0233 -0.0426 0.0418  -0.0008 -0.0098 0.0051  -0.0213 0.0252
POOL_P2_154_06RL        POOL_P2_154_06RL        0       0       0       2       2       NA      NA      6       0.0164  0.0064  0.0211  0.0186  0.0552  -0.0092 -0.0025 -0.0114 0.0258  -0.0670
POOL_P2_158_12RL        POOL_P2_158_12RL        0       0       0       2       2       NA      NA      12      0.0138  0.0202  0.0158  0.0525  -0.0343 -0.0234 0.0504  0.0353  0.0001  0.0148
POOL_P2_166_09RL        POOL_P2_166_09RL        0       0       0       1       2       2       NA      9       0.0140  0.0055  -0.0177 0.0377  -0.0096 -0.0398 -0.0454 0.0344  0.0272  -0.0319
POOL_P2_149_02RL        POOL_P2_149_02RL        0       0       0       2       1       NA      NA      2       0.0112  0.0123  0.0139  0.0321  -0.0063 -0.0345 -0.0308 0.0064  0.0266  0.0154
POOL_P2_67_06RL POOL_P2_67_06RL 0       0       0       2       2       NA      NA      6       0.0145  0.0064  0.0266  -0.0201 -0.0388 0.0020  0.0063  -0.0134 0.0308  0.0171
POOL_P2_75_02RL POOL_P2_75_02RL 0       0       0       2       1       NA      NA      2       0.0106  0.0043  -0.0382 0.0131  -0.0260 0.0097  -0.0206 -0.0053 0.0158  -0.0481
POOL_P2_60_03RL POOL_P2_60_03RL 0       0       0       2       1       NA      NA      3       0.0120  0.0105  0.0506  -0.0095 -0.0124 -0.0044 0.0005  -0.0130 0.0393  -0.0497
POOL_P2_91_12RL POOL_P2_91_12RL 0       0       0       1       1       NA      NA      12      0.0177  0.0039  -0.0203 -0.0247 -0.0437 -0.0020 -0.0005 0.0200  0.0203  -0.0326
POOL_P2_86_10RL POOL_P2_86_10RL 0       0       0       2       NA      NA      NA      10      0.0111  0.0091  -0.0263 0.0054  0.0154  0.0027  -0.0567 -0.0416 -0.0390 0.0190
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm2.vs3.ped | head -n 150 | tail -n 10
POOL_P2_80_01RL POOL_P2_80_01RL 0       0       0       2       NA      NA      NA      1       0.0148  0.0059  -0.0233 -0.0426 0.0418  -0.0008 -0.0098 0.0051  -0.0213 0.0252
POOL_P2_154_06RL        POOL_P2_154_06RL        0       0       0       1       2       NA      NA      6       0.0164  0.0064  0.0211  0.0186  0.0552  -0.0092 -0.0025 -0.0114 0.0258  -0.0670
POOL_P2_158_12RL        POOL_P2_158_12RL        0       0       0       2       2       NA      NA      12      0.0138  0.0202  0.0158  0.0525  -0.0343 -0.0234 0.0504  0.0353  0.0001  0.0148
POOL_P2_166_09RL        POOL_P2_166_09RL        0       0       0       1       1       2       NA      9       0.0140  0.0055  -0.0177 0.0377  -0.0096 -0.0398 -0.0454 0.0344  0.0272  -0.0319
POOL_P2_149_02RL        POOL_P2_149_02RL        0       0       0       2       2       NA      NA      2       0.0112  0.0123  0.0139  0.0321  -0.0063 -0.0345 -0.0308 0.0064  0.0266  0.0154
POOL_P2_67_06RL POOL_P2_67_06RL 0       0       0       2       2       NA      NA      6       0.0145  0.0064  0.0266  -0.0201 -0.0388 0.0020  0.0063  -0.0134 0.0308  0.0171
POOL_P2_75_02RL POOL_P2_75_02RL 0       0       0       2       2       NA      NA      2       0.0106  0.0043  -0.0382 0.0131  -0.0260 0.0097  -0.0206 -0.0053 0.0158  -0.0481
POOL_P2_60_03RL POOL_P2_60_03RL 0       0       0       1       2       NA      NA      3       0.0120  0.0105  0.0506  -0.0095 -0.0124 -0.0044 0.0005  -0.0130 0.0393  -0.0497
POOL_P2_91_12RL POOL_P2_91_12RL 0       0       0       2       1       NA      NA      12      0.0177  0.0039  -0.0203 -0.0247 -0.0437 -0.0020 -0.0005 0.0200  0.0203  -0.0326
POOL_P2_86_10RL POOL_P2_86_10RL 0       0       0       2       NA      NA      NA      10      0.0111  0.0091  -0.0263 0.0054  0.0154  0.0027  -0.0567 -0.0416 -0.0390 0.0190
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm3.vs3.ped | head -n 150 | tail -n 10
POOL_P2_80_01RL POOL_P2_80_01RL 0       0       0       2       NA      NA      NA      1       0.0148  0.0059  -0.0233 -0.0426 0.0418  -0.0008 -0.0098 0.0051  -0.0213 0.0252
POOL_P2_154_06RL        POOL_P2_154_06RL        0       0       0       2       2       NA      NA      6       0.0164  0.0064  0.0211  0.0186  0.0552  -0.0092 -0.0025 -0.0114 0.0258  -0.0670
POOL_P2_158_12RL        POOL_P2_158_12RL        0       0       0       1       1       NA      NA      12      0.0138  0.0202  0.0158  0.0525  -0.0343 -0.0234 0.0504  0.0353  0.0001  0.0148
POOL_P2_166_09RL        POOL_P2_166_09RL        0       0       0       2       2       2       NA      9       0.0140  0.0055  -0.0177 0.0377  -0.0096 -0.0398 -0.0454 0.0344  0.0272  -0.0319
POOL_P2_149_02RL        POOL_P2_149_02RL        0       0       0       2       2       NA      NA      2       0.0112  0.0123  0.0139  0.0321  -0.0063 -0.0345 -0.0308 0.0064  0.0266  0.0154
POOL_P2_67_06RL POOL_P2_67_06RL 0       0       0       1       2       NA      NA      6       0.0145  0.0064  0.0266  -0.0201 -0.0388 0.0020  0.0063  -0.0134 0.0308  0.0171
POOL_P2_75_02RL POOL_P2_75_02RL 0       0       0       2       1       NA      NA      2       0.0106  0.0043  -0.0382 0.0131  -0.0260 0.0097  -0.0206 -0.0053 0.0158  -0.0481
POOL_P2_60_03RL POOL_P2_60_03RL 0       0       0       2       2       NA      NA      3       0.0120  0.0105  0.0506  -0.0095 -0.0124 -0.0044 0.0005  -0.0130 0.0393  -0.0497
POOL_P2_91_12RL POOL_P2_91_12RL 0       0       0       1       2       NA      NA      12      0.0177  0.0039  -0.0203 -0.0247 -0.0437 -0.0020 -0.0005 0.0200  0.0203  -0.0326
POOL_P2_86_10RL POOL_P2_86_10RL 0       0       0       1       NA      NA      NA      10      0.0111  0.0091  -0.0263 0.0054  0.0154  0.0027  -0.0567 -0.0416 -0.0390 0.0190
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm4.vs3.ped | head -n 150 | tail -n 10
POOL_P2_80_01RL POOL_P2_80_01RL 0       0       0       1       NA      NA      NA      1       0.0148  0.0059  -0.0233 -0.0426 0.0418  -0.0008 -0.0098 0.0051  -0.0213 0.0252
POOL_P2_154_06RL        POOL_P2_154_06RL        0       0       0       2       2       NA      NA      6       0.0164  0.0064  0.0211  0.0186  0.0552  -0.0092 -0.0025 -0.0114 0.0258  -0.0670
POOL_P2_158_12RL        POOL_P2_158_12RL        0       0       0       2       2       NA      NA      12      0.0138  0.0202  0.0158  0.0525  -0.0343 -0.0234 0.0504  0.0353  0.0001  0.0148
POOL_P2_166_09RL        POOL_P2_166_09RL        0       0       0       1       1       1       NA      9       0.0140  0.0055  -0.0177 0.0377  -0.0096 -0.0398 -0.0454 0.0344  0.0272  -0.0319
POOL_P2_149_02RL        POOL_P2_149_02RL        0       0       0       2       2       NA      NA      2       0.0112  0.0123  0.0139  0.0321  -0.0063 -0.0345 -0.0308 0.0064  0.0266  0.0154
POOL_P2_67_06RL POOL_P2_67_06RL 0       0       0       2       2       NA      NA      6       0.0145  0.0064  0.0266  -0.0201 -0.0388 0.0020  0.0063  -0.0134 0.0308  0.0171
POOL_P2_75_02RL POOL_P2_75_02RL 0       0       0       2       2       NA      NA      2       0.0106  0.0043  -0.0382 0.0131  -0.0260 0.0097  -0.0206 -0.0053 0.0158  -0.0481
POOL_P2_60_03RL POOL_P2_60_03RL 0       0       0       1       1       NA      NA      3       0.0120  0.0105  0.0506  -0.0095 -0.0124 -0.0044 0.0005  -0.0130 0.0393  -0.0497
POOL_P2_91_12RL POOL_P2_91_12RL 0       0       0       1       1       NA      NA      12      0.0177  0.0039  -0.0203 -0.0247 -0.0437 -0.0020 -0.0005 0.0200  0.0203  -0.0326
POOL_P2_86_10RL POOL_P2_86_10RL 0       0       0       1       NA      NA      NA      10      0.0111  0.0091  -0.0263 0.0054  0.0154  0.0027  -0.0567 -0.0416 -0.0390 0.0190
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.perm5.vs3.ped | head -n 150 | tail -n 10
POOL_P2_80_01RL POOL_P2_80_01RL 0       0       0       2       NA      NA      NA      1       0.0148  0.0059  -0.0233 -0.0426 0.0418  -0.0008 -0.0098 0.0051  -0.0213 0.0252
POOL_P2_154_06RL        POOL_P2_154_06RL        0       0       0       1       2       NA      NA      6       0.0164  0.0064  0.0211  0.0186  0.0552  -0.0092 -0.0025 -0.0114 0.0258  -0.0670
POOL_P2_158_12RL        POOL_P2_158_12RL        0       0       0       2       2       NA      NA      12      0.0138  0.0202  0.0158  0.0525  -0.0343 -0.0234 0.0504  0.0353  0.0001  0.0148
POOL_P2_166_09RL        POOL_P2_166_09RL        0       0       0       2       2       1       NA      9       0.0140  0.0055  -0.0177 0.0377  -0.0096 -0.0398 -0.0454 0.0344  0.0272  -0.0319
POOL_P2_149_02RL        POOL_P2_149_02RL        0       0       0       1       2       NA      NA      2       0.0112  0.0123  0.0139  0.0321  -0.0063 -0.0345 -0.0308 0.0064  0.0266  0.0154
POOL_P2_67_06RL POOL_P2_67_06RL 0       0       0       2       1       NA      NA      6       0.0145  0.0064  0.0266  -0.0201 -0.0388 0.0020  0.0063  -0.0134 0.0308  0.0171
POOL_P2_75_02RL POOL_P2_75_02RL 0       0       0       2       2       NA      NA      2       0.0106  0.0043  -0.0382 0.0131  -0.0260 0.0097  -0.0206 -0.0053 0.0158  -0.0481
POOL_P2_60_03RL POOL_P2_60_03RL 0       0       0       2       2       NA      NA      3       0.0120  0.0105  0.0506  -0.0095 -0.0124 -0.0044 0.0005  -0.0130 0.0393  -0.0497
POOL_P2_91_12RL POOL_P2_91_12RL 0       0       0       2       1       NA      NA      12      0.0177  0.0039  -0.0203 -0.0247 -0.0437 -0.0020 -0.0005 0.0200  0.0203  -0.0326
POOL_P2_86_10RL POOL_P2_86_10RL 0       0       0       2       NA      NA      NA      10      0.0111  0.0091  -0.0263 0.0054  0.0154  0.0027  -0.0567 -0.0416 -0.0390 0.0190
~~~

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs

#mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/*Pheno*run* /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/.

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs





#20150214
#2nd run -- investigating applying a more stringent imiss threshold (just to see how it may or may not change results....this run may not be used, just investigating)
#Using .Vs2. in titles

#mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs2/GATK
#mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs2/GATK/PLINKfiles
#mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs2/GATK/EPACTSFiles

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.vcf.gz --geno .95 --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95

gzip -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.recode.vcf.gz --hwe .0001 --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.recode.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf

#CHECK_0
#/home/shared/software/java/jre1.7.0_09/bin/java -Xmx10g -jar /home/shared/software/GATK/2.5.2/GenomeAnalysisTK.jar -T VariantAnnotator -R /home/michaelt/Data/HumanGenome/GRCh37/Version4/human_g1k_v37.fasta -I /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/AllPools.P2.Vs2.AllPoolsMerged.bam -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.HRun.vcf -A HomopolymerRun --variant /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf -nt 32

#mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.HRun.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf

#mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.HRun.vcf.idx /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.idx

gzip -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf

#zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | perl -lane 'my @vals1 = split(/;/, $F[7]); foreach my $SNPinfo (@vals1) { if ($SNPinfo =~ m/HRun/) { print $SNPinfo; } };' | sort | uniq -c 

#1 HRun=8
#1 HRun=9

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --plink --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --missing --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4

plink --file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4

#Final SNP-QC'ed .vcf created by this point
#Indv QC processes next

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.imiss | awk '{ if ($5 > .99) { print $1, "\t", $1 } } ' | grep -v INDV > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.imiss.Above99Miss.justFIDIIDs

#Dropping these individuals for --genome just so the program finishes running; with individuals that have >.99 missingness the program fails to run. This is consistent with P1 because no individuals had missingness >.99 so the program was not interrupted (also empirically no individuals were close to .99 missingness)
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --remove /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.imiss.Above99Miss.justFIDIIDs --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.Indv99

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/VLookUp.Python.DataHapMap3Overlap.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.map --file2 /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.liftOver.hg18tohg19.map | awk '{ print $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.rsIDs 

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.rsIDs --recode --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3

plink --file /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3 --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.map | awk '{ print $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.map.rsIDs

plink --bfile /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.map.rsIDs --keep /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.CEUCHBYRI.Unrelated.FIDIIDs --recode --out /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Array2.Vs2

plink --file /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Array2.Vs2 --make-bed /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Array2.Vs2

paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.bim /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Array2.Vs2.bim  | awk '{ if (($5 == $11) || ($5 == $12) ) { print $0 } } ' | awk '{ print $2 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.FinalListwDrops.rsIDs 

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3 --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.FinalListwDrops.rsIDs --recode --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.vs2

plink --bfile /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly --extract /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.FinalListwDrops.rsIDs --keep /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.CEUCHBYRI.Unrelated.FIDIIDs --recode --out /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Array2.Vs2.vs2

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.vs2.ped /home/michaelt/Data/HapMap/HapMap3/hapmap3_r2_b36_fwd.consensus.qc.poly.UnrelatedCEUCHBYRI.overlapHIVGeneExomeData.Array2.Vs2.vs2.ped > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.ped

plink --ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.ped --map /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.overlapHapMap3.vs2.map --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.ped | perl -lane '$F[5] = "1"; print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.phenoColEdit.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.phenoColEdit.ped | perl -lane 'print join("\t", @F[0..5]);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.phenoColEdit.pedind 

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.bim /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.pedsnp

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.parfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.parfile

/home/michaelt/Software/EIG5.0.1/bin/smartpca -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.parfile

#Troubleshooting
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_100_06RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_101_08RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_101_11RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_112_01RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_112_08RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_119_02RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_149_10RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_154_02RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_154_12RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_172_02RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_172_03RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_172_04RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_172_05RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_172_06RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_67_10RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_72_06RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_73_09RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_87_09RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_87_12RL)
#Error 1a -- indvID not found in hash1 (indvID: POOL_P2_89_11RL)


python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.vs1.ForP2.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca | cat - <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca | grep -E 'POOL_P2_100_06RL|POOL_P2_101_08RL|POOL_P2_101_11RL|POOL_P2_112_01RL|POOL_P2_112_08RL|POOL_P2_119_02RL|POOL_P2_149_10RL|POOL_P2_154_02RL|POOL_P2_154_12RL|POOL_P2_172_02RL|POOL_P2_172_03RL|POOL_P2_172_04RL|POOL_P2_172_05RL|POOL_P2_172_06RL|POOL_P2_67_10RL|POOL_P2_71_03RL|POOL_P2_72_06RL|POOL_P2_73_09RL|POOL_P2_87_09RL|POOL_P2_87_12RL|POOL_P2_88_10RL|POOL_P2_89_11RL' | perl -lane '$F[$#F] = "NoPhenotype"; print join("\t", @F); ') > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.pt1

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.EditEigenstratPCAfileForPlotting.HapMap3vs.py --file1 /home/michaelt/Data/HapMap/HapMap3/draft2_samples_QC+.txt --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.pt2

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.pt1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.pt2 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race

sed -i 's/American_Indian_or_Alaskan_Native/AIndian_or_Alaskan_Native/g' /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race -c 1:2 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.plot.1vs2.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race -c 3:4 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.plot.3vs4.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race -c 5:6 -p White_non-Hispanic:White_Hispanic:Asian_or_Pacific_Islander:AIndian_or_Alaskan_Native:Asian_or_Pacific_Islander:Black_non-Hispanic:Black_Hispanic:Other:CEU:CHB:YRI:NoPhenotype -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.plot.5vs6.xtxt

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.plot.*.pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/.

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race | awk '{ if ((($2 >= -.001) && ($2 <= .22)) && (($3 >= -.005) && ($3 <= .021))) { print $0 } } ' | grep White_non-Hispanic > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed

~~~
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles]$cmp <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed | awk '{ print $1 }') <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed | awk '{ print $1 }')
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles]$paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed | awk '{ print $1 }') <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed | awk '{ print $1 }') | head -n 10
POOL_P2_100_01RL:POOL_P2_100_01RL       POOL_P2_100_01RL:POOL_P2_100_01RL
POOL_P2_100_03RL:POOL_P2_100_03RL       POOL_P2_100_03RL:POOL_P2_100_03RL
POOL_P2_100_05RL:POOL_P2_100_05RL       POOL_P2_100_05RL:POOL_P2_100_05RL
POOL_P2_100_09RL:POOL_P2_100_09RL       POOL_P2_100_09RL:POOL_P2_100_09RL
POOL_P2_100_10RL:POOL_P2_100_10RL       POOL_P2_100_10RL:POOL_P2_100_10RL
~~~

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed -c 1:2 -p White_non-Hispanic -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.plot.1vs2.xtxt

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.plot.*.pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/.

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed | perl -lane 'my @vals1 = split(/:/, $F[0]); print $vals1[0];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed | perl -lane 'my @vals1 = split(/:/, $F[0]); print $vals1[0], "\t", $vals1[0];' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDIIDs --genome --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed

#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome | sort -rg -k 9,9 | head -n 20

~~~
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome | sort -rg -k 9,9 | head -n 20
POOL_P2_103_03RL  POOL_P2_103_03RL  POOL_P2_105_03RL  POOL_P2_105_03RL UN    NA  0.0000  0.0570  0.9430  0.9715  -1  0.998377  1.0000      NA
POOL_P2_101_01RL  POOL_P2_101_01RL  POOL_P2_104_01RL  POOL_P2_104_01RL UN    NA  0.0238  0.1964  0.7798  0.8780  -1  0.993273  1.0000 77.6667
POOL_P2_163_12RL  POOL_P2_163_12RL   POOL_P2_69_08RL   POOL_P2_69_08RL UN    NA  0.0409  0.2277  0.7314  0.8453  -1  0.991571  1.0000 67.0000
POOL_P2_140_10RL  POOL_P2_140_10RL   POOL_P2_87_11RL   POOL_P2_87_11RL UN    NA  0.0241  0.2550  0.7210  0.8485  -1  0.991593  1.0000 75.1667
POOL_P2_145_11RL  POOL_P2_145_11RL  POOL_P2_150_12RL  POOL_P2_150_12RL UN    NA  0.0456  0.2393  0.7151  0.8348  -1  0.991017  1.0000 67.5714
POOL_P2_164_05RL  POOL_P2_164_05RL   POOL_P2_86_04RL   POOL_P2_86_04RL UN    NA  0.0011  0.6676  0.3313  0.6651  -1  0.980924  1.0000      NA
POOL_P2_59_09RL   POOL_P2_59_09RL   POOL_P2_61_12RL   POOL_P2_61_12RL UN    NA  0.0101  0.6867  0.3032  0.6466  -1  0.979954  1.0000 164.3333
POOL_P2_155_08RL  POOL_P2_155_08RL   POOL_P2_66_03RL   POOL_P2_66_03RL UN    NA  0.7268  0.0000  0.2732  0.2732  -1  0.965022  0.1030  1.3684
POOL_P2_66_03RL   POOL_P2_66_03RL   POOL_P2_93_07RL   POOL_P2_93_07RL UN    NA  0.7278  0.0000  0.2722  0.2722  -1  0.964485  0.1792  1.5263
POOL_P2_66_03RL   POOL_P2_66_03RL   POOL_P2_97_03RL   POOL_P2_97_03RL UN    NA  0.7384  0.0000  0.2616  0.2616  -1  0.963792  0.3400  1.7647

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | grep -E 'POOL_P2_103_03RL|POOL_P2_105_03RL'
POOL_P2_103_03RL        POOL_P2_103_03RL        2       2       -9      -9
POOL_P2_105_03RL        POOL_P2_105_03RL        2       2       -9      -9

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | grep -E 'POOL_P2_101_01RL|POOL_P2_104_01RL'
POOL_P2_101_01RL        POOL_P2_101_01RL        1       1       -9      -9
POOL_P2_104_01RL        POOL_P2_104_01RL        1       1       -9      -9

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | grep -E 'POOL_P2_163_12RL|POOL_P2_69_08RL'
POOL_P2_69_08RL POOL_P2_69_08RL 1       1       -9      -9
POOL_P2_163_12RL        POOL_P2_163_12RL        1       -9      -9      -9

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | grep -E 'POOL_P2_140_10RL|POOL_P2_87_11RL'
POOL_P2_87_11RL POOL_P2_87_11RL 1       -9      -9      -9
POOL_P2_140_10RL        POOL_P2_140_10RL        2       2       -9      -9

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | grep -E 'POOL_P2_145_11RL|POOL_P2_150_12RL'
POOL_P2_150_12RL        POOL_P2_150_12RL        2       2       1       -9
POOL_P2_145_11RL        POOL_P2_145_11RL        2       2       1       -9
~~~

perl -e 'print "POOL_P2_103_03RL\nPOOL_P2_105_03RL\nPOOL_P2_101_01RL\nPOOL_P2_104_01RL\nPOOL_P2_163_12RL\nPOOL_P2_69_08RL\nPOOL_P2_140_10RL\nPOOL_P2_87_11RL\nPOOL_P2_145_11RL\nPOOL_P2_150_12RL\n";' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.FIDs

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.FIDs --depth --missing --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs

~~~
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.idepth
INDV    N_SITES MEAN_DEPTH
POOL_P2_101_01RL        94775   11.7298
POOL_P2_103_03RL        94881   18.6866
POOL_P2_104_01RL        93535   6.26491
POOL_P2_105_03RL        94879   18.3223
POOL_P2_140_10RL        92567   5.24853
POOL_P2_145_11RL        94480   9.1442
POOL_P2_150_12RL        93373   5.90921
POOL_P2_163_12RL        94183   7.61423
POOL_P2_69_08RL 94066   8.19753
POOL_P2_87_11RL 94851   14.0026

[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.imiss
INDV    N_DATA  N_GENOTYPES_FILTERED    N_MISS  F_MISS
POOL_P2_101_01RL        94889   0       114     0.0012014
POOL_P2_103_03RL        94889   0       8       8.4309e-05
POOL_P2_104_01RL        94889   0       1354    0.0142693
POOL_P2_105_03RL        94889   0       10      0.000105386
POOL_P2_140_10RL        94889   0       2322    0.0244707
POOL_P2_145_11RL        94889   0       409     0.0043103
POOL_P2_150_12RL        94889   0       1516    0.0159766
POOL_P2_163_12RL        94889   0       706     0.00744027
POOL_P2_69_08RL 94889   0       823     0.00867329
POOL_P2_87_11RL 94889   0       38      0.000400468
~~~

So choosing:
POOL_P2_105_03RL
POOL_P2_101_01RL
POOL_P2_145_11RL

perl -e 'print "POOL_P2_103_03RL\nPOOL_P2_104_01RL\nPOOL_P2_140_10RL\nPOOL_P2_150_12RL\nPOOL_P2_163_12RL\nPOOL_P2_69_08RL\nPOOL_P2_87_11RL\n";' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome | sort -rg -k 9,9 | head -n 10000 | awk '{ print $1, "\n", $3 }' | perl -lane 'chomp(@F); print join("\t", @F);' | sort | uniq -c | sort -g -k 1,1 | awk '{ if ($1 >= 110 ) { print $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ2Countgte110.FIDs  

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome | sort -rg -k 8,8 | head -n 10000 | awk '{ print $1, "\n", $3 }' | perl -lane 'chomp(@F); print join("\t", @F);' | sort | uniq -c | sort -g -k 1,1 | awk '{ if ($1 >= 100) { print $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.FIDs

#NOTE -- dropping these individuals because they are starkly greater than anyone else in the list. Individuals from the 'high z2' count list are not being dropped because they appear as just to be the tail of a distribution versus being such a group of outliers

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs

cat <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ2Countgte110.FIDs | perl -lane 'push(@F, "Z2"); print join("\t", @F);') <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.FIDs | perl -lane 'push(@F, "Z1"); print join("\t", @F);') > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.OddHighZ1OrZ2.FID_ZNums

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race | awk 'BEGIN{ while(getline < "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.OddHighZ1OrZ2.FID_ZNums") { FID = $1":"$1; FIDList[FID] = $2; }} { if (FIDList[$1]) { if (FIDList[$1] == "Z1") { $12 = "Z1High"; } else if (FIDList[$1] == "Z2") { $12 = "Z2High"; } else{ print "blah" }  } else { $12 = "NormalIBD"; } print $0; } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.IBDissuesTagged 

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.IBDissuesTagged -c 1:2 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.IBDissuesTagged.plot.1vs2.xtxt

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed | awk 'BEGIN{ while(getline < "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.OddHighZ1OrZ2.FID_ZNums") { FID = $1":"$1; FIDList[FID] = $2; }} { if (FIDList[$1]) { if (FIDList[$1] == "Z1") { $12 = "Z1High"; } else if (FIDList[$1] == "Z2") { $12 = "Z2High"; } else{ print "blah" }  } else { $12 = "NormalIBD"; } print $0; } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged

/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 1:2 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.1vs2.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 3:4 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.3vs4.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 5:6 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.5vs6.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 7:8 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.7vs8.xtxt
/home/michaelt/Software/EIG5.0.1/bin/ploteig -i /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged -c 9:10 -p NormalIBD:Z1High:Z2High -x -o /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.plot.9vs10.xtxt

scp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.IBDissuesTagged.*pdf mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/.

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.covar > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.5PCs.covar

~~~
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.5PCs.covar | wc
890    8900   69165
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.5PCs.covar | wc
889    8890   69081
~~~

#Edit code in between runs
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.AddPCComponentsToCovarFile.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.covar > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar

~~~
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles]$head -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar
POOL_P2_137_06RL        POOL_P2_137_06RL        137     6       3       -0.0601 0.0109  0.0150  0.0031  0.0413  -0.0396 0.0464  0.0252  -0.0499 -0.0633
POOL_P2_83_08RL POOL_P2_83_08RL 83      8       1       0.0156  0.0127  -0.0161 -0.0306 -0.0460 0.0286  0.0071  -0.0276 0.0150  -0.0125
POOL_P2_156_08RL        POOL_P2_156_08RL        156     8       1       0.0126  0.0086  -0.0303 0.0166  -0.0120 -0.0239 -0.0156 -0.0252 -0.0331 -0.0006
POOL_P2_85_09RL POOL_P2_85_09RL 85      9       2       0.0023  -0.0281 0.0279  0.0265  -0.0593 0.0079  -0.0928 0.0291  -0.0548 0.0129
POOL_P2_147_06RL        POOL_P2_147_06RL        147     6       3       -0.0632 0.0108  0.0065  0.0254  -0.0178 0.0509  0.0797  -0.0028 -0.0256 0.0074
POOL_P2_135_03RL        POOL_P2_135_03RL        135     3       1       0.0088  0.0041  0.0147  0.0344  0.0045  -0.0063 0.0018  -0.0247 0.0196  -0.0216
POOL_P2_141_03RL        POOL_P2_141_03RL        141     3       1       0.0128  0.0106  -0.0328 0.0322  -0.0089 -0.0540 -0.0393 0.0170  0.0503  0.0290
POOL_P2_157_04RL        POOL_P2_157_04RL        157     4       1       0.0124  0.0006  -0.0328 0.0068  0.0043  0.0227  -0.0153 -0.0068 -0.0462 -0.0168
POOL_P2_131_05RL        POOL_P2_131_05RL        131     5       1       0.0134  0.0059  0.0123  0.0390  0.0145  -0.0268 0.0012  0.0261  -0.0256 0.0273
POOL_P2_88_02RL POOL_P2_88_02RL 88      2       1       0.0116  0.0033  -0.0360 0.0206  0.0383  -0.0404 -0.0117 -0.0159 0.0066  0.0251
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles]$tail -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar
POOL_P2_163_10RL        POOL_P2_163_10RL        163     10      1       0.0138  0.0081  0.0540  0.0400  -0.0431 -0.0153 0.0136  -0.0065 0.0269  0.0156
POOL_P2_175_01RL        POOL_P2_175_01RL        175     1       1       0.0027  0.0034  0.0113  0.0076  0.0042  0.0168  0.0104  0.0005  -0.0106 0.0102
POOL_P2_57_04RL POOL_P2_57_04RL 57      4       1       0.0139  0.0090  0.0151  0.0513  -0.0042 -0.0391 0.0460  0.0427  -0.0206 -0.0140
POOL_P2_135_06RL        POOL_P2_135_06RL        135     6       1       0.0114  0.0113  0.0157  -0.0421 0.0131  0.0041  -0.0043 0.0104  -0.0126 -0.0211
POOL_P2_167_03RL        POOL_P2_167_03RL        167     3       3       -0.0373 0.0066  -0.0036 -0.0227 0.0269  0.0191  -0.0293 -0.0145 -0.0032 0.0271
POOL_P2_161_12RL        POOL_P2_161_12RL        161     12      1       0.0114  0.0051  0.0079  0.0162  0.0471  0.0437  0.0110  -0.0227 -0.0313 -0.0262
POOL_P2_88_11RL POOL_P2_88_11RL 88      11      2       0.0014  -0.0190 0.0395  -0.0262 0.0197  -0.0256 -0.0052 0.0645  0.0065  0.0502
POOL_P2_62_11RL POOL_P2_62_11RL 62      11      1       0.0158  0.0099  -0.0148 -0.0108 -0.0054 0.0104  0.0120  0.0468  -0.0107 0.0272
POOL_P2_82_02RL POOL_P2_82_02RL 82      2       1       0.0111  0.0044  -0.0247 0.0102  0.0175  -0.0198 -0.0190 -0.0370 -0.0294 -0.0593
POOL_P2_173_01RL        POOL_P2_173_01RL        173     1       1       0.0084  0.0039  0.0102  0.0066  0.0002  0.0389  -0.0032 0.0030  0.0288  0.0046
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles]$tail -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar
POOL_P2_163_10RL        POOL_P2_163_10RL        163     10      1       0.0137  0.0081  0.0540  0.0399  -0.0426 -0.0161 0.0133  -0.0057 0.0275  0.0134
POOL_P2_175_01RL        POOL_P2_175_01RL        175     1       1       0.0027  0.0035  0.0112  0.0077  0.0038  0.0172  0.0103  0.0004  -0.0102 0.0110
POOL_P2_57_04RL POOL_P2_57_04RL 57      4       1       0.0139  0.0089  0.0152  0.0513  -0.0036 -0.0389 0.0455  0.0418  -0.0215 -0.0127
POOL_P2_135_06RL        POOL_P2_135_06RL        135     6       1       0.0114  0.0112  0.0158  -0.0421 0.0132  0.0042  -0.0037 0.0099  -0.0143 -0.0211
POOL_P2_167_03RL        POOL_P2_167_03RL        167     3       3       -0.0370 0.0066  -0.0035 -0.0226 0.0263  0.0195  -0.0307 -0.0125 -0.0016 0.0295
POOL_P2_161_12RL        POOL_P2_161_12RL        161     12      1       0.0114  0.0051  0.0079  0.0164  0.0461  0.0446  0.0108  -0.0239 -0.0310 -0.0235
POOL_P2_88_11RL POOL_P2_88_11RL 88      11      2       0.0014  -0.0188 0.0394  -0.0265 0.0194  -0.0251 -0.0056 0.0647  0.0063  0.0488
POOL_P2_62_11RL POOL_P2_62_11RL 62      11      1       0.0157  0.0099  -0.0147 -0.0106 -0.0055 0.0104  0.0118  0.0472  -0.0113 0.0257
POOL_P2_82_02RL POOL_P2_82_02RL 82      2       1       0.0111  0.0042  -0.0245 0.0101  0.0180  -0.0200 -0.0182 -0.0384 -0.0303 -0.0577
POOL_P2_173_01RL        POOL_P2_173_01RL        173     1       1       0.0083  0.0039  0.0101  0.0070  -0.0010 0.0391  -0.0028 0.0032  0.0291  0.0023
~~~

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss | awk '{ if ($5 >= .2) { print $1 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.pheno 

~~~
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.pheno | wc
747    4482   32288
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.pheno | wc
756    4536   32680
~~~

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar | awk '{ print $1, "\t", $2 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar | awk '{ print $1 }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDs

#CHECK_0
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDs  --non-ref-ac 1 --recode --recode-INFO-all --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.recode.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf

gzip -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.allSNPs
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar --covar-number 2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.10PCs
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar --covar-number 2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.10PCs.maf05
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 2 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno2
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 3 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno3
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 4 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno4

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.assoc.logistic Pheno1_justWhite_Logistic_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.assoc.logistic.QQplot.jpeg
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.assoc.logistic Pheno1_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.assoc.logistic.QQplot.jpeg
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.10PCs.assoc.logistic Pheno1_justWhite_Logistic_10PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.10PCs.assoc.logistic.QQplot.jpeg
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.10PCs.maf05.assoc.logistic Pheno1_justWhite_Logistic_maf05_10PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.10PCs.maf05.assoc.logistic.QQplot.jpeg
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno2.assoc.logistic Pheno2_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno2.assoc.logistic.QQplot.jpeg
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno3.assoc.logistic Pheno3_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno3.assoc.logistic.QQplot.jpeg
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno4.assoc.logistic Pheno4_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno4.assoc.logistic.QQplot.jpeg

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/.

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ print $1 }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.FIDs 
~~~
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.FIDs | wc
747     747   12415
[  michaelt@node18  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.FIDs | wc
756     756   12567
~~~
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ if ($3 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroNeg.FIDs 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ if ($4 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroHighlyExposedNeg.FIDs 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ if ($3 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroPos.FIDs 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ if ($5 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justRapidVRapid.FIDs 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ if ($5 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSlowVSlow.FIDs 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ if ($6 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVeryRapid.FIDs 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ if ($6 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVerySlow.FIDs 

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80 &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroNeg.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroHighlyExposedNeg.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroPos.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justRapidVRapid.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSlowVSlow.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVeryRapid.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVerySlow.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80 &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroHighlyExposedNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroPos.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justRapidVRapid.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSlowVSlow.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVeryRapid.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVerySlow.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow & 

paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow.frq.count | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.AllPhenos.frq.AndCount.gz
gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow.frq.count 
#gunzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow.frq.count.gz 

#Technically I cannot create the first file below because frequencies are only for the QC'ed, white subset individuals
##python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.AllPhenos.frq.AndCount.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 30; $i <= 40; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs6.txt.gz 
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.AllPhenos.frq.AndCount.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 30; $i <= 40; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz 

~~~
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v \# | wc
85200 64411200 1485394397
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | wc
87510   97379 38910992
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/\_2/) { print join("\t", @F); }' | wc
2272   74181 1012838
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/\_3/) { print join("\t", @F); }' | wc
35    1133   15436
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/\_4/) { print join("\t", @F); }' | wc
2      64     844
[  michaelt@node7  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/\_5/) { print join("\t", @F); }' | wc
1      32     412
~~~

R -q -e 'val1 = 2272 + 35 + 2 + 1; print (val1);'
> val1 = 2272 + 35 + 2 + 1; print (val1);
[1] 2310
>
>


R -q -e 'val1 = 2272 + 35 + 2 + 1; print(val1); val2 = 87510 - 2310; print(val2);'
> val1 = 2272 + 35 + 2 + 1; print(val1); val2 = 87510 - 2310; print(val2);
[1] 2310
[1] 85200
>
>

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --make-bed --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80 

##python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs3.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs6.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs3.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing") || ($7 == "UTR5;UTR3")) { print $8, "\t", $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.All
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "splicing") || ($7 == "exonic;splicing")) { print $8, "\t", $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.Exonic
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") )  { print $8, "\t", $2 } } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if ((($7 == "exonic") && (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV"))) || (($7 == "splicing") || ($7 == "exonic;splicing"))) { print $8, "\t", $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.wSplice
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($12 == "P") || ($12 == "D") ) { print $8, "\t", $2 } } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_PD
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $8, "\t", $2 } } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_D

~~~
[  michaelt@node6  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if ((($7 == "exonic") && (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV"))) || (($7 == "splicing") || ($7 == "exonic;splicing"))) { print $9 } } ' | sort | uniq -c
67
5296 nonsynonymous_SNV
119 stopgain_SNV                                                                                                                                                                                                                                                                                                       
10 stoploss_SNV
1 synonymous_SNV 
1 unknown
[  michaelt@node6  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if ((($7 == "exonic") && (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV"))) || (($7 == "splicing") || ($7 == "exonic;splicing"))) { print $7 } } ' | sort | uniq -c
5423 exonic
4 exonic;splicing
67 splicing
[  michaelt@node6  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if ((($7 == "exonic") && (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV"))) || (($7 == "splicing") || ($7 == "exonic;splicing"))) { print $0 } } ' | grep exonic\;splicing
7,rs114540180,0,150325378,T,C,exonic;splicing,GIMAP6,nonsynonymous_SNV,NM_001244072:c.A518G:p.Q173R,T,B,N,N
9,9:136521661,0,136521661,T,C,exonic;splicing,DBH,nonsynonymous_SNV,NM_000787:c.T1451C:p.L484P,T,B,D,N
15,rs75010033,0,101905203,C,T,exonic;splicing,PCSK6,unknown,UNKNOWN,,,,
20,rs139806443,0,56188385,T,C,exonic;splicing,ZBP1,synonymous_SNV,NM_001160418:c.A279G:p.E93E,,,,
~~~

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing") || ($7 == "UTR5;UTR3")) { print $0 } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.All 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "splicing") || ($7 == "exonic;splicing")) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $0 } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if ((($7 == "exonic") && (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV"))) || (($7 == "splicing") || ($7 == "exonic;splicing"))) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($12 == "P") || ($12 == "D") ) { print $0 } } } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $0 } } } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($13 == "D") { print $0 } } } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing") || ($7 == "UTR5;UTR3")) { print $0 } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.All 
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "splicing") || ($7 == "exonic;splicing")) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $0 } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if ((($7 == "exonic") && (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV"))) || (($7 == "splicing") || ($7 == "exonic;splicing"))) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.wSplices
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($12 == "P") || ($12 == "D") ) { print $0 } } } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_PD
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $0 } } } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.PP2_D
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.Sift_D
#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($13 == "D") { print $0 } } } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.NetworkKevinVs.vs1.py --file1 - --file2 /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Complexes_from_Array_1.txt > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.Complexes_from_Array_1.GroupFile.Exonic.Nonsynonymous.LRT_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.pheno | awk '{ print $1, "\t", $2, "\t0\t0\t0\t", $3, "\t", $4, "\t", $5, "\t", $6 } ' | sed 's/-9/NA/g' | perl -lane 'print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit.ped 

#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar | awk '{ print $4, "\t", $6, "\t", $7 "\t", $8, "\t", $9, "\t", $10, "\t", $11,  "\t", $12, "\t", $13, "\t", $14,  "\t", $15 }' | paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit.ped  -  | perl -lane 'print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit.ped.vs3 
join /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar | perl -lane 'splice(@F, 9, 2); splice(@F, 10, 1); print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit.ped.vs2 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit.ped.vs2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit.ped

~~~
[  michaelt@node6  ~]$cmp /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit.ped.vs3
[  michaelt@node6  ~]$                                                                                                                                                  
~~~

#Add line #FAM_ID IND_ID FAT_ID MOT_ID SEX HIVPROG HIVPROGHE AIDS AIDSEXTR RMID PC1 PC2 PC3 PC4 PC5 PC6 PC7 PC8 PC9 PC10 to /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit.ped 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | awk '{ if ($1 == "23") { $1 = "X"} print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip

/home/shared/software/tabix-0.2.5/bgzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip
/home/shared/software/tabix-0.2.5/tabix -pvcf -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz

perl /home/michaelt/Scripts/Perl/RandomList.pl --total 10000 --seed 98560234 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.PermutationList.ForP2.vs1.txt

for i in {1..1000}
do
	SEED1=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.PermutationList.ForP2.vs1.txt | awk '{ if (NR == '$i') { print $0 } } '`
	perl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs3.pl --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit.ped --seed $SEED1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit.perm${i}.vs3.ped

done

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs 

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT 
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/SUMSTAT 
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/SUMSTAT 
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/SUMSTAT 
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/SUMSTAT 
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/SUMSTAT 
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/SUMSTAT 
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/SUMSTAT 

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs TruePheno Pheno1 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs TruePheno Pheno2 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs TruePheno Pheno3 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs TruePheno Pheno4 Exonic.Nonsynonymous

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs TruePheno Pheno1 Exonic.Nonsynonymous OLD1
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs TruePheno Pheno2 Exonic.Nonsynonymous OLD1
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs TruePheno Pheno3 Exonic.Nonsynonymous OLD1
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs TruePheno Pheno4 Exonic.Nonsynonymous OLD1

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs FakePheno Pheno1 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs FakePheno Pheno2 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs FakePheno Pheno3 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs FakePheno Pheno4 Exonic.Nonsynonymous

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs FakePheno Pheno1 Exonic.Nonsynonymous OLD1
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs FakePheno Pheno2 Exonic.Nonsynonymous OLD1
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs FakePheno Pheno3 Exonic.Nonsynonymous OLD1
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs FakePheno Pheno4 Exonic.Nonsynonymous OLD1

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno1 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno2 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno3 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno4 Exonic.Nonsynonymous

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno1 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno2 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno3 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno4 Exonic.Nonsynonymous

for i in {1..1000}
do 
        if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm${i}.epacts ] ; then
		echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno1.Exonic.Nonsynonymous
	fi
        if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm${i}.epacts ] ; then
		echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno2.Exonic.Nonsynonymous
	fi
        if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm${i}.epacts ] ; then
		echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno3.Exonic.Nonsynonymous
	fi
        if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm${i}.epacts ] ; then
		echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno4.Exonic.Nonsynonymous
	fi
done

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.QQPlotUsingPermutations.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs2.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg P2_HIVAcquisition_Nonsyn_SkatO 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.QQPlotUsingPermutations.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs2.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg P2_HIVAcquisitionHE_Nonsyn_SkatO 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.QQPlotUsingPermutations.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs2.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg P2_AIDSProgression_Nonsyn_SkatO 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.QQPlotUsingPermutations.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs2.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg P2_AIDSProgressionExtr_Nonsyn_SkatO 

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/.

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/.

#mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/WRONG1.AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs
#mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/OLD1.NoRecodeInfoAll.AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs2.sh 
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs2.sh 
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs2.sh 
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs2.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs2.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno1 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs2.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno2 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs2.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno3 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs2.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno4 Exonic.Nonsynonymous

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs2.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno1 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs2.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno2 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs2.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno3 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs2.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno4 Exonic.Nonsynonymous

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Results
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Results/Checkpoints
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Results/Checkpoints/2ndRun
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Results/Checkpoints/2ndRun/20150302

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno*/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno*.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno*/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno*.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno*/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno*.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Results/Checkpoints/2ndRun/20150302/.

#Should have done this with all four phenotypes but only did with first phenotype, accidentally clobbered the remaining three. Moved all the MM5 files though correctly so can see the before/after there, and if need be can recreate the original files by using the 'gene+var groupings' based on the improper .bim.wGenes file from the wrong annovar file
#mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/WRONG1.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts

~~~
1150  cat TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ print $1 }' > jndfgondfg
1151  cat TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -vF jndfgondfg 
1152  cat TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -vF jndfgondfg  | wc
1153  cat TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -v -F jndfgondfg  
1154  cat TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -v -F jndfgondfg  | wc
1155  cat jndfgondfg | head -n 10
1156  grep --help
1157  cat TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -v -f jndfgondfg  | wc
1158  cat TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -v -f jndfgondfg
1159  cat TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output | grep CTGF
1160  cat TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output | grep LUZP6
1161  history
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations]$cat TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -v -f jndfgondfg
ID
DNAJC7
HS3ST4
AICDA
IFITM2
CALM2
SFPQ
TIMP3
RNF5
CTGF
NCS1
LUZP6
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations]$cat TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output | grep CTGF
CTGF    NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NANA       NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NANA       NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations]$cat TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output | grep LUZP6
LUZP6   NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NANA       NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NANA       NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA
~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.permAll.epacts.NullDistrQQPlotCheck.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.permAll.epacts.NullDistrQQPlotCheck.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.permAll.epacts.NullDistrQQPlotCheck.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.permAll.epacts.NullDistrQQPlotCheck.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/.

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/.

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justpVals /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justpVals /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justpVals /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justpVals /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/.

#20150330
#Retroactively including code for these 'gene/variant level result summary outputs', following the logic of excel sheet exploration, here

/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplicecat

echo ".5" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".6" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".7" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".8" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".85" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".9" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .9) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .9) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`

~~~
[  michaelt@node1  ~]$echo ".5" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.5 24 184
[  michaelt@node1  ~]$echo ".6" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'` 
.6 51 400
[  michaelt@node1  ~]$echo ".7" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'` 
.7 91 654
[  michaelt@node1  ~]$echo ".8" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'` 
.8 135 1011
[  michaelt@node1  ~]$echo ".85" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'` 
.85 188 1384
[  michaelt@node1  ~]$echo ".9" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .9) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .9) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'` 
.9 376 2763
~~~

#GetperFastQC
#GetperRegionpostQC
#GetperFullBam
#GetVCF? for all indvs, just white indvs, then subgroups
#for full file, for per gene, for per indv (and then subgroups)

#QC Stuff

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetFastqQCStats.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output

#CHECK_0: Github backup/hard-copy current QC output files for P1 and P2

#Removed empty lines in /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetFastqQCStats.vs2.20150315_234317.output with /0.00000.0000 0.0000 and dd in vi 
#PreMerge IEUL1WW01.RL4 0.00000.0000 0.0000               totalDropped(%):   minDropped(%):   UpperBoundThresh:  maxDropped(%):
#PreMerge IEUL1WW01.RL5 0.00000.0000 0.0000               totalDropped(%):   minDropped(%):   UpperBoundThresh:  maxDropped(%):
#PreMerge IEUL1WW01.RL6 0.00000.0000 0.0000               totalDropped(%):   minDropped(%):   UpperBoundThresh:  maxDropped(%):
#PreMerge IEUL1WW01.RL7 0.00000.0000 0.0000               totalDropped(%):   minDropped(%):   UpperBoundThresh:  maxDropped(%):
#PreMerge IEUL1WW01.RL9 0.00000.0000 0.0000               totalDropped(%):   minDropped(%):   UpperBoundThresh:  maxDropped(%):
#PreMerge IEUL1WW01.RL10 0.00000.0000 0.0000               totalDropped(%):   minDropped(%):   UpperBoundThresh:  maxDropped(%):
#PreMerge IEUL1WW01.RL11 0.00000.0000 0.0000               totalDropped(%):   minDropped(%):   UpperBoundThresh:  maxDropped(%):
#PreMerge IEUL1WW01.RL12 0.00000.0000 0.0000               totalDropped(%):   minDropped(%):   UpperBoundThresh:  maxDropped(%):

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetFastqQCStats.vs2.20150315_234317.output | R -q -e "Data1 <- read.table(file('stdin'), header=F); sum(as.numeric(Data1[,4])); sum(as.numeric(Data1[,6]));"
[1] 172890654704
[1] 288065985
>
>

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetFastqQCStats.vs2.20150315_234317.output | wc
   3776  109504  828231

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output.GetperRegionpostQCStats.vs3.20150316_\ 00554.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output.GetperRegionpostQCStats.vs3.20150316_00554.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output.GetperRegionpostQCStats.vs3.20150316_00554.output | perl -lane 'my @array1; push(@array1, $F[1]); foreach my $entry (@F) { if ($entry !~ m/[a-zA-Z]/) { push (@array1, $entry); } } print join(",", @array1);' | awk 'BEGIN{ print "ID,TotalReadsPreQC,TotalReadsQ10,TotalReadsQ10Prcnt,TotalReadsF4,TotalReadsF4Prcnt,TotalReadsF256,TotalReadsF256Prcnt,TotalReadsPostQC,TotalReadsDropped,TotalReadsDroppedPrcnt,AverageReadLengthPreQC,AverageReadLengthPostQC,BedFileCoveragePostQC" } { print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output.GetperRegionpostQCStats.vs3.20150316_00554.output.reformatted

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs3.ForP2.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs3.ForP2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected.GetperFullBampostQCStats.vs3.ForP2.20150316_02534.output | perl -lane 'my @array1; push(@array1, $F[1]); foreach my $entry (@F) { if ($entry !~ m/[a-zA-Z]/) { push (@array1, $entry); } } print join(",", @array1);' | awk 'BEGIN{ print "ID,NumberOfOriginalBamFiles,TotalReadsPreQC,TotalReadsPreQCForward,TotalReadsPreQCReverse,TotalReadsPreQCForward_Reverse,PreQCMAPQmean,PreQCMAPQsd,PreQCMean,PreQCSD,PreQCMax,PreQCMin,CoveragePreQC,TotalReadsPostRmdup,TotalReadsPostRmdupDropped,TotalReadsPostRmdupDroppedPercentage,TotalReadsPostQC,TotalReadsPostQCForward,TotalReadsPostQCReverse,TotalReadsPostQCForward_Reverse,PostQCMAPQmean,PostQCMAPQsd,PostQCMean,PostQCSD,PostQCMax,PostQCMin,CoveragePostQC" } { print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected.GetperFullBampostQCStats.vs3.ForP2.20150316_02534.output.reformatted 

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetFastqQCStats.vs2.20150315_234317.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output.GetperRegionpostQCStats.vs3.20150316_00554.output.reformatted /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected.GetperFullBampostQCStats.vs3.ForP2.20150316_02534.output.reformatted 

~~~
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations]$/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetFastqQCStats.vs2.20150315_234317.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output.GetperRegionpostQCStats.vs3.20150316_00554.output.reformatted /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected.GetperFullBampostQCStats.vs3.ForP2.20150316_02534.output.reformatted
[1] 6.75321
[1] 6.294703
[1] 0.2864592
[1] 0.9647855
null device
	1
[1] 254981
[1] 617.3847
[1] 239.1003
[1] 13.57618
[1] 38316.19
[1] 12.64409
null device
	1
~~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/OutputFiles/Graphics/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs2.2015-03-16_00:48:03.SummaryRegionPlots.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/OutputFiles/Graphics/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/OutputFiles/Graphics/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs1.vs2.2015-03-16_00:48:06.SummaryIndvPlots.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/OutputFiles/Graphics/.

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz 
#Edited code in /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs2.py
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.GetVCFQCStats.vs2.output

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.GetVCFQCStats.vs2.output

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz | grep exonic | grep -v ncRNA | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.GetVCFQCStats.vs2.output

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.PerGene.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.PerGene.vs2.py

#python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.PerGene.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.perGene.vs2.output
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz | grep -v intergenic | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.PerGene.vs2.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.perGene.vs2.output

#Coverage/DP is column 18

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz | grep exonic | grep -v ncRNA | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.PerGene.vs2.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.Exonic.perGene.vs2.output

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs3.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz | grep -E '^#|exonic' | grep -v ncRNA | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs3.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroNeg 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroHighlyExposedNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroHighlyExposedNeg 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroPos.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroPos 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justRapidVRapid.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justRapidVRapid 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSlowVSlow.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSlowVSlow 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVeryRapid.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justVeryRapid 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVerySlow.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justVerySlow 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroNeg 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroHighlyExposedNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroHighlyExposedNeg 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroPos.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroPos 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justRapidVRapid.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justRapidVRapid 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSlowVSlow.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSlowVSlow 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVeryRapid.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justVeryRapid 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVerySlow.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justVerySlow 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 12.8567603748327 MeanDoubletons: 21.6559571619813 MeanCommon: 6842.5890227577 MeanRare: 593.401606425703 MeanHeterozy: 4661.906291834 MeanCoverage: 12.5231536370384 sdCoverage: 4.89279817826893 Exonic: 653.61311914324 Intronic: 5008.33065595716 Nonsynonymous: 257.259705488621 Synonymous: 390.409638554217 PP2_D: 28.1954484605087 PP2_P: 18.7028112449799 Sift_D: 41.3373493975904 LRT_D: 30.6398929049531"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 12.4660766961652 MeanDoubletons: 21.7256637168142 MeanCommon: 6828.32448377581 MeanRare: 596.554572271386 MeanHeterozy: 4648.38348082596 MeanCoverage: 12.3044067539243 sdCoverage: 4.93589584565892 Exonic: 654.171091445428 Intronic: 5000.91150442478 Nonsynonymous: 257.262536873156 Synonymous: 390.896755162242 PP2_D: 28.0766961651917 PP2_P: 18.6873156342183 Sift_D: 41.7079646017699 LRT_D: 30.5486725663717"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroHighlyExposedNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 12.796178343949 MeanDoubletons: 21.0382165605096 MeanCommon: 6830.45859872611 MeanRare: 586.955414012739 MeanHeterozy: 4643.3949044586 MeanCoverage: 12.8067167758079 sdCoverage: 4.95028888995383 Exonic: 653.949044585987 Intronic: 4996.75159235669 Nonsynonymous: 257.757961783439 Synonymous: 390.235668789809 PP2_D: 27.6624203821656 PP2_P: 18.6751592356688 Sift_D: 41.5923566878981 LRT_D: 30.4458598726115"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 13.1813725490196 MeanDoubletons: 21.5980392156863 MeanCommon: 6854.44117647059 MeanRare: 590.781862745098 MeanHeterozy: 4673.14215686274 MeanCoverage: 12.7049065619788 sdCoverage: 4.85525920103541 Exonic: 653.149509803922 Intronic: 5014.49509803922 Nonsynonymous: 257.257352941176 Synonymous: 390.004901960784 PP2_D: 28.2941176470588 PP2_P: 18.7156862745098 Sift_D: 41.0294117647059 LRT_D: 30.7156862745098"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justRapidVRapid | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 12.7777777777778 MeanDoubletons: 20.3333333333333 MeanCommon: 6868.83333333333 MeanRare: 579.740740740741 MeanHeterozy: 4691.14814814815 MeanCoverage: 13.1737065707157 sdCoverage: 3.78571833702897 Exonic: 652.111111111111 Intronic: 5019.25925925926 Nonsynonymous: 255.351851851852 Synonymous: 390.666666666667 PP2_D: 27.8703703703704 PP2_P: 18.3333333333333 Sift_D: 39.9814814814815 LRT_D: 29.5740740740741"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSlowVSlow | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 13.0246913580247 MeanDoubletons: 22.6049382716049 MeanCommon: 6866.0987654321 MeanRare: 598.456790123457 MeanHeterozy: 4689.83950617284 MeanCoverage: 12.8425294439885 sdCoverage: 6.90751896295189 Exonic: 656.444444444444 Intronic: 5027.16049382716 Nonsynonymous: 259.679012345679 Synonymous: 391.074074074074 PP2_D: 28.4938271604938 PP2_P: 18.7037037037037 Sift_D: 41.8765432098765 LRT_D: 31.1604938271605"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justVeryRapid | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 14.3809523809524 MeanDoubletons: 19.6666666666667 MeanCommon: 6923.42857142857 MeanRare: 573.714285714286 MeanHeterozy: 4751.57142857143 MeanCoverage: 13.398101176621 sdCoverage: 3.24540374835087 Exonic: 651.333333333333 Intronic: 5064.47619047619 Nonsynonymous: 257.333333333333 Synonymous: 387.952380952381 PP2_D: 29 PP2_P: 18.8095238095238 Sift_D: 40.6190476190476 LRT_D: 31.4285714285714"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justVerySlow | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 13.1578947368421 MeanDoubletons: 21 MeanCommon: 6848.76315789474 MeanRare: 595.631578947368 MeanHeterozy: 4674.86842105263 MeanCoverage: 14.4278295823871 sdCoverage: 9.32553269310467 Exonic: 653.5 Intronic: 5009.71052631579 Nonsynonymous: 259.184210526316 Synonymous: 388.921052631579 PP2_D: 28.1842105263158 PP2_P: 18.4473684210526 Sift_D: 41.6052631578947 LRT_D: 30.6842105263158"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 1.95850066934404 MeanDoubletons: 3.33868808567604 MeanCommon: 591.321285140562 MeanRare: 62.3092369477912 MeanHeterozy: 403.706827309237 MeanCoverage: 14.8163023694426 sdCoverage: 5.89911671487659 Exonic: 653.61311914324 Intronic: 0 Nonsynonymous: 257.259705488621 Synonymous: 390.409638554217 PP2_D: 28.1954484605087 PP2_P: 18.7028112449799 Sift_D: 41.0883534136546 LRT_D: 30.6398929049531"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 1.87610619469027 MeanDoubletons: 3.25663716814159 MeanCommon: 591.082595870207 MeanRare: 63.1061946902655 MeanHeterozy: 404.362831858407 MeanCoverage: 14.6616951893061 sdCoverage: 5.96973879035129 Exonic: 654.171091445428 Intronic: 0 Nonsynonymous: 257.262536873156 Synonymous: 390.896755162242 PP2_D: 28.0766961651917 PP2_P: 18.6873156342183 Sift_D: 41.4660766961652 LRT_D: 30.5486725663717"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroHighlyExposedNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 1.82165605095541 MeanDoubletons: 3.18471337579618 MeanCommon: 591.808917197452 MeanRare: 62.1592356687898 MeanHeterozy: 404.146496815287 MeanCoverage: 15.3278965971024 sdCoverage: 6.0433291355643 Exonic: 653.949044585987 Intronic: 0 Nonsynonymous: 257.757961783439 Synonymous: 390.235668789809 PP2_D: 27.6624203821656 PP2_P: 18.6751592356688 Sift_D: 41.3630573248408 LRT_D: 30.4458598726115"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.02696078431373 MeanDoubletons: 3.40686274509804 MeanCommon: 591.519607843137 MeanRare: 61.6470588235294 MeanHeterozy: 403.161764705882 MeanCoverage: 14.944762747056 sdCoverage: 5.84401646067709 Exonic: 653.149509803922 Intronic: 0 Nonsynonymous: 257.257352941176 Synonymous: 390.004901960784 PP2_D: 28.2941176470588 PP2_P: 18.7156862745098 Sift_D: 40.7745098039216 LRT_D: 30.7156862745098"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justRapidVRapid | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 1.74074074074074 MeanDoubletons: 3.40740740740741 MeanCommon: 592.814814814815 MeanRare: 59.3148148148148 MeanHeterozy: 399.259259259259 MeanCoverage: 15.4838212840091 sdCoverage: 4.5829707196305 Exonic: 652.111111111111 Intronic: 0 Nonsynonymous: 255.351851851852 Synonymous: 390.666666666667 PP2_D: 27.8703703703704 PP2_P: 18.3333333333333 Sift_D: 39.6851851851852 LRT_D: 29.5740740740741"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSlowVSlow | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.08641975308642 MeanDoubletons: 3.5679012345679 MeanCommon: 593.246913580247 MeanRare: 63.2222222222222 MeanHeterozy: 406.345679012346 MeanCoverage: 15.0660673762291 sdCoverage: 8.2663373355639 Exonic: 656.444444444444 Intronic: 0 Nonsynonymous: 259.679012345679 Synonymous: 391.074074074074 PP2_D: 28.4938271604938 PP2_P: 18.7037037037037 Sift_D: 41.5185185185185 LRT_D: 31.1604938271605"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justVeryRapid | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.0952380952381 MeanDoubletons: 3.28571428571429 MeanCommon: 592.142857142857 MeanRare: 59.2380952380952 MeanHeterozy: 397.619047619048 MeanCoverage: 15.6974873657967 sdCoverage: 3.9523316947915 Exonic: 651.333333333333 Intronic: 0 Nonsynonymous: 257.333333333333 Synonymous: 387.952380952381 PP2_D: 29 PP2_P: 18.8095238095238 Sift_D: 40.1904761904762 LRT_D: 31.4285714285714"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justVerySlow | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 1.94736842105263 MeanDoubletons: 3.28947368421053 MeanCommon: 591.631578947368 MeanRare: 61.9210526315789 MeanHeterozy: 404.631578947368 MeanCoverage: 16.9681495161637 sdCoverage: 11.1593773020957 Exonic: 653.5 Intronic: 0 Nonsynonymous: 259.184210526316 Synonymous: 388.921052631579 PP2_D: 28.1842105263158 PP2_P: 18.4473684210526 Sift_D: 41.2894736842105 LRT_D: 30.6842105263158"
> 
> 


#Comparing P2 to P1
~~~
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.GetVCFQCStats.vs2.output
NumVar: 9844 Exonic: 9840 Intronic: 0 Intergenic: 0 UTR5: 0 UTR3: 0                                                                                                                                                                                                                                                        
Synonymous: 4411 Nonsynonymous: 5233 Stopgain: 116 Stoploss: 10
PP2_D: 1572 PP2_P: 842 PP2_B: 2059 PP2_-9: 467 Sift_D: 1991 Sift_T: 2691 Sift_-9: 258 LRT_D: 2050 LRT_N: 2343 LRT_U: 234 LRT_-9: 313                                                                                                                                                                                       
Common: 1144 Rare: 8700 Singletons: 1463 Doubletons: 1547
MeanQuality: 17860.6449553 QualityStd: 65869.6226881 MeanDepth: 14.9857453366 DepthStd: 4.77660674994 TiTvRatio: 4.24453915823 Transitions: 7967 Transversions: 1877                                                                                                                                                       
MeanHapScore: 1.79340668427 HapScoreStd: 1.06580996397 HapScore0Quart: 0.2499 HapScore25Quart: 1.071825 HapScore50Quart: 1.5222 HapScore75Quart: 2.202675 HapScore100Quart: 8.7767
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGenePerVariant.Exonic.GetVCFQCStats.vs1.output
NumVar: 7617 Exonic: 7614 Intronic: 0 Intergenic: 0 UTR5: 0 UTR3: 0
Synonymous: 3339 Nonsynonymous: 4083 Stopgain: 54 Stoploss: 13
PP2_D: 1129 PP2_P: 660 PP2_B: 1546 PP2_-9: 379 Sift_D: 1477 Sift_T: 1927 Sift_-9: 310 LRT_D: 1852 LRT_N: 1624 LRT_U: 96 LRT_-9: 142
Common: 588 Rare: 7029 Singletons: 2592 Doubletons: 1387
MeanQuality: 16993.8342957 QualityStd: 79340.471072 MeanDepth: 23.3232939376 DepthStd: 8.87203927559 TiTvRatio: 3.39780600462 Transitions: 5885 Transversions: 1732
MeanHapScore: 2.75564990154 HapScoreStd: 1.67229214079 HapScore0Quart: 0.181 HapScore25Quart: 1.5964 HapScore50Quart: 2.3674 HapScore75Quart: 3.4397 HapScore100Quart: 14.6402

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 1.87610619469027 MeanDoubletons: 3.25663716814159 MeanCommon: 591.082595870207 MeanRare: 63.1061946902655 MeanHeterozy: 404.362831858407 MeanCoverage: 14.6616951893061 sdCoverage: 5.96973879035129 Exonic: 654.171091445428 Intronic: 0 Nonsynonymous: 257.262536873156 Synonymous: 390.896755162242 PP2_D: 28.0766961651917 PP2_P: 18.6873156342183 Sift_D: 41.4660766961652 LRT_D: 30.5486725663717"
>
>
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output.justSeroNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.43909348441926 MeanDoubletons: 2.86685552407932 MeanCommon: 299.28045325779 MeanRare: 35.2549575070822 MeanHeterozy: 219.773371104816 MeanCoverage: 21.9810675566897 sdCoverage: 8.58968008315038 Exonic: 334.082152974504 Intronic: 0 Nonsynonymous: 122.872521246459 Synonymous: 208.801699716714 PP2_D: 9.90934844192635 PP2_P: 12.6345609065156 Sift_D: 22.9206798866856 LRT_D: 21.6345609065156"
>
>

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.02696078431373 MeanDoubletons: 3.40686274509804 MeanCommon: 591.519607843137 MeanRare: 61.6470588235294 MeanHeterozy: 403.161764705882 MeanCoverage: 14.944762747056 sdCoverage: 5.84401646067709 Exonic: 653.149509803922 Intronic: 0 Nonsynonymous: 257.257352941176 Synonymous: 390.004901960784 PP2_D: 28.2941176470588 PP2_P: 18.7156862745098 Sift_D: 40.7745098039216 LRT_D: 30.7156862745098"
>
>
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.53537735849057 MeanDoubletons: 2.54009433962264 MeanCommon: 298.695754716981 MeanRare: 34.4528301886792 MeanHeterozy: 219.007075471698 MeanCoverage: 22.9559819110576 sdCoverage: 9.25012765969131 Exonic: 332.719339622641 Intronic: 0 Nonsynonymous: 122.200471698113 Synonymous: 207.785377358491 PP2_D: 9.78301886792453 PP2_P: 12.6768867924528 Sift_D: 22.2452830188679 LRT_D: 21.8915094339623"
>
>
~~~

#20150325
#Adding new annotated .vcf format option now using vcf-annotate (http://vcftools.sourceforge.net/perl_module.html#vcf-annotate)
#Currently mainly being used to help pass on information to Sudhir for evaluation cost/design-strategy for variant follow-up genotyping
#CHECK_0: Getting vcf-annotate to work properly for the AC_... columns if this setup/format is desired to be continually used

cat <(echo "#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        REF     ALT     AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW") \
<(join <( python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | sort -k 1,1) <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^\# | perl -lane 'print join("_", @F[0..1]), "\t", $F[3], "\t", $F[4];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
perl -lane 'splice(@F, 0, 1, (((split(/_/, $F[0]))[0]), ((split(/_/, $F[0]))[1]), ((split(/_/, $F[0]))[1]))); print join("\t", @F);' |
perl -ane 'print join("\t", @F[0..8]); my $alt = $F[8]; my $totAC = join("", ((split(/:/, $F[9]))[1])) + join("", ((split(/:/, $F[10]))[1])); print "\t", $totAC; for (my $i=9; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", $F[$i]; } } print "\n";' | sort -g -k 1,1 -k 2,2 ) |
bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs1.wACs.txt.gz

cat <(echo "#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        REF     ALT     AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW") <(join <( python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | sort -k 1,1) <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^\# | perl -lane 'print join("_", @F[0..1]), "\t", $F[3], "\t", $F[4];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | perl -lane 'splice(@F, 0, 1, (((split(/_/, $F[0]))[0]), ((split(/_/, $F[0]))[1]), ((split(/_/, $F[0]))[1]))); print join("\t", @F);' | perl -ane 'print join("\t", @F[0..8]); my $alt = $F[8]; my $totAC = join("", ((split(/:/, $F[9]))[1])) + join("", ((split(/:/, $F[10]))[1])); print "\t", $totAC; for (my $i=9; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", $F[$i]; } } print "\n";' | sort -g -k 1,1 -k 2,2 ) | bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs1.wACs.txt.gz

tabix -s 1 -b 2 -e 3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs1.wACs.txt.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | vcf-annotate -a /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs1.wACs.txt.gz -d /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.PrepFor_vcfannotate.vs1.desc.txt -c CHROM,FROM,TO,INFO/LOC,INFO/GENE,INFO/VARTYPE,INFO/AACHANGE,INFO/AC_ALL,INFO/AC_TOTAL,INFO/AC_SERONEG,INFO/AC_HESERONEG,INFO/AC_SEROPOS,INFO/AC_RAPID,INFO/AC_SLOW | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.annotate.vs1.wACs.vcf.gz

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO

cat <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.annotate.vs1.wACs.vcf.gz | grep ^# | perl -lane 'if ($F[0] =~ /#CHROM/) { print join("\t", @F[0..7]); } else { print join(" ", @F); }' ) \
<(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.annotate.vs1.wACs.vcf.gz | perl -lane 'print join("\t", @F[0..7]);' | grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1; } }' | grep -vw NA ) | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq )) | \
gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.P2.Vs2.qVal_lte85.annotated.vcf.gz 

~~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1; } }'i | grep -vw NA | wc                                                                                                                                                                                                                                                         
    188     188    1154
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.annotate.vs1.wACs.vcf.gz | perl -lane 'print join("\t", @F[0..7]);' | grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1; } }' | grep -vw NA ) | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq ) | wc
   1384   11072  668030                                                                                                                                                                                                                                                                                                    
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.P2.Vs2.qVal_lte85.annotated.vcf.gz | awk '{ print $1, "\t", $2}' | grep -v ^# | wc
   1384    2768   18998
~~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.P2.Vs2.qVal_lte85.annotated.vcf.gz mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/.

cat <(echo "#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        PVAL	QVAL	REF     ALT	AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW") \
<(join <( python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | sort -k 1,1) <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^\# | perl -lane 'print join("_", @F[0..1]), "\t", $F[3], "\t", $F[4];' | sort -k 1,1) | \
perl -lane '$F[1] = join("|", split(/[\;,\,]/, $F[1])); $F[2] = join("|", split(/[\;,\,]/, $F[2])); print join("\t", @F);' | \
perl -lane 'my @geneEntries = split(/\|/, $F[2]); my @pVals; my @qVals; foreach my $gene (@geneEntries) { $gene = ((split(/\(/, $gene))[0]); my $geneEntry = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w "$gene"`; if ($geneEntry) { my @geneVals1 = split(/\s+/, $geneEntry); push(@pVals, $geneVals1[1]); push(@qVals, $geneVals1[3]); } else { push(@pVals, "NA"); push(@qVals, "NA"); } } splice(@F, 5, 0, (join("|", @pVals), join("|", @qVals))); print join("\t", @F);' | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
perl -lane 'splice(@F, 0, 1, (((split(/_/, $F[0]))[0]), ((split(/_/, $F[0]))[1]), ((split(/_/, $F[0]))[1]))); print join("\t", @F);' | \
perl -ane 'print join("\t", @F[0..10]); my $alt = $F[10]; my $totAC = join("", ((split(/:/, $F[11]))[1])) + join("", ((split(/:/, $F[12]))[1])); print "\t", $totAC; for (my $i=11; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", ((split(/\:/, $F[$i]))[1]); } } print "\n";' | sort -g -k 1,1 -k 2,2 ) | \
bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz

cat <(echo "#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        PVAL	QVAL	REF     ALT     AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW") <(join <( python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | sort -k 1,1) <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^\# | perl -lane 'print join("_", @F[0..1]), "\t", $F[3], "\t", $F[4];' | sort -k 1,1) | perl -lane '$F[1] = join("|", split(/[\;,\,]/, $F[1])); $F[2] = join("|", split(/[\;,\,]/, $F[2])); print join("\t", @F);' | perl -lane 'my @geneEntries = split(/\|/, $F[2]); my @pVals; my @qVals; foreach my $gene (@geneEntries) { $gene = ((split(/\(/, $gene))[0]); my $geneEntry = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w "$gene"`; if ($geneEntry) { my @geneVals1 = split(/\s+/, $geneEntry); push(@pVals, $geneVals1[1]); push(@qVals, $geneVals1[3]); } else { push(@pVals, "NA"); push(@qVals, "NA"); } } splice(@F, 5, 0, (join("|", @pVals), join("|", @qVals))); print join("\t", @F);' | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | perl -lane 'splice(@F, 0, 1, (((split(/_/, $F[0]))[0]), ((split(/_/, $F[0]))[1]), ((split(/_/, $F[0]))[1]))); print join("\t", @F);' | perl -ane 'print join("\t", @F[0..10]); my $alt = $F[10]; my $totAC = join("", ((split(/:/, $F[11]))[1])) + join("", ((split(/:/, $F[12]))[1])); print "\t", $totAC; for (my $i=11; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", ((split(/\:/, $F[$i]))[1]); } } print "\n";' | sort -g -k 1,1 -k 2,2 ) | /home/shared/software/tabix-0.2.5/bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz
perl -ane 'print join("\t", @F[0..8]); my $alt = $F[8]; my $totAC = join("", ((split(/:/, $F[9]))[1])) + join("", ((split(/:/, $F[10]))[1])); print "\t", $totAC; for (my $i=9; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", $F[$i]; } } print "\n";' | sort -g -k 1,1 -k 2,2 ) | /home/shared/software/tabix-0.2.5/bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz

~~~
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | head -n 10
#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        PVAL    QVAL    REF     ALT     AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW
X       2821584 2821584 downstream      ARSD    .       .       0.6429427000    0.90298 C       G       1466    2       2       0       0       0       0
X       2821782 2821782 downstream      ARSD    .       .       0.6429427000    0.90298 G       A       1478    1       0       0       1       0       0
X       2821848 2821848 downstream      ARSD    .       .       0.6429427000    0.90298 G       A       1484    3       3       3       0       0       0
X       2822103 2822103 UTR3    ARSD    .       .       0.6429427000    0.90298 C       G       1486    2       0       0       2       0       2
X       2823133 2823133 UTR3    ARSD    .       .       0.6429427000    0.90298 G       A       1486    2       1       1       1       0       0
X       2824218 2824218 UTR3    ARSD    .       .       0.6429427000    0.90298 T       C       1486    15      7       3       8       2       2
X       2825026 2825026 UTR3    ARSD    .       .       0.6429427000    0.90298 C       T       1474    5       2       0       3       0       0
X       2825083 2825083 UTR3    ARSD    .       .       0.6429427000    0.90298 G       A       1476    12      6       4       6       2       2
X       2825149 2825149 UTR3    ARSD    .       .       0.6429427000    0.90298 C       T       1474    4       2       0       2       1       0
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | perl -lane 'print $#F;' | sort | uniq -c
  85201 17
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | wc                                      
  85201 1533618 7483870
~~~

tabix -s 1 -b 2 -e 3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | vcf-annotate -a /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz -d /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.PrepFor_vcfannotate.vs2.desc.txt -c CHROM,FROM,TO,INFO/LOC,INFO/GENE,INFO/VARTYPE,INFO/AACHANGE,INFO/PVAL,INFO/QVAL,REF,ALT,INFO/AC_ALL,INFO/AC_TOTAL,INFO/AC_SERONEG,INFO/AC_HESERONEG,INFO/AC_SEROPOS,INFO/AC_RAPID,INFO/AC_SLOW | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz

~~~
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz | perl -lane 'print $#F;' | sort | uniq -c
87 0
3 1
3 10
1 109
7 11
5 12
1 13
4 14
1 16
2 2
1 20
2 24
2 3
1 38
3 4
1 41
4 5
3 6
1 69
85201 755
4 8
3 9
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | wc
85201 1533618 7483870
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz | wc                         
85340 64412794 1498116300
~~~

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.P2.Vs2.annotated.vcf.gz

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.P2.Vs2.annotated.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.ForSudhir.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/.

##
##20150825 CHECK_0: Redo certain analyses because of the single "30430   POOL87_02RL" individual. Check subset side analysis below for details regarding these revisions and what this is referring to specifically
##20150907 CHECK_0: Redo all this in P2 as well
##

#20150825 CHECK_0: Also figure out how to check alignments of reads/bam file stuff around specific/certain variants, such as those that look interesting on the lollipop charts, to make sure misaligning reads are not the cause of multiple variants popping up in localized places (and aberrantly causing a region to look interesting)
#20150825 CHECK_0: Really need to redo those subsampling-to-P2 analyses, but on the individual .bam level files first?
#20150825 CHECK_0: Do those p-value comparisons or rank comparison between top hits in my analyses and the other analyses the molecular people have done?
#20150914 CEHCK_0: Look at variants that are not present in ExAC or 1000G, but are present in my dataset, to see if there are any systematic qualities to them (poorly mapped reads, certain regions of the genome, correlation or overabundance in significant-trending genes?..correlations with gene p-values perhaps?)
#20150825 CHECK_0: Regarding the fastqc stuff -- Check this stuff?

#20150909 CHECK_0: Ask Sudhir about timetoART (i.e. is it 'time to Anti-Retroviral Therapy'?)

#See /data/userdata/pg/michaelt/Data/ALL_MAPPING/MainScript1.PG_HIVArray2.sh for analyses and creationg of new phenotype file
#From /data/userdata/pg/michaelt/Data/ALL_MAPPING/MainScript1.PG_HIVArray2.sh
~~~
...
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs2.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs3.py

#20150909 -- Changing the ...DataProcessing.Pipeline.CreatePhenotypeFile.vs3.py file to now check whether posage is >0 to determine seropositive status, and I will be doing this as the initial determining step. I will also now add an extra 'else' step that define a phenotype as 'NA' if none of the conditions are met anywhere in the rules provided

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs3.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.vs3.pheno

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/PLINKfiles/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.vs2.pheno
...
~~~

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs3.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs1.ForP2.vs2.py

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs1.pheno

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs1.ForP2.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4 }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $5 }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $6 }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $5, "\t", $6 }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $7 }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $8 }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $9 }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $10 }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $5 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $6 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $5, "\t", $6 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $7 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $8 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $9 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $10 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $5 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $6 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $5, "\t", $6 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $7 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $8 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $9 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $10 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{print $3, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $8, "\t", $9, "\t", $10 }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $8, "\t", $9, "\t", $10 } }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $8, "\t", $9, "\t", $10 } }' | sort | uniq -c

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | awk '{ print $3 }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | awk '{ print $4 }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | awk '{ print $5 }' | sort | uniq -c
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | awk '{ print $6 }' | sort | uniq -c

~~~
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | head -n 10
20446,6,50.35,POOL_P2_137_06RL,Black non-Hispanic,35.2,0,0,0,1,0,0,0,0,3.42
41011,8,19.13,POOL_P2_83_08RL,White non-Hispanic,0,0,1,0,0,0,0,0,0,0
41013,8,67.85,POOL_P2_156_08RL,White non-Hispanic,0,0,1,0,0,0,0,0,0,0
40157,9,88.62,POOL_P2_85_09RL,White Hispanic,0,1,0,0,0,0,0,0,0,0
10290,6,24.94,POOL_P2_147_06RL,Black non-Hispanic,39.3,0,0,0,1,1,1,0,0,1.62
10292,3,50.78,POOL_P2_135_03RL,White non-Hispanic,0,1,0,0,0,0,0,0,0,0
10453,3,82.8,POOL_P2_141_03RL,White non-Hispanic,28.3,0,0,0,1,0,0,0,0,7.91
10519,4,5.75,POOL_P2_157_04RL,White non-Hispanic,0,0,1,0,0,0,0,0,0,0
20501,5,56.03,POOL_P2_131_05RL,White non-Hispanic,0,1,0,0,0,0,0,0,0,0
40553,2,13.47,POOL_P2_88_02RL,White non-Hispanic,31.4,0,0,0,1,0,0,0,0,4.09
[  michaelt@login2  ~]$cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Sample_status_list_12-10-13_CT.edited.noMeta.csv | head -n 10 | perl -F\, -lane 'print join("\t", @F[10..19]);'
posage  hesn    lesn    lesc    othersc vrapid  rapid   vslow   slow    timetoART
36.4    0       0       0       1       0       0       0       0       2.13
40.2    0       0       1       1       0       1       0       0       1.25
0       0       1       0       0       0       0       0       0       0
35.6    0       0       0       0       0       0       1       1       24.03
34      0       0       0       1       0       0       0       0       2.42
45.7    0       0       1       1       0       0       0       0       1.24
0       0       1       0       0       0       0       0       0       0
47.9    0       0       0       1       1       1       0       0       3.15
0       1       0       0       0       0       0       0       0       0
[  michaelt@login2  ~]$head -n 10 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno
POOL_P2_137_06RL        POOL_P2_137_06RL        2       2       -9      -9
POOL_P2_83_08RL POOL_P2_83_08RL 1       -9      -9      -9
POOL_P2_156_08RL        POOL_P2_156_08RL        1       -9      -9      -9
POOL_P2_85_09RL POOL_P2_85_09RL 1       1       -9      -9
POOL_P2_147_06RL        POOL_P2_147_06RL        2       2       1       1
POOL_P2_135_03RL        POOL_P2_135_03RL        1       1       -9      -9
POOL_P2_141_03RL        POOL_P2_141_03RL        2       2       -9      -9
POOL_P2_157_04RL        POOL_P2_157_04RL        1       -9      -9      -9
POOL_P2_131_05RL        POOL_P2_131_05RL        1       1       -9      -9
POOL_P2_88_02RL POOL_P2_88_02RL 2       2       -9      -9
[  michaelt@login2  ~]$head -n 19 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print join("\t", @F[5..14]);' | head -n 10
35.2    0       0       0       1       0       0       0       0       3.42
0       0       1       0       0       0       0       0       0       0
0       0       1       0       0       0       0       0       0       0
0       1       0       0       0       0       0       0       0       0
39.3    0       0       0       1       1       1       0       0       1.62
0       1       0       0       0       0       0       0       0       0
28.3    0       0       0       1       0       0       0       0       7.91
0       0       1       0       0       0       0       0       0       0
0       1       0       0       0       0       0       0       0       0
31.4    0       0       0       1       0       0       0       0       4.09
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4 }' | sort | uniq -c
529 0        0
204 0        1
200 1        0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $5 }' | sort | uniq -c
346 0        0       0
183 0        0       1
204 0        1       0
200 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $6 }' | sort | uniq -c
124 0        0       0
405 0        0       1
204 0        1       0
200 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $5, "\t", $6 }' | sort | uniq -c
113 0        0       0       0
233 0        0       0       1
11 0        0       1       0
172 0        0       1       1
204 0        1       0       0
200 1        0       0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $7 }' | sort | uniq -c
504 0        0       0
25 0        0       1
204 0        1       0
200 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $8 }' | sort | uniq -c
460 0        0       0
69 0        0       1
204 0        1       0
200 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $9 }' | sort | uniq -c
460 0        0       0
69 0        0       1
204 0        1       0
200 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ print $3, "\t", $4, "\t", $10 }' | sort | uniq -c
401 0        0       0
128 0        0       1
204 0        1       0
200 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4 } }' | sort | uniq -c
529 0        0
1 1        0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $5 } }' | sort | uniq -c
346 0        0       0
183 0        0       1
1 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $6 } }' | sort | uniq -c
124 0        0       0
405 0        0       1
1 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $5, "\t", $6 } }' | sort | uniq -c
113 0        0       0       0
233 0        0       0       1
11 0        0       1       0
172 0        0       1       1
1 1        0       0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $7 } }' | sort | uniq -c
504 0        0       0
25 0        0       1
1 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $8 } }' | sort | uniq -c
460 0        0       0
69 0        0       1
1 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $9 } }' | sort | uniq -c
460 0        0       0
69 0        0       1
1 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $10 } }' | sort | uniq -c
401 0        0       0
128 0        0       1
1 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4 } }' | sort | uniq -c
527 0        0
1 1        0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $5 } }' | sort | uniq -c
344 0        0       0
183 0        0       1
1 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $6 } }' | sort | uniq -c
124 0        0       0
403 0        0       1
1 1        0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $5, "\t", $6 } }' | sort | uniq -c
113 0        0       0       0
231 0        0       0       1
11 0        0       1       0
172 0        0       1       1
1 1        0       0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{print $3, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $8, "\t", $9, "\t", $10 }' | sort | uniq -c
53 0        0       0       0       0       0       0       1
52 0        0       0       0       0       0       1       1
7 0        0       0       0       0       1       0       0
1 0        0       0       0       1       1       0       0
181 0        0       0       1       0       0       0       0
6 0        0       0       1       0       0       0       1
8 0        0       0       1       0       0       1       1
20 0        0       0       1       0       1       0       0
18 0        0       0       1       1       1       0       0
10 0        0       1       0       0       0       0       0
1 0        0       1       0       0       1       0       0
141 0        0       1       1       0       0       0       0
9 0        0       1       1       0       0       1       1
16 0        0       1       1       0       1       0       0
6 0        0       1       1       1       1       0       0
204 0        1       0       0       0       0       0       0
200 1        0       0       0       0       0       0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $8, "\t", $9, "\t", $10 } }' | sort | uniq -c
53 0        0       0       0       0       0       0       1
52 0        0       0       0       0       0       1       1
7 0        0       0       0       0       1       0       0
1 0        0       0       0       1       1       0       0
181 0        0       0       1       0       0       0       0
6 0        0       0       1       0       0       0       1
8 0        0       0       1       0       0       1       1
20 0        0       0       1       0       1       0       0
18 0        0       0       1       1       1       0       0
10 0        0       1       0       0       0       0       0
1 0        0       1       0       0       1       0       0
141 0        0       1       1       0       0       0       0
9 0        0       1       1       0       0       1       1
16 0        0       1       1       0       1       0       0
6 0        0       1       1       1       1       0       0
1 1        0       0       0       0       0       0       0
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $8, "\t", $9, "\t", $10 } }' | sort | uniq -c
53 0        0       0       0       0       0       0       1
52 0        0       0       0       0       0       1       1
7 0        0       0       0       0       1       0       0
1 0        0       0       0       1       1       0       0
179 0        0       0       1       0       0       0       0
6 0        0       0       1       0       0       0       1
8 0        0       0       1       0       0       1       1
20 0        0       0       1       0       1       0       0
18 0        0       0       1       1       1       0       0
10 0        0       1       0       0       0       0       0
1 0        0       1       0       0       1       0       0
141 0        0       1       1       0       0       0       0
9 0        0       1       1       0       0       1       1
16 0        0       1       1       0       1       0       0
6 0        0       1       1       1       1       0       0
1 1        0       0       0       0       0       0       0
[  michaelt@login2  ~]$cmp <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($2 > 0) { print $3, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $8, "\t", $9, "\t", $10 } }' | sort | uniq -c) <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | perl -F\, -lane 'print $F[0], "\t", join("\t", @F[5..14]);' | awk '{ if ($11 > 0) { print $3, "\t", $4, "\t", $5, "\t", $6, "\t", $7, "\t", $8, "\t", $9, "\t", $10 } }' | sort | uniq -c)
/dev/fd/63 /dev/fd/62 differ: byte 158, line 5
[  michaelt@login2  ~]$python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs1.ForP2.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno
[  michaelt@login2  ~]$ 
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | awk '{ print $3 }' | sort | uniq -c
403 1
530 2
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | awk '{ print $4 }' | sort | uniq -c
199 1
530 2
204 -9
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | awk '{ print $5 }' | sort | uniq -c
69 1
128 2
736 -9
[  michaelt@login2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | awk '{ print $6 }' | sort | uniq -c
25 1
69 2
839 -9
~~~























###
### Current end of main analyses
###
### Below begin specific subset side analyses
###



















##20140925
##Getting variants present in specific group of invidiuals for Kevin

#20150418
#Getting any variant whose alternative allele copies only exist in these subset of individuals. By definition then only including variants whose alternative ACs are equal to or less than 24 (b/c 12 individuals are specifically being looked at here)
#Still using /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.ReduceReads.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz file since geno80 step was done post subsetting to QCed white non-hispanic individuals
#E.g. there is no /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz -- only a ...Vs2... exists

#cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss | grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs2.20140322_POP_genomics_sample.status_list_12-10-13.output | grep -E '20572|20306|20219|20170|20123|20093|20501|20698|20132|20661|20471|20575' | awk -F, '{ print $4 }' | perl -lane 'if ($F[0] =~ m/(POOL)(\d+)-(\d+RL)/) { print "POOL_P2_", $2, "_", $3; } ')
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss | grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | grep -E '20572|20306|20219|20170|20123|20093|20501|20698|20132|20661|20471|20575' | awk -F, '{ print $4 }')

~~~
[  michaelt@node2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss | grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | grep -E '20572|20306|20219|20170|20123|20093|20501|20698|20132|20661|20471|20575' | awk -F, '{ print $4 }')
POOL_P2_102_06RL        94889   0       30      0.000316159
POOL_P2_109_07RL        94889   0       772     0.00813582
POOL_P2_109_09RL        94889   0       684     0.00720842
POOL_P2_114_02RL        94889   0       128     0.00134894
POOL_P2_131_05RL        94889   0       122     0.00128571
POOL_P2_144_09RL        94889   0       2       2.10773e-05
POOL_P2_153_07RL        94889   0       1084    0.0114239
POOL_P2_159_01RL        94889   0       97      0.00102225
POOL_P2_163_12RL        94889   0       706     0.00744027
POOL_P2_174_02RL        94889   0       51      0.00053747
POOL_P2_87_11RL 94889   0       38      0.000400468
POOL_P2_92_11RL 94889   0       208     0.00219203
~~~

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.ExtractPatientIDsFromVCF.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --IDs 20572,20306,20219,20170,20123,20093,20501,20698,20132,20661,20471,20575 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf.gz | wc

~~~
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz | wc
  95015 88153459 2022066686
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf.gz | wc
  95015 1993339 56484166
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf.gz | perl -lane 'print $#F;' | sort | uniq -c
87 0
3 1
1 10
1 109
5 11
5 12
1 13
4 14
1 16
2 2
94891 20
2 24
2 3
2 4
3 5
1 6
1 69
1 8
2 9
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf.gz | tail -n +127 | perl -lane 'print $#F;' | sort | uniq -c
  94889 20
~~~

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf.gz | perl -lane 'my @SNPinfo = split(/;/, $F[7]); my $AC; foreach my $info1 (@SNPinfo) { if ($info1 =~ /^AC\=/) { $AC = ((split(/=/, $info1))[1]); } } my $genoCount = 0; foreach (my $i = 9; $i <= $#F; $i++) { my @indvInfo = split(/\:/, $F[$i]); my @genoInfo = split(/\//, $indvInfo[0]); if ($genoInfo[0] == 1) { $genoCount++; } if ($genoInfo[1] == 1) { $genoCount++; } } if ($genoCount == $AC) { print join("\t", @F); } ' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.vcf.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.AddAnnotationsToVCF.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs6.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.wAnnotVs6.vs2.vcf.gz

~~~
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.wAnnotVs6.vs2.vcf.gz | wc                     289    4257  114803
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.vcf.gz | wc                                   284    3988  107801
~~~

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85 ) { print $1 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.qVal_lte85.GeneIDs

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.wAnnotVs6.vs2.vcf.gz | perl -lane 'my @snpInfo2 = split(/;/, $F[9]); push(@F, $snpInfo2[1]); print join("\t", @F);' | grep -w -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.qVal_lte85.GeneIDs | perl -lane 'pop(@F); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.wAnnotVs6.vs2.TopGenes_qVallte85.vcf.gz 

cat <(echo "CHR BP SNPID REF ALT LOC GENE VARTYPE AACHANGE NUMAC 20123   20219   20572   20575   20501   20132   20306   20471   20661   20170   20093   20698") <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.wAnnotVs6.vs2.TopGenes_qVallte85.vcf.gz | grep -v ^# | perl -ane 'print join("\t", @F[0..4]);
my @SNPinfo2 = split(/;/, $F[9]); for (my $j = 0; $j <= 3; $j++) { if ($SNPinfo2[$j]) { print "\t", $SNPinfo2[$j]; } else { print "\t", "."; } }
my @SNPinfo1 = split(/;/, $F[7]); my $AC; foreach my $info1 (@SNPinfo1) { if ($info1 =~ /^AC\=/) { $AC = ((split(/=/, $info1))[1]); } } print "\t", $AC;
for (my $i = 10; $i <= $#F; $i++) { my @indvInfo1 = split(/\:/, $F[$i]); print "\t", $indvInfo1[0]; }
print "\n";') > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.wAnnotVs6.vs2.TopGenes_qVallte85.ForKevin

~~~
[  michaelt@node2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.wAnnotVs6.vs2.TopGenes_qVallte85.ForKevin | perl -lane 'print $#F;' | sort | uniq -c
    34 21
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf.gz | perl -lane 'my @SNPinfo1 = split(/;/, $F[7]); my $AC; foreach my $info1 (@SNPinfo1) { if ($info1 =~ /^AC\=/) { $AC = ((split(/=/, $info1))[1]); } } if ($AC == 2) { print join("\t", @F); }' | grep \/1 | perl -lane 'my $genoCount = 0; foreach (my $i = 9; $i <= $#F; $i++) { my @indvInfo = split(/\:/, $F[$i]); my @genoInfo = split(/\//, $indvInfo[0]); if ($genoInfo[0] == 1) { $genoCount++; } if ($genoInfo[1] == 1) { $genoCount++; } } print $genoCount; ' | sort | uniq -c
    279 1
    3 2
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.vcf.gz | perl -lane 'my @SNPinfo1 = split(/;/, $F[7]); my $AC; foreach my $info1 (@SNPinfo1) { if ($info1 =~ /^AC\=/) { $AC = ((split(/=/, $info1))[1]); } } if ($AC == 2) { print join("\t", @F); }' | grep 1\/1 | wc
    2      42    1149
~~~

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.20140925KevinIndvRequest.OnlySNPsWithVarInSubset.wAnnotVs6.vs2.TopGenes_qVallte85.ForKevin /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/HESNIndvs.P2.PrivateVariants.AllVariants.TopGenes_qVal85.txt

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/HESNIndvs.P2.PrivateVariants.AllVariants.TopGenes_qVal85.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/. 


#20150427
#Specifically pulling out variant level information for SULT1C3 based on Kevin's request

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz | grep -E '^#|SULT1C3' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/P2.ForKevin.SULT1C3.ACs.txt 


scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/P2.ForKevin.SULT1C3.ACs.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO/.

~~~
[  michaelt@node2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroNeg.FIDs | wc
339     339    5617
[  michaelt@node2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroHighlyExposedNeg.FIDs | wc
157     157    2633
[  michaelt@node2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroPos.FIDs | wc
408     408    6798
[  michaelt@node2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justRapidVRapid.FIDs | wc
54      54     901
[  michaelt@node2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSlowVSlow.FIDs | wc
81      81    1368
~~~





##20141114 
#Redoing hesn stuff because of issue found previously

1160  vi /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs1.ForP2.vs1.py
#Made edit --
if (line1[7] == "1"): #Note -- 1 is highly-exposed sero-negative and 2 is sero-positive                                                                                                                                                                                                                            
	Pheno2 = "1"
elif (line1[6] == "1"):                                                                                                                                                                                                                                                                                            
	Pheno2 = "-9"

to

if (line1[6] == "1"): #Note -- 1 is highly-exposed sero-negative and 2 is sero-positive                                                                                                                                                                                                                            
	Pheno2 = "1"
elif (line1[7] == "1"):                                                                                                                                                                                                                                                                                            
	Pheno2 = "-9"

1161  cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.pheno
  1162  plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 2 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno2 &
   1163  cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ if ($4 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs
    1164  history

1167  /home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno2.assoc.logistic Pheno2_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno2.assoc.logistic.QQplot.jpeg &

1007  gunzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq.count
 1008  vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg &
  1009  vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg &
   1010  paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq.count | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.AllPhenos.frq.AndCount.gz
    1011  gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq.count

1013  python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs5.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.AllPhenos.frq.AndCount.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 30; $i <= 40; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs5.txt.gz

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.pheno | awk '{ print $1, "\t", $2, "\t0\t0\t0\t", $3, "\t", $4, "\t", $5, "\t", $6 } ' | sed 's/-9/NA/g' | perl -lane 'print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped                                                                                                                                                         

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar | awk '{ print $4, "\t", $6, "\t", $7 "\t", $8, "\t", $9, "\t", $10, "\t", $11,  "\t", $12, "\t", $13, "\t", $14,  "\t", $15 }' | paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped.vs2                                                                                                                                                                               

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped.vs2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped
																																							   #Add line #FAM_ID IND_ID FAT_ID MOT_ID SEX HIVPROG HIVPROGHE AIDS AIDSEXTR RMID PC1 PC2 PC3 PC4 PC5 PC6 PC7 PC8 PC9 PC10 to /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped

1162  bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs

#20150108 didn't actually rerun /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs1.ForP2.vs1.py code after making the edit described above.....so had to rerun my rerun of everything

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs1.ForP2.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.pheno

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno --mpheno 2 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno2 &

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno | awk '{ if ($4 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno2.assoc.logistic Pheno2_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno2.assoc.logistic.QQplot.jpeg &

gunzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq.count

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg &

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg &

paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq.count | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.AllPhenos.frq.AndCount.gz

gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.justVerySlow.frq.count

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.AllPhenos.frq.AndCount.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 30; $i <= 40; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.FreqInfo.AnnovarAnnotation.vs6.txt.gz

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.pheno | awk '{ print $1, "\t", $2, "\t0\t0\t0\t", $3, "\t", $4, "\t", $5, "\t", $6 } ' | sed 's/-9/NA/g' | perl -lane 'print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar | awk '{ print $4, "\t", $6, "\t", $7 "\t", $8, "\t", $9, "\t", $10, "\t", $11,  "\t", $12, "\t", $13, "\t", $14,  "\t", $15 }' | paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped.vs2

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped.vs2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped

#Add line #FAM_ID IND_ID FAT_ID MOT_ID SEX HIVPROG HIVPROGHE AIDS AIDSEXTR RMID PC1 PC2 PC3 PC4 PC5 PC6 PC7 PC8 PC9 PC10 to /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped

1049  ls -lrt | grep Pheno2 | grep skat0 | grep -v flat | grep .epacts$
 1050  mv AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno2.maf1.skat0.epacts HESNError.AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno2.maf1.skat0.epacts
  1051  mv AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts HESNError.AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts
   1052  mv AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno2.maf1.skat0.epacts HESNError.AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_PD.Pheno2.maf1.skat0.epacts
    1053  mv AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno2.maf1.skat0.epacts HESNError.AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.LRT_D.Pheno2.maf1.skat0.epacts
     1054  mv AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno2.maf1.skat0.epacts HESNError.AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Sift_D.Pheno2.maf1.skat0.epacts
      1055  mv AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno2.maf1.skat0.epacts HESNError.AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.PP2_D.Pheno2.maf1.skat0.epacts
       1056  mv AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno2.maf1.skat0.perm1.epacts HESNError.AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno2.maf1.skat0.perm1.epacts
        1057  mv AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno2.maf1.skat0.perm1.epacts HESNError.AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno2.maf1.skat0.perm1.epacts
	 1058  mv AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts HESNError.AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts
	  1059  ls -lrt | grep Pheno2 | grep skat0 | grep -v flat | grep .epacts$
	   1060  mv HESNError.AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno2.maf1.skat0.perm1.epacts AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno2.maf1.skat0.perm1.epacts
	    1061  mv HESNError.AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno2.maf1.skat0.perm1.epacts AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.All.Pheno2.maf1.skat0.perm1.epacts
	     1062  ls -lrt | grep Pheno2 | grep skat0 | grep -v flat | grep .epacts$ | grep maf1
	      1063  mv AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno2.maf1.skat0.epacts HESNError.AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Pheno2.maf1.skat0.epacts

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs




##20141114
##Getting Malim gene results for Steve from P2

scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/Second_capture_P2_gene-list.MalimSpecific.csv /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/.
scp -p mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/Second_capture_P2_gene-list.MalimSpecific.txt /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/.

cat /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Second_capture_P2_gene-list.MalimSpecific.noMeta.txt | sed 's/\r/\n/g' | perl -lane 'print join("\t", @F);' > /home/michaelt/LabStuff/Stephe/HIVStuff/PG/Second_capture_P2_gene-list.MalimSpecific.noMeta.noMeta.txt

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.epacts | grep -f /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Second_capture_P2_gene-list.MalimSpecific.noMeta.txt | awk '{ if ($10 != "NA") { print $0 } } ' | sort -rg -k 10,10 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.epacts.MalimGenes 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts | grep -f /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Second_capture_P2_gene-list.MalimSpecific.noMeta.txt | awk '{ if ($10 != "NA") { print $0 } } ' | sort -rg -k 10,10 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts.MalimGenes 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.epacts | grep -f /home/michaelt/LabStuff/StephensLab/HIVStuff/PG/Second_capture_P2_gene-list.MalimSpecific.noMeta.txt | awk '{ if ($10 != "NA") { print $0 } } ' | sort -rg -k 10,10 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.epacts.MalimGenes 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.epacts.MalimGenes /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.epacts.MalimGenes | sort -rg -k 10,10 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); print join("\t", @F); }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/ForSteve.MalimGenes.HIVAcqui.Nonsynonymous.skat0.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts.MalimGenes /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.epacts.MalimGenes | sort -rg -k 10,10 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); print join("\t", @F); }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/ForSteve.MalimGenes.HIVAcquiHighlyExposed.Nonsynonymous.skat0.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.epacts.MalimGenes /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/EPACTSFiles/AllPools.Vs2.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.epacts.MalimGenes | sort -rg -k 10,10 | R -q -e "library(qvalue); Data1 <- read.table(file('stdin'), header=F); Data1New <- cbind(Data1, signif(qvalue(Data1[,10])\$qvalues, 5)); write.table(Data1New, file="", quote=FALSE, row.name=FALSE, col.name=FALSE);" | perl -lane 'if ($F[0] !~ /\>/) { ($F[$#F], $F[$#F-1]) = ($F[$#F-1], $F[$#F]); print join("\t", @F); }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/ForSteve.MalimGenes.AIDSProg.Nonsynonymous.skat0.output

#Added in header for above 3 'MalimGenes' file results from both P2 and MM5
#chrom  BEGIN   END     MARKER_ID       NS      FRAC_WITH_RARE  NUM_ALL_VARS    NUM_PASS_VARS   NUM_SING_VARS   PVALUE	QVALUE  STATRHO

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/ForSteve.MalimGenes* mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/.

[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/ForSteve.MalimGenes.HIVAcqui.Nonsynonymous.skat0.output | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' | sort | uniq -c | awk '{ if ($1 > 1) { print $0 } } '
	2 APOBEC3A
	2 CBFB
	2 CUL5
	2 DDB1
	2 GIMAP1-GIMAP5
	2 GIMAP5
	2 HEATR1
	2 LGALS3BP
	2 PARP9
	2 TCEB2
	2 TNPO3
	2 VPRBP
	2 ZEB2

[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/ForSteve.MalimGenes.HIVAcquiHighlyExposed.Nonsynonymous.skat0.output | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' | sort | uniq -c | awk '{ if ($1 > 1) { print $0 } } '
	2 APOBEC3A
	2 CBFB
	2 CUL5
	2 DDB1
	2 GIMAP1-GIMAP5
	2 GIMAP5
	2 HEATR1
	2 LGALS3BP
	2 PARP9
	2 TCEB2
	2 TNPO3
	2 VPRBP
	2 ZEB2

[  michaelt@node3  /data/userdata/pg/michaelt/Data/ALL_MAPPING]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/ForSteve.MalimGenes.AIDSProg.Nonsynonymous.skat0.output | perl -lane 'my @vals1 = split(/_/, $F[3]); print $vals1[1];' | sort | uniq -c | awk '{ if ($1 > 1) { print $0 } } '
	2 APOBEC3A
	2 CUL5
	2 DDB1
	2 GIMAP1-GIMAP5
	2 GIMAP5
	2 HEATR1
	2 LGALS3BP
	2 PARP9
	2 TCEB2
	2 TNPO3
	2 ZEB2


##20150106
##Creating code/scripts to get list of individuals per variant that contain the alternative allele for that variant, given a list of specific variants

#Make note about doing the grep nonsynonymous vs. grep -E 'nonsynonymous|stop' thing with the wAnnovar files matching up to the EPACTS # of variants/passed variants results

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs

#python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.vcf.gz 
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz | grep -E '^#|XAF1' | grep -E '^#|nonsynonymous|stop' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.vs2.XAF1.Exonic.Nonsyn.vcf.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.vs2.XAF1.Exonic.Nonsyn.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.vs1.XAF1.Exonic.Nonsyn.output 

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.vs1.XAF1.Exonic.Nonsyn.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs1.XAF1.Exonic.Nonsyn.output

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.vs1.XAF1.Exonic.Nonsyn.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/MACIDs.XAF1.Exonic.Nonsyn.vcf

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs1.XAF1.Exonic.Nonsyn.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/MACIDs.XAF1.Exonic.Nonsyn.condensed

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/MACIDs.XAF1.Exonic.Nonsyn.vcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/MACIDs.XAF1.Exonic.Nonsyn.condensed mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/. 

#20150108 -- realized the below issues with HESN while doing the 'SpecificGeneVCFs' analyses so leaving this code here, but after I ran these lines I redid the code again 'officially' up above
 911  python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreatePhenotypeFile.vs1.ForP2.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.pheno
   912  paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.pheno | vi -
     914  cat AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.pheno | head -n 20
       915  cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.pheno | head -n 20
         921  cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.pheno | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.pheno
	   923  cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.pheno | wc
	     925  paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.pheno | vi -
	       926  cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.pheno  | awk '{ print $1, "\t", $2, "\t0\t0\t0\t", $3, "\t", $4, "\t", $5, "\t", $6 } ' | sed 's/-9/NA/g' | perl -lane 'print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.EPACTSedit.pheno
	         927  cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.pheno  | awk '{ print $1, "\t", $2, "\t0\t0\t0\t", $3, "\t", $4, "\t", $5, "\t", $6 } ' | sed 's/-9/NA/g' | perl -lane 'print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.EPACTSedit.ped
		   928  rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.EPACTSedit.pheno
		     929  cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar | awk '{ print $4, "\t", $6, "\t", $7 "\t", $8, "\t", $9, "\t", $10, "\t", $11,  "\t", $12, "\t", $13, "\t", $14,  "\t", $15 }' | paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.EPACTSedit.ped - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.EPACTSedit.ped.vs2
		       930  mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.EPACTSedit.ped.vs2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.EPACTSedit.ped
1188  bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.HESNUpdate.justWhite.QCed.DropIBD.EPACTS.RMID5PCs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.pheno | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.TopDuplIndvs.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.pheno 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.pheno  | awk '{ print $1, "\t", $2, "\t0\t0\t0\t", $3, "\t", $4, "\t", $5, "\t", $6 } ' | sed 's/-9/NA/g' | perl -lane 'print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.EPACTSedit.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.covar | awk '{ print $4, "\t", $6, "\t", $7 "\t", $8, "\t", $9, "\t", $10, "\t", $11,  "\t", $12, "\t", $13, "\t", $14,  "\t", $15 }' | paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.EPACTSedit.ped - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.EPACTSedit.ped.vs2 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.EPACTSedit.ped.vs2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.EPACTSedit.ped

#Add line #FAM_ID IND_ID FAT_ID MOT_ID SEX HIVPROG HIVPROGHE AIDS AIDSEXTR RMID PC1 PC2 PC3 PC4 PC5 PC6 PC7 PC8 PC9 PC10 to /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.EPACTSedit.ped

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.HESNUpdate.justWhite.QCed.DropIBD.EPACTS.RMID5PCs
#20150109

#20150129 -- it appears that one -rw-rw-r-- 1 michaelt pg    20 Jan 22 23:24 AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.XAF1.Exonic.Nonsyn.vcf.gz a number of files were '/dev/null/'ed or something, so to say, because files that were used on or around Jan 22 (2015) were similarly 'wiped' at this time point of 23:24, e.g.

~~~
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK]$ls -lrt Analyses/SpecificGeneVCFs/.
total 768                                                                                                                                                                                                                                                                                                                  
lrwxrwxrwx 1 michaelt pg 216 Jan  8 15:55 MACIDs.XAF1.Exonic.Nonsyn.vcf -> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.vs1.XAF1.Exonic.Nonsyn.output
lrwxrwxrwx 1 michaelt pg 225 Jan  8 15:55 MACIDs.XAF1.Exonic.Nonsyn.condensed -> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs1.XAF1.Exonic.Nonsyn.output         
-rw-rw-r-- 1 michaelt pg  20 Jan 22 23:24 AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.XAF1.Exonic.Nonsyn.vcf.gz
-rw-rw-r-- 1 michaelt pg  20 Jan 22 23:24 AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.wAnnovarForQC.AllGenesPerVariant.vs2.XAF1.Exonic.Nonsyn.vcf.gz                                                                      
-rw-rw-r-- 1 michaelt pg   0 Jan 22 23:24 DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.vs1.XAF1.Exonic.Nonsyn.output
-rw-rw-r-- 1 michaelt pg   0 Jan 22 23:24 DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs1.XAF1.Exonic.Nonsyn.output           

[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK]$ls -lrt PLINKfiles/.
total 216832
-rw-rw-r-- 1 michaelt pg     40309 Jan  8 18:44 AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.pheno                                                                                                                                                                                                                            
-rw-rw-r-- 1 michaelt pg     32680 Jan  8 22:35 AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.pheno
-rw-rw-r-- 1 michaelt pg    109260 Jan  8 22:46 AllPools.P2.Vs1.AllPoolsMerged.HESNupdate.justWhite.QCed.DropIBD.EPACTSedit.ped                                                                                                                                                                                            
-rw-rw-r-- 1 michaelt pg     30404 Jan  9 13:54 AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno2.nosex
-rw-rw-r-- 1 michaelt pg   8404937 Jan  9 13:54 AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno2.assoc.logistic                                                                      
-rw-rw-r-- 1 michaelt pg      6263 Jan  9 13:54 AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno2.log
-rw-rw-r-- 1 michaelt pg    157039 Jan  9 13:59 AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.CEUCHBYRI.10PCs.justWhite.QCed.DropIBD.5PCs.maf05.pheno2.assoc.logistic.QQplot.jpeg                                                          
-rw-rw-r-- 1 michaelt pg         0 Jan 22 23:24 AllPools.P2.Vs1.AllPoolsMerged.pheno
-rw-rw-r-- 1 michaelt pg         0 Jan 22 23:24 AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.pheno                                                                                                                                                                                                                
-rw-rw-r-- 1 michaelt pg         0 Jan 22 23:24 AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.Indv99.geno95.hwe1e4.wHapMap3Data.justWhite.QCed.DropIBD.pheno.justSeroHighlyExposedNeg.FIDs
-rw-rw-r-- 1 michaelt pg         0 Jan 22 23:24 AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped                                                                                                                                                                                                       
-rw-rw-r-- 1 michaelt pg         0 Jan 22 23:24 AllPools.P2.Vs1.AllPoolsMerged.justWhite.QCed.DropIBD.EPACTSedit.ped.vs2
~~~

#etc...
#so just reran them on 20150129 -- doesn't appear to be many and saw this in MM5 folder in a few files but didn't realize it had happened in P2 too, and a bit more extensively

#20150223
#Rerunning the 'grep XAF1 individuals' specifically again with the 2nd run of the data

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz | grep -E '^#|XAF1' | grep -E '^#|nonsynonymous|stop|splicing' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.XAF1.Exonic.Nonsyn.vcf.gz

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.XAF1.Exonic.Nonsyn.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.vs1.AllPools.P2.Vs2.XAF1.Exonic.Nonsyn.output

#Including PrintLine1.extend([",".join(line1[8].split(",")[0:3])]) line to show information about specific variant along with the individual IDs per variant
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs2.py

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.vs1.AllPools.P2.Vs2.XAF1.Exonic.Nonsyn.output > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs2.AllPools.P2.Vs2.XAF1.Exonic.Nonsyn.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs2.AllPools.P2.Vs2.XAF1.Exonic.Nonsyn.output | perl -ane 'my @vals1 = split(/,/, $F[2]); foreach my $indv1 (@vals1) { print $indv1, "\n"; } ;' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs2.AllPools.P2.Vs2.XAF1.Exonic.Nonsyn.output.IIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.CreateProperIndvPhenoLinkFile.vs3.ForP2.vs1.20140529_P2_master_pool_sheet.status_list_12-10-13.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs2.AllPools.P2.Vs2.XAF1.Exonic.Nonsyn.output.IIDs  

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs2.AllPools.P2.Vs2.XAF1.Exonic.Nonsyn.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/MACIDs.Vs2.XAF1.Exonic.Nonsyn.condensed

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/DataProcessing.Pipeline.Utility.GrabIndvsWithAltACofSpecificVariants.FollowUp.vs2.AllPools.P2.Vs2.XAF1.Exonic.Nonsyn.output.IIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/MACIDs.Vs2.XAF1.Exonic.Nonsyn.condensed.IIDs

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/MACIDs.Vs2.XAF1.Exonic.Nonsyn.condensed /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/MACIDs.Vs2.XAF1.Exonic.Nonsyn.condensed.IIDs mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/SpecificGeneVCFs/.



##20150318
##Adding up and/or binning rho values at different cutoffs/thresholds to just get an idea if there's any information on top gene architecture in relation to HIV-infection                                                                                                                                                 


cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane '$hash1{$F[10]}++; if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };' 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .3) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'


~~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarC
ounts | perl -lane '$hash1{$F[10]}++; if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'                                                                                                                                                                                                 
0.09    11
0.25    20                                                                                                                                                                                                                                                                                                                 
1       354
0.01    4                                                                                                                                                                                                                                                                                                                  
0       243
0.04    8                                                                                                                                                                                                                                                                                                                  
0.5     28
PVALUE  1                                                                                                                                                                                                                                                                                                                  
NA      128
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .3) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
NA      11
PVALUE  1
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .4) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
1       3
0.01    1
0.09    1
0.04    1
NA      11
PVALUE  1
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .5) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.09    2
0.25    1
1       9
0.01    2
0       2
0.04    3
0.5     3
PVALUE  1
NA      13
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .6) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.09    5
0.25    6
1       16
0.01    3
0       8
0.04    4
0.5     4
PVALUE  1
NA      16
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .7) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.09    7
0.25    9
1       31
0.01    3
0       22
0.04    6
0.5     6
PVALUE  1
NA      18
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .8) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.09    8
0.25    11
1       47
0.01    4
0       40
0.04    6
0.5     7
PVALUE  1
NA      23
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .85) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.09    10
0.25    13
1       64
0.01    4
0       55
0.04    7
0.5     11
PVALUE  1
NA      35
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .9) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.09    10
0.25    20
1       147
0.01    4
0       113
0.04    8
0.5     26
PVALUE  1
NA      59
~~~

#Pheno2
~~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane '$hash1{$F[10]}++; if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.09    7
0.25    7
1       300
0.01    5
0       318
0.04    9
0.5     19
PVALUE  1
NA      131
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .4) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.5     1
NA      11
PVALUE  1
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .5) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.5     1
NA      11
PVALUE  1
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .6) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.25    1
1       7
0.01    2
0       12
0.04    2
0.5     1
PVALUE  1
NA      11
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .7) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.25    1
1       7
0.01    2
0       12
0.04    2
0.5     1
PVALUE  1
NA      12
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .8) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.25    2
1       20
0.01    4
0       17
0.04    3
0.5     1
PVALUE  1
NA      22
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .85) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.25    2
1       21
0.01    4
0       17
0.04    3
0.5     1
PVALUE  1
NA      22
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .9) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.09    2
0.25    3
1       26
0.01    5
0       24
0.04    3
0.5     1
PVALUE  1
NA      23
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .95) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.09    7
0.25    7
1       271
0.01    5
0       256
0.04    9
0.5     19
PVALUE  1
NA      101
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .93) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.09    7
0.25    7
1       260
0.01    5
0       246
0.04    9
0.5     19
PVALUE  1
NA      100
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane ' if ($F[3] <= .92) { $hash1{$F[10]}++; } if(eof()) { foreach my $key1 (keys %hash1) { print $key1, "\t", $hash1{$key1}; } };'
0.09    4
0.25    7
1       202
0.01    5
0       160
0.04    6
0.5     19
PVALUE  1
NA      57
~~~






##20150320
##Looking for APOE on P2 for Sudhir

#From P2
~~~
[  michaelt@login2  ~]$vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --chr 19 --from-bp 45409039 --to-bp 45412650 --recode --recode-INFO-all --out nasasnoas.vcf

VCFtools - v0.1.11
(C) Adam Auton 2009

Parameters as interpreted:
--gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz
--chr 19
--to-bp 45412650
--recode-INFO-all
--out nasasnoas.vcf
--recode
--from-bp 45409039

Using zlib version: 1.2.3
Versions of zlib >= 1.2.4 will be *much* faster when reading zipped VCF files.
Reading Index file.
File contains 85200 entries and 747 individuals.
Filtering by chromosome.
Chromosome: 1
Chromosome: 2
Chromosome: 3
Chromosome: 4
Chromosome: 5
Chromosome: 6
Chromosome: 7
Chromosome: 8
Chromosome: 9
Chromosome: 10
Chromosome: 11
Chromosome: 12
Chromosome: 13
Chromosome: 14
Chromosome: 15
Chromosome: 16
Chromosome: 17
Chromosome: 18
Chromosome: 19
Chromosome: 20
Skipping Remainder.
Keeping 3459 entries on specified chromosomes.
Applying Required Filters.
Filtering sites by chromosome and/or position
After filtering, kept 747 out of 747 Individuals
After filtering, kept 13 out of a possible 3459 Sites
Outputting VCF file... Done
Run Time = 17.00 seconds
~~~

#From MM5
~~~
[  michaelt@node1  ~]$vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --chr 19 --from-bp 45409039 --to-bp 45492650 --recode --recode-INFO-all --out nasasnoas_MM5_vs2

VCFtools - v0.1.11
(C) Adam Auton 2009

Parameters as interpreted:
--gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz
--chr 19
--to-bp 45492650
--recode-INFO-all
--out nasasnoas_MM5_vs2
--recode
--from-bp 45409039

Using zlib version: 1.2.3
Versions of zlib >= 1.2.4 will be *much* faster when reading zipped VCF files.
Reading Index file.
File contains 63863 entries and 775 individuals.
Filtering by chromosome.
Chromosome: 1
Chromosome: 2
Chromosome: 3
Chromosome: 4
Chromosome: 5
Chromosome: 6
Chromosome: 7
Chromosome: 8
Chromosome: 9
Chromosome: 10
Chromosome: 11
Chromosome: 12
Chromosome: 13
Chromosome: 14
Chromosome: 15
Chromosome: 16
Chromosome: 17
Chromosome: 18
Chromosome: 19
Chromosome: 20
Skipping Remainder.
Keeping 2467 entries on specified chromosomes.
Applying Required Filters.
Filtering sites by chromosome and/or position
After filtering, kept 775 out of 775 Individuals
After filtering, kept 0 out of a possible 2467 Sites
Error:No data left for analysis!
~~~




##20150315
##Downsampling MM5 array to allSample.bam wc level comparable to P2 and rerunning GATK protocol as a real rough way to see if 'lower coverage' would affect the results we see in MM5 vs. P2
##Should also do QC level things on P2 to see singleton, doubleton calls, SFS, compared to MM5 results
##Downsampling randomly from allSample.bam file as a real, real rough first-attempt at this. Making assumption that 'all reads across all idnividuals' are more or less uniformly distributed so can just remove a 'proportion' of them from the overall .bam file to recapitulate a similar level of coverage between MM5 and P2

samtools view /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/AllPools.P2.Vs1.AllPoolsMerged.bam | wc
234582491 4458515601 618142147413

samtools flagstat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/AllPools.P2.Vs1.AllPoolsMerged.bam
234582491 + 0 in total (QC-passed reads + QC-failed reads)
0 + 0 duplicates
234582491 + 0 mapped (100.00%:-nan%)
0 + 0 paired in sequencing
0 + 0 read1
0 + 0 read2
0 + 0 properly paired (-nan%:-nan%)
0 + 0 with itself and mate mapped
0 + 0 singletons (-nan%:-nan%)
0 + 0 with mate mapped to a different chr
0 + 0 with mate mapped to a different chr (mapQ>=5)



##20150324
##Downloading and comparing overlap between ExAC data and my own for QC sake
##Data/ExAC data overlap QC analysis

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons 

python /home/michaelt/Scripts/Python/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --file2 /home/michaelt/Data/ExAC/ExAC.r0.3.sites.vep.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.gz 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.gz | grep VCF1 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF1.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.gz | grep VCF3a | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.gz | grep VCF3b | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3b.gz 

[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF1.gz | wc
69426 52555482 1206293273
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a.gz | wc
15774 11940918 279542898
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3b.gz | wc
15774  141966 49346091

paste <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a.gz | perl -ane 'print $F[1], "_", $F[2], "\t"; my @vals1 = split(/\;/, $F[8]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { print $entry1; } } print "\n";') <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3b.gz | perl -ane 'print $F[1], "_", $F[2], "\t"; my @vals1 = split(/\;/, $F[8]); my $AC_NFE; my $AN_NFE; foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AC_NFE=/) { $AC_NFE = (split(/,/, (split(/=/, $entry1))[1]))[0]; } if ($entry1 =~ m/^AN_NFE=/) { $AN_NFE = (split(/=/, $entry1))[1]; } } if (($AC_NFE) && ($AN_NFE)) { print "AF_NFE=", $AC_NFE/$AN_NFE; } else { print "NA"} print "\n";') > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs | grep -v NA | perl -lane '$F[1] = (split(/=/, $F[1]))[1]; $F[3] = (split(/=/, $F[3]))[1]; print join("\t", @F);' | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs.jpeg\", width=4000, height=5000, res=300); par(mfrow=c(3,2)); Data2 <- Data1[,1:2]; head(Data1); head(Data2); Data3a <- Data1[,3:4]; head(Data3a); Data3b <- merge(Data2, Data3a, by.x='V1', by.y='V3'); colnames(Data3b) <- c(\"Pos\", \"HIV\", \"ExAC\"); head(Data3b); plot(Data3b[,2], Data3b[,3], xlab=\"P2_HIV AF\", ylab=\"ExAC_NFE AF\", main=\"P2_HIV vs. ExAC_NFE AF\"); abline(a=0,b=1); plot(-log10(Data3b[,2]), -log10(Data3b[,3]), xlab=\"P2_HIV -log10(AF)\", ylab=\"ExAC_NFE -log10(AF)\", main=\"P2_HIV vs. ExAC_NFE -log10(AF)\"); abline(a=0,b=1); hist(Data3b[,2], xlab=\"P2_HIV AF\", main=\"P2_HIV AF Histogram\", breaks=100); hist(-log10(Data3b[,2]), xlab=\"P2_HIV -log10(AF)\", main=\"P2_HIV -log10(AF) Histogram\", breaks=100); hist(Data3b[,3], xlab=\"ExAC_NFE AF\", main=\"ExAC_NFE AF Histogram\", breaks=100); hist(-log10(Data3b[,3]), xlab=\"ExAC_NFE -log10(AF)\", main=\"ExAC_NFE -log10(AF) Histogram\", breaks=100); dev.off()"

~~
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs | grep -v NA | perl -lane '$F[1] = (split(/=/, $F[1]))[1]; $F[3] = (split(/=/, $F[3]))[1]; print join("\t", @F);' | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs.jpeg\", width=4000, height=5000, res=300); par(mfrow=c(3,2)); Data2 <- Data1[,1:2]; head(Data1); head(Data2); Data3a <- Data1[,3:4]; head(Data3a); Data3b <- merge(Data2, Data3a, by.x='V1', by.y='V3'); colnames(Data3b) <- c(\"Pos\", \"HIV\", \"ExAC\"); head(Data3b); plot(Data3b[,2], Data3b[,3], xlab=\"P2_HIV AF\", ylab=\"ExAC_NFE AF\", main=\"P2_HIV vs. ExAC_NFE AF\"); abline(a=0,b=1); plot(-log10(Data3b[,2]), -log10(Data3b[,3]), xlab=\"P2_HIV -log10(AF)\", ylab=\"ExAC_NFE -log10(AF)\", main=\"P2_HIV vs. ExAC_NFE -log10(AF)\"); abline(a=0,b=1); hist(Data3b[,2], xlab=\"P2_HIV AF\", main=\"P2_HIV AF Histogram\", breaks=100); hist(-log10(Data3b[,2]), xlab=\"P2_HIV -log10(AF)\", main=\"P2_HIV -log10(AF) Histogram\", breaks=100); hist(Data3b[,3], xlab=\"ExAC_NFE AF\", main=\"ExAC_NFE AF Histogram\", breaks=100); hist(-log10(Data3b[,3]), xlab=\"ExAC_NFE -log10(AF)\", main=\"ExAC_NFE -log10(AF) Histogram\", breaks=100); dev.off()"
> Data1 <- read.table(file('stdin'), header=F); jpeg("/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs.jpeg", width=4000, height=5000, res=300); par(mfrow=c(3,2)); Data2 <- Data1[,1:2]; head(Data1); head(Data2); Data3a <- Data1[,3:4]; head(Data3a); Data3b <- merge(Data2, Data3a, by.x='V1', by.y='V3'); colnames(Data3b) <- c("Pos", "HIV", "ExAC"); head(Data3b); plot(Data3b[,2], Data3b[,3], xlab="P2_HIV AF", ylab="ExAC_NFE AF", main="P2_HIV vs. ExAC_NFE AF"); abline(a=0,b=1); plot(-log10(Data3b[,2]), -log10(Data3b[,3]), xlab="P2_HIV -log10(AF)", ylab="ExAC_NFE -log10(AF)", main="P2_HIV vs. ExAC_NFE -log10(AF)"); abline(a=0,b=1); hist(Data3b[,2], xlab="P2_HIV AF", main="P2_HIV AF Histogram", breaks=100); hist(-log10(Data3b[,2]), xlab="P2_HIV -log10(AF)", main="P2_HIV -log10(AF) Histogram", breaks=100); hist(Data3b[,3], xlab="ExAC_NFE AF", main="ExAC_NFE AF Histogram", breaks=100); hist(-log10(Data3b[,3]), xlab="ExAC_NFE -log10(AF)", main="ExAC_NFE -log10(AF) Histogram", breaks=100); dev.off()
	V1        V2           V3           V4
1   7_38502677 0.0022030   7_38502677 7.550133e-05
2 12_113446944 0.0005525 12_113446944 3.747452e-04
3  17_66524802 0.0033260  5_149677245 1.962227e-03
4  5_149677245 0.0038500  11_45672261 8.466093e-01
5  19_40430459 0.0027780  19_40430459 5.187658e-04
6  9_140036587 0.0016540  9_140036587 2.998950e-05
	V1        V2
1   7_38502677 0.0022030
2 12_113446944 0.0005525
3  17_66524802 0.0033260
4  5_149677245 0.0038500
5  19_40430459 0.0027780
6  9_140036587 0.0016540
	V3           V4
1   7_38502677 7.550133e-05
2 12_113446944 3.747452e-04
3  5_149677245 1.962227e-03
4  11_45672261 8.466093e-01
5  19_40430459 5.187658e-04
6  9_140036587 2.998950e-05
	Pos       HIV         ExAC
1 10_112262561 0.0016560 4.180652e-03
2 10_112262625 0.0005519 2.998501e-05
3 10_112266771 0.3130000 3.108683e-01
4 10_112266805 0.0011040 8.991728e-05
5 10_112266827 0.0038630 4.422126e-03
6 10_112269846 0.0022050 1.509525e-04
null device 
	1 
> 
> 
~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF3a3b.justAFs.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/.

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF1.gz | perl -lane 'my @vals1 = split(/\;/, $F[8]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { print ((split(/=/, $entry1))[1]); } }' | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF1.AFplots.jpeg\", width=4000, height=1750, res=300); par(mfrow=c(1,2)); hist(Data1[,1], xlab=\"P2_HIV AF\", main=\"P2_HIV AF non-ExAC Overlap\", breaks=50); hist(-log10(Data1[,1]), xlab=\"P2_HIV -log10(AF)\", main=\"P2_HIV -log10(AF) non-ExAC Overlap\", breaks=50); dev.off();"
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF1.gz | perl -lane 'my @vals1 = split(/\;/, $F[8]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AC=/) { print ((split(/=/, $entry1))[1]); } }' | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF1.ACplots.jpeg\", width=4000, height=1750, res=300); par(mfrow=c(1,2)); hist(Data1[,1], xlab=\"P2_HIV AC\", main=\"P2_HIV AC non-ExAC Overlap\", breaks=50); hist(-log10(Data1[,1]), xlab=\"P2_HIV -log10(AC)\", main=\"P2_HIV -log10(AC) non-ExAC Overlap\", breaks=50); dev.off();"

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF1.AFplots.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.ExAC_r0.3.output.VCF1.ACplots.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/.



##20150324
##Downloading and comparing overlap between 1000G data and my own for QC sake
##Data/1000G data overlap QC analysis

python /home/michaelt/Scripts/Python/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --file2 /home/michaelt/Data/1000G/VCFs/ALL.chrAll.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.gz 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.gz | grep VCF1 | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.gz | grep VCF3a | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a.gz
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.gz | fgrep VCF3b | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3b.gz

[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.gz | wc
30290 22929530 528593026
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a.gz | wc
54910 41566870 957282281
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3b.gz | wc
54910 138043740 558622312

paste <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a.gz | perl -ane 'print $F[1], "_", $F[2], "\t"; my @vals1 = split(/\;/, $F[8]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { print $entry1; } } print "\n";') <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3b.gz | perl -ane 'print $F[1], "_", $F[2], "\t"; my @vals1 = split(/\;/, $F[8]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^EUR_AF=/) { print $entry1; } } print "\n";') > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | grep -v NA | perl -lane '$F[1] = (split(/=/, $F[1]))[1]; $F[3] = (split(/,/, (split(/=/, $F[3]))[1]))[0]; print join("\t", @F);' | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs.AFplots.jpeg\", width=4000, height=5000, res=300); par(mfrow=c(3,2)); Data2 <- Data1[,1:2]; head(Data1); head(Data2); Data3a <- Data1[,3:4]; head(Data3a); Data3b <- merge(Data2, Data3a, by.x='V1', by.y='V3'); colnames(Data3b) <- c(\"Pos\", \"HIV\", \"1000G\"); head(Data3b); plot(Data3b[,2], Data3b[,3], xlab=\"P2_HIV AF\", ylab=\"1000G_EUR AF\", main=\"P2_HIV vs. 1000G_EUR AF\"); abline(a=0,b=1); plot(-log10(Data3b[,2]), -log10(Data3b[,3]), xlab=\"P2_HIV -log10(AF)\", ylab=\"1000G_EUR -log10(AF)\", main=\"P2_HIV vs. 1000G_EUR -log10(AF)\"); abline(a=0,b=1); hist(Data3b[,2], xlab=\"P2_HIV AF\", main=\"P2_HIV AF Histogram\", breaks=100); hist(-log10(Data3b[,2]), xlab=\"P2_HIV -log10(AF)\", main=\"P2_HIV -log10(AF) Histogram\", breaks=100); hist(Data3b[,3], xlab=\"1000G_EUR AF\", main=\"1000G_EUR AF Histogram\", breaks=100); hist(-log10(Data3b[,3]), xlab=\"1000G_EUR -log10(AF)\", main=\"1000G_EUR -log10(AF) Histogram\", breaks=100); dev.off()"

~~~
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs | grep -v NA | perl -lane '$F[1] = (split(/=/, $F[1]))[1]; $F[3] = (split(/,/, (split(/=/, $F[3]))[1]))[0]; print join("\t", @F);' | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs.AFplots.jpeg\", width=4000, height=5000, res=300); par(mfrow=c(3,2)); Data2 <- Data1[,1:2]; head(Data1); head(Data2); Data3a <- Data1[,3:4]; head(Data3a); Data3b <- merge(Data2, Data3a, by.x='V1', by.y='V3'); colnames(Data3b) <- c(\"Pos\", \"HIV\", \"1000G\"); head(Data3b); plot(Data3b[,2], Data3b[,3], xlab=\"P2_HIV AF\", ylab=\"1000G_EUR AF\", main=\"P2_HIV vs. 1000G_EUR AF\"); abline(a=0,b=1); plot(-log10(Data3b[,2]), -log10(Data3b[,3]), xlab=\"P2_HIV -log10(AF)\", ylab=\"1000G_EUR -log10(AF)\", main=\"P2_HIV vs. 1000G_EUR -log10(AF)\"); abline(a=0,b=1); hist(Data3b[,2], xlab=\"P2_HIV AF\", main=\"P2_HIV AF Histogram\", breaks=100); hist(-log10(Data3b[,2]), xlab=\"P2_HIV -log10(AF)\", main=\"P2_HIV -log10(AF) Histogram\", breaks=100); hist(Data3b[,3], xlab=\"1000G_EUR AF\", main=\"1000G_EUR AF Histogram\", breaks=100); hist(-log10(Data3b[,3]), xlab=\"1000G_EUR -log10(AF)\", main=\"1000G_EUR -log10(AF) Histogram\", breaks=100); dev.off()"
> Data1 <- read.table(file('stdin'), header=F); jpeg("/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs.AFplots.jpeg", width=4000, height=5000, res=300); par(mfrow=c(3,2)); Data2 <- Data1[,1:2]; head(Data1); head(Data2); Data3a <- Data1[,3:4]; head(Data3a); Data3b <- merge(Data2, Data3a, by.x='V1', by.y='V3'); colnames(Data3b) <- c("Pos", "HIV", "1000G"); head(Data3b); plot(Data3b[,2], Data3b[,3], xlab="P2_HIV AF", ylab="1000G_EUR AF", main="P2_HIV vs. 1000G_EUR AF"); abline(a=0,b=1); plot(-log10(Data3b[,2]), -log10(Data3b[,3]), xlab="P2_HIV -log10(AF)", ylab="1000G_EUR -log10(AF)", main="P2_HIV vs. 1000G_EUR -log10(AF)"); abline(a=0,b=1); hist(Data3b[,2], xlab="P2_HIV AF", main="P2_HIV AF Histogram", breaks=100); hist(-log10(Data3b[,2]), xlab="P2_HIV -log10(AF)", main="P2_HIV -log10(AF) Histogram", breaks=100); hist(Data3b[,3], xlab="1000G_EUR AF", main="1000G_EUR AF Histogram", breaks=100); hist(-log10(Data3b[,3]), xlab="1000G_EUR -log10(AF)", main="1000G_EUR -log10(AF) Histogram", breaks=100); dev.off()
	V1       V2          V3     V4
1  6_32818629 0.032000  6_32818629 0.0338
2  20_1974571 0.002778  1_79109917 0.0010
3  1_79109917 0.002193  1_25881114 0.0050
4  1_25881114 0.004391 19_50868981 0.0050
5 12_49333248 0.137000 12_49333248 0.1441
6 4_108576106 0.004400 17_66528427 0.0268
	V1       V2
1  6_32818629 0.032000
2  20_1974571 0.002778
3  1_79109917 0.002193
4  1_25881114 0.004391
5 12_49333248 0.137000
6 4_108576106 0.004400
	V3     V4
1  6_32818629 0.0338
2  1_79109917 0.0010
3  1_25881114 0.0050
4 19_50868981 0.0050
5 12_49333248 0.1441
6 17_66528427 0.0268
	Pos      HIV  1000G
1 10_112261958 0.002765 0.0050
2 10_112262017 0.018000 0.0239
3 10_112262028 0.004425 0.0080
4 10_112262086 0.003315 0.0040
5 10_112262125 0.670000 0.6998
6 10_112262561 0.001656 0.0080
null device 
	1 
> 
> 
~~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF3a3b.justAFs.AFplots.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/. 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.gz | perl -lane 'my @vals1 = split(/\;/, $F[8]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AF=/) { print ((split(/=/, $entry1))[1]); } }' | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.AFplots.jpeg\", width=4000, height=1750, res=300); par(mfrow=c(1,2)); hist(Data1[,1], xlab=\"P2_HIV AF\", main=\"P2_HIV AF non-1000G Overlap\", breaks=50); hist(-log10(Data1[,1]), xlab=\"P2_HIV -log10(AF)\", main=\"P2_HIV -log10(AF) non-1000G Overlap\", breaks=50); dev.off();"
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.gz | perl -lane 'my @vals1 = split(/\;/, $F[8]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AC=/) { print ((split(/=/, $entry1))[1]); } }' | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.ACplots.jpeg\", width=4000, height=1750, res=300); par(mfrow=c(1,2)); hist(Data1[,1], xlab=\"P2_HIV AC\", main=\"P2_HIV AC non-1000G Overlap\", breaks=50); hist(-log10(Data1[,1]), xlab=\"P2_HIV -log10(AC)\", main=\"P2_HIV -log10(AC) non-1000G Overlap\", breaks=50); dev.off(); print(c(mean(Data1[,1]),sd(Data1[,1]),min(Data1[,1]),max(Data1[,1])));"

~~~
[  michaelt@node1  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.gz | perl -lane 'my @vals1 = split(/\;/, $F[8]); foreach my $entry1 (@vals1) { if ($entry1 =~ m/^AC=/) { print ((split(/=/, $entry1))[1]); } }' | /home/shared/software/R-3.0.1/bin/R -q -e "Data1 <- read.table(file('stdin'), header=F); jpeg(\"/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.ACplots.jpeg\", width=4000, height=1750, res=300); par(mfrow=c(1,2)); hist(Data1[,1], xlab=\"P2_HIV AC\", main=\"P2_HIV AC non-1000G Overlap\", breaks=50); hist(-log10(Data1[,1]), xlab=\"P2_HIV -log10(AC)\", main=\"P2_HIV -log10(AC) non-1000G Overlap\", breaks=50); dev.off(); print(c(mean(Data1[,1]),sd(Data1[,1]),min(Data1[,1]),max(Data1[,1])));"
> Data1 <- read.table(file('stdin'), header=F); jpeg("/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.ACplots.jpeg", width=4000, height=1750, res=300); par(mfrow=c(1,2)); hist(Data1[,1], xlab="P2_HIV AC", main="P2_HIV AC non-1000G Overlap", breaks=50); hist(-log10(Data1[,1]), xlab="P2_HIV -log10(AC)", main="P2_HIV -log10(AC) non-1000G Overlap", breaks=50); dev.off(); print(c(mean(Data1[,1]),sd(Data1[,1]),min(Data1[,1]),max(Data1[,1])));
null device
	1
[1]    4.938825   41.328417    1.000000 1798.000000
>
>
~~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.AFplots.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/DataAnalysis.All.CompareTwoVCFDatasets.vs1.ForExACComp.P2_GATK_DropOffTarg_1kb_geno95_hwe1e4_wHM3_justWhite_QCed_DropIBD_indv80.1000G_phase3_shapeit2_mvncall_integrated_v5a_20130502.output.VCF1.ACplots.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/VCFComparisons/.



##20150324
##Trying to use /home/michaelt/Software/lollipops-v0.96-linux64 on top20/35 or so genes to see how things look and how they come out
##First run of lollipop graphs on top20/25 genes

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | sort -rg -k 2,2 |  head -n -12 | awk '{ print $1 }' | tail -n 35 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top35 

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/LollipopPlots

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top35 | perl -ane 'print $F[0], "\t"; for (my $i = 1; $i<=$#F; $i++) { my $chrBP1 = (split(/_/, $F[$i]))[0]; $chrBP1 =~ s/:/,/g; my @annoInfo1 = split(/,/, `zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | grep $F[0] | grep $chrBP1`); chomp(@annoInfo1); print ((split(/\./, ((split(/:/, $annoInfo1[28]))[2])))[1]); print "\t";} print "\n";' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/LollipopPlots/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top35.LollipopFormat 

OLD_IFS=$IFS
IFS=$'\n'

for command1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/LollipopPlots/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top35.LollipopFormat`
do

	IFS=$OLD_IFS

	geneID1=`echo $command1 | perl -ane 'print $F[0];'` 

#	echo $geneID1 $command1

	/home/michaelt/Software/lollipops-v0.96-linux64/lollipops -o=/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/LollipopPlots/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top35.${geneID1}.LollipopPlot.svg $command1
#	/home/michaelt/Software/lollipops-v0.96-linux64/lollipops -o=/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/LollipopPlots/P2.LollipopPlot.${geneID1}.Nonsyn.svg $command1

#	cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/LollipopPlots/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top35.${geneID1}.LollipopPlot.svg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/LollipopPlots/P2.LollipopPlot.${geneID1}.Nonsyn.svg

done

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/LollipopPlots/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.pValSort.justGeneIDs.top35.*.LollipopPlot.svg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/LollipopPlots/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/LollipopPlots/P2.LollipopPlot.*.Nonsyn.svg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/LollipopPlots/.


##20150330
##Getting 'excel sheets' prepped for sending on to Sumit/Kevin
##Also editting them to send a version to Sudhir as well

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ print $1 } ' | grep -vw NA | grep -vw NS > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs | head -n 5 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs.top5

rm -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/P2.skatO.nonsyn.qVals.ACs.Orig.txt

echo "Gene      pValue  qValue  NumVar  NumVar  NumVar  rho     AC_Total        AC_SeroNeg      AC_HESeroNeg    AC_SeroPos      AC_Rapid        AC_Slow" >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/P2.skatO.nonsyn.qVals.ACs.Orig.txt

for gene1 in `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs.top5`
do

	paste <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | perl -lane 'if ($F[0] =~ m/^'$gene1'$/) { print join("\t", @F); }' | awk '{ print $1, "\t", $2, "\t", $4, "\t", $5, "\t", $8, "\t", $9, "\t", $11 }' ) \
	<(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs1.wACs.txt.gz | \
	grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | perl -lane 'if ($F[0] =~ m/^'$gene1'$/) { print join("\t", @F); }' | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }') | \
	perl -lane 'print join("", ((split(/:/, $F[10]))[1])), "\t", ((split(/:/, $F[11]))[1]), "\t", ((split(/:/, $F[12]))[1]), "\t", ((split(/:/, $F[13]))[1]), "\t", ((split(/:/, $F[14]))[1]), "\t", ((split(/:/, $F[15]))[1]);' | \
	R -q -e "Data1 <- read.table(file('stdin'), header=F); print(paste(apply(Data1, 2, sum)), quote=FALSE);" | grep -v ^\> | perl -lane 'splice(@F, 0, 1); print join("\t", @F);') \
	>> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/P2.skatO.nonsyn.qVals.ACs.Orig.txt

done

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/P2.skatO.nonsyn.qVals.ACs.Orig.txt | perl -lane 'splice(@F, 4, 2); print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.ForSudhir.txt

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.ForSudhir.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/.

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/P2.skatO.nonsyn.qVals.ACs.Orig.txt | perl -lane 'splice(@F, 4, 3); print join("\t", @F);' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/P2.skatO.nonsyn.qVals.ACs.txt

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/P2.skatO.nonsyn.qVals.ACs.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/.

#QC Checks on file

~~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/P2.skatO.nonsyn.qVals.ACs.txt | wc        
    786    7860   35959

#Check_0: Once have annotation file working properly (e.g. Total_AC working properly), go back and check why one variant is dropping out of the analysis for this one gene here?
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/P2.skatO.nonsyn.qVals.ACs.Orig.txt | perl -lane 'if (($F[3] == $F[4]) && ($F[3] == $F[5]) && ($F[4] == $F[5])) { print join("\t", @F); } ' | wc                                   
    785   10205   41299
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/P2.skatO.nonsyn.qVals.ACs.Orig.txt | perl -lane 'if (($F[3] != $F[4]) || ($F[3] != $F[5]) || ($F[4] != $F[5])) { print join("\t", @F); } ' 
WNK1    0.4272612000    0.88854 42      42      41      0       3926    1800    826     2126    290     422

[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/P2.skatO.nonsyn.qVals.ACs.txt | perl -lane 'my $val2 = $F[5] + $F[7]; if ($F[4] != $val2) { print join("\t", @F); }'
[  michaelt@node1  ~]$
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/P2.skatO.nonsyn.qVals.ACs.txt | perl -lane 'my $val2 = $F[5] + $F[7]; if ($F[4] == $val2) { print join("\t", @F); }' | wc
    786    7860   35959

[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/P2.skatO.nonsyn.qVals.ACs.txt | perl -lane 'my $val2 = $F[5] + $F[7]; if ($F[6] > $F[5]) { print join("\t", @F); }'
[  michaelt@node1  ~]$
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/P2.skatO.nonsyn.qVals.ACs.txt | perl -lane 'my $val2 = $F[5] + $F[7]; if ($F[6] <= $F[5]) { print join("\t", @F); }' | wc                                                         
    786    7860   35959

[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/P2.skatO.nonsyn.qVals.ACs.txt | perl -lane 'my $val2 = $F[8] + $F[9]; if ($F[7] < $val2) { print join("\t", @F); }'
[  michaelt@node1  ~]$
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSumit/P2.skatO.nonsyn.qVals.ACs.txt | perl -lane 'my $val2 = $F[8] + $F[9]; if ($F[7] >= $val2) { print join("\t", @F); }' | wc
    786    7860   35959
~~~

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.ForSudhir.txt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.ForSudhir.txt



##20150423
##Working up code to run SKAT-O on msigdb pathways as a form of pathway analysis
##Also looking at prepping some of Kevin O's code/groupings to be used for this as well

cat /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt | perl -ane 'print $F[0], "\t"; my @genes; my $geneCount = 0; my @vars; my @varsPerGene; for (my $i = 2; $i <= $#F; $i++) {my $geneGrep = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -P "^$F[$i]\\s"`; if ($geneGrep) { my @geneGrepArray = split(/\s+/, $geneGrep); push(@genes, $F[$i]); $geneCount++; push(@varsPerGene, $#geneGrepArray); push(@vars, @geneGrepArray[1..$#geneGrepArray]); } } if ($geneCount == 0) { print $geneCount, "\t.\t.\t.\t.\n"; } else { print $geneCount, "\t", scalar(@vars), "\t", join(",", @varsPerGene), "\t", join(",", @genes), "\t", join("\t", @vars), "\n"; }' > /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt.P2.GroupFile.Exonic.Nonsynonymous.wSplice.pre 

cat /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt.P2.GroupFile.Exonic.Nonsynonymous.wSplice.pre | perl -lane 'if ($F[1] >= 2) { print $F[0], "\t", join("\t", @F[5..$#F]); }' > /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt.P2.GroupFile.Exonic.Nonsynonymous.wSplice

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/TruePheno/
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/TruePheno/Pheno1/
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/TruePheno/Pheno2/
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt.P2.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.msigdb_c2

P2TopPathways=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.msigdb_c2.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | sort -rg -k 10,10 | tail -n 11 | head -n 10 | awk '{ print $4 }' | perl -ane 'my @pathwayArray = split(/_/, $F[0]); if (eof()) { print join("_", @pathwayArray[1..$#pathwayArray]), "\n"; } else { print join("_", @pathwayArray[1..$#pathwayArray]), ";"; }'`

#rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.P2.Vs2.EPACTS.RMID5PCs.msigdb_c2.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneLevel.AllPathways

OLD_IFS=$IFS
IFS=$';'

for P2Pathway in $P2TopPathways
do
	IFS=$OLD_IFS
	echo $P2Pathway

#	rm /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.P2.Vs2.EPACTS.RMID5PCs.msigdb_c2.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneLevel.$P2Pathway

	echo $P2Pathway `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.msigdb_c2.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts | grep $P2Pathway | awk '{ print $10 }'` >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.P2.Vs2.EPACTS.RMID5PCs.msigdb_c2.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneLevel.AllPathways

	GenesInPathway=`cat /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt.P2.GroupFile.Exonic.Nonsynonymous.wSplice.pre | grep -w $P2Pathway | awk '{ print $5 }' | perl -lane 'print join(";", split(/,/, $F[0]));'`

	OLD_IFS=$IFS
	IFS=$';'

	for GenePathway in $GenesInPathway
	do
		IFS=$OLD_IFS

		echo $GenePathway

#		cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.ForSudhir.txt | grep -w -P "^$GenePathway\\s" >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.P2.Vs2.EPACTS.RMID5PCs.msigdb_c2.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneLevel.$P2Pathway 
		cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.ForSudhir.txt | grep -w -P "^$GenePathway\\s" >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.P2.Vs2.EPACTS.RMID5PCs.msigdb_c2.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneLevel.AllPathways

	done

	echo "" >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.P2.Vs2.EPACTS.RMID5PCs.msigdb_c2.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneLevel.AllPathways

done

~~~
[  michaelt@node2  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.P2.Vs2.EPACTS.RMID5PCs.msigdb_c2.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneLevel.* | awk '{ print $1 }' | sort | uniq -c | sort -k 1,1 | tail -n 25
1 RGS2
1 RSAD2
1 SAMD9L
1 SLC1A3
1 SLC2A3
1 SOD2
1 SPARC
1 SPP1
1 STXBP2
1 TCEB2
1 TNFSF10
1 TRA2A
1 UBE2J1
1 UBE2M
1 VPRBP
1 XAF1
2 COL1A2
2 COL6A1
2 LBH
2 MME
2 PLAU
2 PRDM1
2 PRRC2A
2 PSME4
3 ENPP2
~~~

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /home/michaelt/Data/GSEA/c2.all.v4.0.symbols.gmt.P2.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/msigdb_c2/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.msigdb_c2




##20150825
##Redoing .pheno file and specific, associated analyses, or at least creating a log here and editting them back in the main file above, because of the single indinvidual mix-up Sudhir identified
##Also going to be creating a specific script here, or at least a one-liner, to check various aspects of the file on which the phenotype codes are based on for QC/cross-checking purposes
##Going to be copying/pasting all the lines of code I am rerunning from above, so there may a large number of duplicate entries in terms of 'logging' specifically-run code. But this section will provide lines of code I did specifically rerun even if naming is same as before

#20150825 CHECK_0: Do this stuff

#From Sudhir 20150403 email chain -- redoing/fixing/correcting "30430   POOL87_02RL" indivdual -- only individual found with issue
~~
...
I checked the CAMACS_phenotypes against the master status list and found one discrepancy.

        (1)     MACSID  30430   POOL87_02RL     is listed in the CAMACS dataset as HESN and is seropositive.  I moved to seropositive in my reconstruction.

	In deidentifying the sequencing pool-midtag IDâ<80><99>s from Michaelâ<80><99>s phenotype data, I could not resolve three samples.
	...
~~

#20150825? 20150913 CHECK_1: Create phenotype file QC check script #Ran a bunch of a one-liners, which can be seen in the main log portion around this time frame, that looks at and QCs the entries both for the original phenotype file and the coded phenotype file

#Put pheno redo correction here?
#
#Have to redo group-pheno level QC stats
#Perm files
#Pathway analyses
#Mktest?
#20150825
#Current to-do list
#
#Redo MultinRun TarAndClean and PermutationRun TarAndClean


#CHECK_0: Really need to redo those subsampling-to-P2 analyses, but on the individual .bam level files first?

#20150907 -- First is to just redo all the prep code and prep files, worry about redoing analyses once all the prep work/steps have been redone properly
#Then run the base-level SKAT-O analyses, run the scripts that collect that information, then the analyses that operate on top of those results
#Note -- all the code below was determined by copying/pasting a large portion of the log from '7th run...' onwards and determining which lines needed to be rerun. Additionaly lines of code were also included, generally referring to renaming previous '...pheno' files to '...vs2.pheno' files to begin this organization system of 'vs2', 'vs3', etc...


mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs1.pheno

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.pheno

plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar --covar-number 2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.10PCs
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar --covar-number 2,4,5,6,7,8,9,10,11,12,13 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.10PCs.maf05
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno --mpheno 2 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno2
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno --mpheno 3 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno3
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno --mpheno 1 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05
plink --bfile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4 --pheno /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno --mpheno 4 --covar /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.covar --covar-number 2,4,5,6,7,8 --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar.FIDIIDs --maf .05 --logistic --allow-no-sex --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno4

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.assoc.logistic Pheno1_justWhite_Logistic_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.assoc.logistic.QQplot.jpeg
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.assoc.logistic Pheno1_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.assoc.logistic.QQplot.jpeg
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.10PCs.assoc.logistic Pheno1_justWhite_Logistic_10PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.10PCs.assoc.logistic.QQplot.jpeg
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.10PCs.maf05.assoc.logistic Pheno1_justWhite_Logistic_maf05_10PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.10PCs.maf05.assoc.logistic.QQplot.jpeg
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno2.assoc.logistic Pheno2_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno2.assoc.logistic.QQplot.jpeg
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno3.assoc.logistic Pheno3_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno3.assoc.logistic.QQplot.jpeg
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.QQPlot.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno4.assoc.logistic Pheno4_justWhite_Logistic_maf05_5PCs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.5PCs.maf05.pheno4.assoc.logistic.QQplot.jpeg

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.*jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/.

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.pheno.FIDs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroNeg.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.pheno.justSeroNeg.FIDs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroHighlyExposedNeg.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.pheno.justSeroHighlyExposedNeg.FIDs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroPos.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.pheno.justSeroPos.FIDs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justRapidVRapid.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.pheno.justRapidVRapid.FIDs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSlowVSlow.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.pheno.justSlowVSlow.FIDs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVeryRapid.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.pheno.justVeryRapid.FIDs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVerySlow.FIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.pheno.justVerySlow.FIDs

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | awk '{ print $1 }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.FIDs 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | awk '{ if ($3 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroNeg.FIDs 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | awk '{ if ($4 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroHighlyExposedNeg.FIDs 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | awk '{ if ($3 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroPos.FIDs 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | awk '{ if ($5 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justRapidVRapid.FIDs 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | awk '{ if ($5 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSlowVSlow.FIDs 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | awk '{ if ($6 == "1") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justVeryRapid.FIDs 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs1.AllPoolsMerged.vs2.pheno | awk '{ if ($6 == "2") { print $1 } }' | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.Results.pca.altrd.race.justWhiteQCed.justFIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.TopDuplIndvs.IndvsTopDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.justWhite.QCed.genome.HighZ1Countgte100.IndvsToDrop.FIDs | grep -v -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.imiss.indv80.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justVerySlow.FIDs 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.frq.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.frq.count.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.justSeroNeg.frq.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.justSeroNeg.frq.count.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.justSeroHighlyExposedNeg.frq.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.justSeroHighlyExposedNeg.frq.count.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.justSeroPos.frq.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.justSeroPos.frq.count.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.justRapidVRapid.frq.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.justRapidVRapid.frq.count.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.justSlowVSlow.frq.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.justSlowVSlow.frq.count.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.justVeryRapid.frq.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVeryRapid.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.justVeryRapid.frq.count.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.justVerySlow.frq.gz
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justVerySlow.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.justVerySlow.frq.count.gz

vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2 &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroNeg.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroNeg &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroHighlyExposedNeg.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroHighlyExposedNeg &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroPos.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroPos &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justRapidVRapid.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justRapidVRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSlowVSlow.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSlowVSlow &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justVeryRapid.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVeryRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justVerySlow.FIDs --freq --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVerySlow & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2 &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroNeg &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroHighlyExposedNeg.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroHighlyExposedNeg &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSeroPos.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroPos &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justRapidVRapid.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justRapidVRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justSlowVSlow.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSlowVSlow &
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justVeryRapid.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVeryRapid & 
vcftools --gzvcf /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.vcf.gz --keep /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.pheno.justVerySlow.FIDs --counts --out /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVerySlow & 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.AllPhenos.frq.AndCount.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.AllPhenos.frq.AndCount.gz

paste /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVerySlow.frq.count | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.AllPhenos.frq.AndCount.gz
gzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroHighlyExposedNeg.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroPos.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justRapidVRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSlowVSlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVeryRapid.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVerySlow.frq /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroHighlyExposedNeg.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroPos.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justRapidVRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSlowVSlow.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVeryRapid.frq.count /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVerySlow.frq.count 
#gunzip /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroNeg.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroHighlyExposedNeg.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroPos.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justRapidVRapid.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSlowVSlow.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVeryRapid.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVerySlow.frq.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroNeg.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroHighlyExposedNeg.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSeroPos.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justRapidVRapid.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justSlowVSlow.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVeryRapid.frq.count.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.justVerySlow.frq.count.gz 












python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.AllPhenos.frq.AndCount.gz --file3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv | sort -t, -k 1,1 -k 2,2 | sed 's/"//g' | perl -F, -lane 'for (my $i = 30; $i <= 40; $i++) { if ($F[$i] eq "NA") { $F[$i] = "-9"; } } print join(",", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz 

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v \# | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/\_2/) { print join("\t", @F); }' | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/\_3/) { print join("\t", @F); }' | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/\_4/) { print join("\t", @F); }' | wc
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/\_5/) { print join("\t", @F); }' | wc

~~~
[  michaelt@node18  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v \# | wc
85200 64411200 1485394397
[  michaelt@node18  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz | wc
87510   97379 38909004
[  michaelt@node18  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/\_2/) { print join("\t", @F); }' | wc
2272   74181 1012781
[  michaelt@node18  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/\_3/) { print join("\t", @F); }' | wc
35    1133   15428
[  michaelt@node18  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/\_4/) { print join("\t", @F); }' | wc
2      64     844
[  michaelt@node18  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz | perl -F, -lane 'if ($F[1] =~ m/\_5/) { print join("\t", @F); }' | wc
1      32     412
R -q -e 'val1 = 2272 + 35 + 2 + 1; print (val1);'
> val1 = 2272 + 35 + 2 + 1; print (val1);
[1] 2310
>
>
R -q -e 'val1 = 2272 + 35 + 2 + 1; print(val1); val2 = 87510 - 2310; print(val2);'
> val1 = 2272 + 35 + 2 + 1; print(val1); val2 = 87510 - 2310; print(val2);
[1] 2310
[1] 85200
>
>
~~~

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.bim.wGeneIDs

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.PythonVLookUp.AddAnnotationsToBimFile.vs3.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.FreqInfo.AnnovarAnnotation.vs6.txt.gz --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.All /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.bim.wGeneIDs.SetIDFile.All 
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.wSplice /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.wSplice
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_PD /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_PD
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_D /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing") || ($7 == "UTR5;UTR3")) { print $8, "\t", $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.SetIDFile.All
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "splicing") || ($7 == "exonic;splicing")) { print $8, "\t", $2 } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.SetIDFile.Exonic
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") )  { print $8, "\t", $2 } } }' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs | awk -F, '{ if ((($7 == "exonic") && (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV"))) || (($7 == "splicing") || ($7 == "exonic;splicing"))) { print $8, "\t", $2 } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.wSplice
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($12 == "P") || ($12 == "D") ) { print $8, "\t", $2 } } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_PD
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $8, "\t", $2 } } } } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/SKATFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.SetIDFile.Exonic.Nonsynonymous.PP2_D

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.All /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.bim.wGeneIDs.GroupFile.All
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.bim.wGeneIDs.GroupFile.Exonic
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs1.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "intronic") || ($7 == "splicing") || ($7 == "UTR5") || ($7 == "UTR3") || ($7 == "exonic;splicing") || ($7 == "UTR5;UTR3")) { print $0 } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.All 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "splicing") || ($7 == "exonic;splicing")) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { print $0 } } } ' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs | awk -F, '{ if ((($7 == "exonic") && (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV"))) || (($7 == "splicing") || ($7 == "exonic;splicing"))) { print $0 } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if (($12 == "P") || ($12 == "D") ) { print $0 } } } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_PD
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($12 == "D") { print $0 } } } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.PP2_D
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($11 == "D") { print $0 } } } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.Sift_D
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs | awk -F, '{ if (($7 == "exonic") || ($7 == "exonic;splicing")) { if (($9 == "nonsynonymous_SNV") || ($9 == "stoploss_SNV") || ($9 == "stopgain_SNV") ) { if ($13 == "D") { print $0 } } } }' | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/DataProcessing.Pipeline.Utility.PrepareGroupFileForEPACTS.vs1.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.LRT_D

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs1.EPACTSedit.ped

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.pheno | awk '{ print $1, "\t", $2, "\t0\t0\t0\t", $3, "\t", $4, "\t", $5, "\t", $6 } ' | sed 's/-9/NA/g' | perl -lane 'print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.EPACTSedit.ped 

join /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EigenstratFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.10PCs.justWhite.QCed.DropIBD.indv80.covar | perl -lane 'splice(@F, 9, 2); splice(@F, 10, 1); print join("\t", @F); ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.EPACTSedit.ped.vs2 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.EPACTSedit.ped.vs2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.EPACTSedit.ped

#Add line #FAM_ID IND_ID FAT_ID MOT_ID SEX HIVPROG HIVPROGHE AIDS AIDSEXTR RMID PC1 PC2 PC3 PC4 PC5 PC6 PC7 PC8 PC9 PC10 to /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.EPACTSedit.ped 

ls -lrt /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/. | awk '{ print $9 }' | perl -lane 'if ($F[0] =~ m/(AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.)(EPACTSedit.perm\d+.vs3.ped)/) { my $val1 = "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/" . $1 . $2; my $val2 = "/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/" . $1 . "vs1." . $2; system("mv $val1 $val2"); }' 

for i in {1..1500}
do
	SEED1=`cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/DataAnalysis.GATK.PermutationList.ForP2.vs1.txt | awk '{ if (NR == '$i') { print $0 } } '`
	perl /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.PermuteEPACTSpedFile.vs3.pl --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.EPACTSedit.ped --seed $SEED1 > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.EPACTSedit.perm${i}.vs3.ped

done

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.EPACTSedit.ped /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs 

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.bgzip.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.bim.wGeneIDs.GroupFile /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.justWhite.Qced.DropIBD.indv80.vs2.EPACTSedit /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vs2.EPACTS.RMID5PCs

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno1 Exonic.Nonsynonymous OLD2
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs.top5  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justGeneIDs.top5
mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.qVal_lte85.GeneIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/OLD2.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.qVal_lte85.GeneIDs
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno2 Exonic.Nonsynonymous OLD2
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno3 Exonic.Nonsynonymous OLD2
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno4 Exonic.Nonsynonymous OLD2

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno1 Exonic.Nonsynonymous OLD2
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno2 Exonic.Nonsynonymous OLD2
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno3 Exonic.Nonsynonymous OLD2
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno4 Exonic.Nonsynonymous OLD2














bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno1 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno2 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno3 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno4 Exonic.Nonsynonymous

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno1 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno2 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno3 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno4 Exonic.Nonsynonymous

for i in {1..1000}
do 
        if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.perm${i}.epacts ] ; then
		echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno1.Exonic.Nonsynonymous
	fi
        if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.perm${i}.epacts ] ; then
		echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno2.Exonic.Nonsynonymous
	fi
        if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.perm${i}.epacts ] ; then
		echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno3.Exonic.Nonsynonymous
	fi
        if [ ! -e /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.perm${i}.epacts ] ; then
		echo $i >> /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.Launcher.vs2.pt1.RedoList.FakePheno.Pheno4.Exonic.Nonsynonymous
	fi
done

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.QQPlotUsingPermutations.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs2.FakePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg P2_HIVAcquisition_Nonsyn_SkatO 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.QQPlotUsingPermutations.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs2.FakePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg P2_HIVAcquisitionHE_Nonsyn_SkatO 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.QQPlotUsingPermutations.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs2.FakePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg P2_AIDSProgression_Nonsyn_SkatO 
/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.QQPlotUsingPermutations.vs1.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerRankAcrossMultRuns.vs2.FakePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg P2_AIDSProgressionExtr_Nonsyn_SkatO 

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/.

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/.

#mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/WRONG1.AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs
#mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/OLD1.NoRecodeInfoAll.AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.TarAndCleanPrevRun.vs2.sh 
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.TarAndCleanPrevRun.vs2.sh 
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs2.sh 
cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs1.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs2.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs2.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno1 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs2.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno2 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs2.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno3 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.MultiRuns.MvAndProcessResults.vs2.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs TruePheno Pheno4 Exonic.Nonsynonymous

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs2.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno1 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs2.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno2 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs2.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno3 Exonic.Nonsynonymous
bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.pt6.All.ptf.EPACTruns.PermutationRuns.MvAndProcessResults.vs2.sh P2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/ AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs FakePheno Pheno4 Exonic.Nonsynonymous

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Results
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Results/Checkpoints
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Results/Checkpoints/2ndRun
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Results/Checkpoints/2ndRun/20150302

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno*/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno*.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno*/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno*.Exonic.Nonsynonymous.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno*/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno*.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Results/Checkpoints/2ndRun/20150302/.

#Should have done this with all four phenotypes but only did with first phenotype, accidentally clobbered the remaining three. Moved all the MM5 files though correctly so can see the before/after there, and if need be can recreate the original files by using the 'gene+var groupings' based on the improper .bim.wGenes file from the wrong annovar file
#mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/WRONG1.DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts

~~~
1150  cat TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ print $1 }' > jndfgondfg
1151  cat TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -vF jndfgondfg 
1152  cat TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -vF jndfgondfg  | wc
1153  cat TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -v -F jndfgondfg  
1154  cat TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -v -F jndfgondfg  | wc
1155  cat jndfgondfg | head -n 10
1156  grep --help
1157  cat TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -v -f jndfgondfg  | wc
1158  cat TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -v -f jndfgondfg
1159  cat TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output | grep CTGF
1160  cat TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output | grep LUZP6
1161  history
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations]$cat TruePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.run1.epacts.GeneIDList | grep -v -f jndfgondfg
ID
DNAJC7
HS3ST4
AICDA
IFITM2
CALM2
SFPQ
TIMP3
RNF5
CTGF
NCS1
LUZP6
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations]$cat TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output | grep CTGF
CTGF    NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NANA       NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NANA       NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations]$cat TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output | grep LUZP6
LUZP6   NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NANA       NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NANA       NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA      NA
~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno1.maf1.skat0.permAll.epacts.NullDistrQQPlotCheck.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno1/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno2.maf1.skat0.permAll.epacts.NullDistrQQPlotCheck.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno2/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno3.maf1.skat0.permAll.epacts.NullDistrQQPlotCheck.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno3/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.EPACTS.RMID5PCs.Exonic.Nonsynonymous.Pheno4.maf1.skat0.permAll.epacts.NullDistrQQPlotCheck.vs1.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/FakePheno/Pheno4/Exonic.Nonsynonymous/.

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/.

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justpVals /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justpVals /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno2.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno2/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justpVals /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno3.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno3/Exonic.Nonsynonymous/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.QQPlotUsingPermutations.vs1.jpeg /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts.justpVals /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno4.Exonic.Nonsynonymous.output.noNAs.obspValHist.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno4/Exonic.Nonsynonymous/.

#20150330
#Retroactively including code for these 'gene/variant level result summary outputs', following the logic of excel sheet exploration, here

/data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplicecat

echo ".5" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".6" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".7" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".8" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".85" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
echo ".9" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .9) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .9) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`

~~~
[  michaelt@node1  ~]$echo ".5" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .5) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'`
.5 24 184
[  michaelt@node1  ~]$echo ".6" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .6) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'` 
.6 51 400
[  michaelt@node1  ~]$echo ".7" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .7) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'` 
.7 91 654
[  michaelt@node1  ~]$echo ".8" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .8) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'` 
.8 135 1011
[  michaelt@node1  ~]$echo ".85" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'` 
.85 188 1384
[  michaelt@node1  ~]$echo ".9" `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .9) { print $1; } }' | grep -vw NA | wc | awk '{ print $1 }'` `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .9) { print $1; } }') | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq | wc | awk '{ print $1 }'` 
.9 376 2763
~~~

#GetperFastQC
#GetperRegionpostQC
#GetperFullBam
#GetVCF? for all indvs, just white indvs, then subgroups
#for full file, for per gene, for per indv (and then subgroups)

#QC Stuff

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetFastqQCStats.vs2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output

#CHECK_0: Github backup/hard-copy current QC output files for P1 and P2

#Removed empty lines in /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetFastqQCStats.vs2.20150315_234317.output with /0.00000.0000 0.0000 and dd in vi 
#PreMerge IEUL1WW01.RL4 0.00000.0000 0.0000               totalDropped(%):   minDropped(%):   UpperBoundThresh:  maxDropped(%):
#PreMerge IEUL1WW01.RL5 0.00000.0000 0.0000               totalDropped(%):   minDropped(%):   UpperBoundThresh:  maxDropped(%):
#PreMerge IEUL1WW01.RL6 0.00000.0000 0.0000               totalDropped(%):   minDropped(%):   UpperBoundThresh:  maxDropped(%):
#PreMerge IEUL1WW01.RL7 0.00000.0000 0.0000               totalDropped(%):   minDropped(%):   UpperBoundThresh:  maxDropped(%):
#PreMerge IEUL1WW01.RL9 0.00000.0000 0.0000               totalDropped(%):   minDropped(%):   UpperBoundThresh:  maxDropped(%):
#PreMerge IEUL1WW01.RL10 0.00000.0000 0.0000               totalDropped(%):   minDropped(%):   UpperBoundThresh:  maxDropped(%):
#PreMerge IEUL1WW01.RL11 0.00000.0000 0.0000               totalDropped(%):   minDropped(%):   UpperBoundThresh:  maxDropped(%):
#PreMerge IEUL1WW01.RL12 0.00000.0000 0.0000               totalDropped(%):   minDropped(%):   UpperBoundThresh:  maxDropped(%):

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetFastqQCStats.vs2.20150315_234317.output | R -q -e "Data1 <- read.table(file('stdin'), header=F); sum(as.numeric(Data1[,4])); sum(as.numeric(Data1[,6]));"
[1] 172890654704
[1] 288065985
>
>

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetFastqQCStats.vs2.20150315_234317.output | wc
   3776  109504  828231

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperRegionpostQCStats.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output 

mv /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output.GetperRegionpostQCStats.vs3.20150316_\ 00554.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output.GetperRegionpostQCStats.vs3.20150316_00554.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output.GetperRegionpostQCStats.vs3.20150316_00554.output | perl -lane 'my @array1; push(@array1, $F[1]); foreach my $entry (@F) { if ($entry !~ m/[a-zA-Z]/) { push (@array1, $entry); } } print join(",", @array1);' | awk 'BEGIN{ print "ID,TotalReadsPreQC,TotalReadsQ10,TotalReadsQ10Prcnt,TotalReadsF4,TotalReadsF4Prcnt,TotalReadsF256,TotalReadsF256Prcnt,TotalReadsPostQC,TotalReadsDropped,TotalReadsDroppedPrcnt,AverageReadLengthPreQC,AverageReadLengthPostQC,BedFileCoveragePostQC" } { print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output.GetperRegionpostQCStats.vs3.20150316_00554.output.reformatted

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs3.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs3.ForP2.sh

bash /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetperFullBampostQCStats.vs3.ForP2.sh /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected.GetperFullBampostQCStats.vs3.ForP2.20150316_02534.output | perl -lane 'my @array1; push(@array1, $F[1]); foreach my $entry (@F) { if ($entry !~ m/[a-zA-Z]/) { push (@array1, $entry); } } print join(",", @array1);' | awk 'BEGIN{ print "ID,NumberOfOriginalBamFiles,TotalReadsPreQC,TotalReadsPreQCForward,TotalReadsPreQCReverse,TotalReadsPreQCForward_Reverse,PreQCMAPQmean,PreQCMAPQsd,PreQCMean,PreQCSD,PreQCMax,PreQCMin,CoveragePreQC,TotalReadsPostRmdup,TotalReadsPostRmdupDropped,TotalReadsPostRmdupDroppedPercentage,TotalReadsPostQC,TotalReadsPostQCForward,TotalReadsPostQCReverse,TotalReadsPostQCForward_Reverse,PostQCMAPQmean,PostQCMAPQsd,PostQCMean,PostQCSD,PostQCMax,PostQCMin,CoveragePostQC" } { print $0 } ' > /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected.GetperFullBampostQCStats.vs3.ForP2.20150316_02534.output.reformatted 

/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetFastqQCStats.vs2.20150315_234317.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output.GetperRegionpostQCStats.vs3.20150316_00554.output.reformatted /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected.GetperFullBampostQCStats.vs3.ForP2.20150316_02534.output.reformatted 

~~~
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations]$/home/shared/software/R-3.0.1/bin/R --vanilla --slave --args < /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs2.R /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetFastqQCStats.vs2.20150315_234317.output /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CollectAllIndvMIDRLsffFiles.vs3.ForPG1.vs1.WorkingMainFile.output.GetperRegionpostQCStats.vs3.20150316_00554.output.reformatted /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataOrganization.CreatebamMergeList.vs3.ForP2.vs1.20140521_113419.output.multPOOLperIndvCollected.GetperFullBampostQCStats.vs3.ForP2.20150316_02534.output.reformatted
[1] 6.75321
[1] 6.294703
[1] 0.2864592
[1] 0.9647855
null device
	1
[1] 254981
[1] 617.3847
[1] 239.1003
[1] 13.57618
[1] 38316.19
[1] 12.64409
null device
	1
~~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/OutputFiles/Graphics/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs2.2015-03-16_00:48:03.SummaryRegionPlots.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/OutputFiles/Graphics/.
scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/OutputFiles/Graphics/DataProcessing.Pipeline.Utility.EvaluateSummaryFiles.vs1.vs2.2015-03-16_00:48:06.SummaryIndvPlots.jpeg mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/OutputFiles/Graphics/.

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz 
#Edited code in /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs2.py
python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.FullVCFwAnnovar.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs1.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.vcf.AnnovarFormat.genome_summary.csv --file2 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | sed 's/"//g' | perl -lane 'my @vals1 = split(/,/, $F[8]); for (my $i = 0; $i <= $#vals1; $i++) { if ($vals1[$i] eq "NA") { $vals1[$i] = "-9"; } } $F[8] = join(",", @vals1); print join("\t", @F);' | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.GetVCFQCStats.vs2.output

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.GetVCFQCStats.vs2.output

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz | grep exonic | grep -v ncRNA | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.vs2.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.GetVCFQCStats.vs2.output

cp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.PerGene.vs1.py /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.PerGene.vs2.py

#python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.PerGene.vs2.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.perGene.vs2.output
zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz | grep -v intergenic | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.PerGene.vs2.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.perGene.vs2.output

#Coverage/DP is column 18

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.vcf.gz | grep exonic | grep -v ncRNA | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.PerGene.vs2.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.AllGenesPerVariant.vs2.Exonic.perGene.vs2.output

python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs3.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.vcf.gz | grep -E '^#|exonic' | grep -v ncRNA | python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.GetVCFQCStats.perIndv.vs3.py --file1 - > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroNeg 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroHighlyExposedNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroHighlyExposedNeg 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroPos.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroPos 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justRapidVRapid.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justRapidVRapid 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSlowVSlow.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSlowVSlow 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVeryRapid.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justVeryRapid 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVerySlow.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justVerySlow 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroNeg 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroHighlyExposedNeg.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroHighlyExposedNeg 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSeroPos.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroPos 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justRapidVRapid.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justRapidVRapid 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justSlowVSlow.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSlowVSlow 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVeryRapid.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justVeryRapid 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output | grep -f /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/PLINKfiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.pheno.justVerySlow.FIDs > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justVerySlow 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 12.8567603748327 MeanDoubletons: 21.6559571619813 MeanCommon: 6842.5890227577 MeanRare: 593.401606425703 MeanHeterozy: 4661.906291834 MeanCoverage: 12.5231536370384 sdCoverage: 4.89279817826893 Exonic: 653.61311914324 Intronic: 5008.33065595716 Nonsynonymous: 257.259705488621 Synonymous: 390.409638554217 PP2_D: 28.1954484605087 PP2_P: 18.7028112449799 Sift_D: 41.3373493975904 LRT_D: 30.6398929049531"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 12.4660766961652 MeanDoubletons: 21.7256637168142 MeanCommon: 6828.32448377581 MeanRare: 596.554572271386 MeanHeterozy: 4648.38348082596 MeanCoverage: 12.3044067539243 sdCoverage: 4.93589584565892 Exonic: 654.171091445428 Intronic: 5000.91150442478 Nonsynonymous: 257.262536873156 Synonymous: 390.896755162242 PP2_D: 28.0766961651917 PP2_P: 18.6873156342183 Sift_D: 41.7079646017699 LRT_D: 30.5486725663717"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroHighlyExposedNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 12.796178343949 MeanDoubletons: 21.0382165605096 MeanCommon: 6830.45859872611 MeanRare: 586.955414012739 MeanHeterozy: 4643.3949044586 MeanCoverage: 12.8067167758079 sdCoverage: 4.95028888995383 Exonic: 653.949044585987 Intronic: 4996.75159235669 Nonsynonymous: 257.757961783439 Synonymous: 390.235668789809 PP2_D: 27.6624203821656 PP2_P: 18.6751592356688 Sift_D: 41.5923566878981 LRT_D: 30.4458598726115"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 13.1813725490196 MeanDoubletons: 21.5980392156863 MeanCommon: 6854.44117647059 MeanRare: 590.781862745098 MeanHeterozy: 4673.14215686274 MeanCoverage: 12.7049065619788 sdCoverage: 4.85525920103541 Exonic: 653.149509803922 Intronic: 5014.49509803922 Nonsynonymous: 257.257352941176 Synonymous: 390.004901960784 PP2_D: 28.2941176470588 PP2_P: 18.7156862745098 Sift_D: 41.0294117647059 LRT_D: 30.7156862745098"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justRapidVRapid | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 12.7777777777778 MeanDoubletons: 20.3333333333333 MeanCommon: 6868.83333333333 MeanRare: 579.740740740741 MeanHeterozy: 4691.14814814815 MeanCoverage: 13.1737065707157 sdCoverage: 3.78571833702897 Exonic: 652.111111111111 Intronic: 5019.25925925926 Nonsynonymous: 255.351851851852 Synonymous: 390.666666666667 PP2_D: 27.8703703703704 PP2_P: 18.3333333333333 Sift_D: 39.9814814814815 LRT_D: 29.5740740740741"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justSlowVSlow | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 13.0246913580247 MeanDoubletons: 22.6049382716049 MeanCommon: 6866.0987654321 MeanRare: 598.456790123457 MeanHeterozy: 4689.83950617284 MeanCoverage: 12.8425294439885 sdCoverage: 6.90751896295189 Exonic: 656.444444444444 Intronic: 5027.16049382716 Nonsynonymous: 259.679012345679 Synonymous: 391.074074074074 PP2_D: 28.4938271604938 PP2_P: 18.7037037037037 Sift_D: 41.8765432098765 LRT_D: 31.1604938271605"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justVeryRapid | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 14.3809523809524 MeanDoubletons: 19.6666666666667 MeanCommon: 6923.42857142857 MeanRare: 573.714285714286 MeanHeterozy: 4751.57142857143 MeanCoverage: 13.398101176621 sdCoverage: 3.24540374835087 Exonic: 651.333333333333 Intronic: 5064.47619047619 Nonsynonymous: 257.333333333333 Synonymous: 387.952380952381 PP2_D: 29 PP2_P: 18.8095238095238 Sift_D: 40.6190476190476 LRT_D: 31.4285714285714"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.perIndv.vs3.output.justVerySlow | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 13.1578947368421 MeanDoubletons: 21 MeanCommon: 6848.76315789474 MeanRare: 595.631578947368 MeanHeterozy: 4674.86842105263 MeanCoverage: 14.4278295823871 sdCoverage: 9.32553269310467 Exonic: 653.5 Intronic: 5009.71052631579 Nonsynonymous: 259.184210526316 Synonymous: 388.921052631579 PP2_D: 28.1842105263158 PP2_P: 18.4473684210526 Sift_D: 41.6052631578947 LRT_D: 30.6842105263158"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 1.95850066934404 MeanDoubletons: 3.33868808567604 MeanCommon: 591.321285140562 MeanRare: 62.3092369477912 MeanHeterozy: 403.706827309237 MeanCoverage: 14.8163023694426 sdCoverage: 5.89911671487659 Exonic: 653.61311914324 Intronic: 0 Nonsynonymous: 257.259705488621 Synonymous: 390.409638554217 PP2_D: 28.1954484605087 PP2_P: 18.7028112449799 Sift_D: 41.0883534136546 LRT_D: 30.6398929049531"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 1.87610619469027 MeanDoubletons: 3.25663716814159 MeanCommon: 591.082595870207 MeanRare: 63.1061946902655 MeanHeterozy: 404.362831858407 MeanCoverage: 14.6616951893061 sdCoverage: 5.96973879035129 Exonic: 654.171091445428 Intronic: 0 Nonsynonymous: 257.262536873156 Synonymous: 390.896755162242 PP2_D: 28.0766961651917 PP2_P: 18.6873156342183 Sift_D: 41.4660766961652 LRT_D: 30.5486725663717"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroHighlyExposedNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 1.82165605095541 MeanDoubletons: 3.18471337579618 MeanCommon: 591.808917197452 MeanRare: 62.1592356687898 MeanHeterozy: 404.146496815287 MeanCoverage: 15.3278965971024 sdCoverage: 6.0433291355643 Exonic: 653.949044585987 Intronic: 0 Nonsynonymous: 257.757961783439 Synonymous: 390.235668789809 PP2_D: 27.6624203821656 PP2_P: 18.6751592356688 Sift_D: 41.3630573248408 LRT_D: 30.4458598726115"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.02696078431373 MeanDoubletons: 3.40686274509804 MeanCommon: 591.519607843137 MeanRare: 61.6470588235294 MeanHeterozy: 403.161764705882 MeanCoverage: 14.944762747056 sdCoverage: 5.84401646067709 Exonic: 653.149509803922 Intronic: 0 Nonsynonymous: 257.257352941176 Synonymous: 390.004901960784 PP2_D: 28.2941176470588 PP2_P: 18.7156862745098 Sift_D: 40.7745098039216 LRT_D: 30.7156862745098"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justRapidVRapid | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 1.74074074074074 MeanDoubletons: 3.40740740740741 MeanCommon: 592.814814814815 MeanRare: 59.3148148148148 MeanHeterozy: 399.259259259259 MeanCoverage: 15.4838212840091 sdCoverage: 4.5829707196305 Exonic: 652.111111111111 Intronic: 0 Nonsynonymous: 255.351851851852 Synonymous: 390.666666666667 PP2_D: 27.8703703703704 PP2_P: 18.3333333333333 Sift_D: 39.6851851851852 LRT_D: 29.5740740740741"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSlowVSlow | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.08641975308642 MeanDoubletons: 3.5679012345679 MeanCommon: 593.246913580247 MeanRare: 63.2222222222222 MeanHeterozy: 406.345679012346 MeanCoverage: 15.0660673762291 sdCoverage: 8.2663373355639 Exonic: 656.444444444444 Intronic: 0 Nonsynonymous: 259.679012345679 Synonymous: 391.074074074074 PP2_D: 28.4938271604938 PP2_P: 18.7037037037037 Sift_D: 41.5185185185185 LRT_D: 31.1604938271605"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justVeryRapid | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.0952380952381 MeanDoubletons: 3.28571428571429 MeanCommon: 592.142857142857 MeanRare: 59.2380952380952 MeanHeterozy: 397.619047619048 MeanCoverage: 15.6974873657967 sdCoverage: 3.9523316947915 Exonic: 651.333333333333 Intronic: 0 Nonsynonymous: 257.333333333333 Synonymous: 387.952380952381 PP2_D: 29 PP2_P: 18.8095238095238 Sift_D: 40.1904761904762 LRT_D: 31.4285714285714"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justVerySlow | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 1.94736842105263 MeanDoubletons: 3.28947368421053 MeanCommon: 591.631578947368 MeanRare: 61.9210526315789 MeanHeterozy: 404.631578947368 MeanCoverage: 16.9681495161637 sdCoverage: 11.1593773020957 Exonic: 653.5 Intronic: 0 Nonsynonymous: 259.184210526316 Synonymous: 388.921052631579 PP2_D: 28.1842105263158 PP2_P: 18.4473684210526 Sift_D: 41.2894736842105 LRT_D: 30.6842105263158"
> 
> 

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 1.95850066934404 MeanDoubletons: 3.33868808567604 MeanCommon: 591.321285140562 MeanRare: 62.3092369477912 MeanHeterozy: 403.706827309237 MeanCoverage: 14.8163023694426 sdCoverage: 5.89911671487659 Exonic: 653.61311914324 Intronic: 0 Nonsynonymous: 257.259705488621 Synonymous: 390.409638554217 PP2_D: 28.1954484605087 PP2_P: 18.7028112449799 Sift_D: 41.0883534136546 LRT_D: 30.6398929049531"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 1.87610619469027 MeanDoubletons: 3.25663716814159 MeanCommon: 591.082595870207 MeanRare: 63.1061946902655 MeanHeterozy: 404.362831858407 MeanCoverage: 14.6616951893061 sdCoverage: 5.96973879035129 Exonic: 654.171091445428 Intronic: 0 Nonsynonymous: 257.262536873156 Synonymous: 390.896755162242 PP2_D: 28.0766961651917 PP2_P: 18.6873156342183 Sift_D: 41.4660766961652 LRT_D: 30.5486725663717"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroHighlyExposedNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 1.82165605095541 MeanDoubletons: 3.18471337579618 MeanCommon: 591.808917197452 MeanRare: 62.1592356687898 MeanHeterozy: 404.146496815287 MeanCoverage: 15.3278965971024 sdCoverage: 6.0433291355643 Exonic: 653.949044585987 Intronic: 0 Nonsynonymous: 257.757961783439 Synonymous: 390.235668789809 PP2_D: 27.6624203821656 PP2_P: 18.6751592356688 Sift_D: 41.3630573248408 LRT_D: 30.4458598726115"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.02696078431373 MeanDoubletons: 3.40686274509804 MeanCommon: 591.519607843137 MeanRare: 61.6470588235294 MeanHeterozy: 403.161764705882 MeanCoverage: 14.944762747056 sdCoverage: 5.84401646067709 Exonic: 653.149509803922 Intronic: 0 Nonsynonymous: 257.257352941176 Synonymous: 390.004901960784 PP2_D: 28.2941176470588 PP2_P: 18.7156862745098 Sift_D: 40.7745098039216 LRT_D: 30.7156862745098"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justRapidVRapid | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 1.74074074074074 MeanDoubletons: 3.40740740740741 MeanCommon: 592.814814814815 MeanRare: 59.3148148148148 MeanHeterozy: 399.259259259259 MeanCoverage: 15.4838212840091 sdCoverage: 4.5829707196305 Exonic: 652.111111111111 Intronic: 0 Nonsynonymous: 255.351851851852 Synonymous: 390.666666666667 PP2_D: 27.8703703703704 PP2_P: 18.3333333333333 Sift_D: 39.6851851851852 LRT_D: 29.5740740740741"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSlowVSlow | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.08641975308642 MeanDoubletons: 3.5679012345679 MeanCommon: 593.246913580247 MeanRare: 63.2222222222222 MeanHeterozy: 406.345679012346 MeanCoverage: 15.0660673762291 sdCoverage: 8.2663373355639 Exonic: 656.444444444444 Intronic: 0 Nonsynonymous: 259.679012345679 Synonymous: 391.074074074074 PP2_D: 28.4938271604938 PP2_P: 18.7037037037037 Sift_D: 41.5185185185185 LRT_D: 31.1604938271605"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justVeryRapid | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.0952380952381 MeanDoubletons: 3.28571428571429 MeanCommon: 592.142857142857 MeanRare: 59.2380952380952 MeanHeterozy: 397.619047619048 MeanCoverage: 15.6974873657967 sdCoverage: 3.9523316947915 Exonic: 651.333333333333 Intronic: 0 Nonsynonymous: 257.333333333333 Synonymous: 387.952380952381 PP2_D: 29 PP2_P: 18.8095238095238 Sift_D: 40.1904761904762 LRT_D: 31.4285714285714"
> 
> 
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justVerySlow | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 1.94736842105263 MeanDoubletons: 3.28947368421053 MeanCommon: 591.631578947368 MeanRare: 61.9210526315789 MeanHeterozy: 404.631578947368 MeanCoverage: 16.9681495161637 sdCoverage: 11.1593773020957 Exonic: 653.5 Intronic: 0 Nonsynonymous: 259.184210526316 Synonymous: 388.921052631579 PP2_D: 28.1842105263158 PP2_P: 18.4473684210526 Sift_D: 41.2894736842105 LRT_D: 30.6842105263158"
> 
> 


#Comparing P2 to P1
~~~
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.GetVCFQCStats.vs2.output
NumVar: 9844 Exonic: 9840 Intronic: 0 Intergenic: 0 UTR5: 0 UTR3: 0                                                                                                                                                                                                                                                        
Synonymous: 4411 Nonsynonymous: 5233 Stopgain: 116 Stoploss: 10
PP2_D: 1572 PP2_P: 842 PP2_B: 2059 PP2_-9: 467 Sift_D: 1991 Sift_T: 2691 Sift_-9: 258 LRT_D: 2050 LRT_N: 2343 LRT_U: 234 LRT_-9: 313                                                                                                                                                                                       
Common: 1144 Rare: 8700 Singletons: 1463 Doubletons: 1547
MeanQuality: 17860.6449553 QualityStd: 65869.6226881 MeanDepth: 14.9857453366 DepthStd: 4.77660674994 TiTvRatio: 4.24453915823 Transitions: 7967 Transversions: 1877                                                                                                                                                       
MeanHapScore: 1.79340668427 HapScoreStd: 1.06580996397 HapScore0Quart: 0.2499 HapScore25Quart: 1.071825 HapScore50Quart: 1.5222 HapScore75Quart: 2.202675 HapScore100Quart: 8.7767
[  michaelt@node2  /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wAnnovarForQC.RanGenePerVariant.Exonic.GetVCFQCStats.vs1.output
NumVar: 7617 Exonic: 7614 Intronic: 0 Intergenic: 0 UTR5: 0 UTR3: 0
Synonymous: 3339 Nonsynonymous: 4083 Stopgain: 54 Stoploss: 13
PP2_D: 1129 PP2_P: 660 PP2_B: 1546 PP2_-9: 379 Sift_D: 1477 Sift_T: 1927 Sift_-9: 310 LRT_D: 1852 LRT_N: 1624 LRT_U: 96 LRT_-9: 142
Common: 588 Rare: 7029 Singletons: 2592 Doubletons: 1387
MeanQuality: 16993.8342957 QualityStd: 79340.471072 MeanDepth: 23.3232939376 DepthStd: 8.87203927559 TiTvRatio: 3.39780600462 Transitions: 5885 Transversions: 1732
MeanHapScore: 2.75564990154 HapScoreStd: 1.67229214079 HapScore0Quart: 0.181 HapScore25Quart: 1.5964 HapScore50Quart: 2.3674 HapScore75Quart: 3.4397 HapScore100Quart: 14.6402

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 1.87610619469027 MeanDoubletons: 3.25663716814159 MeanCommon: 591.082595870207 MeanRare: 63.1061946902655 MeanHeterozy: 404.362831858407 MeanCoverage: 14.6616951893061 sdCoverage: 5.96973879035129 Exonic: 654.171091445428 Intronic: 0 Nonsynonymous: 257.262536873156 Synonymous: 390.896755162242 PP2_D: 28.0766961651917 PP2_P: 18.6873156342183 Sift_D: 41.4660766961652 LRT_D: 30.5486725663717"
>
>
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output.justSeroNeg | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.43909348441926 MeanDoubletons: 2.86685552407932 MeanCommon: 299.28045325779 MeanRare: 35.2549575070822 MeanHeterozy: 219.773371104816 MeanCoverage: 21.9810675566897 sdCoverage: 8.58968008315038 Exonic: 334.082152974504 Intronic: 0 Nonsynonymous: 122.872521246459 Synonymous: 208.801699716714 PP2_D: 9.90934844192635 PP2_P: 12.6345609065156 Sift_D: 22.9206798866856 LRT_D: 21.6345609065156"
>
>

cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/Analyses/QCResults/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.wAnnovarForQC.RanGenePerVariant.vs2.Exonic.perIndv.vs3.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.02696078431373 MeanDoubletons: 3.40686274509804 MeanCommon: 591.519607843137 MeanRare: 61.6470588235294 MeanHeterozy: 403.161764705882 MeanCoverage: 14.944762747056 sdCoverage: 5.84401646067709 Exonic: 653.149509803922 Intronic: 0 Nonsynonymous: 257.257352941176 Synonymous: 390.004901960784 PP2_D: 28.2941176470588 PP2_P: 18.7156862745098 Sift_D: 40.7745098039216 LRT_D: 30.7156862745098"
>
>
cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/QCResults/AllPools.Vs2.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3Data.justWhite.QCed.DropIBD.wAnnovarForQC.RanGene.vs2.Exonic.perIndv.vs3.output.justSeroPos | R -q -e "Data1 <- read.table(file('stdin'), header=F); \
print(paste(\"MeanSingletons:\", mean(Data1[,6]), \"MeanDoubletons:\", mean(Data1[,8]), \"MeanCommon:\", mean(Data1[,10]), \"MeanRare:\", mean(Data1[,12]), \"MeanHeterozy:\", mean(Data1[,14]), \"MeanCoverage:\", mean(Data1[,16]), \"sdCoverage:\", sd(Data1[,16]), \"Exonic:\", mean(Data1[,22]), \"Intronic:\", mean(Data1[,24]), \"Nonsynonymous:\", mean(Data1[,32]), \"Synonymous:\", mean(Data1[,34]), \"PP2_D:\", mean(Data1[,40]), \"PP2_P:\", mean(Data1[,42]), \"Sift_D:\", mean(Data1[,48]), \"LRT_D:\", mean(Data1[,54]), sep=\" \"))"
> Data1 <- read.table(file('stdin'), header=F); print(paste("MeanSingletons:", mean(Data1[,6]), "MeanDoubletons:", mean(Data1[,8]), "MeanCommon:", mean(Data1[,10]), "MeanRare:", mean(Data1[,12]), "MeanHeterozy:", mean(Data1[,14]), "MeanCoverage:", mean(Data1[,16]), "sdCoverage:", sd(Data1[,16]), "Exonic:", mean(Data1[,22]), "Intronic:", mean(Data1[,24]), "Nonsynonymous:", mean(Data1[,32]), "Synonymous:", mean(Data1[,34]), "PP2_D:", mean(Data1[,40]), "PP2_P:", mean(Data1[,42]), "Sift_D:", mean(Data1[,48]), "LRT_D:", mean(Data1[,54]), sep=" "))
[1] "MeanSingletons: 2.53537735849057 MeanDoubletons: 2.54009433962264 MeanCommon: 298.695754716981 MeanRare: 34.4528301886792 MeanHeterozy: 219.007075471698 MeanCoverage: 22.9559819110576 sdCoverage: 9.25012765969131 Exonic: 332.719339622641 Intronic: 0 Nonsynonymous: 122.200471698113 Synonymous: 207.785377358491 PP2_D: 9.78301886792453 PP2_P: 12.6768867924528 Sift_D: 22.2452830188679 LRT_D: 21.8915094339623"
>
>
~~~

#20150325
#Adding new annotated .vcf format option now using vcf-annotate (http://vcftools.sourceforge.net/perl_module.html#vcf-annotate)
#Currently mainly being used to help pass on information to Sudhir for evaluation cost/design-strategy for variant follow-up genotyping
#CHECK_0: Getting vcf-annotate to work properly for the AC_... columns if this setup/format is desired to be continually used

cat <(echo "#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        REF     ALT     AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW") \
<(join <( python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | sort -k 1,1) <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^\# | perl -lane 'print join("_", @F[0..1]), "\t", $F[3], "\t", $F[4];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
perl -lane 'splice(@F, 0, 1, (((split(/_/, $F[0]))[0]), ((split(/_/, $F[0]))[1]), ((split(/_/, $F[0]))[1]))); print join("\t", @F);' |
perl -ane 'print join("\t", @F[0..8]); my $alt = $F[8]; my $totAC = join("", ((split(/:/, $F[9]))[1])) + join("", ((split(/:/, $F[10]))[1])); print "\t", $totAC; for (my $i=9; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", $F[$i]; } } print "\n";' | sort -g -k 1,1 -k 2,2 ) |
bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs1.wACs.txt.gz

cat <(echo "#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        REF     ALT     AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW") <(join <( python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | sort -k 1,1) <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^\# | perl -lane 'print join("_", @F[0..1]), "\t", $F[3], "\t", $F[4];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | perl -lane 'splice(@F, 0, 1, (((split(/_/, $F[0]))[0]), ((split(/_/, $F[0]))[1]), ((split(/_/, $F[0]))[1]))); print join("\t", @F);' | perl -ane 'print join("\t", @F[0..8]); my $alt = $F[8]; my $totAC = join("", ((split(/:/, $F[9]))[1])) + join("", ((split(/:/, $F[10]))[1])); print "\t", $totAC; for (my $i=9; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", $F[$i]; } } print "\n";' | sort -g -k 1,1 -k 2,2 ) | bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs1.wACs.txt.gz

tabix -s 1 -b 2 -e 3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs1.wACs.txt.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | vcf-annotate -a /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs1.wACs.txt.gz -d /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.PrepFor_vcfannotate.vs1.desc.txt -c CHROM,FROM,TO,INFO/LOC,INFO/GENE,INFO/VARTYPE,INFO/AACHANGE,INFO/AC_ALL,INFO/AC_TOTAL,INFO/AC_SERONEG,INFO/AC_HESERONEG,INFO/AC_SEROPOS,INFO/AC_RAPID,INFO/AC_SLOW | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.annotate.vs1.wACs.vcf.gz

mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir
mkdir /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForKevinO

cat <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.annotate.vs1.wACs.vcf.gz | grep ^# | perl -lane 'if ($F[0] =~ /#CHROM/) { print join("\t", @F[0..7]); } else { print join(" ", @F); }' ) \
<(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.annotate.vs1.wACs.vcf.gz | perl -lane 'print join("\t", @F[0..7]);' | grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1; } }' | grep -vw NA ) | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq )) | \
gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.P2.Vs2.qVal_lte85.annotated.vcf.gz 

~~~
[  michaelt@node1  ~]$cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1; } }'i | grep -vw NA | wc                                                                                                                                                                                                                                                         
    188     188    1154
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.annotate.vs1.wACs.vcf.gz | perl -lane 'print join("\t", @F[0..7]);' | grep -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.bim.wGeneIDs.GroupFile.Exonic.Nonsynonymous.wSplice | grep -w -f <(cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | awk '{ if ($4 <= .85) { print $1; } }' | grep -vw NA ) | perl -lane 'for (my $i = 1; $i <= $#F; $i++) { my @vals1 = split(/_/, $F[$i]); my @vals2 = split(/:/, $vals1[0]); print $vals2[0], "\t", $vals2[1]; }' | sort | uniq ) | wc
   1384   11072  668030                                                                                                                                                                                                                                                                                                    
[  michaelt@node1  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.P2.Vs2.qVal_lte85.annotated.vcf.gz | awk '{ print $1, "\t", $2}' | grep -v ^# | wc
   1384    2768   18998
~~~

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.P2.Vs2.qVal_lte85.annotated.vcf.gz mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/.

cat <(echo "#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        PVAL	QVAL	REF     ALT	AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW") \
<(join <( python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | sort -k 1,1) <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^\# | perl -lane 'print join("_", @F[0..1]), "\t", $F[3], "\t", $F[4];' | sort -k 1,1) | \
perl -lane '$F[1] = join("|", split(/[\;,\,]/, $F[1])); $F[2] = join("|", split(/[\;,\,]/, $F[2])); print join("\t", @F);' | \
perl -lane 'my @geneEntries = split(/\|/, $F[2]); my @pVals; my @qVals; foreach my $gene (@geneEntries) { $gene = ((split(/\(/, $gene))[0]); my $geneEntry = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w "$gene"`; if ($geneEntry) { my @geneVals1 = split(/\s+/, $geneEntry); push(@pVals, $geneVals1[1]); push(@qVals, $geneVals1[3]); } else { push(@pVals, "NA"); push(@qVals, "NA"); } } splice(@F, 5, 0, (join("|", @pVals), join("|", @qVals))); print join("\t", @F);' | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | \
perl -lane 'splice(@F, 0, 1, (((split(/_/, $F[0]))[0]), ((split(/_/, $F[0]))[1]), ((split(/_/, $F[0]))[1]))); print join("\t", @F);' | \
perl -ane 'print join("\t", @F[0..10]); my $alt = $F[10]; my $totAC = join("", ((split(/:/, $F[11]))[1])) + join("", ((split(/:/, $F[12]))[1])); print "\t", $totAC; for (my $i=11; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", ((split(/\:/, $F[$i]))[1]); } } print "\n";' | sort -g -k 1,1 -k 2,2 ) | \
bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz

cat <(echo "#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        PVAL	QVAL	REF     ALT     AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW") <(join <( python /data/userdata/pg/michaelt/Data/ALL_MAPPING/DataProcessing.Pipeline.Utility.CombineVCFwFreqsAnnovarInfo.vs6.PrepFor_vcfannotate.vs1.py --file1 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.txt.gz | sort -k 1,1) <( zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | grep -v ^\# | perl -lane 'print join("_", @F[0..1]), "\t", $F[3], "\t", $F[4];' | sort -k 1,1) | perl -lane '$F[1] = join("|", split(/[\;,\,]/, $F[1])); $F[2] = join("|", split(/[\;,\,]/, $F[2])); print join("\t", @F);' | perl -lane 'my @geneEntries = split(/\|/, $F[2]); my @pVals; my @qVals; foreach my $gene (@geneEntries) { $gene = ((split(/\(/, $gene))[0]); my $geneEntry = `cat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/EPACTSFiles/Permutations/TruePheno/Pheno1/Exonic.Nonsynonymous/DataProcessing.Pipeline.Utility.CollectPValsPerGeneAcrossMultRuns.vs2.TruePheno.Pheno1.Exonic.Nonsynonymous.output.noNAs.Routput.wVarCounts | grep -w "$gene"`; if ($geneEntry) { my @geneVals1 = split(/\s+/, $geneEntry); push(@pVals, $geneVals1[1]); push(@qVals, $geneVals1[3]); } else { push(@pVals, "NA"); push(@qVals, "NA"); } } splice(@F, 5, 0, (join("|", @pVals), join("|", @qVals))); print join("\t", @F);' | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroHighlyExposedNeg.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSeroPos.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justRapidVRapid.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | join - <(zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/FreqInfo/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.justSlowVSlow.frq.count.gz | perl -lane 'print join("_", @F[0..1]), "\t", $F[4], "\t", $F[5];' | sort -k 1,1) | perl -lane 'splice(@F, 0, 1, (((split(/_/, $F[0]))[0]), ((split(/_/, $F[0]))[1]), ((split(/_/, $F[0]))[1]))); print join("\t", @F);' | perl -ane 'print join("\t", @F[0..10]); my $alt = $F[10]; my $totAC = join("", ((split(/:/, $F[11]))[1])) + join("", ((split(/:/, $F[12]))[1])); print "\t", $totAC; for (my $i=11; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", ((split(/\:/, $F[$i]))[1]); } } print "\n";' | sort -g -k 1,1 -k 2,2 ) | /home/shared/software/tabix-0.2.5/bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz
perl -ane 'print join("\t", @F[0..8]); my $alt = $F[8]; my $totAC = join("", ((split(/:/, $F[9]))[1])) + join("", ((split(/:/, $F[10]))[1])); print "\t", $totAC; for (my $i=9; $i <= $#F; $i++) { if ($F[$i] =~ m/$alt/) { print "\t", $F[$i]; } } print "\n";' | sort -g -k 1,1 -k 2,2 ) | /home/shared/software/tabix-0.2.5/bgzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz

~~~
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | head -n 10
#CHROM      FROM    TO      LOC     GENE    VARTYPE AACHANGE        PVAL    QVAL    REF     ALT     AC_ALL  AC_TOTAL        AC_SERONEG      AC_HESERONEG    AC_SEROPOS      AC_RAPID        AC_SLOW
X       2821584 2821584 downstream      ARSD    .       .       0.6429427000    0.90298 C       G       1466    2       2       0       0       0       0
X       2821782 2821782 downstream      ARSD    .       .       0.6429427000    0.90298 G       A       1478    1       0       0       1       0       0
X       2821848 2821848 downstream      ARSD    .       .       0.6429427000    0.90298 G       A       1484    3       3       3       0       0       0
X       2822103 2822103 UTR3    ARSD    .       .       0.6429427000    0.90298 C       G       1486    2       0       0       2       0       2
X       2823133 2823133 UTR3    ARSD    .       .       0.6429427000    0.90298 G       A       1486    2       1       1       1       0       0
X       2824218 2824218 UTR3    ARSD    .       .       0.6429427000    0.90298 T       C       1486    15      7       3       8       2       2
X       2825026 2825026 UTR3    ARSD    .       .       0.6429427000    0.90298 C       T       1474    5       2       0       3       0       0
X       2825083 2825083 UTR3    ARSD    .       .       0.6429427000    0.90298 G       A       1476    12      6       4       6       2       2
X       2825149 2825149 UTR3    ARSD    .       .       0.6429427000    0.90298 C       T       1474    4       2       0       2       1       0
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | perl -lane 'print $#F;' | sort | uniq -c
  85201 17
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | wc                                      
  85201 1533618 7483870
~~~

tabix -s 1 -b 2 -e 3 /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz

zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcf.gz | vcf-annotate -a /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz -d /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/MM5/PostMerge/mapping_pool_merged/GATK/AllPools.Vs3.QCed.preGATK.QCed.samplesMerged.rmdup.BQSR.calmd.AllPoolsMerged.ChrAll.GATK.RRs.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.PrepFor_vcfannotate.vs2.desc.txt -c CHROM,FROM,TO,INFO/LOC,INFO/GENE,INFO/VARTYPE,INFO/AACHANGE,INFO/PVAL,INFO/QVAL,REF,ALT,INFO/AC_ALL,INFO/AC_TOTAL,INFO/AC_SERONEG,INFO/AC_HESERONEG,INFO/AC_SEROPOS,INFO/AC_RAPID,INFO/AC_SLOW | gzip > /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz

~~~
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz | perl -lane 'print $#F;' | sort | uniq -c
87 0
3 1
3 10
1 109
7 11
5 12
1 13
4 14
1 16
2 2
1 20
2 24
2 3
1 38
3 4
1 41
4 5
3 6
1 69
85201 755
4 8
3 9
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz | wc
85201 1533618 7483870
[  michaelt@node2  ~]$zcat /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz | wc                         
85340 64412794 1498116300
~~~

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.vcfannotate.vs3.wACs.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.P2.Vs2.annotated.vcf.gz

ln -s /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/AllPools.P2.Vs2.AllPoolsMerged.ChrAll.GATK.RR.UG.VQSR.SNP.PASSts99_9.wAA.Bi.DropOffTarg_1kb.geno95.hwe1e4.wHM3.justWhite.QCed.DropIBD.indv80.FreqInfo.AnnovarAnnotation.vs6.Prepvcfannotate.vs3.wACs.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz

scp -p /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.AllPools.P2.Vs2.annotated.vcf.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.VariantLevel.txt.gz /data/userdata/pg/michaelt/Data/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/P2.skatO.nonsyn.qVals.ACs.GeneLevel.ForSudhir.txt mturchin20@wolfy.uchicago.edu:/Users/mturchin20/LabStuff/StephensLab/HIVStuff/PG/ALL_MAPPING/Pools/P2/PostMerge/mapping_pool_merged/Vs1/GATK/ForPeople/ForSudhir/.




